

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/365/2469965","@type":"ScholarlyArticle","name":" A gene-based map of the Nod factor-independent  Aeschynomene evenia  sheds new light on the evolution of nodulation and legume genomes ","datePublished":"2016-06-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw020","keywords":["<italic>Aeschynomene evenia</italic>","symbiosis","legume","genetic map","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/365/2469965","author":[{"name":"Chaintreuil, Clémence","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Rivallan, Ronan","affiliation":"CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France","@type":"Person"},{"name":"Bertioli, David J.","affiliation":"University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil","@type":"Person"},{"name":"Klopp, Christophe","affiliation":"INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France","@type":"Person"},{"name":"Gouzy, Jérôme","affiliation":"INRA, UMR441 LIPM, INRA Auzeville, F-31326 Castanet-Tolosan, France","@type":"Person"},{"name":"Courtois, Brigitte","affiliation":"CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France","@type":"Person"},{"name":"Leleux, Philippe","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France    INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France","@type":"Person"},{"name":"Martin, Guillaume","affiliation":"CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France","@type":"Person"},{"name":"Rami, Jean-François","affiliation":"CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France","@type":"Person"},{"name":"Gully, Djamel","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Parrinello, Hugues","affiliation":"MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France","@type":"Person"},{"name":"Séverac, Dany","affiliation":"MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France","@type":"Person"},{"name":"Patrel, Delphine","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France    IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France","@type":"Person"},{"name":"Fardoux, Joël","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Ribière, William","affiliation":"IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France","@type":"Person"},{"name":"Boursot, Marc","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Cartieaux, Fabienne","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Czernic, Pierre","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Ratet, Pascal","affiliation":"Institute of Plant Sciences Paris Saclay IPS2, CNRS, INRA, Université Paris-Sud, Université Evry, Université Paris-Saclay, 91405 Orsay, France    Institute of Plant Sciences Paris-Saclay IPS2, Paris Diderot, Sorbonne Paris-Cité, 91405 Orsay, France","@type":"Person"},{"name":"Mournet, Pierre","affiliation":"CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France","@type":"Person"},{"name":"Giraud, Eric","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"},{"name":"Arrighi, Jean-François","affiliation":"IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France","@type":"Person"}],"description":"Abstract.  Aeschynomene evenia has emerged as a new model legume for the deciphering of the molecular mechanisms of an alternative symbiotic process that is ind","pageStart":"365","pageEnd":"376","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" A gene-based map of the Nod factor-independent  Aeschynomene evenia  sheds new light on the evolution of nodulation and legume genomes ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" A gene-based map of the Nod factor-independent  Aeschynomene evenia  sheds new light on the evolution of nodulation and legume genomes " />
    <meta property="og:description" content="Abstract.  Aeschynomene evenia has emerged as a new model legume for the deciphering of the molecular mechanisms of an alternative symbiotic process that is ind" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/365/2469965" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-06-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chaintreuil, Clémence" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Rivallan, Ronan" /><meta name="citation_author_institution" content=" CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France " /><meta name="citation_author" content="Bertioli, David J." /><meta name="citation_author_institution" content=" University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil " /><meta name="citation_author" content="Klopp, Christophe" /><meta name="citation_author_institution" content=" INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France " /><meta name="citation_author" content="Gouzy, Jérôme" /><meta name="citation_author_institution" content=" INRA, UMR441 LIPM, INRA Auzeville, F-31326 Castanet-Tolosan, France " /><meta name="citation_author" content="Courtois, Brigitte" /><meta name="citation_author_institution" content=" CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France " /><meta name="citation_author" content="Leleux, Philippe" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author_institution" content=" INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France " /><meta name="citation_author" content="Martin, Guillaume" /><meta name="citation_author_institution" content=" CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France " /><meta name="citation_author" content="Rami, Jean-François" /><meta name="citation_author_institution" content=" CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France " /><meta name="citation_author" content="Gully, Djamel" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Parrinello, Hugues" /><meta name="citation_author_institution" content=" MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France " /><meta name="citation_author" content="Séverac, Dany" /><meta name="citation_author_institution" content=" MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France " /><meta name="citation_author" content="Patrel, Delphine" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author_institution" content=" IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France " /><meta name="citation_author" content="Fardoux, Joël" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Ribière, William" /><meta name="citation_author_institution" content=" IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France " /><meta name="citation_author" content="Boursot, Marc" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Cartieaux, Fabienne" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Czernic, Pierre" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Ratet, Pascal" /><meta name="citation_author_institution" content=" Institute of Plant Sciences Paris Saclay IPS2, CNRS, INRA, Université Paris-Sud, Université Evry, Université Paris-Saclay, 91405 Orsay, France " /><meta name="citation_author_institution" content=" Institute of Plant Sciences Paris-Saclay IPS2, Paris Diderot, Sorbonne Paris-Cité, 91405 Orsay, France " /><meta name="citation_author" content="Mournet, Pierre" /><meta name="citation_author_institution" content=" CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France " /><meta name="citation_author" content="Giraud, Eric" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_author" content="Arrighi, Jean-François" /><meta name="citation_author_institution" content=" IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France " /><meta name="citation_title" content=" A gene-based map of the Nod factor-independent Aeschynomene evenia sheds new light on the evolution of nodulation and legume genomes " /><meta name="citation_firstpage" content="365" /><meta name="citation_lastpage" content="376" /><meta name="citation_doi" content="10.1093/dnares/dsw020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Legumes: importance and constraints to greater use,; citation_author=Graham  P.H.; citation_author=Vance  C.P.;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=872-7; " /><meta name="citation_reference" content="citation_title=Evolving ideas of legume evolution and diversity: a taxonomic perspective on the occurrence of nodulation; citation_author=Sprent  J.I.;  citation_journal_title=New Phytol;  citation_year=2007;  citation_volume=174;  citation_pages=11-25; " /><meta name="citation_reference" content="citation_title=A simplified method for differential staining of aborted and non-aborted pollen grains; citation_author=Peterson  R.; citation_author=Slovin  J.P.; citation_author=Chen  C.;  citation_journal_title=Int. J. Plant Biol;  citation_year=2010;  citation_volume=1;  citation_pages=e13" /><meta name="citation_reference" content="citation_title=An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure, stability and evolution of legume genomes; citation_author=Bertioli  D.J.; citation_author=Moretzsohn  M.C.; citation_author=Madsen  L.H.;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=45-55; " /><meta name="citation_reference" content="citation_title=Common and not so common entry; citation_author=Held  M.; citation_author=Hossain  M.S.; citation_author=Yokota  K.,;  citation_journal_title=Trends Plant Sci;  citation_year=2010;  citation_volume=15;  citation_pages=540-5; " /><meta name="citation_reference" content="citation_title=Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the Tertiary; citation_author=Lavin  M.; citation_author=Herendeen  P.S.; citation_author=Wojciechowski  M.F.;  citation_journal_title=Syst. Biol;  citation_year=2005;  citation_volume=54;  citation_pages=575-594; " /><meta name="citation_reference" content="citation_title=The rest of the iceberg. Legume diversity and evolution in a phylogenetic context,; citation_author=Doyle  J.J.; citation_author=Luckow  M.A.;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=900-10; " /><meta name="citation_reference" content="citation_title=The rules of engagement in the legume-rhizobial symbiosis,; citation_author=Oldroyd  G.E.D.; citation_author=Murray  J.D.; citation_author=Poole  P.S.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2011;  citation_volume=45;  citation_pages=119-44; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species,; citation_author=Choi  H.K.; citation_author=Mun  J.H.; citation_author=Kim  D.J.;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2004;  citation_volume=101;  citation_pages=15289-94; " /><meta name="citation_reference" content="citation_title=Comparative genetic mapping and discovery of linkage disequilibrium across linkage groups in white clover ( Trifolium repens L.); citation_author=Isobe  S.N.; citation_author=Hisano  H.; citation_author=Sato  S.;  citation_journal_title=G3;  citation_year=2012;  citation_volume=2;  citation_pages=607-17; " /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; citation_author=Sato  S.; citation_author=Nakamura  Y.; citation_author=Kaneko  T.;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean,; citation_author=Schmutz  J.; citation_author=Cannon  S.B.; citation_author=Schlueter  J.;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses; citation_author=Young  N.D.; citation_author=Debellé  F.; citation_author=Oldroyd  G.E.;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=480;  citation_pages=520-4; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea ( Cajanus cajan ), an orphan legume crop of resource-poor farmers; citation_author=Varshney  R.K.; citation_author=Chen  W.; citation_author=Li  Y.;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=Bridging model and crop legumes through comparative genomics; citation_author=Zhu  H.; citation_author=Choi  H.K.; citation_author=Cook  D.R.; citation_author=Shoemaker  R.C.;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=137;  citation_pages=1189-96; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Chickpea ( Cicer arietinum ) provides a resource for trait improvement; citation_author=Varshney  R.K.; citation_author=Song  C.; citation_author=Saxena  R.K.;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=31;  citation_pages=240-6; " /><meta name="citation_reference" content="citation_title=A reference genome for common bean and genome wide analysis of dual domestications; citation_author=Schmutz  J.; citation_author=Mc Clean  P.E.; citation_author=Mamido  S.;  citation_journal_title=Nature Genet;  citation_year=2014;  citation_volume=46;  citation_pages=707-13; " /><meta name="citation_reference" content="citation_title=The genome sequences of Arachis duranensis and Arachis ipaensis the diploid ancestors of cultivated peanut; citation_author=Bertioli  D.J.; citation_author=Cannon  S.B.; citation_author=Froenicke  L.;  citation_journal_title=Nature Genet;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia; citation_author=Giraud  E.; citation_author=Moulin  L.; citation_author=Vallenet  D.;  citation_journal_title=Science;  citation_year=2007;  citation_volume=316;  citation_pages=1307-1312; " /><meta name="citation_reference" content="citation_title=Legume-rhizobial symbiosis: an anorexic model?; citation_author=Sprent  J.I.; citation_author=James  E.K.;  citation_journal_title=New Phytol;  citation_year=2008;  citation_volume=179;  citation_pages=3-5; " /><meta name="citation_reference" content="citation_title=Aeschynomene evenia , a model plant for studying the molecular genetics of the Nod-independent rhizobium-legume symbiosis; citation_author=Arrighi  J.F.; citation_author=Cartieaux  F.; citation_author=Brown  S.C.;  citation_journal_title=Mol. Plant-Microbe Interact;  citation_year=2012;  citation_volume=25;  citation_pages=851-861; " /><meta name="citation_reference" content="citation_title=Genotype delimitation in the Nod-independent model legume Aeschynomene evenia; citation_author=Arrighi  J.F.; citation_author=Cartieaux  F.; citation_author=Chaintreuil  C.;  citation_journal_title=PloS One;  citation_year=2013;  citation_volume=8;  citation_pages=e63836" /><meta name="citation_reference" content="citation_title=Isolation of total RNA from tissues rich in polyphenols and polysaccharides of mangrove plants; citation_author=Rubio-Piña  J.A.; citation_author=Zapata-Pérez  O.;  citation_journal_title=Plant Biotech;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Contribution of subgenomes to the transcriptomes and their intertwined regulation in the allopolyploid Coffea arabica grown at contrasted temperatures; citation_author=Combes  M.C.; citation_author=Dereeper  A.; citation_author=Severac  D.;  citation_journal_title=New Phytol;  citation_year=2013;  citation_volume=200;  citation_pages=251-60; " /><meta name="citation_reference" content="citation_title=Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of Theobroma cacao L; citation_author=Allegre  M.; citation_author=Argoult  X.; citation_author=Boccara  M.;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=23-35; " /><meta name="citation_reference" content="citation_title=The molecular genetic linkage map of the model legume Medicago truncatula : an essential tool for comparative legume genomics and the isolation of agronomically important genes; citation_author=Thoquet  P.; citation_author=Ghérardi  M.; citation_author=Journet  E.P.;  citation_journal_title=BMC Plant Biology;  citation_year=2002;  citation_volume=2;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Rhizobium-legume symbioses: the crucial role of plant immunity; citation_author=Gourion  B.; citation_author=Berrabah  F.; citation_author=Ratet  P.;  citation_journal_title=Trends Plant Sci;  citation_year=2014;  citation_volume=20;  citation_pages=186-94; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms controlling legume autoregulation of nodulation; citation_author=Reid  D.E.; citation_author=Ferguson  B.J.; citation_author=Hayashi  S.;  citation_journal_title=Ann. Bot;  citation_year=2011;  citation_volume=108;  citation_pages=789-95; " /><meta name="citation_reference" content="citation_title=NODULE ROOT and COCHLEATA maintain nodule development and are legume orthologs of Arabidopsis BLADE-ON-PETIOLE genes; citation_author=Couzigou  J.M.; citation_author=Zhukov  V.; citation_author=Mondy  S.;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=4498-510; " /><meta name="citation_reference" content="citation_title=Endoreduplication-mediated initiation of symbiotic organ development in Lotus japonicus; citation_author=Suzaki  T.; citation_author=Ito  M.; citation_author=Yoro  E.;  citation_journal_title=Development;  citation_year=2014;  citation_volume=141;  citation_pages=2441-5; " /><meta name="citation_reference" content="citation_title=Lotus japonicus SUNERGOS1 encodes a predicted subunit A of a DNA topoisomerase VI that is required for nodule differentiation and accommodation of rhizobial infection; citation_author=Yoon  H.J.; citation_author=Hossain  M.S.; citation_author=Held  M.;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=78;  citation_pages=811-21; " /><meta name="citation_reference" content="citation_title=Receptor-mediated exopolysaccharide perception controls bacterial infection; citation_author=Kawaharada  Y.; citation_author=Kelly  S.; citation_author=Nielsen  M.W.;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=523;  citation_pages=308-12; " /><meta name="citation_reference" content="citation_title=Lipo-chitooligosaccharidic symbiotic signals are recognized by LysM receptor-like kinase LYR3 in the legume Medicago truncatula; citation_author=Fliegmann  J.; citation_author=Canova  S.; citation_author=Lachaud  C.;  citation_journal_title=ACS Chem. Biol;  citation_year=2013;  citation_volume=8;  citation_pages=1900-6; " /><meta name="citation_reference" content="citation_title=The Nod factor-independent nodulation in Aeschynomene evenia required the common plant-microbe symbiotic “toolkit”; citation_author=Fabre  S.; citation_author=Gully  D.; citation_author=Poitout  A.;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=2654-64; " /><meta name="citation_reference" content="citation_title=Convergent evolution of endosymbiont diffenrentiation in Dalbergioid and IRLC legumes mediated by nodule-specific cysteine-rich peptides; citation_author=Czernic  P.; citation_author=Gully  D.; citation_author=Cartieaux  F.;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=1254-65; " /><meta name="citation_reference" content="citation_title=A SNP and SSR based genetic map of Asparagus Bean ( Vigna unguiculata ssp. sesquipedialis ) and comparison with the broader species; citation_author=Xu  P.; citation_author=Wu  X.; citation_author=Wang  B.;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e15952" /><meta name="citation_reference" content="citation_title=Ancient orphan crop joins modern era: gene-based SNP discovery and mapping in lentil; citation_author=Sharpe  A.G.; citation_author=Ramsay  L; citation_author=Sanderson  L.A.;  citation_journal_title=BMC Genomics;  citation_volume=14;  citation_pages=192" /><meta name="citation_reference" content="citation_title=The Medicago truncatula reference accession A17 has an aberrant chromosomal configuration; citation_author=Kamphuis  L.G.; citation_author=Williams  A.H.; citation_author=D’Souza  N.K.;  citation_journal_title=New Phytol;  citation_year=2007;  citation_volume=174;  citation_pages=299-303; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map of the model legume Lotus japonicus using an intraspecific F2 population; citation_author=Hayashi  M.; citation_author=Miyahara  A.; citation_author=Sato  S.;  citation_journal_title=DNA Res;  citation_year=2001;  citation_volume=8;  citation_pages=301-10; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the model legume Lotus japonicus and strategies for fast mapping of new loci; citation_author=Sandal  N.; citation_author=Krusell  L.; citation_author=Radutoiu  S.;  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=1673-83; " /><meta name="citation_reference" content="citation_title=Genetic mapping of legume orthologs reveals high conservation of synteny between lentil species and the sequenced genomes of Medicago and chickpea; citation_author=Gujaria-Verma  N.; citation_author=Vail  S.L.; citation_author=Carrasquilla-Garcia  N.;  citation_journal_title=Front. Plant. Sci;  citation_year=2014;  citation_volume=5;  citation_pages=676" /><meta name="citation_reference" content="citation_title=Matita , a new retroelement from peanut: characterization and evolutionary context in the light of the Arachis A-B genome divergence; citation_author=Nielen  S.; citation_author=Vigidal  B.S.; citation_author=Leal-Bertioli  S.C.M.;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2012;  citation_volume=287;  citation_pages=21-38; " /><meta name="citation_reference" content="citation_title=Genetics of symbiosis in Lotus japonicus : recombinant inbred lines, comparative genomic maps, and map position of 35 symbiotic loci; citation_author=Sandal  N.; citation_author=Petersen  T.R.; citation_author=Murray  J.;  citation_journal_title=MPMI;  citation_year=2006;  citation_volume=16;  citation_pages=80-91; " /><meta name="citation_reference" content="citation_title=A set of Lotus japonicus Gifu x Lotus burttii recombinant inbred lines facilitates map-based cloning and QTL mapping; citation_author=Sandal  N.; citation_author=Jin  H.; citation_author=Rodriguez-Navarro  D.N.;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=317-23; " /><meta name="citation_reference" content="citation_title=Genetic suppressors of the Lotus japonicus har1-1 hypernodulation phenotype; citation_author=Murray  J.; citation_author=Karas  B.; citation_author=Ross  L.;  citation_journal_title=MPMI;  citation_year=2006;  citation_volume=19;  citation_pages=1082-91; " /><meta name="citation_reference" content="citation_title=An antimicrobial peptide essential for bacterial survival in the nitrogen-fixing symbiosis; citation_author=Kim  M.; citation_author=Chen  Y.; citation_author=Xi  J;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_volume=112;  citation_pages=15238-43; " /><meta name="citation_reference" content="citation_title=SHOREmap: simultaneous mapping and mutation identification by deep sequencing; citation_author=Schneeberger  K.; citation_author=Ossowski  S.; citation_author=Lanz  C.;  citation_journal_title=Nature Methods;  citation_year=2009;  citation_volume=6;  citation_pages=550-1; " /><meta name="citation_reference" content="citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap; citation_author=Aba  A.; citation_author=Kosugi  S.; citation_author=Yoshida  K.;  citation_journal_title=Nat. Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=174-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/365/17463035/dsw020.pdf" /><meta name="description" content="Abstract.  Aeschynomene evenia has emerged as a new model legume for the deciphering of the molecular mechanisms of an alternative symbiotic process that is ind" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/365/2469965" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/365/2469965" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/365/2469965");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw020';
    </script>
<title>
	 A gene-based map of the Nod factor-independent Aeschynomene evenia sheds new light on the evolution of nodulation and legume genomes  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469965" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYqvATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hhaW50cmV1aWwsIENsw6ltZW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJUkQsIFVNUiBMU1RNLCBDYW1wdXMgSW50ZXJuYXRpb25hbCBkZSBCYWlsbGFyZ3VldCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJpdmFsbGFuLCBSb25hbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBDSVJBRCwgVU1SIEFHQVAsIENhbXB1cyBkZSBMYXZhbGV0dGUsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCZXJ0aW9saSwgRGF2aWQgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgVW5pdmVyc2l0eSBvZiBCcmFzw61saWEsIEluc3RpdHV0ZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBDYW1wdXMgRGFyY3kgUmliZWlybywgNzA5MTAtOTAwIEJyYXPDrWxpYSwgREYsIEJyYXppbCAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLbG9wcCwgQ2hyaXN0b3BoZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJTlJBLCBQbGF0ZWZvcm1lIEdlbm9Ub3VsIEJpb2luZm8sIFVSIDg3NSwgSU5SQSBBdXpldmlsbGUsIEYtMzEzMjYgQ2FzdGFuZXQtVG9sb3NhbiwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdvdXp5LCBKw6lyw7RtZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJTlJBLCBVTVI0NDEgTElQTSwgSU5SQSBBdXpldmlsbGUsIEYtMzEzMjYgQ2FzdGFuZXQtVG9sb3NhbiwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNvdXJ0b2lzLCBCcmlnaXR0ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBDSVJBRCwgVU1SIEFHQVAsIENhbXB1cyBkZSBMYXZhbGV0dGUsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWxldXgsIFBoaWxpcHBlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIElSRCwgVU1SIExTVE0sIENhbXB1cyBJbnRlcm5hdGlvbmFsIGRlIEJhaWxsYXJndWV0LCBGLTM0Mzk4IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIElOUkEsIFBsYXRlZm9ybWUgR2Vub1RvdWwgQmlvaW5mbywgVVIgODc1LCBJTlJBIEF1emV2aWxsZSwgRi0zMTMyNiBDYXN0YW5ldC1Ub2xvc2FuLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFydGluLCBHdWlsbGF1bWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQ0lSQUQsIFVNUiBBR0FQLCBDYW1wdXMgZGUgTGF2YWxldHRlLCBGLTM0Mzk4IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFtaSwgSmVhbi1GcmFuw6dvaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQ0lSQUQsIFVNUiBBR0FQLCBDYW1wdXMgZGUgTGF2YWxldHRlLCBGLTM0Mzk4IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VsbHksIERqYW1lbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJUkQsIFVNUiBMU1RNLCBDYW1wdXMgSW50ZXJuYXRpb25hbCBkZSBCYWlsbGFyZ3VldCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhcnJpbmVsbG8sIEh1Z3VlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBNR1gtTW9udHBlbGxpZXIgR2Vub21pWCwgSW5zdGl0dXQgZGUgR8Opbm9taXF1ZSBGb25jdGlvbm5lbGxlLCBGLTM0MDk0IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU8OpdmVyYWMsIERhbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgTUdYLU1vbnRwZWxsaWVyIEdlbm9taVgsIEluc3RpdHV0IGRlIEfDqW5vbWlxdWUgRm9uY3Rpb25uZWxsZSwgRi0zNDA5NCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhdHJlbCwgRGVscGhpbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSVJELCBVTVIgTFNUTSwgQ2FtcHVzIEludGVybmF0aW9uYWwgZGUgQmFpbGxhcmd1ZXQsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSVJELCBDZW50cmUgSVJEIGRlIE1vbnRwZWxsaWVyIEZyYW5jZSBTdWQsIEYtMzQzOTQgTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYXJkb3V4LCBKb8OrbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJUkQsIFVNUiBMU1RNLCBDYW1wdXMgSW50ZXJuYXRpb25hbCBkZSBCYWlsbGFyZ3VldCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJpYmnDqHJlLCBXaWxsaWFtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIElSRCwgQ2VudHJlIElSRCBkZSBNb250cGVsbGllciBGcmFuY2UgU3VkLCBGLTM0Mzk0IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm91cnNvdCwgTWFyYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJUkQsIFVNUiBMU1RNLCBDYW1wdXMgSW50ZXJuYXRpb25hbCBkZSBCYWlsbGFyZ3VldCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhcnRpZWF1eCwgRmFiaWVubmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSVJELCBVTVIgTFNUTSwgQ2FtcHVzIEludGVybmF0aW9uYWwgZGUgQmFpbGxhcmd1ZXQsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDemVybmljLCBQaWVycmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSVJELCBVTVIgTFNUTSwgQ2FtcHVzIEludGVybmF0aW9uYWwgZGUgQmFpbGxhcmd1ZXQsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYXRldCwgUGFzY2FsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIEluc3RpdHV0ZSBvZiBQbGFudCBTY2llbmNlcyBQYXJpcyBTYWNsYXkgSVBTMiwgQ05SUywgSU5SQSwgVW5pdmVyc2l0w6kgUGFyaXMtU3VkLCBVbml2ZXJzaXTDqSBFdnJ5LCBVbml2ZXJzaXTDqSBQYXJpcy1TYWNsYXksIDkxNDA1IE9yc2F5LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIEluc3RpdHV0ZSBvZiBQbGFudCBTY2llbmNlcyBQYXJpcy1TYWNsYXkgSVBTMiwgUGFyaXMgRGlkZXJvdCwgU29yYm9ubmUgUGFyaXMtQ2l0w6ksIDkxNDA1IE9yc2F5LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW91cm5ldCwgUGllcnJlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIENJUkFELCBVTVIgQUdBUCwgQ2FtcHVzIGRlIExhdmFsZXR0ZSwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdpcmF1ZCwgRXJpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJUkQsIFVNUiBMU1RNLCBDYW1wdXMgSW50ZXJuYXRpb25hbCBkZSBCYWlsbGFyZ3VldCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFycmlnaGksIEplYW4tRnJhbsOnb2lzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIElSRCwgVU1SIExTVE0sIENhbXB1cyBJbnRlcm5hdGlvbmFsIGRlIEJhaWxsYXJndWV0LCBGLTM0Mzk4IE1vbnRwZWxsaWVyLCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSIgQSBnZW5lLWJhc2VkIG1hcCBvZiB0aGUgTm9kIGZhY3Rvci1pbmRlcGVuZGVudCBBZXNjaHlub21lbmUgZXZlbmlhIHNoZWRzIG5ldyBsaWdodCBvbiB0aGUgZXZvbHV0aW9uIG9mIG5vZHVsYXRpb24gYW5kIGxlZ3VtZSBnZW5vbWVzICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWVzOiBpbXBvcnRhbmNlIGFuZCBjb25zdHJhaW50cyB0byBncmVhdGVyIHVzZSw7IGNpdGF0aW9uX2F1dGhvcj1HcmFoYW0gIFAuSC47IGNpdGF0aW9uX2F1dGhvcj1WYW5jZSAgQy5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz04NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHZpbmcgaWRlYXMgb2YgbGVndW1lIGV2b2x1dGlvbiBhbmQgZGl2ZXJzaXR5OiBhIHRheG9ub21pYyBwZXJzcGVjdGl2ZSBvbiB0aGUgb2NjdXJyZW5jZSBvZiBub2R1bGF0aW9uOyBjaXRhdGlvbl9hdXRob3I9U3ByZW50ICBKLkkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NDsgIGNpdGF0aW9uX3BhZ2VzPTExLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsaWZpZWQgbWV0aG9kIGZvciBkaWZmZXJlbnRpYWwgc3RhaW5pbmcgb2YgYWJvcnRlZCBhbmQgbm9uLWFib3J0ZWQgcG9sbGVuIGdyYWluczsgY2l0YXRpb25fYXV0aG9yPVBldGVyc29uICBSLjsgY2l0YXRpb25fYXV0aG9yPVNsb3ZpbiAgSi5QLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWUxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFuYWx5c2lzIG9mIHN5bnRlbnkgb2YgQXJhY2hpcyB3aXRoIExvdHVzIGFuZCBNZWRpY2FnbyBzaGVkcyBuZXcgbGlnaHQgb24gdGhlIHN0cnVjdHVyZSwgc3RhYmlsaXR5IGFuZCBldm9sdXRpb24gb2YgbGVndW1lIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1CZXJ0aW9saSAgRC5KLjsgY2l0YXRpb25fYXV0aG9yPU1vcmV0enNvaG4gIE0uQy47IGNpdGF0aW9uX2F1dGhvcj1NYWRzZW4gIEwuSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tb24gYW5kIG5vdCBzbyBjb21tb24gZW50cnk7IGNpdGF0aW9uX2F1dGhvcj1IZWxkICBNLjsgY2l0YXRpb25fYXV0aG9yPUhvc3NhaW4gIE0uUy47IGNpdGF0aW9uX2F1dGhvcj1Zb2tvdGEgIEsuLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU0MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgcmF0ZXMgYW5hbHlzaXMgb2YgTGVndW1pbm9zYWUgaW1wbGljYXRlcyBhIHJhcGlkIGRpdmVyc2lmaWNhdGlvbiBvZiBsaW5lYWdlcyBkdXJpbmcgdGhlIFRlcnRpYXJ5OyBjaXRhdGlvbl9hdXRob3I9TGF2aW4gIE0uOyBjaXRhdGlvbl9hdXRob3I9SGVyZW5kZWVuICBQLlMuOyBjaXRhdGlvbl9hdXRob3I9V29qY2llY2hvd3NraSAgTS5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTU3NS01OTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZXN0IG9mIHRoZSBpY2ViZXJnLiBMZWd1bWUgZGl2ZXJzaXR5IGFuZCBldm9sdXRpb24gaW4gYSBwaHlsb2dlbmV0aWMgY29udGV4dCw7IGNpdGF0aW9uX2F1dGhvcj1Eb3lsZSAgSi5KLjsgY2l0YXRpb25fYXV0aG9yPUx1Y2tvdyAgTS5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz05MDAtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBydWxlcyBvZiBlbmdhZ2VtZW50IGluIHRoZSBsZWd1bWUtcmhpem9iaWFsIHN5bWJpb3Npcyw7IGNpdGF0aW9uX2F1dGhvcj1PbGRyb3lkICBHLkUuRC47IGNpdGF0aW9uX2F1dGhvcj1NdXJyYXkgIEouRC47IGNpdGF0aW9uX2F1dGhvcj1Qb29sZSAgUC5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTExOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyBnZW5vbWUgY29uc2VydmF0aW9uIGJldHdlZW4gY3JvcCBhbmQgbW9kZWwgbGVndW1lIHNwZWNpZXMsOyBjaXRhdGlvbl9hdXRob3I9Q2hvaSAgSC5LLjsgY2l0YXRpb25fYXV0aG9yPU11biAgSi5ILjsgY2l0YXRpb25fYXV0aG9yPUtpbSAgRC5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE1Mjg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5ldGljIG1hcHBpbmcgYW5kIGRpc2NvdmVyeSBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGFjcm9zcyBsaW5rYWdlIGdyb3VwcyBpbiB3aGl0ZSBjbG92ZXIgKCBUcmlmb2xpdW0gcmVwZW5zIEwuKTsgY2l0YXRpb25fYXV0aG9yPUlzb2JlICBTLk4uOyBjaXRhdGlvbl9hdXRob3I9SGlzYW5vICBILjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NjA3LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBsZWd1bWUsIExvdHVzIGphcG9uaWN1czsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyBjaXRhdGlvbl9hdXRob3I9TmFrYW11cmEgIFkuOyBjaXRhdGlvbl9hdXRob3I9S2FuZWtvICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBwYWxhZW9wb2x5cGxvaWQgc295YmVhbiw7IGNpdGF0aW9uX2F1dGhvcj1TY2htdXR6ICBKLjsgY2l0YXRpb25fYXV0aG9yPUNhbm5vbiAgUy5CLjsgY2l0YXRpb25fYXV0aG9yPVNjaGx1ZXRlciAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBNZWRpY2FnbyBnZW5vbWUgcHJvdmlkZXMgaW5zaWdodCBpbnRvIHRoZSBldm9sdXRpb24gb2Ygcmhpem9iaWFsIHN5bWJpb3NlczsgY2l0YXRpb25fYXV0aG9yPVlvdW5nICBOLkQuOyBjaXRhdGlvbl9hdXRob3I9RGViZWxsw6kgIEYuOyBjaXRhdGlvbl9hdXRob3I9T2xkcm95ZCAgRy5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ4MDsgIGNpdGF0aW9uX3BhZ2VzPTUyMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgcGlnZW9ucGVhICggQ2FqYW51cyBjYWphbiApLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgcmVzb3VyY2UtcG9vciBmYXJtZXJzOyBjaXRhdGlvbl9hdXRob3I9VmFyc2huZXkgIFIuSy47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBXLjsgY2l0YXRpb25fYXV0aG9yPUxpICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9ODMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJpZGdpbmcgbW9kZWwgYW5kIGNyb3AgbGVndW1lcyB0aHJvdWdoIGNvbXBhcmF0aXZlIGdlbm9taWNzOyBjaXRhdGlvbl9hdXRob3I9Wmh1ICBILjsgY2l0YXRpb25fYXV0aG9yPUNob2kgIEguSy47IGNpdGF0aW9uX2F1dGhvcj1Db29rICBELlIuOyBjaXRhdGlvbl9hdXRob3I9U2hvZW1ha2VyICBSLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzNzsgIGNpdGF0aW9uX3BhZ2VzPTExODktOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBDaGlja3BlYSAoIENpY2VyIGFyaWV0aW51bSApIHByb3ZpZGVzIGEgcmVzb3VyY2UgZm9yIHRyYWl0IGltcHJvdmVtZW50OyBjaXRhdGlvbl9hdXRob3I9VmFyc2huZXkgIFIuSy47IGNpdGF0aW9uX2F1dGhvcj1Tb25nICBDLjsgY2l0YXRpb25fYXV0aG9yPVNheGVuYSAgUi5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjQwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmVmZXJlbmNlIGdlbm9tZSBmb3IgY29tbW9uIGJlYW4gYW5kIGdlbm9tZSB3aWRlIGFuYWx5c2lzIG9mIGR1YWwgZG9tZXN0aWNhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1TY2htdXR6ICBKLjsgY2l0YXRpb25fYXV0aG9yPU1jIENsZWFuICBQLkUuOyBjaXRhdGlvbl9hdXRob3I9TWFtaWRvICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NzA3LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlcyBvZiBBcmFjaGlzIGR1cmFuZW5zaXMgYW5kIEFyYWNoaXMgaXBhZW5zaXMgdGhlIGRpcGxvaWQgYW5jZXN0b3JzIG9mIGN1bHRpdmF0ZWQgcGVhbnV0OyBjaXRhdGlvbl9hdXRob3I9QmVydGlvbGkgIEQuSi47IGNpdGF0aW9uX2F1dGhvcj1DYW5ub24gIFMuQi47IGNpdGF0aW9uX2F1dGhvcj1Gcm9lbmlja2UgIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmUgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZXMgc3ltYmlvc2VzOiBhYnNlbmNlIG9mIE5vZCBnZW5lcyBpbiBwaG90b3N5bnRoZXRpYyBicmFkeXJoaXpvYmlhOyBjaXRhdGlvbl9hdXRob3I9R2lyYXVkICBFLjsgY2l0YXRpb25fYXV0aG9yPU1vdWxpbiAgTC47IGNpdGF0aW9uX2F1dGhvcj1WYWxsZW5ldCAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE2OyAgY2l0YXRpb25fcGFnZXM9MTMwNy0xMzEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWUtcmhpem9iaWFsIHN5bWJpb3NpczogYW4gYW5vcmV4aWMgbW9kZWw/OyBjaXRhdGlvbl9hdXRob3I9U3ByZW50ICBKLkkuOyBjaXRhdGlvbl9hdXRob3I9SmFtZXMgIEUuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZXNjaHlub21lbmUgZXZlbmlhICwgYSBtb2RlbCBwbGFudCBmb3Igc3R1ZHlpbmcgdGhlIG1vbGVjdWxhciBnZW5ldGljcyBvZiB0aGUgTm9kLWluZGVwZW5kZW50IHJoaXpvYml1bS1sZWd1bWUgc3ltYmlvc2lzOyBjaXRhdGlvbl9hdXRob3I9QXJyaWdoaSAgSi5GLjsgY2l0YXRpb25fYXV0aG9yPUNhcnRpZWF1eCAgRi47IGNpdGF0aW9uX2F1dGhvcj1Ccm93biAgUy5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudC1NaWNyb2JlIEludGVyYWN0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9ODUxLTg2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub3R5cGUgZGVsaW1pdGF0aW9uIGluIHRoZSBOb2QtaW5kZXBlbmRlbnQgbW9kZWwgbGVndW1lIEFlc2NoeW5vbWVuZSBldmVuaWE7IGNpdGF0aW9uX2F1dGhvcj1BcnJpZ2hpICBKLkYuOyBjaXRhdGlvbl9hdXRob3I9Q2FydGllYXV4ICBGLjsgY2l0YXRpb25fYXV0aG9yPUNoYWludHJldWlsICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MzgzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiB0b3RhbCBSTkEgZnJvbSB0aXNzdWVzIHJpY2ggaW4gcG9seXBoZW5vbHMgYW5kIHBvbHlzYWNjaGFyaWRlcyBvZiBtYW5ncm92ZSBwbGFudHM7IGNpdGF0aW9uX2F1dGhvcj1SdWJpby1QacOxYSAgSi5BLjsgY2l0YXRpb25fYXV0aG9yPVphcGF0YS1Qw6lyZXogIE8uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNoOyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cmlidXRpb24gb2Ygc3ViZ2Vub21lcyB0byB0aGUgdHJhbnNjcmlwdG9tZXMgYW5kIHRoZWlyIGludGVydHdpbmVkIHJlZ3VsYXRpb24gaW4gdGhlIGFsbG9wb2x5cGxvaWQgQ29mZmVhIGFyYWJpY2EgZ3Jvd24gYXQgY29udHJhc3RlZCB0ZW1wZXJhdHVyZXM7IGNpdGF0aW9uX2F1dGhvcj1Db21iZXMgIE0uQy47IGNpdGF0aW9uX2F1dGhvcj1EZXJlZXBlciAgQS47IGNpdGF0aW9uX2F1dGhvcj1TZXZlcmFjICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDA7ICBjaXRhdGlvbl9wYWdlcz0yNTEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBhbmQgbWFwcGluZyBvZiBhIG5ldyBleHByZXNzZWQgc2VxdWVuY2UgdGFnLXNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmQgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBwYW5lbCBmb3IgbGFyZ2Utc2NhbGUgZ2VuZXRpYyBzdHVkaWVzIGFuZCBicmVlZGluZyBvZiBUaGVvYnJvbWEgY2FjYW8gTDsgY2l0YXRpb25fYXV0aG9yPUFsbGVncmUgIE0uOyBjaXRhdGlvbl9hdXRob3I9QXJnb3VsdCAgWC47IGNpdGF0aW9uX2F1dGhvcj1Cb2NjYXJhICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTIzLTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBNZWRpY2FnbyB0cnVuY2F0dWxhIDogYW4gZXNzZW50aWFsIHRvb2wgZm9yIGNvbXBhcmF0aXZlIGxlZ3VtZSBnZW5vbWljcyBhbmQgdGhlIGlzb2xhdGlvbiBvZiBhZ3Jvbm9taWNhbGx5IGltcG9ydGFudCBnZW5lczsgY2l0YXRpb25fYXV0aG9yPVRob3F1ZXQgIFAuOyBjaXRhdGlvbl9hdXRob3I9R2jDqXJhcmRpICBNLjsgY2l0YXRpb25fYXV0aG9yPUpvdXJuZXQgIEUuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umhpem9iaXVtLWxlZ3VtZSBzeW1iaW9zZXM6IHRoZSBjcnVjaWFsIHJvbGUgb2YgcGxhbnQgaW1tdW5pdHk7IGNpdGF0aW9uX2F1dGhvcj1Hb3VyaW9uICBCLjsgY2l0YXRpb25fYXV0aG9yPUJlcnJhYmFoICBGLjsgY2l0YXRpb25fYXV0aG9yPVJhdGV0ICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE4Ni05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1lY2hhbmlzbXMgY29udHJvbGxpbmcgbGVndW1lIGF1dG9yZWd1bGF0aW9uIG9mIG5vZHVsYXRpb247IGNpdGF0aW9uX2F1dGhvcj1SZWlkICBELkUuOyBjaXRhdGlvbl9hdXRob3I9RmVyZ3Vzb24gIEIuSi47IGNpdGF0aW9uX2F1dGhvcj1IYXlhc2hpICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9Nzg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OT0RVTEUgUk9PVCBhbmQgQ09DSExFQVRBIG1haW50YWluIG5vZHVsZSBkZXZlbG9wbWVudCBhbmQgYXJlIGxlZ3VtZSBvcnRob2xvZ3Mgb2YgQXJhYmlkb3BzaXMgQkxBREUtT04tUEVUSU9MRSBnZW5lczsgY2l0YXRpb25fYXV0aG9yPUNvdXppZ291ICBKLk0uOyBjaXRhdGlvbl9hdXRob3I9Wmh1a292ICBWLjsgY2l0YXRpb25fYXV0aG9yPU1vbmR5ICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OTgtNTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvcmVkdXBsaWNhdGlvbi1tZWRpYXRlZCBpbml0aWF0aW9uIG9mIHN5bWJpb3RpYyBvcmdhbiBkZXZlbG9wbWVudCBpbiBMb3R1cyBqYXBvbmljdXM7IGNpdGF0aW9uX2F1dGhvcj1TdXpha2kgIFQuOyBjaXRhdGlvbl9hdXRob3I9SXRvICBNLjsgY2l0YXRpb25fYXV0aG9yPVlvcm8gIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0yNDQxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdHVzIGphcG9uaWN1cyBTVU5FUkdPUzEgZW5jb2RlcyBhIHByZWRpY3RlZCBzdWJ1bml0IEEgb2YgYSBETkEgdG9wb2lzb21lcmFzZSBWSSB0aGF0IGlzIHJlcXVpcmVkIGZvciBub2R1bGUgZGlmZmVyZW50aWF0aW9uIGFuZCBhY2NvbW1vZGF0aW9uIG9mIHJoaXpvYmlhbCBpbmZlY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1Zb29uICBILkouOyBjaXRhdGlvbl9hdXRob3I9SG9zc2FpbiAgTS5TLjsgY2l0YXRpb25fYXV0aG9yPUhlbGQgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9ODExLTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlcHRvci1tZWRpYXRlZCBleG9wb2x5c2FjY2hhcmlkZSBwZXJjZXB0aW9uIGNvbnRyb2xzIGJhY3RlcmlhbCBpbmZlY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1LYXdhaGFyYWRhICBZLjsgY2l0YXRpb25fYXV0aG9yPUtlbGx5ICBTLjsgY2l0YXRpb25fYXV0aG9yPU5pZWxzZW4gIE0uVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01MjM7ICBjaXRhdGlvbl9wYWdlcz0zMDgtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpcG8tY2hpdG9vbGlnb3NhY2NoYXJpZGljIHN5bWJpb3RpYyBzaWduYWxzIGFyZSByZWNvZ25pemVkIGJ5IEx5c00gcmVjZXB0b3ItbGlrZSBraW5hc2UgTFlSMyBpbiB0aGUgbGVndW1lIE1lZGljYWdvIHRydW5jYXR1bGE7IGNpdGF0aW9uX2F1dGhvcj1GbGllZ21hbm4gIEouOyBjaXRhdGlvbl9hdXRob3I9Q2Fub3ZhICBTLjsgY2l0YXRpb25fYXV0aG9yPUxhY2hhdWQgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BQ1MgQ2hlbS4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTkwMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTm9kIGZhY3Rvci1pbmRlcGVuZGVudCBub2R1bGF0aW9uIGluIEFlc2NoeW5vbWVuZSBldmVuaWEgcmVxdWlyZWQgdGhlIGNvbW1vbiBwbGFudC1taWNyb2JlIHN5bWJpb3RpYyDigJx0b29sa2l04oCdOyBjaXRhdGlvbl9hdXRob3I9RmFicmUgIFMuOyBjaXRhdGlvbl9hdXRob3I9R3VsbHkgIEQuOyBjaXRhdGlvbl9hdXRob3I9UG9pdG91dCAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY5OyAgY2l0YXRpb25fcGFnZXM9MjY1NC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udmVyZ2VudCBldm9sdXRpb24gb2YgZW5kb3N5bWJpb250IGRpZmZlbnJlbnRpYXRpb24gaW4gRGFsYmVyZ2lvaWQgYW5kIElSTEMgbGVndW1lcyBtZWRpYXRlZCBieSBub2R1bGUtc3BlY2lmaWMgY3lzdGVpbmUtcmljaCBwZXB0aWRlczsgY2l0YXRpb25fYXV0aG9yPUN6ZXJuaWMgIFAuOyBjaXRhdGlvbl9hdXRob3I9R3VsbHkgIEQuOyBjaXRhdGlvbl9hdXRob3I9Q2FydGllYXV4ICBGLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0xMjU0LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIFNOUCBhbmQgU1NSIGJhc2VkIGdlbmV0aWMgbWFwIG9mIEFzcGFyYWd1cyBCZWFuICggVmlnbmEgdW5ndWljdWxhdGEgc3NwLiBzZXNxdWlwZWRpYWxpcyApIGFuZCBjb21wYXJpc29uIHdpdGggdGhlIGJyb2FkZXIgc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPVh1ICBQLjsgY2l0YXRpb25fYXV0aG9yPVd1ICBYLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE1OTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5jaWVudCBvcnBoYW4gY3JvcCBqb2lucyBtb2Rlcm4gZXJhOiBnZW5lLWJhc2VkIFNOUCBkaXNjb3ZlcnkgYW5kIG1hcHBpbmcgaW4gbGVudGlsOyBjaXRhdGlvbl9hdXRob3I9U2hhcnBlICBBLkcuOyBjaXRhdGlvbl9hdXRob3I9UmFtc2F5ICBMOyBjaXRhdGlvbl9hdXRob3I9U2FuZGVyc29uICBMLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xOTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTWVkaWNhZ28gdHJ1bmNhdHVsYSByZWZlcmVuY2UgYWNjZXNzaW9uIEExNyBoYXMgYW4gYWJlcnJhbnQgY2hyb21vc29tYWwgY29uZmlndXJhdGlvbjsgY2l0YXRpb25fYXV0aG9yPUthbXBodWlzICBMLkcuOyBjaXRhdGlvbl9hdXRob3I9V2lsbGlhbXMgIEEuSC47IGNpdGF0aW9uX2F1dGhvcj1E4oCZU291emEgIE4uSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc0OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyB1c2luZyBhbiBpbnRyYXNwZWNpZmljIEYyIHBvcHVsYXRpb247IGNpdGF0aW9uX2F1dGhvcj1IYXlhc2hpICBNLjsgY2l0YXRpb25fYXV0aG9yPU1peWFoYXJhICBBLjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMDEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyBhbmQgc3RyYXRlZ2llcyBmb3IgZmFzdCBtYXBwaW5nIG9mIG5ldyBsb2NpOyBjaXRhdGlvbl9hdXRob3I9U2FuZGFsICBOLjsgY2l0YXRpb25fYXV0aG9yPUtydXNlbGwgIEwuOyBjaXRhdGlvbl9hdXRob3I9UmFkdXRvaXUgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjE7ICBjaXRhdGlvbl9wYWdlcz0xNjczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2YgbGVndW1lIG9ydGhvbG9ncyByZXZlYWxzIGhpZ2ggY29uc2VydmF0aW9uIG9mIHN5bnRlbnkgYmV0d2VlbiBsZW50aWwgc3BlY2llcyBhbmQgdGhlIHNlcXVlbmNlZCBnZW5vbWVzIG9mIE1lZGljYWdvIGFuZCBjaGlja3BlYTsgY2l0YXRpb25fYXV0aG9yPUd1amFyaWEtVmVybWEgIE4uOyBjaXRhdGlvbl9hdXRob3I9VmFpbCAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPUNhcnJhc3F1aWxsYS1HYXJjaWEgIE4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9Njc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF0aXRhICwgYSBuZXcgcmV0cm9lbGVtZW50IGZyb20gcGVhbnV0OiBjaGFyYWN0ZXJpemF0aW9uIGFuZCBldm9sdXRpb25hcnkgY29udGV4dCBpbiB0aGUgbGlnaHQgb2YgdGhlIEFyYWNoaXMgQS1CIGdlbm9tZSBkaXZlcmdlbmNlOyBjaXRhdGlvbl9hdXRob3I9TmllbGVuICBTLjsgY2l0YXRpb25fYXV0aG9yPVZpZ2lkYWwgIEIuUy47IGNpdGF0aW9uX2F1dGhvcj1MZWFsLUJlcnRpb2xpICBTLkMuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBzeW1iaW9zaXMgaW4gTG90dXMgamFwb25pY3VzIDogcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzLCBjb21wYXJhdGl2ZSBnZW5vbWljIG1hcHMsIGFuZCBtYXAgcG9zaXRpb24gb2YgMzUgc3ltYmlvdGljIGxvY2k7IGNpdGF0aW9uX2F1dGhvcj1TYW5kYWwgIE4uOyBjaXRhdGlvbl9hdXRob3I9UGV0ZXJzZW4gIFQuUi47IGNpdGF0aW9uX2F1dGhvcj1NdXJyYXkgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NUE1JOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODAtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2V0IG9mIExvdHVzIGphcG9uaWN1cyBHaWZ1IHggTG90dXMgYnVydHRpaSByZWNvbWJpbmFudCBpbmJyZWQgbGluZXMgZmFjaWxpdGF0ZXMgbWFwLWJhc2VkIGNsb25pbmcgYW5kIFFUTCBtYXBwaW5nOyBjaXRhdGlvbl9hdXRob3I9U2FuZGFsICBOLjsgY2l0YXRpb25fYXV0aG9yPUppbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1Sb2RyaWd1ZXotTmF2YXJybyAgRC5OLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTMxNy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzdXBwcmVzc29ycyBvZiB0aGUgTG90dXMgamFwb25pY3VzIGhhcjEtMSBoeXBlcm5vZHVsYXRpb24gcGhlbm90eXBlOyBjaXRhdGlvbl9hdXRob3I9TXVycmF5ICBKLjsgY2l0YXRpb25fYXV0aG9yPUthcmFzICBCLjsgY2l0YXRpb25fYXV0aG9yPVJvc3MgIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NUE1JOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTA4Mi05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYW50aW1pY3JvYmlhbCBwZXB0aWRlIGVzc2VudGlhbCBmb3IgYmFjdGVyaWFsIHN1cnZpdmFsIGluIHRoZSBuaXRyb2dlbi1maXhpbmcgc3ltYmlvc2lzOyBjaXRhdGlvbl9hdXRob3I9S2ltICBNLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFkuOyBjaXRhdGlvbl9hdXRob3I9WGkgIEo7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTE1MjM4LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSE9SRW1hcDogc2ltdWx0YW5lb3VzIG1hcHBpbmcgYW5kIG11dGF0aW9uIGlkZW50aWZpY2F0aW9uIGJ5IGRlZXAgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPVNjaG5lZWJlcmdlciAgSy47IGNpdGF0aW9uX2F1dGhvcj1Pc3Nvd3NraSAgUy47IGNpdGF0aW9uX2F1dGhvcj1MYW56ICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTU1MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyByZXZlYWxzIGFncm9ub21pY2FsbHkgaW1wb3J0YW50IGxvY2kgaW4gcmljZSB1c2luZyBNdXRNYXA7IGNpdGF0aW9uX2F1dGhvcj1BYmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9S29zdWdpICBTLjsgY2l0YXRpb25fYXV0aG9yPVlvc2hpZGEgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2guOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTc0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy80LzM2NS8xNzQ2MzAzNS9kc3cwMjAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFlc2NoeW5vbWVuZSBldmVuaWEgaGFzIGVtZXJnZWQgYXMgYSBuZXcgbW9kZWwgbGVndW1lIGZvciB0aGUgZGVjaXBoZXJpbmcgb2YgdGhlIG1vbGVjdWxhciBtZWNoYW5pc21zIG9mIGFuIGFsdGVybmF0aXZlIHN5bWJpb3RpYyBwcm9jZXNzIHRoYXQgaXMgaW5kIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDY5OTY1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZMCaImAaIiJvYLwcRTzfXywthBfo" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f365%2f2469965" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f365%2f2469965" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198502">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198502">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198504">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198504">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198510">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198510">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198529">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198529">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198599">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198599">4. Conclusions and perspectives</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198606">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198606">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198610">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198610">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198613">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198613">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198618">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198618">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198622">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198622">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/353/2469968">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/4/377/2469962">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> A gene-based map of the Nod factor-independent <em>Aeschynomene evenia</em> sheds new light on the evolution of nodulation and legume genomes 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cl&#233;mence Chaintreuil</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cl%c3%a9mence+Chaintreuil">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chaintreuil C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chaintreuil C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Clémence Chaintreuil</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ronan Rivallan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ronan+Rivallan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rivallan R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rivallan R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ronan Rivallan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David J. Bertioli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div> University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+J.+Bertioli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bertioli D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bertioli D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">David J. Bertioli</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christophe Klopp</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div> INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christophe+Klopp">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Klopp C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Klopp C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Christophe Klopp</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J&#233;r&#244;me Gouzy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>5</sup></div> INRA, UMR441 LIPM, INRA Auzeville, F-31326 Castanet-Tolosan, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J%c3%a9r%c3%b4me+Gouzy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gouzy J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gouzy J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jérôme Gouzy</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Brigitte Courtois</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Brigitte+Courtois">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Courtois B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Courtois B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Brigitte Courtois</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Philippe Leleux</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div><div class="aff"><div class="label -label"><sup>4</sup></div> INRA, Plateforme GenoToul Bioinfo, UR 875, INRA Auzeville, F-31326 Castanet-Tolosan, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Philippe+Leleux">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Leleux P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Leleux P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Philippe Leleux</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guillaume Martin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guillaume+Martin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Martin G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Martin G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Guillaume Martin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean-Fran&#231;ois Rami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean-Fran%c3%a7ois+Rami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rami J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rami J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jean-François Rami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Djamel Gully</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Djamel+Gully">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gully D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gully D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Djamel Gully</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hugues Parrinello</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>6</sup></div> MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hugues+Parrinello">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parrinello H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parrinello H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hugues Parrinello</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dany S&#233;verac</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>6</sup></div> MGX-Montpellier GenomiX, Institut de Génomique Fonctionnelle, F-34094 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dany+S%c3%a9verac">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=S&#233;verac D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;S&#233;verac D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Dany Séverac</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Delphine Patrel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div><div class="aff"><div class="label -label"><sup>7</sup></div> IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Delphine+Patrel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Patrel D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Patrel D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Delphine Patrel</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jo&#235;l Fardoux</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jo%c3%abl+Fardoux">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fardoux J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fardoux J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Joël Fardoux</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">William Ribi&#232;re</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>7</sup></div> IRD, Centre IRD de Montpellier France Sud, F-34394 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=William+Ribi%c3%a8re">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ribi&#232;re W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ribi&#232;re W&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">William Ribière</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marc Boursot</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marc+Boursot">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boursot M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boursot M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Marc Boursot</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fabienne Cartieaux</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fabienne+Cartieaux">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cartieaux F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cartieaux F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fabienne Cartieaux</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pierre Czernic</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pierre+Czernic">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Czernic P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Czernic P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Pierre Czernic</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pascal Ratet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>8</sup></div> Institute of Plant Sciences Paris Saclay IPS2, CNRS, INRA, Université Paris-Sud, Université Evry, Université Paris-Saclay, 91405 Orsay, France </div><div class="aff"><div class="label -label"><sup>9</sup></div> Institute of Plant Sciences Paris-Saclay IPS2, Paris Diderot, Sorbonne Paris-Cité, 91405 Orsay, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pascal+Ratet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ratet P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ratet P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Pascal Ratet</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pierre Mournet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> CIRAD, UMR AGAP, Campus de Lavalette, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pierre+Mournet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mournet P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mournet P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Pierre Mournet</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric Giraud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+Giraud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Giraud E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Giraud E&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Eric Giraud</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean-Fran&#231;ois Arrighi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> IRD, UMR LSTM, Campus International de Baillarguet, F-34398 Montpellier, France </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw020-cor1" data-open="dsw020-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel. +33 04 67 59 38 82. Fax. +33 04 67 59 38 02. Email: <a href="mailto:jean-francois.arrighi@ird.fr" target="_blank">jean-francois.arrighi@ird.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean-Fran%c3%a7ois+Arrighi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arrighi J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arrighi J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jean-François Arrighi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="footnote-content">Edited by Dr Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 365–376, <a href='https://doi.org/10.1093/dnares/dsw020'>https://doi.org/10.1093/dnares/dsw020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 June 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 March 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 May 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469965" href="/dnaresearch/article-pdf/23/4/365/17463035/dsw020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Clémence Chaintreuil, Ronan Rivallan, David J. Bertioli, Christophe Klopp, Jérôme Gouzy, Brigitte Courtois, Philippe Leleux, Guillaume Martin, Jean-François Rami, Djamel Gully, Hugues Parrinello, Dany Séverac, Delphine Patrel, Joël Fardoux, William Ribière, Marc Boursot, Fabienne Cartieaux, Pierre Czernic, Pascal Ratet, Pierre Mournet, Eric Giraud, Jean-François Arrighi;   A gene-based map of the Nod factor-independent <em>Aeschynomene evenia</em> sheds new light on the evolution of nodulation and legume genomes , <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 365–376, <a href='https://doi.org/10.1093/dnares/dsw020'>https://doi.org/10.1093/dnares/dsw020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469965?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20A%20gene-based%20map%20of%20the%20Nod%20factor-independent%20%26lt%3Bem%26gt%3BAeschynomene%20evenia%26lt%3B%2Fem%26gt%3B%20sheds%20new%20light%20on%20the%20evolution%20of%20nodulation%20and%20legume%20genomes%20&amp;publicationDate=2016-06-13&amp;volumeNum=23&amp;issueNum=4&amp;author=Chaintreuil%2C%20Cl%C3%A9mence%3B%20Rivallan%2C%20Ronan&amp;startPage=365&amp;endPage=376&amp;contentId=10.1093%2Fdnares%2Fdsw020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f365%2f2469965" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198502 id="87198502" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Aeschynomene evenia</em> has emerged as a new model legume for the deciphering of the molecular mechanisms of an alternative symbiotic process that is independent of the Nod factors. Whereas most of the research on nitrogen-fixing symbiosis, legume genetics and genomics has so far focused on Galegoid and Phaseolid legumes, <em>A. evenia</em> falls in the more basal and understudied Dalbergioid clade along with peanut ( <em>Arachis hypogaea</em> ). To provide insights into the symbiotic genes content and the structure of the <em>A. evenia</em> genome, we established a gene-based genetic map for this species. Firstly, an RNAseq analysis was performed on the two parental lines selected to generate a F <sub>2</sub> mapping population. The transcriptomic data were used to develop molecular markers and they allowed the identification of most symbiotic genes. The resulting map comprised 364 markers arranged in 10 linkage groups (2 <em>n</em>  = 20). A comparative analysis with the sequenced genomes of <em>Arachis duranensis</em> and <em>A. ipaensis</em> , the diploid ancestors of peanut, indicated blocks of conserved macrosynteny. Altogether, these results provided important clues regarding the evolution of symbiotic genes in a Nod factor-independent context. They provide a basis for a genome sequencing project and pave the way for forward genetic analysis of symbiosis in <em>A. evenia</em> . </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aeschynomene evenia&quot;"><em>Aeschynomene evenia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="symbiosis">symbiosis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="legume">legume</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=87198504 id="87198504" class="section-title" >1. Introduction</h2>
<p> Fabaceae, or legumes, account for ∼27% of the world’s primary crop production. The capacity of most legumes to form nitrogen fixing nodules with rhizobia underlies their importance as a source of protein for human and animal diet and of nitrogen in both natural and agricultural ecosystems. <a reveal-id="dsw020-B1" data-open="dsw020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Among the 20,000 legume species, around 14,000 fall into the Papilionoideae subfamily that is divided into four main clades Genistoids, Dalbergioids, Phaseoloids and Galegoids ( <a reveal-id="dsw020-F1" data-open="dsw020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). <a reveal-id="dsw020-B7" data-open="dsw020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p>                        <a id="87198506" scrollto-destination="87198506"></a>
<div id="dsw020-F1" data-id="dsw020-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f1p.png?Expires=2147483647&Signature=TamNMERnd269QwuJCbH1GVUEcf6vEKnOnbfGQT53qF0jWdRnjOovV-Yvof~iv7pUtxe7klwywzwV7eIjOd5JQQZEtn7jMwiSy9jJtS72Amj41QQgoaaCWAVoAHD--CHCI8mzGD4F8xvczqDtze3DubanLyWIdlLNT4NjqoMuWqdxryp4fnYYa962N8d7SDkEOvNMUaiRiVu6MHNBMTEqjiuYkGO6QI3hYFENj76QRy0tP2W-kF2lVWM4MDa5uxG25ljT4Wl4F0EVGUi8mzOMspUCA3CeA-HoyvHV-xBXmq6Zxna17HvjvIFypDGPquLs3Rrg05Nc-vd-L-3NSAzJVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phylogeny of the Papilionoid lineage and evolution of the symbiotic infection process. The tree is a simplified representation with triangle representing the major clades and the two subclades of the Galegoids: the Robinoids and the plastid DNA inverted repeat-lacking clade (IRLC). Known divergence times are indicated in MYA (million years ago) for some nodes (circles) and notable species are listed for each clade. Symbiotic infection processes encountered in the different clades are reported, excepted for the basal clades for which occurrence of nodulation is restricted to few understudied species. Note that Aeschynomene is the only legume genus where a Nod-independent symbiotic infection process (*NI*) has been reported for several species, including A. evenia . Phylogeny and drawings are adaptations, date estimates and nodulation properties come from previous publications. 2-6" path-from-xml="dsw020f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198506/dsw020f1p.png" path-from-xml="dsw020f1p.png" target="_blank">View large</a><a section="87198506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f1p.png?Expires=2147483647&Signature=rM3tIQt1EaCebOb5IikFbeir0Vf4m4yPA3TkYDYn2NO1mKw3IwnQsER3tMEPEoGR4KmmrdRVe-75CYBDgYwwDzA4ZZw3UPLPbS~Qr3JVKkSjVE05SYMtD5ucCkeVSlhVQXVIhY7EY47TCoLoi9hxtViWxZ3UhS9gT1kCAkmp1CcaqwnhMaxcLDNnD3KrqIvBxmNAwrtO~ystOPVpyOj6obGOd1b7dQggB3AGE02E53SUb70mSYdHLHOFI~X0jYAdnh1PWGWbA5RAO8Wm9yoarrx1wEeQpb7qzBge-5pOstM4j-ZGgvbcGaXCFix6hu8-QXTiw4FcEiYdnJkI7W7yEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198506&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phylogeny of the Papilionoid lineage and evolution of the symbiotic infection process. The tree is a simplified representation with triangle representing the major clades and the two subclades of the Galegoids: the Robinoids and the plastid DNA inverted repeat-lacking clade (IRLC). Known divergence times are indicated in MYA (million years ago) for some nodes (circles) and notable species are listed for each clade. Symbiotic infection processes encountered in the different clades are reported, excepted for the basal clades for which occurrence of nodulation is restricted to few understudied species. Note that <em>Aeschynomene</em> is the only legume genus where a Nod-independent symbiotic infection process (*NI*) has been reported for several species, including <em>A. evenia</em> . Phylogeny and drawings are adaptations, date estimates and nodulation properties come from previous publications. <a reveal-id="dsw020-B2" data-open="dsw020-B2" class="link link-ref link-reveal xref-bibr"><sup>2-</sup></a><sup>,</sup><a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p></div></div><div content-id="dsw020-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f1p.png?Expires=2147483647&Signature=TamNMERnd269QwuJCbH1GVUEcf6vEKnOnbfGQT53qF0jWdRnjOovV-Yvof~iv7pUtxe7klwywzwV7eIjOd5JQQZEtn7jMwiSy9jJtS72Amj41QQgoaaCWAVoAHD--CHCI8mzGD4F8xvczqDtze3DubanLyWIdlLNT4NjqoMuWqdxryp4fnYYa962N8d7SDkEOvNMUaiRiVu6MHNBMTEqjiuYkGO6QI3hYFENj76QRy0tP2W-kF2lVWM4MDa5uxG25ljT4Wl4F0EVGUi8mzOMspUCA3CeA-HoyvHV-xBXmq6Zxna17HvjvIFypDGPquLs3Rrg05Nc-vd-L-3NSAzJVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Phylogeny of the Papilionoid lineage and evolution of the symbiotic infection process. The tree is a simplified representation with triangle representing the major clades and the two subclades of the Galegoids: the Robinoids and the plastid DNA inverted repeat-lacking clade (IRLC). Known divergence times are indicated in MYA (million years ago) for some nodes (circles) and notable species are listed for each clade. Symbiotic infection processes encountered in the different clades are reported, excepted for the basal clades for which occurrence of nodulation is restricted to few understudied species. Note that Aeschynomene is the only legume genus where a Nod-independent symbiotic infection process (*NI*) has been reported for several species, including A. evenia . Phylogeny and drawings are adaptations, date estimates and nodulation properties come from previous publications. 2-6" path-from-xml="dsw020f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198506/dsw020f1p.png" path-from-xml="dsw020f1p.png" target="_blank">View large</a><a section="87198506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f1p.png?Expires=2147483647&Signature=rM3tIQt1EaCebOb5IikFbeir0Vf4m4yPA3TkYDYn2NO1mKw3IwnQsER3tMEPEoGR4KmmrdRVe-75CYBDgYwwDzA4ZZw3UPLPbS~Qr3JVKkSjVE05SYMtD5ucCkeVSlhVQXVIhY7EY47TCoLoi9hxtViWxZ3UhS9gT1kCAkmp1CcaqwnhMaxcLDNnD3KrqIvBxmNAwrtO~ystOPVpyOj6obGOd1b7dQggB3AGE02E53SUb70mSYdHLHOFI~X0jYAdnh1PWGWbA5RAO8Wm9yoarrx1wEeQpb7qzBge-5pOstM4j-ZGgvbcGaXCFix6hu8-QXTiw4FcEiYdnJkI7W7yEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198506&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Phylogeny of the Papilionoid lineage and evolution of the symbiotic infection process. The tree is a simplified representation with triangle representing the major clades and the two subclades of the Galegoids: the Robinoids and the plastid DNA inverted repeat-lacking clade (IRLC). Known divergence times are indicated in MYA (million years ago) for some nodes (circles) and notable species are listed for each clade. Symbiotic infection processes encountered in the different clades are reported, excepted for the basal clades for which occurrence of nodulation is restricted to few understudied species. Note that <em>Aeschynomene</em> is the only legume genus where a Nod-independent symbiotic infection process (*NI*) has been reported for several species, including <em>A. evenia</em> . Phylogeny and drawings are adaptations, date estimates and nodulation properties come from previous publications. <a reveal-id="dsw020-B2" data-open="dsw020-B2" class="link link-ref link-reveal xref-bibr"><sup>2-</sup></a><sup>,</sup><a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p></div></div><p> To advance in the understanding of how legumes have evolved to nitrogen-fixing symbiosis with rhizobia, two model species belonging to the Galegoid clade and with favorable genetic attributes, <em>Medicago truncatula</em> and <em>Lotus japonicus</em> , have been extensively studied. This study has revealed a sophisticated symbiotic process that is triggered by the recognition of rhizobial signal molecules called Nod factors (NF) and that involves both the formation of an infection thread guiding the bacteria inside the root and a distant induction of a nodule primordium. Furthermore, the genetic dissection of the nodulation process has allowed the identification and elucidation of the role of many genes that are essential for the different steps of nodule development, some of them being also involved in mycorrhizal symbiosis. <a reveal-id="dsw020-B8" data-open="dsw020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Such mechanisms are likely to be highly conserved in the Phaseolid and Galegoid clades that share similar symbiotic infection and nodule organogenesis processes ( <a reveal-id="dsw020-F1" data-open="dsw020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). In addition, since most of the legumes used as crops are members of the Phaseoloids and Galegoids, genetic and genomic studies, including genome sequencing and comparative genomics, have greatly advanced in these two clades. <a reveal-id="dsw020-B9 dsw020-B10 dsw020-B11 dsw020-B12 dsw020-B13 dsw020-B14 dsw020-B15 dsw020-B16 dsw020-B17" data-open="dsw020-B9 dsw020-B10 dsw020-B11 dsw020-B12 dsw020-B13 dsw020-B14 dsw020-B15 dsw020-B16 dsw020-B17" class="link link-ref link-reveal xref-bibr"><sup>9–17</sup></a></p><p> Conversely, the Genistoid and the Dalbergioid clades, which are more basal in their divergence within the Papilionoideae, have lagged far behind, even though they contain important crop legumes such as the genistoid lupine ( <em>Lupinus angustifolius</em> ) and the dalbergioid peanut ( <em>Arachis hypogaea</em> ). This is partly due to the large and complex nature of their genomes that makes their study challenging. Hence, in the case of peanut, which is a recent allotetraploid species, the choice was made to sequence first the genomes of its two diploid progenitors, <em>Arachis duranensis</em> and <em>A. ipaensis</em> . <a reveal-id="dsw020-B18" data-open="dsw020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Similarly, genistoid and dalbergioid legumes have been understudied for their symbiotic properties although they display a distinct infection process that is initiated in an intercellular fashion and by the formation of nodules that originate directly from dividing infected cells ( <a reveal-id="dsw020-F1" data-open="dsw020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). <a reveal-id="dsw020-B2" data-open="dsw020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> But most surprisingly, in some <em>Aeschynomene</em> species, which are phylogenetically related to <em>Arachis</em> , an unconventional symbiotic process has also been described where some <em>Bradyrhizobium</em> strains are able to form nodules in the absence of NF synthesis ( <a reveal-id="dsw020-F1" data-open="dsw020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). <a reveal-id="dsw020-B19" data-open="dsw020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Unravelling the molecular mechanisms of this NF-independent process would bring important insights on the evolution of rhizobium-legume symbiosis. <a reveal-id="dsw020-B20" data-open="dsw020-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> For this purpose, <em>Aeschynomene evenia</em> was proposed as a new model legume because of its advantageous genetic and developmental characteristics for molecular genetics. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p><p> To provide insights into the symbiotic gene content and on the structure of the NF-independent <em>A. evenia</em> genome, we undertook the development of a gene-based genetic map. RNAseq data obtained for each parental line were mined for symbiotic gene discovery and molecular marker development. These markers were used to genotype a F <sub>2</sub> mapping population and construct a high-density genetic map that was subjected to comparative analysis with the <em>Arachis duranensis</em> and <em>A. ipaensis</em> genomes. </p>                            <h2 scrollto-destination=87198510 id="87198510" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=87198511 id="87198511" class="section-title" >2.1. Plant material, culture, crossing and observation</h3>
<p> Seed germination, plant culture and hybridizations were performed as indicated. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> To develop the mapping population, individual seeds of the <em>A. evenia</em> accessions CIAT8232 (Brazil) and CIAT22838 (Malawi) obtained from CIAT (Colombia) were selfed three times to generate inbred lines and then crossed manually. The hybrid nature of the F <sub>1</sub> plants was confirmed with molecular markers as described. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Two F <sub>1</sub> plants obtained by bi-directional hybridizations were selfed to develop the F <sub>2</sub> mapping population in greenhouse. For pollen viability analysis, buds were fixed in Carnoy’s fixative prior to anthesis, when pollen was mature but anthers non-dehiscent, and then stained using a simplified method of the Alexander’s stain as detailed. <a reveal-id="dsw020-B3" data-open="dsw020-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Pollen viability was scored under the light microscope for at least three flowers per plant by counting aborted pollen grains, which stained pale turquoise blue and nonaborted pollen grains, which stained dark blue or purple. </p>                            <h3 scrollto-destination=87198513 id="87198513" class="section-title" >2.2 DNA and RNA isolation</h3>
<p> Genomic DNA was extracted from young leaves using the CTAB method with the addition of β-mercaptoethanol 2% and PVPP 2% to the CTAB solution in order to limit polysaccharides and polyphenols co-extraction. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> DNA quality and quantity were evaluated in 1% agarose gel electrophoresis and by spectrophotometer before DNA normalization to a concentration of 10 ng/µl. </p><p> For RNA extractions, tissue samples were collected from <em>in vitro</em> cultured plants: roots and leaves on 7-days old un-inoculated plants, nodules at 4, 7 and 14-days after inoculation with the <em>Bradyrhizobium</em> strain ORS278. Total RNA was extracted using the SV total RNA Isolation System (Promega) excepted for leaves for which RNA was extracted using a CTAB protocol. <a reveal-id="dsw020-B23" data-open="dsw020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> RNA was quantified using a NanoDrop ND-1000 spectrophotometer and its quality verified using a 2100 Bioanalyzer RNA Nanochip (Agilent, Santa Clara, CA, USA). For each parental line, a total of 12 µg of RNA was pooled equally from the five tissues for Illumina library construction. </p>                            <h3 scrollto-destination=87198516 id="87198516" class="section-title" >2.3 Development of Illumina transcriptomes</h3>
<p> Two mRNA libraries were built and sequenced in one lane as described. <a reveal-id="dsw020-B24" data-open="dsw020-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The Illumina paired-end sequencing technology generated 2× 100 bp independent reads from the 200 bp insert libraries. RNA-seq data were processed for the <em>de novo</em> transcriptome assemblies using the Velvet 1.2.07 and CAP3 08/06/13 softwares and orthologous relationships between contigs of the two transcriptome assemblies were identified by best reciprocal hit search. To evaluate the quality of the assembly, all usable reads were realigned to the contigs using BWA-MEM 0.7.12-r1039 and SAMtools idxstats 1.1. To assess the depth of gene coverage through comparative genome analysis, ORFs were searched in <em>A. evenia</em> transcripts with TransDcoder_r20140704 and used for BLASTX alignment ( <em>E</em> value &lt; 10 <sup>−</sup><sup>20</sup> ) with Blastall 2.2.26 onto the protein database of <em>A. duranensis</em> (Araip.V14167.a1.M1.peptide.fa file downloaded from <a class="link link-uri" href="http://www.peanut.org" target="_blank">www.peanut.org</a> ). Raw Illumina sequence data have been deposited in the NCBI Sequence Read Archive (SRA) under the accession numbers SRR3276128 and SRR3285082, for the CIAT8232 and CIAT22838 lines respectively. </p>                            <h3 scrollto-destination=87198518 id="87198518" class="section-title" >2.4 Sequence search and comparison</h3>
<p> Symbiotic gene sequences were obtained from <em>M. truncatula</em> and <em>L. japonicus</em> by searching in the NCBI database. These sequences were used to find orthologous unigenes in an <em>A. evenia</em> local database developed with a Galaxy work bench (INRA, Toulouse, France) ( <a class="link link-uri" href="https://bbric-pipelines.toulouse.inra.fr/galaxy/" target="_blank">https://bbric-pipelines.toulouse.inra.fr/galaxy/</a> ) and containing the CIAT8232 and CIAT22838 transcriptome data ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1, S3</a></span> ; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> ). </p><p> For estimation of divergence time, single-copy genes identified in the CIAT8232 and CIAT22838 transcriptomes were used to obtain orthologous sequences for <em>A. evenia</em> IRLFL6945 from its transcriptome database ( <a class="link link-uri" href="http://esttik.cirad.fr" target="_blank">http://esttik.cirad.fr</a> ) and for <em>A. duranensis</em> and <em>A. ipaensis</em> from their genome sequence database ( <a class="link link-uri" href="http://peanutbase.org/" target="_blank">http://peanutbase.org/</a> ) ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> ). Coding sequences were cropped, aligned and used to calculate Ks values with the DnaSP 5.10 software ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). Divergence times were obtained based on a simple molecular clock assumption and by using the previously known ∼49 MYA divergence time estimate between <em>Aeschynomene</em> and <em>Arachis</em> . <a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>                            <h3 scrollto-destination=87198521 id="87198521" class="section-title" >2.5 SSR and INDEL genotyping</h3>
<p> Detection of microsatellites in the transcriptome data were performed using the SPUTNIK software by searching for di- to hexanucleotide SSRs with a minimum of three to six repetitions depending on the motifs. Orthologous sequences in the two transcriptome assemblies comprising different lengths of the same SSR motif, with a contig length over 500 nt and with at least 50 nt surrounding the SSR motifs were selected as <em>in silico</em> polymorphic SSRs ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). For specific gene mapping, SSR and INDELs were searched by sequence comparison from the two transcriptomes. Primer pairs were designed using Primer3 with tree parameters: (i) a primer length of 20 bp with an annealing temperature of 55 °C, (ii) three ranges of PCR product size of 90–120, 150–180 and 220–250 bp, (iii) the addition of a 5′-end M13 tail (5′-CACGACGTTGTAAAACGAC-3′) on the forward primer ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> ). SSR markers were checked for PCR-amplification and co-dominant behavior, organized in 12-SSR multiplexes by combining both the three ranges of fragment sizes and the four fluorochromes (FAM, VIC, NED and PET), and then used to genotype the F <sub>2</sub> mapping population as described. <a reveal-id="dsw020-B25" data-open="dsw020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>                            <h3 scrollto-destination=87198523 id="87198523" class="section-title" >2.6 SNP genotyping</h3>
<p> To detect single nucleotide polymorphisms for a set of target genes, sequences of orthologous contigs identified in the two transcriptome datasets were aligned with the MUSCLE program ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> ). Positions of the SNPs were determined by examining the intron/exon structure of orthologous genes in <em>Arachis</em> ( <a class="link link-uri" href="http://peanutbase.org/" target="_blank">http://peanutbase.org/</a> ). Whenever necessary, additional SNPs were searched by intron sequencing. Up to three SNPs were selected for each target gene and flanking sequences 50 bp both upstream and downstream each target SNP were sent to KBioscience for primer design ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> ). Genotyping of the F <sub>2</sub> mapping population was performed with the KASP genotyping chemistry (KBioscience Ltd., Hoddesdon, UK) on Fluidigm 96.96 Dynamic Arrays using the BioMark HD system (Fluidigm Corp, San Francisco, CA). On the basis of obtained fluorescence, allele call data were viewed graphically as a scatter plot for each marker assayed by using Fluidigm SNP Genotyping Analysis software and checked manually. </p>                            <h3 scrollto-destination=87198525 id="87198525" class="section-title" >2.7 Genetic map construction</h3>
<p> Linkage analysis was performed on segregated genotypic data from the F <sub>2</sub> mapping population using JoinMap® version 4 (Plant Research International BV, Wageningen, Netherlands). The marker loci were attributed to linkage groups using the JoinMap® grouping module with a logarithm of odds (LOD) scores of 6.0. Marker order and genetic distance (in cM) were calculated using a regression mapping algorithm with the following parameters: Kosambi’s mapping function, recombination frequency ≤0.40, and LOD threshold of 1.0 ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> ). The segregation data were tested for deviation from the expected Mendelian ratio using a Chi-square test. LG numbering and orientation were attributed to best fit the <em>A. duranensis and A. ipaensis</em> chromosomes. </p>                            <h3 scrollto-destination=87198527 id="87198527" class="section-title" >2.8 Comparative genomics</h3>
<p> Gene sequences underlying the molecular marker loci assigned to the linkage map were blasted against the <em>A. duranensis</em> and <em>A. ipaensis</em> genome sequences ( <a class="link link-uri" href="http://peanutbase.org/" target="_blank">http://peanutbase.org/</a> ) to detect significant matches at minimum threshold <em>E</em> values &lt; 10 <sup>−</sup><sup>60</sup> ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> ). For the purpose of graphic preparation, cM distance on the <em>A. evenia</em> linkage groups were scaled by factors calculated on the basis of the genome size of <em>A. ipaensis</em> , to provide matching chromosomal lengths in base pairs. The graphical comparative maps were drawn using the Circos program ( <a class="link link-uri" href="http://circos.ca/" target="_blank">http://circos.ca/</a> ) and the SpiderMap software (JF Rami, unpublished). </p>                            <h2 scrollto-destination=87198529 id="87198529" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=87198530 id="87198530" class="section-title" >3.1 Selection of parental genotypes to develop a mapping population</h3>
                            <h4 scrollto-destination=87198531 id="87198531" class="section-title" >3.1.1 Choice of the reference line</h4>
<p> The <em>A. evenia</em> species was proposed as a new model legume to investigate the evolution of the nitrogen-fixing symbiosis because it possesses a number of interesting characteristics for both classical and molecular genetics. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Key attributes of <em>A. evenia</em> include diploidy (2 <em>n</em>  = 2× = 20), self-pollination, a small genome for a legume, a short growth cycle and a high level of diversity. In addition it is efficiently nodulated by the well-characterized strains <em>Bradyrhizobium</em> ORS278 and ORS285 and it can be transformed using the hairy root system. First nodulation studies were performed on the cultivar IRFL6945 belonging to the <em>A. evenia</em> ssp. <em>serrulata</em> . <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> But subsequent analysis of the intra-specific diversity revealed that <em>A. evenia</em> ssp. <em>evenia</em> is more appropriate due to a smaller genome size (415 vs 465 Mb) and the presence of genotypes with a more rapid reproductive cycle and a non-branching habit that facilitate plant management ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig. 2A,B</a> ). <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> We made use of this knowledge to select from within the subspecies <em>evenia</em> var. <em>evenia</em> the genotype Mbao CIAT22838, which displays these advantageous characteristics, as the reference genotype. Its nodulation properties were previously shown to be representative of the species with ∼11nodules/plant, and its ability to be transformed by <em>Agrobacterium rhizogenes</em> was found to be similar to what was described for the line IRLF6945 with a 60% root transformation rate and subsequent 90% nodulation rate upon inoculation with <em>Bradyrhizobium</em> ORS278. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Even though <em>A. evenia</em> is a self-pollinated plant and was previously shown to be predominantly homozygous, a single seed from the reference genotype was selfed three times to ensure high homozygosity. </p>                        <a id="87198533" scrollto-destination="87198533"></a>
<div id="dsw020-F2" data-id="dsw020-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f2p.png?Expires=2147483647&Signature=2QTlE2RGZATT-CpHYNiOkudHXFHj9PoANutSuh8Z0HI24E36Pnbx8x-UHt9ygLUcvI5iaQkBE2yHutzjQxQsENRiKXcb0o63QYvQBlsNpU-MQC7~ryL66bL6qhjqGVtgqaovoHM3rhMAZcoTgLq9yVx1DCSPmb-p1Wo2OOuTRj8uzJgK~4QaRoJxo41z012HcNgv1hMKTL7nR0~75L8-bHADDYtoFnRPCFWtr~uWmBYvDoz9~UQCTLRQ8bp4J-KEyKg5wOUeafLSq6~az8zXY-1tOkzH2KfchFJ-Q9eX2nbnoMzrnFzt2MYj1jO5IFtlT0Sr4qd7YkQZgLaOAvwiQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Differentiation within the A. evenia species and its use for genomic and genetic studies. (A) Schematic phylogeny depicting the genetic differentiation within the A. evenia species and the relationships with Arachis sp. 18,22 Dash lines connect the tetraploid A. hypogaea to its most probable diploid genome donors A. duranensis and A. ipaensis . Divergence times at certain nodes (circles) were estimated using a relaxed molecular clock ( Table 3 ). (B) Phenotypic patterns which distinguish CIAT22838 and CIAT8232 lines and impact on the F1 hybrid obtained by cross-pollination: plant habit (ramified or not), time of flower opening (early and late), flower shapes (standard shape and dark throat indicated by an arrow), pod production. Bars = 5 mm. " path-from-xml="dsw020f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198533/dsw020f2p.png" path-from-xml="dsw020f2p.png" target="_blank">View large</a><a section="87198533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f2p.png?Expires=2147483647&Signature=uc69ao2DeD5sWAH04C373VAEYJtYg-QiXJ-gJ7RG3RLn7VwMkjyjHHAog7bGbOVAtM1AESt9~r51A9mQEAYU~9bTXRx30V8Tz9H52FMcVvyJguwueR63fjZS9QpN3lh4Pj7naEF7X9P9qEU2qZhmLBKQGMSbHrsClgU~ahKoTFMvHToKAUBwTWoIkMtVLfutAH-yxGSKbeLZ9QlUGYH0~vOu0~Z18cE4F6fGrqQttGELoJf5o~88gvvv-WL2XI7N8XowZWbUZAJkbYKnJRR-phKKzBygPrfnKys1-mzEZXhV4anRh5jzPlmpWmlEaRL8IdcG1c4Sgt6EWO7SNIA95w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198533&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Differentiation within the <em>A. evenia</em> species and its use for genomic and genetic studies. (A) Schematic phylogeny depicting the genetic differentiation within the <em>A. evenia</em> species and the relationships with <em>Arachis</em> sp. <a reveal-id="dsw020-B18" data-open="dsw020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Dash lines connect the tetraploid <em>A. hypogaea</em> to its most probable diploid genome donors <em>A. duranensis</em> and <em>A. ipaensis</em> . Divergence times at certain nodes (circles) were estimated using a relaxed molecular clock ( <a reveal-id="dsw020-T3" data-open="dsw020-T3" class="link link-reveal link-table xref-fig">Table 3</a> ). (B) Phenotypic patterns which distinguish CIAT22838 and CIAT8232 lines and impact on the F1 hybrid obtained by cross-pollination: plant habit (ramified or not), time of flower opening (early and late), flower shapes (standard shape and dark throat indicated by an arrow), pod production. Bars = 5 mm. </p></div></div><div content-id="dsw020-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f2p.png?Expires=2147483647&Signature=2QTlE2RGZATT-CpHYNiOkudHXFHj9PoANutSuh8Z0HI24E36Pnbx8x-UHt9ygLUcvI5iaQkBE2yHutzjQxQsENRiKXcb0o63QYvQBlsNpU-MQC7~ryL66bL6qhjqGVtgqaovoHM3rhMAZcoTgLq9yVx1DCSPmb-p1Wo2OOuTRj8uzJgK~4QaRoJxo41z012HcNgv1hMKTL7nR0~75L8-bHADDYtoFnRPCFWtr~uWmBYvDoz9~UQCTLRQ8bp4J-KEyKg5wOUeafLSq6~az8zXY-1tOkzH2KfchFJ-Q9eX2nbnoMzrnFzt2MYj1jO5IFtlT0Sr4qd7YkQZgLaOAvwiQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Differentiation within the A. evenia species and its use for genomic and genetic studies. (A) Schematic phylogeny depicting the genetic differentiation within the A. evenia species and the relationships with Arachis sp. 18,22 Dash lines connect the tetraploid A. hypogaea to its most probable diploid genome donors A. duranensis and A. ipaensis . Divergence times at certain nodes (circles) were estimated using a relaxed molecular clock ( Table 3 ). (B) Phenotypic patterns which distinguish CIAT22838 and CIAT8232 lines and impact on the F1 hybrid obtained by cross-pollination: plant habit (ramified or not), time of flower opening (early and late), flower shapes (standard shape and dark throat indicated by an arrow), pod production. Bars = 5 mm. " path-from-xml="dsw020f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198533/dsw020f2p.png" path-from-xml="dsw020f2p.png" target="_blank">View large</a><a section="87198533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f2p.png?Expires=2147483647&Signature=uc69ao2DeD5sWAH04C373VAEYJtYg-QiXJ-gJ7RG3RLn7VwMkjyjHHAog7bGbOVAtM1AESt9~r51A9mQEAYU~9bTXRx30V8Tz9H52FMcVvyJguwueR63fjZS9QpN3lh4Pj7naEF7X9P9qEU2qZhmLBKQGMSbHrsClgU~ahKoTFMvHToKAUBwTWoIkMtVLfutAH-yxGSKbeLZ9QlUGYH0~vOu0~Z18cE4F6fGrqQttGELoJf5o~88gvvv-WL2XI7N8XowZWbUZAJkbYKnJRR-phKKzBygPrfnKys1-mzEZXhV4anRh5jzPlmpWmlEaRL8IdcG1c4Sgt6EWO7SNIA95w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198533&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Differentiation within the <em>A. evenia</em> species and its use for genomic and genetic studies. (A) Schematic phylogeny depicting the genetic differentiation within the <em>A. evenia</em> species and the relationships with <em>Arachis</em> sp. <a reveal-id="dsw020-B18" data-open="dsw020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Dash lines connect the tetraploid <em>A. hypogaea</em> to its most probable diploid genome donors <em>A. duranensis</em> and <em>A. ipaensis</em> . Divergence times at certain nodes (circles) were estimated using a relaxed molecular clock ( <a reveal-id="dsw020-T3" data-open="dsw020-T3" class="link link-reveal link-table xref-fig">Table 3</a> ). (B) Phenotypic patterns which distinguish CIAT22838 and CIAT8232 lines and impact on the F1 hybrid obtained by cross-pollination: plant habit (ramified or not), time of flower opening (early and late), flower shapes (standard shape and dark throat indicated by an arrow), pod production. Bars = 5 mm. </p></div></div>                            <h4 scrollto-destination=87198534 id="87198534" class="section-title" >3.1.2 Choice of a mapping parent</h4>
<p> The establishment of a mapping population required the identification of a second parent that has favorable polymorphism while the resulting F <sub>2</sub> population should not have developmental and genetic abnormalities. Characterization of the diversity in <em>A. evenia</em> had previously revealed that the genetic polymorphism was the highest between the subspecies <em>evenia</em> and <em>serrulata</em> but that inter-subspecies crosses led to sterile F <sub>1</sub> hybrids. Conversely, intra-subspecific hybridizations were shown to generate fully fertile F <sub>1</sub> progenies. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Therefore, to maximize genetic polymorphism, the genotype Bahia CIAT8232 that belongs to the subspecies <em>evenia</em> var. <em>pauciciliata</em> and that presents a distinct geographic origin (African for Mbao versus American for Bahia) was developed as an inbred line and used as the second mapping parent ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ). The Mbao and Bahia genotypes present a similar DNA content (415 and 405 Mb, respectively) but they were shown to display a 32% SSR polymorphism. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> This situation is similar to the one observed in <em>Medicago truncatula</em> with the two mapping parents Jemalong and DZA315.16 and between three ecotypes of <em>Arabidopsis thaliana</em> , indicating it is convenient for generating genetic maps. <a reveal-id="dsw020-B26" data-open="dsw020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> In addition, the two genotypes display phenotypic differences making them readily distinguishable such as the branching habit, the timing of flower openings and the presence of a dark throat on the upper petal ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> ). </p>                            <h4 scrollto-destination=87198536 id="87198536" class="section-title" >3.1.3 Development of a F2 mapping population</h4>
<p> Artificial hybridizations were performed using the genotype Mbao CIAT22838 as female and male alternatively and producing F <sub>1A</sub> and F <sub>1B</sub> progenies respectively. These F <sub>1</sub> plants showed hybrid vigor, intermediate flower characteristics (opening time and shape) and were self-fertile ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> ). In particular, they displayed equivalent pod set (Bahia: 8.6 ± 0.9, Mbao: 8.2 ± 1, F <sub>1</sub> : 8.8 ± 1.1 seeds/pod). The resulting F <sub>2A</sub> and F <sub>2B</sub> offsprings had no albino or dwarf individuals. Altogether, this suggested that Bahia CIAT8232 was a suitable crossing partner to Mbao CIAT22838 and that the obtained F <sub>2</sub> offsprings could be used as mapping population to develop a genetic map. </p>                            <h3 scrollto-destination=87198539 id="87198539" class="section-title" >3.2 Transcriptome sequencing for identification of symbiotic genes and genetic markers</h3>
                            <h4 scrollto-destination=87198541 id="87198541" class="section-title" >3.2.1 Illumina paired-end sequencing and <em>de novo</em> transcriptome assembly</h4>
<p> High-throughput transcriptomic sequencing allows generating a large transcript sequence dataset for gene discovery and development of molecular markers that serve for gene-based map construction. To be able to achieve a broad survey of genes associated with symbiosis, RNA was extracted and pooled from various organs: roots, leaves and nodules at 4, 7 and 14-dpi. In addition, to increase the efficiency of molecular marker development by prior <em>in silico</em> polymorphic analysis, a RNA library was developed for each parental genotype. </p><p> Using Illumina paired-end sequencing and after stringent quality check of the raw data, ∼72 million reads were kept for the Mbao genotype and ∼66 million for the Bahia genotype. Based on these high-quality reads, a total of 51,763 and 52,648 contigs were assembled respectively, with a common N50 of 1,800 bp and a consistent contig size distribution ( <a reveal-id="dsw020-T1" data-open="dsw020-T1" class="link link-reveal link-table xref-fig">Table 1</a> ). This indicated that a similar transcript content was obtained for both mapping parents. Furthermore, realignment of all the usable reads to the contigs revealed that ∼35,000 contigs were remapped by more than 100 reads and, among them, ∼24,500 contigs were realigned with more than 500 reads, suggesting the contigs were well overlapped by the sequencing reads ( <a reveal-id="dsw020-F3" data-open="dsw020-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> ). <em>A. evenia</em> being a member of the Dalbergioid clade along with the recently sequenced <em>A. duranensis</em> and <em>A. ipaensis</em> , their genomic data could be also used to assess the quality and coverage of the assembled transcriptomes. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Since <em>A. duranensis</em> has not experienced the local gene duplications evidenced for <em>A. ipaensis</em> , the former species was better suited for this purpose. Thus, comparative analysis revealed that 60% of total contigs (∼31,850) and 83% of contigs over 500 bp in length (∼26,378) had BLAST matches ( <em>E</em> -value &lt; 10 <sup>−</sup><sup>5</sup> ) onto the ∼36,700 predicted genes in the <em>A. duranensis</em> genome ( <a reveal-id="dsw020-F3" data-open="dsw020-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> ). In addition, plotting the contigs onto top-hit homologous coding sequences of <em>A. duranensis</em> ( <em>E</em> -value &lt; 10 <sup>−</sup><sup>30</sup> ) indicated a high proportion of genes with complete coding sequence ( <a reveal-id="dsw020-F3" data-open="dsw020-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> ). Altogether, these data suggested that a comprehensive gene capture was achieved for <em>A. evenia</em> . </p>                        <a id="87198547" scrollto-destination="87198547"></a>
<div id="dsw020-F3" data-id="dsw020-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f3p.png?Expires=2147483647&Signature=3dD-btyfZ58iW-AwvjBkDToEx6N2UgJHpszM9V37HTyvlPOrJSSGef~2jpSqdjysagAklM5kRyM9TtbEDopOwsHaJimoJ-sTae4CCCGwN164o67-2mwde~cmqXy5bWv7CgTn5XXkhMUjTfSk9RQ0U3h7a1~2Wy974IemFmJs0h88GcpxiW6T0TBb1ZOdGlG-iD-gIyaRAgG2rqGTB3O6aypfU8h0MZVX5leCkQyb~PuxQ7d~9YZfSDS72x37PI7iXOf7STyioOpTKh3fWPqYRUSJFjZLscm1D-XgMlVtdYZ7aMD-4cSCc-p4GZ1tClWW6oMH-Tdin8Dlu1nxG-tHVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Assessment of assembly quality and gene coverage for the Illumina transcriptomes. (A) Distribution of mapped reads within the assembled contigs for the Illumina transcriptomes of the two parental genotypes Bahia and Mbao. B, Comparison of contig length between hit and no-hit contigs from the Mbao transcriptome assembly after tBLASTX analysis with the protein database of A. ipaensis . C, Comparison of Mbao contigs to putative orthologous A. ipaensis sequences by evaluating the % of alignment of corresponding coding sequences. " path-from-xml="dsw020f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198547/dsw020f3p.png" path-from-xml="dsw020f3p.png" target="_blank">View large</a><a section="87198547" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f3p.png?Expires=2147483647&Signature=KAINHQXDV38Xe-3irbS92a-4kJsJytB4qiujqOTHC1iEA6~C0KbjzhgiiweZ3N4FzYJo6W6bhr5caGcaRgj-VPK97Yn3NfMMYTlhoZfy3ili9Ib4yl5WU1XLHK5U7G2MHOg4aMIbpsWS~xD4XR1qDVITU1SZHOUIvqm2hLu6XOVv9lPJs4NgGSZmGou8-G3J9EiqpN0Y1ZwRK-4DLMtauPrrmDqxTxXR-4O3KgU7eFyHOvbY~B4NUkj-FQHzusnzJMXJuY4oHsaUcmBmooBYU4YW2dTt7vOycSyxVXQSR77Jp8JIZLaeXjqB7g9Zch-CQfdo~Q5c6A7TShvAf6i8iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198547&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Assessment of assembly quality and gene coverage for the Illumina transcriptomes. (A) Distribution of mapped reads within the assembled contigs for the Illumina transcriptomes of the two parental genotypes Bahia and Mbao. B, Comparison of contig length between hit and no-hit contigs from the Mbao transcriptome assembly after tBLASTX analysis with the protein database of <em>A. ipaensis</em> . C, Comparison of Mbao contigs to putative orthologous <em>A. ipaensis</em> sequences by evaluating the % of alignment of corresponding coding sequences. </p></div></div><div content-id="dsw020-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f3p.png?Expires=2147483647&Signature=3dD-btyfZ58iW-AwvjBkDToEx6N2UgJHpszM9V37HTyvlPOrJSSGef~2jpSqdjysagAklM5kRyM9TtbEDopOwsHaJimoJ-sTae4CCCGwN164o67-2mwde~cmqXy5bWv7CgTn5XXkhMUjTfSk9RQ0U3h7a1~2Wy974IemFmJs0h88GcpxiW6T0TBb1ZOdGlG-iD-gIyaRAgG2rqGTB3O6aypfU8h0MZVX5leCkQyb~PuxQ7d~9YZfSDS72x37PI7iXOf7STyioOpTKh3fWPqYRUSJFjZLscm1D-XgMlVtdYZ7aMD-4cSCc-p4GZ1tClWW6oMH-Tdin8Dlu1nxG-tHVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Assessment of assembly quality and gene coverage for the Illumina transcriptomes. (A) Distribution of mapped reads within the assembled contigs for the Illumina transcriptomes of the two parental genotypes Bahia and Mbao. B, Comparison of contig length between hit and no-hit contigs from the Mbao transcriptome assembly after tBLASTX analysis with the protein database of A. ipaensis . C, Comparison of Mbao contigs to putative orthologous A. ipaensis sequences by evaluating the % of alignment of corresponding coding sequences. " path-from-xml="dsw020f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198547/dsw020f3p.png" path-from-xml="dsw020f3p.png" target="_blank">View large</a><a section="87198547" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f3p.png?Expires=2147483647&Signature=KAINHQXDV38Xe-3irbS92a-4kJsJytB4qiujqOTHC1iEA6~C0KbjzhgiiweZ3N4FzYJo6W6bhr5caGcaRgj-VPK97Yn3NfMMYTlhoZfy3ili9Ib4yl5WU1XLHK5U7G2MHOg4aMIbpsWS~xD4XR1qDVITU1SZHOUIvqm2hLu6XOVv9lPJs4NgGSZmGou8-G3J9EiqpN0Y1ZwRK-4DLMtauPrrmDqxTxXR-4O3KgU7eFyHOvbY~B4NUkj-FQHzusnzJMXJuY4oHsaUcmBmooBYU4YW2dTt7vOycSyxVXQSR77Jp8JIZLaeXjqB7g9Zch-CQfdo~Q5c6A7TShvAf6i8iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198547&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Assessment of assembly quality and gene coverage for the Illumina transcriptomes. (A) Distribution of mapped reads within the assembled contigs for the Illumina transcriptomes of the two parental genotypes Bahia and Mbao. B, Comparison of contig length between hit and no-hit contigs from the Mbao transcriptome assembly after tBLASTX analysis with the protein database of <em>A. ipaensis</em> . C, Comparison of Mbao contigs to putative orthologous <em>A. ipaensis</em> sequences by evaluating the % of alignment of corresponding coding sequences. </p></div></div>                        <a id="87198551" scrollto-destination="87198551"></a>
<div content-id="dsw020-T1" class="table-modal"><div class="table-wrap"><div id="dsw020-T1" class="table-wrap-title" data-id="dsw020-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of the sequencing and assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Nucleotide length (pb) </th><th colspan="2"><em>A. evenia</em> Illumina libraries <hr /> </th></tr><tr><th>Bahia CIAT8232 </th><th>Mbao CIAT22838 </th></tr></thead><tbody><tr><td>150–500 </td><td char=".">21,020 </td><td char=".">20,269 </td></tr><tr><td>501–1,000 </td><td char=".">10,109 </td><td char=".">9,990 </td></tr><tr><td>1,001–2,000 </td><td char=".">13,127 </td><td char=".">13,216 </td></tr><tr><td>2,001–3,000 </td><td char=".">5,673 </td><td char=".">5,561 </td></tr><tr><td>3,001–4,000 </td><td char=".">1,767 </td><td char=".">1,769 </td></tr><tr><td>4,001–5,000 </td><td char=".">570 </td><td char=".">578 </td></tr><tr><td>&gt;5,000 </td><td char=".">382 </td><td char=".">379 </td></tr><tr><td>Raw reads </td><td char=".">68,945,277 </td><td char=".">75,745,315 </td></tr><tr><td>Filtered reads </td><td char=".">65,865,857 </td><td char=".">72,387,963 </td></tr><tr><td>Number of contigs </td><td char=".">51,763 </td><td char=".">52,648 </td></tr><tr><td>min length (bp) </td><td char=".">150 </td><td char=".">150 </td></tr><tr><td>max length (bp) </td><td char=".">16,895 </td><td char=".">16,886 </td></tr><tr><td>N50 (bp) </td><td char=".">1,799 </td><td char=".">1,803 </td></tr><tr><td>Average length (bp) </td><td char=".">1,092 </td><td char=".">1,082 </td></tr><tr><td>Total nucleotide length (bp) </td><td char=".">56,526,002 </td><td char=".">57,007,824 </td></tr></tbody></table></div><div content-id="dsw020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Nucleotide length (pb) </th><th colspan="2"><em>A. evenia</em> Illumina libraries <hr /> </th></tr><tr><th>Bahia CIAT8232 </th><th>Mbao CIAT22838 </th></tr></thead><tbody><tr><td>150–500 </td><td char=".">21,020 </td><td char=".">20,269 </td></tr><tr><td>501–1,000 </td><td char=".">10,109 </td><td char=".">9,990 </td></tr><tr><td>1,001–2,000 </td><td char=".">13,127 </td><td char=".">13,216 </td></tr><tr><td>2,001–3,000 </td><td char=".">5,673 </td><td char=".">5,561 </td></tr><tr><td>3,001–4,000 </td><td char=".">1,767 </td><td char=".">1,769 </td></tr><tr><td>4,001–5,000 </td><td char=".">570 </td><td char=".">578 </td></tr><tr><td>&gt;5,000 </td><td char=".">382 </td><td char=".">379 </td></tr><tr><td>Raw reads </td><td char=".">68,945,277 </td><td char=".">75,745,315 </td></tr><tr><td>Filtered reads </td><td char=".">65,865,857 </td><td char=".">72,387,963 </td></tr><tr><td>Number of contigs </td><td char=".">51,763 </td><td char=".">52,648 </td></tr><tr><td>min length (bp) </td><td char=".">150 </td><td char=".">150 </td></tr><tr><td>max length (bp) </td><td char=".">16,895 </td><td char=".">16,886 </td></tr><tr><td>N50 (bp) </td><td char=".">1,799 </td><td char=".">1,803 </td></tr><tr><td>Average length (bp) </td><td char=".">1,092 </td><td char=".">1,082 </td></tr><tr><td>Total nucleotide length (bp) </td><td char=".">56,526,002 </td><td char=".">57,007,824 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198551" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw020-T1" class="table-wrap-title" data-id="dsw020-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of the sequencing and assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Nucleotide length (pb) </th><th colspan="2"><em>A. evenia</em> Illumina libraries <hr /> </th></tr><tr><th>Bahia CIAT8232 </th><th>Mbao CIAT22838 </th></tr></thead><tbody><tr><td>150–500 </td><td char=".">21,020 </td><td char=".">20,269 </td></tr><tr><td>501–1,000 </td><td char=".">10,109 </td><td char=".">9,990 </td></tr><tr><td>1,001–2,000 </td><td char=".">13,127 </td><td char=".">13,216 </td></tr><tr><td>2,001–3,000 </td><td char=".">5,673 </td><td char=".">5,561 </td></tr><tr><td>3,001–4,000 </td><td char=".">1,767 </td><td char=".">1,769 </td></tr><tr><td>4,001–5,000 </td><td char=".">570 </td><td char=".">578 </td></tr><tr><td>&gt;5,000 </td><td char=".">382 </td><td char=".">379 </td></tr><tr><td>Raw reads </td><td char=".">68,945,277 </td><td char=".">75,745,315 </td></tr><tr><td>Filtered reads </td><td char=".">65,865,857 </td><td char=".">72,387,963 </td></tr><tr><td>Number of contigs </td><td char=".">51,763 </td><td char=".">52,648 </td></tr><tr><td>min length (bp) </td><td char=".">150 </td><td char=".">150 </td></tr><tr><td>max length (bp) </td><td char=".">16,895 </td><td char=".">16,886 </td></tr><tr><td>N50 (bp) </td><td char=".">1,799 </td><td char=".">1,803 </td></tr><tr><td>Average length (bp) </td><td char=".">1,092 </td><td char=".">1,082 </td></tr><tr><td>Total nucleotide length (bp) </td><td char=".">56,526,002 </td><td char=".">57,007,824 </td></tr></tbody></table></div><div content-id="dsw020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Nucleotide length (pb) </th><th colspan="2"><em>A. evenia</em> Illumina libraries <hr /> </th></tr><tr><th>Bahia CIAT8232 </th><th>Mbao CIAT22838 </th></tr></thead><tbody><tr><td>150–500 </td><td char=".">21,020 </td><td char=".">20,269 </td></tr><tr><td>501–1,000 </td><td char=".">10,109 </td><td char=".">9,990 </td></tr><tr><td>1,001–2,000 </td><td char=".">13,127 </td><td char=".">13,216 </td></tr><tr><td>2,001–3,000 </td><td char=".">5,673 </td><td char=".">5,561 </td></tr><tr><td>3,001–4,000 </td><td char=".">1,767 </td><td char=".">1,769 </td></tr><tr><td>4,001–5,000 </td><td char=".">570 </td><td char=".">578 </td></tr><tr><td>&gt;5,000 </td><td char=".">382 </td><td char=".">379 </td></tr><tr><td>Raw reads </td><td char=".">68,945,277 </td><td char=".">75,745,315 </td></tr><tr><td>Filtered reads </td><td char=".">65,865,857 </td><td char=".">72,387,963 </td></tr><tr><td>Number of contigs </td><td char=".">51,763 </td><td char=".">52,648 </td></tr><tr><td>min length (bp) </td><td char=".">150 </td><td char=".">150 </td></tr><tr><td>max length (bp) </td><td char=".">16,895 </td><td char=".">16,886 </td></tr><tr><td>N50 (bp) </td><td char=".">1,799 </td><td char=".">1,803 </td></tr><tr><td>Average length (bp) </td><td char=".">1,092 </td><td char=".">1,082 </td></tr><tr><td>Total nucleotide length (bp) </td><td char=".">56,526,002 </td><td char=".">57,007,824 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198551" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=87198552 id="87198552" class="section-title" >3.2.2 Identification of putative orthologs of symbiotic genes</h4>
<p> To determine which symbiotic genes can be identified in <em>A. evenia</em> , a BLAST search was performed in the Bahia and Mbao transcriptomes using the sequence of symbiotic genes characterized in the two model legumes <em>M. truncatula</em> and <em>L. japonicus</em> . <a reveal-id="dsw020-B8" data-open="dsw020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw020-B27 dsw020-B28 dsw020-B29 dsw020-B30 dsw020-B31 dsw020-B32 dsw020-B33" data-open="dsw020-B27 dsw020-B28 dsw020-B29 dsw020-B30 dsw020-B31 dsw020-B32 dsw020-B33" class="link link-ref link-reveal xref-bibr"><sup>27–33</sup></a> This allowed the identification of a single sequence in <em>A. evenia</em> for 36 out of 47 symbiotic genes analysed, for which the putative orthology was checked by reciprocal BLAST and sequence alignment ( <a reveal-id="dsw020-F4" data-open="dsw020-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ; Files S1, S2). These genes completely covered the different steps of the nodulation process: Nod signaling, activation and regulation of transcription, nodule organogenesis and regulation of nodulation ( <a reveal-id="dsw020-F4" data-open="dsw020-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). This coverage suggested that these steps were conserved in <em>A. evenia</em> , but it could not be excluded that some of these genes were only involved in the arbuscular mycorrhizal symbiosis or in other biological processes. So far, only the <em>A. evenia SYMRK</em> , <em>CCaMK</em> , <em>CRE1</em> and <em>DNF1</em> orthologs were effectively demonstrated to play a role in nodulation. <a reveal-id="dsw020-B34" data-open="dsw020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsw020-B35" data-open="dsw020-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                        <a id="87198555" scrollto-destination="87198555"></a>
<div id="dsw020-F4" data-id="dsw020-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f4p.png?Expires=2147483647&Signature=cLs7hwgzf~yoSogqzv2uV5-xOsamvIt0sF7tr~2f2Rsx1TaV5T96B9OX-1pcanCZoS1fmbQTGpXfhVJTdOt7OfZIYd9I54JSg-2qiYBU3sqgsVugzMXsMRXK5UwwB9OLk2JyrVdpJYp-CRVf6vxSe4YfUE5Tz0HxLI6ewzur5PRiAdXAn45STUNTlP7Ut07Eal5dQuGBOLvKuP1x1p3vnNDNwx4mj588wwFauktOEVNbIoB9S4nLP-z8ZKyggTTqmk2pd~5LsG~qGN8xwc5~TMbkHC4IItPXVLVkOzEIYj9BhkxWYjblJShYVH9aup7WM9FmBEU8IzbFksdA2cgmtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Simplified model for the symbiotic signaling pathway in legumes. Symbiotic genes identified in M. truncatula and L. japonicus are listed and classified in main symbiotic functions. 8,27-33 Putative orthologs were searched in the Illumina transcriptome of the two mapping parents. Those that could not be identified are between parentheses. Genes highlighted in bold characters are those which were demonstrated to be involved in the Nod-independent nodulation process in A. evenia . 33,34" path-from-xml="dsw020f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198555/dsw020f4p.png" path-from-xml="dsw020f4p.png" target="_blank">View large</a><a section="87198555" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f4p.png?Expires=2147483647&Signature=SZnlGSKz8SdG2OEwkMUJF1hoTeyyrUHmBy9x3tXLPt60JQtIkm-Pl2aQgEF9Tt5rL1q6JxFKlNoOT4neWEbQrWlGFVbEzjnd4Tv125Y265iqjAuc22KXEaSDXI5S7ByA664RbT-LmppT8yHizXtHstyFz-IfcGR5oae--TFXBZ755G-WG9c8fKR4M~saLR99c29xM2Nf0N~2Rc9Ki-N81FmfhwVsY9fpHQ~SdcEJdWwKspD67vq~BVs18-Fp57kZqIXzKK8e3FYLw4uri6s0M-zq6LjCk6SeDGCTquyFGkPuUOio62KONIlNTYmrsY26v~qbYLoNNZ-qG5eHUn0GuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198555&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Simplified model for the symbiotic signaling pathway in legumes. Symbiotic genes identified in <em>M. truncatula</em> and <em>L. japonicus</em> are listed and classified in main symbiotic functions. <a reveal-id="dsw020-B8" data-open="dsw020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw020-B27" data-open="dsw020-B27" class="link link-ref link-reveal xref-bibr"><sup>27-33</sup></a> Putative orthologs were searched in the Illumina transcriptome of the two mapping parents. Those that could not be identified are between parentheses. Genes highlighted in bold characters are those which were demonstrated to be involved in the Nod-independent nodulation process in <em>A. evenia</em> . <a reveal-id="dsw020-B33" data-open="dsw020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw020-B34" data-open="dsw020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p></div></div><div content-id="dsw020-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f4p.png?Expires=2147483647&Signature=cLs7hwgzf~yoSogqzv2uV5-xOsamvIt0sF7tr~2f2Rsx1TaV5T96B9OX-1pcanCZoS1fmbQTGpXfhVJTdOt7OfZIYd9I54JSg-2qiYBU3sqgsVugzMXsMRXK5UwwB9OLk2JyrVdpJYp-CRVf6vxSe4YfUE5Tz0HxLI6ewzur5PRiAdXAn45STUNTlP7Ut07Eal5dQuGBOLvKuP1x1p3vnNDNwx4mj588wwFauktOEVNbIoB9S4nLP-z8ZKyggTTqmk2pd~5LsG~qGN8xwc5~TMbkHC4IItPXVLVkOzEIYj9BhkxWYjblJShYVH9aup7WM9FmBEU8IzbFksdA2cgmtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Simplified model for the symbiotic signaling pathway in legumes. Symbiotic genes identified in M. truncatula and L. japonicus are listed and classified in main symbiotic functions. 8,27-33 Putative orthologs were searched in the Illumina transcriptome of the two mapping parents. Those that could not be identified are between parentheses. Genes highlighted in bold characters are those which were demonstrated to be involved in the Nod-independent nodulation process in A. evenia . 33,34" path-from-xml="dsw020f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198555/dsw020f4p.png" path-from-xml="dsw020f4p.png" target="_blank">View large</a><a section="87198555" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f4p.png?Expires=2147483647&Signature=SZnlGSKz8SdG2OEwkMUJF1hoTeyyrUHmBy9x3tXLPt60JQtIkm-Pl2aQgEF9Tt5rL1q6JxFKlNoOT4neWEbQrWlGFVbEzjnd4Tv125Y265iqjAuc22KXEaSDXI5S7ByA664RbT-LmppT8yHizXtHstyFz-IfcGR5oae--TFXBZ755G-WG9c8fKR4M~saLR99c29xM2Nf0N~2Rc9Ki-N81FmfhwVsY9fpHQ~SdcEJdWwKspD67vq~BVs18-Fp57kZqIXzKK8e3FYLw4uri6s0M-zq6LjCk6SeDGCTquyFGkPuUOio62KONIlNTYmrsY26v~qbYLoNNZ-qG5eHUn0GuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198555&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Simplified model for the symbiotic signaling pathway in legumes. Symbiotic genes identified in <em>M. truncatula</em> and <em>L. japonicus</em> are listed and classified in main symbiotic functions. <a reveal-id="dsw020-B8" data-open="dsw020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw020-B27" data-open="dsw020-B27" class="link link-ref link-reveal xref-bibr"><sup>27-33</sup></a> Putative orthologs were searched in the Illumina transcriptome of the two mapping parents. Those that could not be identified are between parentheses. Genes highlighted in bold characters are those which were demonstrated to be involved in the Nod-independent nodulation process in <em>A. evenia</em> . <a reveal-id="dsw020-B33" data-open="dsw020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw020-B34" data-open="dsw020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p></div></div><p> Interestingly, for some other symbiotic genes, transcripts were not detected despite the use of root and nodule materials for the transcriptomes. Missing genes are involved in three symbiotic processes: bacterial recognition (i.e. <em>LYK3</em> and <em>LYR3</em> ), rhizobial infection (i.e. <em>ANN1</em> , <em>EPR3</em> , <em>FLOT</em> and <em>RPG</em> ) as well as nodule functioning and immunity (i.e. <em>DNF2</em> , <em>FEN1</em> , <em>SUNERGOS1</em> , <em>SymCRK</em> and <em>VAG1</em> ) ( <a reveal-id="dsw020-F4" data-open="dsw020-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). In the absence of genomic data for <em>A. evenia</em> , it was not possible to determine whether these genes were absent, not expressed or missed when mining the transcriptome data. But, it is worth noting that the concerned genes are involved in symbiotic processes that present sharp differences between <em>A. evenia</em> , <em>M. truncatula</em> and <em>L. japonicus</em> ( <a reveal-id="dsw020-F1" data-open="dsw020-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). <a reveal-id="dsw020-B2" data-open="dsw020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw020-B19" data-open="dsw020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> As a consequence, these first observations represented important cues to further investigate the evolution of rhizobial symbiosis. </p>                            <h4 scrollto-destination=87198560 id="87198560" class="section-title" >3.2.3 Development of EST-based molecular markers</h4>
<p> To allow the development of molecular markers, the Bahia and Mbao transcriptomes were searched for microsatellites. These were subsequently analysed for <em>in silico</em> polymorphism by comparing the orthologous SSR-containing contigs of the two parental genotypes and filtered to allow primer design on flanking sequences. 1,500 microsatellites complying with these criteria were identified, the most abundant type of repeat motif being tri-nucleotides (46.4%), followed by di-nucleotides (35.9%) ( <a reveal-id="dsw020-T2" data-open="dsw020-T2" class="link link-reveal link-table xref-fig">Table 2</a> ). 500 SSR sites were selected for primer design and used for assessment of the polymorphism between the two parental genotypes and the F <sub>1</sub> progeny by capillary sequencing. Of the 412 successfully amplified SSRs, 335 (81%) were co-dominant, 50 (12%) were dominant and 27 (7%) were not polymorphic. Hence, thanks to the <em>in silico</em> analysis, the efficiency of polymorphic SSR development was increased by 2.5-fold compared to previous work with no prior filtering of the genic SSR tested on the same Bahia and Mbao genotypes. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Finally, a set of 318 co-dominant SSR markers was retained for the genotyping of the F <sub>2</sub> mapping population ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1, S2</a></span> ). </p>                        <a id="87198562" scrollto-destination="87198562"></a>
<div content-id="dsw020-T2" class="table-modal"><div class="table-wrap"><div id="dsw020-T2" class="table-wrap-title" data-id="dsw020-T2"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of repeat types and number of repeats within the Mbao library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Repeat type </th><th colspan="9"> Number of repeat units <hr /> </th><th rowspan="2">Total </th><th rowspan="2">% </th></tr><tr><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>&gt;10 </th></tr></thead><tbody><tr><td>Di </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">64 </td><td char=".">105 </td><td char=".">95 </td><td char=".">107 </td><td char=".">68 </td><td char=".">120 </td><td char=".">559 </td><td char=".">35.9 </td></tr><tr><td>Tri </td><td char=".">– </td><td char=".">68 </td><td char=".">141 </td><td char=".">181 </td><td char=".">132 </td><td char=".">93 </td><td char=".">49 </td><td char=".">29 </td><td char=".">31 </td><td char=".">724 </td><td char=".">46.4 </td></tr><tr><td>Tetra </td><td char=".">12 </td><td char=".">58 </td><td char=".">53 </td><td char=".">29 </td><td char=".">11 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">– </td><td char=".">168 </td><td char=".">10.8 </td></tr><tr><td>Penta </td><td char=".">25 </td><td char=".">51 </td><td char=".">27 </td><td char=".">1 </td><td char=".">1 </td><td char=".">1 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">106 </td><td char=".">6.8 </td></tr><tr><td>Hexa </td><td char=".">– </td><td char=".">2 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">2 </td><td char=".">0.1 </td></tr></tbody></table></div><div content-id="dsw020-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Repeat type </th><th colspan="9"> Number of repeat units <hr /> </th><th rowspan="2">Total </th><th rowspan="2">% </th></tr><tr><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>&gt;10 </th></tr></thead><tbody><tr><td>Di </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">64 </td><td char=".">105 </td><td char=".">95 </td><td char=".">107 </td><td char=".">68 </td><td char=".">120 </td><td char=".">559 </td><td char=".">35.9 </td></tr><tr><td>Tri </td><td char=".">– </td><td char=".">68 </td><td char=".">141 </td><td char=".">181 </td><td char=".">132 </td><td char=".">93 </td><td char=".">49 </td><td char=".">29 </td><td char=".">31 </td><td char=".">724 </td><td char=".">46.4 </td></tr><tr><td>Tetra </td><td char=".">12 </td><td char=".">58 </td><td char=".">53 </td><td char=".">29 </td><td char=".">11 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">– </td><td char=".">168 </td><td char=".">10.8 </td></tr><tr><td>Penta </td><td char=".">25 </td><td char=".">51 </td><td char=".">27 </td><td char=".">1 </td><td char=".">1 </td><td char=".">1 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">106 </td><td char=".">6.8 </td></tr><tr><td>Hexa </td><td char=".">– </td><td char=".">2 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">2 </td><td char=".">0.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198562" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw020-T2" class="table-wrap-title" data-id="dsw020-T2"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of repeat types and number of repeats within the Mbao library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Repeat type </th><th colspan="9"> Number of repeat units <hr /> </th><th rowspan="2">Total </th><th rowspan="2">% </th></tr><tr><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>&gt;10 </th></tr></thead><tbody><tr><td>Di </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">64 </td><td char=".">105 </td><td char=".">95 </td><td char=".">107 </td><td char=".">68 </td><td char=".">120 </td><td char=".">559 </td><td char=".">35.9 </td></tr><tr><td>Tri </td><td char=".">– </td><td char=".">68 </td><td char=".">141 </td><td char=".">181 </td><td char=".">132 </td><td char=".">93 </td><td char=".">49 </td><td char=".">29 </td><td char=".">31 </td><td char=".">724 </td><td char=".">46.4 </td></tr><tr><td>Tetra </td><td char=".">12 </td><td char=".">58 </td><td char=".">53 </td><td char=".">29 </td><td char=".">11 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">– </td><td char=".">168 </td><td char=".">10.8 </td></tr><tr><td>Penta </td><td char=".">25 </td><td char=".">51 </td><td char=".">27 </td><td char=".">1 </td><td char=".">1 </td><td char=".">1 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">106 </td><td char=".">6.8 </td></tr><tr><td>Hexa </td><td char=".">– </td><td char=".">2 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">2 </td><td char=".">0.1 </td></tr></tbody></table></div><div content-id="dsw020-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Repeat type </th><th colspan="9"> Number of repeat units <hr /> </th><th rowspan="2">Total </th><th rowspan="2">% </th></tr><tr><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>&gt;10 </th></tr></thead><tbody><tr><td>Di </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">64 </td><td char=".">105 </td><td char=".">95 </td><td char=".">107 </td><td char=".">68 </td><td char=".">120 </td><td char=".">559 </td><td char=".">35.9 </td></tr><tr><td>Tri </td><td char=".">– </td><td char=".">68 </td><td char=".">141 </td><td char=".">181 </td><td char=".">132 </td><td char=".">93 </td><td char=".">49 </td><td char=".">29 </td><td char=".">31 </td><td char=".">724 </td><td char=".">46.4 </td></tr><tr><td>Tetra </td><td char=".">12 </td><td char=".">58 </td><td char=".">53 </td><td char=".">29 </td><td char=".">11 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">– </td><td char=".">168 </td><td char=".">10.8 </td></tr><tr><td>Penta </td><td char=".">25 </td><td char=".">51 </td><td char=".">27 </td><td char=".">1 </td><td char=".">1 </td><td char=".">1 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">106 </td><td char=".">6.8 </td></tr><tr><td>Hexa </td><td char=".">– </td><td char=".">2 </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">– </td><td char=".">2 </td><td char=".">0.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198562" target="_blank">View Large</a></div></div><p> In order to map the symbiotic genes, polymorphisms were also searched for by comparing the orthologous sequences from the two parental genotypes and were converted into molecular markers. When no polymorphic SSR and no INDEL could be detected, Single Nucleotide Polymorphism (SNP) was used instead for KASP assay design ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1–S4</a></span> ). Subsequently, at least one molecular marker could be satisfactorily developed for each identified symbiotic gene, except for two genes, <em>SEN1</em> and <em>RSD</em> , that lack of polymorphism. A few additional markers were also used, notably to locate the 5S and 45S loci that were previously identified on <em>A. evenia</em> chromosomes by cytogenetics ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1, S2</a></span> ). <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p>                            <h3 scrollto-destination=87198565 id="87198565" class="section-title" >3.3 Construction of the genetic map and mapping of symbiotic genes</h3>
                            <h4 scrollto-destination=87198566 id="87198566" class="section-title" >3.3.1 Genotyping of the 220 F <sub>2</sub> mapping population</h4>
<p> 110 F <sub>2A</sub> and the 110 F <sub>2B</sub> individuals, which were obtained by bi-directional crossings of the Bahia and Mbao genotypes, formed the mapping population. DNAs were extracted from the 220 F <sub>2</sub> individuals and were genotyped for the selected 318 SSR markers by capillary sequencing. Multiplexes of 12 SSR markers were chosen so as to combine both three ranges of fragment sizes and four fluorochromes (FAM, VIC, NED and PET). 46 additional markers comprising SSRs, INDELs and SNPs, which target notable putative symbiotic genes, were also used for genotyping by capillary sequencing or KASPAR technology. This genotyping generated a matrix with &lt;0.6% of missing data for the 364 co-dominant markers used. </p>                            <h4 scrollto-destination=87198569 id="87198569" class="section-title" >3.3.2 Building of the genetic map</h4>
<p> The <em>A. evenia</em> genetic map was built using the JOINMAP software with a minimum LOD score value of 6. The markers were assembled into 10 linkage groups, corresponding to the haploid chromosome set of <em>A. evenia</em> (2 <em>n</em>  = 20). Interestingly, no marker remained unlinked and no linkage group could be split by increasing the LOD score value up to a LOD of 15. In addition, manual checking of the marker positions using color-mapping revealed no discrepancy in the genotyping data, thus corroborating the robustness of the 10 linkage groups obtained. They were thus numbered AeLG1 to AeLG10 and orientated as explained below ( <a reveal-id="dsw020-F5" data-open="dsw020-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). </p>                        <a id="87198571" scrollto-destination="87198571"></a>
<div id="dsw020-F5" data-id="dsw020-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f5p.png?Expires=2147483647&Signature=XU-7eSdKS-2G0oSmbLhAycmM~g~zPN4uYCrcIS0nOuNXwBVX422vRz-YIWwm7-tgOvwcUf-hOVe3ZBL6q6qdJuft~hT0k8tUP3wFaeqi3AaSvqWWd2tbXZVh~DE4h5yBmbuzqqmlkBeAB4jNiGlal6Vo5WNxY-6HnjWAAfsop7FU4stBlpF-siiXSKuXJgBZfG97-~TwbetPRV7unTLSLy55Wf3A52X8uiaD8iD8~zY8jhaT0Sk15EO3gXJf3axBSzaL-Mfr7QFhLdiZJ45us1TCIgjPsNzobef8aK9kq5Rj10RbIRprQOPxW8rLplT722Q0RotNLdBQJf9kJmgZLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Genetic map of Aeschynomene evenia . The genetic map based on F2 mapping population CIAT22838 × CIAT8232 is comprised of 364 gene based markers including SSR, INDEL and SNP (KASP) markers. The ten linkage groups are designated as AeLG1-AeLG10. The code to the right of the linkage groups refers to the marker or symbiotic gene name. The numbers to the left of the linkage groups refers to the genetic distances (Kosambi cM) from the top. " path-from-xml="dsw020f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198571/dsw020f5p.png" path-from-xml="dsw020f5p.png" target="_blank">View large</a><a section="87198571" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f5p.png?Expires=2147483647&Signature=T2liKG33yLMQMeS6-Qy-v3cQEcsgNkhYDleIWDjvqwVqu7OhZeRklz4sXH5i9OURftmTNTL0p7GAx80ZHhnAjJiz6VNWrVyxnQnLIDYFsDATpmAKKA4D6J2V4MXDB-rrN-ITgcm8Q9tGTV6rDEpcLcOkgPWWmuvsJakGBkUBQw8gt23s0FuZICqI2nut52jzhlKBEJE~cJJMUsLcFC7wvIR~OmqkPts1X60LOSMjuCBzrW1KDQ7fWtPl9LFFhm-pKmNhBK-SqS~qf7sv-29XB5x5zJ2Dw3bSxRZKPr8szp7z~TMbm6Q5Sx6siH4etd2I2lUrc1xuzAz~PPpnzYesPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198571&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Genetic map of <em>Aeschynomene evenia</em> . The genetic map based on F2 mapping population CIAT22838 × CIAT8232 is comprised of 364 gene based markers including SSR, INDEL and SNP (KASP) markers. The ten linkage groups are designated as AeLG1-AeLG10. The code to the right of the linkage groups refers to the marker or symbiotic gene name. The numbers to the left of the linkage groups refers to the genetic distances (Kosambi cM) from the top. </p></div></div><div content-id="dsw020-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f5p.png?Expires=2147483647&Signature=XU-7eSdKS-2G0oSmbLhAycmM~g~zPN4uYCrcIS0nOuNXwBVX422vRz-YIWwm7-tgOvwcUf-hOVe3ZBL6q6qdJuft~hT0k8tUP3wFaeqi3AaSvqWWd2tbXZVh~DE4h5yBmbuzqqmlkBeAB4jNiGlal6Vo5WNxY-6HnjWAAfsop7FU4stBlpF-siiXSKuXJgBZfG97-~TwbetPRV7unTLSLy55Wf3A52X8uiaD8iD8~zY8jhaT0Sk15EO3gXJf3axBSzaL-Mfr7QFhLdiZJ45us1TCIgjPsNzobef8aK9kq5Rj10RbIRprQOPxW8rLplT722Q0RotNLdBQJf9kJmgZLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Genetic map of Aeschynomene evenia . The genetic map based on F2 mapping population CIAT22838 × CIAT8232 is comprised of 364 gene based markers including SSR, INDEL and SNP (KASP) markers. The ten linkage groups are designated as AeLG1-AeLG10. The code to the right of the linkage groups refers to the marker or symbiotic gene name. The numbers to the left of the linkage groups refers to the genetic distances (Kosambi cM) from the top. " path-from-xml="dsw020f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198571/dsw020f5p.png" path-from-xml="dsw020f5p.png" target="_blank">View large</a><a section="87198571" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f5p.png?Expires=2147483647&Signature=T2liKG33yLMQMeS6-Qy-v3cQEcsgNkhYDleIWDjvqwVqu7OhZeRklz4sXH5i9OURftmTNTL0p7GAx80ZHhnAjJiz6VNWrVyxnQnLIDYFsDATpmAKKA4D6J2V4MXDB-rrN-ITgcm8Q9tGTV6rDEpcLcOkgPWWmuvsJakGBkUBQw8gt23s0FuZICqI2nut52jzhlKBEJE~cJJMUsLcFC7wvIR~OmqkPts1X60LOSMjuCBzrW1KDQ7fWtPl9LFFhm-pKmNhBK-SqS~qf7sv-29XB5x5zJ2Dw3bSxRZKPr8szp7z~TMbm6Q5Sx6siH4etd2I2lUrc1xuzAz~PPpnzYesPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198571&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Genetic map of <em>Aeschynomene evenia</em> . The genetic map based on F2 mapping population CIAT22838 × CIAT8232 is comprised of 364 gene based markers including SSR, INDEL and SNP (KASP) markers. The ten linkage groups are designated as AeLG1-AeLG10. The code to the right of the linkage groups refers to the marker or symbiotic gene name. The numbers to the left of the linkage groups refers to the genetic distances (Kosambi cM) from the top. </p></div></div><p> The resulting genetic map spans 1,036 Kosambi cM, with an average of 400 kb/cM. The 10 linkage groups have different genetic sizes ranging from 82 to 130.8 cM and contained 13–49 markers ( <a reveal-id="dsw020-T3" data-open="dsw020-T3" class="link link-reveal link-table xref-fig">Table 3</a> ). This leads to average distances between two adjacent markers varying from 2.6 to 7.5 cM but with maximum interval sizes per linkage group ranging from 10.7 to 25.1 cM ( <a reveal-id="dsw020-T3" data-open="dsw020-T3" class="link link-reveal link-table xref-fig">Table 3</a> ). This situation is reminiscent to what has been already observed in <em>Vigna unguiculata</em> and <em>Lens culinaris</em> for which genetic maps were also developed with gene-based markers. <a reveal-id="dsw020-B36" data-open="dsw020-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsw020-B37" data-open="dsw020-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The gaps have been proposed to correspond to gene-poor regions or to recombination hotspots. In addition, mapping of the 5S and 45S rDNA loci further enabled distinguishing two linkage groups. Thus the 5S rDNA locus was found to locate on linkage group AeLG8 with a central position (at 55.6 cM on total 96.2 cM length) that is in accordance with the proximal localization previously observed on metaphase chromosomes by GISH. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Conversely, two 45S rDNA loci were shown to locate to in secondary constrictions of satellite chromosomes. <a reveal-id="dsw020-B21" data-open="dsw020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Sequence polymorphism for the ITS region allowed mapping only one of the two loci on the upper part of the linkage group AeLG10 (at 19.4 cM on a total 82 cM length) ( <a reveal-id="dsw020-F5" data-open="dsw020-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Finally, gene-specific markers corresponding to symbiotic genes were found to be distributed in all but one (AeLG2) linkage groups, with AeLG5, AeLG6, AeLG7 and AeLG9 containing the majority of them ( <a reveal-id="dsw020-F5" data-open="dsw020-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). </p>                        <a id="87198573" scrollto-destination="87198573"></a>
<div content-id="dsw020-T3" class="table-modal"><div class="table-wrap"><div id="dsw020-T3" class="table-wrap-title" data-id="dsw020-T3"><div class="label -label">Table 3.</div><div class="caption"><p> Characteristics of the <em>Aeschynomene evenia</em> genetic map </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Total length (cM) </th><th>Number of loci </th><th>Number of markers </th><th>Average distance (cM) </th><th>Min–max distances (cM) </th><th>Distorted markers (P &lt; 0.001) </th></tr></thead><tbody><tr><td>AeLG1 </td><td char=".">107.1 </td><td char=".">25 </td><td char=".">29 </td><td char=".">4.3 </td><td char=".">0–13.5 </td><td char=".">0 </td></tr><tr><td>AeLG2 </td><td char=".">98.1 </td><td char=".">13 </td><td char=".">13 </td><td char=".">7.5 </td><td char=".">0–22.1 </td><td char=".">10 </td></tr><tr><td>AeLG3 </td><td char=".">113.1 </td><td char=".">44 </td><td char=".">49 </td><td char=".">2.6 </td><td char=".">0–25.1 </td><td char=".">0 </td></tr><tr><td>AeLG4 </td><td char=".">105.2 </td><td char=".">38 </td><td char=".">46 </td><td char=".">2.8 </td><td char=".">0–13.3 </td><td char=".">0 </td></tr><tr><td>AeLG5 </td><td char=".">124.7 </td><td char=".">42 </td><td char=".">48 </td><td char=".">3.0 </td><td char=".">0–19.8 </td><td char=".">0 </td></tr><tr><td>AeLG6 </td><td char=".">130.8 </td><td char=".">37 </td><td char=".">41 </td><td char=".">3.5 </td><td char=".">0–21.8 </td><td char=".">0 </td></tr><tr><td>AeLG7 </td><td char=".">83.6 </td><td char=".">31 </td><td char=".">34 </td><td char=".">2.7 </td><td char=".">0–10.7 </td><td char=".">0 </td></tr><tr><td>AeLG8 </td><td char=".">96.2 </td><td char=".">37 </td><td char=".">43 </td><td char=".">2.6 </td><td char=".">0–12.6 </td><td char=".">0 </td></tr><tr><td>AeLG9 </td><td char=".">95.3 </td><td char=".">33 </td><td char=".">36 </td><td char=".">2.9 </td><td char=".">0–13.9 </td><td char=".">15 </td></tr><tr><td>AeLG10 </td><td char=".">82 </td><td char=".">24 </td><td char=".">25 </td><td char=".">3.4 </td><td char=".">0–20.6 </td><td char=".">0 </td></tr></tbody></table></div><div content-id="dsw020-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Total length (cM) </th><th>Number of loci </th><th>Number of markers </th><th>Average distance (cM) </th><th>Min–max distances (cM) </th><th>Distorted markers (P &lt; 0.001) </th></tr></thead><tbody><tr><td>AeLG1 </td><td char=".">107.1 </td><td char=".">25 </td><td char=".">29 </td><td char=".">4.3 </td><td char=".">0–13.5 </td><td char=".">0 </td></tr><tr><td>AeLG2 </td><td char=".">98.1 </td><td char=".">13 </td><td char=".">13 </td><td char=".">7.5 </td><td char=".">0–22.1 </td><td char=".">10 </td></tr><tr><td>AeLG3 </td><td char=".">113.1 </td><td char=".">44 </td><td char=".">49 </td><td char=".">2.6 </td><td char=".">0–25.1 </td><td char=".">0 </td></tr><tr><td>AeLG4 </td><td char=".">105.2 </td><td char=".">38 </td><td char=".">46 </td><td char=".">2.8 </td><td char=".">0–13.3 </td><td char=".">0 </td></tr><tr><td>AeLG5 </td><td char=".">124.7 </td><td char=".">42 </td><td char=".">48 </td><td char=".">3.0 </td><td char=".">0–19.8 </td><td char=".">0 </td></tr><tr><td>AeLG6 </td><td char=".">130.8 </td><td char=".">37 </td><td char=".">41 </td><td char=".">3.5 </td><td char=".">0–21.8 </td><td char=".">0 </td></tr><tr><td>AeLG7 </td><td char=".">83.6 </td><td char=".">31 </td><td char=".">34 </td><td char=".">2.7 </td><td char=".">0–10.7 </td><td char=".">0 </td></tr><tr><td>AeLG8 </td><td char=".">96.2 </td><td char=".">37 </td><td char=".">43 </td><td char=".">2.6 </td><td char=".">0–12.6 </td><td char=".">0 </td></tr><tr><td>AeLG9 </td><td char=".">95.3 </td><td char=".">33 </td><td char=".">36 </td><td char=".">2.9 </td><td char=".">0–13.9 </td><td char=".">15 </td></tr><tr><td>AeLG10 </td><td char=".">82 </td><td char=".">24 </td><td char=".">25 </td><td char=".">3.4 </td><td char=".">0–20.6 </td><td char=".">0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198573" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw020-T3" class="table-wrap-title" data-id="dsw020-T3"><div class="label -label">Table 3.</div><div class="caption"><p> Characteristics of the <em>Aeschynomene evenia</em> genetic map </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Total length (cM) </th><th>Number of loci </th><th>Number of markers </th><th>Average distance (cM) </th><th>Min–max distances (cM) </th><th>Distorted markers (P &lt; 0.001) </th></tr></thead><tbody><tr><td>AeLG1 </td><td char=".">107.1 </td><td char=".">25 </td><td char=".">29 </td><td char=".">4.3 </td><td char=".">0–13.5 </td><td char=".">0 </td></tr><tr><td>AeLG2 </td><td char=".">98.1 </td><td char=".">13 </td><td char=".">13 </td><td char=".">7.5 </td><td char=".">0–22.1 </td><td char=".">10 </td></tr><tr><td>AeLG3 </td><td char=".">113.1 </td><td char=".">44 </td><td char=".">49 </td><td char=".">2.6 </td><td char=".">0–25.1 </td><td char=".">0 </td></tr><tr><td>AeLG4 </td><td char=".">105.2 </td><td char=".">38 </td><td char=".">46 </td><td char=".">2.8 </td><td char=".">0–13.3 </td><td char=".">0 </td></tr><tr><td>AeLG5 </td><td char=".">124.7 </td><td char=".">42 </td><td char=".">48 </td><td char=".">3.0 </td><td char=".">0–19.8 </td><td char=".">0 </td></tr><tr><td>AeLG6 </td><td char=".">130.8 </td><td char=".">37 </td><td char=".">41 </td><td char=".">3.5 </td><td char=".">0–21.8 </td><td char=".">0 </td></tr><tr><td>AeLG7 </td><td char=".">83.6 </td><td char=".">31 </td><td char=".">34 </td><td char=".">2.7 </td><td char=".">0–10.7 </td><td char=".">0 </td></tr><tr><td>AeLG8 </td><td char=".">96.2 </td><td char=".">37 </td><td char=".">43 </td><td char=".">2.6 </td><td char=".">0–12.6 </td><td char=".">0 </td></tr><tr><td>AeLG9 </td><td char=".">95.3 </td><td char=".">33 </td><td char=".">36 </td><td char=".">2.9 </td><td char=".">0–13.9 </td><td char=".">15 </td></tr><tr><td>AeLG10 </td><td char=".">82 </td><td char=".">24 </td><td char=".">25 </td><td char=".">3.4 </td><td char=".">0–20.6 </td><td char=".">0 </td></tr></tbody></table></div><div content-id="dsw020-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Total length (cM) </th><th>Number of loci </th><th>Number of markers </th><th>Average distance (cM) </th><th>Min–max distances (cM) </th><th>Distorted markers (P &lt; 0.001) </th></tr></thead><tbody><tr><td>AeLG1 </td><td char=".">107.1 </td><td char=".">25 </td><td char=".">29 </td><td char=".">4.3 </td><td char=".">0–13.5 </td><td char=".">0 </td></tr><tr><td>AeLG2 </td><td char=".">98.1 </td><td char=".">13 </td><td char=".">13 </td><td char=".">7.5 </td><td char=".">0–22.1 </td><td char=".">10 </td></tr><tr><td>AeLG3 </td><td char=".">113.1 </td><td char=".">44 </td><td char=".">49 </td><td char=".">2.6 </td><td char=".">0–25.1 </td><td char=".">0 </td></tr><tr><td>AeLG4 </td><td char=".">105.2 </td><td char=".">38 </td><td char=".">46 </td><td char=".">2.8 </td><td char=".">0–13.3 </td><td char=".">0 </td></tr><tr><td>AeLG5 </td><td char=".">124.7 </td><td char=".">42 </td><td char=".">48 </td><td char=".">3.0 </td><td char=".">0–19.8 </td><td char=".">0 </td></tr><tr><td>AeLG6 </td><td char=".">130.8 </td><td char=".">37 </td><td char=".">41 </td><td char=".">3.5 </td><td char=".">0–21.8 </td><td char=".">0 </td></tr><tr><td>AeLG7 </td><td char=".">83.6 </td><td char=".">31 </td><td char=".">34 </td><td char=".">2.7 </td><td char=".">0–10.7 </td><td char=".">0 </td></tr><tr><td>AeLG8 </td><td char=".">96.2 </td><td char=".">37 </td><td char=".">43 </td><td char=".">2.6 </td><td char=".">0–12.6 </td><td char=".">0 </td></tr><tr><td>AeLG9 </td><td char=".">95.3 </td><td char=".">33 </td><td char=".">36 </td><td char=".">2.9 </td><td char=".">0–13.9 </td><td char=".">15 </td></tr><tr><td>AeLG10 </td><td char=".">82 </td><td char=".">24 </td><td char=".">25 </td><td char=".">3.4 </td><td char=".">0–20.6 </td><td char=".">0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198573" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=87198574 id="87198574" class="section-title" >3.3.3 Regions of segregating distortion (RSD)</h4>
<p> Although almost all markers complied with the 1:2:1 expected Mendelian ratio for co-dominant genetic markers, two regions of segregation distortion were observed ( <a reveal-id="dsw020-T3" data-open="dsw020-T3" class="link link-reveal link-table xref-fig">Table 3</a> , <a reveal-id="dsw020-F6" data-open="dsw020-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a> ). The AeLG2 RSD comprised up to 10 markers covering a map distance of up 70 cM and the AeLG9 RSD contained 11 markers (among which the symbiotic <em>NSP1</em> and <em>EFD</em> genes) distributed over 45 cM, altogether representing 10% of the genetic map. In both cases, there was a strong overrepresentation of the Bahia allele and a near complete disappearance of the Mbao allele where the distortion culminated, whereas the proportion of heterozygotes remained unaltered. Judged from these skewed genotypic frequencies, a strong selection in favor of the Bahia alleles occurs at these RSD. </p>                        <a id="87198577" scrollto-destination="87198577"></a>
<div id="dsw020-F6" data-id="dsw020-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f6p.png?Expires=2147483647&Signature=jXtD7aK2sFcqn1FJJkwwDu~kQpAj2E6iH~dnNE1djKP4ZEdsqELnTGU8aERLn4YvDTwTFbO7EwM9irPDNxs4vI2r8EUbj-Adl~dHAH~uT3PFk2YvTTULHeH1IwhIdFZlwRF8NarKmQsm7Y7RHv-~rYrbFKDj3DAAqFbvt8DXi4BoiJ5oi~fjcVH78g38mqqsYBxuDfZqSxa-ahqxrxVkF5B7~jle90toMwqgtl5dS2fswlOLcgoVl5mRsRLudVHMa36V5zr1rRUE9JZRoNJamOXNeig3burMNjPX7TTT2XaXDNYpEIO6dfumgrs3LrdLGW2FsXpcWAEyVWRIya~rtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Segregation distortion and pollen viability of F1 hybrids. (A) Segregation distortion of the co-dominant markers along linkage groups AeLG 2 and AeLG 9 in the F2 mapping population. Triangles refer to the CIAT22838 genotype, lozenges to the CIAT8232 genotype and squares to the heterozygote. X -axis: genetic distance from the top of the linkage group in Kosambi cM. Y -axis: frequency (%) of segregation of the different genotypes. If no distortion occurs, the segregation values should be 25%/50%/25%. B, Proportion of non-aborted pollen grains in the mapping parents and the F1 hybrid. C, Anther from the CIAT22838 line (upper left panel) and pollen grains of the mapping parents and the F1 hybrid. Aborted pollen stains pale and nonaborted pollen stains dark. Bars = 50 µm. " path-from-xml="dsw020f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198577/dsw020f6p.png" path-from-xml="dsw020f6p.png" target="_blank">View large</a><a section="87198577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f6p.png?Expires=2147483647&Signature=eOF4joQcREb~x5Zn5sN3LhLftAq4vS0HFtcbnekwr2RfUh-JMkzhXHXq2HmV7OxtwOtV9QfvzohEq0~CqG0MVbCs8MHhkA2~Sxb0xBeZ473op4q1-IZlHtWqBrkhCEE6Bq9uMRPDGFGTOJ6fx3wHhAm-ixlu49kdTaf6fe6jIw6Ut6LQ9mdH3arOlVxKCAXpgzknDSxKOAz-rTh4UhnnYsNKJBoJ38WRUoAZ8A5LN1e5w9-9DgIDI3qPI-XqmD5Mbxkc2cqtFqyEnzUZPJw4YksDespLrdJXNUIwx8Ttcy3KgQD-K9jUPNObvMOopjqPcxr8N3Uhojlk0Qeu58yZpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198577&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Segregation distortion and pollen viability of F1 hybrids. (A) Segregation distortion of the co-dominant markers along linkage groups <em>AeLG</em> 2 and <em>AeLG</em> 9 in the F2 mapping population. Triangles refer to the CIAT22838 genotype, lozenges to the CIAT8232 genotype and squares to the heterozygote. <em>X</em> -axis: genetic distance from the top of the linkage group in Kosambi cM. <em>Y</em> -axis: frequency (%) of segregation of the different genotypes. If no distortion occurs, the segregation values should be 25%/50%/25%. B, Proportion of non-aborted pollen grains in the mapping parents and the F1 hybrid. C, Anther from the CIAT22838 line (upper left panel) and pollen grains of the mapping parents and the F1 hybrid. Aborted pollen stains pale and nonaborted pollen stains dark. Bars = 50 µm. </p></div></div><div content-id="dsw020-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f6p.png?Expires=2147483647&Signature=jXtD7aK2sFcqn1FJJkwwDu~kQpAj2E6iH~dnNE1djKP4ZEdsqELnTGU8aERLn4YvDTwTFbO7EwM9irPDNxs4vI2r8EUbj-Adl~dHAH~uT3PFk2YvTTULHeH1IwhIdFZlwRF8NarKmQsm7Y7RHv-~rYrbFKDj3DAAqFbvt8DXi4BoiJ5oi~fjcVH78g38mqqsYBxuDfZqSxa-ahqxrxVkF5B7~jle90toMwqgtl5dS2fswlOLcgoVl5mRsRLudVHMa36V5zr1rRUE9JZRoNJamOXNeig3burMNjPX7TTT2XaXDNYpEIO6dfumgrs3LrdLGW2FsXpcWAEyVWRIya~rtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Segregation distortion and pollen viability of F1 hybrids. (A) Segregation distortion of the co-dominant markers along linkage groups AeLG 2 and AeLG 9 in the F2 mapping population. Triangles refer to the CIAT22838 genotype, lozenges to the CIAT8232 genotype and squares to the heterozygote. X -axis: genetic distance from the top of the linkage group in Kosambi cM. Y -axis: frequency (%) of segregation of the different genotypes. If no distortion occurs, the segregation values should be 25%/50%/25%. B, Proportion of non-aborted pollen grains in the mapping parents and the F1 hybrid. C, Anther from the CIAT22838 line (upper left panel) and pollen grains of the mapping parents and the F1 hybrid. Aborted pollen stains pale and nonaborted pollen stains dark. Bars = 50 µm. " path-from-xml="dsw020f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198577/dsw020f6p.png" path-from-xml="dsw020f6p.png" target="_blank">View large</a><a section="87198577" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f6p.png?Expires=2147483647&Signature=eOF4joQcREb~x5Zn5sN3LhLftAq4vS0HFtcbnekwr2RfUh-JMkzhXHXq2HmV7OxtwOtV9QfvzohEq0~CqG0MVbCs8MHhkA2~Sxb0xBeZ473op4q1-IZlHtWqBrkhCEE6Bq9uMRPDGFGTOJ6fx3wHhAm-ixlu49kdTaf6fe6jIw6Ut6LQ9mdH3arOlVxKCAXpgzknDSxKOAz-rTh4UhnnYsNKJBoJ38WRUoAZ8A5LN1e5w9-9DgIDI3qPI-XqmD5Mbxkc2cqtFqyEnzUZPJw4YksDespLrdJXNUIwx8Ttcy3KgQD-K9jUPNObvMOopjqPcxr8N3Uhojlk0Qeu58yZpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198577&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Segregation distortion and pollen viability of F1 hybrids. (A) Segregation distortion of the co-dominant markers along linkage groups <em>AeLG</em> 2 and <em>AeLG</em> 9 in the F2 mapping population. Triangles refer to the CIAT22838 genotype, lozenges to the CIAT8232 genotype and squares to the heterozygote. <em>X</em> -axis: genetic distance from the top of the linkage group in Kosambi cM. <em>Y</em> -axis: frequency (%) of segregation of the different genotypes. If no distortion occurs, the segregation values should be 25%/50%/25%. B, Proportion of non-aborted pollen grains in the mapping parents and the F1 hybrid. C, Anther from the CIAT22838 line (upper left panel) and pollen grains of the mapping parents and the F1 hybrid. Aborted pollen stains pale and nonaborted pollen stains dark. Bars = 50 µm. </p></div></div><p> Distortions can be caused by chromosomal structural rearrangements, differences in DNA content, gene incompatibilities or deleterious recessive alleles. Inversion or translocation events have been shown to be the cause of distortions in genetic maps of other legume such as <em>M. truncatula</em> , <em>L. japonicus</em> and <em>L. culinaris</em> . <a reveal-id="dsw020-B13" data-open="dsw020-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw020-B26" data-open="dsw020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsw020-B38 dsw020-B39 dsw020-B40 dsw020-B41" data-open="dsw020-B38 dsw020-B39 dsw020-B40 dsw020-B41" class="link link-ref link-reveal xref-bibr"><sup>38–41</sup></a> Such events are more likely to occur when performing interspecific crosses than when using related genotypes, but one cannot exclude that a chromosomal rearrangement occurred within the subspecies <em>evenia</em> to which both the Bahia and Mbao genotypes belong. In addition, even if they display a similar overall DNA content, this DNA content may not be identical in the two RSD regions. Comparison of the genotypic data for the F <sub>2A</sub> and F <sub>2B</sub> mapping populations, revealed similar allele skewing in the two RSD regions, indicating that the observed distortions were independent of maternal and paternal effects. However, pollen viability analysis showed that the F <sub>1</sub> progeny contained ∼38% aborted pollen in contrast to the Bahia and Mbao parental genotypes that showed more than 99% pollen viability ( <a reveal-id="dsw020-F6" data-open="dsw020-F6" class="link link-reveal link-table xref-fig">Fig. 6B,C</a> ). This F <sub>1</sub> semisterility may account for the observed skewed genetic frequencies, as it has been reported for some <em>M. truncatula</em> crosses. <a reveal-id="dsw020-B38" data-open="dsw020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Although, this RSD can alter genetic distances, it does not compromise marker ordering in the SDR regions. </p>                            <h3 scrollto-destination=87198579 id="87198579" class="section-title" >3.4  Comparative analysis with <em>Arachis</em></h3>
                            <h4 scrollto-destination=87198580 id="87198580" class="section-title" >3.4.1 Estimate of time divergences</h4>
<p> With the aim of fostering genomic and symbiotic comparative analysis, we wanted to further our knowledge of the genetic differentiation in <em>A. evenia</em> and of the evolution within the Dalbergioid clade that contains <em>Aeschynomene</em> and <em>Arachis</em> . A ∼49 MYA divergence of <em>Aeschynomene</em> and <em>Arachis</em> was previously estimated from chloroplast gene phylogenies ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig 2A</a> ). <a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Using this date for calibration and assuming a simple molecular clock, we made a rough estimate for the divergence times between different <em>A. evenia</em> genotypes and between the two <em>Arachis</em> ancestral species of peanut. The coding reading frame of seven single-copy nuclear genes was obtained from the transcript databases available for <em>A. evenia</em> genotypes Bahia (CIAT8232), Goias (IRFL6945) and Mbao (CIAT22838) and their <em>Arachis</em> orthologs in the genome sequences of <em>A. duranensis</em> and <em>A. ipaensis</em> ( <a reveal-id="dsw020-F2" data-open="dsw020-F2" class="link link-reveal link-table xref-fig">Fig 2A</a> , <a reveal-id="dsw020-T4" data-open="dsw020-T4" class="link link-reveal link-table xref-fig">Table 4</a> , <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> ). Synonymous substitution rates (Ks) were calculated for binned values and an average mutation rate of 5.34 × 10 <sup>−</sup><sup>3</sup> substitutions/synonymous site/Mya was obtained by using the 49 MYA value for the <em>Aeschynomene</em> – <em>Arachis</em> divergence. <a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>                        <a id="87198582" scrollto-destination="87198582"></a>
<div content-id="dsw020-T4" class="table-modal"><div class="table-wrap"><div id="dsw020-T4" class="table-wrap-title" data-id="dsw020-T4"><div class="label -label">Table 4.</div><div class="caption"><p> Estimation of divergence times between <em>Aeschynomene</em> and <em>Arachis</em> genomes </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Evolutionary divergence </th><th colspan="7"> Substitution per site (Ks) <hr /> </th><th rowspan="2">Average (SR) </th><th colspan="2"> Divergence <hr /> </th></tr><tr><th>APX2 (738 pb) </th><th>AS2 (1,629 pb) </th><th>β-Fructofuranosidase (1,536 pb) </th><th>PIP2;7 (864 pb) </th><th>Proline-tRNA ligase (1,098 pb) </th><th>SPAG1 (861 pb) </th><th>TIP1;1 (750 pb) </th><th>(SR/Mya) </th><th>(Mya) </th></tr></thead><tbody><tr><td><em>A. duranensis</em> – <em>A. evenia</em> Mbao  </td><td char=".">0.581 </td><td char=".">0.556 </td><td char=".">0.504 </td><td char=".">0.797 </td><td char=".">0.380 </td><td char=".">0.431 </td><td char=".">0.442 </td><td char=".">0.527 </td><td rowspan="2"> 0.00534 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td><td rowspan="2"> 49 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td><em>A. duranensis</em> – <em>A. evenia Mbao</em> </td><td char=".">0.569 </td><td char=".">0.576 </td><td char=".">0.531 </td><td char=".">0.758 </td><td char=".">0.367 </td><td char=".">0.421 </td><td char=".">0.419 </td><td char=".">0.520 </td></tr><tr><td><em>A. duranensis</em> – <em>A. ipaensis</em> </td><td char=".">0.029 </td><td char=".">0.039 </td><td char=".">0.037 </td><td char=".">0.053 </td><td char=".">0.025 </td><td char=".">0.049 </td><td char=".">0.026 </td><td char=".">0.037 </td><td char=".">0.00534 </td><td char="."><strong>3.5</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Bahia  </td><td char=".">0.035 </td><td char=".">0.056 </td><td char=".">0.039 </td><td char=".">0.024 </td><td char=".">0.029 </td><td char=".">0.038 </td><td char=".">0.026 </td><td char=".">0.035 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Mbao  </td><td char=".">0.035 </td><td char=".">0.050 </td><td char=".">0.045 </td><td char=".">0.024 </td><td char=".">0.038 </td><td char=".">0.032 </td><td char=".">0.026 </td><td char=".">0.036 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Bahia-Mbao  </td><td char=".">0.000 </td><td char=".">0.011 </td><td char=".">0.008 </td><td char=".">0.000 </td><td char=".">0.008 </td><td char=".">0.005 </td><td char=".">0.000 </td><td char=".">0.005 </td><td char=".">0.00534 </td><td char="."><strong>0.4</strong> </td></tr></tbody></table></div><div content-id="dsw020-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Evolutionary divergence </th><th colspan="7"> Substitution per site (Ks) <hr /> </th><th rowspan="2">Average (SR) </th><th colspan="2"> Divergence <hr /> </th></tr><tr><th>APX2 (738 pb) </th><th>AS2 (1,629 pb) </th><th>β-Fructofuranosidase (1,536 pb) </th><th>PIP2;7 (864 pb) </th><th>Proline-tRNA ligase (1,098 pb) </th><th>SPAG1 (861 pb) </th><th>TIP1;1 (750 pb) </th><th>(SR/Mya) </th><th>(Mya) </th></tr></thead><tbody><tr><td><em>A. duranensis</em> – <em>A. evenia</em> Mbao  </td><td char=".">0.581 </td><td char=".">0.556 </td><td char=".">0.504 </td><td char=".">0.797 </td><td char=".">0.380 </td><td char=".">0.431 </td><td char=".">0.442 </td><td char=".">0.527 </td><td rowspan="2"> 0.00534 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td><td rowspan="2"> 49 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td><em>A. duranensis</em> – <em>A. evenia Mbao</em> </td><td char=".">0.569 </td><td char=".">0.576 </td><td char=".">0.531 </td><td char=".">0.758 </td><td char=".">0.367 </td><td char=".">0.421 </td><td char=".">0.419 </td><td char=".">0.520 </td></tr><tr><td><em>A. duranensis</em> – <em>A. ipaensis</em> </td><td char=".">0.029 </td><td char=".">0.039 </td><td char=".">0.037 </td><td char=".">0.053 </td><td char=".">0.025 </td><td char=".">0.049 </td><td char=".">0.026 </td><td char=".">0.037 </td><td char=".">0.00534 </td><td char="."><strong>3.5</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Bahia  </td><td char=".">0.035 </td><td char=".">0.056 </td><td char=".">0.039 </td><td char=".">0.024 </td><td char=".">0.029 </td><td char=".">0.038 </td><td char=".">0.026 </td><td char=".">0.035 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Mbao  </td><td char=".">0.035 </td><td char=".">0.050 </td><td char=".">0.045 </td><td char=".">0.024 </td><td char=".">0.038 </td><td char=".">0.032 </td><td char=".">0.026 </td><td char=".">0.036 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Bahia-Mbao  </td><td char=".">0.000 </td><td char=".">0.011 </td><td char=".">0.008 </td><td char=".">0.000 </td><td char=".">0.008 </td><td char=".">0.005 </td><td char=".">0.000 </td><td char=".">0.005 </td><td char=".">0.00534 </td><td char="."><strong>0.4</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw020-TF1"><div class="fn"><p> *Average divergence rate obtained by using the known 49 MYA divergence time between <em>Aeschynomene</em> and <em>Arachis</em> . <a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Numbers in bold are inferred values. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198582" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw020-T4" class="table-wrap-title" data-id="dsw020-T4"><div class="label -label">Table 4.</div><div class="caption"><p> Estimation of divergence times between <em>Aeschynomene</em> and <em>Arachis</em> genomes </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Evolutionary divergence </th><th colspan="7"> Substitution per site (Ks) <hr /> </th><th rowspan="2">Average (SR) </th><th colspan="2"> Divergence <hr /> </th></tr><tr><th>APX2 (738 pb) </th><th>AS2 (1,629 pb) </th><th>β-Fructofuranosidase (1,536 pb) </th><th>PIP2;7 (864 pb) </th><th>Proline-tRNA ligase (1,098 pb) </th><th>SPAG1 (861 pb) </th><th>TIP1;1 (750 pb) </th><th>(SR/Mya) </th><th>(Mya) </th></tr></thead><tbody><tr><td><em>A. duranensis</em> – <em>A. evenia</em> Mbao  </td><td char=".">0.581 </td><td char=".">0.556 </td><td char=".">0.504 </td><td char=".">0.797 </td><td char=".">0.380 </td><td char=".">0.431 </td><td char=".">0.442 </td><td char=".">0.527 </td><td rowspan="2"> 0.00534 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td><td rowspan="2"> 49 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td><em>A. duranensis</em> – <em>A. evenia Mbao</em> </td><td char=".">0.569 </td><td char=".">0.576 </td><td char=".">0.531 </td><td char=".">0.758 </td><td char=".">0.367 </td><td char=".">0.421 </td><td char=".">0.419 </td><td char=".">0.520 </td></tr><tr><td><em>A. duranensis</em> – <em>A. ipaensis</em> </td><td char=".">0.029 </td><td char=".">0.039 </td><td char=".">0.037 </td><td char=".">0.053 </td><td char=".">0.025 </td><td char=".">0.049 </td><td char=".">0.026 </td><td char=".">0.037 </td><td char=".">0.00534 </td><td char="."><strong>3.5</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Bahia  </td><td char=".">0.035 </td><td char=".">0.056 </td><td char=".">0.039 </td><td char=".">0.024 </td><td char=".">0.029 </td><td char=".">0.038 </td><td char=".">0.026 </td><td char=".">0.035 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Mbao  </td><td char=".">0.035 </td><td char=".">0.050 </td><td char=".">0.045 </td><td char=".">0.024 </td><td char=".">0.038 </td><td char=".">0.032 </td><td char=".">0.026 </td><td char=".">0.036 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Bahia-Mbao  </td><td char=".">0.000 </td><td char=".">0.011 </td><td char=".">0.008 </td><td char=".">0.000 </td><td char=".">0.008 </td><td char=".">0.005 </td><td char=".">0.000 </td><td char=".">0.005 </td><td char=".">0.00534 </td><td char="."><strong>0.4</strong> </td></tr></tbody></table></div><div content-id="dsw020-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Evolutionary divergence </th><th colspan="7"> Substitution per site (Ks) <hr /> </th><th rowspan="2">Average (SR) </th><th colspan="2"> Divergence <hr /> </th></tr><tr><th>APX2 (738 pb) </th><th>AS2 (1,629 pb) </th><th>β-Fructofuranosidase (1,536 pb) </th><th>PIP2;7 (864 pb) </th><th>Proline-tRNA ligase (1,098 pb) </th><th>SPAG1 (861 pb) </th><th>TIP1;1 (750 pb) </th><th>(SR/Mya) </th><th>(Mya) </th></tr></thead><tbody><tr><td><em>A. duranensis</em> – <em>A. evenia</em> Mbao  </td><td char=".">0.581 </td><td char=".">0.556 </td><td char=".">0.504 </td><td char=".">0.797 </td><td char=".">0.380 </td><td char=".">0.431 </td><td char=".">0.442 </td><td char=".">0.527 </td><td rowspan="2"> 0.00534 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td><td rowspan="2"> 49 <a reveal-id="dsw020-TF1" data-open="dsw020-TF1" class="link link-ref link-reveal xref-fn">*</a> </td></tr><tr><td><em>A. duranensis</em> – <em>A. evenia Mbao</em> </td><td char=".">0.569 </td><td char=".">0.576 </td><td char=".">0.531 </td><td char=".">0.758 </td><td char=".">0.367 </td><td char=".">0.421 </td><td char=".">0.419 </td><td char=".">0.520 </td></tr><tr><td><em>A. duranensis</em> – <em>A. ipaensis</em> </td><td char=".">0.029 </td><td char=".">0.039 </td><td char=".">0.037 </td><td char=".">0.053 </td><td char=".">0.025 </td><td char=".">0.049 </td><td char=".">0.026 </td><td char=".">0.037 </td><td char=".">0.00534 </td><td char="."><strong>3.5</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Bahia  </td><td char=".">0.035 </td><td char=".">0.056 </td><td char=".">0.039 </td><td char=".">0.024 </td><td char=".">0.029 </td><td char=".">0.038 </td><td char=".">0.026 </td><td char=".">0.035 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Goias-Mbao  </td><td char=".">0.035 </td><td char=".">0.050 </td><td char=".">0.045 </td><td char=".">0.024 </td><td char=".">0.038 </td><td char=".">0.032 </td><td char=".">0.026 </td><td char=".">0.036 </td><td char=".">0.00534 </td><td char="."><strong>3.3</strong> </td></tr><tr><td><em>A. evenia</em> Bahia-Mbao  </td><td char=".">0.000 </td><td char=".">0.011 </td><td char=".">0.008 </td><td char=".">0.000 </td><td char=".">0.008 </td><td char=".">0.005 </td><td char=".">0.000 </td><td char=".">0.005 </td><td char=".">0.00534 </td><td char="."><strong>0.4</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw020-TF1"><div class="fn"><p> *Average divergence rate obtained by using the known 49 MYA divergence time between <em>Aeschynomene</em> and <em>Arachis</em> . <a reveal-id="dsw020-B6" data-open="dsw020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Numbers in bold are inferred values. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198582" target="_blank">View Large</a></div></div><p> This enabled an estimate of divergence time between the two mapping parents that belong to <em>A. evenia</em> ssp. <em>evenia</em> var. <em>evenia</em> and var. <em>pauciciliata</em> at ∼0.4 Mya, reflecting both the low difference observed in polymorphism and the cross-compatibility between the two varieties. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> In contrast, a much older divergence time of ∼3.3 Mya was deduced for <em>A. evenia</em> ssp. <em>evenia</em> – <em>A. evenia</em> ssp. <em>serrulata</em> . This is in agreement with the pronounced genetic differentiation of the two <em>A. evenia</em> subspecies evidenced by the difficulty of making successful crosses and the almost completely sterile hybrids. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> This divergence time is similar to the ∼3.5 Mya obtained for <em>A. duranensis</em> – <em>A. ipaensis</em> , which are genetically isolated, thus supporting the idea that the two <em>A. evenia</em> subspecies are at the final stage of the speciation process. Interestingly, the value calculated for the <em>Arachis</em> species couple is in accordance with other estimates (2.1–3.5 Mya). <a reveal-id="dsw020-B18" data-open="dsw020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw020-B42" data-open="dsw020-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Therefore, our approach seems to be valid to provide a timeframe for <em>A. evenia</em> evolution. </p>                            <h4 scrollto-destination=87198586 id="87198586" class="section-title" >3.4.2 Genome conservation and evolution with <em>Arachis ipaensis</em></h4>
<p> Several studies have undertaken comparative genomics within the Papilionoids revealing that the degree of synteny is correlated with phylogenetic distance of the legume species. <a reveal-id="dsw020-B9 dsw020-B10 dsw020-B11 dsw020-B12 dsw020-B13 dsw020-B14 dsw020-B15 dsw020-B16 dsw020-B17" data-open="dsw020-B9 dsw020-B10 dsw020-B11 dsw020-B12 dsw020-B13 dsw020-B14 dsw020-B15 dsw020-B16 dsw020-B17" class="link link-ref link-reveal xref-bibr"><sup>9–17</sup></a> The first glimpses of genome evolution within the Dalbergioid clade were only recently obtained by the genome sequencing of the two putative genome donors of peanut. This revealed a mostly one-to-one correspondence between the two species, with the genome of <em>A. duranensis</em> , but not of <em>A. ipaensis</em> , having undergone several major rearrangements. <a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> To perform the <em>Aeschynomene-Arachis</em> genome comparison, we made use of the gene-based nature of the markers mapped on the genetic map of <em>A. evenia</em> (2 <em>n</em>  = 20) to locate orthologous sequences on the chromosomes of <em>A. duranensis</em> and <em>A. ipaensis</em> (2 <em>n</em>  = 20) ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> ). Although all the transcripts used to develop the markers did not necessarily contain predicted coding reading frames, ∼97% of them (353 out of 364) showed significant matches to <em>Arachis</em> sequences. To facilitate comparison, the <em>Aeschynomene</em> linkage groups have been numbered and oriented to best match the corresponding <em>Arachis</em> chromosomes ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> A,B). </p><p> When the <em>A. evenia</em> linkage groups and the <em>A</em> . <em>ipaensis</em> chromosomes were aligned, extensive stretches of shared colinearity between them were evident ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> , <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> A). Some linkage groups of <em>A. evenia</em> showed hits mainly with one chromosome of <em>A. ipaensis</em> , notably AeLG1 that matches Araip.A01, AeLG2 with Araip.A02, AeLG5 with Araip.05, AeLG6 with Araip.06 and AeLG9 with Araip.09 ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> ). However, detailed analysis of macrosynteny between homologous chromosomes evidence internal rearrangements such as translocations and inversions ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a> ). In addition, even though <em>A. evenia</em> and <em>A. ipaensis</em> share the same base chromosome number of 10, one-to-one relationships do not hold true for the remaining <em>A. evenia</em> linkage groups. Instead, they appear to be composed of segmental syntenic blocks matching different chromosomal positions in <em>A. ipaensis</em> as exemplified by AeLG4 that contains three conserved blocks found in the chromosomes Araip.A03, A04 and A10. It is worth noting that some gaps observed within <em>Aeschynomene</em> linkage groups are coincident with syntenic block junctions and thus may correspond to more variable regions prone to genome restructuring as suggested earlier. <a reveal-id="dsw020-B4" data-open="dsw020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In addition, <em>Aeschynomene</em> loci mostly match and cover the euchromatic arms of <em>Arachis</em> chromosomes while only few loci map pericentromeric regions ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> ). <a reveal-id="dsw020-B18" data-open="dsw020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw020-B22" data-open="dsw020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> This observation is consistent with all the euchromatic regions of the <em>A. evenia</em> genome being captured by the gene-based markers composing the genetic map. </p>                        <a id="87198591" scrollto-destination="87198591"></a>
<div id="dsw020-F7" data-id="dsw020-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f7p.png?Expires=2147483647&Signature=ZQzQUkAc-hpF9lzGoRmjYggikpHyBsPg7ZajC1L4~7uhIpLqS75nGuzumW-B1pj9slEXYK2zW-yfLFhXydS2cKuM1FzIVlbaGgNy-Jbmq2L-zneJct9w6QNMf3ioOK6w9o3rmVooDhuZE3Ggkgz3frl2FTv9Yj9K1p51IaGyGtQxihdHaqQfZEgJtEu9UMP6n2KtPLuiCbEatFnX4I6EDS91aXzDOZ-oXz60On8y~oBUFnvWGD5b3tiryYvRvePiEcbvPWUsXivHgRaCOqc3BJL0Dbc7cgenWNA3zQVTAkDHxocLkPdrJqJCAG3whR1yP9RNMFaW4SgV4Wi0zjqWWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Syntenic relationships of A. evenia with the sequenced Dalbergioid legume Arachis ipaiensis . (A) Comparison of the genetic map of A. evenia with the genome sequence of A. ipaensis . Circled bars on the left side correspond to the linkage groups of A. evenia while those on the right side represent the chromosomes of A. ipaensis . Homologous loci are connected by lines and spots correspond to symbiotic genes. Spots indicate conserved macrosyntenic locations and asterisks non-conserved ones. (B) Detailed example of macrosynteny between A. evenia and A. ipaensis. Alignment of linkage groups AeLG1 and AeLG8 from the developed genetic map of A. evenia with the corresponding A1 and A8 chromosomes of A. ipaensis . Dot lines connect orthologous loci between the two species. The dot line link the non-macrosyntenic location of the AeCcaMK gene. " path-from-xml="dsw020f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198591/dsw020f7p.png" path-from-xml="dsw020f7p.png" target="_blank">View large</a><a section="87198591" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f7p.png?Expires=2147483647&Signature=vAxErl9j1Ls35L6PaMHYNskwIETaaVNYyuvJfSYFnMRDt9Fl-ZtU6ClMBQjuA2yHqRMhJfubkoIRSmhJWyEZSYCIPViKuiv0GlimV95cJaGbQ1TLVPnEE9WC9MHF~C7-rORGPXNza7Y5Y3t2Rfq6iTY68fbspnu9MaUtvZEd-ytx-rgUerkZbzHBRP0-OJO~Q8AHrwzEADWyPYH5P96QWk6jo5WlbNP1gI8hICJ6O3B7Iau0LEP~EyCjHi5y81zoGpTJXIvhUwfStlaIvXiv46~jHrCXEN3FfDMx8uy05YlSSoZQXgzyUiR7REczNQUjtlA8N4tPlC~jPzbr1xBn6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198591&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Syntenic relationships of <em>A. evenia</em> with the sequenced Dalbergioid legume <em>Arachis ipaiensis</em> . (A) Comparison of the genetic map of <em>A. evenia</em> with the genome sequence of <em>A. ipaensis</em> . Circled bars on the left side correspond to the linkage groups of <em>A. evenia</em> while those on the right side represent the chromosomes of <em>A. ipaensis</em> . Homologous loci are connected by lines and spots correspond to symbiotic genes. Spots indicate conserved macrosyntenic locations and asterisks non-conserved ones. (B) Detailed example of macrosynteny between <em>A. evenia</em> and <em>A. ipaensis.</em> Alignment of linkage groups AeLG1 and AeLG8 from the developed genetic map of <em>A. evenia</em> with the corresponding A1 and A8 chromosomes of <em>A. ipaensis</em> . Dot lines connect orthologous loci between the two species. The dot line link the non-macrosyntenic location of the <em>AeCcaMK</em> gene. </p></div></div><div content-id="dsw020-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/m_dsw020f7p.png?Expires=2147483647&Signature=ZQzQUkAc-hpF9lzGoRmjYggikpHyBsPg7ZajC1L4~7uhIpLqS75nGuzumW-B1pj9slEXYK2zW-yfLFhXydS2cKuM1FzIVlbaGgNy-Jbmq2L-zneJct9w6QNMf3ioOK6w9o3rmVooDhuZE3Ggkgz3frl2FTv9Yj9K1p51IaGyGtQxihdHaqQfZEgJtEu9UMP6n2KtPLuiCbEatFnX4I6EDS91aXzDOZ-oXz60On8y~oBUFnvWGD5b3tiryYvRvePiEcbvPWUsXivHgRaCOqc3BJL0Dbc7cgenWNA3zQVTAkDHxocLkPdrJqJCAG3whR1yP9RNMFaW4SgV4Wi0zjqWWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Syntenic relationships of A. evenia with the sequenced Dalbergioid legume Arachis ipaiensis . (A) Comparison of the genetic map of A. evenia with the genome sequence of A. ipaensis . Circled bars on the left side correspond to the linkage groups of A. evenia while those on the right side represent the chromosomes of A. ipaensis . Homologous loci are connected by lines and spots correspond to symbiotic genes. Spots indicate conserved macrosyntenic locations and asterisks non-conserved ones. (B) Detailed example of macrosynteny between A. evenia and A. ipaensis. Alignment of linkage groups AeLG1 and AeLG8 from the developed genetic map of A. evenia with the corresponding A1 and A8 chromosomes of A. ipaensis . Dot lines connect orthologous loci between the two species. The dot line link the non-macrosyntenic location of the AeCcaMK gene. " path-from-xml="dsw020f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198591/dsw020f7p.png" path-from-xml="dsw020f7p.png" target="_blank">View large</a><a section="87198591" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020f7p.png?Expires=2147483647&Signature=vAxErl9j1Ls35L6PaMHYNskwIETaaVNYyuvJfSYFnMRDt9Fl-ZtU6ClMBQjuA2yHqRMhJfubkoIRSmhJWyEZSYCIPViKuiv0GlimV95cJaGbQ1TLVPnEE9WC9MHF~C7-rORGPXNza7Y5Y3t2Rfq6iTY68fbspnu9MaUtvZEd-ytx-rgUerkZbzHBRP0-OJO~Q8AHrwzEADWyPYH5P96QWk6jo5WlbNP1gI8hICJ6O3B7Iau0LEP~EyCjHi5y81zoGpTJXIvhUwfStlaIvXiv46~jHrCXEN3FfDMx8uy05YlSSoZQXgzyUiR7REczNQUjtlA8N4tPlC~jPzbr1xBn6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198591&ar=2469965&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw020f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Syntenic relationships of <em>A. evenia</em> with the sequenced Dalbergioid legume <em>Arachis ipaiensis</em> . (A) Comparison of the genetic map of <em>A. evenia</em> with the genome sequence of <em>A. ipaensis</em> . Circled bars on the left side correspond to the linkage groups of <em>A. evenia</em> while those on the right side represent the chromosomes of <em>A. ipaensis</em> . Homologous loci are connected by lines and spots correspond to symbiotic genes. Spots indicate conserved macrosyntenic locations and asterisks non-conserved ones. (B) Detailed example of macrosynteny between <em>A. evenia</em> and <em>A. ipaensis.</em> Alignment of linkage groups AeLG1 and AeLG8 from the developed genetic map of <em>A. evenia</em> with the corresponding A1 and A8 chromosomes of <em>A. ipaensis</em> . Dot lines connect orthologous loci between the two species. The dot line link the non-macrosyntenic location of the <em>AeCcaMK</em> gene. </p></div></div>                            <h4 scrollto-destination=87198596 id="87198596" class="section-title" >3.4.3 Macrosynteny analysis for the symbiotic genes</h4>
<p> Considering the divergence time between <em>Aeschynomene</em> and <em>Arachis</em> (∼49 Mya) relative to the isolation time of the Dalbergioid clade from the Phaseolids and Galegoids (∼55 Mya), along with the 3 to 3.5-fold larger genome of <em>A. duranensis</em> and <em>A. ipaensis</em> (1,250 and 1,560 Mb, respectively) compared to <em>A. evenia</em> (415 Mb), the high level of macrosynteny documented here between <em>Arachis</em> and <em>Aeschynomene</em> is remarkable. Much of the expansion in <em>Arachis</em> genomes may be due to retroelements, but it has done little to disrupt macrosynteny that is dominated by large chromosome arms-size rearrangements. Such macrosynteny was found to be maintained for the symbiotic genes mapped onto the <em>A. evenia</em> linkage groups, suggesting that true orthologous relationships were identified ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> ). One noticeable exception concerns the <em>CCaMK</em> gene that is mapped onto AeLG8 and is positioned between two syntenic blocks, while its <em>A. ipaensis</em> ortholog locates on Araip.A01 chromosome within a block that is syntenic to AeLG1 ( <a reveal-id="dsw020-F7" data-open="dsw020-F7" class="link link-reveal link-table xref-fig">Fig. 7A,B</a> ). Although it is not possible to determine the extent to which macrosyntenic relationships identified by genetic mapping are indicative of conserved microsynteny, it is interesting to note that a retrotransposon insertion site was previously identified 564-bp upstream of the <em>AeCCamK</em> gene. <a reveal-id="dsw020-B34" data-open="dsw020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Since retrotransposons are known to be involved in genome restructuring and also to induce gene mobility, further studies may help determining their potential impact on the genomic environment of this key symbiotic gene in <em>A. evenia</em> . </p>                            <h2 scrollto-destination=87198599 id="87198599" class="section-title" >4. Conclusions and perspectives</h2>
<p> To advance in our understanding of the NF-independent symbiosis in the model legume <em>A. evenia</em> , we developed a genetic map incorporating expressed genes that are likely to be involved in nodulation. Previous characterization of different genotypes in <em>A. evenia</em> allowed us to select the Mbao genotype as a reference for molecular genetics and the Bahia genotype as a suitable crossing parent to develop a F <sub>2</sub> mapping population. Transcriptome <em>in silico</em> analysis for the two parental genotypes enabled an efficient selection of polymorphic molecular markers that were used to genotype the mapping population. The choice of genotyping methods, i.e. multiplexing and capillary sequencing for SSR and INDEL markers along with the KASP assays for SNP markers resulted in 99.4% complete genotyping data. The genetic map was resolved in 10 robust linkage groups that most likely represented the 10 chromosome pairs in <em>A. evenia</em> . The availability of a high-density map containing easy-to-use and cost-effective markers will facilitate gene and QTL mapping work in <em>A. evenia</em> . </p><p> Given the gene-based nature of the markers used, this genetic map provided the first insights into the structure of the genome gene-space and, in particular, uncovered the distribution of expressed orthologs of known symbiotic genes. Comparative genomic analysis with <em>Arachis</em> further evidenced that the <em>Aeschynomene</em> genome constituted of macrosyntenic blocks. Therefore, this genetic map of <em>Aeschynomene</em> could form the basis for future efforts in the scaffold anchoring and assembly in an <em>Aeschynomene</em> genome sequencing initiative. The availability of a whole genome sequence is now feasible thanks to the next-generation sequencing technologies. This will allow much better comparison with <em>Arachis</em> to characterize genome evolution within the Dalbergioid clade. In particular, microsynteny analysis will allow investigating if chromosome rearrangements and transposable elements had an impact on certain symbiotic genes as suspected for the <em>AeCCaMK</em> gene and those that could not be found in the transcriptome datasets. Understanding how these symbiotic genes evolved (by changes of gene expression or by gene loss) in a NF-independent context would be an important advance in understanding the evolution of nodulation. </p><p> In addition, an <em>a priori</em> identification of symbiotic genes in <em>A. evenia</em> using a mutagenesis approach should be a powerful tool. In this perspective, a strategy combining gene mapping with sequencing at gene or whole genome-level will allow an efficient identification of new alleles and new symbiotic loci among mutant lines, as has been successfully performed in <em>M. truncatula</em> and <em>L. japonicus</em> . <a reveal-id="dsw020-B31" data-open="dsw020-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw020-B43 dsw020-B44 dsw020-B45 dsw020-B46" data-open="dsw020-B43 dsw020-B44 dsw020-B45 dsw020-B46" class="link link-ref link-reveal xref-bibr"><sup>43–46</sup></a> New methods, such as SHOREmap and MutMap, now integrate simultaneous mapping and mutation identification in a single step by deep whole-genome resequencing of pooled DNA from a F <sub>2</sub> mutant population. <a reveal-id="dsw020-B47" data-open="dsw020-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw020-B48" data-open="dsw020-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> This population can be obtained either by crossing the mutant to the mapping parent or to the original WT parental line used for mutagenesis. In the latter case, mapping relies on the detection of mutagenesis-induced polymorphisms. The similar genetic backgrounds present the advantage of both simplifying segregation patterns and suppressing the occurrence of distortions. As a result, applying this strategy to <em>A. evenia</em> , the identification of new symbiotic genes should be straightforward and should facilitate the deciphering of the molecular mechanisms of the NF-independent process. </p>                            <h2 scrollto-destination=87198606 id="87198606" class="section-title" >Acknowledgements</h2>
<p>We thank A.M. Risterucci and F. de Bellis (AGAP, CIRAD, France) for valuable suggestions on genetic map construction and L. Legrand (LIPM, INRA/CNRS, France) for informatics assistance.</p>                            <h2 scrollto-destination=87198610 id="87198610" class="section-title" >Accession numbers</h2>
<p>SRR3276128, SRR3285082</p>                            <h2 scrollto-destination=87198613 id="87198613" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=87198618 id="87198618" class="section-title" >Funding</h2>
<p>This work was supported by Agropolis Fondation through the « Investissements d’avenir » programme (ANR-10-LABX-0001-01) under the reference ID “AeschyMap” AA1202-009, and by a grant from the French National Research Agency (ANR-AeschyNod-14-CE19-0005-01). Funding to pay the Open Access publication charges for this article was provided by the French National Research Agency.</p>                            <h2 scrollto-destination=87198622 id="87198622" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw020-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Graham</div>
               <div class="given-names">P.H.</div>
            </div>
             <div class="name">
              <div class="surname">Vance</div>
               <div class="given-names">C.P.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Legumes: importance and constraints to greater use,</div><div class="source">Plant Physiol</div> 
            ., 
            <div class="volume">131</div>
            , 
            <div class="fpage">872</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%3A%20importance%20and%20constraints%20to%20greater%20use%2C&amp;author=P.H.%20Graham&amp;author=C.P.%20Vance&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.017004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.017004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.017004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sprent</div>
               <div class="given-names">J.I.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Evolving ideas of legume evolution and diversity: a taxonomic perspective on the occurrence of nodulation</div>
            . 
            <div class="source">New Phytol</div> 
            ., 
            <div class="volume">174</div>
            , 
            <div class="fpage">11</div>
            –
            <div class="lpage">25</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolving%20ideas%20of%20legume%20evolution%20and%20diversity%3A%20a%20taxonomic%20perspective%20on%20the%20occurrence%20of%20nodulation&amp;author=J.I.%20Sprent&amp;publication_year=2007&amp;journal=New%20Phytol&amp;volume=174&amp;pages=11-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2007.02015.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2007.02015.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2007.02015.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17335493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Peterson</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Slovin</div>
               <div class="given-names">J.P.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">A simplified method for differential staining of aborted and non-aborted pollen grains</div>
            . 
            <div class="source">Int. J. Plant Biol</div> 
            ., 
            <div class="volume">1</div>
            :
            <div class="fpage">e13</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simplified%20method%20for%20differential%20staining%20of%20aborted%20and%20non-aborted%20pollen%20grains&amp;author=R.%20Peterson&amp;author=J.P.%20Slovin&amp;author=C.%20Chen&amp;publication_year=2010&amp;journal=Int.%20J.%20Plant%20Biol&amp;volume=1&amp;pages=e13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4081/pb.2010.e13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4081%2Fpb.2010.e13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4081%2Fpb.2010.e13"> </span></div> </div></div></div></div></div><div content-id="dsw020-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bertioli</div>
               <div class="given-names">D.J.</div>
            </div>
             <div class="name">
              <div class="surname">Moretzsohn</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Madsen</div>
               <div class="given-names">L.H.</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2009</div>
            , 
            <div class="article-title">
              An analysis of synteny of 
              <em>Arachis</em>
               with 
              <em>Lotus</em>
               and 
              <em>Medicago</em>
               sheds new light on the structure, stability and evolution of legume genomes
            </div>
            . 
            <div class="source">BMC Genomics</div> 
            , 
            <div class="volume">10</div>
            , 
            <div class="fpage">45</div>
            –
            <div class="lpage">55</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20synteny%20of%20Arachis%20with%20Lotus%20and%20Medicago%20sheds%20new%20light%20on%20the%20structure%2C%20stability%20and%20evolution%20of%20legume%20genomes&amp;author=D.J.%20Bertioli&amp;author=M.C.%20Moretzsohn&amp;author=L.H.%20Madsen&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=45-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Held</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Hossain</div>
               <div class="given-names">M.S.</div>
            </div>
             <div class="name">
              <div class="surname">Yokota</div>
               <div class="given-names">K.,</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Common and not so common entry</div>
            . 
            <div class="source">Trends Plant Sci</div> 
            ., 
            <div class="volume">15</div>
            , 
            <div class="fpage">540</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20and%20not%20so%20common%20entry&amp;author=M.%20Held&amp;author=M.S.%20Hossain&amp;author=K.%2C%20Yokota&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci&amp;volume=15&amp;pages=540-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.08.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.08.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.08.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20829094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lavin</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Herendeen</div>
               <div class="given-names">P.S.</div>
            </div>
             <div class="name">
              <div class="surname">Wojciechowski</div>
               <div class="given-names">M.F.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the Tertiary</div>
            . 
            <div class="source">Syst. Biol</div> 
            ., 
            <div class="volume">54</div>
            , 
            <div class="fpage">575</div>
            –
            <div class="lpage">594</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20rates%20analysis%20of%20Leguminosae%20implicates%20a%20rapid%20diversification%20of%20lineages%20during%20the%20Tertiary&amp;author=M.%20Lavin&amp;author=P.S.%20Herendeen&amp;author=M.F.%20Wojciechowski&amp;publication_year=2005&amp;journal=Syst.%20Biol&amp;volume=54&amp;pages=575-594" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590947131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590947131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150590947131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Doyle</div>
               <div class="given-names">J.J.</div>
            </div>
             <div class="name">
              <div class="surname">Luckow</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">The rest of the iceberg. Legume diversity and evolution in a phylogenetic context,</div><div class="source">Plant Physiol</div> 
            ., 
            <div class="volume">131</div>
            , 
            <div class="fpage">900</div>
            –
            <div class="lpage">10</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rest%20of%20the%20iceberg.%20Legume%20diversity%20and%20evolution%20in%20a%20phylogenetic%20context%2C&amp;author=J.J.%20Doyle&amp;author=M.A.%20Luckow&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=900-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.018150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.018150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.018150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oldroyd</div>
               <div class="given-names">G.E.D.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">J.D.</div>
            </div>
             <div class="name">
              <div class="surname">Poole</div>
               <div class="given-names">P.S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">The rules of engagement in the legume-rhizobial symbiosis,</div><div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">45</div>
            , 
            <div class="fpage">119</div>
            –
            <div class="lpage">44</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rules%20of%20engagement%20in%20the%20legume-rhizobial%20symbiosis%2C&amp;author=G.E.D.%20Oldroyd&amp;author=J.D.%20Murray&amp;author=P.S.%20Poole&amp;publication_year=2011&amp;journal=Annu.%20Rev.%20Genet&amp;volume=45&amp;pages=119-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110410-132549" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110410-132549" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110410-132549"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21838550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Choi</div>
               <div class="given-names">H.K.</div>
            </div>
             <div class="name">
              <div class="surname">Mun</div>
               <div class="given-names">J.H.</div>
            </div>
             <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2004</div>
            , 
            <div class="article-title">Estimating genome conservation between crop and model legume species,</div><div class="source">Proc. Natl. Acad. Sci</div> 
            ., 
            <div class="volume">101</div>
            , 
            <div class="fpage">15289</div>
            –
            <div class="lpage">94</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species%2C&amp;author=H.K.%20Choi&amp;author=J.H.%20Mun&amp;author=D.J.%20Kim&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=101&amp;pages=15289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div> </div></div></div></div></div><div content-id="dsw020-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Isobe</div>
               <div class="given-names">S.N.</div>
            </div>
             <div class="name">
              <div class="surname">Hisano</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Comparative genetic mapping and discovery of linkage disequilibrium across linkage groups in white clover (
              <em>Trifolium repens</em>
               L.)
            </div>
            . 
            <div class="source">G3</div> 
            , 
            <div class="volume">2</div>
            , 
            <div class="fpage">607</div>
            –
            <div class="lpage">17</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20mapping%20and%20discovery%20of%20linkage%20disequilibrium%20across%20linkage%20groups%20in%20white%20clover%20(%20Trifolium%20repens%20L.)&amp;author=S.N.%20Isobe&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=607-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.002600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.002600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.002600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22670230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Nakamura</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">
              Genome structure of the legume, 
              <em>Lotus japonicus</em></div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">15</div>
            , 
            <div class="fpage">227</div>
            –
            <div class="lpage">39</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schmutz</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Cannon</div>
               <div class="given-names">S.B.</div>
            </div>
             <div class="name">
              <div class="surname">Schlueter</div>
               <div class="given-names">J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Genome sequence of the palaeopolyploid soybean,</div><div class="source">Nature</div> 
            , 
            <div class="volume">463</div>
            , 
            <div class="fpage">178</div>
            –
            <div class="lpage">83</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean%2C&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Young</div>
               <div class="given-names">N.D.</div>
            </div>
             <div class="name">
              <div class="surname">Debellé</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Oldroyd</div>
               <div class="given-names">G.E.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            . 
            <div class="article-title">The Medicago genome provides insight into the evolution of rhizobial symbioses</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">480</div>
            , 
            <div class="fpage">520</div>
            –
            <div class="lpage">4</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses&amp;author=N.D.%20Young&amp;author=F.%20Debell%C3%A9&amp;author=G.E.%20Oldroyd&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=520-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/480162a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F480162a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F480162a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Draft genome sequence of pigeonpea (
              <em>Cajanus cajan</em>
              ), an orphan legume crop of resource-poor farmers
            </div>
            . 
            <div class="source">Nat. Biotechnol</div> 
            ., 
            <div class="volume">30</div>
            , 
            <div class="fpage">83</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(%20Cajanus%20cajan%20)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div> </div></div></div></div></div><div content-id="dsw020-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhu</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Choi</div>
               <div class="given-names">H.K.</div>
            </div>
             <div class="name">
              <div class="surname">Cook</div>
               <div class="given-names">D.R.</div>
            </div>
             <div class="name">
              <div class="surname">Shoemaker</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Bridging model and crop legumes through comparative genomics</div>
            . 
            <div class="source">Plant Physiol</div> 
            ., 
            <div class="volume">137</div>
            , 
            <div class="fpage">1189</div>
            –
            <div class="lpage">96</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bridging%20model%20and%20crop%20legumes%20through%20comparative%20genomics&amp;author=H.%20Zhu&amp;author=H.K.%20Choi&amp;author=D.R.%20Cook&amp;author=R.C.%20Shoemaker&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=137&amp;pages=1189-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.058891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.058891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.058891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15824281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>
             <div class="name">
              <div class="surname">Song</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Saxena</div>
               <div class="given-names">R.K.</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Draft genome sequence of Chickpea (
              <em>Cicer arietinum</em>
              ) provides a resource for trait improvement
            </div>
            . 
            <div class="source">Nat. Biotechnol</div> 
            ., 
            <div class="volume">31</div>
            , 
            <div class="fpage">240</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Chickpea%20(%20Cicer%20arietinum%20)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.K.%20Varshney&amp;author=C.%20Song&amp;author=R.K.%20Saxena&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=240-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div> </div></div></div></div></div><div content-id="dsw020-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schmutz</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Mc Clean</div>
               <div class="given-names">P.E.</div>
            </div>
             <div class="name">
              <div class="surname">Mamido</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">A reference genome for common bean and genome wide analysis of dual domestications</div>
            . 
            <div class="source">Nature Genet</div> 
            ., 
            <div class="volume">46</div>
            , 
            <div class="fpage">707</div>
            –
            <div class="lpage">13</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20reference%20genome%20for%20common%20bean%20and%20genome%20wide%20analysis%20of%20dual%20domestications&amp;author=J.%20Schmutz&amp;author=P.E.%20Mc%20Clean&amp;author=S.%20Mamido&amp;publication_year=2014&amp;journal=Nature%20Genet&amp;volume=46&amp;pages=707-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24908249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bertioli</div>
               <div class="given-names">D.J.</div>
            </div>
             <div class="name">
              <div class="surname">Cannon</div>
               <div class="given-names">S.B.</div>
            </div>
             <div class="name">
              <div class="surname">Froenicke</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2016</div>
            , 
            <div class="article-title">
              The genome sequences of 
              <em>Arachis duranensis</em>
               and 
              <em>Arachis ipaensis</em>
               the diploid ancestors of cultivated peanut
            </div>
            . 
            <div class="source">Nature Genet</div> 
            . doi:10.1038/ng.3517.
          </div></div></div></div><div content-id="dsw020-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Giraud</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Moulin</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Vallenet</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">316</div>
            , 
            <div class="fpage">1307</div>
            –
            <div class="lpage">1312</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%20symbioses%3A%20absence%20of%20Nod%20genes%20in%20photosynthetic%20bradyrhizobia&amp;author=E.%20Giraud&amp;author=L.%20Moulin&amp;author=D.%20Vallenet&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1307-1312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1139548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1139548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1139548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sprent</div>
               <div class="given-names">J.I.</div>
            </div>
             <div class="name">
              <div class="surname">James</div>
               <div class="given-names">E.K.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Legume-rhizobial symbiosis: an anorexic model?</div><div class="source">New Phytol</div> 
            ., 
            <div class="volume">179</div>
            , 
            <div class="fpage">3</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume-rhizobial%20symbiosis%3A%20an%20anorexic%20model%3F&amp;author=J.I.%20Sprent&amp;author=E.K.%20James&amp;publication_year=2008&amp;journal=New%20Phytol&amp;volume=179&amp;pages=3-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18557873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Arrighi</div>
               <div class="given-names">J.F.</div>
            </div>
             <div class="name">
              <div class="surname">Cartieaux</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Brown</div>
               <div class="given-names">S.C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title"><em>Aeschynomene evenia</em>
              , a model plant for studying the molecular genetics of the Nod-independent rhizobium-legume symbiosis
            </div>
            . 
            <div class="source">Mol. Plant-Microbe Interact</div> 
            ., 
            <div class="volume">25</div>
            , 
            <div class="fpage">851</div>
            –
            <div class="lpage">861</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aeschynomene%20evenia%20%2C%20a%20model%20plant%20for%20studying%20the%20molecular%20genetics%20of%20the%20Nod-independent%20rhizobium-legume%20symbiosis&amp;author=J.F.%20Arrighi&amp;author=F.%20Cartieaux&amp;author=S.C.%20Brown&amp;publication_year=2012&amp;journal=Mol.%20Plant-Microbe%20Interact&amp;volume=25&amp;pages=851-861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-02-12-0045-TA" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-02-12-0045-TA" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-02-12-0045-TA"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22475377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Arrighi</div>
               <div class="given-names">J.F.</div>
            </div>
             <div class="name">
              <div class="surname">Cartieaux</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Chaintreuil</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Genotype delimitation in the Nod-independent model legume 
              <em>Aeschynomene evenia</em></div>
            . 
            <div class="source">PloS One</div> 
            , 
            <div class="volume">8</div>
            , 
            <div class="fpage">e63836</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%20delimitation%20in%20the%20Nod-independent%20model%20legume%20Aeschynomene%20evenia&amp;author=J.F.%20Arrighi&amp;author=F.%20Cartieaux&amp;author=C.%20Chaintreuil&amp;publication_year=2013&amp;journal=PloS%20One&amp;volume=8&amp;pages=e63836" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063836" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063836" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063836"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23717496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rubio-Piña</div>
               <div class="given-names">J.A.</div>
            </div>
             <div class="name">
              <div class="surname">Zapata-Pérez</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Isolation of total RNA from tissues rich in polyphenols and polysaccharides of mangrove plants</div>
            . 
            <div class="source">Plant Biotech</div> 
            ., doi:10.2225/Vol02304-issue5-fulltext-8.
          </div></div></div></div><div content-id="dsw020-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Combes</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Dereeper</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Severac</div>
               <div class="given-names">D.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Contribution of subgenomes to the transcriptomes and their intertwined regulation in the allopolyploid 
              <em>Coffea arabica</em>
               grown at contrasted temperatures
            </div>
            . 
            <div class="source">New Phytol</div> 
            ., 
            <div class="volume">200</div>
            , 
            <div class="fpage">251</div>
            –
            <div class="lpage">60</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contribution%20of%20subgenomes%20to%20the%20transcriptomes%20and%20their%20intertwined%20regulation%20in%20the%20allopolyploid%20Coffea%20arabica%20grown%20at%20contrasted%20temperatures&amp;author=M.C.%20Combes&amp;author=A.%20Dereeper&amp;author=D.%20Severac&amp;publication_year=2013&amp;journal=New%20Phytol&amp;volume=200&amp;pages=251-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23790161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Allegre</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Argoult</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Boccara</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of 
              <em>Theobroma cacao</em>
               L
            </div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">19</div>
            , 
            <div class="fpage">23</div>
            –
            <div class="lpage">35</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20mapping%20of%20a%20new%20expressed%20sequence%20tag-single%20nucleotide%20polymorphism%20and%20simple%20sequence%20repeat%20panel%20for%20large-scale%20genetic%20studies%20and%20breeding%20of%20Theobroma%20cacao%20L&amp;author=M.%20Allegre&amp;author=X.%20Argoult&amp;author=M.%20Boccara&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=23-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22210604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Thoquet</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Ghérardi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Journet</div>
               <div class="given-names">E.P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">
              The molecular genetic linkage map of the model legume 
              <em>Medicago truncatula</em>
              : an essential tool for comparative legume genomics and the isolation of agronomically important genes
            </div>
            . 
            <div class="source">BMC Plant Biology</div> 
            , 
            <div class="volume">2</div>
            , 
            <div class="fpage">1</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20genetic%20linkage%20map%20of%20the%20model%20legume%20Medicago%20truncatula%20%3A%20an%20essential%20tool%20for%20comparative%20legume%20genomics%20and%20the%20isolation%20of%20agronomically%20important%20genes&amp;author=P.%20Thoquet&amp;author=M.%20Gh%C3%A9rardi&amp;author=E.P.%20Journet&amp;publication_year=2002&amp;journal=BMC%20Plant%20Biology&amp;volume=2&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-2-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-2-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-2-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11825338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gourion</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Berrabah</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Ratet</div>
               <div class="given-names">P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">Rhizobium-legume symbioses: the crucial role of plant immunity</div>
            . 
            <div class="source">Trends Plant Sci</div> 
            ., 
            <div class="volume">20</div>
            , 
            <div class="fpage">186</div>
            –
            <div class="lpage">94</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhizobium-legume%20symbioses%3A%20the%20crucial%20role%20of%20plant%20immunity&amp;author=B.%20Gourion&amp;author=F.%20Berrabah&amp;author=P.%20Ratet&amp;publication_year=2014&amp;journal=Trends%20Plant%20Sci&amp;volume=20&amp;pages=186-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2014.11.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2014.11.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2014.11.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25543258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Reid</div>
               <div class="given-names">D.E.</div>
            </div>
             <div class="name">
              <div class="surname">Ferguson</div>
               <div class="given-names">B.J.</div>
            </div>
             <div class="name">
              <div class="surname">Hayashi</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Molecular mechanisms controlling legume autoregulation of nodulation</div>
            . 
            <div class="source">Ann. Bot</div> 
            ., 
            <div class="volume">108</div>
            , 
            <div class="fpage">789</div>
            –
            <div class="lpage">95</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20controlling%20legume%20autoregulation%20of%20nodulation&amp;author=D.E.%20Reid&amp;author=B.J.%20Ferguson&amp;author=S.%20Hayashi&amp;publication_year=2011&amp;journal=Ann.%20Bot&amp;volume=108&amp;pages=789-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcr205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcr205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcr205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21856632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Couzigou</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Zhukov</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Mondy</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              NODULE ROOT and COCHLEATA maintain nodule development and are legume orthologs of 
              <em>Arabidopsis</em>
               BLADE-ON-PETIOLE genes
            </div>
            . 
            <div class="source">Plant Cell</div> 
            ., 
            <div class="volume">24</div>
            , 
            <div class="fpage">4498</div>
            –
            <div class="lpage">510</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NODULE%20ROOT%20and%20COCHLEATA%20maintain%20nodule%20development%20and%20are%20legume%20orthologs%20of%20Arabidopsis%20BLADE-ON-PETIOLE%20genes&amp;author=J.M.%20Couzigou&amp;author=V.%20Zhukov&amp;author=S.%20Mondy&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=4498-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.103747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.103747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.103747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Suzaki</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Ito</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Yoro</div>
               <div class="given-names">E.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              Endoreduplication-mediated initiation of symbiotic organ development in 
              <em>Lotus japonicus</em></div>
            . 
            <div class="source">Development</div> 
            , 
            <div class="volume">141</div>
            , 
            <div class="fpage">2441</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endoreduplication-mediated%20initiation%20of%20symbiotic%20organ%20development%20in%20Lotus%20japonicus&amp;author=T.%20Suzaki&amp;author=M.%20Ito&amp;author=E.%20Yoro&amp;publication_year=2014&amp;journal=Development&amp;volume=141&amp;pages=2441-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.107946" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.107946" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.107946"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24850853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yoon</div>
               <div class="given-names">H.J.</div>
            </div>
             <div class="name">
              <div class="surname">Hossain</div>
               <div class="given-names">M.S.</div>
            </div>
             <div class="name">
              <div class="surname">Held</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title"><em>Lotus japonicus</em>
               SUNERGOS1 encodes a predicted subunit A of a DNA topoisomerase VI that is required for nodule differentiation and accommodation of rhizobial infection
            </div>
            . 
            <div class="source">Plant J</div> 
            ., 
            <div class="volume">78</div>
            , 
            <div class="fpage">811</div>
            –
            <div class="lpage">21</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20SUNERGOS1%20encodes%20a%20predicted%20subunit%20A%20of%20a%20DNA%20topoisomerase%20VI%20that%20is%20required%20for%20nodule%20differentiation%20and%20accommodation%20of%20rhizobial%20infection&amp;author=H.J.%20Yoon&amp;author=M.S.%20Hossain&amp;author=M.%20Held&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=78&amp;pages=811-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.78.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.78.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.78.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24661810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kawaharada</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kelly</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Nielsen</div>
               <div class="given-names">M.W.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Receptor-mediated exopolysaccharide perception controls bacterial infection</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">523</div>
            , 
            <div class="fpage">308</div>
            –
            <div class="lpage">12</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Receptor-mediated%20exopolysaccharide%20perception%20controls%20bacterial%20infection&amp;author=Y.%20Kawaharada&amp;author=S.%20Kelly&amp;author=M.W.%20Nielsen&amp;publication_year=2015&amp;journal=Nature&amp;volume=523&amp;pages=308-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature14611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature14611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature14611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26153863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fliegmann</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Canova</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Lachaud</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            . 
            <div class="article-title">
              Lipo-chitooligosaccharidic symbiotic signals are recognized by LysM receptor-like kinase LYR3 in the legume 
              <em>Medicago truncatula</em></div>
            . 
            <div class="source">ACS Chem. Biol</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">1900</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipo-chitooligosaccharidic%20symbiotic%20signals%20are%20recognized%20by%20LysM%20receptor-like%20kinase%20LYR3%20in%20the%20legume%20Medicago%20truncatula&amp;author=J.%20Fliegmann&amp;author=S.%20Canova&amp;author=C.%20Lachaud&amp;publication_year=2013&amp;journal=ACS%20Chem.%20Biol&amp;volume=8&amp;pages=1900-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/cb400369u" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fcb400369u" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fcb400369u"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23808871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fabre</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Gully</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Poitout</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              The Nod factor-independent nodulation in 
              <em>Aeschynomene evenia</em>
               required the common plant-microbe symbiotic “toolkit”
            </div>
            . 
            <div class="source">Plant Physiol</div> 
            ., 
            <div class="volume">169</div>
            , 
            <div class="fpage">2654</div>
            –
            <div class="lpage">64</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Nod%20factor-independent%20nodulation%20in%20Aeschynomene%20evenia%20required%20the%20common%20plant-microbe%20symbiotic%20%E2%80%9Ctoolkit%E2%80%9D&amp;author=S.%20Fabre&amp;author=D.%20Gully&amp;author=A.%20Poitout&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=2654-64" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26446590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Czernic</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Gully</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Cartieaux</div>
               <div class="given-names">F.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Convergent evolution of endosymbiont diffenrentiation in Dalbergioid and IRLC legumes mediated by nodule-specific cysteine-rich peptides</div>
            . 
            <div class="source">Plant Physiol</div> 
            ., 
            <div class="volume">169</div>
            , 
            <div class="fpage">1254</div>
            –
            <div class="lpage">65</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Convergent%20evolution%20of%20endosymbiont%20diffenrentiation%20in%20Dalbergioid%20and%20IRLC%20legumes%20mediated%20by%20nodule-specific%20cysteine-rich%20peptides&amp;author=P.%20Czernic&amp;author=D.%20Gully&amp;author=F.%20Cartieaux&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=1254-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.00584" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.00584" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.00584"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26286718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Xu</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">B.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">
              A SNP and SSR based genetic map of Asparagus Bean (
              <em>Vigna unguiculata</em>
               ssp. 
              <em>sesquipedialis</em>
              ) and comparison with the broader species
            </div>
            . 
            <div class="source">PLoS ONE</div> 
            , 
            <div class="volume">6</div>
            , 
            <div class="fpage">e15952</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20SNP%20and%20SSR%20based%20genetic%20map%20of%20Asparagus%20Bean%20(%20Vigna%20unguiculata%20ssp.%20sesquipedialis%20)%20and%20comparison%20with%20the%20broader%20species&amp;author=P.%20Xu&amp;author=X.%20Wu&amp;author=B.%20Wang&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e15952" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0015952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0015952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0015952"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21253606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharpe</div>
               <div class="given-names">A.G.</div>
            </div>
             <div class="name">
              <div class="surname">Ramsay</div>
               <div class="given-names">L</div>
            </div>
             <div class="name">
              <div class="surname">Sanderson</div>
               <div class="given-names">L.A.</div>
            </div>
          </div>
            , 
            et al.  
            , 
            <div class="article-title">Ancient orphan crop joins modern era: gene-based SNP discovery and mapping in lentil</div>
            . 
            <div class="source">BMC Genomics</div> 
            , 
            <div class="volume">14</div>
            , 
            <div class="fpage">192</div>
            .
          <div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23506258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kamphuis</div>
               <div class="given-names">L.G.</div>
            </div>
             <div class="name">
              <div class="surname">Williams</div>
               <div class="given-names">A.H.</div>
            </div>
             <div class="name">
              <div class="surname">D’Souza</div>
               <div class="given-names">N.K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">
              The 
              <em>Medicago truncatula</em>
               reference accession A17 has an aberrant chromosomal configuration
            </div>
            . 
            <div class="source">New Phytol</div> 
            ., 
            <div class="volume">174</div>
            , 
            <div class="fpage">299</div>
            –
            <div class="lpage">303</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20truncatula%20reference%20accession%20A17%20has%20an%20aberrant%20chromosomal%20configuration&amp;author=L.G.%20Kamphuis&amp;author=A.H.%20Williams&amp;author=N.K.%20D%E2%80%99Souza&amp;publication_year=2007&amp;journal=New%20Phytol&amp;volume=174&amp;pages=299-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2007.174.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2007.174.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2007.174.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17388892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hayashi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Miyahara</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2001</div>
            , 
            <div class="article-title">
              Construction of a genetic linkage map of the model legume 
              <em>Lotus japonicus</em>
               using an intraspecific F2 population
            </div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">301</div>
            –
            <div class="lpage">10</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20using%20an%20intraspecific%20F2%20population&amp;author=M.%20Hayashi&amp;author=A.%20Miyahara&amp;author=S.%20Sato&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=301-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Krusell</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Radutoiu</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">
              A genetic linkage map of the model legume 
              <em>Lotus japonicus</em>
               and strategies for fast mapping of new loci
            </div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">161</div>
            , 
            <div class="fpage">1673</div>
            –
            <div class="lpage">83</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20and%20strategies%20for%20fast%20mapping%20of%20new%20loci&amp;author=N.%20Sandal&amp;author=L.%20Krusell&amp;author=S.%20Radutoiu&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1673-83" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gujaria-Verma</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Vail</div>
               <div class="given-names">S.L.</div>
            </div>
             <div class="name">
              <div class="surname">Carrasquilla-Garcia</div>
               <div class="given-names">N.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              Genetic mapping of legume orthologs reveals high conservation of synteny between lentil species and the sequenced genomes of 
              <em>Medicago</em>
               and chickpea
            </div>
            . 
            <div class="source">Front. Plant. Sci</div> 
            ., 
            <div class="volume">5</div>
            , 
            <div class="fpage">676</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20legume%20orthologs%20reveals%20high%20conservation%20of%20synteny%20between%20lentil%20species%20and%20the%20sequenced%20genomes%20of%20Medicago%20and%20chickpea&amp;author=N.%20Gujaria-Verma&amp;author=S.L.%20Vail&amp;author=N.%20Carrasquilla-Garcia&amp;publication_year=2014&amp;journal=Front.%20Plant.%20Sci&amp;volume=5&amp;pages=676" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00676" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00676" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00676"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25538716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nielen</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Vigidal</div>
               <div class="given-names">B.S.</div>
            </div>
             <div class="name">
              <div class="surname">Leal-Bertioli</div>
               <div class="given-names">S.C.M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title"><em>Matita</em>
              , a new retroelement from peanut: characterization and evolutionary context in the light of the 
              <em>Arachis</em>
               A-B genome divergence
            </div>
            . 
            <div class="source">Mol. Genet. Genomics</div> 
            , 
            <div class="volume">287</div>
            , 
            <div class="fpage">21</div>
            –
            <div class="lpage">38</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Matita%20%2C%20a%20new%20retroelement%20from%20peanut%3A%20characterization%20and%20evolutionary%20context%20in%20the%20light%20of%20the%20Arachis%20A-B%20genome%20divergence&amp;author=S.%20Nielen&amp;author=B.S.%20Vigidal&amp;author=S.C.M.%20Leal-Bertioli&amp;publication_year=2012&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=287&amp;pages=21-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-011-0656-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-011-0656-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-011-0656-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22120641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Petersen</div>
               <div class="given-names">T.R.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2006</div>
            , 
            <div class="article-title">
              Genetics of symbiosis in 
              <em>Lotus japonicus</em>
              : recombinant inbred lines, comparative genomic maps, and map position of 35 symbiotic loci
            </div>
            . 
            <div class="source">MPMI</div> 
            , 
            <div class="volume">16</div>
            , 
            <div class="fpage">80</div>
            –
            <div class="lpage">91</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20symbiosis%20in%20Lotus%20japonicus%20%3A%20recombinant%20inbred%20lines%2C%20comparative%20genomic%20maps%2C%20and%20map%20position%20of%2035%20symbiotic%20loci&amp;author=N.%20Sandal&amp;author=T.R.%20Petersen&amp;author=J.%20Murray&amp;publication_year=2006&amp;journal=MPMI&amp;volume=16&amp;pages=80-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0080"> </span></div> </div></div></div></div></div><div content-id="dsw020-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Jin</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Rodriguez-Navarro</div>
               <div class="given-names">D.N.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              A set of 
              <em>Lotus japonicus</em>
               Gifu x 
              <em>Lotus burttii</em>
               recombinant inbred lines facilitates map-based cloning and QTL mapping
            </div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">19</div>
            , 
            <div class="fpage">317</div>
            –
            <div class="lpage">23</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20set%20of%20Lotus%20japonicus%20Gifu%20x%20Lotus%20burttii%20recombinant%20inbred%20lines%20facilitates%20map-based%20cloning%20and%20QTL%20mapping&amp;author=N.%20Sandal&amp;author=H.%20Jin&amp;author=D.N.%20Rodriguez-Navarro&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=317-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22619310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Karas</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Ross</div>
               <div class="given-names">L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2006</div>
            . 
            <div class="article-title">
              Genetic suppressors of the 
              <em>Lotus japonicus har1-1</em>
               hypernodulation phenotype
            </div>
            . 
            <div class="source">MPMI</div> 
            , 
            <div class="volume">19</div>
            , 
            <div class="fpage">1082</div>
            –
            <div class="lpage">91</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20suppressors%20of%20the%20Lotus%20japonicus%20har1-1%20hypernodulation%20phenotype&amp;author=J.%20Murray&amp;author=B.%20Karas&amp;author=L.%20Ross&amp;publication_year=2006&amp;journal=MPMI&amp;volume=19&amp;pages=1082-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-1082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-1082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-1082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17022172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Xi</div>
               <div class="given-names">J</div>
            </div>
          </div>
            ., 
            et al.  
            . 
            <div class="article-title">An antimicrobial peptide essential for bacterial survival in the nitrogen-fixing symbiosis</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">112</div>
            , 
            <div class="fpage">15238</div>
            –
            <div class="lpage">43</div>
            .
          <div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1500123112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1500123112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1500123112"> </span></div> </div></div></div></div></div><div content-id="dsw020-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schneeberger</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Ossowski</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Lanz</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2009</div>
            , 
            <div class="article-title">SHOREmap: simultaneous mapping and mutation identification by deep sequencing</div>
            . 
            <div class="source">Nature Methods</div> 
            , 
            <div class="volume">6</div>
            , 
            <div class="fpage">550</div>
            –
            <div class="lpage">1</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SHOREmap%3A%20simultaneous%20mapping%20and%20mutation%20identification%20by%20deep%20sequencing&amp;author=K.%20Schneeberger&amp;author=S.%20Ossowski&amp;author=C.%20Lanz&amp;publication_year=2009&amp;journal=Nature%20Methods&amp;volume=6&amp;pages=550-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth0809-550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth0809-550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth0809-550"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19644454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw020-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw020-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Aba</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Kosugi</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Yoshida</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">Genome sequencing reveals agronomically important loci in rice using MutMap</div>
            . 
            <div class="source">Nat. Biotech.</div> <div class="volume">30</div>
            , 
            <div class="fpage">174</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20reveals%20agronomically%20important%20loci%20in%20rice%20using%20MutMap&amp;author=A.%20Aba&amp;author=S.%20Kosugi&amp;author=K.%20Yoshida&amp;publication_year=2012&amp;journal=Nat.%20Biotech.&amp;volume=30&amp;pages=174-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2095"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a> ), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469965&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw020/2/dsw020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=y93nEI212uPMw24Xl4Kd-OcKIHD2UXvcSsVL0AwzbNmogfU2bu6jUISy~8wup5mX0QAUQcVAvus8UkPva9O5MqF6~7v2zi05vixtoXX5IysmHLNq9~OrGu8lHz8DOZUb9V6TqGZp0nHF0AFj1asg3GjVfA1zwH3MKZ8Fw~Xstz-YLSAxLTWKvfDoK8zeBAgJy52J0jhBEmYolajXKusP2cU4PGqIz2WQsVsuviDVKmDT-vX88c2-vh0Ywty5tvExVdYmKb9iGmsx9Tj4bY-SfgLMxFbfJy-iM9wLDj6OhiJU6b7v4cuPmNp4M1exvHfqRxbaYCpPj-1f7jCPXEtKMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">617</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469965"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f365%2f2469965">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000384768700006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F365%2F2469965" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469965'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" A gene-based map of the Nod factor-independent Aeschynomene evenia sheds new light on the evolution of nodulation and legume genomes " />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f365%2f2469965">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

