

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/3/103/577859","@type":"ScholarlyArticle","name":"Genome-Wide Analysis of  LIM  Gene Family in  Populus trichocarpa ,  Arabidopsis thaliana , and  Oryza sativa ","datePublished":"2007-06-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm013","keywords":["poplar","<italic>Arabidopsis</italic>","rice","LIM domain protein","tension wood"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/3/103/577859","author":[{"name":"Arnaud, Dominique","affiliation":"Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières ,  INRA-Orléans ,  Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex ,  France","@type":"Person"},{"name":"Déjardin, Annabelle","affiliation":"Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières ,  INRA-Orléans ,  Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex ,  France","@type":"Person"},{"name":"Leplé, Jean-Charles","affiliation":"Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières ,  INRA-Orléans ,  Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex ,  France","@type":"Person"},{"name":"Lesage-Descauses, Marie-Claude","affiliation":"Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières ,  INRA-Orléans ,  Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex ,  France","@type":"Person"},{"name":"Pilate, Gilles","affiliation":"Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières ,  INRA-Orléans ,  Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex ,  France","@type":"Person"}],"description":"Abstract.  In Eukaryotes, LIM proteins act as developmental regulators in basic cellular processes such as regulating the transcription or organizing the cytosk","pageStart":"103","pageEnd":"116","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Analysis of  LIM  Gene Family in  Populus trichocarpa ,  Arabidopsis thaliana , and  Oryza sativa ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Analysis of  LIM  Gene Family in  Populus trichocarpa ,  Arabidopsis thaliana , and  Oryza sativa " />
    <meta property="og:description" content="Abstract.  In Eukaryotes, LIM proteins act as developmental regulators in basic cellular processes such as regulating the transcription or organizing the cytosk" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/3/103/577859" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-06-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Arnaud, Dominique" /><meta name="citation_author_institution" content="Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières, INRA-Orléans, Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex, France" /><meta name="citation_author" content="Déjardin, Annabelle" /><meta name="citation_author_institution" content="Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières, INRA-Orléans, Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex, France" /><meta name="citation_author" content="Leplé, Jean-Charles" /><meta name="citation_author_institution" content="Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières, INRA-Orléans, Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex, France" /><meta name="citation_author" content="Lesage-Descauses, Marie-Claude" /><meta name="citation_author_institution" content="Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières, INRA-Orléans, Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex, France" /><meta name="citation_author" content="Pilate, Gilles" /><meta name="citation_author_institution" content="Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières, INRA-Orléans, Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex, France" /><meta name="citation_title" content="Genome-Wide Analysis of LIM Gene Family in Populus trichocarpa, Arabidopsis thaliana, and Oryza sativa" /><meta name="citation_firstpage" content="103" /><meta name="citation_lastpage" content="116" /><meta name="citation_doi" content="10.1093/dnares/dsm013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Novel cysteine-rich motif and homeodomain in the product of the Caenorhabditis elegans cell lineage gene lin-II; Nature;  citation_year=1990;  citation_volume=344;  citation_pages=876-879; " /><meta name="citation_reference" content="citation_title=Insulin gene enhancer binding protein Isl-1 is a member of a novel class of proteins containing both a homeo- and a Cys–His domain; Nature;  citation_year=1990;  citation_volume=344;  citation_pages=879-882; " /><meta name="citation_reference" content="citation_title=mec-3, a homeobox-containing gene that specifies differentiation of the touch receptor neurons in C. elegans; Cell;  citation_year=1988;  citation_volume=54;  citation_pages=5-16; " /><meta name="citation_reference" content="citation_title=LIM domains: multiple roles as adapters and functional modifiers in protein interactions; Trends Genet.;  citation_year=1998;  citation_volume=14;  citation_pages=156-162; " /><meta name="citation_reference" content="citation_title=The CRP/MLP/TLP family of LIM domain proteins: acting by connecting; BioEssays;  citation_year=2003;  citation_volume=25;  citation_pages=152-162; " /><meta name="citation_reference" content="citation_title=Structure of the carboxy-terminal LIM domain from the cysteine rich protein CRP; Nat. Struct. Biol.;  citation_year=1994;  citation_volume=1;  citation_pages=388-398; " /><meta name="citation_reference" content="citation_title=Molecular and expression analysis of a LIM protein gene family from flowering plants; Mol. Gen. Genet.;  citation_year=2000;  citation_volume=264;  citation_pages=257-267; " /><meta name="citation_reference" content="citation_title=Characterization of a pollen-specific cDNA from sunflower encoding a zinc finger protein; Plant J.;  citation_year=1992;  citation_volume=2;  citation_pages=713-721; " /><meta name="citation_reference" content="citation_title=A LIM motif is present in a pollen-specific protein; Plant Cell;  citation_year=1992;  citation_volume=4;  citation_pages=1465-1466; " /><meta name="citation_reference" content="citation_title=A LIM-domain protein from sunflower is localized to the cytoplasm and/or nucleus in a wide variety of tissues and is associated with the phragmoplast in dividing cells; Plant Mol. Biol.;  citation_year=2000;  citation_volume=42;  citation_pages=291-302; " /><meta name="citation_reference" content="citation_title=Is the LIM-domain protein HaWLIM1 associated with cortical microtubules in sunflower protoplasts?; Plant Cell Physiol.;  citation_year=2003;  citation_volume=44;  citation_pages=1055-1063; " /><meta name="citation_reference" content="citation_title=Tobacco WLIM1 is a novel F-Actin binding protein involved in actin cytoskeleton remodeling; Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=2194-2206; " /><meta name="citation_reference" content="citation_title=Differential localization of the LIM domain protein PLIM-1 in microspores and mature pollen grains from sunflower; Sex. Plant Reprod.;  citation_year=1999;  citation_volume=12;  citation_pages=60-65; " /><meta name="citation_reference" content="citation_title=The pollen-specific LIM protein PLIM-1 from sunflower binds nucleic acids in vitro; Sex. Plant Reprod.;  citation_year=1996;  citation_volume=9;  citation_pages=264-268; " /><meta name="citation_reference" content="citation_title=Functional analysis of tobacco LIM protein Ntlim1 involved in lignin biosynthesis; Plant J.;  citation_year=2000;  citation_volume=22;  citation_pages=289-301; " /><meta name="citation_reference" content="citation_title=Expressed sequence tags from poplar wood tissues—a comparative analysis from multiple libraries; Plant Biol.;  citation_year=2004;  citation_volume=6;  citation_pages=55-64; " /><meta name="citation_reference" content="citation_title=Biosynthesis of cellulose-enriched tension wood in Populus: global analysis of transcripts and metabolites identifies biochemical and developmental regulators in secondary wall biosynthesis; Plant J.;  citation_year=2006;  citation_volume=45;  citation_pages=144-165; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-1604; " /><meta name="citation_reference" content="citation_title=Poplar genes encoding fasciclin-like arabinogalactan proteins are highly expressed in tension wood; New Phytol.;  citation_year=2004;  citation_volume=164;  citation_pages=107-121; " /><meta name="citation_reference" content="citation_title=A Populus EST resource for plant functional genomics; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=13951-13956; " /><meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3497-3500; " /><meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences; Comput. Appl. Biosci.;  citation_year=1992;  citation_volume=8;  citation_pages=275-282; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; " /><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood; Syst. Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; " /><meta name="citation_reference" content="citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data; Mol. Biol. Evol.;  citation_year=1997;  citation_volume=14;  citation_pages=685-695; " /><meta name="citation_reference" content="citation_title=Tree View: an application to display phylogenetic trees on personal computers; Comput. Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=357-358; " /><meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers; Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1994;  citation_volume=2;  citation_pages=28-36; " /><meta name="citation_reference" content="citation_title=Genomics of hybrid poplar (Populus trichocarpa × deltoides) interacting with forest tent caterpillars (Malacosoma disstria): normalized and full-length cDNA libraries, expressed sequence tags, and a cDNA microarray for the study of insect-induced defences in poplar; Mol. Ecol.;  citation_year=2006;  citation_volume=15;  citation_pages=1275-1297; " /><meta name="citation_reference" content="citation_title=The genome sequence of black cottonwood (Populus trichocarpa) reveals 18 conserved cellulose synthase (CesA) genes; Planta;  citation_year=2005;  citation_volume=221;  citation_pages=739-746; " /><meta name="citation_reference" content="citation_title=EST data suggest that poplar is an ancient polyploid; New Phytol.;  citation_year=2005;  citation_volume=167;  citation_pages=165-170; " /><meta name="citation_reference" content="citation_title=Transcription factor genes with expression correlated to nitrate-related root plasticity of Arabidopsis thaliana; Plant Cell Environ.;  citation_year=2003;  citation_volume=26;  citation_pages=459-469; " /><meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice; Science;  citation_year=2003;  citation_volume=301;  citation_pages=376-379; " /><meta name="citation_reference" content="citation_title=Isolation and characterisation of two pollen-specific LIM domain protein cDNAs from Nicotiana tabacum; Sex. Plant Reprod.;  citation_year=2000;  citation_volume=12;  citation_pages=339-345; " /><meta name="citation_reference" content="citation_title=Analysis of the floral transcriptome uncovers new regulators of organ determination and gene families related to flower organ differentiation in Gerbera hybrida (Asteraceae); Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=475-486; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of cDNAs expressed in the early stages of flavonol-induced pollen germination in Petunia; Plant Physiol.;  citation_year=2000;  citation_volume=123;  citation_pages=699-710; " /><meta name="citation_reference" content="citation_title=Analysis of xylem formation in pine by cDNA sequencing; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=9693-9698; " /><meta name="citation_reference" content="citation_title=Coordinate repression of regulators of embryonic identity by PICKLE during germination in Arabidopsis; Plant J.;  citation_year=2003;  citation_volume=35;  citation_pages=33-43; " /><meta name="citation_reference" content="citation_title=Large-scale identification of leaf senescence-associated genes; Plant J.;  citation_year=2003;  citation_volume=36;  citation_pages=629-642; " /><meta name="citation_reference" content="citation_title=Transcriptional profiling of Arabidopsis tissues reveals the unique characteristics of the pollen transcriptome; Plant Physiol.;  citation_year=2003;  citation_volume=133;  citation_pages=713-725; " /><meta name="citation_reference" content="citation_title=cDNA microarray analysis of gene expression changes during pollination, pollen-tube elongation, fertilization, and early embryogenesis in rice pistils; Sex. Plant Reprod.;  citation_year=2005;  citation_volume=17;  citation_pages=269-275; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/3/103/17462710/dsm013.pdf" /><meta name="description" content="Abstract.  In Eukaryotes, LIM proteins act as developmental regulators in basic cellular processes such as regulating the transcription or organizing the cytosk" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/3/103/577859" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/3/103/577859" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/3/103/577859");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm013';
    </script>
<title>
	Genome-Wide Analysis of LIM Gene Family in Populus trichocarpa, Arabidopsis thaliana, and Oryza sativa | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./577859" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf1fPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcm5hdWQsIERvbWluaXF1ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVxdWlwZSDigJhYeWzDqG1l4oCZIFVuaXTDqSBBbcOpbGlvcmF0aW9uLCBHw6luw6l0aXF1ZSBldCBQaHlzaW9sb2dpZSBGb3Jlc3Rpw6hyZXMsIElOUkEtT3Jsw6lhbnMsIEF2ZW51ZSBkZSBsYSBQb21tZSBkZSBQaW4sIEJQIDIwNjE5IEFyZG9uLCBGLTQ1MTY2IE9saXZldCBDZWRleCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRMOpamFyZGluLCBBbm5hYmVsbGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFcXVpcGUg4oCYWHlsw6htZeKAmSBVbml0w6kgQW3DqWxpb3JhdGlvbiwgR8OpbsOpdGlxdWUgZXQgUGh5c2lvbG9naWUgRm9yZXN0acOocmVzLCBJTlJBLU9ybMOpYW5zLCBBdmVudWUgZGUgbGEgUG9tbWUgZGUgUGluLCBCUCAyMDYxOSBBcmRvbiwgRi00NTE2NiBPbGl2ZXQgQ2VkZXgsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlcGzDqSwgSmVhbi1DaGFybGVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRXF1aXBlIOKAmFh5bMOobWXigJkgVW5pdMOpIEFtw6lsaW9yYXRpb24sIEfDqW7DqXRpcXVlIGV0IFBoeXNpb2xvZ2llIEZvcmVzdGnDqHJlcywgSU5SQS1PcmzDqWFucywgQXZlbnVlIGRlIGxhIFBvbW1lIGRlIFBpbiwgQlAgMjA2MTkgQXJkb24sIEYtNDUxNjYgT2xpdmV0IENlZGV4LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZXNhZ2UtRGVzY2F1c2VzLCBNYXJpZS1DbGF1ZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFcXVpcGUg4oCYWHlsw6htZeKAmSBVbml0w6kgQW3DqWxpb3JhdGlvbiwgR8OpbsOpdGlxdWUgZXQgUGh5c2lvbG9naWUgRm9yZXN0acOocmVzLCBJTlJBLU9ybMOpYW5zLCBBdmVudWUgZGUgbGEgUG9tbWUgZGUgUGluLCBCUCAyMDYxOSBBcmRvbiwgRi00NTE2NiBPbGl2ZXQgQ2VkZXgsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBpbGF0ZSwgR2lsbGVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRXF1aXBlIOKAmFh5bMOobWXigJkgVW5pdMOpIEFtw6lsaW9yYXRpb24sIEfDqW7DqXRpcXVlIGV0IFBoeXNpb2xvZ2llIEZvcmVzdGnDqHJlcywgSU5SQS1PcmzDqWFucywgQXZlbnVlIGRlIGxhIFBvbW1lIGRlIFBpbiwgQlAgMjA2MTkgQXJkb24sIEYtNDUxNjYgT2xpdmV0IENlZGV4LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS1XaWRlIEFuYWx5c2lzIG9mIExJTSBHZW5lIEZhbWlseSBpbiBQb3B1bHVzIHRyaWNob2NhcnBhLCBBcmFiaWRvcHNpcyB0aGFsaWFuYSwgYW5kIE9yeXphIHNhdGl2YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNtMDEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDcvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBjeXN0ZWluZS1yaWNoIG1vdGlmIGFuZCBob21lb2RvbWFpbiBpbiB0aGUgcHJvZHVjdCBvZiB0aGUgQ2Flbm9yaGFiZGl0aXMgZWxlZ2FucyBjZWxsIGxpbmVhZ2UgZ2VuZSBsaW4tSUk7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDQ7ICBjaXRhdGlvbl9wYWdlcz04NzYtODc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnN1bGluIGdlbmUgZW5oYW5jZXIgYmluZGluZyBwcm90ZWluIElzbC0xIGlzIGEgbWVtYmVyIG9mIGEgbm92ZWwgY2xhc3Mgb2YgcHJvdGVpbnMgY29udGFpbmluZyBib3RoIGEgaG9tZW8tIGFuZCBhIEN5c+KAk0hpcyBkb21haW47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDQ7ICBjaXRhdGlvbl9wYWdlcz04NzktODgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1tZWMtMywgYSBob21lb2JveC1jb250YWluaW5nIGdlbmUgdGhhdCBzcGVjaWZpZXMgZGlmZmVyZW50aWF0aW9uIG9mIHRoZSB0b3VjaCByZWNlcHRvciBuZXVyb25zIGluIEMuIGVsZWdhbnM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MSU0gZG9tYWluczogbXVsdGlwbGUgcm9sZXMgYXMgYWRhcHRlcnMgYW5kIGZ1bmN0aW9uYWwgbW9kaWZpZXJzIGluIHByb3RlaW4gaW50ZXJhY3Rpb25zOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTU2LTE2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENSUC9NTFAvVExQIGZhbWlseSBvZiBMSU0gZG9tYWluIHByb3RlaW5zOiBhY3RpbmcgYnkgY29ubmVjdGluZzsgQmlvRXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTUyLTE2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIG9mIHRoZSBjYXJib3h5LXRlcm1pbmFsIExJTSBkb21haW4gZnJvbSB0aGUgY3lzdGVpbmUgcmljaCBwcm90ZWluIENSUDsgTmF0LiBTdHJ1Y3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0zODgtMzk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYW5kIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgYSBMSU0gcHJvdGVpbiBnZW5lIGZhbWlseSBmcm9tIGZsb3dlcmluZyBwbGFudHM7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjY0OyAgY2l0YXRpb25fcGFnZXM9MjU3LTI2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBhIHBvbGxlbi1zcGVjaWZpYyBjRE5BIGZyb20gc3VuZmxvd2VyIGVuY29kaW5nIGEgemluYyBmaW5nZXIgcHJvdGVpbjsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTcxMy03MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgTElNIG1vdGlmIGlzIHByZXNlbnQgaW4gYSBwb2xsZW4tc3BlY2lmaWMgcHJvdGVpbjsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTQ2NS0xNDY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIExJTS1kb21haW4gcHJvdGVpbiBmcm9tIHN1bmZsb3dlciBpcyBsb2NhbGl6ZWQgdG8gdGhlIGN5dG9wbGFzbSBhbmQvb3IgbnVjbGV1cyBpbiBhIHdpZGUgdmFyaWV0eSBvZiB0aXNzdWVzIGFuZCBpcyBhc3NvY2lhdGVkIHdpdGggdGhlIHBocmFnbW9wbGFzdCBpbiBkaXZpZGluZyBjZWxsczsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzIHRoZSBMSU0tZG9tYWluIHByb3RlaW4gSGFXTElNMSBhc3NvY2lhdGVkIHdpdGggY29ydGljYWwgbWljcm90dWJ1bGVzIGluIHN1bmZsb3dlciBwcm90b3BsYXN0cz87IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMDU1LTEwNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvYmFjY28gV0xJTTEgaXMgYSBub3ZlbCBGLUFjdGluIGJpbmRpbmcgcHJvdGVpbiBpbnZvbHZlZCBpbiBhY3RpbiBjeXRvc2tlbGV0b24gcmVtb2RlbGluZzsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTIxOTQtMjIwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGxvY2FsaXphdGlvbiBvZiB0aGUgTElNIGRvbWFpbiBwcm90ZWluIFBMSU0tMSBpbiBtaWNyb3Nwb3JlcyBhbmQgbWF0dXJlIHBvbGxlbiBncmFpbnMgZnJvbSBzdW5mbG93ZXI7IFNleC4gUGxhbnQgUmVwcm9kLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYwLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcG9sbGVuLXNwZWNpZmljIExJTSBwcm90ZWluIFBMSU0tMSBmcm9tIHN1bmZsb3dlciBiaW5kcyBudWNsZWljIGFjaWRzIGluIHZpdHJvOyBTZXguIFBsYW50IFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI2NC0yNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgdG9iYWNjbyBMSU0gcHJvdGVpbiBOdGxpbTEgaW52b2x2ZWQgaW4gbGlnbmluIGJpb3N5bnRoZXNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yODktMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzZWQgc2VxdWVuY2UgdGFncyBmcm9tIHBvcGxhciB3b29kIHRpc3N1ZXPigJRhIGNvbXBhcmF0aXZlIGFuYWx5c2lzIGZyb20gbXVsdGlwbGUgbGlicmFyaWVzOyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb3N5bnRoZXNpcyBvZiBjZWxsdWxvc2UtZW5yaWNoZWQgdGVuc2lvbiB3b29kIGluIFBvcHVsdXM6IGdsb2JhbCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0cyBhbmQgbWV0YWJvbGl0ZXMgaWRlbnRpZmllcyBiaW9jaGVtaWNhbCBhbmQgZGV2ZWxvcG1lbnRhbCByZWd1bGF0b3JzIGluIHNlY29uZGFyeSB3YWxsIGJpb3N5bnRoZXNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0xNDQtMTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIGJsYWNrIGNvdHRvbndvb2QsIFBvcHVsdXMgdHJpY2hvY2FycGEgKFRvcnIuICZhbXA7IEdyYXkpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtMTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wbGFyIGdlbmVzIGVuY29kaW5nIGZhc2NpY2xpbi1saWtlIGFyYWJpbm9nYWxhY3RhbiBwcm90ZWlucyBhcmUgaGlnaGx5IGV4cHJlc3NlZCBpbiB0ZW5zaW9uIHdvb2Q7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0xMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgUG9wdWx1cyBFU1QgcmVzb3VyY2UgZm9yIHBsYW50IGZ1bmN0aW9uYWwgZ2Vub21pY3M7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEzOTUxLTEzOTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgd2l0aCB0aGUgQ2x1c3RhbCBzZXJpZXMgb2YgcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM0OTctMzUwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJhcGlkIGdlbmVyYXRpb24gb2YgbXV0YXRpb24gZGF0YSBtYXRyaWNlcyBmcm9tIHByb3RlaW4gc2VxdWVuY2VzOyBDb21wdXQuIEFwcGwuIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI3NS0yODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi00MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlLCBmYXN0LCBhbmQgYWNjdXJhdGUgYWxnb3JpdGhtIHRvIGVzdGltYXRlIGxhcmdlIHBoeWxvZ2VuaWVzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz02OTYtNzA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CSU9OSjogYW4gaW1wcm92ZWQgdmVyc2lvbiBvZiB0aGUgTkogYWxnb3JpdGhtIGJhc2VkIG9uIGEgc2ltcGxlIG1vZGVsIG9mIHNlcXVlbmNlIGRhdGE7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02ODUtNjk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmVlIFZpZXc6IGFuIGFwcGxpY2F0aW9uIHRvIGRpc3BsYXkgcGh5bG9nZW5ldGljIHRyZWVzIG9uIHBlcnNvbmFsIGNvbXB1dGVyczsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzU3LTM1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rml0dGluZyBhIG1peHR1cmUgbW9kZWwgYnkgZXhwZWN0YXRpb24gbWF4aW1pemF0aW9uIHRvIGRpc2NvdmVyIG1vdGlmcyBpbiBiaW9wb2x5bWVyczsgUHJvYy4gSW50LiBDb25mLiBJbnRlbGwuIFN5c3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTI4LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBoeWJyaWQgcG9wbGFyIChQb3B1bHVzIHRyaWNob2NhcnBhIMOXIGRlbHRvaWRlcykgaW50ZXJhY3Rpbmcgd2l0aCBmb3Jlc3QgdGVudCBjYXRlcnBpbGxhcnMgKE1hbGFjb3NvbWEgZGlzc3RyaWEpOiBub3JtYWxpemVkIGFuZCBmdWxsLWxlbmd0aCBjRE5BIGxpYnJhcmllcywgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MsIGFuZCBhIGNETkEgbWljcm9hcnJheSBmb3IgdGhlIHN0dWR5IG9mIGluc2VjdC1pbmR1Y2VkIGRlZmVuY2VzIGluIHBvcGxhcjsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNzUtMTI5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBibGFjayBjb3R0b253b29kIChQb3B1bHVzIHRyaWNob2NhcnBhKSByZXZlYWxzIDE4IGNvbnNlcnZlZCBjZWxsdWxvc2Ugc3ludGhhc2UgKENlc0EpIGdlbmVzOyBQbGFudGE7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjIxOyAgY2l0YXRpb25fcGFnZXM9NzM5LTc0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNUIGRhdGEgc3VnZ2VzdCB0aGF0IHBvcGxhciBpcyBhbiBhbmNpZW50IHBvbHlwbG9pZDsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9MTY1LTE3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBmYWN0b3IgZ2VuZXMgd2l0aCBleHByZXNzaW9uIGNvcnJlbGF0ZWQgdG8gbml0cmF0ZS1yZWxhdGVkIHJvb3QgcGxhc3RpY2l0eSBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgQ2VsbCBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQ1OS00Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGxlY3Rpb24sIG1hcHBpbmcsIGFuZCBhbm5vdGF0aW9uIG9mIG92ZXIgMjgsMDAwIGNETkEgY2xvbmVzIGZyb20gamFwb25pY2EgcmljZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDE7ICBjaXRhdGlvbl9wYWdlcz0zNzYtMzc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3RlcmlzYXRpb24gb2YgdHdvIHBvbGxlbi1zcGVjaWZpYyBMSU0gZG9tYWluIHByb3RlaW4gY0ROQXMgZnJvbSBOaWNvdGlhbmEgdGFiYWN1bTsgU2V4LiBQbGFudCBSZXByb2QuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzM5LTM0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGZsb3JhbCB0cmFuc2NyaXB0b21lIHVuY292ZXJzIG5ldyByZWd1bGF0b3JzIG9mIG9yZ2FuIGRldGVybWluYXRpb24gYW5kIGdlbmUgZmFtaWxpZXMgcmVsYXRlZCB0byBmbG93ZXIgb3JnYW4gZGlmZmVyZW50aWF0aW9uIGluIEdlcmJlcmEgaHlicmlkYSAoQXN0ZXJhY2VhZSk7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NDc1LTQ4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGNETkFzIGV4cHJlc3NlZCBpbiB0aGUgZWFybHkgc3RhZ2VzIG9mIGZsYXZvbm9sLWluZHVjZWQgcG9sbGVuIGdlcm1pbmF0aW9uIGluIFBldHVuaWE7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTY5OS03MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHh5bGVtIGZvcm1hdGlvbiBpbiBwaW5lIGJ5IGNETkEgc2VxdWVuY2luZzsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz05NjkzLTk2OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvb3JkaW5hdGUgcmVwcmVzc2lvbiBvZiByZWd1bGF0b3JzIG9mIGVtYnJ5b25pYyBpZGVudGl0eSBieSBQSUNLTEUgZHVyaW5nIGdlcm1pbmF0aW9uIGluIEFyYWJpZG9wc2lzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBpZGVudGlmaWNhdGlvbiBvZiBsZWFmIHNlbmVzY2VuY2UtYXNzb2NpYXRlZCBnZW5lczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz02MjktNjQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcHJvZmlsaW5nIG9mIEFyYWJpZG9wc2lzIHRpc3N1ZXMgcmV2ZWFscyB0aGUgdW5pcXVlIGNoYXJhY3RlcmlzdGljcyBvZiB0aGUgcG9sbGVuIHRyYW5zY3JpcHRvbWU7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTcxMy03MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWNETkEgbWljcm9hcnJheSBhbmFseXNpcyBvZiBnZW5lIGV4cHJlc3Npb24gY2hhbmdlcyBkdXJpbmcgcG9sbGluYXRpb24sIHBvbGxlbi10dWJlIGVsb25nYXRpb24sIGZlcnRpbGl6YXRpb24sIGFuZCBlYXJseSBlbWJyeW9nZW5lc2lzIGluIHJpY2UgcGlzdGlsczsgU2V4LiBQbGFudCBSZXByb2QuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjY5LTI3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE0LzMvMTAzLzE3NDYyNzEwL2RzbTAxMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSW4gRXVrYXJ5b3RlcywgTElNIHByb3RlaW5zIGFjdCBhcyBkZXZlbG9wbWVudGFsIHJlZ3VsYXRvcnMgaW4gYmFzaWMgY2VsbHVsYXIgcHJvY2Vzc2VzIHN1Y2ggYXMgcmVndWxhdGluZyB0aGUgdHJhbnNjcmlwdGlvbiBvciBvcmdhbml6aW5nIHRoZSBjeXRvc2siIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTU3Nzg1OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRJEVvzPONhcR7qBTP4lZH+n/baDw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f103%2f577859" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f103%2f577859" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23256" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&amp;Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/3">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195968">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195968">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195970">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195970">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195974">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195974">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195982">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195982">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196077">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196077">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196079">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196079">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/3/91/577454">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/3/117/578022">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Analysis of <em>LIM</em> Gene Family in <em>Populus trichocarpa</em>, <em>Arabidopsis thaliana</em>, and <em>Oryza sativa</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dominique Arnaud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières</div>, <div class="insititution">INRA-Orléans</div>, <div class="addr-line">Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dominique+Arnaud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arnaud D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arnaud D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dominique Arnaud</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Annabelle D&#233;jardin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières</div>, <div class="insititution">INRA-Orléans</div>, <div class="addr-line">Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Annabelle+D%c3%a9jardin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=D&#233;jardin A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;D&#233;jardin A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Annabelle Déjardin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean-Charles Lepl&#233;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières</div>, <div class="insititution">INRA-Orléans</div>, <div class="addr-line">Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean-Charles+Lepl%c3%a9">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lepl&#233; J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lepl&#233; J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jean-Charles Leplé</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marie-Claude Lesage-Descauses</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières</div>, <div class="insititution">INRA-Orléans</div>, <div class="addr-line">Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marie-Claude+Lesage-Descauses">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lesage-Descauses M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lesage-Descauses M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marie-Claude Lesage-Descauses</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gilles Pilate</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Equipe ‘Xylème’ Unité Amélioration, Génétique et Physiologie Forestières</div>, <div class="insititution">INRA-Orléans</div>, <div class="addr-line">Avenue de la Pomme de Pin, BP 20619 Ardon, F-45166 Olivet Cedex</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +33 2-38417875. Fax: <div class="fax">+33 2-38417879</div>. E-mail: <a href="mailto:pilate@orleans.inra.fr" target="_blank">pilate@orleans.inra.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gilles+Pilate">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pilate G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pilate G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gilles Pilate</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 103–116, <a href='https://doi.org/10.1093/dnares/dsm013'>https://doi.org/10.1093/dnares/dsm013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 June 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 January 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 May 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="577859" href="/dnaresearch/article-pdf/14/3/103/17462710/dsm013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Dominique Arnaud, Annabelle Déjardin, Jean-Charles Leplé, Marie-Claude Lesage-Descauses, Gilles Pilate;  Genome-Wide Analysis of <em>LIM</em> Gene Family in <em>Populus trichocarpa</em>, <em>Arabidopsis thaliana</em>, and <em>Oryza sativa</em>, <em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 103–116, <a href='https://doi.org/10.1093/dnares/dsm013'>https://doi.org/10.1093/dnares/dsm013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/577859?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/577859?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Analysis%20of%20%26lt%3Bem%26gt%3BLIM%26lt%3B%2Fem%26gt%3B%20Gene%20Family%20in%20%26lt%3Bem%26gt%3BPopulus%20trichocarpa%26lt%3B%2Fem%26gt%3B%2C%20%26lt%3Bem%26gt%3BArabidopsis%20thaliana%26lt%3B%2Fem%26gt%3B%2C%20and%20%26lt%3Bem%26gt%3BOryza%20sativa%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2007-06-15&amp;volumeNum=14&amp;issueNum=3&amp;author=Arnaud%2C%20Dominique%3B%20D%C3%A9jardin%2C%20Annabelle&amp;startPage=103&amp;endPage=116&amp;contentId=10.1093%2Fdnares%2Fdsm013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f103%2f577859" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195968 id="87195968" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In Eukaryotes, LIM proteins act as developmental regulators in basic cellular processes such as regulating the transcription or organizing the cytoskeleton. The LIM domain protein family in plants has mainly been studied in sunflower and tobacco plants, where several of its members exhibit a specific pattern of expression in pollen. In this paper, we finely characterized in poplar six transcripts encoding these proteins. In <em>Populus trichocarpa</em> genome, the 12 <em>LIM</em> gene models identified all appear to be duplicated genes. In addition, we describe several new LIM domain proteins deduced from <em>Arabidopsis</em> and rice genomes, raising the number of <em>LIM</em> gene models to six for both species. Plant <em>LIM</em> genes have a core structure of four introns with highly conserved coding regions. We also identified new LIM domain proteins in several other species, and a phylogenetic analysis of plant LIM proteins reveals that they have undergone one or several duplication events during the evolution. We gathered several LIM protein members within new monophyletic groups. We propose to classify the plant LIM proteins into four groups: αLIM1, βLIM1, γLIM2, and δLIM2, subdivided according to their specificity to a taxonomic class and/or to their tissue-specific expression. Our investigation of the structure of the LIM domain proteins revealed that they contain many conserved motifs potentially involved in their function.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="poplar">poplar</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis"><em>Arabidopsis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rice">rice</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;LIM domain protein&quot;">LIM domain protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tension wood&quot;">tension wood</a></div>                            <h2 scrollto-destination=87195970 id="87195970" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>LIM proteins have been named by the initials of the three first discovered LIM homeodomain proteins: LIN11, ISL1, and MEC3.<sup><a reveal-id="DSM013C1  DSM013C3" data-open="DSM013C1  DSM013C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> In Eukaryotes, LIM proteins contain one or more LIM domains, which are, in some cases, associated with a protein kinase domain or a homeodomain. The LIM domain is a cysteine–histidine-rich, zinc-coordinating domain consisting of two zinc fingers repeated in tandem.<sup><a reveal-id="DSM013C4" data-open="DSM013C4" class="link link-ref link-reveal xref-bibr">4</a></sup> It is conserved over a wide variety of species. The cysteine-rich protein (CRP) family in animals is a subclass of LIM proteins characterized by their two LIM domains, with the consensus sequence [C-X<sub>2</sub>-C-X<sub>16–23</sub>-H-X<sub>2</sub>-C]-X<sub>2</sub>-[C-X<sub>2</sub>-C-X<sub>16–21</sub>C-X<sub>2–3</sub>-(C/D/H)], that are both followed by a short glycine-rich repeat. The CRP family in vertebrates involves four proteins: CRP1, CRP2, CRP3/MLP, and TLP, which act as molecular adapters.<sup><a reveal-id="DSM013C5" data-open="DSM013C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Indeed, CRP1, CRP2, and CRP3 are able to bind α-actin and zyxin, two components of the cytoskeleton. Although the LIM domains from animal CRP proteins are structurally similar to the GATA-type zinc finger transcription factor,<sup><a reveal-id="DSM013C6" data-open="DSM013C6" class="link link-ref link-reveal xref-bibr">6</a></sup> their DNA-binding activity is yet unproven and CRP may rather be involved in protein–protein interactions.</p><p>The LIM protein family in plants is CRP-related proteins containing two LIM domains separated by a long inter-LIM domain. On the contrary to the animal CRP family, plant LIM proteins have a longer C-terminal domain and lack the glycine-rich region (GRR) following each LIM domain. For all plant LIM proteins, the two LIM domains of 52 residues have the following characteristic structure: [C-X<sub>2</sub>-C-X<sub>17</sub>-H-X<sub>2</sub>-C]-X<sub>2</sub>-[C-X<sub>2</sub>-C-X<sub>17</sub>-C-X<sub>2</sub>-H].<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The first gene encoding a LIM domain protein in plants has been named <em>SF3</em>. Later renamed <em>HaPLIM1</em>, <em>SF3</em> expression was found specific to sunflower pollen.<sup><a reveal-id="DSM013C8" data-open="DSM013C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSM013C9" data-open="DSM013C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The LIM proteins from sunflower, tobacco, and <em>Arabidopsis</em> have been classified into four groups: PLIM1 and PLIM2 specifically expressed in pollen and WLIM1 and WLIM2 expressed in the whole plant.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Like animal CRP, most plant LIM proteins are present in the cytoplasm and/or in the nucleus. This is the case for the sunflower protein HaWLIM1 that, for many different cell types, localizes either in the cytoplasm, in the nucleus, or in both.<sup><a reveal-id="DSM013C10" data-open="DSM013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Moreover, in protoplasts, HaWLIM1 seems to be associated with cortical microtubules, and it is also observed in the nucleus during the interphase.<sup><a reveal-id="DSM013C11" data-open="DSM013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> As for CRP proteins, the tobacco NtWLIM1 binds F-actin and may be involved in actin cytoskeleton stability.<sup><a reveal-id="DSM013C12" data-open="DSM013C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The sunflower protein HaPLIM1 has been detected both in small cytoplasmic structures located in the microspores and in the cortical region of mature pollen grains, where it concentrates in the actin-enriched germination cones, suggesting its interaction with the actin cytoskeleton.<sup><a reveal-id="DSM013C13" data-open="DSM013C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Although HaPLIM1 was never found in the nucleus of vegetative cells, it exhibits a non-specific DNA- and RNA-binding activity.<sup><a reveal-id="DSM013C14" data-open="DSM013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Hence, the function of HaWLIM1 and HaPLIM1 in the transcriptional regulation remains unclear. In tobacco, NtLIM1 is clearly a transcription factor that binds to the PAL-box, a conserved motif present in the promoter of a number of genes from the phenylpropanoid pathway.<sup><a reveal-id="DSM013C15" data-open="DSM013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Transgenic tobacco plants with a reduced <em>NtLIM1</em> expression also present reduced lignin content and a decreased expression of PAL, 4CL, and CAD, three enzymes involved in the lignin biosynthesis. In poplar, the distribution of expressed sequence tags (ESTs) in different wood tissues indicates a rather high expression of an LIM protein homologue in tension wood.<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Accordingly, microarray analyses also indicate a higher expression of some LIM transcription factor homologues in tension wood.<sup><a reveal-id="DSM013C17" data-open="DSM013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Tension wood, formed on the upper side of bent stems, is enriched in cellulose due to the formation of a supplementary gelatinous layer.</p><p>Our study focused on the plant LIM domain proteins containing only two LIM domains homologous to the animal CRP family. First, we finely describe, in this paper, the poplar <em>LIM</em> gene family. We determined the complete sequence of six cDNAs encoding LIM proteins and searched for LIM domain protein encoding genes in the <em>Populus trichocarpa</em> genome sequence. Secondly, we completed the inventory of the <em>Arabidopsis</em> and rice <em>LIM</em> gene family. To get a global overview of the plant LIM domain family, cDNAs and ESTs encoding LIM domain proteins have extensively been researched in plant sequence databases. Sequence analyses and phylogenetic studies revealed the structural diversity in plant LIM proteins. We named the genes coding for LIM domain proteins by following a nomenclature stemming from the phylogenetic analysis.</p>                            <h2 scrollto-destination=87195974 id="87195974" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195975 id="87195975" class="section-title" data-legacy-id=s2a>2.1. Characterization of poplar LIM cDNA and gene sequences</h3>
<p>The transcripts similar to LIM domain proteins were searched by basic local alignment search tool (BlastN) within a collection of 10 062 ESTs obtained from <em>Populus tremula</em> × <em>Populus alba</em> (clone INRA #717-1-B4, <em>Populus</em> section) wood cDNA libraries.<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Among the <em>LIM</em> ESTs grouped in a same contig, we selected the longest cDNA clone and sequenced it. The entire cDNA was amplified by polymerase chain reaction (PCR) using the forward primer TriplexA and the reverse primer pTriplexB1<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> of the pTriplex vector (Clontech, Laboratories Inc., Mountain View, CA, USA). We further completed the forward and reverse sequencing with new primers specific to each <em>PtaLIM</em> gene (<em>Pta</em> stands for <em>P. tremula</em> × <em>P. alba</em>; Supplementary Table S1). The different <em>PtLIM</em> genes (<em>Pt</em> stands for <em>P. trichocarpa</em>, <em>Tacamahaca</em> section) were identified in the <em>Populus</em> genome database (<a class="link link-uri" href="http://www.genome.jgi-psf.org/Poptr1_1/Poptr1_1.home.html">http://www.genome.jgi-psf.org/Poptr1_1/Poptr1_1.home.html</a>)<sup><a reveal-id="DSM013C18" data-open="DSM013C18" class="link link-ref link-reveal xref-bibr">18</a></sup> by keyword search, using the InterPro LIM domain annotation (IPR001781) as a query, and using the six cDNAs from <em>P. tremula</em> × <em>P. alba</em> as query for BlastN searches. We selected only the proteins containing two LIM domains. The prediction of exon/intron splicing in <em>P. trichocarpa LIM</em> genes was verified by sequencing reverse transcriptase (RT)–PCR fragments with specific primers against poplar <em>LIM</em> genes (see Supplementary Table S1). These fragments were produced after amplification of polyA RNA prepared from poplar wood samples.<sup><a reveal-id="DSM013C19" data-open="DSM013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Finally, the poplar ESTs homologous to the various <em>PtaLIM</em> transcripts identified were also searched against the GenBank database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) and against the PopulusDB database (<a class="link link-uri" href="http://www.populus.db.umu.se">http://www.populus.db.umu.se</a>), where cDNA libraries were built from <em>P. tremula</em> or <em>P. tremula</em> × <em>Populus tremuloides</em> (<em>Populus</em> section) samples.<sup><a reveal-id="DSM013C20" data-open="DSM013C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h3 scrollto-destination=87195977 id="87195977" class="section-title" data-legacy-id=s2b>2.2. Database search for sequences coding for LIM domain protein</h3>
<p>Until August 2006, we have used several ways to search in multiple databases all plant LIM proteins containing only two LIM domains. First, known genes and full-length cDNAs encoding LIM domain proteins from sunflower, tobacco, and <em>Arabidopsis</em> were collected from the literature and GenBank database.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Additional genes and full-length cDNA annotated as ‘LIM domain protein’, or ‘LIM transcription factor’ were found by keyword searches or using the InterPro LIM domain annotation (IPR001781) as a query. Alternatively, the poplar LIM protein sequences were used for BLAST (TBlastN and BlastP) searches at the GenBank non-redundant database. The genomic and cDNA sequences of <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em> were obtained from the GenBank, TIGR plant genomic group (<a class="link link-uri" href="http://plantgenomics.tigr.org/">http://plantgenomics.tigr.org/</a>), and TAIR (<a class="link link-uri" href="http://arabidopsis.org">http://arabidopsis.org</a>) databases. Finally, to get more sequences encoding LIM domain proteins, ESTs homologous to plant LIM domain proteins were searched by BlastN at the GenBank EST database. This process was repeated with each newly identified set of plant <em>LIM</em> genes until no further sequences with significant similarity were identified. For each gene, the longest EST was translated in all reading frames using the EMBOSS Transeq program at the EMBL–EBI, and only those carrying the entire coding sequence (CDS) and some part of 3′ and 5′ untranslated region (UTR) were chosen for further phylogenetic analyses. When EST sequences were too short or did not contain the entire CDS, a consensus sequence was deduced using the Bioedit software. Finally, the deduced amino acid sequences were verified for those carrying the entire two LIM domains using the PROSITE database (<a class="link link-uri" href="http://www.expasy.org/prosite">http://www.expasy.org/prosite</a>). The selected ESTs with their consensus sequences are listed in Supplementary Table S2.</p>                            <h3 scrollto-destination=87195979 id="87195979" class="section-title" data-legacy-id=s2c>2.3. Sequence and phylogenetic analysis of LIM domain proteins</h3>
<p>The selected protein sequences were aligned using the ClustalW software package (<a class="link link-uri" href="http://www.ebi.ac.uk/clustalw/">http://www.ebi.ac.uk/clustalw/</a>)<sup><a reveal-id="DSM013C21" data-open="DSM013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with minor adjustments. Phylogenetic analyses were carried out using the Phylogenetic Interference Package (PHYLIP) program, version 3.63 (<a class="link link-uri" href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu/phylip.html</a>). Genetic distance matrices using protein polymorphism were calculated using PROTDIST software with the JTT amino acid substitution matrix as measure of distances.<sup><a reveal-id="DSM013C22" data-open="DSM013C22" class="link link-ref link-reveal xref-bibr">22</a></sup> A phylogenetic tree was then constructed using the neighbor-joining method thanks to NEIGHBOR software.<sup><a reveal-id="DSM013C23" data-open="DSM013C23" class="link link-ref link-reveal xref-bibr">23</a></sup> To estimate the statistical robustness of nodes, 1000 bootstrap samples were generated with SEQBOOT software, and the majority rule consensus tree was generated by CONSENSE software. The plant LIM family was also analyzed through a parsimonious method using the PROTPARS program with 1000 bootstrap replicates. Maximum likelihood analyses were performed using Phyml v2.4.1<sup><a reveal-id="DSM013C24" data-open="DSM013C24" class="link link-ref link-reveal xref-bibr">24</a></sup> with the JTT matrix and 100 bootstrap replicates. Maximum likelihood trees were generated with BIONJ, a modified neighbor-joining algorithm.<sup><a reveal-id="DSM013C25" data-open="DSM013C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Trees were viewed and edited with Tree View,<sup><a reveal-id="DSM013C26" data-open="DSM013C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and bootstrap values &lt; 50% were not reported.</p><p>Conserved motifs in LIM domain proteins were detected using the ClustalW alignment with few manual corrections, and the MEME program (<a class="link link-uri" href="http://www.meme.sdsc.edu/meme/meme.html">http://www.meme.sdsc.edu/meme/meme.html</a>).<sup><a reveal-id="DSM013C27" data-open="DSM013C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The aligned protein sequences were shaded using the Bioedit software with a threshold of 90% for identical residues and a BLOSUM62 matrix for shading similar residues. Isoelectric point (p<em>I</em>) and molecular weight (Mw) were predicted using the p<em>I</em>/Mw tool at expasy (<a class="link link-uri" href="http://www.expasy.org/tools/pi_tool.html">http://www.expasy.org/tools/pi_tool.html</a>). PROSITE results were used to find putative ASN-glycosylation and phosphorylation sites.</p>                            <h2 scrollto-destination=87195982 id="87195982" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87195983 id="87195983" class="section-title" data-legacy-id=s3a>3.1. Survey and characterization of the LIM domain proteins in poplar, <em>Arabidopsis</em> and rice</h3>
                            <h4 scrollto-destination=87195984 id="87195984" class="section-title" data-legacy-id=s3a1>3.1.1. Isolation of the cDNAs coding for poplar LIM domain proteins and identification of 12 gene models in the <em>Populus trichocarpa</em> genome</h4>
<p>The distribution of 10 062 poplar ESTs in different wood cDNA libraries<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> revealed that a cDNA, named <em>PtaGLIM1a</em> (EF035035), homologous to the LIM protein SF3<sup><a reveal-id="DSM013C8" data-open="DSM013C8" class="link link-ref link-reveal xref-bibr">8</a></sup> was abundant in differentiating xylem from tension wood. Five other <em>PtaLIM</em> cDNAs present in this EST collection were found, and a complete sequence was determined for each. We named them <em>PtaGLIM1b</em> (EF035036), <em>PtaWLIM2a</em> (EF035040), <em>PtaPLIM2a</em> (EF035037), <em>PtaWLIM1a</em> (EF035038), and <em>PtaWLIM1b</em> (EF035039). All of the full-length cDNAs obtained contain an entire CDS, a 5′ and a 3′ UTR, and encode protein with two LIM domains (Fig. <a reveal-id="DSM013F1" data-open="DSM013F1" class="link link-reveal link-table xref-fig">1</a>; Supplementary Fig. S1).</p>                        <a id="87195986" scrollto-destination="87195986"></a>
<div id="DSM013F1" data-id="DSM013F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01301.jpeg?Expires=2147483647&Signature=Wjvps1H8~ILSYAojaDoYPUQQlPQj6cSrokE6a2cndmpVD7O-G7r4lZlWX-9b1wZeDqKuW4LjICnRetqgVoy23HQluUNBjHY1achvqb2KMbXNhyB7OG2nQBrVT~T21xxK67YhntQvhp-ApKwzRuFq03qsRPq-39GEHSXJKv7wyS-k7JXVpdBKzXOkhrG3I7IavloUqOu7MKVqUcYOBR5MPZW~bOp~K9Jw~qIYJDVQMAURzIcSsN0VXCXXBvoy67-r~xB8LUCyI0XX-GYPWj~4n6qDEY8anhMoZH6vNRAYF65U-C4PWq3rfzKgvQhtjvy8an5iJ3EJS8ik5UP9hI3DfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram of the structure of the poplar LIM domain proteins. The conserved LIM1 and LIM2 domains are indicated by gray boxes. Open boxes show the N-terminal, inter-LIM, and C-terminal domains. Arrows indicate the respective position of the four introns (I1–I4)." path-from-xml="dsm01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195986/dsm01301.jpeg" path-from-xml="dsm01301.jpeg" target="_blank">View large</a><a section="87195986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01301.jpeg?Expires=2147483647&Signature=WmbVo3D4kJwJVXev8BdAUZs5~n22H4nf6UFQ-9OmH~Wz4pnyDzR2-MkmYQ9KVDTy2KovEV4f3YUTNsJZg1vsHF77FziIbI7XiBbkDoWWcxDxodzIw1jLO7MIwR3H9pnSqEFhKBjy7nDKQpn1XxxClHPHaGHPdtTsB83jfyTbFiuHeJqLYYRNdQNI81gjRyK4otjPnuFbNEMWAV3DF4XEsIniidhD8L1lfj-bsuNbqMTkz-TWRkQwPNHQRZ0dzirFESJFqueqaAjhnGmoydaZijRK9MaggKtAzoONk942geijbDNsRxDQeoiLCDj8BQ4vKh~TxuospI4rhFj6wiS33A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195986&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the structure of the poplar LIM domain proteins. The conserved LIM1 and LIM2 domains are indicated by gray boxes. Open boxes show the N-terminal, inter-LIM, and C-terminal domains. Arrows indicate the respective position of the four introns (I1–I4).</p></div></div><div content-id="DSM013F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01301.jpeg?Expires=2147483647&Signature=Wjvps1H8~ILSYAojaDoYPUQQlPQj6cSrokE6a2cndmpVD7O-G7r4lZlWX-9b1wZeDqKuW4LjICnRetqgVoy23HQluUNBjHY1achvqb2KMbXNhyB7OG2nQBrVT~T21xxK67YhntQvhp-ApKwzRuFq03qsRPq-39GEHSXJKv7wyS-k7JXVpdBKzXOkhrG3I7IavloUqOu7MKVqUcYOBR5MPZW~bOp~K9Jw~qIYJDVQMAURzIcSsN0VXCXXBvoy67-r~xB8LUCyI0XX-GYPWj~4n6qDEY8anhMoZH6vNRAYF65U-C4PWq3rfzKgvQhtjvy8an5iJ3EJS8ik5UP9hI3DfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic diagram of the structure of the poplar LIM domain proteins. The conserved LIM1 and LIM2 domains are indicated by gray boxes. Open boxes show the N-terminal, inter-LIM, and C-terminal domains. Arrows indicate the respective position of the four introns (I1–I4)." path-from-xml="dsm01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195986/dsm01301.jpeg" path-from-xml="dsm01301.jpeg" target="_blank">View large</a><a section="87195986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01301.jpeg?Expires=2147483647&Signature=WmbVo3D4kJwJVXev8BdAUZs5~n22H4nf6UFQ-9OmH~Wz4pnyDzR2-MkmYQ9KVDTy2KovEV4f3YUTNsJZg1vsHF77FziIbI7XiBbkDoWWcxDxodzIw1jLO7MIwR3H9pnSqEFhKBjy7nDKQpn1XxxClHPHaGHPdtTsB83jfyTbFiuHeJqLYYRNdQNI81gjRyK4otjPnuFbNEMWAV3DF4XEsIniidhD8L1lfj-bsuNbqMTkz-TWRkQwPNHQRZ0dzirFESJFqueqaAjhnGmoydaZijRK9MaggKtAzoONk942geijbDNsRxDQeoiLCDj8BQ4vKh~TxuospI4rhFj6wiS33A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195986&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the structure of the poplar LIM domain proteins. The conserved LIM1 and LIM2 domains are indicated by gray boxes. Open boxes show the N-terminal, inter-LIM, and C-terminal domains. Arrows indicate the respective position of the four introns (I1–I4).</p></div></div><p>We searched the <em>P. trichocarpa</em> genome sequence for all the gene models coding for LIM domain proteins. We excluded several gene models coding for proteins containing only one LIM domain linked to either a cytochrome P450 domain or an ubiquitin interaction motif and focused on gene models with two LIM domains. Besides the gene models corresponding to the six cDNAs isolated in our laboratory, we identified six other gene models encoding LIM domain proteins, raising to twelve the number of <em>LIM</em> gene models in the <em>Populus</em> genome. In accordance with their phylogenetic relationship with other known plant LIM domain proteins, we named these genes <em>PtWLIM1a</em>, <em>PtWLIM1b</em>, <em>PtGLIM1a</em>, <em>PtGLIM1b</em>, <em>PtβLIM1a</em>, <em>PtβLIM1b</em>, <em>PtWLIM2a</em>, <em>PtWLIM2b</em>, <em>PtPLIM2a</em>, <em>PtPLIM2b</em>, <em>PtδLIM2a</em>, and <em>PtδLIM2b</em> (Fig. <a reveal-id="DSM013F2" data-open="DSM013F2" class="link link-reveal link-table xref-fig">2</a>A; Supplementary Table S3). With new information from both full-length <em>PtaLIM</em> cDNAs and <em>P. trichocarpa</em> transcript sequences,<sup><a reveal-id="DSM013C28" data-open="DSM013C28" class="link link-ref link-reveal xref-bibr">28</a></sup> we have corrected and improved the annotated poplar genomic sequence for the <em>PtLIM</em> genes (Supplementary Data 1).</p>                        <a id="87195988" scrollto-destination="87195988"></a>
<div id="DSM013F2" data-id="DSM013F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01302.jpeg?Expires=2147483647&Signature=KsHwY83wc3geQeabYdG1hiyRCfW5-ACa0Yx~XhDAKwa1uoBD5KWmKoFVhYYlr3t8Sj3QZX4YeFRI0MOz0XGC1D6ESXBmQKb6hfBFnpiiVrlKxSBYpOM9wZPEypzf331sZIShwapSbqRDpZaEG5yYrmHd4bn3mQ~GBd~11SkCpsSHTxuamoBoEMdRPaWN6gmS3Z-kd5SzDtOaCF7yl3uxDp6BhvMcVudtn9JEHQ6mX3bwTJ3yGvpAaXeFNQzo~Sk98kdKdR0wKpy5zrHvo25MQ2SR1KLkRtqylEz7cyZABmxp8y~eXzpz-HSuicnX8kbBvWGfWRKWpLE8hf2rKNW1iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of P. trichocarpa, A. thaliana, and O. sativa LIM domain proteins. Four phylogenetic trees of (A) 12 poplar, (B) six Arabidopsis, (C) six rice, and (D) all deduced LIM domain proteins are shown. Amino acid sequences of LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent percent of bootstraps values (≥50%) based on 1000 replications. The length of the branches is proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the trees. A species acronym is added before each LIM protein name: At, A. thaliana; Os, O. sativa; Pt, P. trichocarpa." path-from-xml="dsm01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195988/dsm01302.jpeg" path-from-xml="dsm01302.jpeg" target="_blank">View large</a><a section="87195988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01302.jpeg?Expires=2147483647&Signature=gGqGVQRVx0R89WGwkrntxbr7-fuM2Us7bFkhGYLVZX5OAr9Gq3WsVAOcMnrLQI9EhDuUNd-wuXKAI44hLmratEf0Jdi4mUfAS2m9wBYwAuoHuDNrFy3~phXgzElBVScW95KEaWS-QyNREaRWIdSsPgt3cHJPnuf1WpSUYiA2D0OQWa5KmNSZ2bYMriFD-jNTBi82YoehOiDTJ9PMWKgy6S2A0wtcLCFWCDEATS3xByLOzzLYNUEOrVwXAjdK0MJsj12qEtWRfsAlDtI-B0DUBfFF~dMMwJ-bgId7uCDwHBRksD1ql62PIT961047b707gyxCNMytV-xYkeK821GsAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195988&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of <em>P. trichocarpa</em>, <em>A. thaliana</em>, and <em>O. sativa</em> LIM domain proteins. Four phylogenetic trees of (<strong>A</strong>) 12 poplar, (<strong>B</strong>) six <em>Arabidopsis</em>, (<strong>C</strong>) six rice, and (<strong>D</strong>) all deduced LIM domain proteins are shown. Amino acid sequences of LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent percent of bootstraps values (≥50%) based on 1000 replications. The length of the branches is proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the trees. A species acronym is added before each LIM protein name: At, <em>A. thaliana</em>; Os, <em>O. sativa</em>; Pt, <em>P. trichocarpa</em>.</p></div></div><div content-id="DSM013F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01302.jpeg?Expires=2147483647&Signature=KsHwY83wc3geQeabYdG1hiyRCfW5-ACa0Yx~XhDAKwa1uoBD5KWmKoFVhYYlr3t8Sj3QZX4YeFRI0MOz0XGC1D6ESXBmQKb6hfBFnpiiVrlKxSBYpOM9wZPEypzf331sZIShwapSbqRDpZaEG5yYrmHd4bn3mQ~GBd~11SkCpsSHTxuamoBoEMdRPaWN6gmS3Z-kd5SzDtOaCF7yl3uxDp6BhvMcVudtn9JEHQ6mX3bwTJ3yGvpAaXeFNQzo~Sk98kdKdR0wKpy5zrHvo25MQ2SR1KLkRtqylEz7cyZABmxp8y~eXzpz-HSuicnX8kbBvWGfWRKWpLE8hf2rKNW1iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees of P. trichocarpa, A. thaliana, and O. sativa LIM domain proteins. Four phylogenetic trees of (A) 12 poplar, (B) six Arabidopsis, (C) six rice, and (D) all deduced LIM domain proteins are shown. Amino acid sequences of LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent percent of bootstraps values (≥50%) based on 1000 replications. The length of the branches is proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the trees. A species acronym is added before each LIM protein name: At, A. thaliana; Os, O. sativa; Pt, P. trichocarpa." path-from-xml="dsm01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195988/dsm01302.jpeg" path-from-xml="dsm01302.jpeg" target="_blank">View large</a><a section="87195988" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01302.jpeg?Expires=2147483647&Signature=gGqGVQRVx0R89WGwkrntxbr7-fuM2Us7bFkhGYLVZX5OAr9Gq3WsVAOcMnrLQI9EhDuUNd-wuXKAI44hLmratEf0Jdi4mUfAS2m9wBYwAuoHuDNrFy3~phXgzElBVScW95KEaWS-QyNREaRWIdSsPgt3cHJPnuf1WpSUYiA2D0OQWa5KmNSZ2bYMriFD-jNTBi82YoehOiDTJ9PMWKgy6S2A0wtcLCFWCDEATS3xByLOzzLYNUEOrVwXAjdK0MJsj12qEtWRfsAlDtI-B0DUBfFF~dMMwJ-bgId7uCDwHBRksD1ql62PIT961047b707gyxCNMytV-xYkeK821GsAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195988&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees of <em>P. trichocarpa</em>, <em>A. thaliana</em>, and <em>O. sativa</em> LIM domain proteins. Four phylogenetic trees of (<strong>A</strong>) 12 poplar, (<strong>B</strong>) six <em>Arabidopsis</em>, (<strong>C</strong>) six rice, and (<strong>D</strong>) all deduced LIM domain proteins are shown. Amino acid sequences of LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent percent of bootstraps values (≥50%) based on 1000 replications. The length of the branches is proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the trees. A species acronym is added before each LIM protein name: At, <em>A. thaliana</em>; Os, <em>O. sativa</em>; Pt, <em>P. trichocarpa</em>.</p></div></div><p>Each pair of genes (<em>a</em> and <em>b</em>) exhibits a high sequence similarity from 85% amino acids identity between PtδLIM2a and b to 95% amino acids identity between PtaGLIM1a and b (Fig. <a reveal-id="DSM013F2" data-open="DSM013F2" class="link link-reveal link-table xref-fig">2</a>A; Supplementary Table S4). This high similarity strongly suggests a gene duplication, as previously observed in a recent study for the poplar cellulose synthase <em>CesA</em> gene family.<sup><a reveal-id="DSM013C29" data-open="DSM013C29" class="link link-ref link-reveal xref-bibr">29</a></sup> These observations are in accordance with the hypothesis that poplar is an ancient polyploid and that a large-scale duplication event has occurred in the ancestor of poplar.<sup><a reveal-id="DSM013C18" data-open="DSM013C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM013C30" data-open="DSM013C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The duplicated LIM proteins may have kept similar functions, but the distribution of ESTs as well as the expression pattern between duplicated genes (data not shown) differs slightly, suggesting some other differences within their regulatory regions. All the deduced proteins have the features of the plant LIM domain protein family as described previously,<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> with two LIM domains separated by a long-spacer named the inter-LIM region, a short N-ter domain, and a C-ter domain variable in length (Supplementary Fig. S1). The length of PtLIM proteins is rather constant, between 194 and 216 amino acids, and their Mw varies between 20.9 and 24.1 kDa (data not shown). Poplar LIM proteins share a level of homology ranging from 43 to 95% of amino acid identity (Supplementary Table S4). The highest divergence between poplar LIM protein sequences is mainly localized to the inter-LIM and C-ter domains. From the single analysis of amino acid identity between the different poplar LIM domain proteins, we differentiated these proteins into two major groups: LIM1 and LIM2. Within each group, the percentage of identity at the amino acid level is a lot higher (from 58 to 95%) than between the two groups (from 43 to 55%).</p>                            <h4 scrollto-destination=87195990 id="87195990" class="section-title" data-legacy-id=s3a2>3.1.2. The <em>Arabidopsis</em> and rice genome contain six LIM gene models</h4>
<p>A previous study reported the identification of three <em>LIM</em> genes in the <em>A. thaliana</em> genome: <em>AtWLIM1</em> (At1g10200), <em>AtPLIM2</em> (At2g45800), and <em>AtWLIM2</em> (At2g39900).<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Because the sequences of both the <em>Arabidopsis</em> and rice genomes were publicly available, we had the opportunity to identify extensively all the genes coding for LIM domain proteins. Bioinformatics analyses performed against the GenBank and TIGR databases show that the <em>Arabidopsis</em> genome contains three other <em>AtLIM</em> genes. Because these genes seem to be duplicated, we named them <em>AtPLIM2b</em> (At1g01780), <em>AtPLIM2c</em> (At3g61230), and <em>AtWLIM2b</em> (At3g55770, also named <em>AtL2</em><sup><a reveal-id="DSM013C31" data-open="DSM013C31" class="link link-ref link-reveal xref-bibr">31</a></sup>), whereas the previous <em>AtWLIM2</em> and <em>AtPLIM2</em> genes have been renamed <em>AtWLIM2a</em> and <em>AtPLIM2a</em>, respectively (Fig. <a reveal-id="DSM013F2" data-open="DSM013F2" class="link link-reveal link-table xref-fig">2</a>B). The Supplementary Table S5 lists genomic, cDNA, and EST accessions for the six <em>Arabidopsis LIM</em> genes, as well as the cDNA and genomic clones that contain errors or encode partial LIM domain proteins. It should be pointed out that the three related genes <em>AtPLIM2a</em>, <em>b</em>, and <em>c</em> overlap, respectively, with a gene encoding a phosphomannomutase (At2g45790), a gene encoding an unknown protein (At1g01770), and a gene encoding an oxydoreductase (At3g61220) that may affect the transcription of these <em>AtPLIM2</em> genes.</p><p>As in <em>Arabidopsis</em>, the rice (<em>Oryza sativa</em>) genome contains six genes coding for protein with two LIM domains: <em>OsWLIM1</em> (LOC_Os12g32620), <em>OsWLIM2</em> (LOC_Os03g15940), <em>OsPLIM2a</em> (LOC_Os02g42820), <em>OsPLIM2b</em> (LOC_Os04g45010), <em>OsPLIM2c</em> (LOC_Os10g35930), and <em>OsLIM</em> (LOC_Os06g13030) (Fig. <a reveal-id="DSM013F2" data-open="DSM013F2" class="link link-reveal link-table xref-fig">2</a>C; Supplementary Table S5). <em>OsPLIM2a</em>, <em>b</em>, and <em>c</em> are very similar in their sequences and, therefore, may be considered in-paralogs genes. Unlike poplar, duplication is not the rule in <em>Arabidopsis</em> and rice (Fig. <a reveal-id="DSM013F2" data-open="DSM013F2" class="link link-reveal link-table xref-fig">2</a>D). The genes <em>OsWLIM2</em>, <em>OsPLIM2b</em>, and <em>OsPLIM2c</em> are well supported by ESTs and are represented by full-length cDNAs.<sup><a reveal-id="DSM013C32" data-open="DSM013C32" class="link link-ref link-reveal xref-bibr">32</a></sup> For <em>OsWLIM1</em>, the identified cDNA (AK058220) is truncated. <em>OsPLIM2a</em> is also poorly represented at the mRNA level, with only two ESTs found and no published full-length cDNA. Therefore, for these two genes, we used their genomic sequences for the sequence alignment. In the case of <em>OsLIM</em>, we identified a very long transcript (AK102383) that encodes an unusual LIM protein of 1303 amino acids with two classical LIM domains and a very long C-ter domain that has no homology for any known protein. Only one (CI584223) of the 22 ESTs found by a BlastN search localizes in the 5′ part of the transcript at the level of the first LIM domain. Because of its unusually long C-ter domain, we did not include this LIM sequence in the phylogenetic analysis.</p>                            <h4 scrollto-destination=87195993 id="87195993" class="section-title" data-legacy-id=s3a3>3.1.3. Genomic organization of poplar, <em>Arabidopsis</em>, and rice genes</h4>
<p>From the genomic analysis, we can infer that all plant <em>LIM</em> genes have a core gene structure with four introns within the coding sequence with the exception of <em>AtPLIM2a</em> and <em>c</em> genes that have two and three introns, respectively (Fig. <a reveal-id="DSM013F1" data-open="DSM013F1" class="link link-reveal link-table xref-fig">1</a>; Supplementary Fig. S2). The position of the first and last introns is strictly conserved in the first and second LIM domain, respectively. In poplar and <em>Arabidopsis</em>, the <em>WLIM2</em> genes diverge from the other <em>LIM</em> genes by the occurrence of one (<em>AtWLIM2a</em>, <em>PtWLIM2a</em>, and <em>b</em>) or two (<em>AtWLIM2b</em>) supplementary introns in the 5′ UTR, before the ATG initiation codon. For these <em>WLIM2</em> genes in eudicots species, a mechanism of alternative splicing of the first intron may be involved in post-translational regulation. This is supported by northern-blot experiments revealing two hybridizing bands for <em>AtWLIM2b</em> only in the shoot and not in the root.<sup><a reveal-id="DSM013C31" data-open="DSM013C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>The length of coding regions CR1, CR2, and CR4 is highly conserved between plant <em>LIM</em> genes, indicative of a strict conservation in the length of the LIM domains during evolution (Supplementary Fig. S2). The first coding region is 135–138 bp long and only <em>OsWLIM1</em>, <em>OsWLIM2</em>, and <em>OsLIM</em> have a variable CR1 length. The second coding region is 97 bp long for genes belonging to the LIM1 group and 100 bp long for genes from the LIM2 group. The fourth exon, localized within the second LIM domain, is the most conserved exon, with a length of 90 bp for all <em>LIM</em> genes. The CR3 and CR5 are highly variable in length resulting in the differences observed at the amino acid level, respectively, in the interLIM region and the C-ter domain. Finally, the <em>PLIM2</em> and δ<em>LIM2</em> genes contain the longest fifth coding region that reflects the extensive length of the C-ter region of the deduced proteins.</p>                            <h4 scrollto-destination=87195996 id="87195996" class="section-title" data-legacy-id=s3a4>3.1.4. Identification of ESTs homologous to LIM domain protein</h4>
<p>In a previous study, LIM proteins from sunflower, tobacco, and <em>Arabidopsis</em> have been classified into two groups, LIM1 and LIM2, and subdivided into four subgroups: PLIM1 and PLIM2, specifically expressed in pollen, and WLIM1 and WLIM2, widely expressed in plant.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Because of the availability of the genome sequence for <em>P. trichocarpa</em>, <em>A. thaliana</em>, and <em>O. sativa</em>, we found an increased number of genes belonging to the plant LIM domain family. The newly discovered LIM proteins may define new LIM subgroups or be related to the previously identified subgroups. To approach the diversity of the <em>LIM</em> gene family, an extensive BlastN search of cDNAs and ESTs encoding proteins with two LIM domains has been performed in NCBI plant sequence databases. In plants, we found 165 unigenes homologous to LIM domain protein, but we did not include 49 of them in the phylogenetic analysis because they contained only partial CDS (data not shown). Within the 116 unigenes coding for an entire LIM domain protein, 90 have a representative EST containing a full length CDS, whereas for each of the remaining 26 unigenes, a consensus was built to determine the complete CDS (Supplementary Table S2). We identified ESTs from a wide range of species in the different groups of plants: bryophytes, conifers, piperales, monocotyledons, and eudicotyledons including rosids, asterids, caryophyllales, and banunculales subclasses. In tobacco (<em>Nicotiana tabacum</em>) and in the sunflower (<em>Helianthus annuus</em>), the number of different <em>LIM</em> transcripts has been raised to eight with the finding of novel ESTs, notably the <em>HpWLIM2</em> EST from <em>Helianthus petiolaris</em>, whose existence was previously suspected in this genus by others.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                            <h3 scrollto-destination=87195998 id="87195998" class="section-title" data-legacy-id=s3b>3.2. Phylogenetic analysis of plant LIM proteins with regard to expression data</h3>
                            <h4 scrollto-destination=87196013 id="87196013" class="section-title" data-legacy-id=s3b1>3.2.1. Four different groups</h4>
<p>Phylogenetic trees have been constructed with the deduced amino acid sequence of the genes encoding LIM domain proteins (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). We renamed the plant LIM proteins according to their phylogenetic relationship. The LIM1 and LIM2 groups identified previously<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> are clearly separated and supported by a high bootstrap value at the level of the TrLIM and PpLIM proteins from the mosses <em>Physcomitrella patens</em> and <em>Tortula ruralis</em> (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). We were unable to place these two LIM proteins within either group because their sequence shares similarities with both LIM groups. Therefore, the phylogenetic trees have been rooted using sequences of these two Bryophyte LIM proteins. The plant LIM family can be divided into four groups, αLIM1, βLIM1, γLIM2, and δLIM2 resulting from the division of the LIM1 and LIM2 groups. These four groups are supported by high bootstrap values. This phylogenetic analysis confirms the existence of the PLIM1, WLIM1, PLIM2, and WLIM2 subgroups as described previously.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> βLIM1 is a new group that has not been identified before, whereas the αLIM1 group includes PLIM1 and WLIM1 subgroups. The WLIM2 and PLIM2 subgroups belong, respectively, to the γLIM2 and δLIM2 groups. In addition, each group contains new subgroups, which are described below.</p>                        <a id="87196017" scrollto-destination="87196017"></a>
<div id="DSM013F3" data-id="DSM013F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01303.jpeg?Expires=2147483647&Signature=jqtnHIfGCdHMcmaIsqaO5YW0TEJ~vKRuDgqrz70y2ntz9bXAElCXFgfnpHz7jOtN-M76NuUCa6ssm-k2fNwJNNkYxskTwPUMZbrI2qxvF-X3yQFP1DtO5et7ISaBt5wtu5v2IZAUwEd6ISYW8SJmeHA9duzU7-qjZ8S40LbE6mV2-Uxjc3uFw0kY04u-pDlFQMeeDDYDYImU4agf8hCsDxLiK2AvKEjvg-0m5hEBHmivSLrnahA0V1leQlgLkEspe69hXC21ze768H7veR1TdMNP1XLpgiIJaN3U2-YAiFIK-o5u2dosarN4XX5sIkijTo4VpWq-9hqjlAlt3eth6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of plant LIM domain proteins. Amino acid sequences of 149 LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent bootstraps values (≥500) based on 1000 replications. The lengths of the branches are proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the tree. LIM proteins have been renamed following their belonging to groups or subgroups WLIM1, PLIM1, FLIM1, αLIM1, βLIM1, WLIM2, PLIM2, or δLIM2 that are in boxes. Poplar LIM proteins are in blue color, and plant LIM proteins whose expression pattern has been determined are in red color. A species acronym is added before each LIM protein name: Afp, Aquilegia formosa × A. pubescens; Am, Antirrhinum majus; At, A. thaliana; Bd, Brachypodium distachyon; Bn, Brassica napus; Bv, Beta vulgaris; Cc, Coffea canephora; Cl, Curcuma longa; Cs, Citrus sinensis; Cr, Ceratopteris richardii; Ec, Eleusine coracana; Ee, Euphorbia esula; Et, Euphorbia tirucalli; Eg, Eucalyptus globulus; Fv, Fragaria vesca; Geh, Gerbera hybrida; Gm, Glycine max; Gh, Gossypium hirsutum; Gr, Gossypium raimondii; Ga, Gossypium arboretum; Ha, Helianthus annuus; Hp, Helianthus petiolaris; Hv, Hordeum vulgare; In, Ipomoea nil; Le; Lycopersicon esculentum; Lp, Lycopersicon pennellii; Ls, Lactuca sativa; Md, Malus × domestica; Mt, Medicago truncatula; Mc, Mesembryanthemum crystallinum; Nt, Nicotiana tabacum; Ob, Ocimum basilicum; Pa, Prunus armeniaca; Pg, Picea glauca; Ph, Petunia × hybrida; Pit, Pinus taeda; Pp, Physcomitrella patens; Pt, Populus trichocarpa; Pv, Panicum virgatum; Sea, Senecio aethnensis; Sec, Senecio chrysanthemifolius; Sb, Sorghum bicolor; Sh, Saruma henryi; So, Saccharum officinarum; St, Solanum tuberosum; Sc, Solanum chacoense; Ta, Triticum aestivum; To, Taraxacum officinale; Tk, Taraxacum kok-saghyz; Tr, Tortula ruralis; Vv, Vitis vinifera; Zo, Zingiber officinale; Zm, Zea mays. For additional details on each gene, see Supplementary Table S2." path-from-xml="dsm01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196017/dsm01303.jpeg" path-from-xml="dsm01303.jpeg" target="_blank">View large</a><a section="87196017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01303.jpeg?Expires=2147483647&Signature=5QV22FiUk49JoiNZWg47x994nhKc6ndvW8~geC4SLa3dVlspP4SV3OliWn5wEBPEY2rDVQpKdcYZfvpvbqlLxtxk~hJ8v9~pKpUWcfuC5lxibBNYqiF0aZZ6LLIPWcWEvKWjQKMCRK51tQu8ZlK97kWpB60w-12~4mX8NF5gIrGGzZwdnH5OpJXliIHq24vpMicTsfWhl3GufUBb5n6ZMeLV8eTGHBrUiVwbd7p6O2PjuQ5iJXTi1nKrCIjjemiRX1dFYwO~va-csgab-RSAgtlSqwkq2bwkXA8ROhPLCUU3UCtc-~UunCz7SGrTvc87IQmS2ecHGpX0KtwmPfNAsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196017&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of plant LIM domain proteins. Amino acid sequences of 149 LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent bootstraps values (≥500) based on 1000 replications. The lengths of the branches are proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the tree. LIM proteins have been renamed following their belonging to groups or subgroups WLIM1, PLIM1, FLIM1, αLIM1, βLIM1, WLIM2, PLIM2, or δLIM2 that are in boxes. Poplar LIM proteins are in blue color, and plant LIM proteins whose expression pattern has been determined are in red color. A species acronym is added before each LIM protein name: Afp, <em>Aquilegia formosa</em> × <em>A. pubescens</em>; Am, <em>Antirrhinum majus</em>; At, <em>A. thaliana</em>; Bd, <em>Brachypodium distachyon</em>; Bn, <em>Brassica napus</em>; Bv, <em>Beta vulgaris</em>; Cc, <em>Coffea canephora</em>; Cl, <em>Curcuma longa</em>; Cs, <em>Citrus sinensis</em>; Cr, <em>Ceratopteris richardii</em>; Ec, <em>Eleusine coracana</em>; Ee, <em>Euphorbia esula</em>; Et, <em>Euphorbia tirucalli</em>; Eg, <em>Eucalyptus globulus</em>; Fv, <em>Fragaria vesca</em>; Geh<em>, Gerbera hybrida</em>; Gm, <em>Glycine max</em>; Gh, <em>Gossypium hirsutum</em>; Gr, <em>Gossypium raimondii</em>; Ga, <em>Gossypium arboretum</em>; Ha, <em>Helianthus annuus</em>; Hp, <em>Helianthus petiolaris</em>; Hv, <em>Hordeum vulgare</em>; In, <em>Ipomoea nil</em>; Le; <em>Lycopersicon esculentum</em>; Lp, <em>Lycopersicon pennellii</em>; Ls, <em>Lactuca sativa</em>; Md, <em>Malus</em> × <em>domestica</em>; Mt, <em>Medicago truncatula</em>; Mc, <em>Mesembryanthemum crystallinum</em>; Nt, <em>Nicotiana tabacum</em>; Ob, <em>Ocimum basilicum</em>; Pa, <em>Prunus armeniaca</em>; Pg, <em>Picea glauca</em>; Ph, <em>Petunia</em> × <em>hybrida</em>; Pit, <em>Pinus taeda</em>; Pp, <em>Physcomitrella patens</em>; Pt, <em>Populus trichocarpa</em>; Pv, <em>Panicum virgatum</em>; Sea, <em>Senecio aethnensis</em>; Sec, <em>Senecio chrysanthemifolius</em>; Sb, <em>Sorghum bicolor</em>; Sh, <em>Saruma henryi</em>; So, <em>Saccharum officinarum</em>; St, <em>Solanum tuberosum</em>; Sc, <em>Solanum chacoense</em>; Ta, <em>Triticum aestivum</em>; To, <em>Taraxacum officinale</em>; Tk, <em>Taraxacum kok-saghyz</em>; Tr, <em>Tortula ruralis</em>; Vv, <em>Vitis vinifera</em>; Zo<em>, Zingiber officinale</em>; Zm<em>, Zea mays</em>. For additional details on each gene, see Supplementary Table S2.</p></div></div><div content-id="DSM013F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01303.jpeg?Expires=2147483647&Signature=jqtnHIfGCdHMcmaIsqaO5YW0TEJ~vKRuDgqrz70y2ntz9bXAElCXFgfnpHz7jOtN-M76NuUCa6ssm-k2fNwJNNkYxskTwPUMZbrI2qxvF-X3yQFP1DtO5et7ISaBt5wtu5v2IZAUwEd6ISYW8SJmeHA9duzU7-qjZ8S40LbE6mV2-Uxjc3uFw0kY04u-pDlFQMeeDDYDYImU4agf8hCsDxLiK2AvKEjvg-0m5hEBHmivSLrnahA0V1leQlgLkEspe69hXC21ze768H7veR1TdMNP1XLpgiIJaN3U2-YAiFIK-o5u2dosarN4XX5sIkijTo4VpWq-9hqjlAlt3eth6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of plant LIM domain proteins. Amino acid sequences of 149 LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent bootstraps values (≥500) based on 1000 replications. The lengths of the branches are proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the tree. LIM proteins have been renamed following their belonging to groups or subgroups WLIM1, PLIM1, FLIM1, αLIM1, βLIM1, WLIM2, PLIM2, or δLIM2 that are in boxes. Poplar LIM proteins are in blue color, and plant LIM proteins whose expression pattern has been determined are in red color. A species acronym is added before each LIM protein name: Afp, Aquilegia formosa × A. pubescens; Am, Antirrhinum majus; At, A. thaliana; Bd, Brachypodium distachyon; Bn, Brassica napus; Bv, Beta vulgaris; Cc, Coffea canephora; Cl, Curcuma longa; Cs, Citrus sinensis; Cr, Ceratopteris richardii; Ec, Eleusine coracana; Ee, Euphorbia esula; Et, Euphorbia tirucalli; Eg, Eucalyptus globulus; Fv, Fragaria vesca; Geh, Gerbera hybrida; Gm, Glycine max; Gh, Gossypium hirsutum; Gr, Gossypium raimondii; Ga, Gossypium arboretum; Ha, Helianthus annuus; Hp, Helianthus petiolaris; Hv, Hordeum vulgare; In, Ipomoea nil; Le; Lycopersicon esculentum; Lp, Lycopersicon pennellii; Ls, Lactuca sativa; Md, Malus × domestica; Mt, Medicago truncatula; Mc, Mesembryanthemum crystallinum; Nt, Nicotiana tabacum; Ob, Ocimum basilicum; Pa, Prunus armeniaca; Pg, Picea glauca; Ph, Petunia × hybrida; Pit, Pinus taeda; Pp, Physcomitrella patens; Pt, Populus trichocarpa; Pv, Panicum virgatum; Sea, Senecio aethnensis; Sec, Senecio chrysanthemifolius; Sb, Sorghum bicolor; Sh, Saruma henryi; So, Saccharum officinarum; St, Solanum tuberosum; Sc, Solanum chacoense; Ta, Triticum aestivum; To, Taraxacum officinale; Tk, Taraxacum kok-saghyz; Tr, Tortula ruralis; Vv, Vitis vinifera; Zo, Zingiber officinale; Zm, Zea mays. For additional details on each gene, see Supplementary Table S2." path-from-xml="dsm01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196017/dsm01303.jpeg" path-from-xml="dsm01303.jpeg" target="_blank">View large</a><a section="87196017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01303.jpeg?Expires=2147483647&Signature=5QV22FiUk49JoiNZWg47x994nhKc6ndvW8~geC4SLa3dVlspP4SV3OliWn5wEBPEY2rDVQpKdcYZfvpvbqlLxtxk~hJ8v9~pKpUWcfuC5lxibBNYqiF0aZZ6LLIPWcWEvKWjQKMCRK51tQu8ZlK97kWpB60w-12~4mX8NF5gIrGGzZwdnH5OpJXliIHq24vpMicTsfWhl3GufUBb5n6ZMeLV8eTGHBrUiVwbd7p6O2PjuQ5iJXTi1nKrCIjjemiRX1dFYwO~va-csgab-RSAgtlSqwkq2bwkXA8ROhPLCUU3UCtc-~UunCz7SGrTvc87IQmS2ecHGpX0KtwmPfNAsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196017&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of plant LIM domain proteins. Amino acid sequences of 149 LIM domain proteins were analyzed by neighbor-joining method with genetic distance calculated by the JTT model of amino acids change. The numbers at the nodes represent bootstraps values (≥500) based on 1000 replications. The lengths of the branches are proportional to the expected numbers of amino acid substitutions per site with a scale provided at the bottom of the tree. LIM proteins have been renamed following their belonging to groups or subgroups WLIM1, PLIM1, FLIM1, αLIM1, βLIM1, WLIM2, PLIM2, or δLIM2 that are in boxes. Poplar LIM proteins are in blue color, and plant LIM proteins whose expression pattern has been determined are in red color. A species acronym is added before each LIM protein name: Afp, <em>Aquilegia formosa</em> × <em>A. pubescens</em>; Am, <em>Antirrhinum majus</em>; At, <em>A. thaliana</em>; Bd, <em>Brachypodium distachyon</em>; Bn, <em>Brassica napus</em>; Bv, <em>Beta vulgaris</em>; Cc, <em>Coffea canephora</em>; Cl, <em>Curcuma longa</em>; Cs, <em>Citrus sinensis</em>; Cr, <em>Ceratopteris richardii</em>; Ec, <em>Eleusine coracana</em>; Ee, <em>Euphorbia esula</em>; Et, <em>Euphorbia tirucalli</em>; Eg, <em>Eucalyptus globulus</em>; Fv, <em>Fragaria vesca</em>; Geh<em>, Gerbera hybrida</em>; Gm, <em>Glycine max</em>; Gh, <em>Gossypium hirsutum</em>; Gr, <em>Gossypium raimondii</em>; Ga, <em>Gossypium arboretum</em>; Ha, <em>Helianthus annuus</em>; Hp, <em>Helianthus petiolaris</em>; Hv, <em>Hordeum vulgare</em>; In, <em>Ipomoea nil</em>; Le; <em>Lycopersicon esculentum</em>; Lp, <em>Lycopersicon pennellii</em>; Ls, <em>Lactuca sativa</em>; Md, <em>Malus</em> × <em>domestica</em>; Mt, <em>Medicago truncatula</em>; Mc, <em>Mesembryanthemum crystallinum</em>; Nt, <em>Nicotiana tabacum</em>; Ob, <em>Ocimum basilicum</em>; Pa, <em>Prunus armeniaca</em>; Pg, <em>Picea glauca</em>; Ph, <em>Petunia</em> × <em>hybrida</em>; Pit, <em>Pinus taeda</em>; Pp, <em>Physcomitrella patens</em>; Pt, <em>Populus trichocarpa</em>; Pv, <em>Panicum virgatum</em>; Sea, <em>Senecio aethnensis</em>; Sec, <em>Senecio chrysanthemifolius</em>; Sb, <em>Sorghum bicolor</em>; Sh, <em>Saruma henryi</em>; So, <em>Saccharum officinarum</em>; St, <em>Solanum tuberosum</em>; Sc, <em>Solanum chacoense</em>; Ta, <em>Triticum aestivum</em>; To, <em>Taraxacum officinale</em>; Tk, <em>Taraxacum kok-saghyz</em>; Tr, <em>Tortula ruralis</em>; Vv, <em>Vitis vinifera</em>; Zo<em>, Zingiber officinale</em>; Zm<em>, Zea mays</em>. For additional details on each gene, see Supplementary Table S2.</p></div></div>                            <h4 scrollto-destination=87196020 id="87196020" class="section-title" data-legacy-id=s3b2>3.2.2. The αLIM1 group</h4>
<p>The previous WLIM1 and PLIM1 subgroups are gathered within the αLIM1 group. With regards to the low bootstrap value supporting the node, the definition of these two subgroups remains questionable (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). In contrast, the monocots WLIM1 subgroup clearly forms a statistically significant new monophyletic clade distinct from the eudicots WLIM1 subgroup. Additionally, a fourth subgroup FLIM1 could also be assigned to the αLIM1 group. However, the respective position of the PLIM1, FLIM1, and monocots and eudicots WLIM1 subgroups within the αLIM1 group needs to be clarified. Indeed, the neighbor-joining trees generated from a matrix of distances calculated by the JTT method or from the maximum likelihood (PhyML) method gave us an FLIM1 subgroup close to the eudicots and monocots WLIM1 subgroups and clearly separated from the PLIM1 subgroup (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>; Supplementary Fig. S3). However, when using the parsimonious method, the FLIM1 subgroup takes place within the PLIM1 subgroup, between the PLIM1 proteins from the Solanaceae and the Asteraceae families (Supplementary Fig. S3). Furthermore, this last method favors the hypothesis of a common ancestor for the monocots WLIM1 and PLIM1 subgroups. Presently, the FLIM1 subgroup includes only a few proteins, namely PtGLIM1a and b, GhαLIM1, and EeαLIM1. Although additional sequences are definitely needed to strengthen this classification, there are strong arguments for the creation of a FLIM1 subgroup: (i) PtGLIM1a and b share 81% of amino acid identity with GhαLIM1 and 79% with EeαLIM1; (ii) RT–PCR analyses indicate that the <em>PtGLIM1a</em> and <em>b</em> genes are not expressed in pollen (data not shown), which could be an argument against grouping <em>PtGLIM1a</em> and <em>b</em> within the PLIM1 subgroup. With 58 ESTs coming mostly from different xylem cDNA libraries (Fig. <a reveal-id="DSM013F4" data-open="DSM013F4" class="link link-reveal link-table xref-fig">4</a>), <em>PtGLIM1a</em> is probably highly expressed in xylem tissues and to a lesser extent in other organs with vascular tissues, such as leaves. In contrast to <em>PtGLIM1a</em>, <em>PtGLIM1b</em> ESTs are preferentially found in cambial zone and phloem. The <em>PtGLIM1a</em> and <em>b</em> names include a G, indicative of their abundance in tension wood with G-fibers.<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSM013C17" data-open="DSM013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Similarly, all of the 31 <em>GhαLIM1</em> ESTs were found in mature cotton fibers and a majority (29 ESTs) came from fibers harvested during secondary cell wall formation (Supplementary Table S2). Accordingly, we named this subgroup FLIM1 with the F, indicative of the high expression of its members in fibers. We can speculate that during the evolution of poplar and cotton species, these three proteins may have gained a novel function important for fiber differentiation or maturation and more particularly in fibers containing a cellulose-enriched secondary cell wall.</p>                        <a id="87196022" scrollto-destination="87196022"></a>
<div id="DSM013F4" data-id="DSM013F4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01304.jpeg?Expires=2147483647&Signature=uustqMvRc7XUsSP0nGWxVSf8XWzQXpr5FZCjtNCK2L-ugDDcpVaFBhPgd~-g0XX0~EYnON5Sawp9rI8X0-AROxXkFophldDiUAXVjvHa4nkp6I4LRXG8M0o1nZoWgYSarnZzjU5LSIHWPJFUrrCTzvsOAfzlrFsdmHY0izJ7iqA0dSpIJkfzpQSQPMYIH3wsjtsZBsGS0QS4YLiKfMK-AJah3Z2bFvJwsZ9Kopc6-OjQcySc2sMDi7OsEUHh85n253Iib8QqHtUUXNMbxn9Uoujj2aBYkQclHvNJ41VzWuACfHc3r~6hugchJ7J1WGdQEpezNXz1lMVTLDua86X3Ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of poplar LIM EST in different poplar cDNA libraries. (A) Libraries from INRA-Orléans.16 (B) Libraries from Umea Plant Science Centre (UPSC).20 (C) Libraries from University of British Columbia (UBC):28 cDNA libraries prepared from young and mature leaves, along with green shoot tips (PT-GT-FL-A-3), local mature leaves harvested after continuous feeding by M. disstria (PTxD-IL-N-A-9), outer xylem harvested biweekly between April and October 2002 (PT-DX-A-7 and PT-DX-N-A-10), phloem and cambium (PT-P-FL-A-2), bark (with phloem and cambium attached) harvested after continuous feeding by C. lapathi (PTxN-IB-N-A-11), roots harvested from 3-month-old trees grown in hydroponic media without nitrogen source for 24 and 48 h, as well as trees grown in regular media (PTxD-NR-A-8), cultured cells grown in media supplemented with salicylic acid, benzothiadazole, methyl jasmonate, chitosan, or Pollacia radiosa extract (PTxD-ICC-N-A-14); for more details, see Ralph et al.28" path-from-xml="dsm01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196022/dsm01304.jpeg" path-from-xml="dsm01304.jpeg" target="_blank">View large</a><a section="87196022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01304.jpeg?Expires=2147483647&Signature=3bEzjEDxL-xXwygJzyUVRynVIZZGTHfAnLibuKZIjZrmh42tJ5CpjwCx5Z-vvoyHwsNxtZOEb0YM6ljgehBjZSsR-TWWINLBe0eXZBm6D~MmSnUahsBCEnfupAMKAhGVXGBzG3apAcfJplx~FD-d4n4S3iuN2odtfzwjsHDLnlAZUiT1JvuvE~j6XGuOwcvr76PAexgLhKhhDlEtWRUlcsEvPYSSxASm5nVQTWHwtM4AeM-iDszOxdrPwkAlccVLCQZwing98cwwFYaPHk66kxiWYqPv-i1Jd0f0Oc~lUuXTF51lJpwmYprYSzlfJPh2w-leCH6h9ib4rqhDpNDcbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196022&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of poplar LIM EST in different poplar cDNA libraries. (<strong>A</strong>) Libraries from INRA-Orléans.<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (<strong>B</strong>) Libraries from Umea Plant Science Centre (UPSC).<sup><a reveal-id="DSM013C20" data-open="DSM013C20" class="link link-ref link-reveal xref-bibr">20</a></sup> (<strong>C</strong>) Libraries from University of British Columbia (UBC):<sup><a reveal-id="DSM013C28" data-open="DSM013C28" class="link link-ref link-reveal xref-bibr">28</a></sup> cDNA libraries prepared from young and mature leaves, along with green shoot tips (PT-GT-FL-A-3), local mature leaves harvested after continuous feeding by <em>M. disstria</em> (PTxD-IL-N-A-9), outer xylem harvested biweekly between April and October 2002 (PT-DX-A-7 and PT-DX-N-A-10), phloem and cambium (PT-P-FL-A-2), bark (with phloem and cambium attached) harvested after continuous feeding by <em>C. lapathi</em> (PTxN-IB-N-A-11), roots harvested from 3-month-old trees grown in hydroponic media without nitrogen source for 24 and 48 h, as well as trees grown in regular media (PTxD-NR-A-8), cultured cells grown in media supplemented with salicylic acid, benzothiadazole, methyl jasmonate, chitosan, or <em>Pollacia radiosa</em> extract (PTxD-ICC-N-A-14); for more details, see Ralph et al.<sup><a reveal-id="DSM013C28" data-open="DSM013C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p></div></div><div content-id="DSM013F4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01304.jpeg?Expires=2147483647&Signature=uustqMvRc7XUsSP0nGWxVSf8XWzQXpr5FZCjtNCK2L-ugDDcpVaFBhPgd~-g0XX0~EYnON5Sawp9rI8X0-AROxXkFophldDiUAXVjvHa4nkp6I4LRXG8M0o1nZoWgYSarnZzjU5LSIHWPJFUrrCTzvsOAfzlrFsdmHY0izJ7iqA0dSpIJkfzpQSQPMYIH3wsjtsZBsGS0QS4YLiKfMK-AJah3Z2bFvJwsZ9Kopc6-OjQcySc2sMDi7OsEUHh85n253Iib8QqHtUUXNMbxn9Uoujj2aBYkQclHvNJ41VzWuACfHc3r~6hugchJ7J1WGdQEpezNXz1lMVTLDua86X3Ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of poplar LIM EST in different poplar cDNA libraries. (A) Libraries from INRA-Orléans.16 (B) Libraries from Umea Plant Science Centre (UPSC).20 (C) Libraries from University of British Columbia (UBC):28 cDNA libraries prepared from young and mature leaves, along with green shoot tips (PT-GT-FL-A-3), local mature leaves harvested after continuous feeding by M. disstria (PTxD-IL-N-A-9), outer xylem harvested biweekly between April and October 2002 (PT-DX-A-7 and PT-DX-N-A-10), phloem and cambium (PT-P-FL-A-2), bark (with phloem and cambium attached) harvested after continuous feeding by C. lapathi (PTxN-IB-N-A-11), roots harvested from 3-month-old trees grown in hydroponic media without nitrogen source for 24 and 48 h, as well as trees grown in regular media (PTxD-NR-A-8), cultured cells grown in media supplemented with salicylic acid, benzothiadazole, methyl jasmonate, chitosan, or Pollacia radiosa extract (PTxD-ICC-N-A-14); for more details, see Ralph et al.28" path-from-xml="dsm01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196022/dsm01304.jpeg" path-from-xml="dsm01304.jpeg" target="_blank">View large</a><a section="87196022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01304.jpeg?Expires=2147483647&Signature=3bEzjEDxL-xXwygJzyUVRynVIZZGTHfAnLibuKZIjZrmh42tJ5CpjwCx5Z-vvoyHwsNxtZOEb0YM6ljgehBjZSsR-TWWINLBe0eXZBm6D~MmSnUahsBCEnfupAMKAhGVXGBzG3apAcfJplx~FD-d4n4S3iuN2odtfzwjsHDLnlAZUiT1JvuvE~j6XGuOwcvr76PAexgLhKhhDlEtWRUlcsEvPYSSxASm5nVQTWHwtM4AeM-iDszOxdrPwkAlccVLCQZwing98cwwFYaPHk66kxiWYqPv-i1Jd0f0Oc~lUuXTF51lJpwmYprYSzlfJPh2w-leCH6h9ib4rqhDpNDcbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196022&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of poplar LIM EST in different poplar cDNA libraries. (<strong>A</strong>) Libraries from INRA-Orléans.<sup><a reveal-id="DSM013C16" data-open="DSM013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (<strong>B</strong>) Libraries from Umea Plant Science Centre (UPSC).<sup><a reveal-id="DSM013C20" data-open="DSM013C20" class="link link-ref link-reveal xref-bibr">20</a></sup> (<strong>C</strong>) Libraries from University of British Columbia (UBC):<sup><a reveal-id="DSM013C28" data-open="DSM013C28" class="link link-ref link-reveal xref-bibr">28</a></sup> cDNA libraries prepared from young and mature leaves, along with green shoot tips (PT-GT-FL-A-3), local mature leaves harvested after continuous feeding by <em>M. disstria</em> (PTxD-IL-N-A-9), outer xylem harvested biweekly between April and October 2002 (PT-DX-A-7 and PT-DX-N-A-10), phloem and cambium (PT-P-FL-A-2), bark (with phloem and cambium attached) harvested after continuous feeding by <em>C. lapathi</em> (PTxN-IB-N-A-11), roots harvested from 3-month-old trees grown in hydroponic media without nitrogen source for 24 and 48 h, as well as trees grown in regular media (PTxD-NR-A-8), cultured cells grown in media supplemented with salicylic acid, benzothiadazole, methyl jasmonate, chitosan, or <em>Pollacia radiosa</em> extract (PTxD-ICC-N-A-14); for more details, see Ralph et al.<sup><a reveal-id="DSM013C28" data-open="DSM013C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p></div></div><p>Interestingly, the PLIM1 subgroup contains only LIM proteins from Solanaceae and Asteraceae species from the asterids subclass (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). Many expression studies have demonstrated the pollen-specific expression of these <em>PLIM1</em> genes. For example, the sunflower HaPLIM1a (SF3) is a protein exclusively and highly expressed in maturing pollen,<sup><a reveal-id="DSM013C13" data-open="DSM013C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and its colocalization with actin in the germinating cone suggests that HaPLIM1a participates in pollen germination or pollen tube elongation. The tobacco <em>NtPLIM1a</em> gene has also been shown to be highly expressed in mature pollen,<sup><a reveal-id="DSM013C33" data-open="DSM013C33" class="link link-ref link-reveal xref-bibr">33</a></sup> whereas in <em>Gerberea hybrida</em>, the expression of <em>GhPLIM1a</em> gene is highly upregulated in stamens as evidenced in microarray studies.<sup><a reveal-id="DSM013C34" data-open="DSM013C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Likewise, the petunia <em>PGPS/D1</em> gene encoding a PLIM1 protein has been found to be highly expressed in germinating pollen,<sup><a reveal-id="DSM013C35" data-open="DSM013C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and the accumulation of the <em>PhPLIM1</em> transcript increases during anther maturation suggesting again an involvement of PLIM1 proteins in pollen tube elongation. PLIM1 proteins from Solanaceae species are clearly separated from those belonging to the Asteraceae species, suggesting that PLIM1 proteins have diverged during the speciation of these two families.</p><p>The monocots WLIM1 subgroup contains proteins with high homology (from 89 to 99% of amino acid identity). Monocots <em>WLIM1</em> ESTs originate from a large amount of tissue including inflorescences. Only the closely related <em>SoWLIM1a</em> and <em>b</em> genes from sugarcane seem to be duplicated, as the two maize <em>WLIM1</em> transcripts described are probably alleles of the same gene. Surprisingly, a high number of <em>WLIM1</em> ESTs has been sequenced from the maize (200) and sugarcane (128) cDNA libraries. In particular, <em>SoWLIM1a</em> and <em>b</em> with 76 ESTs is the biggest cluster of the ‘shoot-root transition zone from adult plant’ cDNA library.</p><p>All plant species from core eudicotyledons are represented within the large eudicots WLIM1 subgroup. Within this subgroup, only poplar, <em>Ipomoea</em>, and soybean species have in-paralogs WLIM1 proteins (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). In general, each plant <em>WLIM1</em> gene is represented by a large number of ESTs originating from a wide variety of tissues, sometimes including inflorescence organs. The <em>WLIM1</em> genes have been the most studied <em>LIM</em> genes in plants, but their precise function is yet to be defined. In the sunflower, the HaWLIM1 protein is localized in the nucleus and/or the cytoplasm for different cell types,<sup><a reveal-id="DSM013C10" data-open="DSM013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and may be associated with the tubulin cytoskeleton in protoplasts.<sup><a reveal-id="DSM013C11" data-open="DSM013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> A tobacco <em>WLIM1</em> gene, <em>NtLIM1</em>, highly expressed in the stem, acts as a transcription activator. Indeed, the binding of the NtLIM1 protein to the PAL-box motif, present on the promoter of several genes from the lignin biosynthesis pathway, leads to the activation of the transcription of these genes. Moreover, an important reduction in the lignin content has been observed in antisense <em>Ntlim1</em> transgenic tobacco.<sup><a reveal-id="DSM013C15" data-open="DSM013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> NtLIM1 protein mostly differs from NtWLIM1 protein<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> by seven additional residues in the C-ter of the protein. The NtWLIM1 protein binds actin directly with a high affinity, enhances the stability of actin cytoskeleton, and promotes the bundling of actin filaments.<sup><a reveal-id="DSM013C12" data-open="DSM013C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The poplar ESTs homologous to <em>PtWLIM1a</em> and <em>b</em> are more represented in vascular tissues with many ESTs from phloem, cambium, and xylem (Fig. <a reveal-id="DSM013F4" data-open="DSM013F4" class="link link-reveal link-table xref-fig">4</a>). In <em>Eucalyptus globulus</em>, the homologous <em>EgLIM1</em> gene (AB208710) and the corresponding cDNA (AB208709) have also been isolated, but there is no expression information available. We have been unable to position the ShαLIM1, AfpαLIM1, VvαLIM1, and CsαLIM1 proteins in either subgroup of the αLIM1 group, as illustrated by the low bootstrap values supporting the nodes.</p><p>In conifer trees, the LIM1 proteins are very similar in structure in white spruce (<em>Picea glauca</em>) and loblolly pine (<em>Pinus taeda</em>). For both species, the LIM1 proteins have undergone a probable duplication event, and it is difficult to class them into either the αLIM1 or the βLIM1 subgroup (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). In <em>Pinus taeda</em>, the <em>PitLIM1b</em> (clone 6C12H) gene is more highly expressed in the xylem than in any other part of the tree.<sup><a reveal-id="DSM013C36" data-open="DSM013C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Therefore, at least in tree species, the LIM domain proteins from the WLIM1 subgroup, as well as those from the FLIM1 subgroup, are preferentially expressed during vascular development.</p>                            <h4 scrollto-destination=87196035 id="87196035" class="section-title" data-legacy-id=s3b3>3.2.3. The new βLIM1 group</h4>
<p>The new βLIM1 group contains LIM proteins from species mostly belonging to the asterids and rosids subclasses, including the poplar duplicated PtβLIM1a and b proteins. Interestingly, there are no <em>Arabidopsis</em> or rice proteins into this group (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). The proteins from Solanaceae and Asteraceae form two distinct smaller clades within this group. Within the asterids subclass, only the Solanaceae species (tobacco, potato, and tomato) have in-paralogs βLIM1 proteins. The expression or function of the <em>βLIM1</em> genes is yet unknown. Like the genes from the WLIM1 and WLIM2 subgroups, <em>βLIM1</em> ESTs originate from a variety of tissues including flower and fruit (data not shown), but they are preferentially found in roots, in cells that undergo differentiation like immature cotton fibers, and in undifferentiated cells like callus. In poplar, the <em>PtβLIM1a</em> ESTs almost exclusively originate from cell suspension cultures and the <em>PtβLIM1b</em> ESTs are more widely represented in xylem, vegetative buds, and roots (Fig. <a reveal-id="DSM013F4" data-open="DSM013F4" class="link link-reveal link-table xref-fig">4</a>). Although <em>βLIM1</em> ESTs originate from all parts of the plant, a significant number has been sequenced from undifferentiated cell samples.</p>                            <h4 scrollto-destination=87196045 id="87196045" class="section-title" data-legacy-id=s3b4>3.2.4. The γLIM2 group</h4>
<p>The γLIM2 group is clearly separated from the δLIM2 group with a high bootstrap value (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). The γLIM2 group contains the monocots and eudicots WLIM2 subgroups that are clearly separated. As in the WLIM1 subgroup, the short branches are indicative of a high conservation between WLIM2 protein sequences within each subgroup. In several eudicots such as <em>Arabidopsis</em>, poplar, cotton, and lettuce, <em>WLIM2</em> genes are duplicated, whereas this is never the case in monocot species. For both monocots and eudicots WLIM2 subgroups, a high number of ESTs from various tissues, including inflorescences, have been sequenced. This is the case in <em>Arabidopsis</em> for the <em>AtWLIM2a</em> gene<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> as well as in poplar, for the <em>PtWLIM2a</em> and <em>b</em> genes (Fig. <a reveal-id="DSM013F4" data-open="DSM013F4" class="link link-reveal link-table xref-fig">4</a>). The wide range of expression and the high conservation of <em>WLIM2</em> genes suggest that they may be involved in basic cellular processes. This is substantiated by the difficulty to produce homozygous mutants for <em>AtWLIM2a</em> gene.<sup><a reveal-id="DSM013C37" data-open="DSM013C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In a recent microarray study, <em>AtWLIM2a</em> appeared highly expressed in siliques, and its expression was induced in the <em>Arabidopsis pkl</em> mutant.<sup><a reveal-id="DSM013C37" data-open="DSM013C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The PKL protein is a negative regulator of transcription that acts as a repressor of embryonic identity during seed germination, and <em>AtWLIM2a</em> gene may be involved in the regulation of embryo development. In another study, <em>AtWLIM2a</em> expression is also gradually induced during leaf senescence.<sup><a reveal-id="DSM013C38" data-open="DSM013C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Taken together, the <em>AtWLIM2a</em> gene may be more precisely involved in the seed maturation. The duplicate of <em>AtWLIM2a</em>, <em>AtWLIM2b</em> (<em>AtL2</em>), is more highly expressed in roots than in shoots. In addition, this expression seems to be affected by nitrogen availability.<sup><a reveal-id="DSM013C31" data-open="DSM013C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                            <h4 scrollto-destination=87196052 id="87196052" class="section-title" data-legacy-id=s3b5>3.2.5. The δLIM2 group</h4>
<p>Because of the increased number of LIM sequences available, the δLIM2 group is considerably enlarged. The δLIM2 group is divided into three monophyletic subgroups: the eudicots PLIM2, monocots PLIM2, and Asterids δLIM2 that contains the sequences from the new LIM domain proteins identified in sunflower and tobacco plants (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). Several duplication events have occurred within the δ<em>LIM2</em> genes, indicative of an important diversification within this group. Because of the long length of branches, the δLIM2 proteins are on average more divergent each other than proteins from any other groups. Most eudicots <em>PLIM2</em> genes appear preferentially expressed during pollen development. This is the case for <em>AtPLIM2a</em> expressed in hydrated pollen grains,<sup><a reveal-id="DSM013C39" data-open="DSM013C39" class="link link-ref link-reveal xref-bibr">39</a></sup> for the tobacco <em>NtPLIM2</em>, and the sunflower <em>HaPLIM2</em> genes.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Likewise, in <em>Gerbera hybrida</em>, the ESTs encoding an homologue to <em>HaPLIM2</em> also appeared exclusively expressed in stamens <sup><a reveal-id="DSM013C34" data-open="DSM013C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and in tomato, all the EST sequences orthologous to the <em>NtPLIM2</em> gene originated from a pollen cDNA library (data not shown). Surprisingly, a poplar cDNA library prepared from male catkins<sup><a reveal-id="DSM013C20" data-open="DSM013C20" class="link link-ref link-reveal xref-bibr">20</a></sup> does not contain any <em>PtPLIM2a</em> and <em>b</em> EST, but we observed for the <em>PtPLIM2a</em> gene a strong expression in mature anthers (data not shown). It should be noted that a few <em>PtPLIM2a</em> ESTs originate from cambial zone, tension wood, and root poplar libraries (Fig. <a reveal-id="DSM013F4" data-open="DSM013F4" class="link link-reveal link-table xref-fig">4</a>), but no <em>PtPLIM2b</em> EST has been sequenced yet.</p><p>Monocots PLIM2 proteins are phylogenetically separated from eudicots PLIM2 proteins supported by a high bootstrap value. In monocots, the PLIM2 proteins are generally found in triplicate, resulting from at least two duplication events. Contrary to the highly conserved monocots WLIM2 proteins, monocots PLIM2 proteins are more divergent. As for eudicots, the monocots <em>PLIM2</em> genes are generally highly expressed in pollen, but not exclusively. Of the 200 <em>ZmPLIM2a</em> ESTs found in public databases, 156 come from flower cDNA libraries, and 66 from these ESTs originate from mature pollen. The majority of <em>OsPLIM2c</em> and <em>OsPLIM2b</em> ESTs also come from flower cDNA libraries. <em>OsPLIM2c</em> (AK072520) expression has been studied using microarray analysis during pollination and early embryogenesis in the rice pistil.<sup><a reveal-id="DSM013C40" data-open="DSM013C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Their maximal accumulation in the pistil occurs at anthesis and decreases gradually during the following 24 h, when the pollen tube reaches the micropyle. However, the same study reveals a wider expression of the <em>OsPLIM2c</em> gene, suggesting an involvement in some other processes than pollen tube elongation. We speculate that within monocots and rosids species, the extensive duplication of their <em>PLIM2</em> genes may be a counterpart to the lack of specialization of the genes from the αLIM1 group toward pollen-specific expression.</p><p>In asterids species, <em>LIM</em> genes also seem duplicated within the δLIM2 group. Contrary to the <em>Arabidopsis PLIM2</em> genes found in triplicate within the PLIM2 subgroup, the newly discovered tobacco and sunflower δLIM2 proteins are more distantly related from the previously identified HaPLIM2 and NtPLIM2 and therefore belong to a new Asterids δLIM2 subgroup (Fig. <a reveal-id="DSM013F3" data-open="DSM013F3" class="link link-reveal link-table xref-fig">3</a>). The Asterids δLIM2 subgroup is surrounded by the two monocots and eudicots PLIM2 subgroups. Because dicots and monocots <em>PLIM2</em> genes are strongly expressed in pollen, it is probable that these asterids δ<em>LIM2</em> genes are also strongly expressed in pollen, but it is not clear if these duplication events have brought any new functions in pollen development or in other processes. Poplar PtδLIM2a and b protein sequences deduced from the <em>P. trichocarpa</em> genome are phylogenetically distant from the PLIM2 subgroups. So far there is no expression data for their potential pollen-specific expression.</p>                            <h3 scrollto-destination=87196069 id="87196069" class="section-title" data-legacy-id=s3c>3.3. Sequence and structural analysis of plant LIM domain protein</h3>
                            <h4 scrollto-destination=87196070 id="87196070" class="section-title" data-legacy-id=s3c1>3.3.1. Toward a new characterization of the two LIM domains consensus sequences in plant</h4>
<p>The plant LIM domain proteins present a lot of common features with the CRP proteins, but also several specificities. This is the case for the GRR following each LIM domain found in the animal CRP proteins that lacks in all the plant LIM domain proteins.<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> However, for all plant LIM domain proteins, a glycine residue is strictly conserved nine amino acids after the last zinc ligand for each LIM domain (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>B). The potential nuclear targeting signal (KKYGPK) present in the CRP family of animal LIM proteins is also missing in the plant LIM domain family. The LIM domains contain two zinc finger repeated in tandem with the characteristic structure [C-X<sub>2</sub>-C-X<sub>17</sub>-H-X<sub>2</sub>-C]-X<sub>2</sub>-[C-X<sub>2</sub>-C-X<sub>17</sub>-C-X<sub>2</sub>-H] (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>).<sup><a reveal-id="DSM013C7" data-open="DSM013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Compared with the CRP family in animals, the second LIM domain in plants is atypical with the second ligand of the second zinc finger replaced by a conserved glycine. It has been proposed that closer histidine or cysteine makes the formation of two alternative structures -C-X-H-X<sub>18</sub>-C-X<sub>2</sub>-H- or -C-X<sub>4</sub>-C-X<sub>15</sub>-C-X<sub>2</sub>-H- possible for the zinc coordination.<sup><a reveal-id="DSM013C10" data-open="DSM013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the increased number of protein sequences analyzed reveals that this histidine residue is sometimes lacking in the LIM proteins from the βLIM1 and δLIM2 groups (Supplementary Fig. S4). Only the cysteine residue is conserved in all plant LIM domain proteins, strongly suggesting that the -C-X<sub>4</sub>-C-X<sub>15</sub>-C-X<sub>2</sub>-H- is the most probable structure for the second zinc finger. Interestingly, the first cysteine of the first LIM domain is missing only for the sugarcane-duplicated SoWLIM1b protein and in maize for the second allelic version of ZmWLIM1 protein supported by the cDNA AY112454 (data not shown). If this cysteine is compulsory for the formation of the zinc finger, the functional significance of this absence remains to be elucidated. In both animal CRP and plant LIM families, with the exception of the monocots PLIM2 protein, a highly conserved K[T/A]VY motif is found in the first LIM domain close to the second cysteine of the first zinc finger (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>B). This common motif may be important in the correct folding of the LIM1 domain, and is also found, but conserved to a lesser extent, at the same position in the second LIM domain. In general, the amino acids surrounding the LIM domains are also highly conserved between LIM proteins of each subgroup with the noticeable exception of the δLIM2 proteins (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>A and B). It should be pointed out that the LIM1 domain is generally more conserved than the LIM2 domain. However, in WLIM2 proteins, both LIM domains are highly conserved.</p>                        <a id="87196072" scrollto-destination="87196072"></a>
<div id="DSM013F5" data-id="DSM013F5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01305.jpeg?Expires=2147483647&Signature=2-Hbe4dENLS~BlxjHTJTqNgjbnugyu2YPftVmipQGPYo9Ac2cD5dE0Z79Sx89gEHOagJ89Pn4W3kOEUkMULGVrTqCYN7bNBza9aSmfB1Pceu833URyF2GmtvA03zsciV6ZZM67wEi4BtnyCRZBOuvP2dCbQTBdeiFV9ukUb2t2O0vq2iHwda0vj3xEKwi6QPxTRDmfvXVU7w1q-xa8mLNB3n2nyACTbh9161Rt6Cy5aL7qt5692F~qjesNgKawmGpaOEgo2z4wwm0VjfeBeXlSV3QbiYdipk-qB34wlN3CJu2bOXLAUGbZUC5jMn4D8vTTh~2SvT8qcjPm~nlQAjEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Diagrammatic representation of motifs found in the sequences of the plant LIM family proteins. The LIM1 and LIM2 domains are represented by black boxes. ASN glycosylation sites are shown in triangles. cAMP- and cGMP-dependent protein kinase (cAMP), CKII, Tyr, and PKC phosphorylation sites are, respectively, shown in grey and white diamonds, grey and white circles. Non-conserved motifs are striped. Conserved motifs surrounding the LIM domains and identified with ClustalW21 and MEME27 programs are in gray boxes. (B) Consensus sequences of the two plant LIM domains and the 15 motifs identified. Cysteine and histidine coordinating zinc finger are highlighted in bold. The last conserved glycine after each LIM domain is underlined and the conserved K[T/A]VY motif is boxed. X represents any amino acid residue." path-from-xml="dsm01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196072/dsm01305.jpeg" path-from-xml="dsm01305.jpeg" target="_blank">View large</a><a section="87196072" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01305.jpeg?Expires=2147483647&Signature=X28-T06qTamLfsSWLjsrsMyjkaECwI2k0T-kdScszlTNt2jE~AIKN67kXiAAzOCrUDIlUXeYor3nq8t-rnccdFmuYlYpNjNYEuS9BgzHYbPtVrYwvHqnFUQa96L9EDCv0IQNiHokQ0UHTHGG28XgKwrsR55eW129hYbozbi3XXFohjlpVDcXSaQ3AUswIasZHf-tLSjBdtiz7QvpK5iIYSxugUzRuKwObB~6VzCdlPi19X1HOnTSsZ9jz-khraeYNgVdiewek466t1xSke32X2GjEdHviITHOY3xjzS9E6WEMCvz4YQeNiOCAfFa3vI-WuKulhPK-QiOAG2j-aFUhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196072&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Diagrammatic representation of motifs found in the sequences of the plant LIM family proteins. The LIM1 and LIM2 domains are represented by black boxes. ASN glycosylation sites are shown in triangles. cAMP- and cGMP-dependent protein kinase (cAMP), CKII, Tyr, and PKC phosphorylation sites are, respectively, shown in grey and white diamonds, grey and white circles. Non-conserved motifs are striped. Conserved motifs surrounding the LIM domains and identified with ClustalW<sup><a reveal-id="DSM013C21" data-open="DSM013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and MEME<sup><a reveal-id="DSM013C27" data-open="DSM013C27" class="link link-ref link-reveal xref-bibr">27</a></sup> programs are in gray boxes. (<strong>B</strong>) Consensus sequences of the two plant LIM domains and the 15 motifs identified. Cysteine and histidine coordinating zinc finger are highlighted in bold. The last conserved glycine after each LIM domain is underlined and the conserved K[T/A]VY motif is boxed. X represents any amino acid residue.</p></div></div><div content-id="DSM013F5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/m_dsm01305.jpeg?Expires=2147483647&Signature=2-Hbe4dENLS~BlxjHTJTqNgjbnugyu2YPftVmipQGPYo9Ac2cD5dE0Z79Sx89gEHOagJ89Pn4W3kOEUkMULGVrTqCYN7bNBza9aSmfB1Pceu833URyF2GmtvA03zsciV6ZZM67wEi4BtnyCRZBOuvP2dCbQTBdeiFV9ukUb2t2O0vq2iHwda0vj3xEKwi6QPxTRDmfvXVU7w1q-xa8mLNB3n2nyACTbh9161Rt6Cy5aL7qt5692F~qjesNgKawmGpaOEgo2z4wwm0VjfeBeXlSV3QbiYdipk-qB34wlN3CJu2bOXLAUGbZUC5jMn4D8vTTh~2SvT8qcjPm~nlQAjEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Diagrammatic representation of motifs found in the sequences of the plant LIM family proteins. The LIM1 and LIM2 domains are represented by black boxes. ASN glycosylation sites are shown in triangles. cAMP- and cGMP-dependent protein kinase (cAMP), CKII, Tyr, and PKC phosphorylation sites are, respectively, shown in grey and white diamonds, grey and white circles. Non-conserved motifs are striped. Conserved motifs surrounding the LIM domains and identified with ClustalW21 and MEME27 programs are in gray boxes. (B) Consensus sequences of the two plant LIM domains and the 15 motifs identified. Cysteine and histidine coordinating zinc finger are highlighted in bold. The last conserved glycine after each LIM domain is underlined and the conserved K[T/A]VY motif is boxed. X represents any amino acid residue." path-from-xml="dsm01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196072/dsm01305.jpeg" path-from-xml="dsm01305.jpeg" target="_blank">View large</a><a section="87196072" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm01305.jpeg?Expires=2147483647&Signature=X28-T06qTamLfsSWLjsrsMyjkaECwI2k0T-kdScszlTNt2jE~AIKN67kXiAAzOCrUDIlUXeYor3nq8t-rnccdFmuYlYpNjNYEuS9BgzHYbPtVrYwvHqnFUQa96L9EDCv0IQNiHokQ0UHTHGG28XgKwrsR55eW129hYbozbi3XXFohjlpVDcXSaQ3AUswIasZHf-tLSjBdtiz7QvpK5iIYSxugUzRuKwObB~6VzCdlPi19X1HOnTSsZ9jz-khraeYNgVdiewek466t1xSke32X2GjEdHviITHOY3xjzS9E6WEMCvz4YQeNiOCAfFa3vI-WuKulhPK-QiOAG2j-aFUhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196072&ar=577859&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Diagrammatic representation of motifs found in the sequences of the plant LIM family proteins. The LIM1 and LIM2 domains are represented by black boxes. ASN glycosylation sites are shown in triangles. cAMP- and cGMP-dependent protein kinase (cAMP), CKII, Tyr, and PKC phosphorylation sites are, respectively, shown in grey and white diamonds, grey and white circles. Non-conserved motifs are striped. Conserved motifs surrounding the LIM domains and identified with ClustalW<sup><a reveal-id="DSM013C21" data-open="DSM013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and MEME<sup><a reveal-id="DSM013C27" data-open="DSM013C27" class="link link-ref link-reveal xref-bibr">27</a></sup> programs are in gray boxes. (<strong>B</strong>) Consensus sequences of the two plant LIM domains and the 15 motifs identified. Cysteine and histidine coordinating zinc finger are highlighted in bold. The last conserved glycine after each LIM domain is underlined and the conserved K[T/A]VY motif is boxed. X represents any amino acid residue.</p></div></div>                            <h4 scrollto-destination=87196073 id="87196073" class="section-title" data-legacy-id=s3c2>3.3.2. Structural variation may reflect functional differences</h4>
<p>As for animals, plant LIM proteins may have a function in the regulation of cell differentiation or development. LIM proteins could have multiple partners in the cell, such as actin or tubulin cytoskeleton, DNA and other potential proteins. It has been proposed that plant LIM proteins may serve as a shuttle between the cytoplasm and the nucleus for yet unknown functions. In this respect, we searched for putative phosphorylation and glycosylation sites in the LIM protein sequences using the Prosite database. Thanks to the high number of sequences collected, we have been able to perform ortholog sequence comparisons within subgroups to find conserved motifs using the MEME program. Most of the differences observed between these LIM proteins are concentrated either in the inter-LIM region or in the C-ter domain (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>). Within the αLIM1 group, PLIM1 proteins are more divergent each other than WLIM1 and FLIM1 proteins. The C-ter domain of PLIM1 proteins is more different than in WLIM1 proteins and contains additional proline residues (Supplementary Fig. S4). FLIM1 proteins mostly resemble WLIM1 proteins, particularly at the level of the LIM domains. Their inter-LIM domain is more similar to those of monocots WLIM1 proteins, whereas their C-ter domain differs clearly from that of the other subgroups. Proteins within the βLIM1 group carry on their LIM domains, the structural properties characteristic of both WLIM2 and αLIM1 groups, but the inter-LIM and C-ter domains are specific to this group. WLIM2 proteins have highly conserved sequences, even at the level of the inter-LIM region, except for WLIM2 proteins from conifers and caryophyllales. The δLIM2 group is the most heterogeneous LIM group even at the level of LIM domains. The δLIM2 proteins are characterized by a long C-ter domain highly variable in amino acid composition, and composed of many acidic amino acids, particularly glutamic acid (Supplementary Fig. S4). It has been suggested for tobacco NtLIM1 protein that this acidic domain may function as a transcription activator.<sup><a reveal-id="DSM013C15" data-open="DSM013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Interestingly, on the contrary to PLIM2 and αLIM1 proteins, the C-ter domain of WLIM2 proteins is rather basic and contains non-polar amino acids except one conserved acidic amino acid at the end of the protein. The C-ter domain of βLIM1 proteins has a variable p<em>I</em>. The p<em>I</em> is rather basic in the Solanaceae-duplicated proteins, and acid to neutral for the Asteraceae proteins (data not shown). Different kinds of motifs like the ASN glycosylation site, the casein kinase II (CKII), the tyrosine (Tyr), and the protein kinase C (PKC) phosphorylation sites are localized inside and outside the LIM domains (Fig. <a reveal-id="DSM013F5" data-open="DSM013F5" class="link link-reveal link-table xref-fig">5</a>). For example, in αLIM1 and βLIM1 proteins and only in these proteins, a putative PKC phosphorylation site is found just before the first zinc ligand of the LIM1 domain. However, the FLIM1 proteins (from the αLIM1 group) do not possess this phosphorylation site. Finally, another PKC phosphorylation site is exclusively found at the C-ter domain in WLIM2 proteins from angiosperm and also gymnosperm species, suggesting a key function for this site.</p>                            <h3 scrollto-destination=87196075 id="87196075" class="section-title" data-legacy-id=s3d>3.4. Concluding remarks</h3>
<p>In this work, we report on the first analysis of poplar LIM domain protein family with the characterization of six <em>PtaLIM</em> cDNAs. Moreover, we updated the <em>LIM</em> genomic sequences in <em>P. trichocarpa</em>, <em>A. thaliana</em>, and <em>O. sativa.</em> The set of ESTs collected and the subsequent phylogenetic classification of LIM proteins provide a useful database for future researches on this family. We have defined four phylogenetic groups, αLIM1, βLIM1, γLIM2, and δLIM2, and positioned into these groups, the previously characterized PLIM1, WLIM1, PLIM2 and WLIM2 subgroups. The βLIM1 is a newly identified group, whose genes have an apparent preferential expression in undifferentiated cells. However, more detailed expression studies are needed for the functional analysis of these βLIM1 proteins. Besides, the pollen-specific expression of <em>PLIM1</em> and <em>PLIM2</em> genes, several subgroups seem to be specific to a plant class or subclass. This is the case for the PLIM1 subgroup that is represented only by LIM protein from the asterids subclass. Additionally, it appears that LIM proteins from monocots and eudicots plant are phylogenetically distant, suggesting specific functions within each taxonomic class. Finally, a new FLIM1 subgroup was created within the αLIM1 group, and the corresponding genes appear to be highly expressed in poplar G-fibers or cotton fibers. In poplar, the distribution of ESTs corresponding to <em>WLIM1</em> and <em>GLIM1</em> genes suggests the involvement of LIM proteins in wood fibers formation and/or vascular development. Despite the fact that plant LIM proteins could both bind to actin and act as a transcription factor, the function of LIM proteins remains largely unknown. Structural analysis is the first step to the functional characterization of LIM proteins in plants. In the future, we will need to determine whether all plant LIM proteins are able to bind actin cytoskeleton and/or DNA, and to examine the function of PLIM1 and PLIM2 proteins in pollen development or pollen tube elongation.</p>                            <h2 scrollto-destination=87196077 id="87196077" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to J.-F. Arnaud and V. Castric (Laboratoire de Génétique et Evolution des Populations Végétales, Université de Lille 1, France) for their advice in phylogenetic analysis, and I. Bourgait for her help in bioinformatics analysis. We also thank Kory Wein, Assistant Professor at the University of Platteville (WI, USA) for the English editing. D. Arnaud was supported by a fellowship grant from the Conseil Régional de la Région Centre.</p>                            <h2 scrollto-destination=87196079 id="87196079" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM013C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Freyd</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Horvitz</div> <div class="given-names">H. R</div></div>. </div><div class="article-title">Novel cysteine-rich motif and homeodomain in the product of the <em>Caenorhabditis elegans</em> cell lineage gene <em>lin-II</em></div>, <div class="source">Nature</div> , <div class="year">1990</div>, vol. <div class="volume">344</div> (pg. <div class="fpage">876</div>-<div class="lpage">879</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20cysteine-rich%20motif%20and%20homeodomain%20in%20the%20product%20of%20the%20Caenorhabditis%20elegans%20cell%20lineage%20gene%20lin-II&amp;author=G.%20Freyd&amp;author=S.%20K.%20Kim&amp;author=H.%20R%20Horvitz&amp;publication_year=1990&amp;journal=Nature&amp;volume=344&amp;pages=876-879" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/344876a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F344876a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F344876a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1970421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlsson</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Thor</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Norberg</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ohlsson</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Edlund</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Insulin gene enhancer binding protein Isl-1 is a member of a novel class of proteins containing both a homeo- and a Cys–His domain</div>, <div class="source">Nature</div> , <div class="year">1990</div>, vol. <div class="volume">344</div> (pg. <div class="fpage">879</div>-<div class="lpage">882</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insulin%20gene%20enhancer%20binding%20protein%20Isl-1%20is%20a%20member%20of%20a%20novel%20class%20of%20proteins%20containing%20both%20a%20homeo-%20and%20a%20Cys%E2%80%93His%20domain&amp;author=O.%20Karlsson&amp;author=S.%20Thor&amp;author=T.%20Norberg&amp;author=H.%20Ohlsson&amp;author=T.%20Edlund&amp;publication_year=1990&amp;journal=Nature&amp;volume=344&amp;pages=879-882" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/344879a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F344879a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F344879a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1691825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Way</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Chalfie</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>mec-3</em>, a homeobox-containing gene that specifies differentiation of the touch receptor neurons in <em>C. elegans</em></div>, <div class="source">Cell</div> , <div class="year">1988</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">5</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mec-3%2C%20a%20homeobox-containing%20gene%20that%20specifies%20differentiation%20of%20the%20touch%20receptor%20neurons%20in%20C.%20elegans&amp;author=J.%20C.%20Way&amp;author=M.%20Chalfie&amp;publication_year=1988&amp;journal=Cell&amp;volume=54&amp;pages=5-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(88)90174-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(88)90174-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(88)90174-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2898300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dawid</div> <div class="given-names">I. B.</div></div>,  <div class="name"><div class="surname">Breen</div> <div class="given-names">J. J.</div></div>,  <div class="name"><div class="surname">Toyama</div> <div class="given-names">R.</div></div>. </div><div class="article-title">LIM domains: multiple roles as adapters and functional modifiers in protein interactions</div>, <div class="source">Trends Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">156</div>-<div class="lpage">162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LIM%20domains%3A%20multiple%20roles%20as%20adapters%20and%20functional%20modifiers%20in%20protein%20interactions&amp;author=I.%20B.%20Dawid&amp;author=J.%20J.%20Breen&amp;author=R.%20Toyama&amp;publication_year=1998&amp;journal=Trends%20Genet.&amp;volume=14&amp;pages=156-162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(98)01424-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(98)01424-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(98)01424-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9594664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weiskirchen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Günther</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The CRP/MLP/TLP family of LIM domain proteins: acting by connecting</div>, <div class="source">BioEssays</div> , <div class="year">2003</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">152</div>-<div class="lpage">162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20CRP%2FMLP%2FTLP%20family%20of%20LIM%20domain%20proteins%3A%20acting%20by%20connecting&amp;author=R.%20Weiskirchen&amp;author=K.%20G%C3%BCnther&amp;publication_year=2003&amp;journal=BioEssays&amp;volume=25&amp;pages=152-162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bies.10226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.10226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbies.10226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12539241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez-Alvarado</div> <div class="given-names">G. C.</div></div>,  <div class="name"><div class="surname">Miles</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Michelsen</div> <div class="given-names">J. W.</div></div>, et al. </div><div class="article-title">Structure of the carboxy-terminal LIM domain from the cysteine rich protein CRP</div>, <div class="source">Nat. Struct. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">388</div>-<div class="lpage">398</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20carboxy-terminal%20LIM%20domain%20from%20the%20cysteine%20rich%20protein%20CRP&amp;author=G.%20C.%20Perez-Alvarado&amp;author=C.%20Miles&amp;author=J.%20W.%20Michelsen&amp;publication_year=1994&amp;journal=Nat.%20Struct.%20Biol.&amp;volume=1&amp;pages=388-398" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsb0694-388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsb0694-388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsb0694-388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7664053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eliasson</div> <div class="given-names">Å.</div></div>,  <div class="name"><div class="surname">Gass</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mundel</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Molecular and expression analysis of a LIM protein gene family from flowering plants</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">264</div> (pg. <div class="fpage">257</div>-<div class="lpage">267</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20expression%20analysis%20of%20a%20LIM%20protein%20gene%20family%20from%20flowering%20plants&amp;author=%C3%85.%20Eliasson&amp;author=N.%20Gass&amp;author=C.%20Mundel&amp;publication_year=2000&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=264&amp;pages=257-267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380000312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380000312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380000312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11085265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baltz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Domon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pillay</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Characterization of a pollen-specific cDNA from sunflower encoding a zinc finger protein</div>, <div class="source">Plant J.</div> , <div class="year">1992</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">713</div>-<div class="lpage">721</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20pollen-specific%20cDNA%20from%20sunflower%20encoding%20a%20zinc%20finger%20protein&amp;author=R.%20Baltz&amp;author=C.%20Domon&amp;author=D.%20Pillay&amp;author=A.%20Steinmetz&amp;publication_year=1992&amp;journal=Plant%20J.&amp;volume=2&amp;pages=713-721" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1302629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baltz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Evrard</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Domon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A LIM motif is present in a pollen-specific protein</div>, <div class="source">Plant Cell</div> , <div class="year">1992</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1465</div>-<div class="lpage">1466</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20LIM%20motif%20is%20present%20in%20a%20pollen-specific%20protein&amp;author=R.%20Baltz&amp;author=J.%20L.%20Evrard&amp;author=C.%20Domon&amp;author=A.%20Steinmetz&amp;publication_year=1992&amp;journal=Plant%20Cell&amp;volume=4&amp;pages=1465-1466" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.4.12.1465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.4.12.1465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.4.12.1465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1467648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mundel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Baltz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Eliasson</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A LIM-domain protein from sunflower is localized to the cytoplasm and/or nucleus in a wide variety of tissues and is associated with the phragmoplast in dividing cells</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">291</div>-<div class="lpage">302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20LIM-domain%20protein%20from%20sunflower%20is%20localized%20to%20the%20cytoplasm%20and%2For%20nucleus%20in%20a%20wide%20variety%20of%20tissues%20and%20is%20associated%20with%20the%20phragmoplast%20in%20dividing%20cells&amp;author=C.%20Mundel&amp;author=R.%20Baltz&amp;author=A.%20Eliasson&amp;publication_year=2000&amp;journal=Plant%20Mol.%20Biol.&amp;volume=42&amp;pages=291-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1006333611189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1006333611189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1006333611189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10794529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Briere</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bordel</div> <div class="given-names">A.-C.</div></div>,  <div class="name"><div class="surname">Barthou</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Is the LIM-domain protein HaWLIM1 associated with cortical microtubules in sunflower protoplasts?</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">1055</div>-<div class="lpage">1063</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20the%20LIM-domain%20protein%20HaWLIM1%20associated%20with%20cortical%20microtubules%20in%20sunflower%20protoplasts%3F&amp;author=C.%20Briere&amp;author=A.-C.%20Bordel&amp;author=H.%20Barthou&amp;publication_year=2003&amp;journal=Plant%20Cell%20Physiol.&amp;volume=44&amp;pages=1055-1063" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcg126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcg126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcg126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14581630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dieterle</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Van Troys</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ampe</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Tobacco WLIM1 is a novel F-Actin binding protein involved in actin cytoskeleton remodeling</div>, <div class="source">Plant Cell</div> , <div class="year">2006</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">2194</div>-<div class="lpage">2206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tobacco%20WLIM1%20is%20a%20novel%20F-Actin%20binding%20protein%20involved%20in%20actin%20cytoskeleton%20remodeling&amp;author=C.%20Thomas&amp;author=C.%20Hoffmann&amp;author=M.%20Dieterle&amp;author=M.%20Van%20Troys&amp;author=C.%20Ampe&amp;author=A.%20Steinmetz&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=2194-2206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.040956" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.040956" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.040956"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16905656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baltz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schmit</div> <div class="given-names">A.-C.</div></div>,  <div class="name"><div class="surname">Kohnen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hentges</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Differential localization of the LIM domain protein PLIM-1 in microspores and mature pollen grains from sunflower</div>, <div class="source">Sex. Plant Reprod.</div> , <div class="year">1999</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">60</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20localization%20of%20the%20LIM%20domain%20protein%20PLIM-1%20in%20microspores%20and%20mature%20pollen%20grains%20from%20sunflower&amp;author=R.%20Baltz&amp;author=A.-C.%20Schmit&amp;author=M.%20Kohnen&amp;author=F.%20Hentges&amp;author=A.%20Steinmetz&amp;publication_year=1999&amp;journal=Sex.%20Plant%20Reprod.&amp;volume=12&amp;pages=60-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004970050172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004970050172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004970050172"> </span></div> </div></div></div></div></div><div content-id="DSM013C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baltz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Evrard</div> <div class="given-names">J.-L.</div></div>,  <div class="name"><div class="surname">Bourdon</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The pollen-specific LIM protein PLIM-1 from sunflower binds nucleic acids in vitro</div>, <div class="source">Sex. Plant Reprod.</div> , <div class="year">1996</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">264</div>-<div class="lpage">268</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pollen-specific%20LIM%20protein%20PLIM-1%20from%20sunflower%20binds%20nucleic%20acids%20in%20vitro&amp;author=R.%20Baltz&amp;author=J.-L.%20Evrard&amp;author=V.%20Bourdon&amp;author=A.%20Steinmetz&amp;publication_year=1996&amp;journal=Sex.%20Plant%20Reprod.&amp;volume=9&amp;pages=264-268" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02152700" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02152700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02152700"> </span></div> </div></div></div></div></div><div content-id="DSM013C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaoka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kaothien</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Endo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ebinuma</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Functional analysis of tobacco LIM protein Ntlim1 involved in lignin biosynthesis</div>, <div class="source">Plant J.</div> , <div class="year">2000</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">289</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20analysis%20of%20tobacco%20LIM%20protein%20Ntlim1%20involved%20in%20lignin%20biosynthesis&amp;author=A.%20Kawaoka&amp;author=P.%20Kaothien&amp;author=K.%20Yoshida&amp;author=S.%20Endo&amp;author=K.%20Yamada&amp;author=H.%20Ebinuma&amp;publication_year=2000&amp;journal=Plant%20J.&amp;volume=22&amp;pages=289-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.2000.00737.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.2000.00737.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.2000.00737.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10849346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Déjardin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Leplé</div> <div class="given-names">J.-C.</div></div>,  <div class="name"><div class="surname">Lesage-Descauses</div> <div class="given-names">M.-C.</div></div>,  <div class="name"><div class="surname">Costa</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pilate</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Expressed sequence tags from poplar wood tissues—a comparative analysis from multiple libraries</div>, <div class="source">Plant Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">55</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20sequence%20tags%20from%20poplar%20wood%20tissues%E2%80%94a%20comparative%20analysis%20from%20multiple%20libraries&amp;author=A.%20D%C3%A9jardin&amp;author=J.-C.%20Lepl%C3%A9&amp;author=M.-C.%20Lesage-Descauses&amp;author=G.%20Costa&amp;author=G.%20Pilate&amp;publication_year=2004&amp;journal=Plant%20Biol.&amp;volume=6&amp;pages=55-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1055/s-2003-44744" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1055%2Fs-2003-44744" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1055%2Fs-2003-44744"> </span></div> </div></div></div></div></div><div content-id="DSM013C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson-Gunneras</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mellerowicz</div> <div class="given-names">E. J.</div></div>,  <div class="name"><div class="surname">Love</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Biosynthesis of cellulose-enriched tension wood in <em>Populus</em>: global analysis of transcripts and metabolites identifies biochemical and developmental regulators in secondary wall biosynthesis</div>, <div class="source">Plant J.</div> , <div class="year">2006</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">144</div>-<div class="lpage">165</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20cellulose-enriched%20tension%20wood%20in%20Populus%3A%20global%20analysis%20of%20transcripts%20and%20metabolites%20identifies%20biochemical%20and%20developmental%20regulators%20in%20secondary%20wall%20biosynthesis&amp;author=S.%20Andersson-Gunneras&amp;author=E.%20J.%20Mellerowicz&amp;author=J.%20Love&amp;publication_year=2006&amp;journal=Plant%20J.&amp;volume=45&amp;pages=144-165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2006.45.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2006.45.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2006.45.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16367961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G. A.</div></div>,  <div class="name"><div class="surname">DiFazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">1604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.%20A.%20Tuskan&amp;author=S.%20DiFazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-1604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafarguette</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Leplé</div> <div class="given-names">J.-C.</div></div>,  <div class="name"><div class="surname">Déjardin</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Poplar genes encoding fasciclin-like arabinogalactan proteins are highly expressed in tension wood</div>, <div class="source">New Phytol.</div> , <div class="year">2004</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">107</div>-<div class="lpage">121</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poplar%20genes%20encoding%20fasciclin-like%20arabinogalactan%20proteins%20are%20highly%20expressed%20in%20tension%20wood&amp;author=F.%20Lafarguette&amp;author=J.-C.%20Lepl%C3%A9&amp;author=A.%20D%C3%A9jardin&amp;publication_year=2004&amp;journal=New%20Phytol.&amp;volume=164&amp;pages=107-121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2004.01175.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2004.01175.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2004.01175.x"> </span></div> </div></div></div></div></div><div content-id="DSM013C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sterky</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bhalerao</div> <div class="given-names">R. R.</div></div>,  <div class="name"><div class="surname">Unneberg</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A <em>Populus</em> EST resource for plant functional genomics</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">13951</div>-<div class="lpage">13956</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Populus%20EST%20resource%20for%20plant%20functional%20genomics&amp;author=F.%20Sterky&amp;author=R.%20R.%20Bhalerao&amp;author=P.%20Unneberg&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=13951-13956" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0401641101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0401641101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0401641101"> </span></div> </div></div></div></div></div><div content-id="DSM013C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chenna</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Multiple sequence alignment with the Clustal series of programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3497</div>-<div class="lpage">3500</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sequence%20alignment%20with%20the%20Clustal%20series%20of%20programs&amp;author=R.%20Chenna&amp;author=H.%20Sugawara&amp;author=T.%20Koike&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3497-3500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">D. T.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">W. R.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">The rapid generation of mutation data matrices from protein sequences</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1992</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">275</div>-<div class="lpage">282</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=D.%20T.%20Jones&amp;author=W.%20R.%20Taylor&amp;author=J.%20M.%20Thornton&amp;publication_year=1992&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=8&amp;pages=275-282" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1633570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-425" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>. </div><div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>, <div class="source">Syst. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">696</div>-<div class="lpage">704</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2003&amp;journal=Syst.%20Biol.&amp;volume=52&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>. </div><div class="article-title">BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">685</div>-<div class="lpage">695</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;author=O.%20Gascuel&amp;publication_year=1997&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=14&amp;pages=685-695" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Page</div> <div class="given-names">R. D. M.</div></div>. </div><div class="article-title">Tree View: an application to display phylogenetic trees on personal computers</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">357</div>-<div class="lpage">358</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tree%20View%3A%20an%20application%20to%20display%20phylogenetic%20trees%20on%20personal%20computers&amp;author=R.%20D.%20M.%20Page&amp;publication_year=1996&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=12&amp;pages=357-358" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8902363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Elkan</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers</div>, <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">28</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fitting%20a%20mixture%20model%20by%20expectation%20maximization%20to%20discover%20motifs%20in%20biopolymers&amp;author=T.%20L.%20Bailey&amp;author=C.%20Elkan&amp;publication_year=1994&amp;journal=Proc.%20Int.%20Conf.%20Intell.%20Syst.%20Mol.%20Biol.&amp;volume=2&amp;pages=28-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ralph</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oddy</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Genomics of hybrid poplar (<em>Populus trichocarpa</em> × <em>deltoides</em>) interacting with forest tent caterpillars (<em>Malacosoma disstria</em>): normalized and full-length cDNA libraries, expressed sequence tags, and a cDNA microarray for the study of insect-induced defences in poplar</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1275</div>-<div class="lpage">1297</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20hybrid%20poplar%20(Populus%20trichocarpa%20%C3%97%20deltoides)%20interacting%20with%20forest%20tent%20caterpillars%20(Malacosoma%20disstria)%3A%20normalized%20and%20full-length%20cDNA%20libraries%2C%20expressed%20sequence%20tags%2C%20and%20a%20cDNA%20microarray%20for%20the%20study%20of%20insect-induced%20defences%20in%20poplar&amp;author=S.%20Ralph&amp;author=C.%20Oddy&amp;author=D.%20Cooper&amp;publication_year=2006&amp;journal=Mol.%20Ecol.&amp;volume=15&amp;pages=1275-1297" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2006.02824.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2006.02824.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2006.02824.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16626454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Djerbi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lindskog</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Arvestad</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sterky</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Teeri</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The genome sequence of black cottonwood (<em>Populus trichocarpa</em>) reveals 18 conserved cellulose synthase (<em>CesA</em>) genes</div>, <div class="source">Planta</div> , <div class="year">2005</div>, vol. <div class="volume">221</div> (pg. <div class="fpage">739</div>-<div class="lpage">746</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20black%20cottonwood%20(Populus%20trichocarpa)%20reveals%2018%20conserved%20cellulose%20synthase%20(CesA)%20genes&amp;author=S.%20Djerbi&amp;author=M.%20Lindskog&amp;author=L.%20Arvestad&amp;author=F.%20Sterky&amp;author=T.%20Teeri&amp;publication_year=2005&amp;journal=Planta&amp;volume=221&amp;pages=739-746" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-005-1498-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-005-1498-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-005-1498-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15940463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sterky</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Rouze</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">EST data suggest that poplar is an ancient polyploid</div>, <div class="source">New Phytol.</div> , <div class="year">2005</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">165</div>-<div class="lpage">170</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%20data%20suggest%20that%20poplar%20is%20an%20ancient%20polyploid&amp;author=L.%20Sterck&amp;author=S.%20Rombauts&amp;author=S.%20Jansson&amp;author=F.%20Sterky&amp;author=P.%20Rouze&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2005&amp;journal=New%20Phytol.&amp;volume=167&amp;pages=165-170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01378.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2005.01378.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2005.01378.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15948839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tranbarger</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">Al-Ghazi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Muller</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Teyssendier de la Serve</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Doumas</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Touraine</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Transcription factor genes with expression correlated to nitrate-related root plasticity of <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell Environ.</div> , <div class="year">2003</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">459</div>-<div class="lpage">469</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factor%20genes%20with%20expression%20correlated%20to%20nitrate-related%20root%20plasticity%20of%20Arabidopsis%20thaliana&amp;author=T.%20J.%20Tranbarger&amp;author=Y.%20Al-Ghazi&amp;author=B.%20Muller&amp;author=B.%20Teyssendier%20de%20la%20Serve&amp;author=P.%20Doumas&amp;author=B.%20Touraine&amp;publication_year=2003&amp;journal=Plant%20Cell%20Environ.&amp;volume=26&amp;pages=459-469" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-3040.2003.00977.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-3040.2003.00977.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-3040.2003.00977.x"> </span></div> </div></div></div></div></div><div content-id="DSM013C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">376</div>-<div class="lpage">379</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%2C%20mapping%2C%20and%20annotation%20of%20over%2028%2C000%20cDNA%20clones%20from%20japonica%20rice&amp;author=S.%20Kikuchi&amp;author=K.%20Satoh&amp;author=T.%20Nagata&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=376-379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1081288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1081288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1081288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sweetman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Spurr</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Eliasson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gass</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Twell</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Isolation and characterisation of two pollen-specific LIM domain protein cDNAs from Nicotiana tabacum</div>, <div class="source">Sex. Plant Reprod.</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">339</div>-<div class="lpage">345</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterisation%20of%20two%20pollen-specific%20LIM%20domain%20protein%20cDNAs%20from%20Nicotiana%20tabacum&amp;author=J.%20Sweetman&amp;author=C.%20Spurr&amp;author=A.%20Eliasson&amp;author=N.%20Gass&amp;author=A.%20Steinmetz&amp;author=D.%20Twell&amp;publication_year=2000&amp;journal=Sex.%20Plant%20Reprod.&amp;volume=12&amp;pages=339-345" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00009836" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00009836" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00009836"> </span></div> </div></div></div></div></div><div content-id="DSM013C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laitinen</div> <div class="given-names">R. A. E.</div></div>,  <div class="name"><div class="surname">Immanen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Auvinen</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Analysis of the floral transcriptome uncovers new regulators of organ determination and gene families related to flower organ differentiation in <em>Gerbera hybrida</em> (Asteraceae)</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">475</div>-<div class="lpage">486</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20floral%20transcriptome%20uncovers%20new%20regulators%20of%20organ%20determination%20and%20gene%20families%20related%20to%20flower%20organ%20differentiation%20in%20Gerbera%20hybrida%20(Asteraceae)&amp;author=R.%20A.%20E.%20Laitinen&amp;author=J.%20Immanen&amp;author=P.%20Auvinen&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=475-486" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3043705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3043705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3043705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15781570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guyon</div> <div class="given-names">V. N.</div></div>,  <div class="name"><div class="surname">Astwood</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Garner</div> <div class="given-names">E. C.</div></div>,  <div class="name"><div class="surname">Dunker</div> <div class="given-names">A. K.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">L. P.</div></div>. </div><div class="article-title">Isolation and characterization of cDNAs expressed in the early stages of flavonol-induced pollen germination in Petunia</div>, <div class="source">Plant Physiol.</div> , <div class="year">2000</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">699</div>-<div class="lpage">710</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20cDNAs%20expressed%20in%20the%20early%20stages%20of%20flavonol-induced%20pollen%20germination%20in%20Petunia&amp;author=V.%20N.%20Guyon&amp;author=J.%20D.%20Astwood&amp;author=E.%20C.%20Garner&amp;author=A.%20K.%20Dunker&amp;author=L.%20P.%20Taylor&amp;publication_year=2000&amp;journal=Plant%20Physiol.&amp;volume=123&amp;pages=699-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.123.2.699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.123.2.699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.123.2.699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10859200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allona</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Quinn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shoop</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Analysis of xylem formation in pine by cDNA sequencing</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">9693</div>-<div class="lpage">9698</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20xylem%20formation%20in%20pine%20by%20cDNA%20sequencing&amp;author=I.%20Allona&amp;author=M.%20Quinn&amp;author=E.%20Shoop&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=9693-9698" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9693" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9693" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9693"> </span></div> </div></div></div></div></div><div content-id="DSM013C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dean Rider</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Henderson</div> <div class="given-names">J. T.</div></div>,  <div class="name"><div class="surname">Jerome</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Edenberg</div> <div class="given-names">H. J.</div></div>,  <div class="name"><div class="surname">Romero-Severson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ogas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Coordinate repression of regulators of embryonic identity by <em>PICKLE</em> during germination in <em>Arabidopsis</em></div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">33</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinate%20repression%20of%20regulators%20of%20embryonic%20identity%20by%20PICKLE%20during%20germination%20in%20Arabidopsis&amp;author=S.%20Dean%20Rider&amp;author=J.%20T.%20Henderson&amp;author=R.%20E.%20Jerome&amp;author=H.%20J.%20Edenberg&amp;author=J.%20Romero-Severson&amp;author=J.%20Ogas&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=35&amp;pages=33-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01783.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01783.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01783.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12834400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gepstein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sabehi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Carp</div> <div class="given-names">M.-J.</div></div>, et al. </div><div class="article-title">Large-scale identification of leaf senescence-associated genes</div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">629</div>-<div class="lpage">642</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20identification%20of%20leaf%20senescence-associated%20genes&amp;author=S.%20Gepstein&amp;author=G.%20Sabehi&amp;author=M.-J.%20Carp&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=36&amp;pages=629-642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01908.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01908.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01908.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14617064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Boavida</div> <div class="given-names">L. C.</div></div>,  <div class="name"><div class="surname">Carneiro</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Haury</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Feijo</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Transcriptional profiling of <em>Arabidopsis</em> tissues reveals the unique characteristics of the pollen transcriptome</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">713</div>-<div class="lpage">725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20of%20Arabidopsis%20tissues%20reveals%20the%20unique%20characteristics%20of%20the%20pollen%20transcriptome&amp;author=J.%20D.%20Becker&amp;author=L.%20C.%20Boavida&amp;author=J.%20Carneiro&amp;author=M.%20Haury&amp;author=J.%20A.%20Feijo&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=133&amp;pages=713-725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.028241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.028241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.028241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM013C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM013C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Endo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakazono</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">cDNA microarray analysis of gene expression changes during pollination, pollen-tube elongation, fertilization, and early embryogenesis in rice pistils</div>, <div class="source">Sex. Plant Reprod.</div> , <div class="year">2005</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">269</div>-<div class="lpage">275</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cDNA%20microarray%20analysis%20of%20gene%20expression%20changes%20during%20pollination%2C%20pollen-tube%20elongation%2C%20fertilization%2C%20and%20early%20embryogenesis%20in%20rice%20pistils&amp;author=K.%20Yoshida&amp;author=M.%20Endo&amp;author=M.%20Nakazono&amp;publication_year=2005&amp;journal=Sex.%20Plant%20Reprod.&amp;volume=17&amp;pages=269-275" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00497-004-0238-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00497-004-0238-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00497-004-0238-4"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=577859&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm013/1/dsm013_Supplementary_Data.zip?Expires=2147483647&amp;Signature=VTAN~1bzY89OA~ySLppsYh-8qBrNTPV7WMM-b-tJPSRDgtYZZBcjndUKFpdCyd5npJAPZ6iKYVFFkqgcYeHX1Nb49tF~EXsyKEGitlRKVZGzJeTeRXVQS0Pd2nNfnYpvARmAL88LHhGgCnTodM0cc5fQ0omY9O8Ci3CQxaThTPGTvVb7Y4SSQ1Bo1DFXM0QzelYpns1gV-21Zvp2V9BOFqT615z5LePuhjD~b1Qx0Ho3leV3oquTR9NGkHp~t0bg~oBHVn0t74GJgrQFLAEkbqFsmM4Wjz80CDo0YC4OQJSlsEhVfoLh2DKDuN7DDdMIc0qUlH0sKfaonczhFuVU0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">177</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">45</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="577859"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f3%2f103%2f577859">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248634100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (45)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F3%2F103%2F577859" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/577859'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Analysis of LIM Gene Family in Populus trichocarpa, Arabidopsis thaliana, and Oryza sativa" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f103%2f577859">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

