

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/4/259/348246","@type":"ScholarlyArticle","name":"Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds","datePublished":"2015-06-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2015-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv011","keywords":["whole-genome sequencing","Pig","SNP","signature of selection"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/4/259/348246","author":[{"name":"Choi, Jung-Woo","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Chung, Won-Hyong","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea    2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Kyung-Tai","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Cho, Eun-Seok","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Si-Woo","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Bong-Hwan","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Sang-Heon","affiliation":"2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  Korea University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Lim, Wonjun","affiliation":"2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  Korea University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Lim, Dajeong","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Yun-Gyeong","affiliation":"2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  Korea University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Hong, Joon-Ki","affiliation":"4  Swine Division ,  National Institute of Animal Science, Rural Development Administration ,  Suwon 441-706 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Doo-Wan","affiliation":"4  Swine Division ,  National Institute of Animal Science, Rural Development Administration ,  Suwon 441-706 ,  Republic of Korea","@type":"Person"},{"name":"Jeon, Hyeon-Jeong","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Jiwoong","affiliation":"2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Namshin","affiliation":"2  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  Korea University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Tae-Hun","affiliation":"1  Animal Genomics and Bioinformatics Division ,  National Institute of Animal Science, Rural Development Administration ,  Jeonju 565-851 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Pigs have been one of the most important sources of meat for humans, and their productivity has been substantially improved by recent strong selectio","pageStart":"259","pageEnd":"267","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds" />
    <meta property="og:description" content="Abstract.  Pigs have been one of the most important sources of meat for humans, and their productivity has been substantially improved by recent strong selectio" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/4/259/348246" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-06-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Choi, Jung-Woo" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Chung, Won-Hyong" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Lee, Kyung-Tai" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Cho, Eun-Seok" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Lee, Si-Woo" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Choi, Bong-Hwan" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Lee, Sang-Heon" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, Korea University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Lim, Wonjun" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, Korea University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Lim, Dajeong" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Lee, Yun-Gyeong" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, Korea University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Hong, Joon-Ki" /><meta name="citation_author_institution" content="Swine Division, National Institute of Animal Science, Rural Development Administration, Suwon 441-706, Republic of Korea" /><meta name="citation_author" content="Kim, Doo-Wan" /><meta name="citation_author_institution" content="Swine Division, National Institute of Animal Science, Rural Development Administration, Suwon 441-706, Republic of Korea" /><meta name="citation_author" content="Jeon, Hyeon-Jeong" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_author" content="Kim, Jiwoong" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Kim, Namshin" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, Korea University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Kim, Tae-Hun" /><meta name="citation_author_institution" content="Animal Genomics and Bioinformatics Division, National Institute of Animal Science, Rural Development Administration, Jeonju 565-851, Republic of Korea" /><meta name="citation_title" content="Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds" /><meta name="citation_firstpage" content="259" /><meta name="citation_lastpage" content="267" /><meta name="citation_doi" content="10.1093/dnares/dsv011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2015/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Analyses of pig genomes provide insight into porcine demography and evolution;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=393-8; " /><meta name="citation_reference" content="citation_title=The origin of the domestic pig: independent domestication and subsequent introgression;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1785-91; " /><meta name="citation_reference" content="United States Department of Agriculture (USDA). 2014 , The Hogs and Pigs report. http://www.usda.gov/ ( 9 September 2014, date last accessed ). " /><meta name="citation_reference" content="Canadian Centre for Swine Improvement (CCSI). 2012 , Canadian Centre for Swine Improvement Annual Report. https://www.ccsi.ca/ ( 5 September 2014, date last accessed ). " /><meta name="citation_reference" content="citation_title=Genetic manipulation in pigs;  citation_journal_title=Curr. Opin. Organ Transplant.;  citation_year=2009;  citation_volume=14;  citation_pages=148-53; " /><meta name="citation_reference" content="citation_title=The miniature pig as an animal model in biomedical research;  citation_journal_title=Ann. NY Acad. Sci.;  citation_year=2005;  citation_volume=10,49;  citation_pages=161-71; " /><meta name="citation_reference" content="Food and Agriculture Organization (FAO). 2014 , Domestic Animal Diversity Information Service (DAD-IS). http://dad.fao.org/ ( 5 September 2014, date last accessed ). " /><meta name="citation_reference" content="citation_title=Comparison of carcass characteristics, meat quality, amino acids contents, and fatty acid profiles of Korea native pig by gender;  citation_journal_title=Rep. Dev. Biol.;  citation_year=2013;  citation_volume=37;  citation_pages=129-34; " /><meta name="citation_reference" content="citation_title=Carcass yields and meat quality by live weight of Korean native black pigs;  citation_journal_title=J. Anim. Sci. Technol.;  citation_year=2007;  citation_volume=49;  citation_pages=523-30; " /><meta name="citation_reference" content="citation_title=National Management System for conservation of livestock genetic resources: an overview;  citation_journal_title=Ann. Anim. Resour. Sci.;  citation_year=2012;  citation_volume=23;  citation_pages=142-8; " /><meta name="citation_reference" content="citation_title=Molecular genetic evaluation of Korean native pig populations based on microsatellite markers;  citation_journal_title=J. Anim. Sci. Technol.;  citation_year=2011;  citation_volume=53;  citation_pages=35-42; " /><meta name="citation_reference" content="citation_title=Genetic structure of pig breeds from Korea and China using microsatellite loci analysis;  citation_journal_title=J. Anim. Sci.;  citation_year=2005;  citation_volume=83;  citation_pages=2255-63; " /><meta name="citation_reference" content="citation_title=Strong signatures of selection in the domestic pig genome;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=19529-36; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of Berkshire (European native pig) provides insights into its origin and domestication;  citation_journal_title=Sci. Rep.;  citation_year=2014;  citation_volume=4;  citation_pages=4678" /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3;  citation_journal_title=Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=Using Drosophila melanogaster as a model for genotoxic chemical mutational studies with a new program, SnpSift;  citation_journal_title=Front. Genet.;  citation_year=2012;  citation_volume=3;  citation_pages=35" /><meta name="citation_reference" content="citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2009;  citation_volume=84;  citation_pages=210-23; " /><meta name="citation_reference" content="citation_title=APE: analyses of phylogenetics and evolution in R language;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=289-90; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Inferring weak population structure with the assistance of sample group information;  citation_journal_title=Mol. Ecol. Resour.;  citation_year=2009;  citation_volume=9;  citation_pages=1322-32; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies;  citation_journal_title=Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1567-87; " /><meta name="citation_reference" content="citation_title=DISTRUCT: a program for the graphical display of population structure;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2004;  citation_volume=4;  citation_pages=137-8; " /><meta name="citation_reference" content="citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=587-91; " /><meta name="citation_reference" content="citation_title=Whole-genome analyses of Korean native and Holstein cattle breeds by massively parallel sequencing;  citation_journal_title=PloS One;  citation_year=2014;  citation_volume=9;  citation_pages=e101127" /><meta name="citation_reference" content="citation_title=An integrated map of genetic variation from 1,092 human genomes; citation_author=Genomes Project Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=56-65; " /><meta name="citation_reference" content="citation_title=Massively parallel sequencing of Chikso (Korean brindle cattle) to discover genome-wide SNPs and InDels;  citation_journal_title=Mol. Cells;  citation_year=2013;  citation_volume=36;  citation_pages=203-11; " /><meta name="citation_reference" content="citation_title=SIFT: predicting amino acid changes that affect protein function;  citation_journal_title=Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3812-4; " /><meta name="citation_reference" content="citation_title=Targeted disruption of the mouse phospholipase C beta3 gene results in early embryonic lethality;  citation_journal_title=FEBS Lett.;  citation_year=1998;  citation_volume=441;  citation_pages=261-5; " /><meta name="citation_reference" content="citation_title=Phospholipase C in mouse oocytes: characterization of beta and gamma isoforms and their possible involvement in sperm-induced Ca2+ spiking;  citation_journal_title=Biochem. J.;  citation_year=1996;  citation_volume=316;  citation_issue=(Pt 2);  citation_pages=583-91; " /><meta name="citation_reference" content="citation_title=Essential role of the inositol 1,4,5-trisphosphate receptor/Ca2+ release channel in Ca2+ waves and Ca2+ oscillations at fertilization of mammalian eggs;  citation_journal_title=Dev. Biol.;  citation_year=1993;  citation_volume=158;  citation_pages=62-78; " /><meta name="citation_reference" content="citation_title=Genetic associations for gilt growth, compositional, and structural soundness traits with sow longevity and lifetime reproductive performance;  citation_journal_title=J. Anim. Sci.;  citation_year=2013;  citation_volume=91;  citation_pages=1570-9; " /><meta name="citation_reference" content="citation_title=Genetics of length of productive life and lifetime prolificacy in the Finnish Landrace and Large White pig populations;  citation_journal_title=J. Anim. Sci.;  citation_year=2004;  citation_volume=82;  citation_pages=3111-7; " /><meta name="citation_reference" content="citation_title=Neurexophilins form a conserved family of neuropeptide-like glycoproteins;  citation_journal_title=J. Neurosci.;  citation_year=1998;  citation_volume=18;  citation_pages=3630-8; " /><meta name="citation_reference" content="citation_title=Uterine gene expression in the live-bearing lizard, Chalcides ocellatus, reveals convergence of squamate reptile and mammalian pregnancy mechanisms;  citation_journal_title=Genome Biol. Evol.;  citation_year=2012;  citation_volume=4;  citation_pages=394-411; " /><meta name="citation_reference" content="citation_title=Embryonic signals direct the formation of tight junctional permeability barrier in the decidualizing stroma during embryo implantation;  citation_journal_title=J. Cell Sci.;  citation_year=2004;  citation_volume=117;  citation_pages=53-62; " /><meta name="citation_reference" content="citation_title=Endometrial gene expression profiling in pregnant Meishan and Yorkshire pigs on day 12 of gestation;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=156" /><meta name="citation_reference" content="citation_title=Saethre-Chotzen syndrome caused by TWIST 1 gene mutations: functional differentiation from Muenke coronal synostosis syndrome;  citation_journal_title=Eur. J. Hum. Genet.;  citation_year=2006;  citation_volume=14;  citation_pages=39-48; " /><meta name="citation_reference" content="citation_title=Expression of the transcription factors snail, slug, and twist and their clinical significance in human breast cancer;  citation_journal_title=Ann. Surg. Oncol.;  citation_year=2005;  citation_volume=12;  citation_pages=488-96; " /><meta name="citation_reference" content="citation_title=Twist 1 regulates the expression of PPARgamma during hormone-induced 3T3-L1 preadipocyte differentiation: a possible role in obesity and associated diseases;  citation_journal_title=Lipids Health Dis.;  citation_year=2014;  citation_volume=13;  citation_pages=132" /><meta name="citation_reference" content="citation_title=Twist1 in human white adipose tissue and obesity;  citation_journal_title=J. Clin. Endocrinol. Metab.;  citation_year=2011;  citation_volume=96;  citation_pages=133-41; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/4/259/997950/dsv011.pdf" /><meta name="description" content="Abstract.  Pigs have been one of the most important sources of meat for humans, and their productivity has been substantially improved by recent strong selectio" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/4/259/348246" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/4/259/348246" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/4/259/348246");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv011';
    </script>
<title>
	Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348246" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BayEATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgSnVuZy1Xb28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgR2Vub21pY3MgYW5kIEJpb2luZm9ybWF0aWNzIERpdmlzaW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIFJ1cmFsIERldmVsb3BtZW50IEFkbWluaXN0cmF0aW9uLCBKZW9uanUgNTY1LTg1MSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaHVuZywgV29uLUh5b25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbm9taWNzIGFuZCBCaW9pbmZvcm1hdGljcyBEaXZpc2lvbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFuaW1hbCBTY2llbmNlLCBSdXJhbCBEZXZlbG9wbWVudCBBZG1pbmlzdHJhdGlvbiwgSmVvbmp1IDU2NS04NTEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIEt5dW5nLVRhaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFuaW1hbCBHZW5vbWljcyBhbmQgQmlvaW5mb3JtYXRpY3MgRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIEplb25qdSA1NjUtODUxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNobywgRXVuLVNlb2siIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgR2Vub21pY3MgYW5kIEJpb2luZm9ybWF0aWNzIERpdmlzaW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIFJ1cmFsIERldmVsb3BtZW50IEFkbWluaXN0cmF0aW9uLCBKZW9uanUgNTY1LTg1MSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIFNpLVdvbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFuaW1hbCBHZW5vbWljcyBhbmQgQmlvaW5mb3JtYXRpY3MgRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIEplb25qdSA1NjUtODUxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob2ksIEJvbmctSHdhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFuaW1hbCBHZW5vbWljcyBhbmQgQmlvaW5mb3JtYXRpY3MgRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIEplb25qdSA1NjUtODUxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlZSwgU2FuZy1IZW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBLb3JlYSBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaW0sIFdvbmp1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDMwNS04MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcywgS29yZWEgVW5pdmVyc2l0eSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBEYWVqZW9uIDMwNS04MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGltLCBEYWplb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbm9taWNzIGFuZCBCaW9pbmZvcm1hdGljcyBEaXZpc2lvbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFuaW1hbCBTY2llbmNlLCBSdXJhbCBEZXZlbG9wbWVudCBBZG1pbmlzdHJhdGlvbiwgSmVvbmp1IDU2NS04NTEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVlLCBZdW4tR3llb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBLb3JlYSBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb25nLCBKb29uLUtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3dpbmUgRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIFN1d29uIDQ0MS03MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBEb28tV2FuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3dpbmUgRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIFN1d29uIDQ0MS03MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmVvbiwgSHllb24tSmVvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbmltYWwgR2Vub21pY3MgYW5kIEJpb2luZm9ybWF0aWNzIERpdmlzaW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIFJ1cmFsIERldmVsb3BtZW50IEFkbWluaXN0cmF0aW9uLCBKZW9uanUgNTY1LTg1MSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEppd29vbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLb3JlYW4gQmlvaW5mb3JtYXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgRGFlamVvbiAzMDUtODA2LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbSwgTmFtc2hpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDMwNS04MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcywgS29yZWEgVW5pdmVyc2l0eSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBEYWVqZW9uIDMwNS04MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBUYWUtSHVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbm9taWNzIGFuZCBCaW9pbmZvcm1hdGljcyBEaXZpc2lvbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFuaW1hbCBTY2llbmNlLCBSdXJhbCBEZXZlbG9wbWVudCBBZG1pbmlzdHJhdGlvbiwgSmVvbmp1IDU2NS04NTEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJXaG9sZS1nZW5vbWUgcmVzZXF1ZW5jaW5nIGFuYWx5c2VzIG9mIGZpdmUgcGlnIGJyZWVkcywgaW5jbHVkaW5nIEtvcmVhbiB3aWxkIGFuZCBuYXRpdmUsIGFuZCB0aHJlZSBFdXJvcGVhbiBvcmlnaW4gYnJlZWRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3YwMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2VzIG9mIHBpZyBnZW5vbWVzIHByb3ZpZGUgaW5zaWdodCBpbnRvIHBvcmNpbmUgZGVtb2dyYXBoeSBhbmQgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9MzkzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBvcmlnaW4gb2YgdGhlIGRvbWVzdGljIHBpZzogaW5kZXBlbmRlbnQgZG9tZXN0aWNhdGlvbiBhbmQgc3Vic2VxdWVudCBpbnRyb2dyZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzg1LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJVbml0ZWQgU3RhdGVzIERlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmUgKFVTREEpLiAyMDE0ICwgVGhlIEhvZ3MgYW5kIFBpZ3MgcmVwb3J0LiBodHRwOi8vd3d3LnVzZGEuZ292LyAoIDkgU2VwdGVtYmVyIDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCApLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJDYW5hZGlhbiBDZW50cmUgZm9yIFN3aW5lIEltcHJvdmVtZW50IChDQ1NJKS4gMjAxMiAsIENhbmFkaWFuIENlbnRyZSBmb3IgU3dpbmUgSW1wcm92ZW1lbnQgQW5udWFsIFJlcG9ydC4gaHR0cHM6Ly93d3cuY2NzaS5jYS8gKCA1IFNlcHRlbWJlciAyMDE0LCBkYXRlIGxhc3QgYWNjZXNzZWQgKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYW5pcHVsYXRpb24gaW4gcGlnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gT3JnYW4gVHJhbnNwbGFudC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtaW5pYXR1cmUgcGlnIGFzIGFuIGFuaW1hbCBtb2RlbCBpbiBiaW9tZWRpY2FsIHJlc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIE5ZIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAsNDk7ICBjaXRhdGlvbl9wYWdlcz0xNjEtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkZvb2QgYW5kIEFncmljdWx0dXJlIE9yZ2FuaXphdGlvbiAoRkFPKS4gMjAxNCAsIERvbWVzdGljIEFuaW1hbCBEaXZlcnNpdHkgSW5mb3JtYXRpb24gU2VydmljZSAoREFELUlTKS4gaHR0cDovL2RhZC5mYW8ub3JnLyAoIDUgU2VwdGVtYmVyIDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCApLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGNhcmNhc3MgY2hhcmFjdGVyaXN0aWNzLCBtZWF0IHF1YWxpdHksIGFtaW5vIGFjaWRzIGNvbnRlbnRzLCBhbmQgZmF0dHkgYWNpZCBwcm9maWxlcyBvZiBLb3JlYSBuYXRpdmUgcGlnIGJ5IGdlbmRlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmVwLiBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXJjYXNzIHlpZWxkcyBhbmQgbWVhdCBxdWFsaXR5IGJ5IGxpdmUgd2VpZ2h0IG9mIEtvcmVhbiBuYXRpdmUgYmxhY2sgcGlnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW5pbS4gU2NpLiBUZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTUyMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmF0aW9uYWwgTWFuYWdlbWVudCBTeXN0ZW0gZm9yIGNvbnNlcnZhdGlvbiBvZiBsaXZlc3RvY2sgZ2VuZXRpYyByZXNvdXJjZXM6IGFuIG92ZXJ2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEFuaW0uIFJlc291ci4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE0Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZ2VuZXRpYyBldmFsdWF0aW9uIG9mIEtvcmVhbiBuYXRpdmUgcGlnIHBvcHVsYXRpb25zIGJhc2VkIG9uIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFuaW0uIFNjaS4gVGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0zNS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzdHJ1Y3R1cmUgb2YgcGlnIGJyZWVkcyBmcm9tIEtvcmVhIGFuZCBDaGluYSB1c2luZyBtaWNyb3NhdGVsbGl0ZSBsb2NpIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbmltLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MjI1NS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Ryb25nIHNpZ25hdHVyZXMgb2Ygc2VsZWN0aW9uIGluIHRoZSBkb21lc3RpYyBwaWcgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE5NTI5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2luZyBvZiBCZXJrc2hpcmUgKEV1cm9wZWFuIG5hdGl2ZSBwaWcpIHByb3ZpZGVzIGluc2lnaHRzIGludG8gaXRzIG9yaWdpbiBhbmQgZG9tZXN0aWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00Njc4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgbG9uZy1yZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2Vub21lIEFuYWx5c2lzIFRvb2xraXQ6IGEgTWFwUmVkdWNlIGZyYW1ld29yayBmb3IgYW5hbHl6aW5nIG5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb2dyYW0gZm9yIGFubm90YXRpbmcgYW5kIHByZWRpY3RpbmcgdGhlIGVmZmVjdHMgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcywgU25wRWZmOiBTTlBzIGluIHRoZSBnZW5vbWUgb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgc3RyYWluIHcxMTE4OyBpc28tMjsgaXNvLTM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZseTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODAtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGFzIGEgbW9kZWwgZm9yIGdlbm90b3hpYyBjaGVtaWNhbCBtdXRhdGlvbmFsIHN0dWRpZXMgd2l0aCBhIG5ldyBwcm9ncmFtLCBTbnBTaWZ0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdW5pZmllZCBhcHByb2FjaCB0byBnZW5vdHlwZSBpbXB1dGF0aW9uIGFuZCBoYXBsb3R5cGUtcGhhc2UgaW5mZXJlbmNlIGZvciBsYXJnZSBkYXRhIHNldHMgb2YgdHJpb3MgYW5kIHVucmVsYXRlZCBpbmRpdmlkdWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MjEwLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUEU6IGFuYWx5c2VzIG9mIHBoeWxvZ2VuZXRpY3MgYW5kIGV2b2x1dGlvbiBpbiBSIGxhbmd1YWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcnJpbmcgd2VhayBwb3B1bGF0aW9uIHN0cnVjdHVyZSB3aXRoIHRoZSBhc3Npc3RhbmNlIG9mIHNhbXBsZSBncm91cCBpbmZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLiBSZXNvdXIuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzIyLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOiBsaW5rZWQgbG9jaSBhbmQgY29ycmVsYXRlZCBhbGxlbGUgZnJlcXVlbmNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURJU1RSVUNUOiBhIHByb2dyYW0gZm9yIHRoZSBncmFwaGljYWwgZGlzcGxheSBvZiBwb3B1bGF0aW9uIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTM3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmcgcmV2ZWFscyBsb2NpIHVuZGVyIHNlbGVjdGlvbiBkdXJpbmcgY2hpY2tlbiBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY0OyAgY2l0YXRpb25fcGFnZXM9NTg3LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgYW5hbHlzZXMgb2YgS29yZWFuIG5hdGl2ZSBhbmQgSG9sc3RlaW4gY2F0dGxlIGJyZWVkcyBieSBtYXNzaXZlbHkgcGFyYWxsZWwgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDExMjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIG1hcCBvZiBnZW5ldGljIHZhcmlhdGlvbiBmcm9tIDEsMDkyIGh1bWFuIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1HZW5vbWVzIFByb2plY3QgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTU2LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlbHkgcGFyYWxsZWwgc2VxdWVuY2luZyBvZiBDaGlrc28gKEtvcmVhbiBicmluZGxlIGNhdHRsZSkgdG8gZGlzY292ZXIgZ2Vub21lLXdpZGUgU05QcyBhbmQgSW5EZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGxzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MjAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSUZUOiBwcmVkaWN0aW5nIGFtaW5vIGFjaWQgY2hhbmdlcyB0aGF0IGFmZmVjdCBwcm90ZWluIGZ1bmN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zODEyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIGRpc3J1cHRpb24gb2YgdGhlIG1vdXNlIHBob3NwaG9saXBhc2UgQyBiZXRhMyBnZW5lIHJlc3VsdHMgaW4gZWFybHkgZW1icnlvbmljIGxldGhhbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NDE7ICBjaXRhdGlvbl9wYWdlcz0yNjEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3Bob2xpcGFzZSBDIGluIG1vdXNlIG9vY3l0ZXM6IGNoYXJhY3Rlcml6YXRpb24gb2YgYmV0YSBhbmQgZ2FtbWEgaXNvZm9ybXMgYW5kIHRoZWlyIHBvc3NpYmxlIGludm9sdmVtZW50IGluIHNwZXJtLWluZHVjZWQgQ2EyKyBzcGlraW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTY7ICBjaXRhdGlvbl9pc3N1ZT0oUHQgMik7ICBjaXRhdGlvbl9wYWdlcz01ODMtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzc2VudGlhbCByb2xlIG9mIHRoZSBpbm9zaXRvbCAxLDQsNS10cmlzcGhvc3BoYXRlIHJlY2VwdG9yL0NhMisgcmVsZWFzZSBjaGFubmVsIGluIENhMisgd2F2ZXMgYW5kIENhMisgb3NjaWxsYXRpb25zIGF0IGZlcnRpbGl6YXRpb24gb2YgbWFtbWFsaWFuIGVnZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldi4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTU4OyAgY2l0YXRpb25fcGFnZXM9NjItNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYXNzb2NpYXRpb25zIGZvciBnaWx0IGdyb3d0aCwgY29tcG9zaXRpb25hbCwgYW5kIHN0cnVjdHVyYWwgc291bmRuZXNzIHRyYWl0cyB3aXRoIHNvdyBsb25nZXZpdHkgYW5kIGxpZmV0aW1lIHJlcHJvZHVjdGl2ZSBwZXJmb3JtYW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW5pbS4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTE1NzAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2YgbGVuZ3RoIG9mIHByb2R1Y3RpdmUgbGlmZSBhbmQgbGlmZXRpbWUgcHJvbGlmaWNhY3kgaW4gdGhlIEZpbm5pc2ggTGFuZHJhY2UgYW5kIExhcmdlIFdoaXRlIHBpZyBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW5pbS4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTMxMTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV1cmV4b3BoaWxpbnMgZm9ybSBhIGNvbnNlcnZlZCBmYW1pbHkgb2YgbmV1cm9wZXB0aWRlLWxpa2UgZ2x5Y29wcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTmV1cm9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzYzMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VdGVyaW5lIGdlbmUgZXhwcmVzc2lvbiBpbiB0aGUgbGl2ZS1iZWFyaW5nIGxpemFyZCwgQ2hhbGNpZGVzIG9jZWxsYXR1cywgcmV2ZWFscyBjb252ZXJnZW5jZSBvZiBzcXVhbWF0ZSByZXB0aWxlIGFuZCBtYW1tYWxpYW4gcHJlZ25hbmN5IG1lY2hhbmlzbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9Mzk0LTQxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1icnlvbmljIHNpZ25hbHMgZGlyZWN0IHRoZSBmb3JtYXRpb24gb2YgdGlnaHQganVuY3Rpb25hbCBwZXJtZWFiaWxpdHkgYmFycmllciBpbiB0aGUgZGVjaWR1YWxpemluZyBzdHJvbWEgZHVyaW5nIGVtYnJ5byBpbXBsYW50YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENlbGwgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz01My02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kb21ldHJpYWwgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGluZyBpbiBwcmVnbmFudCBNZWlzaGFuIGFuZCBZb3Jrc2hpcmUgcGlncyBvbiBkYXkgMTIgb2YgZ2VzdGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWV0aHJlLUNob3R6ZW4gc3luZHJvbWUgY2F1c2VkIGJ5IFRXSVNUIDEgZ2VuZSBtdXRhdGlvbnM6IGZ1bmN0aW9uYWwgZGlmZmVyZW50aWF0aW9uIGZyb20gTXVlbmtlIGNvcm9uYWwgc3lub3N0b3NpcyBzeW5kcm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVyLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIHRoZSB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgc25haWwsIHNsdWcsIGFuZCB0d2lzdCBhbmQgdGhlaXIgY2xpbmljYWwgc2lnbmlmaWNhbmNlIGluIGh1bWFuIGJyZWFzdCBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gU3VyZy4gT25jb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDg4LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud2lzdCAxIHJlZ3VsYXRlcyB0aGUgZXhwcmVzc2lvbiBvZiBQUEFSZ2FtbWEgZHVyaW5nIGhvcm1vbmUtaW5kdWNlZCAzVDMtTDEgcHJlYWRpcG9jeXRlIGRpZmZlcmVudGlhdGlvbjogYSBwb3NzaWJsZSByb2xlIGluIG9iZXNpdHkgYW5kIGFzc29jaWF0ZWQgZGlzZWFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxpcGlkcyBIZWFsdGggRGlzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3aXN0MSBpbiBodW1hbiB3aGl0ZSBhZGlwb3NlIHRpc3N1ZSBhbmQgb2Jlc2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gRW5kb2NyaW5vbC4gTWV0YWIuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9MTMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNC8yNTkvOTk3OTUwL2RzdjAxMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUGlncyBoYXZlIGJlZW4gb25lIG9mIHRoZSBtb3N0IGltcG9ydGFudCBzb3VyY2VzIG9mIG1lYXQgZm9yIGh1bWFucywgYW5kIHRoZWlyIHByb2R1Y3Rpdml0eSBoYXMgYmVlbiBzdWJzdGFudGlhbGx5IGltcHJvdmVkIGJ5IHJlY2VudCBzdHJvbmcgc2VsZWN0aW8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0ODI0NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTYSAgdvAjNKEOxb2+1tCeIQbPdKQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f259%2f348246" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f259%2f348246" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13443" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&amp;Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/4">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4942269">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942269">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942271">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942271">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942276">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942293">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942293">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942313">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942313">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942315">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942315">Authors’ contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942317">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942317">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942319">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942319">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942321">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942321">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/4/245/348293">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/4/269/348206">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jung-Woo Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jung-Woo+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jung-Woo Choi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Won-Hyong Chung</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Won-Hyong+Chung">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chung W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chung W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Won-Hyong Chung</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyung-Tai Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyung-Tai+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyung-Tai Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eun-Seok Cho</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eun-Seok+Cho">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cho E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cho E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eun-Seok Cho</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Si-Woo Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Si-Woo+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Si-Woo Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bong-Hwan Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bong-Hwan+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Bong-Hwan Choi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sang-Heon Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Korea University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sang-Heon+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sang-Heon Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wonjun Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Korea University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wonjun+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wonjun Lim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dajeong Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dajeong+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Dajeong Lim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun-Gyeong Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Korea University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun-Gyeong+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yun-Gyeong Lee</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Joon-Ki Hong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Swine Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Suwon 441-706</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Joon-Ki+Hong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hong J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hong J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Joon-Ki Hong</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Doo-Wan Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Swine Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Suwon 441-706</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Doo-Wan+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Doo-Wan Kim</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hyeon-Jeong Jeon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hyeon-Jeong+Jeon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeon H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeon H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hyeon-Jeong Jeon</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiwoong Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiwoong+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jiwoong Kim</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Namshin Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Korea University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 63-238-7300. Fax. +82 63-238-7347. E-mail: <a href="mailto:thkim63@korea.kr" target="_blank">thkim63@korea.kr</a> (T.-H.K.); Tel. +82 42-879-8162. Fax. +82 42-879-8493. E-mail: <a href="mailto:n@rna.kr" target="_blank">n@rna.kr</a> (N.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Namshin+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Namshin Kim</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tae-Hun Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Animal Genomics and Bioinformatics Division</div>, <div class="insititution">National Institute of Animal Science, Rural Development Administration</div>, <div class="addr-line">Jeonju 565-851</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 63-238-7300. Fax. +82 63-238-7347. E-mail: <a href="mailto:thkim63@korea.kr" target="_blank">thkim63@korea.kr</a> (T.-H.K.); Tel. +82 42-879-8162. Fax. +82 42-879-8493. E-mail: <a href="mailto:n@rna.kr" target="_blank">n@rna.kr</a> (N.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tae-Hun+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tae-Hun Kim</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Quantitative Biomedical Research Center, Department of Clinical Sciences, University of Texas Southwestern Medical Center, Suite NC8.512, 6000 Harry Hines Boulevard, Dallas, TX 75390, USA.</div><div class="footnote-content">Harold C. Simmons Comprehensive Cancer Center, University of Texas Southwestern Medical Center, Suite NC8.512, 6000 Harry Hines Boulevard, Dallas, TX 75390, USA.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="footnote-content">Edited by Dr Takao Sekiya</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 259–267, <a href='https://doi.org/10.1093/dnares/dsv011'>https://doi.org/10.1093/dnares/dsv011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 June 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 November 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 May 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348246" href="/dnaresearch/article-pdf/22/4/259/997950/dsv011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jung-Woo Choi, Won-Hyong Chung, Kyung-Tai Lee, Eun-Seok Cho, Si-Woo Lee, Bong-Hwan Choi, Sang-Heon Lee, Wonjun Lim, Dajeong Lim, Yun-Gyeong Lee, Joon-Ki Hong, Doo-Wan Kim, Hyeon-Jeong Jeon, Jiwoong Kim, Namshin Kim, Tae-Hun Kim;  Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds, <em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 259–267, <a href='https://doi.org/10.1093/dnares/dsv011'>https://doi.org/10.1093/dnares/dsv011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348246?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348246?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Whole-genome%20resequencing%20analyses%20of%20five%20pig%20breeds%2C%20including%20Korean%20wild%20and%20native%2C%20and%20three%20European%20origin%20breeds&amp;publicationDate=2015-06-27&amp;volumeNum=22&amp;issueNum=4&amp;author=Choi%2C%20Jung-Woo%3B%20Chung%2C%20Won-Hyong&amp;startPage=259&amp;endPage=267&amp;contentId=10.1093%2Fdnares%2Fdsv011&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f259%2f348246" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4942269 id="4942269" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Pigs have been one of the most important sources of meat for humans, and their productivity has been substantially improved by recent strong selection. Here, we present whole-genome resequencing analyses of 55 pigs of five breeds representing Korean native pigs, wild boar and three European origin breeds. 1,673.1 Gb of sequence reads were mapped to the Swine reference assembly, covering ∼99.2% of the reference genome, at an average of ∼11.7-fold coverage. We detected 20,123,573 single-nucleotide polymorphisms (SNPs), of which 25.5% were novel. We extracted 35,458 of non-synonymous SNPs in 9,904 genes, which may contribute to traits of interest. The whole SNP sets were further used to access the population structures of the breeds, using multiple methodologies, including phylogenetic, similarity matrix, and population structure analysis. They showed clear population clusters with respect to each breed. Furthermore, we scanned the whole genomes to identify signatures of selection throughout the genome. The result revealed several promising loci that might underlie economically important traits in pigs, such as the <em>CLDN1</em> and <em>TWIST1</em> genes. These discoveries provide useful genomic information for further study of the discrete genetic mechanisms associated with economically important traits in pigs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome sequencing&quot;">whole-genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Pig">Pig</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;signature of selection&quot;">signature of selection</a></div>                            <h2 scrollto-destination=4942271 id="4942271" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Domestication of pigs originated from local wild boar populations in Europe and Asia circa 10,000 yrs ago<sup><a reveal-id="DSV011C1" data-open="DSV011C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and occurred independently from wild boar subspecies in Europe and Asia.<sup><a reveal-id="DSV011C2" data-open="DSV011C2" class="link link-ref link-reveal xref-bibr">2</a></sup> There are hundreds of domestic pig breeds available across the world, and pigs have served as one of the most important sources of animal protein for humans. Since the last century, traditional selective breeding has contributed significantly to the genetic improvement of economically important traits in pigs. For example, substantial improvements were achieved in pigs per sow per year (annually +0.25) and the feed conversion ratio (annually −0.009 to −0.070 kg of the dry matter intake per live mass gain kg) during recent decades.<sup><a reveal-id="DSV011C3" data-open="DSV011C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV011C4" data-open="DSV011C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Furthermore, some pig breeds have been used as mammalian model animals for the biomedical research because of their physiological and anatomical similarities to humans, and for developing several swine lines, including highly inbred miniature pig breeds.<sup><a reveal-id="DSV011C5" data-open="DSV011C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV011C6" data-open="DSV011C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, most of the achievements were made in European origin breeds, whereas, there are fewer systematic genetic improvement programs for diverse Asian local breeds.</p><p>There are two types of Korean indigenous pigs currently registered with the worldwide Domestic Animal Diversity Information System of the Food and Agriculture Organization of the United Nations: Chookjin Chamdon and Jeju native pig.<sup><a reveal-id="DSV011C7" data-open="DSV011C7" class="link link-ref link-reveal xref-bibr">7</a></sup> However, they are generally termed as the Korean native pig because of their same origin until the middle of the last century. Korean native pigs are known for their peculiar characteristics including higher intramuscular fat contents and chewy texture, which are appealing palatability factors to Korean pork consumers. However, they also have poorer productivities, such as lower growth rates and litters per sow per year than typical imported breeds, such as Duroc and Yorkshire.<sup><a reveal-id="DSV011C8" data-open="DSV011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV011C9" data-open="DSV011C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Since the beginning of the last century, the native breeds have been endangered, especially because of the extensive crossbreeding with economically promising imported breeds. To conserve the native genetic resource, recent efforts have been made by the National Institute of Animal Science (NIAS) in Korea to restore and manage the Korean native population.<sup><a reveal-id="DSV011C10" data-open="DSV011C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The Korean native pig is genetically more closely related to other Asian native pigs than to most European origin pigs.<sup><a reveal-id="DSV011C11" data-open="DSV011C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Kim et al.<sup><a reveal-id="DSV011C12" data-open="DSV011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> further showed that it might have a distinct genetic distance even from some Chinese native pig breeds.</p><p>The completion of the swine sequencing project has resulted in many single-nucleotide polymorphisms (SNPs) being identified throughout the genome.<sup><a reveal-id="DSV011C1" data-open="DSV011C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The recent advances in next-generation sequencing (NGS) technologies have led to further cataloguing of SNPs by resequencing of diverse pig breeds or multiple animals in comparison with the reference sequence assembly. Furthermore, the SNPs derived from NGS have been used successfully to dissect genomic characteristics of pigs. For example, Rubin et al. revealed significant signatures of selection in pig genomes by scanning whole genomes using SNPs.<sup><a reveal-id="DSV011C13" data-open="DSV011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In addition, three Berkshire pigs were resequenced recently to explore their genetic relationship with 38 pig genomes obtained from the public database, providing useful information on the breed's origin and domestication.<sup><a reveal-id="DSV011C14" data-open="DSV011C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>In this study, we present whole-genome sequencing analyses of 55 pigs of five breeds, including Korean wild boar, Korean native, Duroc, Landrace, and Yorkshire. Substantial numbers of SNPs were identified across the genome using the Swine reference assembly (<em>Sus scrofa</em> 10.2). SNP sets derived from the whole-genome sequencing were used to explore genomic characteristics among the diverse pig breeds and to detect genomic regions potentially implicated in the strong selective breeding applied to the pig populations in this study.</p>                            <h2 scrollto-destination=4942276 id="4942276" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4942277 id="4942277" class="section-title" data-legacy-id=s2a>2.1. Sampling and DNA extraction</h3>
<p>We sequenced 55 animals from five pig breeds, including 10 Korean wild boars, 10 Korean native pigs (4 boars and 6 sows), 6 Duroc pigs (3 boars and 3 sows), 14 Landrace pigs (7 boars and 7 sows), and 15 Yorkshire pigs (7 boars and 8 sows) for this study (Table <a reveal-id="DSV011TB1" data-open="DSV011TB1" class="link link-reveal link-table xref-fig">1</a>). The Korean native pigs were accessed at the Swine Science Division, NIAS, Rural Development Administration (RDA), Suwon; and the Subtropical Animal Experiment Station in NIAS, RDA, Jeju. The Duroc, Landrace, and Yorkshire pig samples were collected at the NIAS and Sunjin Co., Ltd in Korea. The Korean wild boars were captured in either Gyeonggi or Gyeongsangnam Provinces in South Korea, sampling five animals from each province, respectively. Genomic DNA was extracted from blood samples in EDTA using a Wizard Genomic DNA kit (Promega, Madison, WI, USA). The genomic DNA of the Korean wild boars was extracted from meat samples. The quality and quantity of the DNA were evaluated using a NanoDrop spectrophotometer (NanoDrop Technologies, USA) and gel electrophoresis in 1% agarose gel. The concentration of double-stranded DNA was checked using a Qubit dsDNA HS Assay (Invitrogen, USA). The National Institute of Animal Science's Institutional Animal Care and Use Committee reviewed and approved the study protocol and standard operating procedures (No. 2009-077, C-grade). </p>                        <a id="4942279" scrollto-destination="4942279"></a>
<div content-id="DSV011TB1" class="table-modal"><div class="table-wrap"><div id="DSV011TB1" class="table-wrap-title" data-id="DSV011TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of sequencing results and SNPs from the five pig breeds including KWB, KNP, DUR, LAN, and YOR used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample name </th><th>No. of sample </th><th>Raw_Reads </th><th>Mapped reads </th><th>A_Coverage<sup>a</sup> </th><th>Ave_Fold<sup>b</sup> </th></tr></thead><tbody><tr><td>Korean wild boar </td><td>10 </td><td>4,637,563,674 </td><td>2,881,666,665 </td><td>98.72% </td><td>11.21 X </td></tr><tr><td>Korean native pig </td><td>10 </td><td>4,738,183,150 </td><td>3,017,226,894 </td><td>98.94% </td><td>11.74 X </td></tr><tr><td>Duroc </td><td>6 </td><td>2,134,670,142 </td><td>1,788,806,414 </td><td>98.16% </td><td>11.60 X </td></tr><tr><td>Landrace </td><td>14 </td><td>6,092,030,408 </td><td>4,652,217,189 </td><td>99.18% </td><td>12.93 X </td></tr><tr><td>Yorkshire </td><td>15 </td><td>7,330,204,916 </td><td>4,225,021,030 </td><td>98.86% </td><td>10.96 X </td></tr><tr><td>Total </td><td><strong>55</strong> </td><td><strong>24,932,652,290</strong> </td><td><strong>16,564,938,192</strong> </td><td><strong>99.18%</strong> </td><td><strong>11.71 X</strong> </td></tr></tbody></table></div><div content-id="DSV011TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample name </th><th>No. of sample </th><th>Raw_Reads </th><th>Mapped reads </th><th>A_Coverage<sup>a</sup> </th><th>Ave_Fold<sup>b</sup> </th></tr></thead><tbody><tr><td>Korean wild boar </td><td>10 </td><td>4,637,563,674 </td><td>2,881,666,665 </td><td>98.72% </td><td>11.21 X </td></tr><tr><td>Korean native pig </td><td>10 </td><td>4,738,183,150 </td><td>3,017,226,894 </td><td>98.94% </td><td>11.74 X </td></tr><tr><td>Duroc </td><td>6 </td><td>2,134,670,142 </td><td>1,788,806,414 </td><td>98.16% </td><td>11.60 X </td></tr><tr><td>Landrace </td><td>14 </td><td>6,092,030,408 </td><td>4,652,217,189 </td><td>99.18% </td><td>12.93 X </td></tr><tr><td>Yorkshire </td><td>15 </td><td>7,330,204,916 </td><td>4,225,021,030 </td><td>98.86% </td><td>10.96 X </td></tr><tr><td>Total </td><td><strong>55</strong> </td><td><strong>24,932,652,290</strong> </td><td><strong>16,564,938,192</strong> </td><td><strong>99.18%</strong> </td><td><strong>11.71 X</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>A_Coverage, assembly coverage calculated as the proportion of bases in the genome assembly that were covered by at least one read.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Ave_Fold, average fold that was calculated as the average depth of coverage across the whole genome.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942279" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV011TB1" class="table-wrap-title" data-id="DSV011TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of sequencing results and SNPs from the five pig breeds including KWB, KNP, DUR, LAN, and YOR used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample name </th><th>No. of sample </th><th>Raw_Reads </th><th>Mapped reads </th><th>A_Coverage<sup>a</sup> </th><th>Ave_Fold<sup>b</sup> </th></tr></thead><tbody><tr><td>Korean wild boar </td><td>10 </td><td>4,637,563,674 </td><td>2,881,666,665 </td><td>98.72% </td><td>11.21 X </td></tr><tr><td>Korean native pig </td><td>10 </td><td>4,738,183,150 </td><td>3,017,226,894 </td><td>98.94% </td><td>11.74 X </td></tr><tr><td>Duroc </td><td>6 </td><td>2,134,670,142 </td><td>1,788,806,414 </td><td>98.16% </td><td>11.60 X </td></tr><tr><td>Landrace </td><td>14 </td><td>6,092,030,408 </td><td>4,652,217,189 </td><td>99.18% </td><td>12.93 X </td></tr><tr><td>Yorkshire </td><td>15 </td><td>7,330,204,916 </td><td>4,225,021,030 </td><td>98.86% </td><td>10.96 X </td></tr><tr><td>Total </td><td><strong>55</strong> </td><td><strong>24,932,652,290</strong> </td><td><strong>16,564,938,192</strong> </td><td><strong>99.18%</strong> </td><td><strong>11.71 X</strong> </td></tr></tbody></table></div><div content-id="DSV011TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample name </th><th>No. of sample </th><th>Raw_Reads </th><th>Mapped reads </th><th>A_Coverage<sup>a</sup> </th><th>Ave_Fold<sup>b</sup> </th></tr></thead><tbody><tr><td>Korean wild boar </td><td>10 </td><td>4,637,563,674 </td><td>2,881,666,665 </td><td>98.72% </td><td>11.21 X </td></tr><tr><td>Korean native pig </td><td>10 </td><td>4,738,183,150 </td><td>3,017,226,894 </td><td>98.94% </td><td>11.74 X </td></tr><tr><td>Duroc </td><td>6 </td><td>2,134,670,142 </td><td>1,788,806,414 </td><td>98.16% </td><td>11.60 X </td></tr><tr><td>Landrace </td><td>14 </td><td>6,092,030,408 </td><td>4,652,217,189 </td><td>99.18% </td><td>12.93 X </td></tr><tr><td>Yorkshire </td><td>15 </td><td>7,330,204,916 </td><td>4,225,021,030 </td><td>98.86% </td><td>10.96 X </td></tr><tr><td>Total </td><td><strong>55</strong> </td><td><strong>24,932,652,290</strong> </td><td><strong>16,564,938,192</strong> </td><td><strong>99.18%</strong> </td><td><strong>11.71 X</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>A_Coverage, assembly coverage calculated as the proportion of bases in the genome assembly that were covered by at least one read.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Ave_Fold, average fold that was calculated as the average depth of coverage across the whole genome.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942279" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4942280 id="4942280" class="section-title" data-legacy-id=s2b>2.2. Library construction and sequencing</h3>
<p>DNA libraries were constructed to have insert sizes of ∼300 bp, according to the manufacturer's instruction (Illumina, San Diego, CA, USA). The libraries were sequenced using the Illumina HiSeq 2000 platform (paired-end 101-bp reads) and Illumina GA IIx platform (paired-end 100-bp reads). We generated 24,932,652,290 reads (2,518 Gb) from 55 samples. Each sample was sequenced to have over &gt;34 Gb initial reads to produce a minimum of 10-fold genomic coverage.</p>                            <h3 scrollto-destination=4942282 id="4942282" class="section-title" data-legacy-id=s2c>2.3. Mapping, SNP calling, and annotation</h3>
<p>The sequenced reads were mapped to the Swine reference genome assembly (Sscrofa10.2) using Burrows Wheeler Aligner version 0.5.9 with default options.<sup><a reveal-id="DSV011C15" data-open="DSV011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The genome sequence was downloaded from the UCSC genome browser (<a class="link link-uri" href="http://hgdownload.soe.ucsc.edu/goldenPath/susScr3/bigZips/susScr3.fa.gz">http://hgdownload.soe.ucsc.edu/goldenPath/susScr3/bigZips/susScr3.fa.gz</a>). We also screened the unmapped reads and the reads that were aligned to unplaced scaffolds. Of the aligned reads, removal of polymerase chain reaction duplicates and re-synchronization of the mate information were performed using MarkDuplicates and FixMateInformation in the Picard software package version 1.48 (<a class="link link-uri" href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a>), respectively. Local alignment around indels was performed to the duplication-removed reads using RealignerTargetCreator and IndelRealigner in the Genome Analysis Toolkit16 (GATK; version 1.0.5974). Furthermore, basepair quality scores were recalibrated using CountCovariates and TableRecalibration in the GATK.<sup><a reveal-id="DSV011C16" data-open="DSV011C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Multi-sample SNP genotyping was performed to identify SNPs using UnifiedGenotyper in GATK. To reduce the false discovery rate, the filtering steps followed these criteria: QUAL &lt; 30.0, QD &lt; 2.0, MQ &lt; 40.0, FS &gt; 60.0, HaplotypeScore &gt; 13.0, MQRankSum &lt; −12.5, and ReadPosRankSum &lt; −8.0. All the filtered SNPs were annotated to belong to 12 functional categories (Table <a reveal-id="DSV011TB2" data-open="DSV011TB2" class="link link-reveal link-table xref-fig">2</a>) using an SNP annotation tool, snpEff and snpSift version 3.6c,<sup><a reveal-id="DSV011C17" data-open="DSV011C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV011C18" data-open="DSV011C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and the Ensembl <em>Sus scrofa</em> gene set version 75 (Sscrofa10.2.75). For the gene set, the canonical genes were applied to the annotation with the ‘-canon’ option in the snpEff program. Non-synonymous SNPs that had a SIFT score &lt;0.05 were classified as potentially damaging variants. SIFT scores for the Sus scrofa gene set were downloaded from SIFT 4G (SIFT Databases for Genomes; <a class="link link-uri" href="http://sift-dna.org">http://sift-dna.org</a>). </p>                        <a id="4942284" scrollto-destination="4942284"></a>
<div content-id="DSV011TB2" class="table-modal"><div class="table-wrap"><div id="DSV011TB2" class="table-wrap-title" data-id="DSV011TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of all the detected SNPs identified from all of the five breeds used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fields </th><th>Total </th><th>KWB </th><th>KNP </th><th>DUR </th><th>LAN </th><th>YOR </th></tr></thead><tbody><tr><td>Sample counts </td><td>55 </td><td>10 </td><td>10 </td><td>6 </td><td>14 </td><td>15 </td></tr><tr><td>SNP </td><td>20,123,573 </td><td>13,973,333 </td><td>9,592,404 </td><td>6,625,918 </td><td>10,872,881 </td><td>11,032,246 </td></tr><tr><td>Transition </td><td>13,352,480 </td><td>9,266,133 </td><td>6,304,688 </td><td>4,351,967 </td><td>7,166,396 </td><td>7,259,773 </td></tr><tr><td>Transversion </td><td>6,771,093 </td><td>4,707,200 </td><td>3,287,716 </td><td>2,273,951 </td><td>3,706,485 </td><td>3,772,473 </td></tr><tr><td colspan="7">SNP categories<sup>a</sup> </td></tr><tr><td> Synonymous coding </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr><tr><td> Non-synonymous coding </td><td>34,990 </td><td>22,267 </td><td>16,146 </td><td>10,871 </td><td>18,058 </td><td>19,101 </td></tr><tr><td> Start lost </td><td>26 </td><td>15 </td><td>15 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td> Stop gained </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Stop lost </td><td>49 </td><td>39 </td><td>25 </td><td>22 </td><td>31 </td><td>28 </td></tr><tr><td> Non-coding exon </td><td>96,102 </td><td>60,277 </td><td>46,017 </td><td>26,706 </td><td>51,131 </td><td>51,305 </td></tr><tr><td> UTR 5′ </td><td>11,888 </td><td>7,430 </td><td>5,562 </td><td>2,886 </td><td>6,188 </td><td>6,602 </td></tr><tr><td> UTR 3′ </td><td>68,432 </td><td>46,001 </td><td>31,269 </td><td>21,670 </td><td>35,645 </td><td>36,640 </td></tr><tr><td> Splice site acceptor </td><td>266 </td><td>175 </td><td>137 </td><td>94 </td><td>159 </td><td>165 </td></tr><tr><td> Splice site donor </td><td>275 </td><td>193 </td><td>133 </td><td>96 </td><td>161 </td><td>160 </td></tr><tr><td> Intron </td><td>4,955,607 </td><td>3,445,188 </td><td>2,337,787 </td><td>1,602,182 </td><td>2,660,083 </td><td>2,690,109 </td></tr><tr><td> Intergenic </td><td>14,902,488 </td><td>10,356,001 </td><td>7,130,757 </td><td>4,945,950 </td><td>8,074,319 </td><td>8,199,477 </td></tr><tr><td colspan="7">Functional categories<sup>b</sup> </td></tr><tr><td> Missense </td><td>35,065 </td><td>22,321 </td><td>16,186 </td><td>10,902 </td><td>18,103 </td><td>19,144 </td></tr><tr><td> Nonsense </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Silent </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr></tbody></table></div><div content-id="DSV011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Fields </th><th>Total </th><th>KWB </th><th>KNP </th><th>DUR </th><th>LAN </th><th>YOR </th></tr></thead><tbody><tr><td>Sample counts </td><td>55 </td><td>10 </td><td>10 </td><td>6 </td><td>14 </td><td>15 </td></tr><tr><td>SNP </td><td>20,123,573 </td><td>13,973,333 </td><td>9,592,404 </td><td>6,625,918 </td><td>10,872,881 </td><td>11,032,246 </td></tr><tr><td>Transition </td><td>13,352,480 </td><td>9,266,133 </td><td>6,304,688 </td><td>4,351,967 </td><td>7,166,396 </td><td>7,259,773 </td></tr><tr><td>Transversion </td><td>6,771,093 </td><td>4,707,200 </td><td>3,287,716 </td><td>2,273,951 </td><td>3,706,485 </td><td>3,772,473 </td></tr><tr><td colspan="7">SNP categories<sup>a</sup> </td></tr><tr><td> Synonymous coding </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr><tr><td> Non-synonymous coding </td><td>34,990 </td><td>22,267 </td><td>16,146 </td><td>10,871 </td><td>18,058 </td><td>19,101 </td></tr><tr><td> Start lost </td><td>26 </td><td>15 </td><td>15 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td> Stop gained </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Stop lost </td><td>49 </td><td>39 </td><td>25 </td><td>22 </td><td>31 </td><td>28 </td></tr><tr><td> Non-coding exon </td><td>96,102 </td><td>60,277 </td><td>46,017 </td><td>26,706 </td><td>51,131 </td><td>51,305 </td></tr><tr><td> UTR 5′ </td><td>11,888 </td><td>7,430 </td><td>5,562 </td><td>2,886 </td><td>6,188 </td><td>6,602 </td></tr><tr><td> UTR 3′ </td><td>68,432 </td><td>46,001 </td><td>31,269 </td><td>21,670 </td><td>35,645 </td><td>36,640 </td></tr><tr><td> Splice site acceptor </td><td>266 </td><td>175 </td><td>137 </td><td>94 </td><td>159 </td><td>165 </td></tr><tr><td> Splice site donor </td><td>275 </td><td>193 </td><td>133 </td><td>96 </td><td>161 </td><td>160 </td></tr><tr><td> Intron </td><td>4,955,607 </td><td>3,445,188 </td><td>2,337,787 </td><td>1,602,182 </td><td>2,660,083 </td><td>2,690,109 </td></tr><tr><td> Intergenic </td><td>14,902,488 </td><td>10,356,001 </td><td>7,130,757 </td><td>4,945,950 </td><td>8,074,319 </td><td>8,199,477 </td></tr><tr><td colspan="7">Functional categories<sup>b</sup> </td></tr><tr><td> Missense </td><td>35,065 </td><td>22,321 </td><td>16,186 </td><td>10,902 </td><td>18,103 </td><td>19,144 </td></tr><tr><td> Nonsense </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Silent </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>UTR, untranslated region.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>SNP categories: categorized by the effects of SNPs.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>b</sup>Functional categories: categorized by functional effects of coding SNPs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942284" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV011TB2" class="table-wrap-title" data-id="DSV011TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of all the detected SNPs identified from all of the five breeds used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Fields </th><th>Total </th><th>KWB </th><th>KNP </th><th>DUR </th><th>LAN </th><th>YOR </th></tr></thead><tbody><tr><td>Sample counts </td><td>55 </td><td>10 </td><td>10 </td><td>6 </td><td>14 </td><td>15 </td></tr><tr><td>SNP </td><td>20,123,573 </td><td>13,973,333 </td><td>9,592,404 </td><td>6,625,918 </td><td>10,872,881 </td><td>11,032,246 </td></tr><tr><td>Transition </td><td>13,352,480 </td><td>9,266,133 </td><td>6,304,688 </td><td>4,351,967 </td><td>7,166,396 </td><td>7,259,773 </td></tr><tr><td>Transversion </td><td>6,771,093 </td><td>4,707,200 </td><td>3,287,716 </td><td>2,273,951 </td><td>3,706,485 </td><td>3,772,473 </td></tr><tr><td colspan="7">SNP categories<sup>a</sup> </td></tr><tr><td> Synonymous coding </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr><tr><td> Non-synonymous coding </td><td>34,990 </td><td>22,267 </td><td>16,146 </td><td>10,871 </td><td>18,058 </td><td>19,101 </td></tr><tr><td> Start lost </td><td>26 </td><td>15 </td><td>15 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td> Stop gained </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Stop lost </td><td>49 </td><td>39 </td><td>25 </td><td>22 </td><td>31 </td><td>28 </td></tr><tr><td> Non-coding exon </td><td>96,102 </td><td>60,277 </td><td>46,017 </td><td>26,706 </td><td>51,131 </td><td>51,305 </td></tr><tr><td> UTR 5′ </td><td>11,888 </td><td>7,430 </td><td>5,562 </td><td>2,886 </td><td>6,188 </td><td>6,602 </td></tr><tr><td> UTR 3′ </td><td>68,432 </td><td>46,001 </td><td>31,269 </td><td>21,670 </td><td>35,645 </td><td>36,640 </td></tr><tr><td> Splice site acceptor </td><td>266 </td><td>175 </td><td>137 </td><td>94 </td><td>159 </td><td>165 </td></tr><tr><td> Splice site donor </td><td>275 </td><td>193 </td><td>133 </td><td>96 </td><td>161 </td><td>160 </td></tr><tr><td> Intron </td><td>4,955,607 </td><td>3,445,188 </td><td>2,337,787 </td><td>1,602,182 </td><td>2,660,083 </td><td>2,690,109 </td></tr><tr><td> Intergenic </td><td>14,902,488 </td><td>10,356,001 </td><td>7,130,757 </td><td>4,945,950 </td><td>8,074,319 </td><td>8,199,477 </td></tr><tr><td colspan="7">Functional categories<sup>b</sup> </td></tr><tr><td> Missense </td><td>35,065 </td><td>22,321 </td><td>16,186 </td><td>10,902 </td><td>18,103 </td><td>19,144 </td></tr><tr><td> Nonsense </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Silent </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr></tbody></table></div><div content-id="DSV011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Fields </th><th>Total </th><th>KWB </th><th>KNP </th><th>DUR </th><th>LAN </th><th>YOR </th></tr></thead><tbody><tr><td>Sample counts </td><td>55 </td><td>10 </td><td>10 </td><td>6 </td><td>14 </td><td>15 </td></tr><tr><td>SNP </td><td>20,123,573 </td><td>13,973,333 </td><td>9,592,404 </td><td>6,625,918 </td><td>10,872,881 </td><td>11,032,246 </td></tr><tr><td>Transition </td><td>13,352,480 </td><td>9,266,133 </td><td>6,304,688 </td><td>4,351,967 </td><td>7,166,396 </td><td>7,259,773 </td></tr><tr><td>Transversion </td><td>6,771,093 </td><td>4,707,200 </td><td>3,287,716 </td><td>2,273,951 </td><td>3,706,485 </td><td>3,772,473 </td></tr><tr><td colspan="7">SNP categories<sup>a</sup> </td></tr><tr><td> Synonymous coding </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr><tr><td> Non-synonymous coding </td><td>34,990 </td><td>22,267 </td><td>16,146 </td><td>10,871 </td><td>18,058 </td><td>19,101 </td></tr><tr><td> Start lost </td><td>26 </td><td>15 </td><td>15 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td> Stop gained </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Stop lost </td><td>49 </td><td>39 </td><td>25 </td><td>22 </td><td>31 </td><td>28 </td></tr><tr><td> Non-coding exon </td><td>96,102 </td><td>60,277 </td><td>46,017 </td><td>26,706 </td><td>51,131 </td><td>51,305 </td></tr><tr><td> UTR 5′ </td><td>11,888 </td><td>7,430 </td><td>5,562 </td><td>2,886 </td><td>6,188 </td><td>6,602 </td></tr><tr><td> UTR 3′ </td><td>68,432 </td><td>46,001 </td><td>31,269 </td><td>21,670 </td><td>35,645 </td><td>36,640 </td></tr><tr><td> Splice site acceptor </td><td>266 </td><td>175 </td><td>137 </td><td>94 </td><td>159 </td><td>165 </td></tr><tr><td> Splice site donor </td><td>275 </td><td>193 </td><td>133 </td><td>96 </td><td>161 </td><td>160 </td></tr><tr><td> Intron </td><td>4,955,607 </td><td>3,445,188 </td><td>2,337,787 </td><td>1,602,182 </td><td>2,660,083 </td><td>2,690,109 </td></tr><tr><td> Intergenic </td><td>14,902,488 </td><td>10,356,001 </td><td>7,130,757 </td><td>4,945,950 </td><td>8,074,319 </td><td>8,199,477 </td></tr><tr><td colspan="7">Functional categories<sup>b</sup> </td></tr><tr><td> Missense </td><td>35,065 </td><td>22,321 </td><td>16,186 </td><td>10,902 </td><td>18,103 </td><td>19,144 </td></tr><tr><td> Nonsense </td><td>393 </td><td>230 </td><td>189 </td><td>92 </td><td>189 </td><td>214 </td></tr><tr><td> Silent </td><td>53,057 </td><td>35,517 </td><td>24,367 </td><td>15,340 </td><td>26,903 </td><td>28,430 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>UTR, untranslated region.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>SNP categories: categorized by the effects of SNPs.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>b</sup>Functional categories: categorized by functional effects of coding SNPs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942284" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4942285 id="4942285" class="section-title" data-legacy-id=s2d>2.4. Phylogeny and admixture analysis</h3>
<p>To perform phylogenetic and admixture analyses, haplotypes of all the animals in this study were inferred by an imputation tool, BEAGLE, version 3.3.2.<sup><a reveal-id="DSV011C19" data-open="DSV011C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The phased haplotypes based on the SNPs were used to conduct phylogenetic analysis. The phylogenetic tree was generated using neighbour-joining tree estimation with pairwise distance, and Kimura's two-parameter distance method implemented in Analysis of Phylogenetic and Evolution (version 3.0.6) in R language.<sup><a reveal-id="DSV011C20" data-open="DSV011C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The tree was plotted using MEGA5,<sup><a reveal-id="DSV011C21" data-open="DSV011C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and the population structures were accessed by admixture analysis using the STRUCTURE version 2.3.4, which is based on the maximum likelihood method.<sup><a reveal-id="DSV011C22" data-open="DSV011C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We used different values of <em>K</em> (number of putative ancestral clusters of allelic similarity) and an admixture model with a correlated allele frequency to assign the <em>K</em> clusters.<sup><a reveal-id="DSV011C23" data-open="DSV011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> A 30,000 burn-in period of Chain Monte Carlo searches followed by 20,000 replicate runs were performed at each <em>K</em> from 2 to 5. The result was then plotted using DISTRUCT version 1.1.<sup><a reveal-id="DSV011C24" data-open="DSV011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The pairwise similarity between each animal was computed by the number of the same SNP genotypes divided by the total number of SNPs. The order of the 55 animals for the STRUCTURE plot and the similarity matrix followed the same order used in the phylogenetic tree.</p>                            <h3 scrollto-destination=4942287 id="4942287" class="section-title" data-legacy-id=s2e>2.5. Identification of selective sweeps</h3>
<p>To identify putative selective sweeps, we searched genomic regions with higher degrees of fixation, following the method by Rubin et al.<sup><a reveal-id="DSV011C13" data-open="DSV011C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV011C25" data-open="DSV011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The number of major and minor allele sequence reads were counted at each of the identified SNP in each breed. We filtered the SNP positions whose minor allele frequency was &lt;0.05. We then scanned the genome using 50% overlapping windows of 150 kb in size, which was successfully adopted in the previous pig resequencing study.<sup><a reveal-id="DSV011C25" data-open="DSV011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Among the 150 kb windows, we excluded windows with fewer than 10 of the number of heterozygous SNPs in each breed. Briefly, the Pooled heterozygosity (Hp) was initially computed by an equation: 2*<em>sum_maj</em>*<em>sum_min</em>/(<em>sum_maj</em>+ <em>sum_min</em>),<sup><a reveal-id="DSV011C2" data-open="DSV011C2" class="link link-ref link-reveal xref-bibr">2</a></sup> where <em>sum_maj</em> and <em>sum_min</em> are the sum of major and minor allele frequency at the given 150 kb window, using the numbers of reference and alternative allele sequence reads observed in each of the breeds. The ZHp score was calculated from <em>Z</em> transformation of the Hp score as described by Rubin et al.<sup><a reveal-id="DSV011C13" data-open="DSV011C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV011C25" data-open="DSV011C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>                            <h3 scrollto-destination=4942289 id="4942289" class="section-title" data-legacy-id=s2f>2.6. Validation of SNPs by genotyping chip array</h3>
<p>To validate the accuracy of SNP calling from whole-genome sequencing of five pig breeds, we genotyped five animals per each breed using the Illumina PorcineSNP60 v2 Genotyping BeadChip. Of the 64,232 SNP probes in the chip, we selected 44,903 probes that were successfully located on the autosomal and sex chromosomes. The reference allele of a chip SNP was set to the allele that is equal to the base at the reference genome, Sus scrofa 10.2. The genotype concordance was measured for each animal by the number of concordant genotypes between the SNP chip and the whole-genome sequencing derived SNPs divided by the total number of the SNP probes. The method was referred from GATK online documentation for VariantEval (<a class="link link-uri" href="http://gatkforums.broadinstitute.org/discussion/48/using-varianteval">http://gatkforums.broadinstitute.org/discussion/48/using-varianteval</a>).</p>                            <h3 scrollto-destination=4942291 id="4942291" class="section-title" data-legacy-id=s2g>2.7. Data access</h3>
<p>All SNPs detected in this study have been submitted to the NCBI dbSNP with the accession numbers: ss1754731760–ss1774855430.</p>                            <h2 scrollto-destination=4942293 id="4942293" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4942294 id="4942294" class="section-title" data-legacy-id=s3a>3.1. Sequencing and read mapping</h3>
<p>We extracted genomic DNA from 55 pigs of five breeds including Korean wild boar (KWB), Korean native (KNP), Duroc (DUR), Landrace (LAN), and Yorkshire (YOR). Approximately 24.9 billion reads were generated by short-read sequencing technology including the Illumina GA IIx and HiSeq 2000 sequencing platforms. This equated to ∼2.52 terabases and 897-fold coverage compared with the expected genome size of the Swine reference genome (Sscrofa 10.2; size ∼2.8 Gb). To detect high-quality SNPs from all the samples, we sequenced at least 34 Gb for each of the samples. Of the 55 sequenced animals, the least amount of sequence reads was 12.1-fold for the ‘DUR5’ individual. After mapping the reads to the reference genome, we removed erroneous sequence reads that were caused by potential polymerase chain reaction duplication (17.5%). This yielded ∼16.6 billion reads (1,673.1 Gb), covering 99.2% of the reference assembly at an average of 11.7-fold coverage across the covered region (Table <a reveal-id="DSV011TB1" data-open="DSV011TB1" class="link link-reveal link-table xref-fig">1</a>). There was no outstanding coverage bias observed on any specific chromosome within each of the animals.</p>                            <h3 scrollto-destination=4942296 id="4942296" class="section-title" data-legacy-id=s3b>3.2. SNP detection</h3>
<p>We identified a total of 20,123,573 SNPs throughout the genomes from all the 55 sequenced pigs of the five breeds in this study. The number of SNPs in each breed population varied from 6.6 to 14.0 million; the least number of the SNP set was observed in DUR, while the most values were found in KWB (Table <a reveal-id="DSV011TB2" data-open="DSV011TB2" class="link link-reveal link-table xref-fig">2</a>). This was as expected, because the DUR was the breed used to construct the Sscrofa 10.2 reference assembly, and the KWB is the most genetically distant of the pig breeds in this study.<sup><a reveal-id="DSV011C1" data-open="DSV011C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV011C25" data-open="DSV011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In addition, we do not rule out the possibility that parts of the SNPs could be caused by the within-population variation in this study. Of the total SNPs, ∼25.5% were found to be novel, as assessed by the dbSNP build 137 (Fig. <a reveal-id="DSV011F1" data-open="DSV011F1" class="link link-reveal link-table xref-fig">1</a>a). Correspondingly, KWB showed the highest percentage of novel SNPs (24.9%) compared with the other four breeds (16.4, 13.6, 16.7, and 16.1% for KNP, DUR, LAN, and YOR, respectively). These values indicated that many SNPs remain to be identified by further sequencing efforts, although numerous SNPs have accumulated since the recent completion of the swine sequencing project. The quality of the detected SNPs was examined by calculating the transition-to-transversion ratio (Ti/Tv) for each SNP (Table <a reveal-id="DSV011TB2" data-open="DSV011TB2" class="link link-reveal link-table xref-fig">2</a>) as well as experimental validation performing a concordance test. The Ti/Tv ratio has been used as an indicator of potential sequencing errors, and it has been empirically approximated to around 2.1 and 2.2 in recent resequencing studies in Humans and cattle, respectively.<sup><a reveal-id="DSV011C26" data-open="DSV011C26" class="link link-ref link-reveal xref-bibr">26</a>–<a reveal-id="DSV011C28" data-open="DSV011C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Following the previous studies, Ti/Tv ratios for most of the pigs are in accordance with the values observed in the Human and cattle, with no outstanding values (KWB: 1.97, KNP: 1.92, DUR: 1.91, LAN: 1.93, YOR: 1.92). Furthermore, we genotyped a part of the same individuals sequenced in this study, using the Illumina PorcineSNP60K BeadChip to perform a concordance test between the SNP panel genotype and SNPs derived from this study. A total of 44,903 SNPs were used for the concordance test to show high concordance rates (98.8–99.4%), which is reliable enough for further investigations (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These results well suggested that most of the SNPs identified in this study were reasonably accurate. </p>                        <a id="4942298" scrollto-destination="4942298"></a>
<div id="DSV011F1" data-id="DSV011F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01101.png?Expires=2147483647&Signature=EEFCig~uXqzJWY6gBqaZ42az1sknZlC0MSl8nl1SrvvdWI4SGQpAwHG3mmx34O6K6iCQd~yeOHcJiw-i6qMeSLJlSwPEUVyR9UFQMSUVECP9srZTUQ0jS22lQWNV-NNNzxGw0DURrQO6PGAWnGYkJj8L5ED1hRtaSKmxh6trQBKZu9GlkbbmSC8~yNMno0lWC74YGne9STyj0bkcFlZl4Me87f-W6ZuakkpTQiPagYHu5-z-JWNr2gmg8gxbwXJQVA-xrfZqxau8vjYp9lLGA7oW1chkO-ehjNNgCzTYEA8ReJHl89qsggD7rxoOI564gXbvxO4Q2lDURV2dKDwPaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the identified SNPs. (a) The number of SNPs identified from 55 pigs of the five breeds used in this study (KWB, KNP, DUR, LAN, and YOR). (b) The pie chart showing functional categories of all the detected SNPs. In the functional categories, the ‘Others’ represents the sum of three functional categories including ‘start lost’, ‘stop gained’, and ‘stop lost’. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942298/dsv01101.tiff" path-from-xml="dsv01101.tiff" target="_blank">View large</a><a section="4942298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01101.png?Expires=2147483647&Signature=O8cG4y8UPEoJb8BIF5UYC3lAXdgm6vnwv8n1xcKq3wt8AGX-9dtbI0Xu2j8dKZiJejJf1XB-2lP0oAOSgISaEfFB0uT0qy-DQFe3sHYjtRS1uXNU6XQGKCPR2lk7rmapN2xgFw-~xXKCfcELJSsfzPlJK3dWm8jq7MN3GpDpp5aTVMKEF-q4F6aZjgUc4kIizJVdQ~0ZbP9RhBsstB28ZacSVF8rL9Q2JCRi4A~tQ-QNvSqdb0nYdGRiFjbB4tCvDUUZxEYKjVNAewsEHEamZ-L2qO1uB4PQojUh4DZtg12f8Q4HvpN9gj7A4DcXsw6vVlXm047bij8M1thXCByz8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942298&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the identified SNPs. (a) The number of SNPs identified from 55 pigs of the five breeds used in this study (KWB, KNP, DUR, LAN, and YOR). (b) The pie chart showing functional categories of all the detected SNPs. In the functional categories, the ‘Others’ represents the sum of three functional categories including ‘start lost’, ‘stop gained’, and ‘stop lost’. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV011F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01101.png?Expires=2147483647&Signature=EEFCig~uXqzJWY6gBqaZ42az1sknZlC0MSl8nl1SrvvdWI4SGQpAwHG3mmx34O6K6iCQd~yeOHcJiw-i6qMeSLJlSwPEUVyR9UFQMSUVECP9srZTUQ0jS22lQWNV-NNNzxGw0DURrQO6PGAWnGYkJj8L5ED1hRtaSKmxh6trQBKZu9GlkbbmSC8~yNMno0lWC74YGne9STyj0bkcFlZl4Me87f-W6ZuakkpTQiPagYHu5-z-JWNr2gmg8gxbwXJQVA-xrfZqxau8vjYp9lLGA7oW1chkO-ehjNNgCzTYEA8ReJHl89qsggD7rxoOI564gXbvxO4Q2lDURV2dKDwPaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the identified SNPs. (a) The number of SNPs identified from 55 pigs of the five breeds used in this study (KWB, KNP, DUR, LAN, and YOR). (b) The pie chart showing functional categories of all the detected SNPs. In the functional categories, the ‘Others’ represents the sum of three functional categories including ‘start lost’, ‘stop gained’, and ‘stop lost’. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942298/dsv01101.tiff" path-from-xml="dsv01101.tiff" target="_blank">View large</a><a section="4942298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01101.png?Expires=2147483647&Signature=O8cG4y8UPEoJb8BIF5UYC3lAXdgm6vnwv8n1xcKq3wt8AGX-9dtbI0Xu2j8dKZiJejJf1XB-2lP0oAOSgISaEfFB0uT0qy-DQFe3sHYjtRS1uXNU6XQGKCPR2lk7rmapN2xgFw-~xXKCfcELJSsfzPlJK3dWm8jq7MN3GpDpp5aTVMKEF-q4F6aZjgUc4kIizJVdQ~0ZbP9RhBsstB28ZacSVF8rL9Q2JCRi4A~tQ-QNvSqdb0nYdGRiFjbB4tCvDUUZxEYKjVNAewsEHEamZ-L2qO1uB4PQojUh4DZtg12f8Q4HvpN9gj7A4DcXsw6vVlXm047bij8M1thXCByz8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942298&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the identified SNPs. (a) The number of SNPs identified from 55 pigs of the five breeds used in this study (KWB, KNP, DUR, LAN, and YOR). (b) The pie chart showing functional categories of all the detected SNPs. In the functional categories, the ‘Others’ represents the sum of three functional categories including ‘start lost’, ‘stop gained’, and ‘stop lost’. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4942299 id="4942299" class="section-title" data-legacy-id=s3c>3.3. SNP annotation and gene of interest</h3>
<p>All the detected SNPs were annotated functionally using Ensembl gene annotation and dbSNP databases, assigning them to 12 functional classes. In agreement with previous studies, most of the SNPs were located in intergenic and intron regions (74.1 and 24.6% respectively), while fewer SNPs (1.3%) resided within exonic region, including exonic, splice sites, and untranslated regions (Fig. <a reveal-id="DSV011F1" data-open="DSV011F1" class="link link-reveal link-table xref-fig">1</a>b and Table <a reveal-id="DSV011TB2" data-open="DSV011TB2" class="link link-reveal link-table xref-fig">2</a>). Among the genic SNPs, we identified a substantial number (35,458) of non-synonymous SNPs (nsSNPs) in a total of 9,904 genes, which could be associated with traits of interest in pigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). To prioritize the nsSNPs, all the nsSNPs were further assigned a SIFT score to classify the effects of each nsSNP on phenotypic functions, based on the amino acid conservation.<sup><a reveal-id="DSV011C29" data-open="DSV011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> According to the typical interpretation of the score, a score of ≤0.05 classifies an nsSNP as ‘damaging’ and a score of &gt;0.05 corresponds to ‘tolerant’. There were 6,849 damaging and 27,215 tolerant nsSNPs in our data (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Furthermore, we extracted nsSNP sets that did not overlap among the breeds in this study (breed-specific nsSNP) and identified 126, 26, 11, 10, and 8 for KWB, KNP, DUR, LAN, and YOR, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>Of the breed-specific nsSNP, we identified several interesting nsSNPs that may be implicated with economically important traits in the pig. For example, a YOR-specific novel nsSNP (SIFT score: 0.03; 6,916,234 bp on chromosome 2) was located in the phospholipase C, beta 3 gene (<em>PLCB3</em>). <em>PLCB3</em> is involved in the mammalian oocyte at fertilization, spermatozoa activation, and embryonic lethality in sea urchin and mouse studies.<sup><a reveal-id="DSV011C30  DSV011C32" data-open="DSV011C30  DSV011C32" class="link link-ref link-reveal xref-bibr">30–32</a></sup> Notably, the Yorkshire (YOR) has been widely used as a maternal line breed, particularly for its large litter size; thus, the SNP in <em>PLCB3</em> is a candidate SNP to account for genetic variation in the reproduction trait, whose genetic effects have been difficult to predict. Little is known about which genes affect litter size, which has a low heritability and is a highly appreciated trait in the pig industry.<sup><a reveal-id="DSV011C33" data-open="DSV011C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSV011C34" data-open="DSV011C34" class="link link-ref link-reveal xref-bibr">34</a></sup> A homozygous SNP was found in all 10 KWB individuals sequenced in this study (GG against the A reference allele). Interestingly, the same pattern was observed in the European wild boar samples, while the G allele frequency is around 33% in the Asian wild boars in a recent independent study (Dr Martien Groenen, personal communication). The gene affected by the SNP has homology to Neurexophilin and PC-esterase domain family, member 4 (<em>NXPE4</em>). Members of the NXPE family are neuropeptides that potentially signal via alpha-neurexins.<sup><a reveal-id="DSV011C35" data-open="DSV011C35" class="link link-ref link-reveal xref-bibr">35</a></sup> It could be hypothesized that the NXPE4 homolog is involved in neurological mechanisms associated with temperament or fear, whose genetic mechanisms are poorly understood in pigs. Although it is beyond the scope of this study to conclude how each of the genes are affected by the nsSNPs, the many damaging nsSNPs identified provide ample information to identify promising candidates for further studies to dissect genetic mechanisms of diverse traits in pigs.</p>                            <h3 scrollto-destination=4942303 id="4942303" class="section-title" data-legacy-id=s3d>3.4. Phylogenetic and admixture analysis</h3>
<p>To examine the population structure and genetic relationships of the five breeds, we carried out multiple analyses, including phylogenetic analysis by neighbour-joining estimation, similarity estimation by comparison of genotypes, and admixture analysis using the STRUCTURE program. All five breeds were consistently separated by those three methods, as expected. The wild boar (KWB) was mostly distinguished from the four domesticated breeds (KNP, DUR, LAN, and YOR), and a clear genetic distance was also observed within the domesticated breeds with the respect to each breed (Fig. <a reveal-id="DSV011F2" data-open="DSV011F2" class="link link-reveal link-table xref-fig">2</a>a). The result was further supported by the similarity matrix, which showed genotyping similarity of &lt;41% between all the KWBs and the domesticated breed individuals. KWB appears to be genetically closer to KNP than any other breed, as shown in both the phylogenetic tree and similarity matrix. The similarity level between KWB and KNP varied from 38.8 to 41%, while that between KWB and the other three breeds was 36.7–38.9%. We observed the maximum value of the genotyping similarity (∼70%) across the domestic breeds, while the minimum values within each breed were &gt;70%, except for the KNP (minimum value = 68.1%). In the STRUCTURE analysis, we found the best-fit value of <em>K</em> as 5 to estimate the most likely number of genetic clusters, which corresponded to the number of breeds used in the study. While observing obvious genetic differentiation among the breeds, LAN and YOR have partially shared genetic differentiation signals; YOR shared 4–12.5% of LAN and LAN shared 8.3–31.5% of the YOR. Interestingly, among the 10 KNP individuals, six animals (Chookjin Chamdon, KNP Nos. 1–6) appeared to be genetically distant from the other four KNPs (Jeju native pig, KNP Nos. 7–10) that share other breed populations (11.3–14.6, 12.8–17.3, and 11.4–13.3% for DUR, LAN, and YOR, respectively). The result was found consistently in the phylogenetic tree, similarity matrix, and STRUCTURE analyses in this study. KNPs can be classified as two subgroups with respect to their geographical location in Korea, and they have been bred separately for ∼50 yrs. Despite their same origin until the last century, this result may imply that the Chookjin Chamdon and Jeju native have discernable genetic differences that may be caused by their geographical isolation and different environments. In addition, we did not rule out the possibility that the native breed may have been affected by unrecorded crossbreeding with imported breeds before the systematic management of the native genetic resources. </p>                        <a id="4942307" scrollto-destination="4942307"></a>
<div id="DSV011F2" data-id="DSV011F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01102.png?Expires=2147483647&Signature=Iz0yCHzgE5kjY5Jd1L-NKDtMSU-lovMTPU8-rDU6VWd7u35DnTj3mpszJODoac6sQTIVHCRdENfInc~R969uyTbQpda9i97xNwW1PSpGjisqR5vbeVvOlfslpTu7fyOZmsw5Bf5ddfsytknZ0XM7q7XLnYX46F-a7NJUVVf69OpWbibHOAZ7tF3I81S-EdOhfJlE~4XOsU~7OocXJgKv9pMEIUbxesk3n-ofOS~vO6ialMb3KtXcztux20-DRH8TeEQEN5aSaa4hSo9DUjHn8zGhwGbIast9V4Vrgg9D2BiuU0WnZdNMcisXTzeoBw8MNJIIgp4JpEU0BXc9xS-N4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic structure of the 55 pigs of the five pig breeds (KWB, KNP, DUR, LAN, and YOR) in this study. (a) Neighbour-joining phylogenetic tree of all the 55 pigs. (b) Population structure using the STRUCTURE program, which Bayesian cluster all 55 samples from the five breeds. (c) The similarity matrix of the 55 pigs based on the calculated SNP identity. Each of all the 55 pigs was presented in the same order on those three (a–c) plots. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942307/dsv01102.tiff" path-from-xml="dsv01102.tiff" target="_blank">View large</a><a section="4942307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01102.png?Expires=2147483647&Signature=4TYvBpSHu8I7kCKapRihrmxImMHn~AE5JbUMg7gHOs2evQXlLExJhBioHub4M0zzRSwJDOHTvR16tgwu1Vk2gpDWPZBluGTELzq4drQ8~pmVz5v8egn-gQWBK6jq8lMzUTb9uas7crl6YzKm3XQsbvlL4v3vhAItmUP6nEp5LYIA4Vx7Jh-ifUdVZvgJiNKoGyHzxW-4waMA3JC8~AJA4~fn6eY4AQhrF4Ms3vwL5AvbHQHIQB3oE1flrZJXB4EW~tixwt--2nyyyDmZaYMalZwUef72E-itAQWWkyEp3GmsF6zCY4wnjVhxqwpCQbO5H4Lj4qgFIxplNACbuV7ybg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942307&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic structure of the 55 pigs of the five pig breeds (KWB, KNP, DUR, LAN, and YOR) in this study. (a) Neighbour-joining phylogenetic tree of all the 55 pigs. (b) Population structure using the STRUCTURE program, which Bayesian cluster all 55 samples from the five breeds. (c) The similarity matrix of the 55 pigs based on the calculated SNP identity. Each of all the 55 pigs was presented in the same order on those three (a–c) plots. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV011F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01102.png?Expires=2147483647&Signature=Iz0yCHzgE5kjY5Jd1L-NKDtMSU-lovMTPU8-rDU6VWd7u35DnTj3mpszJODoac6sQTIVHCRdENfInc~R969uyTbQpda9i97xNwW1PSpGjisqR5vbeVvOlfslpTu7fyOZmsw5Bf5ddfsytknZ0XM7q7XLnYX46F-a7NJUVVf69OpWbibHOAZ7tF3I81S-EdOhfJlE~4XOsU~7OocXJgKv9pMEIUbxesk3n-ofOS~vO6ialMb3KtXcztux20-DRH8TeEQEN5aSaa4hSo9DUjHn8zGhwGbIast9V4Vrgg9D2BiuU0WnZdNMcisXTzeoBw8MNJIIgp4JpEU0BXc9xS-N4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic structure of the 55 pigs of the five pig breeds (KWB, KNP, DUR, LAN, and YOR) in this study. (a) Neighbour-joining phylogenetic tree of all the 55 pigs. (b) Population structure using the STRUCTURE program, which Bayesian cluster all 55 samples from the five breeds. (c) The similarity matrix of the 55 pigs based on the calculated SNP identity. Each of all the 55 pigs was presented in the same order on those three (a–c) plots. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942307/dsv01102.tiff" path-from-xml="dsv01102.tiff" target="_blank">View large</a><a section="4942307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01102.png?Expires=2147483647&Signature=4TYvBpSHu8I7kCKapRihrmxImMHn~AE5JbUMg7gHOs2evQXlLExJhBioHub4M0zzRSwJDOHTvR16tgwu1Vk2gpDWPZBluGTELzq4drQ8~pmVz5v8egn-gQWBK6jq8lMzUTb9uas7crl6YzKm3XQsbvlL4v3vhAItmUP6nEp5LYIA4Vx7Jh-ifUdVZvgJiNKoGyHzxW-4waMA3JC8~AJA4~fn6eY4AQhrF4Ms3vwL5AvbHQHIQB3oE1flrZJXB4EW~tixwt--2nyyyDmZaYMalZwUef72E-itAQWWkyEp3GmsF6zCY4wnjVhxqwpCQbO5H4Lj4qgFIxplNACbuV7ybg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942307&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic structure of the 55 pigs of the five pig breeds (KWB, KNP, DUR, LAN, and YOR) in this study. (a) Neighbour-joining phylogenetic tree of all the 55 pigs. (b) Population structure using the STRUCTURE program, which Bayesian cluster all 55 samples from the five breeds. (c) The similarity matrix of the 55 pigs based on the calculated SNP identity. Each of all the 55 pigs was presented in the same order on those three (a–c) plots. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4942308 id="4942308" class="section-title" data-legacy-id=s3e>3.5. Selective sweep analysis</h3>
<p>The whole genomes were scanned to identify genomic regions with excess homozygosity as indicative of a signature of selection in each breed. By applying 50% overlapping windows of 150 kb in size, 32,651 windows were used in the genome scanning. All the detected SNPs in each breed were used to calculate the ZHp. Altogether, 236, 472, 1,568, 383, and 363 windows were removed that had fewer than 10 SNPs in KWB, KNP, DUR, LAN, and YOR, respectively. In each breed, the ZHp values varied from −4.83 to 2.73, −3.19 to 1.58, −2.57 to 1.45, −3.60 to 2.07, and −3.54 to 1.98 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>), and we observed 1,277, 1,219, 656, 759, and 904 windows with ZHp values lower than −2 for KWB, KNP, DUR, LAN, and YOR, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4–S8</a>). An extremely low ZHp score is indicative of a selective sweep; therefore, we further accessed windows having low ZHp scores. Thus, we located several convincing loci that are potentially implicated with the selection applied on each breed. For example, the Claudin-1 (<em>CLDN1</em>) gene is located on chromosome 13 (Position: 136,971,662–136,987,446 bp), and windows including the gene (bin#: 1,815 and 1,826) exhibited a putative signature of selection in both LAN (ZHp = −1.81 and −2.96) and YOR (ZHp = −2.64 and −2.53) pigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5 and S6</a>). The <em>CLDN1</em> genomic region was further examined by accessing pooled heterozygosity (Hp) calculated from each allele of the 142 SNPs that were identified in the region (Fig. <a reveal-id="DSV011F3" data-open="DSV011F3" class="link link-reveal link-table xref-fig">3</a>c). Given that Hp scores of 0 and 0.5 indicate complete homozygosity and heterozygosity, respectively, three breeds other than the LAN and YOR showed distinctively lower homozygosity at the <em>CLDN1</em>. The <em>CLDN1</em> is a dominant tight junction protein that plays a role in invasion activity during metastasis, and its biological functions are known to maintain pregnancy and facilitate conceptus attachment.<sup><a reveal-id="DSV011C36" data-open="DSV011C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSV011C37" data-open="DSV011C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Note that, of the five breeds in this study, both LAN and YOR are outstanding maternal line breeds, with superior reproduction traits. In addition, a recent report showed that the Claudin family members are expressed more abundantly in Yorkshire pigs, as assessed using the porcine Affymetrix-Chip, suggesting that it may be implicated with successful conceptus attachment in this breed.<sup><a reveal-id="DSV011C38" data-open="DSV011C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p>                        <a id="4942310" scrollto-destination="4942310"></a>
<div id="DSV011F3" data-id="DSV011F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01103.png?Expires=2147483647&Signature=o2RE6fekTr27~ioAd9vaDlR9jj-g2qOl~ADMz6Kj2I~moZIAsItaJM-yD7QA9TtLfABQ9F1ySm3IiodHWcgEh-Vw9Hcrj0H5pxab~O8EoSSsSqrve6AdlAjb6Je6amuQvqbvsPtVBksy1U125cleCPkwvEUUYOMxkibrqQxFVQIVxLT8i4V7e4F77vydlSeMo53fRgEHoVV~32HG4nTl2pl3E3zVPyKMrCuurwdFnq9AmRhvexU9TFD~pQpOHTAOoJGSzDBmtxyOaigSUgHDt5X0gq9LHmLSd~n32MaflMmsxpMY6VLmxXb6zqKtLc9~KxGURaBFlw3i1ExRUQYirQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z-scores of pooled heterozygosity (ZHp) overlapping the CLDN1 gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the CLDN1 gene. (a) ZHp scores at the region of 136–139 Mb on Swine chromosome 13. Each point represents a ZHp score estimated from a 150 kb window. Five different shapes of the points represent each breed, respectively (see the legend for details). (b) The structure of RefSeq genes included in the 136–139 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the CLDN1. Colour gradation was used to show the degree of homozygosity: red (Hp = 0) and green (Hp = 0.5) colours represent the complete homozygosity and heterozygosity, respectively. Namely, high level of homozygosity represents the deepest red colour, while high level of heterozygosity increases green colour intensity. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942310/dsv01103.tiff" path-from-xml="dsv01103.tiff" target="_blank">View large</a><a section="4942310" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01103.png?Expires=2147483647&Signature=BCFfbS6dAI84N48Vfw2jT12-PRAGpS~lfkzT3GYTguKVCUBYmMXq3Z2lsqjpQmNDyiSuRExuC0CHyfaUFXssscS3soYAac8nr1v6~WrfEuh-GPcsfm8OEwBNlD9UQ2zB7qY-ohbY1YpIqAHFmmAdmVSgczMVrr8figYoE6D13HGvLs2ClfxNrXK~-pYrUlRKZmedUiXcDCy5VUPHHGxssGa7tf0d1lFGoyr7WdLnV7mb7NBn8iMRv0JoY0otCLY6sYeA2eUtzA11k7GRorf1iImHFHXn5jWTCKl9tSaCZR7pKd1Xvr7KHQaHUPbF-Ta7-ixnMBv4z~Jnz8ifQ5nf7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942310&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Z</em>-scores of pooled heterozygosity (ZHp) overlapping the <em>CLDN1</em> gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the <em>CLDN1</em> gene. (a) ZHp scores at the region of 136–139 Mb on Swine chromosome 13. Each point represents a ZHp score estimated from a 150 kb window. Five different shapes of the points represent each breed, respectively (see the legend for details). (b) The structure of RefSeq genes included in the 136–139 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the <em>CLDN1</em>. Colour gradation was used to show the degree of homozygosity: red (Hp = 0) and green (Hp = 0.5) colours represent the complete homozygosity and heterozygosity, respectively. Namely, high level of homozygosity represents the deepest red colour, while high level of heterozygosity increases green colour intensity. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV011F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01103.png?Expires=2147483647&Signature=o2RE6fekTr27~ioAd9vaDlR9jj-g2qOl~ADMz6Kj2I~moZIAsItaJM-yD7QA9TtLfABQ9F1ySm3IiodHWcgEh-Vw9Hcrj0H5pxab~O8EoSSsSqrve6AdlAjb6Je6amuQvqbvsPtVBksy1U125cleCPkwvEUUYOMxkibrqQxFVQIVxLT8i4V7e4F77vydlSeMo53fRgEHoVV~32HG4nTl2pl3E3zVPyKMrCuurwdFnq9AmRhvexU9TFD~pQpOHTAOoJGSzDBmtxyOaigSUgHDt5X0gq9LHmLSd~n32MaflMmsxpMY6VLmxXb6zqKtLc9~KxGURaBFlw3i1ExRUQYirQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z-scores of pooled heterozygosity (ZHp) overlapping the CLDN1 gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the CLDN1 gene. (a) ZHp scores at the region of 136–139 Mb on Swine chromosome 13. Each point represents a ZHp score estimated from a 150 kb window. Five different shapes of the points represent each breed, respectively (see the legend for details). (b) The structure of RefSeq genes included in the 136–139 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the CLDN1. Colour gradation was used to show the degree of homozygosity: red (Hp = 0) and green (Hp = 0.5) colours represent the complete homozygosity and heterozygosity, respectively. Namely, high level of homozygosity represents the deepest red colour, while high level of heterozygosity increases green colour intensity. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942310/dsv01103.tiff" path-from-xml="dsv01103.tiff" target="_blank">View large</a><a section="4942310" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01103.png?Expires=2147483647&Signature=BCFfbS6dAI84N48Vfw2jT12-PRAGpS~lfkzT3GYTguKVCUBYmMXq3Z2lsqjpQmNDyiSuRExuC0CHyfaUFXssscS3soYAac8nr1v6~WrfEuh-GPcsfm8OEwBNlD9UQ2zB7qY-ohbY1YpIqAHFmmAdmVSgczMVrr8figYoE6D13HGvLs2ClfxNrXK~-pYrUlRKZmedUiXcDCy5VUPHHGxssGa7tf0d1lFGoyr7WdLnV7mb7NBn8iMRv0JoY0otCLY6sYeA2eUtzA11k7GRorf1iImHFHXn5jWTCKl9tSaCZR7pKd1Xvr7KHQaHUPbF-Ta7-ixnMBv4z~Jnz8ifQ5nf7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942310&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Z</em>-scores of pooled heterozygosity (ZHp) overlapping the <em>CLDN1</em> gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the <em>CLDN1</em> gene. (a) ZHp scores at the region of 136–139 Mb on Swine chromosome 13. Each point represents a ZHp score estimated from a 150 kb window. Five different shapes of the points represent each breed, respectively (see the legend for details). (b) The structure of RefSeq genes included in the 136–139 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the <em>CLDN1</em>. Colour gradation was used to show the degree of homozygosity: red (Hp = 0) and green (Hp = 0.5) colours represent the complete homozygosity and heterozygosity, respectively. Namely, high level of homozygosity represents the deepest red colour, while high level of heterozygosity increases green colour intensity. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>As another example, we found a signature of selection for a locus on Chromosome 9 (window #1296), which includes the twist family bHLH transcription factor 1 gene (<em>TWIST1</em>). The genomic region showed a distinctly lower ZHp of −3.02 in KNP (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>) compared with the other four breed populations (0.78, −1.17, 0.85, and 0.53 for KWB, DUR, LAN, and YOR, respectively). <em>TWIST1</em> is implicated in cell lineage differentiation and is associated with breast cancer and Saethre–Chotzen syndrome in humans.<sup><a reveal-id="DSV011C39" data-open="DSV011C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSV011C40" data-open="DSV011C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Furthermore, its expression was low in obese subjects and increased after weight loss, suggesting a prospective role in obesity.<sup><a reveal-id="DSV011C41" data-open="DSV011C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSV011C42" data-open="DSV011C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Despite its lower growth rate and litter size, KNP is known to have the higher intramuscular fat content (marbling) than typical imported breeds. The marbling is an important palatability factor appealing to many Korean pork consumers; however, limited numbers of genes have been identified to account for genetic variations in the trait in KNP. These examples show the potential of our analysis to identify candidate genes to dissect genetic mechanisms implicated in economically important traits in diverse pig breeds (Fig. <a reveal-id="DSV011F4" data-open="DSV011F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="4942312" scrollto-destination="4942312"></a>
<div id="DSV011F4" data-id="DSV011F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01104.png?Expires=2147483647&Signature=YUHfKy26mIVKwF1V51ymK3JNs4Adzmmvo3pVXkXPO77xQGZzvyCEaEZL6ZaBeHF~E7lBnyT~D0NMivLIm1n0R2BsTpgEfjHap~ZHp1PmjyTR97oQwOToLf2Tqu8Hf-ZQfqC2vSlQmsqpq1uKL-AHUxj415hOemYHrnZ-4XNJB9V1uGpr~4ff6Lj-8Z3MzEGQAcfstNrEScw2w6v6Sl8dm0p5RZZMxrIF5Wp~x6ApaZBbI5LngR2Cih5jnWBkDIThqFWQkgSMMHwKP7Wxe8TB-WlrthDavfacrPjRcQdb~WwFp39DyqvjyQFtD3LL8gp7xcp6u00MDrE6oCmT8g0fgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z-scores of pooled heterozygosity (ZHp) overlapping the TWIST1 gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the TWIST1 gene. (a) ZHp scores at the region of 96–99 Mb on Swine chromosome 9. Each point represents a ZHp score estimated from a 150 kb window. (b) The structure of Ensembl and RefSeq genes included in the 96–99 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the TWIST1. The presented region includes 5 kb flanking regions at both sides of the TWIST1 gene. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942312/dsv01104.tiff" path-from-xml="dsv01104.tiff" target="_blank">View large</a><a section="4942312" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01104.png?Expires=2147483647&Signature=tCok1un-~BW1R2ZSylc475igzicMqVkEPc5fVUUxCfpFb69WueS-R8F1ugM49JSn06Rkcapf6rs6XjaIZAhDLvI8eJMj~m4rHfqjZnpys8BW~nyLkZln-CQrqR2zbcsJq3ebTlwzIXuMiZpapGz60fm8VsWbK5GajfJckMQAPPc7vhboPgcK0fullhF-bSmy3JDEZZIyAgHj0YiKmmqqrHq0zhb2vRB6plkQABXb5UgQ7CmFE5r3dmHoMauj4y3PGEy9PtRrga0f5MvTdL3u2xB5LFP9MH14RzMEJRPxNfdMn7vL1OKP67G7YKmEo3~f9~TIeIxcOAhzS2L4Q4S9-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942312&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Z</em>-scores of pooled heterozygosity (ZHp) overlapping the <em>TWIST1</em> gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the <em>TWIST1</em> gene. (a) ZHp scores at the region of 96–99 Mb on Swine chromosome 9. Each point represents a ZHp score estimated from a 150 kb window. (b) The structure of Ensembl and RefSeq genes included in the 96–99 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the <em>TWIST1</em>. The presented region includes 5 kb flanking regions at both sides of the <em>TWIST1</em> gene. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV011F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/m_dsv01104.png?Expires=2147483647&Signature=YUHfKy26mIVKwF1V51ymK3JNs4Adzmmvo3pVXkXPO77xQGZzvyCEaEZL6ZaBeHF~E7lBnyT~D0NMivLIm1n0R2BsTpgEfjHap~ZHp1PmjyTR97oQwOToLf2Tqu8Hf-ZQfqC2vSlQmsqpq1uKL-AHUxj415hOemYHrnZ-4XNJB9V1uGpr~4ff6Lj-8Z3MzEGQAcfstNrEScw2w6v6Sl8dm0p5RZZMxrIF5Wp~x6ApaZBbI5LngR2Cih5jnWBkDIThqFWQkgSMMHwKP7Wxe8TB-WlrthDavfacrPjRcQdb~WwFp39DyqvjyQFtD3LL8gp7xcp6u00MDrE6oCmT8g0fgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z-scores of pooled heterozygosity (ZHp) overlapping the TWIST1 gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the TWIST1 gene. (a) ZHp scores at the region of 96–99 Mb on Swine chromosome 9. Each point represents a ZHp score estimated from a 150 kb window. (b) The structure of Ensembl and RefSeq genes included in the 96–99 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the TWIST1. The presented region includes 5 kb flanking regions at both sides of the TWIST1 gene. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942312/dsv01104.tiff" path-from-xml="dsv01104.tiff" target="_blank">View large</a><a section="4942312" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv01104.png?Expires=2147483647&Signature=tCok1un-~BW1R2ZSylc475igzicMqVkEPc5fVUUxCfpFb69WueS-R8F1ugM49JSn06Rkcapf6rs6XjaIZAhDLvI8eJMj~m4rHfqjZnpys8BW~nyLkZln-CQrqR2zbcsJq3ebTlwzIXuMiZpapGz60fm8VsWbK5GajfJckMQAPPc7vhboPgcK0fullhF-bSmy3JDEZZIyAgHj0YiKmmqqrHq0zhb2vRB6plkQABXb5UgQ7CmFE5r3dmHoMauj4y3PGEy9PtRrga0f5MvTdL3u2xB5LFP9MH14RzMEJRPxNfdMn7vL1OKP67G7YKmEo3~f9~TIeIxcOAhzS2L4Q4S9-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942312&ar=348246&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Z</em>-scores of pooled heterozygosity (ZHp) overlapping the <em>TWIST1</em> gene region. The highlighted orange-coloured box in the dotted line indicates genomic regions including the <em>TWIST1</em> gene. (a) ZHp scores at the region of 96–99 Mb on Swine chromosome 9. Each point represents a ZHp score estimated from a 150 kb window. (b) The structure of Ensembl and RefSeq genes included in the 96–99 Mb region, which were presented by the UCSC genome browser. (c) Hp score of each SNP detected in the <em>TWIST1</em>. The presented region includes 5 kb flanking regions at both sides of the <em>TWIST1</em> gene. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=4942313 id="4942313" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>Here, we present extensive whole-genome resequencing analyses for 55 pigs of five breeds: Korean wild boar, Korean native, Duroc, Landrace, and Yorkshire. This study had two main findings. First, we identified a significant number of SNPs across the genomes (∼20.1 million), of which substantial numbers were novel. Furthermore, we located numerous nsSNPs using a deep annotation process, which could be candidate genetic markers for predicting genetic variations in traits of interest. Second, multiple methodologies were applied to the detected SNPs to dissect genomic features of the pig breeds, revealing obvious signals of genetic differentiation among the breeds. Furthermore, the whole genomes were scanned to detect signatures of selection by accessing excess homozygosity throughout the genome. We observed many higher homozygosity regions, which are unevenly distributed across the genome. The results allowed us to retrieve several interesting genomic regions that may be associated with economically important traits in pigs.</p>                            <h2 scrollto-destination=4942315 id="4942315" class="section-title" data-legacy-id=s5>Authors’ contribution</h2>
<p>J.W.C., W.H.C., and K.T.L. wrote the article. N.K. and T.H.K. conceived and designed the experiments. J.W.C., W.H.C., K.T.L., S.H.L., W.L., D.L., Y.G.L., J.K., N.K., and T.H.K. carried out the analysis of the genome sequences. K.T.L., E.S.C., and T.H.K. performed the experiments.</p>                            <h2 scrollto-destination=4942317 id="4942317" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a></p>                            <h2 scrollto-destination=4942319 id="4942319" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by 2-7-10 Agenda Research (PJ00670701) from the National Institute of Animal Science; a grant (PJ008068) from the Next-generation BioGreen 21 Program, Rural Development Administration, Republic of Korea; and the 2014 Post-doctoral Fellowship Program of the Rural Development Administration, Republic of Korea. Funding to pay the Open Access publication charges for this article was provided by the National Institute of Animal Science.</p>                            <h2 scrollto-destination=4942321 id="4942321" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV011C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Groenen</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Archibald</div>  <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Uenishi</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Analyses of pig genomes provide insight into porcine demography and evolution</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">393</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20pig%20genomes%20provide%20insight%20into%20porcine%20demography%20and%20evolution&amp;author=M.A.%20Groenen&amp;author=A.L.%20Archibald&amp;author=H.%20Uenishi&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=393-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Giuffra</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kijas</div>  <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Amarger</div>  <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Carlborg</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Jeon</div>  <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Andersson</div>  <div class="given-names">L.</div></div></div> <div class="year">2000</div>, <div class="article-title">The origin of the domestic pig: independent domestication and subsequent introgression</div>, <div class="source">Genetics</div> , <div class="volume">154</div>, <div class="fpage">1785</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20the%20domestic%20pig%3A%20independent%20domestication%20and%20subsequent%20introgression&amp;author=E.%20Giuffra&amp;author=J.M.%20Kijas&amp;author=V.%20Amarger&amp;author=O.%20Carlborg&amp;author=J.T.%20Jeon&amp;author=L.%20Andersson&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1785-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="collab">United States Department of Agriculture (USDA)</div>. <div class="year">2014</div>, <div class="article-title">The Hogs and Pigs report</div>. <div class="comment"><a class="link link-uri" href="http://www.usda.gov/">http://www.usda.gov/</a></div> (<div class="comment">9 September 2014, date last accessed</div>).</div></div></div></div><div content-id="DSV011C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="collab">Canadian Centre for Swine Improvement (CCSI)</div>. <div class="year">2012</div>, <div class="article-title">Canadian Centre for Swine Improvement Annual Report</div>. <div class="comment"><a class="link link-uri" href="https://www.ccsi.ca/">https://www.ccsi.ca/</a></div> (<div class="comment">5 September 2014, date last accessed</div>).</div></div></div></div><div content-id="DSV011C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sachs</div>  <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Galli</div>  <div class="given-names">C.</div></div></div> <div class="year">2009</div>, <div class="article-title">Genetic manipulation in pigs</div>, <div class="source">Curr. Opin. Organ Transplant.</div> , <div class="volume">14</div>, <div class="fpage">148</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20manipulation%20in%20pigs&amp;author=D.H.%20Sachs&amp;author=C.%20Galli&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Organ%20Transplant.&amp;volume=14&amp;pages=148-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/MOT.0b013e3283292549" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2FMOT.0b013e3283292549" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2FMOT.0b013e3283292549"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19469029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vodicka</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Smetana</div>  <div class="given-names">K.</div>Jr.</div>, <div class="name string-name"><div class="surname">Dvorankova</div>  <div class="given-names">B.</div></div> et al.  </div>. <div class="year">2005</div>, <div class="article-title">The miniature pig as an animal model in biomedical research</div>, <div class="source">Ann. NY Acad. Sci.</div> , <div class="volume">10,49</div>, <div class="fpage">161</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20miniature%20pig%20as%20an%20animal%20model%20in%20biomedical%20research&amp;author=P.%20Vodicka&amp;author=K.%20Smetana&amp;author=B.%20Dvorankova&amp;publication_year=2005&amp;journal=Ann.%20NY%20Acad.%20Sci.&amp;volume=10,49&amp;pages=161-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1196/annals.1334.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1196%2Fannals.1334.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1196%2Fannals.1334.015"> </span></div> </div></div></div></div></div><div content-id="DSV011C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="collab">Food and Agriculture Organization (FAO)</div>. <div class="year">2014</div>, <div class="article-title">Domestic Animal Diversity Information Service (DAD-IS)</div>. <div class="comment"><a class="link link-uri" href="http://dad.fao.org/">http://dad.fao.org/</a></div> (<div class="comment">5 September 2014, date last accessed</div>).</div></div></div></div><div content-id="DSV011C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Hong</div>  <div class="given-names">J.K.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Comparison of carcass characteristics, meat quality, amino acids contents, and fatty acid profiles of Korea native pig by gender</div>, <div class="source">Rep. Dev. Biol.</div> , <div class="volume">37</div>, <div class="fpage">129</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20carcass%20characteristics%2C%20meat%20quality%2C%20amino%20acids%20contents%2C%20and%20fatty%20acid%20profiles%20of%20Korea%20native%20pig%20by%20gender&amp;author=D.W.%20Kim&amp;author=K.H.%20Kim&amp;author=J.K.%20Hong&amp;publication_year=2013&amp;journal=Rep.%20Dev.%20Biol.&amp;volume=37&amp;pages=129-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.12749/RDB." target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.12749%2FRDB." target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.12749%2FRDB."> </span></div> </div></div></div></div></div><div content-id="DSV011C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cho</div>  <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Park</div>  <div class="given-names">B.Y.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">J.H.</div></div> et al.  </div>. <div class="year">2007</div>, <div class="article-title">Carcass yields and meat quality by live weight of Korean native black pigs</div>, <div class="source">J. Anim. Sci. Technol.</div> , <div class="volume">49</div>, <div class="fpage">523</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carcass%20yields%20and%20meat%20quality%20by%20live%20weight%20of%20Korean%20native%20black%20pigs&amp;author=S.H.%20Cho&amp;author=B.Y.%20Park&amp;author=J.H.%20Kim&amp;publication_year=2007&amp;journal=J.%20Anim.%20Sci.%20Technol.&amp;volume=49&amp;pages=523-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5187/JAST.2007.49.4.523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5187%2FJAST.2007.49.4.523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5187%2FJAST.2007.49.4.523"> </span></div> </div></div></div></div></div><div content-id="DSV011C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div>  <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Byun</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">Y.S.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">National Management System for conservation of livestock genetic resources: an overview</div>, <div class="source">Ann. Anim. Resour. Sci.</div> , <div class="volume">23</div>, <div class="fpage">142</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=National%20Management%20System%20for%20conservation%20of%20livestock%20genetic%20resources%3A%20an%20overview&amp;author=S.B.%20Choi&amp;author=M.J.%20Byun&amp;author=Y.S.%20Kim&amp;publication_year=2012&amp;journal=Ann.%20Anim.%20Resour.%20Sci.&amp;volume=23&amp;pages=142-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.12718/AARS" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.12718%2FAARS" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.12718%2FAARS"> </span></div> </div></div></div></div></div><div content-id="DSV011C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wee</div>  <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Ko</div>  <div class="given-names">Y.G.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Molecular genetic evaluation of Korean native pig populations based on microsatellite markers</div>, <div class="source">J. Anim. Sci. Technol.</div> , <div class="volume">53</div>, <div class="fpage">35</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20genetic%20evaluation%20of%20Korean%20native%20pig%20populations%20based%20on%20microsatellite%20markers&amp;author=P.%20Lee&amp;author=M.S.%20Wee&amp;author=Y.G.%20Ko&amp;publication_year=2011&amp;journal=J.%20Anim.%20Sci.%20Technol.&amp;volume=53&amp;pages=35-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5187/JAST.2011.53.1.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5187%2FJAST.2011.53.1.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5187%2FJAST.2011.53.1.35"> </span></div> </div></div></div></div></div><div content-id="DSV011C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">T.H.</div></div>, <div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Choi</div>  <div class="given-names">B.H.</div></div> et al.  </div>. <div class="year">2005</div>, <div class="article-title">Genetic structure of pig breeds from Korea and China using microsatellite loci analysis</div>, <div class="source">J. Anim. Sci.</div> , <div class="volume">83</div>, <div class="fpage">2255</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20of%20pig%20breeds%20from%20Korea%20and%20China%20using%20microsatellite%20loci%20analysis&amp;author=T.H.%20Kim&amp;author=K.S.%20Kim&amp;author=B.H.%20Choi&amp;publication_year=2005&amp;journal=J.%20Anim.%20Sci.&amp;volume=83&amp;pages=2255-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/2005.83102255x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2F2005.83102255x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2F2005.83102255x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16160034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rubin</div>  <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Megens</div>  <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Martinez Barrio</div>  <div class="given-names">A.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Strong signatures of selection in the domestic pig genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">19529</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20signatures%20of%20selection%20in%20the%20domestic%20pig%20genome&amp;author=C.J.%20Rubin&amp;author=H.J.%20Megens&amp;author=A.%20Martinez%20Barrio&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=19529-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217149109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217149109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217149109"> </span></div> </div></div></div></div></div><div content-id="DSV011C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tian</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yeung</div>  <div class="given-names">C.K.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Whole-genome sequencing of Berkshire (European native pig) provides insights into its origin and domestication</div>, <div class="source">Sci. Rep.</div> , <div class="volume">4</div>, <div class="fpage">4678</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20Berkshire%20(European%20native%20pig)%20provides%20insights%20into%20its%20origin%20and%20domestication&amp;author=M.%20Li&amp;author=S.%20Tian&amp;author=C.K.%20Yeung&amp;publication_year=2014&amp;journal=Sci.%20Rep.&amp;volume=4&amp;pages=4678" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24728479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div>  <div class="given-names">R.</div></div></div> <div class="year">2010</div>, <div class="article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McKenna</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hanna</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Banks</div>  <div class="given-names">E.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cingolani</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Platts</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang le</div>  <div class="given-names">L.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</div>, <div class="source">Fly</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20Wang%20le&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cingolani</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Patel</div>  <div class="given-names">V.M.</div></div>, <div class="name string-name"><div class="surname">Coon</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Using <em>Drosophila melanogaster</em> as a model for genotoxic chemical mutational studies with a new program, SnpSift</div>, <div class="source">Front. Genet.</div> , <div class="volume">3</div>, <div class="fpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20Drosophila%20melanogaster%20as%20a%20model%20for%20genotoxic%20chemical%20mutational%20studies%20with%20a%20new%20program%2C%20SnpSift&amp;author=P.%20Cingolani&amp;author=V.M.%20Patel&amp;author=M.%20Coon&amp;publication_year=2012&amp;journal=Front.%20Genet.&amp;volume=3&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2012.00035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2012.00035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2012.00035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22435069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Browning</div>  <div class="given-names">B.L.</div></div>, <div class="name string-name"><div class="surname">Browning</div>  <div class="given-names">S.R.</div></div></div> <div class="year">2009</div>, <div class="article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">84</div>, <div class="fpage">210</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20approach%20to%20genotype%20imputation%20and%20haplotype-phase%20inference%20for%20large%20data%20sets%20of%20trios%20and%20unrelated%20individuals&amp;author=B.L.%20Browning&amp;author=S.R.%20Browning&amp;publication_year=2009&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=84&amp;pages=210-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajhg.2009.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2009.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajhg.2009.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19200528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paradis</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Claude</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Strimmer</div>  <div class="given-names">K.</div></div></div> <div class="year">2004</div>, <div class="article-title">APE: analyses of phylogenetics and evolution in R language</div>, <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">289</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=APE%3A%20analyses%20of%20phylogenetics%20and%20evolution%20in%20R%20language&amp;author=E.%20Paradis&amp;author=J.%20Claude&amp;author=K.%20Strimmer&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=289-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg412" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg412" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg412"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14734327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Peterson</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Peterson</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Stecher</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nei</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">S.</div></div></div> <div class="year">2011</div>, <div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">28</div>, <div class="fpage">2731</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hubisz</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Falush</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stephens</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div>  <div class="given-names">J.K.</div></div></div> <div class="year">2009</div>, <div class="article-title">Inferring weak population structure with the assistance of sample group information</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="volume">9</div>, <div class="fpage">1322</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information&amp;author=M.J.%20Hubisz&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2009&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=9&amp;pages=1322-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2009.9.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2009.9.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2009.9.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21564903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Falush</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stephens</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div>  <div class="given-names">J.K.</div></div></div> <div class="year">2003</div>, <div class="article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</div>, <div class="source">Genetics</div> , <div class="volume">164</div>, <div class="fpage">1567</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1567-87" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rosenberg</div>  <div class="given-names">N.A.</div></div></div> <div class="year">2004</div>, <div class="article-title">DISTRUCT: a program for the graphical display of population structure</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="volume">4</div>, <div class="fpage">137</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DISTRUCT%3A%20a%20program%20for%20the%20graphical%20display%20of%20population%20structure&amp;author=N.A.%20Rosenberg&amp;publication_year=2004&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=4&amp;pages=137-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00566.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00566.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00566.x"> </span></div> </div></div></div></div></div><div content-id="DSV011C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rubin</div>  <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Zody</div>  <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Eriksson</div>  <div class="given-names">J.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Whole-genome resequencing reveals loci under selection during chicken domestication</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">587</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20resequencing%20reveals%20loci%20under%20selection%20during%20chicken%20domestication&amp;author=C.J.%20Rubin&amp;author=M.C.%20Zody&amp;author=J.%20Eriksson&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=587-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20220755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div>  <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Liao</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Stothard</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Whole-genome analyses of Korean native and Holstein cattle breeds by massively parallel sequencing</div>, <div class="source">PloS One</div> , <div class="volume">9</div>, <div class="fpage">e101127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20analyses%20of%20Korean%20native%20and%20Holstein%20cattle%20breeds%20by%20massively%20parallel%20sequencing&amp;author=J.W.%20Choi&amp;author=X.%20Liao&amp;author=P.%20Stothard&amp;publication_year=2014&amp;journal=PloS%20One&amp;volume=9&amp;pages=e101127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0101127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0101127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0101127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24992012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="collab">Genomes Project Consortium</div><div class="person-group"><div class="name string-name"><div class="surname">Abecasis</div>  <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Auton</div>  <div class="given-names">A.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">An integrated map of genetic variation from 1,092 human genomes</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">56</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20map%20of%20genetic%20variation%20from%201%2C092%20human%20genomes&amp;author=G.R.%20Abecasis&amp;author=A.%20Auton&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=56-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23128226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div>  <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Liao</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Park</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Massively parallel sequencing of Chikso (Korean brindle cattle) to discover genome-wide SNPs and InDels</div>, <div class="source">Mol. Cells</div> , <div class="volume">36</div>, <div class="fpage">203</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massively%20parallel%20sequencing%20of%20Chikso%20(Korean%20brindle%20cattle)%20to%20discover%20genome-wide%20SNPs%20and%20InDels&amp;author=J.W.%20Choi&amp;author=X.%20Liao&amp;author=S.%20Park&amp;publication_year=2013&amp;journal=Mol.%20Cells&amp;volume=36&amp;pages=203-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10059-013-2347-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10059-013-2347-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10059-013-2347-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23912596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ng</div>  <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Henikoff</div>  <div class="given-names">S.</div></div></div> <div class="year">2003</div>, <div class="article-title">SIFT: predicting amino acid changes that affect protein function</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">31</div>, <div class="fpage">3812</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SIFT%3A%20predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&amp;author=P.C.%20Ng&amp;author=S.%20Henikoff&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3812-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gebre-Medhin</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Betsholtz</div>  <div class="given-names">C.</div></div> et al.  </div>. <div class="year">1998</div>, <div class="article-title">Targeted disruption of the mouse phospholipase C beta3 gene results in early embryonic lethality</div>, <div class="source">FEBS Lett.</div> , <div class="volume">441</div>, <div class="fpage">261</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20disruption%20of%20the%20mouse%20phospholipase%20C%20beta3%20gene%20results%20in%20early%20embryonic%20lethality&amp;author=S.%20Wang&amp;author=S.%20Gebre-Medhin&amp;author=C.%20Betsholtz&amp;publication_year=1998&amp;journal=FEBS%20Lett.&amp;volume=441&amp;pages=261-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(98)01518-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(98)01518-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(98)01518-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9883896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dupont</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">McGuinness</div>  <div class="given-names">O.M.</div></div>, <div class="name string-name"><div class="surname">Johnson</div>  <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Berridge</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Borgese</div>  <div class="given-names">F.</div></div></div> <div class="year">1996</div>, <div class="article-title">Phospholipase C in mouse oocytes: characterization of beta and gamma isoforms and their possible involvement in sperm-induced Ca2+ spiking</div>, <div class="source">Biochem. J.</div> , <div class="volume">316</div> <div class="issue">(Pt 2)</div>, <div class="fpage">583</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phospholipase%20C%20in%20mouse%20oocytes%3A%20characterization%20of%20beta%20and%20gamma%20isoforms%20and%20their%20possible%20involvement%20in%20sperm-induced%20Ca2%2B%20spiking&amp;author=G.%20Dupont&amp;author=O.M.%20McGuinness&amp;author=M.H.%20Johnson&amp;author=M.J.%20Berridge&amp;author=F.%20Borgese&amp;publication_year=1996&amp;journal=Biochem.%20J.&amp;volume=316&amp;pages=583-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj3160583" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj3160583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj3160583"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8687404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miyazaki</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shirakawa</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nakada</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Honda</div>  <div class="given-names">Y.</div></div></div> <div class="year">1993</div>, <div class="article-title">Essential role of the inositol 1,4,5-trisphosphate receptor/Ca2+ release channel in Ca2+ waves and Ca2+ oscillations at fertilization of mammalian eggs</div>, <div class="source">Dev. Biol.</div> , <div class="volume">158</div>, <div class="fpage">62</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essential%20role%20of%20the%20inositol%201%2C4%2C5-trisphosphate%20receptor%2FCa2%2B%20release%20channel%20in%20Ca2%2B%20waves%20and%20Ca2%2B%20oscillations%20at%20fertilization%20of%20mammalian%20eggs&amp;author=S.%20Miyazaki&amp;author=H.%20Shirakawa&amp;author=K.%20Nakada&amp;author=Y.%20Honda&amp;publication_year=1993&amp;journal=Dev.%20Biol.&amp;volume=158&amp;pages=62-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/dbio.1993.1168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fdbio.1993.1168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fdbio.1993.1168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8392472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nikkila</div>  <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Stalder</div>  <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Mote</div>  <div class="given-names">B.E.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genetic associations for gilt growth, compositional, and structural soundness traits with sow longevity and lifetime reproductive performance</div>, <div class="source">J. Anim. Sci.</div> , <div class="volume">91</div>, <div class="fpage">1570</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20associations%20for%20gilt%20growth%2C%20compositional%2C%20and%20structural%20soundness%20traits%20with%20sow%20longevity%20and%20lifetime%20reproductive%20performance&amp;author=M.T.%20Nikkila&amp;author=K.J.%20Stalder&amp;author=B.E.%20Mote&amp;publication_year=2013&amp;journal=J.%20Anim.%20Sci.&amp;volume=91&amp;pages=1570-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/jas.2012-5723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2Fjas.2012-5723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2Fjas.2012-5723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23449861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Serenius</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Stalder</div>  <div class="given-names">K.J.</div></div></div> <div class="year">2004</div>, <div class="article-title">Genetics of length of productive life and lifetime prolificacy in the Finnish Landrace and Large White pig populations</div>, <div class="source">J. Anim. Sci.</div> , <div class="volume">82</div>, <div class="fpage">3111</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20length%20of%20productive%20life%20and%20lifetime%20prolificacy%20in%20the%20Finnish%20Landrace%20and%20Large%20White%20pig%20populations&amp;author=T.%20Serenius&amp;author=K.J.%20Stalder&amp;publication_year=2004&amp;journal=J.%20Anim.%20Sci.&amp;volume=82&amp;pages=3111-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/2004.82113111x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2F2004.82113111x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2F2004.82113111x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15542456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Missler</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sudhof</div>  <div class="given-names">T.C.</div></div></div> <div class="year">1998</div>, <div class="article-title">Neurexophilins form a conserved family of neuropeptide-like glycoproteins</div>, <div class="source">J. Neurosci.</div> , <div class="volume">18</div>, <div class="fpage">3630</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neurexophilins%20form%20a%20conserved%20family%20of%20neuropeptide-like%20glycoproteins&amp;author=M.%20Missler&amp;author=T.C.%20Sudhof&amp;publication_year=1998&amp;journal=J.%20Neurosci.&amp;volume=18&amp;pages=3630-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9570794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brandley</div>  <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Young</div>  <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Warren</div>  <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Thompson</div>  <div class="given-names">M.B.</div></div>, <div class="name string-name"><div class="surname">Wagner</div>  <div class="given-names">G.P.</div></div></div> <div class="year">2012</div>, <div class="article-title">Uterine gene expression in the live-bearing lizard, Chalcides ocellatus, reveals convergence of squamate reptile and mammalian pregnancy mechanisms</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">4</div>, <div class="fpage">394</div>–<div class="lpage">411</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uterine%20gene%20expression%20in%20the%20live-bearing%20lizard%2C%20Chalcides%20ocellatus%2C%20reveals%20convergence%20of%20squamate%20reptile%20and%20mammalian%20pregnancy%20mechanisms&amp;author=M.C.%20Brandley&amp;author=R.L.%20Young&amp;author=D.L.%20Warren&amp;author=M.B.%20Thompson&amp;author=G.P.%20Wagner&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol.&amp;volume=4&amp;pages=394-411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22333490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhao</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Das</div>  <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Paria</div>  <div class="given-names">B.C.</div></div></div> <div class="year">2004</div>, <div class="article-title">Embryonic signals direct the formation of tight junctional permeability barrier in the decidualizing stroma during embryo implantation</div>, <div class="source">J. Cell Sci.</div> , <div class="volume">117</div>, <div class="fpage">53</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Embryonic%20signals%20direct%20the%20formation%20of%20tight%20junctional%20permeability%20barrier%20in%20the%20decidualizing%20stroma%20during%20embryo%20implantation&amp;author=X.%20Wang&amp;author=H.%20Matsumoto&amp;author=X.%20Zhao&amp;author=S.K.%20Das&amp;author=B.C.%20Paria&amp;publication_year=2004&amp;journal=J.%20Cell%20Sci.&amp;volume=117&amp;pages=53-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.00826" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.00826" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.00826"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14627626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gu</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Schroyen</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Endometrial gene expression profiling in pregnant Meishan and Yorkshire pigs on day 12 of gestation</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">156</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endometrial%20gene%20expression%20profiling%20in%20pregnant%20Meishan%20and%20Yorkshire%20pigs%20on%20day%2012%20of%20gestation&amp;author=T.%20Gu&amp;author=M.J.%20Zhu&amp;author=M.%20Schroyen&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24564230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kress</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Schropp</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lieb</div>  <div class="given-names">G.</div></div> et al.  </div>. <div class="year">2006</div>, <div class="article-title">Saethre-Chotzen syndrome caused by TWIST 1 gene mutations: functional differentiation from Muenke coronal synostosis syndrome</div>, <div class="source">Eur. J. Hum. Genet.</div> , <div class="volume">14</div>, <div class="fpage">39</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saethre-Chotzen%20syndrome%20caused%20by%20TWIST%201%20gene%20mutations%3A%20functional%20differentiation%20from%20Muenke%20coronal%20synostosis%20syndrome&amp;author=W.%20Kress&amp;author=C.%20Schropp&amp;author=G.%20Lieb&amp;publication_year=2006&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;volume=14&amp;pages=39-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16251895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div>  <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Goyal</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Watkins</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Jiang</div>  <div class="given-names">W.G.</div></div></div> <div class="year">2005</div>, <div class="article-title">Expression of the transcription factors snail, slug, and twist and their clinical significance in human breast cancer</div>, <div class="source">Ann. Surg. Oncol.</div> , <div class="volume">12</div>, <div class="fpage">488</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20transcription%20factors%20snail%2C%20slug%2C%20and%20twist%20and%20their%20clinical%20significance%20in%20human%20breast%20cancer&amp;author=T.A.%20Martin&amp;author=A.%20Goyal&amp;author=G.%20Watkins&amp;author=W.G.%20Jiang&amp;publication_year=2005&amp;journal=Ann.%20Surg.%20Oncol.&amp;volume=12&amp;pages=488-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1245/ASO.2005.04.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1245%2FASO.2005.04.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1245%2FASO.2005.04.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15864483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Lu</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sun</div>  <div class="given-names">T.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Twist 1 regulates the expression of PPARgamma during hormone-induced 3T3-L1 preadipocyte differentiation: a possible role in obesity and associated diseases</div>, <div class="source">Lipids Health Dis.</div> , <div class="volume">13</div>, <div class="fpage">132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twist%201%20regulates%20the%20expression%20of%20PPARgamma%20during%20hormone-induced%203T3-L1%20preadipocyte%20differentiation%3A%20a%20possible%20role%20in%20obesity%20and%20associated%20diseases&amp;author=W.%20Ma&amp;author=S.%20Lu&amp;author=T.%20Sun&amp;publication_year=2014&amp;journal=Lipids%20Health%20Dis.&amp;volume=13&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1476-511X-13-132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1476-511X-13-132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1476-511X-13-132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25128964" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV011C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV011C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pettersson</div>  <div class="given-names">A.T.</div></div>, <div class="name string-name"><div class="surname">Mejhert</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Jernas</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Twist1 in human white adipose tissue and obesity</div>, <div class="source">J. Clin. Endocrinol. Metab.</div> , <div class="volume">96</div>, <div class="fpage">133</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twist1%20in%20human%20white%20adipose%20tissue%20and%20obesity&amp;author=A.T.%20Pettersson&amp;author=N.%20Mejhert&amp;author=M.%20Jernas&amp;publication_year=2011&amp;journal=J.%20Clin.%20Endocrinol.%20Metab.&amp;volume=96&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1210/jc.2010-0929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1210%2Fjc.2010-0929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1210%2Fjc.2010-0929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20943789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348246&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table8.xlsx?Expires=2147483647&amp;Signature=sVK5p3AIQXgfVa8WNutlIJcNYeUeAtp3ZCe73Rxw6iwzUnwrhJmsI-povuXN0BLNmfdf5VD~mtNMX6oQJz26-4milCE2EeQQ8VHdYuf0wLpkWdA1zlSuCbyz3Z11Ewk0urSCQ6LtC2nHNE2oB88NqGY4A84LQMVJKX1vnpIpkD8kS~frD-kQBX47PXfvxmjz50lFP611~FHbWqRPOwGfJeHi5ljmF1NSObHHZkxWtUc~8z3skv0JNpEkIaU~y31w~fA7nYK52p~K5ieTqoa3F19MbsmB3q~ATzgEgwTIPVOk~XwaC0bFF3iYxha3AFev2UXY49GJchLJi6HZ-94VMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table7.xlsx?Expires=2147483647&amp;Signature=3RaYnhweumZs9m9g~9zhOB1lGgIAGeyaFU-SbkiL5YCEjq9LCN7FM~9UWraXacTfy8z5Wnjt~0xXw8YmkKuc9u4P6nGpQx0aSAvanMAQKyIFXBP6b-Ntxfvn0HAiPJ~9Dni-vBKcpTAC1iiFW5JYooiaBctEOCJiIQAqdUx1aBTYRe67Fi3srHcsWXYvQLVkZCQlXMu8W3MkQyZfwpURQpd-gJGe~v9LgPt2Q1qQ8ICdXiwxWVtQXdCf7iI1P9x8g-XX8TZXI6j5b0DngbU4-tRuwq8adKIfbv6-AdWldW1j8OZEpVHRG9VbeueQEpAuju125e2-GLpGv92ijfN9pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table6.xlsx?Expires=2147483647&amp;Signature=ZUl7SG3lEFen2LC1dlu4akzyy9UcbuPQ6MTDCknfA1l9QYifRR0FsXyvX1HjyKKVwgy61SO-wC-R759TKFew2m00XJydZWhPJG7fC7QLq7otkZRjdD6nywT6inmu343pw4~xTTM4jdhfrN-V7d5K18GRL-nG0YqaRwyWw698DJCxYJJWInCI8zCpT9HGYoo5wxfNzWjUZ4t3LK1OZcPa7OGEHIp1Q8B9QEhmUXrRBYzIHs5iwl5c1t6G4xWnae7hdd6UAtIw6YTSB3tq1VVjy~2KJy3iuqEPpoMzW5VgsINNj4cVao5c4vdNBpLrIBMyapiq~b8tPtGYX~7uncVEgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table5.xlsx?Expires=2147483647&amp;Signature=ccnqtESDYQ9iwVcsLBn1VXZmTVpH6AMjp7VW33muTNM9-k1r9kvRtVJ8FiciINnV2LH~mvs05GikVVwiTkpP0dF3Kgjd1ne~RjQTeEvasSTaMfxN8mBO0-jW~0eSQUuj8I84qb3DdN3EIrspQ5UGEeYiMWGThn1csNmDT4RWBDEOAASi8wLaizLWmnjK3HBnX4OD~px0UTxs2nLVy2VVVC7rFhV8edR1s6bddC6FDNNfLwKE6zbt147QdXtvLmruBp6WK14DLYTXRNEtK2QZikI0QT0mBhT4nK8xUuIm-DXqxYRdQNCwOCRXBUNTYv7GaPIE0~ciQ3ygTQIZpmlkHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table4.xlsx?Expires=2147483647&amp;Signature=Zf1fqWs05MHeJS5BzU3t5LaWQR3qUHTezg-t1c~49KsapeP6oIJTZuyRRHE7LRlXOPsBZwyvRS-gseMqQOMAPAIPKOM1755DVraJnMTlUSuJqmX-RFwXKkfGXLppa19JdNWGq-s9vUICeq1uDYuPbxgSvBIYzU~~a~2HBd39ARF8WAsIZiRUrtlSgLEXJEiETJZ2KCufiQ7q2Z5~s5NsmHw1U6fC5GFk0vrXFpp~JHcyItEIYAOcMksRP8YTUls2aM3f5qqri~8tUKskoZaBbYoQpZ3bJW1y6WGXxlceG~WBiooRc53sWfgaRQXB0OR118ZY8aUm-~NuxefQEMdT3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table3.xlsx?Expires=2147483647&amp;Signature=I3M22vYVBsFNh2m179Hy5XqxSpZv9q-JfWQnJXF~tI-Xq2VZC5821DGsxzj~n5LanzCc2yndmCWKU7V1u93lRDQjX52rvljHOAdAFSCp19un9T68963ikV9tDn~7Th58kL2C~mn5Tul5TjNa6vSGff7dg1fapBPk4QK8n3MGqXFYe5wY3CrDsUvzkW4kfF7iwMV2aZVqRMfPomBfZ1hkE6fQGU9kz02Bsi0gQ9LajmT6HBtgP11xtqBHIVo9qTXyAMF7G55hVOlhGk4ZjukrKg51jByhr-SFU8CANSOJhMeUTBUM8d3Nt-oh5PB2PJsWPyXK1aPKMBLDmm9Fi1-aAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table2.xlsx?Expires=2147483647&amp;Signature=uq37CMvO37I7-5bHTlQhmMnKzh~jeXJP9cWxjV-Y0h8kdU1wSzSgvRtPy20fblWY8eAHnxxfu-~jfbAegn6fwGdGbCO4s4bltvIAHha0u-OGZdtgLqRhgKHxdMSv3-xuKIeztGJi9yOAKaVkkCrBDnHjgfrAR-4u6EX1TF1QVY1x6ekW1oo8DaiXpncNq7vxYo17hSkCTZfUy6H0no-8OZD9sPHbj1OWghn2nWFEiZVffEwPH~iN4u3g9YBcm1zwWTJ6vQyQECG7iSyrqIvZNNkHdfJVOyta2VPdTgVq1pDlOaAPgO0caolovAGM6CR6zVaAhW~5M2osvYNIaxZVUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_table1.xlsx?Expires=2147483647&amp;Signature=HLKj2KxmgqK2QWWobXfBkhYLYVgLwI9-sXfb2s6o2zOMmkgx0PUNkgCGTrBGXcB0uomgoRw4xe~n7X2U9Vig1MHB~E6xxu2cpk4ndyVcrlt-oUrSa8PxSLHCZIzJMPVLbsLgsQpQnZFwqV3l7g~5RW0f0gz7KSfYMIbh2NEhvXM42MPgkQv6aHGHyZ2hVVmwmYaI8-xzuzKCNHSeHSy9Rmt5Uqjxj-dryIfI2bgjdJRHMIzt0TG3dxbxo3bmyPbowuYUU9ECKlf5uAqm7TmCFXSyOlgt9ofCL936sWdiRVLRZVbc4rys-Zj4LmrX~nhFSukuLGmwl5QIXY-vf6CUYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv011/2/dsv011-dsv011supp_fig1.ppt?Expires=2147483647&amp;Signature=r0yEszQOrckMf4MxH2DKWXBLggTxy8YgM3dQTE6GL2uvg5Zp8pH7S5RYqD~94-4bGV0Y4V4hJL5A0~Qy1MJJJ0xLjWXMe9aEleHl59MFPzukJfZk6Wo07XWoTtHAbOy0lhjyjz99dqpl-bJpIaNUiWl0RJzamArzl8U6-x2FyFDuly0FdSl2jXjmCyBffSx3bkTLA~q7Xl5NUBm70JVjQPnWEQ~mbMOoUQ2IXRXnhgzQoPcI12yrs2QOIFOUNOsutwqE3bKDeCPYmTuZWyISIzhaRPdxD7lfOOloiYFJ1JqYrPN55Nq~twVZUIsIz2XUXuOnQKZskZiGwkUDQrP0~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">882</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348246"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f4%2f259%2f348246">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362346000002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F4%2F259%2F348246" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348246'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f259%2f348246">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

