

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/421/5020793","@type":"ScholarlyArticle","name":"A survey of transcriptome complexity in  Sus scrofa  using single-molecule long-read sequencing","datePublished":"2018-05-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy014","keywords":["single-molecule sequencing","full-length","novel gene","alternative splicing","methylation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/421/5020793","author":[{"name":"Li, Yao","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Fang, Chengchi","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Fu, Yuhua","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Hu, An","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Li, Cencen","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Zou, Cheng","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Li, Xinyun","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Zhao, Shuhong","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"},{"name":"Zhang, Chengjun","affiliation":"Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China","@type":"Person"},{"name":"Li, Changchun","affiliation":"Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) and fusion transcripts produce a vast expansion of transcriptomes and proteomes diversity. However, the reliability of thes","pageStart":"421","pageEnd":"437","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A survey of transcriptome complexity in  Sus scrofa  using single-molecule long-read sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="survey of transcriptome complexity in  Sus scrofa  using single-molecule long-read sequencing" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) and fusion transcripts produce a vast expansion of transcriptomes and proteomes diversity. However, the reliability of thes" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/421/5020793" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-05-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Yao" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Fang, Chengchi" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Fu, Yuhua" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Hu, An" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Li, Cencen" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Zou, Cheng" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Li, Xinyun" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Zhao, Shuhong" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_author" content="Zhang, Chengjun" /><meta name="citation_author_institution" content="Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China" /><meta name="citation_author" content="Li, Changchun" /><meta name="citation_author_institution" content="Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China" /><meta name="citation_title" content="A survey of transcriptome complexity in Sus scrofa using single-molecule long-read sequencing" /><meta name="citation_firstpage" content="421" /><meta name="citation_lastpage" content="437" /><meta name="citation_doi" content="10.1093/dnares/dsy014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Proceedings of the conference on swine in biomedical research;  citation_journal_title=ILAR J;  citation_year=2009;  citation_volume=50;  citation_pages=89-94; " /><meta name="citation_reference" content="citation_title=Developmental origins of health and disease in swine: implications for animal production and biomedical research;  citation_journal_title=Theriogenology;  citation_year=2016;  citation_volume=86;  citation_pages=110-9; " /><meta name="citation_reference" content="citation_title=Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=259-67; " /><meta name="citation_reference" content="citation_title=Molecular Pap smear: hPV genotype and DNA methylation of ADCY8, CDH8, and ZNF582 as an integrated biomarker for high-grade cervical cytology;  citation_journal_title=Clin. Epigenetics;  citation_year=2016;  citation_volume=8;  citation_pages=96" /><meta name="citation_reference" content="citation_title=Characterization of the human ESC transcriptome by hybrid sequencing;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2013;  citation_volume=110;  citation_pages=E4821-30; " /><meta name="citation_reference" content="citation_title=The Pig Genome Database (PiGenome): an integrated database for pig genome research;  citation_journal_title=Mamm. Genome;  citation_year=2009;  citation_volume=20;  citation_pages=60-6; " /><meta name="citation_reference" content="citation_title=A study of alternative splicing in the pig;  citation_journal_title=BMC Res. Notes;  citation_year=2010;  citation_volume=3;  citation_pages=123" /><meta name="citation_reference" content="citation_title=Alternative isoform regulation in human tissue transcriptomes;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=470-6; " /><meta name="citation_reference" content="citation_title=An amazing sequence arrangement at the 5' ends of adenovirus 2 messenger RNA;  citation_journal_title=Cell;  citation_year=1977;  citation_volume=12;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing;  citation_journal_title=Nat. Genet;  citation_year=2008;  citation_volume=40;  citation_pages=1413-5; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of alternative splicing in cow: implications in bovine as a model for human diseases;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=S11" /><meta name="citation_reference" content="citation_title=A genomic view of alternative splicing;  citation_journal_title=Nat. Genet;  citation_year=2002;  citation_volume=30;  citation_pages=13-9; " /><meta name="citation_reference" content="citation_title=Alternative splicing and cell survival: from tissue homeostasis to disease;  citation_journal_title=Cell Death Differ;  citation_year=2016;  citation_volume=23;  citation_pages=1919-29; " /><meta name="citation_reference" content="citation_title=Antisense oligonucleotide-mediated MDM4 exon 6 skipping impairs tumor growth;  citation_journal_title=J. Clin. Invest;  citation_year=2015;  citation_volume=126;  citation_pages=68-84; " /><meta name="citation_reference" content="citation_title=Survey of programs used to detect alternative splicing isoforms from deep sequencing data in silico;  citation_journal_title=Biomed. Res. Int;  citation_year=2015;  citation_volume=2015;  citation_pages=1" /><meta name="citation_reference" content="citation_title=RNA-mediated epigenetic regulation of gene expression;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=71-84; " /><meta name="citation_reference" content="citation_title=Transcript isoform variation associated with cytosine modification in human lymphoblastoid cell lines;  citation_journal_title=Genetics;  citation_year=2016;  citation_volume=203;  citation_pages=985-95; " /><meta name="citation_reference" content="citation_title=Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing;  citation_journal_title=Nat. Comms;  citation_year=2016;  citation_volume=7;  citation_pages=11708" /><meta name="citation_reference" content="citation_title=Complete telomere-to-telomere de novo assembly of the Plasmodium falciparum genome through long-read (&gt;11 kb), single molecule, real-time sequencing;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=339-51; " /><meta name="citation_reference" content="citation_title=A single-molecule long-read survey of the human transcriptome;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=1009-14; " /><meta name="citation_reference" content="citation_title=Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=736-42; " /><meta name="citation_reference" content="citation_title=Long-read sequencing of chicken transcripts and identification of new transcript isoforms; citation_author=Bench To Basinet CvDC I.S;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e94650" /><meta name="citation_reference" content="citation_title=Widespread polycistronic transcripts in fungi revealed by single-molecule mRNA sequencing;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0132628" /><meta name="citation_reference" content="citation_title=Exploiting single-molecule transcript sequencing for eukaryotic gene prediction;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=184" /><meta name="citation_reference" content="citation_title=proovread: large-scale high-accuracy PacBio correction through iterative short read consensus;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3004-11; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=Integrative genomics viewer;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=24-6; " /><meta name="citation_reference" content="citation_title=How did alternative splicing evolve?;  citation_journal_title=Nat. Rev. Genet;  citation_year=2004;  citation_volume=5;  citation_pages=773-82; " /><meta name="citation_reference" content="citation_title=ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W297-9; " /><meta name="citation_reference" content="citation_title=IBS: an illustrator for the presentation and visualization of biological sequences;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3359-61; " /><meta name="citation_reference" content="citation_title=PLEK: a tool for predicting long non-coding RNAs and messenger RNAs based on an improved k-mer scheme;  citation_journal_title=BMC Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=311" /><meta name="citation_reference" content="citation_title=ALDB: a domestic-animal long noncoding RNA database;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0124003" /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=HTSeq–a Python framework to work with high-throughput sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=GFOLD: a generalized fold change for ranking differentially expressed genes from RNA-seq data;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=2782-8; " /><meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; " /><meta name="citation_reference" content="citation_title=Structure and regulation of the porcine skeletal alpha-actin-encoding gene;  citation_journal_title=Gene;  citation_year=1996;  citation_volume=180;  citation_pages=23-8; " /><meta name="citation_reference" content="citation_title=A survey of the sorghum transcriptome using single-molecule long reads;  citation_journal_title=Nat. Comms;  citation_year=2016;  citation_volume=7;  citation_pages=11706" /><meta name="citation_reference" content="citation_title=Expansion of the eukaryotic proteome by alternative splicing;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=457-63; " /><meta name="citation_reference" content="citation_title=Granulomatous angiitis of the CNS revealing a Hodgkin lymphoma;  citation_journal_title=Neurology;  citation_year=2013;  citation_volume=80;  citation_pages=323-4; " /><meta name="citation_reference" content="citation_title=Mutation-specific effects on thin filament length in thin filament myopathy;  citation_journal_title=Ann. Neurol;  citation_year=2016;  citation_volume=79;  citation_pages=959-69; " /><meta name="citation_reference" content="citation_title=Generation of an efficient artificial promoter of bovine skeletal muscle alpha-actin gene (ACTA1) through addition of cis-acting element;  citation_journal_title=Cell Mol. Biol. Lett;  citation_year=2015;  citation_volume=20;  citation_pages=160-76; " /><meta name="citation_reference" content="citation_title=DNA-methylation effect on cotranscriptional splicing is dependent on GC architecture of the exon-intron structure;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=789-99; " /><meta name="citation_reference" content="citation_title=Human DNA methylomes at base resolution show widespread epigenomic differences;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=462;  citation_pages=315-22; " /><meta name="citation_reference" content="citation_title=Dynamic changes in the human methylome during differentiation;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=320-31; " /><meta name="citation_reference" content="citation_title=When epigenetics meets alternative splicing: the roles of DNA methylation and GC architecture;  citation_journal_title=Epigenomics;  citation_year=2013;  citation_volume=5;  citation_pages=351-3; " /><meta name="citation_reference" content="citation_title=Conservation and divergence of methylation patterning in plants and animals;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2010;  citation_volume=107;  citation_pages=8689-94; " /><meta name="citation_reference" content="citation_title=DNA methylation landscapes: provocative insights from epigenomics;  citation_journal_title=Nat. Rev. Genet;  citation_year=2008;  citation_volume=9;  citation_pages=465-76; " /><meta name="citation_reference" content="citation_title=Characterization of fusion genes and the significantly expressed fusion isoforms in breast cancer by hybrid sequencing;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=e116" /><meta name="citation_reference" content="citation_title=mTORC1-mediated downregulation of COX2 restrains tumor growth caused by TSC2 deficiency;  citation_journal_title=Oncotarget;  citation_year=2016;  citation_volume=7;  citation_pages=28435-47; " /><meta name="citation_reference" content="citation_title=Targeted Cox2 gene deletion in intestinal epithelial cells decreases tumorigenesis in female, but not male, ApcMin/+ mice;  citation_journal_title=Mol. Oncol;  citation_year=2014;  citation_volume=8;  citation_pages=169-77; " /><meta name="citation_reference" content="citation_title=Binding of ATP to UAP56 is necessary for mRNA export;  citation_journal_title=J. Cell Sci;  citation_year=2008;  citation_volume=121;  citation_pages=1526-37; " /><meta name="citation_reference" content="citation_title=Structure of tumor necrosis factor-alpha haploblocks in European populations;  citation_journal_title=Immunogenetics;  citation_year=2013;  citation_volume=65;  citation_pages=543-52; " /><meta name="citation_reference" content="citation_title=STEM: a tool for the analysis of short time series gene expression data;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=191" /><meta name="citation_reference" content="citation_title=TUSC5 regulates insulin-mediated adipose tissue glucose uptake by modulation of GLUT4 recycling;  citation_journal_title=Mol. Metab;  citation_year=2015;  citation_volume=4;  citation_pages=795-810; " /><meta name="citation_reference" content="citation_title=Mice deficient in the respiratory chain gene Cox6a2 are protected against high-fat diet-induced obesity and insulin resistance;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e56719" /><meta name="citation_reference" content="citation_title=A missense mutation in MYH1 is associated with susceptibility to immune-mediated myositis in Quarter Horses;  citation_journal_title=Skelet. Muscle;  citation_year=2018;  citation_volume=8;  citation_pages=7" /><meta name="citation_reference" content="citation_title=Identification of Xin-repeat proteins as novel ligands of the SH3 domains of nebulin and nebulette and analysis of their interaction during myofibril formation and remodeling;  citation_journal_title=Mol. Biol. Cell;  citation_year=2013;  citation_volume=24;  citation_pages=3215-26; " /><meta name="citation_reference" content="citation_title=A global DNA methylation and gene expression analysis of early human B-cell development reveals a demethylation signature and transcription factor network;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=11339-51; " /><meta name="citation_reference" content="citation_title=Functional consequences of developmentally regulated alternative splicing;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=715-29; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of gene and isoform regulation in Mammalian tissues;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1593-9; " /><meta name="citation_reference" content="citation_title=The evolutionary landscape of alternative splicing in vertebrate species;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1587-93; " /><meta name="citation_reference" content="citation_title=Human epigenome data reveal increased CpG methylation in alternatively spliced sites and putative exonic splicing enhancers;  citation_journal_title=DNA Cell Biol;  citation_year=2011;  citation_volume=30;  citation_pages=267-75; " /><meta name="citation_reference" content="citation_title=In silico prediction of long intergenic non-coding RNAs in sheep;  citation_journal_title=Genome;  citation_year=2016;  citation_volume=59;  citation_pages=263-75; " /><meta name="citation_reference" content="citation_title=A catalogue of novel bovine long noncoding RNA across 18 tissues;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0141225" /><meta name="citation_reference" content="citation_title=DNA methylation signatures of long intergenic noncoding RNAs in porcine adipose and muscle tissues;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=15435" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/421/25501695/dsy014.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) and fusion transcripts produce a vast expansion of transcriptomes and proteomes diversity. However, the reliability of thes" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/421/5020793" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/421/5020793" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/421/5020793");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy014';
    </script>
<title>
	survey of transcriptome complexity in Sus scrofa using single-molecule long-read sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5020793" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdqfATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWIgb2YgQWdyaWN1bHR1cmUgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZywgYW5kIFJlcHJvZHVjdGlvbiBvZiBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYW5nLCBDaGVuZ2NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWIgb2YgQWdyaWN1bHR1cmUgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZywgYW5kIFJlcHJvZHVjdGlvbiBvZiBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdSwgWXVodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFiIG9mIEFncmljdWx0dXJlIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcsIGFuZCBSZXByb2R1Y3Rpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHUsIEFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYiBvZiBBZ3JpY3VsdHVyZSBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nLCBhbmQgUmVwcm9kdWN0aW9uIG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBDZW5jZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFiIG9mIEFncmljdWx0dXJlIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcsIGFuZCBSZXByb2R1Y3Rpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWm91LCBDaGVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWIgb2YgQWdyaWN1bHR1cmUgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZywgYW5kIFJlcHJvZHVjdGlvbiBvZiBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWGlueXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYiBvZiBBZ3JpY3VsdHVyZSBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nLCBhbmQgUmVwcm9kdWN0aW9uIG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW8sIFNodWhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFiIG9mIEFncmljdWx0dXJlIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcsIGFuZCBSZXByb2R1Y3Rpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIENoZW5nanVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2VybXBsYXNtIEJhbmsgb2YgV2lsZCBTcGVjaWVzLCBLdW5taW5nIEluc3RpdHV0ZSBvZiBCb3RhbnksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgS3VubWluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgQ2hhbmdjaHVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYiBvZiBBZ3JpY3VsdHVyZSBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nLCBhbmQgUmVwcm9kdWN0aW9uIG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBzdXJ2ZXkgb2YgdHJhbnNjcmlwdG9tZSBjb21wbGV4aXR5IGluIFN1cyBzY3JvZmEgdXNpbmcgc2luZ2xlLW1vbGVjdWxlIGxvbmctcmVhZCBzZXF1ZW5jaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2NlZWRpbmdzIG9mIHRoZSBjb25mZXJlbmNlIG9uIHN3aW5lIGluIGJpb21lZGljYWwgcmVzZWFyY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlMQVIgSjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTg5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudGFsIG9yaWdpbnMgb2YgaGVhbHRoIGFuZCBkaXNlYXNlIGluIHN3aW5lOiBpbXBsaWNhdGlvbnMgZm9yIGFuaW1hbCBwcm9kdWN0aW9uIGFuZCBiaW9tZWRpY2FsIHJlc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVyaW9nZW5vbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTExMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgcmVzZXF1ZW5jaW5nIGFuYWx5c2VzIG9mIGZpdmUgcGlnIGJyZWVkcywgaW5jbHVkaW5nIEtvcmVhbiB3aWxkIGFuZCBuYXRpdmUsIGFuZCB0aHJlZSBFdXJvcGVhbiBvcmlnaW4gYnJlZWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjU5LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgUGFwIHNtZWFyOiBoUFYgZ2Vub3R5cGUgYW5kIEROQSBtZXRoeWxhdGlvbiBvZiBBRENZOCwgQ0RIOCwgYW5kIFpORjU4MiBhcyBhbiBpbnRlZ3JhdGVkIGJpb21hcmtlciBmb3IgaGlnaC1ncmFkZSBjZXJ2aWNhbCBjeXRvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2xpbi4gRXBpZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgaHVtYW4gRVNDIHRyYW5zY3JpcHRvbWUgYnkgaHlicmlkIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVSBTIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9RTQ4MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQaWcgR2Vub21lIERhdGFiYXNlIChQaUdlbm9tZSk6IGFuIGludGVncmF0ZWQgZGF0YWJhc2UgZm9yIHBpZyBnZW5vbWUgcmVzZWFyY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1hbW0uIEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTYwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3R1ZHkgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gdGhlIHBpZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFJlcy4gTm90ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIGlzb2Zvcm0gcmVndWxhdGlvbiBpbiBodW1hbiB0aXNzdWUgdHJhbnNjcmlwdG9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTY7ICBjaXRhdGlvbl9wYWdlcz00NzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYW1hemluZyBzZXF1ZW5jZSBhcnJhbmdlbWVudCBhdCB0aGUgNScgZW5kcyBvZiBhZGVub3ZpcnVzIDIgbWVzc2VuZ2VyIFJOQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MTk3NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBzdXJ2ZXlpbmcgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgY29tcGxleGl0eSBpbiB0aGUgaHVtYW4gdHJhbnNjcmlwdG9tZSBieSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTE0MTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gY293OiBpbXBsaWNhdGlvbnMgaW4gYm92aW5lIGFzIGEgbW9kZWwgZm9yIGh1bWFuIGRpc2Vhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1TMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgdmlldyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGFuZCBjZWxsIHN1cnZpdmFsOiBmcm9tIHRpc3N1ZSBob21lb3N0YXNpcyB0byBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIERlYXRoIERpZmZlcjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE5MTktMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFudGlzZW5zZSBvbGlnb251Y2xlb3RpZGUtbWVkaWF0ZWQgTURNNCBleG9uIDYgc2tpcHBpbmcgaW1wYWlycyB0dW1vciBncm93dGg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENsaW4uIEludmVzdDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz02OC04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VydmV5IG9mIHByb2dyYW1zIHVzZWQgdG8gZGV0ZWN0IGFsdGVybmF0aXZlIHNwbGljaW5nIGlzb2Zvcm1zIGZyb20gZGVlcCBzZXF1ZW5jaW5nIGRhdGEgaW4gc2lsaWNvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9tZWQuIFJlcy4gSW50OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIwMTU7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLW1lZGlhdGVkIGVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGlzb2Zvcm0gdmFyaWF0aW9uIGFzc29jaWF0ZWQgd2l0aCBjeXRvc2luZSBtb2RpZmljYXRpb24gaW4gaHVtYW4gbHltcGhvYmxhc3RvaWQgY2VsbCBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjAzOyAgY2l0YXRpb25fcGFnZXM9OTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnZlaWxpbmcgdGhlIGNvbXBsZXhpdHkgb2YgdGhlIG1haXplIHRyYW5zY3JpcHRvbWUgYnkgc2luZ2xlLW1vbGVjdWxlIGxvbmctcmVhZCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW1zOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMTcwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHRlbG9tZXJlLXRvLXRlbG9tZXJlIGRlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBnZW5vbWUgdGhyb3VnaCBsb25nLXJlYWQgKCZndDsxMSBrYiksIHNpbmdsZSBtb2xlY3VsZSwgcmVhbC10aW1lIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zMzktNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2luZ2xlLW1vbGVjdWxlIGxvbmctcmVhZCBzdXJ2ZXkgb2YgdGhlIGh1bWFuIHRyYW5zY3JpcHRvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDktMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyB1c2luZyBzeW50aGV0aWMgbG9uZy1yZWFkIHNlcXVlbmNpbmcgcmV2ZWFscyBtb2xlY3VsYXIgY28tYXNzb2NpYXRpb24gb2YgZGlzdGFudCBzcGxpY2luZyBldmVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTczNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZy1yZWFkIHNlcXVlbmNpbmcgb2YgY2hpY2tlbiB0cmFuc2NyaXB0cyBhbmQgaWRlbnRpZmljYXRpb24gb2YgbmV3IHRyYW5zY3JpcHQgaXNvZm9ybXM7IGNpdGF0aW9uX2F1dGhvcj1CZW5jaCBUbyBCYXNpbmV0IEN2REMgSS5TOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTk0NjUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBwb2x5Y2lzdHJvbmljIHRyYW5zY3JpcHRzIGluIGZ1bmdpIHJldmVhbGVkIGJ5IHNpbmdsZS1tb2xlY3VsZSBtUk5BIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMzI2MjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb2l0aW5nIHNpbmdsZS1tb2xlY3VsZSB0cmFuc2NyaXB0IHNlcXVlbmNpbmcgZm9yIGV1a2FyeW90aWMgZ2VuZSBwcmVkaWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE4NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXByb292cmVhZDogbGFyZ2Utc2NhbGUgaGlnaC1hY2N1cmFjeSBQYWNCaW8gY29ycmVjdGlvbiB0aHJvdWdoIGl0ZXJhdGl2ZSBzaG9ydCByZWFkIGNvbnNlbnN1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zMDA0LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HTUFQOiBhIGdlbm9taWMgbWFwcGluZyBhbmQgYWxpZ25tZW50IHByb2dyYW0gZm9yIG1STkEgYW5kIEVTVCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTg1OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgZ2Vub21pY3Mgdmlld2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yNC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgZGlkIGFsdGVybmF0aXZlIHNwbGljaW5nIGV2b2x2ZT87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NzczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BU1RBTEFWSVNUQTogZHluYW1pYyBhbmQgZmxleGlibGUgYW5hbHlzaXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzIGluIGN1c3RvbSBnZW5lIGRhdGFzZXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcyOTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SUJTOiBhbiBpbGx1c3RyYXRvciBmb3IgdGhlIHByZXNlbnRhdGlvbiBhbmQgdmlzdWFsaXphdGlvbiBvZiBiaW9sb2dpY2FsIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMzU5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QTEVLOiBhIHRvb2wgZm9yIHByZWRpY3RpbmcgbG9uZyBub24tY29kaW5nIFJOQXMgYW5kIG1lc3NlbmdlciBSTkFzIGJhc2VkIG9uIGFuIGltcHJvdmVkIGstbWVyIHNjaGVtZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MzExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUxEQjogYSBkb21lc3RpYy1hbmltYWwgbG9uZyBub25jb2RpbmcgUk5BIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTI0MDAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IVFNlceKAk2EgUHl0aG9uIGZyYW1ld29yayB0byB3b3JrIHdpdGggaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xNjYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0ZPTEQ6IGEgZ2VuZXJhbGl6ZWQgZm9sZCBjaGFuZ2UgZm9yIHJhbmtpbmcgZGlmZmVyZW50aWFsbHkgZXhwcmVzc2VkIGdlbmVzIGZyb20gUk5BLXNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3ODItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiB0cmFuc2NyaXB0IHJlY29uc3RydWN0aW9uIG1ldGhvZHMgZm9yIFJOQS1zZXE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTExNzctODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSBhbmQgcmVndWxhdGlvbiBvZiB0aGUgcG9yY2luZSBza2VsZXRhbCBhbHBoYS1hY3Rpbi1lbmNvZGluZyBnZW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3VydmV5IG9mIHRoZSBzb3JnaHVtIHRyYW5zY3JpcHRvbWUgdXNpbmcgc2luZ2xlLW1vbGVjdWxlIGxvbmcgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTExNzA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwYW5zaW9uIG9mIHRoZSBldWthcnlvdGljIHByb3Rlb21lIGJ5IGFsdGVybmF0aXZlIHNwbGljaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9NDU3LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HcmFudWxvbWF0b3VzIGFuZ2lpdGlzIG9mIHRoZSBDTlMgcmV2ZWFsaW5nIGEgSG9kZ2tpbiBseW1waG9tYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV1cm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9MzIzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uLXNwZWNpZmljIGVmZmVjdHMgb24gdGhpbiBmaWxhbWVudCBsZW5ndGggaW4gdGhpbiBmaWxhbWVudCBteW9wYXRoeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBOZXVyb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz05NTktNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYXRpb24gb2YgYW4gZWZmaWNpZW50IGFydGlmaWNpYWwgcHJvbW90ZXIgb2YgYm92aW5lIHNrZWxldGFsIG11c2NsZSBhbHBoYS1hY3RpbiBnZW5lIChBQ1RBMSkgdGhyb3VnaCBhZGRpdGlvbiBvZiBjaXMtYWN0aW5nIGVsZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgTW9sLiBCaW9sLiBMZXR0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTYwLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEtbWV0aHlsYXRpb24gZWZmZWN0IG9uIGNvdHJhbnNjcmlwdGlvbmFsIHNwbGljaW5nIGlzIGRlcGVuZGVudCBvbiBHQyBhcmNoaXRlY3R1cmUgb2YgdGhlIGV4b24taW50cm9uIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTc4OS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gRE5BIG1ldGh5bG9tZXMgYXQgYmFzZSByZXNvbHV0aW9uIHNob3cgd2lkZXNwcmVhZCBlcGlnZW5vbWljIGRpZmZlcmVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYyOyAgY2l0YXRpb25fcGFnZXM9MzE1LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIGNoYW5nZXMgaW4gdGhlIGh1bWFuIG1ldGh5bG9tZSBkdXJpbmcgZGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzIwLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGVuIGVwaWdlbmV0aWNzIG1lZXRzIGFsdGVybmF0aXZlIHNwbGljaW5nOiB0aGUgcm9sZXMgb2YgRE5BIG1ldGh5bGF0aW9uIGFuZCBHQyBhcmNoaXRlY3R1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVwaWdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zNTEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VydmF0aW9uIGFuZCBkaXZlcmdlbmNlIG9mIG1ldGh5bGF0aW9uIHBhdHRlcm5pbmcgaW4gcGxhbnRzIGFuZCBhbmltYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTg2ODktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBsYW5kc2NhcGVzOiBwcm92b2NhdGl2ZSBpbnNpZ2h0cyBmcm9tIGVwaWdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQ2NS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBmdXNpb24gZ2VuZXMgYW5kIHRoZSBzaWduaWZpY2FudGx5IGV4cHJlc3NlZCBmdXNpb24gaXNvZm9ybXMgaW4gYnJlYXN0IGNhbmNlciBieSBoeWJyaWQgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1lMTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bVRPUkMxLW1lZGlhdGVkIGRvd25yZWd1bGF0aW9uIG9mIENPWDIgcmVzdHJhaW5zIHR1bW9yIGdyb3d0aCBjYXVzZWQgYnkgVFNDMiBkZWZpY2llbmN5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PbmNvdGFyZ2V0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yODQzNS00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgQ294MiBnZW5lIGRlbGV0aW9uIGluIGludGVzdGluYWwgZXBpdGhlbGlhbCBjZWxscyBkZWNyZWFzZXMgdHVtb3JpZ2VuZXNpcyBpbiBmZW1hbGUsIGJ1dCBub3QgbWFsZSwgQXBjTWluLysgbWljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBPbmNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTY5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW5kaW5nIG9mIEFUUCB0byBVQVA1NiBpcyBuZWNlc3NhcnkgZm9yIG1STkEgZXhwb3J0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDZWxsIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz0xNTI2LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgdHVtb3IgbmVjcm9zaXMgZmFjdG9yLWFscGhhIGhhcGxvYmxvY2tzIGluIEV1cm9wZWFuIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbW11bm9nZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTU0My01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1RFTTogYSB0b29sIGZvciB0aGUgYW5hbHlzaXMgb2Ygc2hvcnQgdGltZSBzZXJpZXMgZ2VuZSBleHByZXNzaW9uIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VFVTQzUgcmVndWxhdGVzIGluc3VsaW4tbWVkaWF0ZWQgYWRpcG9zZSB0aXNzdWUgZ2x1Y29zZSB1cHRha2UgYnkgbW9kdWxhdGlvbiBvZiBHTFVUNCByZWN5Y2xpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWV0YWI7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTc5NS04MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY2UgZGVmaWNpZW50IGluIHRoZSByZXNwaXJhdG9yeSBjaGFpbiBnZW5lIENveDZhMiBhcmUgcHJvdGVjdGVkIGFnYWluc3QgaGlnaC1mYXQgZGlldC1pbmR1Y2VkIG9iZXNpdHkgYW5kIGluc3VsaW4gcmVzaXN0YW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU1NjcxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWlzc2Vuc2UgbXV0YXRpb24gaW4gTVlIMSBpcyBhc3NvY2lhdGVkIHdpdGggc3VzY2VwdGliaWxpdHkgdG8gaW1tdW5lLW1lZGlhdGVkIG15b3NpdGlzIGluIFF1YXJ0ZXIgSG9yc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ta2VsZXQuIE11c2NsZTsgIGNpdGF0aW9uX3llYXI9MjAxODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIFhpbi1yZXBlYXQgcHJvdGVpbnMgYXMgbm92ZWwgbGlnYW5kcyBvZiB0aGUgU0gzIGRvbWFpbnMgb2YgbmVidWxpbiBhbmQgbmVidWxldHRlIGFuZCBhbmFseXNpcyBvZiB0aGVpciBpbnRlcmFjdGlvbiBkdXJpbmcgbXlvZmlicmlsIGZvcm1hdGlvbiBhbmQgcmVtb2RlbGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MzIxNS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnbG9iYWwgRE5BIG1ldGh5bGF0aW9uIGFuZCBnZW5lIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgZWFybHkgaHVtYW4gQi1jZWxsIGRldmVsb3BtZW50IHJldmVhbHMgYSBkZW1ldGh5bGF0aW9uIHNpZ25hdHVyZSBhbmQgdHJhbnNjcmlwdGlvbiBmYWN0b3IgbmV0d29yazsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xMTMzOS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBjb25zZXF1ZW5jZXMgb2YgZGV2ZWxvcG1lbnRhbGx5IHJlZ3VsYXRlZCBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NzE1LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgZ2VuZSBhbmQgaXNvZm9ybSByZWd1bGF0aW9uIGluIE1hbW1hbGlhbiB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzODsgIGNpdGF0aW9uX3BhZ2VzPTE1OTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbmFyeSBsYW5kc2NhcGUgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gdmVydGVicmF0ZSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzODsgIGNpdGF0aW9uX3BhZ2VzPTE1ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIGVwaWdlbm9tZSBkYXRhIHJldmVhbCBpbmNyZWFzZWQgQ3BHIG1ldGh5bGF0aW9uIGluIGFsdGVybmF0aXZlbHkgc3BsaWNlZCBzaXRlcyBhbmQgcHV0YXRpdmUgZXhvbmljIHNwbGljaW5nIGVuaGFuY2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI2Ny03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gc2lsaWNvIHByZWRpY3Rpb24gb2YgbG9uZyBpbnRlcmdlbmljIG5vbi1jb2RpbmcgUk5BcyBpbiBzaGVlcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9MjYzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNhdGFsb2d1ZSBvZiBub3ZlbCBib3ZpbmUgbG9uZyBub25jb2RpbmcgUk5BIGFjcm9zcyAxOCB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTQxMjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIHNpZ25hdHVyZXMgb2YgbG9uZyBpbnRlcmdlbmljIG5vbmNvZGluZyBSTkFzIGluIHBvcmNpbmUgYWRpcG9zZSBhbmQgbXVzY2xlIHRpc3N1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTQzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS80LzQyMS8yNTUwMTY5NS9kc3kwMTQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGVybmF0aXZlIHNwbGljaW5nIChBUykgYW5kIGZ1c2lvbiB0cmFuc2NyaXB0cyBwcm9kdWNlIGEgdmFzdCBleHBhbnNpb24gb2YgdHJhbnNjcmlwdG9tZXMgYW5kIHByb3Rlb21lcyBkaXZlcnNpdHkuIEhvd2V2ZXIsIHRoZSByZWxpYWJpbGl0eSBvZiB0aGVzIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01MDIwNzkzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZEIgIY0MckaEy50iWIMRL4EAV0ay" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f421%2f5020793" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f421%2f5020793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120122143">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122143">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122145">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122145">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122150">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122150">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122176">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122176">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122217">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122217">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122226">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122226">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122232">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122232">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/409/5003450">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/439/5035912">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A survey of transcriptome complexity in <em>Sus scrofa</em> using single-molecule long-read sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yao Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yao+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yao Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chengchi Fang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chengchi+Fang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fang C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fang C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chengchi Fang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuhua Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuhua+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fu Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuhua Fu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">An Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=An+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">An Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cencen Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cencen+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cencen Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cheng Zou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cheng+Zou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zou C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zou C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cheng Zou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinyun Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xinyun+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xinyun Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuhong Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuhong+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shuhong Zhao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chengjun Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Germplasm Bank of Wild Species, Kunming Institute of Botany, Chinese Academy of Sciences, Kunming, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chengjun+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chengjun Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Changchun Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Lab of Agriculture Animal Genetics, Breeding, and Reproduction of Ministry of Education, College of Animal Science and Technology, Huazhong Agricultural University, Wuhan, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy014-cor1" data-open="dsy014-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 27-87388470. Email: <a href="mailto:lichangchun@mail.hzau.edu.cn" target="_blank">lichangchun@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Changchun+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Changchun Li</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy014-FM2"><div class="fn"><div class="footnote-content">Yao Li and Chengchi Fang authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy014-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 421–437, <a href='https://doi.org/10.1093/dnares/dsy014'>https://doi.org/10.1093/dnares/dsy014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 May 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 January 2018</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 May 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5020793" href="/dnaresearch/article-pdf/25/4/421/25501695/dsy014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yao Li, Chengchi Fang, Yuhua Fu, An Hu, Cencen Li, Cheng Zou, Xinyun Li, Shuhong Zhao, Chengjun Zhang, Changchun Li;  A survey of transcriptome complexity in <em>Sus scrofa</em> using single-molecule long-read sequencing, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 421–437, <a href='https://doi.org/10.1093/dnares/dsy014'>https://doi.org/10.1093/dnares/dsy014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5020793?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20survey%20of%20transcriptome%20complexity%20in%20%26lt%3Bem%26gt%3BSus%20scrofa%26lt%3B%2Fem%26gt%3B%20using%20single-molecule%20long-read%20sequencing&amp;publicationDate=2018-05-29&amp;volumeNum=25&amp;issueNum=4&amp;author=Li%2C%20Yao%3B%20Fang%2C%20Chengchi&amp;startPage=421&amp;endPage=437&amp;contentId=10.1093%2Fdnares%2Fdsy014&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f421%2f5020793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120122143 id="120122143" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) and fusion transcripts produce a vast expansion of transcriptomes and proteomes diversity. However, the reliability of these events and the extend of epigenetic mechanisms have not been adequately addressed due to its limitation of uncertainties about the complete structure of mRNA. Here we combined single-molecule real-time sequencing, Illumina RNA-seq and DNA methylation data to characterize the landscapes of DNA methylation on AS, fusion isoforms formation and lncRNA feature and further to unveil the transcriptome complexity of pig. Our analysis identified an unprecedented scale of high-quality full-length isoforms with over 28,127 novel isoforms from 26,881 novel genes. More than 92,000 novel AS events were detected and intron retention predominated in AS model, followed by exon skipping. Interestingly, we found that DNA methylation played an important role in generating various AS isoforms by regulating splicing sites, promoter regions and first exons. Furthermore, we identified a large of fusion transcripts and novel lncRNAs, and found that DNA methylation of the promoter and gene body could regulate lncRNA expression. Our results significantly improved existed gene models of pig and unveiled that pig AS and epigenetic modify were more complex than previously thought.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single-molecule sequencing&quot;">single-molecule sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="full-length">full-length</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;novel gene&quot;">novel gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="methylation">methylation</a></div>                            <h2 scrollto-destination=120122145 id="120122145" class="section-title" >1. Introduction</h2>
<p>Domestic pig (<em>Sus scrofa</em>) is an agriculturally important species and an attractive biomedical model because of its anatomical, physiological, pathological and genomic similarities to humans.<a reveal-id="dsy014-B1" data-open="dsy014-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy014-B2" data-open="dsy014-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> However, increasing number of studies have shown that reference genomes are often incomplete and has annotation and structural defects.<a reveal-id="dsy014-B3" data-open="dsy014-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy014-B4" data-open="dsy014-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Thus, reference assembly and gene annotations require refinement. Obtained through short-read sequencing, the sequence data of several species have been accumulated in recent years. But the knowledge on full-length (FL) sequences of mRNAs remains scarce. Furthermore, in some cases, low-quality transcripts derived from short-read sequencing can result in incorrect annotations.<a reveal-id="dsy014-B5" data-open="dsy014-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> FL transcripts can significantly increase the accuracy of genome annotation and transcriptome characterization. Several expressed sequence tags (ESTs) from FL studies have been performed in pigs,<a reveal-id="dsy014-B6" data-open="dsy014-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy014-B7" data-open="dsy014-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> which improved genome annotation and was beneficial to downstream analysis such as expression quantification and alternative splicing (AS) identification.</p><p>AS increases the variability of the cells and tissues proteome according to different splice modes in a single animal, thereby changing the composition of transcribed genes without massively increasing the number of genes.<a reveal-id="dsy014-B8" data-open="dsy014-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Since AS was discovered in 1977,<a reveal-id="dsy014-B9" data-open="dsy014-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> a large number of AS events were identified in the reference genomes of human and other animals.<a reveal-id="dsy014-B10" data-open="dsy014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy014-B11" data-open="dsy014-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> In humans, ∼20% of multi-exon genes are tissue specific and ∼95% of multi-exon genes are alternatively spliced.<a reveal-id="dsy014-B10" data-open="dsy014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In porcine, ∼30% of the genes undergo AS, and 31% of the identified splice events appear to be species specific.<a reveal-id="dsy014-B7" data-open="dsy014-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> AS is an important regulatory mechanism involved in gene expression and proteome diversity in individual<a reveal-id="dsy014-B12" data-open="dsy014-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and related to many diseases such as cancer and chemoresistance.<a reveal-id="dsy014-B13" data-open="dsy014-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Depending on a specific AS switch, enhancing the specific exon inclusion has potential as a clinically compatible therapeutic target.<a reveal-id="dsy014-B14" data-open="dsy014-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Accordingly, a more comprehensive and accurate identification of AS event will facilitate further cognition of the AS regulatory mechanism, scholars in medicine, genetics, bioinformatics and other fields.<a reveal-id="dsy014-B15" data-open="dsy014-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p>Diverse classes of epigenetic regulation, ranging from ncRNA to methylation, have emerged as key regulators of gene expression, genome stability and defence against foreign genetic elements.<a reveal-id="dsy014-B16" data-open="dsy014-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Epigenetics can mediate disease aetiology through isoform variations of cytosine modification-specific transcript which are attributable to AS.<a reveal-id="dsy014-B17" data-open="dsy014-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Recently, it is revealed that CHG methylation could repress AS while CG methylation promoted AS in plant.<a reveal-id="dsy014-B18" data-open="dsy014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> However, the comprehensive relationship between DNA methylation and AS or lncRNA remains unclear in animals. Because uncertainties about the complete structure of mRNA transcripts limited the identification of splice sites and lncRNA.</p><p>Single-molecule real-time (SMRT) sequencing carried out in Pacific Bioscience RS (PacBio, <a class="link link-uri" href="http://www.pacificbiosciences.com/" target="_blank">http://www.pacificbiosciences.com/</a>) provides a third-generation sequencing platform widely used in genome sequencing because of its long reads (average: 12 kb).<a reveal-id="dsy014-B19" data-open="dsy014-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> SMRT technology without assembling sequencing read provides direct evidence for comprehensive analysis of splice isoforms of each gene and can improve the annotation of existing gene models.<a reveal-id="dsy014-B20 dsy014-B21 dsy014-B22" data-open="dsy014-B20 dsy014-B21 dsy014-B22" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> Recently, Iso-Seq is used to analyse FL splice isoforms in humans<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and chickens,<a reveal-id="dsy014-B22" data-open="dsy014-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and indicates that identification of genes and splice isoforms are far from being complete even in a highly characterized transcriptome. In this study, we combined SMRT sequencing and short-read next-generation sequencing technology to generate a more complete FL porcine transcriptome further to analyse features of AS, fusion isoforms and lncRNAs. Accordingly, this study highlights the splice isoforms and transcriptome diversity and dynamics, provides a valuable resource for further investigation of genome annotation and increases our understanding of the porcine transcriptome.</p>                            <h2 scrollto-destination=120122150 id="120122150" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120122151 id="120122151" class="section-title" >2.1 Animal materials</h3>
<p>A total of 38 porcine tissues of Large White sow were collected, including 20 tissues [heart, liver, spleen, lung, kidney, stomach, duodenum, cecum, inguinal lymph nodes, precaval vein blood, ovary, uterus, corpus luteum, inner ear, subcutaneous fat, longissimus muscle, psoas muscle, soleus muscle, extensor digitorum longus (EDL) and tongue] from adult sow, and 17 tissues [heart, liver, spleen, lung, stomach, duodenum, inguinal lymph nodes, precaval vein blood, uterus, thymus, skin (dorsum), subcutaneous fat, longissimus muscle, psoas muscle, soleus muscle, EDL and tongue] from one-day-old sow and one organ (26-day-old embryo). For each tissue, total RNA was extracted using TRIzol reagent (REF15596026, Invitrogen) and processed following the manufacturer’s protocol.</p>                            <h3 scrollto-destination=120122153 id="120122153" class="section-title" >2.2. PacBio library construction and sequencing</h3>
<p>Equimolar rations of 38 samples were pooled together. Total RNA (1 μg) was reverse-transcribed into cDNA using the SMARTer™ PCR cDNA synthesis kit (Takara Biotechnology, Dalian, China) and optimized to prepare high-quality and FL cDNAs. Subsequently, size fractionation (0.6–1, 1–2 and &gt;2 kb) was conducted using the BluePippin™ Size-Selection System (Sage Science, Beverly, MA). Another amplification was performed using 12–14 PCR cycles. Large-scale PCR products were purified with AMPure PB magnetic beads. Each SMRTbell library was constructed using selected cDNA (500 ng) with the Pacific Biosciences DNA Template Prep Kit 2.0. The SMRTbell templates were bound to polymerases using the DNA/Polymerase Binding Kit P6 and v2 primers. The polymerase-bound template was bound to zero-mode waveguide using Magbeadbingding kit (part 100-133-600). A total of 20 SMRT cells, composed of three SMRTbell libraries (0.6–1 kb: 7 cells; 1–2 kb: 7 cells; &gt;2kb: 6 cells), were prepared on the Pacific Bioscience RS II platform by Frasergen Inc. (Wuhan, China) using C4 reagents with 240 min movies.</p>                            <h3 scrollto-destination=120122155 id="120122155" class="section-title" >2.3 Illumina RNA-seq library construction</h3>
<p>In parallel, eight tissues (subcutaneous back fat, soleus muscle, EDL and endometria from adult and one-day-old sows) from the 38 tissues were sequenced respectively using PE125 sequencing on the Illumina HiSeq 2500 platform to quantify gene/isoform expression. HiSeq library was constructed using NEB kit. Briefly, poly(A) + RNA transcript was isolated from the total RNA (1 μg). Libraries were prepared using the NEBNext Ultra RNA Library Prep Kit for Illumina (<em>NEB</em> #E7530).</p>                            <h3 scrollto-destination=120122157 id="120122157" class="section-title" >2.4 Subread processing and error correction</h3>
<p>Effective subreads were obtained using the P_Fetch and P_Filter function (parameters: minSubReadLength =50, readScore =0.75 and minLength =50) in the SMRT Analysis Software v2.3 Suite (<a class="link link-uri" href="http://www.pacb.com/devnet/" target="_blank">http://www.pacb.com/devnet/</a>). The FL transcript sequence was obtained using ToFU pipeline<a reveal-id="dsy014-B23" data-open="dsy014-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). Briefly, circular consensus (CCS) read was obtained from the P_CCS module using the parameter MinFullPasses =2 and MinPredictedAccuracy =0. After examining for poly(A) signal, 5′ and 3′ adaptors, only the CCS with all three signals was considered a FL non-chimetric (FLNC) read.<a reveal-id="dsy014-B24" data-open="dsy014-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> To improve consensus accuracy, we used an isoform-level clustering algorithm, namely, iterative clustering for error correction (ICE), and polished FL consensus sequences from ICE using Quiver with the following cut-off criteria: isoform length &gt;200; high-quality &gt;0.99. Additional nucleotide errors in FLNC reads were corrected using the Illumina RNA-seq data with the software Proovread<a reveal-id="dsy014-B25" data-open="dsy014-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> using the parameter coverage of 127. The untrimmed sequence was regarded as the result of error correction. </p>                        <a id="120122159" scrollto-destination="120122159"></a>
<div id="dsy014-F1" data-id="dsy014-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f1.png?Expires=2147483647&Signature=MW6IoSLAK2WWr2rV8SvalsC~pdqIqzR7FO9OrXSZwH~hlAkq6uVZY1xpvtMtvFAmtlIlhoNQm0XgD1DVohAWhFUODq8uyOUAm5DDIKSN~QTRo1En-ozx160JWOqXC2RfYtzhk3g5bPujhcJdFgc0g6etJHftETx8app5BZ-XuIEKM~8y-U0ME7lDOlGSk1OESezgYA5UixFT9tt2nRU730jERklFF3soHxtuLZWLuUIHONzQTZAmfPkVp4QNoXMDfXyfIzAVPuPo3HRFyemnGGqkTmjdv6KoqXRp-PJHENroEZD5hS~HU4oBLoUDRfHcZfbfDfkdVwONWQtbc3hZCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of methods. (a) Iso-Seq workflow for data processing (ToFU). (b) Mapping of PacBio data. (c) Identification of full-length. Same 5′ and 3′ ends: the first and last splice sites of FLNC were as same as the reference annotation transcripts. With 5′ and 3′ ends: the first and last splice sites of the reference transcripts were presented in the splice sites of the FLNC sequence. Same 5′ end: The first splice sites of 5′ end was identical between annotated transcript and FLNC sequence. With 5′ end: the first splice site of the 5′ end of the annotated transcript was presented in the FLNC sequence. The criteria of full-length were the type ‘with 5′ end’. (d) Criteria for assessing a new isoform. (e) Fusion transcript identification from PacBio sequences." path-from-xml="dsy014f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122159/dsy014f1.tif" path-from-xml="dsy014f1.tif" target="_blank">View large</a><a section="120122159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f1.png?Expires=2147483647&Signature=e~4EeYPcj0itQzyHfwlHntR8X2e3DdZYljC2vtyjCie3KnhjVbIHaxR~V2JMXiH5HK~O92xHC-VR4bNOMYOUO98ar4NktkL2DjJm3M60GJxOY8aXu549GrZOoCCYmnP-uvZrsLlOFr0mDCRyEngFzhsuim02mbSbQeSGYsPNUnf-9BIxtFhpIgYb1dcX6n53TvUFlbTRYfZpujp8cbn6KHDdNTKj9Lcqkcg3Zj9aZ~b39ZWbnD~cF3l6yLOQTOFDQ1bSStzzqCCiI~dZDSyDpVmFimX6FrMRKbnmMmapI8U9rG7rGcE~fYpbdRsvYoFJydlIgsc1EUdfLvG8hXj4dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122159&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of methods. (a) Iso-Seq workflow for data processing (ToFU). (b) Mapping of PacBio data. (c) Identification of full-length. Same 5′ and 3′ ends: the first and last splice sites of FLNC were as same as the reference annotation transcripts. With 5′ and 3′ ends: the first and last splice sites of the reference transcripts were presented in the splice sites of the FLNC sequence. Same 5′ end: The first splice sites of 5′ end was identical between annotated transcript and FLNC sequence. With 5′ end: the first splice site of the 5′ end of the annotated transcript was presented in the FLNC sequence. The criteria of full-length were the type ‘with 5′ end’. (d) Criteria for assessing a new isoform. (e) Fusion transcript identification from PacBio sequences.</p></div></div><div content-id="dsy014-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f1.png?Expires=2147483647&Signature=MW6IoSLAK2WWr2rV8SvalsC~pdqIqzR7FO9OrXSZwH~hlAkq6uVZY1xpvtMtvFAmtlIlhoNQm0XgD1DVohAWhFUODq8uyOUAm5DDIKSN~QTRo1En-ozx160JWOqXC2RfYtzhk3g5bPujhcJdFgc0g6etJHftETx8app5BZ-XuIEKM~8y-U0ME7lDOlGSk1OESezgYA5UixFT9tt2nRU730jERklFF3soHxtuLZWLuUIHONzQTZAmfPkVp4QNoXMDfXyfIzAVPuPo3HRFyemnGGqkTmjdv6KoqXRp-PJHENroEZD5hS~HU4oBLoUDRfHcZfbfDfkdVwONWQtbc3hZCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of methods. (a) Iso-Seq workflow for data processing (ToFU). (b) Mapping of PacBio data. (c) Identification of full-length. Same 5′ and 3′ ends: the first and last splice sites of FLNC were as same as the reference annotation transcripts. With 5′ and 3′ ends: the first and last splice sites of the reference transcripts were presented in the splice sites of the FLNC sequence. Same 5′ end: The first splice sites of 5′ end was identical between annotated transcript and FLNC sequence. With 5′ end: the first splice site of the 5′ end of the annotated transcript was presented in the FLNC sequence. The criteria of full-length were the type ‘with 5′ end’. (d) Criteria for assessing a new isoform. (e) Fusion transcript identification from PacBio sequences." path-from-xml="dsy014f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122159/dsy014f1.tif" path-from-xml="dsy014f1.tif" target="_blank">View large</a><a section="120122159" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f1.png?Expires=2147483647&Signature=e~4EeYPcj0itQzyHfwlHntR8X2e3DdZYljC2vtyjCie3KnhjVbIHaxR~V2JMXiH5HK~O92xHC-VR4bNOMYOUO98ar4NktkL2DjJm3M60GJxOY8aXu549GrZOoCCYmnP-uvZrsLlOFr0mDCRyEngFzhsuim02mbSbQeSGYsPNUnf-9BIxtFhpIgYb1dcX6n53TvUFlbTRYfZpujp8cbn6KHDdNTKj9Lcqkcg3Zj9aZ~b39ZWbnD~cF3l6yLOQTOFDQ1bSStzzqCCiI~dZDSyDpVmFimX6FrMRKbnmMmapI8U9rG7rGcE~fYpbdRsvYoFJydlIgsc1EUdfLvG8hXj4dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122159&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of methods. (a) Iso-Seq workflow for data processing (ToFU). (b) Mapping of PacBio data. (c) Identification of full-length. Same 5′ and 3′ ends: the first and last splice sites of FLNC were as same as the reference annotation transcripts. With 5′ and 3′ ends: the first and last splice sites of the reference transcripts were presented in the splice sites of the FLNC sequence. Same 5′ end: The first splice sites of 5′ end was identical between annotated transcript and FLNC sequence. With 5′ end: the first splice site of the 5′ end of the annotated transcript was presented in the FLNC sequence. The criteria of full-length were the type ‘with 5′ end’. (d) Criteria for assessing a new isoform. (e) Fusion transcript identification from PacBio sequences.</p></div></div>                            <h3 scrollto-destination=120122160 id="120122160" class="section-title" >2.5 Mapping of PacBio data</h3>
<p>The error rectified FLNC reads were mapped to the pig genome sequence (Sus_scrofa.10.2.84) from ENSEMBL databases (Release 84) using GMAP<a reveal-id="dsy014-B26" data-open="dsy014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> with the options—no-chimeras-n 20 (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). The best mapped locus was selected for each FLNC read based on both identity and coverage values. Genome mapping results of FLNC reads were visualized using the Integrative Genome Viewer.<a reveal-id="dsy014-B27" data-open="dsy014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The high percent of identity (PID) aligned FLNC reads were used to annotate loci and isoform. For loci, two sequences, which overlapped 20% and at least one overlapping exon to more than 20%, were identified as the same loci transcript. For isoform, single-exon sequence with overlap was determined as the same isoform. Such sequence can be identified as the same isoform when all the splicing sites of the multiple-exon sequences were identical. The redundant and false positive gene structure was removed as follows: (i) the missing 5′ end was removed; the sequence structure was a subset of other sequences (sequence structure refers to the ordered sets of all remaining cleavage sites, excluding the initiation and termination sites); the 5′ last exon spanning the intron region was determined, and the sequence was retained when it spanned the intron region; (ii) region PID &lt;99: each transcript model kept at least two PacBio sequences; otherwise, all junctions of this sequence were annotated or supported by the junction of second-generation sequencing and (iii) the longest one was retained when the structure of two sequences was the same.</p>                            <h3 scrollto-destination=120122162 id="120122162" class="section-title" >2.6 Novel isoform</h3>
<p>The gene structure annotation results were compared with those of reference annotation to determine the new gene following these criteria: (1) results showed no overlap or overlapped by less than 20% of the annotated gene site, or (2) the gene overlap was more than 20%, but the gene direction was not consistent. The criteria used for a single transcript to identify novel isoform were as follows: (i) the final splice site of 3′ end changed and (ii) new intron or new exon emerged (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1d</a>).</p>                            <h3 scrollto-destination=120122164 id="120122164" class="section-title" >2.7 Alternative splicing classification</h3>
<p>The relative importance of the main models of AS<a reveal-id="dsy014-B28" data-open="dsy014-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and the comprehensive distribution of AS structure in pig transcriptome were ascertained using Astalavista.<a reveal-id="dsy014-B29" data-open="dsy014-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Astalavista was also used for the classification analysis of splice type for the gene model after removing redundancy, and the simplified model constructed by IBS was visualized.<a reveal-id="dsy014-B30" data-open="dsy014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h3 scrollto-destination=120122166 id="120122166" class="section-title" >2.8 Methylation data analysis</h3>
<p>Methylome data from our research group (the GEO accession is GSE92417) were remapped to the pig genome assembly, and the methylation levels were calculated as previously reported.<a reveal-id="dsy014-B18" data-open="dsy014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Briefly, all splice junctions from PacBio Iso-Seq transcripts were stacked (50 bp exon +50 bp intron for donor and 50 bp intron +50 bp exon for acceptor). The methylation level of each base pair was calculated as C/(C + T). The methylation of the donor site was calculated from the first nucleotide of both strands on 5′ end of the intron as C/(C + T). The methylation of the acceptor site was calculated from the last nucleotide of both strands on the 3′ end of the intron using the same formula. For lncRNA and non-lncRNA methylations, three regions were used for methylation study: 5 kb upstream transcription start site (TSS), transcript body and 5 kb downstream transcription termination site (TTS). Each region was divided into 100 bins. Each methylation ratio was calculated from the corresponding bins from all genes.</p>                            <h3 scrollto-destination=120122168 id="120122168" class="section-title" >2.9 Fusion transcript identification</h3>
<p>The criteria used to identify candidate fusion transcripts for a single transcript were as follows (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1e</a>): (i) FLNC transcript mapped to two or more annotation loci in the genome; (ii) each mapped locus must align with at least 10% of the transcript; (iii) the combined alignment coverage must be at least 99%; (iv) each mapped locus must be at least 10 kb apart and (v) a certain amount of Illumina reads should support the fusion regions.</p>                            <h3 scrollto-destination=120122170 id="120122170" class="section-title" >2.10 LncRNA identification from PacBio data</h3>
<p>LncRNA identification was performed as previously reported.<a reveal-id="dsy014-B18" data-open="dsy014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The known high-confidence 27,692 long non-coding RNA transcript sequences and 94,359 protein-coding transcript sequences of human downloaded from GENCODE (Release 25, GRCh38.p7) were used to build the model using PLEK.<a reveal-id="dsy014-B31" data-open="dsy014-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> All PacBio isoforms were predicted on the basis of the model. The open reading frames (ORFs) of candidate lncRNAs were predicted by EMBOSS (<a class="link link-uri" href="http://emboss.bioinformatics.nl/" target="_blank">http://emboss.bioinformatics.nl/</a>). The transcripts encoding ORFs that were longer than 100 amino acids were filtered. The remaining transcripts were further screened by BLASTX (<em>e</em>-value ≤1<em>e</em>−10) against protein sequences of all species from NR database. Finally, Basic Local Alignment Search Tool (BLASTN) was used to eliminate the previously discovered lincRNAs of pig in ALDB<a reveal-id="dsy014-B32" data-open="dsy014-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> under a criterion of <em>e</em>-value of ≤1e−10, min-identity of 90% and min-coverage of 85%.</p>                            <h3 scrollto-destination=120122172 id="120122172" class="section-title" >2.11 Identification of tissue-specific and period-specific PacBio isoforms by Illumina data</h3>
<p>Isoforms detected by PacBio was used as the template combined with the Illumina data to detect the junction of different tissue and period for multi-exon isoforms of Iso-Seq data. We used a new modified GFF file to estimate the expression level of each isoforms. The new GFF file contained junction positions of each isoform which were extracted from the GMAP mapping data. The expressed junction information was obtained as follows: the Illumina raw reads were under the quality control by ng_QC (in-house software of Frasergen Bioinformatics Inc., Wuhan, China) with parameters read_len 125 and default settings for other parameters. Then, the clean paired reads were aligned to the pig genome sequence using TopHat2.1.0<a reveal-id="dsy014-B33" data-open="dsy014-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> following these options: G—library-type fr-first strand. When all the junctions of one isoform were supported by the Illumina data, it was defined as an expressed isoform (with ‘1’) in the corresponding period or tissue. Conversely, when no Illumina data supported the junction of the Iso-Seq data, or the junction was only partially supported by the Illumina data, it was defined as an unexpressed isoform (with ‘0’) in the corresponding period or tissue. In addition, the read counts for expressed genes were calculated with the HTSeq-count software<a reveal-id="dsy014-B34" data-open="dsy014-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and the differential expression level of genes were determined using GFOLD<a reveal-id="dsy014-B35" data-open="dsy014-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> with the default parameters.</p>                            <h3 scrollto-destination=120122174 id="120122174" class="section-title" >2.12 Validation of isoforms by RT-PCR</h3>
<p>We randomly selected 13 isoforms including AS, novel gene, lncRNA and fusion gene for experimental validation. Transcript-specific primers were designed to span the predicted splicing events based on FL sequences using Primer-BLAST (<a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/" target="_blank">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>; <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). PCR amplification was monitored on 1.5% agarose gel and followed by Sanger sequencing. The 18srRNA was amplified as an endogenous control.</p>                            <h2 scrollto-destination=120122176 id="120122176" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120122177 id="120122177" class="section-title" >3.1 Pig transcriptome by PacBio Iso-Seq</h3>
<p>Short-read sequencing from the Illumina platform is effective in qualifying gene expression and detecting AS events. However, its capacity to accurately detect FL splice variants of genes is limited.<a reveal-id="dsy014-B36" data-open="dsy014-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> To avoid underestimating isoform diversity, pig transcriptomes were sequenced using the PacBio Iso-Seq platform. This platform can provide long reads often up to several transcript lengths, thus the acquisition of accurately reconstructed FL splice variants is possible. To identify as many transcripts as possible, high-quality total RNA was extracted from 38 tissues at two different developmental stages. The method for detecting FLNC, new AS and fusion transcripts was depicted in the flow chart shown in <a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Figure 1</a>.</p><p>SMRT bell libraries were constructed and sequenced on the PacBio RSII using the latest P6-C4 chemistry with 20 SMRT cells. In total, we detected 1,898,155 polymerase reads representing more than 36 G bases, with a mean length of ∼12 kb (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). After processing raw data by ToFU pipeline (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>), we obtained 14,868,653 filtered subread data with a mean length of 2,596 bp (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and 1,300,544 CCS reads with average depth of 6–10 passes in three libraries (median, <a reveal-id="dsy014-F2" data-open="dsy014-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>). Then CCS reads were classified into five types as follows: with 5′ adaptor, with 3′ adaptor, with poly-A tail, FL and FLNC (<a reveal-id="dsy014-F1" data-open="dsy014-F1" class="link link-reveal link-table xref-fig">Fig. 1b and c</a>). We detected 517,462 FL reads [containing 5′ and 3′ cDNA synthesis primers and a distinct poly(A) tail], of which 99.46% (514,659/517,462) were defined as FLNC (<a reveal-id="dsy014-T1" data-open="dsy014-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Interestingly, we found that only 1.10% (2,254/206,756) FLNCs were &lt;1 kb in length in 0.6–1 kb library (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), indicating that swine FL transcripts might be more than 1 kb in length while short fragments mainly consisted of non-coding RNAs with a single exon. Thus, future research on FL transcript can mainly focus on longer than 0.6 kb reads. By analysing the distribution of transcripts’ length, we found that PacBio data set could retrieve much longer transcripts than those described in the current SSC10.2 reference annotations (<a reveal-id="dsy014-F2" data-open="dsy014-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). <div content-id="dsy014-T1" class="table-modal"><div class="table-wrap"><div id="dsy014-T1" data-id="dsy014-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Sequence summary of PacBio CCS reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library </th><th>Cell </th><th>CCS </th><th>5' </th><th>3' </th><th>poly-A </th><th>FL </th><th>FLNC </th><th>Average FLNC read length (bp) </th></tr></thead><tbody><tr><td>0.6–1 kb </td><td>7 </td><td>456,861 </td><td>240,085 </td><td>286,237 </td><td>279,184 </td><td>207,721 </td><td>206,756 </td><td>1,560 </td></tr><tr><td>1–2 kb </td><td>7 </td><td>472,850 </td><td>225,442 </td><td>268,959 </td><td>261,714 </td><td>189,761 </td><td>188,735 </td><td>2,087 </td></tr><tr><td>&gt;2 kb </td><td>6 </td><td>370,833 </td><td>155,580 </td><td>192,158 </td><td>185,582 </td><td>119,980 </td><td>119,168 </td><td>2,853 </td></tr><tr><td>Total </td><td>20 </td><td>1,300,544 </td><td>621,107 </td><td>747,354 </td><td>726,480 </td><td>517,462 </td><td>514,659 </td><td> </td></tr></tbody></table></div><div content-id="dsy014-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library </th><th>Cell </th><th>CCS </th><th>5' </th><th>3' </th><th>poly-A </th><th>FL </th><th>FLNC </th><th>Average FLNC read length (bp) </th></tr></thead><tbody><tr><td>0.6–1 kb </td><td>7 </td><td>456,861 </td><td>240,085 </td><td>286,237 </td><td>279,184 </td><td>207,721 </td><td>206,756 </td><td>1,560 </td></tr><tr><td>1–2 kb </td><td>7 </td><td>472,850 </td><td>225,442 </td><td>268,959 </td><td>261,714 </td><td>189,761 </td><td>188,735 </td><td>2,087 </td></tr><tr><td>&gt;2 kb </td><td>6 </td><td>370,833 </td><td>155,580 </td><td>192,158 </td><td>185,582 </td><td>119,980 </td><td>119,168 </td><td>2,853 </td></tr><tr><td>Total </td><td>20 </td><td>1,300,544 </td><td>621,107 </td><td>747,354 </td><td>726,480 </td><td>517,462 </td><td>514,659 </td><td> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy014-T1" data-id="dsy014-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Sequence summary of PacBio CCS reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library </th><th>Cell </th><th>CCS </th><th>5' </th><th>3' </th><th>poly-A </th><th>FL </th><th>FLNC </th><th>Average FLNC read length (bp) </th></tr></thead><tbody><tr><td>0.6–1 kb </td><td>7 </td><td>456,861 </td><td>240,085 </td><td>286,237 </td><td>279,184 </td><td>207,721 </td><td>206,756 </td><td>1,560 </td></tr><tr><td>1–2 kb </td><td>7 </td><td>472,850 </td><td>225,442 </td><td>268,959 </td><td>261,714 </td><td>189,761 </td><td>188,735 </td><td>2,087 </td></tr><tr><td>&gt;2 kb </td><td>6 </td><td>370,833 </td><td>155,580 </td><td>192,158 </td><td>185,582 </td><td>119,980 </td><td>119,168 </td><td>2,853 </td></tr><tr><td>Total </td><td>20 </td><td>1,300,544 </td><td>621,107 </td><td>747,354 </td><td>726,480 </td><td>517,462 </td><td>514,659 </td><td> </td></tr></tbody></table></div><div content-id="dsy014-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library </th><th>Cell </th><th>CCS </th><th>5' </th><th>3' </th><th>poly-A </th><th>FL </th><th>FLNC </th><th>Average FLNC read length (bp) </th></tr></thead><tbody><tr><td>0.6–1 kb </td><td>7 </td><td>456,861 </td><td>240,085 </td><td>286,237 </td><td>279,184 </td><td>207,721 </td><td>206,756 </td><td>1,560 </td></tr><tr><td>1–2 kb </td><td>7 </td><td>472,850 </td><td>225,442 </td><td>268,959 </td><td>261,714 </td><td>189,761 </td><td>188,735 </td><td>2,087 </td></tr><tr><td>&gt;2 kb </td><td>6 </td><td>370,833 </td><td>155,580 </td><td>192,158 </td><td>185,582 </td><td>119,980 </td><td>119,168 </td><td>2,853 </td></tr><tr><td>Total </td><td>20 </td><td>1,300,544 </td><td>621,107 </td><td>747,354 </td><td>726,480 </td><td>517,462 </td><td>514,659 </td><td> </td></tr></tbody></table></div></div></p>                        <a id="120122180" scrollto-destination="120122180"></a>
<div id="dsy014-F2" data-id="dsy014-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f2.png?Expires=2147483647&Signature=Ull~cIHdaWiG3wn48Xw6sNyw5LE2ggYgmlnElpwR~Q65FB50quT0xEFNGvDKNISYidpIdkDkkNXo3Jg4xVeoFmvywmzilDLBMJZiB~aYa0LGTr~eZ9w5imOb0WRLTX77zTozedwCNfZ31SWQ9ZJamBm~Z6W9iqCs5j2ZOFtJN-4RgclmjAiLWteuatDnCV8pZy~VsPfvqMeA7pEP0jfkAF8PWAudyFiwfV7LS2u0d8246JKUbTCu3fOBFrueXpExJd8eA~kHQR-Tdzda~sD1uOcYVTLRsIRX84IpjPOxl66GnXIQj-S8qL~dSKUHdzfAWdnEP7NMubmenHKOCg6D8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length analysis with Iso-seq reads. (a) Mean number of passes of CCS read. (b) Comparison of length between reference annotation and FLNC." path-from-xml="dsy014f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122180/dsy014f2.tif" path-from-xml="dsy014f2.tif" target="_blank">View large</a><a section="120122180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f2.png?Expires=2147483647&Signature=LyVH12z~tAQGcJnv6YJ3WdGkTtOtR0QfaMXBp6AC6Y5rG2jXq3Ks7Y-k90Ru9k08R~~VFDE-gNWXJgI2wGJAj2vpV3BUHY5paFTwc7HgZOdsBKAbHgIQ0oSn~6blGXFt-zvDmG~FXE-mUaeKQs8J-xq3iOOyGvcozYl10Jhin~40Qy2A2uJsnZxhsFtJ3kZoDzfNbcU2cdzkNygxgkyR5z84RTY3uCXWhRSHR0Ep8V~y9rpE~mBEbKbKXUHlIT7jOHcey-XtGGwKOtDKQxbvovhjn6Wp-MqV5jDF1-mYPf8xWKbvEiRsJB~E5WFA4Wy1fNtvucriAS~KX9GAdCacGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122180&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length analysis with Iso-seq reads. (a) Mean number of passes of CCS read. (b) Comparison of length between reference annotation and FLNC.</p></div></div><div content-id="dsy014-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f2.png?Expires=2147483647&Signature=Ull~cIHdaWiG3wn48Xw6sNyw5LE2ggYgmlnElpwR~Q65FB50quT0xEFNGvDKNISYidpIdkDkkNXo3Jg4xVeoFmvywmzilDLBMJZiB~aYa0LGTr~eZ9w5imOb0WRLTX77zTozedwCNfZ31SWQ9ZJamBm~Z6W9iqCs5j2ZOFtJN-4RgclmjAiLWteuatDnCV8pZy~VsPfvqMeA7pEP0jfkAF8PWAudyFiwfV7LS2u0d8246JKUbTCu3fOBFrueXpExJd8eA~kHQR-Tdzda~sD1uOcYVTLRsIRX84IpjPOxl66GnXIQj-S8qL~dSKUHdzfAWdnEP7NMubmenHKOCg6D8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length analysis with Iso-seq reads. (a) Mean number of passes of CCS read. (b) Comparison of length between reference annotation and FLNC." path-from-xml="dsy014f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122180/dsy014f2.tif" path-from-xml="dsy014f2.tif" target="_blank">View large</a><a section="120122180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f2.png?Expires=2147483647&Signature=LyVH12z~tAQGcJnv6YJ3WdGkTtOtR0QfaMXBp6AC6Y5rG2jXq3Ks7Y-k90Ru9k08R~~VFDE-gNWXJgI2wGJAj2vpV3BUHY5paFTwc7HgZOdsBKAbHgIQ0oSn~6blGXFt-zvDmG~FXE-mUaeKQs8J-xq3iOOyGvcozYl10Jhin~40Qy2A2uJsnZxhsFtJ3kZoDzfNbcU2cdzkNygxgkyR5z84RTY3uCXWhRSHR0Ep8V~y9rpE~mBEbKbKXUHlIT7jOHcey-XtGGwKOtDKQxbvovhjn6Wp-MqV5jDF1-mYPf8xWKbvEiRsJB~E5WFA4Wy1fNtvucriAS~KX9GAdCacGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122180&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length analysis with Iso-seq reads. (a) Mean number of passes of CCS read. (b) Comparison of length between reference annotation and FLNC.</p></div></div><p>Considering a high base error rate of SMRT sequencing technology, we used high-quality Illumina short reads to correct erroneous SMRT long reads by Proovread software.<a reveal-id="dsy014-B25" data-open="dsy014-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The FLNC sequences before and after correction were respectively aligned to pig genome sequence through GMAP. After correction, we obtained 389,781 high-quality FLNC for further study (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p>                            <h3 scrollto-destination=120122182 id="120122182" class="section-title" >3.2 Isoform detection and characterization</h3>
<p>To evaluate the density and length of isoforms, we compared the loci coverage of PacBio FLNC and swine SSC 10.2 annotation. In PacBio data set, a total of 389,781 high-quality FLNCs covered 77,075 isoforms and were allocated to 39,940 loci (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). About 96.65% (38,604/39,940) gene transcripts were ≥1 kb in length. In reference annotation, 30,585 isoforms covered 25,322 loci and only 58.72% (14,872/25,322) gene transcripts were ≥1 kb. Our unique isoforms covered about 51.57% (13,059/25,322) of reference annotation loci. In addition, out of 77,075 isoforms, 29,992 (38.91%) were single-exon isoforms and 47,083 (61.09%) were multiple-exon isoforms. Approximately 8,830 loci could produce more than one transcript, accounting for a total of 45,695 isoforms (<a reveal-id="dsy014-F3" data-open="dsy014-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). In contrary, in reference annotation, around half (51.57%, 13,059/25,322) could be detected in Iso-Seq reads, of which only 8,062 genes (∼61.74%) could produce at least two splice isoforms (<a reveal-id="dsy014-F3" data-open="dsy014-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). The gene <em>ACTA1 (</em>gene ID: 14.1882, chr14: 65, 236, 181-65, 239 and 183<em>)</em> had the largest number of isoforms ∼337, which played a vital role in the development of skeletal muscle myofibrils in pig.<a reveal-id="dsy014-B37" data-open="dsy014-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Thus, PacBio FLNC data set provided higher isoform density and longer isoform length than SSC 10.2 annotation, which would be propitious to unveil the comprehensive assessment of the true complexity of the transcriptome for the gene structure annotation.<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p>                        <a id="120122184" scrollto-destination="120122184"></a>
<div id="dsy014-F3" data-id="dsy014-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f3.png?Expires=2147483647&Signature=v0TBcOnAAZW5drJ3AfO0cphY6gp1iw1wQAU74MmOcyP2KpIzI8xDtJGSFdqqe3r7Q2UQ-xODbm6cxnOcoVF-AbQ~vtMjKbbVDzTM8kWNEst3KU66w5cVuAUiSQTmSlknBrbN~yS9Fhj6Uhc2-dPjSpIPFh9Eg4UcndFRxcsVs1EbVXE8Fx5C4o6wNBFK6JMpE9EOI~fGVpRxmVjOz8GBZvOndRSexCrSCLPcdWdasQl7qiWNjDQWtU4d2MHiAXu5J3MUTKNXHqoljTFCZuXkJ1mw1Sij0mGT41K1tzypQ51cja4ciZoHzyuDstoNM57spnwVgtF8iqWj~e9LRj2Cuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of genes that produce splice isoform and AS event with different models. (a) The total number of loci according to isoforms of FLNC in PacBio Iso-deq data. (b) Annotated loci and isoforms were based on FLNC mapped to reference genome. (c) Five basic models of AS: exon skipping, intron retention, alternative donor site, alternative acceptor site and mutually exclusive exons. (d) The total number of AS events in genes based on Iso-seq data compared with the annotated gene models." path-from-xml="dsy014f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122184/dsy014f3.tif" path-from-xml="dsy014f3.tif" target="_blank">View large</a><a section="120122184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f3.png?Expires=2147483647&Signature=dN5qB8Px6i3ExZnu3w-oimkOfmB4NtkdSpJMPeE7fCMNSaP4ktPLx3IPoMg0c3VOLquq09fTA2QdHh~LtSrQJSZcPI~ijaay9K2w5BOjSLB5uEXnlBwuk~aCz4MsMOWLT01YLkZu2bs7MCEUctrAXAufXb4uorjsw4mMUmxeYXvMzzX7jl9l1ZtPsCeWA-4nYYa4gTVovyqLEGO~zZ9KqUkcFpoTEU08ge9bKRe7bsXkwJEyh8t1MLodA38wBEu47fkCTNnHliwf~hes-PLzrcrDVBDzUBECXqOctZd63kyXL04WNZrClw5HHlyNc5CNDtC5zGdrAvFGgGAQII213A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122184&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of genes that produce splice isoform and AS event with different models. (a) The total number of loci according to isoforms of FLNC in PacBio Iso-deq data. (b) Annotated loci and isoforms were based on FLNC mapped to reference genome. (c) Five basic models of AS: exon skipping, intron retention, alternative donor site, alternative acceptor site and mutually exclusive exons. (d) The total number of AS events in genes based on Iso-seq data compared with the annotated gene models.</p></div></div><div content-id="dsy014-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f3.png?Expires=2147483647&Signature=v0TBcOnAAZW5drJ3AfO0cphY6gp1iw1wQAU74MmOcyP2KpIzI8xDtJGSFdqqe3r7Q2UQ-xODbm6cxnOcoVF-AbQ~vtMjKbbVDzTM8kWNEst3KU66w5cVuAUiSQTmSlknBrbN~yS9Fhj6Uhc2-dPjSpIPFh9Eg4UcndFRxcsVs1EbVXE8Fx5C4o6wNBFK6JMpE9EOI~fGVpRxmVjOz8GBZvOndRSexCrSCLPcdWdasQl7qiWNjDQWtU4d2MHiAXu5J3MUTKNXHqoljTFCZuXkJ1mw1Sij0mGT41K1tzypQ51cja4ciZoHzyuDstoNM57spnwVgtF8iqWj~e9LRj2Cuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of genes that produce splice isoform and AS event with different models. (a) The total number of loci according to isoforms of FLNC in PacBio Iso-deq data. (b) Annotated loci and isoforms were based on FLNC mapped to reference genome. (c) Five basic models of AS: exon skipping, intron retention, alternative donor site, alternative acceptor site and mutually exclusive exons. (d) The total number of AS events in genes based on Iso-seq data compared with the annotated gene models." path-from-xml="dsy014f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122184/dsy014f3.tif" path-from-xml="dsy014f3.tif" target="_blank">View large</a><a section="120122184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f3.png?Expires=2147483647&Signature=dN5qB8Px6i3ExZnu3w-oimkOfmB4NtkdSpJMPeE7fCMNSaP4ktPLx3IPoMg0c3VOLquq09fTA2QdHh~LtSrQJSZcPI~ijaay9K2w5BOjSLB5uEXnlBwuk~aCz4MsMOWLT01YLkZu2bs7MCEUctrAXAufXb4uorjsw4mMUmxeYXvMzzX7jl9l1ZtPsCeWA-4nYYa4gTVovyqLEGO~zZ9KqUkcFpoTEU08ge9bKRe7bsXkwJEyh8t1MLodA38wBEu47fkCTNnHliwf~hes-PLzrcrDVBDzUBECXqOctZd63kyXL04WNZrClw5HHlyNc5CNDtC5zGdrAvFGgGAQII213A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122184&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of genes that produce splice isoform and AS event with different models. (a) The total number of loci according to isoforms of FLNC in PacBio Iso-deq data. (b) Annotated loci and isoforms were based on FLNC mapped to reference genome. (c) Five basic models of AS: exon skipping, intron retention, alternative donor site, alternative acceptor site and mutually exclusive exons. (d) The total number of AS events in genes based on Iso-seq data compared with the annotated gene models.</p></div></div>                            <h3 scrollto-destination=120122185 id="120122185" class="section-title" >3.3 Novel genes identification</h3>
<p>Compared with reference annotation, over 69% (193,318/277,842) of high-quality aligned FLNC had the same initial or terminated sites as mRNAs in annotated database, implying a relatively high integrity in structure. Considering the preferable integrity of the FLNC at the 3′ end based on the characteristics of library construction, gene structure integrity of FLNC was only estimated at the 5′ end of the sequence. After removing redundancies and false positives, a total of 237,580 FLNC (∼85.51%) contained the same initial splice site sequences with the reference annotation, covering 29,036 isoforms (∼68.34%, <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Moreover, a total of 14,792 FLNC isoforms from 6,105 genes maintained the same structures as annotation, which is similar to previous report that ∼1,4000 FL personal transcripts from complement of a pooled set of 20 human organs and tissues in a single-molecule level.<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p><p>The published pig genome annotation contains ∼25,322 gene models with 30,585 isoforms. In PacBio data set, 26,881 unique transcript clusters did not overlap with any annotated gene, which likely originated from novel genes (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s7.xlsx?Expires=2147483647&amp;Signature=E0cuVHQ1hjFCC~AI5VrDwRmuylYgQvz2shcXcnA~zA9kZePxlm5zcLQeYapVFj1Hu2ntq~A5cX-6NT1pj0lg-NNOxHgIHXzPGAuWojT8O9ofWuPzQVBK56RqQjxo1DtWfh7Vqd4pzpahnPurdY6mg3~hE3eOG1U~E02S4DitkdMgZhCl13-ysduUQClpGrPZQt1IjYYkkkptpUJzotZ3PkbHxHLfkJDRlWPPKRoRkRKHO0ZUp6Vr1UByd9JYNZiHOP1CcCxXOgySOeyq1gBneDPqXaRqiF4eM15LC2GADXoDXQgisoDwdTq8qz5voGUtHm4F7KsoVZQTd~n4AUpgbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Even so, about 13,000 known loci (32.70%) could be mapped to 12,278 reference annotated genes (48.49% of the 25,322 gene). Thus, Iso-Seq data displayed a good example that one gene was annotated to produce a single transcript but was found to generate four splice variants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1a</a></span>). Novel annotation of Iso-Seq could rectify the incorrect position annotation of exon of the reference (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1b</a></span>). Novel genes have emerged as a new structure and filled the blank position without annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1c</a></span>).</p><p>Gene annotation for the 26,881 new loci showed that 23,712 loci were single-exon loci and 3,169 were multiple-exon loci (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s7.xlsx?Expires=2147483647&Signature=E0cuVHQ1hjFCC~AI5VrDwRmuylYgQvz2shcXcnA~zA9kZePxlm5zcLQeYapVFj1Hu2ntq~A5cX-6NT1pj0lg-NNOxHgIHXzPGAuWojT8O9ofWuPzQVBK56RqQjxo1DtWfh7Vqd4pzpahnPurdY6mg3~hE3eOG1U~E02S4DitkdMgZhCl13-ysduUQClpGrPZQt1IjYYkkkptpUJzotZ3PkbHxHLfkJDRlWPPKRoRkRKHO0ZUp6Vr1UByd9JYNZiHOP1CcCxXOgySOeyq1gBneDPqXaRqiF4eM15LC2GADXoDXQgisoDwdTq8qz5voGUtHm4F7KsoVZQTd~n4AUpgbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). To validate the unannotated novel loci, we searched these 26,881 new genes in NR, KOG, KO and GO databases using BLASTX (<em>e</em>-value ≤1<em>e</em>−5). It showed that 10,299 (38.31%), 800 (2.98%), 1,567 (5.83%) and 2,803 (10.43%) of 26,881 new genes could be found in NR, KOG, KO and GO databases, respectively. A total of 417 novel genes had significant hits in the four databases (<a reveal-id="dsy014-F4" data-open="dsy014-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). At the same time, a large number of unannotated single-exon genes might contain some non-coding RNAs, because coding genes usually had multiple exons which were overwhelmingly alternatively spliced.<a reveal-id="dsy014-B10" data-open="dsy014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy014-B38" data-open="dsy014-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p>                        <a id="120122189" scrollto-destination="120122189"></a>
<div id="dsy014-F4" data-id="dsy014-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f4.png?Expires=2147483647&Signature=JNesOvVgJMChUNY1lsjGjFbKityj~lSvD~YmVufGniMVdbyeX8rOjOCw0jP2chGlrL12h~zjO9p~LjojYr~XsToQM1hLxbWfvJBNZ04JZTfGS~eNYlJA7oj42l3Y-JUTZN6aAJlXNwwa~8kyW2PF6aN47tq7HTvPFdEAGkGokPT28eZp0ud2MjW6C8xt1DuJtLeLbnx7~TIgPKeC3aj7ia3KXvrNR0cseNVMxP~81UbNmXV1EvzeuoxLx3VdFg~nKWdUijhlv9It2MXN5n4~sKm90M1zjLrm~F1b4zsZqGR8KOPk0iSSkvqU-jpviBlh8tOqtz63Tyd4FlHtcCtGVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Novel genes identified in Iso-seq data. The number represents novel genes identified in NR, KOG, KO and GO databases." path-from-xml="dsy014f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122189/dsy014f4.tif" path-from-xml="dsy014f4.tif" target="_blank">View large</a><a section="120122189" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f4.png?Expires=2147483647&Signature=X8H8g-IVvntTjWtuvx3LuIkGQArHNJNX9dUN-SIImqUcGAIQuvwxpSvHcIVzhm2RR9hKU3cieWd9zCLScW8u35nK9i~aNTZRVVQnGcsnjP-XoII4QNkbk2kcjKkN3Ss7zQED7zCwY8Fz6BKpjdl5SvSwrAaQtroMoS1PhJAXlO6UTeoPrHQeChbyT3qoSgguOleh48YVnPx~kAZahUnEzqL0pbDW5Ce7Db7IkaNpUqWVZtQHwJrI3TAjPOsrcp4lAaLhr~ZBl0bE7R-zlIaH5cE0D2FpIDn~3V8hYODFsnyh4lI3bzEjzuUWT9GXgL~OSpofHNxs-0WYvPtf786iYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122189&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Novel genes identified in Iso-seq data. The number represents novel genes identified in NR, KOG, KO and GO databases.</p></div></div><div content-id="dsy014-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f4.png?Expires=2147483647&Signature=JNesOvVgJMChUNY1lsjGjFbKityj~lSvD~YmVufGniMVdbyeX8rOjOCw0jP2chGlrL12h~zjO9p~LjojYr~XsToQM1hLxbWfvJBNZ04JZTfGS~eNYlJA7oj42l3Y-JUTZN6aAJlXNwwa~8kyW2PF6aN47tq7HTvPFdEAGkGokPT28eZp0ud2MjW6C8xt1DuJtLeLbnx7~TIgPKeC3aj7ia3KXvrNR0cseNVMxP~81UbNmXV1EvzeuoxLx3VdFg~nKWdUijhlv9It2MXN5n4~sKm90M1zjLrm~F1b4zsZqGR8KOPk0iSSkvqU-jpviBlh8tOqtz63Tyd4FlHtcCtGVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Novel genes identified in Iso-seq data. The number represents novel genes identified in NR, KOG, KO and GO databases." path-from-xml="dsy014f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122189/dsy014f4.tif" path-from-xml="dsy014f4.tif" target="_blank">View large</a><a section="120122189" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f4.png?Expires=2147483647&Signature=X8H8g-IVvntTjWtuvx3LuIkGQArHNJNX9dUN-SIImqUcGAIQuvwxpSvHcIVzhm2RR9hKU3cieWd9zCLScW8u35nK9i~aNTZRVVQnGcsnjP-XoII4QNkbk2kcjKkN3Ss7zQED7zCwY8Fz6BKpjdl5SvSwrAaQtroMoS1PhJAXlO6UTeoPrHQeChbyT3qoSgguOleh48YVnPx~kAZahUnEzqL0pbDW5Ce7Db7IkaNpUqWVZtQHwJrI3TAjPOsrcp4lAaLhr~ZBl0bE7R-zlIaH5cE0D2FpIDn~3V8hYODFsnyh4lI3bzEjzuUWT9GXgL~OSpofHNxs-0WYvPtf786iYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122189&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Novel genes identified in Iso-seq data. The number represents novel genes identified in NR, KOG, KO and GO databases.</p></div></div>                            <h3 scrollto-destination=120122190 id="120122190" class="section-title" >3.4 Various types of alternative splicing</h3>
<p>AS is an important mechanism of generating regulatory function for trait expression of eukaryotes.<a reveal-id="dsy014-B39" data-open="dsy014-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Through alternative recognition of exon and splice site during splicing, a single gene can generate functionally distinct mRNA and diverse protein isoforms. AS formation has five basic types as follows (<a reveal-id="dsy014-F3" data-open="dsy014-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>): exon skipping (ES), intron retention (IR), mutually exclusive exons (MEE), alternative donor site (AD) and alternative acceptor site (AA). Previously, ∼30% of porcine genes were reported to undergo AS using ESTs.<a reveal-id="dsy014-B7" data-open="dsy014-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In this study, we explored a more comprehensive formation and distribution of AS event in pig. In reference annotation, a total of 7.77% (1,967/25,322) genes corresponding to 17.45% (5,336/30,585) isoforms experienced 5,417 AS events (2.71 isoforms and 2.75 AS in each gene). In contrary, 17.66% loci (7,053/39,940 loci) corresponding to 42.38% (32,662/77,075) isoforms underwent 97,727 AS events in PacBio data set (4.63 isoforms and 13.86 AS in each gene, <a reveal-id="dsy014-F3" data-open="dsy014-F3" class="link link-reveal link-table xref-fig">Fig. 3d</a>, <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s8.xlsx?Expires=2147483647&amp;Signature=xXOmHahVg~Uy0JLL4Z9~uvjrTzQX4Gm63dmSkjOD27f2YXBXR~3lDIbtDKWRtQ07HhS665pzo-XDDmCCCaDN3bYmI18eSwndTkns95OLndYT2F94mZ~B7T7RmS7btFt2Oq7k0NLjcj2F6gG5ZilwiDfpYkoO4iIxkcGefstoAg2v4K7nbWOquVClBmNr3UoPpiuDjAScfQ0FmcuhQ66LV4vGPewf23CET-wbn8mLJtS11sHvhfRsEIrgG3RkmiqeHihkUdK2gdr-Z53SCl1l~bc14Q6Ypihro9UJenf-bGtx04-XdB0NefKThLy7~4Zrvyfw-xwYwRblCTuEs9HUFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). This result indicated that the prevalence of AS event was much higher than previously thought. Interestingly, five basic AS models only accounted for 21.42% (20,930/97,727) of AS events. IR predominated, accounting for 10.16% of alternative transcripts, while MEE (0.34%) was the least frequent. More AS events evolved from combinations of five basic patterns. For example, IR event could appear in different sites within a gene, AA event might be more prone to the 3′ terminal exon and ES event would appear in different combinations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1d–f</a></span>).</p><p>On whole-genome level, 1,150 AS models in reference annotation and 7,132 models in Iso-Seq reads were detected (<a reveal-id="dsy014-F5" data-open="dsy014-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In contrary, by removing the duplication among different chromosomes, 97,727 AS events with 2,637 models in PacBio Iso-Seq data were identified, of which more than 92,000 were new AS events with 2,100 modes (<a reveal-id="dsy014-F3" data-open="dsy014-F3" class="link link-reveal link-table xref-fig">Fig. 3d</a>). In detail, <em>GANS</em> gene has the largest number of splice variants (∼23 transcripts) in present reference annotation. However, we found 35 novel transcripts for <em>GANS</em> gene in Iso-Seq data, which implied a more complex imprinted expression pattern closely related to tumour therapy.<a reveal-id="dsy014-B40" data-open="dsy014-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Moreover, <em>ACTA1</em> gene expressed only one annotated transcript in reference but exhibited 337 splice variants in our analysis. Its specific expression in muscle provides therapeutic strategies for thin filament myopathy patients<a reveal-id="dsy014-B41" data-open="dsy014-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and a method for improving muscle mass in livestock.<a reveal-id="dsy014-B42" data-open="dsy014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> These results suggested that AS events were severely underestimated in present pig genome annotations. </p>                        <a id="120122193" scrollto-destination="120122193"></a>
<div id="dsy014-F5" data-id="dsy014-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f5.png?Expires=2147483647&Signature=Dnkkg79MCZ8so1gJ-kT8q-4za6SnmHIJwl4kR76FPxGw9o5OktzE8wn9L-05t5Z7ESPmpU3aXCteGzJ5QDFmFdZ6~moZSJTbqMj92GpQG6mN4qfXvr3fYf3Izx8QGzO-LxrXiAOheGMZPkoW~bMXUifgEKFx6NfyPOqjWhdKRqj5OtLP3499w5R0s9pNxFwCNjpep0Mk0xYUeKzYNqycdM3x3Huz-B7xlqqNY24YCn-gbv35vRr8MokV~rjFXAKOPiw-upmmlKlTy9UdL~fx07mqaz4tASqFlh3lfLZ5DZrLopnk-uXMpfKzcpNnFFlGI22nerllT1KvTtAHU7~1cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributed visualization using Circos of different data at chromosome level. The total length of the half circle corresponding to each label is the sum of all the values corresponding to the label. The connection between the different half circles indicates the value expressed by the two tags. A and B represent the number distribution of AS events in reference annotation and Iso-seq data, respectively. C and D correspond to number distribution of AS models in reference annotation and Iso-seq data, respectively. E and F exhibit number of gene in annotation and Iso-seq data, respectively. G and H denote fusion gene and fusion isoform in Iso-seq data, respectively. Chr represents chromosome; JH and GL represent the scaffolds that have not assembled on the chromosome." path-from-xml="dsy014f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122193/dsy014f5.tif" path-from-xml="dsy014f5.tif" target="_blank">View large</a><a section="120122193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f5.png?Expires=2147483647&Signature=0ZjTwkXBWg34DkFF9OmbX0DlhFLNY6M81rFE4zL4urdIY-ApiwoVbrPGOyVbIXBaQSyFNkMIfZblcp~KmjvBAYfcndavoDwFDCyIkamIEsGaQj-wdWbg2aGcZbfaDQgqrOD1hxFkLGGgx~~n78qetc3qNAXNfqwM3rl-Ehc~GYgSpFVhYd960mcZlT~e2aXMpa3L7nZ2hByXriZsDF4oadnm7b~ZX6eO62vshsSQxsrNt76-bpnPY2~MeCrIxeJnDIdbu7Qm~0acpCpuvhK1bADgHUBQLf1Fk9LwncNkeOicKxq-ImSQu2QdNlbHB61YEIZcOAi8Gh9oMfU9vjWZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122193&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributed visualization using Circos of different data at chromosome level. The total length of the half circle corresponding to each label is the sum of all the values corresponding to the label. The connection between the different half circles indicates the value expressed by the two tags. A and B represent the number distribution of AS events in reference annotation and Iso-seq data, respectively. C and D correspond to number distribution of AS models in reference annotation and Iso-seq data, respectively. E and F exhibit number of gene in annotation and Iso-seq data, respectively. G and H denote fusion gene and fusion isoform in Iso-seq data, respectively. Chr represents chromosome; JH and GL represent the scaffolds that have not assembled on the chromosome.</p></div></div><div content-id="dsy014-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f5.png?Expires=2147483647&Signature=Dnkkg79MCZ8so1gJ-kT8q-4za6SnmHIJwl4kR76FPxGw9o5OktzE8wn9L-05t5Z7ESPmpU3aXCteGzJ5QDFmFdZ6~moZSJTbqMj92GpQG6mN4qfXvr3fYf3Izx8QGzO-LxrXiAOheGMZPkoW~bMXUifgEKFx6NfyPOqjWhdKRqj5OtLP3499w5R0s9pNxFwCNjpep0Mk0xYUeKzYNqycdM3x3Huz-B7xlqqNY24YCn-gbv35vRr8MokV~rjFXAKOPiw-upmmlKlTy9UdL~fx07mqaz4tASqFlh3lfLZ5DZrLopnk-uXMpfKzcpNnFFlGI22nerllT1KvTtAHU7~1cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distributed visualization using Circos of different data at chromosome level. The total length of the half circle corresponding to each label is the sum of all the values corresponding to the label. The connection between the different half circles indicates the value expressed by the two tags. A and B represent the number distribution of AS events in reference annotation and Iso-seq data, respectively. C and D correspond to number distribution of AS models in reference annotation and Iso-seq data, respectively. E and F exhibit number of gene in annotation and Iso-seq data, respectively. G and H denote fusion gene and fusion isoform in Iso-seq data, respectively. Chr represents chromosome; JH and GL represent the scaffolds that have not assembled on the chromosome." path-from-xml="dsy014f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122193/dsy014f5.tif" path-from-xml="dsy014f5.tif" target="_blank">View large</a><a section="120122193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f5.png?Expires=2147483647&Signature=0ZjTwkXBWg34DkFF9OmbX0DlhFLNY6M81rFE4zL4urdIY-ApiwoVbrPGOyVbIXBaQSyFNkMIfZblcp~KmjvBAYfcndavoDwFDCyIkamIEsGaQj-wdWbg2aGcZbfaDQgqrOD1hxFkLGGgx~~n78qetc3qNAXNfqwM3rl-Ehc~GYgSpFVhYd960mcZlT~e2aXMpa3L7nZ2hByXriZsDF4oadnm7b~ZX6eO62vshsSQxsrNt76-bpnPY2~MeCrIxeJnDIdbu7Qm~0acpCpuvhK1bADgHUBQLf1Fk9LwncNkeOicKxq-ImSQu2QdNlbHB61YEIZcOAi8Gh9oMfU9vjWZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122193&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distributed visualization using Circos of different data at chromosome level. The total length of the half circle corresponding to each label is the sum of all the values corresponding to the label. The connection between the different half circles indicates the value expressed by the two tags. A and B represent the number distribution of AS events in reference annotation and Iso-seq data, respectively. C and D correspond to number distribution of AS models in reference annotation and Iso-seq data, respectively. E and F exhibit number of gene in annotation and Iso-seq data, respectively. G and H denote fusion gene and fusion isoform in Iso-seq data, respectively. Chr represents chromosome; JH and GL represent the scaffolds that have not assembled on the chromosome.</p></div></div><p>Interestingly, we found that AS events (23,440 with 555 models) predominantly took place in chromosome 14. To detect whether such a situation existed in other species, we downloaded reference annotations of six species, including horse, cattle, sheep, human, mouse and pig, from ensemble (release-84, <a class="link link-uri" href="http://ftp://ftp.ensembl.org/pub/release-84/gtf/" target="_blank">ftp://ftp.ensembl.org/pub/release-84/gtf/</a>) for AS analysis (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s2.png?Expires=2147483647&Signature=pL21NAjwIvUzKk5qnDoMMVQNBfsUqXNgB9Q03ZtkcTTsEnsUEBk3G4t7yexITMfctZl7vpyNW6VP-bNFYpNNxb7mc3K-zfM6fmielPdqa~91wJ0tw2L93ycwpV1Jp-F0~302tWJ6XOMvdpbqwwii6BLY5tlTc640YCbET0RjDXCJfn56UmPWNgCc5F0XD~ph6mpD7tXsvcX7os6pVDaEqwbOp3Oem1vZFGBCOh2bKnegcUHYZOCmOxCKyFOSL8OH2LVkdvYrgPRRCYCvjbACOSUIowHAV23g3h0Tn7mrFr0rAk7j~bmLfNy9P2DozD4HoPyQ69rs7WiL15H6QGJDPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). In other five species, correlation coefficient between the number of AS events and gene numbers was very high (<em>r</em> = 0.83–0.94). However, the correlation coefficient in pig from reference annotation was fairly low (<em>r</em> = 0.35), which was similar to the results in Iso-Seq data (<em>r</em> = 0.49). The number of AS events did not significantly grow with the increase of the number of genes on chromosome in pig, indicating that a species specificity was presented on the pig.</p>                            <h3 scrollto-destination=120122195 id="120122195" class="section-title" >3.5 DNA methylation regulates alternative splicing variation</h3>
<p>Given that exons are often more highly methylated than introns,<a reveal-id="dsy014-B43" data-open="dsy014-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsy014-B44" data-open="dsy014-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> DNA methylation is often considered as a strong marker for exon–intron (EI) boundaries during splicing.<a reveal-id="dsy014-B45" data-open="dsy014-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsy014-B46" data-open="dsy014-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The large number of isoforms identified via long-read sequencing in this work provided a good opportunity to investigate the relationship between DNA methylation and AS. To determine whether methylation level is related to AS, DNA methylation level of various isoforms was investigated. We stacked all splice junctions of the PacBio isoforms and measured the methylation levels of three types of cytosines on both strands in each methylation context at donor and acceptor sites. The results revealed that CHG methylation was enriched in acceptor sites, whereas CG methylation was elevated at donor sites (<a reveal-id="dsy014-F6" data-open="dsy014-F6" class="link link-reveal link-table xref-fig">Fig. 6a–c</a>). The CHH and CHG methylation levels were extremely low, whereas CG methylation predominated in pig. This result was consistent with a previous study that three types of methylated motif existed in plants, whereas more CG methylation existed in vertebrates.<a reveal-id="dsy014-B47" data-open="dsy014-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Simultaneously, we found that methylation levels of the longest transcripts were consistent with the methylation of all transcripts. The methylation levels at splice sites did not change significantly with the increase of the number of isoforms (<a reveal-id="dsy014-F6" data-open="dsy014-F6" class="link link-reveal link-table xref-fig">Fig. 6d–f</a>). However, we discovered that CG methylation levels dropped sharply (at least 10 times) at splice sites and then quickly increased in the range of sense strand with 3 bp and antisense strand with 2 bp [both EI and intron–exon (IE)] (<a reveal-id="dsy014-F7" data-open="dsy014-F7" class="link link-reveal link-table xref-fig">Fig. 7a</a>). The tremendous changes might be closely related to AS. </p>                        <a id="120122197" scrollto-destination="120122197"></a>
<div id="dsy014-F6" data-id="dsy014-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f6.png?Expires=2147483647&Signature=DZdxAl2Ht4geN0qWcwagpQ6KhLgzylhKtaT9aimuS5-xmYrJd-DYZaJUWkqiFfrGGYQDXX-yEI~J1XWbqp15UlKxfxxfhZEJ7IUSm1N~MfMfk-vcJTuku-Miu2pg5EWy03hVcYYT1SJ3-dMtVEOL-FSNVbEfKD6v1o95LdDezQ-89NhuvpzmFWw2dcy9LgXHT9FslrAgylVKhgVU-exgUiC0p5tXQSqMjCxVJC~HsEkTNI77OpfGbWxVXHDrsxVzOpCZ-fuM2H8yzHak1v03jPwdpFjSqoREEFOmTIjc3zZ34glJRqVhJUikkVF7X4XSGukpFkTgN4SLlmqapUNZqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Level of DNA methylation at splice sites (exon–intron, intron–exon). (a)–(c): For each row, level of DNA methylation, combing sense and antisense strand, on the sense strand, on the antisense strand. (d)–(f): For each row, level of DNA methylation in genes with only one isoform, in all isoforms of genes with 2–10 isoforms, in all isoforms of genes with more than 10 isoforms." path-from-xml="dsy014f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122197/dsy014f6.tif" path-from-xml="dsy014f6.tif" target="_blank">View large</a><a section="120122197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f6.png?Expires=2147483647&Signature=yiLDI7RCjFFQLauqnlsV16OmIZgwnYHGmZBtBvyJulMkckd6RxGNGf9lnZpooMlccvmyCJGd5LfYD4WxUOkZSi67NSEDfS7HrhtCS3xSs6kLQkEXsXYblQ0zOUTzY8ul2z0zVHhvVu52zUKWEYZWTf6-bkLXQDhtkEmebyv8WD~JD800Dw3Tt8SUH3MJ8BGOYExGgwDbCYx4PQGNyqb9d9otuvFE881WkFPu6rTT~ss44I5xsciH3v8q4suZuXHt~d07JYQthDBWVRWWe2n4~w783f82F3WO4suDZH2VX20KPfzmd27AyzqKlD9UADkKIPxJOOspO3migYYV-wG2nQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122197&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Level of DNA methylation at splice sites (exon–intron, intron–exon). (a)–(c): For each row, level of DNA methylation, combing sense and antisense strand, on the sense strand, on the antisense strand. (d)–(f): For each row, level of DNA methylation in genes with only one isoform, in all isoforms of genes with 2–10 isoforms, in all isoforms of genes with more than 10 isoforms.</p></div></div><div content-id="dsy014-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f6.png?Expires=2147483647&Signature=DZdxAl2Ht4geN0qWcwagpQ6KhLgzylhKtaT9aimuS5-xmYrJd-DYZaJUWkqiFfrGGYQDXX-yEI~J1XWbqp15UlKxfxxfhZEJ7IUSm1N~MfMfk-vcJTuku-Miu2pg5EWy03hVcYYT1SJ3-dMtVEOL-FSNVbEfKD6v1o95LdDezQ-89NhuvpzmFWw2dcy9LgXHT9FslrAgylVKhgVU-exgUiC0p5tXQSqMjCxVJC~HsEkTNI77OpfGbWxVXHDrsxVzOpCZ-fuM2H8yzHak1v03jPwdpFjSqoREEFOmTIjc3zZ34glJRqVhJUikkVF7X4XSGukpFkTgN4SLlmqapUNZqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Level of DNA methylation at splice sites (exon–intron, intron–exon). (a)–(c): For each row, level of DNA methylation, combing sense and antisense strand, on the sense strand, on the antisense strand. (d)–(f): For each row, level of DNA methylation in genes with only one isoform, in all isoforms of genes with 2–10 isoforms, in all isoforms of genes with more than 10 isoforms." path-from-xml="dsy014f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122197/dsy014f6.tif" path-from-xml="dsy014f6.tif" target="_blank">View large</a><a section="120122197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f6.png?Expires=2147483647&Signature=yiLDI7RCjFFQLauqnlsV16OmIZgwnYHGmZBtBvyJulMkckd6RxGNGf9lnZpooMlccvmyCJGd5LfYD4WxUOkZSi67NSEDfS7HrhtCS3xSs6kLQkEXsXYblQ0zOUTzY8ul2z0zVHhvVu52zUKWEYZWTf6-bkLXQDhtkEmebyv8WD~JD800Dw3Tt8SUH3MJ8BGOYExGgwDbCYx4PQGNyqb9d9otuvFE881WkFPu6rTT~ss44I5xsciH3v8q4suZuXHt~d07JYQthDBWVRWWe2n4~w783f82F3WO4suDZH2VX20KPfzmd27AyzqKlD9UADkKIPxJOOspO3migYYV-wG2nQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122197&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Level of DNA methylation at splice sites (exon–intron, intron–exon). (a)–(c): For each row, level of DNA methylation, combing sense and antisense strand, on the sense strand, on the antisense strand. (d)–(f): For each row, level of DNA methylation in genes with only one isoform, in all isoforms of genes with 2–10 isoforms, in all isoforms of genes with more than 10 isoforms.</p></div></div>                        <a id="120122198" scrollto-destination="120122198"></a>
<div id="dsy014-F7" data-id="dsy014-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f7.png?Expires=2147483647&Signature=QGR06UKXYhmxxITjK0sC651gnLdpLy9SzUrsbJ~M8PEb0-JzZ0KFewdrIL5NWGA9J1c0XmRFvEI4OG7783zejPLGnHoOrhaEfHuWislSzWNfhCwYBKVwh6mytg5CSldhKFyLqNqaQGBpJEgQs0Pc2R4zmwA7FvPvo3HFR~JAFJntgMrn~-~FlotZYyIhZnKhnrVKJTHpl2C2k54c7Li5ZAbmZtYdRDtfVoDWfILcSXr9hCOE6YtDDTWstlvhf9Tkkasw627sk6y3v6tQvhOp~QcqdmDmpmVTjgI~JiJ6JzXXhuASky64ghS~mRitU1LmZtslREmfARsESgeLWczuog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CG methylation and isoform. (a) Level of CG methylation on the sense strand and antisense strand at splice sites. (b) The relationship between methylation of gene and isoform. (c) Methylation of exon. (exon1: the first exon; exon: other exons excluding the first exon). (d) The relationship between gene length and number of isoform. In (b), (c) and (d), type a: one isoform; type b: 2–10 isoforms; type c: more than 10 isoforms." path-from-xml="dsy014f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122198/dsy014f7.tif" path-from-xml="dsy014f7.tif" target="_blank">View large</a><a section="120122198" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f7.png?Expires=2147483647&Signature=n5oyR7u9AjpCw-us5YLEkRcWwdXorzI4xB-pnZZhHRkaQ~PIrGw~GY0b-1lp7NNkAl9J5kuyH9BpeUgfv51vOa0~p3H25UOHCfcsbgmsQ9AcYJk3vgR9tUmqWfeiJMPOzMNMmnxfZVYSYr5sRdKRibCMwkH2MbC2k~J-5x3YMnx0bhn8IBAOP4Hin6B3oDLTOwGZBwvtlj-~9Zmdgg~-68yM9RG~LMwGsb~1dypDC~cInEwp15HimJWTlqxZmMg1lriePfqZ7INJMAB68pJw98yQgjHNIoSGbUY3YmR57KQhMHdhNo~FB5Ec98vfRyY4~HIH02piulK~gFNyh7G1ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122198&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CG methylation and isoform. (a) Level of CG methylation on the sense strand and antisense strand at splice sites. (b) The relationship between methylation of gene and isoform. (c) Methylation of exon. (exon1: the first exon; exon: other exons excluding the first exon). (d) The relationship between gene length and number of isoform. In (b), (c) and (d), type a: one isoform; type b: 2–10 isoforms; type c: more than 10 isoforms.</p></div></div><div content-id="dsy014-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f7.png?Expires=2147483647&Signature=QGR06UKXYhmxxITjK0sC651gnLdpLy9SzUrsbJ~M8PEb0-JzZ0KFewdrIL5NWGA9J1c0XmRFvEI4OG7783zejPLGnHoOrhaEfHuWislSzWNfhCwYBKVwh6mytg5CSldhKFyLqNqaQGBpJEgQs0Pc2R4zmwA7FvPvo3HFR~JAFJntgMrn~-~FlotZYyIhZnKhnrVKJTHpl2C2k54c7Li5ZAbmZtYdRDtfVoDWfILcSXr9hCOE6YtDDTWstlvhf9Tkkasw627sk6y3v6tQvhOp~QcqdmDmpmVTjgI~JiJ6JzXXhuASky64ghS~mRitU1LmZtslREmfARsESgeLWczuog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CG methylation and isoform. (a) Level of CG methylation on the sense strand and antisense strand at splice sites. (b) The relationship between methylation of gene and isoform. (c) Methylation of exon. (exon1: the first exon; exon: other exons excluding the first exon). (d) The relationship between gene length and number of isoform. In (b), (c) and (d), type a: one isoform; type b: 2–10 isoforms; type c: more than 10 isoforms." path-from-xml="dsy014f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122198/dsy014f7.tif" path-from-xml="dsy014f7.tif" target="_blank">View large</a><a section="120122198" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f7.png?Expires=2147483647&Signature=n5oyR7u9AjpCw-us5YLEkRcWwdXorzI4xB-pnZZhHRkaQ~PIrGw~GY0b-1lp7NNkAl9J5kuyH9BpeUgfv51vOa0~p3H25UOHCfcsbgmsQ9AcYJk3vgR9tUmqWfeiJMPOzMNMmnxfZVYSYr5sRdKRibCMwkH2MbC2k~J-5x3YMnx0bhn8IBAOP4Hin6B3oDLTOwGZBwvtlj-~9Zmdgg~-68yM9RG~LMwGsb~1dypDC~cInEwp15HimJWTlqxZmMg1lriePfqZ7INJMAB68pJw98yQgjHNIoSGbUY3YmR57KQhMHdhNo~FB5Ec98vfRyY4~HIH02piulK~gFNyh7G1ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122198&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CG methylation and isoform. (a) Level of CG methylation on the sense strand and antisense strand at splice sites. (b) The relationship between methylation of gene and isoform. (c) Methylation of exon. (exon1: the first exon; exon: other exons excluding the first exon). (d) The relationship between gene length and number of isoform. In (b), (c) and (d), type a: one isoform; type b: 2–10 isoforms; type c: more than 10 isoforms.</p></div></div><p>Previous research reported that the CG locus in the genome was highly methylated and the methylation of CpG islands in the promoter region of the gene could regulate gene expression.<a reveal-id="dsy014-B47" data-open="dsy014-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> To determine whether the methylation level of gene was related to the number of isoforms, we divided the isoforms into three groups as follows: type a, genes with only one isoform; type b, genes with two to ten isoforms and type c, genes with more than 10 isoforms. Gene methylation was divided into upstream region (promoters encompass 2 kb upstream of the transcriptional start site), gene body and downstream region. Interestingly, CG methylation of promoter was negatively correlated with number of isoforms, suggesting that a high level of CG methylation at promoter region repressed AS and gene expression (<a reveal-id="dsy014-F7" data-open="dsy014-F7" class="link link-reveal link-table xref-fig">Fig. 7b</a>). Some evidence proved that the region near the promoters of many active genes tended to be unmethylated.<a reveal-id="dsy014-B48" data-open="dsy014-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> We also found that the downward trend of methylation level of exon and promoter was insignificant when isoforms were more than 10. To determine whether the first exon was affected by CG methylation, we analysed the relationship between CG methylation of the first exon and the number of isoforms. These results showed that AS was negatively correlated with CG methylation of the first exon, suggesting that a low level of CG methylation at first exon could promote AS formations (<a reveal-id="dsy014-F7" data-open="dsy014-F7" class="link link-reveal link-table xref-fig">Fig. 7c</a>). Furthermore, we determined whether the number of isoforms was affected by the gene length. The results showed that the longer the gene, the more isoforms were produced, but the produced isoforms would be stable when the number of isoforms arrived at a certain level (<a reveal-id="dsy014-F7" data-open="dsy014-F7" class="link link-reveal link-table xref-fig">Fig. 7d</a>). The results showed that AS was not correlated with the number of genes on chromosome in pig but was closely linked to gene length and promoter methylation.</p>                            <h3 scrollto-destination=120122200 id="120122200" class="section-title" >3.6 Fusion isoform identification and characterization</h3>
<p>A fusion gene was an aberrant gene formed by the concatenation of two separate genes. Long-read alignments can actually determine the EI structure of fusion genes. In this study, we identified 711 isoforms and they could simultaneously cover two or more annotated genes (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s9.xlsx?Expires=2147483647&amp;Signature=LIDyG6fFeriAmmKf6xq7lMd6oRxqVZ7q2XfRCcNZ4QtfzscvguU~nKDeeVW9zAvanBkfSmeNaynCtfzCmknHGQ3waUcricTgUcUuYiPbjLGPwc2PgKbhyoIqoxq2lBmoo-kSiN~2PZjzdlmz4hjVNjMwhehQuD-YPS0-qfpqMxxNGSoSr7BqTCbazi6JZ43tPGrjDcGijN09hy7~D0xJGCJvjkpym52PQF2-q-Bap~jXvxX8YxEHru6bb0FcRmzbA2jp52DwRwNM~gJWmsVahH-UUZX1MoWf4Pbzze7sJZdai244XWM2ItHUVD~h0H6hMjuZcYYATYv5U5mg7Y3cQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). The 711 fusion isoforms corresponded to 269 FLNC fusion loci and were involved in 622 genes in the reference annotation. Distribution of fusion isoforms revealed that most of the fusion isoforms were present on chr7, followed by chr14 and chr6 (<a reveal-id="dsy014-F5" data-open="dsy014-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In 269 FLNC fusion loci, 31 chimeric genes (∼11.52%) corresponding to 90 fusion isoforms (∼12.66%) originated from at least three candidate genes, whereas the remaining from two candidate ones. Previous studies showed that fusion events were mostly comprised of two genes.<a reveal-id="dsy014-B49" data-open="dsy014-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> However, our results revealed that fusion event was not limited to two genes and could include more than two genes, or even ncRNAs. For example, MT.3.1 originated from 10 candidates (<em>MT-CO2; MT_tRNA; MT-ATP8; MT-ATP6; MT-CO3; MT-tRNA; MT-ND3; MT-tRNA; MT-ND4L; MT-ND4;</em>). <em>MT-CO2</em> is usually unexpressed and has the characteristic of inducible expression. Once the transcript originates from <em>MT-CO2</em> expression, it can induce tumour occurrence<a reveal-id="dsy014-B50" data-open="dsy014-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> and increase the probability of inducing tumour in females.<a reveal-id="dsy014-B51" data-open="dsy014-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Moreover, fusion event could not only span two distinct genes but also produce variant isoforms with AS events. For example, fusion isoform 11.112 originated form two genes (EXOSC8-ENSSSCG00000026613) and acquired another two novel exons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1g</a></span>). This result was consistent with previously reported results that FL fusion could produce fusion genes that acquired novel genes or exons, suggested their isoforms independent of a reference annotation library.<a reveal-id="dsy014-B49" data-open="dsy014-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Thus, fusion isoforms identified by Iso-Seq would provide more direct evidence for further studies of aberrant transcription which were from <em>trans</em>-splicing of distinct genes or splicing of fusion genes formed by from genome rearrangements.</p><p>To test whether these fusion events were formed stochastically by abnormal AS or necessarily by functional requirement, we analysed the functional relationship between precursor genes and each fusion isoform (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s9.xlsx?Expires=2147483647&Signature=LIDyG6fFeriAmmKf6xq7lMd6oRxqVZ7q2XfRCcNZ4QtfzscvguU~nKDeeVW9zAvanBkfSmeNaynCtfzCmknHGQ3waUcricTgUcUuYiPbjLGPwc2PgKbhyoIqoxq2lBmoo-kSiN~2PZjzdlmz4hjVNjMwhehQuD-YPS0-qfpqMxxNGSoSr7BqTCbazi6JZ43tPGrjDcGijN09hy7~D0xJGCJvjkpym52PQF2-q-Bap~jXvxX8YxEHru6bb0FcRmzbA2jp52DwRwNM~gJWmsVahH-UUZX1MoWf4Pbzze7sJZdai244XWM2ItHUVD~h0H6hMjuZcYYATYv5U5mg7Y3cQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). After excluding unannotated genes, we obtained 539 fusion isoforms and 352 precursor genes. Interestingly, we found that a majority of fusion isoforms (397/539,73.65%) were formed from the combinations of separate precursor transcripts in a same gene family. For example, 58 isoforms were from different duplicates of the Myosin heavy chain gene family (<em>MYH</em>), 18 isoforms from DEAD-box helicase family (<em>DDX</em>) and 15 isoforms from interferon induced protein with tetratricopeptide repeats family (<em>IFIT</em>). Thus, fusion isoforms from a same gene family implied that the formation of fusing events were likely due to special biological functions such as functional conserved between conserved duplicates or functional complementation between subfunctional duplicates. For other fusion isoforms that originated from different family genes, precursor transcripts of which were probably functionally related. For example, a fusion gene 7.1323 originated from two precursor transcripts (<em>ATP6V1G2</em> and <em>DDX39B</em>). <em>DDX39B</em> encodes a member of the <em>DEAD</em> box family of RNA-dependent ATPases that mediates adenosine triphosphate (ATP) hydrolysis during pre-mRNA splicing.<a reveal-id="dsy014-B52" data-open="dsy014-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> The fusion gene 7.1323 could produce 17 fusion isoforms in pigs while it only produced two transcripts in humans and might induce immunologic and infectious diseases.<a reveal-id="dsy014-B53" data-open="dsy014-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p><p>Moreover, we perform a more precise BLAST analysis using 711 fusion isoforms against NCBI Reference RNA Sequences database to validate fusion isoforms. Results showed that, 270 fusion genes (∼38%) are supported by at least one Reference RNA Sequences in pig (with identity &gt;95% and overlap &gt;90%). In addition, we searched these fusion genes in human. Interestingly, 86 are supported by at least one Reference RNA Sequences (with identity &gt;88% and overlap &gt;90%). And 84 fusion genes were both supported in these two databases, suggested that fusion gene may exist in different species and evolve conserved functions (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s9.xlsx?Expires=2147483647&Signature=LIDyG6fFeriAmmKf6xq7lMd6oRxqVZ7q2XfRCcNZ4QtfzscvguU~nKDeeVW9zAvanBkfSmeNaynCtfzCmknHGQ3waUcricTgUcUuYiPbjLGPwc2PgKbhyoIqoxq2lBmoo-kSiN~2PZjzdlmz4hjVNjMwhehQuD-YPS0-qfpqMxxNGSoSr7BqTCbazi6JZ43tPGrjDcGijN09hy7~D0xJGCJvjkpym52PQF2-q-Bap~jXvxX8YxEHru6bb0FcRmzbA2jp52DwRwNM~gJWmsVahH-UUZX1MoWf4Pbzze7sJZdai244XWM2ItHUVD~h0H6hMjuZcYYATYv5U5mg7Y3cQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>).</p>                            <h3 scrollto-destination=120122204 id="120122204" class="section-title" >3.7 Tissue-specific and period-specific isoform</h3>
<p>Identification of specific isoform is among the primary analysis accomplished through Illumina RNA-seq data of the same eight tissues. We used a new modified GFF file to estimate the expression level of each isoforms (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s11.xlsx?Expires=2147483647&amp;Signature=e78bgYQbpHN6NQ0GjDoufRwNvw2gJ~sOYeJwNFDhgdT~CydnbHo9T7CaOcbgoGOjJR58vf~stacSFAJyIRaQ5ia6~KRl97L0~uTuqC~Bp3rmeINRhhf-Mv9OJcaEIC-4T0nUicttZCwCMoMjphn5~e8rMXHDY6Ac9-PV5SghKaU8L~gbD1d1Vo-JF5uocIseOqK5z7c6~dZC9igcL2oGkdJra~9ZB32yv2--u96E6-NEe8YqHXbcWNGgJMvLNiUeucOz7fVLZJeGap7rO2ooT4O8oJOmpiibgBrLGHuPsArjETXECRfttmhlXkwLSKZBTow2xG5nnt2B6MiJ87BJ~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). The new GFF file contained junction positions of each isoform which were extracted from the GMAP mapping data. We obtained more than 4 G bases clean paired data each tissue, which was adequate for quantify gene expression (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&amp;Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). Additionally, rarefaction analysis revealed that sequencing depth had reached saturation of gene and isoform discovery in each tissue (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s3.png?Expires=2147483647&amp;Signature=x3i7228YriMM1jcR6-pjCUooKhJMbfKZGPHrWzhCWLvUvgztrNSRjLiUOXvUXnzRSsPAbOwMMeoQYF2aB46LrLu2QOogtXhEAqyp3RFUpY2IBN18TDKIwy~uV2KFX7alJCfwAfC3RipvNWSAowmos-P-0odcD2KY4G9ie1~4yeWoiMLIE6E-r1xPH4pcfawUqsoZLIxI4Us9Kj9IR-LJIytBHoC48tSiNUCnOipWju-ztpbS3rSp8mofFt6FwGw28VPVSWEDT0x6SVZH64aqOpcGjUh~LHnufs5NZlBeLODMBj6u-TiA5JMP6LAzkiOI7rjT0K4WCaYwX3oOqI~VwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3a and b</a></span>). STEM<a reveal-id="dsy014-B54" data-open="dsy014-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> expression analysis for the 39,940 loci revealed that 25,018 loci were expressed in eight tissues. Moreover, about 46.25% (12,432/26,881) of novel gene loci and 48.45% (4,282/8838) of lncRNA could be validated from these eight tissues (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&amp;Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>).</p><p>For detecting the tissue-specific expressed isoform, a total of 47,083 multi-exon isoforms were analysed in different tissues and periods (<a reveal-id="dsy014-F8" data-open="dsy014-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). Uniquely expressed isoforms were 451 in subcutaneous fat of back (SFB), 416 in SM, 341 in EDL and 1,337 in EN respectively, and 16,501 isoforms were simultaneously expressed in the four tissues (<a reveal-id="dsy014-F9" data-open="dsy014-F9" class="link link-reveal link-table xref-fig">Fig. 9a</a>). For the 2,545 isoforms uniquely expressed in each tissue, GO terms were primarily (top six) related to cellular and single-organism processes, cell, cell part, metabolic process, binding (<a reveal-id="dsy014-F9" data-open="dsy014-F9" class="link link-reveal link-table xref-fig">Fig. 9b</a>). Function enrichments of pathway (<a reveal-id="dsy014-F9" data-open="dsy014-F9" class="link link-reveal link-table xref-fig">Fig. 9c</a>) varied slightly in each tissue such as catabolism and carbohydrate metabolism in SFB; digestive system and lipid metabolism in SM; sorting and degradation and digestive system in EDL and endocrine system, immune system and transport and catabolism in EN. These results showed that genes played the same biological functions in different tissues by the same or unique isoforms and the isoforms with tissue specificity might lead to various roles via different pathways. </p>                        <a id="120122207" scrollto-destination="120122207"></a>
<div id="dsy014-F8" data-id="dsy014-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f8.png?Expires=2147483647&Signature=hG095zHlJCvbTRlL2RsJ4S-q6tuL8NjTZDoSBrCTlbPMWCmm5saz-QiE0BgFG3xTYXqv2H-kyJnVLTH1n498MfnF1r-bGpiAVNhqLnWTwWClLAym77tj2qlRv9E88D1DHM2ClJxyXQGASnbG34lOz~DKf54LkOf3uy3Fw6CSPgaOTt~PdNc0tfN-nTkVLBNqUfkHQehA9Srig9AI1S3MWE939dfZMB0zeOaTLBT5uc81l7J0~kAi9gYHFk9FmjmnRVR0yfgBNFM8tv~o7UaXLIysV5Ux5N26-LAQBEmK261VTyK1y56pwY2w-mslLJGBW0p-wVKCY~QLiSBOkxIG-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heatmap of tissue-specific and period-specific isoforms. (a) Tissue-specific isoforms. (b) Period-specific isoforms. SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day). The coloured box on the left means the cluster analysis of whole-gene expression levels in eight tissues." path-from-xml="dsy014f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122207/dsy014f8.tif" path-from-xml="dsy014f8.tif" target="_blank">View large</a><a section="120122207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f8.png?Expires=2147483647&Signature=v0C9mfnDOSLOvQtVf8acvo3~bMym5l9V1Nunx-22wgVZceAG9GhGiwnKuW7JRDWHJvczXESBZwrE4UEOEpJDVPQwG4rhUTbhq3qEdtUd2jwuiWCRW9ajJ1x-ZgVEdUDqsH1neBgVaG6yOiTe6j-H0rTvz9zN26ftjNUZtSHR5EXoEVw8CW-BLpHklcGC2qbwoR4Lf0dvQHNBHmZnWirW3EbBWjH6mlkJ2q0EM88NEIaK48sWQYC~wJKuNdkrF0bmNuOHc4xCvRQuwamRZiDuWo8BAgPOwGYqwjScOdMHU9ktJPEnFTv~wYGsMHLp3GJqyHc~9HdKkolCMfWkh~bVcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122207&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heatmap of tissue-specific and period-specific isoforms. (a) Tissue-specific isoforms. (b) Period-specific isoforms. SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day). The coloured box on the left means the cluster analysis of whole-gene expression levels in eight tissues.</p></div></div><div content-id="dsy014-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f8.png?Expires=2147483647&Signature=hG095zHlJCvbTRlL2RsJ4S-q6tuL8NjTZDoSBrCTlbPMWCmm5saz-QiE0BgFG3xTYXqv2H-kyJnVLTH1n498MfnF1r-bGpiAVNhqLnWTwWClLAym77tj2qlRv9E88D1DHM2ClJxyXQGASnbG34lOz~DKf54LkOf3uy3Fw6CSPgaOTt~PdNc0tfN-nTkVLBNqUfkHQehA9Srig9AI1S3MWE939dfZMB0zeOaTLBT5uc81l7J0~kAi9gYHFk9FmjmnRVR0yfgBNFM8tv~o7UaXLIysV5Ux5N26-LAQBEmK261VTyK1y56pwY2w-mslLJGBW0p-wVKCY~QLiSBOkxIG-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heatmap of tissue-specific and period-specific isoforms. (a) Tissue-specific isoforms. (b) Period-specific isoforms. SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day). The coloured box on the left means the cluster analysis of whole-gene expression levels in eight tissues." path-from-xml="dsy014f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122207/dsy014f8.tif" path-from-xml="dsy014f8.tif" target="_blank">View large</a><a section="120122207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f8.png?Expires=2147483647&Signature=v0C9mfnDOSLOvQtVf8acvo3~bMym5l9V1Nunx-22wgVZceAG9GhGiwnKuW7JRDWHJvczXESBZwrE4UEOEpJDVPQwG4rhUTbhq3qEdtUd2jwuiWCRW9ajJ1x-ZgVEdUDqsH1neBgVaG6yOiTe6j-H0rTvz9zN26ftjNUZtSHR5EXoEVw8CW-BLpHklcGC2qbwoR4Lf0dvQHNBHmZnWirW3EbBWjH6mlkJ2q0EM88NEIaK48sWQYC~wJKuNdkrF0bmNuOHc4xCvRQuwamRZiDuWo8BAgPOwGYqwjScOdMHU9ktJPEnFTv~wYGsMHLp3GJqyHc~9HdKkolCMfWkh~bVcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122207&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heatmap of tissue-specific and period-specific isoforms. (a) Tissue-specific isoforms. (b) Period-specific isoforms. SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day). The coloured box on the left means the cluster analysis of whole-gene expression levels in eight tissues.</p></div></div>                        <a id="120122208" scrollto-destination="120122208"></a>
<div id="dsy014-F9" data-id="dsy014-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f9.png?Expires=2147483647&Signature=wKDycFnwzN3GHlbBwWkHFc-lgtbIHCKWKNs9Y4Yuu3-4drUr1QfbFkftQYMn2dw5JdUKgDVi4sqBO7BSaYuceeIc6Q53Gfdr4a4snai0dzjueN3s3LMMenffVTlCKiD8gD2vUAy7DD6SDhoBmFFbvKsznApDAkwciTOAC8bP5eijBSs~abJ5bLVpE5WjEyNnfTC3vJ5KNQSZT8cahSQq1Lo6t7aaxfrOOp9Ub7zJm4yiAg~u8Um6cseEszaGX~wc1lbeWfEJ8pLiAVLr~mqrJUel3x-w7mkozD6I3EK50axclo1s4uJKvDH0CtFBGFgdwIcZj1fWTYnwYEmHY7eaIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Function annotation of tissue-specific isoforms. (a) Tissue-specific isoforms. (b) Classification of gene ontology annotation for isoform uniquely expressed in the four tissues. (c) Classification of KEGG pathways annotation for isoform uniquely expressed in the four tissues." path-from-xml="dsy014f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122208/dsy014f9.tif" path-from-xml="dsy014f9.tif" target="_blank">View large</a><a section="120122208" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f9.png?Expires=2147483647&Signature=ulqI6Nc1pSvg4XCKQAVxWzSuFfh6FaqCQzpFLMYbsYR9uR58nznVJg1sEX9BfU3E7ifC72oVFLQybUPxkPYaG0WHDf4qk93gEPtZpIPTCrt4PWEJpf5fZEVNgn9O1ch4jFjeNvQ76VZztWt~IzzrLJ8M~KAbhImXn9RfOuZaf2jCUKSgI7w5EPmapPg07gDfXhOfv6QdjhbX3arCcwQ69gFyL51qbV8bSE~cBEnnhUNXISY3wjFo5qPe-omSMYOJ9flR9pw4bQ1m0WRjy2Yd8LzJz5nbj4F4dH35vjC7~KivXx6kkzBZ20WrThJDcEh-upGeM4u2zElNCCa277Xulg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122208&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Function annotation of tissue-specific isoforms. (a) Tissue-specific isoforms. (b) Classification of gene ontology annotation for isoform uniquely expressed in the four tissues. (c) Classification of KEGG pathways annotation for isoform uniquely expressed in the four tissues.</p></div></div><div content-id="dsy014-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f9.png?Expires=2147483647&Signature=wKDycFnwzN3GHlbBwWkHFc-lgtbIHCKWKNs9Y4Yuu3-4drUr1QfbFkftQYMn2dw5JdUKgDVi4sqBO7BSaYuceeIc6Q53Gfdr4a4snai0dzjueN3s3LMMenffVTlCKiD8gD2vUAy7DD6SDhoBmFFbvKsznApDAkwciTOAC8bP5eijBSs~abJ5bLVpE5WjEyNnfTC3vJ5KNQSZT8cahSQq1Lo6t7aaxfrOOp9Ub7zJm4yiAg~u8Um6cseEszaGX~wc1lbeWfEJ8pLiAVLr~mqrJUel3x-w7mkozD6I3EK50axclo1s4uJKvDH0CtFBGFgdwIcZj1fWTYnwYEmHY7eaIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Function annotation of tissue-specific isoforms. (a) Tissue-specific isoforms. (b) Classification of gene ontology annotation for isoform uniquely expressed in the four tissues. (c) Classification of KEGG pathways annotation for isoform uniquely expressed in the four tissues." path-from-xml="dsy014f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122208/dsy014f9.tif" path-from-xml="dsy014f9.tif" target="_blank">View large</a><a section="120122208" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f9.png?Expires=2147483647&Signature=ulqI6Nc1pSvg4XCKQAVxWzSuFfh6FaqCQzpFLMYbsYR9uR58nznVJg1sEX9BfU3E7ifC72oVFLQybUPxkPYaG0WHDf4qk93gEPtZpIPTCrt4PWEJpf5fZEVNgn9O1ch4jFjeNvQ76VZztWt~IzzrLJ8M~KAbhImXn9RfOuZaf2jCUKSgI7w5EPmapPg07gDfXhOfv6QdjhbX3arCcwQ69gFyL51qbV8bSE~cBEnnhUNXISY3wjFo5qPe-omSMYOJ9flR9pw4bQ1m0WRjy2Yd8LzJz5nbj4F4dH35vjC7~KivXx6kkzBZ20WrThJDcEh-upGeM4u2zElNCCa277Xulg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122208&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Function annotation of tissue-specific isoforms. (a) Tissue-specific isoforms. (b) Classification of gene ontology annotation for isoform uniquely expressed in the four tissues. (c) Classification of KEGG pathways annotation for isoform uniquely expressed in the four tissues.</p></div></div><p>For the period-specific expressed isoforms, 47,083 multi-exon isoforms were classified into five categories (<a reveal-id="dsy014-T2" data-open="dsy014-T2" class="link link-reveal link-table xref-fig">Table 2</a>). A total of 14,706 and 12,766 isoforms simultaneously expressed in four tissues in one-day-old and adult period, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s4.png?Expires=2147483647&amp;Signature=Gb3qPOpGxq71-hSZlu6-~zows4F~5jwZ5ZCVNNnf2L4NQT0SMFATM0PB7hwMjl3Zt-9HiheTDNBJcFkJKrdykl3~VJbaz6~AHaqOltnaQiHjQSUFBDiBeYVHQjN-Ryzv3n96YDTohT-5fFLz64WPXaVgzKg~ELZGAHAbdzVfQvCpi1yttSCUNZNr~SUI6fdMrVf3UJ8-RH~3dvug9LQE12xr3TC7xZsGdkTyvqMiqGycC2b6SfOhwXUhMv-c~X2SDKab7ivCIiJ5Tt6wQbevzPzDiPRitP1lwUBUE0ln57H7ZxWP1zQH16W~oIbcRCkUV5cVhy2rbppDgS4Fqe3ijw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4a</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s5.png?Expires=2147483647&amp;Signature=n6PKBzPrNmQ8wqAD2wbQV3VgrSE53Y8~uFXB~z53BDGo5-DHWJnVWiI-0L6TLtFS3ZOy4oX4R1THnsBzAlO8Z8sbZ55o6dzD-UtKz7nPoplrj~vpB7d6WQYhd3KCRUZCDu97GgCWFpuwtUuJfpd9a~Dch5DhQo4VQWA0sHekwExhBj-XtEl4FUYG3bofwNlthJ2A-FZ9DQZx7xaVfxa7Y7L2hZJ3TJrGtcrjP1FO~v4St2Rfg4wKyWsQOeTNZUE~4TAQ25bALc31IEvMUb4KjMu9aaGJuKY~rQ5Wc8Yuu7~~WyGoz7yRqR9vry9mcefY47L-Gwl0ihBRiL3ovvuyeQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S5a</a></span>). The number of isoforms expressed only in single tissue between one-day-old and adult also maintained high consistency (<em>r</em><strong><em> </em></strong>= 0.974). Results from GO and Kyoto Encyclopaedia of Genes and Genomes <strong>(</strong>KEGG) analyses of the isoforms expressed only in single tissue between one-day-old and adult revealed that the tissue diversity of the expression abundance could promote varieties of protein, although GO terms were similar (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s4.png?Expires=2147483647&amp;Signature=Gb3qPOpGxq71-hSZlu6-~zows4F~5jwZ5ZCVNNnf2L4NQT0SMFATM0PB7hwMjl3Zt-9HiheTDNBJcFkJKrdykl3~VJbaz6~AHaqOltnaQiHjQSUFBDiBeYVHQjN-Ryzv3n96YDTohT-5fFLz64WPXaVgzKg~ELZGAHAbdzVfQvCpi1yttSCUNZNr~SUI6fdMrVf3UJ8-RH~3dvug9LQE12xr3TC7xZsGdkTyvqMiqGycC2b6SfOhwXUhMv-c~X2SDKab7ivCIiJ5Tt6wQbevzPzDiPRitP1lwUBUE0ln57H7ZxWP1zQH16W~oIbcRCkUV5cVhy2rbppDgS4Fqe3ijw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4b and c</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s5.png?Expires=2147483647&amp;Signature=n6PKBzPrNmQ8wqAD2wbQV3VgrSE53Y8~uFXB~z53BDGo5-DHWJnVWiI-0L6TLtFS3ZOy4oX4R1THnsBzAlO8Z8sbZ55o6dzD-UtKz7nPoplrj~vpB7d6WQYhd3KCRUZCDu97GgCWFpuwtUuJfpd9a~Dch5DhQo4VQWA0sHekwExhBj-XtEl4FUYG3bofwNlthJ2A-FZ9DQZx7xaVfxa7Y7L2hZJ3TJrGtcrjP1FO~v4St2Rfg4wKyWsQOeTNZUE~4TAQ25bALc31IEvMUb4KjMu9aaGJuKY~rQ5Wc8Yuu7~~WyGoz7yRqR9vry9mcefY47L-Gwl0ihBRiL3ovvuyeQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S5b and c</a></span>). For example, isoform 12.550.1 (TUSC5) showed specific and high expression in SF adult period, which could regulate insulin-mediated adipose tissue glucose uptake by modulation of GLUT4 recycling.<a reveal-id="dsy014-B55" data-open="dsy014-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Isoform 3.161.1 (COX6A2), 12.685.1(MYH13) and 13.197.8 (XIRP1), with specific and high expression in SM and EDL adult period, could play important roles in skeletal muscle fibre type switch,<a reveal-id="dsy014-B56" data-open="dsy014-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> immune-mediated myositis<a reveal-id="dsy014-B57" data-open="dsy014-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and muscle development,<a reveal-id="dsy014-B58" data-open="dsy014-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> respectively. <div content-id="dsy014-T2" class="table-modal"><div class="table-wrap"><div id="dsy014-T2" data-id="dsy014-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Analysis of period-specific isoforms</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Isoform </th><th>1 day </th><th>Adult </th></tr></thead><tbody><tr><td>Total multiple-exon isoform </td><td>47,083 </td><td>47,083 </td></tr><tr><td>Expressed in at least one tissue </td><td>22,116 </td><td>21,986 </td></tr><tr><td>Expressed in 4 tissues </td><td>14,681 </td><td>12,700 </td></tr><tr><td>Expressed only in single tissue </td><td>2,675 </td><td>3,314 </td></tr><tr><td>Unexpressed only in single tissue </td><td>2,810 </td><td>2,937 </td></tr><tr><td>Unexpressed in 4 tissues </td><td>24,967 </td><td>25,097 </td></tr></tbody></table></div><div content-id="dsy014-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Isoform </th><th>1 day </th><th>Adult </th></tr></thead><tbody><tr><td>Total multiple-exon isoform </td><td>47,083 </td><td>47,083 </td></tr><tr><td>Expressed in at least one tissue </td><td>22,116 </td><td>21,986 </td></tr><tr><td>Expressed in 4 tissues </td><td>14,681 </td><td>12,700 </td></tr><tr><td>Expressed only in single tissue </td><td>2,675 </td><td>3,314 </td></tr><tr><td>Unexpressed only in single tissue </td><td>2,810 </td><td>2,937 </td></tr><tr><td>Unexpressed in 4 tissues </td><td>24,967 </td><td>25,097 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy014-T2" data-id="dsy014-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Analysis of period-specific isoforms</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Isoform </th><th>1 day </th><th>Adult </th></tr></thead><tbody><tr><td>Total multiple-exon isoform </td><td>47,083 </td><td>47,083 </td></tr><tr><td>Expressed in at least one tissue </td><td>22,116 </td><td>21,986 </td></tr><tr><td>Expressed in 4 tissues </td><td>14,681 </td><td>12,700 </td></tr><tr><td>Expressed only in single tissue </td><td>2,675 </td><td>3,314 </td></tr><tr><td>Unexpressed only in single tissue </td><td>2,810 </td><td>2,937 </td></tr><tr><td>Unexpressed in 4 tissues </td><td>24,967 </td><td>25,097 </td></tr></tbody></table></div><div content-id="dsy014-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Isoform </th><th>1 day </th><th>Adult </th></tr></thead><tbody><tr><td>Total multiple-exon isoform </td><td>47,083 </td><td>47,083 </td></tr><tr><td>Expressed in at least one tissue </td><td>22,116 </td><td>21,986 </td></tr><tr><td>Expressed in 4 tissues </td><td>14,681 </td><td>12,700 </td></tr><tr><td>Expressed only in single tissue </td><td>2,675 </td><td>3,314 </td></tr><tr><td>Unexpressed only in single tissue </td><td>2,810 </td><td>2,937 </td></tr><tr><td>Unexpressed in 4 tissues </td><td>24,967 </td><td>25,097 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=120122210 id="120122210" class="section-title" >3.8 LncRNA identification</h3>
<p>A classification model of a high-confidence set of known lncRNAs was built using PLEK<a reveal-id="dsy014-B31" data-open="dsy014-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> to identify lncRNAs in the PacBio data. Among the 77,075 PacBio isoforms, 15,049 candidate lncRNAs were obtained via PLEK model. To obtain a high-confidence set of lncRNA genes, we eliminated transcripts with ORFs exceeding 100 codons and used BLASTX to screen the 15,049 candidates for homology with proteins of all species in NR data, thereby obtaining 8,838 lncRNAs with a mean length of 2 kb. Of the 8,838 lncRNAs, including 7,928 single-exon lncRNAs (89.7%) and 910 multi-exon lncRNAs (10.3%), 5,058 lincRNAs existed. BLASTN found that 664 of the 5,058 candidates corresponded to previously reported lincRNA of pig in ALDB.<a reveal-id="dsy014-B32" data-open="dsy014-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The remaining 4,394 novel lincRNAs had a mean length of 2.024 kb (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10a</a>). LncRNAs were classified into five groups according to their biogenesis positions relative to protein-coding genes of SSC10.2.84 annotations: 57.23% of them were generated from intergenic regions, 31.81% from the intronic regions, 7.24% from the sense strand, 3.33% from the antisense strand and 0.40% belong to bidirectional lncRNA (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10b</a>). In addition, majority (89.7%) of the lncRNAs were single exon, and this percentage was significantly higher than the non-lncRNAs (<em>P</em> &lt; 0.01, <a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10c</a>). </p>                        <a id="120122212" scrollto-destination="120122212"></a>
<div id="dsy014-F10" data-id="dsy014-F10" class="fig fig-section"><div class="label fig-label">Figure 10</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f10.png?Expires=2147483647&Signature=y-e-SNIqcgaAvq8qN9wqqA06aqc0D8l~3hbNITdKpzIFS-90mIG~Mopg6Y4Jr-WzMzSjSqqnDhZeBMz2MV9B1YIUZKiJsdjrvnVBM36buaSZw5YTtIP1lZ~v0yk1GXRc--veswqDgrPDVAlYadUHrRAcp~eNh9x8whWrG3OhOb3TpyYav-7ybPXTP-FTpCOP5Aae-h~J59I3gDafXT7BoU2bWTC8NA7MwtUU2~4Wr1RVEictgmECPkurGfcIwqF1nbaIwPwY1YOMmLdpfnt2q2C~0Ob~vt7QrzaljdEKXbbRLHZKwlLuPXxfyAUsfp2rCkJfJ4WxJd4fc-AJv~IIuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characters of identified lncRNAs. (a) Comparison of lengths of previously reported lincRNAs with novel lincRNAs identified in our study. (b) Proportions of five kinds of lncRNAs, classified according to their position relative to protein-coding genes. (c) Number of exons in lncRNAs and non-lncRNAs. (d) Overlap of lncRNAs among tissue and period [SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day)]. (e) Heatmap of lncRNA expression in eight tissues. (f) Heatmap of non-lncRNA expression in eight tissues. (g) Comparison of overall expression between lncRNAs and non-lncRNAs. (h) Comparison of overall expression between single-exon lncRNAs and multi-exon lncRNAs. (i) Comparison of DNA methylation level on lncRNAs and non-lncRNAs." path-from-xml="dsy014f10.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122212/dsy014f10.tif" path-from-xml="dsy014f10.tif" target="_blank">View large</a><a section="120122212" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f10.png?Expires=2147483647&Signature=3BDo0Njnc~iGMBAi~hesECmJnYd1dI4x1JyhJpapD8f9UPQxEAbZfEkpI1mr5ZDMhrOqSiJ3~ZpI1EdZxA2OtWtWP09r6xOCY6VrG8edFhQD65gzuRS33vgJ7iSSp7~mrhaRkKKKQ3jNd9Not1LVHVw9g0ybTxFVM0TwAP2ptT8IGzPA7u1eQKF7FXGyQZ4nMP19QAPu9HlF8S2sMUQPCXcwVabMDVEa0Xg2JZfFuW8~k9X-Oa8h5cR4hvAZr87MWb7LBTod2M2mhmqBsQ4~xubAWRHz0MaECQkMoJbgCW1n8Apaq24N4UmE4ymFaX4FZtICxp-dZqYLEo3bSXq2cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122212&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f10.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characters of identified lncRNAs. (a) Comparison of lengths of previously reported lincRNAs with novel lincRNAs identified in our study. (b) Proportions of five kinds of lncRNAs, classified according to their position relative to protein-coding genes. (c) Number of exons in lncRNAs and non-lncRNAs. (d) Overlap of lncRNAs among tissue and period [SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day)]. (e) Heatmap of lncRNA expression in eight tissues. (f) Heatmap of non-lncRNA expression in eight tissues. (g) Comparison of overall expression between lncRNAs and non-lncRNAs. (h) Comparison of overall expression between single-exon lncRNAs and multi-exon lncRNAs. (i) Comparison of DNA methylation level on lncRNAs and non-lncRNAs.</p></div></div><div content-id="dsy014-F10" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 10</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f10.png?Expires=2147483647&Signature=y-e-SNIqcgaAvq8qN9wqqA06aqc0D8l~3hbNITdKpzIFS-90mIG~Mopg6Y4Jr-WzMzSjSqqnDhZeBMz2MV9B1YIUZKiJsdjrvnVBM36buaSZw5YTtIP1lZ~v0yk1GXRc--veswqDgrPDVAlYadUHrRAcp~eNh9x8whWrG3OhOb3TpyYav-7ybPXTP-FTpCOP5Aae-h~J59I3gDafXT7BoU2bWTC8NA7MwtUU2~4Wr1RVEictgmECPkurGfcIwqF1nbaIwPwY1YOMmLdpfnt2q2C~0Ob~vt7QrzaljdEKXbbRLHZKwlLuPXxfyAUsfp2rCkJfJ4WxJd4fc-AJv~IIuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characters of identified lncRNAs. (a) Comparison of lengths of previously reported lincRNAs with novel lincRNAs identified in our study. (b) Proportions of five kinds of lncRNAs, classified according to their position relative to protein-coding genes. (c) Number of exons in lncRNAs and non-lncRNAs. (d) Overlap of lncRNAs among tissue and period [SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day)]. (e) Heatmap of lncRNA expression in eight tissues. (f) Heatmap of non-lncRNA expression in eight tissues. (g) Comparison of overall expression between lncRNAs and non-lncRNAs. (h) Comparison of overall expression between single-exon lncRNAs and multi-exon lncRNAs. (i) Comparison of DNA methylation level on lncRNAs and non-lncRNAs." path-from-xml="dsy014f10.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122212/dsy014f10.tif" path-from-xml="dsy014f10.tif" target="_blank">View large</a><a section="120122212" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f10.png?Expires=2147483647&Signature=3BDo0Njnc~iGMBAi~hesECmJnYd1dI4x1JyhJpapD8f9UPQxEAbZfEkpI1mr5ZDMhrOqSiJ3~ZpI1EdZxA2OtWtWP09r6xOCY6VrG8edFhQD65gzuRS33vgJ7iSSp7~mrhaRkKKKQ3jNd9Not1LVHVw9g0ybTxFVM0TwAP2ptT8IGzPA7u1eQKF7FXGyQZ4nMP19QAPu9HlF8S2sMUQPCXcwVabMDVEa0Xg2JZfFuW8~k9X-Oa8h5cR4hvAZr87MWb7LBTod2M2mhmqBsQ4~xubAWRHz0MaECQkMoJbgCW1n8Apaq24N4UmE4ymFaX4FZtICxp-dZqYLEo3bSXq2cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122212&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f10.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characters of identified lncRNAs. (a) Comparison of lengths of previously reported lincRNAs with novel lincRNAs identified in our study. (b) Proportions of five kinds of lncRNAs, classified according to their position relative to protein-coding genes. (c) Number of exons in lncRNAs and non-lncRNAs. (d) Overlap of lncRNAs among tissue and period [SFB: subcutaneous fat of back, SM: soleus muscle, EDL: extensor digitorum longus, EN: endometria, A: adult, B: birth (one day)]. (e) Heatmap of lncRNA expression in eight tissues. (f) Heatmap of non-lncRNA expression in eight tissues. (g) Comparison of overall expression between lncRNAs and non-lncRNAs. (h) Comparison of overall expression between single-exon lncRNAs and multi-exon lncRNAs. (i) Comparison of DNA methylation level on lncRNAs and non-lncRNAs.</p></div></div><p>A total of 388 lncRNAs were simultaneously expressed in the eight tissues. Endometria at adult expressed the most specific lncRNAs (132), whereas adult soleus muscle expressed the fewest (9) (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10d</a>). Heatmap of lncRNA and non-lncRNA expression confirmed that both lncRNA and non-lncRNA unfolded tissue-specific expression, particularly in SFB_A, EN_A and EN_B (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10e and f</a>). Comparison of overall expression between lncRNA and non-lncRNA showed that lncRNA were significantly less expressed than non-lncRNA (<em>P</em> &lt; 0.01, <a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10g</a>). Comparison revealed that multi-exon lncRNAs expressed a higher level than single-exon lncRNAs (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10h</a>). Furthermore, we monitored the level of CG methylation within and surrounding (5 kb upstream and 5 kb downstream) lncRNA and non-lncRNA isoforms (<a reveal-id="dsy014-F10" data-open="dsy014-F10" class="link link-reveal link-table xref-fig">Fig. 10i</a>). DNA methylation decreased near the TSS and TTS of both lncRNA and non-lncRNA genes; DNA methylation was also particularly significant for the TSS of non-lncRNA genes (<em>P</em> &lt; 0.01). Non-lncRNA genes presented higher CG methylation within the gene body than lncRNAs, whereas lncRNAs showed higher methylation level at upstream and downstream regions than non-lncRNAs. Finally, we discovered that multi-exon lncRNAs exhibited higher CG methylation within the gene body and the lower level at upstream and downstream regions than single-exon lncRNAs. DNA methylation of the promoter and gene body was strongly correlated with lncRNA expression,<a reveal-id="dsy014-B59" data-open="dsy014-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> which might reflect the diverse expression levels of non-lncRNA genes and lncRNAs, as well as that of single-exon and multi-exon isoforms.</p>                            <h3 scrollto-destination=120122214 id="120122214" class="section-title" >3.9 Validation of isoforms by RT-PCR</h3>
<p>To verify the expression of novel isoforms and genes in eight tissues, we randomly selected six genes (with or without exons and with or without AS) and performed reverse transcription (RT) PCR following by Sanger sequencing (primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). As shown in <a reveal-id="dsy014-F11" data-open="dsy014-F11" class="link link-reveal link-table xref-fig">Figure 11</a>, expression and splicing were confirmed in our expression analysis. In addition, two fusion transcripts (7.168.3 and 1.2355.7) and two transcripts of novel lncRNAs (1.1171.1 and x.631.1) were randomly selected and experimentally validated. The results confirmed the authenticity of two chimeric RNA and tow lncRNAs. The validation above also showed that AS and fusion isoforms had differential expression and tissue and period specificity, thus increasing diversity of gene regulation as well as complexity of transcriptome. </p>                        <a id="120122216" scrollto-destination="120122216"></a>
<div id="dsy014-F11" data-id="dsy014-F11" class="fig fig-section"><div class="label fig-label">Figure 11</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f11.png?Expires=2147483647&Signature=eF1A4rNJGo4a4l-G1iHkBd3vOrQ9X9y5sOfydUZLtK4HZzg1iiTqo4cQosqF0DQSPZrMB0uHN-RaoF8VNhWNcyPHcxw-VTOypOTFQsHTgF0s2CZUCuu~mNnEJHI3GKaO-1xbXga-GRqYl0RJeZdO5pldqXeVFvZScKU-4jaJw~Am3z9Kjw1PXoWW0bYGxFXHEDGqo5q8g4yzET9eL5Y4ntZ54ZSa1soULb~OdaWHBtwlMJ2-Z4PSYpH-d63AutI9qR54Q2dCx-E4MANzDWe6g-Rd8cnizCL6Y~vvwLmb61KzKIvwzULxs41eVeaNxrWa68zYYWG12D-NnkD-i-LClA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of differential expressed isoforms. (a) Heatmap of differential expressed isoforms in eight tissues. (b) Validation of differential expressed isoforms by RT-PCR. The isoform symbol was explained, for example, 8.1373.1 represented the first transcript of loci 8.1373 on the chromosome 8. 8.1373.1, 14.1634.2 and 15.158.2 belonged to the transcripts of novel loci; 5.203.1, 4.1478.1 and 1.1932.4 were known isoforms of annotated genes of SSC10.2 reference; 13.681.3, 1.3863.6 and 11.632.1 were new isoforms came from the annotated genes of SSC10.2 reference; 1.1177.1 and x.631.1 were novel isoforms of novel lncRNAs; 1.2355.7 and 7.168.3 were fusion transcripts and the genes below them represented their precursor genes." path-from-xml="dsy014f11.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122216/dsy014f11.tif" path-from-xml="dsy014f11.tif" target="_blank">View large</a><a section="120122216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f11.png?Expires=2147483647&Signature=F~S62d5mKAJy0ptb42LvFUHTIsCypWmQ0qBsKjh8H2EP0LV6j~zXzw15Kn215PjfTTRfcW3ub~vjJUrGJuWuWrEl4z1tf8NvGrY3dKWmlt-O2ACfPjqGo35ziDvNWBviHeQito74z5ZJn~w06fsG~kXRCzPwPDC0NPJw5BHN23lvnAE~THKiOlgFkcA7s6biiChHDvfMNJ4SC39s3M8ltn0ZUbdhgihbGYTjMmGYkecD-hkp-SWYftUbOwIW1Go~B0BDVY8vLps54ELAGDFuXH9sx21PSOsy1msO~Y43ZQubESsej80wEoXvaERkKJ-KlA9Kd5dI0k2L6Skm5LPf0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122216&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f11.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of differential expressed isoforms. (a) Heatmap of differential expressed isoforms in eight tissues. (b) Validation of differential expressed isoforms by RT-PCR. The isoform symbol was explained, for example, 8.1373.1 represented the first transcript of loci 8.1373 on the chromosome 8. 8.1373.1, 14.1634.2 and 15.158.2 belonged to the transcripts of novel loci; 5.203.1, 4.1478.1 and 1.1932.4 were known isoforms of annotated genes of SSC10.2 reference; 13.681.3, 1.3863.6 and 11.632.1 were new isoforms came from the annotated genes of SSC10.2 reference; 1.1177.1 and x.631.1 were novel isoforms of novel lncRNAs; 1.2355.7 and 7.168.3 were fusion transcripts and the genes below them represented their precursor genes.</p></div></div><div content-id="dsy014-F11" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 11</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/m_dsy014f11.png?Expires=2147483647&Signature=eF1A4rNJGo4a4l-G1iHkBd3vOrQ9X9y5sOfydUZLtK4HZzg1iiTqo4cQosqF0DQSPZrMB0uHN-RaoF8VNhWNcyPHcxw-VTOypOTFQsHTgF0s2CZUCuu~mNnEJHI3GKaO-1xbXga-GRqYl0RJeZdO5pldqXeVFvZScKU-4jaJw~Am3z9Kjw1PXoWW0bYGxFXHEDGqo5q8g4yzET9eL5Y4ntZ54ZSa1soULb~OdaWHBtwlMJ2-Z4PSYpH-d63AutI9qR54Q2dCx-E4MANzDWe6g-Rd8cnizCL6Y~vvwLmb61KzKIvwzULxs41eVeaNxrWa68zYYWG12D-NnkD-i-LClA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of differential expressed isoforms. (a) Heatmap of differential expressed isoforms in eight tissues. (b) Validation of differential expressed isoforms by RT-PCR. The isoform symbol was explained, for example, 8.1373.1 represented the first transcript of loci 8.1373 on the chromosome 8. 8.1373.1, 14.1634.2 and 15.158.2 belonged to the transcripts of novel loci; 5.203.1, 4.1478.1 and 1.1932.4 were known isoforms of annotated genes of SSC10.2 reference; 13.681.3, 1.3863.6 and 11.632.1 were new isoforms came from the annotated genes of SSC10.2 reference; 1.1177.1 and x.631.1 were novel isoforms of novel lncRNAs; 1.2355.7 and 7.168.3 were fusion transcripts and the genes below them represented their precursor genes." path-from-xml="dsy014f11.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122216/dsy014f11.tif" path-from-xml="dsy014f11.tif" target="_blank">View large</a><a section="120122216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014f11.png?Expires=2147483647&Signature=F~S62d5mKAJy0ptb42LvFUHTIsCypWmQ0qBsKjh8H2EP0LV6j~zXzw15Kn215PjfTTRfcW3ub~vjJUrGJuWuWrEl4z1tf8NvGrY3dKWmlt-O2ACfPjqGo35ziDvNWBviHeQito74z5ZJn~w06fsG~kXRCzPwPDC0NPJw5BHN23lvnAE~THKiOlgFkcA7s6biiChHDvfMNJ4SC39s3M8ltn0ZUbdhgihbGYTjMmGYkecD-hkp-SWYftUbOwIW1Go~B0BDVY8vLps54ELAGDFuXH9sx21PSOsy1msO~Y43ZQubESsej80wEoXvaERkKJ-KlA9Kd5dI0k2L6Skm5LPf0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122216&ar=5020793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy014f11.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of differential expressed isoforms. (a) Heatmap of differential expressed isoforms in eight tissues. (b) Validation of differential expressed isoforms by RT-PCR. The isoform symbol was explained, for example, 8.1373.1 represented the first transcript of loci 8.1373 on the chromosome 8. 8.1373.1, 14.1634.2 and 15.158.2 belonged to the transcripts of novel loci; 5.203.1, 4.1478.1 and 1.1932.4 were known isoforms of annotated genes of SSC10.2 reference; 13.681.3, 1.3863.6 and 11.632.1 were new isoforms came from the annotated genes of SSC10.2 reference; 1.1177.1 and x.631.1 were novel isoforms of novel lncRNAs; 1.2355.7 and 7.168.3 were fusion transcripts and the genes below them represented their precursor genes.</p></div></div>                            <h2 scrollto-destination=120122217 id="120122217" class="section-title" >4. Discussion</h2>
<p>In this work, we applied PacBio sequencing technique to investigate transcripts, provided the first comprehensive view of splice variants in pig and illustrated the advantage of Iso-Seq in identifying FL splice isoforms. Following the latest methodologies in analysing PacBio transcriptome data, we obtained 389,781 high-quality FLNC reads, with 77,075 isoforms covering 39,940 loci, 97,727 AS events corresponding to 2,637 models, 711 fusion isoforms and 4,394 novel lincRNAs that were not previously annotated in pig. The new resource and transcriptional information would be of great value to improve pig genome annotation and livestock transcriptome research.</p><p>This experimental design aimed at maximizing transcript diversity and investigating comprehensive splice isoforms by broadly sampling 38 tissues and organs. Compared with previous studies, this experiment by far used the most number of tissues to comprehensively research transcriptome isoform via SMRT methodology.<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Twenty cells in our study might not well uncover more low-abundance isoforms, but the data depth was adequate to explore as many FL splice isoforms as possible when depending on the porcine genome size and comparing with previous similar studies of SMRT transcriptomes.<a reveal-id="dsy014-B18" data-open="dsy014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsy014-B22" data-open="dsy014-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> In this study, only 0.6% of FLNCs was excavated in the 0–600 bp library, indicated that swine FL transcript might be more than 1 kb in length and short fragments mainly consisted of non-coding RNAs with a single exon. Thus, future research on FL transcript can mainly focus on longer than 0.6 kb reads.</p><p>AS plays important roles in regulating molecular, cellular, physiological and developmental processes/pathways in eukaryotes.<a reveal-id="dsy014-B60" data-open="dsy014-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> However, the difficulty of identifying the combinations of splice-site usage by Illumina short reads limits gene model prediction. Previous studies show that ∼30% of genes were alternatively spliced in pig, compared with 68% in humans, 57% in mouse and 21% in bovine.<a reveal-id="dsy014-B7" data-open="dsy014-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy014-B11" data-open="dsy014-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Our analysis shows that ∼17.66% loci (7,053/39,940) are alternatively spliced, and ∼42.38% (32,662/77,075) transcripts are associated with AS in Iso-Seq data, while only 7.77% (1,967/25,322) genes corresponding to 17.45% (5,336/30,585) isoforms are in pig reference annotation. Thus, a large of AS gene identified in this study will greatly improve reference annotation. Relatively, gene proportion of AS in Iso-Seq data is low, owing to slightly high single-exon isoform with little coding capacity and thus may represent lncRNAs.<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Furthermore, 7,051 multi-exon genes (99.97%) are alternatively spliced in Iso-Seq data, which was consistent with previous reports on humans,<a reveal-id="dsy014-B10" data-open="dsy014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> confirming that almost all multi-exon genes undergo AS to increase transcriptome diversity.</p><p>Hypothetically, variations in the EI methylation dynamics may result in ES. In fact, alternative exon recognition mechanisms may have evolved in genes with equal exon-to-intron DNA methylation ratios, and this phenomenon is one of the biological explanations given for the variation in AS patterns between species.<a reveal-id="dsy014-B61" data-open="dsy014-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsy014-B62" data-open="dsy014-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Interestingly, DNA methylation is enriched in AS sites and splicing regulatory motifs.<a reveal-id="dsy014-B63" data-open="dsy014-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> We found that DNA methylation enriching in AS sites varied from acceptor to donor sites and from sense strand to antisense strand. The trend of CG methylation is not as significant like that in plant, revealing that CG methylation promoted AS.<a reveal-id="dsy014-B18" data-open="dsy014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The sharp decline of cytosine coverage of EI and IE structure sites indicated that the change of cytosine content at 2–3 bp around the AS sites caused the change of methylation level and affected the occurrence of AS events. This finding provides evidence for the methylation of AS events depending on the structure of the AS sites. Moreover, our results revealed that CG methylation in promoter region repressed AS and regulated the first exon and that methylation of first exon also repressed AS in pig. Alternative first exon affected by the usage of alternative promoter was found to result in mRNA isoforms with distinct 5′ UTRs.<a reveal-id="dsy014-B8" data-open="dsy014-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> This phenomenon indicates that promoter methylation can enhance the first exon to repress the AS events in animals.</p><p>In this study, we identified 8,838 high-confidence lncRNA, including 4,394 novel lincRNA in pig. Owing to the lack of a good database, we constructed the PLEK model using human data, which included the most complete lncRNAs. Relatively, media length of lncRNA in pig (2,024 bp) was only half of that of human (4,096 bp). ∼90% of the lncRNAs belonging to the single exon confirmed that candidate genes with few introns showed little coding capacity and thus might represent lncRNAs, and isoforms with more introns corresponded to known (mostly protein coding) genes.<a reveal-id="dsy014-B20" data-open="dsy014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The expression of lncRNA is usually low, and we speculate that one reason is that lncRNAs with little exons rarely occur AS events. Consistent with previous work, we uncovered a high degree of tissue specificity among lncRNAs, a feature shared by other animals.<a reveal-id="dsy014-B64" data-open="dsy014-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsy014-B65" data-open="dsy014-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> In addition, we discovered that multi-exon lncRNAs expressed a higher level than single-exon lncRNAs. Methylation levels of the lncRNA genomic regions were significantly higher than that for the mRNA genes.<a reveal-id="dsy014-B66" data-open="dsy014-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Meanwhile, our work found that methylation levels of lncRNAs were higher than non-lncRNAs in the upstream region of TSS and downstream region of TTS, whereas non-lncRNAs with higher methylation levels existed in gene body. This phenomenon also appeared in single-exon and multi-exon lncRNAs, indicating that gene with high expression or coding capacity corresponds with low methylation level in TSS and TTS. However, high methylation level in gene body needs future research.</p><p>Overall, our study demonstrates that long-read sequencing complements short-read sequencing for cataloguing and quantifying eukaryotic transcripts. Based on FL transcripts, a large number of AS models and isoforms provide a more comprehensive foundation to explore transcriptome diversity. Our results revealed the unique species specificity of AS, the rule of fusion event, the specificity of isoform, the difference of lncRNA, the DNA methylation regulation on AS and lncRNA. These results not only significantly improve existing gene models of pig, but have revealed important rules and generated novel resource and information with positive implications for agricultural production or disease prevention.</p>                            <h3 scrollto-destination=120122224 id="120122224" class="section-title" >4.1 Availability of data and material</h3>
<p>The sequence data reported in this paper have been deposited in the Genome Sequence Archive (GSA; <a class="link link-uri" href="http://gsa.big.ac.cn/" target="_blank">http://gsa.big.ac.cn/</a>) of Beijing Institute of Genomics, Chinese Academy of Sciences, under accession number PRJCA000349.</p>                            <h2 scrollto-destination=120122226 id="120122226" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Many thanks to Frasergen Inc. (<a class="link link-uri" href="http://www.frasergen.com/" target="_blank">http://www.frasergen.com/</a>) for providing sequencing platform and technical assistance in data analyses.</p>                            <h3 scrollto-destination=120122228 id="120122228" class="section-title" >Conflict of interest</h3>
<p>The authors declare no competing financial interests.</p>                            <h3 scrollto-destination=120122230 id="120122230" class="section-title" >Funding</h3>
<p>This work was supported by National Natural Science Foundation of China (NSFC, 31472076), the Science Fund for Distinguished Young Scholars of Hubei Province of China (Grant No. 2014CFA024) and the Research Project of Chinese Ministry of Education (Grant No. 113048A).</p>                            <h2 scrollto-destination=120122232 id="120122232" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy014-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Critser</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Laughlin</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Prather</div> <div class="given-names">R.S.</div></div>, <div class="name string-name"><div class="surname">Riley</div> <div class="given-names">L.K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Proceedings of the conference on swine in biomedical research</div>, <div class="source">ILAR J</div> ., <div class="volume">50</div>, <div class="fpage">89</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%20conference%20on%20swine%20in%20biomedical%20research&amp;author=J.K.%20Critser&amp;author=M.H.%20Laughlin&amp;author=R.S.%20Prather&amp;author=L.K.%20Riley&amp;publication_year=2009&amp;journal=ILAR%20J&amp;volume=50&amp;pages=89-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/ilar.50.1.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Filar.50.1.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Filar.50.1.89"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19106456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gonzalez-Bulnes</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Astiz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ovilo</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Developmental origins of health and disease in swine: implications for animal production and biomedical research</div>, <div class="source">Theriogenology</div> , <div class="volume">86</div>, <div class="fpage">110</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20origins%20of%20health%20and%20disease%20in%20swine%3A%20implications%20for%20animal%20production%20and%20biomedical%20research&amp;author=A.%20Gonzalez-Bulnes&amp;author=S.%20Astiz&amp;author=C.%20Ovilo&amp;publication_year=2016&amp;journal=Theriogenology&amp;volume=86&amp;pages=110-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.theriogenology.2016.03.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.theriogenology.2016.03.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.theriogenology.2016.03.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27238437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Chung</div> <div class="given-names">W.H.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">K.T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Whole-genome resequencing analyses of five pig breeds, including Korean wild and native, and three European origin breeds</div>, <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">259</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20resequencing%20analyses%20of%20five%20pig%20breeds%2C%20including%20Korean%20wild%20and%20native%2C%20and%20three%20European%20origin%20breeds&amp;author=J.W.%20Choi&amp;author=W.H.%20Chung&amp;author=K.T.%20Lee&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=259-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26117497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shen-Gunther</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Poage</div> <div class="given-names">G.M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Molecular Pap smear: hPV genotype and DNA methylation of ADCY8, CDH8, and ZNF582 as an integrated biomarker for high-grade cervical cytology</div>, <div class="source">Clin. Epigenetics</div> , <div class="volume">8</div>, <div class="fpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20Pap%20smear%3A%20hPV%20genotype%20and%20DNA%20methylation%20of%20ADCY8%2C%20CDH8%2C%20and%20ZNF582%20as%20an%20integrated%20biomarker%20for%20high-grade%20cervical%20cytology&amp;author=J.%20Shen-Gunther&amp;author=C.M.%20Wang&amp;author=G.M.%20Poage&amp;publication_year=2016&amp;journal=Clin.%20Epigenetics&amp;volume=8&amp;pages=96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13148-016-0263-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13148-016-0263-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13148-016-0263-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27651839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Au</div> <div class="given-names">K.F.</div></div>, <div class="name string-name"><div class="surname">Sebastiano</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Afshar</div> <div class="given-names">P.T.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Characterization of the human ESC transcriptome by hybrid sequencing</div>, <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">110</div>, <div class="fpage">E4821</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20human%20ESC%20transcriptome%20by%20hybrid%20sequencing&amp;author=K.F.%20Au&amp;author=V.%20Sebastiano&amp;author=P.T.%20Afshar&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=110&amp;pages=E4821-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1320101110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1320101110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1320101110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">Y.M.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">K.T.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The Pig Genome Database (PiGenome): an integrated database for pig genome research</div>, <div class="source">Mamm. Genome</div> ., <div class="volume">20</div>, <div class="fpage">60</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pig%20Genome%20Database%20(PiGenome)%3A%20an%20integrated%20database%20for%20pig%20genome%20research&amp;author=D.%20Lim&amp;author=Y.M.%20Cho&amp;author=K.T.%20Lee&amp;publication_year=2009&amp;journal=Mamm.%20Genome&amp;volume=20&amp;pages=60-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-008-9156-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-008-9156-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-008-9156-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19082661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nygard</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Cirera</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gilchrist</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Gorodkin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jorgensen</div> <div class="given-names">C.B.</div></div>, <div class="name string-name"><div class="surname">Fredholm</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A study of alternative splicing in the pig</div>, <div class="source">BMC Res. Notes</div> , <div class="volume">3</div>, <div class="fpage">123</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20study%20of%20alternative%20splicing%20in%20the%20pig&amp;author=A.B.%20Nygard&amp;author=S.%20Cirera&amp;author=M.J.%20Gilchrist&amp;author=J.%20Gorodkin&amp;author=C.B.%20Jorgensen&amp;author=M.%20Fredholm&amp;publication_year=2010&amp;journal=BMC%20Res.%20Notes&amp;volume=3&amp;pages=123" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-3-123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-3-123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-3-123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">E.T.</div></div>, <div class="name string-name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Alternative isoform regulation in human tissue transcriptomes</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">470</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=470-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chow</div> <div class="given-names">L.T.</div></div>, <div class="name string-name"><div class="surname">Gelinas</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Broker</div> <div class="given-names">T.R.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">R.J.</div></div></div> <div class="year">1977</div>, 
<div class="article-title">An amazing sequence arrangement at the 5' ends of adenovirus 2 messenger RNA</div>, <div class="source">Cell</div> , <div class="volume">12</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20amazing%20sequence%20arrangement%20at%20the%205'%20ends%20of%20adenovirus%202%20messenger%20RNA&amp;author=L.T.%20Chow&amp;author=R.E.%20Gelinas&amp;author=T.R.%20Broker&amp;author=R.J.%20Roberts&amp;publication_year=1977&amp;journal=Cell&amp;volume=12&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(77)90180-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(77)90180-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(77)90180-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/902310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Shai</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">L.J.</div></div>, <div class="name string-name"><div class="surname">Frey</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genet</div> ., <div class="volume">40</div>, <div class="fpage">1413</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.J.%20Lee&amp;author=B.J.%20Frey&amp;author=B.J.%20Blencowe&amp;publication_year=2008&amp;journal=Nat.%20Genet&amp;volume=40&amp;pages=1413-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chacko</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Ranganathan</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide analysis of alternative splicing in cow: implications in bovine as a model for human diseases</div>, <div class="source">BMC Genomics</div> , <div class="volume">10</div>, <div class="fpage">S11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20alternative%20splicing%20in%20cow%3A%20implications%20in%20bovine%20as%20a%20model%20for%20human%20diseases&amp;author=E.%20Chacko&amp;author=S.%20Ranganathan&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=S11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-S3-S11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-S3-S11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-S3-S11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19958474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">A genomic view of alternative splicing</div>, <div class="source">Nat. Genet</div> ., <div class="volume">30</div>, <div class="fpage">13</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20view%20of%20alternative%20splicing&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2002&amp;journal=Nat.%20Genet&amp;volume=30&amp;pages=13-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0102-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0102-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0102-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paronetto</div> <div class="given-names">M.P.</div></div>, <div class="name string-name"><div class="surname">Passacantilli</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Sette</div> <div class="given-names">C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Alternative splicing and cell survival: from tissue homeostasis to disease</div>, <div class="source">Cell Death Differ</div> ., <div class="volume">23</div>, <div class="fpage">1919</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20cell%20survival%3A%20from%20tissue%20homeostasis%20to%20disease&amp;author=M.P.%20Paronetto&amp;author=I.%20Passacantilli&amp;author=C.%20Sette&amp;publication_year=2016&amp;journal=Cell%20Death%20Differ&amp;volume=23&amp;pages=1919-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cdd.2016.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcdd.2016.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcdd.2016.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27689872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dewaele</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tabaglio</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Willekens</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Antisense oligonucleotide-mediated MDM4 exon 6 skipping impairs tumor growth</div>, <div class="source">J. Clin. Invest</div> ., <div class="volume">126</div>, <div class="fpage">68</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antisense%20oligonucleotide-mediated%20MDM4%20exon%206%20skipping%20impairs%20tumor%20growth&amp;author=M.%20Dewaele&amp;author=T.%20Tabaglio&amp;author=K.%20Willekens&amp;publication_year=2015&amp;journal=J.%20Clin.%20Invest&amp;volume=126&amp;pages=68-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI82534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI82534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI82534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26595814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Min</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Survey of programs used to detect alternative splicing isoforms from deep sequencing data in silico</div>, <div class="source">Biomed. Res. Int</div> ., <div class="volume">2015</div>, <div class="fpage">1</div>.</div></div></div></div><div content-id="dsy014-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holoch</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Moazed</div> <div class="given-names">D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">RNA-mediated epigenetic regulation of gene expression</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">16</div>, <div class="fpage">71</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-mediated%20epigenetic%20regulation%20of%20gene%20expression&amp;author=D.%20Holoch&amp;author=D.%20Moazed&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=71-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3863"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25554358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Transcript isoform variation associated with cytosine modification in human lymphoblastoid cell lines</div>, <div class="source">Genetics</div> , <div class="volume">203</div>, <div class="fpage">985</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20isoform%20variation%20associated%20with%20cytosine%20modification%20in%20human%20lymphoblastoid%20cell%20lines&amp;author=X.%20Zhang&amp;author=W.%20Zhang&amp;publication_year=2016&amp;journal=Genetics&amp;volume=203&amp;pages=985-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.115.185504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.115.185504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.115.185504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27029734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Tseng</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Regulski</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing</div>, <div class="source">Nat. Comms</div> ., <div class="volume">7</div>, <div class="fpage">11708</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unveiling%20the%20complexity%20of%20the%20maize%20transcriptome%20by%20single-molecule%20long-read%20sequencing&amp;author=B.%20Wang&amp;author=E.%20Tseng&amp;author=M.%20Regulski&amp;publication_year=2016&amp;journal=Nat.%20Comms&amp;volume=7&amp;pages=11708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms11708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms11708"> </span></div> </div></div></div></div></div><div content-id="dsy014-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vembar</div> <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Seetin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lambert</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Complete telomere-to-telomere de novo assembly of the <em>Plasmodium falciparum</em> genome through long-read (&gt;11 kb), single molecule, real-time sequencing</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">339</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20telomere-to-telomere%20de%20novo%20assembly%20of%20the%20Plasmodium%20falciparum%20genome%20through%20long-read%20(%3E11%20kb)%2C%20single%20molecule%2C%20real-time%20sequencing&amp;author=S.S.%20Vembar&amp;author=M.%20Seetin&amp;author=C.%20Lambert&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=339-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27345719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharon</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tilgner</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Grubert</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Snyder</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">A single-molecule long-read survey of the human transcriptome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">1009</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single-molecule%20long-read%20survey%20of%20the%20human%20transcriptome&amp;author=D.%20Sharon&amp;author=H.%20Tilgner&amp;author=F.%20Grubert&amp;author=M.%20Snyder&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=1009-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24108091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tilgner</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jahanbani</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Blauwkamp</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">736</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20transcriptome%20analysis%20using%20synthetic%20long-read%20sequencing%20reveals%20molecular%20co-association%20of%20distant%20splicing%20events&amp;author=H.%20Tilgner&amp;author=F.%20Jahanbani&amp;author=T.%20Blauwkamp&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=736-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25985263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Underwood</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Tseng</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Holloway</div> <div class="given-names">A.K.</div></div></div> and <div class="collab">Bench To Basinet CvDC I.S</div>. <div class="year">2014</div>, 
<div class="article-title">Long-read sequencing of chicken transcripts and identification of new transcript isoforms</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e94650</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-read%20sequencing%20of%20chicken%20transcripts%20and%20identification%20of%20new%20transcript%20isoforms&amp;author=S.%20Thomas&amp;author=J.G.%20Underwood&amp;author=E.%20Tseng&amp;author=A.K.%20Holloway&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e94650" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0094650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0094650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0094650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24736250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordon</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Tseng</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Salamov</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Widespread polycistronic transcripts in fungi revealed by single-molecule mRNA sequencing</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0132628</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20polycistronic%20transcripts%20in%20fungi%20revealed%20by%20single-molecule%20mRNA%20sequencing&amp;author=S.P.%20Gordon&amp;author=E.%20Tseng&amp;author=A.%20Salamov&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0132628" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0132628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0132628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0132628"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26177194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Minoche</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Dohm</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Schneider</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Exploiting single-molecule transcript sequencing for eukaryotic gene prediction</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">184</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20single-molecule%20transcript%20sequencing%20for%20eukaryotic%20gene%20prediction&amp;author=A.E.%20Minoche&amp;author=J.C.%20Dohm&amp;author=J.%20Schneider&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0729-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0729-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0729-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26328666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hackl</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hedrich</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Forster</div> <div class="given-names">F.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">proovread: large-scale high-accuracy PacBio correction through iterative short read consensus</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3004</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=proovread%3A%20large-scale%20high-accuracy%20PacBio%20correction%20through%20iterative%20short%20read%20consensus&amp;author=T.%20Hackl&amp;author=R.%20Hedrich&amp;author=J.%20Schultz&amp;author=F.%20Forster&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3004-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25015988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Thorvaldsdottir</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Winckler</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Integrative genomics viewer</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">24</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20genomics%20viewer&amp;author=J.T.%20Robinson&amp;author=H.%20Thorvaldsdottir&amp;author=W.%20Winckler&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=24-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21221095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ast</div> <div class="given-names">G.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">How did alternative splicing evolve?</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">5</div>, <div class="fpage">773</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20did%20alternative%20splicing%20evolve%3F&amp;author=G.%20Ast&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet&amp;volume=5&amp;pages=773-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15510168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foissac</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sammeth</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">ASTALAVISTA: dynamic and flexible analysis of alternative splicing events in custom gene datasets</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W297</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ASTALAVISTA%3A%20dynamic%20and%20flexible%20analysis%20of%20alternative%20splicing%20events%20in%20custom%20gene%20datasets&amp;author=S.%20Foissac&amp;author=M.%20Sammeth&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W297-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">IBS: an illustrator for the presentation and visualization of biological sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3359</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IBS%3A%20an%20illustrator%20for%20the%20presentation%20and%20visualization%20of%20biological%20sequences&amp;author=W.%20Liu&amp;author=Y.%20Xie&amp;author=J.%20Ma&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3359-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26069263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">PLEK: a tool for predicting long non-coding RNAs and messenger RNAs based on an improved k-mer scheme</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">15</div>, <div class="fpage">311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLEK%3A%20a%20tool%20for%20predicting%20long%20non-coding%20RNAs%20and%20messenger%20RNAs%20based%20on%20an%20improved%20k-mer%20scheme&amp;author=A.%20Li&amp;author=J.%20Zhang&amp;author=Z.%20Zhou&amp;publication_year=2014&amp;journal=BMC%20Bioinformatics&amp;volume=15&amp;pages=311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-15-311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-15-311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-15-311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25239089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">ALDB: a domestic-animal long noncoding RNA database</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0124003</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ALDB%3A%20a%20domestic-animal%20long%20noncoding%20RNA%20database&amp;author=A.%20Li&amp;author=J.%20Zhang&amp;author=Z.%20Zhou&amp;author=L.%20Wang&amp;author=Y.%20Liu&amp;author=Y.%20Liu&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0124003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0124003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0124003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0124003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25853886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pyl</div> <div class="given-names">P.T.</div></div>, <div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">HTSeq–a Python framework to work with high-throughput sequencing data</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">166</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HTSeq%E2%80%93a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&amp;author=S.%20Anders&amp;author=P.T.%20Pyl&amp;author=W.%20Huber&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu638" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu638" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu638"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25260700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Shirley Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">GFOLD: a generalized fold change for ranking differentially expressed genes from RNA-seq data</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">2782</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GFOLD%3A%20a%20generalized%20fold%20change%20for%20ranking%20differentially%20expressed%20genes%20from%20RNA-seq%20data&amp;author=J.%20Feng&amp;author=C.A.%20Meyer&amp;author=Q.%20Wang&amp;author=J.S.%20Liu&amp;author=X.%20Shirley%20Liu&amp;author=Y.%20Zhang&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2782-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts515" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts515" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts515"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Steijger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Abril</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Engstrom</div> <div class="given-names">P.G.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1177</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.F.%20Abril&amp;author=P.G.%20Engstrom&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reecy</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Bidwell</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Briley</div> <div class="given-names">G.P.</div></div>, <div class="name string-name"><div class="surname">Grant</div> <div class="given-names">A.L.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Structure and regulation of the porcine skeletal alpha-actin-encoding gene</div>, <div class="source">Gene</div> , <div class="volume">180</div>, <div class="fpage">23</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20regulation%20of%20the%20porcine%20skeletal%20alpha-actin-encoding%20gene&amp;author=J.M.%20Reecy&amp;author=C.A.%20Bidwell&amp;author=G.P.%20Briley&amp;author=A.L.%20Grant&amp;publication_year=1996&amp;journal=Gene&amp;volume=180&amp;pages=23-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(96)00394-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(96)00394-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(96)00394-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8973342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abdel-Ghany</div> <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Hamilton</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jacobi</div> <div class="given-names">J.L.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A survey of the sorghum transcriptome using single-molecule long reads</div>, <div class="source">Nat. Comms</div> ., <div class="volume">7</div>, <div class="fpage">11706</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20the%20sorghum%20transcriptome%20using%20single-molecule%20long%20reads&amp;author=S.E.%20Abdel-Ghany&amp;author=M.%20Hamilton&amp;author=J.L.%20Jacobi&amp;publication_year=2016&amp;journal=Nat.%20Comms&amp;volume=7&amp;pages=11706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms11706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms11706"> </span></div> </div></div></div></div></div><div content-id="dsy014-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nilsen</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Graveley</div> <div class="given-names">B.R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Expansion of the eukaryotic proteome by alternative splicing</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">457</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20of%20the%20eukaryotic%20proteome%20by%20alternative%20splicing&amp;author=T.W.%20Nilsen&amp;author=B.R.%20Graveley&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=457-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le Guennec</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Roos-Weil</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mokhtari</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Granulomatous angiitis of the CNS revealing a Hodgkin lymphoma</div>, <div class="source">Neurology</div> , <div class="volume">80</div>, <div class="fpage">323</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Granulomatous%20angiitis%20of%20the%20CNS%20revealing%20a%20Hodgkin%20lymphoma&amp;author=L.%20Le%20Guennec&amp;author=D.%20Roos-Weil&amp;author=K.%20Mokhtari&amp;publication_year=2013&amp;journal=Neurology&amp;volume=80&amp;pages=323-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1212/WNL.0b013e31827deb26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1212%2FWNL.0b013e31827deb26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1212%2FWNL.0b013e31827deb26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23269593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winter</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Joureau</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">E.J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Mutation-specific effects on thin filament length in thin filament myopathy</div>, <div class="source">Ann. Neurol</div> ., <div class="volume">79</div>, <div class="fpage">959</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation-specific%20effects%20on%20thin%20filament%20length%20in%20thin%20filament%20myopathy&amp;author=J.M.%20Winter&amp;author=B.%20Joureau&amp;author=E.J.%20Lee&amp;publication_year=2016&amp;journal=Ann.%20Neurol&amp;volume=79&amp;pages=959-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ana.24654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fana.24654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fana.24654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27074222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Tong</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Generation of an efficient artificial promoter of bovine skeletal muscle alpha-actin gene (ACTA1) through addition of cis-acting element</div>, <div class="source">Cell Mol. Biol. Lett</div> ., <div class="volume">20</div>, <div class="fpage">160</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20an%20efficient%20artificial%20promoter%20of%20bovine%20skeletal%20muscle%20alpha-actin%20gene%20(ACTA1)%20through%20addition%20of%20cis-acting%20element&amp;author=Q.%20Hu&amp;author=H.%20Tong&amp;author=D.%20Zhao&amp;publication_year=2015&amp;journal=Cell%20Mol.%20Biol.%20Lett&amp;volume=20&amp;pages=160-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1515/cmble-2015-0009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2Fcmble-2015-0009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1515%2Fcmble-2015-0009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26204400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gelfman</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cohen</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Yearim</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ast</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">DNA-methylation effect on cotranscriptional splicing is dependent on GC architecture of the exon-intron structure</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">789</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-methylation%20effect%20on%20cotranscriptional%20splicing%20is%20dependent%20on%20GC%20architecture%20of%20the%20exon-intron%20structure&amp;author=S.%20Gelfman&amp;author=N.%20Cohen&amp;author=A.%20Yearim&amp;author=G.%20Ast&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=789-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.143503.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.143503.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.143503.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23502848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lister</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Pelizzola</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dowen</div> <div class="given-names">R.H.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Human DNA methylomes at base resolution show widespread epigenomic differences</div>, <div class="source">Nature</div> , <div class="volume">462</div>, <div class="fpage">315</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20DNA%20methylomes%20at%20base%20resolution%20show%20widespread%20epigenomic%20differences&amp;author=R.%20Lister&amp;author=M.%20Pelizzola&amp;author=R.H.%20Dowen&amp;publication_year=2009&amp;journal=Nature&amp;volume=462&amp;pages=315-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08514" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08514" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08514"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19829295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laurent</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Dynamic changes in the human methylome during differentiation</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">320</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20changes%20in%20the%20human%20methylome%20during%20differentiation&amp;author=L.%20Laurent&amp;author=E.%20Wong&amp;author=G.%20Li&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=320-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.101907.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.101907.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.101907.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20133333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gelfman</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ast</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">When epigenetics meets alternative splicing: the roles of DNA methylation and GC architecture</div>, <div class="source">Epigenomics</div> , <div class="volume">5</div>, <div class="fpage">351</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=When%20epigenetics%20meets%20alternative%20splicing%3A%20the%20roles%20of%20DNA%20methylation%20and%20GC%20architecture&amp;author=S.%20Gelfman&amp;author=G.%20Ast&amp;publication_year=2013&amp;journal=Epigenomics&amp;volume=5&amp;pages=351-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2217/epi.13.32" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2217%2Fepi.13.32" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2217%2Fepi.13.32"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23895647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Conservation and divergence of methylation patterning in plants and animals</div>, <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">107</div>, <div class="fpage">8689</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20divergence%20of%20methylation%20patterning%20in%20plants%20and%20animals&amp;author=S.%20Feng&amp;author=S.J.%20Cokus&amp;author=X.%20Zhang&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=107&amp;pages=8689-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1002720107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1002720107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1002720107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20395551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Bird</div> <div class="given-names">A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">DNA methylation landscapes: provocative insights from epigenomics</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">9</div>, <div class="fpage">465</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20landscapes%3A%20provocative%20insights%20from%20epigenomics&amp;author=M.M.%20Suzuki&amp;author=A.%20Bird&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet&amp;volume=9&amp;pages=465-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weirather</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Afshar</div> <div class="given-names">P.T.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">T.A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Characterization of fusion genes and the significantly expressed fusion isoforms in breast cancer by hybrid sequencing</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">e116</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20fusion%20genes%20and%20the%20significantly%20expressed%20fusion%20isoforms%20in%20breast%20cancer%20by%20hybrid%20sequencing&amp;author=J.L.%20Weirather&amp;author=P.T.%20Afshar&amp;author=T.A.%20Clark&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=e116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26040699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">mTORC1-mediated downregulation of COX2 restrains tumor growth caused by TSC2 deficiency</div>, <div class="source">Oncotarget</div> , <div class="volume">7</div>, <div class="fpage">28435</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mTORC1-mediated%20downregulation%20of%20COX2%20restrains%20tumor%20growth%20caused%20by%20TSC2%20deficiency&amp;author=H.%20Li&amp;author=F.%20Jin&amp;author=K.%20Jiang&amp;publication_year=2016&amp;journal=Oncotarget&amp;volume=7&amp;pages=28435-47" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27078846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cherukuri</div> <div class="given-names">D.P.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">T.O.</div></div>, <div class="name string-name"><div class="surname">Chun</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Targeted Cox2 gene deletion in intestinal epithelial cells decreases tumorigenesis in female, but not male, ApcMin/+ mice</div>, <div class="source">Mol. Oncol</div> ., <div class="volume">8</div>, <div class="fpage">169</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20Cox2%20gene%20deletion%20in%20intestinal%20epithelial%20cells%20decreases%20tumorigenesis%20in%20female%2C%20but%20not%20male%2C%20ApcMin%2F%2B%20mice&amp;author=D.P.%20Cherukuri&amp;author=T.O.%20Ishikawa&amp;author=P.%20Chun&amp;publication_year=2014&amp;journal=Mol.%20Oncol&amp;volume=8&amp;pages=169-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molonc.2013.10.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molonc.2013.10.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molonc.2013.10.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24268915" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kota</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Wagner</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Huerta</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Underwood</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Nickerson</div> <div class="given-names">J.A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Binding of ATP to UAP56 is necessary for mRNA export</div>, <div class="source">J. Cell Sci</div> ., <div class="volume">121</div>, <div class="fpage">1526</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Binding%20of%20ATP%20to%20UAP56%20is%20necessary%20for%20mRNA%20export&amp;author=K.P.%20Kota&amp;author=S.R.%20Wagner&amp;author=E.%20Huerta&amp;author=J.M.%20Underwood&amp;author=J.A.%20Nickerson&amp;publication_year=2008&amp;journal=J.%20Cell%20Sci&amp;volume=121&amp;pages=1526-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.021055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.021055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.021055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18411249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Merino</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kaslow</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Aissani</div> <div class="given-names">B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Structure of tumor necrosis factor-alpha haploblocks in European populations</div>, <div class="source">Immunogenetics</div> , <div class="volume">65</div>, <div class="fpage">543</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20tumor%20necrosis%20factor-alpha%20haploblocks%20in%20European%20populations&amp;author=A.M.%20Merino&amp;author=K.%20Zhang&amp;author=R.A.%20Kaslow&amp;author=B.%20Aissani&amp;publication_year=2013&amp;journal=Immunogenetics&amp;volume=65&amp;pages=543-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00251-013-0700-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00251-013-0700-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00251-013-0700-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23579626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ernst</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bar-Joseph</div> <div class="given-names">Z.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">STEM: a tool for the analysis of short time series gene expression data</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">7</div>, <div class="fpage">191</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STEM%3A%20a%20tool%20for%20the%20analysis%20of%20short%20time%20series%20gene%20expression%20data&amp;author=J.%20Ernst&amp;author=Z.%20Bar-Joseph&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=191" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beaton</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Rudigier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Moest</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">TUSC5 regulates insulin-mediated adipose tissue glucose uptake by modulation of GLUT4 recycling</div>, <div class="source">Mol. Metab</div> ., <div class="volume">4</div>, <div class="fpage">795</div>–<div class="lpage">810</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TUSC5%20regulates%20insulin-mediated%20adipose%20tissue%20glucose%20uptake%20by%20modulation%20of%20GLUT4%20recycling&amp;author=N.%20Beaton&amp;author=C.%20Rudigier&amp;author=H.%20Moest&amp;publication_year=2015&amp;journal=Mol.%20Metab&amp;volume=4&amp;pages=795-810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molmet.2015.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molmet.2015.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molmet.2015.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26629404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quintens</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lemaire</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Mice deficient in the respiratory chain gene Cox6a2 are protected against high-fat diet-induced obesity and insulin resistance</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e56719</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mice%20deficient%20in%20the%20respiratory%20chain%20gene%20Cox6a2%20are%20protected%20against%20high-fat%20diet-induced%20obesity%20and%20insulin%20resistance&amp;author=R.%20Quintens&amp;author=S.%20Singh&amp;author=K.%20Lemaire&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e56719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0056719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0056719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0056719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23460811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finno</div> <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Gianino</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Perumbakkam</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2018</div>, 
<div class="article-title">A missense mutation in MYH1 is associated with susceptibility to immune-mediated myositis in Quarter Horses</div>, <div class="source">Skelet. Muscle</div> , <div class="volume">8</div>, <div class="fpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20missense%20mutation%20in%20MYH1%20is%20associated%20with%20susceptibility%20to%20immune-mediated%20myositis%20in%20Quarter%20Horses&amp;author=C.J.%20Finno&amp;author=G.%20Gianino&amp;author=S.%20Perumbakkam&amp;publication_year=2018&amp;journal=Skelet.%20Muscle&amp;volume=8&amp;pages=7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13395-018-0155-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13395-018-0155-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13395-018-0155-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29510741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eulitz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sauer</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Pelissier</div> <div class="given-names">M.C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification of Xin-repeat proteins as novel ligands of the SH3 domains of nebulin and nebulette and analysis of their interaction during myofibril formation and remodeling</div>, <div class="source">Mol. Biol. Cell</div> , <div class="volume">24</div>, <div class="fpage">3215</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Xin-repeat%20proteins%20as%20novel%20ligands%20of%20the%20SH3%20domains%20of%20nebulin%20and%20nebulette%20and%20analysis%20of%20their%20interaction%20during%20myofibril%20formation%20and%20remodeling&amp;author=S.%20Eulitz&amp;author=F.%20Sauer&amp;author=M.C.%20Pelissier&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Cell&amp;volume=24&amp;pages=3215-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.e13-04-0202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.e13-04-0202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.e13-04-0202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23985323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.T.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Muench</div> <div class="given-names">M.O.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A global DNA methylation and gene expression analysis of early human B-cell development reveals a demethylation signature and transcription factor network</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">11339</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20DNA%20methylation%20and%20gene%20expression%20analysis%20of%20early%20human%20B-cell%20development%20reveals%20a%20demethylation%20signature%20and%20transcription%20factor%20network&amp;author=S.T.%20Lee&amp;author=Y.%20Xiao&amp;author=M.O.%20Muench&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=11339-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks957" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks957"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23074194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kalsotra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cooper</div> <div class="given-names">T.A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Functional consequences of developmentally regulated alternative splicing</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">715</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20consequences%20of%20developmentally%20regulated%20alternative%20splicing&amp;author=A.%20Kalsotra&amp;author=T.A.%20Cooper&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=715-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Merkin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1593</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20gene%20and%20isoform%20regulation%20in%20Mammalian%20tissues&amp;author=J.%20Merkin&amp;author=C.%20Russell&amp;author=P.%20Chen&amp;author=C.B.%20Burge&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1593-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1228186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1228186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1228186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbosa-Morais</div> <div class="given-names">N.L.</div></div>, <div class="name string-name"><div class="surname">Irimia</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The evolutionary landscape of alternative splicing in vertebrate species</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1587</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20landscape%20of%20alternative%20splicing%20in%20vertebrate%20species&amp;author=N.L.%20Barbosa-Morais&amp;author=M.%20Irimia&amp;author=Q.%20Pan&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1587-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1230612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1230612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1230612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anastasiadou</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Malousi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Maglaveras</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kouidou</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Human epigenome data reveal increased CpG methylation in alternatively spliced sites and putative exonic splicing enhancers</div>, <div class="source">DNA Cell Biol</div> ., <div class="volume">30</div>, <div class="fpage">267</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20epigenome%20data%20reveal%20increased%20CpG%20methylation%20in%20alternatively%20spliced%20sites%20and%20putative%20exonic%20splicing%20enhancers&amp;author=C.%20Anastasiadou&amp;author=A.%20Malousi&amp;author=N.%20Maglaveras&amp;author=S.%20Kouidou&amp;publication_year=2011&amp;journal=DNA%20Cell%20Biol&amp;volume=30&amp;pages=267-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/dna.2010.1094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fdna.2010.1094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fdna.2010.1094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21545276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bakhtiarizadeh</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Hosseinpour</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Arefnezhad</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Shamabadi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Salami</div> <div class="given-names">S.A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">In silico prediction of long intergenic non-coding RNAs in sheep</div>, <div class="source">Genome</div> , <div class="volume">59</div>, <div class="fpage">263</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20prediction%20of%20long%20intergenic%20non-coding%20RNAs%20in%20sheep&amp;author=M.R.%20Bakhtiarizadeh&amp;author=B.%20Hosseinpour&amp;author=B.%20Arefnezhad&amp;author=N.%20Shamabadi&amp;author=S.A.%20Salami&amp;publication_year=2016&amp;journal=Genome&amp;volume=59&amp;pages=263-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-2015-0141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-2015-0141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-2015-0141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27002388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koufariotis</div> <div class="given-names">L.T.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.P.</div></div>, <div class="name string-name"><div class="surname">Chamberlain</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vander Jagt</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hayes</div> <div class="given-names">B.J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A catalogue of novel bovine long noncoding RNA across 18 tissues</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0141225</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20catalogue%20of%20novel%20bovine%20long%20noncoding%20RNA%20across%2018%20tissues&amp;author=L.T.%20Koufariotis&amp;author=Y.P.%20Chen&amp;author=A.%20Chamberlain&amp;author=C.%20Vander%20Jagt&amp;author=B.J.%20Hayes&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0141225" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0141225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0141225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0141225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26496443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy014-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy014-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.G.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">DNA methylation signatures of long intergenic noncoding RNAs in porcine adipose and muscle tissues</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">15435</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20signatures%20of%20long%20intergenic%20noncoding%20RNAs%20in%20porcine%20adipose%20and%20muscle%20tissues&amp;author=Z.Y.%20Zhou&amp;author=A.%20Li&amp;author=L.G.%20Wang&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=15435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep15435" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep15435" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep15435"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26493951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5020793&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s1.png?Expires=2147483647&amp;Signature=ghSl9yYfAWLoZRQd1ZL5ht8gfvc4qnI5CtWTGAI2r4c74Q5Q6jB9hsWEJ~gZnLZKymfO-Ejt1LseOFcLUmaX3DzStukC5Z6wVPxZa17NfzwoMLDmGYMxD6pq8lBkxFxLT64SLMu4PRJV6YOb9GlihKLbz-QzE2szyueH3CB0~DN7M~ZzKX0ae-ph37W8-mEN~ug6HGiDT5WpbYs4dDRZm4d91BcfmJeG1dZP3Z9Kgo4g2tfmYHrNUEi5i-TwNv820ayw2ufcu-rZ6~pwMrBda~3FgpDcAOFq9Lh4czdlAKnHGQ9lnYC84TB-NLBOh3HTipdQbtr8hOzEVpggABbGFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s2.png?Expires=2147483647&amp;Signature=pL21NAjwIvUzKk5qnDoMMVQNBfsUqXNgB9Q03ZtkcTTsEnsUEBk3G4t7yexITMfctZl7vpyNW6VP-bNFYpNNxb7mc3K-zfM6fmielPdqa~91wJ0tw2L93ycwpV1Jp-F0~302tWJ6XOMvdpbqwwii6BLY5tlTc640YCbET0RjDXCJfn56UmPWNgCc5F0XD~ph6mpD7tXsvcX7os6pVDaEqwbOp3Oem1vZFGBCOh2bKnegcUHYZOCmOxCKyFOSL8OH2LVkdvYrgPRRCYCvjbACOSUIowHAV23g3h0Tn7mrFr0rAk7j~bmLfNy9P2DozD4HoPyQ69rs7WiL15H6QGJDPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s3.png?Expires=2147483647&amp;Signature=x3i7228YriMM1jcR6-pjCUooKhJMbfKZGPHrWzhCWLvUvgztrNSRjLiUOXvUXnzRSsPAbOwMMeoQYF2aB46LrLu2QOogtXhEAqyp3RFUpY2IBN18TDKIwy~uV2KFX7alJCfwAfC3RipvNWSAowmos-P-0odcD2KY4G9ie1~4yeWoiMLIE6E-r1xPH4pcfawUqsoZLIxI4Us9Kj9IR-LJIytBHoC48tSiNUCnOipWju-ztpbS3rSp8mofFt6FwGw28VPVSWEDT0x6SVZH64aqOpcGjUh~LHnufs5NZlBeLODMBj6u-TiA5JMP6LAzkiOI7rjT0K4WCaYwX3oOqI~VwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s4.png?Expires=2147483647&amp;Signature=Gb3qPOpGxq71-hSZlu6-~zows4F~5jwZ5ZCVNNnf2L4NQT0SMFATM0PB7hwMjl3Zt-9HiheTDNBJcFkJKrdykl3~VJbaz6~AHaqOltnaQiHjQSUFBDiBeYVHQjN-Ryzv3n96YDTohT-5fFLz64WPXaVgzKg~ELZGAHAbdzVfQvCpi1yttSCUNZNr~SUI6fdMrVf3UJ8-RH~3dvug9LQE12xr3TC7xZsGdkTyvqMiqGycC2b6SfOhwXUhMv-c~X2SDKab7ivCIiJ5Tt6wQbevzPzDiPRitP1lwUBUE0ln57H7ZxWP1zQH16W~oIbcRCkUV5cVhy2rbppDgS4Fqe3ijw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S4</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_figure_s5.png?Expires=2147483647&amp;Signature=n6PKBzPrNmQ8wqAD2wbQV3VgrSE53Y8~uFXB~z53BDGo5-DHWJnVWiI-0L6TLtFS3ZOy4oX4R1THnsBzAlO8Z8sbZ55o6dzD-UtKz7nPoplrj~vpB7d6WQYhd3KCRUZCDu97GgCWFpuwtUuJfpd9a~Dch5DhQo4VQWA0sHekwExhBj-XtEl4FUYG3bofwNlthJ2A-FZ9DQZx7xaVfxa7Y7L2hZJ3TJrGtcrjP1FO~v4St2Rfg4wKyWsQOeTNZUE~4TAQ25bALc31IEvMUb4KjMu9aaGJuKY~rQ5Wc8Yuu7~~WyGoz7yRqR9vry9mcefY47L-Gwl0ihBRiL3ovvuyeQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S5</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s1-13.docx?Expires=2147483647&amp;Signature=ae8Di~GvU0WTPJf3YjFijXYFj0pElfMuKdMIVDsxtBWeKiNepJ35ieF~zYAlXhX7z2xIlhzqHxZXS5F~AZE5LpxkRhJEQGu799hZ6n7wI0SbfswJpyVvPelcybC0IqrMe-uEIL-GhvyWq0vly5af~NWTPjI0cHxRh1XBC7noC9SIEBgCQbyJ5aUPQllEFJNjKoWW9rhyqJY-G8-UhSswyaUawSf7ssTsGTYG48tVPUwPJOdNFp85iCdfez51aMzo8T7ajrG6YJpgsz8VVHuqJL331RYQAgO8wlYcCWfCAcvpVEPrOfuK8uHdkKrlstZm579jBQjiTwdmW031CMkehQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1-13</div></a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s7.xlsx?Expires=2147483647&amp;Signature=E0cuVHQ1hjFCC~AI5VrDwRmuylYgQvz2shcXcnA~zA9kZePxlm5zcLQeYapVFj1Hu2ntq~A5cX-6NT1pj0lg-NNOxHgIHXzPGAuWojT8O9ofWuPzQVBK56RqQjxo1DtWfh7Vqd4pzpahnPurdY6mg3~hE3eOG1U~E02S4DitkdMgZhCl13-ysduUQClpGrPZQt1IjYYkkkptpUJzotZ3PkbHxHLfkJDRlWPPKRoRkRKHO0ZUp6Vr1UByd9JYNZiHOP1CcCxXOgySOeyq1gBneDPqXaRqiF4eM15LC2GADXoDXQgisoDwdTq8qz5voGUtHm4F7KsoVZQTd~n4AUpgbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S7</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s8.xlsx?Expires=2147483647&amp;Signature=xXOmHahVg~Uy0JLL4Z9~uvjrTzQX4Gm63dmSkjOD27f2YXBXR~3lDIbtDKWRtQ07HhS665pzo-XDDmCCCaDN3bYmI18eSwndTkns95OLndYT2F94mZ~B7T7RmS7btFt2Oq7k0NLjcj2F6gG5ZilwiDfpYkoO4iIxkcGefstoAg2v4K7nbWOquVClBmNr3UoPpiuDjAScfQ0FmcuhQ66LV4vGPewf23CET-wbn8mLJtS11sHvhfRsEIrgG3RkmiqeHihkUdK2gdr-Z53SCl1l~bc14Q6Ypihro9UJenf-bGtx04-XdB0NefKThLy7~4Zrvyfw-xwYwRblCTuEs9HUFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s9.xlsx?Expires=2147483647&amp;Signature=LIDyG6fFeriAmmKf6xq7lMd6oRxqVZ7q2XfRCcNZ4QtfzscvguU~nKDeeVW9zAvanBkfSmeNaynCtfzCmknHGQ3waUcricTgUcUuYiPbjLGPwc2PgKbhyoIqoxq2lBmoo-kSiN~2PZjzdlmz4hjVNjMwhehQuD-YPS0-qfpqMxxNGSoSr7BqTCbazi6JZ43tPGrjDcGijN09hy7~D0xJGCJvjkpym52PQF2-q-Bap~jXvxX8YxEHru6bb0FcRmzbA2jp52DwRwNM~gJWmsVahH-UUZX1MoWf4Pbzze7sJZdai244XWM2ItHUVD~h0H6hMjuZcYYATYv5U5mg7Y3cQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S9</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy014/1/dsy014_table_s11.xlsx?Expires=2147483647&amp;Signature=e78bgYQbpHN6NQ0GjDoufRwNvw2gJ~sOYeJwNFDhgdT~CydnbHo9T7CaOcbgoGOjJR58vf~stacSFAJyIRaQ5ia6~KRl97L0~uTuqC~Bp3rmeINRhhf-Mv9OJcaEIC-4T0nUicttZCwCMoMjphn5~e8rMXHDY6Ac9-PV5SghKaU8L~gbD1d1Vo-JF5uocIseOqK5z7c6~dZC9igcL2oGkdJra~9ZB32yv2--u96E6-NEe8YqHXbcWNGgJMvLNiUeucOz7fVLZJeGap7rO2ooT4O8oJOmpiibgBrLGHuPsArjETXECRfttmhlXkwLSKZBTow2xG5nnt2B6MiJ87BJ~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S11</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,218</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5020793"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f421%2f5020793">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F421%2F5020793" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5020793'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="survey of transcriptome complexity in Sus scrofa using single-molecule long-read sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f421%2f5020793">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

