

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/395/4989276","@type":"ScholarlyArticle","name":"Detection of genome-edited cells by oligoribonucleotide interference-PCR","datePublished":"2018-04-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy012","keywords":["PCR","ORNi-PCR","genome editing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/395/4989276","author":[{"name":"Fujita, Toshitsugu","affiliation":"Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan  Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan","@type":"Person"},{"name":"Yuno, Miyuki","affiliation":"Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan","@type":"Person"},{"name":"Kitaura, Fusako","affiliation":"Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan","@type":"Person"},{"name":"Fujii, Hodaka","affiliation":"Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan  Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan","@type":"Person"}],"description":"Abstract.  Genome editing by engineered sequence-specific nucleases, such as the clustered regularly interspaced short palindromic repeats (CRISPR) system is wi","pageStart":"395","pageEnd":"407","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Detection of genome-edited cells by oligoribonucleotide interference-PCR","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Detection of genome-edited cells by oligoribonucleotide interference-PCR" />
    <meta property="og:description" content="Abstract.  Genome editing by engineered sequence-specific nucleases, such as the clustered regularly interspaced short palindromic repeats (CRISPR) system is wi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/395/4989276" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-04-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Fujita, Toshitsugu" /><meta name="citation_author_institution" content="Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan" /><meta name="citation_author_institution" content="Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan" /><meta name="citation_author" content="Yuno, Miyuki" /><meta name="citation_author_institution" content="Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan" /><meta name="citation_author" content="Kitaura, Fusako" /><meta name="citation_author_institution" content="Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan" /><meta name="citation_author" content="Fujii, Hodaka" /><meta name="citation_author_institution" content="Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan" /><meta name="citation_author_institution" content="Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan" /><meta name="citation_title" content="Detection of genome-edited cells by oligoribonucleotide interference-PCR" /><meta name="citation_firstpage" content="395" /><meta name="citation_lastpage" content="407" /><meta name="citation_doi" content="10.1093/dnares/dsy012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering;  citation_journal_title=Trends Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=397-405; " /><meta name="citation_reference" content="citation_title=Genome editing. The new frontier of genome engineering with CRISPR-Cas9;  citation_journal_title=Science;  citation_year=2014;  citation_volume=346;  citation_pages=1258096" /><meta name="citation_reference" content="citation_title=A CRISPR view of development;  citation_journal_title=Genes Dev;  citation_year=2014;  citation_volume=28;  citation_pages=1859-72; " /><meta name="citation_reference" content="citation_title=Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases;  citation_journal_title=Biotechnol. Adv;  citation_year=2017;  citation_volume=35;  citation_pages=95-104; " /><meta name="citation_reference" content="citation_title=An efficient genotyping method for genome-modified animals and human cells generated with CRISPR/Cas9 system;  citation_journal_title=Sci. Rep;  citation_year=2014;  citation_volume=4;  citation_pages=6420" /><meta name="citation_reference" content="citation_title=Fast and sensitive detection of indels induced by precise gene targeting;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=e59" /><meta name="citation_reference" content="citation_title=High-throughput genotyping of CRISPR/Cas9-mediated mutants using fluorescent PCR-capillary gel electrophoresis;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=15587" /><meta name="citation_reference" content="citation_title=Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002861" /><meta name="citation_reference" content="citation_title=A PCR based protocol for detecting indel mutations induced by TALENs and CRISPR/Cas9 in zebrafish;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e98282" /><meta name="citation_reference" content="citation_title=mRNA transfection of a novel TAL effector nuclease (TALEN) facilitates efficient knockout of HIV co-receptor CCR5;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=5560-71; " /><meta name="citation_reference" content="citation_title=Isolation of single-base genome-edited human iPS cells without antibiotic selection;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=291-3; " /><meta name="citation_reference" content="citation_title=A digital PCR-based method for efficient and highly specific screening of genome edited cells;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0153901" /><meta name="citation_reference" content="citation_title=A simple and efficient method for CRISPR/Cas9-induced mutant screening;  citation_journal_title=J. Genet. Genomics;  citation_year=2017;  citation_volume=44;  citation_pages=207-13; " /><meta name="citation_reference" content="citation_title=Detection of genome-edited mutant clones by a simple competition-based PCR method;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0179165" /><meta name="citation_reference" content="citation_title=Oligoribonucleotide (ORN) interference-PCR (ORNi-PCR): a simple method for suppressing PCR amplification of specific DNA sequences using ORNs;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e113345" /><meta name="citation_reference" content="citation_title=RNA-guided human genome engineering via Cas9;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=823-6; " /><meta name="citation_reference" content="citation_title=Allele-specific locus binding and genome editing by CRISPR at the p16INK4a locus;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=30485" /><meta name="citation_reference" content="citation_title=A programmable dual-RNA- guided DNA endonuclease in adaptive bacterial immunity;  citation_journal_title=Science;  citation_year=2012;  citation_volume=337;  citation_pages=816-21; " /><meta name="citation_reference" content="citation_title=Characterization of DNA polymerase from Pyrococcus sp. strain KOD1 and its application to PCR;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1997;  citation_volume=63;  citation_pages=4504-10; " /><meta name="citation_reference" content="citation_title=Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1012-9; " /><meta name="citation_reference" content="citation_title=Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=73-80; " /><meta name="citation_reference" content="citation_title=Locus-specific ChIP combined with NGS analysis reveals genomic regulatory regions that physically interact with the Pax5 promoter in a chicken B cell line;  citation_journal_title=DNA Res;  citation_year=2017;  citation_volume=24;  citation_pages=537-48; " /><meta name="citation_reference" content="citation_title=Mutations and altered expression of p16INK4 in human cancer;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=11045-9; " /><meta name="citation_reference" content="citation_title=Cancer translocations in human cells induced by zinc finger and TALE nucleases;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1182-93; " /><meta name="citation_reference" content="citation_title=CpG island hypermethylation and tumor suppressor genes: a booming present, a brighter future;  citation_journal_title=Oncogene;  citation_year=2002;  citation_volume=21;  citation_pages=5427-40; " /><meta name="citation_reference" content="citation_title=The role of genomic imprinting in biology and disease: an expanding view;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=517-30; " /><meta name="citation_reference" content="citation_title=X-chromosome inactivation in development and cancer;  citation_journal_title=FEBS Lett;  citation_year=2014;  citation_volume=588;  citation_pages=2514-22; " /><meta name="citation_reference" content="citation_title=OligoCalc: an online oligonucleotide properties calculator;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W43-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/395/25501727/dsy012.pdf" /><meta name="description" content="Abstract.  Genome editing by engineered sequence-specific nucleases, such as the clustered regularly interspaced short palindromic repeats (CRISPR) system is wi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/395/4989276" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/395/4989276" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/395/4989276");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy012';
    </script>
<title>
	Detection of genome-edited cells by oligoribonucleotide interference-PCR | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4989276" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcpKPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppdGEsIFRvc2hpdHN1Z3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSBhbmQgR2Vub21lIEJpb2xvZ3ksIEhpcm9zYWtpIFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljaW5lLCBIaXJvc2FraSwgQW9tb3JpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNocm9tYXRpbiBCaW9jaGVtaXN0cnkgUmVzZWFyY2ggR3JvdXAsIENvbWJpbmVkIFByb2dyYW0gb24gTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgT3Nha2EgVW5pdmVyc2l0eSwgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1bm8sIE1peXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNocm9tYXRpbiBCaW9jaGVtaXN0cnkgUmVzZWFyY2ggR3JvdXAsIENvbWJpbmVkIFByb2dyYW0gb24gTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgT3Nha2EgVW5pdmVyc2l0eSwgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpdGF1cmEsIEZ1c2FrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNocm9tYXRpbiBCaW9jaGVtaXN0cnkgUmVzZWFyY2ggR3JvdXAsIENvbWJpbmVkIFByb2dyYW0gb24gTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgT3Nha2EgVW5pdmVyc2l0eSwgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1amlpLCBIb2Rha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSBhbmQgR2Vub21lIEJpb2xvZ3ksIEhpcm9zYWtpIFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljaW5lLCBIaXJvc2FraSwgQW9tb3JpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNocm9tYXRpbiBCaW9jaGVtaXN0cnkgUmVzZWFyY2ggR3JvdXAsIENvbWJpbmVkIFByb2dyYW0gb24gTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIE1pY3JvYmlhbCBEaXNlYXNlcywgT3Nha2EgVW5pdmVyc2l0eSwgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGV0ZWN0aW9uIG9mIGdlbm9tZS1lZGl0ZWQgY2VsbHMgYnkgb2xpZ29yaWJvbnVjbGVvdGlkZSBpbnRlcmZlcmVuY2UtUENSIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVpGTiwgVEFMRU4sIGFuZCBDUklTUFIvQ2FzLWJhc2VkIG1ldGhvZHMgZm9yIGdlbm9tZSBlbmdpbmVlcmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zOTctNDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZWRpdGluZy4gVGhlIG5ldyBmcm9udGllciBvZiBnZW5vbWUgZW5naW5lZXJpbmcgd2l0aCBDUklTUFItQ2FzOTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDY7ICBjaXRhdGlvbl9wYWdlcz0xMjU4MDk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBDUklTUFIgdmlldyBvZiBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTg1OS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIG9uLXRhcmdldCBhbmQgb2ZmLXRhcmdldCBtdXRhdGlvbnMgZ2VuZXJhdGVkIGJ5IENSSVNQUi9DYXM5IGFuZCBvdGhlciBzZXF1ZW5jZS1zcGVjaWZpYyBudWNsZWFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEFkdjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTk1LTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZWZmaWNpZW50IGdlbm90eXBpbmcgbWV0aG9kIGZvciBnZW5vbWUtbW9kaWZpZWQgYW5pbWFscyBhbmQgaHVtYW4gY2VsbHMgZ2VuZXJhdGVkIHdpdGggQ1JJU1BSL0Nhczkgc3lzdGVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NjQyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIHNlbnNpdGl2ZSBkZXRlY3Rpb24gb2YgaW5kZWxzIGluZHVjZWQgYnkgcHJlY2lzZSBnZW5lIHRhcmdldGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1lNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZ2Vub3R5cGluZyBvZiBDUklTUFIvQ2FzOS1tZWRpYXRlZCBtdXRhbnRzIHVzaW5nIGZsdW9yZXNjZW50IFBDUi1jYXBpbGxhcnkgZ2VsIGVsZWN0cm9waG9yZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE1NTg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIG1ldGhvZHMgZm9yIGdlbmVyYXRpbmcgYW5kIGRldGVjdGluZyBsb2N1cy1zcGVjaWZpYyBtdXRhdGlvbnMgaW5kdWNlZCB3aXRoIFRBTEVOcyBpbiB0aGUgemVicmFmaXNoIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI4NjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIFBDUiBiYXNlZCBwcm90b2NvbCBmb3IgZGV0ZWN0aW5nIGluZGVsIG11dGF0aW9ucyBpbmR1Y2VkIGJ5IFRBTEVOcyBhbmQgQ1JJU1BSL0NhczkgaW4gemVicmFmaXNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTk4MjgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bVJOQSB0cmFuc2ZlY3Rpb24gb2YgYSBub3ZlbCBUQUwgZWZmZWN0b3IgbnVjbGVhc2UgKFRBTEVOKSBmYWNpbGl0YXRlcyBlZmZpY2llbnQga25vY2tvdXQgb2YgSElWIGNvLXJlY2VwdG9yIENDUjU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NTU2MC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIHNpbmdsZS1iYXNlIGdlbm9tZS1lZGl0ZWQgaHVtYW4gaVBTIGNlbGxzIHdpdGhvdXQgYW50aWJpb3RpYyBzZWxlY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRpZ2l0YWwgUENSLWJhc2VkIG1ldGhvZCBmb3IgZWZmaWNpZW50IGFuZCBoaWdobHkgc3BlY2lmaWMgc2NyZWVuaW5nIG9mIGdlbm9tZSBlZGl0ZWQgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTAxNTM5MDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBhbmQgZWZmaWNpZW50IG1ldGhvZCBmb3IgQ1JJU1BSL0NhczktaW5kdWNlZCBtdXRhbnQgc2NyZWVuaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBvZiBnZW5vbWUtZWRpdGVkIG11dGFudCBjbG9uZXMgYnkgYSBzaW1wbGUgY29tcGV0aXRpb24tYmFzZWQgUENSIG1ldGhvZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1lMDE3OTE2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9saWdvcmlib251Y2xlb3RpZGUgKE9STikgaW50ZXJmZXJlbmNlLVBDUiAoT1JOaS1QQ1IpOiBhIHNpbXBsZSBtZXRob2QgZm9yIHN1cHByZXNzaW5nIFBDUiBhbXBsaWZpY2F0aW9uIG9mIHNwZWNpZmljIEROQSBzZXF1ZW5jZXMgdXNpbmcgT1JOczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTMzNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtZ3VpZGVkIGh1bWFuIGdlbm9tZSBlbmdpbmVlcmluZyB2aWEgQ2FzOTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzk7ICBjaXRhdGlvbl9wYWdlcz04MjMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxsZWxlLXNwZWNpZmljIGxvY3VzIGJpbmRpbmcgYW5kIGdlbm9tZSBlZGl0aW5nIGJ5IENSSVNQUiBhdCB0aGUgcDE2SU5LNGEgbG9jdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMDQ4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvZ3JhbW1hYmxlIGR1YWwtUk5BLSBndWlkZWQgRE5BIGVuZG9udWNsZWFzZSBpbiBhZGFwdGl2ZSBiYWN0ZXJpYWwgaW1tdW5pdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM3OyAgY2l0YXRpb25fcGFnZXM9ODE2LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIEROQSBwb2x5bWVyYXNlIGZyb20gUHlyb2NvY2N1cyBzcC4gc3RyYWluIEtPRDEgYW5kIGl0cyBhcHBsaWNhdGlvbiB0byBQQ1I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTQ1MDQtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBlZmZpY2llbnQgUk5BLWd1aWRlZCBnZW5vbWUgZWRpdGluZyBpbiBodW1hbiBjZWxscyB2aWEgZGVsaXZlcnkgb2YgcHVyaWZpZWQgQ2FzOSByaWJvbnVjbGVvcHJvdGVpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDEyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhdGlvbmljIGxpcGlkLW1lZGlhdGVkIGRlbGl2ZXJ5IG9mIHByb3RlaW5zIGVuYWJsZXMgZWZmaWNpZW50IHByb3RlaW4tYmFzZWQgZ2Vub21lIGVkaXRpbmcgaW4gdml0cm8gYW5kIGluIHZpdm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb2N1cy1zcGVjaWZpYyBDaElQIGNvbWJpbmVkIHdpdGggTkdTIGFuYWx5c2lzIHJldmVhbHMgZ2Vub21pYyByZWd1bGF0b3J5IHJlZ2lvbnMgdGhhdCBwaHlzaWNhbGx5IGludGVyYWN0IHdpdGggdGhlIFBheDUgcHJvbW90ZXIgaW4gYSBjaGlja2VuIEIgY2VsbCBsaW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NTM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbnMgYW5kIGFsdGVyZWQgZXhwcmVzc2lvbiBvZiBwMTZJTks0IGluIGh1bWFuIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xMTA0NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYW5jZXIgdHJhbnNsb2NhdGlvbnMgaW4gaHVtYW4gY2VsbHMgaW5kdWNlZCBieSB6aW5jIGZpbmdlciBhbmQgVEFMRSBudWNsZWFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMTgyLTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcEcgaXNsYW5kIGh5cGVybWV0aHlsYXRpb24gYW5kIHR1bW9yIHN1cHByZXNzb3IgZ2VuZXM6IGEgYm9vbWluZyBwcmVzZW50LCBhIGJyaWdodGVyIGZ1dHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T25jb2dlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01NDI3LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBnZW5vbWljIGltcHJpbnRpbmcgaW4gYmlvbG9neSBhbmQgZGlzZWFzZTogYW4gZXhwYW5kaW5nIHZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTUxNy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WC1jaHJvbW9zb21lIGluYWN0aXZhdGlvbiBpbiBkZXZlbG9wbWVudCBhbmQgY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTg4OyAgY2l0YXRpb25fcGFnZXM9MjUxNC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2xpZ29DYWxjOiBhbiBvbmxpbmUgb2xpZ29udWNsZW90aWRlIHByb3BlcnRpZXMgY2FsY3VsYXRvcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNDMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzQvMzk1LzI1NTAxNzI3L2RzeTAxMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2Vub21lIGVkaXRpbmcgYnkgZW5naW5lZXJlZCBzZXF1ZW5jZS1zcGVjaWZpYyBudWNsZWFzZXMsIHN1Y2ggYXMgdGhlIGNsdXN0ZXJlZCByZWd1bGFybHkgaW50ZXJzcGFjZWQgc2hvcnQgcGFsaW5kcm9taWMgcmVwZWF0cyAoQ1JJU1BSKSBzeXN0ZW0gaXMgd2kiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ5ODkyNzYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkyc2fZ394KPeN239pOx/06J32Z0U=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f395%2f4989276" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f395%2f4989276" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120122312">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122312">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122314">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122314">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122320">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122320">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122331">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122331">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122361">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122361">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122370">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122370">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122376">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/383/4967678">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/409/5003450">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Detection of genome-edited cells by oligoribonucleotide interference-PCR
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshitsugu Fujita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan</div><div class="aff">Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshitsugu+Fujita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujita T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujita T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshitsugu Fujita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Miyuki Yuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Miyuki+Yuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yuno M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yuno M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Miyuki Yuno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fusako Kitaura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fusako+Kitaura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kitaura F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kitaura F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fusako Kitaura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hodaka Fujii</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biochemistry and Genome Biology, Hirosaki University Graduate School of Medicine, Hirosaki, Aomori, Japan</div><div class="aff">Chromatin Biochemistry Research Group, Combined Program on Microbiology and Immunology, Research Institute for Microbial Diseases, Osaka University, Suita, Osaka, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy012-cor1" data-open="dsy012-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 172 39 5018. Fax. +81 172 39 5020. Email: <a href="mailto:hodaka@hirosaki-u.ac.jp" target="_blank">hodaka@hirosaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hodaka+Fujii">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujii H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujii H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hodaka Fujii</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy012-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 395407, <a href='https://doi.org/10.1093/dnares/dsy012'>https://doi.org/10.1093/dnares/dsy012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 April 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 November 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 April 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4989276" href="/dnaresearch/article-pdf/25/4/395/25501727/dsy012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Toshitsugu Fujita, Miyuki Yuno, Fusako Kitaura, Hodaka Fujii;  Detection of genome-edited cells by oligoribonucleotide interference-PCR, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 395407, <a href='https://doi.org/10.1093/dnares/dsy012'>https://doi.org/10.1093/dnares/dsy012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4989276?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Detection%20of%20genome-edited%20cells%20by%20oligoribonucleotide%20interference-PCR&amp;publicationDate=2018-04-27&amp;volumeNum=25&amp;issueNum=4&amp;author=Fujita%2C%20Toshitsugu%3B%20Yuno%2C%20Miyuki&amp;startPage=395&amp;endPage=407&amp;contentId=10.1093%2Fdnares%2Fdsy012&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f395%2f4989276" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120122312 id="120122312" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genome editing by engineered sequence-specific nucleases, such as the clustered regularly interspaced short palindromic repeats (CRISPR) system is widely used for analysis of gene functions. Several techniques have been developed for detection of genome-edited cells, but simple, cost-effective, and positive detection methods remain limited. Recently, we developed oligoribonucleotide (ORN) interference-PCR (ORNi-PCR), in which hybridization of an ORN with a complementary DNA sequence inhibits amplification across the sequence. Here, we investigated whether ORNi-PCR can be used to detect genome-edited cells. First, we showed that ORNs that hybridize to a CRISPR target site in the <em>THYN1</em> locus inhibited amplification across the target site, but no longer inhibited amplification after the target site was edited, resulting in mismatches. Importantly, ORNi-PCR could distinguish even single-nucleotide differences. These features of ORNi-PCR enabled detection of genome-edited cells by positive PCR amplification. In addition, ORNi-PCR was successful in discriminating genome-edited cells from wild-type cells, and multiplex ORNi-PCR simultaneously detected indel mutations at multiple loci. However, endpoint ORNi-PCR may not be able to distinguish between mono- and bi-allelic mutations, which may limit its utility. Taken together, these results demonstrate the potential utility of ORNi-PCR for the screening of genome-edited cells.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="PCR">PCR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ORNi-PCR">ORNi-PCR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome editing&quot;">genome editing</a></div>                            <h2 scrollto-destination=120122314 id="120122314" class="section-title" >1. Introduction</h2>
<p>Genome editing is an essential biotechnology for medical and biological research. Zinc finger nucleases (ZFNs), transcription activator-like effector nucleases (TALENs), and the clustered regularly interspaced short palindromic repeats (CRISPRs) system have been widely used as genome-editing tools.<a reveal-id="dsy012-B1 dsy012-B2 dsy012-B3" data-open="dsy012-B1 dsy012-B2 dsy012-B3" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Because CRISPR is the most convenient tool, it is rapidly becoming the predominant technique for genome editing.<a reveal-id="dsy012-B1 dsy012-B2 dsy012-B3" data-open="dsy012-B1 dsy012-B2 dsy012-B3" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> CRISPR also makes it much easier to perform genome editing in various cell types.<a reveal-id="dsy012-B3" data-open="dsy012-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>For such approaches to be successful, screening methods capable of positively identifying genome-edited cells are indispensable. Although DNA sequencing can unambiguously detect genome-edited cells, it is time-consuming, costly, and unsuitable for initial screening of a large number of clones. Therefore, various methods have been developed for rapid and inexpensive detection of the desired cells.<a reveal-id="dsy012-B4" data-open="dsy012-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> For example, mismatch cleavage assays using T7 endonuclease 1 (T7E1) or Surveyor nuclease have been used to evaluate mutation frequency. In these techniques, following PCR amplification of a DNA sequence spanning a CRISPR target site, the amplicons are denatured, and re-hybridized to form heteroduplexes of wild-type (WT) and mutated strands. Mismatches between edited and unedited sites in the heteroduplexes are cleaved by T7E1 or Surveyor nuclease, and the resultant DNA fragments are electrophoresed to assess mutation frequency. The re-hybridized DNA strands can also be directly analysed by native polyacrylamide gel electrophoresis.<a reveal-id="dsy012-B5" data-open="dsy012-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> It is also possible to analyse mobility of the amplicons of WT and mutated strands by capillary electrophoresis.<a reveal-id="dsy012-B6" data-open="dsy012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy012-B7" data-open="dsy012-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Real-time PCR has also been used to evaluate the success of genome editing. For example, in high-resolution melting analysis, melting curves of PCR amplicons are analysed to distinguish nucleotide-level differences.<a reveal-id="dsy012-B8" data-open="dsy012-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Alternatively, by monitoring fluorescence emission of SYBR Green, real-time PCR with a primer designed against the target site can be used to assess the presence of indel mutations.<a reveal-id="dsy012-B9" data-open="dsy012-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In this approach, inhibition of PCR amplification indicates introduction of indel mutations into the primer-binding site (i.e. the target site). Real-time PCR with fluorescent probes complementary to the target site (e.g. TaqMan probes) can also be used to assess the presence of indel mutations.<a reveal-id="dsy012-B10" data-open="dsy012-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Digital PCR with complementary fluorescent probes can be used for more accurate analysis.<a reveal-id="dsy012-B10 dsy012-B11 dsy012-B12" data-open="dsy012-B10 dsy012-B11 dsy012-B12" class="link link-ref link-reveal xref-bibr"><sup>1012</sup></a> Although those methods are useful for detecting genome-edited cells, some require expensive equipment, such as real-time PCR machines or capillary gel electrophoresis systems, and they are also time-consuming. To address this issue, simpler methods using endpoint PCR were developed,<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy012-B14" data-open="dsy012-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> in which a primer designed against the target site is used in nearly the same way as in the real-time PCR-based method.<a reveal-id="dsy012-B9" data-open="dsy012-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Endpoint PCR-based approaches can also be more cost-effective for the evaluation of indel mutations.</p><p>We recently developed oligoribonucleotide (ORN) interference-PCR (ORNi-PCR) to inhibit PCR amplification in a sequence-specific manner (<a reveal-id="dsy012-F1" data-open="dsy012-F1" class="link link-reveal link-table xref-fig">Fig.1A</a>).<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> In ORNi-PCR, an ORN [not an oligodeoxyribonucleotide (ODN)] with a length of 1729 bases inhibits PCR amplification of a target DNA sequence that contains the DNA sequence complementary to the ORN (<a reveal-id="dsy012-F1" data-open="dsy012-F1" class="link link-reveal link-table xref-fig">Fig.1A</a>). We hypothesized that ORNi-PCR could be applied to detection of genome-edited cells (<a reveal-id="dsy012-F1" data-open="dsy012-F1" class="link link-reveal link-table xref-fig">Fig.1B</a>). In this approach, an ORN is designed to hybridize with the target site for genome editing. Next, PCR is performed in the presence of the ORN on genomic DNAs (gDNAs) extracted from the genome-edited cells. If a mutation has been successfully introduced into the target site, the ORN fails to hybridize, resulting in amplification of the target DNA. In other words, positive PCR amplification indicates that the target site is mutated, whereas no PCR amplification indicates that the target site is intact (<a reveal-id="dsy012-F1" data-open="dsy012-F1" class="link link-reveal link-table xref-fig">Fig.1B</a>). Because chemical synthesis of ORNs is inexpensive, it is possible to screen economically a large number of potential genome-edited cells by ORNi-PCR. In addition, endpoint ORNi-PCR does not require expensive equipment such as real-time PCR machines. </p>                        <a id="120122318" scrollto-destination="120122318"></a>
<div id="dsy012-F1" data-id="dsy012-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f1.png?Expires=2147483647&Signature=4JXloQNpXryb4wO2PArZ7hwxWAWbG4INSm-TosXuC93ahIH8jcVvWReebZMtmIjiO4IGkx4alNiTlGJob21tZgRmNoikObwY14mp2xKUJCy-BO0TuOTCGgBhig99DsrW9~ztUHASiBjdmSNVR-bJcdTlx8IGjJz4N6eVu0jWAln96SPdEbMsaJT8~1Rf6pyeBbGqEA8xS0eZPJhbsWYcq6i7uq2AXt5COSiaAszIsXE8Fyh42Qp2Xi65HaFCq4Vhc86DlgL0idDRa1TPpjdiS5dI5X7xyDJR3knW3ExIUG~b2ElUUoaavsDvSlIAMz2SSY739aMJxDkMv~yq7dsQ3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of genome-edited cells by ORNi-PCR. (A) Schematic of ORNi-PCR. An ORN [not ODN] inhibits PCR amplification of a target DNA sequence that contains the DNA sequence complementary to the ORN. (B) Application of ORNi-PCR to detection of genome-edited cells." path-from-xml="dsy012f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122318/dsy012f1.tif" path-from-xml="dsy012f1.tif" target="_blank">View large</a><a section="120122318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f1.png?Expires=2147483647&Signature=Z5npdeIvWoFbSMC74VJPK6hLQ20BEmQReq9Z6c0bQUaJe20Q1ZkFIE929ZaxR1XmZB60q5Lb18kBWNdjFfzDdas~3epyCPeMHIfbvPDuOfc7ShyduugpTh-rR8ekmZdr2aw9LSSg~USQMNP7KtwjA~7qqXuuVQElgluWNvhFDb4lKYj5ZuO66LQ0r8GUwn4Lp-nQ~vomJb-b~MclDJ7VOGI-XaUNcVSDTncf0NyH3b-PpPqzt9HVQfpSU-jdaRxv-A1uzezpwYlrruShmKuc1kGBFknjWBOCoWi9C27Z7oM7p2WqIPxTULP8Kij3MEYI-iBnKsh6O0bLnuKcVDKw6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122318&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of genome-edited cells by ORNi-PCR. (A) Schematic of ORNi-PCR. An ORN [not ODN] inhibits PCR amplification of a target DNA sequence that contains the DNA sequence complementary to the ORN. (B) Application of ORNi-PCR to detection of genome-edited cells.</p></div></div><div content-id="dsy012-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f1.png?Expires=2147483647&Signature=4JXloQNpXryb4wO2PArZ7hwxWAWbG4INSm-TosXuC93ahIH8jcVvWReebZMtmIjiO4IGkx4alNiTlGJob21tZgRmNoikObwY14mp2xKUJCy-BO0TuOTCGgBhig99DsrW9~ztUHASiBjdmSNVR-bJcdTlx8IGjJz4N6eVu0jWAln96SPdEbMsaJT8~1Rf6pyeBbGqEA8xS0eZPJhbsWYcq6i7uq2AXt5COSiaAszIsXE8Fyh42Qp2Xi65HaFCq4Vhc86DlgL0idDRa1TPpjdiS5dI5X7xyDJR3knW3ExIUG~b2ElUUoaavsDvSlIAMz2SSY739aMJxDkMv~yq7dsQ3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of genome-edited cells by ORNi-PCR. (A) Schematic of ORNi-PCR. An ORN [not ODN] inhibits PCR amplification of a target DNA sequence that contains the DNA sequence complementary to the ORN. (B) Application of ORNi-PCR to detection of genome-edited cells." path-from-xml="dsy012f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122318/dsy012f1.tif" path-from-xml="dsy012f1.tif" target="_blank">View large</a><a section="120122318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f1.png?Expires=2147483647&Signature=Z5npdeIvWoFbSMC74VJPK6hLQ20BEmQReq9Z6c0bQUaJe20Q1ZkFIE929ZaxR1XmZB60q5Lb18kBWNdjFfzDdas~3epyCPeMHIfbvPDuOfc7ShyduugpTh-rR8ekmZdr2aw9LSSg~USQMNP7KtwjA~7qqXuuVQElgluWNvhFDb4lKYj5ZuO66LQ0r8GUwn4Lp-nQ~vomJb-b~MclDJ7VOGI-XaUNcVSDTncf0NyH3b-PpPqzt9HVQfpSU-jdaRxv-A1uzezpwYlrruShmKuc1kGBFknjWBOCoWi9C27Z7oM7p2WqIPxTULP8Kij3MEYI-iBnKsh6O0bLnuKcVDKw6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122318&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of genome-edited cells by ORNi-PCR. (A) Schematic of ORNi-PCR. An ORN [not ODN] inhibits PCR amplification of a target DNA sequence that contains the DNA sequence complementary to the ORN. (B) Application of ORNi-PCR to detection of genome-edited cells.</p></div></div><p>In this study, we investigated whether ORNi-PCR can be used for detection of genome-edited cells. We demonstrated that ORNi-PCR can distinguish indel mutations with even a single-nucleotide difference relative to the intact DNA sequences of target sites. However, we also found that it may be difficult to distinguish between mono- and bi-allelic mutations by endpoint ORNi-PCR. Thus, the primary use of ORNi-PCR could be the detection of genome-edited cells possessing at least mono-allelic mutations at a target site before subsequent determination of mono- or bi-allelic mutations by other methods.</p>                            <h2 scrollto-destination=120122320 id="120122320" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120122321 id="120122321" class="section-title" >2.1 Oligonucleotides</h3>
<p>The primers used in this study are listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=xbV7fxPtlQnbXi~~MuqHf2UVvhcYwDVrUqVbNn82jvh8b8cfVlKJnMvsaIeYMBt2pQMJYzWi-AMnOZDrnvKnhWVFSsopv0RRc7lfK~iaQaX4U17BbJxA338xw88mkbBBOI6akuH5nohXhzPlm08~eKNQ1XN26JLjdumJfT-xXiaAbfgWC5FjIAHLs4e0PG6RgLm9x1DN7V~X2BqFFxYTDckeQ09Tf071MSDmjwxeNkMT6iWy76V9aT4O2X8KqjV6EVM-UpcncSBwqVJRzApAfDCdQv0HzHitSJiqY9hOi5QjY8nIh6GJuJbrBkBMClWapuUqUVMtqG13aaD-Ud9vCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. ORNs were chemically synthesized (Greiner) and are listed in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_table_s2.xlsx?Expires=2147483647&amp;Signature=lXoJ-eMt-9KILoaohq1ZmMCHkYm-~LDxKyvRvb-I-wHnWKTvct9qADQJhrAXNYNMrP2FpxDWhITSiGrsC~-PJ8aVoazHeOsEWlHTCtXR9fhkzowNvgzFUC7jCxbjje~7ZXxw1~4ELojHoPreONUO3maAStqqqO~NMzc1EnyBN1us9VPR5FP9jmyvdcdubz5HkadvoePJvYSV5lI3EUxUeiS4nHtju1JV5c1RfDfbjSzvO3Oces46GN2934fBK8gJ3v7h~q3AqcQVAOhHD2oYdVXDjKOyPOXP8mtQcHjmwVkzFh569Rh5rAVkG-HcihEfyZ6hF1wlI7V0TCZw61CHTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=120122323 id="120122323" class="section-title" >2.2 Cell culture and extraction of gDNAs</h3>
<p>Raji cells were cultured in RPMI-1640 (Wako) supplemented with 10% foetal bovine serum (FBS). 293T cells were cultured in Dulbecco's modified Eagle medium (DMEM) (Wako) supplemented with 10% FBS. HCT116 cells were cultured in McCoys 5A (Thermo Fisher Scientific) supplemented with 10% FBS. gDNAs were extracted from cells by standard phenol/chloroform extraction.</p>                            <h3 scrollto-destination=120122325 id="120122325" class="section-title" >2.3 ORNi-PCR</h3>
<p>ORNi-PCR reactions were performed using KOD-Plus-Ver.2 (Toyobo). For ORNi-PCR targeting the human <em>THYN1</em> locus, ORNi-PCR reaction mixtures containing 20 ng of Raji gDNA, 0.3 M of each primer, and 0.12 M of an ORN were prepared in a 10 l volume according to the manufacturers protocol. The reactions were carried out with an initial denaturation at 94C for 2 min, followed by 35 cycles of 98C for 10 s, 62C for 30 s, and 68 C for 1 min. For ORNi-PCR targeting the human <em>CDKN2A(p16)</em> locus, ORNi-PCR reaction mixtures containing 20 ng of 293T or HCT116 gDNA, 0.3 M of each primer, and 1 M of an ORN were prepared in a 10 l volume. The reactions were carried out with an initial denaturation at 94C for 2 min, followed by 30 cycles of 98C for 10 s, 62C for 30 s, and 68C for 1 min (for <a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6</a>), or an initial denaturation at 94C for 2 min, followed by 30 cycles of 98C for 10 s, and 6272C for 20 s (for <a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Figs8 and 9</a>). For ORNi-PCR targeting both the human <em>THYN1</em> and <em>CDKN2A(p16)</em> loci, ORNi-PCR reaction mixtures containing 20 ng of 293T gDNA, 0.3 M of each primer, and 1 M of each ORN were prepared in a 10 l volume. The reactions were carried out with an initial denaturation at 94C for 2 min, followed by 30 cycles of 98C for 10 s, 62C for 30 s, and 68C for 1 min. ORNi-PCR products were electrophoresed on 1% or 2% agarose gels and, if necessary, subjected to DNA sequencing. DNA-sequencing data were analysed using the Applied Biosystems Sequence Scanner Software v2.0 (Thermo Fisher Scientific).</p>                            <h3 scrollto-destination=120122327 id="120122327" class="section-title" >2.4 Plasmids</h3>
<p>The Cas9 expression plasmid (Addgene no. 41815)<a reveal-id="dsy012-B16" data-open="dsy012-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and chimeric single guide RNA (sgRNA) expression plasmid (Addgene no. 41824)<a reveal-id="dsy012-B16" data-open="dsy012-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> were provided by Dr George Church through Addgene. To construct a sgRNA expression plasmid targeting the human <em>THYN1</em> locus, a CRISPR target sequence was cloned downstream of the U6 promoter in the sgRNA expression plasmid according to the hCRISPR gRNA synthesis protocol (<a class="link link-uri" href="https://media.addgene.org/data/93/40/adf4a4fe-5e77-11e2-9c30-003048dd6500.pdf" target="_blank">https://media.addgene.org/data/93/40/adf4a4fe-5e77-11e2-9c30-003048dd6500.pdf</a> (19 April 2018, date last accessed)). To construct a Cas9 plus sgRNA expression plasmid targeting the human <em>CDKN2A(p16)</em> locus, the sgRNA expression cassette for <em>CDKN2A(p16)</em> (Gx4 no. 2)<a reveal-id="dsy012-B17" data-open="dsy012-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> was cloned upstream of the Cas9 expression cassette in the Cas9 expression plasmid.</p>                            <h3 scrollto-destination=120122329 id="120122329" class="section-title" >2.5 CRISPR-mediated genome editing</h3>
<p>For genome editing of the human <em>THYN1</em> locus, Raji cells (1  10<sup>7</sup>) were transfected with Cas9 expression plasmid (120 g), sgRNA expression plasmid targeting the human <em>THYN1</em> locus (120 g), and pEGFP-N3 (0.3 g, Clontech) by electroporation on a Gene Pulser II (Bio-Rad) at 250 V and 950 F. One day later, GFP-positive cells were individually sorted and expanded. For genome editing of the human <em>CDKN2A(p16)</em> locus, 293 T cells (4  10<sup>5</sup>) were transfected with the Cas9 plus sgRNA expression plasmid targeting the human <em>CDKN2A(p16)</em> locus (4 g) and pcDNA3.1/Hygro () (0.4 g, Thermo Fisher Scientific) using Lipofectamine 3000 (Thermo Fisher Scientific). Two days later, hygromycin was added (0.4 mg/ml), and hygromycin-resistant colonies were picked and cultured. For genome editing of the human <em>THYN1</em> and <em>CDKN2A(p16)</em> loci, 293 T cells (4  10<sup>5</sup>) were transfected with the Cas9 plus sgRNA expression plasmid targeting the human <em>CDKN2A(p16)</em> locus (4 g), sgRNA expression plasmid targeting the human <em>THYN1</em> locus (4 g), and pcDNA3.1/Hygro () (0.4 g) using Lipofectamine 3000. Two days later, hygromycin was added (0.4 mg/ml), and hygromycin-resistant colonies were picked and cultured.</p>                            <h2 scrollto-destination=120122331 id="120122331" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120122332 id="120122332" class="section-title" >3.1 Determination of effective concentrations of ORNs for ORNi-PCR</h3>
<p>First, we asked whether ORNi-PCR could suppress amplification across a CRISPR target site in the human <em>THYN1</em> locus. We designed three ORNs: ORN_20b, ORN_24b, and ORN_Target. ORN_20b (20 bases) and ORN_24b (24 bases) hybridize with the target site such that the CRISPR cleavage position, which is 3 bp upstream of the protospacer adjacent motif (PAM),<a reveal-id="dsy012-B18" data-open="dsy012-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> is in the centre of their sequences. ORN_Target (23 bases) matches the 20 bp sequence of the sgRNA as well as the PAM sequence used for genome editing (<a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2A</a>). Because DNA polymerases that retain 35 exonuclease activity, but not those that retain 53 exonuclease activity, can be utilized for ORNi-PCR,<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> we used the KOD DNA polymerase<a reveal-id="dsy012-B19" data-open="dsy012-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> to amplify a 0.9 kb region surrounding the target site (<a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2A</a>). As shown in <a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2B</a>, the 0.9 kb region was specifically amplified when gDNA extracted from human Raji cells was used for PCR in the absence of ORNs. Addition of ORN_20b (1 or 2 M), ORN_24b (0.12 M), or ORN_Target (0.52 M) to the reactions strongly suppressed PCR amplification. Lower concentrations of ORN_20b (0.1 or 0.5 M) and ORN_Target (0.5 M) were less effective in this respect. In contrast, ORN_306F(NC), a 25-base ORN hybridizing with an irrelevant locus (human <em>IRF-1</em> locus),<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> did not affect amplification (<a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2B</a>). Thus, these <em>THYN1</em>-specific ORNs can be used to specifically suppress PCR amplification of the human <em>THYN1</em> locus. </p>                        <a id="120122334" scrollto-destination="120122334"></a>
<div id="dsy012-F2" data-id="dsy012-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f2.png?Expires=2147483647&Signature=ukXk3Nt96UzcDiR7iZjw7kLtOplyqCk8CiohqevgvRSsU4P697z2G69EzUALXALl0lp~yUGn-1I4r-Mq4oV3jyhX6L5qRaBo~euoUwv~Tb1Dbouvvl74e-HOhHDSwD3Jb65D4tjND25uU6P0K4ymZrWn-yUxSunxXRZII88ssR7jNL03jA1OYtJ43mfLRv6t1PHUNhK~gd3KfXJmpJnmUFG2esOPg7U8HhHxYwLwhNSKp2Qn8IX7bvcKghniwhwqK2VKBbGnwpQNKivnxHKSJxpwuk5IIxSyLsOCsbPCdJ2LsZejLNcAIZJA9sFEdZzSYgzx9yZOEEhIuoXocW4b-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Suppression of PCR amplification of a target DNA sequence by ORNs. (A) Target positions and sequences of ORNs. The forward DNA sequence of the allele is shown. CRISPR target sites are underlined, and PAM positions are shown in red. Black arrows indicate the CRISPR cleavage sites. (B) Dose responses of ORNs. PCR amplification was performed in the presence or absence of various concentrations of ORNs. M, molecular weight marker." path-from-xml="dsy012f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122334/dsy012f2.tif" path-from-xml="dsy012f2.tif" target="_blank">View large</a><a section="120122334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f2.png?Expires=2147483647&Signature=nXL~5iXuIb5V9MmHggKgHhdTxVItFYKHcOWv0gxYVsSAy3PYLeVlwcAdEQKUDQM0XD1ph7m27~JWwSOeS4x~CdN0AV870QPFy0OrkyWB~HY9RdJTy2ZquFLmMp9x8I8eKy3-~ooqYI9nvq969nLc3W8a9OjlHRMNx~uwO9xVwn~ax~d83w5gAt~DGueB1ry2imegX-uzmCnzbrPrJrCOTPwwazwcatcSpn~ZuEB~~Dp52cG5vAxnLbw8JpPwtlI108YjfGWYoKrUrJ~09tN6p54~ObUu0sTZwN54Ps667OsBRI5wp~0fAbhDIf-0nJ3EBe3akoEq5GrbZoMut4HVdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122334&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Suppression of PCR amplification of a target DNA sequence by ORNs. (A) Target positions and sequences of ORNs. The forward DNA sequence of the allele is shown. CRISPR target sites are underlined, and PAM positions are shown in red. Black arrows indicate the CRISPR cleavage sites. (B) Dose responses of ORNs. PCR amplification was performed in the presence or absence of various concentrations of ORNs. M, molecular weight marker.</p></div></div><div content-id="dsy012-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f2.png?Expires=2147483647&Signature=ukXk3Nt96UzcDiR7iZjw7kLtOplyqCk8CiohqevgvRSsU4P697z2G69EzUALXALl0lp~yUGn-1I4r-Mq4oV3jyhX6L5qRaBo~euoUwv~Tb1Dbouvvl74e-HOhHDSwD3Jb65D4tjND25uU6P0K4ymZrWn-yUxSunxXRZII88ssR7jNL03jA1OYtJ43mfLRv6t1PHUNhK~gd3KfXJmpJnmUFG2esOPg7U8HhHxYwLwhNSKp2Qn8IX7bvcKghniwhwqK2VKBbGnwpQNKivnxHKSJxpwuk5IIxSyLsOCsbPCdJ2LsZejLNcAIZJA9sFEdZzSYgzx9yZOEEhIuoXocW4b-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Suppression of PCR amplification of a target DNA sequence by ORNs. (A) Target positions and sequences of ORNs. The forward DNA sequence of the allele is shown. CRISPR target sites are underlined, and PAM positions are shown in red. Black arrows indicate the CRISPR cleavage sites. (B) Dose responses of ORNs. PCR amplification was performed in the presence or absence of various concentrations of ORNs. M, molecular weight marker." path-from-xml="dsy012f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122334/dsy012f2.tif" path-from-xml="dsy012f2.tif" target="_blank">View large</a><a section="120122334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f2.png?Expires=2147483647&Signature=nXL~5iXuIb5V9MmHggKgHhdTxVItFYKHcOWv0gxYVsSAy3PYLeVlwcAdEQKUDQM0XD1ph7m27~JWwSOeS4x~CdN0AV870QPFy0OrkyWB~HY9RdJTy2ZquFLmMp9x8I8eKy3-~ooqYI9nvq969nLc3W8a9OjlHRMNx~uwO9xVwn~ax~d83w5gAt~DGueB1ry2imegX-uzmCnzbrPrJrCOTPwwazwcatcSpn~ZuEB~~Dp52cG5vAxnLbw8JpPwtlI108YjfGWYoKrUrJ~09tN6p54~ObUu0sTZwN54Ps667OsBRI5wp~0fAbhDIf-0nJ3EBe3akoEq5GrbZoMut4HVdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122334&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Suppression of PCR amplification of a target DNA sequence by ORNs. (A) Target positions and sequences of ORNs. The forward DNA sequence of the allele is shown. CRISPR target sites are underlined, and PAM positions are shown in red. Black arrows indicate the CRISPR cleavage sites. (B) Dose responses of ORNs. PCR amplification was performed in the presence or absence of various concentrations of ORNs. M, molecular weight marker.</p></div></div>                            <h3 scrollto-destination=120122335 id="120122335" class="section-title" >3.2 Detection of genome-edited cells using ORNi-PCR</h3>
<p>To demonstrate that this approach can detect genome-edited cells (<a reveal-id="dsy012-F1" data-open="dsy012-F1" class="link link-reveal link-table xref-fig">Fig.1B</a>), we next investigated whether ORNs could suppress PCR amplification across an intact CRISPR target site, but not an edited variant of the same site. To this end, we performed CRISPR-mediated genome editing of the <em>THYN1</em> locus in the Raji line and established genome-edited clones in which the target site was mutated in both alleles (<a reveal-id="dsy012-F3" data-open="dsy012-F3" class="link link-reveal link-table xref-fig">Fig.3A</a>). The genome-edited cells contained indel mutations in the range of 3501 bp (<a reveal-id="dsy012-F3" data-open="dsy012-F3" class="link link-reveal link-table xref-fig">Fig.3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S2</a></span>). Clone T6 possesses the shortest indel mutation (3 bp deletion) in one allele, whereas T9 has the longest indel (501 bp insertion) in both alleles. T4 and T9 harboured homozygous indel mutations. PCR with the <em>THYN1</em>-specific primer set (<a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2A</a>) amplified the target region from gDNAs of all cells in the absence of ORNs (<a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Fig.4A</a>). In contrast, ORN_20b completely suppressed the PCR amplification from gDNA of WT Raji but not the genome-edited cells (<a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Fig.4A</a>). ORN_306F(NC) had no effect on amplification. </p>                        <a id="120122337" scrollto-destination="120122337"></a>
<div id="dsy012-F3" data-id="dsy012-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f3.png?Expires=2147483647&Signature=IXkUc1LeQiEAiaBODEpf-9BKmAJNpxxP4RLOkWyXuCus6TQwTCcNfZB1B0RqaoQirt4rC-uxYMRzRuY3s526aiR5g8p28U2d~tFjHtmSh5FOe5W6tdWcObvnOnC-sk5ryCopvYgr2JCH~HEF33nN~d-nhpylyn4WN4sTC3rsgwWuk7Rl4GBIYp9tXeVCajpi3hDUw-ytJE-NWlRue50~8SvFtxcKlpj7dqyjxGfFznHrWJ69obVFAoIS9zHQTES3S-f9wW3BXYb7JLs5WVkj3Mvq06zcXJ5BKPzmf4s5yoW7Q-I7-e7dbILdluTjwDuILBkMbW5MgUZku7tqJIEA7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Indel mutations introduced by CRISPR-mediated genome editing. (A) Schematic of CRISPR-mediated genome editing in Raji cells. (B) DNA sequences around the CRISPR target site in the THYN1 locus in WT and genome-edited cells. The forward DNA sequences of both alleles are shown. The CRISPR target sites in WT are underlined, and PAM positions are shown in red. An arrow indicates the CRISPR cleavage sites." path-from-xml="dsy012f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122337/dsy012f3.tif" path-from-xml="dsy012f3.tif" target="_blank">View large</a><a section="120122337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f3.png?Expires=2147483647&Signature=erBo3iGuKLbdyRxdjOoSALB897H8e4nqcZdCD9oWNl7kN3wXY3qUjka7T7NDh8A5N2xbGjsx2zqU8aZbrdcugRWIEGGE~Lg3w0ef6ap0WtMS~wdzS~Sst02h4ZniB54wpxdisGnsQisg8ggAbSj2gRHEivPFTZsaGyMlkOEZv0yRnuvw4cLUbGoqFC5EU-EklEyzgxAu5CLECCD0lWjGhUePxhfjHm9GIST0~AoKxRF7FlvoMiwEJ3Tsfw1TaM835044j92YZsXYdiGtbU53y13kQX48QLzsMnJbA-QOsKkxtwdAw8GbWVTx5eRIxCcXSehm3BVPix7TWeOGG5YSyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122337&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Indel mutations introduced by CRISPR-mediated genome editing. (A) Schematic of CRISPR-mediated genome editing in Raji cells. (B) DNA sequences around the CRISPR target site in the <em>THYN1</em> locus in WT and genome-edited cells. The forward DNA sequences of both alleles are shown. The CRISPR target sites in WT are underlined, and PAM positions are shown in red. An arrow indicates the CRISPR cleavage sites.</p></div></div><div content-id="dsy012-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f3.png?Expires=2147483647&Signature=IXkUc1LeQiEAiaBODEpf-9BKmAJNpxxP4RLOkWyXuCus6TQwTCcNfZB1B0RqaoQirt4rC-uxYMRzRuY3s526aiR5g8p28U2d~tFjHtmSh5FOe5W6tdWcObvnOnC-sk5ryCopvYgr2JCH~HEF33nN~d-nhpylyn4WN4sTC3rsgwWuk7Rl4GBIYp9tXeVCajpi3hDUw-ytJE-NWlRue50~8SvFtxcKlpj7dqyjxGfFznHrWJ69obVFAoIS9zHQTES3S-f9wW3BXYb7JLs5WVkj3Mvq06zcXJ5BKPzmf4s5yoW7Q-I7-e7dbILdluTjwDuILBkMbW5MgUZku7tqJIEA7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Indel mutations introduced by CRISPR-mediated genome editing. (A) Schematic of CRISPR-mediated genome editing in Raji cells. (B) DNA sequences around the CRISPR target site in the THYN1 locus in WT and genome-edited cells. The forward DNA sequences of both alleles are shown. The CRISPR target sites in WT are underlined, and PAM positions are shown in red. An arrow indicates the CRISPR cleavage sites." path-from-xml="dsy012f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122337/dsy012f3.tif" path-from-xml="dsy012f3.tif" target="_blank">View large</a><a section="120122337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f3.png?Expires=2147483647&Signature=erBo3iGuKLbdyRxdjOoSALB897H8e4nqcZdCD9oWNl7kN3wXY3qUjka7T7NDh8A5N2xbGjsx2zqU8aZbrdcugRWIEGGE~Lg3w0ef6ap0WtMS~wdzS~Sst02h4ZniB54wpxdisGnsQisg8ggAbSj2gRHEivPFTZsaGyMlkOEZv0yRnuvw4cLUbGoqFC5EU-EklEyzgxAu5CLECCD0lWjGhUePxhfjHm9GIST0~AoKxRF7FlvoMiwEJ3Tsfw1TaM835044j92YZsXYdiGtbU53y13kQX48QLzsMnJbA-QOsKkxtwdAw8GbWVTx5eRIxCcXSehm3BVPix7TWeOGG5YSyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122337&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Indel mutations introduced by CRISPR-mediated genome editing. (A) Schematic of CRISPR-mediated genome editing in Raji cells. (B) DNA sequences around the CRISPR target site in the <em>THYN1</em> locus in WT and genome-edited cells. The forward DNA sequences of both alleles are shown. The CRISPR target sites in WT are underlined, and PAM positions are shown in red. An arrow indicates the CRISPR cleavage sites.</p></div></div>                        <a id="120122338" scrollto-destination="120122338"></a>
<div id="dsy012-F4" data-id="dsy012-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f4.png?Expires=2147483647&Signature=wK-VAgpEYpaebg-diS6QznZUoMiOZRj1PqR1qbOnUldLDNg6tYhHXfMXCi1yMYceq2FR~RjIc3YUK8wg619Jgu-t58Ui8e1rTmGHbcOVAlHSVirfsD1WAEeReNzUVClZNbmStvwKMqnK7p0cdw8MhrzxduvLN~ftvA5D3yhH5zPppkeEFjUP10qAmRve2xuEySXRslZAkzSf881A-ir0f4zkqjvr7biAgArF0GmLRHQtOY3OpaD7U4mBMC2p~PfzOyz0rW-2NXxcOfj8Qf4C7SWzk9ofNKGgD-oTV2T6WGfrvEP0thFqAfRRJJAAVWfxfyh4xF2wQr7IjjgSmiSXYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of bi-allelic genome editing by ORNi-PCR. (A and C) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of T1, T6, and T7 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing." path-from-xml="dsy012f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122338/dsy012f4.tif" path-from-xml="dsy012f4.tif" target="_blank">View large</a><a section="120122338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f4.png?Expires=2147483647&Signature=d3pzyhWz1oJivCG6w00QFlxHz-tJKqqZa0IX29kEloOs9zG6PF525cXluRsH2nSDyzGHGM2bh0mORtSEMflWuPwr~lT46vJ37nOLoC6--MzxbeVJbtJV840SKCb~jMY763tHHzjOWueI86LQC2fT1F8CoI-NtYmZY9aqvGNeyS3E5xUu51TnEPChikjBKIrOTv2qoqP6V4KBuwQyAyUnNfoBbeK5w4GoH2J64~YD~UYUSQYPfC-oc6mAMTDxZIrXaS5TKmpGXwzQiQYhAoApQa3oGPhSaysNv-o7yI-QSrVwMzLStXbKF3PvTP3Sw0UEhlWCt8CnYBnuuvrNUgYL5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122338&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of bi-allelic genome editing by ORNi-PCR. (A and C) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of T1, T6, and T7 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing.</p></div></div><div content-id="dsy012-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f4.png?Expires=2147483647&Signature=wK-VAgpEYpaebg-diS6QznZUoMiOZRj1PqR1qbOnUldLDNg6tYhHXfMXCi1yMYceq2FR~RjIc3YUK8wg619Jgu-t58Ui8e1rTmGHbcOVAlHSVirfsD1WAEeReNzUVClZNbmStvwKMqnK7p0cdw8MhrzxduvLN~ftvA5D3yhH5zPppkeEFjUP10qAmRve2xuEySXRslZAkzSf881A-ir0f4zkqjvr7biAgArF0GmLRHQtOY3OpaD7U4mBMC2p~PfzOyz0rW-2NXxcOfj8Qf4C7SWzk9ofNKGgD-oTV2T6WGfrvEP0thFqAfRRJJAAVWfxfyh4xF2wQr7IjjgSmiSXYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of bi-allelic genome editing by ORNi-PCR. (A and C) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of T1, T6, and T7 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing." path-from-xml="dsy012f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122338/dsy012f4.tif" path-from-xml="dsy012f4.tif" target="_blank">View large</a><a section="120122338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f4.png?Expires=2147483647&Signature=d3pzyhWz1oJivCG6w00QFlxHz-tJKqqZa0IX29kEloOs9zG6PF525cXluRsH2nSDyzGHGM2bh0mORtSEMflWuPwr~lT46vJ37nOLoC6--MzxbeVJbtJV840SKCb~jMY763tHHzjOWueI86LQC2fT1F8CoI-NtYmZY9aqvGNeyS3E5xUu51TnEPChikjBKIrOTv2qoqP6V4KBuwQyAyUnNfoBbeK5w4GoH2J64~YD~UYUSQYPfC-oc6mAMTDxZIrXaS5TKmpGXwzQiQYhAoApQa3oGPhSaysNv-o7yI-QSrVwMzLStXbKF3PvTP3Sw0UEhlWCt8CnYBnuuvrNUgYL5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122338&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of bi-allelic genome editing by ORNi-PCR. (A and C) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of T1, T6, and T7 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing.</p></div></div><p>Because ORN_20b might allele-specifically suppress PCR amplification from gDNAs of T1, T6, and T7 cells, in which the CRISPR target site is differentially mutated in each allele (<a reveal-id="dsy012-F3" data-open="dsy012-F3" class="link link-reveal link-table xref-fig">Fig.3B</a>), we sequenced the ORNi-PCR product. As shown in <a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Fig.4B</a>, the sequencing signals of the ORNi-PCR products were comparable to those of PCR products generated in the absence of an ORN [() ORN], demonstrating that ORN_20b did not affect amplification from those gDNAs. Next, we tested other ORNs. ORNi-PCR with ORN_24b or ORN_Target, but not ORN_302F(NC), another irrelevant 21-base ORN,<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> yielded PCR patterns identical to those of ORNi-PCR with ORN_20b (<a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Fig.4C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Thus, these results showed that ORNi-PCR can be used to distinguish bi-allelic indel mutations from an intact target sequence. Amplification of T6 gDNA was not affected by the <em>THYN1</em>-specific ORNs, indicating that ORNi-PCR can distinguish indel mutations of more than 3 bp in size.</p><p>Next, we sought to determine whether ORNi-PCR can also distinguish mono-allelic mutations from an intact target sequence. To this end, we mixed gDNAs extracted from WT and T4 or T9 cells at 1:1 ratio to mimic mono-allelic mutations (<a reveal-id="dsy012-F5" data-open="dsy012-F5" class="link link-reveal link-table xref-fig">Fig.5</a>). When the mixture of WT and T9 gDNAs was used in ORNi-PCR with ORN_20b, ORN_24b, or ORN_Target, PCR amplification was suppressed only when the samples contained WT gDNA, but not mutated gDNA (<a reveal-id="dsy012-F5" data-open="dsy012-F5" class="link link-reveal link-table xref-fig">Fig.5A</a>). When the same experiment was performed with the mixture of WT and T4 gDNAs, a single band (0.9 kb) was detected (<a reveal-id="dsy012-F5" data-open="dsy012-F5" class="link link-reveal link-table xref-fig">Fig.5A</a>); DNA sequencing of the ORNi-PCR product using ORN_20b detected the signal derived from T4 but not WT (<a reveal-id="dsy012-F5" data-open="dsy012-F5" class="link link-reveal link-table xref-fig">Fig.5B</a>), clearly showing that ORNi-PCR amplified the edited genomic region but not the intact site. We also obtained the same results using ORNi-PCR with other <em>THYN1</em>-specific ORNs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Thus, these results demonstrated that ORNi-PCR can be used to distinguish mono-allelic indel mutations from an intact target sequence. </p>                        <a id="120122341" scrollto-destination="120122341"></a>
<div id="dsy012-F5" data-id="dsy012-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f5.png?Expires=2147483647&Signature=mi56tv8ZaoA0T~dtpfCFBvy9~XhrzYeXpi7M2L0SM8e2eZKcXQdMls0hl52fyDStqfZdDML3PuHAbezS2oHiCKiEhMa14JC5yHarDzC0kMV4hOsu3dnAv3SeCf5UPwkuD30FW0JCbE-fXvUYBhJG9vzcvGvYUtKS9pqPZnINkSthNOMonUL6IcUbuikNqOMuaiz87tB8g5XW9mkH2vp69tMZOeXFt8bxevudKYe0G9ZmeRiNn9~yudZgtiXHp95bkP9x~lenyqVcLhnYIpvby5XAIDNRkJrUr7USsNloBsVtjwUgrq2XIexObfjzgpWHsEn7FqPwy8062g2F2eVC3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of mono-allelic genome editing by ORNi-PCR. (A) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. To mimic mono-allelic mutations, gDNAs of WT and T4 or T9 were mixed at 1:1 ratio. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of WT + T4 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing." path-from-xml="dsy012f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122341/dsy012f5.tif" path-from-xml="dsy012f5.tif" target="_blank">View large</a><a section="120122341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f5.png?Expires=2147483647&Signature=hDh~UcSp~dzz3QqO92NKdZrwReE9XaiSz6QPQo3CDng~YQgdppFLtKMvbLUKdzaFfhUinEU-34x1x5OEVonYq3uZSu4JB0sFG55lT6LNHLQSy4rt~NPNtqSodbSre7wQom-RAp1i7-EnUNcHfXnmu4ui~lyADN~UAJGkpB0SF0Fha1yAl2qat26kkTxRq3Rno24NSJXETCLc~c811PMpXUOU~AgYuGhAwoFLqIfU~GwRfpPSwGwQS~ru5~~10EJgrzCAI1AbtM-xhvZVlvZQ884Ta7iARDYN3I~cw7OEefrKUhlWfJzK7VCDVyALZ8YTnCjfREodTEofDV4BsBoJKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122341&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of mono-allelic genome editing by ORNi-PCR. (A) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. To mimic mono-allelic mutations, gDNAs of WT and T4 or T9 were mixed at 1:1 ratio. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of WT + T4 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing.</p></div></div><div content-id="dsy012-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f5.png?Expires=2147483647&Signature=mi56tv8ZaoA0T~dtpfCFBvy9~XhrzYeXpi7M2L0SM8e2eZKcXQdMls0hl52fyDStqfZdDML3PuHAbezS2oHiCKiEhMa14JC5yHarDzC0kMV4hOsu3dnAv3SeCf5UPwkuD30FW0JCbE-fXvUYBhJG9vzcvGvYUtKS9pqPZnINkSthNOMonUL6IcUbuikNqOMuaiz87tB8g5XW9mkH2vp69tMZOeXFt8bxevudKYe0G9ZmeRiNn9~yudZgtiXHp95bkP9x~lenyqVcLhnYIpvby5XAIDNRkJrUr7USsNloBsVtjwUgrq2XIexObfjzgpWHsEn7FqPwy8062g2F2eVC3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of mono-allelic genome editing by ORNi-PCR. (A) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. To mimic mono-allelic mutations, gDNAs of WT and T4 or T9 were mixed at 1:1 ratio. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of WT + T4 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing." path-from-xml="dsy012f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122341/dsy012f5.tif" path-from-xml="dsy012f5.tif" target="_blank">View large</a><a section="120122341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f5.png?Expires=2147483647&Signature=hDh~UcSp~dzz3QqO92NKdZrwReE9XaiSz6QPQo3CDng~YQgdppFLtKMvbLUKdzaFfhUinEU-34x1x5OEVonYq3uZSu4JB0sFG55lT6LNHLQSy4rt~NPNtqSodbSre7wQom-RAp1i7-EnUNcHfXnmu4ui~lyADN~UAJGkpB0SF0Fha1yAl2qat26kkTxRq3Rno24NSJXETCLc~c811PMpXUOU~AgYuGhAwoFLqIfU~GwRfpPSwGwQS~ru5~~10EJgrzCAI1AbtM-xhvZVlvZQ884Ta7iARDYN3I~cw7OEefrKUhlWfJzK7VCDVyALZ8YTnCjfREodTEofDV4BsBoJKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122341&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of mono-allelic genome editing by ORNi-PCR. (A) ORNi-PCR with gDNAs extracted from WT and genome-edited Raji cells. To mimic mono-allelic mutations, gDNAs of WT and T4 or T9 were mixed at 1:1 ratio. PCR was performed in the presence (1 M) or absence of ORNs. M: molecular weight marker. (B) DNA-sequencing signals of PCR products. PCR products of WT + T4 in (A) [() ORN and ORN_20b] were purified from agarose gels and subjected to DNA sequencing.</p></div></div><p>Taken together, these findings show that ORNi-PCR is a useful method for distinguishing mono- and bi-allelic indel mutations from an intact target sequence. However, endpoint ORNi-PCR may not be suitable to discriminate mono- from bi-allelic indel mutations, largely because it may be difficult to quantitatively distinguish PCR products amplified from one- and two-copy templates.</p>                            <h3 scrollto-destination=120122343 id="120122343" class="section-title" >3.3 ORNi-PCR using the <em>Pfu</em> DNA polymerase, a real-time PCR machine, and crRNAs</h3>
<p>The <em>Pfu</em> DNA polymerase retains 35 exonuclease activity<a reveal-id="dsy012-B19" data-open="dsy012-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and can be used for ORNi-PCR.<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> We therefore examined whether <em>Pfu</em> DNA polymerase could also be used to detect genome-edited cells. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>, ORN_20b or ORN_24b prevented <em>Pfu</em> DNA polymerase from amplifying WT gDNA, but not mutant gDNA. In contrast to ORNi-PCR using the KOD DNA polymerase, the ORN_Target had no suppressive effects (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A and B</a></span>). Thus, <em>Pfu</em> DNA polymerase can also be used to detect genome-edited cells if an appropriate ORN is utilized.</p><p>One of the main aims of this study was to apply endpoint ORNi-PCR to detection of genome-edited cells. Nevertheless, it was also of interest to determine whether real-time (quantitative) ORNi-PCR is applicable for this purpose. To address this question, we performed ORNi-PCR on a real-time PCR machine. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>, in real-time ORNi-PCR, PCR amplification of the <em>THYN1</em> locus from WT gDNA was inhibited in the presence of ORN_24b, whereas amplification from T4 and T6 gDNAs was not affected. In addition, with a template mimicking a mono-allelic indel mutation (WT + T4), amplification was suppressed by 60% in the presence of the ORN (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In contrast, PCR amplification was not affected in the presence of an irrelevant ORN, ORN_302F(NC) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). These results suggested that intact target sequences, mono-, and bi-allelic mutations can be discriminated by analysis of amplification patterns of real-time ORNi-PCR.</p><p>Genome editing can be performed using recombinant CRISPR ribonucleoproteins (RNPs)<a reveal-id="dsy012-B20" data-open="dsy012-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsy012-B21" data-open="dsy012-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>. In this approach, synthesized sgRNAs or complexes of CRISPR RNAs (crRNAs) plus trans-activating crRNAs (tracrRNAs) are used as gRNAs. When performing genome editing using CRISPR RNPs, it would be more cost-effective to use crRNAs rather than ORNs in ORNi-PCR to detect genome-edited cells. Therefore, we tested the feasibility of using ORNi-PCR with crRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). The patterns of PCR amplification in the presence of crRNA_Target, a crRNA containing an RNA sequence complementary to the CRISPR target site (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>), were comparable to those obtained with <em>THYN1</em>-specific ORNs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B and C</a></span> and <a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Figs4 and 5</a>). crRNA_NC, a crRNA targeting an irrelevant locus (chicken <em>Pax5</em> locus),<a reveal-id="dsy012-B22" data-open="dsy012-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> did not affect amplification (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B and C</a></span>). These results suggested that crRNAs can also be used for ORNi-PCR. In addition, an ORN containing irrelevant sequences in addition to a target-specific sequence could also be used for ORNi-PCR.</p>                            <h3 scrollto-destination=120122347 id="120122347" class="section-title" >3.4 Screening of genome-edited cells by ORNi-PCR</h3>
<p>Next, we applied ORNi-PCR to screening of genome-edited cells. Using CRISPR, we attempted to edit the <em>CDKN2A(p16)</em> locus in human 293T cells (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6A</a>). After transfection of a CRISPR complex targeting this locus and single-colony isolation, gDNAs were extracted from 12 individual clones (C1C12) and subjected to ORNi-PCR to detect genome-editing events. ORNi-PCR with ORN_p16, a <em>CDKN2A(p16)</em>-specific 20-base ORN (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6B and C</a>), yielded amplification of the <em>CDKN2A(p16)</em> locus from 11 of the 12 samples (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6D</a>), suggesting that genome editing had occurred in these clones. Two ORNi-PCR products were detected from the gDNAs of C9, C11, and C12, implying that these clones harbour distinct bi-allelic genome edits. The amplicon from C10 was of higher molecular weight (1 kb), suggesting that an insertional mutation occurred in this clone. </p>                        <a id="120122349" scrollto-destination="120122349"></a>
<div id="dsy012-F6" data-id="dsy012-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f6.png?Expires=2147483647&Signature=aMJDFHaa5vnUhfGhM-2a6KxtQFNUXZTRNoN-3DSOSIxfiAh9rIVJCQU5eS20ATg1az70IktYkvZJfWX8MJmWZacFKfPnPDVWLnjA5WmCmPWp73QmvVeaTXtphsm453BX1jb5lIoQ~1wZoLCygf8LlUHPrwYcGz1d6wW5vzgGPZDkhWs4bvHgQwF3q0en7b5YB2NVKIqaIGd8vMy4Uog1LDYIes6TvJRUh3IoftNvusbO4b0b6mgf4H8d0pHkOYHaaKKBc5MSm2Ap~S~Yhbotx3SGSKS57qQgDj7oSgx5AgsVebZBL478pjPwN9vy6R9fS5TqJk5MFGrpwORS-13AEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of genome-edited cells by ORNi-PCR. (A) Schematic of screening of genome-edited cells by ORNi-PCR. (B) Target position of genome editing and sequence of a CDKN2A(p16)-specific ORN. The forward and reverse DNA sequences of the allele are shown. The CRISPR target site in 293T is underlined, and PAM positions are shown in red. A black arrow indicates the cleavage sites of CRISPR. (C) Setting of ORNi-PCR with gDNA extracted from 293T. M: molecular weight marker. (D) Results of ORNi-PCR (1 M ORN). (E) Schematic of confirmation of indel mutations. (F) DNA sequences around the CRISPR target site in the CDKN2A(p16) locus in genome-edited cells. The forward DNA sequences of both alleles are shown." path-from-xml="dsy012f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122349/dsy012f6.tif" path-from-xml="dsy012f6.tif" target="_blank">View large</a><a section="120122349" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f6.png?Expires=2147483647&Signature=YE58rDtgHVX4WrnXaVKBdCSdyZAfaUQvdhYS2MYvmms0R9Z10zdrBF0tw5tbSRatyiiOkJUmITaSV806U~FVbmCBUOWyKOlavdv6b-Kq27T0YjJbMp0JJ~EXD7W6YaxKwWpmC4qzzzzq3JnRGqTpwGMXrYppfwJT83Tsj0OLs9ixVSX0fKC5ZYb9ukdFHukx6iXVQ-vCmQ3tqPSY4~sbTM7y2NoZsbbRbuH7roic1vGWxaxhvjFeH6TgeAGWamFMxnCgG79GxVU0fbpnbEVd69XA6I4AZPlFZ2wYy8aQ5cC29TYs2I4RyYg2jbvKbTk8ef~neJwQZRQo6InfLVxOkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122349&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screening of genome-edited cells by ORNi-PCR. (A) Schematic of screening of genome-edited cells by ORNi-PCR. (B) Target position of genome editing and sequence of a <em>CDKN2A(p16)</em>-specific ORN. The forward and reverse DNA sequences of the allele are shown. The CRISPR target site in 293T is underlined, and PAM positions are shown in red. A black arrow indicates the cleavage sites of CRISPR. (C) Setting of ORNi-PCR with gDNA extracted from 293T. M: molecular weight marker. (D) Results of ORNi-PCR (1 M ORN). (E) Schematic of confirmation of indel mutations. (F) DNA sequences around the CRISPR target site in the <em>CDKN2A(p16)</em> locus in genome-edited cells. The forward DNA sequences of both alleles are shown.</p></div></div><div content-id="dsy012-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f6.png?Expires=2147483647&Signature=aMJDFHaa5vnUhfGhM-2a6KxtQFNUXZTRNoN-3DSOSIxfiAh9rIVJCQU5eS20ATg1az70IktYkvZJfWX8MJmWZacFKfPnPDVWLnjA5WmCmPWp73QmvVeaTXtphsm453BX1jb5lIoQ~1wZoLCygf8LlUHPrwYcGz1d6wW5vzgGPZDkhWs4bvHgQwF3q0en7b5YB2NVKIqaIGd8vMy4Uog1LDYIes6TvJRUh3IoftNvusbO4b0b6mgf4H8d0pHkOYHaaKKBc5MSm2Ap~S~Yhbotx3SGSKS57qQgDj7oSgx5AgsVebZBL478pjPwN9vy6R9fS5TqJk5MFGrpwORS-13AEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of genome-edited cells by ORNi-PCR. (A) Schematic of screening of genome-edited cells by ORNi-PCR. (B) Target position of genome editing and sequence of a CDKN2A(p16)-specific ORN. The forward and reverse DNA sequences of the allele are shown. The CRISPR target site in 293T is underlined, and PAM positions are shown in red. A black arrow indicates the cleavage sites of CRISPR. (C) Setting of ORNi-PCR with gDNA extracted from 293T. M: molecular weight marker. (D) Results of ORNi-PCR (1 M ORN). (E) Schematic of confirmation of indel mutations. (F) DNA sequences around the CRISPR target site in the CDKN2A(p16) locus in genome-edited cells. The forward DNA sequences of both alleles are shown." path-from-xml="dsy012f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122349/dsy012f6.tif" path-from-xml="dsy012f6.tif" target="_blank">View large</a><a section="120122349" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f6.png?Expires=2147483647&Signature=YE58rDtgHVX4WrnXaVKBdCSdyZAfaUQvdhYS2MYvmms0R9Z10zdrBF0tw5tbSRatyiiOkJUmITaSV806U~FVbmCBUOWyKOlavdv6b-Kq27T0YjJbMp0JJ~EXD7W6YaxKwWpmC4qzzzzq3JnRGqTpwGMXrYppfwJT83Tsj0OLs9ixVSX0fKC5ZYb9ukdFHukx6iXVQ-vCmQ3tqPSY4~sbTM7y2NoZsbbRbuH7roic1vGWxaxhvjFeH6TgeAGWamFMxnCgG79GxVU0fbpnbEVd69XA6I4AZPlFZ2wYy8aQ5cC29TYs2I4RyYg2jbvKbTk8ef~neJwQZRQo6InfLVxOkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122349&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screening of genome-edited cells by ORNi-PCR. (A) Schematic of screening of genome-edited cells by ORNi-PCR. (B) Target position of genome editing and sequence of a <em>CDKN2A(p16)</em>-specific ORN. The forward and reverse DNA sequences of the allele are shown. The CRISPR target site in 293T is underlined, and PAM positions are shown in red. A black arrow indicates the cleavage sites of CRISPR. (C) Setting of ORNi-PCR with gDNA extracted from 293T. M: molecular weight marker. (D) Results of ORNi-PCR (1 M ORN). (E) Schematic of confirmation of indel mutations. (F) DNA sequences around the CRISPR target site in the <em>CDKN2A(p16)</em> locus in genome-edited cells. The forward DNA sequences of both alleles are shown.</p></div></div><p>ORNi-PCR amplified 0.8 kb products from gDNAs of C1 and C3C8 (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6D</a>), the same size as the PCR product obtained when using WT gDNA as the template in the absence of ORNs (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6C</a>). To characterize the types of mutations, we amplified the <em>CDKN2A(p16)</em> locus from gDNAs from clones with positive signals in ORNi-PCR in the absence of ORNs and directly sequenced the amplicons (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6E</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). No DNA-sequencing signals corresponding to the intact CRISPR target site were detected in the amplicons from C1 and C3C8 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8C</a></span>), suggesting that bi-allelic mutations had been introduced into the target sites in these clones. These results of DNA sequencing were consistent with those of ORNi-PCR. Thus, ORNi-PCR can be applied to screening of genome-edited cells. Notably in this regard, no DNA fragments were amplified from C2 gDNA even in the absence of ORNs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8B</a></span>), potentially due to deletion of primer-binding sites, insertion of a DNA sequence too long to be amplified, or translocation of the <em>CDKN2A(p16)</em> locus.</p><p>At least three patterns of DNA-sequencing signals were detected in the amplicons of C3, C5, C7, and C8 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8C</a></span>). Such mosaicism may have occurred in a step-by-step manner during genome editing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), or have been caused by a mixture of different types of genome-edited cells (i.e. failed isolation of single-cell clones). Alternatively, it is also possible that such mosaicism was caused by aneuploidy in the cell cultures. In contrast, two patterns of DNA-sequencing signals were detected in the amplicons of C1, C4, and C6 (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6F</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8C</a></span>), suggesting that these cells were true clones of bi-allelically genome-edited cells.</p>                            <h3 scrollto-destination=120122352 id="120122352" class="section-title" >3.5 Screening of genome-edited cells by multiplex ORNi-PCR</h3>
<p>Genome editing can also occur simultaneously at multiple loci in a single cell. In this regard, it would be useful if genome-editing at multiple loci could be detected in a single-tube ORNi-PCR. We therefore examined whether multiplex ORNi-PCR can be used for screening of cells in which multiple target sites have been edited. Using CRISPR, we attempted to simultaneously edit the <em>CDKN2A(p16)</em> and <em>THYN1</em> loci in 293T cells (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7A</a>). Multiplex ORNi-PCR with ORN_p16 and ORN_24b yielded PCR products from the <em>CDKN2A(p16)</em> locus from gDNAs of 9 out of 11 clones (CT1, CT3CT8, CT10, and CT11), and from the <em>THYN1</em> locus from all gDNAs (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7BD</a>), suggesting that successful genome editing had occurred at each locus. The amplicons [<em>CDKN2A(p16)</em> and/or <em>THYN1</em>] of CT3, CT7, and CT10 were of different lengths, implying that the target loci were mutated mono-allelically or bi-allelically (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7D</a>). To determine the types of indel mutations in the other samples, we amplified the <em>CDKN2A(p16)</em> and <em>THYN1</em> loci in the absence of ORNs and directly sequenced the amplicons (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7E</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). As summarized in <a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7F</a>, CT2 and CT9 harboured mutations in the <em>THYN1</em> locus, but not in the <em>CDKN2A(p16)</em> locus, whereas the other clones harboured mutations in both loci. The types of genome editing (i.e. mono- or bi-allelic mutations) were confirmed in some samples (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7F</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S10C and S11</a></span>). The results of DNA sequencing were consistent with those of ORNi-PCR (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7D</a>). Thus, multiplex ORNi-PCR can be applied to screening of cells in which multiple target sites have been edited. </p>                        <a id="120122354" scrollto-destination="120122354"></a>
<div id="dsy012-F7" data-id="dsy012-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f7.png?Expires=2147483647&Signature=4ciVNGPPxwQ-3dBEewg~BxtLOTWyc4OT91qEidZGtUKnjT5enJRb5AMRCGU72lK-mWgbtoru7Z-BaJ95xsotp4ewpjB~r18YyZ5d3djZG872tSHslzTD1nN7uxE1ZTKmNGlCoB3zm3SYN67nZrMFm5lr6k-RZ64Lo-zRfgm5PTG~mhUyyU14fc9wyU8FTSO6V2rL5yfXvAHxDyQ6zzw~NvXwf5i1CmF1y~bAkRqnifgIyOCuImbvG4N72V33EaqgXHZ87et7g7wl~ZHqBlQg5l1tGrN4zkHuEOFhf5hDQtncKbzmOfSO3nlY39DwqUytFqjU0IZIczdMJulE6MOodw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of genome-edited cells by multiplex ORNi-PCR. (A) Schematic of screening of genome-edited cells by multiplex ORNi-PCR. (B) Target positions of ORNs. Black arrows indicate the cleavage sites of CRISPR. (C and D) Multiplex ORNi-PCR (1 M each ORN) with gDNA extracted from 293T WT and genome-edited cells. Arrows indicate positions of PCR amplicons of CDKN2A(p16) (orange) and THYN1 (blue). M, molecular weight marker. (E) Schematic of confirmation of indel mutations. (F) Types of indel mutations. N.C., not confirmed." path-from-xml="dsy012f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122354/dsy012f7.tif" path-from-xml="dsy012f7.tif" target="_blank">View large</a><a section="120122354" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f7.png?Expires=2147483647&Signature=mGdAROkswDJB0mdh9ldd-1T13oMEHE34i7jVTMS~mV8PISD3-sruDnYDsrLA7fcvo2z0b5TQAkPp74X~a7t1hizcquEsvbiha79aP5qoFxTSGykdXz-c5N3yWDxRafMAhPrz7Tv1FWtzUP3dlW0A5sGAbJ8QrMqgWZ1L5E-p7ieaWuyex~jWubNL9o0LVxbOZvDko4oSbMGt2gsUz4hI9oMB66fPivQE8gsuH96djy-TDzcpTqNvgsc8LtUS3Ft98GtatjB2CyCXnwpquW7ctrc7KXzBQHfH4~MnGFdGn5~bZZ62IbQYmAj8SBd~179ONndVBNl7RcQS725gNBplmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122354&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screening of genome-edited cells by multiplex ORNi-PCR. (A) Schematic of screening of genome-edited cells by multiplex ORNi-PCR. (B) Target positions of ORNs. Black arrows indicate the cleavage sites of CRISPR. (C and D) Multiplex ORNi-PCR (1 M each ORN) with gDNA extracted from 293T WT and genome-edited cells. Arrows indicate positions of PCR amplicons of <em>CDKN2A(p16)</em> (orange) and <em>THYN1</em> (blue). M, molecular weight marker. (E) Schematic of confirmation of indel mutations. (F) Types of indel mutations. N.C., not confirmed.</p></div></div><div content-id="dsy012-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f7.png?Expires=2147483647&Signature=4ciVNGPPxwQ-3dBEewg~BxtLOTWyc4OT91qEidZGtUKnjT5enJRb5AMRCGU72lK-mWgbtoru7Z-BaJ95xsotp4ewpjB~r18YyZ5d3djZG872tSHslzTD1nN7uxE1ZTKmNGlCoB3zm3SYN67nZrMFm5lr6k-RZ64Lo-zRfgm5PTG~mhUyyU14fc9wyU8FTSO6V2rL5yfXvAHxDyQ6zzw~NvXwf5i1CmF1y~bAkRqnifgIyOCuImbvG4N72V33EaqgXHZ87et7g7wl~ZHqBlQg5l1tGrN4zkHuEOFhf5hDQtncKbzmOfSO3nlY39DwqUytFqjU0IZIczdMJulE6MOodw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of genome-edited cells by multiplex ORNi-PCR. (A) Schematic of screening of genome-edited cells by multiplex ORNi-PCR. (B) Target positions of ORNs. Black arrows indicate the cleavage sites of CRISPR. (C and D) Multiplex ORNi-PCR (1 M each ORN) with gDNA extracted from 293T WT and genome-edited cells. Arrows indicate positions of PCR amplicons of CDKN2A(p16) (orange) and THYN1 (blue). M, molecular weight marker. (E) Schematic of confirmation of indel mutations. (F) Types of indel mutations. N.C., not confirmed." path-from-xml="dsy012f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122354/dsy012f7.tif" path-from-xml="dsy012f7.tif" target="_blank">View large</a><a section="120122354" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f7.png?Expires=2147483647&Signature=mGdAROkswDJB0mdh9ldd-1T13oMEHE34i7jVTMS~mV8PISD3-sruDnYDsrLA7fcvo2z0b5TQAkPp74X~a7t1hizcquEsvbiha79aP5qoFxTSGykdXz-c5N3yWDxRafMAhPrz7Tv1FWtzUP3dlW0A5sGAbJ8QrMqgWZ1L5E-p7ieaWuyex~jWubNL9o0LVxbOZvDko4oSbMGt2gsUz4hI9oMB66fPivQE8gsuH96djy-TDzcpTqNvgsc8LtUS3Ft98GtatjB2CyCXnwpquW7ctrc7KXzBQHfH4~MnGFdGn5~bZZ62IbQYmAj8SBd~179ONndVBNl7RcQS725gNBplmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122354&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screening of genome-edited cells by multiplex ORNi-PCR. (A) Schematic of screening of genome-edited cells by multiplex ORNi-PCR. (B) Target positions of ORNs. Black arrows indicate the cleavage sites of CRISPR. (C and D) Multiplex ORNi-PCR (1 M each ORN) with gDNA extracted from 293T WT and genome-edited cells. Arrows indicate positions of PCR amplicons of <em>CDKN2A(p16)</em> (orange) and <em>THYN1</em> (blue). M, molecular weight marker. (E) Schematic of confirmation of indel mutations. (F) Types of indel mutations. N.C., not confirmed.</p></div></div>                            <h3 scrollto-destination=120122355 id="120122355" class="section-title" >3.6 Detection of point mutations by ORNi-PCR</h3>
<p>Given that genome editing can be used to introduce a point mutation, and it is of practical importance to detect such mutations, we investigated whether ORNi-PCR is also capable of distinguishing point mutations. DNA sequencing of ORNi-PCR amplicons of C4 and C6 (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6D</a>) yielded one and two patterns of DNA-sequencing signals, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). C4 and C6 possess 1 and 2 bp deletions, respectively, in the CRISPR target site in one allele (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Fig.6F</a>). Thus, these results suggested that ORNi-PCR can detect a 2 bp deletion but not a 1 bp deletion of C4 under these experimental conditions (hybridization at 62C, see Section 2). We next examined whether a higher hybridization temperature would enable detection of the 1 bp deletion of C4 (<a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Fig.8</a>). To this end, we performed ORNi-PCR with ORN_p16 at an annealing temperature of 70 C (<a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Fig.8A and B</a>). Under these conditions, ORNi-PCR amplified the target product from C4, but not WT gDNA. Sequencing of the ORNi-PCR amplicon of C4 yielded a signal from the 1 bp deleted template. Similar conditions enabled detection of a 1 bp insertion (the <em>THYN1</em> locus in CT11, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S11 and S13</a></span>). Together, these results indicate that ORNi-PCR can distinguish a single-nucleotide indel mutation if the annealing temperature is optimized (<a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Fig.8D</a>). </p>                        <a id="120122357" scrollto-destination="120122357"></a>
<div id="dsy012-F8" data-id="dsy012-F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f8.png?Expires=2147483647&Signature=JUA5aEmtpKTcWsh1kR6pazY7FtP06mK7oc7Oht31VGKgHCpy-rm1~f~6w4Bwntyr7w1wi80DI2zLWGtiA48oEXWCkUNdCCNi8P2su9ZBXh8UKpg9QdcGjcoiIw5k60dWuEE0A0z5PNrrpVvyXHnF8vYKLeFZHrNlN9vqVJPE2IublyNOB~YLTyyBDaZxrMcs-PqwkLyo8FP4jrAfNpZz2-CFaUCMYNA6VDUdANlM2gUePceA0hvigyT6klTjrAwcBtAXJqGtukJamz~ecZ7Kv7OB2wkiMg2DWRdBeTH5-X1UN-YyUIsIC4heFXB7UErEeNFRNmbQhAOfEjnKASrHZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Discrimination of a single-nucleotide difference by ORNi-PCR. (A) Target positions of a CDKN2A(p16)-specific ORN, ORN_p16. Black arrow indicates the CRISPR cleavage sites. The forward DNA sequences around the CRISPR target site in the CDKN2A(p16) locus of both alleles are shown. The target site of the ORN is highlighted in light green. (B) Results of ORNi-PCR. PCR amplification was performed at two annealing temperatures (62 and 70C) in the presence (1 M) or absence of ORN_p16. M, molecular weight marker. (C) DNA-sequencing signals of ORNi-PCR products. ORNi-PCR products in (B) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of discrimination of a single-nucleotide difference by ORNi-PCR. At an annealing temperature of 62C, the ORN (green) hybridized with the target site containing a 1 bp mutation, but not a 2 bp mutation. Hybridization with the 1 bp-mutated target site was incomplete at an annealing temperature of 70C, resulting in successful amplification. The ORN hybridized with the intact target site and strongly inhibited amplification at both temperatures." path-from-xml="dsy012f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122357/dsy012f8.tif" path-from-xml="dsy012f8.tif" target="_blank">View large</a><a section="120122357" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f8.png?Expires=2147483647&Signature=4AQadxG7xS5G6ujsN7Z~B583xgAF8cqZMKJRUmY~SFekvXAv9BnB~kBqnmnyiBTsDBqDIiLQwbzDDN7R74mL~vXlOO55ZjkK7WG2uqANJIHg2hfDyZ8mGIfP1LTvzAkwj1M5i0CZcbfnQlm-2wumVYizDqQDx9-V3Z7YbvMWHoBw6C-A79A7eUfiyXXG5-r5FGMcuqz3S-I9euntAWyKXo5dsK5zBQKsBsUhl~oEQmyrtJnk6-TpgANJSQJbX8SeGkKNc8MGA10SnDTiEct3Y0xRuoxESiI~gowV6IcXurQFe8GAcXxeV1OcD7ShyQzTin7OlJGEsjNpT916BKqktQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122357&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Discrimination of a single-nucleotide difference by ORNi-PCR. (A) Target positions of a <em>CDKN2A(p16)</em>-specific ORN, ORN_p16. Black arrow indicates the CRISPR cleavage sites. The forward DNA sequences around the CRISPR target site in the <em>CDKN2A(p16)</em> locus of both alleles are shown. The target site of the ORN is highlighted in light green. (B) Results of ORNi-PCR. PCR amplification was performed at two annealing temperatures (62 and 70C) in the presence (1 M) or absence of ORN_p16. M, molecular weight marker. (C) DNA-sequencing signals of ORNi-PCR products. ORNi-PCR products in (B) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of discrimination of a single-nucleotide difference by ORNi-PCR. At an annealing temperature of 62C, the ORN (green) hybridized with the target site containing a 1 bp mutation, but not a 2 bp mutation. Hybridization with the 1 bp-mutated target site was incomplete at an annealing temperature of 70C, resulting in successful amplification. The ORN hybridized with the intact target site and strongly inhibited amplification at both temperatures.</p></div></div><div content-id="dsy012-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f8.png?Expires=2147483647&Signature=JUA5aEmtpKTcWsh1kR6pazY7FtP06mK7oc7Oht31VGKgHCpy-rm1~f~6w4Bwntyr7w1wi80DI2zLWGtiA48oEXWCkUNdCCNi8P2su9ZBXh8UKpg9QdcGjcoiIw5k60dWuEE0A0z5PNrrpVvyXHnF8vYKLeFZHrNlN9vqVJPE2IublyNOB~YLTyyBDaZxrMcs-PqwkLyo8FP4jrAfNpZz2-CFaUCMYNA6VDUdANlM2gUePceA0hvigyT6klTjrAwcBtAXJqGtukJamz~ecZ7Kv7OB2wkiMg2DWRdBeTH5-X1UN-YyUIsIC4heFXB7UErEeNFRNmbQhAOfEjnKASrHZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Discrimination of a single-nucleotide difference by ORNi-PCR. (A) Target positions of a CDKN2A(p16)-specific ORN, ORN_p16. Black arrow indicates the CRISPR cleavage sites. The forward DNA sequences around the CRISPR target site in the CDKN2A(p16) locus of both alleles are shown. The target site of the ORN is highlighted in light green. (B) Results of ORNi-PCR. PCR amplification was performed at two annealing temperatures (62 and 70C) in the presence (1 M) or absence of ORN_p16. M, molecular weight marker. (C) DNA-sequencing signals of ORNi-PCR products. ORNi-PCR products in (B) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of discrimination of a single-nucleotide difference by ORNi-PCR. At an annealing temperature of 62C, the ORN (green) hybridized with the target site containing a 1 bp mutation, but not a 2 bp mutation. Hybridization with the 1 bp-mutated target site was incomplete at an annealing temperature of 70C, resulting in successful amplification. The ORN hybridized with the intact target site and strongly inhibited amplification at both temperatures." path-from-xml="dsy012f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122357/dsy012f8.tif" path-from-xml="dsy012f8.tif" target="_blank">View large</a><a section="120122357" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f8.png?Expires=2147483647&Signature=4AQadxG7xS5G6ujsN7Z~B583xgAF8cqZMKJRUmY~SFekvXAv9BnB~kBqnmnyiBTsDBqDIiLQwbzDDN7R74mL~vXlOO55ZjkK7WG2uqANJIHg2hfDyZ8mGIfP1LTvzAkwj1M5i0CZcbfnQlm-2wumVYizDqQDx9-V3Z7YbvMWHoBw6C-A79A7eUfiyXXG5-r5FGMcuqz3S-I9euntAWyKXo5dsK5zBQKsBsUhl~oEQmyrtJnk6-TpgANJSQJbX8SeGkKNc8MGA10SnDTiEct3Y0xRuoxESiI~gowV6IcXurQFe8GAcXxeV1OcD7ShyQzTin7OlJGEsjNpT916BKqktQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122357&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Discrimination of a single-nucleotide difference by ORNi-PCR. (A) Target positions of a <em>CDKN2A(p16)</em>-specific ORN, ORN_p16. Black arrow indicates the CRISPR cleavage sites. The forward DNA sequences around the CRISPR target site in the <em>CDKN2A(p16)</em> locus of both alleles are shown. The target site of the ORN is highlighted in light green. (B) Results of ORNi-PCR. PCR amplification was performed at two annealing temperatures (62 and 70C) in the presence (1 M) or absence of ORN_p16. M, molecular weight marker. (C) DNA-sequencing signals of ORNi-PCR products. ORNi-PCR products in (B) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of discrimination of a single-nucleotide difference by ORNi-PCR. At an annealing temperature of 62C, the ORN (green) hybridized with the target site containing a 1 bp mutation, but not a 2 bp mutation. Hybridization with the 1 bp-mutated target site was incomplete at an annealing temperature of 70C, resulting in successful amplification. The ORN hybridized with the intact target site and strongly inhibited amplification at both temperatures.</p></div></div><p>We also investigated whether ORNi-PCR can detect point mutations in another sample. Here, we utilized the <em>CDKN2A</em>(<em>p16</em>) locus in human HCT116 cells as a model locus, in which an insertion of a single nucleotide (guanine) is present in the first exon in one allele (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9A</a>).<a reveal-id="dsy012-B17" data-open="dsy012-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy012-B23" data-open="dsy012-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> We designed an ORN, ORN_Gx5, complementary to the sequence of the G-insertion allele (Gx5) but with a single-nucleotide mismatch at the 3 position with the corresponding DNA sequence in the other allele (Gx4) (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9A</a>). As shown in <a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9B</a>, ORN_Gx5 suppressed amplification of the <em>CDKN2A</em>(<em>p16</em>) locus when the annealing temperature was 64C, suggesting that the ORN hybridized with the target DNA sequence in both alleles. In contrast, amplification was not clearly suppressed when the annealing temperature was raised to 68C (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9B</a>). Sequencing of the PCR product revealed that ORN_Gx5 suppressed PCR amplification of the <em>CDKN2A</em>(<em>p16</em>) locus from the Gx5 allele, but not the Gx4 allele (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9C</a>). Such suppressive effects were not observed when the annealing temperature was raised to 72C (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9B and C</a>). Thus, these results provide further confirmation that ORNi-PCR can distinguish a point mutation if the annealing temperature is optimized (<a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">Fig.9D</a>). </p>                        <a id="120122359" scrollto-destination="120122359"></a>
<div id="dsy012-F9" data-id="dsy012-F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f9.png?Expires=2147483647&Signature=PmCuqmt94Xt4ArYfs0owA3TcV1etdp8TrREV~4pAAAU~8KYNMlNPBINyAFGjahVzceWBG01eDHW~tVun6Sj2dJfaacTwc9v6ZwFKZEmhsaL2Yct3Vpv6eLpfv2tCA5FNAw~opowP6ZYMPTFV8ZpXSeNutkC7xxhiMxBn5EZnvv9yFyRFHlN8RKUKZURAyni72aIfmKp6UJlHABJ3Em7CZoNtsA~afe9alocTJo9cPGyEtI3a35nOqA87~ew4zkeQGk52HPS3nHicGwQfeJ61Ks5INvGKK9pr5n0wr4FZAVUbIQPaBHIMRCs~6EPHHZs~u-qMMkT-fpCBipzjcxtlyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of a point mutation by ORNi-PCR. Target position and sequence of a CDKN2A(p16)-specific ORN, ORN_Gx5. The forward DNA sequences of both alleles (Gx4 and Gx5) in HCT116 are shown. (B) Results of ORNi-PCR with gDNA extracted from HCT116. PCR was performed at various hybridization temperatures in the presence (1 M) or absence of the ORN. M, molecular weight marker. (C) DNA-sequencing signals of PCR products. PCR products in (B) (68 and 72C) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of detection of point mutations by ORNi-PCR. (Left) At an annealing temperature of 64C, ORN_Gx5 (green) hybridized with the target site in the Gx5 allele and the corresponding site in the Gx4 allele in the CDKN2A(p16) locus. (Middle) At an annealing temperature of 68C, ORN_Gx5 hybridized with the target site in a Gx5 allele-specific manner. Because of a single-base mismatch of the 3 cytosine of ORN_Gx5, the ORN cannot hybridize with the corresponding site in the Gx4 allele at this temperature. (Right) At an annealing temperature of 72C, ORN_Gx5 did not hybridize even with the target sites." path-from-xml="dsy012f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122359/dsy012f9.tif" path-from-xml="dsy012f9.tif" target="_blank">View large</a><a section="120122359" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f9.png?Expires=2147483647&Signature=Kjxc~zsC4nvG3ltID82CqUUTOxECbckN~aTGYNqTyBUejH~OfJa~c6OqODAy2-BBuWL4W5j6dXiVmd4SjAYae-LLyJZqNZ148a7ra5T34xbBcR~VVs2lIC4K-3q3dkDj6yGo~mNbQvvO47D6tlszYoCpyRc0RDzzS8Dja8JlIfFxfTUiY3GupYe~B-zTYEihAnGZfRsVL66PNDL8XYTuXDoPnaH~~rvlJtDhAfznWGpwiaz3Mzf3DJYdDklnvvRn19H3rdVMnNNs9Yqc5nW6eq2O7qf9DU4A2M8udXb7s3MC5co-a6LelJyfLwlX79D5RagCv56cAXfquhCuiR-1GQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122359&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of a point mutation by ORNi-PCR. Target position and sequence of a <em>CDKN2A(p16)</em>-specific ORN, ORN_Gx5. The forward DNA sequences of both alleles (Gx4 and Gx5) in HCT116 are shown. (B) Results of ORNi-PCR with gDNA extracted from HCT116. PCR was performed at various hybridization temperatures in the presence (1 M) or absence of the ORN. M, molecular weight marker. (C) DNA-sequencing signals of PCR products. PCR products in (B) (68 and 72C) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of detection of point mutations by ORNi-PCR. (Left) At an annealing temperature of 64C, ORN_Gx5 (green) hybridized with the target site in the Gx5 allele and the corresponding site in the Gx4 allele in the <em>CDKN2A(p16)</em> locus. (Middle) At an annealing temperature of 68C, ORN_Gx5 hybridized with the target site in a Gx5 allele-specific manner. Because of a single-base mismatch of the 3 cytosine of ORN_Gx5, the ORN cannot hybridize with the corresponding site in the Gx4 allele at this temperature. (Right) At an annealing temperature of 72C, ORN_Gx5 did not hybridize even with the target sites.</p></div></div><div content-id="dsy012-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/m_dsy012f9.png?Expires=2147483647&Signature=PmCuqmt94Xt4ArYfs0owA3TcV1etdp8TrREV~4pAAAU~8KYNMlNPBINyAFGjahVzceWBG01eDHW~tVun6Sj2dJfaacTwc9v6ZwFKZEmhsaL2Yct3Vpv6eLpfv2tCA5FNAw~opowP6ZYMPTFV8ZpXSeNutkC7xxhiMxBn5EZnvv9yFyRFHlN8RKUKZURAyni72aIfmKp6UJlHABJ3Em7CZoNtsA~afe9alocTJo9cPGyEtI3a35nOqA87~ew4zkeQGk52HPS3nHicGwQfeJ61Ks5INvGKK9pr5n0wr4FZAVUbIQPaBHIMRCs~6EPHHZs~u-qMMkT-fpCBipzjcxtlyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of a point mutation by ORNi-PCR. Target position and sequence of a CDKN2A(p16)-specific ORN, ORN_Gx5. The forward DNA sequences of both alleles (Gx4 and Gx5) in HCT116 are shown. (B) Results of ORNi-PCR with gDNA extracted from HCT116. PCR was performed at various hybridization temperatures in the presence (1 M) or absence of the ORN. M, molecular weight marker. (C) DNA-sequencing signals of PCR products. PCR products in (B) (68 and 72C) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of detection of point mutations by ORNi-PCR. (Left) At an annealing temperature of 64C, ORN_Gx5 (green) hybridized with the target site in the Gx5 allele and the corresponding site in the Gx4 allele in the CDKN2A(p16) locus. (Middle) At an annealing temperature of 68C, ORN_Gx5 hybridized with the target site in a Gx5 allele-specific manner. Because of a single-base mismatch of the 3 cytosine of ORN_Gx5, the ORN cannot hybridize with the corresponding site in the Gx4 allele at this temperature. (Right) At an annealing temperature of 72C, ORN_Gx5 did not hybridize even with the target sites." path-from-xml="dsy012f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122359/dsy012f9.tif" path-from-xml="dsy012f9.tif" target="_blank">View large</a><a section="120122359" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012f9.png?Expires=2147483647&Signature=Kjxc~zsC4nvG3ltID82CqUUTOxECbckN~aTGYNqTyBUejH~OfJa~c6OqODAy2-BBuWL4W5j6dXiVmd4SjAYae-LLyJZqNZ148a7ra5T34xbBcR~VVs2lIC4K-3q3dkDj6yGo~mNbQvvO47D6tlszYoCpyRc0RDzzS8Dja8JlIfFxfTUiY3GupYe~B-zTYEihAnGZfRsVL66PNDL8XYTuXDoPnaH~~rvlJtDhAfznWGpwiaz3Mzf3DJYdDklnvvRn19H3rdVMnNNs9Yqc5nW6eq2O7qf9DU4A2M8udXb7s3MC5co-a6LelJyfLwlX79D5RagCv56cAXfquhCuiR-1GQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122359&ar=4989276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy012f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of a point mutation by ORNi-PCR. Target position and sequence of a <em>CDKN2A(p16)</em>-specific ORN, ORN_Gx5. The forward DNA sequences of both alleles (Gx4 and Gx5) in HCT116 are shown. (B) Results of ORNi-PCR with gDNA extracted from HCT116. PCR was performed at various hybridization temperatures in the presence (1 M) or absence of the ORN. M, molecular weight marker. (C) DNA-sequencing signals of PCR products. PCR products in (B) (68 and 72C) were purified from agarose gels and subjected to DNA sequencing using a forward primer. (D) Mode of detection of point mutations by ORNi-PCR. (Left) At an annealing temperature of 64C, ORN_Gx5 (green) hybridized with the target site in the Gx5 allele and the corresponding site in the Gx4 allele in the <em>CDKN2A(p16)</em> locus. (Middle) At an annealing temperature of 68C, ORN_Gx5 hybridized with the target site in a Gx5 allele-specific manner. Because of a single-base mismatch of the 3 cytosine of ORN_Gx5, the ORN cannot hybridize with the corresponding site in the Gx4 allele at this temperature. (Right) At an annealing temperature of 72C, ORN_Gx5 did not hybridize even with the target sites.</p></div></div><p>Taken together, our findings show that ORNi-PCR can distinguish point mutations if the DNA/RNA hybridization is performed at the optimal temperature. Therefore, the optimal ORN hybridization temperature should be determined in advance for the detection of point mutations during the screening of genome-edited cells.</p>                            <h2 scrollto-destination=120122361 id="120122361" class="section-title" >4. Discussion</h2>
<p>In this study, we demonstrated that ORNi-PCR can be used to detect genome-edited cells without using expensive equipment such as real-time PCR machines. ORNi-PCR could discriminate an intact target DNA sequence from bi- and mono-allelic mutations (<a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Figs4 and 5</a>). In addition, ORNi-PCR could be applied to screening of genome-edited cells (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Figs6 and 7</a>). However, because it may be difficult to distinguish between mono- and bi-allelic mutations using endpoint ORNi-PCR, endpoint ORNi-PCR should be primarily used for the detection of genome-edited cells possessing at least mono-allelic mutations at a target site before subsequent determination of mono- or bi-allelic mutations by other methods (see detailed discussions on this below). In this study, we evaluated the introduction of CRISPR-mediated genome editing by ORNi-PCR. However, genome edits introduced by ZFNs or TALENs could also be detected by ORNi-PCR because the cleavage sites of these enzymes are also well defined.<a reveal-id="dsy012-B24" data-open="dsy012-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p><p>Mismatch cleavage assays using T7E1 or Surveyor nuclease are often used for the evaluation of indel mutations. In these approaches, after performing PCR of the target sequences, PCR products are denatured and reannealed, and DNA bulges at mismatch sites in the heteroduplexes are cleaved by these nucleases. These approaches are suitable for the evaluation of indel mutations in pools of genome-edited cells (i.e. in cells containing mixtures of WT and/or various types mutant sequences that easily form DNA bulges at mismatch sites). However, mismatch cleavage assays would not be suitable for screening of genome-edited cells because the addition of WT gDNA to the reaction would be required to detect a homozygous bi-allelic mutation in a single clone. In addition, it may be difficult to distinguish mono-allelic mutations from heterozygous bi-allelic mutations using this method. In this context, ORNi-PCR is more advantageous than mismatch cleavage assays (<a reveal-id="dsy012-T1" data-open="dsy012-T1" class="link link-reveal link-table xref-fig">Table1</a>). On the other hand, because ORNi-PCR generates positive PCR signals when one allele of the genome of a target cell is edited, it may be difficult to distinguish bi- and mono-allelic mutations by endpoint ORNi-PCR. This feature is one of the drawbacks of ORNi-PCR not found in endpoint PCR-based methods<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy012-B14" data-open="dsy012-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> (<a reveal-id="dsy012-T1" data-open="dsy012-T1" class="link link-reveal link-table xref-fig">Table1</a>). Thus, endpoint ORNi-PCR should be primarily used to detect genome-edited cells possessing at least mono-allelic mutations at a target site before subsequent detection of mono- or bi-allelic mutations by other methods. However, endpoint ORNi-PCR would still be useful if hemi-knockout is also required, as in studies of X-chromosome inactivation, genomic imprinting, and cancer.<a reveal-id="dsy012-B25 dsy012-B26 dsy012-B27" data-open="dsy012-B25 dsy012-B26 dsy012-B27" class="link link-ref link-reveal xref-bibr"><sup>2527</sup></a> In addition, ORNi-PCR could be applied when optimal primers cannot be designed in some PCR-based methods,<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> especially as ORNs can be designed flexibly on their design.<a reveal-id="dsy012-B15" data-open="dsy012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> We demonstrated that genome-editing events at multiple loci can be detected in a single-tube endpoint ORNi-PCR (<a reveal-id="dsy012-F7" data-open="dsy012-F7" class="link link-reveal link-table xref-fig">Fig.7</a>), which has not been achievable before using endpoint PCR-based methods.<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy012-B14" data-open="dsy012-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In this regard, the primer combination should be more carefully considered to avoid overlap of amplicons in the endpoint PCR-based methods. <div content-id="dsy012-T1" class="table-modal"><div class="table-wrap"><div id="dsy012-T1" data-id="dsy012-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of genome-editing detection methods</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Methods</th><th>Time (h)</th><th>Accuracy</th><th>Labor</th><th>Cost</th><th>Limitation</th></tr></thead><tbody><tr><td>Mismatch cleavage assay<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>&gt;4</td><td>Moderate</td><td>Moderate</td><td>Moderate</td><td>Not suitable for detection of mutations that do not form DNA bulges at mismatches in the assay (e.g. homozygous bi-allelic mutations).</td></tr><tr><td>PCR-based method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>12</td><td>High</td><td>Low</td><td>Low</td><td></td></tr><tr><td>ORNi-PCR</td><td>12</td><td>High</td><td>Low</td><td>Low</td><td>It may be difficult to distinguish mono- and bi-allelic mutations by endpoint ORNi-PCR.</td></tr></tbody></table></div><div content-id="dsy012-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Methods</th><th>Time (h)</th><th>Accuracy</th><th>Labor</th><th>Cost</th><th>Limitation</th></tr></thead><tbody><tr><td>Mismatch cleavage assay<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>&gt;4</td><td>Moderate</td><td>Moderate</td><td>Moderate</td><td>Not suitable for detection of mutations that do not form DNA bulges at mismatches in the assay (e.g. homozygous bi-allelic mutations).</td></tr><tr><td>PCR-based method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>12</td><td>High</td><td>Low</td><td>Low</td><td></td></tr><tr><td>ORNi-PCR</td><td>12</td><td>High</td><td>Low</td><td>Low</td><td>It may be difficult to distinguish mono- and bi-allelic mutations by endpoint ORNi-PCR.</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Properties of these methods have been previously described.<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsy012-T1" data-id="dsy012-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of genome-editing detection methods</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Methods</th><th>Time (h)</th><th>Accuracy</th><th>Labor</th><th>Cost</th><th>Limitation</th></tr></thead><tbody><tr><td>Mismatch cleavage assay<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>&gt;4</td><td>Moderate</td><td>Moderate</td><td>Moderate</td><td>Not suitable for detection of mutations that do not form DNA bulges at mismatches in the assay (e.g. homozygous bi-allelic mutations).</td></tr><tr><td>PCR-based method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>12</td><td>High</td><td>Low</td><td>Low</td><td></td></tr><tr><td>ORNi-PCR</td><td>12</td><td>High</td><td>Low</td><td>Low</td><td>It may be difficult to distinguish mono- and bi-allelic mutations by endpoint ORNi-PCR.</td></tr></tbody></table></div><div content-id="dsy012-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Methods</th><th>Time (h)</th><th>Accuracy</th><th>Labor</th><th>Cost</th><th>Limitation</th></tr></thead><tbody><tr><td>Mismatch cleavage assay<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>&gt;4</td><td>Moderate</td><td>Moderate</td><td>Moderate</td><td>Not suitable for detection of mutations that do not form DNA bulges at mismatches in the assay (e.g. homozygous bi-allelic mutations).</td></tr><tr><td>PCR-based method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a></td><td>12</td><td>High</td><td>Low</td><td>Low</td><td></td></tr><tr><td>ORNi-PCR</td><td>12</td><td>High</td><td>Low</td><td>Low</td><td>It may be difficult to distinguish mono- and bi-allelic mutations by endpoint ORNi-PCR.</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Properties of these methods have been previously described.<a reveal-id="dsy012-B13" data-open="dsy012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p></div></div></div></div></p><p>On the other hand, we also demonstrated that real-time ORNi-PCR could detect genome-edited cells (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>) and distinguish between bi- and mono-allelic mutations. In real-time or digital PCR with fluorescent probes complementary to a target site of genome editing (e.g. TaqMan probes), the decline in fluorescent emission can be taken as an indication of genome-editing events.<a reveal-id="dsy012-B10 dsy012-B11 dsy012-B12" data-open="dsy012-B10 dsy012-B11 dsy012-B12" class="link link-ref link-reveal xref-bibr"><sup>1012</sup></a> In contrast, the fluorescence emission of SYBR Green can be taken as an indication of genome-editing events in real-time ORNi-PCR. Notably, ORNi-PCR does not require modification of each ORN with fluorescent chemicals, representing an economic advantage over other detection methods that use fluorescent probes.</p><p>In our proof-of-principle experiments, genome-edited cells were efficiently isolated because of stable expression of the CRISPR complex (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Figs6 and 7</a>, and see Section 2). Although this feature is useful for the characterization of genome editing performed by CRISPR, it is an undesirable feature in biological and medical applications. In such cases, transient expression of the CRISPR complex might be an optional strategy. However, sometimes large numbers of candidate clones must be screened if the efficiency of genome-editing is low, e.g. when the target region is not readily accessible to the genome-editing machinery, as in the case of heterochromatin. Because ORNi-PCR can detect various types of indel mutations (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Figs 68</a>), it would be useful for the detection of small numbers of genome-edited cells among large numbers of candidate cells.</p><p>We found that ORNi-PCR can distinguish point mutations if DNA/RNA hybridization is performed under optimal temperature (<a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Figs8 and 9</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&amp;Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). This property of ORNi-PCR can also be used for detection of single-nucleotide polymorphisms (SNPs), mutations in pathogenic cells such as cancer cells, and other subtle differences in genome sequences. In this study, we estimated the hybridization temperatures of ORNs using a <em>T</em><sub>m</sub> calculation program, Oligo Calc<a reveal-id="dsy012-B28" data-open="dsy012-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_table_s2.xlsx?Expires=2147483647&amp;Signature=lXoJ-eMt-9KILoaohq1ZmMCHkYm-~LDxKyvRvb-I-wHnWKTvct9qADQJhrAXNYNMrP2FpxDWhITSiGrsC~-PJ8aVoazHeOsEWlHTCtXR9fhkzowNvgzFUC7jCxbjje~7ZXxw1~4ELojHoPreONUO3maAStqqqO~NMzc1EnyBN1us9VPR5FP9jmyvdcdubz5HkadvoePJvYSV5lI3EUxUeiS4nHtju1JV5c1RfDfbjSzvO3Oces46GN2934fBK8gJ3v7h~q3AqcQVAOhHD2oYdVXDjKOyPOXP8mtQcHjmwVkzFh569Rh5rAVkG-HcihEfyZ6hF1wlI7V0TCZw61CHTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), and annealing was performed at 62C as a default setting in most ORNi-PCR experiments. This temperature was suitable for detection of 2 bp mutations (<a reveal-id="dsy012-F4" data-open="dsy012-F4" class="link link-reveal link-table xref-fig">Figs4</a>, <a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">8</a>, and <a reveal-id="dsy012-F9" data-open="dsy012-F9" class="link link-reveal link-table xref-fig">9</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&amp;Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). Several bases are usually edited in most cases of CRISPR-mediated genome editing in mammalian cells. In fact, we found 2 bp mutations in many cases (<a reveal-id="dsy012-F6" data-open="dsy012-F6" class="link link-reveal link-table xref-fig">Figs6 and 7</a>). Thus, in the standard knock-out strategy, optimization of the annealing temperature might be unnecessary when screening genome-edited mammalian cells by ORNi-PCR. On the other hand, higher annealing temperatures were required to detect 1 bp mutations (<a reveal-id="dsy012-F8" data-open="dsy012-F8" class="link link-reveal link-table xref-fig">Figs8 and 9</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&amp;Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). In this regard, if 1 bp mutations occurred in both alleles during genome editing, the method may have failed to detect such cells at an annealing temperature of 62C. To avoid such false negatives, it is recommended to empirically determine the optimal hybridization temperature in advance by performing ORNi-PCR on WT gDNA. To distinguish single-nucleotide differences, it may be preferable to use the highest temperature at which the ORN can hybridize with the target sequence.</p><p>PCR can be perturbed by unpredicted contaminants when impure gDNAs are used. In such cases, it would be difficult to judge whether the absence of amplification in ORNi-PCR reflects sequence-specific suppression by ORNs or such an undesirable perturbation. In this regard, it would be useful to amplify an irrelevant locus as an internal control in parallel with ORNi-PCR in the same reaction mixture; amplification of the internal control would indicate that the reaction conditions were suitable for PCR. In fact, we showed that it was possible to evaluate introduction of genome editing using such a multiplex PCR system, in which a target primer set, a target-specific ORN, and another primer set that amplified an internal control were mixed in the same tube (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). These results also ensure target specificity of the designed ORNs, and ORNi-PCR with an internal control PCR could more reliably detect genome-editing events.</p><p>In this study, we used ORNs at 1 M to screen genome-edited cells (see Section 2) because PCR amplification of a target locus in WT gDNA was completely suppressed by ORNs at 1 or 2 M (<a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2B</a>). Lower concentrations of some ORNs were less effective and resulted in the appearance of faint amplicons (e.g. with ORN_20b at 0.1 or 0.5 M in <a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2B</a>). Because such amplicons might lead to false positive results, effective concentrations of ORNs should be determined by titration as shown in <a reveal-id="dsy012-F2" data-open="dsy012-F2" class="link link-reveal link-table xref-fig">Fig.2</a>. Additionally, we used 20 ng purified human gDNAs in 10 l reaction mixtures for 3035 cycles of ORNi-PCR. ORN suppression might be insufficient if excess amounts of gDNAs or more cycles are used, and might also skew the results. Concerning the DNA purification method, we used gDNAs purified by the standard phenol/chloroform extraction, which could be laborious when many clones need to be handled. Crude extracts might provide an easier option, but potential RNase activity in the crude extracts could degrade ORNs in the ORNi-PCR reactions. However, this could be avoided by adding RNase inhibitors to the reactions. To increase the flexibility of ORNi-PCR, the ORNi-PCR protocol should be further optimized in the future.</p><p>In summary, we showed that ORNi-PCR can be applied to screening of genome-edited cells without expensive equipment. We believe that, despite its drawbacks, ORNi-PCR is a potentially useful tool that will make genome editing easier. In addition, ORNi-PCR enables detection of SNPs, mutations in pathogenic cells such as cancer cells, and other subtle differences in genome sequences.</p>                            <h2 scrollto-destination=120122370 id="120122370" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank G.M. Church for providing plasmids (Addgene Nos. 41815 and 41824).</p>                            <h3 scrollto-destination=120122372 id="120122372" class="section-title" >Funding</h3>
<p>This work was supported by Hirosaki University Graduate School of Medicine (T.F. and H.F.).</p>                            <h3 scrollto-destination=120122374 id="120122374" class="section-title" >Conflict of interest</h3>
<p>T.F. and H.F. filed patent applications on ORNi-PCR. The patent details are as follows: Name: Method for suppressing amplification of specific nucleic acid sequences; number: Japanese Patent Application No. 2014-176018; and name: Method for detecting differences in target nucleic acid region; number: Japanese Patent Application No. 201881752.</p>                            <h2 scrollto-destination=120122376 id="120122376" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy012-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaj</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gersbach</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Barbas</div> <div class="given-names">C. F.</div>3rd</div></div>. <div class="year">2013</div>, 
<div class="article-title">ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering</div>, <div class="source">Trends Biotechnol</div>., <div class="volume">31</div>, <div class="fpage">397</div><div class="lpage">405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ZFN%2C%20TALEN%2C%20and%20CRISPR%2FCas-based%20methods%20for%20genome%20engineering&amp;author=T.%20Gaj&amp;author=C.%20A.%20Gersbach&amp;author=C.%20F.%20Barbas&amp;publication_year=2013&amp;journal=Trends%20Biotechnol&amp;volume=31&amp;pages=397-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2013.04.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2013.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2013.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23664777" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doudna</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Charpentier</div> <div class="given-names">E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome editing. The new frontier of genome engineering with CRISPR-Cas9</div>, <div class="source">Science</div>, <div class="volume">346</div>, <div class="fpage">1258096</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20editing.%20The%20new%20frontier%20of%20genome%20engineering%20with%20CRISPR-Cas9&amp;author=J.%20A.%20Doudna&amp;author=E.%20Charpentier&amp;publication_year=2014&amp;journal=Science&amp;volume=346&amp;pages=1258096" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1258096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1258096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1258096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25430774" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harrison</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">B. V.</div></div>, <div class="name string-name"><div class="surname">OConnor-Giles</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Wildonger</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A CRISPR view of development</div>, <div class="source">Genes Dev</div>., <div class="volume">28</div>, <div class="fpage">1859</div><div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20CRISPR%20view%20of%20development&amp;author=M.%20M.%20Harrison&amp;author=B.%20V.%20Jenkins&amp;author=K.%20M.%20O%E2%80%99Connor-Giles&amp;author=J.%20Wildonger&amp;publication_year=2014&amp;journal=Genes%20Dev&amp;volume=28&amp;pages=1859-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.248252.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.248252.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.248252.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184674" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zischewski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fischer</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bortesi</div> <div class="given-names">L.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases</div>, <div class="source">Biotechnol. Adv</div>., <div class="volume">35</div>, <div class="fpage">95</div><div class="lpage">104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20on-target%20and%20off-target%20mutations%20generated%20by%20CRISPR%2FCas9%20and%20other%20sequence-specific%20nucleases&amp;author=J.%20Zischewski&amp;author=R.%20Fischer&amp;author=L.%20Bortesi&amp;publication_year=2017&amp;journal=Biotechnol.%20Adv&amp;volume=35&amp;pages=95-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2016.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2016.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2016.12.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28011075" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">An efficient genotyping method for genome-modified animals and human cells generated with CRISPR/Cas9 system</div>, <div class="source">Sci. Rep</div>., <div class="volume">4</div>, <div class="fpage">6420</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20genotyping%20method%20for%20genome-modified%20animals%20and%20human%20cells%20generated%20with%20CRISPR%2FCas9%20system&amp;author=X.%20Zhu&amp;author=Y.%20Xu&amp;author=S.%20Yu&amp;publication_year=2014&amp;journal=Sci.%20Rep&amp;volume=4&amp;pages=6420" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep06420" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep06420" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep06420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25236476" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Steentoft</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hauge</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Fast and sensitive detection of indels induced by precise gene targeting</div>, <div class="source">Nucleic Acids Res</div>., <div class="volume">43</div>, <div class="fpage">e59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20sensitive%20detection%20of%20indels%20induced%20by%20precise%20gene%20targeting&amp;author=Z.%20Yang&amp;author=C.%20Steentoft&amp;author=C.%20Hauge&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=e59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25753669" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramlee</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cheung</div> <div class="given-names">A. M. S.</div></div>, <div class="name string-name"><div class="surname">Chuah</div> <div class="given-names">C. T. H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">High-throughput genotyping of CRISPR/Cas9-mediated mutants using fluorescent PCR-capillary gel electrophoresis</div>, <div class="source">Sci. Rep</div>., <div class="volume">5</div>, <div class="fpage">15587</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20of%20CRISPR%2FCas9-mediated%20mutants%20using%20fluorescent%20PCR-capillary%20gel%20electrophoresis&amp;author=M.%20K.%20Ramlee&amp;author=T.%20Yan&amp;author=A.%20M.%20S.%20Cheung&amp;author=C.%20T.%20H.%20Chuah&amp;author=S.%20Li&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=15587" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep15587" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep15587" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep15587"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26498861" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dahlem</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Hoshijima</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jurynec</div> <div class="given-names">M. J.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome</div>, <div class="source">PLoS Genet</div>., <div class="volume">8</div>, <div class="fpage">e1002861</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20generating%20and%20detecting%20locus-specific%20mutations%20induced%20with%20TALENs%20in%20the%20zebrafish%20genome&amp;author=T.%20J.%20Dahlem&amp;author=K.%20Hoshijima&amp;author=M.%20J.%20Jurynec&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22916025" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">Y.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A PCR based protocol for detecting indel mutations induced by TALENs and CRISPR/Cas9 in zebrafish</div>, <div class="source">PLoS One</div>, <div class="volume">9</div>, <div class="fpage">e98282</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20PCR%20based%20protocol%20for%20detecting%20indel%20mutations%20induced%20by%20TALENs%20and%20CRISPR%2FCas9%20in%20zebrafish&amp;author=C.%20Yu&amp;author=Y.%20Zhang&amp;author=S.%20Yao&amp;author=Y.%20Wei&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e98282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0098282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0098282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0098282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24901507" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mock</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Machowicz</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hauber</div> <div class="given-names">I.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">mRNA transfection of a novel TAL effector nuclease (TALEN) facilitates efficient knockout of HIV co-receptor CCR5</div>, <div class="source">Nucleic Acids Res</div>., <div class="volume">43</div>, <div class="fpage">5560</div><div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNA%20transfection%20of%20a%20novel%20TAL%20effector%20nuclease%20(TALEN)%20facilitates%20efficient%20knockout%20of%20HIV%20co-receptor%20CCR5&amp;author=U.%20Mock&amp;author=R.%20Machowicz&amp;author=I.%20Hauber&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=5560-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25964300" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miyaoka</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">A. H.</div></div>, <div class="name string-name"><div class="surname">Judge</div> <div class="given-names">L. M.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Isolation of single-base genome-edited human iPS cells without antibiotic selection</div>, <div class="source">Nat. Methods</div>, <div class="volume">11</div>, <div class="fpage">291</div><div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20single-base%20genome-edited%20human%20iPS%20cells%20without%20antibiotic%20selection&amp;author=Y.%20Miyaoka&amp;author=A.%20H.%20Chan&amp;author=L.%20M.%20Judge&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=291-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24509632" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Findlay</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Vincent</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Berman</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Postovit</div> <div class="given-names">L. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A digital PCR-based method for efficient and highly specific screening of genome edited cells</div>, <div class="source">PLoS One</div>, <div class="volume">11</div>, <div class="fpage">e0153901</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20digital%20PCR-based%20method%20for%20efficient%20and%20highly%20specific%20screening%20of%20genome%20edited%20cells&amp;author=S.%20D.%20Findlay&amp;author=K.%20M.%20Vincent&amp;author=J.%20R.%20Berman&amp;author=L.%20M.%20Postovit&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0153901" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0153901" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0153901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0153901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27089539" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hua</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">A simple and efficient method for CRISPR/Cas9-induced mutant screening</div>, <div class="source">J. Genet. Genomics</div>, <div class="volume">44</div>, <div class="fpage">207</div><div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20and%20efficient%20method%20for%20CRISPR%2FCas9-induced%20mutant%20screening&amp;author=Y.%20Hua&amp;author=C.%20Wang&amp;author=J.%20Huang&amp;author=K.%20Wang&amp;publication_year=2017&amp;journal=J.%20Genet.%20Genomics&amp;volume=44&amp;pages=207-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jgg.2017.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jgg.2017.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jgg.2017.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28416245" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harayama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Riezman</div> <div class="given-names">H.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Detection of genome-edited mutant clones by a simple competition-based PCR method</div>, <div class="source">PLoS One</div>, <div class="volume">12</div>, <div class="fpage">e0179165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20genome-edited%20mutant%20clones%20by%20a%20simple%20competition-based%20PCR%20method&amp;author=T.%20Harayama&amp;author=H.%20Riezman&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0179165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0179165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0179165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0179165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28586390" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tanigawa</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Oligoribonucleotide (ORN) interference-PCR (ORNi-PCR): a simple method for suppressing PCR amplification of specific DNA sequences using ORNs</div>, <div class="source">PLoS One</div>, <div class="volume">9</div>, <div class="fpage">e113345</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligoribonucleotide%20(ORN)%20interference-PCR%20(ORNi-PCR)%3A%20a%20simple%20method%20for%20suppressing%20PCR%20amplification%20of%20specific%20DNA%20sequences%20using%20ORNs&amp;author=N.%20Tanigawa&amp;author=T.%20Fujita&amp;author=H.%20Fujii&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e113345" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0113345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0113345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0113345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25405983" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mali</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Esvelt</div> <div class="given-names">K. M.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">RNA-guided human genome engineering via Cas9</div>, <div class="source">Science</div>, <div class="volume">339</div>, <div class="fpage">823</div><div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-guided%20human%20genome%20engineering%20via%20Cas9&amp;author=P.%20Mali&amp;author=L.%20Yang&amp;author=K.%20M.%20Esvelt&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=823-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1232033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1232033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1232033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yuno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Allele-specific locus binding and genome editing by CRISPR at the p16INK4a locus</div>, <div class="source">Sci. Rep</div>., <div class="volume">6</div>, <div class="fpage">30485</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allele-specific%20locus%20binding%20and%20genome%20editing%20by%20CRISPR%20at%20the%20p16INK4a%20locus&amp;author=T.%20Fujita&amp;author=M.%20Yuno&amp;author=H.%20Fujii&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=30485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep30485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep30485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep30485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27465215" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jinek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chylinski</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fonfara</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hauer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Doudna</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Charpentier</div> <div class="given-names">E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A programmable dual-RNA- guided DNA endonuclease in adaptive bacterial immunity</div>, <div class="source">Science</div>, <div class="volume">337</div>, <div class="fpage">816</div><div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20programmable%20dual-RNA-%20guided%20DNA%20endonuclease%20in%20adaptive%20bacterial%20immunity&amp;author=M.%20Jinek&amp;author=K.%20Chylinski&amp;author=I.%20Fonfara&amp;author=M.%20Hauer&amp;author=J.%20A.%20Doudna&amp;author=E.%20Charpentier&amp;publication_year=2012&amp;journal=Science&amp;volume=337&amp;pages=816-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1225829" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1225829" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1225829"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745249" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nishioka</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kakihara</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">1997</div>, 
<div class="article-title">Characterization of DNA polymerase from <em>Pyrococcus</em> sp. strain KOD1 and its application to PCR</div>, <div class="source">Appl. Environ. Microbiol</div>., <div class="volume">63</div>, <div class="fpage">4504</div><div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20DNA%20polymerase%20from%20Pyrococcus%20sp.%20strain%20KOD1%20and%20its%20application%20to%20PCR&amp;author=M.%20Takagi&amp;author=M.%20Nishioka&amp;author=H.%20Kakihara&amp;publication_year=1997&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=63&amp;pages=4504-10" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9361436" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J. S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins</div>, <div class="source">Genome Res</div>., <div class="volume">24</div>, <div class="fpage">1012</div><div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20efficient%20RNA-guided%20genome%20editing%20in%20human%20cells%20via%20delivery%20of%20purified%20Cas9%20ribonucleoproteins&amp;author=S.%20Kim&amp;author=D.%20Kim&amp;author=S.%20W.%20Cho&amp;author=J.%20Kim&amp;author=J.%20S.%20Kim&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1012-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.171322.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.171322.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.171322.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24696461" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zuris</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">D. B.</div></div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo</div>, <div class="source">Nat. Biotechnol</div>., <div class="volume">33</div>, <div class="fpage">73</div><div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cationic%20lipid-mediated%20delivery%20of%20proteins%20enables%20efficient%20protein-based%20genome%20editing%20in%20vitro%20and%20in%20vivo&amp;author=J.%20A.%20Zuris&amp;author=D.%20B.%20Thompson&amp;author=Y.%20Shu&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=73-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3081"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25357182" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kitaura</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Yuno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">H.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Locus-specific ChIP combined with NGS analysis reveals genomic regulatory regions that physically interact with the Pax5 promoter in a chicken B cell line</div>, <div class="source">DNA Res</div>., <div class="volume">24</div>, <div class="fpage">537</div><div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locus-specific%20ChIP%20combined%20with%20NGS%20analysis%20reveals%20genomic%20regulatory%20regions%20that%20physically%20interact%20with%20the%20Pax5%20promoter%20in%20a%20chicken%20B%20cell%20line&amp;author=T.%20Fujita&amp;author=F.%20Kitaura&amp;author=M.%20Yuno&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;author=H.%20Fujii&amp;publication_year=2017&amp;journal=DNA%20Res&amp;volume=24&amp;pages=537-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsx023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsx023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsx023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28586432" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okamoto</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Demetrick</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Spillare</div> <div class="given-names">E. A.</div></div></div>, et al.  <div class="year">1994</div>, 
<div class="article-title">Mutations and altered expression of p16INK4 in human cancer</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div>., <div class="volume">91</div>, <div class="fpage">11045</div><div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20and%20altered%20expression%20of%20p16INK4%20in%20human%20cancer&amp;author=A.%20Okamoto&amp;author=D.%20J.%20Demetrick&amp;author=E.%20A.%20Spillare&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=11045-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.23.11045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.23.11045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.23.11045"> </span></div></div></div></div></div></div><div content-id="dsy012-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Piganeau</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ghezraoui</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">De Cian</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Cancer translocations in human cells induced by zinc finger and TALE nucleases</div>, <div class="source">Genome Res</div>., <div class="volume">23</div>, <div class="fpage">1182</div><div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%20translocations%20in%20human%20cells%20induced%20by%20zinc%20finger%20and%20TALE%20nucleases&amp;author=M.%20Piganeau&amp;author=H.%20Ghezraoui&amp;author=A.%20De%20Cian&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1182-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.147314.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.147314.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.147314.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23568838" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Esteller</div> <div class="given-names">M.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">CpG island hypermethylation and tumor suppressor genes: a booming present, a brighter future</div>, <div class="source">Oncogene</div>, <div class="volume">21</div>, <div class="fpage">5427</div><div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20island%20hypermethylation%20and%20tumor%20suppressor%20genes%3A%20a%20booming%20present%2C%20a%20brighter%20future&amp;author=M.%20Esteller&amp;publication_year=2002&amp;journal=Oncogene&amp;volume=21&amp;pages=5427-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1205600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1205600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1205600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12154405" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peters</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The role of genomic imprinting in biology and disease: an expanding view</div>, <div class="source">Nat. Rev. Genet</div>., <div class="volume">15</div>, <div class="fpage">517</div><div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20genomic%20imprinting%20in%20biology%20and%20disease%3A%20an%20expanding%20view&amp;author=J.%20Peters&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=517-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3766"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24958438" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chalign</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Heard</div> <div class="given-names">E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">X-chromosome inactivation in development and cancer</div>, <div class="source">FEBS Lett</div>., <div class="volume">588</div>, <div class="fpage">2514</div><div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=X-chromosome%20inactivation%20in%20development%20and%20cancer&amp;author=R.%20Chalign%C3%A9&amp;author=E.%20Heard&amp;publication_year=2014&amp;journal=FEBS%20Lett&amp;volume=588&amp;pages=2514-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2014.06.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2014.06.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2014.06.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24937141" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="dsy012-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy012-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kibbe</div> <div class="given-names">W. A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">OligoCalc: an online oligonucleotide properties calculator</div>, <div class="source">Nucleic Acids Res</div>., <div class="volume">35</div>, <div class="fpage">W43</div><div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OligoCalc%3A%20an%20online%20oligonucleotide%20properties%20calculator&amp;author=W.%20A.%20Kibbe&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W43-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452344" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4989276&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_figures_s1-s14.pdf?Expires=2147483647&amp;Signature=VNuDzHQR1O91Tw0wV-SQKOEZgPoTFPLKPcEzB09EVDC7nlb25dfv6U74i1YLQZ5H0zAS0ftojyYp8U0IFEJhhaiYWp8XBlMovANTJc3t-2cogeGVYb~vOq-exZ6701p932IC7x3CpQzGOJnFPawaJTi-Q3DBxU~CH865gOfk5H42ZHFRBZrxLIjoKZOOgIvrzBAumLsLDnrDs27qLsxc0hDXYYFxvl~evZb2D~JhujgmVkyfq72MT-FKoZ6m7fVQ5EEmQtvOcMeKiJz6wEsegRWYfDp0vIuZDLE5275P9qvm8iazGyycRlm9c3cSK7od~SdxjHoTCZdVVwNthSX4vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures S1-S14</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_table_s1.xlsx?Expires=2147483647&amp;Signature=xbV7fxPtlQnbXi~~MuqHf2UVvhcYwDVrUqVbNn82jvh8b8cfVlKJnMvsaIeYMBt2pQMJYzWi-AMnOZDrnvKnhWVFSsopv0RRc7lfK~iaQaX4U17BbJxA338xw88mkbBBOI6akuH5nohXhzPlm08~eKNQ1XN26JLjdumJfT-xXiaAbfgWC5FjIAHLs4e0PG6RgLm9x1DN7V~X2BqFFxYTDckeQ09Tf071MSDmjwxeNkMT6iWy76V9aT4O2X8KqjV6EVM-UpcncSBwqVJRzApAfDCdQv0HzHitSJiqY9hOi5QjY8nIh6GJuJbrBkBMClWapuUqUVMtqG13aaD-Ud9vCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_table_s2.xlsx?Expires=2147483647&amp;Signature=lXoJ-eMt-9KILoaohq1ZmMCHkYm-~LDxKyvRvb-I-wHnWKTvct9qADQJhrAXNYNMrP2FpxDWhITSiGrsC~-PJ8aVoazHeOsEWlHTCtXR9fhkzowNvgzFUC7jCxbjje~7ZXxw1~4ELojHoPreONUO3maAStqqqO~NMzc1EnyBN1us9VPR5FP9jmyvdcdubz5HkadvoePJvYSV5lI3EUxUeiS4nHtju1JV5c1RfDfbjSzvO3Oces46GN2934fBK8gJ3v7h~q3AqcQVAOhHD2oYdVXDjKOyPOXP8mtQcHjmwVkzFh569Rh5rAVkG-HcihEfyZ6hF1wlI7V0TCZw61CHTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy012/1/dsy012_supplementary_text.doc?Expires=2147483647&amp;Signature=Vn5ZPCXQ013-zu6onsddkhyqf3nKGu-9IvSAMDDvcpLGqnORmLJRWAtT8X9k~Lw50TvHsT4MoyjJZckgdw5P7YY6Y3cD4LlBZ8PHoH4L~yzILIrnEHeCGdC8H3fDwXvnoxdeOPMLu448kwuITqRlE6J~m9CzrnimJkiJlhZN9FpMwBmkdEnPu8ygCJv8Zn~cCt~d6-f8XS4XuJvKZm90I3mTEVrtgeCPqsRsxLoQme8AEyooqa5l~THxixbLLuvUAcoF9s30fCBlQAllGGD7v0OFJDtdz3ucxbTs-7MgTFiq5Blq-ola1uFzIqhORZzoOm5qiJ-0KwH7rbFu4OEL2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Text</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,159</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4989276"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f395%2f4989276">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F395%2F4989276" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4989276'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Detection of genome-edited cells by oligoribonucleotide interference-PCR" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f395%2f4989276">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

