

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/2/161/332908","@type":"ScholarlyArticle","name":"Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder ( Paralichthys olivaceus ): applications to QTL mapping of  Vibrio anguillarum  disease resistance and comparative genomic analysis","datePublished":"2015-03-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2015-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv001","keywords":["Japanese flounder","RAD-seq-based SNP","high-resolution linkage map","QTL mapping","genome synteny"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/2/161/332908","author":[{"name":"Shao, Changwei","affiliation":"1  Ministry of Agriculture ,  Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries ,  Qingdao 266071 ,  China    2  Function Laboratory for Marine Fisheries Science and Food Production Processes ,  National Lab for Ocean Science and Technology ,  Qingdao 266071 ,  China    3  Faculty of Marine Science ,  Tokyo University of Marine Science and Technology ,  Minato, Tokyo 108-8477 ,  Japan","@type":"Person"},{"name":"Niu, Yongchao","affiliation":"4  BGI-Shenzhen ,  Shenzhen 518000 ,  China","@type":"Person"},{"name":"Rastas, Pasi","affiliation":"5  Department of Biosciences, Metapopulation Research Group ,  University of Helsinki ,  Helsinki FI-00014 ,  Finland","@type":"Person"},{"name":"Liu, Yang","affiliation":"1  Ministry of Agriculture ,  Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries ,  Qingdao 266071 ,  China    2  Function Laboratory for Marine Fisheries Science and Food Production Processes ,  National Lab for Ocean Science and Technology ,  Qingdao 266071 ,  China","@type":"Person"},{"name":"Xie, Zhiyuan","affiliation":"4  BGI-Shenzhen ,  Shenzhen 518000 ,  China","@type":"Person"},{"name":"Li, Hengde","affiliation":"6  Chinese Academy of Fisheries Science ,  Beijing 100039 ,  China","@type":"Person"},{"name":"Wang, Lei","affiliation":"1  Ministry of Agriculture ,  Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries ,  Qingdao 266071 ,  China    2  Function Laboratory for Marine Fisheries Science and Food Production Processes ,  National Lab for Ocean Science and Technology ,  Qingdao 266071 ,  China","@type":"Person"},{"name":"Jiang, Yong","affiliation":"7  National Oceanographic Center ,  Qingdao 266071 ,  China","@type":"Person"},{"name":"Tai, Shuaishuai","affiliation":"4  BGI-Shenzhen ,  Shenzhen 518000 ,  China","@type":"Person"},{"name":"Tian, Yongsheng","affiliation":"1  Ministry of Agriculture ,  Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries ,  Qingdao 266071 ,  China    2  Function Laboratory for Marine Fisheries Science and Food Production Processes ,  National Lab for Ocean Science and Technology ,  Qingdao 266071 ,  China","@type":"Person"},{"name":"Sakamoto, Takashi","affiliation":"3  Faculty of Marine Science ,  Tokyo University of Marine Science and Technology ,  Minato, Tokyo 108-8477 ,  Japan","@type":"Person"},{"name":"Chen, Songlin","affiliation":"1  Ministry of Agriculture ,  Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries ,  Qingdao 266071 ,  China    2  Function Laboratory for Marine Fisheries Science and Food Production Processes ,  National Lab for Ocean Science and Technology ,  Qingdao 266071 ,  China","@type":"Person"}],"description":"Abstract.  High-resolution genetic maps are essential for fine mapping of complex traits, genome assembly, and comparative genomic analysis. Single-nucleotide p","pageStart":"161","pageEnd":"170","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder ( Paralichthys olivaceus ): applications to QTL mapping of  Vibrio anguillarum  disease resistance and comparative genomic analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder ( Paralichthys olivaceus ): applications to QTL mapping of  Vibrio anguillarum  disease resistance and comparative genomic analysis" />
    <meta property="og:description" content="Abstract.  High-resolution genetic maps are essential for fine mapping of complex traits, genome assembly, and comparative genomic analysis. Single-nucleotide p" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/2/161/332908" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-03-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shao, Changwei" /><meta name="citation_author_institution" content="Ministry of Agriculture, Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries, Qingdao 266071, China" /><meta name="citation_author_institution" content="Function Laboratory for Marine Fisheries Science and Food Production Processes, National Lab for Ocean Science and Technology, Qingdao 266071, China" /><meta name="citation_author_institution" content="Faculty of Marine Science, Tokyo University of Marine Science and Technology, Minato, Tokyo 108-8477, Japan" /><meta name="citation_author" content="Niu, Yongchao" /><meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen 518000, China" /><meta name="citation_author" content="Rastas, Pasi" /><meta name="citation_author_institution" content="Department of Biosciences, Metapopulation Research Group, University of Helsinki, Helsinki FI-00014, Finland" /><meta name="citation_author" content="Liu, Yang" /><meta name="citation_author_institution" content="Ministry of Agriculture, Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries, Qingdao 266071, China" /><meta name="citation_author_institution" content="Function Laboratory for Marine Fisheries Science and Food Production Processes, National Lab for Ocean Science and Technology, Qingdao 266071, China" /><meta name="citation_author" content="Xie, Zhiyuan" /><meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen 518000, China" /><meta name="citation_author" content="Li, Hengde" /><meta name="citation_author_institution" content="Chinese Academy of Fisheries Science, Beijing 100039, China" /><meta name="citation_author" content="Wang, Lei" /><meta name="citation_author_institution" content="Ministry of Agriculture, Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries, Qingdao 266071, China" /><meta name="citation_author_institution" content="Function Laboratory for Marine Fisheries Science and Food Production Processes, National Lab for Ocean Science and Technology, Qingdao 266071, China" /><meta name="citation_author" content="Jiang, Yong" /><meta name="citation_author_institution" content="National Oceanographic Center, Qingdao 266071, China" /><meta name="citation_author" content="Tai, Shuaishuai" /><meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen 518000, China" /><meta name="citation_author" content="Tian, Yongsheng" /><meta name="citation_author_institution" content="Ministry of Agriculture, Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries, Qingdao 266071, China" /><meta name="citation_author_institution" content="Function Laboratory for Marine Fisheries Science and Food Production Processes, National Lab for Ocean Science and Technology, Qingdao 266071, China" /><meta name="citation_author" content="Sakamoto, Takashi" /><meta name="citation_author_institution" content="Faculty of Marine Science, Tokyo University of Marine Science and Technology, Minato, Tokyo 108-8477, Japan" /><meta name="citation_author" content="Chen, Songlin" /><meta name="citation_author_institution" content="Ministry of Agriculture, Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries, Qingdao 266071, China" /><meta name="citation_author_institution" content="Function Laboratory for Marine Fisheries Science and Food Production Processes, National Lab for Ocean Science and Technology, Qingdao 266071, China" /><meta name="citation_title" content="Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis" /><meta name="citation_firstpage" content="161" /><meta name="citation_lastpage" content="170" /><meta name="citation_doi" content="10.1093/dnares/dsv001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2015/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Characterization of Japanese flounder karyotype by chromosome bandings and fluorescence in situ hybridization with DNA markers; citation_author=Fujiwara  A.; citation_author=Fujiwara  M.; citation_author=Nishida-Umehara  C.; citation_author=Abe  S.; citation_author=Masaoka  T.;  citation_journal_title=Genetica;  citation_year=2007;  citation_volume=131;  citation_pages=267-74; " /><meta name="citation_reference" content="citation_title=Fish production estimates and trends 2012–2013; citation_author=Nystoyl  R.; citation_author=Tveteras  R.;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Flounder culture and its challenges in Asia; citation_author=Seikai  T.;  citation_journal_title=Rev. Fish. Sci.;  citation_year=2002;  citation_volume=3–4;  citation_pages=421-32; " /><meta name="citation_reference" content="citation_title=A genomic scan for selection reveals candidates for genes involved in the evolution of cultivated sunflower (Helianthus annuus); citation_author=Chapman  M.A.; citation_author=Pashley  C.H.; citation_author=Wenzler  J.;  citation_journal_title=Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=2931-45; " /><meta name="citation_reference" content="citation_title=Genetic map construction and quantitative trait locus (QTL) detection of growth-related traits in Litopenaeus vannamei for selective breeding applications; citation_author=Andriantahina  F.; citation_author=Liu  X.; citation_author=Huang  H.;  citation_journal_title=PloS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e75206" /><meta name="citation_reference" content="citation_title=Recent advances of genome mapping and marker-assisted selection in aquaculture; citation_author=Yue  G.H.;  citation_journal_title=Fish Fish.;  citation_year=2013;  citation_volume=15;  citation_pages=376-96; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the Japanese flounder, Paralichthys olivaceus; citation_author=Coimbra  M.R.; citation_author=Kobayashi  K.; citation_author=Koretsugu  S.;  citation_journal_title=Aquaculture;  citation_year=2003;  citation_volume=1–4;  citation_pages=203-18; " /><meta name="citation_reference" content="citation_title=Genetic linkage map of olive flounder, Paralichthys olivaceus; citation_author=Kang  J.H.; citation_author=Kim  W.J.; citation_author=Lee  W.J.;  citation_journal_title=Int. J. Biol. Sci.;  citation_year=2008;  citation_volume=4;  citation_pages=143-9; " /><meta name="citation_reference" content="citation_title=A second generation genetic linkage map of Japanese flounder (Paralichthys olivaceus); citation_author=Castano-Sanchez  C.; citation_author=Fuji  K.; citation_author=Ozaki  A.;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=554" /><meta name="citation_reference" content="citation_title=Construction of high-density genetic linkage maps and mapping of growth-related quantitative trail loci in the Japanese flounder (Paralichthys olivaceus); citation_author=Song  W.; citation_author=Pang  R.; citation_author=Niu  Y.;  citation_journal_title=Plos ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e50404" /><meta name="citation_reference" content="citation_title=Identification of a single major genetic locus controlling the resistance to lymphocystis disease in Japanese flounder (Paralichthys olivaceus); citation_author=Fuji  K.; citation_author=Kobayashi  K.; citation_author=Hasegawa  O.;  citation_journal_title=Aquaculture;  citation_year=2006;  citation_volume=1–4;  citation_pages=203-10; " /><meta name="citation_reference" content="citation_title=Linkage mapping of toll-like receptors (TLRs) in Japanese flounder, Paralichthys olivaceus; citation_author=Hwang  S.D.; citation_author=Fuji  K.; citation_author=Takano  T.;  citation_journal_title=Mar. Biotechnol.;  citation_year=2011;  citation_volume=13;  citation_pages=1086-91; " /><meta name="citation_reference" content="citation_title=Marker-assisted breeding of a lymphocystis disease-resistant Japanese flounder (Pralichthys olivaceus); citation_author=Fuji  K.H.O.; citation_author=Honda  K.; citation_author=Kumasaka  K.; citation_author=Sakamoto  T.; citation_author=Okamoto  N.;  citation_journal_title=Aquaculture;  citation_year=2007;  citation_volume=1–4;  citation_pages=291-5; " /><meta name="citation_reference" content="citation_title=Comparison of C. elegans and C. briggsae genome sequences reveals extensive conservation of chromosome organization and synteny; citation_author=Hillier  L.W.; citation_author=Miller  R.D.; citation_author=Baird  S.E.;  citation_journal_title=Plos Biol.;  citation_year=2007;  citation_volume=5;  citation_pages=e167" /><meta name="citation_reference" content="citation_title=A gene-based linkage map for Bicyclus anynana butterflies allows for a comprehensive analysis of synteny with the lepidopteran reference genome; citation_author=Beldade  P.; citation_author=Saenko  S.V.; citation_author=Pul  N.; citation_author=Long  A.D.;  citation_journal_title=PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000366" /><meta name="citation_reference" content="citation_title=Review of massively parallel DNA sequencing technologies; citation_author=Moorthie  S.; citation_author=Mattocks  C.J.; citation_author=Wright  C.F.;  citation_journal_title=HUGO J;  citation_year=2011;  citation_volume=5;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Reliability of high-throughput genotyping of whole genome amplified DNA in SNP genotyping studies; citation_author=Berthier-Schaad  Y.; citation_author=Kao  W.H.; citation_author=Coresh  J.;  citation_journal_title=Electrophoresis;  citation_year=2007;  citation_volume=28;  citation_pages=2812-7; " /><meta name="citation_reference" content="citation_title=RAD in the realm of next-generation sequencing technologies; citation_author=Rowe  H.C.; citation_author=Renaut  S.; citation_author=Guggisberg  A.;  citation_journal_title=Mol. Ecol.;  citation_year=2011;  citation_volume=20;  citation_pages=3499-502; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers; citation_author=Baird  N.A.; citation_author=Etter  P.D.; citation_author=Atwood  T.S.;  citation_journal_title=PloS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=RAD sequencing yields a high success rate for westslope cutthroat and rainbow trout species-diagnostic SNP assays; citation_author=Amish  S.J.; citation_author=Hohenlohe  P.A.; citation_author=Painter  S.;  citation_journal_title=Mol. Ecol. Resour.;  citation_year=2012;  citation_volume=12;  citation_pages=653-60; " /><meta name="citation_reference" content="citation_title=A RAD tag derived marker based eggplant linkage map and the location of QTLs determining anthocyanin pigmentation; citation_author=Barchi  L.; citation_author=Lanteri  S.; citation_author=Portis  E.;  citation_journal_title=PloS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e43740" /><meta name="citation_reference" content="citation_title=Construction and application for QTL analysis of a Restriction Site Associated DNA (RAD) linkage map in barley; citation_author=Chutimanitsakun  Y.; citation_author=Nipper  R.W.; citation_author=Cuesta-Marcos  A.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=4" /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (Cicer arietinum L.); citation_author=Gaur  R.; citation_author=Azam  S.; citation_author=Jeena  G.;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=357-73; " /><meta name="citation_reference" content="citation_title=Paired-end RAD-seq for de novo assembly and marker design without available reference; citation_author=Willing  E.M.; citation_author=Hoffmann  M.; citation_author=Klein  J.D.; citation_author=Weigel  D.; citation_author=Dreyer  C.;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2187-93; " /><meta name="citation_reference" content="citation_title=Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in Lupinus angustifolius L; citation_author=Yang  H.; citation_author=Tao  Y.; citation_author=Zheng  Z.; citation_author=Li  C.; citation_author=Sweetingham  M.W.; citation_author=Howieson  J.G.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=318" /><meta name="citation_reference" content="citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags; citation_author=Hohenlohe  P.A.; citation_author=Bassham  S.; citation_author=Etter  P.D.; citation_author=Stiffler  N.; citation_author=Johnson  E.A.; citation_author=Cresko  W.A.;  citation_journal_title=PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000862" /><meta name="citation_reference" content="citation_title=Analysis of growth performance and breeding value of ‘Ping You No.1’ Japanese flounder and selection of parents; citation_author=Liu  F.; citation_author=Chen  S.L.; citation_author=Wang  L.;  citation_journal_title=J. Fish. Sci. China;  citation_year=2013;  citation_volume=3;  citation_pages=521-7; " /><meta name="citation_reference" content="citation_title=MHC polymorphism and disease resistance to Vibrio anguillarum in 12 selective Japanese flounder (Paralichthys olivaceus) families; citation_author=Xu  T.J.; citation_author=Chen  S.L.; citation_author=Ji  X.S.; citation_author=Tian  Y.S.;  citation_journal_title=Fish Shellfish Immun.;  citation_year=2008;  citation_volume=25;  citation_pages=213-21; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping Loci de novo from short-read sequences; citation_author=Catchen  J.M.; citation_author=Amores  A.; citation_author=Hohenlohe  P.; citation_author=Cresko  W.; citation_author=Postlethwait  J.H.;  citation_journal_title=G3-Genes Genom.Genet.;  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=Stacks: an analysis tool set for population genomics; citation_author=Catchen  J.; citation_author=Hohenlohe  P.A.; citation_author=Bassham  S.; citation_author=Amores  A.; citation_author=Cresko  W.A.;  citation_journal_title=Mol. Ecol.;  citation_year=2013;  citation_volume=22;  citation_pages=3124-40; " /><meta name="citation_reference" content="citation_title=Lep-MAP: fast and accurate linkage map construction for large SNP datasets; citation_author=Rastas  P.; citation_author=Paulin  L.; citation_author=Hanski  I.; citation_author=Lehtonen  R.; citation_author=Auvinen  P.;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=3128-34; " /><meta name="citation_reference" content="citation_title=Composite interval mapping and multiple interval mapping: procedures and guidelines for using Windows QTL Cartographer; citation_author=Silva Lda  C.; citation_author=Wang  S.; citation_author=Zeng  Z.B.;  citation_journal_title=Methods Mol. Biol.;  citation_year=2012;  citation_volume=871;  citation_pages=75-119; " /><meta name="citation_reference" content="citation_title=Assembly and function of the MHC I peptide-loading complex are conserved across higher vertebrates; citation_author=Hinz  A.; citation_author=Jedamzick  J.; citation_author=Herbring  V.;  citation_journal_title=J. Biol. Chem.;  citation_year=2014;  citation_volume=289;  citation_pages=33109-17; " /><meta name="citation_reference" content="citation_title=Heat shock protein 60 activates B cells via the TLR4-MyD88 pathway; citation_author=Cohen-Sfady  M.; citation_author=Nussbaum  G.; citation_author=Pevsner-Fischer  M.;  citation_journal_title=J. Immunol.;  citation_year=2005;  citation_volume=175;  citation_pages=3594-602; " /><meta name="citation_reference" content="citation_title=DNA transposons and the role of recombination in mutation accumulation in Daphnia pulex; citation_author=Schaack  S.; citation_author=Choi  E.; citation_author=Lynch  M.; citation_author=Pritham  E.J.;  citation_journal_title=Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R46" /><meta name="citation_reference" content="citation_title=The evolution of restricted recombination in sex chromosomes; citation_author=Bergero  R.; citation_author=Charlesworth  D.;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2009;  citation_volume=24;  citation_pages=94-102; " /><meta name="citation_reference" content="citation_title=Progressive recombination suppression and differentiation in recently evolved neo-sex chromosomes; citation_author=Natri  H.M.; citation_author=Shikano  T.; citation_author=Merila  J.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=1131-44; " /><meta name="citation_reference" content="citation_title=Joinmap 4: software for the calculation of genetic linkage maps in experimental populations; citation_author=van Ooijen  J.W.;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The Glanville fritillary genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera; citation_author=Ahola  V.; citation_author=Lehtonen  R.; citation_author=Somervuo  P.;  citation_journal_title=Nature Commun.;  citation_year=2006;  citation_volume=5;  citation_pages=4737" /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species; citation_author=Elshire  R.J.; citation_author=Glaubitz  J.C.; citation_author=Sun  Q.;  citation_journal_title=PloS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=A genome scan for quantitative trait loci associated with Vibrio anguillarum infection resistance in Japanese flounder (Paralichthys olivaceus) by bulked segregant analysis; citation_author=Wang  L.; citation_author=Fan  C.; citation_author=Liu  Y.;  citation_journal_title=Mar. Biotechnol.;  citation_year=2014;  citation_volume=16;  citation_pages=513-21; " /><meta name="citation_reference" content="citation_title=QTL analysis for plant height and leaf angle by using different populations of maize; citation_author=Yu  Y.T.; citation_author=Zhang  J.M.; citation_author=Shi  Y.S.; citation_author=Song  Y.C.; citation_author=Wang  T.Y.; citation_author=Li  Y.;  citation_journal_title=J. Maize Sci.;  citation_year=2006;  citation_volume=14;  citation_pages=88-92; " /><meta name="citation_reference" content="citation_title=Detection of QTL with effects on osmoregulation capacities in the rainbow trout (Oncorhynchus mykiss); citation_author=Le Bras  Y.; citation_author=Dechamp  N.; citation_author=Krieg  F.;  citation_journal_title=BMC Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=46" /><meta name="citation_reference" content="citation_title=The adaptable major histocompatibility complex (MHC) fold: structure and function of nonclassical and MHC class I-like molecules; citation_author=Adams  E.J.; citation_author=Luoma  A.M.;  citation_journal_title=Annu. Rev. Immunol.;  citation_year=2013;  citation_volume=31;  citation_pages=529-61; " /><meta name="citation_reference" content="citation_title=MHC class IIalpha gene polymorphism and its association with resistance/susceptibility to Vibrio anguillarum in Japanese flounder (Paralichthys olivaceus); citation_author=Xu  T.J.; citation_author=Chen  S.L.; citation_author=Zhang  Y.X.;  citation_journal_title=Dev. Comp. Immunol.;  citation_year=2010;  citation_volume=34;  citation_pages=1042-50; " /><meta name="citation_reference" content="citation_title=Allelic variation, balancing selection and positive selected sites detected from MHC class Ialpha gene of olive flounder; citation_author=Xu  T.J.; citation_author=Sun  Y.N.; citation_author=Chen  S.L.;  citation_journal_title=Genetica;  citation_year=2010;  citation_volume=138;  citation_pages=1251-9; " /><meta name="citation_reference" content="citation_title=Polymorphism and balancing selection of MHC class II DAB gene in 7 selective flounder (Paralichthys olivaceus) families; citation_author=Du  M.; citation_author=Chen  S.L.; citation_author=Liang  Y.;  citation_journal_title=Evid. Based Complement. Alternat. Med.;  citation_year=2011;  citation_volume=20,11;  citation_pages=613629" /><meta name="citation_reference" content="citation_title=Next generation sequencing technology: advances and applications; citation_author=Buermans  H.P.; citation_author=den Dunnen  J.T.;  citation_journal_title=Biochim. Biophys. Acta.;  citation_year=2014;  citation_volume=10;  citation_pages=1932-41; " /><meta name="citation_reference" content="citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions; citation_author=Treangen  T.J.; citation_author=Salzberg  S.L.;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=36-46; " /><meta name="citation_reference" content="citation_title=Whole genome comparative analysis of channel catfish (Ictalurus punctatus) with four model fish species; citation_author=Jiang  Y.; citation_author=Gao  X.; citation_author=Liu  S.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=780" /><meta name="citation_reference" content="citation_title=Differential expression of anti-Mullerian hormone (amh) and anti-Mullerian hormone receptor type II (amhrII) in the teleost medaka; citation_author=Kluver  N.; citation_author=Pfennig  F.; citation_author=Pala  I.;  citation_journal_title=Dev. Dynam.;  citation_year=2007;  citation_volume=236;  citation_pages=271-81; " /><meta name="citation_reference" content="citation_title=Neighboring genes show interchromosomal colocalization after their separation; citation_author=Dai  Z.; citation_author=Xiong  Y.; citation_author=Dai  X.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2014;  citation_volume=31;  citation_pages=1166-72; " /><meta name="citation_reference" content="citation_title=Comparative genomic analysis of catfish linkage group 8 reveals two homologous chromosomes in zebrafish and other teleosts with extensive inter-chromosomal rearrangements; citation_author=Zhang  Y.; citation_author=Liu  S.; citation_author=Lu  J.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=387" /><meta name="citation_reference" content="citation_title=The zebrafish gene map defines ancestral vertebrate chromosomes; citation_author=Woods  I.G.; citation_author=Wilson  C.; citation_author=Friedlander  B.;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1307-14; " /><meta name="citation_reference" content="citation_title=Zebrafish comparative genomics and the origins of vertebrate chromosomes; citation_author=Postlethwait  J.H.; citation_author=Woods  I.G.; citation_author=Ngo-Hazelett  P.;  citation_journal_title=Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1890-902; " /><meta name="citation_reference" content="citation_title=Whole-genome duplication in teleost fishes and its evolutionary consequences; citation_author=Glasauer  S.M.; citation_author=Neuhauss  S.C.;  citation_journal_title=Mol. Genet. Genomics.;  citation_year=2014;  citation_volume=289;  citation_pages=1045-60; " /><meta name="citation_reference" content="citation_title=Gene loss and evolutionary rates following whole-genome duplication in teleost fishes; citation_author=Brunet  F.G.; citation_author=Roest Crollius  H.; citation_author=Paris  M.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1808-16; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/2/161/951904/dsv001.pdf" /><meta name="description" content="Abstract.  High-resolution genetic maps are essential for fine mapping of complex traits, genome assembly, and comparative genomic analysis. Single-nucleotide p" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/2/161/332908" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/2/161/332908" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/2/161/332908");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv001';
    </script>
<title>
	Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./332908" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BejEATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hhbywgQ2hhbmd3ZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgWWVsbG93IFNlYSBGaXNoZXJpZXMgUmVzZWFyY2ggSW5zdGl0dXRlLCBDQUZTLCBLZXkgTGFiIGZvciBTdXN0YWluYWJsZSBEZXZlbG9wbWVudCBvZiBNYXJpbmUgRmlzaGVyaWVzLCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbiBMYWJvcmF0b3J5IGZvciBNYXJpbmUgRmlzaGVyaWVzIFNjaWVuY2UgYW5kIEZvb2QgUHJvZHVjdGlvbiBQcm9jZXNzZXMsIE5hdGlvbmFsIExhYiBmb3IgT2NlYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgUWluZ2RhbyAyNjYwNzEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBNYXJpbmUgU2NpZW5jZSwgVG9reW8gVW5pdmVyc2l0eSBvZiBNYXJpbmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgTWluYXRvLCBUb2t5byAxMDgtODQ3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaXUsIFlvbmdjaGFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQkdJLVNoZW56aGVuLCBTaGVuemhlbiA1MTgwMDAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFzdGFzLCBQYXNpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlcywgTWV0YXBvcHVsYXRpb24gUmVzZWFyY2ggR3JvdXAsIFVuaXZlcnNpdHkgb2YgSGVsc2lua2ksIEhlbHNpbmtpIEZJLTAwMDE0LCBGaW5sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBZYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIFllbGxvdyBTZWEgRmlzaGVyaWVzIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ0FGUywgS2V5IExhYiBmb3IgU3VzdGFpbmFibGUgRGV2ZWxvcG1lbnQgb2YgTWFyaW5lIEZpc2hlcmllcywgUWluZ2RhbyAyNjYwNzEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb24gTGFib3JhdG9yeSBmb3IgTWFyaW5lIEZpc2hlcmllcyBTY2llbmNlIGFuZCBGb29kIFByb2R1Y3Rpb24gUHJvY2Vzc2VzLCBOYXRpb25hbCBMYWIgZm9yIE9jZWFuIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFFpbmdkYW8gMjY2MDcxLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlhpZSwgWmhpeXVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJHSS1TaGVuemhlbiwgU2hlbnpoZW4gNTE4MDAwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBIZW5nZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDaGluZXNlIEFjYWRlbXkgb2YgRmlzaGVyaWVzIFNjaWVuY2UsIEJlaWppbmcgMTAwMDM5LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIExlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBZZWxsb3cgU2VhIEZpc2hlcmllcyBSZXNlYXJjaCBJbnN0aXR1dGUsIENBRlMsIEtleSBMYWIgZm9yIFN1c3RhaW5hYmxlIERldmVsb3BtZW50IG9mIE1hcmluZSBGaXNoZXJpZXMsIFFpbmdkYW8gMjY2MDcxLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1bmN0aW9uIExhYm9yYXRvcnkgZm9yIE1hcmluZSBGaXNoZXJpZXMgU2NpZW5jZSBhbmQgRm9vZCBQcm9kdWN0aW9uIFByb2Nlc3NlcywgTmF0aW9uYWwgTGFiIGZvciBPY2VhbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaWFuZywgWW9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIE9jZWFub2dyYXBoaWMgQ2VudGVyLCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWksIFNodWFpc2h1YWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCR0ktU2hlbnpoZW4sIFNoZW56aGVuIDUxODAwMCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUaWFuLCBZb25nc2hlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgWWVsbG93IFNlYSBGaXNoZXJpZXMgUmVzZWFyY2ggSW5zdGl0dXRlLCBDQUZTLCBLZXkgTGFiIGZvciBTdXN0YWluYWJsZSBEZXZlbG9wbWVudCBvZiBNYXJpbmUgRmlzaGVyaWVzLCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbiBMYWJvcmF0b3J5IGZvciBNYXJpbmUgRmlzaGVyaWVzIFNjaWVuY2UgYW5kIEZvb2QgUHJvZHVjdGlvbiBQcm9jZXNzZXMsIE5hdGlvbmFsIExhYiBmb3IgT2NlYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgUWluZ2RhbyAyNjYwNzEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrYW1vdG8sIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIE1hcmluZSBTY2llbmNlLCBUb2t5byBVbml2ZXJzaXR5IG9mIE1hcmluZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBNaW5hdG8sIFRva3lvIDEwOC04NDc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIFNvbmdsaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgWWVsbG93IFNlYSBGaXNoZXJpZXMgUmVzZWFyY2ggSW5zdGl0dXRlLCBDQUZTLCBLZXkgTGFiIGZvciBTdXN0YWluYWJsZSBEZXZlbG9wbWVudCBvZiBNYXJpbmUgRmlzaGVyaWVzLCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGdW5jdGlvbiBMYWJvcmF0b3J5IGZvciBNYXJpbmUgRmlzaGVyaWVzIFNjaWVuY2UgYW5kIEZvb2QgUHJvZHVjdGlvbiBQcm9jZXNzZXMsIE5hdGlvbmFsIExhYiBmb3IgT2NlYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgUWluZ2RhbyAyNjYwNzEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtd2lkZSBTTlAgaWRlbnRpZmljYXRpb24gZm9yIHRoZSBjb25zdHJ1Y3Rpb24gb2YgYSBoaWdoLXJlc29sdXRpb24gZ2VuZXRpYyBtYXAgb2YgSmFwYW5lc2UgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpOiBhcHBsaWNhdGlvbnMgdG8gUVRMIG1hcHBpbmcgb2YgVmlicmlvIGFuZ3VpbGxhcnVtIGRpc2Vhc2UgcmVzaXN0YW5jZSBhbmQgY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIEphcGFuZXNlIGZsb3VuZGVyIGthcnlvdHlwZSBieSBjaHJvbW9zb21lIGJhbmRpbmdzIGFuZCBmbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIHdpdGggRE5BIG1hcmtlcnM7IGNpdGF0aW9uX2F1dGhvcj1GdWppd2FyYSAgQS47IGNpdGF0aW9uX2F1dGhvcj1GdWppd2FyYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1OaXNoaWRhLVVtZWhhcmEgIEMuOyBjaXRhdGlvbl9hdXRob3I9QWJlICBTLjsgY2l0YXRpb25fYXV0aG9yPU1hc2Fva2EgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz0yNjctNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpc2ggcHJvZHVjdGlvbiBlc3RpbWF0ZXMgYW5kIHRyZW5kcyAyMDEy4oCTMjAxMzsgY2l0YXRpb25fYXV0aG9yPU55c3RveWwgIFIuOyBjaXRhdGlvbl9hdXRob3I9VHZldGVyYXMgIFIuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG91bmRlciBjdWx0dXJlIGFuZCBpdHMgY2hhbGxlbmdlcyBpbiBBc2lhOyBjaXRhdGlvbl9hdXRob3I9U2Vpa2FpICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmV2LiBGaXNoLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTPigJM0OyAgY2l0YXRpb25fcGFnZXM9NDIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgc2NhbiBmb3Igc2VsZWN0aW9uIHJldmVhbHMgY2FuZGlkYXRlcyBmb3IgZ2VuZXMgaW52b2x2ZWQgaW4gdGhlIGV2b2x1dGlvbiBvZiBjdWx0aXZhdGVkIHN1bmZsb3dlciAoSGVsaWFudGh1cyBhbm51dXMpOyBjaXRhdGlvbl9hdXRob3I9Q2hhcG1hbiAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPVBhc2hsZXkgIEMuSC47IGNpdGF0aW9uX2F1dGhvcj1XZW56bGVyICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI5MzEtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwIGNvbnN0cnVjdGlvbiBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIChRVEwpIGRldGVjdGlvbiBvZiBncm93dGgtcmVsYXRlZCB0cmFpdHMgaW4gTGl0b3BlbmFldXMgdmFubmFtZWkgZm9yIHNlbGVjdGl2ZSBicmVlZGluZyBhcHBsaWNhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1BbmRyaWFudGFoaW5hICBGLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgWC47IGNpdGF0aW9uX2F1dGhvcj1IdWFuZyAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzUyMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgYWR2YW5jZXMgb2YgZ2Vub21lIG1hcHBpbmcgYW5kIG1hcmtlci1hc3Npc3RlZCBzZWxlY3Rpb24gaW4gYXF1YWN1bHR1cmU7IGNpdGF0aW9uX2F1dGhvcj1ZdWUgIEcuSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2ggRmlzaC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zNzYtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgSmFwYW5lc2UgZmxvdW5kZXIsIFBhcmFsaWNodGh5cyBvbGl2YWNldXM7IGNpdGF0aW9uX2F1dGhvcj1Db2ltYnJhICBNLlIuOyBjaXRhdGlvbl9hdXRob3I9S29iYXlhc2hpICBLLjsgY2l0YXRpb25fYXV0aG9yPUtvcmV0c3VndSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTHigJM0OyAgY2l0YXRpb25fcGFnZXM9MjAzLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGxpbmthZ2UgbWFwIG9mIG9saXZlIGZsb3VuZGVyLCBQYXJhbGljaHRoeXMgb2xpdmFjZXVzOyBjaXRhdGlvbl9hdXRob3I9S2FuZyAgSi5ILjsgY2l0YXRpb25fYXV0aG9yPUtpbSAgVy5KLjsgY2l0YXRpb25fYXV0aG9yPUxlZSAgVy5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZWNvbmQgZ2VuZXJhdGlvbiBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIEphcGFuZXNlIGZsb3VuZGVyIChQYXJhbGljaHRoeXMgb2xpdmFjZXVzKTsgY2l0YXRpb25fYXV0aG9yPUNhc3Rhbm8tU2FuY2hleiAgQy47IGNpdGF0aW9uX2F1dGhvcj1GdWppICBLLjsgY2l0YXRpb25fYXV0aG9yPU96YWtpICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGhpZ2gtZGVuc2l0eSBnZW5ldGljIGxpbmthZ2UgbWFwcyBhbmQgbWFwcGluZyBvZiBncm93dGgtcmVsYXRlZCBxdWFudGl0YXRpdmUgdHJhaWwgbG9jaSBpbiB0aGUgSmFwYW5lc2UgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpOyBjaXRhdGlvbl9hdXRob3I9U29uZyAgVy47IGNpdGF0aW9uX2F1dGhvcj1QYW5nICBSLjsgY2l0YXRpb25fYXV0aG9yPU5pdSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb3MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNTA0MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhIHNpbmdsZSBtYWpvciBnZW5ldGljIGxvY3VzIGNvbnRyb2xsaW5nIHRoZSByZXNpc3RhbmNlIHRvIGx5bXBob2N5c3RpcyBkaXNlYXNlIGluIEphcGFuZXNlIGZsb3VuZGVyIChQYXJhbGljaHRoeXMgb2xpdmFjZXVzKTsgY2l0YXRpb25fYXV0aG9yPUZ1amkgIEsuOyBjaXRhdGlvbl9hdXRob3I9S29iYXlhc2hpICBLLjsgY2l0YXRpb25fYXV0aG9yPUhhc2VnYXdhICBPLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MeKAkzQ7ICBjaXRhdGlvbl9wYWdlcz0yMDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2UgbWFwcGluZyBvZiB0b2xsLWxpa2UgcmVjZXB0b3JzIChUTFJzKSBpbiBKYXBhbmVzZSBmbG91bmRlciwgUGFyYWxpY2h0aHlzIG9saXZhY2V1czsgY2l0YXRpb25fYXV0aG9yPUh3YW5nICBTLkQuOyBjaXRhdGlvbl9hdXRob3I9RnVqaSAgSy47IGNpdGF0aW9uX2F1dGhvcj1UYWthbm8gIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NYXIuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTA4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFya2VyLWFzc2lzdGVkIGJyZWVkaW5nIG9mIGEgbHltcGhvY3lzdGlzIGRpc2Vhc2UtcmVzaXN0YW50IEphcGFuZXNlIGZsb3VuZGVyIChQcmFsaWNodGh5cyBvbGl2YWNldXMpOyBjaXRhdGlvbl9hdXRob3I9RnVqaSAgSy5ILk8uOyBjaXRhdGlvbl9hdXRob3I9SG9uZGEgIEsuOyBjaXRhdGlvbl9hdXRob3I9S3VtYXNha2EgIEsuOyBjaXRhdGlvbl9hdXRob3I9U2FrYW1vdG8gIFQuOyBjaXRhdGlvbl9hdXRob3I9T2thbW90byAgTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTHigJM0OyAgY2l0YXRpb25fcGFnZXM9MjkxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgQy4gZWxlZ2FucyBhbmQgQy4gYnJpZ2dzYWUgZ2Vub21lIHNlcXVlbmNlcyByZXZlYWxzIGV4dGVuc2l2ZSBjb25zZXJ2YXRpb24gb2YgY2hyb21vc29tZSBvcmdhbml6YXRpb24gYW5kIHN5bnRlbnk7IGNpdGF0aW9uX2F1dGhvcj1IaWxsaWVyICBMLlcuOyBjaXRhdGlvbl9hdXRob3I9TWlsbGVyICBSLkQuOyBjaXRhdGlvbl9hdXRob3I9QmFpcmQgIFMuRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb3MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmUtYmFzZWQgbGlua2FnZSBtYXAgZm9yIEJpY3ljbHVzIGFueW5hbmEgYnV0dGVyZmxpZXMgYWxsb3dzIGZvciBhIGNvbXByZWhlbnNpdmUgYW5hbHlzaXMgb2Ygc3ludGVueSB3aXRoIHRoZSBsZXBpZG9wdGVyYW4gcmVmZXJlbmNlIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUJlbGRhZGUgIFAuOyBjaXRhdGlvbl9hdXRob3I9U2FlbmtvICBTLlYuOyBjaXRhdGlvbl9hdXRob3I9UHVsICBOLjsgY2l0YXRpb25fYXV0aG9yPUxvbmcgIEEuRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDM2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldmlldyBvZiBtYXNzaXZlbHkgcGFyYWxsZWwgRE5BIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzOyBjaXRhdGlvbl9hdXRob3I9TW9vcnRoaWUgIFMuOyBjaXRhdGlvbl9hdXRob3I9TWF0dG9ja3MgIEMuSi47IGNpdGF0aW9uX2F1dGhvcj1XcmlnaHQgIEMuRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhVR08gSjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsaWFiaWxpdHkgb2YgaGlnaC10aHJvdWdocHV0IGdlbm90eXBpbmcgb2Ygd2hvbGUgZ2Vub21lIGFtcGxpZmllZCBETkEgaW4gU05QIGdlbm90eXBpbmcgc3R1ZGllczsgY2l0YXRpb25fYXV0aG9yPUJlcnRoaWVyLVNjaGFhZCAgWS47IGNpdGF0aW9uX2F1dGhvcj1LYW8gIFcuSC47IGNpdGF0aW9uX2F1dGhvcj1Db3Jlc2ggIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbGVjdHJvcGhvcmVzaXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yODEyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJBRCBpbiB0aGUgcmVhbG0gb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzOyBjaXRhdGlvbl9hdXRob3I9Um93ZSAgSC5DLjsgY2l0YXRpb25fYXV0aG9yPVJlbmF1dCAgUy47IGNpdGF0aW9uX2F1dGhvcj1HdWdnaXNiZXJnICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM0OTktNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBTTlAgZGlzY292ZXJ5IGFuZCBnZW5ldGljIG1hcHBpbmcgdXNpbmcgc2VxdWVuY2VkIFJBRCBtYXJrZXJzOyBjaXRhdGlvbl9hdXRob3I9QmFpcmQgIE4uQS47IGNpdGF0aW9uX2F1dGhvcj1FdHRlciAgUC5ELjsgY2l0YXRpb25fYXV0aG9yPUF0d29vZCAgVC5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUzMzc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFEIHNlcXVlbmNpbmcgeWllbGRzIGEgaGlnaCBzdWNjZXNzIHJhdGUgZm9yIHdlc3RzbG9wZSBjdXR0aHJvYXQgYW5kIHJhaW5ib3cgdHJvdXQgc3BlY2llcy1kaWFnbm9zdGljIFNOUCBhc3NheXM7IGNpdGF0aW9uX2F1dGhvcj1BbWlzaCAgUy5KLjsgY2l0YXRpb25fYXV0aG9yPUhvaGVubG9oZSAgUC5BLjsgY2l0YXRpb25fYXV0aG9yPVBhaW50ZXIgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291ci47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgUkFEIHRhZyBkZXJpdmVkIG1hcmtlciBiYXNlZCBlZ2dwbGFudCBsaW5rYWdlIG1hcCBhbmQgdGhlIGxvY2F0aW9uIG9mIFFUTHMgZGV0ZXJtaW5pbmcgYW50aG9jeWFuaW4gcGlnbWVudGF0aW9uOyBjaXRhdGlvbl9hdXRob3I9QmFyY2hpICBMLjsgY2l0YXRpb25fYXV0aG9yPUxhbnRlcmkgIFMuOyBjaXRhdGlvbl9hdXRob3I9UG9ydGlzICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0Mzc0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgYXBwbGljYXRpb24gZm9yIFFUTCBhbmFseXNpcyBvZiBhIFJlc3RyaWN0aW9uIFNpdGUgQXNzb2NpYXRlZCBETkEgKFJBRCkgbGlua2FnZSBtYXAgaW4gYmFybGV5OyBjaXRhdGlvbl9hdXRob3I9Q2h1dGltYW5pdHNha3VuICBZLjsgY2l0YXRpb25fYXV0aG9yPU5pcHBlciAgUi5XLjsgY2l0YXRpb25fYXV0aG9yPUN1ZXN0YS1NYXJjb3MgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgZm9yIGNvbnN0cnVjdGluZyBhIHNhdHVyYXRlZCBsaW5rYWdlIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgY2l0YXRpb25fYXV0aG9yPUdhdXIgIFIuOyBjaXRhdGlvbl9hdXRob3I9QXphbSAgUy47IGNpdGF0aW9uX2F1dGhvcj1KZWVuYSAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzU3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYWlyZWQtZW5kIFJBRC1zZXEgZm9yIGRlIG5vdm8gYXNzZW1ibHkgYW5kIG1hcmtlciBkZXNpZ24gd2l0aG91dCBhdmFpbGFibGUgcmVmZXJlbmNlOyBjaXRhdGlvbl9hdXRob3I9V2lsbGluZyAgRS5NLjsgY2l0YXRpb25fYXV0aG9yPUhvZmZtYW5uICBNLjsgY2l0YXRpb25fYXV0aG9yPUtsZWluICBKLkQuOyBjaXRhdGlvbl9hdXRob3I9V2VpZ2VsICBELjsgY2l0YXRpb25fYXV0aG9yPURyZXllciAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb24gb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgZm9yIHJhcGlkIG1hcmtlciBkZXZlbG9wbWVudCBpbiBtb2xlY3VsYXIgcGxhbnQgYnJlZWRpbmc6IGEgY2FzZSBzdHVkeSBvbiBhbnRocmFjbm9zZSBkaXNlYXNlIHJlc2lzdGFuY2UgaW4gTHVwaW51cyBhbmd1c3RpZm9saXVzIEw7IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBILjsgY2l0YXRpb25fYXV0aG9yPVRhbyAgWS47IGNpdGF0aW9uX2F1dGhvcj1aaGVuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1MaSAgQy47IGNpdGF0aW9uX2F1dGhvcj1Td2VldGluZ2hhbSAgTS5XLjsgY2l0YXRpb25fYXV0aG9yPUhvd2llc29uICBKLkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGdlbm9taWNzIG9mIHBhcmFsbGVsIGFkYXB0YXRpb24gaW4gdGhyZWVzcGluZSBzdGlja2xlYmFjayB1c2luZyBzZXF1ZW5jZWQgUkFEIHRhZ3M7IGNpdGF0aW9uX2F1dGhvcj1Ib2hlbmxvaGUgIFAuQS47IGNpdGF0aW9uX2F1dGhvcj1CYXNzaGFtICBTLjsgY2l0YXRpb25fYXV0aG9yPUV0dGVyICBQLkQuOyBjaXRhdGlvbl9hdXRob3I9U3RpZmZsZXIgIE4uOyBjaXRhdGlvbl9hdXRob3I9Sm9obnNvbiAgRS5BLjsgY2l0YXRpb25fYXV0aG9yPUNyZXNrbyAgVy5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwODYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgZ3Jvd3RoIHBlcmZvcm1hbmNlIGFuZCBicmVlZGluZyB2YWx1ZSBvZiDigJhQaW5nIFlvdSBOby4x4oCZIEphcGFuZXNlIGZsb3VuZGVyIGFuZCBzZWxlY3Rpb24gb2YgcGFyZW50czsgY2l0YXRpb25fYXV0aG9yPUxpdSAgRi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBTLkwuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZpc2guIFNjaS4gQ2hpbmE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTUyMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NSEMgcG9seW1vcnBoaXNtIGFuZCBkaXNlYXNlIHJlc2lzdGFuY2UgdG8gVmlicmlvIGFuZ3VpbGxhcnVtIGluIDEyIHNlbGVjdGl2ZSBKYXBhbmVzZSBmbG91bmRlciAoUGFyYWxpY2h0aHlzIG9saXZhY2V1cykgZmFtaWxpZXM7IGNpdGF0aW9uX2F1dGhvcj1YdSAgVC5KLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFMuTC47IGNpdGF0aW9uX2F1dGhvcj1KaSAgWC5TLjsgY2l0YXRpb25fYXV0aG9yPVRpYW4gIFkuUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2ggU2hlbGxmaXNoIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIxMy0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhY2tzOiBidWlsZGluZyBhbmQgZ2Vub3R5cGluZyBMb2NpIGRlIG5vdm8gZnJvbSBzaG9ydC1yZWFkIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPUNhdGNoZW4gIEouTS47IGNpdGF0aW9uX2F1dGhvcj1BbW9yZXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9SG9oZW5sb2hlICBQLjsgY2l0YXRpb25fYXV0aG9yPUNyZXNrbyAgVy47IGNpdGF0aW9uX2F1dGhvcj1Qb3N0bGV0aHdhaXQgIEouSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczLUdlbmVzIEdlbm9tLkdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFja3M6IGFuIGFuYWx5c2lzIHRvb2wgc2V0IGZvciBwb3B1bGF0aW9uIGdlbm9taWNzOyBjaXRhdGlvbl9hdXRob3I9Q2F0Y2hlbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1Ib2hlbmxvaGUgIFAuQS47IGNpdGF0aW9uX2F1dGhvcj1CYXNzaGFtICBTLjsgY2l0YXRpb25fYXV0aG9yPUFtb3JlcyAgQS47IGNpdGF0aW9uX2F1dGhvcj1DcmVza28gIFcuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zMTI0LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZXAtTUFQOiBmYXN0IGFuZCBhY2N1cmF0ZSBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gZm9yIGxhcmdlIFNOUCBkYXRhc2V0czsgY2l0YXRpb25fYXV0aG9yPVJhc3RhcyAgUC47IGNpdGF0aW9uX2F1dGhvcj1QYXVsaW4gIEwuOyBjaXRhdGlvbl9hdXRob3I9SGFuc2tpICBJLjsgY2l0YXRpb25fYXV0aG9yPUxlaHRvbmVuICBSLjsgY2l0YXRpb25fYXV0aG9yPUF1dmluZW4gIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMxMjgtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0ZSBpbnRlcnZhbCBtYXBwaW5nIGFuZCBtdWx0aXBsZSBpbnRlcnZhbCBtYXBwaW5nOiBwcm9jZWR1cmVzIGFuZCBndWlkZWxpbmVzIGZvciB1c2luZyBXaW5kb3dzIFFUTCBDYXJ0b2dyYXBoZXI7IGNpdGF0aW9uX2F1dGhvcj1TaWx2YSBMZGEgIEMuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgUy47IGNpdGF0aW9uX2F1dGhvcj1aZW5nICBaLkIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODcxOyAgY2l0YXRpb25fcGFnZXM9NzUtMTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NlbWJseSBhbmQgZnVuY3Rpb24gb2YgdGhlIE1IQyBJIHBlcHRpZGUtbG9hZGluZyBjb21wbGV4IGFyZSBjb25zZXJ2ZWQgYWNyb3NzIGhpZ2hlciB2ZXJ0ZWJyYXRlczsgY2l0YXRpb25fYXV0aG9yPUhpbnogIEEuOyBjaXRhdGlvbl9hdXRob3I9SmVkYW16aWNrICBKLjsgY2l0YXRpb25fYXV0aG9yPUhlcmJyaW5nICBWLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9MzMxMDktMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYXQgc2hvY2sgcHJvdGVpbiA2MCBhY3RpdmF0ZXMgQiBjZWxscyB2aWEgdGhlIFRMUjQtTXlEODggcGF0aHdheTsgY2l0YXRpb25fYXV0aG9yPUNvaGVuLVNmYWR5ICBNLjsgY2l0YXRpb25fYXV0aG9yPU51c3NiYXVtICBHLjsgY2l0YXRpb25fYXV0aG9yPVBldnNuZXItRmlzY2hlciAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3NTsgIGNpdGF0aW9uX3BhZ2VzPTM1OTQtNjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgdHJhbnNwb3NvbnMgYW5kIHRoZSByb2xlIG9mIHJlY29tYmluYXRpb24gaW4gbXV0YXRpb24gYWNjdW11bGF0aW9uIGluIERhcGhuaWEgcHVsZXg7IGNpdGF0aW9uX2F1dGhvcj1TY2hhYWNrICBTLjsgY2l0YXRpb25fYXV0aG9yPUNob2kgIEUuOyBjaXRhdGlvbl9hdXRob3I9THluY2ggIE0uOyBjaXRhdGlvbl9hdXRob3I9UHJpdGhhbSAgRS5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbiBvZiByZXN0cmljdGVkIHJlY29tYmluYXRpb24gaW4gc2V4IGNocm9tb3NvbWVzOyBjaXRhdGlvbl9hdXRob3I9QmVyZ2VybyAgUi47IGNpdGF0aW9uX2F1dGhvcj1DaGFybGVzd29ydGggIEQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgRWNvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz05NC0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2dyZXNzaXZlIHJlY29tYmluYXRpb24gc3VwcHJlc3Npb24gYW5kIGRpZmZlcmVudGlhdGlvbiBpbiByZWNlbnRseSBldm9sdmVkIG5lby1zZXggY2hyb21vc29tZXM7IGNpdGF0aW9uX2F1dGhvcj1OYXRyaSAgSC5NLjsgY2l0YXRpb25fYXV0aG9yPVNoaWthbm8gIFQuOyBjaXRhdGlvbl9hdXRob3I9TWVyaWxhICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTExMzEtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUpvaW5tYXAgNDogc29mdHdhcmUgZm9yIHRoZSBjYWxjdWxhdGlvbiBvZiBnZW5ldGljIGxpbmthZ2UgbWFwcyBpbiBleHBlcmltZW50YWwgcG9wdWxhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj12YW4gT29pamVuICBKLlcuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2xhbnZpbGxlIGZyaXRpbGxhcnkgZ2Vub21lIHJldGFpbnMgYW4gYW5jaWVudCBrYXJ5b3R5cGUgYW5kIHJldmVhbHMgc2VsZWN0aXZlIGNocm9tb3NvbWFsIGZ1c2lvbnMgaW4gTGVwaWRvcHRlcmE7IGNpdGF0aW9uX2F1dGhvcj1BaG9sYSAgVi47IGNpdGF0aW9uX2F1dGhvcj1MZWh0b25lbiAgUi47IGNpdGF0aW9uX2F1dGhvcj1Tb21lcnZ1byAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZSBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00NzM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2J1c3QsIHNpbXBsZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgKEdCUykgYXBwcm9hY2ggZm9yIGhpZ2ggZGl2ZXJzaXR5IHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1FbHNoaXJlICBSLkouOyBjaXRhdGlvbl9hdXRob3I9R2xhdWJpdHogIEouQy47IGNpdGF0aW9uX2F1dGhvcj1TdW4gIFEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE5Mzc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWUgc2NhbiBmb3IgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYXNzb2NpYXRlZCB3aXRoIFZpYnJpbyBhbmd1aWxsYXJ1bSBpbmZlY3Rpb24gcmVzaXN0YW5jZSBpbiBKYXBhbmVzZSBmbG91bmRlciAoUGFyYWxpY2h0aHlzIG9saXZhY2V1cykgYnkgYnVsa2VkIHNlZ3JlZ2FudCBhbmFseXNpczsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9RmFuICBDLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1hci4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01MTMtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBhbmFseXNpcyBmb3IgcGxhbnQgaGVpZ2h0IGFuZCBsZWFmIGFuZ2xlIGJ5IHVzaW5nIGRpZmZlcmVudCBwb3B1bGF0aW9ucyBvZiBtYWl6ZTsgY2l0YXRpb25fYXV0aG9yPVl1ICBZLlQuOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIEouTS47IGNpdGF0aW9uX2F1dGhvcj1TaGkgIFkuUy47IGNpdGF0aW9uX2F1dGhvcj1Tb25nICBZLkMuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgVC5ZLjsgY2l0YXRpb25fYXV0aG9yPUxpICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTWFpemUgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTg4LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgUVRMIHdpdGggZWZmZWN0cyBvbiBvc21vcmVndWxhdGlvbiBjYXBhY2l0aWVzIGluIHRoZSByYWluYm93IHRyb3V0IChPbmNvcmh5bmNodXMgbXlraXNzKTsgY2l0YXRpb25fYXV0aG9yPUxlIEJyYXMgIFkuOyBjaXRhdGlvbl9hdXRob3I9RGVjaGFtcCAgTi47IGNpdGF0aW9uX2F1dGhvcj1LcmllZyAgRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhZGFwdGFibGUgbWFqb3IgaGlzdG9jb21wYXRpYmlsaXR5IGNvbXBsZXggKE1IQykgZm9sZDogc3RydWN0dXJlIGFuZCBmdW5jdGlvbiBvZiBub25jbGFzc2ljYWwgYW5kIE1IQyBjbGFzcyBJLWxpa2UgbW9sZWN1bGVzOyBjaXRhdGlvbl9hdXRob3I9QWRhbXMgIEUuSi47IGNpdGF0aW9uX2F1dGhvcj1MdW9tYSAgQS5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTUyOS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUhDIGNsYXNzIElJYWxwaGEgZ2VuZSBwb2x5bW9ycGhpc20gYW5kIGl0cyBhc3NvY2lhdGlvbiB3aXRoIHJlc2lzdGFuY2Uvc3VzY2VwdGliaWxpdHkgdG8gVmlicmlvIGFuZ3VpbGxhcnVtIGluIEphcGFuZXNlIGZsb3VuZGVyIChQYXJhbGljaHRoeXMgb2xpdmFjZXVzKTsgY2l0YXRpb25fYXV0aG9yPVh1ICBULkouOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLlguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIENvbXAuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTA0Mi01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxsZWxpYyB2YXJpYXRpb24sIGJhbGFuY2luZyBzZWxlY3Rpb24gYW5kIHBvc2l0aXZlIHNlbGVjdGVkIHNpdGVzIGRldGVjdGVkIGZyb20gTUhDIGNsYXNzIElhbHBoYSBnZW5lIG9mIG9saXZlIGZsb3VuZGVyOyBjaXRhdGlvbl9hdXRob3I9WHUgIFQuSi47IGNpdGF0aW9uX2F1dGhvcj1TdW4gIFkuTi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBTLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz0xMjUxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHltb3JwaGlzbSBhbmQgYmFsYW5jaW5nIHNlbGVjdGlvbiBvZiBNSEMgY2xhc3MgSUkgREFCIGdlbmUgaW4gNyBzZWxlY3RpdmUgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpIGZhbWlsaWVzOyBjaXRhdGlvbl9hdXRob3I9RHUgIE0uOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPUxpYW5nICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXZpZC4gQmFzZWQgQ29tcGxlbWVudC4gQWx0ZXJuYXQuIE1lZC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjAsMTE7ICBjaXRhdGlvbl9wYWdlcz02MTM2MjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5OiBhZHZhbmNlcyBhbmQgYXBwbGljYXRpb25zOyBjaXRhdGlvbl9hdXRob3I9QnVlcm1hbnMgIEguUC47IGNpdGF0aW9uX2F1dGhvcj1kZW4gRHVubmVuICBKLlQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE5MzItNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGV0aXRpdmUgRE5BIGFuZCBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzogY29tcHV0YXRpb25hbCBjaGFsbGVuZ2VzIGFuZCBzb2x1dGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1UcmVhbmdlbiAgVC5KLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBTLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzYtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBjaGFubmVsIGNhdGZpc2ggKEljdGFsdXJ1cyBwdW5jdGF0dXMpIHdpdGggZm91ciBtb2RlbCBmaXNoIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1KaWFuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1HYW8gIFguOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gb2YgYW50aS1NdWxsZXJpYW4gaG9ybW9uZSAoYW1oKSBhbmQgYW50aS1NdWxsZXJpYW4gaG9ybW9uZSByZWNlcHRvciB0eXBlIElJIChhbWhySUkpIGluIHRoZSB0ZWxlb3N0IG1lZGFrYTsgY2l0YXRpb25fYXV0aG9yPUtsdXZlciAgTi47IGNpdGF0aW9uX2F1dGhvcj1QZmVubmlnICBGLjsgY2l0YXRpb25fYXV0aG9yPVBhbGEgIEkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIER5bmFtLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzY7ICBjaXRhdGlvbl9wYWdlcz0yNzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5laWdoYm9yaW5nIGdlbmVzIHNob3cgaW50ZXJjaHJvbW9zb21hbCBjb2xvY2FsaXphdGlvbiBhZnRlciB0aGVpciBzZXBhcmF0aW9uOyBjaXRhdGlvbl9hdXRob3I9RGFpICBaLjsgY2l0YXRpb25fYXV0aG9yPVhpb25nICBZLjsgY2l0YXRpb25fYXV0aG9yPURhaSAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMTY2LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljIGFuYWx5c2lzIG9mIGNhdGZpc2ggbGlua2FnZSBncm91cCA4IHJldmVhbHMgdHdvIGhvbW9sb2dvdXMgY2hyb21vc29tZXMgaW4gemVicmFmaXNoIGFuZCBvdGhlciB0ZWxlb3N0cyB3aXRoIGV4dGVuc2l2ZSBpbnRlci1jaHJvbW9zb21hbCByZWFycmFuZ2VtZW50czsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgUy47IGNpdGF0aW9uX2F1dGhvcj1MdSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6ZWJyYWZpc2ggZ2VuZSBtYXAgZGVmaW5lcyBhbmNlc3RyYWwgdmVydGVicmF0ZSBjaHJvbW9zb21lczsgY2l0YXRpb25fYXV0aG9yPVdvb2RzICBJLkcuOyBjaXRhdGlvbl9hdXRob3I9V2lsc29uICBDLjsgY2l0YXRpb25fYXV0aG9yPUZyaWVkbGFuZGVyICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMzA3LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1aZWJyYWZpc2ggY29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIHRoZSBvcmlnaW5zIG9mIHZlcnRlYnJhdGUgY2hyb21vc29tZXM7IGNpdGF0aW9uX2F1dGhvcj1Qb3N0bGV0aHdhaXQgIEouSC47IGNpdGF0aW9uX2F1dGhvcj1Xb29kcyAgSS5HLjsgY2l0YXRpb25fYXV0aG9yPU5nby1IYXplbGV0dCAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTg5MC05MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBkdXBsaWNhdGlvbiBpbiB0ZWxlb3N0IGZpc2hlcyBhbmQgaXRzIGV2b2x1dGlvbmFyeSBjb25zZXF1ZW5jZXM7IGNpdGF0aW9uX2F1dGhvcj1HbGFzYXVlciAgUy5NLjsgY2l0YXRpb25fYXV0aG9yPU5ldWhhdXNzICBTLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbmV0LiBHZW5vbWljcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9MTA0NS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBsb3NzIGFuZCBldm9sdXRpb25hcnkgcmF0ZXMgZm9sbG93aW5nIHdob2xlLWdlbm9tZSBkdXBsaWNhdGlvbiBpbiB0ZWxlb3N0IGZpc2hlczsgY2l0YXRpb25fYXV0aG9yPUJydW5ldCAgRi5HLjsgY2l0YXRpb25fYXV0aG9yPVJvZXN0IENyb2xsaXVzICBILjsgY2l0YXRpb25fYXV0aG9yPVBhcmlzICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE4MDgtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8yLzE2MS85NTE5MDQvZHN2MDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBIaWdoLXJlc29sdXRpb24gZ2VuZXRpYyBtYXBzIGFyZSBlc3NlbnRpYWwgZm9yIGZpbmUgbWFwcGluZyBvZiBjb21wbGV4IHRyYWl0cywgZ2Vub21lIGFzc2VtYmx5LCBhbmQgY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcy4gU2luZ2xlLW51Y2xlb3RpZGUgcCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMyOTA4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJvxXc1ygLFPPQ9iqEqZ/dHu3GJ4" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f161%2f332908" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f161%2f332908" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12675" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&amp;Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/2">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4595056">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595056">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595064">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595064">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595078">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595078">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595101">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595101">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595118">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595118">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595131">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595131">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595135">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595135">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4595139">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4595139">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/2/147/332826">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/2/171/333025">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of <em>Vibrio anguillarum</em> disease resistance and comparative genomic analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Changwei Shao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Ministry of Agriculture</div>, <div class="insititution">Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Function Laboratory for Marine Fisheries Science and Food Production Processes</div>, <div class="insititution">National Lab for Ocean Science and Technology</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Marine Science</div>, <div class="insititution">Tokyo University of Marine Science and Technology</div>, <div class="addr-line">Minato, Tokyo 108-8477</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Changwei+Shao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shao C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shao C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Changwei Shao</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongchao Niu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">BGI-Shenzhen</div>, <div class="addr-line">Shenzhen 518000</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongchao+Niu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Niu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Niu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yongchao Niu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pasi Rastas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Biosciences, Metapopulation Research Group</div>, <div class="insititution">University of Helsinki</div>, <div class="addr-line">Helsinki FI-00014</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pasi+Rastas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rastas P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rastas P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Pasi Rastas</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Ministry of Agriculture</div>, <div class="insititution">Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Function Laboratory for Marine Fisheries Science and Food Production Processes</div>, <div class="insititution">National Lab for Ocean Science and Technology</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yang Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhiyuan Xie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">BGI-Shenzhen</div>, <div class="addr-line">Shenzhen 518000</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhiyuan+Xie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xie Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xie Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zhiyuan Xie</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hengde Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Chinese Academy of Fisheries Science</div>, <div class="addr-line">Beijing 100039</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hengde+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hengde Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lei Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Ministry of Agriculture</div>, <div class="insititution">Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Function Laboratory for Marine Fisheries Science and Food Production Processes</div>, <div class="insititution">National Lab for Ocean Science and Technology</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lei+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lei Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">National Oceanographic Center</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yong Jiang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuaishuai Tai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">BGI-Shenzhen</div>, <div class="addr-line">Shenzhen 518000</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuaishuai+Tai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tai S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tai S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shuaishuai Tai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongsheng Tian</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Ministry of Agriculture</div>, <div class="insititution">Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Function Laboratory for Marine Fisheries Science and Food Production Processes</div>, <div class="insititution">National Lab for Ocean Science and Technology</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongsheng+Tian">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tian Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tian Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yongsheng Tian</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Sakamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Marine Science</div>, <div class="insititution">Tokyo University of Marine Science and Technology</div>, <div class="addr-line">Minato, Tokyo 108-8477</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 0532-85844606. Fax. +86 0532-85811514. E-mail: <a href="mailto:chensl@ysfri.ac.cn" target="_blank">chensl@ysfri.ac.cn</a>; Tel. +81 03-5463-0450. Fax. +81 03-5463-0450. E-mail: <a href="mailto:takashis@kaiyodai.ac.jp" target="_blank">takashis@kaiyodai.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Sakamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakamoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakamoto T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takashi Sakamoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Songlin Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Ministry of Agriculture</div>, <div class="insititution">Yellow Sea Fisheries Research Institute, CAFS, Key Lab for Sustainable Development of Marine Fisheries</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Function Laboratory for Marine Fisheries Science and Food Production Processes</div>, <div class="insititution">National Lab for Ocean Science and Technology</div>, <div class="addr-line">Qingdao 266071</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 0532-85844606. Fax. +86 0532-85811514. E-mail: <a href="mailto:chensl@ysfri.ac.cn" target="_blank">chensl@ysfri.ac.cn</a>; Tel. +81 03-5463-0450. Fax. +81 03-5463-0450. E-mail: <a href="mailto:takashis@kaiyodai.ac.jp" target="_blank">takashis@kaiyodai.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Songlin+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Songlin Chen</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 161–170, <a href='https://doi.org/10.1093/dnares/dsv001'>https://doi.org/10.1093/dnares/dsv001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 March 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 November 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 February 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="332908" href="/dnaresearch/article-pdf/22/2/161/951904/dsv001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Changwei Shao, Yongchao Niu, Pasi Rastas, Yang Liu, Zhiyuan Xie, Hengde Li, Lei Wang, Yong Jiang, Shuaishuai Tai, Yongsheng Tian, Takashi Sakamoto, Songlin Chen;  Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of <em>Vibrio anguillarum</em> disease resistance and comparative genomic analysis, <em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 161–170, <a href='https://doi.org/10.1093/dnares/dsv001'>https://doi.org/10.1093/dnares/dsv001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/332908?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/332908?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20SNP%20identification%20for%20the%20construction%20of%20a%20high-resolution%20genetic%20map%20of%20Japanese%20flounder%20(%26lt%3Bem%26gt%3BParalichthys%20olivaceus%26lt%3B%2Fem%26gt%3B)%3A%20applications%20to%20QTL%20mapping%20of%20%26lt%3Bem%26gt%3BVibrio%20anguillarum%26lt%3B%2Fem%26gt%3B%20disease%20resistance%20and%20comparative%20genomic%20analysis&amp;publicationDate=2015-03-11&amp;volumeNum=22&amp;issueNum=2&amp;author=Shao%2C%20Changwei%3B%20Niu%2C%20Yongchao&amp;startPage=161&amp;endPage=170&amp;contentId=10.1093%2Fdnares%2Fdsv001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f161%2f332908" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4595056 id="4595056" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>High-resolution genetic maps are essential for fine mapping of complex traits, genome assembly, and comparative genomic analysis. Single-nucleotide polymorphisms (SNPs) are the primary molecular markers used for genetic map construction. In this study, we identified 13,362 SNPs evenly distributed across the Japanese flounder (<em>Paralichthys olivaceus</em>) genome. Of these SNPs, 12,712 high-confidence SNPs were subjected to high-throughput genotyping and assigned to 24 consensus linkage groups (LGs). The total length of the genetic linkage map was 3,497.29 cM with an average distance of 0.47 cM between loci, thereby representing the densest genetic map currently reported for Japanese flounder. Nine positive quantitative trait loci (QTLs) forming two main clusters for <em>Vibrio anguillarum</em> disease resistance were detected. All QTLs could explain 5.1–8.38% of the total phenotypic variation. Synteny analysis of the QTL regions on the genome assembly revealed 12 immune-related genes, among them 4 genes strongly associated with <em>V. anguillarum</em> disease resistance. In addition, 246 genome assembly scaffolds with an average size of 21.79 Mb were anchored onto the LGs; these scaffolds, comprising 522.99 Mb, represented 95.78% of assembled genomic sequences. The mapped assembly scaffolds in Japanese flounder were used for genome synteny analyses against zebrafish (<em>Danio rerio</em>) and medaka (<em>Oryzias latipes</em>). Flounder and medaka were found to possess almost one-to-one synteny, whereas flounder and zebrafish exhibited a multi-syntenic correspondence. The newly developed high-resolution genetic map, which will facilitate QTL mapping, scaffold assembly, and genome synteny analysis of Japanese flounder, marks a milestone in the ongoing genome project for this species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Japanese flounder&quot;">Japanese flounder</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RAD-seq-based SNP&quot;">RAD-seq-based SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-resolution linkage map&quot;">high-resolution linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL mapping&quot;">QTL mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome synteny&quot;">genome synteny</a></div>                            <h2 scrollto-destination=4595064 id="4595064" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Japanese flounder (<em>Paralichthys olivaceus</em>), one of the most desirable and highly priced marine fish species, is widely cultured along the coast of Northeast Asian countries such as China, Japan, and Korea.<sup><a reveal-id="DSV001C1" data-open="DSV001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In addition to a booming Chinese market, &gt;60,000 tons of Japanese flounder were artificially produced in Japan and Korea in 2013,<sup><a reveal-id="DSV001C2" data-open="DSV001C2" class="link link-ref link-reveal xref-bibr">2</a></sup> making <em>P. olivaceus</em> one of the most important species in stock enhancement programs in these countries. As a result of long-term, high-intensity stocking and resource management, however, farmed Japanese flounder have depressed immune systems that enhance individual susceptibility to microbial infections.<sup><a reveal-id="DSV001C3" data-open="DSV001C3" class="link link-ref link-reveal xref-bibr">3</a></sup> To increase profitability and sustainability while maintaining genetic variability in the cultured stock, the development of genetic breeding programs, such as marker- and gene-assisted selection, is urgently required. These approaches involve selection of genomic loci or genes related to the economic trait of interest using genetic maps constructed by molecular markers.<sup><a reveal-id="DSV001C4" data-open="DSV001C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>An accurate genetic linkage map is therefore foundational to genetic breeding of a species.<sup><a reveal-id="DSV001C5" data-open="DSV001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> To date, linkage maps have been constructed for at least 28 fish species; in addition, economically important traits have been mapped using these maps in over 13 fish species, including tilapia, rainbow trout, channel catfish, Atlantic salmon, common carp, and Asian seabass.<sup><a reveal-id="DSV001C6" data-open="DSV001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Following the construction of the first generation of Japanese flounder genetic maps based on simple sequence repeat (SSR) and amplified fragment length polymorphism (AFLP) markers by Coimbra <em>et al.</em><sup><a reveal-id="DSV001C7" data-open="DSV001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and SSRs alone by Kang <em>et al</em>.,<sup><a reveal-id="DSV001C8" data-open="DSV001C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the second generation of linkage maps has been developed with various types of molecular markers, such as SSRs and a few single-nucleotide polymorphism (SNP) markers, either alone or in combination.<sup><a reveal-id="DSV001C9" data-open="DSV001C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSV001C10" data-open="DSV001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Such efforts have resulted in the identification of several quantitative trait loci (QTLs) for disease resistance and growth in Japanese flounder. For instance, a single major genetic locus associated with lymphocystis disease resistance has been detected<sup><a reveal-id="DSV001C11" data-open="DSV001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and its candidate gene <em>tlr2</em> identified,<sup><a reveal-id="DSV001C12" data-open="DSV001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> making the development of a lymphocystis disease-resistant strain of Japanese flounder possible by marker-assisted selection.<sup><a reveal-id="DSV001C13" data-open="DSV001C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In addition, four QTLs for growth rate are clustered on genetic linkage group (LG) 14, suggesting that these major loci contributing to growth trait variation are potentially useful for marker-assisted selection in future genetic breeding programs of Japanese flounder.<sup><a reveal-id="DSV001C10" data-open="DSV001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Although previous genetic maps of Japanese flounder have been successfully used to map QTLs for a few economic traits, the number of commonly available markers, restricted to hundreds to a few thousand, makes it difficult to carry out fine-scale mapping to shrink the genomic regions tightly associated with important traits. A genetic map of much higher resolution is thus urgently needed for Japanese flounder.</p><p>A high-resolution genetic linkage map is also an excellent tool for genome map construction, which in turn allows direct comparison of chromosomal organization and evolution.<sup><a reveal-id="DSV001C14" data-open="DSV001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For instance, <em>Caenorhabditis briggsae</em> chromosomes reconstructed by SNP-based genetic mapping were compared with the <em>Caenorhabditis elegans</em> genome. An almost complete conservation of synteny was observed, with many cases of perfect 1:1 orthologues found between these two distantly related species.<sup><a reveal-id="DSV001C14" data-open="DSV001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In addition, construction of a dense gene-based map of the butterfly species <em>Bicyclus anynana</em> enabled broad-coverage analysis of synteny with the lepidopteran reference genome; this analysis suggested strong conservation of gene assignments to chromosomes and numerous large- and small-scale chromosomal rearrangements.<sup><a reveal-id="DSV001C15" data-open="DSV001C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The application of genetic mapping allows genome-level analysis to be extended to non-model species, conferring an advantage in the comparative analysis of chromosomal organization and evolution.</p><p>With the emergence of next-generation (massively parallel) sequencing and associated technological genotyping advancements such as reduced representation library sequencing techniques,<sup><a reveal-id="DSV001C16" data-open="DSV001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> SNP markers, representing the most abundant source of variation in the genome, have become widely used for high-resolution genetic map construction; they are also applied for measurement of genetic diversity and investigation of population structure.<sup><a reveal-id="DSV001C17" data-open="DSV001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> As a reliable, high-throughput, affordable method to reduce genomic complexity, restriction-site associated DNA tag sequencing (RAD-seq) has been particularly attractive for SNP discovery and genotyping.<sup><a reveal-id="DSV001C18" data-open="DSV001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> RAD-seq technology has been successfully applied in the construction of genetic maps for various species, such as stickleback,<sup><a reveal-id="DSV001C19" data-open="DSV001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> rainbow trout,<sup><a reveal-id="DSV001C20" data-open="DSV001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> eggplant,<sup><a reveal-id="DSV001C21" data-open="DSV001C21" class="link link-ref link-reveal xref-bibr">21</a></sup> barley,<sup><a reveal-id="DSV001C22" data-open="DSV001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> chickpea,<sup><a reveal-id="DSV001C23" data-open="DSV001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> guppy,<sup><a reveal-id="DSV001C24" data-open="DSV001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and lupin.<sup><a reveal-id="DSV001C25" data-open="DSV001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Furthermore, the discovery that replicate parallel phenotypic evolution may be occurring in stickleback on a genome-wide scale<sup><a reveal-id="DSV001C26" data-open="DSV001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> as well as the detection of population admixture and improved identification of hybrid and non-hybridized individuals using RAD-seq<sup><a reveal-id="DSV001C20" data-open="DSV001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> suggest that this technique is applicable to population genomics and phylogeography.</p><p>Despite the availability of two generations of genetic maps, the large-scale discovery and utilization of SNPs, the most promising markers for providing sufficiently dense genetic maps,<sup><a reveal-id="DSV001C23" data-open="DSV001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> has not been carried out in Japanese flounder. In the present study, we consequently performed a large-scale identification of genome-wide SNPs derived from RAD-seq of a Japanese flounder mapping population containing 2 parents and 216 offspring. We then used the results to construct a high-density (third generation) SNP-based map. We also successfully detected QTLs for <em>Vibrio anguillarum</em> disease resistance and their related genes in Japanese flounder. Finally, we used our SNP-based genetic map in conjunction with the <em>P. olivaceus</em> Genome Sequencing Project (PoGSP) database, which is to be published in the near future. In particular, the map was used to facilitate the anchoring and orienting of scaffolds generated by whole genome sequence data assembly and for chromosomal-level comparative analysis.</p>                            <h2 scrollto-destination=4595078 id="4595078" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4595081 id="4595081" class="section-title" data-legacy-id=s2a>2.1. Mapping population</h3>
<p>Twenty-four full-sib families of Japanese flounder were bred by Haiyang Yellow Sea Fisheries Co. (Yantai, China) in 2010.<sup><a reveal-id="DSV001C27" data-open="DSV001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> All 216 offspring of one of the families, with an average length of 12.61 ± 1.40 cm and average weight of 18.85 ± 6.28 g, were challenged through intraperitoneal injection with a 0.2-ml bacterial suspension of ∼7.16 × 10<sup>5</sup> colony-forming units (CFU) of <em>V. anguillarum</em>. Another 20 individuals were injected with 1× phosphate-buffered saline as a control. The median lethal concentration had been determined in our laboratory previously.<sup><a reveal-id="DSV001C28" data-open="DSV001C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The offspring were kept in three 0.28-m<sup>3</sup> tanks supplied with fresh seawater at 18 ± 0.5°C. This challenge experiment was performed once and lasted for ∼1 week. Mortality was recorded every 3 h based on the appearance of dead fish (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Genomic DNA was isolated from the fins of the parental fish and 216 offspring using traditional phenol–chloroform extraction in combination with RNase treatment and stored at −20°C.<sup><a reveal-id="DSV001C10" data-open="DSV001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Before construction of RAD-seq libraries, all DNA samples were quantified using a NanoDrop instrument (Thermo Scientific, Wilmington, DE, USA), and their concentrations were adjusted to 50 ng/µl using Tris–EDTA buffer.</p>                            <h3 scrollto-destination=4595086 id="4595086" class="section-title" data-legacy-id=s2b>2.2. RAD library construction and sequencing</h3>
<p>RAD-seq libraries were constructed using a protocol adapted from Baird <em>et al.</em><sup><a reveal-id="DSV001C19" data-open="DSV001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Briefly, 250 ng of genomic DNA from each of 218 library individuals was digested separately with 20 units of PstI and then heat inactivated at 65°C. Various P1 adapters, each with a unique 4–8 bp molecular-identifying sequence (MID), were then ligated to designated individuals, which were then pooled in groups of 24 individuals and randomly sheared to DNA fragments. Sheared DNA was purified, eluted, and separated using gel electrophoresis, and a DNA fraction corresponding to 300–700 bp was excised and purified. After end repair, purification, and elution, dATP overhangs were added to the DNA fraction. A paired-end P2 adapter containing T overhangs was ligated to 20 μl of sheared, size-selected, P1-ligated, and pooled DNA template with a specific adapter. The ligated material was then purified, eluted, and subjected to PCR enrichment. Sequencing of the RAD products from the 218 individuals was performed on a HiSeq2000 next-generation sequencing platform. Sequencing data for each individual were then extracted according to the specific MID.</p>                            <h3 scrollto-destination=4595091 id="4595091" class="section-title" data-legacy-id=s2c>2.3. SNP discovery and genotyping</h3>
<p>We first filtered out Illumina short reads lacking sample-specific MIDs and expected restriction enzyme motifs. All the short reads from each of the samples were then clustered into tag reads on the basis of sequence similarity (allowing one mismatch at most between any two reads within each tag read cluster, with clusters having &lt;2 or &gt;100 reads discarded).<sup><a reveal-id="DSV001C29" data-open="DSV001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Tag reads from the two parents were compared and filtered to remove monomorphic DNA sequences, leaving only the tag reads with SNPs. SNPs were selected according to the following criteria: (i) alleles with a minimum coverage of five reads and a score &gt;20 (<em>P</em> &gt; 0.05) were selected; (ii) the base at the SNP site was unique (if at a homozygous locus) or bimorphic (if at a heterozygous locus, with the minor base represented by at least three reads); and (iii) the ratio of the two kinds of bases (major to minor) at a heterozygous locus ranged from 1 to 5. Regions containing these putative SNPs were used as reference SNP regions. The clean tag reads of offspring individuals were then aligned to reference SNP regions, with genotypes of individuals determined by reference to the parental genotypes. Filtering of reads and SNP selection were performed following previously described algorithms<sup><a reveal-id="DSV001C29" data-open="DSV001C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSV001C30" data-open="DSV001C30" class="link link-ref link-reveal xref-bibr">30</a></sup> as implemented using custom scripts (available from the authors upon request).</p>                            <h3 scrollto-destination=4595095 id="4595095" class="section-title" data-legacy-id=s2d>2.4. Genetic map construction</h3>
<p>A pseudo-testcross population was used to construct the linkage map. For the linkage analysis, RAD-based SNPs were first tested against the expected segregation ratio. Two SNP alleles heterozygous with respect to the two parents were expected to segregate in a 1:2:1 ratio, whereas a pair of SNP alleles, one heterozygous and the other homozygous, were expected to segregate in a 1:1 ratio. To construct a genetic map, markers showing significant segregation distortion (<em>P</em> &lt; 0.01, <em>χ</em><sup>2</sup> test) were removed. The remaining SNPs were then used to construct the genetic map with the recently developed software package Lep-MAP.<sup><a reveal-id="DSV001C31" data-open="DSV001C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The genotype data were first filtered manually to remove obvious Mendelian errors from the offspring. LG assignments were then obtained using the separate chromosomes module with a logarithm of odds (LOD) score limit of 10. The marker order was obtained using the order markers module of Lep-MAP. Because this module uses only paternally informative markers in the ordering, a second genotype data file was constructed by swapping the parents of the original data. An integrated map was constructed by using both genotype files as input, with paternal and maternal maps generated by using only one input file. To speed up the computation, constant rates for genotype errors and recombinations were used. Finally, the marker positions and error parameters were established by order markers module.</p>                            <h3 scrollto-destination=4595097 id="4595097" class="section-title" data-legacy-id=s2e>2.5. QTL mapping for <em>V. anguillarum</em> disease resistance</h3>
<p>QTL analyses were conducted with the WinQTLCart2.5 software program using the composite interval mapping (CIM) method.<sup><a reveal-id="DSV001C32" data-open="DSV001C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The CIM analysis was run using Model 6 with four parameters for forward and backward stepwise regression, a 10-cM window size, five control markers, and a 1-cM step size. The LOD threshold value was determined on the basis of 1,000 permutations at a whole genome-wide significance level of <em>P</em> &lt; 0.05. The location of each QTL was determined according to its LOD peak location and surrounding region. Other QTL reference values, including phenotypic variation and a positive or negative additive effect for <em>V. anguillarum</em> disease resistance, were also calculated by WinQTLCart2.5. Candidate genes for <em>V. anguillarum</em> disease resistance were identified by mapping the corresponding tags of SNPs in QTL regions to the scaffold assembly followed by retrieval of the corresponding gene ID from the gene annotation file.</p>                            <h3 scrollto-destination=4595099 id="4595099" class="section-title" data-legacy-id=s2f>2.6. Genome scaffold assembly and genome synteny</h3>
<p>We used the high-resolution RAD-based SNP genetic map and BLASTN (<em>E</em>-value &lt; 1 × 10<sup>−5</sup>, identity ≥95%, and alignment rate &gt;50%) to map reads containing SNPs with less than two mismatches to PoGSP-derived scaffolds. To reduce the complexity of scaffolds anchored to hundreds of SNP markers, a tag SNP was selected from each scaffold with multiple SNPs. The position and orientation of scaffolds were ordered and assembled onto 24 pseudo-chromosomes corresponding to the 24 LGs based on genetic distances between SNPs. Scaffolds with only one SNP marker could be anchored but not oriented because of the lack of markers. We then used the Japanese flounder reference gene set generated from the PoGSP database along with medaka (<em>Oryzias latipes</em>) and zebrafish (<em>Danio rerio</em>) gene sequences from Ensembl (release 57). After filtering short genes (coding sequence &lt;150 bp), we chose the transcripts with the longest coding sequences to represent each gene and then identified reciprocal best-matching orthologous genes between Japanese flounder and the other fish using BLASTP (<em>E</em>-value = 1 × 10<sup>−10</sup>). Syntenic regions defined by the top hits of the homology search were plotted using an in-house script.</p>                            <h2 scrollto-destination=4595101 id="4595101" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4595102 id="4595102" class="section-title" data-legacy-id=s3a>3.1. RAD-seq library construction and sequencing</h3>
<p>A total of 218 RAD-seq libraries from 2 parents and their 216 offspring were constructed and sequenced on an Illumina HiSeq2000 platform to generate 3.91 billion raw reads. After data trimming, 3.52 billion reads, comprising ∼149.87 Gb of sequencing data, were individually partitioned into RAD tags according to their MIDs. Finally, female and male parental data sets, containing respectively 22.93 million filtered reads (comprising 947.96 Mb of data with a GC% of 47.02) and 17.7 million filtered reads (comprising 796.52 Mb of data with a GC% of 47.21), were correspondingly partitioned into 6,294,857 and 4,843,594 RAD tags. From the 216 offspring, a total of 3.48 billion filtered reads (average of 16.12 million) corresponding to 148,128.63 Mb of data (average of 685.78 Mb) were produced and divided into 939,770,718 RAD tags (ranging from 1,627,621 to 8,329,563 with an average of 4,350,790) for individual SNP discovery (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p>                            <h3 scrollto-destination=4595104 id="4595104" class="section-title" data-legacy-id=s3b>3.2. SNP discovery and genotyping</h3>
<p>After stringent selection according to the above-described method, RAD tags from each individual were clustered and compared. RAD tags containing SNPs of the two parents were identified using a custom k-mer matching algorithm that excluded exact sequence matches (monomorphic loci) per 41-bp sequence. A panel of 13,362 high-fidelity SNPs with fixed genotypes in both parents was identified using these criteria described under ‘Material and methods’, and alleles for each marker were assigned to their respective parental donor. Using an in-house script, these SNPs were then analysed using genotyping data across the 216 offspring and classified into three categories: maternal heterozygous (6,741 SNPs), paternal heterozygous (4,128 SNPs), and heterozygous in both (2,493 SNPs). All 13,362 SNPs and their flanking sequences are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h3 scrollto-destination=4595106 id="4595106" class="section-title" data-legacy-id=s3c>3.3. High-resolution genetic map construction</h3>
<p>A high-resolution RAD-based SNP genetic map of Japanese flounder based on a pseudo-testcross population was first constructed using Lep-MAP. A total of 12,712 segregating SNPs (95.14%) were successfully classified into 24 LGs (Table <a reveal-id="DSV001TB1" data-open="DSV001TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSV001F1" data-open="DSV001F1" class="link link-reveal link-table xref-fig">1</a>). The paternal map contained 8,784 SNPs with a total genetic distance of 2,561.17 cM; the length of each LG ranged from 19.53 cM (LG8) to 145.08 cM (LG19), with an average genetic length of 106.72 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The corresponding maternal map consisted of 6,343 SNPs representing a total length of 2,356.86 cM and ranging from 0.6 cM (LG8) to 187.33 cM (LG1) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). More interestingly, LG8 was the smallest LG on both maternal (0.6 cM) and paternal maps (19.53 cM). LG8 on the maternal map contained only 3 SNPs heterozygous in both parents; on the paternal map, LG8 included 135 paternal heterozygous SNPs and 3 SNPs heterozygous in both parents. The resulting integrated map consisted of 24 LGs including 12,712 SNPs, which corresponded to 7,430 effective loci. The total map length was 3,497.29 cM, with an average inter-locus distance of 0.47 cM; the genetic length of each LG ranged from 21.64 cM (LG8) to 221.14 cM (LG20), with an average inter-locus distance of 0.37–0.75 cM. LG1 was the densest, having 495 effective loci with an average density of 0.42 cM, whereas LG8 had the least number of effective loci (only 59). On average, each LG contained 310 effective loci spanning 145.72 cM (Table <a reveal-id="DSV001TB1" data-open="DSV001TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSV001F1" data-open="DSV001F1" class="link link-reveal link-table xref-fig">1</a>). Locus names and SNP positions on the 24 LGs of the integrated genetic map are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. </p>                        <a id="4595108" scrollto-destination="4595108"></a>
<div content-id="DSV001TB1" class="table-modal"><div class="table-wrap"><div id="DSV001TB1" class="table-wrap-title" data-id="DSV001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of genetic maps and anchoring scaffolds of Japanese flounder</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LG_ID </th><th colspan="2">Paternal map<hr /> </th><th colspan="2">Maternal map<hr /> </th><th colspan="6">Integrated_Map<hr /> </th></tr><tr><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>No. of effective loci </th><th>Distance (cM) </th><th>Average inter-loci distance </th><th>No. of anchored scaffolds </th><th>Length of anchored scaffolds (Mb) </th></tr></thead><tbody><tr><td>1 </td><td>493 </td><td>137.15 </td><td>561 </td><td>187.33 </td><td>875 </td><td>495 </td><td>208.74 </td><td>0.42 </td><td>16 </td><td>27.81 </td></tr><tr><td>2 </td><td>75 </td><td>46.95 </td><td>94 </td><td>9.03 </td><td>127 </td><td>72 </td><td>36.33 </td><td>0.50 </td><td>9 </td><td>18.60 </td></tr><tr><td>3 </td><td>451 </td><td>111.49 </td><td>375 </td><td>137.75 </td><td>706 </td><td>407 </td><td>190.32 </td><td>0.47 </td><td>10 </td><td>23.21 </td></tr><tr><td>4 </td><td>321 </td><td>97.84 </td><td>284 </td><td>95.24 </td><td>527 </td><td>290 </td><td>142.28 </td><td>0.49 </td><td>7 </td><td>18.98 </td></tr><tr><td>5 </td><td>413 </td><td>101.92 </td><td>216 </td><td>80.38 </td><td>584 </td><td>303 </td><td>114.34 </td><td>0.38 </td><td>10 </td><td>25.85 </td></tr><tr><td>6 </td><td>441 </td><td>126.54 </td><td>442 </td><td>181.24 </td><td>742 </td><td>440 </td><td>203.21 </td><td>0.46 </td><td>9 </td><td>24.62 </td></tr><tr><td>7 </td><td>224 </td><td>94.54 </td><td>76 </td><td>72.64 </td><td>253 </td><td>178 </td><td>94.76 </td><td>0.53 </td><td>13 </td><td>29.55 </td></tr><tr><td>8 </td><td>138 </td><td>19.53 </td><td>3 </td><td>0.60 </td><td>138 </td><td>59 </td><td>21.64 </td><td>0.37 </td><td>8 </td><td>11.10 </td></tr><tr><td>9 </td><td>377 </td><td>103.89 </td><td>46 </td><td>9.00 </td><td>407 </td><td>195 </td><td>101.09 </td><td>0.52 </td><td>19 </td><td>14.86 </td></tr><tr><td>10 </td><td>222 </td><td>88.72 </td><td>151 </td><td>76.56 </td><td>333 </td><td>214 </td><td>123.10 </td><td>0.58 </td><td>12 </td><td>21.85 </td></tr><tr><td>11 </td><td>263 </td><td>82.11 </td><td>385 </td><td>127.91 </td><td>573 </td><td>351 </td><td>177.62 </td><td>0.51 </td><td>8 </td><td>22.77 </td></tr><tr><td>12 </td><td>452 </td><td>102.60 </td><td>461 </td><td>121.51 </td><td>787 </td><td>436 </td><td>161.24 </td><td>0.37 </td><td>10 </td><td>25.65 </td></tr><tr><td>13 </td><td>410 </td><td>130.09 </td><td>412 </td><td>107.02 </td><td>686 </td><td>401 </td><td>172.96 </td><td>0.43 </td><td>5 </td><td>19.78 </td></tr><tr><td>14 </td><td>313 </td><td>117.59 </td><td>260 </td><td>88.67 </td><td>421 </td><td>276 </td><td>128.28 </td><td>0.46 </td><td>11 </td><td>11.15 </td></tr><tr><td>15 </td><td>486 </td><td>122.12 </td><td>151 </td><td>91.38 </td><td>509 </td><td>297 </td><td>150.71 </td><td>0.51 </td><td>10 </td><td>23.80 </td></tr><tr><td>16 </td><td>175 </td><td>72.03 </td><td>294 </td><td>100.37 </td><td>402 </td><td>253 </td><td>119.09 </td><td>0.47 </td><td>12 </td><td>17.16 </td></tr><tr><td>17 </td><td>408 </td><td>136.93 </td><td>492 </td><td>169.11 </td><td>717 </td><td>462 </td><td>209.14 </td><td>0.45 </td><td>11 </td><td>25.19 </td></tr><tr><td>18 </td><td>276 </td><td>116.96 </td><td>99 </td><td>84.30 </td><td>287 </td><td>192 </td><td>144.18 </td><td>0.75 </td><td>8 </td><td>19.41 </td></tr><tr><td>19 </td><td>458 </td><td>145.08 </td><td>174 </td><td>97.10 </td><td>484 </td><td>302 </td><td>169.02 </td><td>0.56 </td><td>12 </td><td>26.39 </td></tr><tr><td>20 </td><td>528 </td><td>114.33 </td><td>464 </td><td>159.35 </td><td>824 </td><td>467 </td><td>221.14 </td><td>0.47 </td><td>6 </td><td>22.92 </td></tr><tr><td>21 </td><td>449 </td><td>127.47 </td><td>103 </td><td>32.21 </td><td>492 </td><td>247 </td><td>101.46 </td><td>0.41 </td><td>6 </td><td>25.31 </td></tr><tr><td>22 </td><td>448 </td><td>102.11 </td><td>260 </td><td>106.97 </td><td>590 </td><td>380 </td><td>157.75 </td><td>0.42 </td><td>10 </td><td>19.99 </td></tr><tr><td>23 </td><td>514 </td><td>141.20 </td><td>220 </td><td>101.53 </td><td>603 </td><td>336 </td><td>173.14 </td><td>0.52 </td><td>12 </td><td>24.25 </td></tr><tr><td>24 </td><td>449 </td><td>121.98 </td><td>320 </td><td>119.66 </td><td>645 </td><td>377 </td><td>175.75 </td><td>0.47 </td><td>12 </td><td>22.80 </td></tr><tr><td>Total </td><td>8,784 </td><td>2,561.17 </td><td>6,343 </td><td>2,356.86 </td><td>12,712 </td><td>7,430 </td><td>3,497.29 </td><td>0.47 </td><td>246 </td><td>522.99 </td></tr><tr><td>Average </td><td>366 </td><td>106.72 </td><td>264 </td><td>98.20 </td><td>530 </td><td>310 </td><td>145.72 </td><td>0.47 </td><td>10 </td><td>21.79 </td></tr></tbody></table></div><div content-id="DSV001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LG_ID </th><th colspan="2">Paternal map<hr /> </th><th colspan="2">Maternal map<hr /> </th><th colspan="6">Integrated_Map<hr /> </th></tr><tr><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>No. of effective loci </th><th>Distance (cM) </th><th>Average inter-loci distance </th><th>No. of anchored scaffolds </th><th>Length of anchored scaffolds (Mb) </th></tr></thead><tbody><tr><td>1 </td><td>493 </td><td>137.15 </td><td>561 </td><td>187.33 </td><td>875 </td><td>495 </td><td>208.74 </td><td>0.42 </td><td>16 </td><td>27.81 </td></tr><tr><td>2 </td><td>75 </td><td>46.95 </td><td>94 </td><td>9.03 </td><td>127 </td><td>72 </td><td>36.33 </td><td>0.50 </td><td>9 </td><td>18.60 </td></tr><tr><td>3 </td><td>451 </td><td>111.49 </td><td>375 </td><td>137.75 </td><td>706 </td><td>407 </td><td>190.32 </td><td>0.47 </td><td>10 </td><td>23.21 </td></tr><tr><td>4 </td><td>321 </td><td>97.84 </td><td>284 </td><td>95.24 </td><td>527 </td><td>290 </td><td>142.28 </td><td>0.49 </td><td>7 </td><td>18.98 </td></tr><tr><td>5 </td><td>413 </td><td>101.92 </td><td>216 </td><td>80.38 </td><td>584 </td><td>303 </td><td>114.34 </td><td>0.38 </td><td>10 </td><td>25.85 </td></tr><tr><td>6 </td><td>441 </td><td>126.54 </td><td>442 </td><td>181.24 </td><td>742 </td><td>440 </td><td>203.21 </td><td>0.46 </td><td>9 </td><td>24.62 </td></tr><tr><td>7 </td><td>224 </td><td>94.54 </td><td>76 </td><td>72.64 </td><td>253 </td><td>178 </td><td>94.76 </td><td>0.53 </td><td>13 </td><td>29.55 </td></tr><tr><td>8 </td><td>138 </td><td>19.53 </td><td>3 </td><td>0.60 </td><td>138 </td><td>59 </td><td>21.64 </td><td>0.37 </td><td>8 </td><td>11.10 </td></tr><tr><td>9 </td><td>377 </td><td>103.89 </td><td>46 </td><td>9.00 </td><td>407 </td><td>195 </td><td>101.09 </td><td>0.52 </td><td>19 </td><td>14.86 </td></tr><tr><td>10 </td><td>222 </td><td>88.72 </td><td>151 </td><td>76.56 </td><td>333 </td><td>214 </td><td>123.10 </td><td>0.58 </td><td>12 </td><td>21.85 </td></tr><tr><td>11 </td><td>263 </td><td>82.11 </td><td>385 </td><td>127.91 </td><td>573 </td><td>351 </td><td>177.62 </td><td>0.51 </td><td>8 </td><td>22.77 </td></tr><tr><td>12 </td><td>452 </td><td>102.60 </td><td>461 </td><td>121.51 </td><td>787 </td><td>436 </td><td>161.24 </td><td>0.37 </td><td>10 </td><td>25.65 </td></tr><tr><td>13 </td><td>410 </td><td>130.09 </td><td>412 </td><td>107.02 </td><td>686 </td><td>401 </td><td>172.96 </td><td>0.43 </td><td>5 </td><td>19.78 </td></tr><tr><td>14 </td><td>313 </td><td>117.59 </td><td>260 </td><td>88.67 </td><td>421 </td><td>276 </td><td>128.28 </td><td>0.46 </td><td>11 </td><td>11.15 </td></tr><tr><td>15 </td><td>486 </td><td>122.12 </td><td>151 </td><td>91.38 </td><td>509 </td><td>297 </td><td>150.71 </td><td>0.51 </td><td>10 </td><td>23.80 </td></tr><tr><td>16 </td><td>175 </td><td>72.03 </td><td>294 </td><td>100.37 </td><td>402 </td><td>253 </td><td>119.09 </td><td>0.47 </td><td>12 </td><td>17.16 </td></tr><tr><td>17 </td><td>408 </td><td>136.93 </td><td>492 </td><td>169.11 </td><td>717 </td><td>462 </td><td>209.14 </td><td>0.45 </td><td>11 </td><td>25.19 </td></tr><tr><td>18 </td><td>276 </td><td>116.96 </td><td>99 </td><td>84.30 </td><td>287 </td><td>192 </td><td>144.18 </td><td>0.75 </td><td>8 </td><td>19.41 </td></tr><tr><td>19 </td><td>458 </td><td>145.08 </td><td>174 </td><td>97.10 </td><td>484 </td><td>302 </td><td>169.02 </td><td>0.56 </td><td>12 </td><td>26.39 </td></tr><tr><td>20 </td><td>528 </td><td>114.33 </td><td>464 </td><td>159.35 </td><td>824 </td><td>467 </td><td>221.14 </td><td>0.47 </td><td>6 </td><td>22.92 </td></tr><tr><td>21 </td><td>449 </td><td>127.47 </td><td>103 </td><td>32.21 </td><td>492 </td><td>247 </td><td>101.46 </td><td>0.41 </td><td>6 </td><td>25.31 </td></tr><tr><td>22 </td><td>448 </td><td>102.11 </td><td>260 </td><td>106.97 </td><td>590 </td><td>380 </td><td>157.75 </td><td>0.42 </td><td>10 </td><td>19.99 </td></tr><tr><td>23 </td><td>514 </td><td>141.20 </td><td>220 </td><td>101.53 </td><td>603 </td><td>336 </td><td>173.14 </td><td>0.52 </td><td>12 </td><td>24.25 </td></tr><tr><td>24 </td><td>449 </td><td>121.98 </td><td>320 </td><td>119.66 </td><td>645 </td><td>377 </td><td>175.75 </td><td>0.47 </td><td>12 </td><td>22.80 </td></tr><tr><td>Total </td><td>8,784 </td><td>2,561.17 </td><td>6,343 </td><td>2,356.86 </td><td>12,712 </td><td>7,430 </td><td>3,497.29 </td><td>0.47 </td><td>246 </td><td>522.99 </td></tr><tr><td>Average </td><td>366 </td><td>106.72 </td><td>264 </td><td>98.20 </td><td>530 </td><td>310 </td><td>145.72 </td><td>0.47 </td><td>10 </td><td>21.79 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4595108" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV001TB1" class="table-wrap-title" data-id="DSV001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of genetic maps and anchoring scaffolds of Japanese flounder</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LG_ID </th><th colspan="2">Paternal map<hr /> </th><th colspan="2">Maternal map<hr /> </th><th colspan="6">Integrated_Map<hr /> </th></tr><tr><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>No. of effective loci </th><th>Distance (cM) </th><th>Average inter-loci distance </th><th>No. of anchored scaffolds </th><th>Length of anchored scaffolds (Mb) </th></tr></thead><tbody><tr><td>1 </td><td>493 </td><td>137.15 </td><td>561 </td><td>187.33 </td><td>875 </td><td>495 </td><td>208.74 </td><td>0.42 </td><td>16 </td><td>27.81 </td></tr><tr><td>2 </td><td>75 </td><td>46.95 </td><td>94 </td><td>9.03 </td><td>127 </td><td>72 </td><td>36.33 </td><td>0.50 </td><td>9 </td><td>18.60 </td></tr><tr><td>3 </td><td>451 </td><td>111.49 </td><td>375 </td><td>137.75 </td><td>706 </td><td>407 </td><td>190.32 </td><td>0.47 </td><td>10 </td><td>23.21 </td></tr><tr><td>4 </td><td>321 </td><td>97.84 </td><td>284 </td><td>95.24 </td><td>527 </td><td>290 </td><td>142.28 </td><td>0.49 </td><td>7 </td><td>18.98 </td></tr><tr><td>5 </td><td>413 </td><td>101.92 </td><td>216 </td><td>80.38 </td><td>584 </td><td>303 </td><td>114.34 </td><td>0.38 </td><td>10 </td><td>25.85 </td></tr><tr><td>6 </td><td>441 </td><td>126.54 </td><td>442 </td><td>181.24 </td><td>742 </td><td>440 </td><td>203.21 </td><td>0.46 </td><td>9 </td><td>24.62 </td></tr><tr><td>7 </td><td>224 </td><td>94.54 </td><td>76 </td><td>72.64 </td><td>253 </td><td>178 </td><td>94.76 </td><td>0.53 </td><td>13 </td><td>29.55 </td></tr><tr><td>8 </td><td>138 </td><td>19.53 </td><td>3 </td><td>0.60 </td><td>138 </td><td>59 </td><td>21.64 </td><td>0.37 </td><td>8 </td><td>11.10 </td></tr><tr><td>9 </td><td>377 </td><td>103.89 </td><td>46 </td><td>9.00 </td><td>407 </td><td>195 </td><td>101.09 </td><td>0.52 </td><td>19 </td><td>14.86 </td></tr><tr><td>10 </td><td>222 </td><td>88.72 </td><td>151 </td><td>76.56 </td><td>333 </td><td>214 </td><td>123.10 </td><td>0.58 </td><td>12 </td><td>21.85 </td></tr><tr><td>11 </td><td>263 </td><td>82.11 </td><td>385 </td><td>127.91 </td><td>573 </td><td>351 </td><td>177.62 </td><td>0.51 </td><td>8 </td><td>22.77 </td></tr><tr><td>12 </td><td>452 </td><td>102.60 </td><td>461 </td><td>121.51 </td><td>787 </td><td>436 </td><td>161.24 </td><td>0.37 </td><td>10 </td><td>25.65 </td></tr><tr><td>13 </td><td>410 </td><td>130.09 </td><td>412 </td><td>107.02 </td><td>686 </td><td>401 </td><td>172.96 </td><td>0.43 </td><td>5 </td><td>19.78 </td></tr><tr><td>14 </td><td>313 </td><td>117.59 </td><td>260 </td><td>88.67 </td><td>421 </td><td>276 </td><td>128.28 </td><td>0.46 </td><td>11 </td><td>11.15 </td></tr><tr><td>15 </td><td>486 </td><td>122.12 </td><td>151 </td><td>91.38 </td><td>509 </td><td>297 </td><td>150.71 </td><td>0.51 </td><td>10 </td><td>23.80 </td></tr><tr><td>16 </td><td>175 </td><td>72.03 </td><td>294 </td><td>100.37 </td><td>402 </td><td>253 </td><td>119.09 </td><td>0.47 </td><td>12 </td><td>17.16 </td></tr><tr><td>17 </td><td>408 </td><td>136.93 </td><td>492 </td><td>169.11 </td><td>717 </td><td>462 </td><td>209.14 </td><td>0.45 </td><td>11 </td><td>25.19 </td></tr><tr><td>18 </td><td>276 </td><td>116.96 </td><td>99 </td><td>84.30 </td><td>287 </td><td>192 </td><td>144.18 </td><td>0.75 </td><td>8 </td><td>19.41 </td></tr><tr><td>19 </td><td>458 </td><td>145.08 </td><td>174 </td><td>97.10 </td><td>484 </td><td>302 </td><td>169.02 </td><td>0.56 </td><td>12 </td><td>26.39 </td></tr><tr><td>20 </td><td>528 </td><td>114.33 </td><td>464 </td><td>159.35 </td><td>824 </td><td>467 </td><td>221.14 </td><td>0.47 </td><td>6 </td><td>22.92 </td></tr><tr><td>21 </td><td>449 </td><td>127.47 </td><td>103 </td><td>32.21 </td><td>492 </td><td>247 </td><td>101.46 </td><td>0.41 </td><td>6 </td><td>25.31 </td></tr><tr><td>22 </td><td>448 </td><td>102.11 </td><td>260 </td><td>106.97 </td><td>590 </td><td>380 </td><td>157.75 </td><td>0.42 </td><td>10 </td><td>19.99 </td></tr><tr><td>23 </td><td>514 </td><td>141.20 </td><td>220 </td><td>101.53 </td><td>603 </td><td>336 </td><td>173.14 </td><td>0.52 </td><td>12 </td><td>24.25 </td></tr><tr><td>24 </td><td>449 </td><td>121.98 </td><td>320 </td><td>119.66 </td><td>645 </td><td>377 </td><td>175.75 </td><td>0.47 </td><td>12 </td><td>22.80 </td></tr><tr><td>Total </td><td>8,784 </td><td>2,561.17 </td><td>6,343 </td><td>2,356.86 </td><td>12,712 </td><td>7,430 </td><td>3,497.29 </td><td>0.47 </td><td>246 </td><td>522.99 </td></tr><tr><td>Average </td><td>366 </td><td>106.72 </td><td>264 </td><td>98.20 </td><td>530 </td><td>310 </td><td>145.72 </td><td>0.47 </td><td>10 </td><td>21.79 </td></tr></tbody></table></div><div content-id="DSV001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LG_ID </th><th colspan="2">Paternal map<hr /> </th><th colspan="2">Maternal map<hr /> </th><th colspan="6">Integrated_Map<hr /> </th></tr><tr><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>Distance (cM) </th><th>No. of SNPs </th><th>No. of effective loci </th><th>Distance (cM) </th><th>Average inter-loci distance </th><th>No. of anchored scaffolds </th><th>Length of anchored scaffolds (Mb) </th></tr></thead><tbody><tr><td>1 </td><td>493 </td><td>137.15 </td><td>561 </td><td>187.33 </td><td>875 </td><td>495 </td><td>208.74 </td><td>0.42 </td><td>16 </td><td>27.81 </td></tr><tr><td>2 </td><td>75 </td><td>46.95 </td><td>94 </td><td>9.03 </td><td>127 </td><td>72 </td><td>36.33 </td><td>0.50 </td><td>9 </td><td>18.60 </td></tr><tr><td>3 </td><td>451 </td><td>111.49 </td><td>375 </td><td>137.75 </td><td>706 </td><td>407 </td><td>190.32 </td><td>0.47 </td><td>10 </td><td>23.21 </td></tr><tr><td>4 </td><td>321 </td><td>97.84 </td><td>284 </td><td>95.24 </td><td>527 </td><td>290 </td><td>142.28 </td><td>0.49 </td><td>7 </td><td>18.98 </td></tr><tr><td>5 </td><td>413 </td><td>101.92 </td><td>216 </td><td>80.38 </td><td>584 </td><td>303 </td><td>114.34 </td><td>0.38 </td><td>10 </td><td>25.85 </td></tr><tr><td>6 </td><td>441 </td><td>126.54 </td><td>442 </td><td>181.24 </td><td>742 </td><td>440 </td><td>203.21 </td><td>0.46 </td><td>9 </td><td>24.62 </td></tr><tr><td>7 </td><td>224 </td><td>94.54 </td><td>76 </td><td>72.64 </td><td>253 </td><td>178 </td><td>94.76 </td><td>0.53 </td><td>13 </td><td>29.55 </td></tr><tr><td>8 </td><td>138 </td><td>19.53 </td><td>3 </td><td>0.60 </td><td>138 </td><td>59 </td><td>21.64 </td><td>0.37 </td><td>8 </td><td>11.10 </td></tr><tr><td>9 </td><td>377 </td><td>103.89 </td><td>46 </td><td>9.00 </td><td>407 </td><td>195 </td><td>101.09 </td><td>0.52 </td><td>19 </td><td>14.86 </td></tr><tr><td>10 </td><td>222 </td><td>88.72 </td><td>151 </td><td>76.56 </td><td>333 </td><td>214 </td><td>123.10 </td><td>0.58 </td><td>12 </td><td>21.85 </td></tr><tr><td>11 </td><td>263 </td><td>82.11 </td><td>385 </td><td>127.91 </td><td>573 </td><td>351 </td><td>177.62 </td><td>0.51 </td><td>8 </td><td>22.77 </td></tr><tr><td>12 </td><td>452 </td><td>102.60 </td><td>461 </td><td>121.51 </td><td>787 </td><td>436 </td><td>161.24 </td><td>0.37 </td><td>10 </td><td>25.65 </td></tr><tr><td>13 </td><td>410 </td><td>130.09 </td><td>412 </td><td>107.02 </td><td>686 </td><td>401 </td><td>172.96 </td><td>0.43 </td><td>5 </td><td>19.78 </td></tr><tr><td>14 </td><td>313 </td><td>117.59 </td><td>260 </td><td>88.67 </td><td>421 </td><td>276 </td><td>128.28 </td><td>0.46 </td><td>11 </td><td>11.15 </td></tr><tr><td>15 </td><td>486 </td><td>122.12 </td><td>151 </td><td>91.38 </td><td>509 </td><td>297 </td><td>150.71 </td><td>0.51 </td><td>10 </td><td>23.80 </td></tr><tr><td>16 </td><td>175 </td><td>72.03 </td><td>294 </td><td>100.37 </td><td>402 </td><td>253 </td><td>119.09 </td><td>0.47 </td><td>12 </td><td>17.16 </td></tr><tr><td>17 </td><td>408 </td><td>136.93 </td><td>492 </td><td>169.11 </td><td>717 </td><td>462 </td><td>209.14 </td><td>0.45 </td><td>11 </td><td>25.19 </td></tr><tr><td>18 </td><td>276 </td><td>116.96 </td><td>99 </td><td>84.30 </td><td>287 </td><td>192 </td><td>144.18 </td><td>0.75 </td><td>8 </td><td>19.41 </td></tr><tr><td>19 </td><td>458 </td><td>145.08 </td><td>174 </td><td>97.10 </td><td>484 </td><td>302 </td><td>169.02 </td><td>0.56 </td><td>12 </td><td>26.39 </td></tr><tr><td>20 </td><td>528 </td><td>114.33 </td><td>464 </td><td>159.35 </td><td>824 </td><td>467 </td><td>221.14 </td><td>0.47 </td><td>6 </td><td>22.92 </td></tr><tr><td>21 </td><td>449 </td><td>127.47 </td><td>103 </td><td>32.21 </td><td>492 </td><td>247 </td><td>101.46 </td><td>0.41 </td><td>6 </td><td>25.31 </td></tr><tr><td>22 </td><td>448 </td><td>102.11 </td><td>260 </td><td>106.97 </td><td>590 </td><td>380 </td><td>157.75 </td><td>0.42 </td><td>10 </td><td>19.99 </td></tr><tr><td>23 </td><td>514 </td><td>141.20 </td><td>220 </td><td>101.53 </td><td>603 </td><td>336 </td><td>173.14 </td><td>0.52 </td><td>12 </td><td>24.25 </td></tr><tr><td>24 </td><td>449 </td><td>121.98 </td><td>320 </td><td>119.66 </td><td>645 </td><td>377 </td><td>175.75 </td><td>0.47 </td><td>12 </td><td>22.80 </td></tr><tr><td>Total </td><td>8,784 </td><td>2,561.17 </td><td>6,343 </td><td>2,356.86 </td><td>12,712 </td><td>7,430 </td><td>3,497.29 </td><td>0.47 </td><td>246 </td><td>522.99 </td></tr><tr><td>Average </td><td>366 </td><td>106.72 </td><td>264 </td><td>98.20 </td><td>530 </td><td>310 </td><td>145.72 </td><td>0.47 </td><td>10 </td><td>21.79 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4595108" target="_blank">View Large</a></div></div>                        <a id="4595109" scrollto-destination="4595109"></a>
<div id="DSV001F1" data-id="DSV001F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00101.png?Expires=2147483647&Signature=diMFR7AdvUxPldEBAJOY8DIr5LNlTDdxqDb3PaTLwq~D8Ipy-yi6fIvzOIqWOCDHeEAyZ4Ss5E3LFh9krxXjV7lYp62Pyk5tmgSRpJNgQGUfflr1y3i4oZEwZP5EeTsrsRBeLgmvBjSLg42K3V4j5WuSflxO7NV5g8SUXJxaBwbMsYcgJLcqQojUsbYES470qtKZ3fHpiziNm1Ezau2Jm8rmZ8z~-G7lMccmHQm3IBxeGv1TUWVhsnTD1THL903WOqJEnFJU0VxHs1S~6seMZik99fEv~fmkztEmkT3jvdhdTaSaUyOJnPqwlKSfPrtJLGd6VPCMu8zENoqsC~DIUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage group lengths and marker distribution of the high-resolution restriction site-associated DNA sequencing-based SNP genetic map of Japanese flounder. Within each linkage group, red, blue, and yellow lines, respectively, represent maternal heterozygous SNPs, paternal heterozygous SNPs, and SNPs heterozygous in both parents. Genetic map details are given in Supplementary Table S3." path-from-xml="dsv00101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595109/dsv00101.tiff" path-from-xml="dsv00101.tiff" target="_blank">View large</a><a section="4595109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00101.png?Expires=2147483647&Signature=P4p3lMpHDSGjcuWSjh6zLFAQ350g7YXfOzvkyOVtHvDB7mK8Muie4jQWMLcU4jUBhnfeCC2Z9ND5rreUzlqGImh-w~g5ckEvxhSmXVvIe2ILZ2uOnfVdg-vBfEyGXdjGIdk6IvnPSvIf6U8DyU9ojU91WmkGSI5Da25c-BkyTSiwLQpn1MLVJetdMjLFSpbVKJF2e4TdSI5QOw0pI0s6n~BXF63fLGCLTsmXwmBTvOyMtVVlQ8JY9eUgEvngbH3jrdrggEkIgs~hyDvOd0hzfJVx8-BROnRmzkP7f7NFgXAywWXzSxJwyHgtTNWb0JYVKpzol~TVW-StK8J64A2FXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595109&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage group lengths and marker distribution of the high-resolution restriction site-associated DNA sequencing-based SNP genetic map of Japanese flounder. Within each linkage group, red, blue, and yellow lines, respectively, represent maternal heterozygous SNPs, paternal heterozygous SNPs, and SNPs heterozygous in both parents. Genetic map details are given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>.</p></div></div><div content-id="DSV001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00101.png?Expires=2147483647&Signature=diMFR7AdvUxPldEBAJOY8DIr5LNlTDdxqDb3PaTLwq~D8Ipy-yi6fIvzOIqWOCDHeEAyZ4Ss5E3LFh9krxXjV7lYp62Pyk5tmgSRpJNgQGUfflr1y3i4oZEwZP5EeTsrsRBeLgmvBjSLg42K3V4j5WuSflxO7NV5g8SUXJxaBwbMsYcgJLcqQojUsbYES470qtKZ3fHpiziNm1Ezau2Jm8rmZ8z~-G7lMccmHQm3IBxeGv1TUWVhsnTD1THL903WOqJEnFJU0VxHs1S~6seMZik99fEv~fmkztEmkT3jvdhdTaSaUyOJnPqwlKSfPrtJLGd6VPCMu8zENoqsC~DIUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage group lengths and marker distribution of the high-resolution restriction site-associated DNA sequencing-based SNP genetic map of Japanese flounder. Within each linkage group, red, blue, and yellow lines, respectively, represent maternal heterozygous SNPs, paternal heterozygous SNPs, and SNPs heterozygous in both parents. Genetic map details are given in Supplementary Table S3." path-from-xml="dsv00101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595109/dsv00101.tiff" path-from-xml="dsv00101.tiff" target="_blank">View large</a><a section="4595109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00101.png?Expires=2147483647&Signature=P4p3lMpHDSGjcuWSjh6zLFAQ350g7YXfOzvkyOVtHvDB7mK8Muie4jQWMLcU4jUBhnfeCC2Z9ND5rreUzlqGImh-w~g5ckEvxhSmXVvIe2ILZ2uOnfVdg-vBfEyGXdjGIdk6IvnPSvIf6U8DyU9ojU91WmkGSI5Da25c-BkyTSiwLQpn1MLVJetdMjLFSpbVKJF2e4TdSI5QOw0pI0s6n~BXF63fLGCLTsmXwmBTvOyMtVVlQ8JY9eUgEvngbH3jrdrggEkIgs~hyDvOd0hzfJVx8-BROnRmzkP7f7NFgXAywWXzSxJwyHgtTNWb0JYVKpzol~TVW-StK8J64A2FXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595109&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage group lengths and marker distribution of the high-resolution restriction site-associated DNA sequencing-based SNP genetic map of Japanese flounder. Within each linkage group, red, blue, and yellow lines, respectively, represent maternal heterozygous SNPs, paternal heterozygous SNPs, and SNPs heterozygous in both parents. Genetic map details are given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>.</p></div></div>                            <h3 scrollto-destination=4595110 id="4595110" class="section-title" data-legacy-id=s3d>3.4. <em>Vibrio anguillarum</em> disease resistance-associated QTLs and related genes</h3>
<p>In total, 10 significant QTLs for <em>V. anguillarum</em> disease resistance were distributed on LG6, LG19, and LG21 of Japanese flounder (Table <a reveal-id="DSV001TB2" data-open="DSV001TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSV001F2" data-open="DSV001F2" class="link link-reveal link-table xref-fig">2</a>). Most of these QTLs were clustered together on their respective LGs. One major cluster containing six QTLs (qVA-1, qVA-2, qVA-3, qVA-4, qVA-5, and qVA-6) was detected between the narrow positions of 89.8–108.7 cM on LG6. Among them, qVA-2 located at 95.9–99.3 cM had the highest LOD value, 5, and correspondingly had the highest contribution to phenotypic variation, 8.38%. The other QTLs on LG6 were detected at positions 89.8–90.8, 100.7–102.5, 104.1–105.0, 105.0–105.4, and 107.8–108.7 cM, with LOD values of 3.0–3.6 and contributions to phenotypic variation of 5.1–7.59% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). On LG21, another cluster situated within a short region (98.7–101.3 cM) consisted of three QTLs (qVA-8, qVA-9, and qVA-10) with a LOD value of 3.3–3.7 and was able to explain 6.06–6.56% of the phenotypic variation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Finally, qVA-7 containing a single SNP was centred around 115.6 cM on LG 19; it had a LOD value of 15.8 and explained 1.19% of the phenotypic variation. Although these 10 QTLs explained &gt;59.25% of the total phenotypic variation, no major loci (explaining &gt;20% of the total variation) were detected. The fact that these loci do not independently have higher contributions to such a complicated trait is not unexpected. Several genes related to <em>V. anguillarum</em> disease resistance were identified from the higher density map based on the Japanese flounder genome assembly. As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>, 218 genes were identified from the QTL regions, 12 of which were immune-related genes (<em>tap1</em>, <em>rftn1</em>, <em>satb1</em>, <em>cd40</em>, <em>cd69</em>, <em>aicda</em>, <em>mtss1</em>, <em>ccr4</em>, <em>azi2</em>, <em>mrc1</em>, <em>nod1</em>, and <em>tgfbr2</em>) functioning as key factors in different immune-gene signalling pathways or as antigen receptors. Interestingly, <em>tap1</em> and <em>satb1</em> act as molecular organizers for the major histocompatibility complex (MHC) class I,<sup><a reveal-id="DSV001C33" data-open="DSV001C33" class="link link-ref link-reveal xref-bibr">33</a></sup> while <em>cd40</em> and <em>cd69</em> are accessory molecules of MHC class II<sup><a reveal-id="DSV001C34" data-open="DSV001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> directly involved in <em>V. anguillarum</em> disease resistance in Japanese flounder. </p>                        <a id="4595112" scrollto-destination="4595112"></a>
<div content-id="DSV001TB2" class="table-modal"><div class="table-wrap"><div id="DSV001TB2" class="table-wrap-title" data-id="DSV001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Characteristics of <em>Vibrio anguillarum</em> disease resistance QTLs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>QTL </th><th>Linkage group </th><th>Genetic position </th><th>Associated marker </th><th>LOD </th><th><sup>a</sup>Exp% </th><th>Additive effect </th></tr></thead><tbody><tr><td>qVA-1 </td><td>LG6 </td><td>89.8–90.8 </td><td>record_231777.7 </td><td>3 </td><td>5.10 </td><td>0.21 </td></tr><tr><td>qVA-2 </td><td>LG6 </td><td>95.9–99.3 </td><td>record_245752.29 </td><td>5 </td><td>8.38 </td><td>0.31 </td></tr><tr><td>qVA-3 </td><td>LG6 </td><td>100.7–102.5 </td><td>record_247030.22 </td><td>3.6 </td><td>7.59 </td><td>0.29 </td></tr><tr><td>qVA-4 </td><td>LG6 </td><td>104.1–105 </td><td>record_246301.16 </td><td>3.3 </td><td>5.87 </td><td>0.26 </td></tr><tr><td>qVA-5 </td><td>LG6 </td><td>105–105.4 </td><td>record_254077.24 </td><td>3.5 </td><td>6.19 </td><td>0.26 </td></tr><tr><td>qVA-6 </td><td>LG6 </td><td>107.8–108.7 </td><td>record_254548.37 </td><td>3.4 </td><td>5.95 </td><td>0.26 </td></tr><tr><td>qVA-7 </td><td>LG19 </td><td>115.6–115.8 </td><td>record_255627.24 </td><td>15.8 </td><td>1.19 </td><td>−0.12 </td></tr><tr><td>qVA-8 </td><td>LG21 </td><td>98.7–99.6 </td><td>record_245734.7 </td><td>3.7 </td><td>6.06 </td><td>−0.25 </td></tr><tr><td>qVA-9 </td><td>LG21 </td><td>100.6–100.9 </td><td>record_255692.21 </td><td>3.7 </td><td>6.56 </td><td>−0.27 </td></tr><tr><td>qVA-10 </td><td>LG21 </td><td>100.9–101.3 </td><td>record_245651.14 </td><td>3.3 </td><td>6.36 </td><td>−0.26 </td></tr></tbody></table></div><div content-id="DSV001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>QTL </th><th>Linkage group </th><th>Genetic position </th><th>Associated marker </th><th>LOD </th><th><sup>a</sup>Exp% </th><th>Additive effect </th></tr></thead><tbody><tr><td>qVA-1 </td><td>LG6 </td><td>89.8–90.8 </td><td>record_231777.7 </td><td>3 </td><td>5.10 </td><td>0.21 </td></tr><tr><td>qVA-2 </td><td>LG6 </td><td>95.9–99.3 </td><td>record_245752.29 </td><td>5 </td><td>8.38 </td><td>0.31 </td></tr><tr><td>qVA-3 </td><td>LG6 </td><td>100.7–102.5 </td><td>record_247030.22 </td><td>3.6 </td><td>7.59 </td><td>0.29 </td></tr><tr><td>qVA-4 </td><td>LG6 </td><td>104.1–105 </td><td>record_246301.16 </td><td>3.3 </td><td>5.87 </td><td>0.26 </td></tr><tr><td>qVA-5 </td><td>LG6 </td><td>105–105.4 </td><td>record_254077.24 </td><td>3.5 </td><td>6.19 </td><td>0.26 </td></tr><tr><td>qVA-6 </td><td>LG6 </td><td>107.8–108.7 </td><td>record_254548.37 </td><td>3.4 </td><td>5.95 </td><td>0.26 </td></tr><tr><td>qVA-7 </td><td>LG19 </td><td>115.6–115.8 </td><td>record_255627.24 </td><td>15.8 </td><td>1.19 </td><td>−0.12 </td></tr><tr><td>qVA-8 </td><td>LG21 </td><td>98.7–99.6 </td><td>record_245734.7 </td><td>3.7 </td><td>6.06 </td><td>−0.25 </td></tr><tr><td>qVA-9 </td><td>LG21 </td><td>100.6–100.9 </td><td>record_255692.21 </td><td>3.7 </td><td>6.56 </td><td>−0.27 </td></tr><tr><td>qVA-10 </td><td>LG21 </td><td>100.9–101.3 </td><td>record_245651.14 </td><td>3.3 </td><td>6.36 </td><td>−0.26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>Exp, percentage of explained phenotypic variation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4595112" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV001TB2" class="table-wrap-title" data-id="DSV001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Characteristics of <em>Vibrio anguillarum</em> disease resistance QTLs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>QTL </th><th>Linkage group </th><th>Genetic position </th><th>Associated marker </th><th>LOD </th><th><sup>a</sup>Exp% </th><th>Additive effect </th></tr></thead><tbody><tr><td>qVA-1 </td><td>LG6 </td><td>89.8–90.8 </td><td>record_231777.7 </td><td>3 </td><td>5.10 </td><td>0.21 </td></tr><tr><td>qVA-2 </td><td>LG6 </td><td>95.9–99.3 </td><td>record_245752.29 </td><td>5 </td><td>8.38 </td><td>0.31 </td></tr><tr><td>qVA-3 </td><td>LG6 </td><td>100.7–102.5 </td><td>record_247030.22 </td><td>3.6 </td><td>7.59 </td><td>0.29 </td></tr><tr><td>qVA-4 </td><td>LG6 </td><td>104.1–105 </td><td>record_246301.16 </td><td>3.3 </td><td>5.87 </td><td>0.26 </td></tr><tr><td>qVA-5 </td><td>LG6 </td><td>105–105.4 </td><td>record_254077.24 </td><td>3.5 </td><td>6.19 </td><td>0.26 </td></tr><tr><td>qVA-6 </td><td>LG6 </td><td>107.8–108.7 </td><td>record_254548.37 </td><td>3.4 </td><td>5.95 </td><td>0.26 </td></tr><tr><td>qVA-7 </td><td>LG19 </td><td>115.6–115.8 </td><td>record_255627.24 </td><td>15.8 </td><td>1.19 </td><td>−0.12 </td></tr><tr><td>qVA-8 </td><td>LG21 </td><td>98.7–99.6 </td><td>record_245734.7 </td><td>3.7 </td><td>6.06 </td><td>−0.25 </td></tr><tr><td>qVA-9 </td><td>LG21 </td><td>100.6–100.9 </td><td>record_255692.21 </td><td>3.7 </td><td>6.56 </td><td>−0.27 </td></tr><tr><td>qVA-10 </td><td>LG21 </td><td>100.9–101.3 </td><td>record_245651.14 </td><td>3.3 </td><td>6.36 </td><td>−0.26 </td></tr></tbody></table></div><div content-id="DSV001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>QTL </th><th>Linkage group </th><th>Genetic position </th><th>Associated marker </th><th>LOD </th><th><sup>a</sup>Exp% </th><th>Additive effect </th></tr></thead><tbody><tr><td>qVA-1 </td><td>LG6 </td><td>89.8–90.8 </td><td>record_231777.7 </td><td>3 </td><td>5.10 </td><td>0.21 </td></tr><tr><td>qVA-2 </td><td>LG6 </td><td>95.9–99.3 </td><td>record_245752.29 </td><td>5 </td><td>8.38 </td><td>0.31 </td></tr><tr><td>qVA-3 </td><td>LG6 </td><td>100.7–102.5 </td><td>record_247030.22 </td><td>3.6 </td><td>7.59 </td><td>0.29 </td></tr><tr><td>qVA-4 </td><td>LG6 </td><td>104.1–105 </td><td>record_246301.16 </td><td>3.3 </td><td>5.87 </td><td>0.26 </td></tr><tr><td>qVA-5 </td><td>LG6 </td><td>105–105.4 </td><td>record_254077.24 </td><td>3.5 </td><td>6.19 </td><td>0.26 </td></tr><tr><td>qVA-6 </td><td>LG6 </td><td>107.8–108.7 </td><td>record_254548.37 </td><td>3.4 </td><td>5.95 </td><td>0.26 </td></tr><tr><td>qVA-7 </td><td>LG19 </td><td>115.6–115.8 </td><td>record_255627.24 </td><td>15.8 </td><td>1.19 </td><td>−0.12 </td></tr><tr><td>qVA-8 </td><td>LG21 </td><td>98.7–99.6 </td><td>record_245734.7 </td><td>3.7 </td><td>6.06 </td><td>−0.25 </td></tr><tr><td>qVA-9 </td><td>LG21 </td><td>100.6–100.9 </td><td>record_255692.21 </td><td>3.7 </td><td>6.56 </td><td>−0.27 </td></tr><tr><td>qVA-10 </td><td>LG21 </td><td>100.9–101.3 </td><td>record_245651.14 </td><td>3.3 </td><td>6.36 </td><td>−0.26 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>Exp, percentage of explained phenotypic variation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4595112" target="_blank">View Large</a></div></div>                        <a id="4595113" scrollto-destination="4595113"></a>
<div id="DSV001F2" data-id="DSV001F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00102.png?Expires=2147483647&Signature=F2N-CheAurF1aDY53a3PGn7Ng-0pAPj3WIIuX~OF1PizA9xTnTq~Y1KzReMxYyrvb7JmCEWPeZUJZcLa-jUGzdno6Dm8J0ZxplroVq5hxg~qdZ5NOzkBG~tZoCQtyO4I1lVlcaLp0rHQX1FFAg~n0KfsYP~bmMEdxwsWVrsBoH4OBjsi3V1YgKTyseC1AomAIZUCOQe-IS1VIfKF1rmxS6VB4pKpyGSACEWNRJbF7NOK4shIzp0e4IfKBNuJXX3~IQDv6ATPLlmoDt5XH2kz-SAOoJIy3CpOrcPpjHxXW-TBklD44V20sLcEdwDRTKK7BZBKKaNEVLH1q~CdZfDH4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic location of QTLs for Vibrio anguillarum disease resistance along the Japanese flounder genome. The blue horizontal line represents a linkage group-wise logarithm of odds (LOD) significance threshold of 3.0." path-from-xml="dsv00102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595113/dsv00102.tiff" path-from-xml="dsv00102.tiff" target="_blank">View large</a><a section="4595113" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00102.png?Expires=2147483647&Signature=dUJlIDiBg3LNIbujr63v7GlumHXKfHicdKdyBFYpKn~JKfsvBFNRJDWzETmTDC2ClPmCFXU93eARHLG34qbEvf4CnYFkKmdE01EEMXCQyKzoaJ7g93jPHrMhQRk6RpKMdHvuxecV4QbFcavsBObnONiVRXBP2SfLC7K20LJpCgwY4zCPusy8UyEw1OaaKRGYjf7xwjE7MrC7cOvghRkwKmegLjewDgcrvOwM22CuH9v6Gg7N7IVEYSQNNGsUdDiLiusOK~0mVXqpA8GsTZMFT-a4BvtRq4qf37VASBuNVtFksEYf4ylvwcvYt1GJnwHKCAIlYP4MAhY69z53YX1IHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595113&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic location of QTLs for <em>Vibrio anguillarum</em> disease resistance along the Japanese flounder genome. The blue horizontal line represents a linkage group-wise logarithm of odds (LOD) significance threshold of 3.0.</p></div></div><div content-id="DSV001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00102.png?Expires=2147483647&Signature=F2N-CheAurF1aDY53a3PGn7Ng-0pAPj3WIIuX~OF1PizA9xTnTq~Y1KzReMxYyrvb7JmCEWPeZUJZcLa-jUGzdno6Dm8J0ZxplroVq5hxg~qdZ5NOzkBG~tZoCQtyO4I1lVlcaLp0rHQX1FFAg~n0KfsYP~bmMEdxwsWVrsBoH4OBjsi3V1YgKTyseC1AomAIZUCOQe-IS1VIfKF1rmxS6VB4pKpyGSACEWNRJbF7NOK4shIzp0e4IfKBNuJXX3~IQDv6ATPLlmoDt5XH2kz-SAOoJIy3CpOrcPpjHxXW-TBklD44V20sLcEdwDRTKK7BZBKKaNEVLH1q~CdZfDH4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic location of QTLs for Vibrio anguillarum disease resistance along the Japanese flounder genome. The blue horizontal line represents a linkage group-wise logarithm of odds (LOD) significance threshold of 3.0." path-from-xml="dsv00102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595113/dsv00102.tiff" path-from-xml="dsv00102.tiff" target="_blank">View large</a><a section="4595113" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00102.png?Expires=2147483647&Signature=dUJlIDiBg3LNIbujr63v7GlumHXKfHicdKdyBFYpKn~JKfsvBFNRJDWzETmTDC2ClPmCFXU93eARHLG34qbEvf4CnYFkKmdE01EEMXCQyKzoaJ7g93jPHrMhQRk6RpKMdHvuxecV4QbFcavsBObnONiVRXBP2SfLC7K20LJpCgwY4zCPusy8UyEw1OaaKRGYjf7xwjE7MrC7cOvghRkwKmegLjewDgcrvOwM22CuH9v6Gg7N7IVEYSQNNGsUdDiLiusOK~0mVXqpA8GsTZMFT-a4BvtRq4qf37VASBuNVtFksEYf4ylvwcvYt1GJnwHKCAIlYP4MAhY69z53YX1IHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595113&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic location of QTLs for <em>Vibrio anguillarum</em> disease resistance along the Japanese flounder genome. The blue horizontal line represents a linkage group-wise logarithm of odds (LOD) significance threshold of 3.0.</p></div></div>                            <h3 scrollto-destination=4595114 id="4595114" class="section-title" data-legacy-id=s3e>3.5. Genome assembly and synteny</h3>
<p>The high-resolution genetic map was used in the present study for genome scaffold assembly. Among the 12,712 SNPs on the high-resolution map, 12,463 were successfully used to anchor 246 scaffolds representing 522.99 Mb to create a genome map of Japanese flounder. In total, we constructed 24 pseudo-chromosomes based on the LGs; each chromosome comprised an average of 21.79 Mb (10 scaffolds), corresponding to a mean linkage distance of 145.72 cM and suggesting a physical/genetic distance ratio of 149.53 Kb/cM. Among the LGs, LG7 holding 13 scaffolds corresponded to the largest chromosome (29.55 Mb) and represented 95.78% of the entire assembly, whereas the smallest LGs contained 8 scaffolds representing 67.85% of the assembly (16.36 Mb) (Table <a reveal-id="DSV001TB1" data-open="DSV001TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, 219 scaffolds covering 520.44 Mb were oriented by assigning multiple genetically separated markers located on each scaffold. The orientation of 27 scaffolds covering 2.55 Mb could not be determined, because only single markers were present on those scaffolds.</p><p>Following genome map construction, we compared the chromosomal orders of protein-coding genes of Japanese flounder on each scaffold with their counterparts in medaka and zebrafish. A total of 6,768 1:1 best orthologues between Japanese flounder and zebrafish were identified. As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a> and Fig. <a reveal-id="DSV001F3" data-open="DSV001F3" class="link link-reveal link-table xref-fig">3</a>, 7 of the 24 chromosomes of flounder (Po. 4, 6, 10, 11, 12, 15, 18, 21, and 22) were in relatively conserved synteny with zebrafish chromosomes Dr. 12, 16, 3, 2, 14, 9, 24, 15, and 19 respectively. Many of the flounder chromosomes were found to have syntenic blocks with more than one chromosome of zebrafish and vice versa. An example of the multi-syntenic correspondence between zebrafish and flounder genomes is that of Po. 20, which exhibited one of the highest degrees of synteny: Po. 20 had 519 hits on zebrafish chromosomes, most of which (171, 142, and 136) were located on Dr. 05, 10, and 21, respectively. Similarly, Dr. 06 was found to be highly syntenic with Po. 01, 05, and 24, whereas Po. 01, 05, and 24 were in synteny with Dr. 11, 01, and 22, respectively. With respect to flounder–medaka genome synteny, the alignment of 6,463 orthologues on the 24 flounder chromosomes revealed an obvious syntenic relationship with the 24 corresponding chromosomes of medaka. Except for some chromosomes with minor multi-chromosome hits (&lt;50 genes), the results of this comparison suggest that flounder is more closely related to medaka than to zebrafish (Fig. <a reveal-id="DSV001F3" data-open="DSV001F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="4595117" scrollto-destination="4595117"></a>
<div id="DSV001F3" data-id="DSV001F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00103.png?Expires=2147483647&Signature=LYZYuyV3JB9uvmSDkagQzhqcZeS-l-O2S9vWlceU3xcxCrynuJpKjr083rSVRFLq1hue4UccOgIaLLpj7uYrbl3zrJyOffBWcvN6qubtL58htYWZcotKpZxj6cI4nhVgYCdOqj-hSd6JA8PXdK-Hx3MOiiY6iXyltY3toXzYJHLLj00t33tiaM-epK3pXi2~nJUPakGLW9m5RTmdFrAcwUYZrkQVT1IahkMnXHIWB4Y5L2rTIJweR7hXwQUEdUVRZbOqXiDHYjYvTdhPmK8YoQCYTkUeIWI7cqAra3jhoKrcxchUMHEJKuW4sYQKnSDAsyCg9PMNgEEM5V5550avsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circos diagram representing syntenic relationships between flounder and (a) zebrafish and (b) medaka, respectively. Each colored arc represents an orthologous match between two species. Pol, Paralichthys olivaceus; Dre, Danio rerio; Ola, Oryzias latipes." path-from-xml="dsv00103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595117/dsv00103.tiff" path-from-xml="dsv00103.tiff" target="_blank">View large</a><a section="4595117" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00103.png?Expires=2147483647&Signature=n3tCMYfwEKNj5cNo7mjt1vH-FIUJ3bRZORN8a7y5aXt9gqtx36OWFUwyQo~y5Xjj4RsDHruO80quR5kifnoDbmmHjS9j3UuM9VhcM6NJOe8orDLaYmdkvi7ZggUxBnv31vEqcDlERBxV2oaI6CKon96bdVEOjvMjsIQKHnqPSmW320~bwiBuBLfsXt-8hc0nJ-q7XMbzsaijgBjFYbkVrNEU5mszHY4DJlTCsUKBGGr-8V2wgn6rNs1NkDS31kxRFzVg8SbIKiklLzKq5k1hiGn37LUByqCDdez3LIi70v6ndgSkYF0PmD6IN6oLj9Ho2Wb7ENe9AbuuO0LUYXArog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595117&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circos diagram representing syntenic relationships between flounder and (a) zebrafish and (b) medaka, respectively. Each colored arc represents an orthologous match between two species. Pol, <em>Paralichthys olivaceus</em>; Dre, <em>Danio rerio</em>; Ola, <em>Oryzias latipes</em>.</p></div></div><div content-id="DSV001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/m_dsv00103.png?Expires=2147483647&Signature=LYZYuyV3JB9uvmSDkagQzhqcZeS-l-O2S9vWlceU3xcxCrynuJpKjr083rSVRFLq1hue4UccOgIaLLpj7uYrbl3zrJyOffBWcvN6qubtL58htYWZcotKpZxj6cI4nhVgYCdOqj-hSd6JA8PXdK-Hx3MOiiY6iXyltY3toXzYJHLLj00t33tiaM-epK3pXi2~nJUPakGLW9m5RTmdFrAcwUYZrkQVT1IahkMnXHIWB4Y5L2rTIJweR7hXwQUEdUVRZbOqXiDHYjYvTdhPmK8YoQCYTkUeIWI7cqAra3jhoKrcxchUMHEJKuW4sYQKnSDAsyCg9PMNgEEM5V5550avsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circos diagram representing syntenic relationships between flounder and (a) zebrafish and (b) medaka, respectively. Each colored arc represents an orthologous match between two species. Pol, Paralichthys olivaceus; Dre, Danio rerio; Ola, Oryzias latipes." path-from-xml="dsv00103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4595117/dsv00103.tiff" path-from-xml="dsv00103.tiff" target="_blank">View large</a><a section="4595117" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv00103.png?Expires=2147483647&Signature=n3tCMYfwEKNj5cNo7mjt1vH-FIUJ3bRZORN8a7y5aXt9gqtx36OWFUwyQo~y5Xjj4RsDHruO80quR5kifnoDbmmHjS9j3UuM9VhcM6NJOe8orDLaYmdkvi7ZggUxBnv31vEqcDlERBxV2oaI6CKon96bdVEOjvMjsIQKHnqPSmW320~bwiBuBLfsXt-8hc0nJ-q7XMbzsaijgBjFYbkVrNEU5mszHY4DJlTCsUKBGGr-8V2wgn6rNs1NkDS31kxRFzVg8SbIKiklLzKq5k1hiGn37LUByqCDdez3LIi70v6ndgSkYF0PmD6IN6oLj9Ho2Wb7ENe9AbuuO0LUYXArog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4595117&ar=332908&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circos diagram representing syntenic relationships between flounder and (a) zebrafish and (b) medaka, respectively. Each colored arc represents an orthologous match between two species. Pol, <em>Paralichthys olivaceus</em>; Dre, <em>Danio rerio</em>; Ola, <em>Oryzias latipes</em>.</p></div></div>                            <h2 scrollto-destination=4595118 id="4595118" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=4595119 id="4595119" class="section-title" data-legacy-id=s4a>4.1. High-resolution genetic map construction using RAD-based SNPs</h3>
<p>Genetic maps constructed using molecular markers are important for genomic and genetic analyses of individual species. SNPs are particularly attractive for genetic map construction, because they represent the most common type of DNA polymorphism in the genome and are amenable to high-throughput genotyping.<sup><a reveal-id="DSV001C17" data-open="DSV001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In the present study, ∼950 million RAD tags were generated from 2 Japanese flounder parents and 216 offspring individuals. We discovered 13,362 novel SNPs evenly distributed throughout the entire Japanese flounder genome, providing thereby a large number of genetic variation resources for future genome selection and genome-wide association studies. Among the identified SNPs, 12,712 SNPs were successfully genotyped and assigned to 24 LGs corresponding to the 24 chromosome pairs of the Japanese flounder genome. The total length of the resulting map was 3,497.29 cM, with an average marker spacing of 0.47 cM. Four genetic maps of Japanese flounder have been previously constructed using various markers,<sup><a reveal-id="DSV001C7  DSV001C10" data-open="DSV001C7  DSV001C10" class="link link-ref link-reveal xref-bibr">7–10</a></sup> but those maps are characterized by relatively larger genetic intervals and/or non-uniform marker distributions. The most recent genetic map<sup><a reveal-id="DSV001C10" data-open="DSV001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> containing 1,487 SSRs with an average interval of 1.22 cM was formerly the densest reported flatfish linkage map and worthy of acclaim. However, whole genome-level analyses such as genome-wide association studies were still not practical using this map because of the limits imposed by the types of markers and their associated genotyping methods.<sup><a reveal-id="DSV001C23" data-open="DSV001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The denser SNP-based genetic map constructed in this study thus has greatly expanded application. It represents a considerable improvement over previous flounder genetic linkage maps based on SSR and AFLP markers.</p><p>Most LGs on our genetic map were considered to be saturated with an even distribution of three types of SNPs, namely, maternal heterozygous, paternal heterozygous, and double heterozygous SNPs; a few LGs, such as LG8, LG9, LG11, LG18, LG19, were not, however, revealing a complex genome containing regions enriched in repeat sequences and transposable elements.<sup><a reveal-id="DSV001C35" data-open="DSV001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Interestingly, LG8 encompassed the smallest genetic distance, contained only paternal and double heterozygous markers, and corresponded to the smallest physical distance (11.1 Mb). The fact that LG8 on the maternal map had only markers heterozygous in both parents whereas the group on the paternal map had only paternal heterozygous and double heterozygous markers suggests specific suppression of homologous recombination in this LG. Similar phenomena have been documented in other species having recently evolved sex chromosomes with large pseudoautosomal regions.<sup><a reveal-id="DSV001C36" data-open="DSV001C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSV001C37" data-open="DSV001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> To overcome this complication, additional markers must be developed and a larger mapping population is required to increase the density of the linkage map.</p><p>The Lep-MAP program used in this study offers several advantages over JoinMap,<sup><a reveal-id="DSV001C38" data-open="DSV001C38" class="link link-ref link-reveal xref-bibr">38</a></sup> currently the most popular linkage-mapping software package. Lep-MAP, unlike JoinMAP, is suitable for use with tens of thousands of markers. Furthermore, Lep-MAP is faster on large data sets, requires no manual input, and handles genotyping errors better because it estimates genotyping error rates for each marker. Since its development in 2013, Lep-MAP has thus been heavily used for genetic map construction, such as for the high-resolution genetic map of <em>Glanville fritillary</em>.<sup><a reveal-id="DSV001C39" data-open="DSV001C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Finally, the RAD-seq technique is proving to be a highly valuable tool for SNP discovery and genotyping followed by genetic map construction and has currently been applied to various species. The success of RAD-seq is owing to advances in next-generation technologies, which reduce the cost of DNA sequencing. Genotyping by sequencing is therefore expected to become increasingly popular for high-throughput genotyping, genetic map construction, and genome analysis.<sup><a reveal-id="DSV001C40" data-open="DSV001C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p>                            <h3 scrollto-destination=4595123 id="4595123" class="section-title" data-legacy-id=s4b>4.2. <em>V. anguillarum</em> disease resistance-related QTLs and associated genes</h3>
<p>The generated high-resolution genetic map allowed us to perform QTL fine mapping for an economically important trait of Japanese flounder. In the present study, 10 QTLs associated with <em>V. anguillarum</em> disease resistance were found to be distributed on three LGs (LG6, LG19, and LG21). Interestingly, most of the QTLs were concentrated within a narrow region (cluster) on the LGs. Six QTLs were clustered together (89.8–108.7 cM) on LG6 corresponding to assembly scaffolds, scaffold196 (1.08 Mb), scaffold103 (1.06 Mb), scaffold31 (4.39 Mb), and scaffold36 (11.37 Mb), and three were found in another cluster (98.7–101.3 cM) on LG21, corresponding to scaffold190 (0.97 Mb) and scaffold70 (2.59 Mb). The quite small genetic and physical distances among QTLs in certain clusters suggest that the individual clusters are highly effective QTLs. One exception is qVA-7 on LG19. Even though it had the largest LOD value (15.8), qVA-7 contained only one SNP located in a physical gap between two scaffolds, indicating that it may be a false-positive QTL or, alternatively, the result of an error arising during genetic map construction due to segregation distortion. In addition, the smallest significant QTL (other than qVA-7) explained 5.1% of the phenotypic variation, whereas the maximum contribution was only 8.38%, reflecting the complexity of this polygenically controlled disease trait in Japanese flounder. Although the possibility exists that additional disease-controlling QTLs are present, the positive QTLs detected in this study, taken together, explained 58.06% of the total disease resistance variation, suggesting that these two QTL clusters play major roles. On the other hand, the results of our QTL analysis are not consistent with a previous report that the major QTL for <em>V. anguillarum</em> disease resistance is located on LG11 (LG7 in that study) in Japanese flounder.<sup><a reveal-id="DSV001C41" data-open="DSV001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> In the previous study, two <em>V. anguillarum</em> disease resistance-associated polymorphic SSR markers were selected using bulked segregant analysis and then directly anchored onto LG11. The QTLs were finally identified by genotyping of 22 SSRs of LG11 in an F<sub>1</sub> population.<sup><a reveal-id="DSV001C41" data-open="DSV001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Because the QTL detection analysis was restricted to a specific LG at the very start, other potential QTLs may have been missed. The other possible explanation for the observed discrepancy is that it was the inevitable result of using a different family under different environmental conditions, just as repeated analysis of the same family treated by two successive challenges would result in lower repeatability for QTL detection for such a complex quantitative trait.<sup><a reveal-id="DSV001C42" data-open="DSV001C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV001C43" data-open="DSV001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> To enhance the accuracy of QTL detection, results from multiple full-sib families should be integrated in the future.</p><p>To identify potential candidate genes, we compared the detected QTLs with the scaffold assembly and annotation of the Japanese flounder reference genome. A total of 12 immune-related genes were discovered, corresponding to the multi-population QTL region. The fact that four of these genes (<em>tap1</em>, <em>satb1</em>, <em>cd40</em>, and <em>cd69</em>) are associated with <em>V. anguillarum</em> disease resistance in Japanese flounder is unsurprising, as they are involved in the molecular function of the MHC class-dependent pathway of antigen presentation. Extensive studies have shown that MHC class I and II molecules play a pivotal role in immune defence systems, because they allow T cells to distinguish self from non-self.<sup><a reveal-id="DSV001C44" data-open="DSV001C44" class="link link-ref link-reveal xref-bibr">44</a></sup> In particular, investigations of allelic polymorphism and the pattern of evolution in MHC genes in Japanese flounder have indicated an association between MHC genes and <em>V. anguillarum</em> disease resistance.<sup><a reveal-id="DSV001C28" data-open="DSV001C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV001C45  DSV001C47" data-open="DSV001C45  DSV001C47" class="link link-ref link-reveal xref-bibr">45–47</a></sup> These latter results also support the quality of our RAD data, because a key issue for QTL detection and associated gene analysis, given adequate genome coverage, is marker quality.</p>                            <h3 scrollto-destination=4595126 id="4595126" class="section-title" data-legacy-id=s4c>4.3. Comparative genome analysis</h3>
<p>Next-generation sequencing technology has enabled generation of draft genomes for individual species as well as rapid development of comparative analyses among multiple genomes based on chromosomal-assembly levels.<sup><a reveal-id="DSV001C48" data-open="DSV001C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Nevertheless, the large amount of repetitive DNA sequences and high heterozygosity levels in most species, especially marine species, has hindered assembly accuracy and also prevented direct assembly into chromosomal performance.<sup><a reveal-id="DSV001C49" data-open="DSV001C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Consequently, a high-quality genetic map is an appropriate way to anchor scaffolds onto chromosomes. In this study, 24 pseudo-chromosomes of Japanese flounder were constructed by anchoring 246 scaffolds totaling 522.99 Mb, accounting for 95.78% of assembly sequences, using a high-resolution SNP genetic map. Such a high level of chromosomal-level genome map integrity is due to the SNPs, which were abundantly distributed across the genome by using the highest-throughput genotyping technique available.<sup><a reveal-id="DSV001C17" data-open="DSV001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The constructed pseudo-chromosomes of Japanese flounder provide a basic foundation to carry out extensive comparative genomics. In the present study, we performed an <em>in silico</em> analysis of orthologous gene pairs, because genes are evolutionarily conserved relative to intergenic regions.<sup><a reveal-id="DSV001C50" data-open="DSV001C50" class="link link-ref link-reveal xref-bibr">50</a></sup> From a general point of view, the orthologues (6,768) of flounder-medaka were more abundant than the orthologues (6,463) of flounder–zebrafish, reflecting the relatively closer phylogenetic relationship of medaka and flounder.<sup><a reveal-id="DSV001C51" data-open="DSV001C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Looking more closely at evolutionary relationships based on the chromosomal segments, multi-syntenic correspondences were exhibited between flounder and zebrafish, whereas almost all flounder and medaka chromosomes had a 1:1 correspondence. This difference in synteny directly reflects that the flounder have a phylogenetic closeness to medaka than to zebrafish and also suggests that a much larger number of evolutionarily significant events, such as fusion, breakage, and translocation, have occurred in the zebrafish lineage. The medaka–flounder comparative analysis uncovered little evidence of such inter-chromosomal rearrangements in the medaka lineage despite the difference in chromosome numbers among those fish species (zebrafish has 25 chromosomes, whereas flounder and medaka have 24).<sup><a reveal-id="DSV001C52  DSV001C54" data-open="DSV001C52  DSV001C54" class="link link-ref link-reveal xref-bibr">52–54</a></sup> Furthermore, we inferred the occurrence of many intra-chromosomal events that have interrupted syntenic linkage blocks both between flounder and medaka and between flounder and zebrafish; these interruptions may have been the result of gene loss or divergence after duplication, which can contribute to deviations from synteny.<sup><a reveal-id="DSV001C55" data-open="DSV001C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Most teleost species have experienced three rounds of whole-genome duplication. Thousands of duplicated genes remaining after divergence of teleost species have subsequently undergone non-reciprocal loss, pseudogenization, or sub-functionalization, and may be associated with unequal microsynteny in flounder and zebrafish.<sup><a reveal-id="DSV001C56" data-open="DSV001C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSV001C57" data-open="DSV001C57" class="link link-ref link-reveal xref-bibr">57</a></sup> From a different perspective, most chromosomal regions among the three analysed teleost species generally had a conserved syntenic relationship. This observation not only supports the marker ordering on the map but also facilitates the functional inference of genes in flounder and further benefits the analysis of genome evolution and comparative genomics.</p><p>In conclusion, we used RAD-seq technology for large-scale identification of SNPs that were then successfully used for high-throughput genotyping and construction of a high-resolution genetic map of Japanese flounder. The developed genetic map is the most comprehensive genetic map to date for this species. Through SNP mapping analysis, we identified 9 positive QTLs for <em>V. anguillarum</em> disease resistance that will be of interest to achieve breeding goals for Japanese flounder. We also anchored the genome scaffolds to pseudo-chromosomes and further identified complex syntenic relationships among flounder, medaka, and zebrafish by comparative genomic analysis. The large numbers of generated SNPs and the dense genetic map, coupled with future re-sequencing of multiple breeding families, should not only lay a foundation for chromosomal-level analysis of the flounder genome but should also provide an excellent resource for future molecular breeding efforts such as genome selection.</p>                            <h2 scrollto-destination=4595131 id="4595131" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at http://www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4595135 id="4595135" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The work was supported by grants from State 863 High-Technology R&amp;D Project (2012AA10A408), National Nature Science Foundation of China (31461163005), JSPS RONPAKU (Dissertation PhD) Program for C.S., and Taishan Scholar Project Fund of Shandong of China. Funding to pay the Open Access publication charges for this article was provided by the State 863 HIgh Technology R&amp;D Project (2012AA10A408).</p>                            <h2 scrollto-destination=4595139 id="4595139" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fujiwara</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Fujiwara</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Nishida-Umehara</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Abe</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Masaoka</div> <div class="given-names">T.</div></div></div> <div class="year">2007</div>, <div class="article-title">Characterization of Japanese flounder karyotype by chromosome bandings and fluorescence in situ hybridization with DNA markers</div>, <div class="source">Genetica</div> , <div class="volume">131</div>, <div class="fpage">267</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Japanese%20flounder%20karyotype%20by%20chromosome%20bandings%20and%20fluorescence%20in%20situ%20hybridization%20with%20DNA%20markers&amp;author=A.%20Fujiwara&amp;author=M.%20Fujiwara&amp;author=C.%20Nishida-Umehara&amp;author=S.%20Abe&amp;author=T.%20Masaoka&amp;publication_year=2007&amp;journal=Genetica&amp;volume=131&amp;pages=267-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-006-9136-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-006-9136-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-006-9136-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17273899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nystoyl</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Tveteras</div> <div class="given-names">R.</div></div></div> <div class="year">2013</div>, <div class="article-title">Fish production estimates and trends 2012–2013</div>. <div class="comment">Available at: <a class="link link-uri" href="http://www.gaalliance.org/cmsAdmin/uploads/tveteras.pdf">http://www.gaalliance.org/cmsAdmin/uploads/tveteras.pdf</a>.</div></div></div></div></div><div content-id="DSV001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Seikai</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, <div class="article-title">Flounder culture and its challenges in Asia</div>, <div class="source">Rev. Fish. Sci.</div> , <div class="volume">3–4</div>, <div class="fpage">421</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flounder%20culture%20and%20its%20challenges%20in%20Asia&amp;author=T.%20Seikai&amp;publication_year=2002&amp;journal=Rev.%20Fish.%20Sci.&amp;volume=3–4&amp;pages=421-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/20026491051721" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F20026491051721" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F20026491051721"> </span></div> </div></div></div></div></div><div content-id="DSV001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chapman</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Pashley</div> <div class="given-names">C.H.</div></div> <div class="name"><div class="surname">Wenzler</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">A genomic scan for selection reveals candidates for genes involved in the evolution of cultivated sunflower (<em>Helianthus annuus</em>)</div>, <div class="source">Plant Cell</div> , <div class="volume">20</div>, <div class="fpage">2931</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20scan%20for%20selection%20reveals%20candidates%20for%20genes%20involved%20in%20the%20evolution%20of%20cultivated%20sunflower%20(Helianthus%20annuus)&amp;author=M.A.%20Chapman&amp;author=C.H.%20Pashley&amp;author=J.%20Wenzler&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=2931-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.059808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.059808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.059808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19017747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Andriantahina</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Huang</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, <div class="article-title">Genetic map construction and quantitative trait locus (QTL) detection of growth-related traits in <em>Litopenaeus vannamei</em> for selective breeding applications</div>, <div class="source">PloS ONE</div> , <div class="volume">8</div>, <div class="fpage">e75206</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20map%20construction%20and%20quantitative%20trait%20locus%20(QTL)%20detection%20of%20growth-related%20traits%20in%20Litopenaeus%20vannamei%20for%20selective%20breeding%20applications&amp;author=F.%20Andriantahina&amp;author=X.%20Liu&amp;author=H.%20Huang&amp;publication_year=2013&amp;journal=PloS%20ONE&amp;volume=8&amp;pages=e75206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0075206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0075206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0075206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yue</div> <div class="given-names">G.H.</div></div></div> <div class="year">2013</div>, <div class="article-title">Recent advances of genome mapping and marker-assisted selection in aquaculture</div>, <div class="source">Fish Fish.</div> , <div class="volume">15</div>, <div class="fpage">376</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20of%20genome%20mapping%20and%20marker-assisted%20selection%20in%20aquaculture&amp;author=G.H.%20Yue&amp;publication_year=2013&amp;journal=Fish%20Fish.&amp;volume=15&amp;pages=376-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/faf.2014.15.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffaf.2014.15.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffaf.2014.15.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSV001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Coimbra</div> <div class="given-names">M.R.</div></div> <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Koretsugu</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">A genetic linkage map of the Japanese flounder, <em>Paralichthys olivaceus</em></div>, <div class="source">Aquaculture</div> , <div class="volume">1–4</div>, <div class="fpage">203</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20Japanese%20flounder%2C%20Paralichthys%20olivaceus&amp;author=M.R.%20Coimbra&amp;author=K.%20Kobayashi&amp;author=S.%20Koretsugu&amp;publication_year=2003&amp;journal=Aquaculture&amp;volume=1–4&amp;pages=203-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0044-8486(02)00353-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0044-8486(02)00353-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0044-8486(02)00353-8"> </span></div> </div></div></div></div></div><div content-id="DSV001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kang</div> <div class="given-names">J.H.</div></div> <div class="name"><div class="surname">Kim</div> <div class="given-names">W.J.</div></div> <div class="name"><div class="surname">Lee</div> <div class="given-names">W.J.</div></div></div> <div class="year">2008</div>, <div class="article-title">Genetic linkage map of olive flounder, <em>Paralichthys olivaceus</em></div>, <div class="source">Int. J. Biol. Sci.</div> , <div class="volume">4</div>, <div class="fpage">143</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20of%20olive%20flounder%2C%20Paralichthys%20olivaceus&amp;author=J.H.%20Kang&amp;author=W.J.%20Kim&amp;author=W.J.%20Lee&amp;publication_year=2008&amp;journal=Int.%20J.%20Biol.%20Sci.&amp;volume=4&amp;pages=143-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7150/ijbs.4.143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7150%2Fijbs.4.143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7150%2Fijbs.4.143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18497873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Castano-Sanchez</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Ozaki</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">A second generation genetic linkage map of Japanese flounder (<em>Paralichthys olivaceus</em>)</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">554</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20generation%20genetic%20linkage%20map%20of%20Japanese%20flounder%20(Paralichthys%20olivaceus)&amp;author=C.%20Castano-Sanchez&amp;author=K.%20Fuji&amp;author=A.%20Ozaki&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=554" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-554" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-554" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-554"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20937088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Pang</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Niu</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Construction of high-density genetic linkage maps and mapping of growth-related quantitative trail loci in the Japanese flounder (<em>Paralichthys olivaceus</em>)</div>, <div class="source">Plos ONE</div> , <div class="volume">7</div>, <div class="fpage">e50404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20high-density%20genetic%20linkage%20maps%20and%20mapping%20of%20growth-related%20quantitative%20trail%20loci%20in%20the%20Japanese%20flounder%20(Paralichthys%20olivaceus)&amp;author=W.%20Song&amp;author=R.%20Pang&amp;author=Y.%20Niu&amp;publication_year=2012&amp;journal=Plos%20ONE&amp;volume=7&amp;pages=e50404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0050404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0050404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0050404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">O.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">Identification of a single major genetic locus controlling the resistance to lymphocystis disease in Japanese flounder (<em>Paralichthys olivaceus</em>)</div>, <div class="source">Aquaculture</div> , <div class="volume">1–4</div>, <div class="fpage">203</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20single%20major%20genetic%20locus%20controlling%20the%20resistance%20to%20lymphocystis%20disease%20in%20Japanese%20flounder%20(Paralichthys%20olivaceus)&amp;author=K.%20Fuji&amp;author=K.%20Kobayashi&amp;author=O.%20Hasegawa&amp;publication_year=2006&amp;journal=Aquaculture&amp;volume=1–4&amp;pages=203-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2005.11.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2005.11.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2005.11.024"> </span></div> </div></div></div></div></div><div content-id="DSV001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">S.D.</div></div> <div class="name"><div class="surname">Fuji</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Takano</div> <div class="given-names">T.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Linkage mapping of toll-like receptors (TLRs) in Japanese flounder, <em>Paralichthys olivaceus</em></div>, <div class="source">Mar. Biotechnol.</div> , <div class="volume">13</div>, <div class="fpage">1086</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20mapping%20of%20toll-like%20receptors%20(TLRs)%20in%20Japanese%20flounder%2C%20Paralichthys%20olivaceus&amp;author=S.D.%20Hwang&amp;author=K.%20Fuji&amp;author=T.%20Takano&amp;publication_year=2011&amp;journal=Mar.%20Biotechnol.&amp;volume=13&amp;pages=1086-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-011-9371-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-011-9371-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-011-9371-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21494881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fuji</div> <div class="given-names">K.H.O.</div></div> <div class="name"><div class="surname">Honda</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Kumasaka</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Okamoto</div> <div class="given-names">N.</div></div></div> <div class="year">2007</div>, <div class="article-title">Marker-assisted breeding of a lymphocystis disease-resistant Japanese flounder (<em>Pralichthys olivaceus</em>)</div>, <div class="source">Aquaculture</div> , <div class="volume">1–4</div>, <div class="fpage">291</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marker-assisted%20breeding%20of%20a%20lymphocystis%20disease-resistant%20Japanese%20flounder%20(Pralichthys%20olivaceus)&amp;author=K.H.O.%20Fuji&amp;author=K.%20Honda&amp;author=K.%20Kumasaka&amp;author=T.%20Sakamoto&amp;author=N.%20Okamoto&amp;publication_year=2007&amp;journal=Aquaculture&amp;volume=1–4&amp;pages=291-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2007.07.210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2007.07.210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2007.07.210"> </span></div> </div></div></div></div></div><div content-id="DSV001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hillier</div> <div class="given-names">L.W.</div></div> <div class="name"><div class="surname">Miller</div> <div class="given-names">R.D.</div></div> <div class="name"><div class="surname">Baird</div> <div class="given-names">S.E.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Comparison of <em>C. elegans</em> and <em>C. briggsae</em> genome sequences reveals extensive conservation of chromosome organization and synteny</div>, <div class="source">Plos Biol.</div> , <div class="volume">5</div>, <div class="fpage">e167</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20C.%20elegans%20and%20C.%20briggsae%20genome%20sequences%20reveals%20extensive%20conservation%20of%20chromosome%20organization%20and%20synteny&amp;author=L.W.%20Hillier&amp;author=R.D.%20Miller&amp;author=S.E.%20Baird&amp;publication_year=2007&amp;journal=Plos%20Biol.&amp;volume=5&amp;pages=e167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17608563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Beldade</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Saenko</div> <div class="given-names">S.V.</div></div> <div class="name"><div class="surname">Pul</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Long</div> <div class="given-names">A.D.</div></div></div> <div class="year">2009</div>, <div class="article-title">A gene-based linkage map for <em>Bicyclus anynana</em> butterflies allows for a comprehensive analysis of synteny with the lepidopteran reference genome</div>, <div class="source">PLoS Genet.</div> , <div class="volume">5</div>, <div class="fpage">e1000366</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene-based%20linkage%20map%20for%20Bicyclus%20anynana%20butterflies%20allows%20for%20a%20comprehensive%20analysis%20of%20synteny%20with%20the%20lepidopteran%20reference%20genome&amp;author=P.%20Beldade&amp;author=S.V.%20Saenko&amp;author=N.%20Pul&amp;author=A.D.%20Long&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000366" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19197358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moorthie</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Mattocks</div> <div class="given-names">C.J.</div></div> <div class="name"><div class="surname">Wright</div> <div class="given-names">C.F.</div></div></div> <div class="year">2011</div>, <div class="article-title">Review of massively parallel DNA sequencing technologies</div>, <div class="source">HUGO J</div> , <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%20of%20massively%20parallel%20DNA%20sequencing%20technologies&amp;author=S.%20Moorthie&amp;author=C.J.%20Mattocks&amp;author=C.F.%20Wright&amp;publication_year=2011&amp;journal=HUGO%20J&amp;volume=5&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11568-011-9156-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11568-011-9156-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11568-011-9156-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23205160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Berthier-Schaad</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Kao</div> <div class="given-names">W.H.</div></div> <div class="name"><div class="surname">Coresh</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Reliability of high-throughput genotyping of whole genome amplified DNA in SNP genotyping studies</div>, <div class="source">Electrophoresis</div> , <div class="volume">28</div>, <div class="fpage">2812</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reliability%20of%20high-throughput%20genotyping%20of%20whole%20genome%20amplified%20DNA%20in%20SNP%20genotyping%20studies&amp;author=Y.%20Berthier-Schaad&amp;author=W.H.%20Kao&amp;author=J.%20Coresh&amp;publication_year=2007&amp;journal=Electrophoresis&amp;volume=28&amp;pages=2812-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17702060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rowe</div> <div class="given-names">H.C.</div></div> <div class="name"><div class="surname">Renaut</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Guggisberg</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, <div class="article-title">RAD in the realm of next-generation sequencing technologies</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">20</div>, <div class="fpage">3499</div>–<div class="lpage">502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAD%20in%20the%20realm%20of%20next-generation%20sequencing%20technologies&amp;author=H.C.%20Rowe&amp;author=S.%20Renaut&amp;author=A.%20Guggisberg&amp;publication_year=2011&amp;journal=Mol.%20Ecol.&amp;volume=20&amp;pages=3499-502" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21991593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Baird</div> <div class="given-names">N.A.</div></div> <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Atwood</div> <div class="given-names">T.S.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PloS ONE</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.A.%20Baird&amp;author=P.D.%20Etter&amp;author=T.S.%20Atwood&amp;publication_year=2008&amp;journal=PloS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Amish</div> <div class="given-names">S.J.</div></div> <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Painter</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">RAD sequencing yields a high success rate for westslope cutthroat and rainbow trout species-diagnostic SNP assays</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="volume">12</div>, <div class="fpage">653</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAD%20sequencing%20yields%20a%20high%20success%20rate%20for%20westslope%20cutthroat%20and%20rainbow%20trout%20species-diagnostic%20SNP%20assays&amp;author=S.J.%20Amish&amp;author=P.A.%20Hohenlohe&amp;author=S.%20Painter&amp;publication_year=2012&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=12&amp;pages=653-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1755-0998.2012.03157.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1755-0998.2012.03157.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1755-0998.2012.03157.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Barchi</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Lanteri</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Portis</div> <div class="given-names">E.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">A RAD tag derived marker based eggplant linkage map and the location of QTLs determining anthocyanin pigmentation</div>, <div class="source">PloS ONE</div> , <div class="volume">7</div>, <div class="fpage">e43740</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20RAD%20tag%20derived%20marker%20based%20eggplant%20linkage%20map%20and%20the%20location%20of%20QTLs%20determining%20anthocyanin%20pigmentation&amp;author=L.%20Barchi&amp;author=S.%20Lanteri&amp;author=E.%20Portis&amp;publication_year=2012&amp;journal=PloS%20ONE&amp;volume=7&amp;pages=e43740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0043740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0043740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0043740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22912903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chutimanitsakun</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Nipper</div> <div class="given-names">R.W.</div></div> <div class="name"><div class="surname">Cuesta-Marcos</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Construction and application for QTL analysis of a Restriction Site Associated DNA (RAD) linkage map in barley</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20application%20for%20QTL%20analysis%20of%20a%20Restriction%20Site%20Associated%20DNA%20(RAD)%20linkage%20map%20in%20barley&amp;author=Y.%20Chutimanitsakun&amp;author=R.W.%20Nipper&amp;author=A.%20Cuesta-Marcos&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21205322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gaur</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Azam</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Jeena</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (<em>Cicer arietinum L</em>.)</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">357</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20genotyping%20for%20constructing%20a%20saturated%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=S.%20Azam&amp;author=G.%20Jeena&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=357-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Willing</div> <div class="given-names">E.M.</div></div> <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Klein</div> <div class="given-names">J.D.</div></div> <div class="name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Dreyer</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, <div class="article-title">Paired-end RAD-seq for de novo assembly and marker design without available reference</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2187</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paired-end%20RAD-seq%20for%20de%20novo%20assembly%20and%20marker%20design%20without%20available%20reference&amp;author=E.M.%20Willing&amp;author=M.%20Hoffmann&amp;author=J.D.%20Klein&amp;author=D.%20Weigel&amp;author=C.%20Dreyer&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2187-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21712251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zheng</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Sweetingham</div> <div class="given-names">M.W.</div></div> <div class="name"><div class="surname">Howieson</div> <div class="given-names">J.G.</div></div></div> <div class="year">2012</div>, <div class="article-title">Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in <em>Lupinus angustifolius L</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">318</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20next-generation%20sequencing%20for%20rapid%20marker%20development%20in%20molecular%20plant%20breeding%3A%20a%20case%20study%20on%20anthracnose%20disease%20resistance%20in%20Lupinus%20angustifolius%20L&amp;author=H.%20Yang&amp;author=Y.%20Tao&amp;author=Z.%20Zheng&amp;author=C.%20Li&amp;author=M.W.%20Sweetingham&amp;author=J.G.%20Howieson&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-318" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-318" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-318"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22805587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Stiffler</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.A.</div></div> <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div></div> <div class="year">2010</div>, <div class="article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags</div>, <div class="source">PLoS Genet.</div> , <div class="volume">6</div>, <div class="fpage">e1000862</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomics%20of%20parallel%20adaptation%20in%20threespine%20stickleback%20using%20sequenced%20RAD%20tags&amp;author=P.A.%20Hohenlohe&amp;author=S.%20Bassham&amp;author=P.D.%20Etter&amp;author=N.%20Stiffler&amp;author=E.A.%20Johnson&amp;author=W.A.%20Cresko&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1000862" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000862" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20195501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Analysis of growth performance and breeding value of ‘Ping You No.1’ Japanese flounder and selection of parents</div>, <div class="source">J. Fish. Sci. China</div> , <div class="volume">3</div>, <div class="fpage">521</div>–<div class="lpage">7</div> <div class="comment">(in Chinese)</div>.</div></div></div></div><div content-id="DSV001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">T.J.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Ji</div> <div class="given-names">X.S.</div></div> <div class="name"><div class="surname">Tian</div> <div class="given-names">Y.S.</div></div></div> <div class="year">2008</div>, <div class="article-title">MHC polymorphism and disease resistance to <em>Vibrio anguillarum</em> in 12 selective Japanese flounder (<em>Paralichthys olivaceus</em>) families</div>, <div class="source">Fish Shellfish Immun.</div> , <div class="volume">25</div>, <div class="fpage">213</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MHC%20polymorphism%20and%20disease%20resistance%20to%20Vibrio%20anguillarum%20in%2012%20selective%20Japanese%20flounder%20(Paralichthys%20olivaceus)%20families&amp;author=T.J.%20Xu&amp;author=S.L.%20Chen&amp;author=X.S.%20Ji&amp;author=Y.S.%20Tian&amp;publication_year=2008&amp;journal=Fish%20Shellfish%20Immun.&amp;volume=25&amp;pages=213-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsi.2008.05.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsi.2008.05.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsi.2008.05.007"> </span></div> </div></div></div></div></div><div content-id="DSV001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Amores</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Postlethwait</div> <div class="given-names">J.H.</div></div></div> <div class="year">2011</div>, <div class="article-title">Stacks: building and genotyping Loci de novo from short-read sequences</div>, <div class="source">G3-Genes Genom.Genet.</div> , <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">82</div>.</div></div></div></div><div content-id="DSV001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Catchen</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Amores</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div></div> <div class="year">2013</div>, <div class="article-title">Stacks: an analysis tool set for population genomics</div>, <div class="source">Mol. Ecol.</div> , <div class="volume">22</div>, <div class="fpage">3124</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20an%20analysis%20tool%20set%20for%20population%20genomics&amp;author=J.%20Catchen&amp;author=P.A.%20Hohenlohe&amp;author=S.%20Bassham&amp;author=A.%20Amores&amp;author=W.A.%20Cresko&amp;publication_year=2013&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=3124-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23701397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rastas</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Paulin</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Hanski</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Lehtonen</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Auvinen</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, <div class="article-title">Lep-MAP: fast and accurate linkage map construction for large SNP datasets</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">3128</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lep-MAP%3A%20fast%20and%20accurate%20linkage%20map%20construction%20for%20large%20SNP%20datasets&amp;author=P.%20Rastas&amp;author=L.%20Paulin&amp;author=I.%20Hanski&amp;author=R.%20Lehtonen&amp;author=P.%20Auvinen&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=3128-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24078685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Silva Lda</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z.B.</div></div></div> <div class="year">2012</div>, <div class="article-title">Composite interval mapping and multiple interval mapping: procedures and guidelines for using Windows QTL Cartographer</div>, <div class="source">Methods Mol. Biol.</div> , <div class="volume">871</div>, <div class="fpage">75</div>–<div class="lpage">119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composite%20interval%20mapping%20and%20multiple%20interval%20mapping%3A%20procedures%20and%20guidelines%20for%20using%20Windows%20QTL%20Cartographer&amp;author=C.%20Silva%20Lda&amp;author=S.%20Wang&amp;author=Z.B.%20Zeng&amp;publication_year=2012&amp;journal=Methods%20Mol.%20Biol.&amp;volume=871&amp;pages=75-119" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22565834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hinz</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Jedamzick</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Herbring</div> <div class="given-names">V.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Assembly and function of the MHC I peptide-loading complex are conserved across higher vertebrates</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">289</div>, <div class="fpage">33109</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20and%20function%20of%20the%20MHC%20I%20peptide-loading%20complex%20are%20conserved%20across%20higher%20vertebrates&amp;author=A.%20Hinz&amp;author=J.%20Jedamzick&amp;author=V.%20Herbring&amp;publication_year=2014&amp;journal=J.%20Biol.%20Chem.&amp;volume=289&amp;pages=33109-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M114.609263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M114.609263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M114.609263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25320083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cohen-Sfady</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Nussbaum</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Pevsner-Fischer</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Heat shock protein 60 activates B cells via the TLR4-MyD88 pathway</div>, <div class="source">J. Immunol.</div> , <div class="volume">175</div>, <div class="fpage">3594</div>–<div class="lpage">602</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heat%20shock%20protein%2060%20activates%20B%20cells%20via%20the%20TLR4-MyD88%20pathway&amp;author=M.%20Cohen-Sfady&amp;author=G.%20Nussbaum&amp;author=M.%20Pevsner-Fischer&amp;publication_year=2005&amp;journal=J.%20Immunol.&amp;volume=175&amp;pages=3594-602" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4049/jimmunol.175.6.3594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4049%2Fjimmunol.175.6.3594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4049%2Fjimmunol.175.6.3594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16148103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schaack</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Choi</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Pritham</div> <div class="given-names">E.J.</div></div></div> <div class="year">2010</div>, <div class="article-title">DNA transposons and the role of recombination in mutation accumulation in <em>Daphnia pulex</em></div>, <div class="source">Genome Biol.</div> , <div class="volume">11</div>, <div class="fpage">R46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20transposons%20and%20the%20role%20of%20recombination%20in%20mutation%20accumulation%20in%20Daphnia%20pulex&amp;author=S.%20Schaack&amp;author=E.%20Choi&amp;author=M.%20Lynch&amp;author=E.J.%20Pritham&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-4-r46" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-4-r46" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-4-r46"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20433697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bergero</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, <div class="article-title">The evolution of restricted recombination in sex chromosomes</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="volume">24</div>, <div class="fpage">94</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20restricted%20recombination%20in%20sex%20chromosomes&amp;author=R.%20Bergero&amp;author=D.%20Charlesworth&amp;publication_year=2009&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=24&amp;pages=94-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2008.09.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2008.09.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2008.09.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19100654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Natri</div> <div class="given-names">H.M.</div></div> <div class="name"><div class="surname">Shikano</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Merila</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, <div class="article-title">Progressive recombination suppression and differentiation in recently evolved neo-sex chromosomes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">30</div>, <div class="fpage">1131</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progressive%20recombination%20suppression%20and%20differentiation%20in%20recently%20evolved%20neo-sex%20chromosomes&amp;author=H.M.%20Natri&amp;author=T.%20Shikano&amp;author=J.%20Merila&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=1131-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23436913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">van Ooijen</div> <div class="given-names">J.W.</div></div></div> <div class="year">2006</div>, <div class="article-title">Joinmap 4: software for the calculation of genetic linkage maps in experimental populations</div>. <div class="comment">Kyazma B.V., Wageningen, Netherlands</div>.</div></div></div></div><div content-id="DSV001C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ahola</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Lehtonen</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Somervuo</div> <div class="given-names">P.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">The <em>Glanville fritillary</em> genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera</div>, <div class="source">Nature Commun.</div> , <div class="volume">5</div>, <div class="fpage">4737</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Glanville%20fritillary%20genome%20retains%20an%20ancient%20karyotype%20and%20reveals%20selective%20chromosomal%20fusions%20in%20Lepidoptera&amp;author=V.%20Ahola&amp;author=R.%20Lehtonen&amp;author=P.%20Somervuo&amp;publication_year=2006&amp;journal=Nature%20Commun.&amp;volume=5&amp;pages=4737" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5737"> </span></div> </div></div></div></div></div><div content-id="DSV001C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Elshire</div> <div class="given-names">R.J.</div></div> <div class="name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</div>, <div class="source">PloS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.J.%20Elshire&amp;author=J.C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PloS%20ONE&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Fan</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">A genome scan for quantitative trait loci associated with Vibrio anguillarum infection resistance in Japanese flounder (Paralichthys olivaceus) by bulked segregant analysis</div>, <div class="source">Mar. Biotechnol.</div> , <div class="volume">16</div>, <div class="fpage">513</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome%20scan%20for%20quantitative%20trait%20loci%20associated%20with%20Vibrio%20anguillarum%20infection%20resistance%20in%20Japanese%20flounder%20(Paralichthys%20olivaceus)%20by%20bulked%20segregant%20analysis&amp;author=L.%20Wang&amp;author=C.%20Fan&amp;author=Y.%20Liu&amp;publication_year=2014&amp;journal=Mar.%20Biotechnol.&amp;volume=16&amp;pages=513-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-014-9569-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-014-9569-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-014-9569-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24562474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">Y.T.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Shi</div> <div class="given-names">Y.S.</div></div> <div class="name"><div class="surname">Song</div> <div class="given-names">Y.C.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">T.Y.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div></div> <div class="year">2006</div>, <div class="article-title">QTL analysis for plant height and leaf angle by using different populations of maize</div>, <div class="source">J. Maize Sci.</div> , <div class="volume">14</div>, <div class="fpage">88</div>–<div class="lpage">92</div> <div class="comment">(in Chinese)</div>.</div></div></div></div><div content-id="DSV001C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Le Bras</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Dechamp</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Krieg</div> <div class="given-names">F.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Detection of QTL with effects on osmoregulation capacities in the rainbow trout (<em>Oncorhynchus mykiss</em>)</div>, <div class="source">BMC Genet.</div> , <div class="volume">12</div>, <div class="fpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20QTL%20with%20effects%20on%20osmoregulation%20capacities%20in%20the%20rainbow%20trout%20(Oncorhynchus%20mykiss)&amp;author=Y.%20Le%20Bras&amp;author=N.%20Dechamp&amp;author=F.%20Krieg&amp;publication_year=2011&amp;journal=BMC%20Genet.&amp;volume=12&amp;pages=46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-12-46" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-12-46" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-12-46"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">E.J.</div></div> <div class="name"><div class="surname">Luoma</div> <div class="given-names">A.M.</div></div></div> <div class="year">2013</div>, <div class="article-title">The adaptable major histocompatibility complex (MHC) fold: structure and function of nonclassical and MHC class I-like molecules</div>, <div class="source">Annu. Rev. Immunol.</div> , <div class="volume">31</div>, <div class="fpage">529</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20adaptable%20major%20histocompatibility%20complex%20(MHC)%20fold%3A%20structure%20and%20function%20of%20nonclassical%20and%20MHC%20class%20I-like%20molecules&amp;author=E.J.%20Adams&amp;author=A.M.%20Luoma&amp;publication_year=2013&amp;journal=Annu.%20Rev.%20Immunol.&amp;volume=31&amp;pages=529-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-immunol-032712-095912" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-immunol-032712-095912" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-immunol-032712-095912"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23298204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">T.J.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.X.</div></div></div> <div class="year">2010</div>, <div class="article-title">MHC class IIalpha gene polymorphism and its association with resistance/susceptibility to Vibrio anguillarum in Japanese flounder (<em>Paralichthys olivaceus</em>)</div>, <div class="source">Dev. Comp. Immunol.</div> , <div class="volume">34</div>, <div class="fpage">1042</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MHC%20class%20IIalpha%20gene%20polymorphism%20and%20its%20association%20with%20resistance%2Fsusceptibility%20to%20Vibrio%20anguillarum%20in%20Japanese%20flounder%20(Paralichthys%20olivaceus)&amp;author=T.J.%20Xu&amp;author=S.L.%20Chen&amp;author=Y.X.%20Zhang&amp;publication_year=2010&amp;journal=Dev.%20Comp.%20Immunol.&amp;volume=34&amp;pages=1042-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dci.2010.05.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dci.2010.05.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dci.2010.05.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20580738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">T.J.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">Y.N.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div></div> <div class="year">2010</div>, <div class="article-title">Allelic variation, balancing selection and positive selected sites detected from MHC class Ialpha gene of olive flounder</div>, <div class="source">Genetica</div> , <div class="volume">138</div>, <div class="fpage">1251</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%2C%20balancing%20selection%20and%20positive%20selected%20sites%20detected%20from%20MHC%20class%20Ialpha%20gene%20of%20olive%20flounder&amp;author=T.J.%20Xu&amp;author=Y.N.%20Sun&amp;author=S.L.%20Chen&amp;publication_year=2010&amp;journal=Genetica&amp;volume=138&amp;pages=1251-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-010-9524-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-010-9524-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-010-9524-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21110065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Liang</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Polymorphism and balancing selection of MHC class II DAB gene in 7 selective flounder (<em>Paralichthys olivaceus</em>) families</div>, <div class="source">Evid. Based Complement. Alternat. Med.</div> , <div class="volume">20,11</div>, <div class="fpage">613629</div>.</div></div></div></div><div content-id="DSV001C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Buermans</div> <div class="given-names">H.P.</div></div> <div class="name"><div class="surname">den Dunnen</div> <div class="given-names">J.T.</div></div></div> <div class="year">2014</div>, <div class="article-title">Next generation sequencing technology: advances and applications</div>, <div class="source">Biochim. Biophys. Acta.</div> , <div class="volume">10</div>, <div class="fpage">1932</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20technology%3A%20advances%20and%20applications&amp;author=H.P.%20Buermans&amp;author=J.T.%20den%20Dunnen&amp;publication_year=2014&amp;journal=Biochim.%20Biophys.%20Acta.&amp;volume=10&amp;pages=1932-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbadis.2014.06.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbadis.2014.06.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbadis.2014.06.015"> </span></div> </div></div></div></div></div><div content-id="DSV001C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Treangen</div> <div class="given-names">T.J.</div></div> <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2012</div>, <div class="article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">13</div>, <div class="fpage">36</div>–<div class="lpage">46</div>.</div></div></div></div><div content-id="DSV001C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Gao</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Whole genome comparative analysis of channel catfish (<em>Ictalurus punctatus</em>) with four model fish species</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">780</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20comparative%20analysis%20of%20channel%20catfish%20(Ictalurus%20punctatus)%20with%20four%20model%20fish%20species&amp;author=Y.%20Jiang&amp;author=X.%20Gao&amp;author=S.%20Liu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=780" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24215161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kluver</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Pfennig</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Pala</div> <div class="given-names">I.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Differential expression of anti-Mullerian hormone (amh) and anti-Mullerian hormone receptor type II (amhrII) in the teleost medaka</div>, <div class="source">Dev. Dynam.</div> , <div class="volume">236</div>, <div class="fpage">271</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20anti-Mullerian%20hormone%20(amh)%20and%20anti-Mullerian%20hormone%20receptor%20type%20II%20(amhrII)%20in%20the%20teleost%20medaka&amp;author=N.%20Kluver&amp;author=F.%20Pfennig&amp;author=I.%20Pala&amp;publication_year=2007&amp;journal=Dev.%20Dynam.&amp;volume=236&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0177" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0177"> </span></div> </div></div></div></div></div><div content-id="DSV001C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dai</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Xiong</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Dai</div> <div class="given-names">X.</div></div></div> <div class="year">2014</div>, <div class="article-title">Neighboring genes show interchromosomal colocalization after their separation</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">31</div>, <div class="fpage">1166</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neighboring%20genes%20show%20interchromosomal%20colocalization%20after%20their%20separation&amp;author=Z.%20Dai&amp;author=Y.%20Xiong&amp;author=X.%20Dai&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=31&amp;pages=1166-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24505120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Comparative genomic analysis of catfish linkage group 8 reveals two homologous chromosomes in zebrafish and other teleosts with extensive inter-chromosomal rearrangements</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">387</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20analysis%20of%20catfish%20linkage%20group%208%20reveals%20two%20homologous%20chromosomes%20in%20zebrafish%20and%20other%20teleosts%20with%20extensive%20inter-chromosomal%20rearrangements&amp;author=Y.%20Zhang&amp;author=S.%20Liu&amp;author=J.%20Lu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23758806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Woods</div> <div class="given-names">I.G.</div></div> <div class="name"><div class="surname">Wilson</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Friedlander</div> <div class="given-names">B.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">The zebrafish gene map defines ancestral vertebrate chromosomes</div>, <div class="source">Genome Res.</div> , <div class="volume">15</div>, <div class="fpage">1307</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zebrafish%20gene%20map%20defines%20ancestral%20vertebrate%20chromosomes&amp;author=I.G.%20Woods&amp;author=C.%20Wilson&amp;author=B.%20Friedlander&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1307-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4134305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4134305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4134305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16109975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Postlethwait</div> <div class="given-names">J.H.</div></div> <div class="name"><div class="surname">Woods</div> <div class="given-names">I.G.</div></div> <div class="name"><div class="surname">Ngo-Hazelett</div> <div class="given-names">P.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">Zebrafish comparative genomics and the origins of vertebrate chromosomes</div>, <div class="source">Genome Res.</div> , <div class="volume">10</div>, <div class="fpage">1890</div>–<div class="lpage">902</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zebrafish%20comparative%20genomics%20and%20the%20origins%20of%20vertebrate%20chromosomes&amp;author=J.H.%20Postlethwait&amp;author=I.G.%20Woods&amp;author=P.%20Ngo-Hazelett&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1890-902" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.164800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.164800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.164800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11116085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Glasauer</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Neuhauss</div> <div class="given-names">S.C.</div></div></div> <div class="year">2014</div>, <div class="article-title">Whole-genome duplication in teleost fishes and its evolutionary consequences</div>, <div class="source">Mol. Genet. Genomics.</div> , <div class="volume">289</div>, <div class="fpage">1045</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20duplication%20in%20teleost%20fishes%20and%20its%20evolutionary%20consequences&amp;author=S.M.%20Glasauer&amp;author=S.C.%20Neuhauss&amp;publication_year=2014&amp;journal=Mol.%20Genet.%20Genomics.&amp;volume=289&amp;pages=1045-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-014-0889-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-014-0889-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-014-0889-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25092473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV001C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV001C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brunet</div> <div class="given-names">F.G.</div></div> <div class="name"><div class="surname">Roest Crollius</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Paris</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">Gene loss and evolutionary rates following whole-genome duplication in teleost fishes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">23</div>, <div class="fpage">1808</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20loss%20and%20evolutionary%20rates%20following%20whole-genome%20duplication%20in%20teleost%20fishes&amp;author=F.G.%20Brunet&amp;author=H.%20Roest%20Crollius&amp;author=M.%20Paris&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1808-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16809621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=332908&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_table5.xls?Expires=2147483647&amp;Signature=I87rlXUYnk~UkEiBcgMX9d4ZLFa9-Rtg3cn8QfG9zvZOsFzfOPcV8zCC5ZZvbX4iIZg7cFs7Jdn7xD42fE84PP~7CZBknxAWiexFQ00yAx-axBE5YlUWaGEjiMzmBrkFSfr4Z6kgQ6ggvrffpbqkoQzSeyE0s41L75u1ki8Zvkg9C8yJiw5eRDrr1WonFT8t3Z~mmJYkQMMXrsbRNQiWGx2Ije9QcYruNMII-Uys5uU2-BY2h-eLeK70r0AeQpB6~jCNMs7uBKKPO7DRbYlMLMvhkM9npG8Ocq-oFAZXlRLdTn0DnNLrbIaNcXvSILwnAHQfUBgMPrmiKw~NW0a-7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_table4.xls?Expires=2147483647&amp;Signature=bgm93ycO9TnchZx7Ck6qdlILzGU35RNqfhyQDFrinna51Y9iKAOSHxBs65xcDb~WxhTdZ8kIt2okfCH53ll0RvO19DNXmR1NYE34a~9cIGD2kdICGLryCKOK53AzMYebYyQu1DXOIsh2dCNPnJM9V92KaAqBkJMgdS-vfeeKBUqVMB4jxIQyi2Yw7HFmV9yskq31B54xTpDfSQWw2H5N~~Vi~qWLPC4t7OBRKZKoG1KGCBZnnWTEf8aSHDS7fM83cbUXqHa-5bCQbiE25KSuCDjP11V71ELWnb4Jt25NC56W-5Lmc9IP-IW4tWneLm4wE0xtL6cf7dbHlGK-r0R6og__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_table3.xls?Expires=2147483647&amp;Signature=U6u4jbg6RpippAdLP9PH5L8Vcqm0ndqzAgB4Y4IYR84detEOvy9Yadz-viaMNa5GPGeVBGZF778dzlwhPDI1nHiC0rLbW~OpfUSllAiuAWkMaW7qGzp6ZCRhe9mEZKIgpMrgTZL4wx9O~kI6GC1fiNevuZF66MYVvgkck~FCDg2jrBexYp1~M4l66vTtDlDBpiPX-uj-5I6Litwcnbq7I1y1DtuzfpogVHtiXtWx7fVdfdsOeOVME9ggPUAQP19dc2fgjJf~64b6xa-An5CPNVTQF~3h~bnpNvkjjQ~MDeysbovNuDtPmcY0Nrve~fMcTLk-mx~~lI~8oe9e8-bjBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_table2.xls?Expires=2147483647&amp;Signature=kfK8liFf0gbL4DNYjeaSC4uDTbkz4hSS2VjfqFxUT1Cw6kHaY1YXwZnp-pVL2a~-4hfLGNa5qmpMc9VITCYJeuFOO31Chovdn9HMgDp0PUK7bDGjoFDg0MvONwDMcuUPH50iGq8Mw3SmVwtEo0xvOaN2XS5UnrdumOgWICv6fYcpePvTcbV3IFEVJtaAONT~biA1JAAVuzBMWDjc5H77m5mm9pRqe1-g6BY3AD1P1EOH6D5W0TQXnVvdFXRDs4V4KxkttZwb0iCjbBoS4DEizVi2qvkoGQrJt~9d1bzlaySP-Fo~fMzKQ9b-tVuVEqG7609N15voUcqSFmXasmrp0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_table1.xls?Expires=2147483647&amp;Signature=rwUtU11T07gPnLvhfCprYax2~7kFlCehQIdkNy7OpaW~tBBhcLD-y~t-h8Y-YfdXceZ1TJW6l8GezPIYjAqNJ7IFxLtrxMPqQSlwtjjDxmGUD2dxFZT9x6-WqLsXZJGiyB4BpMdIyLrDFq-7VGU6xeYSAm3lV1kKfNI6D6gxu7NH8uNqerMO~nERbIQOLRD5mv5uwXwR59xVJvXK2B-mKFOWems6pILbwckqYxl87HOxjMri1PJrUTtkBpmQNOXInqkpqmQ0JgavNNf4bCU9uMc43uVh-FpyzXZs7uR~S7vqUi6UdnKlfmzetHudP97Wt3yccHNzE3DAGK9xFHyhTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_fig3.pdf?Expires=2147483647&amp;Signature=4ExJ1qi-C5eVh5k6OPDX1ekxNEh0tMG~rkIjhQZB4OeDOksHfvCw2f0z2Zv1seyf--Q6CA8J1wWx1rKXz3jf-PO-josFZXPVM1Lzpc3yfnEEdrE9f58dlGx2OyHpPV0tEivQ8-Dxay9JJP6ag9XkV70R1uocEO7KilGPGXDlYvq3W3Acj8C7iXPVrL9bnEexKOjhYXQuLZbmOp~GiGZJ4Hl9RHOVtJ8fzyYR4hwsogYDN-MomeK5CwxqTDCMlByWdoA33OA2GIwj3yTVQ~xxtwjbc9a5X4xfFZ2H2LbYcKtrP0Nd0A9WO6BFOzYem07YwEiNISwDyRM9RCWWCI2nog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_fig2.pdf?Expires=2147483647&amp;Signature=kMPOcrlYRC45X9u2MzXGJ8ei7mE0q7TB5Kgqi2yoJ0w-20RZcPV90w-U9ZlWGE8rAmuoYcEO8uoNVyeMQgKK6TuqDa0Ysc0qhO38Ye4DpkfTXlO3sjArQ7KrqMK7DCYz9bE2c84AKJo4YC-KQF9xgK4XPDiYEavnIOMxlh9Ha1SNDyP1We~Ri06nzxvN5RW28wRdXy~56H1QWA8~SliJuLp9dkRdWiqak0TxF~qDTYO1JBKChHWR3x78wjNczXknpfJoxtDzc~l7XZBwAjDJSRalElDk6HwyfNZaYxLeiLIoBHI1p4gfXkeA4LMCKBRQdFyRFXSqzyoagdkxtCkNXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp_fig1.pdf?Expires=2147483647&amp;Signature=FXUXYNaIonelzT1Ua6lv~i98A0jblXXjzwTAoMTAqjiT4ZWAfshoQLxdzxMVwSFaGa3hpzZPi4BlgCdwNRNOhz-juWe6oXMVU2v2TO6HpsDemj6aRgyW9cl~71X4GyRoKFYnqNb7pdnfZRM3DYrlV~FoJnbHvyOA028WqnZUiPrm13OXoElGyimfFvV05D5IQgxZ5Ey1w0AY7BE-iFYvXJlb7FUd6ZmDkuSpbdcdYPcwc9AJXxQyi5Be3fQbnFgLwg-JdEKrIydQf5vV9JXSEAfO9Vdt3R9pZCFwgvLwBFTQeOFR~SaI72cHesy-aBakLnrOfIxmhbkR-IXn50BqPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsv001/2/dsv001-dsv001supp.docx?Expires=2147483647&amp;Signature=RFUNT4JHLKPjZxjv~T~MYfCl-aDmZ2tJZUBvl9E5aT0rvfcqmisIP37-1sBClqlfxK1-2sB9mz1YCocs5S~CAbUlGEeT4ADBWmfm9Fl32jF-jAvajH~g5trk4n6IzrSdy6O1opkiEWAHZgce5mtL3-ZdKmVwffuA9WWQsOtMW2wq3lPGMgrx6HnWoZJbTwzW3uuAnOGlKza9bMhG0ZgLw1Meee2eEHhwppALLnadu35mqh91sCKwyosJ~Kt9g4iH82uVcI6gdYeq-9MSAOL5jqT9n1svzi7wmssg9-rC~aVNDsHEbjMHaQKRO8pCVkY5VPpiwewCYYw8S2VyWDDpZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,004</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">36</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="332908"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f2%2f161%2f332908">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000353644600004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (36)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F2%2F161%2F332908" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/332908'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f161%2f332908">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

