

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/6/561/2647449","@type":"ScholarlyArticle","name":"A draft genome of the brown alga,  Cladosiphon okamuranus , S-strain: a platform for future studies of ‘mozuku’ biology","datePublished":"2016-08-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2016-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw039","keywords":["brown alga","<italic>Cladosiphon okamuranus</italic>","genome decoding","genes for enzymes of polysaccharide synthesis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/6/561/2647449","author":[{"name":"Nishitsuji, Koki","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Arimoto, Asuka","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Iwai, Kenji","affiliation":"2 Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan","@type":"Person"},{"name":"Sudo, Yusuke","affiliation":"2 Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan","@type":"Person"},{"name":"Hisata, Kanako","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Fujie, Manabu","affiliation":"3 DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Arakaki, Nana","affiliation":"3 DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Kushiro, Tetsuo","affiliation":"4 School of Agriculture, Meiji University, Kawasaki, Kanagawa 214-8571, Japan","@type":"Person"},{"name":"Konishi, Teruko","affiliation":"5 Department of Bioscience and Biotechnology, Faculty of Agriculture, University of the Ryukyus, Nishihara, Okinawa 903-0213, Japan","@type":"Person"},{"name":"Shinzato, Chuya","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Satoh, Noriyuki","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"},{"name":"Shoguchi, Eiichi","affiliation":"1 Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan","@type":"Person"}],"description":"The brown alga, Cladosiphon okamuranus (Okinawa mozuku), is economically one of the most important edible seaweeds, and is cultivated for market primarily in Ok","pageStart":"561","pageEnd":"570","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A draft genome of the brown alga,  Cladosiphon okamuranus , S-strain: a platform for future studies of ‘mozuku’ biology","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="draft genome of the brown alga,  Cladosiphon okamuranus , S-strain: a platform for future studies of ‘mozuku’ biology" />
    <meta property="og:description" content="The brown alga, Cladosiphon okamuranus (Okinawa mozuku), is economically one of the most important edible seaweeds, and is cultivated for market primarily in Ok" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/6/561/2647449" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-08-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nishitsuji, Koki" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Arimoto, Asuka" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Iwai, Kenji" /><meta name="citation_author_institution" content="Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan" /><meta name="citation_author" content="Sudo, Yusuke" /><meta name="citation_author_institution" content="Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan" /><meta name="citation_author" content="Hisata, Kanako" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Fujie, Manabu" /><meta name="citation_author_institution" content="DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Arakaki, Nana" /><meta name="citation_author_institution" content="DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Kushiro, Tetsuo" /><meta name="citation_author_institution" content="School of Agriculture, Meiji University, Kawasaki, Kanagawa 214-8571, Japan" /><meta name="citation_author" content="Konishi, Teruko" /><meta name="citation_author_institution" content="Department of Bioscience and Biotechnology, Faculty of Agriculture, University of the Ryukyus, Nishihara, Okinawa 903-0213, Japan" /><meta name="citation_author" content="Shinzato, Chuya" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Satoh, Noriyuki" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_author" content="Shoguchi, Eiichi" /><meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan" /><meta name="citation_title" content="A draft genome of the brown alga, Cladosiphon okamuranus, S-strain: a platform for future studies of ‘mozuku’ biology" /><meta name="citation_firstpage" content="561" /><meta name="citation_lastpage" content="570" /><meta name="citation_doi" content="10.1093/dnares/dsw039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2016/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The new higher level classification of eukaryotes with emphasis on the taxonomy of protists;  citation_journal_title=J. Eukaryot. Microbiol;  citation_year=2005;  citation_volume=52;  citation_pages=399-451; " /><meta name="citation_reference" content="citation_title=Chromalveolates and the evolution of plastids by secondary endosymbiosis;  citation_journal_title=J Eukaryot. Microbiol;  citation_year=2009;  citation_volume=56;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Principles of protein and lipid targeting in secondary symbiogenesis: euglenoid, dinoflagellate, and sporozoan plastid origins and the eukaryote family tree;  citation_journal_title=J. Eukaryot. Microbiol;  citation_year=1999;  citation_volume=46;  citation_pages=347-66; " /><meta name="citation_reference" content="Van Den Hoek C. Mann D.G. Jahns H.M.  1995 , Algae: An Introduction to Phycology. Cambridge University Press, Cambridge. " /><meta name="citation_reference" content="citation_title=Checklist of marine algae of Japan (revised in 2015);  citation_journal_title=Jpn. J. Phycol (Sôrui);  citation_year=2015;  citation_volume=63;  citation_pages=129-89; " /><meta name="citation_reference" content="citation_title=Chemical characterization of fucoidan from commercially cultured nemacystus decipiens (Itomozuku);  citation_journal_title=Biosci. Biotechnol. Biochem;  citation_year=1999;  citation_volume=63;  citation_pages=1813-5; " /><meta name="citation_reference" content="citation_title=Sulfated polysaccharides are potent and selective inhibitors of various enveloped viruses, including herpes simplex virus, cytomegalovirus, vesicular stomatitis virus, and human immunodeficiency virus;  citation_journal_title=Antimicrob. Agents Chemother;  citation_year=1988;  citation_volume=32;  citation_pages=1742-5; " /><meta name="citation_reference" content="citation_title=Pathway of algnic acid synthesis in the marine brown alga, Fucus gardneri Silva;  citation_journal_title=J. Biol. Chem;  citation_year=1966;  citation_volume=241;  citation_pages=5284-97; " /><meta name="citation_reference" content="citation_title=The cell wall polysaccharide metabolism of the brown alga Ectocarpus siliculosus. Insights into the evolution of extracellular matrix polysaccharides in Eukaryotes;  citation_journal_title=New Phytol;  citation_year=2010;  citation_volume=188;  citation_pages=82-97; " /><meta name="citation_reference" content="citation_title=The Ectocarpus genome and the independent evolution of multicellularity in brown algae;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=465;  citation_pages=617-21; " /><meta name="citation_reference" content="citation_title=Saccharina genomes provide novel insight into kelp biology;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=6986" /><meta name="citation_reference" content="citation_title=Phylogenetic relationships of the brown algal orders Ectocarpales, Chordariales, Dictyosiphonales, and Tilopteridales (Phaeophyceae) based on RUBISCO large subunit and spacer sequences;  citation_journal_title=J. Phycol;  citation_year=1998;  citation_volume=34;  citation_pages=1038-48; " /><meta name="citation_reference" content="citation_title=Whole-genome re-sequencing;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2006;  citation_volume=16;  citation_pages=545-52; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=The sequence and de novo assembly of the giant panda genome;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=311-7; " /><meta name="citation_reference" content="citation_title=HaploMerger: reconstructing allelic relationships for polymorphic diploid genome assemblies;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1581-1588; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1420-8; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1086-92; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=5654-66; " /><meta name="citation_reference" content="citation_title=Draft genome of the pearl oyster Pinctada fucata: a platform for understanding bivalve biology;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=117-30; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21 Suppl 1;  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=CENSOR – a program for identification and elimination of repetitive elements from DNA sequences;  citation_journal_title=Comput. Chem;  citation_year=1996;  citation_volume=20;  citation_pages=119-21; " /><meta name="citation_reference" content="citation_title=Repbase update, a database of eukaryotic repetitive elements;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D247-51; " /><meta name="citation_reference" content="citation_title=Profile hidden Markov models;  citation_journal_title=Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=755-63; " /><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; " /><meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building;  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=221-4; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=JBrowse: a next-generation genome browser;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1630-8; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=Hierarchical clustering using mutual information;  citation_journal_title=Europhys. Lett;  citation_year=2005;  citation_volume=70;  citation_pages=278-84; " /><meta name="citation_reference" content="citation_title=Hierarchical clustering via joint between-within distances: Extending Ward's minimum variance method;  citation_journal_title=J. Classif;  citation_year=2005;  citation_volume=22;  citation_pages=151-83; " /><meta name="citation_reference" content="citation_title=MaxBin: an automated binning method to recover individual genomes from metagenomes using an expectation-maximization algorithm;  citation_journal_title=Microbiome;  citation_year=2014;  citation_volume=2;  citation_pages=26" /><meta name="citation_reference" content="citation_title=Impacts of ambient salinity and copper on brown algae: 2. Interactive effects on phenolic pool and assessment of metal binding capacity of phlorotannin;  citation_journal_title=Aquat. Toxicol;  citation_year=2011;  citation_volume=104;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Potential pharmacological applications of polyphenolic derivatives from marine brown algae;  citation_journal_title=Environ. Toxicol. Pharmacol;  citation_year=2011;  citation_volume=32;  citation_pages=325-35; " /><meta name="citation_reference" content="citation_title=Plant flavonoid: biological molecules for useful exploitation;  citation_journal_title=Aust. J. Plant Physiol;  citation_year=1995;  citation_volume=22;  citation_pages=87-99; " /><meta name="citation_reference" content="citation_title=Slicks associated with Trichodesmium Blooms in the Sargasso Sea;  citation_journal_title=Nature;  citation_year=1965;  citation_volume=205;  citation_pages=830-1; " /><meta name="citation_reference" content="citation_title=Inducible chemical resistance to herbivory in the brown seaweed Ascophyllum nodosum;  citation_journal_title=Ecology;  citation_year=2000;  citation_volume=81;  citation_pages=3215-25; " /><meta name="citation_reference" content="citation_title=Defensive and sensory chemical ecology of brown algae;  citation_journal_title=Adv. Botanical Res;  citation_year=2006;  citation_volume=43;  citation_pages=1-91; " /><meta name="citation_reference" content="citation_title=Role of polyphenolic molecular size in reduction of assimilation efficiency in Xiphister mucosus;  citation_journal_title=Ecology;  citation_year=1993;  citation_volume=74;  citation_pages=891-903; " /><meta name="citation_reference" content="citation_title=Phenology, TPC and size-fractioning phenolics variability in temperate Sargassaceae (Phaeophyceae, Fucales) from Western Brittany: native versus introduced species;  citation_journal_title=Mar. Environ. Res;  citation_year=2012;  citation_volume=80;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Structure/function analysis of a type iii polyketide synthase in the brown alga Ectocarpus siliculosus reveals a biochemical pathway in phlorotannin monomer biosynthesis;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3089-103; " /><meta name="citation_reference" content="citation_title=B2G-FAR, a species-centered GO annotation repository;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=919-24; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/6/561/8325707/dsw039.pdf" /><meta name="description" content="The brown alga, Cladosiphon okamuranus (Okinawa mozuku), is economically one of the most important edible seaweeds, and is cultivated for market primarily in Ok" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/6/561/2647449" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/6/561/2647449" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/6/561/2647449");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw039';
    </script>
<title>
	draft genome of the brown alga, Cladosiphon okamuranus, S-strain: a platform for future studies of ‘mozuku’ biology | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2647449" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be2CATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGl0c3VqaSwgS29raSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1hcmluZSBHZW5vbWljcyBVbml0LCBPa2luYXdhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5IEdyYWR1YXRlIFVuaXZlcnNpdHksIE9ubmEsIE9raW5hd2EgOTA0LTA0OTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQXJpbW90bywgQXN1a2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYXJpbmUgR2Vub21pY3MgVW5pdCwgT2tpbmF3YSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBHcmFkdWF0ZSBVbml2ZXJzaXR5LCBPbm5hLCBPa2luYXdhIDkwNC0wNDk1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl3YWksIEtlbmppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2tpbmF3YSBQcmVmZWN0dXJhbCBGaXNoZXJpZXMgUmVzZWFyY2ggYW5kIEV4dGVuc2lvbiBDZW50ZXIsIEl0b21hbiwgT2tpbmF3YSA5MDEtMDM1NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWRvLCBZdXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPa2luYXdhIFByZWZlY3R1cmFsIEZpc2hlcmllcyBSZXNlYXJjaCBhbmQgRXh0ZW5zaW9uIENlbnRlciwgSXRvbWFuLCBPa2luYXdhIDkwMS0wMzU0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhpc2F0YSwgS2FuYWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFyaW5lIEdlbm9taWNzIFVuaXQsIE9raW5hd2EgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgR3JhZHVhdGUgVW5pdmVyc2l0eSwgT25uYSwgT2tpbmF3YSA5MDQtMDQ5NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppZSwgTWFuYWJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRE5BIFNlcXVlbmNpbmcgU2VjdGlvbiwgT2tpbmF3YSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBHcmFkdWF0ZSBVbml2ZXJzaXR5LCBPbm5hLCBPa2luYXdhIDkwNC0wNDk1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyYWtha2ksIE5hbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJETkEgU2VxdWVuY2luZyBTZWN0aW9uLCBPa2luYXdhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5IEdyYWR1YXRlIFVuaXZlcnNpdHksIE9ubmEsIE9raW5hd2EgOTA0LTA0OTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VzaGlybywgVGV0c3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEFncmljdWx0dXJlLCBNZWlqaSBVbml2ZXJzaXR5LCBLYXdhc2FraSwgS2FuYWdhd2EgMjE0LTg1NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29uaXNoaSwgVGVydWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBVbml2ZXJzaXR5IG9mIHRoZSBSeXVreXVzLCBOaXNoaWhhcmEsIE9raW5hd2EgOTAzLTAyMTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbnphdG8sIENodXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFyaW5lIEdlbm9taWNzIFVuaXQsIE9raW5hd2EgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgR3JhZHVhdGUgVW5pdmVyc2l0eSwgT25uYSwgT2tpbmF3YSA5MDQtMDQ5NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvaCwgTm9yaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYXJpbmUgR2Vub21pY3MgVW5pdCwgT2tpbmF3YSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBHcmFkdWF0ZSBVbml2ZXJzaXR5LCBPbm5hLCBPa2luYXdhIDkwNC0wNDk1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNob2d1Y2hpLCBFaWljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNYXJpbmUgR2Vub21pY3MgVW5pdCwgT2tpbmF3YSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBHcmFkdWF0ZSBVbml2ZXJzaXR5LCBPbm5hLCBPa2luYXdhIDkwNC0wNDk1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBkcmFmdCBnZW5vbWUgb2YgdGhlIGJyb3duIGFsZ2EsIENsYWRvc2lwaG9uIG9rYW11cmFudXMsIFMtc3RyYWluOiBhIHBsYXRmb3JtIGZvciBmdXR1cmUgc3R1ZGllcyBvZiDigJhtb3p1a3XigJkgYmlvbG9neSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjU2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmV3IGhpZ2hlciBsZXZlbCBjbGFzc2lmaWNhdGlvbiBvZiBldWthcnlvdGVzIHdpdGggZW1waGFzaXMgb24gdGhlIHRheG9ub215IG9mIHByb3Rpc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFdWthcnlvdC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9Mzk5LTQ1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21hbHZlb2xhdGVzIGFuZCB0aGUgZXZvbHV0aW9uIG9mIHBsYXN0aWRzIGJ5IHNlY29uZGFyeSBlbmRvc3ltYmlvc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KIEV1a2FyeW90LiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW5jaXBsZXMgb2YgcHJvdGVpbiBhbmQgbGlwaWQgdGFyZ2V0aW5nIGluIHNlY29uZGFyeSBzeW1iaW9nZW5lc2lzOiBldWdsZW5vaWQsIGRpbm9mbGFnZWxsYXRlLCBhbmQgc3Bvcm96b2FuIHBsYXN0aWQgb3JpZ2lucyBhbmQgdGhlIGV1a2FyeW90ZSBmYW1pbHkgdHJlZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXVrYXJ5b3QuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTM0Ny02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVmFuIERlbiBIb2VrIEMuIE1hbm4gRC5HLiBKYWhucyBILk0uICAxOTk1ICwgQWxnYWU6IEFuIEludHJvZHVjdGlvbiB0byBQaHljb2xvZ3kuIENhbWJyaWRnZSBVbml2ZXJzaXR5IFByZXNzLCBDYW1icmlkZ2UuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoZWNrbGlzdCBvZiBtYXJpbmUgYWxnYWUgb2YgSmFwYW4gKHJldmlzZWQgaW4gMjAxNSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUpwbi4gSi4gUGh5Y29sIChTw7RydWkpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MTI5LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGVtaWNhbCBjaGFyYWN0ZXJpemF0aW9uIG9mIGZ1Y29pZGFuIGZyb20gY29tbWVyY2lhbGx5IGN1bHR1cmVkIG5lbWFjeXN0dXMgZGVjaXBpZW5zIChJdG9tb3p1a3UpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0xODEzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1bGZhdGVkIHBvbHlzYWNjaGFyaWRlcyBhcmUgcG90ZW50IGFuZCBzZWxlY3RpdmUgaW5oaWJpdG9ycyBvZiB2YXJpb3VzIGVudmVsb3BlZCB2aXJ1c2VzLCBpbmNsdWRpbmcgaGVycGVzIHNpbXBsZXggdmlydXMsIGN5dG9tZWdhbG92aXJ1cywgdmVzaWN1bGFyIHN0b21hdGl0aXMgdmlydXMsIGFuZCBodW1hbiBpbW11bm9kZWZpY2llbmN5IHZpcnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbnRpbWljcm9iLiBBZ2VudHMgQ2hlbW90aGVyOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTc0Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXRod2F5IG9mIGFsZ25pYyBhY2lkIHN5bnRoZXNpcyBpbiB0aGUgbWFyaW5lIGJyb3duIGFsZ2EsIEZ1Y3VzIGdhcmRuZXJpIFNpbHZhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTY2OyAgY2l0YXRpb25fdm9sdW1lPTI0MTsgIGNpdGF0aW9uX3BhZ2VzPTUyODQtOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjZWxsIHdhbGwgcG9seXNhY2NoYXJpZGUgbWV0YWJvbGlzbSBvZiB0aGUgYnJvd24gYWxnYSBFY3RvY2FycHVzIHNpbGljdWxvc3VzLiBJbnNpZ2h0cyBpbnRvIHRoZSBldm9sdXRpb24gb2YgZXh0cmFjZWxsdWxhciBtYXRyaXggcG9seXNhY2NoYXJpZGVzIGluIEV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9ODItOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBFY3RvY2FycHVzIGdlbm9tZSBhbmQgdGhlIGluZGVwZW5kZW50IGV2b2x1dGlvbiBvZiBtdWx0aWNlbGx1bGFyaXR5IGluIGJyb3duIGFsZ2FlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY1OyAgY2l0YXRpb25fcGFnZXM9NjE3LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWNjaGFyaW5hIGdlbm9tZXMgcHJvdmlkZSBub3ZlbCBpbnNpZ2h0IGludG8ga2VscCBiaW9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9Njk4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyByZWxhdGlvbnNoaXBzIG9mIHRoZSBicm93biBhbGdhbCBvcmRlcnMgRWN0b2NhcnBhbGVzLCBDaG9yZGFyaWFsZXMsIERpY3R5b3NpcGhvbmFsZXMsIGFuZCBUaWxvcHRlcmlkYWxlcyAoUGhhZW9waHljZWFlKSBiYXNlZCBvbiBSVUJJU0NPIGxhcmdlIHN1YnVuaXQgYW5kIHNwYWNlciBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBoeWNvbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTEwMzgtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSByZS1zZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3Rpb24gb2YgdGhlIG9jdG9wbG9pZCBzdHJhd2JlcnJ5IGdlbm9tZSBieSBkZWVwIHNlcXVlbmNpbmcgb2YgdGhlIGdlbm9tZXMgb2YgRnJhZ2FyaWEgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpbW1vbWF0aWM6IGEgZmxleGlibGUgdHJpbW1lciBmb3IgSWxsdW1pbmEgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMTE0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2FmZm9sZGluZyBwcmUtYXNzZW1ibGVkIGNvbnRpZ3MgdXNpbmcgU1NQQUNFOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYW5kIGRlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIGdpYW50IHBhbmRhIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTMxMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IYXBsb01lcmdlcjogcmVjb25zdHJ1Y3RpbmcgYWxsZWxpYyByZWxhdGlvbnNoaXBzIGZvciBwb2x5bW9ycGhpYyBkaXBsb2lkIGdlbm9tZSBhc3NlbWJsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTU4MS0xNTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRUdNQTogYSBwaXBlbGluZSB0byBhY2N1cmF0ZWx5IGFubm90YXRlIGNvcmUgZ2VuZXMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEwNjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SURCQS1VRDogYSBkZSBub3ZvIGFzc2VtYmxlciBmb3Igc2luZ2xlLWNlbGwgYW5kIG1ldGFnZW5vbWljIHNlcXVlbmNpbmcgZGF0YSB3aXRoIGhpZ2hseSB1bmV2ZW4gZGVwdGg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTQyMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZWx2ZXQ6IGFsZ29yaXRobXMgZm9yIGRlIG5vdm8gc2hvcnQgcmVhZCBhc3NlbWJseSB1c2luZyBkZSBCcnVpam4gZ3JhcGhzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9ODIxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9hc2VzOiByb2J1c3QgZGUgbm92byBSTkEtc2VxIGFzc2VtYmx5IGFjcm9zcyB0aGUgZHluYW1pYyByYW5nZSBvZiBleHByZXNzaW9uIGxldmVsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMDg2LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBuYXRpdmUgYW5kIHN5bnRlbmljYWxseSBtYXBwZWQgY0ROQSBhbGlnbm1lbnRzIHRvIGltcHJvdmUgZGUgbm92byBnZW5lIGZpbmRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgdGhlIEFyYWJpZG9wc2lzIGdlbm9tZSBhbm5vdGF0aW9uIHVzaW5nIG1heGltYWwgdHJhbnNjcmlwdCBhbGlnbm1lbnQgYXNzZW1ibGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz01NjU0LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgb2YgdGhlIHBlYXJsIG95c3RlciBQaW5jdGFkYSBmdWNhdGE6IGEgcGxhdGZvcm0gZm9yIHVuZGVyc3RhbmRpbmcgYml2YWx2ZSBiaW9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTE3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0cyBmaW5kZXI6IGEgcHJvZ3JhbSB0byBhbmFseXplIEROQSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBmYW1pbGllcyBpbiBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMSBTdXBwbCAxOyAgY2l0YXRpb25fcGFnZXM9aTM1MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRU5TT1Ig4oCTIGEgcHJvZ3JhbSBmb3IgaWRlbnRpZmljYXRpb24gYW5kIGVsaW1pbmF0aW9uIG9mIHJlcGV0aXRpdmUgZWxlbWVudHMgZnJvbSBETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIENoZW07ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGJhc2UgdXBkYXRlLCBhIGRhdGFiYXNlIG9mIGV1a2FyeW90aWMgcmVwZXRpdGl2ZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZmFtOiBjbGFucywgd2ViIHRvb2xzIGFuZCBzZXJ2aWNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1EMjQ3LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9maWxlIGhpZGRlbiBNYXJrb3YgbW9kZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFGRlQ6IGEgbm92ZWwgbWV0aG9kIGZvciByYXBpZCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYmFzZWQgb24gZmFzdCBGb3VyaWVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zMDU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWFWaWV3IHZlcnNpb24gNDogYSBtdWx0aXBsYXRmb3JtIGdyYXBoaWNhbCB1c2VyIGludGVyZmFjZSBmb3Igc2VxdWVuY2UgYWxpZ25tZW50IGFuZCBwaHlsb2dlbmV0aWMgdHJlZSBidWlsZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjIxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E1OiBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHVzaW5nIG1heGltdW0gbGlrZWxpaG9vZCwgZXZvbHV0aW9uYXJ5IGRpc3RhbmNlLCBhbmQgbWF4aW11bSBwYXJzaW1vbnkgbWV0aG9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1KQnJvd3NlOiBhIG5leHQtZ2VuZXJhdGlvbiBnZW5vbWUgYnJvd3NlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGFzc2VtYmx5IGFuZCBxdWFudGlmaWNhdGlvbiBieSBSTkEtU2VxIHJldmVhbHMgdW5hbm5vdGF0ZWQgdHJhbnNjcmlwdHMgYW5kIGlzb2Zvcm0gc3dpdGNoaW5nIGR1cmluZyBjZWxsIGRpZmZlcmVudGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NTExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZXJhcmNoaWNhbCBjbHVzdGVyaW5nIHVzaW5nIG11dHVhbCBpbmZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVyb3BoeXMuIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yNzgtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZXJhcmNoaWNhbCBjbHVzdGVyaW5nIHZpYSBqb2ludCBiZXR3ZWVuLXdpdGhpbiBkaXN0YW5jZXM6IEV4dGVuZGluZyBXYXJkJ3MgbWluaW11bSB2YXJpYW5jZSBtZXRob2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENsYXNzaWY7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNTEtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1heEJpbjogYW4gYXV0b21hdGVkIGJpbm5pbmcgbWV0aG9kIHRvIHJlY292ZXIgaW5kaXZpZHVhbCBnZW5vbWVzIGZyb20gbWV0YWdlbm9tZXMgdXNpbmcgYW4gZXhwZWN0YXRpb24tbWF4aW1pemF0aW9uIGFsZ29yaXRobTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9tZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXBhY3RzIG9mIGFtYmllbnQgc2FsaW5pdHkgYW5kIGNvcHBlciBvbiBicm93biBhbGdhZTogMi4gSW50ZXJhY3RpdmUgZWZmZWN0cyBvbiBwaGVub2xpYyBwb29sIGFuZCBhc3Nlc3NtZW50IG9mIG1ldGFsIGJpbmRpbmcgY2FwYWNpdHkgb2YgcGhsb3JvdGFubmluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcXVhdC4gVG94aWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3RlbnRpYWwgcGhhcm1hY29sb2dpY2FsIGFwcGxpY2F0aW9ucyBvZiBwb2x5cGhlbm9saWMgZGVyaXZhdGl2ZXMgZnJvbSBtYXJpbmUgYnJvd24gYWxnYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIFRveGljb2wuIFBoYXJtYWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgZmxhdm9ub2lkOiBiaW9sb2dpY2FsIG1vbGVjdWxlcyBmb3IgdXNlZnVsIGV4cGxvaXRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXVzdC4gSi4gUGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbGlja3MgYXNzb2NpYXRlZCB3aXRoIFRyaWNob2Rlc21pdW0gQmxvb21zIGluIHRoZSBTYXJnYXNzbyBTZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk2NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDU7ICBjaXRhdGlvbl9wYWdlcz04MzAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWNpYmxlIGNoZW1pY2FsIHJlc2lzdGFuY2UgdG8gaGVyYml2b3J5IGluIHRoZSBicm93biBzZWF3ZWVkIEFzY29waHlsbHVtIG5vZG9zdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVjb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0zMjE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWZlbnNpdmUgYW5kIHNlbnNvcnkgY2hlbWljYWwgZWNvbG9neSBvZiBicm93biBhbGdhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBCb3RhbmljYWwgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBwb2x5cGhlbm9saWMgbW9sZWN1bGFyIHNpemUgaW4gcmVkdWN0aW9uIG9mIGFzc2ltaWxhdGlvbiBlZmZpY2llbmN5IGluIFhpcGhpc3RlciBtdWNvc3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FY29sb2d5OyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9ODkxLTkwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhlbm9sb2d5LCBUUEMgYW5kIHNpemUtZnJhY3Rpb25pbmcgcGhlbm9saWNzIHZhcmlhYmlsaXR5IGluIHRlbXBlcmF0ZSBTYXJnYXNzYWNlYWUgKFBoYWVvcGh5Y2VhZSwgRnVjYWxlcykgZnJvbSBXZXN0ZXJuIEJyaXR0YW55OiBuYXRpdmUgdmVyc3VzIGludHJvZHVjZWQgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFyLiBFbnZpcm9uLiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUvZnVuY3Rpb24gYW5hbHlzaXMgb2YgYSB0eXBlIGlpaSBwb2x5a2V0aWRlIHN5bnRoYXNlIGluIHRoZSBicm93biBhbGdhIEVjdG9jYXJwdXMgc2lsaWN1bG9zdXMgcmV2ZWFscyBhIGJpb2NoZW1pY2FsIHBhdGh3YXkgaW4gcGhsb3JvdGFubmluIG1vbm9tZXIgYmlvc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzA4OS0xMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUIyRy1GQVIsIGEgc3BlY2llcy1jZW50ZXJlZCBHTyBhbm5vdGF0aW9uIHJlcG9zaXRvcnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTE5LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNi81NjEvODMyNTcwNy9kc3cwMzkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJUaGUgYnJvd24gYWxnYSwgQ2xhZG9zaXBob24gb2thbXVyYW51cyAoT2tpbmF3YSBtb3p1a3UpLCBpcyBlY29ub21pY2FsbHkgb25lIG9mIHRoZSBtb3N0IGltcG9ydGFudCBlZGlibGUgc2Vhd2VlZHMsIGFuZCBpcyBjdWx0aXZhdGVkIGZvciBtYXJrZXQgcHJpbWFyaWx5IGluIE9rIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNjQ3NDQ5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDkmiWJSOkNStuPEfpDypYaHUxsf" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f561%2f2647449" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f561%2f2647449" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=89339" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&amp;Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/6">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="48005397">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005397">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005401">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005401">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005420">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005420">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005459">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005459">4. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005461">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005461">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005463">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005463">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005471">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005471">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/6/547/2647446">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/6/571/2647450">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A draft genome of the brown alga, <em>Cladosiphon okamuranus</em>, S-strain: a platform for future studies of ‘mozuku’ biology
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koki Nishitsuji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw039-cor1" data-open="dsw039-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed: Tel. +81 98 966 8653. Fax. +81 98 966 2890. Email: <a href="mailto:koki.nishitsuji@oist.jp" target="_blank">koki.nishitsuji@oist.jp</a> (Koki Nishitsuji); <a href="mailto:eiichi@oist.jp" target="_blank">eiichi@oist.jp</a> (Eiichi Shoguchi)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koki+Nishitsuji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishitsuji K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishitsuji K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Koki Nishitsuji</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asuka Arimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asuka+Arimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arimoto A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arimoto A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Asuka Arimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Iwai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Iwai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwai K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenji Iwai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yusuke Sudo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Okinawa Prefectural Fisheries Research and Extension Center, Itoman, Okinawa 901-0354, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yusuke+Sudo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sudo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sudo Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yusuke Sudo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanako Hisata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanako+Hisata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hisata K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hisata K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kanako Hisata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Fujie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Fujie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujie M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujie M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Manabu Fujie</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nana Arakaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nana+Arakaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arakaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arakaki N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nana Arakaki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuo Kushiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>School of Agriculture, Meiji University, Kawasaki, Kanagawa 214-8571, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuo+Kushiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kushiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kushiro T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tetsuo Kushiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Teruko Konishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Department of Bioscience and Biotechnology, Faculty of Agriculture, University of the Ryukyus, Nishihara, Okinawa 903-0213, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Teruko+Konishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Konishi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Konishi T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Teruko Konishi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chuya Shinzato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chuya+Shinzato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinzato C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shinzato C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chuya Shinzato</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriyuki Satoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriyuki+Satoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satoh N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satoh N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Noriyuki Satoh</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eiichi Shoguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa 904-0495, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw039-cor1" data-open="dsw039-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed: Tel. +81 98 966 8653. Fax. +81 98 966 2890. Email: <a href="mailto:koki.nishitsuji@oist.jp" target="_blank">koki.nishitsuji@oist.jp</a> (Koki Nishitsuji); <a href="mailto:eiichi@oist.jp" target="_blank">eiichi@oist.jp</a> (Eiichi Shoguchi)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eiichi+Shoguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shoguchi E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shoguchi E&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Eiichi Shoguchi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw039-FM2"><div class="fn"><div class="footnote-content"><sup>†</sup>The first two authors contributed equally to this work</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw039-FM1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 561–570, <a href='https://doi.org/10.1093/dnares/dsw039'>https://doi.org/10.1093/dnares/dsw039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 August 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 March 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2647449" href="/dnaresearch/article-pdf/23/6/561/8325707/dsw039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Koki Nishitsuji, Asuka Arimoto, Kenji Iwai, Yusuke Sudo, Kanako Hisata, Manabu Fujie, Nana Arakaki, Tetsuo Kushiro, Teruko Konishi, Chuya Shinzato, Noriyuki Satoh, Eiichi Shoguchi;  A draft genome of the brown alga, <em>Cladosiphon okamuranus</em>, S-strain: a platform for future studies of ‘mozuku’ biology, <em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 561–570, <a href='https://doi.org/10.1093/dnares/dsw039'>https://doi.org/10.1093/dnares/dsw039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647449?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20draft%20genome%20of%20the%20brown%20alga%2C%20%26lt%3Bem%26gt%3BCladosiphon%20okamuranus%26lt%3B%2Fem%26gt%3B%2C%20S-strain%3A%20a%20platform%20for%20future%20studies%20of%20%E2%80%98mozuku%E2%80%99%20biology&amp;publicationDate=2016-08-08&amp;volumeNum=23&amp;issueNum=6&amp;author=Nishitsuji%2C%20Koki%3B%20Arimoto%2C%20Asuka&amp;startPage=561&amp;endPage=570&amp;contentId=10.1093%2Fdnares%2Fdsw039&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f561%2f2647449" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>The brown alga, <em>Cladosiphon okamuranus</em> (Okinawa mozuku), is economically one of the most important edible seaweeds, and is cultivated for market primarily in Okinawa, Japan. <em>C. okamuranus</em> constitutes a significant source of fucoidan, which has various physiological and biological activities. To facilitate studies of seaweed biology, we decoded the draft genome of <em>C. okamuranus</em> S-strain. The genome size of <em>C. okamuranus</em> was estimated as ∼140 Mbp, smaller than genomes of two other brown algae, <em>Ectocarpus siliculosus</em> and <em>Saccharina japonica</em>. Sequencing with ∼100× coverage yielded an assembly of 541 scaffolds with N50 = 416 kbp. Together with transcriptomic data, we estimated that the <em>C. okamuranus</em> genome contains 13,640 protein-coding genes, approximately 94% of which have been confirmed with corresponding mRNAs. Comparisons with the <em>E. siliculosus</em> genome identified a set of <em>C. okamuranus</em> genes that encode enzymes involved in biosynthetic pathways for sulfated fucans and alginate biosynthesis. In addition, we identified <em>C. okamuranus</em> genes for enzymes involved in phlorotannin biosynthesis. The present decoding of the <em>Cladosiphon okamuranus</em> genome provides a platform for future studies of mozuku biology.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;brown alga&quot;">brown alga</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cladosiphon okamuranus&quot;"><em>Cladosiphon okamuranus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome decoding&quot;">genome decoding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genes for enzymes of polysaccharide synthesis&quot;">genes for enzymes of polysaccharide synthesis</a></div>                            <h2 scrollto-destination=48005397 id="48005397" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Taxonomically, brown algae (Phaeophyceae) belong to the Stramenopiles and include multicellular species.<a reveal-id="dsw039-B1" data-open="dsw039-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> They are photosynthetic organisms with chloroplasts surrounded by four membranes,<a reveal-id="dsw039-B2" data-open="dsw039-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> suggesting that they originated from a symbiotic relationship between two eukaryotes.<a reveal-id="dsw039-B3" data-open="dsw039-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Most brown algae contain the pigment, fucoxanthin, which is responsible for the distinctive greenish-brown color that gives brown algae their common name. Phaeophytes include many types of seaweed in the Northern Hemisphere, and they are important members of marine ecosystems, both because they create habitats for other organisms and because they provide food.<a reveal-id="dsw039-B4" data-open="dsw039-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a></p><p><em>Cladosiphon okamuranus</em> (Chordariales, Phaeophyceae),<a reveal-id="dsw039-B5" data-open="dsw039-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Okinawa mozuku in Japanese, is one of the important edible seaweeds. In Okinawa, <em>C. okamuranus</em> has been cultivated for more than 35 years by several fishermen’s associations, including those in Onna and Chinen Villages. This cultivation history has established several strains of mozuku that have similar morphology and texture. It was reported in the 36th annual report of the Japanese Cabinet Office that approximately 20 kilotons of mozuku (<em>C. okamuranus</em> and <em>Nemacystus decipiens</em> (‘Itomozuku’)) are produced annually, yielding approximately 4 billion Japanese yen in 2006. In addition, <em>C. okamuranus</em> and <em>N. decipiens</em> are sources of fucoidan,<a reveal-id="dsw039-B6" data-open="dsw039-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> a sulfated polysaccharide found in the cell-wall matrix of brown algae that has anti-coagulant, anti-thrombin-like, and tumor-suppressant activities.<a reveal-id="dsw039-B7" data-open="dsw039-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Brown algae also produce alginates.<a reveal-id="dsw039-B8" data-open="dsw039-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>Due to their biological significance, genomes of two species of brown algae have been decoded: <em>Ectocarpus siliculosus</em> (Order Ectocarpales<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a>) and <em>Saccharina japonica</em> (Order Laminariales<a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a>). The genome size of former is approximately 214 Mbp with 16,256 predicted protein-coding genes, while that of the latter is 545 Mbp with 18,733 predicted protein-coding genes. Several genetic features of the two brown algae have been characterized to understand their biology.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> A close phylogenetic relationship between Ectocarpales and Chordariales has been reported.<a reveal-id="dsw039-B5" data-open="dsw039-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw039-B12" data-open="dsw039-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Given its importance for fisheries, food, and possible pharmaceuticals, we decoded the draft genome of <em>Cladosiphon okamuranus</em> S-strain, (Order Chordariales).</p>                            <h2 scrollto-destination=48005401 id="48005401" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=48005402 id="48005402" class="section-title" >2.1. Strain and DNA extraction</h3>
<div class="block-child-p">The S-strain of <em>Cladosiphon okamuranus</em> (‘Shikenjo-kabu’) has been maintained as a stock culture at the Okinawa Prefectural Fisheries Research and Extension Center, Okinawa, Japan. It is cultivated at 22.5° C with a 12-h light-dark cycle in sea water containing 0.5% KW21 (Daiichi Seimo Co., Ltd). The life cycle of <em>C. okamuranus</em> includes both haploid (n) and diploid (2n) generations (<a reveal-id="dsw039-F1" data-open="dsw039-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). 2n germlings mature into sporophytes that are harvested for market. For DNA extraction, 2n germlings of <em>C. okamuranus</em> were frozen in liquid nitrogen and crushed to powder with a frozen-cell-crusher, Cryo-Press (Microtec Co., Ltd). Genomic DNA was extracted from the powder using an extraction kit, DNA-Suisui-VS (Rizo Co., Ltd). <div data-id="dsw039-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f1p.png?Expires=2147483647&Signature=BC8WYFz6~wnwK9e7ItSSqsY77rM5Fq8i5QSM6wq22LuFrMnzdCmz~wPZPaouGl0~tVLZaX5NotEFPp~oNShhTVbnNtEG07Sgz5829wMs4VAVKCeets322I0SG1ImBuwjpLWtzc7lDljJYxt7xhfgB969CrYg~GXtFcMpApjFQj8A0m2ko3pME1rMQf1mKycqH8oB~Eloxn8mIVp~M7lx4i42ygyXtrbi2YY1h9B42gDckF~nwLmThKyH4-cNKeKYOxiLDdnugEmFh8lL6koSvF-gdS~d9YiwT3y3dMqw9g1EXy3WuBDENH964oejZSvKGfjUOzgdCOdc9ZpJgJqtcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A diagram showing the life cycle of the brown alga, Cladosiphon okamuranus. The alga has n and 2n generations. Cladosiphon okamuranus is cultivated and sporophytes are harvested for market. Genomic DNA was extracted from 2n germlings, while RNA was extracted from 2n germlings and 2n sporophytes." path-from-xml="dsw039f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005403/dsw039f1p.tif" path-from-xml="dsw039f1p.tif" target="_blank">View large</a><a section="48005403" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f1p.png?Expires=2147483647&Signature=l4sW7YeaWwOOLE3-oarife3saZ~Ee6Jr1hC6yMzbAtINqASJWzd1KMOXFOov5z2Rn8CSzQIDL-JyUi2HEEQXGR86YNQO0rKf9Gdlymveu4W-OyFBAWfubC7WUCWz3wHXz7jcSnQouTN51bvMbHwMd-qGCFFKZqJtyXWHmwt6OCiinb~BghrOC9biPnlMCtSCnt5FfO-NEHt9yYh4aZ8enxfw20o7jiX4uHa8GV-jAsSUwaj1AbWOx2chFnIp1pqbObPc4QrWaljYH6zsX0ZKYsMIziqnyCS2SEg8wSwCwghpqhfsvVEJ-l-dD0HXnkO5MX2AGiadVySxSQqokXfO5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005403&ar=2647449&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw039f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A diagram showing the life cycle of the brown alga, <em>Cladosiphon okamuranus</em>. The alga has n and 2n generations. <em>Cladosiphon okamuranus</em> is cultivated and sporophytes are harvested for market. Genomic DNA was extracted from 2n germlings, while RNA was extracted from 2n germlings and 2n sporophytes.</p></div></div><div content-id="dsw039-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f1p.png?Expires=2147483647&Signature=BC8WYFz6~wnwK9e7ItSSqsY77rM5Fq8i5QSM6wq22LuFrMnzdCmz~wPZPaouGl0~tVLZaX5NotEFPp~oNShhTVbnNtEG07Sgz5829wMs4VAVKCeets322I0SG1ImBuwjpLWtzc7lDljJYxt7xhfgB969CrYg~GXtFcMpApjFQj8A0m2ko3pME1rMQf1mKycqH8oB~Eloxn8mIVp~M7lx4i42ygyXtrbi2YY1h9B42gDckF~nwLmThKyH4-cNKeKYOxiLDdnugEmFh8lL6koSvF-gdS~d9YiwT3y3dMqw9g1EXy3WuBDENH964oejZSvKGfjUOzgdCOdc9ZpJgJqtcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A diagram showing the life cycle of the brown alga, Cladosiphon okamuranus. The alga has n and 2n generations. Cladosiphon okamuranus is cultivated and sporophytes are harvested for market. Genomic DNA was extracted from 2n germlings, while RNA was extracted from 2n germlings and 2n sporophytes." path-from-xml="dsw039f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005403/dsw039f1p.tif" path-from-xml="dsw039f1p.tif" target="_blank">View large</a><a section="48005403" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f1p.png?Expires=2147483647&Signature=l4sW7YeaWwOOLE3-oarife3saZ~Ee6Jr1hC6yMzbAtINqASJWzd1KMOXFOov5z2Rn8CSzQIDL-JyUi2HEEQXGR86YNQO0rKf9Gdlymveu4W-OyFBAWfubC7WUCWz3wHXz7jcSnQouTN51bvMbHwMd-qGCFFKZqJtyXWHmwt6OCiinb~BghrOC9biPnlMCtSCnt5FfO-NEHt9yYh4aZ8enxfw20o7jiX4uHa8GV-jAsSUwaj1AbWOx2chFnIp1pqbObPc4QrWaljYH6zsX0ZKYsMIziqnyCS2SEg8wSwCwghpqhfsvVEJ-l-dD0HXnkO5MX2AGiadVySxSQqokXfO5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005403&ar=2647449&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw039f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A diagram showing the life cycle of the brown alga, <em>Cladosiphon okamuranus</em>. The alga has n and 2n generations. <em>Cladosiphon okamuranus</em> is cultivated and sporophytes are harvested for market. Genomic DNA was extracted from 2n germlings, while RNA was extracted from 2n germlings and 2n sporophytes.</p></div></div></div>                            <h3 scrollto-destination=48005404 id="48005404" class="section-title" >2.2. Genome sequencing and assembly</h3>
<p>The Illumina platform (Miseq and Hiseq 2500) was used for sequencing.<a reveal-id="dsw039-B13" data-open="dsw039-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Libraries were prepared according to slight modifications of protocols provided by the manufacturer. Fragmented genomic DNA was further purified using Blue Pippin (Sage Science). A paired-end library consisting of clones ∼720 bp was prepared for the Miseq using a TruSeq DNA PCR-Free LT Sample Prep Kit (Illumina), and 3-kb and 8-kb mate-pair libraries were prepared for the Hiseq 2500 using a Nextera Mate Pair Sample Prep Kit (Illumina), respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Longer reads were obtained by using more reagent kits for the Hiseq. K-mer counting and estimation of genome size were performed using JELLYFISH 2.2.0 software.<a reveal-id="dsw039-B14" data-open="dsw039-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw039-B15" data-open="dsw039-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p>Adapter sequences were trimmed from all reads using Trimmomatic-0.30.<a reveal-id="dsw039-B16" data-open="dsw039-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Paired-end reads of high quality (quality value ≥ 20) were assembled <em>de novo</em> using Newbler 2.9 (GS Assembler) to create contigs. Then subsequent scaffolding of the Newbler output was performed using SSPACE 3.0,<a reveal-id="dsw039-B17" data-open="dsw039-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> based on Illumina mate-pair information. Gaps inside scaffolds were closed using GapCloser 1.12.<a reveal-id="dsw039-B18" data-open="dsw039-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Diploid sequences of gap-closed scaffolds were merged with Haplomerger-2-20151124.<a reveal-id="dsw039-B19" data-open="dsw039-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> CEGMA 2.5 software<a reveal-id="dsw039-B20" data-open="dsw039-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> was used to evaluate genome assembly. The mitochondrial genome was generated with the IDBA_UD 1.1.1 assembler.<a reveal-id="dsw039-B21" data-open="dsw039-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p>                            <h3 scrollto-destination=48005407 id="48005407" class="section-title" >2.3. Transcriptome analyses</h3>
<p>RNA was isolated from sporophytes (2–5 cm) and 2n germlings (<a reveal-id="dsw039-F1" data-open="dsw039-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Total RNA was extracted following the instructions of the manufacturer using DNase and an RNeasy Plant mini kit (QIAGEN). Transcriptome libraries were prepared using a TruSeq Stranded mRNA Library Prep kit (Illumina). RNAs were sequenced as per the manufacturer’s instructions for the Illumina Hiseq 2500. Only sequences of high quality (quality value ≥ 20) were assembled, using Velvet 1.2.10<a reveal-id="dsw039-B22" data-open="dsw039-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and Oases 0.2.08.<a reveal-id="dsw039-B23" data-open="dsw039-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p>                            <h3 scrollto-destination=48005409 id="48005409" class="section-title" >2.4. Gene model prediction</h3>
<p>A set of gene model predictions (<em>C. okamuranus</em> Gene Model ver. 1) was generated using AUGUSTUS 3.2.1.<a reveal-id="dsw039-B24" data-open="dsw039-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> AUGUSTUS was trained on the 9120 transcriptome contigs recommended by PASA 2.0.2<a reveal-id="dsw039-B25" data-open="dsw039-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> for this purpose. Gene models were produced by running AUGUSTUS on a repeat-masked genome, produced with RepeatMasker 4.0.6 (<a class="link link-uri" href="http://ftp.genome.washington.edu/RM/RepeatMasker.html" target="_blank">http://ftp.genome.washington.edu/RM/RepeatMasker.html</a>) and refined with PASA.</p>                            <h3 scrollto-destination=48005411 id="48005411" class="section-title" >2.5. Transposable elements and repetitive sequences</h3>
<p>Repetitive sequences were detected as described previously.<a reveal-id="dsw039-B26" data-open="dsw039-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Tandem repeats were detected using Tandem Repeat Finder (version 4.04),<a reveal-id="dsw039-B27" data-open="dsw039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and then classified using RepeatModeler-1.1.8 (<a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler.html" target="_blank">http://www.repeatmasker.org/RepeatModeler.html</a>). A <em>de novo</em> repeats library was generated with RepeatScout (version 1.0.5).<a reveal-id="dsw039-B28" data-open="dsw039-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Transposons and SINE in the scaffold were identified with CENSOR (version 4.2)<a reveal-id="dsw039-B29" data-open="dsw039-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> using Blast searches against the Repbase TE library (version 16.05).<a reveal-id="dsw039-B30" data-open="dsw039-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h3 scrollto-destination=48005413 id="48005413" class="section-title" >2.6. Gene annotation and identification</h3>
<p>Three approaches, individually or in combination, were used to annotate protein-coding genes in the <em>C. okamuranus</em> genome. The primary approach for identification of putative <em>C. okamuranus</em> orthologous genes was reciprocal BLAST analysis. This was carried out using mutual best hits of genes of <em>E. siliculosus</em><a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> against the <em>C. okamuranus</em> gene models (BLASTP) or the assembly (TBLASTN). A second approach used for genes encoding proteins with one or more specific protein domains, was to screen the models against the Pfam database (Pfam-A.hmm, release 24.0; <a class="link link-uri" href="http://pfam.sanger.ac.uk" target="_blank">http://pfam.sanger.ac.uk</a>),<a reveal-id="dsw039-B31" data-open="dsw039-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> which contains approximately 11,000 conserved domains using HMMER (hmmer3).<a reveal-id="dsw039-B32" data-open="dsw039-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p><p>In the case of complex multigene families, a third annotation method was employed; sets of related sequences were subjected to phylogenetic analyses in order to more precisely determine orthologous relationships between proteins of <em>C. okamuranus</em> and <em>E. siliculosus.</em> For this purpose, amino acid sequences were aligned using MAFFT<a reveal-id="dsw039-B33" data-open="dsw039-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with default options. Gaps and ambiguous areas were excluded using SeaView version 4.5.3<a reveal-id="dsw039-B34" data-open="dsw039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> manually. Based on alignment datasets, phylogenetic trees were constructed by the maximum likelihood method. A maximum likelihood phylogenetic tree was constructed with MEGA 5.2<a reveal-id="dsw039-B35" data-open="dsw039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> using the best model with 1,000 bootstrap duplications.</p>                            <h3 scrollto-destination=48005416 id="48005416" class="section-title" >2.7. Genome browser</h3>
<p>A genome browser has been established for the assembled genome sequences using the JavaScript-based Genome Browser (JBrowse) 1.11.6.<a reveal-id="dsw039-B36" data-open="dsw039-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The assembled sequence and gene models are accessible at <a class="link link-uri" href="http://marinegenomics.oist.jp/gallery/" target="_blank">http://marinegenomics.oist.jp/gallery/</a>.</p>                            <h3 scrollto-destination=48005418 id="48005418" class="section-title" >2.8 Gene expression analysis</h3>
<p>RNA-seq reads were aligned to the genome using TopHat-2.0.9,<a reveal-id="dsw039-B37" data-open="dsw039-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> while FPKM values were calculated from the aligned results using Cufflinks v2.0.0.<a reveal-id="dsw039-B38" data-open="dsw039-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Gene expression levels were visualized with R 3.2.2 (<a class="link link-uri" href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a>) using the pheatmap 1.0.8 package.<a reveal-id="dsw039-B39" data-open="dsw039-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsw039-B40" data-open="dsw039-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h2 scrollto-destination=48005420 id="48005420" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=48005421 id="48005421" class="section-title" >3.1. Genome sequencing and assembly</h3>
<p>The Illumina Miseq (average library size, 720 bp) generated a total of 9.75 gigabase pairs (Gbp) of paired-end sequence data (read length, 309 ×2 bases) and the Hiseq2500 generated 5.79 Gbp of mate-pair sequence data (read length, 283 ×2 bases); 5.02 Gbp for the 3-kb library (physical coverage, 177×) and 0.78 Gbp for the 8-kb library (physical coverage, 73×), respectively. A total of 15.6 Gbp of sequence data was obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). As mentioned below, the genome size of <em>Cladosiphon okamuranus</em> was estimated as approximately 140 Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>). Total reads of 15.6 Gbp correspond to approximately 100x sequencing coverage of the genome. <div content-id="dsw039-T1" class="table-modal"><div class="table-wrap"><div id="dsw039-T1" data-id="dsw039-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of draft genome assemblies of three species of brown algae, <em>Cladosiphon okamuranus</em> (Order Chordariales), <em>Ectocarpus siliculosus</em> (Order Ectocarpales), and <em>Saccharina japonica</em> (Order Laminariales)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th> </th><th><em>Cladosiphon okamuranus</em><a reveal-id="dsw039-TF1" data-open="dsw039-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Ectocarpus siliculosus</em><a reveal-id="dsw039-TF2" data-open="dsw039-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><em>Saccharina japonica</em><a reveal-id="dsw039-TF3" data-open="dsw039-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>Genome size (Mb) </td><td char=".">140 </td><td char=".">214 </td><td char=".">545 </td></tr><tr><td>Total assembled length (Mb) </td><td char=".">129.9 </td><td char=".">195.8 </td><td char=".">537 </td></tr><tr><td>Number of scaffolds </td><td char=".">541 </td><td char=".">1,561 </td><td char=".">13,327 </td></tr><tr><td>N50 scaffold size (kb) </td><td char=".">416 </td><td char=".">504 </td><td char=".">252 </td></tr><tr><td>Number of contigs </td><td char=".">31,858 </td><td char=".">14,043 </td><td char=".">29,670 </td></tr><tr><td>N50 Contig size (bp) </td><td char=".">21,705 </td><td char=".">32,862 </td><td char=".">58,867 </td></tr><tr><td>Number of genes </td><td char=".">13,640 </td><td char=".">16,256 </td><td char=".">18,733 </td></tr><tr><td>GC Contents (%) </td><td char=".">54 </td><td char=".">54 </td><td char=".">50 </td></tr><tr><td>Repeated sequences (%) </td><td char=".">4.1 </td><td char=".">22.7 </td><td char=".">39 </td></tr><tr><td>Average gene length (bp) </td><td char=".">7,696 </td><td char=".">6,859 </td><td char=".">9,587 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">2,004 </td><td char=".">1,563 </td><td char=".">— </td></tr><tr><td>Average number of introns per gene </td><td char=".">8.26 </td><td char=".">6.98 </td><td char=".">— </td></tr><tr><td>Average intron length (bp) </td><td char=".">522 </td><td char=".">703.8 </td><td char=".">1,057 </td></tr><tr><td>Cegma Partial (%) </td><td char=".">91.9 </td><td char=".">91.5 </td><td char=".">79.0 </td></tr><tr><td>Cegma Completeness (%) </td><td char=".">84.3 </td><td char=".">77.4 </td><td char=".">45.6 </td></tr></tbody></table></div><div content-id="dsw039-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th> </th><th><em>Cladosiphon okamuranus</em><a reveal-id="dsw039-TF1" data-open="dsw039-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Ectocarpus siliculosus</em><a reveal-id="dsw039-TF2" data-open="dsw039-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><em>Saccharina japonica</em><a reveal-id="dsw039-TF3" data-open="dsw039-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>Genome size (Mb) </td><td char=".">140 </td><td char=".">214 </td><td char=".">545 </td></tr><tr><td>Total assembled length (Mb) </td><td char=".">129.9 </td><td char=".">195.8 </td><td char=".">537 </td></tr><tr><td>Number of scaffolds </td><td char=".">541 </td><td char=".">1,561 </td><td char=".">13,327 </td></tr><tr><td>N50 scaffold size (kb) </td><td char=".">416 </td><td char=".">504 </td><td char=".">252 </td></tr><tr><td>Number of contigs </td><td char=".">31,858 </td><td char=".">14,043 </td><td char=".">29,670 </td></tr><tr><td>N50 Contig size (bp) </td><td char=".">21,705 </td><td char=".">32,862 </td><td char=".">58,867 </td></tr><tr><td>Number of genes </td><td char=".">13,640 </td><td char=".">16,256 </td><td char=".">18,733 </td></tr><tr><td>GC Contents (%) </td><td char=".">54 </td><td char=".">54 </td><td char=".">50 </td></tr><tr><td>Repeated sequences (%) </td><td char=".">4.1 </td><td char=".">22.7 </td><td char=".">39 </td></tr><tr><td>Average gene length (bp) </td><td char=".">7,696 </td><td char=".">6,859 </td><td char=".">9,587 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">2,004 </td><td char=".">1,563 </td><td char=".">— </td></tr><tr><td>Average number of introns per gene </td><td char=".">8.26 </td><td char=".">6.98 </td><td char=".">— </td></tr><tr><td>Average intron length (bp) </td><td char=".">522 </td><td char=".">703.8 </td><td char=".">1,057 </td></tr><tr><td>Cegma Partial (%) </td><td char=".">91.9 </td><td char=".">91.5 </td><td char=".">79.0 </td></tr><tr><td>Cegma Completeness (%) </td><td char=".">84.3 </td><td char=".">77.4 </td><td char=".">45.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw039-TF1"><div class="fn"><p><sup>a</sup>The present study.</p></div></div><div content-id="dsw039-TF2"><div class="fn"><p><sup>b</sup>Cock et al.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p></div></div><div content-id="dsw039-TF3"><div class="fn"><p><sup>c</sup>Ye et al.<a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p></div></div></div></div></div><div class="table-wrap"><div id="dsw039-T1" data-id="dsw039-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Comparison of draft genome assemblies of three species of brown algae, <em>Cladosiphon okamuranus</em> (Order Chordariales), <em>Ectocarpus siliculosus</em> (Order Ectocarpales), and <em>Saccharina japonica</em> (Order Laminariales)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th> </th><th><em>Cladosiphon okamuranus</em><a reveal-id="dsw039-TF1" data-open="dsw039-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Ectocarpus siliculosus</em><a reveal-id="dsw039-TF2" data-open="dsw039-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><em>Saccharina japonica</em><a reveal-id="dsw039-TF3" data-open="dsw039-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>Genome size (Mb) </td><td char=".">140 </td><td char=".">214 </td><td char=".">545 </td></tr><tr><td>Total assembled length (Mb) </td><td char=".">129.9 </td><td char=".">195.8 </td><td char=".">537 </td></tr><tr><td>Number of scaffolds </td><td char=".">541 </td><td char=".">1,561 </td><td char=".">13,327 </td></tr><tr><td>N50 scaffold size (kb) </td><td char=".">416 </td><td char=".">504 </td><td char=".">252 </td></tr><tr><td>Number of contigs </td><td char=".">31,858 </td><td char=".">14,043 </td><td char=".">29,670 </td></tr><tr><td>N50 Contig size (bp) </td><td char=".">21,705 </td><td char=".">32,862 </td><td char=".">58,867 </td></tr><tr><td>Number of genes </td><td char=".">13,640 </td><td char=".">16,256 </td><td char=".">18,733 </td></tr><tr><td>GC Contents (%) </td><td char=".">54 </td><td char=".">54 </td><td char=".">50 </td></tr><tr><td>Repeated sequences (%) </td><td char=".">4.1 </td><td char=".">22.7 </td><td char=".">39 </td></tr><tr><td>Average gene length (bp) </td><td char=".">7,696 </td><td char=".">6,859 </td><td char=".">9,587 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">2,004 </td><td char=".">1,563 </td><td char=".">— </td></tr><tr><td>Average number of introns per gene </td><td char=".">8.26 </td><td char=".">6.98 </td><td char=".">— </td></tr><tr><td>Average intron length (bp) </td><td char=".">522 </td><td char=".">703.8 </td><td char=".">1,057 </td></tr><tr><td>Cegma Partial (%) </td><td char=".">91.9 </td><td char=".">91.5 </td><td char=".">79.0 </td></tr><tr><td>Cegma Completeness (%) </td><td char=".">84.3 </td><td char=".">77.4 </td><td char=".">45.6 </td></tr></tbody></table></div><div content-id="dsw039-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th> </th><th><em>Cladosiphon okamuranus</em><a reveal-id="dsw039-TF1" data-open="dsw039-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><em>Ectocarpus siliculosus</em><a reveal-id="dsw039-TF2" data-open="dsw039-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><em>Saccharina japonica</em><a reveal-id="dsw039-TF3" data-open="dsw039-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td>Genome size (Mb) </td><td char=".">140 </td><td char=".">214 </td><td char=".">545 </td></tr><tr><td>Total assembled length (Mb) </td><td char=".">129.9 </td><td char=".">195.8 </td><td char=".">537 </td></tr><tr><td>Number of scaffolds </td><td char=".">541 </td><td char=".">1,561 </td><td char=".">13,327 </td></tr><tr><td>N50 scaffold size (kb) </td><td char=".">416 </td><td char=".">504 </td><td char=".">252 </td></tr><tr><td>Number of contigs </td><td char=".">31,858 </td><td char=".">14,043 </td><td char=".">29,670 </td></tr><tr><td>N50 Contig size (bp) </td><td char=".">21,705 </td><td char=".">32,862 </td><td char=".">58,867 </td></tr><tr><td>Number of genes </td><td char=".">13,640 </td><td char=".">16,256 </td><td char=".">18,733 </td></tr><tr><td>GC Contents (%) </td><td char=".">54 </td><td char=".">54 </td><td char=".">50 </td></tr><tr><td>Repeated sequences (%) </td><td char=".">4.1 </td><td char=".">22.7 </td><td char=".">39 </td></tr><tr><td>Average gene length (bp) </td><td char=".">7,696 </td><td char=".">6,859 </td><td char=".">9,587 </td></tr><tr><td>Average coding sequence length (bp) </td><td char=".">2,004 </td><td char=".">1,563 </td><td char=".">— </td></tr><tr><td>Average number of introns per gene </td><td char=".">8.26 </td><td char=".">6.98 </td><td char=".">— </td></tr><tr><td>Average intron length (bp) </td><td char=".">522 </td><td char=".">703.8 </td><td char=".">1,057 </td></tr><tr><td>Cegma Partial (%) </td><td char=".">91.9 </td><td char=".">91.5 </td><td char=".">79.0 </td></tr><tr><td>Cegma Completeness (%) </td><td char=".">84.3 </td><td char=".">77.4 </td><td char=".">45.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw039-TF1"><div class="fn"><p><sup>a</sup>The present study.</p></div></div><div content-id="dsw039-TF2"><div class="fn"><p><sup>b</sup>Cock et al.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p></div></div><div content-id="dsw039-TF3"><div class="fn"><p><sup>c</sup>Ye et al.<a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p></div></div></div></div></p><p>Illumina whole-genome, shotgun paired-end reads were assembled <em>de novo</em> with Newbler. The assembler generated 31,858 contigs with an N50 size of 21.7 kb (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The longest contig was 943.8 kb and approximately 40% of the sequences were covered by contigs &gt;2 kb. Subsequent scaffolding of the 31,858 contigs of Newbler output was performed with SSPACE using Illumina mate-pair sequence information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Gaps inside scaffolds were closed with GapCloser. As a result, the final assembly contained 732 scaffolds with an N50 size of 536 kb. The total scaffold length reached 170 Mb.</p><p>Bacterial contaminant scaffolds were identified in the genome of an <em>Ectocarpus siliculosus</em> strain.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> To detect scaffolds of likely bacterial origin, 732 scaffolds were classified into one large group (129.9 Mb containing 541 scaffolds) and seven smaller groups (3.6–8.0 Mb) using MaxBin software,<a reveal-id="dsw039-B41" data-open="dsw039-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> which bins assembled microbial scaffolds, based on tetranucleotide frequencies, scaffold coverage levels and marker sequences. Using transcriptome mapping of <em>C. okamuranus</em> (See later) and similarities to <em>Ectocarpus</em> genes, it was confirmed that the large group contained <em>Cladosiphon okamuranus</em> scaffolds. All scaffolds of smaller groups had no mapped transcriptomes that showed exon–intron structures. For 541 scaffolds of <em>C. okamuranus</em>, the number of contigs was 2,774 and the N50 of contigs was 88 kb. The longest scaffold was 2.8 Mbp, and the scaffold N50 was 416 kb (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Approximately 87% of scaffold sequences were covered with contigs of &gt;20 kb. Chloroplast and mitochondrial genome sequences were included in scaffold IDs, Cok_S_s60 (599,436 bp) and Cok_S_s1074 (36,249 bp), respectively (data not shown).</p><p>CEGMA analysis was carried out to evaluate the accuracy of the assembled genome (129.9 Mb). CEGMA reported 91.9% sequences (CEGMA partial) with an average degree of completeness for all sequences of 84.3%. On the other hand, CEGMA completeness values for genome sequences of <em>E. siliculosus</em> and <em>S. japonica</em> are 77.4% and 45.6% (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>), respectively, suggesting the assembled genome of <em>C. okamuranus</em> is the highest quality brown algal genome to date.</p>                            <h3 scrollto-destination=48005426 id="48005426" class="section-title" >3.2. Genome size</h3>
<p>The genome size of <em>C. okamuranus</em> was estimated by K-mer analysis (K-mer = 25). The peak appeared around 47 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>). The estimated genome size was ∼140 Mb. The <em>C. okamuranus</em> genome is the smallest phaeophyte genome sequenced to date (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The genome size of <em>E. siliculosus</em> was estimated to be 214 Mbp while that of <em>S. japonica</em> was 545 Mb (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>).<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p>                            <h3 scrollto-destination=48005428 id="48005428" class="section-title" >3.3. GC content</h3>
<p>GC content of the <em>C. okamuranus</em> genome was estimated to be ∼54% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span> and <a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). GC contents for <em>E. siliculosus</em> and <em>S. japonica</em> were 54% and 50%, respectively; thus, the three brown algae are similar in this regard (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                            <h3 scrollto-destination=48005430 id="48005430" class="section-title" >3.4. RNAseq, clustering, and mapping</h3>
<p>Transcriptomic analysis is essential to determine which genes are expressed by a given organism. RNAs extracted from germlings and sporophytes were sequenced on an Illumina Hiseq 2500 (read length, 329 ×2 bases from both samples) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). About 44.8 giga nucleotide reads were obtained.</p><p>An assembly of transcriptome sequences yielded 59,590 contigs (92.8 mega nucleotides (nts)) with an N50 size of 2,231 nts. Of those, 58,095 (97.4%) had a blat alignment (with default settings) to the assembled genome sequences (129.9 Mb). 37,299 of the 59,590 contigs had predicted ORFs from start to stop codons of at least 450 nts. Of these putatively full-length RNA-seq contigs, 36,648 (98.6%) had a blat alignment to assembled scaffolds (129.9 Mb). These data were used to produce gene models for annotation.</p>                            <h3 scrollto-destination=48005433 id="48005433" class="section-title" >3.5. Gene modeling</h3>
<p>Assembled RNAseqs of <em>C. okamuranus</em>, algae EST datasets available on NCBI and putative protein coding loci found with PASA, were incorporated as AUGUSTUS ‘hints.’ The number of gene models was 13,640 (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This is somewhat smaller than the 16,256 predicted genes of <em>E. siliculosus</em> and the 18,733 of <em>S. japonica</em>, respectively, although this value is comparable. The 12,813 genes of <em>C. okamuranus</em> were supported by mRNA sequences.</p><p>The average (arithmetic mean) length of <em>C. okamuranus</em> genes that included both exons and introns, was 7,696 bp, and that of exons translated to form proteins (coding sequence) was 2,004 bp. The average lengths of <em>C. okamuranus</em> 5′ and 3′UTRs were 911 bp and 459 bp, respectively. Cock et al.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> noted that <em>E. siliculosus</em> genes are intron-rich, with an average number of introns per gene of 6.98 and an average intron length of 704 bp (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This was the case in the <em>C. okamuranus</em> genome; the average number of introns per gene was 8.26 and the average intron length was 522 bp (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                            <h3 scrollto-destination=48005436 id="48005436" class="section-title" >3.6. Transposable elements and other genomic components</h3>
<p>We examined the proportion of transposable elements and repetitive elements in the assembled mozuku genome. DNA transposons and retrotransposons apparently account for 0.539% and 2.491% of the <em>C. okamuranus</em> genome, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). DNA transposons included hAT (0.104% of the assembled sequences), EnSpm (0.090%), Helitron (0.084%), PIF-Harbinger (0.056%) and others, while retrotransposons included LTR retrotransposons such as Gypsy (1.096%), Copia (0.752%) and Bel_Pao (0.115%), and non-LTR retrotransposons such as CR1 (0.040%) and I (0.018%). The percentages of LINE (long interspersed nuclear elements) such as L1 (0.123%), Tx1, and Jockey, and SINE (short interspersed nuclear elements) were smaller in the <em>C. okamuranus</em> genome, compared to other brown algal genomes.</p><p>Including unclassified repeats, 4.1% of the <em>C. okamuranus</em> genome was composed of repetitive sequences (<a reveal-id="dsw039-T1" data-open="dsw039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This ratio is lower than in the two other brown algae, 22.7% for <em>E. siliculosus</em> and 39% for <em>S. japonica</em>, respectively. It may be that the high frequency of repeated sequences in brown algal genomes is not universal among the Phaeophyceae.</p>                            <h3 scrollto-destination=48005439 id="48005439" class="section-title" >3.7. Identification of <em>Cladosiphon okamuranus</em> genes</h3>
<p>In order to examine the utility of the assembled genome as a platform for future studies in mozuku biology, we searched for genes in the assembled <em>C. okamuranus</em> genome. By comparing genes found in the <em>E. siliculosus</em> genome,<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> we found the corresponding mozuku genes associated with transcription factors, signaling molecules, and enzymes involved in polysaccharide biosynthesis and phlorotannin biosynthesis.</p>                            <h3 scrollto-destination=48005441 id="48005441" class="section-title" data-legacy-id=SEC3.7.1>3.7.1 Transcription factors</h3>
<p>Transcription factors play pivotal roles in many biological processes. We examined transcription factors (TFs) by searching conserved protein domains using hmmer3 and the Pfam database (e-value cutoff &lt; e <sup>−</sup><sup>5</sup>). The number of domains that have been identified in transcription factors was determined and compared with those in the <em>E. siliculosus</em> genome. These domains include HSF, Myb, bZIP, Zinc Finger, bHLH, CCAAT-binding, homeobox, AP2-EREBP, Nin-like, TAF, E2F-DP, CBF/NF-Y/archaeal and Sigma-70 r2/r3/r4 (<a reveal-id="dsw039-T2" data-open="dsw039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). <div content-id="dsw039-T2" class="table-modal"><div class="table-wrap"><div id="dsw039-T2" data-id="dsw039-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Number of transcription factors (TFs) in the brown algae, <em>Cladosiphon okamuranus</em> and <em>Ectocarpus siliculosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th> </th><th colspan="4">Species </th></tr><tr><th>Family </th><th colspan="2"><em>Cladosiphon okamuranus</em> </th><th colspan="2"><em>Ectocarpus siliculosus</em> </th></tr></thead><tbody><tr><td>Myb </td><td char=".">58 </td><td char=".">(27.1%)<a reveal-id="dsw039-TF4" data-open="dsw039-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">59 </td><td char=".">(22.7%) </td></tr><tr><td>CBF/NF-Y/archaeal </td><td char=".">35 </td><td char=".">(16.3%) </td><td char=".">42 </td><td char=".">(16.2%) </td></tr><tr><td>Zn_finger, C2H2-type </td><td char=".">23 </td><td char=".">(10.7%) </td><td char=".">30 </td><td char=".">(11.5%) </td></tr><tr><td>Zn_finger, CCCH-type </td><td char=".">22 </td><td char=".">(10.3%) </td><td char=".">24 </td><td char=".">(9.2%) </td></tr><tr><td>bZIP </td><td char=".">17 </td><td char=".">(7.9%) </td><td char=".">23 </td><td char=".">(8.8%) </td></tr><tr><td>HSF </td><td char=".">13 </td><td char=".">(6.1%) </td><td char=".">22 </td><td char=".">(8.5%) </td></tr><tr><td>TAF </td><td char=".">10 </td><td char=".">(4.7%) </td><td char=".">18 </td><td char=".">(6.9%) </td></tr><tr><td>Sigma-70 r2/r3/r4 </td><td char=".">9 </td><td char=".">(4.2%) </td><td char=".">10 </td><td char=".">(3.8%) </td></tr><tr><td>Zn_finger, TAZ-type </td><td char=".">7 </td><td char=".">(3.3%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>Nin-like </td><td char=".">5 </td><td char=".">(2.3%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>bHLH </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Homeobox </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>AP2-EREBP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>E2F-DP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>CXC/tesmin </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>Zn_finger, GATA-type </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>Fungal TRF </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Total </td><td char=".">214 </td><td char="."> </td><td char=".">260 </td><td char="."> </td></tr></tbody></table></div><div content-id="dsw039-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th> </th><th colspan="4">Species </th></tr><tr><th>Family </th><th colspan="2"><em>Cladosiphon okamuranus</em> </th><th colspan="2"><em>Ectocarpus siliculosus</em> </th></tr></thead><tbody><tr><td>Myb </td><td char=".">58 </td><td char=".">(27.1%)<a reveal-id="dsw039-TF4" data-open="dsw039-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">59 </td><td char=".">(22.7%) </td></tr><tr><td>CBF/NF-Y/archaeal </td><td char=".">35 </td><td char=".">(16.3%) </td><td char=".">42 </td><td char=".">(16.2%) </td></tr><tr><td>Zn_finger, C2H2-type </td><td char=".">23 </td><td char=".">(10.7%) </td><td char=".">30 </td><td char=".">(11.5%) </td></tr><tr><td>Zn_finger, CCCH-type </td><td char=".">22 </td><td char=".">(10.3%) </td><td char=".">24 </td><td char=".">(9.2%) </td></tr><tr><td>bZIP </td><td char=".">17 </td><td char=".">(7.9%) </td><td char=".">23 </td><td char=".">(8.8%) </td></tr><tr><td>HSF </td><td char=".">13 </td><td char=".">(6.1%) </td><td char=".">22 </td><td char=".">(8.5%) </td></tr><tr><td>TAF </td><td char=".">10 </td><td char=".">(4.7%) </td><td char=".">18 </td><td char=".">(6.9%) </td></tr><tr><td>Sigma-70 r2/r3/r4 </td><td char=".">9 </td><td char=".">(4.2%) </td><td char=".">10 </td><td char=".">(3.8%) </td></tr><tr><td>Zn_finger, TAZ-type </td><td char=".">7 </td><td char=".">(3.3%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>Nin-like </td><td char=".">5 </td><td char=".">(2.3%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>bHLH </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Homeobox </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>AP2-EREBP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>E2F-DP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>CXC/tesmin </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>Zn_finger, GATA-type </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>Fungal TRF </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Total </td><td char=".">214 </td><td char="."> </td><td char=".">260 </td><td char="."> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw039-TF4"><div class="fn"><p><sup>a</sup>Percentage of the family within the brown algal genome.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw039-T2" data-id="dsw039-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Number of transcription factors (TFs) in the brown algae, <em>Cladosiphon okamuranus</em> and <em>Ectocarpus siliculosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th> </th><th colspan="4">Species </th></tr><tr><th>Family </th><th colspan="2"><em>Cladosiphon okamuranus</em> </th><th colspan="2"><em>Ectocarpus siliculosus</em> </th></tr></thead><tbody><tr><td>Myb </td><td char=".">58 </td><td char=".">(27.1%)<a reveal-id="dsw039-TF4" data-open="dsw039-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">59 </td><td char=".">(22.7%) </td></tr><tr><td>CBF/NF-Y/archaeal </td><td char=".">35 </td><td char=".">(16.3%) </td><td char=".">42 </td><td char=".">(16.2%) </td></tr><tr><td>Zn_finger, C2H2-type </td><td char=".">23 </td><td char=".">(10.7%) </td><td char=".">30 </td><td char=".">(11.5%) </td></tr><tr><td>Zn_finger, CCCH-type </td><td char=".">22 </td><td char=".">(10.3%) </td><td char=".">24 </td><td char=".">(9.2%) </td></tr><tr><td>bZIP </td><td char=".">17 </td><td char=".">(7.9%) </td><td char=".">23 </td><td char=".">(8.8%) </td></tr><tr><td>HSF </td><td char=".">13 </td><td char=".">(6.1%) </td><td char=".">22 </td><td char=".">(8.5%) </td></tr><tr><td>TAF </td><td char=".">10 </td><td char=".">(4.7%) </td><td char=".">18 </td><td char=".">(6.9%) </td></tr><tr><td>Sigma-70 r2/r3/r4 </td><td char=".">9 </td><td char=".">(4.2%) </td><td char=".">10 </td><td char=".">(3.8%) </td></tr><tr><td>Zn_finger, TAZ-type </td><td char=".">7 </td><td char=".">(3.3%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>Nin-like </td><td char=".">5 </td><td char=".">(2.3%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>bHLH </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Homeobox </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>AP2-EREBP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>E2F-DP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>CXC/tesmin </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>Zn_finger, GATA-type </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>Fungal TRF </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Total </td><td char=".">214 </td><td char="."> </td><td char=".">260 </td><td char="."> </td></tr></tbody></table></div><div content-id="dsw039-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th> </th><th colspan="4">Species </th></tr><tr><th>Family </th><th colspan="2"><em>Cladosiphon okamuranus</em> </th><th colspan="2"><em>Ectocarpus siliculosus</em> </th></tr></thead><tbody><tr><td>Myb </td><td char=".">58 </td><td char=".">(27.1%)<a reveal-id="dsw039-TF4" data-open="dsw039-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">59 </td><td char=".">(22.7%) </td></tr><tr><td>CBF/NF-Y/archaeal </td><td char=".">35 </td><td char=".">(16.3%) </td><td char=".">42 </td><td char=".">(16.2%) </td></tr><tr><td>Zn_finger, C2H2-type </td><td char=".">23 </td><td char=".">(10.7%) </td><td char=".">30 </td><td char=".">(11.5%) </td></tr><tr><td>Zn_finger, CCCH-type </td><td char=".">22 </td><td char=".">(10.3%) </td><td char=".">24 </td><td char=".">(9.2%) </td></tr><tr><td>bZIP </td><td char=".">17 </td><td char=".">(7.9%) </td><td char=".">23 </td><td char=".">(8.8%) </td></tr><tr><td>HSF </td><td char=".">13 </td><td char=".">(6.1%) </td><td char=".">22 </td><td char=".">(8.5%) </td></tr><tr><td>TAF </td><td char=".">10 </td><td char=".">(4.7%) </td><td char=".">18 </td><td char=".">(6.9%) </td></tr><tr><td>Sigma-70 r2/r3/r4 </td><td char=".">9 </td><td char=".">(4.2%) </td><td char=".">10 </td><td char=".">(3.8%) </td></tr><tr><td>Zn_finger, TAZ-type </td><td char=".">7 </td><td char=".">(3.3%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>Nin-like </td><td char=".">5 </td><td char=".">(2.3%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>bHLH </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Homeobox </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">7 </td><td char=".">(2.7%) </td></tr><tr><td>AP2-EREBP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>E2F-DP </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>CXC/tesmin </td><td char=".">3 </td><td char=".">(1.4%) </td><td char=".">4 </td><td char=".">(1.5%) </td></tr><tr><td>Zn_finger, GATA-type </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">1 </td><td char=".">(0.4%) </td></tr><tr><td>Fungal TRF </td><td char=".">0 </td><td char=".">(0.0%) </td><td char=".">2 </td><td char=".">(0.8%) </td></tr><tr><td>Total </td><td char=".">214 </td><td char="."> </td><td char=".">260 </td><td char="."> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw039-TF4"><div class="fn"><p><sup>a</sup>Percentage of the family within the brown algal genome.</p></div></div></div></div></p><div class="block-child-p">This analysis showed that the <em>C. okamuranus</em> genome contains 214 transcription factor genes (<a reveal-id="dsw039-T2" data-open="dsw039-T2" class="link link-reveal link-table xref-fig">Table 2</a>), fewer than the 260 found in the <em>E. siliculosus</em> genome. However, the ratio of transcription factor genes to total genes was ∼1.6% in both genomes. The most abundant TF in both genomes was the Myb family, with 58 and 59 genes in the <em>C. okamuranus</em> and <em>E. siliculosus</em> genomes, respectively. The next most abundant TFs were CBF/NF-Y/archaeal (35), Zinc Finger C2H2-type (23) and Zinc Finger CCCH-type (22). The <em>C. okamuranus</em> genome likely contains three genes for bHLH domains, three for homeobox domains and 17 genes for bZIP domains. The expression level of bZIP genes seems to be similar in germlings and sporophytes, whereas some of CBF/NF-Y/archaeal genes may have different expression levels (<a reveal-id="dsw039-F2" data-open="dsw039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A). Expression and function of phaeophyte TF genes should be expressly examined in future studies. <div data-id="dsw039-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f2p.png?Expires=2147483647&amp;Signature=1homkRjCogAB~7FcVyV8nPSdHtDLSBdbDPOte0D7A6d5XDGOakE4HWMPzg~~u46~ASvA4aqUfh6y1-xEFHZ5RTG3XKuhAC5gq4NznUO45hp0STbrAiqeK2S4K3wh7pK9oybwPhgulWPzP3uhSzRxi3sbqIGGRHM7uiye6gSsfWNk838meM3no5s~5uZ4Gv2xDNOnez4zCY8hgbKfoZotoHWt205WG3f4I7oaZzB5X7YfTNW003VH68pbBNtd73oM3WVyl0iQo0AQdbuM3Z6IcpxpRhg3zXBwTnJKfFlNIwUV4yfM6V8Sffg8oNoI5hJvHhZCkbARjCrmGX01mbreLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat maps compared with gene expression levels of germlings and sporophytes of Cladsiphon okamuranus based on FPKM values. Gene IDs for the genome browser are indicated by sXXX_XXXXX. (A) Expression levels of transcription factors shown in Table 2. (B) Expression levels of receptor kinase genes shown in Supplementary Fig. S2. (C) Expression levels of C. okamuranus genes potentially contributing to biosynthetic pathways of sulfated fucans, alginates, and phlorotannin shown in Figs. 3- -5 and Supplementary Table S3." path-from-xml="dsw039f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005443/dsw039f2p.tif" path-from-xml="dsw039f2p.tif" target="_blank">View large</a><a section="48005443" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f2p.png?Expires=2147483647&amp;Signature=kyfvbfftlQlFiaGmt-~mE4SGfpd2Ido1zvq6m8wgO-QWbI1J62XA4SvCs1i-TK5j1sQpGJUMgWnJZIOapTnPuvMQTcJ5iNBROolJs3EXXzhmvLddkqqWCjKvjOIq7Mflvm1XTf58qgdrXd5Gc696G3c7XTeog7hMtqZrc935rdtlnN16V0WgJUlKzL~9j0YaoqxrrAaxvGbu-kk3a~~TtqYTxY2D~BfDoS0Fzz-YsTSSMsZ~WnojENREhuccBrtNa3ApfNHs2OCwV~b-yqFYqskjk~-9KDQcmTEeSJCFaLkQLsTO4E1tN11VFMKlg0RNgW4x-WYT0Cf3dylrGGiOTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005443&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat maps compared with gene expression levels of germlings and sporophytes of <em>Cladsiphon okamuranus</em> based on FPKM values. Gene IDs for the genome browser are indicated by sXXX_XXXXX. (A) Expression levels of transcription factors shown in <a reveal-id="dsw039-T2" data-open="dsw039-T2" class="link link-reveal link-table xref-fig">Table 2</a>. (B) Expression levels of receptor kinase genes shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. (C) Expression levels of <em>C. okamuranus</em> genes potentially contributing to biosynthetic pathways of sulfated fucans, alginates, and phlorotannin shown in Figs. 3- -5 and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div><div content-id="dsw039-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f2p.png?Expires=2147483647&amp;Signature=1homkRjCogAB~7FcVyV8nPSdHtDLSBdbDPOte0D7A6d5XDGOakE4HWMPzg~~u46~ASvA4aqUfh6y1-xEFHZ5RTG3XKuhAC5gq4NznUO45hp0STbrAiqeK2S4K3wh7pK9oybwPhgulWPzP3uhSzRxi3sbqIGGRHM7uiye6gSsfWNk838meM3no5s~5uZ4Gv2xDNOnez4zCY8hgbKfoZotoHWt205WG3f4I7oaZzB5X7YfTNW003VH68pbBNtd73oM3WVyl0iQo0AQdbuM3Z6IcpxpRhg3zXBwTnJKfFlNIwUV4yfM6V8Sffg8oNoI5hJvHhZCkbARjCrmGX01mbreLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat maps compared with gene expression levels of germlings and sporophytes of Cladsiphon okamuranus based on FPKM values. Gene IDs for the genome browser are indicated by sXXX_XXXXX. (A) Expression levels of transcription factors shown in Table 2. (B) Expression levels of receptor kinase genes shown in Supplementary Fig. S2. (C) Expression levels of C. okamuranus genes potentially contributing to biosynthetic pathways of sulfated fucans, alginates, and phlorotannin shown in Figs. 3- -5 and Supplementary Table S3." path-from-xml="dsw039f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005443/dsw039f2p.tif" path-from-xml="dsw039f2p.tif" target="_blank">View large</a><a section="48005443" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f2p.png?Expires=2147483647&amp;Signature=kyfvbfftlQlFiaGmt-~mE4SGfpd2Ido1zvq6m8wgO-QWbI1J62XA4SvCs1i-TK5j1sQpGJUMgWnJZIOapTnPuvMQTcJ5iNBROolJs3EXXzhmvLddkqqWCjKvjOIq7Mflvm1XTf58qgdrXd5Gc696G3c7XTeog7hMtqZrc935rdtlnN16V0WgJUlKzL~9j0YaoqxrrAaxvGbu-kk3a~~TtqYTxY2D~BfDoS0Fzz-YsTSSMsZ~WnojENREhuccBrtNa3ApfNHs2OCwV~b-yqFYqskjk~-9KDQcmTEeSJCFaLkQLsTO4E1tN11VFMKlg0RNgW4x-WYT0Cf3dylrGGiOTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005443&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat maps compared with gene expression levels of germlings and sporophytes of <em>Cladsiphon okamuranus</em> based on FPKM values. Gene IDs for the genome browser are indicated by sXXX_XXXXX. (A) Expression levels of transcription factors shown in <a reveal-id="dsw039-T2" data-open="dsw039-T2" class="link link-reveal link-table xref-fig">Table 2</a>. (B) Expression levels of receptor kinase genes shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. (C) Expression levels of <em>C. okamuranus</em> genes potentially contributing to biosynthetic pathways of sulfated fucans, alginates, and phlorotannin shown in Figs. 3- -5 and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p></div></div></div>                            <h3 scrollto-destination=48005444 id="48005444" class="section-title" data-legacy-id=SEC3.7.2>3.7.2 Signaling molecules</h3>
<p>Cell–cell signaling molecules play pivotal roles in organismal development and cellular physiological activity. Cock et al.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> specifically analyzed membrane-spanning receptor kinase genes, since receptor kinases have been shown to function in developmental processes in both animals and green plants. They showed that <em>Ectocarpus</em> receptor kinases form a monophyletic clade, as in the case of animals and plants, suggesting that brown algae evolved independently of other taxa. The present domain analysis, using the Pfam database, showed that the <em>C. okamuranus</em> and <em>E. siliculosus</em> genomes contain 286 and 338 genes for protein kinase domains, which are both equivalent to about 2.1% of total genes.</p><p>Cock et al.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> also identified five genes for stramenopile-lineage-specific receptor kinases in <em>E. siliculosus</em>.<a reveal-id="dsw039-B10" data-open="dsw039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The <em>C. okamuranus</em> genome likely contains at least two genes for brown algal-specific receptor kinases (Cok_S_s144_12294 and Cok_S_s043_7434), which have orthologous relationships with <em>E. siliculosus</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Three other genes are additional candidates (Cok_S_s126_11689, Cok_S_s097_10483 and Cok_S_s044_7515). All five genes are found on different scaffolds. RNA-seq reads demonstrate that Cok_S_s144_12294 is expressed in both germlings and sporophytes (<a reveal-id="dsw039-F2" data-open="dsw039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>B).</p>                            <h3 scrollto-destination=48005447 id="48005447" class="section-title" data-legacy-id=SEC3.7.3>3.7.3. Genes associated with biosynthesis of polysaccharides</h3>
<p>Genes encoding enzymes for polysaccharide metabolism in brown algae have been predicted in the <em>E. siliculosus</em> genome.<a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Using informatics methods, we investigated gene families for biosynthesis of sulphated fucans and alginates, compounds characteristic of brown algae. Genes encoding enzymes involved in biosynthetic pathways are apparently conserved between <em>C. okamuranus</em> and <em>E. siliculosus</em>, although downstream enzymes of the polysaccharide biosynthesis pathways are likely to be expanded in each lineage.</p>                            <h3 scrollto-destination=48005449 id="48005449" class="section-title" data-legacy-id=SEC3.7.3.1>3.7.3.1 Fucoidans</h3>
<div class="block-child-p">Fucoidans are a family of sulfated homo- and heteropolysaccharides of brown algae that include L-fucose residues. The family comprises a broad spectrum of polysaccharides, from compounds with high uronic acid content and very little fucose and sulfate, to almost pure α-<span class="small-caps">l</span>-fucan, in which the dominant monosaccharide is fucose. GDP-mannose and <span class="small-caps">l</span>-fucose are original sources of GDP-fucose, which is then transformed from fucan to sulfated fucan (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A). Seven enzymes are likely involved in this pathway (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A). <div data-id="dsw039-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f3p.png?Expires=2147483647&Signature=pezgyKbTFOdEWVouN0HVGQyuBu~O6jLLkwTpZJ2zNSjb50I91LrYqnlP--EseealgdDOZAn~a9ZRKytTFRsIxCU5430a3mQjFFZLXMIDigOENy1lKF6bQkJgDx-wcKQrQx3Vly1i4mz5Yd3eHC~E35yRvMVx3tW3PAAhRWShEAz1iJr-vNgowJpDPqXqzg6iDrhlPe~mw7k84trtR~WfkwrUedd1~aVQJEttxMw5I5eJhpjp-4bznatw2sJiz7MSptqqbbbppkY87OPJlntUTyukgDctrlGDVbXUF8LI0P-wXmKPQefo2DX--t6O9tkxfKWPEVaBCbUjPHyoEt7XUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of all genes in the predicted biosynthetic pathway of sulfated fucans in the draft genome of Cladosiphon okamuranus. (A) A schematic representation of the biosynthetic pathway of sulphated fucans in brown algae, based on the description of Michel et al.9 Genes encoding each of the enzymes in the genomes of Cladosiphon okamuranus (C.ok) and Ectocarpus siliculosus (E.si), and the number of identified genes is shown in broken squares. (B) The location of candidate genes for l-fucosinase (FK) and GDP-fucose pyrophosphorylase (GFPP) in the C. okamuranus genome, suggesting the presence of an enzymatic gene cluster having shared synteny with E. siliculosus (http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi). Two enzymatic genes that contain sulphotransferase and hydrolase domains are clustered with a gene (magenta) for a bifunctional enzyme with FK and GHMP-kinase domains. Genes coloured grey encode conserved proteins with RNA-binding domains." path-from-xml="dsw039f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005450/dsw039f3p.tif" path-from-xml="dsw039f3p.tif" target="_blank">View large</a><a section="48005450" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f3p.png?Expires=2147483647&Signature=2wjjvs68op0xQb-1uxhcTWbCUG0wdEwCf0ZUuT6vbzk3Bk4jijX3bHs2bTh8L17bv16eb1CDNhVgfu0EeIN3lyhYM6r2QMt8lyoLdZvy9wg~q1VJXTbuI2BoZITczSyG~EOASHJmy9EKtn7wf9ExNcQCDrGPs069TpObuo1pHDDNaLOtlvF9YRQjNL4D1sJGy8TrnptVlB1KpasWh1i2FLwGQzVM8jgTxsiD3e8M9hE4Qi8LMMhVHgCcLxy~r08Utg2XSgT1AzJ49wnfBfsCSBvi35rQzj~Av9MEfIR73ykl5fazDORS75KOZLnnsj8ec56uuz~TApvFCQRKo4agIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005450&ar=2647449&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw039f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of all genes in the predicted biosynthetic pathway of sulfated fucans in the draft genome of <em>Cladosiphon okamuranus</em>. (A) A schematic representation of the biosynthetic pathway of sulphated fucans in brown algae, based on the description of Michel et al.<a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Genes encoding each of the enzymes in the genomes of <em>Cladosiphon okamuranus</em> (C.ok) and <em>Ectocarpus siliculosus</em> (E.si), and the number of identified genes is shown in broken squares. (B) The location of candidate genes for <span class="small-caps">l</span>-fucosinase (FK) and GDP-fucose pyrophosphorylase (GFPP) in the <em>C. okamuranus</em> genome, suggesting the presence of an enzymatic gene cluster having shared synteny with <em>E. siliculosus</em> (<a class="link link-uri" href="http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi" target="_blank">http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi</a>). Two enzymatic genes that contain sulphotransferase and hydrolase domains are clustered with a gene (magenta) for a bifunctional enzyme with FK and GHMP-kinase domains. Genes coloured grey encode conserved proteins with RNA-binding domains.</p></div></div><div content-id="dsw039-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f3p.png?Expires=2147483647&Signature=pezgyKbTFOdEWVouN0HVGQyuBu~O6jLLkwTpZJ2zNSjb50I91LrYqnlP--EseealgdDOZAn~a9ZRKytTFRsIxCU5430a3mQjFFZLXMIDigOENy1lKF6bQkJgDx-wcKQrQx3Vly1i4mz5Yd3eHC~E35yRvMVx3tW3PAAhRWShEAz1iJr-vNgowJpDPqXqzg6iDrhlPe~mw7k84trtR~WfkwrUedd1~aVQJEttxMw5I5eJhpjp-4bznatw2sJiz7MSptqqbbbppkY87OPJlntUTyukgDctrlGDVbXUF8LI0P-wXmKPQefo2DX--t6O9tkxfKWPEVaBCbUjPHyoEt7XUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of all genes in the predicted biosynthetic pathway of sulfated fucans in the draft genome of Cladosiphon okamuranus. (A) A schematic representation of the biosynthetic pathway of sulphated fucans in brown algae, based on the description of Michel et al.9 Genes encoding each of the enzymes in the genomes of Cladosiphon okamuranus (C.ok) and Ectocarpus siliculosus (E.si), and the number of identified genes is shown in broken squares. (B) The location of candidate genes for l-fucosinase (FK) and GDP-fucose pyrophosphorylase (GFPP) in the C. okamuranus genome, suggesting the presence of an enzymatic gene cluster having shared synteny with E. siliculosus (http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi). Two enzymatic genes that contain sulphotransferase and hydrolase domains are clustered with a gene (magenta) for a bifunctional enzyme with FK and GHMP-kinase domains. Genes coloured grey encode conserved proteins with RNA-binding domains." path-from-xml="dsw039f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005450/dsw039f3p.tif" path-from-xml="dsw039f3p.tif" target="_blank">View large</a><a section="48005450" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f3p.png?Expires=2147483647&Signature=2wjjvs68op0xQb-1uxhcTWbCUG0wdEwCf0ZUuT6vbzk3Bk4jijX3bHs2bTh8L17bv16eb1CDNhVgfu0EeIN3lyhYM6r2QMt8lyoLdZvy9wg~q1VJXTbuI2BoZITczSyG~EOASHJmy9EKtn7wf9ExNcQCDrGPs069TpObuo1pHDDNaLOtlvF9YRQjNL4D1sJGy8TrnptVlB1KpasWh1i2FLwGQzVM8jgTxsiD3e8M9hE4Qi8LMMhVHgCcLxy~r08Utg2XSgT1AzJ49wnfBfsCSBvi35rQzj~Av9MEfIR73ykl5fazDORS75KOZLnnsj8ec56uuz~TApvFCQRKo4agIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005450&ar=2647449&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw039f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of all genes in the predicted biosynthetic pathway of sulfated fucans in the draft genome of <em>Cladosiphon okamuranus</em>. (A) A schematic representation of the biosynthetic pathway of sulphated fucans in brown algae, based on the description of Michel et al.<a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Genes encoding each of the enzymes in the genomes of <em>Cladosiphon okamuranus</em> (C.ok) and <em>Ectocarpus siliculosus</em> (E.si), and the number of identified genes is shown in broken squares. (B) The location of candidate genes for <span class="small-caps">l</span>-fucosinase (FK) and GDP-fucose pyrophosphorylase (GFPP) in the <em>C. okamuranus</em> genome, suggesting the presence of an enzymatic gene cluster having shared synteny with <em>E. siliculosus</em> (<a class="link link-uri" href="http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi" target="_blank">http://bioinformatics.psb.ugent.be/orcae/overview/Ectsi</a>). Two enzymatic genes that contain sulphotransferase and hydrolase domains are clustered with a gene (magenta) for a bifunctional enzyme with FK and GHMP-kinase domains. Genes coloured grey encode conserved proteins with RNA-binding domains.</p></div></div></div><p>Examination of genes for enzymes in both the <em>C. okamuranus</em> and <em>E. siliculosus</em> genomes identified two genes encoding GDP-mannose 4,6-dehydratase and one for GDP-<span class="small-caps">l</span>-fucose synthase (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The presence of a candidate gene (<em>FK-GFPP</em>) for a bifunctional enzyme possessing both <span class="small-caps">l</span>-fucokinase and GDP-fucose pyrophosphorylase activities was identified in the <em>C. okamuranus</em> genome (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A). These enzyme genes were expressed at germling and sporophyte stages (<a reveal-id="dsw039-F2" data-open="dsw039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C). Comparison of the <em>C. okamuranus</em> and <em>E. siliculosus</em> genomes revealed syntenic localization of enzyme genes, suggesting the presence of an enzymatic gene cluster in both species (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B). Two other enzymatic genes that contain a sulfotransferase domain and a hydrolase domain are clustered with <em>FK-GFPP</em>. The 3′ region of <em>FK-GFPP</em> contains an ankyrin-repeat domain (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B).</p><p>In addition, the <em>C. okamuranus</em> and <em>E. siliculosus</em> genomes contain four genes for fucosyltransferase, and six genes for sulfotransferase, respectively (<a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A). Identification of genes that encode enzymes involved in sulfated fucan biosynthesis suggests that <em>C. okamuranus</em> provides a good source of fucoidans, which should be further characterized in future studies.</p>                            <h3 scrollto-destination=48005453 id="48005453" class="section-title" data-legacy-id=SEC3.7.3.2>3.7.3.2. Alginates</h3>
<div class="block-child-p">Alginates are cell-wall constituents of brown algae. They are chain-forming heteropolysaccharides consisting of blocks of mannuronic acid and glucuronic acid. Composition of the blocks varies by species and by the part of the thallus from which the extraction is made. Enzymes involved in the alginate biosynthetic pathway, including GDP-mannose 6-dehydrogenase (GMD) and mannuronate C5-epimerase (MC5E), have been identified in genomes of <em>E. siliculosus</em><a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and <em>C. okamuranus</em> (<a reveal-id="dsw039-F4" data-open="dsw039-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>A and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The number of enzymatic genes is comparable in both species. Some of MC5E-coding genes likely have different expression levels in germlings and sporophytes (<a reveal-id="dsw039-F2" data-open="dsw039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C). These expression data will be useful for functional analyses of these enzymes. <div data-id="dsw039-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f4p.png?Expires=2147483647&amp;Signature=qxrmj-dj4~lGXjaI5uB69kcm-Mr304G4u1tZMfpkctjeFQ45lQiyaSRwsWEd1YGM6kb-sce8rP4FHE7fGOH1kdeyJzw6XtzBsG9C2bGb0l99aKk9977WGDTEvEnRBmJU5-f~X40NXRJYeIGDod-Kg6GE8aA8bBFYs-HCRTQHmX3SZQKF1lnd5KOwrCSvnxQzgHGReaNx6stcIrF1HdTlvbg7AIXcFeUMfr8DotBXLvnLk-8Jz-8QOloEVGqYzpdXAu5y34dreoCpPEL1ySIr9ugeI2t9Fx2~wcVU8OSTSjv0mhrSmmzBOqo7ToqcszqjWtWbl0LxE36N5UvJ7jTIJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted genes related to the alginate biosynthetic pathway in brown algae and conservation of a key enzyme, GDP-mannose 6-dehydrogenase (GMD). (A) A schematic representation of the alginate biosynthetic pathway in brown algae.9 As in Fig. 3, identified gene numbers in broken squares are followed by species identifiers, C.ok and E.si. (B) A molecular phylogenetic analysis of GMD revealing the conservation of a key enzyme for the alginate biosynthesis pathway between Cladosiphon okamuranus and Ectocarpus siliculosus. The maximum-likelihood (ML) tree was constructed with a WAG + G model. Numbers at nodes indicate more than 70% bootstrap support. Brown stars indicate Cladosiphon GMD proteins that were presumably transferred horizontally from bacteria. UDP glucose 6-dehydrogenase (UGD) is potentially involved in fucoidan biosynthesis, rather than alginate biosynthesis." path-from-xml="dsw039f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005454/dsw039f4p.tif" path-from-xml="dsw039f4p.tif" target="_blank">View large</a><a section="48005454" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f4p.png?Expires=2147483647&amp;Signature=Dftci6-8zRrBPM3b732QhkwIcdPONHA9jhKzkHAwB~IBL8CtMB5J1UpQJvZmy3Dcw5npCVYOhUn3fa71~HxMpjH~ClpQZEL73HcXA1YJpJRLAauiaVJT4HzSDfST97MGbrAnVYDcu68DKiahDyLFnbYpmBrW0JZFvCD8SCj0oPo05O8U2Yd2gwtLwZ6vVZOF4mWxkXKrDBmyTp4zDyA694dkATQfPA~daPAa3AZd1JjNhn6cDFuyJGVG6-h0KtWgFqBXLoq9jaZPCOfETnlxlvAKlmre1cnQQyliAEiM0uNMmB3d5dXHVGWm4d9kno8bATBlhd4YmICvpUffmHQdyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005454&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted genes related to the alginate biosynthetic pathway in brown algae and conservation of a key enzyme, GDP-mannose 6-dehydrogenase (GMD). (A) A schematic representation of the alginate biosynthetic pathway in brown algae.<a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> As in <a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, identified gene numbers in broken squares are followed by species identifiers, C.ok and E.si. (B) A molecular phylogenetic analysis of GMD revealing the conservation of a key enzyme for the alginate biosynthesis pathway between <em>Cladosiphon okamuranus</em> and <em>Ectocarpus siliculosus</em>. The maximum-likelihood (ML) tree was constructed with a WAG + G model. Numbers at nodes indicate more than 70% bootstrap support. Brown stars indicate <em>Cladosiphon</em> GMD proteins that were presumably transferred horizontally from bacteria. UDP glucose 6-dehydrogenase (UGD) is potentially involved in fucoidan biosynthesis, rather than alginate biosynthesis.</p></div></div><div content-id="dsw039-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f4p.png?Expires=2147483647&amp;Signature=qxrmj-dj4~lGXjaI5uB69kcm-Mr304G4u1tZMfpkctjeFQ45lQiyaSRwsWEd1YGM6kb-sce8rP4FHE7fGOH1kdeyJzw6XtzBsG9C2bGb0l99aKk9977WGDTEvEnRBmJU5-f~X40NXRJYeIGDod-Kg6GE8aA8bBFYs-HCRTQHmX3SZQKF1lnd5KOwrCSvnxQzgHGReaNx6stcIrF1HdTlvbg7AIXcFeUMfr8DotBXLvnLk-8Jz-8QOloEVGqYzpdXAu5y34dreoCpPEL1ySIr9ugeI2t9Fx2~wcVU8OSTSjv0mhrSmmzBOqo7ToqcszqjWtWbl0LxE36N5UvJ7jTIJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted genes related to the alginate biosynthetic pathway in brown algae and conservation of a key enzyme, GDP-mannose 6-dehydrogenase (GMD). (A) A schematic representation of the alginate biosynthetic pathway in brown algae.9 As in Fig. 3, identified gene numbers in broken squares are followed by species identifiers, C.ok and E.si. (B) A molecular phylogenetic analysis of GMD revealing the conservation of a key enzyme for the alginate biosynthesis pathway between Cladosiphon okamuranus and Ectocarpus siliculosus. The maximum-likelihood (ML) tree was constructed with a WAG + G model. Numbers at nodes indicate more than 70% bootstrap support. Brown stars indicate Cladosiphon GMD proteins that were presumably transferred horizontally from bacteria. UDP glucose 6-dehydrogenase (UGD) is potentially involved in fucoidan biosynthesis, rather than alginate biosynthesis." path-from-xml="dsw039f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005454/dsw039f4p.tif" path-from-xml="dsw039f4p.tif" target="_blank">View large</a><a section="48005454" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f4p.png?Expires=2147483647&amp;Signature=Dftci6-8zRrBPM3b732QhkwIcdPONHA9jhKzkHAwB~IBL8CtMB5J1UpQJvZmy3Dcw5npCVYOhUn3fa71~HxMpjH~ClpQZEL73HcXA1YJpJRLAauiaVJT4HzSDfST97MGbrAnVYDcu68DKiahDyLFnbYpmBrW0JZFvCD8SCj0oPo05O8U2Yd2gwtLwZ6vVZOF4mWxkXKrDBmyTp4zDyA694dkATQfPA~daPAa3AZd1JjNhn6cDFuyJGVG6-h0KtWgFqBXLoq9jaZPCOfETnlxlvAKlmre1cnQQyliAEiM0uNMmB3d5dXHVGWm4d9kno8bATBlhd4YmICvpUffmHQdyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005454&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted genes related to the alginate biosynthetic pathway in brown algae and conservation of a key enzyme, GDP-mannose 6-dehydrogenase (GMD). (A) A schematic representation of the alginate biosynthetic pathway in brown algae.<a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> As in <a reveal-id="dsw039-F3" data-open="dsw039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, identified gene numbers in broken squares are followed by species identifiers, C.ok and E.si. (B) A molecular phylogenetic analysis of GMD revealing the conservation of a key enzyme for the alginate biosynthesis pathway between <em>Cladosiphon okamuranus</em> and <em>Ectocarpus siliculosus</em>. The maximum-likelihood (ML) tree was constructed with a WAG + G model. Numbers at nodes indicate more than 70% bootstrap support. Brown stars indicate <em>Cladosiphon</em> GMD proteins that were presumably transferred horizontally from bacteria. UDP glucose 6-dehydrogenase (UGD) is potentially involved in fucoidan biosynthesis, rather than alginate biosynthesis.</p></div></div></div><p>Previous studies showed that <em>E. siliculosus</em><a reveal-id="dsw039-B9" data-open="dsw039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and <em>S. japonica</em><a reveal-id="dsw039-B11" data-open="dsw039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> possess three genes for GMDs of possible bacterial origin. Examination of the <em>C. okamuranus</em> genome and further molecular phylogenetic analyses indicated that the <em>C. okamuranus</em> genome also contains three <em>GMD</em> genes for this key enzyme (<a reveal-id="dsw039-F4" data-open="dsw039-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>A and B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Twenty-four <em>MC5E</em> genes of <em>C. okamuranus</em> have homology to <em>E. siliculosus</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Our survey suggests that <em>MC5E</em> genes are independently expanded in each lineage.</p>                            <h3 scrollto-destination=48005456 id="48005456" class="section-title" data-legacy-id=SEC3.7.3.3>3.7.3.3. Phlorotannin biosynthesis</h3>
<p>Brown algal phenols attract considerable attention due to the wide variety of biological activities and potential health benefits of phlorotannins.<a reveal-id="dsw039-B42" data-open="dsw039-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsw039-B43" data-open="dsw039-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Phlorotannins are known only from brown algae and are structural analogs of condensed tannins, such as anthocyanidins and other flavonoid derivatives, a diverse class of metabolites with a vast array of functions in terrestrial plants.<a reveal-id="dsw039-B44" data-open="dsw039-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> They probably serve multiple ecological functions in brown algae, such as antifouling substances<a reveal-id="dsw039-B45" data-open="dsw039-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and chemical defenses against herbivory,<a reveal-id="dsw039-B46" data-open="dsw039-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> in addition to providing UV protection for intertidal seaweeds.<a reveal-id="dsw039-B47" data-open="dsw039-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Their chemical structures are based on aryl-aryl and/or diaryl ether linkages of phloroglucinol (1,3,5-trihydroxybenzene) units and are rather complex. Polymerization leads to a wide range of molecular masses (10 and 100 kDa).<a reveal-id="dsw039-B48" data-open="dsw039-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsw039-B49" data-open="dsw039-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a></p><div class="block-child-p"><em>Polyketide synthase</em> (<em>PKS</em>) is a key gene for phlorotannin biosynthesis.<a reveal-id="dsw039-B50" data-open="dsw039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> It has recently been reported that <em>E. siliculosus</em> PKS1 was classified as a Type-III PKS.<a reveal-id="dsw039-B50" data-open="dsw039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> We surveyed <em>PKS</em> genes in the <em>C. okamuranus</em> genome and found two candidate genes (<a reveal-id="dsw039-F5" data-open="dsw039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Molecular phylogenetic analysis indicated that <em>C. okamuranus</em> Cok_S_597_t3 is the ortholog of <em>E. siliculosus PKS1</em>, a key gene in phlorotannin biosynthesis. <em>E. siliculosus</em> has three <em>type III PKS</em> genes. On the other hand, <em>C. okamuranus</em> has only two (<em>PKS1</em> and <em>PKS2</em>) and has probably lost <em>PKS3</em>. <em>PKS1</em> and <em>PKS2</em> are expressed in germlings and sporophytes of <em>C. okamuranus</em> (<a reveal-id="dsw039-F2" data-open="dsw039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C). <em>Type III PKS</em> genes of brown algae probably originated with bacteria. <div data-id="dsw039-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f5p.png?Expires=2147483647&amp;Signature=V3uhUCP0b1o6ZV~lVcqLgzzDOPxFVoSlyU96v6krOqycsETR1-fmhQcEpQ4TNARiJH7CxVLMhWsND76NSuRyVQROmDTvY8AEK7pelBgmMtYWK9D9Bj5lK0K8ih85kahtzVqR-QZNxLbt88YkqzR1k8YiOGUVSbEAmuAsjgC~2k-ES0F3JCBzdPthxxsj3kU6w4a3SafmlY8KW7buHwyBBvxWUvG7jpsDSAsOsId8V0gpEV-ypcGovRnOfBrlMeZkZStO6b-Hi~Rk5JFnyZ702F4ScE8Z1qMe47OQRo0vD7JC4fQ5tZ0jTe8HywHZwDRYYNyeK6VODqtQ7WJfhlOVFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A molecular phylogenetic tree of two proteins similar to type III PKS in the Cladosiphon okamuranus genome. The ML tree was constructed with a WAG + G + I model. Proteins are indicated by brown stars. Other protein sequences refer to Meslet-Cladiere et al.50." path-from-xml="dsw039f5p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005458/dsw039f5p.tif" path-from-xml="dsw039f5p.tif" target="_blank">View large</a><a section="48005458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f5p.png?Expires=2147483647&amp;Signature=bYwtW5-plpG9gm5JfeB7RFGf2zK4RedanaERLcvQqrfv~YkJiSI3z6Ntfpc3DadOxlk36TK0tsnpGSVuODa7iIPGu20XIzkP5YkceMlcGEZX2Rxnh7MuzCIjqhjJaE3IBh0g3uJvF0t3qB1aWh~xkn~0btQcpcWw3e4sEs-RKWaJol3vcLwgTIhAGAoq2D0hTFukthMEHHTpeNnyhp~uqxwbvfVpzGaXT-Ey2tQN2~XIX7oStKTumDS-m-mkep90rdY36xpnY0pYcnj6oqno7VfH3hPP66Q~eoAVjoMVqylIAYZnBu4Udo~8g9MWp9y0YF3pLqfU~uvTLdfLVrY4mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005458&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f5p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A molecular phylogenetic tree of two proteins similar to type III PKS in the <em>Cladosiphon okamuranus</em> genome. The ML tree was constructed with a WAG + G + I model. Proteins are indicated by brown stars. Other protein sequences refer to Meslet-Cladiere et al.<a reveal-id="dsw039-B50" data-open="dsw039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a>.</p></div></div><div content-id="dsw039-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/m_dsw039f5p.png?Expires=2147483647&amp;Signature=V3uhUCP0b1o6ZV~lVcqLgzzDOPxFVoSlyU96v6krOqycsETR1-fmhQcEpQ4TNARiJH7CxVLMhWsND76NSuRyVQROmDTvY8AEK7pelBgmMtYWK9D9Bj5lK0K8ih85kahtzVqR-QZNxLbt88YkqzR1k8YiOGUVSbEAmuAsjgC~2k-ES0F3JCBzdPthxxsj3kU6w4a3SafmlY8KW7buHwyBBvxWUvG7jpsDSAsOsId8V0gpEV-ypcGovRnOfBrlMeZkZStO6b-Hi~Rk5JFnyZ702F4ScE8Z1qMe47OQRo0vD7JC4fQ5tZ0jTe8HywHZwDRYYNyeK6VODqtQ7WJfhlOVFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A molecular phylogenetic tree of two proteins similar to type III PKS in the Cladosiphon okamuranus genome. The ML tree was constructed with a WAG + G + I model. Proteins are indicated by brown stars. Other protein sequences refer to Meslet-Cladiere et al.50." path-from-xml="dsw039f5p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005458/dsw039f5p.tif" path-from-xml="dsw039f5p.tif" target="_blank">View large</a><a section="48005458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039f5p.png?Expires=2147483647&amp;Signature=bYwtW5-plpG9gm5JfeB7RFGf2zK4RedanaERLcvQqrfv~YkJiSI3z6Ntfpc3DadOxlk36TK0tsnpGSVuODa7iIPGu20XIzkP5YkceMlcGEZX2Rxnh7MuzCIjqhjJaE3IBh0g3uJvF0t3qB1aWh~xkn~0btQcpcWw3e4sEs-RKWaJol3vcLwgTIhAGAoq2D0hTFukthMEHHTpeNnyhp~uqxwbvfVpzGaXT-Ey2tQN2~XIX7oStKTumDS-m-mkep90rdY36xpnY0pYcnj6oqno7VfH3hPP66Q~eoAVjoMVqylIAYZnBu4Udo~8g9MWp9y0YF3pLqfU~uvTLdfLVrY4mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005458&amp;ar=2647449&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw039f5p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A molecular phylogenetic tree of two proteins similar to type III PKS in the <em>Cladosiphon okamuranus</em> genome. The ML tree was constructed with a WAG + G + I model. Proteins are indicated by brown stars. Other protein sequences refer to Meslet-Cladiere et al.<a reveal-id="dsw039-B50" data-open="dsw039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a>.</p></div></div></div>                            <h2 scrollto-destination=48005459 id="48005459" class="section-title" data-legacy-id=SEC4>4. Data availability</h2>
<p>A genome browser has been established at: <a class="link link-uri" href="http://marinegenomics.oist.jp/gallery/" target="_blank">http://marinegenomics.oist.jp/gallery/</a>. Gene annotations from domain searches and Blast2GO<a reveal-id="dsw039-B51" data-open="dsw039-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> are available at: (<a class="link link-uri" href="http://marinegenomics.oist.jp/gallery/" target="_blank">http://marinegenomics.oist.jp/gallery/</a>).</p>                            <h2 scrollto-destination=48005461 id="48005461" class="section-title" >Accession numbers</h2>
<p>All Illumina reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) are available from DRA accession no. DRA004654. Genome sequences of the S-strain have been deposited with accession nos. BDDF01000001-01004525 (contigs) and DF977685-DF978416 (scaffolds).</p>                            <h2 scrollto-destination=48005463 id="48005463" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank all members of Marine Genomics Unit and the DNA Sequencing Section of OIST for their support of this work. We also thank Dr. Steven Aird for editing the manuscript.</p>                            <h3 scrollto-destination=48005465 id="48005465" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=48005467 id="48005467" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=48005469 id="48005469" class="section-title" >Funding</h3>
<p>This study was supported by OIST internal funding for the Marine Genomics Unit.</p>                            <h2 scrollto-destination=48005471 id="48005471" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw039-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Adl</div> <div class="given-names">S.M.</div></div><div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">A.G.</div></div><div class="name string-name"><div class="surname">Farmer</div> <div class="given-names">M.A.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">The new higher level classification of eukaryotes with emphasis on the taxonomy of protists</div>. <div class="source">J. Eukaryot. Microbiol</div> ., <div class="volume">52</div>, <div class="fpage">399</div>–<div class="lpage">451</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20new%20higher%20level%20classification%20of%20eukaryotes%20with%20emphasis%20on%20the%20taxonomy%20of%20protists&amp;author=S.M.%20Adl&amp;author=A.G.%20Simpson&amp;author=M.A.%20Farmer&amp;publication_year=2005&amp;journal=J.%20Eukaryot.%20Microbiol&amp;volume=52&amp;pages=399-451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.2005.52.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.2005.52.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.2005.52.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16248873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keeling</div> <div class="given-names">P.J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Chromalveolates and the evolution of plastids by secondary endosymbiosis</div>. <div class="source">J Eukaryot. Microbiol</div> ., <div class="volume">56</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromalveolates%20and%20the%20evolution%20of%20plastids%20by%20secondary%20endosymbiosis&amp;author=P.J.%20Keeling&amp;publication_year=2009&amp;journal=J%20Eukaryot.%20Microbiol&amp;volume=56&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.2009.56.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.2009.56.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.2009.56.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19335769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cavalier-Smith</div> <div class="given-names">T.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Principles of protein and lipid targeting in secondary symbiogenesis: euglenoid, dinoflagellate, and sporozoan plastid origins and the eukaryote family tree</div>. <div class="source">J. Eukaryot. Microbiol</div> ., <div class="volume">46</div>, <div class="fpage">347</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principles%20of%20protein%20and%20lipid%20targeting%20in%20secondary%20symbiogenesis%3A%20euglenoid%2C%20dinoflagellate%2C%20and%20sporozoan%20plastid%20origins%20and%20the%20eukaryote%20family%20tree&amp;author=T.%20Cavalier-Smith&amp;publication_year=1999&amp;journal=J.%20Eukaryot.%20Microbiol&amp;volume=46&amp;pages=347-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.1999.46.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.1999.46.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.1999.46.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18092388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Den Hoek</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Mann</div> <div class="given-names">D.G.</div></div><div class="name string-name"><div class="surname">Jahns</div> <div class="given-names">H.M.</div></div></div> <div class="year">1995</div>, <div class="collab">Algae: An Introduction to Phycology</div>. Cambridge University Press, Cambridge.</div></div></div></div><div content-id="dsw039-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Yoshinaga</div> <div class="given-names">K.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Checklist of marine algae of Japan (revised in 2015)</div>. <div class="source">Jpn. J. Phycol (Sôrui)</div> ., <div class="volume">63</div>, <div class="fpage">129</div>–<div class="lpage">89</div>.</div></div></div></div><div content-id="dsw039-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tako</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nakada</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Hongou</div> <div class="given-names">F.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Chemical characterization of fucoidan from commercially cultured nemacystus decipiens (Itomozuku)</div>. <div class="source">Biosci. Biotechnol. Biochem</div> ., <div class="volume">63</div>, <div class="fpage">1813</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemical%20characterization%20of%20fucoidan%20from%20commercially%20cultured%20nemacystus%20decipiens%20(Itomozuku)&amp;author=M.%20Tako&amp;author=T.%20Nakada&amp;author=F.%20Hongou&amp;publication_year=1999&amp;journal=Biosci.%20Biotechnol.%20Biochem&amp;volume=63&amp;pages=1813-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.63.1813" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.63.1813" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.63.1813"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26300174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baba</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Snoeck</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Pauwels</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">de Clercq</div> <div class="given-names">E.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">Sulfated polysaccharides are potent and selective inhibitors of various enveloped viruses, including herpes simplex virus, cytomegalovirus, vesicular stomatitis virus, and human immunodeficiency virus</div>. <div class="source">Antimicrob. Agents Chemother</div> ., <div class="volume">32</div>, <div class="fpage">1742</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sulfated%20polysaccharides%20are%20potent%20and%20selective%20inhibitors%20of%20various%20enveloped%20viruses%2C%20including%20herpes%20simplex%20virus%2C%20cytomegalovirus%2C%20vesicular%20stomatitis%20virus%2C%20and%20human%20immunodeficiency%20virus&amp;author=M.%20Baba&amp;author=R.%20Snoeck&amp;author=R.%20Pauwels&amp;author=E.%20de%20Clercq&amp;publication_year=1988&amp;journal=Antimicrob.%20Agents%20Chemother&amp;volume=32&amp;pages=1742-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.32.11.1742" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.32.11.1742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.32.11.1742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2472775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">T.Y.</div></div><div class="name string-name"><div class="surname">Hassid</div> <div class="given-names">W.Z.</div></div></div> <div class="year">1966</div>, 
<div class="article-title">Pathway of algnic acid synthesis in the marine brown alga, <em>Fucus gardneri</em> Silva</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">241</div>, <div class="fpage">5284</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathway%20of%20algnic%20acid%20synthesis%20in%20the%20marine%20brown%20alga%2C%20Fucus%20gardneri%20Silva&amp;author=T.Y.%20Lin&amp;author=W.Z.%20Hassid&amp;publication_year=1966&amp;journal=J.%20Biol.%20Chem&amp;volume=241&amp;pages=5284-97" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5954796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Michel</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Tonon</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Scornet</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Cock</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Kloareg</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The cell wall polysaccharide metabolism of the brown alga <em>Ectocarpus siliculosus</em>. Insights into the evolution of extracellular matrix polysaccharides in Eukaryotes</div>. <div class="source">New Phytol</div> ., <div class="volume">188</div>, <div class="fpage">82</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cell%20wall%20polysaccharide%20metabolism%20of%20the%20brown%20alga%20Ectocarpus%20siliculosus.%20Insights%20into%20the%20evolution%20of%20extracellular%20matrix%20polysaccharides%20in%20Eukaryotes&amp;author=G.%20Michel&amp;author=T.%20Tonon&amp;author=D.%20Scornet&amp;author=J.M.%20Cock&amp;author=B.%20Kloareg&amp;publication_year=2010&amp;journal=New%20Phytol&amp;volume=188&amp;pages=82-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03374.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03374.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03374.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20618907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cock</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Rouze</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">The Ectocarpus genome and the independent evolution of multicellularity in brown algae</div>. <div class="source">Nature</div> , <div class="volume">465</div>, <div class="fpage">617</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ectocarpus%20genome%20and%20the%20independent%20evolution%20of%20multicellularity%20in%20brown%20algae&amp;author=J.M.%20Cock&amp;author=L.%20Sterck&amp;author=P.%20Rouze&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=617-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20520714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Miao</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Saccharina genomes provide novel insight into kelp biology</div>. <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">6986</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharina%20genomes%20provide%20novel%20insight%20into%20kelp%20biology&amp;author=N.%20Ye&amp;author=X.%20Zhang&amp;author=M.%20Miao&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=6986" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms7986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms7986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms7986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25908475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Siemer</div> <div class="given-names">B.L.</div></div><div class="name string-name"><div class="surname">Stam</div> <div class="given-names">W.T.</div></div><div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">J.L.</div></div><div class="name string-name"><div class="surname">Pedersen</div> <div class="given-names">P.M.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Phylogenetic relationships of the brown algal orders Ectocarpales, Chordariales, Dictyosiphonales, and Tilopteridales (Phaeophyceae) based on RUBISCO large subunit and spacer sequences</div>. <div class="source">J. Phycol</div> ., <div class="volume">34</div>, <div class="fpage">1038</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20of%20the%20brown%20algal%20orders%20Ectocarpales%2C%20Chordariales%2C%20Dictyosiphonales%2C%20and%20Tilopteridales%20(Phaeophyceae)%20based%20on%20RUBISCO%20large%20subunit%20and%20spacer%20sequences&amp;author=B.L.%20Siemer&amp;author=W.T.%20Stam&amp;author=J.L.%20Olsen&amp;author=P.M.%20Pedersen&amp;publication_year=1998&amp;journal=J.%20Phycol&amp;volume=34&amp;pages=1038-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1529-8817.1998.341038.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1529-8817.1998.341038.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1529-8817.1998.341038.x"> </span></div> </div></div></div></div></div><div content-id="dsw039-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bentley</div> <div class="given-names">D.R.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Whole-genome re-sequencing</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">16</div>, <div class="fpage">545</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20re-sequencing&amp;author=D.R.%20Bentley&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=16&amp;pages=545-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2006.10.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2006.10.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2006.10.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17055251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcais</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Marcais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A.M.</div></div><div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div><div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div><div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Fan</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">The sequence and de novo assembly of the giant panda genome</div>. <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">311</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20and%20de%20novo%20assembly%20of%20the%20giant%20panda%20genome&amp;author=R.%20Li&amp;author=W.%20Fan&amp;author=G.%20Tian&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=311-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">HaploMerger: reconstructing allelic relationships for polymorphic diploid genome assemblies</div>. <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1581</div>–<div class="lpage">1588</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HaploMerger%3A%20reconstructing%20allelic%20relationships%20for%20polymorphic%20diploid%20genome%20assemblies&amp;author=S.%20Huang&amp;author=Z.%20Chen&amp;author=G.%20Huang&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1581-1588" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.133652.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.133652.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.133652.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22555592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peng</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Leung</div> <div class="given-names">H.C.</div></div><div class="name string-name"><div class="surname">Yiu</div> <div class="given-names">S.M.</div></div><div class="name string-name"><div class="surname">Chin</div> <div class="given-names">F.Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1420</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IDBA-UD%3A%20a%20de%20novo%20assembler%20for%20single-cell%20and%20metagenomic%20sequencing%20data%20with%20highly%20uneven%20depth&amp;author=Y.%20Peng&amp;author=H.C.%20Leung&amp;author=S.M.%20Yiu&amp;author=F.Y.%20Chin&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1420-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>. <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">821</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div><div class="name string-name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div><div class="name string-name"><div class="surname">Vingron</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1086</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oases%3A%20robust%20de%20novo%20RNA-seq%20assembly%20across%20the%20dynamic%20range%20of%20expression%20levels&amp;author=M.H.%20Schulz&amp;author=D.R.%20Zerbino&amp;author=M.%20Vingron&amp;author=E.%20Birney&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1086-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22368243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding</div>. <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div><div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div><div class="name string-name"><div class="surname">Mount</div> <div class="given-names">S.M.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">5654</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;author=B.J.%20Haas&amp;author=A.L.%20Delcher&amp;author=S.M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=5654-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takeuchi</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Kawashima</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Koyanagi</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Draft genome of the pearl oyster <em>Pinctada fucata</em>: a platform for understanding bivalve biology</div>. <div class="source">DNA Res</div> ., <div class="volume">19</div>, <div class="fpage">117</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20pearl%20oyster%20Pinctada%20fucata%3A%20a%20platform%20for%20understanding%20bivalve%20biology&amp;author=T.%20Takeuchi&amp;author=T.%20Kawashima&amp;author=R.%20Koyanagi&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=117-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22315334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benson</div> <div class="given-names">G.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div><div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">De novo identification of repeat families in large genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">21 Suppl 1</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21 Suppl 1&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Dagman</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Pelton</div> <div class="given-names">P.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">CENSOR – a program for identification and elimination of repetitive elements from DNA sequences</div>. <div class="source">Comput. Chem</div> ., <div class="volume">20</div>, <div class="fpage">119</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CENSOR%20%E2%80%93%20a%20program%20for%20identification%20and%20elimination%20of%20repetitive%20elements%20from%20DNA%20sequences&amp;author=J.%20Jurka&amp;author=P.%20Klonowski&amp;author=V.%20Dagman&amp;author=P.%20Pelton&amp;publication_year=1996&amp;journal=Comput.%20Chem&amp;volume=20&amp;pages=119-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0097-8485(96)80013-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(96)80013-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0097-8485(96)80013-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8867843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div><div class="name string-name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Repbase update, a database of eukaryotic repetitive elements</div>. <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">110</div>, <div class="fpage">462</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div><div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Schuster-Bockler</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Pfam: clans, web tools and services</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">D247</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20clans%2C%20web%20tools%20and%20services&amp;author=R.D.%20Finn&amp;author=J.%20Mistry&amp;author=B.%20Schuster-Bockler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D247-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Profile hidden Markov models</div>. <div class="source">Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">755</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20hidden%20Markov%20models&amp;author=S.R.%20Eddy&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=755-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.9.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">221</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeaView%20version%204%3A%20a%20multiplatform%20graphical%20user%20interface%20for%20sequence%20alignment%20and%20phylogenetic%20tree%20building&amp;author=M.%20Gouy&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=221-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2731</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Skinner</div> <div class="given-names">M.E.</div></div><div class="name string-name"><div class="surname">Uzilov</div> <div class="given-names">A.V.</div></div><div class="name string-name"><div class="surname">Stein</div> <div class="given-names">L.D.</div></div><div class="name string-name"><div class="surname">Mungall</div> <div class="given-names">C.J.</div></div><div class="name string-name"><div class="surname">Holmes</div> <div class="given-names">I.H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">JBrowse: a next-generation genome browser</div>. <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1630</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JBrowse%3A%20a%20next-generation%20genome%20browser&amp;author=M.E.%20Skinner&amp;author=A.V.%20Uzilov&amp;author=L.D.%20Stein&amp;author=C.J.%20Mungall&amp;author=I.H.%20Holmes&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1630-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.094607.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.094607.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.094607.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19570905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div><div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kraskov</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Stogbauer</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Andrzejak</div> <div class="given-names">R.G.</div></div><div class="name string-name"><div class="surname">Grassberger</div> <div class="given-names">P.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Hierarchical clustering using mutual information</div>. <div class="source">Europhys. Lett</div> ., <div class="volume">70</div>, <div class="fpage">278</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hierarchical%20clustering%20using%20mutual%20information&amp;author=A.%20Kraskov&amp;author=H.%20Stogbauer&amp;author=R.G.%20Andrzejak&amp;author=P.%20Grassberger&amp;publication_year=2005&amp;journal=Europhys.%20Lett&amp;volume=70&amp;pages=278-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1209/epl/i2004-10483-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1209%2Fepl%2Fi2004-10483-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1209%2Fepl%2Fi2004-10483-y"> </span></div> </div></div></div></div></div><div content-id="dsw039-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Szekely</div> <div class="given-names">G.J.</div></div><div class="name string-name"><div class="surname">Rizzo</div> <div class="given-names">M.L.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Hierarchical clustering via joint between-within distances: Extending Ward's minimum variance method</div>. <div class="source">J. Classif</div> ., <div class="volume">22</div>, <div class="fpage">151</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hierarchical%20clustering%20via%20joint%20between-within%20distances%3A%20Extending%20Ward's%20minimum%20variance%20method&amp;author=G.J.%20Szekely&amp;author=M.L.%20Rizzo&amp;publication_year=2005&amp;journal=J.%20Classif&amp;volume=22&amp;pages=151-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00357-005-0012-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00357-005-0012-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00357-005-0012-9"> </span></div> </div></div></div></div></div><div content-id="dsw039-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Y.W.</div></div><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Y.H.</div></div><div class="name string-name"><div class="surname">Tringe</div> <div class="given-names">S.G.</div></div><div class="name string-name"><div class="surname">Simmons</div> <div class="given-names">B.A.</div></div><div class="name string-name"><div class="surname">Singer</div> <div class="given-names">S.W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">MaxBin: an automated binning method to recover individual genomes from metagenomes using an expectation-maximization algorithm</div>. <div class="source">Microbiome</div> , <div class="volume">2</div>, <div class="fpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MaxBin%3A%20an%20automated%20binning%20method%20to%20recover%20individual%20genomes%20from%20metagenomes%20using%20an%20expectation-maximization%20algorithm&amp;author=Y.W.%20Wu&amp;author=Y.H.%20Tang&amp;author=S.G.%20Tringe&amp;author=B.A.%20Simmons&amp;author=S.W.%20Singer&amp;publication_year=2014&amp;journal=Microbiome&amp;volume=2&amp;pages=26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2049-2618-2-26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2049-2618-2-26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2049-2618-2-26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25136443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Connan</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Stengel</div> <div class="given-names">D.B.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Impacts of ambient salinity and copper on brown algae: 2. Interactive effects on phenolic pool and assessment of metal binding capacity of phlorotannin</div>. <div class="source">Aquat. Toxicol</div> ., <div class="volume">104</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impacts%20of%20ambient%20salinity%20and%20copper%20on%20brown%20algae%3A%202.%20Interactive%20effects%20on%20phenolic%20pool%20and%20assessment%20of%20metal%20binding%20capacity%20of%20phlorotannin&amp;author=S.%20Connan&amp;author=D.B.%20Stengel&amp;publication_year=2011&amp;journal=Aquat.%20Toxicol&amp;volume=104&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquatox.2011.03.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquatox.2011.03.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquatox.2011.03.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21543047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">N.V.</div></div><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.K.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Potential pharmacological applications of polyphenolic derivatives from marine brown algae</div>. <div class="source">Environ. Toxicol. Pharmacol</div> ., <div class="volume">32</div>, <div class="fpage">325</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Potential%20pharmacological%20applications%20of%20polyphenolic%20derivatives%20from%20marine%20brown%20algae&amp;author=N.V.%20Thomas&amp;author=S.K.%20Kim&amp;publication_year=2011&amp;journal=Environ.%20Toxicol.%20Pharmacol&amp;volume=32&amp;pages=325-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.etap.2011.09.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.etap.2011.09.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.etap.2011.09.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22004951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dakora</div> <div class="given-names">F.D.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Plant flavonoid: biological molecules for useful exploitation</div>. <div class="source">Aust. J. Plant Physiol</div> ., <div class="volume">22</div>, <div class="fpage">87</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20flavonoid%3A%20biological%20molecules%20for%20useful%20exploitation&amp;author=F.D.%20Dakora&amp;publication_year=1995&amp;journal=Aust.%20J.%20Plant%20Physiol&amp;volume=22&amp;pages=87-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/PP9950087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FPP9950087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FPP9950087"> </span></div> </div></div></div></div></div><div content-id="dsw039-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sieburth</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Conover</div> <div class="given-names">J.T.</div></div></div> <div class="year">1965</div>, 
<div class="article-title">Slicks associated with Trichodesmium Blooms in the Sargasso Sea</div>. <div class="source">Nature</div> , <div class="volume">205</div>, <div class="fpage">830</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slicks%20associated%20with%20Trichodesmium%20Blooms%20in%20the%20Sargasso%20Sea&amp;author=J.M.%20Sieburth&amp;author=J.T.%20Conover&amp;publication_year=1965&amp;journal=Nature&amp;volume=205&amp;pages=830-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/205830b0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F205830b0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F205830b0"> </span></div> </div></div></div></div></div><div content-id="dsw039-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pavia</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Toth</div> <div class="given-names">B.G.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Inducible chemical resistance to herbivory in the brown seaweed Ascophyllum nodosum</div>. <div class="source">Ecology</div> , <div class="volume">81</div>, <div class="fpage">3215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inducible%20chemical%20resistance%20to%20herbivory%20in%20the%20brown%20seaweed%20Ascophyllum%20nodosum&amp;author=H.%20Pavia&amp;author=B.G.%20Toth&amp;publication_year=2000&amp;journal=Ecology&amp;volume=81&amp;pages=3215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/0012-9658(2000)081[3212:ICRTHI]2.0.CO;2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F0012-9658(2000)081[3212:ICRTHI]2.0.CO;2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F0012-9658(2000)081[3212:ICRTHI]2.0.CO;2"> </span></div> </div></div></div></div></div><div content-id="dsw039-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amsler</div> <div class="given-names">C.D.</div></div><div class="name string-name"><div class="surname">Fairhead</div> <div class="given-names">V.A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Defensive and sensory chemical ecology of brown algae</div>. <div class="source">Adv. Botanical Res</div> ., <div class="volume">43</div>, <div class="fpage">1</div>–<div class="lpage">91</div>.</div></div></div></div><div content-id="dsw039-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boettcher</div> <div class="given-names">A.A.</div></div><div class="name string-name"><div class="surname">Targett</div> <div class="given-names">N.M.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Role of polyphenolic molecular size in reduction of assimilation efficiency in Xiphister mucosus</div>. <div class="source">Ecology</div> , <div class="volume">74</div>, <div class="fpage">891</div>–<div class="lpage">903</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20polyphenolic%20molecular%20size%20in%20reduction%20of%20assimilation%20efficiency%20in%20Xiphister%20mucosus&amp;author=A.A.%20Boettcher&amp;author=N.M.%20Targett&amp;publication_year=1993&amp;journal=Ecology&amp;volume=74&amp;pages=891-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/1940814" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F1940814" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F1940814"> </span></div> </div></div></div></div></div><div content-id="dsw039-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le Lann</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Connan</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Stiger-Pouvreau</div> <div class="given-names">V.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Phenology, TPC and size-fractioning phenolics variability in temperate Sargassaceae (Phaeophyceae, Fucales) from Western Brittany: native versus introduced species</div>. <div class="source">Mar. Environ. Res</div> ., <div class="volume">80</div>, <div class="fpage">1</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenology%2C%20TPC%20and%20size-fractioning%20phenolics%20variability%20in%20temperate%20Sargassaceae%20(Phaeophyceae%2C%20Fucales)%20from%20Western%20Brittany%3A%20native%20versus%20introduced%20species&amp;author=K.%20Le%20Lann&amp;author=S.%20Connan&amp;author=V.%20Stiger-Pouvreau&amp;publication_year=2012&amp;journal=Mar.%20Environ.%20Res&amp;volume=80&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.marenvres.2012.05.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.marenvres.2012.05.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.marenvres.2012.05.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22770761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meslet-Cladiere</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Delage</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Leroux</div> <div class="given-names">C.J.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Structure/function analysis of a type iii polyketide synthase in the brown alga <em>Ectocarpus siliculosus</em> reveals a biochemical pathway in phlorotannin monomer biosynthesis</div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3089</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2Ffunction%20analysis%20of%20a%20type%20iii%20polyketide%20synthase%20in%20the%20brown%20alga%20Ectocarpus%20siliculosus%20reveals%20a%20biochemical%20pathway%20in%20phlorotannin%20monomer%20biosynthesis&amp;author=L.%20Meslet-Cladiere&amp;author=L.%20Delage&amp;author=C.J.%20Leroux&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3089-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.111336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.111336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.111336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23983220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw039-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw039-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Arnold</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Sebastian-Leon</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">B2G-FAR, a species-centered GO annotation repository</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">919</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=B2G-FAR%2C%20a%20species-centered%20GO%20annotation%20repository&amp;author=S.%20Gotz&amp;author=R.%20Arnold&amp;author=P.%20Sebastian-Leon&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=919-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2647449&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw039/2/dsw039_Supp.pdf?Expires=2147483647&amp;Signature=IO92Xie8E1XSDomFJhootwTbwNkswPFnujBa8dYvlFoutjQNLuhD7eNAQhT1JeJIx0FzaDC43Rb0Y9QJMwGSJzcUpTa4gicYdHW208N3JrDLq1wCb4lBJiWKvwumDFkN7tTZJIEue5T7EACNtfUGgW0jDbhe6RitPefvZXHekszvz8kyvHSVw0Uv8IXIzKe~DOzS3iaQKFoOQd3YTwzkkD0AS37twtfPcEIA7IyYUKrx3NYPiHFuvlCUIntRloqVtMXUyk-OoHbRm-WE-g4gXUaC6tN-bX83wwIMcwsqHZH9BzrWTXu4TRVtCCkXNCl6mKSqvw5eq2HcNJmVzij5-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,399</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2647449"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f6%2f561%2f2647449">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000392723300006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F6%2F561%2F2647449" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2647449'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="draft genome of the brown alga, Cladosiphon okamuranus, S-strain: a platform for future studies of ‘mozuku’ biology" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f561%2f2647449">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

