

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/1/346817","@type":"ScholarlyArticle","name":"Comprehensive Transcriptome Profiling and Functional Analysis of the Frog ( Bombina maxima ) Immune System","datePublished":"2013-08-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst035","keywords":["transcriptome","frog","immune system","tissue-specific genes","non-coding RNAs"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/1/346817","author":[{"name":"Zhao, Feng","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China    2  University of Chinese Academy of Sciences ,  Beijing ,  China","@type":"Person"},{"name":"Yan, Chao","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China    2  University of Chinese Academy of Sciences ,  Beijing ,  China","@type":"Person"},{"name":"Wang, Xuan","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China    2  University of Chinese Academy of Sciences ,  Beijing ,  China","@type":"Person"},{"name":"Yang, Yang","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China    2  University of Chinese Academy of Sciences ,  Beijing ,  China","@type":"Person"},{"name":"Wang, Guangyin","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China    2  University of Chinese Academy of Sciences ,  Beijing ,  China","@type":"Person"},{"name":"Lee, Wenhui","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China","@type":"Person"},{"name":"Xiang, Yang","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China","@type":"Person"},{"name":"Zhang, Yun","affiliation":"1  Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province ,  Kunming Institute of Zoology, Chinese Academy of Sciences ,  Kunming, Yunnan 650223 ,  China","@type":"Person"}],"description":"Abstract.  Amphibians occupy a key phylogenetic position in vertebrates and evolution of the immune system. But, the resources of its transcriptome or genome ar","pageStart":"1","pageEnd":"13","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comprehensive Transcriptome Profiling and Functional Analysis of the Frog ( Bombina maxima ) Immune System","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comprehensive Transcriptome Profiling and Functional Analysis of the Frog ( Bombina maxima ) Immune System" />
    <meta property="og:description" content="Abstract.  Amphibians occupy a key phylogenetic position in vertebrates and evolution of the immune system. But, the resources of its transcriptome or genome ar" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/1/346817" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-08-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhao, Feng" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China" /><meta name="citation_author" content="Yan, Chao" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China" /><meta name="citation_author" content="Wang, Xuan" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China" /><meta name="citation_author" content="Yang, Yang" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China" /><meta name="citation_author" content="Wang, Guangyin" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China" /><meta name="citation_author" content="Lee, Wenhui" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author" content="Xiang, Yang" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_author" content="Zhang, Yun" /><meta name="citation_author_institution" content="Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan 650223, China" /><meta name="citation_title" content="Comprehensive Transcriptome Profiling and Functional Analysis of the Frog (Bombina maxima) Immune System" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="13" /><meta name="citation_doi" content="10.1093/dnares/dst035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Insights from Xenopus genomes; Genome Dyn.;  citation_year=2006;  citation_volume=2;  citation_pages=138-53; " /><meta name="citation_reference" content="citation_title=Manipulating the Xenopus genome with transposable elements; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_issue=Suppl 1;  citation_pages=S11" /><meta name="citation_reference" content="citation_title=Sequencing and analysis of 10,967 full-length cDNA clones from Xenopus laevis and Xenopus tropicalis reveals post-tetraploidization transcriptome remodeling; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=796-803; " /><meta name="citation_reference" content="citation_title=The genome of the Western clawed frog Xenopus tropicalis; Science;  citation_year=2010;  citation_volume=328;  citation_pages=633-36; " /><meta name="citation_reference" content="citation_title=Genetics of global gene expression; Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=862-72; " /><meta name="citation_reference" content="citation_title=Bm-TFF2, a trefoil factor protein with platelet activation activity from frog Bombina maxima skin secretions; Biochem. Biophys. Res. Commun.;  citation_year=2005;  citation_volume=330;  citation_pages=1027-33; " /><meta name="citation_reference" content="citation_title=Variety of antimicrobial peptides in the Bombina maxima toad and evidence of their rapid diversification; Eur. J. Immunol.;  citation_year=2005;  citation_volume=35;  citation_pages=1220-9; " /><meta name="citation_reference" content="citation_title=Establishment, characterization and immortalization of a fibroblast cell line from the Chinese red belly toad Bombina maxima skin; Cytotechnology;  citation_year=2012;  citation_volume=64;  citation_pages=95-105; " /><meta name="citation_reference" content="citation_title=A novel non-lens betagamma-crystallin and trefoil factor complex from amphibian skin and its functional implications; PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e1770" /><meta name="citation_reference" content="citation_title=Structural basis for signal transduction by the Toll/interleukin-1 receptor domains; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=111-5; " /><meta name="citation_reference" content="citation_title=The genus Xenopus as a multispecies model for evolutionary and comparative immunobiology of the 21st century; Dev. Comp. Immunol.;  citation_year=2011;  citation_volume=35;  citation_pages=916-23; " /><meta name="citation_reference" content="citation_title=Antimicrobial peptides: the ancient arm of the human immune system; Virulence;  citation_year=2010;  citation_volume=1;  citation_pages=440-64; " /><meta name="citation_reference" content="citation_title=Extremely abundant antimicrobial peptides existed in the skins of nine kinds of Chinese odorous frogs; J. Proteome Res.;  citation_year=2012;  citation_volume=11;  citation_pages=306-19; " /><meta name="citation_reference" content="citation_title=The immune system of Xenopus; Annu. Rev. Immunol.;  citation_year=1989;  citation_volume=7;  citation_pages=251-75; " /><meta name="citation_reference" content="citation_title=An ancient origin for the enigmatic flat-headed frogs (Bombinatoridae: Barbourula) from the islands of Southeast Asia; PloS One;  citation_year=2010;  citation_volume=5;  citation_pages=e12090" /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1135-45; " /><meta name="citation_reference" content="citation_title=RNA sequencing reveals a diverse and dynamic repertoire of the Xenopus tropicalis transcriptome over development; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=201-16; " /><meta name="citation_reference" content="citation_title=A new mathematical model for relative quantification in real-time RT-PCR; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=e45" /><meta name="citation_reference" content="citation_title=Sequencing, de novo annotation and analysis of the first Anguilla anguilla transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=635" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the oriental river prawn, Macrobrachium nipponense using 454 pyrosequencing for discovery of genes and markers; PloS One;  citation_year=2012;  citation_volume=7;  citation_pages=e39727" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of silver carp (Hypophthalmichthys molitrix) by paired-end RNA sequencing; DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=131-42; " /><meta name="citation_reference" content="citation_title=Analysis of muscle and ovary transcriptome of Sus scrofa: assembly, annotation and marker discovery; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=343-51; " /><meta name="citation_reference" content="citation_title=Intracellular metabolic pathways control immune tolerance; Trends Immunol.;  citation_year=2012;  citation_volume=33;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Beyond DNA: zinc finger domains as RNA-binding modules; Methods Mol. Biol.;  citation_year=2010;  citation_volume=649;  citation_pages=479-91; " /><meta name="citation_reference" content="citation_title=Transcriptomics of a giant freshwater prawn (Macrobrachium rosenbergii): de novo assembly, annotation and marker discovery; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e27938" /><meta name="citation_reference" content="citation_title=Systematic characterization of the zinc-finger-containing proteins in the mouse transcriptome; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1430-42; " /><meta name="citation_reference" content="citation_title=IgD, like IgM, is a primordial immunoglobulin class perpetuated in most jawed vertebrates; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=10723-28; " /><meta name="citation_reference" content="citation_title=Endotoxin recognition: in fish or not in fish?; FEBS Lett.;  citation_year=2005;  citation_volume=579;  citation_pages=6519-28; " /><meta name="citation_reference" content="citation_title=Frog urinary bladder epithelial cells express TLR4 and respond to bacterial LPS by increase of iNOS expression and l-arginine uptake; Am. J. Physiol. Regul. Integr. Comp. Physiol.;  citation_year=2012;  citation_volume=303;  citation_pages=R1042-52; " /><meta name="citation_reference" content="citation_title=Toll-like receptor-4 mediates lipopolysaccharide-induced signal transduction; J. Biol. Chem.;  citation_year=1999;  citation_volume=274;  citation_pages=10689-92; " /><meta name="citation_reference" content="citation_title=Mal (MyD88-adapter-like) is required for Toll-like receptor-4 signal transduction; Nature;  citation_year=2001;  citation_volume=413;  citation_pages=78-83; " /><meta name="citation_reference" content="citation_title=The Toll/IL-1 receptor binding protein MyD88 is required for Xenopus axis formation; Mech. Develop.;  citation_year=2000;  citation_volume=97;  citation_pages=85-92; " /><meta name="citation_reference" content="citation_title=WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases; Cell. Mol. Life Sci.;  citation_year=2001;  citation_volume=58;  citation_pages=2085-97; " /><meta name="citation_reference" content="citation_title=Regulator of complement activation (RCA) gene cluster in Xenopus tropicalis; Immunogenetics;  citation_year=2009;  citation_volume=61;  citation_pages=371-84; " /><meta name="citation_reference" content="citation_title=Expression of the third component of complement, C3, in regenerating limb blastema cells of urodeles; J. Immunol.;  citation_year=1998;  citation_volume=161;  citation_pages=6819-24; " /><meta name="citation_reference" content="citation_title=The molecular evolution of the interleukin-1 family of cytokines; IL-18 in teleost fish; Dev. Comp. Immunol.;  citation_year=2004;  citation_volume=28;  citation_pages=395-413; " /><meta name="citation_reference" content="citation_title=[Bactericidal activity of cells of the immune system]; Pathol. Biol.;  citation_year=1993;  citation_volume=41;  citation_pages=887-96; " /><meta name="citation_reference" content="citation_title=Macrophage migration inhibitory factor: a regulator of innate immunity; Nat. Rev. Immunol.;  citation_year=2003;  citation_volume=3;  citation_pages=791-800; " /><meta name="citation_reference" content="citation_title=p47 GTPases: regulators of immunity to intracellular pathogens; Nat. Rev. Immunol.;  citation_year=2004;  citation_volume=4;  citation_pages=100-9; " /><meta name="citation_reference" content="citation_title=A giant GTPase, very large inducible GTPase-1, is inducible by IFNs; J. Immunol.;  citation_year=2003;  citation_volume=171;  citation_pages=1255-65; " /><meta name="citation_reference" content="citation_title=The inflammasomes: guardians of the body; Annu. Rev. Immunol.;  citation_year=2009;  citation_volume=27;  citation_pages=229-65; " /><meta name="citation_reference" content="citation_title=‘Big Bang’ emergence of the combinatorial immune system; Dev. Comp. Immunol.;  citation_year=1999;  citation_volume=23;  citation_pages=107-11; " /><meta name="citation_reference" content="citation_title=Origin and evolution of the adaptive immune system: genetic events and selective pressures; Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=47-59; " /><meta name="citation_reference" content="citation_title=Comparative analyses of immunoglobulin genes: surprises and portents; Nat. Rev. Immunol.;  citation_year=2002;  citation_volume=2;  citation_pages=688-98; " /><meta name="citation_reference" content="citation_title=IgY: clues to the origins of modern antibodies; Immunol. Today;  citation_year=1995;  citation_volume=16;  citation_pages=392-8; " /><meta name="citation_reference" content="citation_title=Genome dynamics of the major histocompatibility complex: insights from genome paralogy; Immunogenetics;  citation_year=1999;  citation_volume=50;  citation_pages=134-45; " /><meta name="citation_reference" content="citation_title=Genetic diversity of MHC class I loci in six non-model frogs is shaped by positive selection and gene duplication; Heredity (Edinb);  citation_year=2012;  citation_volume=109;  citation_pages=146-55; " /><meta name="citation_reference" content="citation_title=Expressed major histocompatibility complex class II loci in fishes; Immunol Rev.;  citation_year=1998;  citation_volume=166;  citation_pages=294-300; " /><meta name="citation_reference" content="citation_title=Evolution of the major histocompatibility complex: molecular cloning of major histocompatibility complex class I from the amphibian Xenopus; Proc. Natl Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=537-41; " /><meta name="citation_reference" content="citation_title=Timing and mechanism of ancient vertebrate genome duplications—the adventure of a hypothesis; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=559-67; " /><meta name="citation_reference" content="citation_title=A novel type of class I gene organization in vertebrates: a large family of non-MHC-linked class I genes is expressed at the RNA level in the amphibian Xenopus; EMBO J.;  citation_year=1993;  citation_volume=12;  citation_pages=4385-96; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/1/992697/dst035.pdf" /><meta name="description" content="Abstract.  Amphibians occupy a key phylogenetic position in vertebrates and evolution of the immune system. But, the resources of its transcriptome or genome ar" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/1/346817" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/1/346817" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/1/1/346817");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst035';
    </script>
<title>
	Comprehensive Transcriptome Profiling and Functional Analysis of the Frog (Bombina maxima) Immune System | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./346817" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Ba18PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFvLCBGZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQW5pbWFsIE1vZGVscyBhbmQgSHVtYW4gRGlzZWFzZSBNZWNoYW5pc21zIG9mIHRoZSBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMgYW5kIFl1bm5hbiBQcm92aW5jZSwgS3VubWluZyBJbnN0aXR1dGUgb2YgWm9vbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBLdW5taW5nLCBZdW5uYW4gNjUwMjIzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbiwgQ2hhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEFuaW1hbCBNb2RlbHMgYW5kIEh1bWFuIERpc2Vhc2UgTWVjaGFuaXNtcyBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzIGFuZCBZdW5uYW4gUHJvdmluY2UsIEt1bm1pbmcgSW5zdGl0dXRlIG9mIFpvb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgS3VubWluZywgWXVubmFuIDY1MDIyMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXR5IG9mIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBYdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQW5pbWFsIE1vZGVscyBhbmQgSHVtYW4gRGlzZWFzZSBNZWNoYW5pc21zIG9mIHRoZSBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMgYW5kIFl1bm5hbiBQcm92aW5jZSwgS3VubWluZyBJbnN0aXR1dGUgb2YgWm9vbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBLdW5taW5nLCBZdW5uYW4gNjUwMjIzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIFlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTW9kZWxzIGFuZCBIdW1hbiBEaXNlYXNlIE1lY2hhbmlzbXMgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcyBhbmQgWXVubmFuIFByb3ZpbmNlLCBLdW5taW5nIEluc3RpdHV0ZSBvZiBab29sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEt1bm1pbmcsIFl1bm5hbiA2NTAyMjMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgR3Vhbmd5aW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTW9kZWxzIGFuZCBIdW1hbiBEaXNlYXNlIE1lY2hhbmlzbXMgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcyBhbmQgWXVubmFuIFByb3ZpbmNlLCBLdW5taW5nIEluc3RpdHV0ZSBvZiBab29sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEt1bm1pbmcsIFl1bm5hbiA2NTAyMjMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVlLCBXZW5odWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTW9kZWxzIGFuZCBIdW1hbiBEaXNlYXNlIE1lY2hhbmlzbXMgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcyBhbmQgWXVubmFuIFByb3ZpbmNlLCBLdW5taW5nIEluc3RpdHV0ZSBvZiBab29sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEt1bm1pbmcsIFl1bm5hbiA2NTAyMjMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGlhbmcsIFlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTW9kZWxzIGFuZCBIdW1hbiBEaXNlYXNlIE1lY2hhbmlzbXMgb2YgdGhlIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcyBhbmQgWXVubmFuIFByb3ZpbmNlLCBLdW5taW5nIEluc3RpdHV0ZSBvZiBab29sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEt1bm1pbmcsIFl1bm5hbiA2NTAyMjMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIFl1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEFuaW1hbCBNb2RlbHMgYW5kIEh1bWFuIERpc2Vhc2UgTWVjaGFuaXNtcyBvZiB0aGUgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzIGFuZCBZdW5uYW4gUHJvdmluY2UsIEt1bm1pbmcgSW5zdGl0dXRlIG9mIFpvb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgS3VubWluZywgWXVubmFuIDY1MDIyMywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvbXByZWhlbnNpdmUgVHJhbnNjcmlwdG9tZSBQcm9maWxpbmcgYW5kIEZ1bmN0aW9uYWwgQW5hbHlzaXMgb2YgdGhlIEZyb2cgKEJvbWJpbmEgbWF4aW1hKSBJbW11bmUgU3lzdGVtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGZyb20gWGVub3B1cyBnZW5vbWVzOyBHZW5vbWUgRHluLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTM4LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW5pcHVsYXRpbmcgdGhlIFhlbm9wdXMgZ2Vub21lIHdpdGggdHJhbnNwb3NhYmxlIGVsZW1lbnRzOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsIDE7ICBjaXRhdGlvbl9wYWdlcz1TMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiAxMCw5NjcgZnVsbC1sZW5ndGggY0ROQSBjbG9uZXMgZnJvbSBYZW5vcHVzIGxhZXZpcyBhbmQgWGVub3B1cyB0cm9waWNhbGlzIHJldmVhbHMgcG9zdC10ZXRyYXBsb2lkaXphdGlvbiB0cmFuc2NyaXB0b21lIHJlbW9kZWxpbmc7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgV2VzdGVybiBjbGF3ZWQgZnJvZyBYZW5vcHVzIHRyb3BpY2FsaXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzI4OyAgY2l0YXRpb25fcGFnZXM9NjMzLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBnbG9iYWwgZ2VuZSBleHByZXNzaW9uOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz04NjItNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJtLVRGRjIsIGEgdHJlZm9pbCBmYWN0b3IgcHJvdGVpbiB3aXRoIHBsYXRlbGV0IGFjdGl2YXRpb24gYWN0aXZpdHkgZnJvbSBmcm9nIEJvbWJpbmEgbWF4aW1hIHNraW4gc2VjcmV0aW9uczsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMjctMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlldHkgb2YgYW50aW1pY3JvYmlhbCBwZXB0aWRlcyBpbiB0aGUgQm9tYmluYSBtYXhpbWEgdG9hZCBhbmQgZXZpZGVuY2Ugb2YgdGhlaXIgcmFwaWQgZGl2ZXJzaWZpY2F0aW9uOyBFdXIuIEouIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTIyMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RhYmxpc2htZW50LCBjaGFyYWN0ZXJpemF0aW9uIGFuZCBpbW1vcnRhbGl6YXRpb24gb2YgYSBmaWJyb2JsYXN0IGNlbGwgbGluZSBmcm9tIHRoZSBDaGluZXNlIHJlZCBiZWxseSB0b2FkIEJvbWJpbmEgbWF4aW1hIHNraW47IEN5dG90ZWNobm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9OTUtMTA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIG5vbi1sZW5zIGJldGFnYW1tYS1jcnlzdGFsbGluIGFuZCB0cmVmb2lsIGZhY3RvciBjb21wbGV4IGZyb20gYW1waGliaWFuIHNraW4gYW5kIGl0cyBmdW5jdGlvbmFsIGltcGxpY2F0aW9uczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUxNzcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBiYXNpcyBmb3Igc2lnbmFsIHRyYW5zZHVjdGlvbiBieSB0aGUgVG9sbC9pbnRlcmxldWtpbi0xIHJlY2VwdG9yIGRvbWFpbnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz0xMTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbnVzIFhlbm9wdXMgYXMgYSBtdWx0aXNwZWNpZXMgbW9kZWwgZm9yIGV2b2x1dGlvbmFyeSBhbmQgY29tcGFyYXRpdmUgaW1tdW5vYmlvbG9neSBvZiB0aGUgMjFzdCBjZW50dXJ5OyBEZXYuIENvbXAuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9OTE2LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpbWljcm9iaWFsIHBlcHRpZGVzOiB0aGUgYW5jaWVudCBhcm0gb2YgdGhlIGh1bWFuIGltbXVuZSBzeXN0ZW07IFZpcnVsZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NDQwLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRyZW1lbHkgYWJ1bmRhbnQgYW50aW1pY3JvYmlhbCBwZXB0aWRlcyBleGlzdGVkIGluIHRoZSBza2lucyBvZiBuaW5lIGtpbmRzIG9mIENoaW5lc2Ugb2Rvcm91cyBmcm9nczsgSi4gUHJvdGVvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTMwNi0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGltbXVuZSBzeXN0ZW0gb2YgWGVub3B1czsgQW5udS4gUmV2LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjUxLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbmNpZW50IG9yaWdpbiBmb3IgdGhlIGVuaWdtYXRpYyBmbGF0LWhlYWRlZCBmcm9ncyAoQm9tYmluYXRvcmlkYWU6IEJhcmJvdXJ1bGEpIGZyb20gdGhlIGlzbGFuZHMgb2YgU291dGhlYXN0IEFzaWE7IFBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTIwOTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmc7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMTM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgc2VxdWVuY2luZyByZXZlYWxzIGEgZGl2ZXJzZSBhbmQgZHluYW1pYyByZXBlcnRvaXJlIG9mIHRoZSBYZW5vcHVzIHRyb3BpY2FsaXMgdHJhbnNjcmlwdG9tZSBvdmVyIGRldmVsb3BtZW50OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTIwMS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbWF0aGVtYXRpY2FsIG1vZGVsIGZvciByZWxhdGl2ZSBxdWFudGlmaWNhdGlvbiBpbiByZWFsLXRpbWUgUlQtUENSOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz1lNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nLCBkZSBub3ZvIGFubm90YXRpb24gYW5kIGFuYWx5c2lzIG9mIHRoZSBmaXJzdCBBbmd1aWxsYSBhbmd1aWxsYSB0cmFuc2NyaXB0b21lOiBFZWVsQmFzZSBvcGVucyBuZXcgcGVyc3BlY3RpdmVzIGZvciB0aGUgc3R1ZHkgb2YgdGhlIGNyaXRpY2FsbHkgZW5kYW5nZXJlZCBFdXJvcGVhbiBlZWw7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTYzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdGhlIG9yaWVudGFsIHJpdmVyIHByYXduLCBNYWNyb2JyYWNoaXVtIG5pcHBvbmVuc2UgdXNpbmcgNDU0IHB5cm9zZXF1ZW5jaW5nIGZvciBkaXNjb3Zlcnkgb2YgZ2VuZXMgYW5kIG1hcmtlcnM7IFBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzk3MjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIHNpbHZlciBjYXJwIChIeXBvcGh0aGFsbWljaHRoeXMgbW9saXRyaXgpIGJ5IHBhaXJlZC1lbmQgUk5BIHNlcXVlbmNpbmc7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTMxLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBtdXNjbGUgYW5kIG92YXJ5IHRyYW5zY3JpcHRvbWUgb2YgU3VzIHNjcm9mYTogYXNzZW1ibHksIGFubm90YXRpb24gYW5kIG1hcmtlciBkaXNjb3Zlcnk7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYWNlbGx1bGFyIG1ldGFib2xpYyBwYXRod2F5cyBjb250cm9sIGltbXVuZSB0b2xlcmFuY2U7IFRyZW5kcyBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmV5b25kIEROQTogemluYyBmaW5nZXIgZG9tYWlucyBhcyBSTkEtYmluZGluZyBtb2R1bGVzOyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjQ5OyAgY2l0YXRpb25fcGFnZXM9NDc5LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pY3Mgb2YgYSBnaWFudCBmcmVzaHdhdGVyIHByYXduIChNYWNyb2JyYWNoaXVtIHJvc2VuYmVyZ2lpKTogZGUgbm92byBhc3NlbWJseSwgYW5ub3RhdGlvbiBhbmQgbWFya2VyIGRpc2NvdmVyeTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyNzkzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgemluYy1maW5nZXItY29udGFpbmluZyBwcm90ZWlucyBpbiB0aGUgbW91c2UgdHJhbnNjcmlwdG9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNDMwLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZ0QsIGxpa2UgSWdNLCBpcyBhIHByaW1vcmRpYWwgaW1tdW5vZ2xvYnVsaW4gY2xhc3MgcGVycGV0dWF0ZWQgaW4gbW9zdCBqYXdlZCB2ZXJ0ZWJyYXRlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xMDcyMy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kb3RveGluIHJlY29nbml0aW9uOiBpbiBmaXNoIG9yIG5vdCBpbiBmaXNoPzsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01Nzk7ICBjaXRhdGlvbl9wYWdlcz02NTE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gcm9nIHVyaW5hcnkgYmxhZGRlciBlcGl0aGVsaWFsIGNlbGxzIGV4cHJlc3MgVExSNCBhbmQgcmVzcG9uZCB0byBiYWN0ZXJpYWwgTFBTIGJ5IGluY3JlYXNlIG9mIGlOT1MgZXhwcmVzc2lvbiBhbmQgbC1hcmdpbmluZSB1cHRha2U7IEFtLiBKLiBQaHlzaW9sLiBSZWd1bC4gSW50ZWdyLiBDb21wLiBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDM7ICBjaXRhdGlvbl9wYWdlcz1SMTA0Mi01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9sbC1saWtlIHJlY2VwdG9yLTQgbWVkaWF0ZXMgbGlwb3BvbHlzYWNjaGFyaWRlLWluZHVjZWQgc2lnbmFsIHRyYW5zZHVjdGlvbjsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc0OyAgY2l0YXRpb25fcGFnZXM9MTA2ODktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hbCAoTXlEODgtYWRhcHRlci1saWtlKSBpcyByZXF1aXJlZCBmb3IgVG9sbC1saWtlIHJlY2VwdG9yLTQgc2lnbmFsIHRyYW5zZHVjdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQxMzsgIGNpdGF0aW9uX3BhZ2VzPTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgVG9sbC9JTC0xIHJlY2VwdG9yIGJpbmRpbmcgcHJvdGVpbiBNeUQ4OCBpcyByZXF1aXJlZCBmb3IgWGVub3B1cyBheGlzIGZvcm1hdGlvbjsgTWVjaC4gRGV2ZWxvcC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz04NS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V0QtcmVwZWF0IHByb3RlaW5zOiBzdHJ1Y3R1cmUgY2hhcmFjdGVyaXN0aWNzLCBiaW9sb2dpY2FsIGZ1bmN0aW9uLCBhbmQgdGhlaXIgaW52b2x2ZW1lbnQgaW4gaHVtYW4gZGlzZWFzZXM7IENlbGwuIE1vbC4gTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjA4NS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdG9yIG9mIGNvbXBsZW1lbnQgYWN0aXZhdGlvbiAoUkNBKSBnZW5lIGNsdXN0ZXIgaW4gWGVub3B1cyB0cm9waWNhbGlzOyBJbW11bm9nZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTM3MS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiB0aGUgdGhpcmQgY29tcG9uZW50IG9mIGNvbXBsZW1lbnQsIEMzLCBpbiByZWdlbmVyYXRpbmcgbGltYiBibGFzdGVtYSBjZWxscyBvZiB1cm9kZWxlczsgSi4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9NjgxOS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vbGVjdWxhciBldm9sdXRpb24gb2YgdGhlIGludGVybGV1a2luLTEgZmFtaWx5IG9mIGN5dG9raW5lczsgSUwtMTggaW4gdGVsZW9zdCBmaXNoOyBEZXYuIENvbXAuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9Mzk1LTQxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9W0JhY3RlcmljaWRhbCBhY3Rpdml0eSBvZiBjZWxscyBvZiB0aGUgaW1tdW5lIHN5c3RlbV07IFBhdGhvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz04ODctOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hY3JvcGhhZ2UgbWlncmF0aW9uIGluaGliaXRvcnkgZmFjdG9yOiBhIHJlZ3VsYXRvciBvZiBpbm5hdGUgaW1tdW5pdHk7IE5hdC4gUmV2LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NzkxLTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9cDQ3IEdUUGFzZXM6IHJlZ3VsYXRvcnMgb2YgaW1tdW5pdHkgdG8gaW50cmFjZWxsdWxhciBwYXRob2dlbnM7IE5hdC4gUmV2LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTAwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2lhbnQgR1RQYXNlLCB2ZXJ5IGxhcmdlIGluZHVjaWJsZSBHVFBhc2UtMSwgaXMgaW5kdWNpYmxlIGJ5IElGTnM7IEouIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbmZsYW1tYXNvbWVzOiBndWFyZGlhbnMgb2YgdGhlIGJvZHk7IEFubnUuIFJldi4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMjktNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPeKAmEJpZyBCYW5n4oCZIGVtZXJnZW5jZSBvZiB0aGUgY29tYmluYXRvcmlhbCBpbW11bmUgc3lzdGVtOyBEZXYuIENvbXAuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTA3LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIGV2b2x1dGlvbiBvZiB0aGUgYWRhcHRpdmUgaW1tdW5lIHN5c3RlbTogZ2VuZXRpYyBldmVudHMgYW5kIHNlbGVjdGl2ZSBwcmVzc3VyZXM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00Ny01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzZXMgb2YgaW1tdW5vZ2xvYnVsaW4gZ2VuZXM6IHN1cnByaXNlcyBhbmQgcG9ydGVudHM7IE5hdC4gUmV2LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9Njg4LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZ1k6IGNsdWVzIHRvIHRoZSBvcmlnaW5zIG9mIG1vZGVybiBhbnRpYm9kaWVzOyBJbW11bm9sLiBUb2RheTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTM5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZHluYW1pY3Mgb2YgdGhlIG1ham9yIGhpc3RvY29tcGF0aWJpbGl0eSBjb21wbGV4OiBpbnNpZ2h0cyBmcm9tIGdlbm9tZSBwYXJhbG9neTsgSW1tdW5vZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0xMzQtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IG9mIE1IQyBjbGFzcyBJIGxvY2kgaW4gc2l4IG5vbi1tb2RlbCBmcm9ncyBpcyBzaGFwZWQgYnkgcG9zaXRpdmUgc2VsZWN0aW9uIGFuZCBnZW5lIGR1cGxpY2F0aW9uOyBIZXJlZGl0eSAoRWRpbmIpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE0Ni01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2VkIG1ham9yIGhpc3RvY29tcGF0aWJpbGl0eSBjb21wbGV4IGNsYXNzIElJIGxvY2kgaW4gZmlzaGVzOyBJbW11bm9sIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9Mjk0LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHRoZSBtYWpvciBoaXN0b2NvbXBhdGliaWxpdHkgY29tcGxleDogbW9sZWN1bGFyIGNsb25pbmcgb2YgbWFqb3IgaGlzdG9jb21wYXRpYmlsaXR5IGNvbXBsZXggY2xhc3MgSSBmcm9tIHRoZSBhbXBoaWJpYW4gWGVub3B1czsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTUzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGltaW5nIGFuZCBtZWNoYW5pc20gb2YgYW5jaWVudCB2ZXJ0ZWJyYXRlIGdlbm9tZSBkdXBsaWNhdGlvbnPigJR0aGUgYWR2ZW50dXJlIG9mIGEgaHlwb3RoZXNpczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTU1OS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCB0eXBlIG9mIGNsYXNzIEkgZ2VuZSBvcmdhbml6YXRpb24gaW4gdmVydGVicmF0ZXM6IGEgbGFyZ2UgZmFtaWx5IG9mIG5vbi1NSEMtbGlua2VkIGNsYXNzIEkgZ2VuZXMgaXMgZXhwcmVzc2VkIGF0IHRoZSBSTkEgbGV2ZWwgaW4gdGhlIGFtcGhpYmlhbiBYZW5vcHVzOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDM4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzEvMS85OTI2OTcvZHN0MDM1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBbXBoaWJpYW5zIG9jY3VweSBhIGtleSBwaHlsb2dlbmV0aWMgcG9zaXRpb24gaW4gdmVydGVicmF0ZXMgYW5kIGV2b2x1dGlvbiBvZiB0aGUgaW1tdW5lIHN5c3RlbS4gQnV0LCB0aGUgcmVzb3VyY2VzIG9mIGl0cyB0cmFuc2NyaXB0b21lIG9yIGdlbm9tZSBhciIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ2ODE3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZK4V3cre0T84h2Vx3grNQH4GGULy" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f1%2f346817" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f1%2f346817" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13381" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&amp;Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4898831">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898831">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898833">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898833">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898836">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898836">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898847">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898880">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898880">Authors' Contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898882">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898882">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898884">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898884">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4898886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4898886">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/21/1/15/347005">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comprehensive Transcriptome Profiling and Functional Analysis of the Frog (<em>Bombina maxima</em>) Immune System
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Feng Zhao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chao Yan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chao+Yan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chao Yan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xuan Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xuan+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xuan Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yang Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guangyin Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guangyin+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guangyin Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wenhui Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wenhui+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wenhui Lee</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Xiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 871-65198515. Fax. +86 871-65198515. Email: <a href="mailto:zhangy@mail.kiz.ac.cn" target="_blank">zhangy@mail.kiz.ac.cn</a> (Y. Zhang); Tel. +86 871-65194279. Fax. +86 871-65194279. Email: <a href="mailto:yang_xiang000@hotmail.com" target="_blank">yang_xiang000@hotmail.com</a> (Y. Xiang).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Xiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xiang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xiang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yang Xiang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Animal Models and Human Disease Mechanisms of the Chinese Academy of Sciences and Yunnan Province</div>, <div class="insititution">Kunming Institute of Zoology, Chinese Academy of Sciences</div>, <div class="addr-line">Kunming, Yunnan 650223</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 871-65198515. Fax. +86 871-65198515. Email: <a href="mailto:zhangy@mail.kiz.ac.cn" target="_blank">zhangy@mail.kiz.ac.cn</a> (Y. Zhang); Tel. +86 871-65194279. Fax. +86 871-65194279. Email: <a href="mailto:yang_xiang000@hotmail.com" target="_blank">yang_xiang000@hotmail.com</a> (Y. Xiang).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yun Zhang</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 1–13, <a href='https://doi.org/10.1093/dnares/dst035'>https://doi.org/10.1093/dnares/dst035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 August 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 March 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 July 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="346817" href="/dnaresearch/article-pdf/21/1/1/992697/dst035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Feng Zhao, Chao Yan, Xuan Wang, Yang Yang, Guangyin Wang, Wenhui Lee, Yang Xiang, Yun Zhang;  Comprehensive Transcriptome Profiling and Functional Analysis of the Frog (<em>Bombina maxima</em>) Immune System, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 1–13, <a href='https://doi.org/10.1093/dnares/dst035'>https://doi.org/10.1093/dnares/dst035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/346817?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/346817?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comprehensive%20Transcriptome%20Profiling%20and%20Functional%20Analysis%20of%20the%20Frog%20(%26lt%3Bem%26gt%3BBombina%20maxima%26lt%3B%2Fem%26gt%3B)%20Immune%20System&amp;publicationDate=2013-08-13&amp;volumeNum=21&amp;issueNum=1&amp;author=Zhao%2C%20Feng%3B%20Yan%2C%20Chao&amp;startPage=1&amp;endPage=13&amp;contentId=10.1093%2Fdnares%2Fdst035&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f1%2f346817" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4898831 id="4898831" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Amphibians occupy a key phylogenetic position in vertebrates and evolution of the immune system. But, the resources of its transcriptome or genome are still little now. <em>Bombina maxima</em> possess strong ability to survival in very harsh environment with a more mature immune system. We obtained a comprehensive transcriptome by RNA-sequencing technology. 14.3% of transcripts were identified to be skin-specific genes, most of which were not isolated from skin secretion in previous works or novel non-coding RNAs. 27.9% of transcripts were mapped into 242 predicted KEGG pathways and 6.16% of transcripts related to human disease and cancer. Of 39 448 transcripts with the coding sequence, at least 1501 transcripts (570 genes) related to the immune system process. The molecules of immune signalling pathway were almost presented, several transcripts with high expression in skin and stomach. Experiments showed that lipopolysaccharide or bacteria challenge stimulated pro-inflammatory cytokine production and activation of pro-inflammatory caspase-1. These frog's data can remarkably expand the existing genome or transcriptome resources of amphibians, especially immunity data. The entity of the data provides a valuable platform for further investigation on more detailed immune response in <em>B. maxima</em> and a comparative study with other amphibians.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="frog">frog</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;immune system&quot;">immune system</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tissue-specific genes&quot;">tissue-specific genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-coding RNAs&quot;">non-coding RNAs</a></div>                            <h2 scrollto-destination=4898833 id="4898833" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Amphibians have been used as vertebrate models for the experimentalist since the 19th century. Since 50 years or so, <em>Xenopus laevis</em> is the most widely used anuran amphibian research organism,<sup><a reveal-id="DST035C1" data-open="DST035C1" class="link link-ref link-reveal xref-bibr">1</a></sup> an important model of vertebrate cell biology and development for many decades.<sup><a reveal-id="DST035C2" data-open="DST035C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, of the thousands of species, only two closely related species (<em>Xenopus silurana tropicalis</em> and <em>X. laevis</em>) have considerable genomic resources: published genome of the Pipid, <em>Xenopus (silurana) tropicalis</em>, and transcript information of <em>X. laevis</em>.<sup><a reveal-id="DST035C3" data-open="DST035C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST035C4" data-open="DST035C4" class="link link-ref link-reveal xref-bibr">4</a></sup><em>Xenopus tropicalis</em>, of which genome analysis results show that at least 1700 predicted genes identified to be human disease genes, occupies a key phylogenetic position among amniots and teleost fish.<sup><a reveal-id="DST035C4" data-open="DST035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> But, understanding of the genome requires large quality of transcripts as reference sequences. The transcriptome is comprised of a comprehensive set of genes active in selected tissues and species. Therefore, understanding global dynamics of the transcriptome is essential for interpreting genome expanded information and unravelling phenotypic variation.<sup><a reveal-id="DST035C5" data-open="DST035C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>In our previous studies, the Chinese red belly frog (<em>Bombina maxima</em>) has various different skin secretions defencing external stimulus, which is an endemic amphibian and lives in very harsh environments of the mountainous regions of southwestern China.<sup><a reveal-id="DST035C6  DST035C9" data-open="DST035C6  DST035C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> This study was motivated by two reasons. First, genome-wide or transcriptome-wide approaches are now being applied to whole-genome wide duplication of Xenopodinae.<sup><a reveal-id="DST035C10" data-open="DST035C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the amphibian is still the lack of genomic resources like genomic sequences and transcriptome dataset, in spite of two <em>Xenopus</em> toads' genome or transcriptome being available. Secondly, more and more investigators found that <em>Xenopus</em> model offers an invaluable research tool for immunological research.<sup><a reveal-id="DST035C11" data-open="DST035C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In addition, the most ancient mechanism of immunity was considered to be direct microbicidal activity of antimicrobial peptides (AMPs) in the human immune system.<sup><a reveal-id="DST035C12" data-open="DST035C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In frogs, extremely abundant and diverse AMPs have been reported in our previous studies.<sup><a reveal-id="DST035C13" data-open="DST035C13" class="link link-ref link-reveal xref-bibr">13</a></sup> A lot of AMPs were also rich in <em>B. maxima</em>.<sup><a reveal-id="DST035C7" data-open="DST035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In recent studies, the immune system of amphibian species is nearly as complex as that of mammals.<sup><a reveal-id="DST035C11" data-open="DST035C11" class="link link-ref link-reveal xref-bibr">11</a></sup> But, research on the immune system of anuran amphibians exclusively relies on <em>X. laevis</em> as a model.<sup><a reveal-id="DST035C14" data-open="DST035C14" class="link link-ref link-reveal xref-bibr">14</a></sup><em>Bombina</em> and <em>Xenoanura</em> diverged in the middle Jurassic and the ancestor of <em>Bombina</em> appeared the Eocene of Palaeogene.<sup><a reveal-id="DST035C15" data-open="DST035C15" class="link link-ref link-reveal xref-bibr">15</a></sup> From the living environment and skin secretions of <em>B. maxima</em>, it could have a more mature immune system compared with <em>Xenopus</em>, potentially suited to be as a new model for identifying the nature of innate or adaptive immunity in vertebrates. Thus, we performed the latest paired-end RNA-sequencing (RNA-seq) techniques<sup><a reveal-id="DST035C16" data-open="DST035C16" class="link link-ref link-reveal xref-bibr">16</a></sup> to transcriptome-wide depict its feature of the immune system. The database of the transcriptome will submit to public databases for free to use.</p>                            <h2 scrollto-destination=4898836 id="4898836" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=4898837 id="4898837" class="section-title" data-legacy-id=s2a>2.1. Animals and ethics</h3>
<p><em>Bombina maxima</em>, used here were obtained from the Kunming Institute of Zoology, which has been used in our previous studies. All the animal studies were reviewed and approved by the animal care and use committee of Kunming Institute of Zoology, The Chinese Academy of Sciences.</p>                            <h3 scrollto-destination=4898839 id="4898839" class="section-title" data-legacy-id=s2b>2.2. Construction of the transcriptome</h3>
<p>Skin and blood tissues were sampled from four adult frogs with no injury. RNA extraction and library construction are described in detail in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S1</a>. Transcriptome-sequencing clean reads is basis for assembly and following analysis (the data are archived at the NCBI Sequence Read Archive under accession number SRA058577; <a class="link link-uri" href="http://ncbi.nlm.nih.gov/sra">http://ncbi.nlm.nih.gov/sra</a>). Transcriptome is assembled in <em>de novo</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S2</a>).</p>                            <h3 scrollto-destination=4898841 id="4898841" class="section-title" data-legacy-id=s2c>2.3. Transcriptome functional annotation</h3>
<p>The transcript sequences were identified with the coding sequence (CDS) to search for gene ontology (GO) terms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S3</a>). All translated protein sequences were identified protein domain prediction. Tissue-specific genes and comparative expression analysis are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S4</a>. Identification of novel non-coding RNAs (ncRNAs) is identified as <em>Tan's</em> methods.<sup><a reveal-id="DST035C17" data-open="DST035C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The detailed description is mentioned in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S5</a>.</p>                            <h3 scrollto-destination=4898843 id="4898843" class="section-title" data-legacy-id=s2d>2.4. Tissue expression of selected immune-related genes</h3>
<p>We used cDNA as a template for reverse transcription PCR (RT-PCR) amplification of the coding regions of several given immune genes. Equal amounts of total RNA (1.05 μg) from each tissue were pooled for cDNA generation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S6</a>). Quantitative RT-PCR (qRT-PCR) was carried out using the LightCycler<sup>®</sup> 480 (Roche, Switzerland) with SYBR premix Ex Taq II (TaKaRa, Japan). Samples were run in triplicates. Primers used in qRT-PCR are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. For each primer, a melting curve of the PCR product was also performed to ensure the absence of artefacts. The comparative crossing points (ΔCP) method<sup><a reveal-id="DST035C18" data-open="DST035C18" class="link link-ref link-reveal xref-bibr">18</a></sup> was used for the relative quantification of the respective transcript expression. Expression values were normalized using an endogenous beta-actin.</p>                            <h3 scrollto-destination=4898845 id="4898845" class="section-title" data-legacy-id=s2e>2.5. Stimulation of immune response <em>in vitro</em> and <em>in vivo</em></h3>
<p><em>In vivo</em> immune response was stimulated by intraperitoneal injection of bacteria or lipopolysaccharide (LPS). <em>In vitro</em> stimulation were carried out on splenocytes from <em>B. maxima</em>. The detailed methods are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S8</a>.</p>                            <h2 scrollto-destination=4898847 id="4898847" class="section-title" data-legacy-id=s3>3. Results and Discussion</h2>
                            <h3 scrollto-destination=4898848 id="4898848" class="section-title" data-legacy-id=s3a>3.1. Sequencing and transcriptome assembly</h3>
<p>cDNAs prepared from skin and blood tissues of <em>B. maxima</em> were sequenced using Illumina HiSeq™ 2000. After sequencing, 59 820 004 and 58 492 250 raw reads was separately obtained from skin and blood tissue in <em>B. maxima</em>. Cleaning the low-quality reads, we acquired 55 533 592 (4 998 023 280 bp from skin tissue) and 54 967 354 (4 947 061 860 bp from blood tissue) clean reads (Table <a reveal-id="DST035TB1" data-open="DST035TB1" class="link link-reveal link-table xref-fig">1</a>). Using the Trinity program to assemble clean reads to contigs in <em>de novo</em>, a total of 172 119 contigs were separately generated from these two tissues. After contigs assembly, a total of 88 213 putative gene objects (all unigenes) were obtained ranging from 200 to 20 882 bp and 602 bp in average length. The size of unigenes &gt;500 bp was 27 661. The largest unigenes had 20 882 bp, and the N50 of unigenes was 868 bp (Table <a reveal-id="DST035TB1" data-open="DST035TB1" class="link link-reveal link-table xref-fig">1</a>). The length of all unigene sequences was &gt;200 bp and 41.0% (36 209) were &gt;499 bp in length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In term of sequence completeness, we used a less stringent but broadly adopted definition, considering a sequence to be full length if it comprises at least the complete CDS.<sup><a reveal-id="DST035C19" data-open="DST035C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Of the 88 213 (41.2%) unigenes, 36 319 had CDS matches with direction possessed significant similarity (cut-off <em>E</em>-value of 1e<sup>−6</sup>) with best-hit blast results. Of 88 213 (3.5%) CDS sequences, 3129 were produced by ESTScan program, which cannot to be aligned to known databases. Finally, a total of 39 448 unigene sequences had the complete CDS (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="4898850" scrollto-destination="4898850"></a>
<div content-id="DST035TB1" class="table-modal"><div class="table-wrap"><div id="DST035TB1" class="table-wrap-title" data-id="DST035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of sequencing, assembly and analysis of <em>B. maxima</em> transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Dataset name </th><th>All </th><th>Skin </th><th>Blood </th></tr></thead><tbody><tr><td>Total no. of bases (bp) </td><td>9 945 085 140 </td><td>4 998 023 280 </td><td>4 947 061 860 </td></tr><tr><td>Average read length (bp) </td><td>90 </td><td>90 </td><td>90 </td></tr><tr><td colspan="4">No. of reads </td></tr><tr><td> Raw reads </td><td>118 312 254 </td><td>59 820 004 </td><td>58 492 250 </td></tr><tr><td> Clean reads </td><td>110 500 946 </td><td>55 533 592 </td><td>54 967 354 </td></tr><tr><td> Q20<sup>a</sup> of clean reads </td><td>97.75% </td><td>97.82% </td><td>97.67% </td></tr><tr><td colspan="4">No. of contigs </td></tr><tr><td> Total contigs </td><td>348 906 </td><td>172 119 </td><td>176 787 </td></tr><tr><td> Average contig read length </td><td>275 </td><td>289 </td><td>260 </td></tr><tr><td colspan="4">No. of unigenes </td></tr><tr><td> Total unigenes </td><td>88 213 </td><td>87 931 </td><td>80 830 </td></tr><tr><td> Distinct clusters </td><td>33 619 </td><td>1762 </td><td>2016 </td></tr><tr><td> Distinct singletons </td><td>54 594 </td><td>86 819 </td><td>78 714 </td></tr><tr><td> N50<sup>b</sup> of unigenes </td><td>868 </td><td>710 </td><td>681 </td></tr><tr><td> Average unigene read length </td><td>602 </td><td>498 </td><td>481 </td></tr><tr><td> Largest unigene </td><td>20 882 </td><td>11 892 </td><td>20 882 </td></tr><tr><td> No. of large unigenes &gt;500 bp </td><td>27 661 </td><td>21 961 </td><td>18 509 </td></tr></tbody></table></div><div content-id="DST035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Dataset name </th><th>All </th><th>Skin </th><th>Blood </th></tr></thead><tbody><tr><td>Total no. of bases (bp) </td><td>9 945 085 140 </td><td>4 998 023 280 </td><td>4 947 061 860 </td></tr><tr><td>Average read length (bp) </td><td>90 </td><td>90 </td><td>90 </td></tr><tr><td colspan="4">No. of reads </td></tr><tr><td> Raw reads </td><td>118 312 254 </td><td>59 820 004 </td><td>58 492 250 </td></tr><tr><td> Clean reads </td><td>110 500 946 </td><td>55 533 592 </td><td>54 967 354 </td></tr><tr><td> Q20<sup>a</sup> of clean reads </td><td>97.75% </td><td>97.82% </td><td>97.67% </td></tr><tr><td colspan="4">No. of contigs </td></tr><tr><td> Total contigs </td><td>348 906 </td><td>172 119 </td><td>176 787 </td></tr><tr><td> Average contig read length </td><td>275 </td><td>289 </td><td>260 </td></tr><tr><td colspan="4">No. of unigenes </td></tr><tr><td> Total unigenes </td><td>88 213 </td><td>87 931 </td><td>80 830 </td></tr><tr><td> Distinct clusters </td><td>33 619 </td><td>1762 </td><td>2016 </td></tr><tr><td> Distinct singletons </td><td>54 594 </td><td>86 819 </td><td>78 714 </td></tr><tr><td> N50<sup>b</sup> of unigenes </td><td>868 </td><td>710 </td><td>681 </td></tr><tr><td> Average unigene read length </td><td>602 </td><td>498 </td><td>481 </td></tr><tr><td> Largest unigene </td><td>20 882 </td><td>11 892 </td><td>20 882 </td></tr><tr><td> No. of large unigenes &gt;500 bp </td><td>27 661 </td><td>21 961 </td><td>18 509 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Q20: percentage is the proportion of nucleotides with a quality value &gt;20 in reads.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>N50: unigene length-weighted median.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4898850" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST035TB1" class="table-wrap-title" data-id="DST035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of sequencing, assembly and analysis of <em>B. maxima</em> transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Dataset name </th><th>All </th><th>Skin </th><th>Blood </th></tr></thead><tbody><tr><td>Total no. of bases (bp) </td><td>9 945 085 140 </td><td>4 998 023 280 </td><td>4 947 061 860 </td></tr><tr><td>Average read length (bp) </td><td>90 </td><td>90 </td><td>90 </td></tr><tr><td colspan="4">No. of reads </td></tr><tr><td> Raw reads </td><td>118 312 254 </td><td>59 820 004 </td><td>58 492 250 </td></tr><tr><td> Clean reads </td><td>110 500 946 </td><td>55 533 592 </td><td>54 967 354 </td></tr><tr><td> Q20<sup>a</sup> of clean reads </td><td>97.75% </td><td>97.82% </td><td>97.67% </td></tr><tr><td colspan="4">No. of contigs </td></tr><tr><td> Total contigs </td><td>348 906 </td><td>172 119 </td><td>176 787 </td></tr><tr><td> Average contig read length </td><td>275 </td><td>289 </td><td>260 </td></tr><tr><td colspan="4">No. of unigenes </td></tr><tr><td> Total unigenes </td><td>88 213 </td><td>87 931 </td><td>80 830 </td></tr><tr><td> Distinct clusters </td><td>33 619 </td><td>1762 </td><td>2016 </td></tr><tr><td> Distinct singletons </td><td>54 594 </td><td>86 819 </td><td>78 714 </td></tr><tr><td> N50<sup>b</sup> of unigenes </td><td>868 </td><td>710 </td><td>681 </td></tr><tr><td> Average unigene read length </td><td>602 </td><td>498 </td><td>481 </td></tr><tr><td> Largest unigene </td><td>20 882 </td><td>11 892 </td><td>20 882 </td></tr><tr><td> No. of large unigenes &gt;500 bp </td><td>27 661 </td><td>21 961 </td><td>18 509 </td></tr></tbody></table></div><div content-id="DST035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Dataset name </th><th>All </th><th>Skin </th><th>Blood </th></tr></thead><tbody><tr><td>Total no. of bases (bp) </td><td>9 945 085 140 </td><td>4 998 023 280 </td><td>4 947 061 860 </td></tr><tr><td>Average read length (bp) </td><td>90 </td><td>90 </td><td>90 </td></tr><tr><td colspan="4">No. of reads </td></tr><tr><td> Raw reads </td><td>118 312 254 </td><td>59 820 004 </td><td>58 492 250 </td></tr><tr><td> Clean reads </td><td>110 500 946 </td><td>55 533 592 </td><td>54 967 354 </td></tr><tr><td> Q20<sup>a</sup> of clean reads </td><td>97.75% </td><td>97.82% </td><td>97.67% </td></tr><tr><td colspan="4">No. of contigs </td></tr><tr><td> Total contigs </td><td>348 906 </td><td>172 119 </td><td>176 787 </td></tr><tr><td> Average contig read length </td><td>275 </td><td>289 </td><td>260 </td></tr><tr><td colspan="4">No. of unigenes </td></tr><tr><td> Total unigenes </td><td>88 213 </td><td>87 931 </td><td>80 830 </td></tr><tr><td> Distinct clusters </td><td>33 619 </td><td>1762 </td><td>2016 </td></tr><tr><td> Distinct singletons </td><td>54 594 </td><td>86 819 </td><td>78 714 </td></tr><tr><td> N50<sup>b</sup> of unigenes </td><td>868 </td><td>710 </td><td>681 </td></tr><tr><td> Average unigene read length </td><td>602 </td><td>498 </td><td>481 </td></tr><tr><td> Largest unigene </td><td>20 882 </td><td>11 892 </td><td>20 882 </td></tr><tr><td> No. of large unigenes &gt;500 bp </td><td>27 661 </td><td>21 961 </td><td>18 509 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Q20: percentage is the proportion of nucleotides with a quality value &gt;20 in reads.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>N50: unigene length-weighted median.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4898850" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4898851 id="4898851" class="section-title" data-legacy-id=s3b>3.2. Function annotation</h3>
<p>Several complementary methods were used to annotate the assembled sequences. To begin with using BLASTX and BLASTN, the assembled sequences were searched against the Nr, Swiss-Prot protein databases and Nt nucleotides database (cut-off <em>E</em>-value of 1e<sup>−6</sup>). Of the 88 213 assembled sequences, 35 457 (Nr, 40.2%), 30 413 (Swiss-Prot, 34.5%) and 27 622 (Nt, 31.3%) had significant matches. The more detailed analysis of the blast top 30 hits is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. We compared the transcriptome with CDS with that of human (Ensemble data release 69) and <em>X. tropicalis</em> (Xenbase, Xentr7_2). The overlap between transcripts from the transcriptome of <em>Bombina</em> and <em>Xenopus</em> is larger than the others (Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>A). In total, above two times more transcripts of <em>B. maxima</em> have the orthologs in <em>X. tropicalis</em>, compared with human. However, the most of transcripts from <em>B. maxima</em> have no orthologs with human and <em>X. tropicalis</em>. Although ∼32% of transcripts from <em>B. maxima</em> have its orthologs in <em>Xenopus</em>, obviously most of the transcripts were identified to be species-specific transcripts or novel-spliced transcripts of this frog and Xenopus. To determine the expression level of the transcripts in skin and blood tissues, we mapped the raw reads to the assembled sequences. The reads per kilobase per million (RPKM) values of the transcripts are shown in Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>B. It depicts that gene with high expression has distinct biases for skin, compared with the blood. </p>                        <a id="4898853" scrollto-destination="4898853"></a>
<div id="DST035F1" data-id="DST035F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03501.png?Expires=2147483647&Signature=Zme~N0g6olT-XymnZrdHDBjIvlK25-lhVB45Z6l2P1Kwa6A3kRVgNz9zBrTKFWiU1IsMspNqf53O9sz8yRQd-TFD3Sz61vcPEDhtEnTYX~BLTfeUaqPIe72jq~2nhME-QCIFoXxuYzCg8mn4BaLRPSP-gosbSl3PmPKR0M1PRbcENyimwNDXRb48hrKRTu7robRd0KQ1kWi2vfwXRVdXrs5vv8IeLgT8~Lxf3-0ZmRIK7zHV1wXojkB~ef3qBNmz3AhkcAi4HscBvYBsrs3KHfuuzDcY6kpkT240TIH4UCcdmw1jQbybuG~r~aNHpABqmnCh5UVLJ4VjohFx6gs5Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional profiles of the transcriptome. (A) The venn diagram shows the comparison of the transcriptome between B. maxima, X. laevis and human. The data of B. maxima are composed of transcripts with CDS from the transcriptome. The transcriptome of X. laevis and human is including of all cDNA sequences. (B) All transcripts expression of skin and blood tissues in B. maxima. (C) Tissue-specific genes were identified in the transcriptome. It shows the overlap of skin- and blood-specific transcripts that passed strict filters. (D) The expression patterns of skin- and blood-specific genes under different reads per kilobase per million (RPKM) values. (E) The novel ncRNAs were revealed from RNA-seq data. A breakdown of all the transcripts into protein-coding, putative ncRNAs, and definite ncRNAs. (F) Heatmap showing distinct profiles of all definite ncRNAs was detected in the transcriptome. The RPKM values from skin and blood tissues were used for mean-centering, normalization and clustering." path-from-xml="dst03501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898853/dst03501.tiff" path-from-xml="dst03501.tiff" target="_blank">View large</a><a section="4898853" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03501.png?Expires=2147483647&Signature=plVTSfhS5cB9wyF8XmQp6MMAkMs~M9nHeN6FNunCV-SkQSHIIiPFcMGEV7gFFviwt-8Q2g04VHQBa7r7PrXWPRGOP1yJ61~QJOsupmDx3QDwzW0bk3b6VDDpqk9hx5oTh4fliRv0AAxj3o2jtWAyhgPAcWcuLx3Iy9-JyIyK3z9RhNCzEAftZOdA9p7ZNr7Z8a3C4mXAxN1KhehtoAh-gS2azArsljEh9Ioeb2RKuu379tW9DdhJMwBnPRUPtf4Pb1GvrlBBE6rWQA9AS-ETI7-1b~U15oTVyVSNxhMVQHrHik~tMLj0HnsTsonlDqgPU~~xXM3jGCfkxTVnTxY86w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898853&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional profiles of the transcriptome. (A) The venn diagram shows the comparison of the transcriptome between <em>B. maxima</em>, <em>X. laevis</em> and human. The data of <em>B. maxima</em> are composed of transcripts with CDS from the transcriptome. The transcriptome of <em>X. laevis</em> and human is including of all cDNA sequences. (B) All transcripts expression of skin and blood tissues in <em>B. maxima</em>. (C) Tissue-specific genes were identified in the transcriptome. It shows the overlap of skin- and blood-specific transcripts that passed strict filters. (D) The expression patterns of skin- and blood-specific genes under different reads per kilobase per million (RPKM) values. (E) The novel ncRNAs were revealed from RNA-seq data. A breakdown of all the transcripts into protein-coding, putative ncRNAs, and definite ncRNAs. (F) Heatmap showing distinct profiles of all definite ncRNAs was detected in the transcriptome. The RPKM values from skin and blood tissues were used for mean-centering, normalization and clustering.</p></div></div><div content-id="DST035F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03501.png?Expires=2147483647&Signature=Zme~N0g6olT-XymnZrdHDBjIvlK25-lhVB45Z6l2P1Kwa6A3kRVgNz9zBrTKFWiU1IsMspNqf53O9sz8yRQd-TFD3Sz61vcPEDhtEnTYX~BLTfeUaqPIe72jq~2nhME-QCIFoXxuYzCg8mn4BaLRPSP-gosbSl3PmPKR0M1PRbcENyimwNDXRb48hrKRTu7robRd0KQ1kWi2vfwXRVdXrs5vv8IeLgT8~Lxf3-0ZmRIK7zHV1wXojkB~ef3qBNmz3AhkcAi4HscBvYBsrs3KHfuuzDcY6kpkT240TIH4UCcdmw1jQbybuG~r~aNHpABqmnCh5UVLJ4VjohFx6gs5Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional profiles of the transcriptome. (A) The venn diagram shows the comparison of the transcriptome between B. maxima, X. laevis and human. The data of B. maxima are composed of transcripts with CDS from the transcriptome. The transcriptome of X. laevis and human is including of all cDNA sequences. (B) All transcripts expression of skin and blood tissues in B. maxima. (C) Tissue-specific genes were identified in the transcriptome. It shows the overlap of skin- and blood-specific transcripts that passed strict filters. (D) The expression patterns of skin- and blood-specific genes under different reads per kilobase per million (RPKM) values. (E) The novel ncRNAs were revealed from RNA-seq data. A breakdown of all the transcripts into protein-coding, putative ncRNAs, and definite ncRNAs. (F) Heatmap showing distinct profiles of all definite ncRNAs was detected in the transcriptome. The RPKM values from skin and blood tissues were used for mean-centering, normalization and clustering." path-from-xml="dst03501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898853/dst03501.tiff" path-from-xml="dst03501.tiff" target="_blank">View large</a><a section="4898853" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03501.png?Expires=2147483647&Signature=plVTSfhS5cB9wyF8XmQp6MMAkMs~M9nHeN6FNunCV-SkQSHIIiPFcMGEV7gFFviwt-8Q2g04VHQBa7r7PrXWPRGOP1yJ61~QJOsupmDx3QDwzW0bk3b6VDDpqk9hx5oTh4fliRv0AAxj3o2jtWAyhgPAcWcuLx3Iy9-JyIyK3z9RhNCzEAftZOdA9p7ZNr7Z8a3C4mXAxN1KhehtoAh-gS2azArsljEh9Ioeb2RKuu379tW9DdhJMwBnPRUPtf4Pb1GvrlBBE6rWQA9AS-ETI7-1b~U15oTVyVSNxhMVQHrHik~tMLj0HnsTsonlDqgPU~~xXM3jGCfkxTVnTxY86w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898853&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional profiles of the transcriptome. (A) The venn diagram shows the comparison of the transcriptome between <em>B. maxima</em>, <em>X. laevis</em> and human. The data of <em>B. maxima</em> are composed of transcripts with CDS from the transcriptome. The transcriptome of <em>X. laevis</em> and human is including of all cDNA sequences. (B) All transcripts expression of skin and blood tissues in <em>B. maxima</em>. (C) Tissue-specific genes were identified in the transcriptome. It shows the overlap of skin- and blood-specific transcripts that passed strict filters. (D) The expression patterns of skin- and blood-specific genes under different reads per kilobase per million (RPKM) values. (E) The novel ncRNAs were revealed from RNA-seq data. A breakdown of all the transcripts into protein-coding, putative ncRNAs, and definite ncRNAs. (F) Heatmap showing distinct profiles of all definite ncRNAs was detected in the transcriptome. The RPKM values from skin and blood tissues were used for mean-centering, normalization and clustering.</p></div></div><p>The unigene sequences that had matches in Nr databases were given GO annotations with Uniprot databases. Of these, 11 905 (33.6%) were assigned to one or more 3423 GO terms. As reported in other <em>Xenopus</em> transcriptome studies,<sup><a reveal-id="DST035C3" data-open="DST035C3" class="link link-ref link-reveal xref-bibr">3</a></sup> the largest proportion of transcripts fell into 2–3 broad and general categories as follows: ‘Biological Process’, ‘Molecular Function’ and ‘Cellular Component’ (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). These results were similarly found in <em>Mytilus edulis</em>,<sup><a reveal-id="DST035C20" data-open="DST035C20" class="link link-ref link-reveal xref-bibr">20</a></sup><em>Hypophthalmichthys molitrix</em>,<sup><a reveal-id="DST035C21" data-open="DST035C21" class="link link-ref link-reveal xref-bibr">21</a></sup><em>Sus scrofa</em><sup><a reveal-id="DST035C22" data-open="DST035C22" class="link link-ref link-reveal xref-bibr">22</a></sup>. As our expectation, a great amount of transcripts were involved in localization process (2383, 20.02%) and development process (2204, 18.51%), corresponding to the results of <em>X. tropicalis</em> transcriptome.<sup><a reveal-id="DST035C17" data-open="DST035C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Meanwhile, annotation of the 88 213 unigene sequences using Clusters of Orthologous Groups (COG) of protein databases generated the results for 9625 putative proteins. The COG annotated putative proteins ranged functionally into at least 25 molecular families, including cell cycle control, cell membrane biogenesis, immune defence and signal transduction (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>), in accordance with the categories of GO annotation described above.</p><p>To further analyse the tissue-specific genes, the strict filter was performed as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary methods</a>. Finally, 12 608 (14.3%) and 6771 (7.7%) unigenes were identified to be skin-specific and blood-specific genes (Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>C). Compared with blood-specific genes, the skin-specific genes have a large proportion of high-expressed genes (Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>D), 70 skin-specific genes whose RPKM values were &gt;50. As our expectation, most of them were related to epithelial cell growth, wound healing and AMPs. One of high-expressed AMP—Maximin S2, isolated from skin secretion in our previous work, was identified to be as the skin-specific gene. Besides the Maximin S2, a fungicide antimicrobial peptide, Holotricin, was also a high-expressed skin-specific protein, which was not isolated from skin secretions in our previous work. In the high-expressed skin-specific genes, more and more epithelial cells' components were identified with highest RPKM values, such as <em>WAP</em>, <em>Desmoglein-1</em>. Our results responses the fact that this frog can be lived well in the harsh environment.</p><p>Next, we examined the unannotated transcripts (47 582, 53.9% of all transcripts) for protein-coding transcripts potential, leaving 47 112 transcripts as potential ncRNAs. As methods of <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary information S5</a>, we finally identified 1183 ncRNAs (Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>E), which possess at least 70% sequence identity to known non-coding genes. These ncRNAs serve as the definite ncRNAs in <em>B. maxima</em>. Clustering analysis reveals that these genes can be broadly divided into three clusters (Fig. <a reveal-id="DST035F1" data-open="DST035F1" class="link link-reveal link-table xref-fig">1</a>F). The first cluster corresponds to the transcripts whose expression levels had great variations between skin and blood tissue. The second cluster contains the genes whose high expressions were gradually increased over skin to blood. It indicates that the ncRNAs play important roles in skin and blood. The third cluster appears to contain the genes that are sharply degraded with high expression in blood, compared with skin. Not all ncRNAs derived from skin and blood tissues, we mapped the raw reads to ncRNAs transcripts and found that there were 146 (12.3%) skin-specific and 46 (3.9%) blood-specific ncRNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Collectively, our analysis has identified with confidence a total of 47 112 potential ncRNAs and 1183 definite ncRNAs. We observed that the majority of these ncRNAs appears to be lowly expressed, with 69.0% of them (or 815 transcripts) having maximum RPKM of &lt;5. In addition, the non-coding transcripts that we found in this study appear to be performing tissue-specific function. This collection of ncRNAs provides a rich resource for future functional studies, which will greatly enhance our understanding the genetic adaption of the frog.</p>                            <h3 scrollto-destination=4898857 id="4898857" class="section-title" data-legacy-id=s3c>3.3. KEGG pathways analysis</h3>
<p>A total of 24 579 assembled sequences were identified to occur in 242 predicted KEGG pathways. The number of sequences ranged from 2 to 2722. The top 20 pathways with the highest number of sequences are shown in Table <a reveal-id="DST035TB2" data-open="DST035TB2" class="link link-reveal link-table xref-fig">2</a>, and the maximum of transcripts was found in the metabolic pathways (<em>n</em> = 2722). Metabolic pathways, implicated the intracellular metabolic pathways control immune tolerance and regulation of stem cell behaviour and life span,<sup><a reveal-id="DST035C23" data-open="DST035C23" class="link link-ref link-reveal xref-bibr">23</a></sup> showed the highest number of transcripts here and corresponded to the results of GO annotation described above and published transcriptome report.<sup><a reveal-id="DST035C21" data-open="DST035C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST035C22" data-open="DST035C22" class="link link-ref link-reveal xref-bibr">22</a></sup> As the previous report of <em>X. tropicalis</em> genome, 7% of protein-coding gene orthologs were identified to be human disease genes.<sup><a reveal-id="DST035C4" data-open="DST035C4" class="link link-ref link-reveal xref-bibr">4</a></sup> We also identified 11.17% (2745) of transcripts related disease using KEGG analysis, related disease including Huntington's disease, Alzheimer's disease (<em>n</em> = 619), Chagas disease (American trypanosomiasis, <em>n</em> = 280), Parkinson's disease (191), Autoimmune thyroid disease (<em>n</em> = 131), Graft-versus-host disease (<em>n</em> = 121), Prion diseases (<em>n</em> = 116). Remarkably, 2693 (10.95% of 24 579 transcripts annotated by KEGG analysis) transcripts were identified to be related to cancer pathways and specified cancer (small-cell lung cancer, prostate cancer, endometrial cancer, non-small-cell lung cancer, pancreatic cancer, colorectal cancer, bladder cancer and thyroid cancer). This analysis and annotation may be useful for further study and great interest for other researchers. </p>                        <a id="4898859" scrollto-destination="4898859"></a>
<div content-id="DST035TB2" class="table-modal"><div class="table-wrap"><div id="DST035TB2" class="table-wrap-title" data-id="DST035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of top 20 pathways with highest gene numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathway </th><th>All genes with pathway annotation (<em>N</em> = 24 579), <em>n</em> (%) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>2722 (11.07) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Huntington's disease </td><td>1287 (5.24) </td><td>ko05016 </td></tr><tr><td>3 </td><td>Ubiquitin-mediated proteolysis </td><td>1223 (4.98) </td><td>ko04120 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>1138 (4.63) </td><td>ko00230 </td></tr><tr><td>5 </td><td>Pathways in cancer </td><td>1073 (4.37) </td><td>ko05200 </td></tr><tr><td>6 </td><td>Focal adhesion </td><td>1014 (4.13) </td><td>ko04510 </td></tr><tr><td>7 </td><td>Regulation of actin cytoskeleton </td><td>946 (3.85) </td><td>ko04810 </td></tr><tr><td>8 </td><td>Calcium signalling pathway </td><td>821 (3.34) </td><td>ko04020 </td></tr><tr><td>9 </td><td>Influenza A </td><td>817 (3.32) </td><td>ko05164 </td></tr><tr><td>10 </td><td>Lysine degradation </td><td>794 (3.23) </td><td>ko00310 </td></tr><tr><td>11 </td><td>Protein processing in endoplasmic reticulum </td><td>706 (2.87) </td><td>ko04141 </td></tr><tr><td>12 </td><td>MAPK signalling pathway </td><td>683 (2.78) </td><td>ko04010 </td></tr><tr><td>13 </td><td>Measles </td><td>674 (2.74) </td><td>ko05162 </td></tr><tr><td>14 </td><td>Endocytosis </td><td>654 (2.66) </td><td>ko04144 </td></tr><tr><td>15 </td><td>Alzheimer's disease </td><td>619 (2.52) </td><td>ko05010 </td></tr><tr><td>16 </td><td>Hepatitis C </td><td>610 (2.48) </td><td>ko05160 </td></tr><tr><td>17 </td><td>Basal transcription factors </td><td>601 (2.45) </td><td>ko03022 </td></tr><tr><td>18 </td><td>Vascular smooth muscle contraction </td><td>598 (2.43) </td><td>ko04270 </td></tr><tr><td>19 </td><td>Ribosome biogenesis in eukaryotes </td><td>592 (2.41) </td><td>ko03008 </td></tr><tr><td>20 </td><td>Tight junction </td><td>550 (2.24) </td><td>ko04530 </td></tr></tbody></table></div><div content-id="DST035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathway </th><th>All genes with pathway annotation (<em>N</em> = 24 579), <em>n</em> (%) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>2722 (11.07) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Huntington's disease </td><td>1287 (5.24) </td><td>ko05016 </td></tr><tr><td>3 </td><td>Ubiquitin-mediated proteolysis </td><td>1223 (4.98) </td><td>ko04120 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>1138 (4.63) </td><td>ko00230 </td></tr><tr><td>5 </td><td>Pathways in cancer </td><td>1073 (4.37) </td><td>ko05200 </td></tr><tr><td>6 </td><td>Focal adhesion </td><td>1014 (4.13) </td><td>ko04510 </td></tr><tr><td>7 </td><td>Regulation of actin cytoskeleton </td><td>946 (3.85) </td><td>ko04810 </td></tr><tr><td>8 </td><td>Calcium signalling pathway </td><td>821 (3.34) </td><td>ko04020 </td></tr><tr><td>9 </td><td>Influenza A </td><td>817 (3.32) </td><td>ko05164 </td></tr><tr><td>10 </td><td>Lysine degradation </td><td>794 (3.23) </td><td>ko00310 </td></tr><tr><td>11 </td><td>Protein processing in endoplasmic reticulum </td><td>706 (2.87) </td><td>ko04141 </td></tr><tr><td>12 </td><td>MAPK signalling pathway </td><td>683 (2.78) </td><td>ko04010 </td></tr><tr><td>13 </td><td>Measles </td><td>674 (2.74) </td><td>ko05162 </td></tr><tr><td>14 </td><td>Endocytosis </td><td>654 (2.66) </td><td>ko04144 </td></tr><tr><td>15 </td><td>Alzheimer's disease </td><td>619 (2.52) </td><td>ko05010 </td></tr><tr><td>16 </td><td>Hepatitis C </td><td>610 (2.48) </td><td>ko05160 </td></tr><tr><td>17 </td><td>Basal transcription factors </td><td>601 (2.45) </td><td>ko03022 </td></tr><tr><td>18 </td><td>Vascular smooth muscle contraction </td><td>598 (2.43) </td><td>ko04270 </td></tr><tr><td>19 </td><td>Ribosome biogenesis in eukaryotes </td><td>592 (2.41) </td><td>ko03008 </td></tr><tr><td>20 </td><td>Tight junction </td><td>550 (2.24) </td><td>ko04530 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4898859" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST035TB2" class="table-wrap-title" data-id="DST035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of top 20 pathways with highest gene numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathway </th><th>All genes with pathway annotation (<em>N</em> = 24 579), <em>n</em> (%) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>2722 (11.07) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Huntington's disease </td><td>1287 (5.24) </td><td>ko05016 </td></tr><tr><td>3 </td><td>Ubiquitin-mediated proteolysis </td><td>1223 (4.98) </td><td>ko04120 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>1138 (4.63) </td><td>ko00230 </td></tr><tr><td>5 </td><td>Pathways in cancer </td><td>1073 (4.37) </td><td>ko05200 </td></tr><tr><td>6 </td><td>Focal adhesion </td><td>1014 (4.13) </td><td>ko04510 </td></tr><tr><td>7 </td><td>Regulation of actin cytoskeleton </td><td>946 (3.85) </td><td>ko04810 </td></tr><tr><td>8 </td><td>Calcium signalling pathway </td><td>821 (3.34) </td><td>ko04020 </td></tr><tr><td>9 </td><td>Influenza A </td><td>817 (3.32) </td><td>ko05164 </td></tr><tr><td>10 </td><td>Lysine degradation </td><td>794 (3.23) </td><td>ko00310 </td></tr><tr><td>11 </td><td>Protein processing in endoplasmic reticulum </td><td>706 (2.87) </td><td>ko04141 </td></tr><tr><td>12 </td><td>MAPK signalling pathway </td><td>683 (2.78) </td><td>ko04010 </td></tr><tr><td>13 </td><td>Measles </td><td>674 (2.74) </td><td>ko05162 </td></tr><tr><td>14 </td><td>Endocytosis </td><td>654 (2.66) </td><td>ko04144 </td></tr><tr><td>15 </td><td>Alzheimer's disease </td><td>619 (2.52) </td><td>ko05010 </td></tr><tr><td>16 </td><td>Hepatitis C </td><td>610 (2.48) </td><td>ko05160 </td></tr><tr><td>17 </td><td>Basal transcription factors </td><td>601 (2.45) </td><td>ko03022 </td></tr><tr><td>18 </td><td>Vascular smooth muscle contraction </td><td>598 (2.43) </td><td>ko04270 </td></tr><tr><td>19 </td><td>Ribosome biogenesis in eukaryotes </td><td>592 (2.41) </td><td>ko03008 </td></tr><tr><td>20 </td><td>Tight junction </td><td>550 (2.24) </td><td>ko04530 </td></tr></tbody></table></div><div content-id="DST035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathway </th><th>All genes with pathway annotation (<em>N</em> = 24 579), <em>n</em> (%) </th><th>Pathway ID </th></tr></thead><tbody><tr><td>1 </td><td>Metabolic pathways </td><td>2722 (11.07) </td><td>ko01100 </td></tr><tr><td>2 </td><td>Huntington's disease </td><td>1287 (5.24) </td><td>ko05016 </td></tr><tr><td>3 </td><td>Ubiquitin-mediated proteolysis </td><td>1223 (4.98) </td><td>ko04120 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>1138 (4.63) </td><td>ko00230 </td></tr><tr><td>5 </td><td>Pathways in cancer </td><td>1073 (4.37) </td><td>ko05200 </td></tr><tr><td>6 </td><td>Focal adhesion </td><td>1014 (4.13) </td><td>ko04510 </td></tr><tr><td>7 </td><td>Regulation of actin cytoskeleton </td><td>946 (3.85) </td><td>ko04810 </td></tr><tr><td>8 </td><td>Calcium signalling pathway </td><td>821 (3.34) </td><td>ko04020 </td></tr><tr><td>9 </td><td>Influenza A </td><td>817 (3.32) </td><td>ko05164 </td></tr><tr><td>10 </td><td>Lysine degradation </td><td>794 (3.23) </td><td>ko00310 </td></tr><tr><td>11 </td><td>Protein processing in endoplasmic reticulum </td><td>706 (2.87) </td><td>ko04141 </td></tr><tr><td>12 </td><td>MAPK signalling pathway </td><td>683 (2.78) </td><td>ko04010 </td></tr><tr><td>13 </td><td>Measles </td><td>674 (2.74) </td><td>ko05162 </td></tr><tr><td>14 </td><td>Endocytosis </td><td>654 (2.66) </td><td>ko04144 </td></tr><tr><td>15 </td><td>Alzheimer's disease </td><td>619 (2.52) </td><td>ko05010 </td></tr><tr><td>16 </td><td>Hepatitis C </td><td>610 (2.48) </td><td>ko05160 </td></tr><tr><td>17 </td><td>Basal transcription factors </td><td>601 (2.45) </td><td>ko03022 </td></tr><tr><td>18 </td><td>Vascular smooth muscle contraction </td><td>598 (2.43) </td><td>ko04270 </td></tr><tr><td>19 </td><td>Ribosome biogenesis in eukaryotes </td><td>592 (2.41) </td><td>ko03008 </td></tr><tr><td>20 </td><td>Tight junction </td><td>550 (2.24) </td><td>ko04530 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4898859" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4898860 id="4898860" class="section-title" data-legacy-id=s3d>3.4. Protein domains prediction</h3>
<p>InterProScan identified 55 758 protein domains (6912 types) among 22 241 unigene sequences. The protein domains abundantly occurred in immunoglobulin-related domains (<em>n</em> = 1802), Zinc finger, C2H2 (<em>n</em> = 864; IPR007087), followed by reverse transcriptase (<em>n</em> = 802; IPR000477), Zinc finger, C2H2-like (<em>n</em> = 748; IPR015880), Zinc finger, C2H2-type/integrase, DNA binding (<em>n</em> = 709; IPR013087) and Zinc finger, RING/FYVE/PHD type (<em>n</em> = 387; IPR013083) (Fig. <a reveal-id="DST035F2" data-open="DST035F2" class="link link-reveal link-table xref-fig">2</a>). A total of 4416 transcripts (5.0% of the transcriptome) with protein domains including Zinc finger keyword were identified, which comprise 80 types of protein domain. Usually, the most common DNA-binding motif and transcription factors contain one or more capable of making sequence-specific recognition of DNA and also binding to RNA and protein targets.<sup><a reveal-id="DST035C24" data-open="DST035C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Comparing with other species' transcriptome or genome,<sup><a reveal-id="DST035C25" data-open="DST035C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DST035C26" data-open="DST035C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Zinc finger protein domain was enriched in all scanned protein domains, such as 3.3% in <em>Macrobrachium</em><em>rosenbergii</em> transcriptome and 2.6% in the mouse transcriptome. </p>                        <a id="4898862" scrollto-destination="4898862"></a>
<div id="DST035F2" data-id="DST035F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03502.png?Expires=2147483647&Signature=YYOkXlGqNvLXcLkZraPLtgqsEg0NnFxA21tXBQ0u6w4gGz6muKw4yAJBdRHMsJ47TCsL-RjihRMOEv2G2zNdI~DlA5Q4PGQ7ZL3rOCEUiKReFeSmJioYWDVVXODvQT5FnUK14LZIiVbLO5K0RfrxTkctZFlQRnKsikLV~Ur1zRGjHOooG5bI18Bu-wqi7XUW5Ub9tnEe1RpTHyjqgZ69D3nWPKGxqZGe9oqV8kqYq3uSszKxY88a3wTj5c7qVaXwhk-6HHsHqXagbz~SnObFPmsQNfFfYIh-dgnjQOnm3qspkWSRZRvlJ5cE2dsVde~nItHPSJ6d9~dD0NPKalZB9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of protein domains predicted in B. maxima unigenes. The protein domain prediction was obtained by the InterProScan software. Remarkably, the sum of immunoglobulin-related domain is higher than others." path-from-xml="dst03502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898862/dst03502.tiff" path-from-xml="dst03502.tiff" target="_blank">View large</a><a section="4898862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03502.png?Expires=2147483647&Signature=BUR~sWhwQ8AhcLYkM2iqol1dtgo4xweoagKycFLzub2-qyfCAf1ude7Y-DvQvNCOkjLaiRkjDABhCN0bWV33mdV8Bd1TdlU7qwiOwk7Cxzqakpu6MTsNAVfKLdK02Km99wTKAQ1RBWsXaOv7qE5r3dVcRZMMpGRcH7XHfRaqHwuNTcdc0QwqFbJ5LdjG-o005eW38OidGeDU6hFU3Bjd~dWifrxJ8mmn2xr5ngz7ltKBWsKAbzpNfz5RYWTr~t9u7KNEvLUiVEePHZuOdUpMNQhQPQ06djm34S0oizMGJX5LdsVTrZ~Nm09hCjwbLLPNwSgxx03sd58FaWewIE~moA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898862&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of protein domains predicted in <em>B. maxima</em> unigenes. The protein domain prediction was obtained by the InterProScan software. Remarkably, the sum of immunoglobulin-related domain is higher than others.</p></div></div><div content-id="DST035F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03502.png?Expires=2147483647&Signature=YYOkXlGqNvLXcLkZraPLtgqsEg0NnFxA21tXBQ0u6w4gGz6muKw4yAJBdRHMsJ47TCsL-RjihRMOEv2G2zNdI~DlA5Q4PGQ7ZL3rOCEUiKReFeSmJioYWDVVXODvQT5FnUK14LZIiVbLO5K0RfrxTkctZFlQRnKsikLV~Ur1zRGjHOooG5bI18Bu-wqi7XUW5Ub9tnEe1RpTHyjqgZ69D3nWPKGxqZGe9oqV8kqYq3uSszKxY88a3wTj5c7qVaXwhk-6HHsHqXagbz~SnObFPmsQNfFfYIh-dgnjQOnm3qspkWSRZRvlJ5cE2dsVde~nItHPSJ6d9~dD0NPKalZB9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of protein domains predicted in B. maxima unigenes. The protein domain prediction was obtained by the InterProScan software. Remarkably, the sum of immunoglobulin-related domain is higher than others." path-from-xml="dst03502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898862/dst03502.tiff" path-from-xml="dst03502.tiff" target="_blank">View large</a><a section="4898862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03502.png?Expires=2147483647&Signature=BUR~sWhwQ8AhcLYkM2iqol1dtgo4xweoagKycFLzub2-qyfCAf1ude7Y-DvQvNCOkjLaiRkjDABhCN0bWV33mdV8Bd1TdlU7qwiOwk7Cxzqakpu6MTsNAVfKLdK02Km99wTKAQ1RBWsXaOv7qE5r3dVcRZMMpGRcH7XHfRaqHwuNTcdc0QwqFbJ5LdjG-o005eW38OidGeDU6hFU3Bjd~dWifrxJ8mmn2xr5ngz7ltKBWsKAbzpNfz5RYWTr~t9u7KNEvLUiVEePHZuOdUpMNQhQPQ06djm34S0oizMGJX5LdsVTrZ~Nm09hCjwbLLPNwSgxx03sd58FaWewIE~moA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898862&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of protein domains predicted in <em>B. maxima</em> unigenes. The protein domain prediction was obtained by the InterProScan software. Remarkably, the sum of immunoglobulin-related domain is higher than others.</p></div></div><p>Among the immunoglobulin-related domains, a total of 653 immunoglobulin (Ig)-like-fold domains (IPR013783) and 318 immunoglobulin-like domains (IPR007110) were predicted (Fig. <a reveal-id="DST035F2" data-open="DST035F2" class="link link-reveal link-table xref-fig">2</a>). Ig-like-fold domains are involved in a variety of functions playing essential roles in the vertebrate immune response and many other processes, involved with protein–protein interactions with their β-sheets. Ig-like-related domains were also found in other published species' transcriptome. However, its proportion in <em>M. rosenbergii</em> (0.9% of Ig-like-fold domains in 8411 detected contigs) was much less than that in <em>B. maxima</em> (2.9% of 22 241 input sequences). Moreover, the immunoglobulin provides further links between vertebrate immune systems. Interestingly, we found a high number of repeating units ending with Trp-Asp (WD)-containing domains (4.6% of the CDS), including WD40/YVTN repeat-like-containing domains (IPR015943; <em>n</em> = 423), WD40-repeat-like-containing domains (IPR011046; <em>n</em> = 303), WD40-repeat subgroup (IPR019781; <em>n</em> = 300), WD40-repeat domains (IPR001680; <em>n</em> = 281), WD40-repeat-containing domains (IPR017986; <em>n</em> = 267) and WD40-repeat 2 (IPR019782; <em>n</em> = 254) domains. In contrast to published domain prediction data, 2.1% of the transcripts with WD40-repeat domains in the <em>M. rosenbergii</em> transcriptome. These domains are involved in functions ranging from signal transduction transcription regulation to cell cycle control and apoptosis.<sup><a reveal-id="DST035C27" data-open="DST035C27" class="link link-ref link-reveal xref-bibr">27</a></sup> In <em>Xenopus</em>, the WD40-repeat domains were present in <em>wdr36</em>, <em>traf51</em> and <em>pik3r4</em> related to transcriptional regulation and epigenetic regulation in the latest report.<sup><a reveal-id="DST035C17" data-open="DST035C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=4898864 id="4898864" class="section-title" data-legacy-id=s3e>3.5. Genes related to the innate immune system</h3>
<p>Innate immune system (IIS) is the ancient defence system and first defence line of multicellular organisms against various infectious agents. We identified 1501transcripts (570 genes) involved in the immune system process. Of these, a total of 32 transcripts and at least 11 different Toll-like receptors (TLRs) were identified in <em>B. maxima</em>, including TLR1, TLR2, TLR3, TLR4, TLR6, TLR7, TLR8 and Toll/interleukin-1 receptor (Fig. <a reveal-id="DST035F3" data-open="DST035F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The TLRs signalling pathway is primarily dependent on TLR2 and TLR4. In humans, TLR2 binds LPS in the presence of LPS-binding protein (LBP) and CD14, and then induces nuclear factor (NF)-κB activation. In <em>B. maxima</em>, lipoprotein and peptidoglycan can induce the TLR signalling pathway in <em>B. maxima</em>, indicated by the predicted pathway map (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>A). The expression of several key molecules activated by the signalling pathway was validated in <em>B. maxima</em> (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). We detected the expression of TLR2 and TLR4 using qRT-PCR, and then found that TLR2 and TLR4 show a peak expression in spleen, kidney, stomach and liver (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>D). In digital expression genes analysis (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>C), the clustered expression of genes in skin and blood was divided into major three clusters. The first cluster contains genes whose expression in skin and blood keep in the medium level, comparing with others. Most of them are downstream genes of the signalling pathway. The second cluster corresponds to genes whose expression in skin was higher than that in blood except for its first branch. The third cluster contains genes with highest expression in the TLR pathway of <em>B. maxima</em>. </p>                        <a id="4898866" scrollto-destination="4898866"></a>
<div id="DST035F3" data-id="DST035F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03503.png?Expires=2147483647&Signature=R4u3Sf1ygtFmeZZvYoM8TRM5diwNJ9drd3jSSXXAqot34pY0eosqElrOC0Waiq9aYRfKylMAwTcTlyxJxHIyLnO7hw1oqBCVha7yg4oBB6zEMD30~LFYQmeT3AQXRX1ZgrrhnYiLXssu-rYkliY~WmlDy5Sx-ps38F44LTq3XTK6m2Zt~Y6upy9blb2IoX6njGaLUAeBDB1BlK9NPx7zD~Cxu-gUl-X3kBdT4Ek4s3YrGIXvccRZBcTj-~Dgw1kmc~fe6lCZKGydkdm2DjjNuT2eBQIiO2RNtLKRWksAFu6PMYObyNdeBG06hv~WximcfB1KjOtEz-ff2nuS2kFujA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The putative genes related to the immune system in B. maxima. Bars with dots represent the number of transcripts with the annotated functions; diamond bars represent the number of different genes (transcripts &lt;10 are not shown)." path-from-xml="dst03503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898866/dst03503.tiff" path-from-xml="dst03503.tiff" target="_blank">View large</a><a section="4898866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03503.png?Expires=2147483647&Signature=gBlF9cpFwnYsZd1hDAnfJK2WANfci3oQfn8DgD2pNxOo592t22nbZKmXkhUCaZeV9fKPoArNBo~p9QoTVTEtaxIwO12FKBF~5PHrRl6OETVOYG7jnp-wSpHE7krPjvzno5n5suBHb7MrlA2-gUxUcdo35daKJQ1iK-bcoK5R3L7U3BWg3CXrUR230c89NJGXunuaXZyroyYNWX~RwYR1pQgVkKApdXCsWqte1G7OjPysd0VlAQ3eIYOOqHumr1YQPIkcNVuffQwzsuJ~1WLemkFFi2CkiP6Snspd4hCgIg08Uhpi82l4greh6UetqrNr2ouESrdQETL~FuVV-B5tNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898866&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The putative genes related to the immune system in <em>B. maxima</em>. Bars with dots represent the number of transcripts with the annotated functions; diamond bars represent the number of different genes (transcripts &lt;10 are not shown).</p></div></div><div content-id="DST035F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03503.png?Expires=2147483647&Signature=R4u3Sf1ygtFmeZZvYoM8TRM5diwNJ9drd3jSSXXAqot34pY0eosqElrOC0Waiq9aYRfKylMAwTcTlyxJxHIyLnO7hw1oqBCVha7yg4oBB6zEMD30~LFYQmeT3AQXRX1ZgrrhnYiLXssu-rYkliY~WmlDy5Sx-ps38F44LTq3XTK6m2Zt~Y6upy9blb2IoX6njGaLUAeBDB1BlK9NPx7zD~Cxu-gUl-X3kBdT4Ek4s3YrGIXvccRZBcTj-~Dgw1kmc~fe6lCZKGydkdm2DjjNuT2eBQIiO2RNtLKRWksAFu6PMYObyNdeBG06hv~WximcfB1KjOtEz-ff2nuS2kFujA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The putative genes related to the immune system in B. maxima. Bars with dots represent the number of transcripts with the annotated functions; diamond bars represent the number of different genes (transcripts &lt;10 are not shown)." path-from-xml="dst03503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898866/dst03503.tiff" path-from-xml="dst03503.tiff" target="_blank">View large</a><a section="4898866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03503.png?Expires=2147483647&Signature=gBlF9cpFwnYsZd1hDAnfJK2WANfci3oQfn8DgD2pNxOo592t22nbZKmXkhUCaZeV9fKPoArNBo~p9QoTVTEtaxIwO12FKBF~5PHrRl6OETVOYG7jnp-wSpHE7krPjvzno5n5suBHb7MrlA2-gUxUcdo35daKJQ1iK-bcoK5R3L7U3BWg3CXrUR230c89NJGXunuaXZyroyYNWX~RwYR1pQgVkKApdXCsWqte1G7OjPysd0VlAQ3eIYOOqHumr1YQPIkcNVuffQwzsuJ~1WLemkFFi2CkiP6Snspd4hCgIg08Uhpi82l4greh6UetqrNr2ouESrdQETL~FuVV-B5tNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898866&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The putative genes related to the immune system in <em>B. maxima</em>. Bars with dots represent the number of transcripts with the annotated functions; diamond bars represent the number of different genes (transcripts &lt;10 are not shown).</p></div></div>                        <a id="4898867" scrollto-destination="4898867"></a>
<div id="DST035F4" data-id="DST035F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03504.png?Expires=2147483647&Signature=RROENyXglAH2VAQqZS95fHXBvuIQNxxan4~rd5xjM-g5mYzvDfujxwLTnr5nT2vywtxB-y-hGa7JM8sscxNIO~WyFpNM6LQU80oL1dMOtd8UtG18GEftbj7YxT0p85WtEp2dZaVXMbxNsbwQwgz1MOiiT2Tb0iFBJWAG3RGUOHYtQYio2~qzeOsMZUUj6DqA-NfZpFEyIPkutv81XIFMOjsT4nB8~aV8ESUS9PQI-YA3PNqtFeQIp~cWK4VdL2CXJHIZQ~0qa-I96tPuKgTfsm-NbAMXfsCFnHZ9Z7Zv7XlrU1P6m7DokA9rnghSCbbBYVIYvLesbZPdv8sA7Wrmtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The signalling pathway activated by TLRs in B. maxima. (A) The putative TLRs pathway of B. maxima predicted by KEGG pathway annotation (KO04620). Red coloured genes on the map were definitely identified in the transcriptome. In MyD88-dependent pathway, it leads to the activation of IRAK. In MyD88-independent pathway, TRIF and Rac1 are the main adaptor protein and interact with TRAF3 or PI3K to activate IRF or AKT. Finally, NF-κB is activated. All the members shown in this figure have homologous proteins in other species. (B) The key molecules of TLR pathway were amplified by reverse transcription PCR (RT-PCR) using RNA from skin and blood tissues. Primers were designed according to the transcripts. (C) Clustered expression profile for 56 genes mapped to the KEGG pathway is organized by different tissues. The RPKM values were normalized, with each row representing a different gene. (D) Expression detection of eight key components of the TLR signalling pathway was measured by quantitative RT-PCR (qRT-PCR). Results are the mean crossing points (ΔCP) value ± SEM (n = 3 technical triplicates)." path-from-xml="dst03504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898867/dst03504.tiff" path-from-xml="dst03504.tiff" target="_blank">View large</a><a section="4898867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03504.png?Expires=2147483647&Signature=NhjUwb4I~tiLRFcXmFcmFncHh6MofQEvwOJixHGyh6RAjMq3Gz103U7zKr6pkXgRvuyni1Jsfm-PQ5YmNWqNsyqv63pUbm-YDT8zJTL4LIxw9P3x269R1H96IUQedziToIYLko0BRTq5mwr-8-n7dJYcoWPTEDDbOazz6HTbc~EP~l7tf5jjOYaalScpL572Lc9fVHxhJA8weQlfC9F9ggxPMfs7cs1-3~yNUntOhHv6Mqzlm9~8bfPUb4bVp-Ao1VT~m15HlYOhoPTpKA2pU4mEf3ghxeoKlnNEHZtu00a1R9bUnAG1B3wv-ygBrsD2ELotLvlDFu0q8wmny8C5hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898867&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The signalling pathway activated by TLRs in <em>B. maxima</em>. (A) The putative TLRs pathway of <em>B. maxima</em> predicted by KEGG pathway annotation (KO04620). Red coloured genes on the map were definitely identified in the transcriptome. In MyD88-dependent pathway, it leads to the activation of IRAK. In MyD88-independent pathway, TRIF and Rac1 are the main adaptor protein and interact with TRAF3 or PI3K to activate IRF or AKT. Finally, NF-κB is activated. All the members shown in this figure have homologous proteins in other species. (B) The key molecules of TLR pathway were amplified by reverse transcription PCR (RT-PCR) using RNA from skin and blood tissues. Primers were designed according to the transcripts. (C) Clustered expression profile for 56 genes mapped to the KEGG pathway is organized by different tissues. The RPKM values were normalized, with each row representing a different gene. (D) Expression detection of eight key components of the TLR signalling pathway was measured by quantitative RT-PCR (qRT-PCR). Results are the mean crossing points (ΔCP) value ± SEM (<em>n</em> = 3 technical triplicates).</p></div></div><div content-id="DST035F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03504.png?Expires=2147483647&Signature=RROENyXglAH2VAQqZS95fHXBvuIQNxxan4~rd5xjM-g5mYzvDfujxwLTnr5nT2vywtxB-y-hGa7JM8sscxNIO~WyFpNM6LQU80oL1dMOtd8UtG18GEftbj7YxT0p85WtEp2dZaVXMbxNsbwQwgz1MOiiT2Tb0iFBJWAG3RGUOHYtQYio2~qzeOsMZUUj6DqA-NfZpFEyIPkutv81XIFMOjsT4nB8~aV8ESUS9PQI-YA3PNqtFeQIp~cWK4VdL2CXJHIZQ~0qa-I96tPuKgTfsm-NbAMXfsCFnHZ9Z7Zv7XlrU1P6m7DokA9rnghSCbbBYVIYvLesbZPdv8sA7Wrmtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The signalling pathway activated by TLRs in B. maxima. (A) The putative TLRs pathway of B. maxima predicted by KEGG pathway annotation (KO04620). Red coloured genes on the map were definitely identified in the transcriptome. In MyD88-dependent pathway, it leads to the activation of IRAK. In MyD88-independent pathway, TRIF and Rac1 are the main adaptor protein and interact with TRAF3 or PI3K to activate IRF or AKT. Finally, NF-κB is activated. All the members shown in this figure have homologous proteins in other species. (B) The key molecules of TLR pathway were amplified by reverse transcription PCR (RT-PCR) using RNA from skin and blood tissues. Primers were designed according to the transcripts. (C) Clustered expression profile for 56 genes mapped to the KEGG pathway is organized by different tissues. The RPKM values were normalized, with each row representing a different gene. (D) Expression detection of eight key components of the TLR signalling pathway was measured by quantitative RT-PCR (qRT-PCR). Results are the mean crossing points (ΔCP) value ± SEM (n = 3 technical triplicates)." path-from-xml="dst03504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898867/dst03504.tiff" path-from-xml="dst03504.tiff" target="_blank">View large</a><a section="4898867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03504.png?Expires=2147483647&Signature=NhjUwb4I~tiLRFcXmFcmFncHh6MofQEvwOJixHGyh6RAjMq3Gz103U7zKr6pkXgRvuyni1Jsfm-PQ5YmNWqNsyqv63pUbm-YDT8zJTL4LIxw9P3x269R1H96IUQedziToIYLko0BRTq5mwr-8-n7dJYcoWPTEDDbOazz6HTbc~EP~l7tf5jjOYaalScpL572Lc9fVHxhJA8weQlfC9F9ggxPMfs7cs1-3~yNUntOhHv6Mqzlm9~8bfPUb4bVp-Ao1VT~m15HlYOhoPTpKA2pU4mEf3ghxeoKlnNEHZtu00a1R9bUnAG1B3wv-ygBrsD2ELotLvlDFu0q8wmny8C5hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898867&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The signalling pathway activated by TLRs in <em>B. maxima</em>. (A) The putative TLRs pathway of <em>B. maxima</em> predicted by KEGG pathway annotation (KO04620). Red coloured genes on the map were definitely identified in the transcriptome. In MyD88-dependent pathway, it leads to the activation of IRAK. In MyD88-independent pathway, TRIF and Rac1 are the main adaptor protein and interact with TRAF3 or PI3K to activate IRF or AKT. Finally, NF-κB is activated. All the members shown in this figure have homologous proteins in other species. (B) The key molecules of TLR pathway were amplified by reverse transcription PCR (RT-PCR) using RNA from skin and blood tissues. Primers were designed according to the transcripts. (C) Clustered expression profile for 56 genes mapped to the KEGG pathway is organized by different tissues. The RPKM values were normalized, with each row representing a different gene. (D) Expression detection of eight key components of the TLR signalling pathway was measured by quantitative RT-PCR (qRT-PCR). Results are the mean crossing points (ΔCP) value ± SEM (<em>n</em> = 3 technical triplicates).</p></div></div><p>CD14 transcripts of <em>B. maxima</em> were not mapped to the pathway determined by KEGG annotation. The similarity of CD14 in <em>B. maxima</em> was closely identical to that of <em>Taeniopygia guttata</em> and also to TLR2 type 2 of <em>Amazona albifrons</em>. In addition to previous studies, fish, of which TLR4 is not response to LPS, are resistant to endotoxic shock.<sup><a reveal-id="DST035C28" data-open="DST035C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In frog, the TLR4 is respond to LPS as a receptor in urinary bladder epithelial cells.<sup><a reveal-id="DST035C29" data-open="DST035C29" class="link link-ref link-reveal xref-bibr">29</a></sup> As the pathway prediction, it showed that TLR4 might be failing to respond to LPS through LBP binding to CD14 induced the pathway. Hence, the fact that TLR4 of <em>B. maxima</em> can recognize the LPS to induce the pathway remains to be elucidated. Then, in addition to <em>in silico</em> analysis, we detected the response to LPS or bacteria both <em>in vitro</em> and <em>in vivo</em>. Interleukin (IL)-1β and IL-8 are pro-inflammatory cytokines that play pivotal role in the innate immunity. In response to LPS stimulation, it can be activated in dependent of TLRs activation.<sup><a reveal-id="DST035C30" data-open="DST035C30" class="link link-ref link-reveal xref-bibr">30</a></sup> We detected the IL-1β and IL8 secretion by <em>B. maxima</em> splenocytes after LPS or bacteria stimulation. The results from RT-PCR demonstrated that the mRNA level elevated than that of unstimulated (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>A). LPS and bacteria stimulation upregulated the intracellular level of pro-IL-1β, but LPS only slightly enhanced the secretion of mature IL-1β. It seems that bacteria strongly enhanced the expression and secretion of IL-1β (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>B). Western blotting of IL-8 revealed that LPS and bacteria enhanced the secretion of IL-8 by splenocytes (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>B). The secretion of IL-1β and IL-8 upon LPS and bacteria stimulation were also tested <em>in vivo</em>. The results demonstrated that after intraperitoneal administration of either LPS or bacteria, IL-1β and -8 could be found in the peritoneum exudate fluids (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>D). LPS and bacteria induced the change of peritoneum cell proportion, and also the upregulated proportion of cells was identified to be monocyte-like cells (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>E). </p>                        <a id="4898869" scrollto-destination="4898869"></a>
<div id="DST035F5" data-id="DST035F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03505.png?Expires=2147483647&Signature=vBxkjM8lf85IAtp6j4xd02M45-FCj7G4wk4ZYw7r3gzxdm0KEAx4EE~9inucd6Zic4Bnus5crVHzmZidOogidks4bm~cfCon1HeJcUrc0vMkYFU6w9zLeMRvgJtH2fXA60TMvild06SbbzjXb5muDl7UFrrSL--WW~YH56nZo4zzclYOMgc1RCXAP4t~laW2HiuURpqK1R8~OKRs1yzMBZDPhSDZQ4UPbLMQOJwzjsXaNksEyPZfeDOttJF8KnfL02Zdcv69g436ikRWxaTEZpe3TWJdDvNqsytp3ZtKrjhKH7ANeAyuWYrbdTxa7nTs~ItbY8RalY1Vuf8lJhdmtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lipopolysaccharide (LPS) and bacteria induced pro-inflammatory caspase activation and pro-inflammatory cytokine secretion in vitro and in vivo. (A) Splenocytes were isolated from B. maxima spleen and stimulated with LPS (8 μg/ml) or bacteria (3 × 108cfu/ml) for 2 h. The cells were collected for RT-PCR of IL-1βand IL-8. Either LPS or bacteria can upregulate the mRNA level of IL-1β and IL-8. (B) Splenocytes were incubated with LPS or bacteria. The cells and the supernatants were collected for western blotting of IL-1β and IL-8. (C) Splenocytes stimulated with LPS or bacteria were lysed for the detection of caspase-1 activation by western blotting. The band of ∼20 kDa represents the presence of activated caspase-1. (D) Bombina maxima weighted 25 ± 5 g were injected intraperitoneally with LPS (50 mg/kg in 0.1 ml PBS) or bacteria (3 × 108cfu in 0.1 ml PBS, Staphylococcus aureus, ATCC25923). Two hours after injection, peritoneum exudates were harvested, IL-1β and IL-8 were detected by western blot. (E) Two hours after injection, peritoneal cells were harvested and analysed by flow cytometry. The mostly changed subset of cells was stained and photographed." path-from-xml="dst03505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898869/dst03505.tiff" path-from-xml="dst03505.tiff" target="_blank">View large</a><a section="4898869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03505.png?Expires=2147483647&Signature=A7PNbPXxhvWPXrTusXOXE22DW4MW3r~mzfL1vTyhZoKCP-Nsk5kUUd~fC9Qg-J-BkRckto8JtePQ4qN2D9jyMYmo0mW0Av2u7JSqjYnS7I4o2elVG4j1cXhcbeWV6v9et41iaLS5YciWi-CE0xay2zAMg92JeoexgfijFpcw6tJHQu3Otgace5GjchTxX312JR4NQk-WMlEeOmEvPu02QugIIba7WR1ZzOw9DVfwyCbA20m3PGpM-5Bn62oR3fqdti7M8MRl-nwATJzAtlRQoZEjVQAhDXscv-sNWjJpOVqIrrhm4k~VeqiYItHVugcKWayT5sx27rvpAM8K-hsAEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898869&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lipopolysaccharide (LPS) and bacteria induced pro-inflammatory caspase activation and pro-inflammatory cytokine secretion <em>in vitro</em> and <em>in vivo</em>. (A) Splenocytes were isolated from <em>B. maxima</em> spleen and stimulated with LPS (8 μg/ml) or bacteria (3 × 10<sup>8</sup>cfu/ml) for 2 h. The cells were collected for RT-PCR of IL-1βand IL-8. Either LPS or bacteria can upregulate the mRNA level of IL-1β and IL-8. (B) Splenocytes were incubated with LPS or bacteria. The cells and the supernatants were collected for western blotting of IL-1β and IL-8. (C) Splenocytes stimulated with LPS or bacteria were lysed for the detection of caspase-1 activation by western blotting. The band of ∼20 kDa represents the presence of activated caspase-1. (D) <em>Bombina maxima</em> weighted 25 ± 5 g were injected intraperitoneally with LPS (50 mg/kg in 0.1 ml PBS) or bacteria (3 × 10<sup>8</sup>cfu in 0.1 ml PBS, <em>Staphylococcus aureus</em>, ATCC25923). Two hours after injection, peritoneum exudates were harvested, IL-1β and IL-8 were detected by western blot. (E) Two hours after injection, peritoneal cells were harvested and analysed by flow cytometry. The mostly changed subset of cells was stained and photographed.</p></div></div><div content-id="DST035F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03505.png?Expires=2147483647&Signature=vBxkjM8lf85IAtp6j4xd02M45-FCj7G4wk4ZYw7r3gzxdm0KEAx4EE~9inucd6Zic4Bnus5crVHzmZidOogidks4bm~cfCon1HeJcUrc0vMkYFU6w9zLeMRvgJtH2fXA60TMvild06SbbzjXb5muDl7UFrrSL--WW~YH56nZo4zzclYOMgc1RCXAP4t~laW2HiuURpqK1R8~OKRs1yzMBZDPhSDZQ4UPbLMQOJwzjsXaNksEyPZfeDOttJF8KnfL02Zdcv69g436ikRWxaTEZpe3TWJdDvNqsytp3ZtKrjhKH7ANeAyuWYrbdTxa7nTs~ItbY8RalY1Vuf8lJhdmtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lipopolysaccharide (LPS) and bacteria induced pro-inflammatory caspase activation and pro-inflammatory cytokine secretion in vitro and in vivo. (A) Splenocytes were isolated from B. maxima spleen and stimulated with LPS (8 μg/ml) or bacteria (3 × 108cfu/ml) for 2 h. The cells were collected for RT-PCR of IL-1βand IL-8. Either LPS or bacteria can upregulate the mRNA level of IL-1β and IL-8. (B) Splenocytes were incubated with LPS or bacteria. The cells and the supernatants were collected for western blotting of IL-1β and IL-8. (C) Splenocytes stimulated with LPS or bacteria were lysed for the detection of caspase-1 activation by western blotting. The band of ∼20 kDa represents the presence of activated caspase-1. (D) Bombina maxima weighted 25 ± 5 g were injected intraperitoneally with LPS (50 mg/kg in 0.1 ml PBS) or bacteria (3 × 108cfu in 0.1 ml PBS, Staphylococcus aureus, ATCC25923). Two hours after injection, peritoneum exudates were harvested, IL-1β and IL-8 were detected by western blot. (E) Two hours after injection, peritoneal cells were harvested and analysed by flow cytometry. The mostly changed subset of cells was stained and photographed." path-from-xml="dst03505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898869/dst03505.tiff" path-from-xml="dst03505.tiff" target="_blank">View large</a><a section="4898869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03505.png?Expires=2147483647&Signature=A7PNbPXxhvWPXrTusXOXE22DW4MW3r~mzfL1vTyhZoKCP-Nsk5kUUd~fC9Qg-J-BkRckto8JtePQ4qN2D9jyMYmo0mW0Av2u7JSqjYnS7I4o2elVG4j1cXhcbeWV6v9et41iaLS5YciWi-CE0xay2zAMg92JeoexgfijFpcw6tJHQu3Otgace5GjchTxX312JR4NQk-WMlEeOmEvPu02QugIIba7WR1ZzOw9DVfwyCbA20m3PGpM-5Bn62oR3fqdti7M8MRl-nwATJzAtlRQoZEjVQAhDXscv-sNWjJpOVqIrrhm4k~VeqiYItHVugcKWayT5sx27rvpAM8K-hsAEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898869&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lipopolysaccharide (LPS) and bacteria induced pro-inflammatory caspase activation and pro-inflammatory cytokine secretion <em>in vitro</em> and <em>in vivo</em>. (A) Splenocytes were isolated from <em>B. maxima</em> spleen and stimulated with LPS (8 μg/ml) or bacteria (3 × 10<sup>8</sup>cfu/ml) for 2 h. The cells were collected for RT-PCR of IL-1βand IL-8. Either LPS or bacteria can upregulate the mRNA level of IL-1β and IL-8. (B) Splenocytes were incubated with LPS or bacteria. The cells and the supernatants were collected for western blotting of IL-1β and IL-8. (C) Splenocytes stimulated with LPS or bacteria were lysed for the detection of caspase-1 activation by western blotting. The band of ∼20 kDa represents the presence of activated caspase-1. (D) <em>Bombina maxima</em> weighted 25 ± 5 g were injected intraperitoneally with LPS (50 mg/kg in 0.1 ml PBS) or bacteria (3 × 10<sup>8</sup>cfu in 0.1 ml PBS, <em>Staphylococcus aureus</em>, ATCC25923). Two hours after injection, peritoneum exudates were harvested, IL-1β and IL-8 were detected by western blot. (E) Two hours after injection, peritoneal cells were harvested and analysed by flow cytometry. The mostly changed subset of cells was stained and photographed.</p></div></div><p>The ligation of TLRs signalling occurs primarily through MyD88 (MyD88-dependent pathway) or other adapter proteins (TRAM, TRIF and MyD88-independent pathway). It leads to the induction of antimicrobial genes, cytokines and chemokines and maturation of dendritic cells (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>A). In TLR4 signal transduction, Mal (MyD88) is a necessary adaptor.<sup><a reveal-id="DST035C31" data-open="DST035C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We surely found MyD88 transcripts and that had nearly 2-fold expression in skin than in blood, containing Toll/interleukin-1 receptor domain. But, the RPKM value of MyD88 in skin was a little great than in blood. In addition, MyD88 is required for <em>Xenopus</em> axis formation as binding protein with Toll/IL-1 receptors.<sup><a reveal-id="DST035C32" data-open="DST035C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In the MyD88-independent pathway, the putative transcripts of components were identified such as TRAF3, TBK1 and several IRFs (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>The complement system is a key subsystem in innate immunity, which is composed of many serum proteins associated with many important activities.<sup><a reveal-id="DST035C33" data-open="DST035C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A total of 157 transcripts were identified including C2, C3, C3d, C4a, C5, C6, factor B, factor D and factor I, etc. The most of key complement components were also detected with high expression in skin, stomach and lung tissues of <em>B. maxima</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5 and Table S3</a>). The complement components usually share the short consensus domains, such as C9 with perforin-like domain. We analysed the domains contained in complement components of <em>B. maxima</em>. Most of the complement components shared CCP domain and Tryp SPc (trypsin-like serine protease) domain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). But, the factor P (properdin factor) and factor B have thrombospondin_1 domain. These domains may play a important role in complement initiation and amplification. The regulator of complement activation (RCA) is essential for excessive C3 activation that often drives the consumption of the complement pathway and damages self tissue.<sup><a reveal-id="DST035C34" data-open="DST035C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We identified two RCA-coding proteins in <em>B. maxima</em>, ARC1 and ARC3, which are closely identical to the ARCs of <em>Xenopus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5D and E</a>). In addition to the lectin pathway, we also identified that a number of 22 transcripts highly expressed in <em>B. Maxima</em>, including Fcn1 (M-ficolin), Fcn2 (L-ficolin) and Fcn3 (H-ficolin). In our results, it shows a peak expression in spleen, kidney and stomach (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5C</a>).</p><p>The cytokine network is an important homeostatic system with potent activities in immune surveillance, growth, developmental and repair processes.<sup><a reveal-id="DST035C35" data-open="DST035C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The important role of cytokines is controlling immune responses.<sup><a reveal-id="DST035C36" data-open="DST035C36" class="link link-ref link-reveal xref-bibr">36</a></sup> We identified 42 interleukin/interleukin receptor transcripts in the <em>B. maxima</em> dataset. We did not found IL-2 transcripts in <em>B. maxima</em> the same as in <em>Xenopus</em> that involved in the bactericidal activity in the adaptive immune system (AIS).<sup><a reveal-id="DST035C37" data-open="DST035C37" class="link link-ref link-reveal xref-bibr">37</a></sup> However, IL-34 and IL-7 receptors were identified in <em>B. Maxima</em>, with no orthologs of <em>Xenopus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Another ancient and conserved cytokine is the macrophage migration inhibitory factor (MIF) in the immune system. We identified one full-length MIF transcripts (345 bp), which is closely identical to the MIF of <em>Xenopus</em>, mouse and human (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). MIF is conserved across taxa from invertebrate to vertebrates,<sup><a reveal-id="DST035C38" data-open="DST035C38" class="link link-ref link-reveal xref-bibr">38</a></sup> emerging as an important mediator of IIS.</p><p>In human, TLRs engage in Type I interferon (IFN) (IFN-α/β) production. In our results, the full-length Type I IFNs were not identified in the transcriptome. But, we found short-length (&lt;200 bp) small contigs with the medium identity (40–60%) to <em>Xenopus</em> Type I IFNs, which have low consensus overlaps for further assembly to long transcripts. In interferon receptor pathways, Interferon-α/β receptor (IFNAR) expressed eight transcripts in the transcriptome and mapped to the pathway (Fig. <a reveal-id="DST035F4" data-open="DST035F4" class="link link-reveal link-table xref-fig">4</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). It shows high similarity to IFNAR of chicken (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6A</a>). The Type I IFNs all bind to IFNAR, which is composed of two chains: IFNAR1 and IFNAR2. The intracellular domain of IFNAR1 associates with Tyk2, whereas IFNAR2 associates with JAK1 and JAK2. Type I IFNs can induce the evolutionary pathway, JAK/STAT pathway. As expected, 12 transcripts with a high identity to JAK2 and STATs were found in <em>B. maxima</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). These cytokines also regulate the expression of over 1200 genes, the products of which function as mediators of host immune responses,<sup><a reveal-id="DST035C39" data-open="DST035C39" class="link link-ref link-reveal xref-bibr">39</a></sup> such as four families of IFN-inducible GTPases including dynamin-like GTPases, Mx proteins, p47 GTPases and very large inducible GTPase-1 (VLIG-1).<sup><a reveal-id="DST035C40" data-open="DST035C40" class="link link-ref link-reveal xref-bibr">40</a></sup> These four families of IFN-inducible GTPases were all expressed in the <em>B. maxima</em> transcriptome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). There were also great amounts of VLIG-1 from two major groups expressed in this frog (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6B</a>).</p><p>NOD-like receptors (NLRs) and RIG-like receptors (RLRs) act as pathogen sensors for the recognition of intracellular bacteria and viruses. Twenty-nine transcripts of NLRs were present in <em>B. maxima</em>, including of 10 members such as NOD1, NLRC3, NLRC5, NAIP. Another intracellular sensor is retinoic acid-inducible gene I (RIG-I) receptors. Two transcripts of RIG-I were identified in this transcriptome, with high similarity to <em>Xenopus</em>, then to mouse and human (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Caspase-1 is considered as a typical member of pro-inflammatory caspases, including caspase-1, caspase-4, caspase-5 and caspase-12 in human and caspase-1, caspase-11 and caspase-12 in mouse.<sup><a reveal-id="DST035C41" data-open="DST035C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Caspase-1 is activated upon the assembly of inflammasome, which is assembled by the activation of NOD-like receptors including NLRP3, NLRP1 and NLRC4. In our present study, we tested whether the pro-inflammatory caspase, caspase-1, could be activated. The pro-enzyme form of caspase-1 is an ∼45 kDa protein and cleaved into a p10 and p20 subunit after activation. The results showed that, after bacteria stimulation, the appearance of p20 was determined by western blotting using a rabbit polyclonal antibody against caspase-1 p20. Although it seems that LPS was also able to induce the activation of caspase-1, the ability is much weaker than bacteria (Fig. <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a>C). As the downstream executor of inflammasome, caspase-1 activation implied the presence of the NLR–inflammasome–caspase-1 pathway in this frog.</p>                            <h3 scrollto-destination=4898875 id="4898875" class="section-title" data-legacy-id=s3f>3.6. Genes related to the adaptive immune system</h3>
<p>Currently available data indicate that the AIS suddenly emerged on vertebrate with jaws like a ‘big-bang’ event.<sup><a reveal-id="DST035C42" data-open="DST035C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST035C43" data-open="DST035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> So, it is fascinating to many scientist and investigators for its origin and evolution. The highly complex of AIS is based on intact B-cell receptor (BCR)–T-cell receptor (TCR)–major histocompatibility complex (MHC)-based AIS.<sup><a reveal-id="DST035C43" data-open="DST035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> We identified the transcripts of BCR, TCR and MHC in <em>B. maxima</em> (Fig. <a reveal-id="DST035F3" data-open="DST035F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Moreover, we amplified some key components of the AIS and sequencing the amplified results (Fig. <a reveal-id="DST035F6" data-open="DST035F6" class="link link-reveal link-table xref-fig">6</a>A). All of validated molecules were high expressed in <em>B. Maxima</em>, excluding IgM compared with the others (Fig. <a reveal-id="DST035F6" data-open="DST035F6" class="link link-reveal link-table xref-fig">6</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). From the components of AIS, the immune system of <em>B. maxima</em> is nearly parallel to the mammals'. BCR is composed of two parts, ligand-binding moiety and signal transduction moiety. We identified that IgM, Ig CH II/IgY, IgG and many segments of heavy chain and light chains were definitely presented in the <em>B. maxima</em> transcriptome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). IgM, generally evolutionarily conserved, is most ancient antibody class and secreted monomer.<sup><a reveal-id="DST035C44" data-open="DST035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> IgY is first found in amphibians, the function of which is equal to IgG and IgE.<sup><a reveal-id="DST035C45" data-open="DST035C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Shorter forms of IgY are believed to neutralize pathogen without activation of inflammatory response.<sup><a reveal-id="DST035C43" data-open="DST035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> In addition, we identified the IgG-gamma heavy chain in <em>B. maxima</em> that is searched with no result in <em>Xenopus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). IgG is involved in high-affinity memory responses. In summary, frog is the most primitive species, whose IgY and IgG play a important role in immune response, with a classical memory response that is based on class switching after antigen stimulation. This result showed that <em>IgY</em> has already taken rearrangement and is essential for immune response in <em>B. maxima</em>. CD79A was also identified in the transcriptome and had two transcripts. Therefore, the component of BCRs is complete in the AIS of <em>B. maxima</em>. TCR is responsible for recognizing antigens bound to MHC molecules. Beta-chain TCR and gamma-chain TCR were identified, but alpha-chain and delta-chain had no transcripts expressed in skin and blood tissues. TCR is a heterodimer, consisting of α/β-chains and γ/δ-chains. A translocon organization of both α- and β-TCR genes is found in all animals, and the δ- and α-TCR gene with close linkage.<sup><a reveal-id="DST035C43" data-open="DST035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Hence, the α- and δ-chain of TCR probably existed in <em>B. maxima</em>. </p>                        <a id="4898877" scrollto-destination="4898877"></a>
<div id="DST035F6" data-id="DST035F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03506.png?Expires=2147483647&Signature=zHbM0DGcZczVg3skV8H41Vn-RylX540ywjkYswXN-AtTJPjJXrGnSgGCrV8dxhNyI4aBe1CR5D3eBvBE5em1aEapLmU5~cGF-US75kDENJ3gHNLy8cHKMqJ3J2tie7BjuIwIJk3YCtwtNgsejeOu0Nd4Qe~O2~Lp3quSQIG3OdPXcswVKLtfjTXiTOyZdcAWwYl~YVPrKaFX-fNJBiQqbUrdsx2rik4l-wOIeYvmSvm0vWba3zCtzAyC3VXko-~v~CCsCGCWk4dk7IyiSu7eRH7RrrzyObxMSdX7jnuQhwER2Yh-gDOjjSrcHmZBxyMNwNHr5gWupMRLbuP~MHwiGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes related to the adaptive immune system. (A) Several selected molecules of the AIS in B. maxima were amplified by RT-PCR using RNA from skin and blood tissues, primers designed according to the assembled sequences of the transcriptome. These molecules were identified by searching against known database and domain prediction. Each two transcripts from 18 MHC I and 11 MHC II were detected by RT-PCR. (B) Five key immune components were detected its expression in seven different tissues using qRT-PCR. Results are the mean ΔCP value ± SEM (n = 3 technical triplicates). (C) Eighteen MHC Class I amino sequences were from B. maxima, and MHC I of Gallus gallus, MHC Ia and Ib of Homo sapiens, MHC Ia and Ib of Mus musculus and MHC I of X. laevis. The bootstrap values next to the nodes represent the percentage of 1000 replicate trees supporting the corresponding clade. (D) Eleven MHC Class II amino sequences were from B. maxima, and other MHC II proteins from above species." path-from-xml="dst03506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898877/dst03506.tiff" path-from-xml="dst03506.tiff" target="_blank">View large</a><a section="4898877" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03506.png?Expires=2147483647&Signature=pOUW-Wm714HDspDEyXfeUSKuqxisaYjBTN6usSEh6L4tbkQ868RWw2oBii0CygcS7X2Xf6WVLL9rAcf5hbLVAuvQMGeCK3BP6G07rCbdgyRrgYJ0MgZTalx2wCc1dQSAd2K39G7eSlaFzshBdX5IpqfEegeEbNuiYHBaHcz9YfdqNjqgPydQ-MWflmTXkTh4fOtPHaQWmGQVKReNkgVjoOe4NU72rrZDA7D--XR44NSprf3DmFOckKf2zlruc-z2XTLqR6KRGGKfOFlfgTqJXPbY2SlwkuGluj0AqWNwfSegae4RV9JRI4N81FDGsSKaonbaLzqjinLD3svXGGvzog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898877&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes related to the adaptive immune system. (A) Several selected molecules of the AIS in <em>B. maxima</em> were amplified by RT-PCR using RNA from skin and blood tissues, primers designed according to the assembled sequences of the transcriptome. These molecules were identified by searching against known database and domain prediction. Each two transcripts from 18 MHC I and 11 MHC II were detected by RT-PCR. (B) Five key immune components were detected its expression in seven different tissues using qRT-PCR. Results are the mean ΔCP value ± SEM (<em>n</em> = 3 technical triplicates). (C) Eighteen MHC Class I amino sequences were from <em>B. maxima</em>, and MHC I of <em>Gallus gallus</em>, MHC Ia and Ib of <em>Homo sapiens</em>, MHC Ia and Ib of <em>Mus musculus</em> and MHC I of <em>X. laevis</em>. The bootstrap values next to the nodes represent the percentage of 1000 replicate trees supporting the corresponding clade. (D) Eleven MHC Class II amino sequences were from <em>B. maxima</em>, and other MHC II proteins from above species.</p></div></div><div content-id="DST035F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/m_dst03506.png?Expires=2147483647&Signature=zHbM0DGcZczVg3skV8H41Vn-RylX540ywjkYswXN-AtTJPjJXrGnSgGCrV8dxhNyI4aBe1CR5D3eBvBE5em1aEapLmU5~cGF-US75kDENJ3gHNLy8cHKMqJ3J2tie7BjuIwIJk3YCtwtNgsejeOu0Nd4Qe~O2~Lp3quSQIG3OdPXcswVKLtfjTXiTOyZdcAWwYl~YVPrKaFX-fNJBiQqbUrdsx2rik4l-wOIeYvmSvm0vWba3zCtzAyC3VXko-~v~CCsCGCWk4dk7IyiSu7eRH7RrrzyObxMSdX7jnuQhwER2Yh-gDOjjSrcHmZBxyMNwNHr5gWupMRLbuP~MHwiGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes related to the adaptive immune system. (A) Several selected molecules of the AIS in B. maxima were amplified by RT-PCR using RNA from skin and blood tissues, primers designed according to the assembled sequences of the transcriptome. These molecules were identified by searching against known database and domain prediction. Each two transcripts from 18 MHC I and 11 MHC II were detected by RT-PCR. (B) Five key immune components were detected its expression in seven different tissues using qRT-PCR. Results are the mean ΔCP value ± SEM (n = 3 technical triplicates). (C) Eighteen MHC Class I amino sequences were from B. maxima, and MHC I of Gallus gallus, MHC Ia and Ib of Homo sapiens, MHC Ia and Ib of Mus musculus and MHC I of X. laevis. The bootstrap values next to the nodes represent the percentage of 1000 replicate trees supporting the corresponding clade. (D) Eleven MHC Class II amino sequences were from B. maxima, and other MHC II proteins from above species." path-from-xml="dst03506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4898877/dst03506.tiff" path-from-xml="dst03506.tiff" target="_blank">View large</a><a section="4898877" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst03506.png?Expires=2147483647&Signature=pOUW-Wm714HDspDEyXfeUSKuqxisaYjBTN6usSEh6L4tbkQ868RWw2oBii0CygcS7X2Xf6WVLL9rAcf5hbLVAuvQMGeCK3BP6G07rCbdgyRrgYJ0MgZTalx2wCc1dQSAd2K39G7eSlaFzshBdX5IpqfEegeEbNuiYHBaHcz9YfdqNjqgPydQ-MWflmTXkTh4fOtPHaQWmGQVKReNkgVjoOe4NU72rrZDA7D--XR44NSprf3DmFOckKf2zlruc-z2XTLqR6KRGGKfOFlfgTqJXPbY2SlwkuGluj0AqWNwfSegae4RV9JRI4N81FDGsSKaonbaLzqjinLD3svXGGvzog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4898877&ar=346817&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes related to the adaptive immune system. (A) Several selected molecules of the AIS in <em>B. maxima</em> were amplified by RT-PCR using RNA from skin and blood tissues, primers designed according to the assembled sequences of the transcriptome. These molecules were identified by searching against known database and domain prediction. Each two transcripts from 18 MHC I and 11 MHC II were detected by RT-PCR. (B) Five key immune components were detected its expression in seven different tissues using qRT-PCR. Results are the mean ΔCP value ± SEM (<em>n</em> = 3 technical triplicates). (C) Eighteen MHC Class I amino sequences were from <em>B. maxima</em>, and MHC I of <em>Gallus gallus</em>, MHC Ia and Ib of <em>Homo sapiens</em>, MHC Ia and Ib of <em>Mus musculus</em> and MHC I of <em>X. laevis</em>. The bootstrap values next to the nodes represent the percentage of 1000 replicate trees supporting the corresponding clade. (D) Eleven MHC Class II amino sequences were from <em>B. maxima</em>, and other MHC II proteins from above species.</p></div></div><p>The MHC is defined by MHC Class I and Class II; the MHC Class I/II molecules are closely involved in antigen presentation and usually has multiple copies in the vertebrates.<sup><a reveal-id="DST035C46" data-open="DST035C46" class="link link-ref link-reveal xref-bibr">46</a></sup> We identified 18 transcripts of MHC Class I and 11 transcripts of MHC Class II expressed in <em>B. maxima</em>. As the previous methods,<sup><a reveal-id="DST035C47" data-open="DST035C47" class="link link-ref link-reveal xref-bibr">47</a></sup> we conservatively estimated that the minimum number of putative expressed MHC Class I and Class II was three and two loci indicated by transcripts clustering into three and two major groups without a reference genome (Fig. <a reveal-id="DST035F6" data-open="DST035F6" class="link link-reveal link-table xref-fig">6</a>C and D). Multiple loci for MHC Class II have a profound impact on peptide binding and anti-infections.<sup><a reveal-id="DST035C48" data-open="DST035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> With multiple loci for MHC Class II, the frog has an ability to battle constantly intracellular infection, such as virus. Most <em>Xenopus</em> species have a single-expressed MHC Class I gene (Class Ia), which encodes receptor molecules to present intracellular pathogen and is closely linked to other genes assisting in antigen processing.<sup><a reveal-id="DST035C49" data-open="DST035C49" class="link link-ref link-reveal xref-bibr">49</a></sup> MHC Class Ib gene belongs to a separate MHC lineage. The expressed multiple loci of MHC Class I/II is considered to be the crucial role acted by one or two rounds of whole-genome duplication (WGD) at stage of fish or amphibians.<sup><a reveal-id="DST035C50" data-open="DST035C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In fact, <em>X. laevis</em> expresses non-classical MHC Ib genes with similarity in sequence and structure to classical genes.<sup><a reveal-id="DST035C51" data-open="DST035C51" class="link link-ref link-reveal xref-bibr">51</a></sup> From the constructed a phylogenetic tree (Fig. <a reveal-id="DST035F6" data-open="DST035F6" class="link link-reveal link-table xref-fig">6</a>C), the MHC Class Ib have rapid rate of evolution compared with classical MHC Class Ia of <em>B. maxima</em> and <em>X. laevis</em>. The multiple sequence alignment of the selected MHC Class I sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7A</a>) also shows that the rate of residue substitution to MHC Class Ia lower than that of MHC Class Ib from this frog and other vertebrates. The MHC Class Ia of <em>X. laevis</em> shows high identity to that of <em>B. maxima</em>, except in very few sites. And the conserved sites of MHC Class I exist great variances in this frog (marked with red box). Counterpart to MHC Class II (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7B</a>, marked with blue box), it shows that the conserved domain of four transcripts in <em>B. maxima</em> is high similarity to the human MHC Class II, instead of <em>Xenopus</em>, which is correspond to the phylogenetic tree analysis (Fig. <a reveal-id="DST035F6" data-open="DST035F6" class="link link-reveal link-table xref-fig">6</a>D). The phylogenetic analysis revealed that these sequences clustered into distinct two clades. One of these clades is grouped into MHC Class Ib genes from higher vertebrates. From bootstrap values, MHC Class Ib was indeed a more rapid evolutional rate than MHC Class Ia. As to a single classical Class I locus expressed in <em>X. tropicalis</em> and <em>X. laevis</em>, the degree to which <em>B. maxima</em> is a good potential model for the anuran MHC.</p><p>In summary, we have developed comprehensive transcriptome for <em>B. maxima</em>. Although there is extensive genome or transcriptome achieved in <em>Xenopus</em>, comparative analysis showed that difference exists between these two species' transcriptome. The data obtained is rich in gene transcripts related to disease and disease pathway, especially to immunity. Based on pathway prediction, we uncover the fact that TLRs can recognize the lipoproteins, peptidoglycan, LPS or bacteria in this frog. Furthermore, our results showed that NLR–inflammasome–caspase-1 pathway is present in this frog, which is not clear in other frogs including of <em>Xenopus</em>. Besides that active IIS, this frog can live well in harsh environment owing to its nearly parallel AIS to the mammal. In addition, tissue-specific genes and ncRNAs analysis initially reveal the genetic adaption of this frog.</p>                            <h2 scrollto-destination=4898880 id="4898880" class="section-title" data-legacy-id=s4>Authors' Contributions</h2>
<p>Y.Z., F.Z. and W.L. conceived the study. Manuscript wrote by F.Z. Sequence assemble, bioinformatics analysis and expression validation were performed by F.Z. Samples and RNA extraction were carried out by Y.Y. and G.W. Figure <a reveal-id="DST035F5" data-open="DST035F5" class="link link-reveal link-table xref-fig">5</a> was performed by C.Y., X.W. and Y.X. Y.X. gives great help to manuscript revision. All authors read and approved the final version of manuscript.</p>                            <h2 scrollto-destination=4898882 id="4898882" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4898884 id="4898884" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by grants from the National Basic Research Program of China (973 Program, 2010CB529800), the National Natural Science Foundation of China (NSFC-Yunnan joint funding U1132601, 31071926 and 31270835) and the Key Research Program of the Chinese Academy of Sciences (KJZD-EW-L03).</p>                            <h2 scrollto-destination=4898886 id="4898886" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST035C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pollet</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mazabraud</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Insights from Xenopus genomes</div>, <div class="source">Genome Dyn.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">138</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20from%20Xenopus%20genomes&amp;author=N.%20Pollet&amp;author=A.%20Mazabraud&amp;publication_year=2006&amp;journal=Genome%20Dyn.&amp;volume=2&amp;pages=138-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18753776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yergeau</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Mead</div> <div class="given-names">P.E.</div></div>. </div><div class="article-title">Manipulating the Xenopus genome with transposable elements</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> <div class="issue">Suppl 1</div>pg. <div class="fpage">S11</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Manipulating%20the%20Xenopus%20genome%20with%20transposable%20elements&amp;author=D.A.%20Yergeau&amp;author=P.E.%20Mead&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=S11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-s1-s11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-s1-s11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-s1-s11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18047688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morin</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Petrescu</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Sequencing and analysis of 10,967 full-length cDNA clones from <em>Xenopus laevis</em> and <em>Xenopus tropicalis</em> reveals post-tetraploidization transcriptome remodeling</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">796</div>-<div class="lpage">803</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20analysis%20of%2010%2C967%20full-length%20cDNA%20clones%20from%20Xenopus%20laevis%20and%20Xenopus%20tropicalis%20reveals%20post-tetraploidization%20transcriptome%20remodeling&amp;author=R.D.%20Morin&amp;author=E.%20Chang&amp;author=A.%20Petrescu&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=796-803" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4871006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4871006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4871006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16672307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hellsten</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Harland</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Gilchrist</div> <div class="given-names">M.J.</div></div>, et al. </div><div class="article-title">The genome of the Western clawed frog <em>Xenopus tropicalis</em></div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">328</div> (pg. <div class="fpage">633</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20Western%20clawed%20frog%20Xenopus%20tropicalis&amp;author=U.%20Hellsten&amp;author=R.M.%20Harland&amp;author=M.J.%20Gilchrist&amp;publication_year=2010&amp;journal=Science&amp;volume=328&amp;pages=633-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1183670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1183670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1183670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rockman</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genetics of global gene expression</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">862</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20global%20gene%20expression&amp;author=M.V.%20Rockman&amp;author=L.%20Kruglyak&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=862-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1964" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1964" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1964"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17047685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wan</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Bm-TFF2, a trefoil factor protein with platelet activation activity from frog <em>Bombina maxima</em> skin secretions</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2005</div>, vol. <div class="volume">330</div> (pg. <div class="fpage">1027</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bm-TFF2%2C%20a%20trefoil%20factor%20protein%20with%20platelet%20activation%20activity%20from%20frog%20Bombina%20maxima%20skin%20secretions&amp;author=J.%20Zhang&amp;author=Y.%20Zhang&amp;author=S.G.%20Wan&amp;author=S.S.%20Wei&amp;author=W.H.%20Lee&amp;publication_year=2005&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=330&amp;pages=1027-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2005.03.077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2005.03.077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2005.03.077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15823546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lai</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Variety of antimicrobial peptides in the <em>Bombina maxima</em> toad and evidence of their rapid diversification</div>, <div class="source">Eur. J. Immunol.</div> , <div class="year">2005</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">1220</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variety%20of%20antimicrobial%20peptides%20in%20the%20Bombina%20maxima%20toad%20and%20evidence%20of%20their%20rapid%20diversification&amp;author=W.H.%20Lee&amp;author=Y.%20Li&amp;author=R.%20Lai&amp;author=S.%20Li&amp;author=Y.%20Zhang&amp;author=W.%20Wang&amp;publication_year=2005&amp;journal=Eur.%20J.%20Immunol.&amp;volume=35&amp;pages=1220-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-4141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-4141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-4141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15770703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Establishment, characterization and immortalization of a fibroblast cell line from the Chinese red belly toad <em>Bombina maxima</em> skin</div>, <div class="source">Cytotechnology</div> , <div class="year">2012</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">95</div>-<div class="lpage">105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%2C%20characterization%20and%20immortalization%20of%20a%20fibroblast%20cell%20line%20from%20the%20Chinese%20red%20belly%20toad%20Bombina%20maxima%20skin&amp;author=Y.%20Xiang&amp;author=Q.%20Gao&amp;author=W.%20Su&amp;publication_year=2012&amp;journal=Cytotechnology&amp;volume=64&amp;pages=95-105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10616-011-9399-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10616-011-9399-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10616-011-9399-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21948116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A novel non-lens betagamma-crystallin and trefoil factor complex from amphibian skin and its functional implications</div>, <div class="source">PLoS One</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e1770</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20non-lens%20betagamma-crystallin%20and%20trefoil%20factor%20complex%20from%20amphibian%20skin%20and%20its%20functional%20implications&amp;author=S.B.%20Liu&amp;author=Y.Y.%20He&amp;author=Y.%20Zhang&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e1770" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18335045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Structural basis for signal transduction by the Toll/interleukin-1 receptor domains</div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">111</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20signal%20transduction%20by%20the%20Toll%2Finterleukin-1%20receptor%20domains&amp;author=Y.%20Xu&amp;author=X.%20Tao&amp;author=B.%20Shen&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=111-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35047056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35047056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35047056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11081518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robert</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">N.</div></div>. </div><div class="article-title">The genus Xenopus as a multispecies model for evolutionary and comparative immunobiology of the 21st century</div>, <div class="source">Dev. Comp. Immunol.</div> , <div class="year">2011</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">916</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genus%20Xenopus%20as%20a%20multispecies%20model%20for%20evolutionary%20and%20comparative%20immunobiology%20of%20the%2021st%20century&amp;author=J.%20Robert&amp;author=N.%20Cohen&amp;publication_year=2011&amp;journal=Dev.%20Comp.%20Immunol.&amp;volume=35&amp;pages=916-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dci.2011.01.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dci.2011.01.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dci.2011.01.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21277325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wiesner</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vilcinskas</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Antimicrobial peptides: the ancient arm of the human immune system</div>, <div class="source">Virulence</div> , <div class="year">2010</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">440</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antimicrobial%20peptides%3A%20the%20ancient%20arm%20of%20the%20human%20immune%20system&amp;author=J.%20Wiesner&amp;author=A.%20Vilcinskas&amp;publication_year=2010&amp;journal=Virulence&amp;volume=1&amp;pages=440-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/viru.1.5.12983" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fviru.1.5.12983" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fviru.1.5.12983"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21178486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Extremely abundant antimicrobial peptides existed in the skins of nine kinds of Chinese odorous frogs</div>, <div class="source">J. Proteome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">306</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extremely%20abundant%20antimicrobial%20peptides%20existed%20in%20the%20skins%20of%20nine%20kinds%20of%20Chinese%20odorous%20frogs&amp;author=X.%20Yang&amp;author=W.H.%20Lee&amp;author=Y.%20Zhang&amp;publication_year=2012&amp;journal=J.%20Proteome%20Res.&amp;volume=11&amp;pages=306-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr200782u" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr200782u" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr200782u"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22029824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du Pasquier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Schwager</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">The immune system of Xenopus</div>, <div class="source">Annu. Rev. Immunol.</div> , <div class="year">1989</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">251</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20immune%20system%20of%20Xenopus&amp;author=L.%20Du%20Pasquier&amp;author=J.%20Schwager&amp;author=M.F.%20Flajnik&amp;publication_year=1989&amp;journal=Annu.%20Rev.%20Immunol.&amp;volume=7&amp;pages=251-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.iy.07.040189.001343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.iy.07.040189.001343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.iy.07.040189.001343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2653371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blackburn</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Bickford</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Diesmos</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Iskandar</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">An ancient origin for the enigmatic flat-headed frogs (Bombinatoridae: Barbourula) from the islands of Southeast Asia</div>, <div class="source">PloS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e12090</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ancient%20origin%20for%20the%20enigmatic%20flat-headed%20frogs%20(Bombinatoridae%3A%20Barbourula)%20from%20the%20islands%20of%20Southeast%20Asia&amp;author=D.C.%20Blackburn&amp;author=D.P.%20Bickford&amp;author=A.C.%20Diesmos&amp;author=D.T.%20Iskandar&amp;author=R.M.%20Brown&amp;publication_year=2010&amp;journal=PloS%20One&amp;volume=5&amp;pages=e12090" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0012090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0012090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0012090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20711504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Next-generation DNA sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1135</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing&amp;author=J.%20Shendure&amp;author=H.%20Ji&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18846087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tan</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Au</div> <div class="given-names">K.F.</div></div>,  <div class="name"><div class="surname">Yablonovitch</div> <div class="given-names">A.L.</div></div>, et al. </div><div class="article-title">RNA sequencing reveals a diverse and dynamic repertoire of the <em>Xenopus tropicalis</em> transcriptome over development</div>, <div class="source">Genome Res.</div> , <div class="year">2013</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">201</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequencing%20reveals%20a%20diverse%20and%20dynamic%20repertoire%20of%20the%20Xenopus%20tropicalis%20transcriptome%20over%20development&amp;author=M.H.%20Tan&amp;author=K.F.%20Au&amp;author=A.L.%20Yablonovitch&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=201-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.141424.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.141424.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.141424.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22960373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pfaffl</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">A new mathematical model for relative quantification in real-time RT-PCR</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> pg. <div class="fpage">e45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR&amp;author=M.W.%20Pfaffl&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=e45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.9.e45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.9.e45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.9.e45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11328886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coppe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pujolar</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Maes</div> <div class="given-names">G.E.</div></div>, et al. </div><div class="article-title">Sequencing, de novo annotation and analysis of the first <em>Anguilla anguilla</em> transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">635</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%2C%20de%20novo%20annotation%20and%20analysis%20of%20the%20first%20Anguilla%20anguilla%20transcriptome%3A%20EeelBase%20opens%20new%20perspectives%20for%20the%20study%20of%20the%20critically%20endangered%20European%20eel&amp;author=A.%20Coppe&amp;author=J.M.%20Pujolar&amp;author=G.E.%20Maes&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21080939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ma</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Transcriptome analysis of the oriental river prawn, <em>Macrobrachium nipponense</em> using 454 pyrosequencing for discovery of genes and markers</div>, <div class="source">PloS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e39727</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20oriental%20river%20prawn%2C%20Macrobrachium%20nipponense%20using%20454%20pyrosequencing%20for%20discovery%20of%20genes%20and%20markers&amp;author=K.%20Ma&amp;author=G.%20Qiu&amp;author=J.%20Feng&amp;author=J.%20Li&amp;publication_year=2012&amp;journal=PloS%20One&amp;volume=7&amp;pages=e39727" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0039727" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0039727" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0039727"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Transcriptome analysis of silver carp (<em>Hypophthalmichthys molitrix</em>) by paired-end RNA sequencing</div>, <div class="source">DNA Res.</div> , <div class="year">2012</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">131</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20silver%20carp%20(Hypophthalmichthys%20molitrix)%20by%20paired-end%20RNA%20sequencing&amp;author=B.%20Fu&amp;author=S.%20He&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=131-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22279088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nie</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jia</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Analysis of muscle and ovary transcriptome of <em>Sus scrofa:</em> assembly, annotation and marker discovery</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">343</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20muscle%20and%20ovary%20transcriptome%20of%20Sus%20scrofa%3A%20assembly%2C%20annotation%20and%20marker%20discovery&amp;author=Q.%20Nie&amp;author=M.%20Fang&amp;author=X.%20Jia&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=343-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21729922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Procaccini</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Galgani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">De Rosa</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Matarese</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Intracellular metabolic pathways control immune tolerance</div>, <div class="source">Trends Immunol.</div> , <div class="year">2012</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intracellular%20metabolic%20pathways%20control%20immune%20tolerance&amp;author=C.%20Procaccini&amp;author=M.%20Galgani&amp;author=V.%20De%20Rosa&amp;author=G.%20Matarese&amp;publication_year=2012&amp;journal=Trends%20Immunol.&amp;volume=33&amp;pages=1-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.it.2011.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.it.2011.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.it.2011.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22075206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Font</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mackay</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Beyond DNA: zinc finger domains as RNA-binding modules</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">649</div> (pg. <div class="fpage">479</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beyond%20DNA%3A%20zinc%20finger%20domains%20as%20RNA-binding%20modules&amp;author=J.%20Font&amp;author=J.P.%20Mackay&amp;publication_year=2010&amp;journal=Methods%20Mol.%20Biol.&amp;volume=649&amp;pages=479-91" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20680853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jung</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lyons</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Dinh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hurwood</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">McWilliam</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mather</div> <div class="given-names">P.B.</div></div>. </div><div class="article-title">Transcriptomics of a giant freshwater prawn (<em>Macrobrachium rosenbergii</em>): de novo assembly, annotation and marker discovery</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e27938</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomics%20of%20a%20giant%20freshwater%20prawn%20(Macrobrachium%20rosenbergii)%3A%20de%20novo%20assembly%2C%20annotation%20and%20marker%20discovery&amp;author=H.%20Jung&amp;author=R.E.%20Lyons&amp;author=H.%20Dinh&amp;author=D.A.%20Hurwood&amp;author=S.%20McWilliam&amp;author=P.B.%20Mather&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e27938" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0027938" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027938" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0027938"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ravasi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Zavolan</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Systematic characterization of the zinc-finger-containing proteins in the mouse transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1430</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20characterization%20of%20the%20zinc-finger-containing%20proteins%20in%20the%20mouse%20transcriptome&amp;author=T.%20Ravasi&amp;author=T.%20Huber&amp;author=M.%20Zavolan&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1430-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.949803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.949803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.949803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12819142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohta</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.</div></div>. </div><div class="article-title">IgD, like IgM, is a primordial immunoglobulin class perpetuated in most jawed vertebrates</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">10723</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IgD%2C%20like%20IgM%2C%20is%20a%20primordial%20immunoglobulin%20class%20perpetuated%20in%20most%20jawed%20vertebrates&amp;author=Y.%20Ohta&amp;author=M.%20Flajnik&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=10723-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0601407103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0601407103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0601407103"> </span></div> </div></div></div></div></div><div content-id="DST035C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iliev</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Roach</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Mackenzie</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Planas</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Goetz</div> <div class="given-names">F.W.</div></div>. </div><div class="article-title">Endotoxin recognition: in fish or not in fish?</div>, <div class="source">FEBS Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">579</div> (pg. <div class="fpage">6519</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endotoxin%20recognition%3A%20in%20fish%20or%20not%20in%20fish%3F&amp;author=D.B.%20Iliev&amp;author=J.C.%20Roach&amp;author=S.%20Mackenzie&amp;author=J.V.%20Planas&amp;author=F.W.%20Goetz&amp;publication_year=2005&amp;journal=FEBS%20Lett.&amp;volume=579&amp;pages=6519-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2005.10.061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2005.10.061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2005.10.061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16297386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nikolaeva</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bachteeva</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fock</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Frog urinary bladder epithelial cells express TLR4 and respond to bacterial LPS by increase of iNOS expression and <span class="small-caps">l</span>-arginine uptake</div>, <div class="source">Am. J. Physiol. <em>Regul. Integr. Comp. Physiol.</em></div> , <div class="year">2012</div>, vol. <div class="volume">303</div> (pg. <div class="fpage">R1042</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Frog%20urinary%20bladder%20epithelial%20cells%20express%20TLR4%20and%20respond%20to%20bacterial%20LPS%20by%20increase%20of%20iNOS%20expression%20and%20l-arginine%20uptake&amp;author=S.%20Nikolaeva&amp;author=V.%20Bachteeva&amp;author=E.%20Fock&amp;publication_year=2012&amp;journal=Am.%20J.%20Physiol.%20Regul.%20Integr.%20Comp.%20Physiol.&amp;volume=303&amp;pages=R1042-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/ajpregu.00045.2012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fajpregu.00045.2012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fajpregu.00045.2012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23019216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chow</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Golenbock</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Christ</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Gusovsky</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Toll-like receptor-4 mediates lipopolysaccharide-induced signal transduction</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1999</div>, vol. <div class="volume">274</div> (pg. <div class="fpage">10689</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toll-like%20receptor-4%20mediates%20lipopolysaccharide-induced%20signal%20transduction&amp;author=J.C.%20Chow&amp;author=D.W.%20Young&amp;author=D.T.%20Golenbock&amp;author=W.J.%20Christ&amp;author=F.%20Gusovsky&amp;publication_year=1999&amp;journal=J.%20Biol.%20Chem.&amp;volume=274&amp;pages=10689-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.274.16.10689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.274.16.10689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.274.16.10689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10196138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fitzgerald</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Palsson-McDermott</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Bowie</div> <div class="given-names">A.G.</div></div>, et al. </div><div class="article-title">Mal (MyD88-adapter-like) is required for Toll-like receptor-4 signal transduction</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">413</div> (pg. <div class="fpage">78</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mal%20(MyD88-adapter-like)%20is%20required%20for%20Toll-like%20receptor-4%20signal%20transduction&amp;author=K.A.%20Fitzgerald&amp;author=E.M.%20Palsson-McDermott&amp;author=A.G.%20Bowie&amp;publication_year=2001&amp;journal=Nature&amp;volume=413&amp;pages=78-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35092578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35092578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35092578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11544529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prothmann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Armstrong</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Rupp</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">The Toll/IL-1 receptor binding protein MyD88 is required for Xenopus axis formation</div>, <div class="source">Mech. Develop.</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">85</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Toll%2FIL-1%20receptor%20binding%20protein%20MyD88%20is%20required%20for%20Xenopus%20axis%20formation&amp;author=C.%20Prothmann&amp;author=N.J.%20Armstrong&amp;author=R.A.%20Rupp&amp;publication_year=2000&amp;journal=Mech.%20Develop.&amp;volume=97&amp;pages=85-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0925-4773(00)00447-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0925-4773(00)00447-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0925-4773(00)00447-0"> </span></div> </div></div></div></div></div><div content-id="DST035C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">R.</div></div>. </div><div class="article-title">WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases</div>, <div class="source">Cell. Mol. Life Sci.</div> , <div class="year">2001</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">2085</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WD-repeat%20proteins%3A%20structure%20characteristics%2C%20biological%20function%2C%20and%20their%20involvement%20in%20human%20diseases&amp;author=D.%20Li&amp;author=R.%20Roberts&amp;publication_year=2001&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;volume=58&amp;pages=2085-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00000838" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00000838" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00000838"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11814058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oshiumi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seya</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Regulator of complement activation (RCA) gene cluster in <em>Xenopus tropicalis</em></div>, <div class="source">Immunogenetics</div> , <div class="year">2009</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">371</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulator%20of%20complement%20activation%20(RCA)%20gene%20cluster%20in%20Xenopus%20tropicalis&amp;author=H.%20Oshiumi&amp;author=Y.%20Suzuki&amp;author=M.%20Matsumoto&amp;author=T.%20Seya&amp;publication_year=2009&amp;journal=Immunogenetics&amp;volume=61&amp;pages=371-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00251-009-0368-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00251-009-0368-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00251-009-0368-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19319518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Del Rio-Tsonis</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsonis</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Zarkadis</div> <div class="given-names">I.K.</div></div>,  <div class="name"><div class="surname">Tsagas</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Lambris</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Expression of the third component of complement, C3, in regenerating limb blastema cells of urodeles</div>, <div class="source">J. Immunol.</div> , <div class="year">1998</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">6819</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20third%20component%20of%20complement%2C%20C3%2C%20in%20regenerating%20limb%20blastema%20cells%20of%20urodeles&amp;author=K.%20Del%20Rio-Tsonis&amp;author=P.A.%20Tsonis&amp;author=I.K.%20Zarkadis&amp;author=A.G.%20Tsagas&amp;author=J.D.%20Lambris&amp;publication_year=1998&amp;journal=J.%20Immunol.&amp;volume=161&amp;pages=6819-24" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huising</div> <div class="given-names">M.O.</div></div>,  <div class="name"><div class="surname">Stet</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Savelkoul</div> <div class="given-names">H.F.</div></div>,  <div class="name"><div class="surname">Verburg-van Kemenade</div> <div class="given-names">B.M.</div></div>. </div><div class="article-title">The molecular evolution of the interleukin-1 family of cytokines; IL-18 in teleost fish</div>, <div class="source">Dev. Comp. Immunol.</div> , <div class="year">2004</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">395</div>-<div class="lpage">413</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20evolution%20of%20the%20interleukin-1%20family%20of%20cytokines%3B%20IL-18%20in%20teleost%20fish&amp;author=M.O.%20Huising&amp;author=R.J.%20Stet&amp;author=H.F.%20Savelkoul&amp;author=B.M.%20Verburg-van%20Kemenade&amp;publication_year=2004&amp;journal=Dev.%20Comp.%20Immunol.&amp;volume=28&amp;pages=395-413" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.dci.2003.09.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.dci.2003.09.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.dci.2003.09.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15062640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wargnier</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lagrange</div> <div class="given-names">P.H.</div></div>. </div><div class="article-title">[Bactericidal activity of cells of the immune system]</div>, <div class="source">Pathol. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">887</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%5BBactericidal%20activity%20of%20cells%20of%20the%20immune%20system%5D&amp;author=A.%20Wargnier&amp;author=P.H.%20Lagrange&amp;publication_year=1993&amp;journal=Pathol.%20Biol.&amp;volume=41&amp;pages=887-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8121721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Calandra</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Roger</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Macrophage migration inhibitory factor: a regulator of innate immunity</div>, <div class="source">Nat. Rev. Immunol.</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">791</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Macrophage%20migration%20inhibitory%20factor%3A%20a%20regulator%20of%20innate%20immunity&amp;author=T.%20Calandra&amp;author=T.%20Roger&amp;publication_year=2003&amp;journal=Nat.%20Rev.%20Immunol.&amp;volume=3&amp;pages=791-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nri1200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnri1200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnri1200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14502271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Taylor</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Sher</div> <div class="given-names">A.</div></div>. </div><div class="article-title">p47 GTPases: regulators of immunity to intracellular pathogens</div>, <div class="source">Nat. Rev. Immunol.</div> , <div class="year">2004</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">100</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=p47%20GTPases%3A%20regulators%20of%20immunity%20to%20intracellular%20pathogens&amp;author=G.A.%20Taylor&amp;author=C.G.%20Feng&amp;author=A.%20Sher&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Immunol.&amp;volume=4&amp;pages=100-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nri1270" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnri1270" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnri1270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15040583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klamp</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Boehm</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Schenk</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pfeffer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Howard</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">A giant GTPase, very large inducible GTPase-1, is inducible by IFNs</div>, <div class="source">J. Immunol.</div> , <div class="year">2003</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1255</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20giant%20GTPase%2C%20very%20large%20inducible%20GTPase-1%2C%20is%20inducible%20by%20IFNs&amp;author=T.%20Klamp&amp;author=U.%20Boehm&amp;author=D.%20Schenk&amp;author=K.%20Pfeffer&amp;author=J.C.%20Howard&amp;publication_year=2003&amp;journal=J.%20Immunol.&amp;volume=171&amp;pages=1255-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4049/jimmunol.171.3.1255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4049%2Fjimmunol.171.3.1255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4049%2Fjimmunol.171.3.1255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12874213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martinon</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Mayor</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tschopp</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The inflammasomes: guardians of the body</div>, <div class="source">Annu. Rev. Immunol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">229</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inflammasomes%3A%20guardians%20of%20the%20body&amp;author=F.%20Martinon&amp;author=A.%20Mayor&amp;author=J.%20Tschopp&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Immunol.&amp;volume=27&amp;pages=229-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.immunol.021908.132715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.immunol.021908.132715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.immunol.021908.132715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19302040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schluter</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Bernstein</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Bernstein</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Marchalonis</div> <div class="given-names">J.J.</div></div>. </div><div class="article-title">‘Big Bang’ emergence of the combinatorial immune system</div>, <div class="source">Dev. Comp. Immunol.</div> , <div class="year">1999</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">107</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%E2%80%98Big%20Bang%E2%80%99%20emergence%20of%20the%20combinatorial%20immune%20system&amp;author=S.F.%20Schluter&amp;author=R.M.%20Bernstein&amp;author=H.%20Bernstein&amp;author=J.J.%20Marchalonis&amp;publication_year=1999&amp;journal=Dev.%20Comp.%20Immunol.&amp;volume=23&amp;pages=107-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0145-305X(99)00002-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0145-305X(99)00002-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0145-305X(99)00002-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10227478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Origin and evolution of the adaptive immune system: genetic events and selective pressures</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">47</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20evolution%20of%20the%20adaptive%20immune%20system%3A%20genetic%20events%20and%20selective%20pressures&amp;author=M.F.%20Flajnik&amp;author=M.%20Kasahara&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=47-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2703" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2703" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2703"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Comparative analyses of immunoglobulin genes: surprises and portents</div>, <div class="source">Nat. Rev. Immunol.</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">688</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20of%20immunoglobulin%20genes%3A%20surprises%20and%20portents&amp;author=M.F.%20Flajnik&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Immunol.&amp;volume=2&amp;pages=688-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nri889" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnri889" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnri889"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12209137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warr</div> <div class="given-names">G.W.</div></div>,  <div class="name"><div class="surname">Magor</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">IgY: clues to the origins of modern antibodies</div>, <div class="source">Immunol. Today</div> , <div class="year">1995</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">392</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IgY%3A%20clues%20to%20the%20origins%20of%20modern%20antibodies&amp;author=G.W.%20Warr&amp;author=K.E.%20Magor&amp;author=D.A.%20Higgins&amp;publication_year=1995&amp;journal=Immunol.%20Today&amp;volume=16&amp;pages=392-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0167-5699(95)80008-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0167-5699(95)80008-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0167-5699(95)80008-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7546196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genome dynamics of the major histocompatibility complex: insights from genome paralogy</div>, <div class="source">Immunogenetics</div> , <div class="year">1999</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">134</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20dynamics%20of%20the%20major%20histocompatibility%20complex%3A%20insights%20from%20genome%20paralogy&amp;author=M.%20Kasahara&amp;publication_year=1999&amp;journal=Immunogenetics&amp;volume=50&amp;pages=134-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002510050589" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002510050589" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002510050589"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10602875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kiemnec-Tyburczy</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Richmond</div> <div class="given-names">J.Q.</div></div>,  <div class="name"><div class="surname">Savage</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Lips</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Zamudio</div> <div class="given-names">K.R.</div></div>. </div><div class="article-title">Genetic diversity of MHC class I loci in six non-model frogs is shaped by positive selection and gene duplication</div>, <div class="source">Heredity (Edinb)</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">146</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20MHC%20class%20I%20loci%20in%20six%20non-model%20frogs%20is%20shaped%20by%20positive%20selection%20and%20gene%20duplication&amp;author=K.M.%20Kiemnec-Tyburczy&amp;author=J.Q.%20Richmond&amp;author=A.E.%20Savage&amp;author=K.R.%20Lips&amp;author=K.R.%20Zamudio&amp;publication_year=2012&amp;journal=Heredity%20(Edinb)&amp;volume=109&amp;pages=146-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2012.22" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2012.22" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2012.22"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22549517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McConnell</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Godwin</div> <div class="given-names">U.B.</div></div>,  <div class="name"><div class="surname">Cuthbertson</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Expressed major histocompatibility complex class II loci in fishes</div>, <div class="source">Immunol Rev.</div> , <div class="year">1998</div>, vol. <div class="volume">166</div> (pg. <div class="fpage">294</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20major%20histocompatibility%20complex%20class%20II%20loci%20in%20fishes&amp;author=T.J.%20McConnell&amp;author=U.B.%20Godwin&amp;author=B.J.%20Cuthbertson&amp;publication_year=1998&amp;journal=Immunol%20Rev.&amp;volume=166&amp;pages=294-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imr.1998.166.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimr.1998.166.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimr.1998.166.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9914920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Canel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kramer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Evolution of the major histocompatibility complex: molecular cloning of major histocompatibility complex class I from the amphibian Xenopus</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1991</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">537</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20major%20histocompatibility%20complex%3A%20molecular%20cloning%20of%20major%20histocompatibility%20complex%20class%20I%20from%20the%20amphibian%20Xenopus&amp;author=M.F.%20Flajnik&amp;author=C.%20Canel&amp;author=J.%20Kramer&amp;author=M.%20Kasahara&amp;publication_year=1991&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=537-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.2.537" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.2.537" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.2.537"> </span></div> </div></div></div></div></div><div content-id="DST035C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panopoulou</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Poustka</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Timing and mechanism of ancient vertebrate genome duplications—the adventure of a hypothesis</div>, <div class="source">Trends Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">559</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Timing%20and%20mechanism%20of%20ancient%20vertebrate%20genome%20duplications%E2%80%94the%20adventure%20of%20a%20hypothesis&amp;author=G.%20Panopoulou&amp;author=A.J.%20Poustka&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=559-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.08.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16099069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST035C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST035C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flajnik</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Kasahara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shum</div> <div class="given-names">B.P.</div></div>,  <div class="name"><div class="surname">Salter-Cid</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Du Pasquier</div> <div class="given-names">L.</div></div>. </div><div class="article-title">A novel type of class I gene organization in vertebrates: a large family of non-MHC-linked class I genes is expressed at the RNA level in the amphibian Xenopus</div>, <div class="source">EMBO J.</div> , <div class="year">1993</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">4385</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20type%20of%20class%20I%20gene%20organization%20in%20vertebrates%3A%20a%20large%20family%20of%20non-MHC-linked%20class%20I%20genes%20is%20expressed%20at%20the%20RNA%20level%20in%20the%20amphibian%20Xenopus&amp;author=M.F.%20Flajnik&amp;author=M.%20Kasahara&amp;author=B.P.%20Shum&amp;author=L.%20Salter-Cid&amp;author=E.%20Taylor&amp;author=L.%20Du%20Pasquier&amp;publication_year=1993&amp;journal=EMBO%20J.&amp;volume=12&amp;pages=4385-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8223448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Shoji Tsuji</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=346817&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table7.xls?Expires=2147483647&amp;Signature=lr1g8RXeFqek9JdTRX~pwEMYKZQn79-e38dpYyArCanhAi24U~BDVihq3hXCROMugLWaS4vO63LQSSEVFjd~FX2ltCjwA9rIfalOK5TYLEnnM32-TPk0lNIobYvm0NUjDkXQaRuR6gNNBU720CSvQ88qqEjWBj5oXJVjVfteaREofmQvQqqSLk6ajm4ZFb5oidypsEM3SoYUYInfhWAK0yGKMCase-6illpA5oPRv79zh000SDIR7R-pZEoK7AgWLm6Lz7n5atey5v19wvk3PCd4WquG18UXe9lfW94jCvtnep2kklFI9xEH8YiV-VhekhCMn3JeULoy3Q7-2EMRWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table6.xls?Expires=2147483647&amp;Signature=Tzy3fr~sK~~e7skJEQ~AE0lP4DGfAocvlHGDqBqkLkD68xWMN8j4~QanWgUNRJkQP4M6uIemptNeIIH7XF6hIaeHLTpLjsTjWi1RzClnOldxUDI904vQ~rRs6EhFN6N-GwvuektwlywNeU-3Bxhpz1-kp81x9h7G5y6AXa6aqsZ~tPEnqxBgY6ZGLCZxGIjG1qBZnadYcavU7mwuyBJU5u5EAWp0og8-oBDzlPcJWZn8P6gL5FRk~SmszWA9iFvB42uKiPwrpGFjyuGycO9VFjTx4CDxyt~JVVtj004oFuhp5vCjtKQE2KeBgu1eZVUyvzStYf72FqlUPm7-97zGgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table5.xls?Expires=2147483647&amp;Signature=Vrw4Mzj72mGgnx9gwOW2isIZKEmYRJjE5YBe5PjqxFuitQHCKx1EOLvWhrPnSZOCEAYB7NysbGsipDmiC5Y19xF9M4dZrjje2D3mCflqmgWe3hFnfTOwrDDMab54CvWIodI-If536~Axkqs4rdqVRCe6w84aI7AKpC7OKga~umy5uUBYa5pCQKppSHN19sR-Z9tIJQQbX1tFMWO4TgSm-RS9MWFDoMgeuye~yyE7hhTJWDmupVbmkQD8XNqKMYesNaTsTcxMerLzMW8OE7mWy65GSXWzL6IEapQvmncJH5XDMDjnTxf2CiNhYtFP-BPUcPgDdu5FuPZvE597tnL0EQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table4.xls?Expires=2147483647&amp;Signature=RQKN3DBHcFOvunuc9KK~GNQMEFrLF4gudBJo5JgYf6BtQmvsv8zdTG2yvb2IR-DOmixET9jCaMB8bLAd3L-1UQT-NxyrSCZ1t62ZVhN03q46EAkXvCr0yxixQa9OJg-Ri6gf22OizEv73MfS46XB6cYCc3K8dDlWpMF9FZtrZTWAg237f7A2zdAyihePMfwbF1qj-NAbBNk-RTRXOptwGlodeImF0jygrJX5OG8ECSCkV7~xV8C5wuf2eVxk0fyNbGwPPuqlLw0Ngh3KvJp0Z5jFwbT3V7FeDsK9kJ3D2ITAz2UPHHLS3kwk9IHBHu0emBrOWj~K-sJXHdH1WvukUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table3.xls?Expires=2147483647&amp;Signature=1XYygVulU-VTwEFWmye~fD0z9pHpmKJvlJ4nlU7tGpdBjkHuWbG9Bxf7lKvF2kTuOHew9Xfq3B4ABAFYpqbveFJUfEzoucxGSMc-zLvzBhg~EZmZpNdW2MEvwrIh1oOjlo1TAArZPwi1NrCcZ6uVZyJuTlwDwEHU~nnF4jeZdy8yavLgyaKdm529OCK42E7Rle8wANB0IhMFlQ6KTsZA6FU9JH92pBwIg--LiJGQhsKcXj0ESIKl-YwOhtD4J9f4X9ccjmC0Weg4a9YTSh6~f3craYzu0jyt6bt-qtRJdm-fyIwbiaycQMRIDpvpNBAMJOGB9WLGM5K~Rt13Wc-KtQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table2.xls?Expires=2147483647&amp;Signature=Mzrqx37fbA6JP1zzq-6ybS7aUg4N2~PRZ52hh7SOwa6t5qV62y-Q~jvMkCksGt~HREd5s-Y-TP73MDhJo7tPrYjkCST~6x4FlMTv5dKLAmPCp5pwGacsxONRgjWFPWsbPnwVGs928JYbDLRr-ICO0mgxuLHJYH7zjLnlv-viX8toP8d9CMBv5OB11fzxqbhYhVBtETWsXTNzwMaDJ3enHVXXkdf94yXkf1EwVEp2UqwFUW4fNJQ5om4dbN4wIQHeMUIE2H9AdmmX8WouoE~z8cJ12FzZWJ-9l3H2rV-kv~J7BtLEWAmc7fU97EzJbmPQ7agpx26l17GsqDeWxHtcJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp_table1.xls?Expires=2147483647&amp;Signature=SV0kzaCVCIjsyxyNr7wi0Ejm~nguKGQ3yj3xqxst6NajS7rqMn8h4vwYzMVwfLLbIW655qNsORS2XVRS1GPoMt-3tUJrqxN9~Yhy4wErV4cvmx-bXJ5EDmE~2dnodKAhPW8vVYY88IBJqSnG5u6xB6IigzaDNLmriMNtXSFzv~~yuH8ASzXgKtCKZkbzrDugDTxclKWy57Cs6QELxqalH1Fg9n8h9Pgn~W3HXKAMsJG7CizuXRhY~MaVgznoAJlzpkv9fzUU1hoZejXlVHIctNMvXcXqBvmniXWW0~bJZi6DHGHIYKRCHPsGfVSHRCvAUx3UqhHxzftlQEPVsZoqLA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst035/2/dst035-dst035supp.pdf?Expires=2147483647&amp;Signature=WStPiYeUcFj4hltvj~1FvpZtk5neGM~wFrvW8ZZIhoXa3MCl8QjgIc49bfvAS6I1s6pk1nsGmZfBDvctZGWh0pIz~jMCEurevJvu9tDKDbAdIJXXqhIVeWlXVLnc2GmRh5zk8~MYeizR0KA2M~za~~TztpuFQ4JCo032bMGf2ZF0SrpyZWBS2FYc9vQ3AN8uLBYClW00PLaP5uufg3xubThM0nEXwqaDi3HDk6RaPv3SYp3si4I6ggfaM~9ZhL6dLR1I7e45xn5vBI9uxTl2zWF-y6i6YxlFcA-ctSQRgzc7oEFQlLrhJLckVpt6H10VXyk4Ctx11jwVgb9C0L8gUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">611</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="346817"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f1%2f346817">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F1%2F346817" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/346817'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comprehensive Transcriptome Profiling and Functional Analysis of the Frog (Bombina maxima) Immune System" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f1%2f346817">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

