

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/4/155/466553","@type":"ScholarlyArticle","name":"Distribution of Stable DnaA-Binding Sites on the  Bacillus Subtilis  Genome Detected using a Modified ChIP-chip Method","datePublished":"2007-10-10","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm017","keywords":["ChIP-chip analysis","<italic>Bacillus subtilis</italic>","DnaA protein","DnaA box","high-density tiling chip"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/4/155/466553","author":[{"name":"Ishikawa, Shu","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Yoshimura, Mika","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Okumura, Hajime","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Cho, Eunha","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Kawai, Yoshikazu","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Oshima, Taku","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Ogasawara, Naotake","affiliation":"Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"}],"description":"Abstract.  We developed a modified ChIP-chip method, designated ChAP-chip (Chromatin Affinity Precipitation coupled with tiling chip). The binding sites of Baci","pageStart":"155","pageEnd":"168","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Distribution of Stable DnaA-Binding Sites on the  Bacillus Subtilis  Genome Detected using a Modified ChIP-chip Method","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Distribution of Stable DnaA-Binding Sites on the  Bacillus Subtilis  Genome Detected using a Modified ChIP-chip Method" />
    <meta property="og:description" content="Abstract.  We developed a modified ChIP-chip method, designated ChAP-chip (Chromatin Affinity Precipitation coupled with tiling chip). The binding sites of Baci" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/4/155/466553" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-10-10" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ishikawa, Shu" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Yoshimura, Mika" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Okumura, Hajime" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Cho, Eunha" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Kawai, Yoshikazu" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Oshima, Taku" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Ogasawara, Naotake" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_title" content="Distribution of Stable DnaA-Binding Sites on the Bacillus Subtilis Genome Detected using a Modified ChIP-chip Method" /><meta name="citation_firstpage" content="155" /><meta name="citation_lastpage" content="168" /><meta name="citation_doi" content="10.1093/dnares/dsm017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Building a bacterial orisome: emergence of new regulatory features for replication origin unwinding; Mol. Microbiol.;  citation_year=2005;  citation_volume=55;  citation_pages=978-985; " /><meta name="citation_reference" content="citation_title=DnaA: controlling the initiation of bacterial DNA replication and more; Annu. Rev. Microbiol.;  citation_year=2006;  citation_volume=60;  citation_pages=351-375; " /><meta name="citation_reference" content="citation_title=DNA replication initiation: mechanisms and regulation in bacteria; Nat. Rev. Microbiol.;  citation_year=2007;  citation_volume=5;  citation_pages=343-354; " /><meta name="citation_reference" content="citation_title=Regulation of the initiation of chromosomal replication in bacteria; FEMS Microbiol. Rev.;  citation_year=2007;  citation_volume=31;  citation_pages=378-387; " /><meta name="citation_reference" content="citation_title=Autoregulation of the DNA replication gene dnaA in E. coli K-12; Cell;  citation_year=1985;  citation_volume=40;  citation_pages=159-169; " /><meta name="citation_reference" content="citation_title=Autoregulation of the dnaA-dnaN operon and effects of DnaA protein levels on replication initiation in Bacillus subtilis; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=3833-3841; " /><meta name="citation_reference" content="citation_title=Structure and regulation of the dnaA promoter region in three Streptomyces species; Mol. Gen. Genet.;  citation_year=2000;  citation_volume=262;  citation_pages=1093-1102; " /><meta name="citation_reference" content="citation_title=Transcription analysis of the dnaA gene and oriC region of the chromosome of Mycobacterium smegmatis and Mycobacterium bovis BCG, and its regulation by the DnaA protein; Microbiology;  citation_year=2003;  citation_volume=149;  citation_pages=773-784; " /><meta name="citation_reference" content="citation_title=Conditional expression of Mycobacterium smegmatis dnaA, an essential DNA replication gene; Microbiology;  citation_year=2002;  citation_volume=148;  citation_pages=3887-3900; " /><meta name="citation_reference" content="citation_title=DnaA initiator—also a transcription factor; Mol. Microbiol.;  citation_year=1997;  citation_volume=24;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=In vitro transcription of the origin region of replication of the Escherichia coli chromosome; J. Biol. Chem.;  citation_year=1988;  citation_volume=263;  citation_pages=14176-14183; " /><meta name="citation_reference" content="citation_title=dnaA protein regulates transcriptions of the rpoH gene of Escherichia coli; J. Biol. Chem.;  citation_year=1989;  citation_volume=264;  citation_pages=7338-7344; " /><meta name="citation_reference" content="citation_title=DnaA protein stimulates polA gene expression in Escherichia coli; Mol. Microbiol.;  citation_year=1997;  citation_volume=23;  citation_pages=1193-1202; " /><meta name="citation_reference" content="citation_title=Correlation of gene transcription with the time of initiation of chromosome replication in Escherichia coli; Mol. Microbiol.;  citation_year=1993;  citation_volume=10;  citation_pages=575-584; " /><meta name="citation_reference" content="citation_title=Cell cycle-dependent transcription from the gid and mioC promoters of Escherichia coli; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=1609-1615; " /><meta name="citation_reference" content="citation_title=A novel regulatory mechanism couples deoxyribonucleotide synthesis and DNA replication in Escherichia coli; EMBO J.;  citation_year=2006;  citation_volume=25;  citation_pages=1137-1147; " /><meta name="citation_reference" content="citation_title=Escherichia coli Fis and DnaA proteins bind specifically to the nrd promoter region and affect expression of an nrd-lac fusion; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=378-387; " /><meta name="citation_reference" content="citation_title=Ribonucleotide reductase and the regulation of DNA replication: an old story and an ancient heritage; Mol. Microbiol.;  citation_year=2007;  citation_volume=63;  citation_pages=22-34; " /><meta name="citation_reference" content="citation_title=The datA locus predominantly contributes to the initiator titration mechanism in the control of replication initiation in Escherichia coli; Mol. Microbiol.;  citation_year=2002;  citation_volume=44;  citation_pages=1367-1375; " /><meta name="citation_reference" content="citation_title=Deletion of the datA site does not affect once-per-cell-cycle timing but induces rifampin-resistant replication; Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=3913-3920; " /><meta name="citation_reference" content="citation_title=Cluster of DnaA boxes involved in regulation of Streptomyces chromosome replication: from in silico to in vivo studies; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=6184-6194; " /><meta name="citation_reference" content="citation_title=Replication initiation proteins regulate a developmental checkpoint in Bacillus subtilis; Cell;  citation_year=2001;  citation_volume=104;  citation_pages=269-279; " /><meta name="citation_reference" content="citation_title=A transcriptional response to replication status mediated by the conserved bacterial replication protein DnaA; Proc. Natl. Acad. Sci. U.S.A.;  citation_year=2005;  citation_volume=102;  citation_pages=12932-12937; " /><meta name="citation_reference" content="citation_title=DnaA coordinates replication initiation and cell cycle transcription in Caulobacter crescentus; Mol. Microbiol.;  citation_year=2005;  citation_volume=58;  citation_pages=1340-1353; " /><meta name="citation_reference" content="citation_title=DnaA couples DNA replication and the expression of two cell cycle master regulators; EMBO J.;  citation_year=2006;  citation_volume=25;  citation_pages=346-356; " /><meta name="citation_reference" content="citation_title=Genomic approach for the understanding of dynamic aspect of chromosome behavior; Methods Enzymol.;  citation_year=2006;  citation_volume=409;  citation_pages=389-410; " /><meta name="citation_reference" content="citation_title=Escherichia coli histone-like protein H-NS preferentially binds to horizontally acquired DNA in association with RNA polymerase; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=141-153; " /><meta name="citation_reference" content="citation_title=Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella; Science;  citation_year=2006;  citation_volume=313;  citation_pages=236-238; " /><meta name="citation_reference" content="citation_title=H-NS mediates the silencing of laterally acquired genes in bacteria; PLoS Pathogens.;  citation_year=2006;  citation_volume=2;  citation_pages=746-752; " /><meta name="citation_reference" content="citation_title=A new FtsZ-interacting protein, YlmF, complements the activity of FtsA during progression of cell division in Bacillus subtilis; Mol. Microbiol.;  citation_year=2006;  citation_volume=60;  citation_pages=1364-1380; " /><meta name="citation_reference" content="citation_title=Identification and characterization of a bacterial chromosome partitioning site; Cell;  citation_year=1998;  citation_volume=92;  citation_pages=675-685; " /><meta name="citation_reference" content="citation_title=Structural basis for ATP-dependent DnaA assembly and replication-origin remodeling; Nat. Struct. Mol. Biol.;  citation_year=2006;  citation_volume=13;  citation_pages=676-683; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-256; " /><meta name="citation_reference" content="citation_title=Regulation of initiation of Bacillus subtilis chromosome replication; Plasmid;  citation_year=1999;  citation_volume=41;  citation_pages=17-29; " /><meta name="citation_reference" content="citation_title=Six GTP-binding proteins of the Era/Obg family are essential for cell growth in Bacillus subtilis; Microbiology;  citation_year=2002;  citation_volume=148;  citation_pages=3539-3552; " /><meta name="citation_reference" content="citation_title=Regulation and characterization of a newly deduced cell wall hydrolase gene (cwlJ) which affects germination of Bacillus subtilis spores; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=1375-1380; " /><meta name="citation_reference" content="citation_title=The bacterial chromosome segregation protein Spo0J spreads along DNA from parS nucleation sites; Mol. Microbiol.;  citation_year=2006;  citation_volume=61;  citation_pages=1352-1361; " /><meta name="citation_reference" content="citation_title=Structure and function of DnaA and the DnaA-box in eubacteria: evolutionary relationships of bacterial replication origins; Mol. Microbiol.;  citation_year=1991;  citation_volume=5;  citation_pages=2589-2597; " /><meta name="citation_reference" content="citation_title=Purification and characterization of an initiation protein for chromosomal replication, DnaA, in Bacillus subtilis; J. Biochem.;  citation_year=1990;  citation_volume=107;  citation_pages=732-739; " /><meta name="citation_reference" content="citation_title=The bacterial replication initiator DnaA. DnaA and oriC, the bacterial mode to initiate DNA replication; FEMS Microbiol. Rev.;  citation_year=2002;  citation_volume=26;  citation_pages=355-374; " /><meta name="citation_reference" content="citation_title=Structural basis of replication origin recognition by the DnaA protein; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=2077-2086; " /><meta name="citation_reference" content="citation_title=Where does bacterial replication start? Rules for predicting the oriC region; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=3781-3791; " /><meta name="citation_reference" content="citation_title=Insights into the quality of DnaA boxes and their cooperativity; J. Mol. Biol.;  citation_year=2006;  citation_volume=355;  citation_pages=85-95; " /><meta name="citation_reference" content="citation_title=Genomic transcriptional response to loss of chromosomal supercoiling in Escherichia coli; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R87" /><meta name="citation_reference" content="citation_title=Homeostatic regulation of supercoiling sensitivity coordinates transcription of the bacterial genome; EMBO Rep.;  citation_year=2006;  citation_volume=7;  citation_pages=710-715; " /><meta name="citation_reference" content="citation_title=The Initiator titration model: computer simulation of chromosome and minichromosome control; Res. Microbiol.;  citation_year=1991;  citation_volume=142;  citation_pages=161-167; " /><meta name="citation_reference" content="citation_title=Sequence characteristics required for cooperative binding and efficient in vivo titration of the replication initiator protein DnaA in E. coli; J. Mol. Biol.;  citation_year=2007;  citation_volume=367;  citation_pages=942-952; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/4/155/17462628/dsm017.pdf" /><meta name="description" content="Abstract.  We developed a modified ChIP-chip method, designated ChAP-chip (Chromatin Affinity Precipitation coupled with tiling chip). The binding sites of Baci" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/4/155/466553" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/4/155/466553" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/4/155/466553");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm017';
    </script>
<title>
	Distribution of Stable DnaA-Binding Sites on the Bacillus Subtilis Genome Detected using a Modified ChIP-chip Method | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./466553" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZdxPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpa2F3YSwgU2h1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2d1cmEsIFlvc2hpdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZb3NoaW11cmEsIE1pa2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa3VtdXJhLCBIYWppbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG8sIEV1bmhhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YWksIFlvc2hpa2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1cm9rYXdhLCBLZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIFRha3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ2FzYXdhcmEsIE5hb3Rha2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpc3RyaWJ1dGlvbiBvZiBTdGFibGUgRG5hQS1CaW5kaW5nIFNpdGVzIG9uIHRoZSBCYWNpbGx1cyBTdWJ0aWxpcyBHZW5vbWUgRGV0ZWN0ZWQgdXNpbmcgYSBNb2RpZmllZCBDaElQLWNoaXAgTWV0aG9kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc20wMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNy8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1aWxkaW5nIGEgYmFjdGVyaWFsIG9yaXNvbWU6IGVtZXJnZW5jZSBvZiBuZXcgcmVndWxhdG9yeSBmZWF0dXJlcyBmb3IgcmVwbGljYXRpb24gb3JpZ2luIHVud2luZGluZzsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9OTc4LTk4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hQTogY29udHJvbGxpbmcgdGhlIGluaXRpYXRpb24gb2YgYmFjdGVyaWFsIEROQSByZXBsaWNhdGlvbiBhbmQgbW9yZTsgQW5udS4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MzUxLTM3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHJlcGxpY2F0aW9uIGluaXRpYXRpb246IG1lY2hhbmlzbXMgYW5kIHJlZ3VsYXRpb24gaW4gYmFjdGVyaWE7IE5hdC4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zNDMtMzU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIHRoZSBpbml0aWF0aW9uIG9mIGNocm9tb3NvbWFsIHJlcGxpY2F0aW9uIGluIGJhY3RlcmlhOyBGRU1TIE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM3OC0zODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9yZWd1bGF0aW9uIG9mIHRoZSBETkEgcmVwbGljYXRpb24gZ2VuZSBkbmFBIGluIEUuIGNvbGkgSy0xMjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTE1OS0xNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9yZWd1bGF0aW9uIG9mIHRoZSBkbmFBLWRuYU4gb3Blcm9uIGFuZCBlZmZlY3RzIG9mIERuYUEgcHJvdGVpbiBsZXZlbHMgb24gcmVwbGljYXRpb24gaW5pdGlhdGlvbiBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz0zODMzLTM4NDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSBhbmQgcmVndWxhdGlvbiBvZiB0aGUgZG5hQSBwcm9tb3RlciByZWdpb24gaW4gdGhyZWUgU3RyZXB0b215Y2VzIHNwZWNpZXM7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9MTA5My0xMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGFuYWx5c2lzIG9mIHRoZSBkbmFBIGdlbmUgYW5kIG9yaUMgcmVnaW9uIG9mIHRoZSBjaHJvbW9zb21lIG9mIE15Y29iYWN0ZXJpdW0gc21lZ21hdGlzIGFuZCBNeWNvYmFjdGVyaXVtIGJvdmlzIEJDRywgYW5kIGl0cyByZWd1bGF0aW9uIGJ5IHRoZSBEbmFBIHByb3RlaW47IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz03NzMtNzg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25kaXRpb25hbCBleHByZXNzaW9uIG9mIE15Y29iYWN0ZXJpdW0gc21lZ21hdGlzIGRuYUEsIGFuIGVzc2VudGlhbCBETkEgcmVwbGljYXRpb24gZ2VuZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE0ODsgIGNpdGF0aW9uX3BhZ2VzPTM4ODctMzkwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hQSBpbml0aWF0b3LigJRhbHNvIGEgdHJhbnNjcmlwdGlvbiBmYWN0b3I7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gdml0cm8gdHJhbnNjcmlwdGlvbiBvZiB0aGUgb3JpZ2luIHJlZ2lvbiBvZiByZXBsaWNhdGlvbiBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBjaHJvbW9zb21lOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjM7ICBjaXRhdGlvbl9wYWdlcz0xNDE3Ni0xNDE4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZG5hQSBwcm90ZWluIHJlZ3VsYXRlcyB0cmFuc2NyaXB0aW9ucyBvZiB0aGUgcnBvSCBnZW5lIG9mIEVzY2hlcmljaGlhIGNvbGk7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI2NDsgIGNpdGF0aW9uX3BhZ2VzPTczMzgtNzM0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hQSBwcm90ZWluIHN0aW11bGF0ZXMgcG9sQSBnZW5lIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTE5My0xMjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBvZiBnZW5lIHRyYW5zY3JpcHRpb24gd2l0aCB0aGUgdGltZSBvZiBpbml0aWF0aW9uIG9mIGNocm9tb3NvbWUgcmVwbGljYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTc1LTU4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbCBjeWNsZS1kZXBlbmRlbnQgdHJhbnNjcmlwdGlvbiBmcm9tIHRoZSBnaWQgYW5kIG1pb0MgcHJvbW90ZXJzIG9mIEVzY2hlcmljaGlhIGNvbGk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9MTYwOS0xNjE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHJlZ3VsYXRvcnkgbWVjaGFuaXNtIGNvdXBsZXMgZGVveHlyaWJvbnVjbGVvdGlkZSBzeW50aGVzaXMgYW5kIEROQSByZXBsaWNhdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTEzNy0xMTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIEZpcyBhbmQgRG5hQSBwcm90ZWlucyBiaW5kIHNwZWNpZmljYWxseSB0byB0aGUgbnJkIHByb21vdGVyIHJlZ2lvbiBhbmQgYWZmZWN0IGV4cHJlc3Npb24gb2YgYW4gbnJkLWxhYyBmdXNpb247IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9Mzc4LTM4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umlib251Y2xlb3RpZGUgcmVkdWN0YXNlIGFuZCB0aGUgcmVndWxhdGlvbiBvZiBETkEgcmVwbGljYXRpb246IGFuIG9sZCBzdG9yeSBhbmQgYW4gYW5jaWVudCBoZXJpdGFnZTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MjItMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkYXRBIGxvY3VzIHByZWRvbWluYW50bHkgY29udHJpYnV0ZXMgdG8gdGhlIGluaXRpYXRvciB0aXRyYXRpb24gbWVjaGFuaXNtIGluIHRoZSBjb250cm9sIG9mIHJlcGxpY2F0aW9uIGluaXRpYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTM2Ny0xMzc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWxldGlvbiBvZiB0aGUgZGF0QSBzaXRlIGRvZXMgbm90IGFmZmVjdCBvbmNlLXBlci1jZWxsLWN5Y2xlIHRpbWluZyBidXQgaW5kdWNlcyByaWZhbXBpbi1yZXNpc3RhbnQgcmVwbGljYXRpb247IEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9MzkxMy0zOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyIG9mIERuYUEgYm94ZXMgaW52b2x2ZWQgaW4gcmVndWxhdGlvbiBvZiBTdHJlcHRvbXljZXMgY2hyb21vc29tZSByZXBsaWNhdGlvbjogZnJvbSBpbiBzaWxpY28gdG8gaW4gdml2byBzdHVkaWVzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTYxODQtNjE5NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljYXRpb24gaW5pdGlhdGlvbiBwcm90ZWlucyByZWd1bGF0ZSBhIGRldmVsb3BtZW50YWwgY2hlY2twb2ludCBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0yNjktMjc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRyYW5zY3JpcHRpb25hbCByZXNwb25zZSB0byByZXBsaWNhdGlvbiBzdGF0dXMgbWVkaWF0ZWQgYnkgdGhlIGNvbnNlcnZlZCBiYWN0ZXJpYWwgcmVwbGljYXRpb24gcHJvdGVpbiBEbmFBOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMjkzMi0xMjkzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG5hQSBjb29yZGluYXRlcyByZXBsaWNhdGlvbiBpbml0aWF0aW9uIGFuZCBjZWxsIGN5Y2xlIHRyYW5zY3JpcHRpb24gaW4gQ2F1bG9iYWN0ZXIgY3Jlc2NlbnR1czsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTM0MC0xMzUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFBIGNvdXBsZXMgRE5BIHJlcGxpY2F0aW9uIGFuZCB0aGUgZXhwcmVzc2lvbiBvZiB0d28gY2VsbCBjeWNsZSBtYXN0ZXIgcmVndWxhdG9yczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM0Ni0zNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgYXBwcm9hY2ggZm9yIHRoZSB1bmRlcnN0YW5kaW5nIG9mIGR5bmFtaWMgYXNwZWN0IG9mIGNocm9tb3NvbWUgYmVoYXZpb3I7IE1ldGhvZHMgRW56eW1vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBoaXN0b25lLWxpa2UgcHJvdGVpbiBILU5TIHByZWZlcmVudGlhbGx5IGJpbmRzIHRvIGhvcml6b250YWxseSBhY3F1aXJlZCBETkEgaW4gYXNzb2NpYXRpb24gd2l0aCBSTkEgcG9seW1lcmFzZTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNDEtMTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3RpdmUgc2lsZW5jaW5nIG9mIGZvcmVpZ24gRE5BIHdpdGggbG93IEdDIGNvbnRlbnQgYnkgdGhlIEgtTlMgcHJvdGVpbiBpbiBTYWxtb25lbGxhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTIzNi0yMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUgtTlMgbWVkaWF0ZXMgdGhlIHNpbGVuY2luZyBvZiBsYXRlcmFsbHkgYWNxdWlyZWQgZ2VuZXMgaW4gYmFjdGVyaWE7IFBMb1MgUGF0aG9nZW5zLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NzQ2LTc1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgRnRzWi1pbnRlcmFjdGluZyBwcm90ZWluLCBZbG1GLCBjb21wbGVtZW50cyB0aGUgYWN0aXZpdHkgb2YgRnRzQSBkdXJpbmcgcHJvZ3Jlc3Npb24gb2YgY2VsbCBkaXZpc2lvbiBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MTM2NC0xMzgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhIGJhY3RlcmlhbCBjaHJvbW9zb21lIHBhcnRpdGlvbmluZyBzaXRlOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9Njc1LTY4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBiYXNpcyBmb3IgQVRQLWRlcGVuZGVudCBEbmFBIGFzc2VtYmx5IGFuZCByZXBsaWNhdGlvbi1vcmlnaW4gcmVtb2RlbGluZzsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NzYtNjgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBncmFtLXBvc2l0aXZlIGJhY3Rlcml1bSBCYWNpbGx1cyBzdWJ0aWxpczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS0yNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgaW5pdGlhdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBjaHJvbW9zb21lIHJlcGxpY2F0aW9uOyBQbGFzbWlkOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTctMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpeCBHVFAtYmluZGluZyBwcm90ZWlucyBvZiB0aGUgRXJhL09iZyBmYW1pbHkgYXJlIGVzc2VudGlhbCBmb3IgY2VsbCBncm93dGggaW4gQmFjaWxsdXMgc3VidGlsaXM7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz0zNTM5LTM1NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBuZXdseSBkZWR1Y2VkIGNlbGwgd2FsbCBoeWRyb2xhc2UgZ2VuZSAoY3dsSikgd2hpY2ggYWZmZWN0cyBnZXJtaW5hdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBzcG9yZXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9MTM3NS0xMzgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFjdGVyaWFsIGNocm9tb3NvbWUgc2VncmVnYXRpb24gcHJvdGVpbiBTcG8wSiBzcHJlYWRzIGFsb25nIEROQSBmcm9tIHBhclMgbnVjbGVhdGlvbiBzaXRlczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MTM1Mi0xMzYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uIG9mIERuYUEgYW5kIHRoZSBEbmFBLWJveCBpbiBldWJhY3RlcmlhOiBldm9sdXRpb25hcnkgcmVsYXRpb25zaGlwcyBvZiBiYWN0ZXJpYWwgcmVwbGljYXRpb24gb3JpZ2luczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yNTg5LTI1OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1cmlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhbiBpbml0aWF0aW9uIHByb3RlaW4gZm9yIGNocm9tb3NvbWFsIHJlcGxpY2F0aW9uLCBEbmFBLCBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgSi4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NzMyLTczOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhY3RlcmlhbCByZXBsaWNhdGlvbiBpbml0aWF0b3IgRG5hQS4gRG5hQSBhbmQgb3JpQywgdGhlIGJhY3RlcmlhbCBtb2RlIHRvIGluaXRpYXRlIEROQSByZXBsaWNhdGlvbjsgRkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zNTUtMzc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGJhc2lzIG9mIHJlcGxpY2F0aW9uIG9yaWdpbiByZWNvZ25pdGlvbiBieSB0aGUgRG5hQSBwcm90ZWluOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yMDc3LTIwODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdoZXJlIGRvZXMgYmFjdGVyaWFsIHJlcGxpY2F0aW9uIHN0YXJ0PyBSdWxlcyBmb3IgcHJlZGljdGluZyB0aGUgb3JpQyByZWdpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTM3ODEtMzc5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byB0aGUgcXVhbGl0eSBvZiBEbmFBIGJveGVzIGFuZCB0aGVpciBjb29wZXJhdGl2aXR5OyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM1NTsgIGNpdGF0aW9uX3BhZ2VzPTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHRyYW5zY3JpcHRpb25hbCByZXNwb25zZSB0byBsb3NzIG9mIGNocm9tb3NvbWFsIHN1cGVyY29pbGluZyBpbiBFc2NoZXJpY2hpYSBjb2xpOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVI4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbWVvc3RhdGljIHJlZ3VsYXRpb24gb2Ygc3VwZXJjb2lsaW5nIHNlbnNpdGl2aXR5IGNvb3JkaW5hdGVzIHRyYW5zY3JpcHRpb24gb2YgdGhlIGJhY3RlcmlhbCBnZW5vbWU7IEVNQk8gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NzEwLTcxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEluaXRpYXRvciB0aXRyYXRpb24gbW9kZWw6IGNvbXB1dGVyIHNpbXVsYXRpb24gb2YgY2hyb21vc29tZSBhbmQgbWluaWNocm9tb3NvbWUgY29udHJvbDsgUmVzLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTE2MS0xNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGNoYXJhY3RlcmlzdGljcyByZXF1aXJlZCBmb3IgY29vcGVyYXRpdmUgYmluZGluZyBhbmQgZWZmaWNpZW50IGluIHZpdm8gdGl0cmF0aW9uIG9mIHRoZSByZXBsaWNhdGlvbiBpbml0aWF0b3IgcHJvdGVpbiBEbmFBIGluIEUuIGNvbGk7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzY3OyAgY2l0YXRpb25fcGFnZXM9OTQyLTk1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE0LzQvMTU1LzE3NDYyNjI4L2RzbTAxNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgZGV2ZWxvcGVkIGEgbW9kaWZpZWQgQ2hJUC1jaGlwIG1ldGhvZCwgZGVzaWduYXRlZCBDaEFQLWNoaXAgKENocm9tYXRpbiBBZmZpbml0eSBQcmVjaXBpdGF0aW9uIGNvdXBsZWQgd2l0aCB0aWxpbmcgY2hpcCkuIFRoZSBiaW5kaW5nIHNpdGVzIG9mIEJhY2kiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ2NjU1MyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSWlhhzBAIx4n5p3o+22LogsvmZqg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f155%2f466553" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f155%2f466553" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18275" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&amp;Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/4">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195293">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195293">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195295">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195295">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195299">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195299">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195312">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195312">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195331">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195331">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195339">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195339">Supplementry Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195341">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195341">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195343">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195343">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/4/141/466067">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/4/169/466883">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Distribution of Stable DnaA-Binding Sites on the <em>Bacillus Subtilis</em> Genome Detected using a Modified ChIP-chip Method
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shu Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shu+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shu Ishikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mika Yoshimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mika+Yoshimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshimura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshimura M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mika Yoshimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hajime Okumura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hajime+Okumura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okumura H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okumura H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hajime Okumura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eunha Cho</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eunha+Cho">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cho E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cho E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eunha Cho</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Kawai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Kawai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshikazu Kawai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taku Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taku+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taku Oshima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naotake Ogasawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspond should be addressed. Tel. +81 743-72-5430. Fax. <div class="fax">+81 743-72-5439</div>. E-mail: <a href="mailto:nogasawa@bs.naist.jp" target="_blank">nogasawa@bs.naist.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naotake+Ogasawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogasawara N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogasawara N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naotake Ogasawara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Current address: Frontier Science Research Center, University of Miyazaki, 5200 Kihara, Kiyotake, Miyazaki 889-1692, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Current address: Faculty of Medical Sciences, Institute for Cell and Molecular Biosciences, University of Newcastle upon Tyne, Framlington Place, NE2 4HH, UK.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 155–168, <a href='https://doi.org/10.1093/dnares/dsm017'>https://doi.org/10.1093/dnares/dsm017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">10 October 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 July 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">10 September 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="466553" href="/dnaresearch/article-pdf/14/4/155/17462628/dsm017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shu Ishikawa, Yoshitoshi Ogura, Mika Yoshimura, Hajime Okumura, Eunha Cho, Yoshikazu Kawai, Ken Kurokawa, Taku Oshima, Naotake Ogasawara;  Distribution of Stable DnaA-Binding Sites on the <em>Bacillus Subtilis</em> Genome Detected using a Modified ChIP-chip Method, <em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 155–168, <a href='https://doi.org/10.1093/dnares/dsm017'>https://doi.org/10.1093/dnares/dsm017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/466553?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/466553?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Distribution%20of%20Stable%20DnaA-Binding%20Sites%20on%20the%20%26lt%3Bem%26gt%3BBacillus%20Subtilis%26lt%3B%2Fem%26gt%3B%20Genome%20Detected%20using%20a%20Modified%20ChIP-chip%20Method&amp;publicationDate=2007-10-10&amp;volumeNum=14&amp;issueNum=4&amp;author=Ishikawa%2C%20Shu%3B%20Ogura%2C%20Yoshitoshi&amp;startPage=155&amp;endPage=168&amp;contentId=10.1093%2Fdnares%2Fdsm017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f155%2f466553" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195293 id="87195293" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We developed a modified ChIP-chip method, designated ChAP-chip (Chromatin Affinity Precipitation coupled with tiling chip). The binding sites of <em>Bacillus subtilis</em> Spo0J determined using this technique were consistent with previous findings. A DNA replication initiator protein, DnaA, formed stable complexes at eight intergenic regions on the <em>B. subtilis</em> genome. Characterization of the binding sequences suggested that two factors—the local density of DnaA boxes and their affinities for DnaA—are critical for stable binding. We further showed that in addition to autoregulation, DnaA directly modulate the expression of <em>sda</em> in a positive, and <em>ywlC</em> and <em>yydA</em> in a negative manner. Examination of possible stable DnaA-binding sequences in other <em>Bacillus</em> species suggested that DnaA-dependent regulation of those genes is maintained in most bacteria examined, supporting their biological significance. In addition, a possible stable DnaA-binding site downstream of <em>gcp</em> is also suggested to be conserved. Furthermore, potential DnaA-binding sequences specific for each bacterium have been identified, generally in close proximity to <em>oriC</em>. These findings suggest that DnaA plays several additional roles, such as control of the level of effective initiator, ATP-DnaA, and/or stabilization of the domain structure of the genome around <em>oriC</em> for the proper initiation of chromosome replication.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ChIP-chip analysis&quot;">ChIP-chip analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Bacillus subtilis&quot;"><em>Bacillus subtilis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DnaA protein&quot;">DnaA protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DnaA box&quot;">DnaA box</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-density tiling chip&quot;">high-density tiling chip</a></div>                            <h2 scrollto-destination=87195295 id="87195295" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The replication initiation protein of eubacteria, DnaA, binds to an asymmetrical 9 bp consensus sequence (TTATNCACA, DnaA box) that is repeated several times in the replication origin (<em>oriC</em>) region, and plays a central role in the formation of the initiation complex for chromosome replication (reviewed by Leonard and Grimwade,<sup><a reveal-id="DSM017C1" data-open="DSM017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Kaguni,<sup><a reveal-id="DSM017C2" data-open="DSM017C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Mott and Berger,<sup><a reveal-id="DSM017C3" data-open="DSM017C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and Zakrzewska-Czerwinska <em>et al.</em><sup><a reveal-id="DSM017C4" data-open="DSM017C4" class="link link-ref link-reveal xref-bibr">4</a></sup>). Additionally, the protein autogenously represses its own expression to maintain the cellular level through binding to DnaA boxes downstream of the <em>dnaA</em> promoter in <em>Escherichia coli</em>,<sup><a reveal-id="DSM017C5" data-open="DSM017C5" class="link link-ref link-reveal xref-bibr">5</a></sup><em>Bacillus subtilis</em>,<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup><em>Streptomyces lividance</em>,<sup><a reveal-id="DSM017C7" data-open="DSM017C7" class="link link-ref link-reveal xref-bibr">7</a></sup><em>Mycobacterium smegmatis</em>, and <em>M. bovis.</em><sup><a reveal-id="DSM017C8" data-open="DSM017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The timing of chromosome replication initiation is tightly regulated, and involves the cellular DnaA level in <em>E. coli</em> (reviewed by Kaguni<sup><a reveal-id="DSM017C2" data-open="DSM017C2" class="link link-ref link-reveal xref-bibr">2</a></sup>), <em>B. subtilis</em><sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and <em>M. smegmatis</em>.<sup><a reveal-id="DSM017C9" data-open="DSM017C9" class="link link-ref link-reveal xref-bibr">9</a></sup> However, the DnaA box sequence is not restricted to the <em>oriC</em> and <em>dnaA</em> promoter regions. A search for the consensus sequence, TTATNCACA, allowing a one base mismatch along the complete genome sequence of <em>E. coli</em> and <em>B. subtilis</em>, reveals the existence of 3742 and 4342 sites, respectively.</p><p>Indeed, several genes are possibly or conclusively regulated directly by DnaA in <em>E. coli</em> (reviewed by Messer and Weigel).<sup><a reveal-id="DSM017C10" data-open="DSM017C10" class="link link-ref link-reveal xref-bibr">10</a></sup> DnaA negatively regulates <em>mioC</em>, <em>rpoH</em>, <em>uvrB</em>, and <em>proS</em>,<sup><a reveal-id="DSM017C5" data-open="DSM017C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM017C11" data-open="DSM017C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSM017C12" data-open="DSM017C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and positively regulates <em>polA</em> expression.<sup><a reveal-id="DSM017C13" data-open="DSM017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Levels of <em>mioC</em> and <em>polA</em> are controlled in a growth phase-dependent manner, and DnaA possibly contributes to the coupling of expression to growth rate.<sup><a reveal-id="DSM017C13  DSM017C15" data-open="DSM017C13  DSM017C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup> A recent study proposes that repression of <em>nrdAB</em> expression by ATP-bound DnaA is associated with coupling of the initiation frequency to elongation rate of chromosome replication,<sup><a reveal-id="DSM017C16" data-open="DSM017C16" class="link link-ref link-reveal xref-bibr">16</a></sup> although DnaA has been demonstrated previously to regulate positively <em>nrdAB</em> promoter.<sup><a reveal-id="DSM017C17" data-open="DSM017C17" class="link link-ref link-reveal xref-bibr">17</a></sup> DnaA might have a dual role in modulating <em>nrdAB</em> expression; low levels of DnaA-ATP stimulate the transcription whereas high levels repress it.<sup><a reveal-id="DSM017C18" data-open="DSM017C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In addition, a high-affinity DnaA-binding sequence containing five DnaA boxes, <em>dat</em> (DnaA Titration), has been identified.<sup><a reveal-id="DSM017C19" data-open="DSM017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Multicopy plasmids harboring the <em>dat</em> sequence interfere with the normal replication cycle, although its exact role is unclear.<sup><a reveal-id="DSM017C20" data-open="DSM017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> A similar high-affinity DnaA-binding region is involved in regulating the initiation of chromosome replication in <em>S. coelicolor</em>.<sup><a reveal-id="DSM017C21" data-open="DSM017C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In <em>B. subtilis</em>, DnaA functions in repressing the initiation of sporulation in actively replicating cells by controlling the expression of Sda that blocks the signal transduction cascade leading to sporulation.<sup><a reveal-id="DSM017C22" data-open="DSM017C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The sequence upstream of the <em>sda</em> promoter includes five DnaA boxes, and DnaA binding to these is essential for <em>sda</em> expression. Transcriptome analysis of replication-inhibited cells and chromatin immunoprecipitation assays reveal that <em>yllB</em> (<em>ftsL</em>), <em>dnaB</em>, <em>ywlC</em>, and <em>yydA</em> are negatively regulated by DnaA binding to their promoter regions,<sup><a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> suggesting that the protein is involved in tolerance against replication stress as a transcriptional regulator. Furthermore, DnaA acts as a transcriptional regulator in <em>Caulobacter crescentus</em>.<sup><a reveal-id="DSM017C24" data-open="DSM017C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSM017C25" data-open="DSM017C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In this case, DnaA stimulates the expression of a master regulator of cell cycle progression, <em>gcrA</em>, which in turn activates multiple genes, including those involved in chromosome replication and segregation, coordinating the initiation of chromosome replication and cell cycle progression.</p><p>Thus, in addition to the initiation of replication and autoregulation of its expression, DnaA appears to participate in the coordination between initiation of chromosome replication and cell cycle progression. However, genome-wide identification of DnaA-binding sites <em>in vivo</em> has not yet been attempted. ChIP-chip analysis, a powerful tool for monitoring the distribution of genome-associated proteins <em>in vivo</em>, employs chromatin immunoprecipitation in combination with a high-resolution oligonucleotide tiling chip.<sup><a reveal-id="DSM017C26" data-open="DSM017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> For instance, we and other groups recently reported the distribution of nucleoid-associated protein, H-NS, on the <em>E. coli</em> and <em>Salmonella</em> genomes, and found that H-NS preferentially binds to and represses horizontally acquired genes.<sup><a reveal-id="DSM017C27  DSM017C29" data-open="DSM017C27  DSM017C29" class="link link-ref link-reveal xref-bibr">27–29</a></sup> However, ChIP-chip analysis requires a specific antibody that efficiently recognizes the target protein in protein–DNA complexes, which may limit its applicability. Previously, we developed a method to isolate protein complexes with high purity, using a 12 × Histidine-tag under denatured conditions.<sup><a reveal-id="DSM017C30" data-open="DSM017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> To improve versatility, this procedure was adapted for the purification of protein–DNA complexes, denoted ChAP-chip (Chromatin Affinity Precipitation coupled with high density tiling chip). The binding sites of Spo0J determined with the modified technique were consistent with previous reports.<sup><a reveal-id="DSM017C31" data-open="DSM017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Clearly, DnaA forms stable complexes at eight intergenic regions on the <em>B. subtilis</em> genome, including <em>oriC</em>, <em>oriC</em>1, and <em>oriC</em>2. Characterization of the DnaA-binding sequences revealed that two factors—local density of the DnaA boxes and their affinities to DnaA—are critical for stable binding. Thus, filament formation of multiple ATP-bound DnaA proposed for the bacterial <em>oriC</em>–DnaA complex<sup><a reveal-id="DSM017C32" data-open="DSM017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> might occur at the stable-binding sites. In combination with transcriptome analysis of cells containing increased or decreased DnaA amounts, our data show that DnaA proteins stably associated with target DNA directly regulate the expression of <em>sda</em> in a positive manner, and <em>ywlC</em> and <em>yydA</em> in a negative manner. Examination of possible stable DnaA-binding sequences in genome sequences of other <em>Bacillus</em> species suggested that DnaA-dependent regulation of those genes is maintained in most of bacteria examined, supporting their biological significance. In addition, a possible stable DnaA-binding site downstream of <em>gcp</em> is also suggested to be conserved. Furthermore, potential DnaA-binding sequences specific for each bacterium have been identified, generally in close proximity to the <em>oriC</em> region. These findings suggest that DnaA plays several additional roles, such as control of the level of effective initiator, ATP-DnaA, and/or stabilization of the domain structure of the genome around <em>oriC</em> for the proper initiation of chromosome replication.</p>                            <h2 scrollto-destination=87195299 id="87195299" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195300 id="87195300" class="section-title" data-legacy-id=s2a>2.1. Design of the B. subtilis tiling chip</h3>
<p>We retrieved the <em>B. subtilis</em> genome sequence and other genetic information from the SubtiList database (<a class="link link-uri" href="http://genolist.pasteur.fr/SubtiList/">http://genolist.pasteur.fr/SubtiList/</a>) for the design of probes, according to the Affymetrix guidelines. Probe sequences were selected for protein coding and intergenic regions using different criteria.<sup><a reveal-id="DSM017C27" data-open="DSM017C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Apart from the <em>rrnA</em> operon, rRNA genes were excluded from probe design. In addition, <em>skin</em> element, two prophage (PNSX and spβ), and seven prophage-like sequences<sup><a reveal-id="DSM017C33" data-open="DSM017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> were excluded. The tRNA genes and those encoding proteins with lengths of &lt;100 amino acids were taken as intergenic regions. As a result, 55 430 25-mer sequences on the coding strand were selected for protein coding regions at 25–30 bp intervals, and 72 218 sequences on both strands were selected for intergenic regions at 2–3 bp intervals. In total, 127 648 probe sets (perfect match and mismatch probes) were synthesized on the chip. Chip probe information is available on our website (<a class="link link-uri" href="http://genome.naist.jp/bacteria/array/ecol.html">http://genome.naist.jp/bacteria/array/ecol.html</a>).</p>                            <h3 scrollto-destination=87195302 id="87195302" class="section-title" data-legacy-id=s2b>2.2. Construction of strains expressing C-terminally histidine-tagged proteins</h3>
<p>Primers used for generating plasmids used in this study are presented in Supplementary Table S1.</p><p>To construct the pMUTinHis-▵spo0J plasmid, a 250 bp fragment encompassing the 3′-region of the <em>spo0J</em> gene (except the stop codon) was amplified by PCR from <em>B. subtilis</em> 168 genomic DNA using the primer set, spo0J.f–spo0J.r, and cloned between <em>Eco</em>RI–<em>Xho</em>I sites of pMUTinHis.<sup><a reveal-id="DSM017C30" data-open="DSM017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The resultant plasmid was integrated into the <em>B. subtilis</em> chromosome by single crossover to generate <em>B. subtilis</em> strains expressing C-terminal histidine-tagged Spo0J (strain SI002).</p><p>The <em>dnaA</em> gene constitutes an operon with the downstream <em>dnaN</em> gene and the intergenic region (<em>oriC2</em>) acts as the <em>B. subtilis oriC</em>, together with a sequence upstream of <em>dnaA</em> (<em>oriC1</em>).<sup><a reveal-id="DSM017C34" data-open="DSM017C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We generated a strain expressing C-terminal histidine-tagged DnaA without plasmid integration between <em>dnaA</em> and <em>dnaN</em> (strain SI003), as schematically shown in Fig. <a reveal-id="DSM017F1" data-open="DSM017F1" class="link link-reveal link-table xref-fig">1</a>. A DNA fragment containing the 3′-region of <em>dnaA</em>, except the stop codon, was amplified using the primers, 1dnaAHiF and 1dnaALinkR, with a partial 12 × Histidine (His-tag) coding sequence (<em>dnaA</em>-side fragment). A fragment containing the His-tag, followed by the stop codon of <em>dnaA</em> and its 500 bp downstream sequence (<em>dnaN</em>-side fragment) was generated by two-step adaptor PCR, using the primer sets, 2dnaAHisF–2dnaABaR, for initial amplification, and Linker12His–2dnaABaR, for the second PCR. The two fragments were ligated by recombinant PCR using the 1dnaAHiF–2dnaABaR primer set, and cloned within <em>Hin</em>dIII and <em>Bam</em>HI sites of pMUTinNC.<sup><a reveal-id="DSM017C35" data-open="DSM017C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Since the <em>dnaN</em> gene essential for cell growth was placed under control of the IPTG-inducible <em>spac</em> promoter in the resultant transformants, pMUTinAhisN was used for transformation of <em>B. subtilis</em> 168 by single crossover with selection on LB medium containing 0.5 µg/ml erythromycin and 1 mM IPTG (first step in Fig. <a reveal-id="DSM017F1" data-open="DSM017F1" class="link link-reveal link-table xref-fig">1</a>B). To induce a second recombination event for removal of the integrated plasmid, the resultant strain was cultured in competent medium containing 1 mM IPTG. Cells that grew in the absence of IPTG were selected on LB medium without erythromycin and IPTG (second step in Fig. <a reveal-id="DSM017F1" data-open="DSM017F1" class="link link-reveal link-table xref-fig">1</a>B). Cells with insertion of the His-tag sequence were selected from IPTG-independent and erythromycin-sensitive colonies by examining amplification using primers complementary to the His-tag sequence (Linker12His) and a <em>dnaN</em> coding sequence (2dnaAbaR), and chromosomal DNA extracted from these colonies. Finally, we confirmed the addition of the 12 × His coding sequence at the 3′ end of <em>dnaA</em> and removal of integrated plasmid by sequencing and the resultant strain expressing the His-tagged DnaA was designated SI003. Negligible changes if any in synchrony of initiation of chromosome replication in SI003 cells were confirmed by flow cytometory (Supplementary Fig. S1).</p>                        <a id="87195306" scrollto-destination="87195306"></a>
<div id="DSM017F1" data-id="DSM017F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01701.jpeg?Expires=2147483647&Signature=3wGSiLG1InHvVdeOF4c9pfW7AokuhN4IErjY-FkT4yiV7fYbKZ6EiQzItdVGXcTdaqHGmelVPwFqOppV5erM1hLQpC-hlTniSiAZlF7HHDJxgTHJi4eP9JzIRr9PrTDZ13WFF2Hvq4mCG7e0VqpJI2UcHSwVIgvZs6imgI-4ydxNeEsatHgqYja8OdyNbAf7oCQmgw44sCBqYl~YPfiVE7nhEgmltSRDK3ngWM4NlhnUGXnIv~z-TX4Q7Yl8GXiy61G3YDx7vsJ-hePSWwU7Ws7sJNsyYcfulP7oi9ug-SyjmiDacmZQfcmEPsPFt9MDqKPq2z5Jsx2xCM9KqZhT1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the generation of strain SI003. (A) Adapter PCR and recombinant PCR products used to create the plasmid, pMUTinAhisN. (B) Transformation of 168 cells with pMUTinAhisN and removal of the integrated plasmid to generate strain SI003. Integration of the plasmid into the chromosome through recombination at region a or b resulted in cells with structure 1 or 2, respectively. Induction of recombination at region c or d in cells with structure 1 resulted in cells with structure 3 or 4, respectively. On the other hand, recombination at region e or f in cells with structure 2 resulted in cells having structure 4 or 3, respectively. Cells displaying structure 4 were selected for ChAP-chip analysis." path-from-xml="dsm01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195306/dsm01701.jpeg" path-from-xml="dsm01701.jpeg" target="_blank">View large</a><a section="87195306" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01701.jpeg?Expires=2147483647&Signature=XdIQ~feVMdlctcEY2SokHP0KBk9Tq94ukkyU9N31Nv562TEUJCQIyJcfGW~-F1fQ9i2L5b2y9byiHU8JWWKTfFZ4S7QIuowrzMCcpXX0uSq-TFCnpQEnqggYHS0W-~lL1OYXyRXt87QxSyqFCwM0MgpvlnHYyyM213EZDR2dpA~jcnYaSEk1Q2tozBjW6wZNJ-YTipjLiooLKacGnBrettLGTVK3dVwDH6HnYVm1dskR5EKq4CRdmRwnJvEamf~xHB8gqy0MOdslV91AABdjN5yR~Y75QuRKFcMXloCEvK-PR2j~TpnoldK0Z5tN-I1Q1bGYsH2KhHslzbDotz5l2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195306&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the generation of strain SI003. (<strong>A</strong>) Adapter PCR and recombinant PCR products used to create the plasmid, pMUTinAhisN. (<strong>B</strong>) Transformation of 168 cells with pMUTinAhisN and removal of the integrated plasmid to generate strain SI003. Integration of the plasmid into the chromosome through recombination at region a or b resulted in cells with structure 1 or 2, respectively. Induction of recombination at region c or d in cells with structure 1 resulted in cells with structure 3 or 4, respectively. On the other hand, recombination at region e or f in cells with structure 2 resulted in cells having structure 4 or 3, respectively. Cells displaying structure 4 were selected for ChAP-chip analysis.</p></div></div><div content-id="DSM017F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01701.jpeg?Expires=2147483647&Signature=3wGSiLG1InHvVdeOF4c9pfW7AokuhN4IErjY-FkT4yiV7fYbKZ6EiQzItdVGXcTdaqHGmelVPwFqOppV5erM1hLQpC-hlTniSiAZlF7HHDJxgTHJi4eP9JzIRr9PrTDZ13WFF2Hvq4mCG7e0VqpJI2UcHSwVIgvZs6imgI-4ydxNeEsatHgqYja8OdyNbAf7oCQmgw44sCBqYl~YPfiVE7nhEgmltSRDK3ngWM4NlhnUGXnIv~z-TX4Q7Yl8GXiy61G3YDx7vsJ-hePSWwU7Ws7sJNsyYcfulP7oi9ug-SyjmiDacmZQfcmEPsPFt9MDqKPq2z5Jsx2xCM9KqZhT1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the generation of strain SI003. (A) Adapter PCR and recombinant PCR products used to create the plasmid, pMUTinAhisN. (B) Transformation of 168 cells with pMUTinAhisN and removal of the integrated plasmid to generate strain SI003. Integration of the plasmid into the chromosome through recombination at region a or b resulted in cells with structure 1 or 2, respectively. Induction of recombination at region c or d in cells with structure 1 resulted in cells with structure 3 or 4, respectively. On the other hand, recombination at region e or f in cells with structure 2 resulted in cells having structure 4 or 3, respectively. Cells displaying structure 4 were selected for ChAP-chip analysis." path-from-xml="dsm01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195306/dsm01701.jpeg" path-from-xml="dsm01701.jpeg" target="_blank">View large</a><a section="87195306" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01701.jpeg?Expires=2147483647&Signature=XdIQ~feVMdlctcEY2SokHP0KBk9Tq94ukkyU9N31Nv562TEUJCQIyJcfGW~-F1fQ9i2L5b2y9byiHU8JWWKTfFZ4S7QIuowrzMCcpXX0uSq-TFCnpQEnqggYHS0W-~lL1OYXyRXt87QxSyqFCwM0MgpvlnHYyyM213EZDR2dpA~jcnYaSEk1Q2tozBjW6wZNJ-YTipjLiooLKacGnBrettLGTVK3dVwDH6HnYVm1dskR5EKq4CRdmRwnJvEamf~xHB8gqy0MOdslV91AABdjN5yR~Y75QuRKFcMXloCEvK-PR2j~TpnoldK0Z5tN-I1Q1bGYsH2KhHslzbDotz5l2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195306&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the generation of strain SI003. (<strong>A</strong>) Adapter PCR and recombinant PCR products used to create the plasmid, pMUTinAhisN. (<strong>B</strong>) Transformation of 168 cells with pMUTinAhisN and removal of the integrated plasmid to generate strain SI003. Integration of the plasmid into the chromosome through recombination at region a or b resulted in cells with structure 1 or 2, respectively. Induction of recombination at region c or d in cells with structure 1 resulted in cells with structure 3 or 4, respectively. On the other hand, recombination at region e or f in cells with structure 2 resulted in cells having structure 4 or 3, respectively. Cells displaying structure 4 were selected for ChAP-chip analysis.</p></div></div>                            <h3 scrollto-destination=87195307 id="87195307" class="section-title" data-legacy-id=s2c>2.3. ChAP-chip analysis</h3>
<p>An overnight culture of <em>B. subtilis</em> cells expressing histidine-tagged protein in LB liquid medium at 37°C was inoculated into 400 mL LB medium to obtain an initial OD<sub>600</sub> value of 0.01. During SI002 culture at 37°C, 0.5 µg/mL erythromycin and 1 mM IPTG were added at every step. At OD<sub>600</sub> of 0.4, the culture was treated with formaldehyde (1% final concentration) for 30 min. Cells were washed with TBS buffer (pH 7.5), and stored at −80°C until use. Next, cells were disrupted by sonication on ice in 3 mL of UT buffer (100 mM HEPES, 50 mM imidazole, 8 M urea, 0.5 M NaCl, 1% Triton X-100, 1 mM DTT, 1 mM PMSF, pH 7.4). After centrifugation at 8000 rpm for 10 min, 200 µL of MagneHis (Promega) was added to the supernatant, followed by overnight incubation at room temperature with gentle shaking. MagneHis was washed five times with UT buffer, and bound proteins eluted with 400 µL of elution buffer (100 mM Tris–HCl, pH 7.5, 0.5 M imidazole, 1% SDS, 10 mM DTT). The eluate was passed through Microcon-100 (Millipore) to remove non-specifically bound uncrossed-linked proteins with a molecular mass lower than 100 kDa. Protein complexes retained on the membrane were washed three times with wash buffer (100 mM Tris–HCl, pH 7.5, 1% SDS, 10 mM DTT), and recovered by the addition of 50 µL buffer. Cross-links were dissociated by heating at 65°C overnight, and DNA purified using Qiaquick (QIAGEN).</p><p>Terminal labeling of purified DNA fragments and hybridization to the oligonucleotide chip were performed essentially as described previously.<sup><a reveal-id="DSM017C27" data-open="DSM017C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Signal intensities of mismatch probes were subtracted from those of perfect match probes. The signal intensities of DNA in the affinity-purified fraction and those of DNA isolated from the whole cell extract fraction before purification (control DNA) were adjusted to confer a signal average of 500. Signal intensities of DNA in the affinity-purified fraction were subtracted by those of control DNA to obtain protein-binding signals. The distribution of protein-binding signals along the genome coordinate was visualized with the <em>In Silico</em> Molecular Cloning program, Array Edition (<em>In Silico</em> Biology).</p>                            <h3 scrollto-destination=87195310 id="87195310" class="section-title" data-legacy-id=s2d>2.4. High-resolution transcriptome analysis</h3>
<p>NIS2022 cells harboring the IPTG-inducible <em>dnaA–dnaN</em> operon at the <em>purA</em> locus<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> were pre-cultured overnight at 30°C on a PAB plate containing 5 µg/ml tetracycline and 10 µg/ml neomycin, with or without 10 µM IPTG. Cells were inoculated into PAB liquid medium with or without 0.1 M IPTG to obtain an initial OD<sub>600</sub> value of 0.01, followed by culture at 30°C. At OD<sub>600</sub> of 0.4, cells were harvested from 40 mL of culture, and total RNA extracted, as described previously.<sup><a reveal-id="DSM017C36" data-open="DSM017C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Expression of DnaA protein in wild-type CRK6000 cells and NIS2022 cells was confirmed by western blotting using a specific antibody, based on an earlier protocol.<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Synthesis of cDNA, terminal labeling, and hybridization to the oligonucleotide chip were performed using established methods.<sup><a reveal-id="DSM017C27" data-open="DSM017C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Amplification of cDNA was avoided to allow the coding strands of each transcript to be distinguished. To compensate for the differences in hybridization efficiency of each 25-mer probe on the chip, we divided the hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA. The distribution of transcriptional signals along the genome coordinate was visualized with the <em>In Silico</em> Molecular Cloning program, Array Edition (<em>In Silico</em> Biology).</p>                            <h2 scrollto-destination=87195312 id="87195312" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87195313 id="87195313" class="section-title" data-legacy-id=s3a>3.1. Accurate mapping of known Spo0J-binding sites by ChAP-chip analysis</h3>
<p>Previously, we developed a method to purify <em>in vivo</em> complexes of cell division proteins using a 12 × histidine- tag under denatured conditions.<sup><a reveal-id="DSM017C30" data-open="DSM017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> We anticipated that the procedure is similarly effective for the purification of <em>in vivo</em> DNA–protein complexes. To verify the sensitivity of the tag-based chromatin precipitation protocol, we initially examined the binding sites of Spo0J involved in chromosome partitioning in <em>B. subtilis</em>. Spo0J binds at eight specific positions (<em>parS</em> sites) on the genome, as observed from the identified binding consensus sequences and chromatin immunoprecipitation analysis.<sup><a reveal-id="DSM017C31" data-open="DSM017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We expressed the Spo0J protein fused to a histidine tag at the C-terminus using SI002 cells. During exponential growth in LB medium under aerobic conditions, Spo0J-12 × His-expressing cells were treated with formaldehyde to cross-link protein and genome DNA, and disrupted by sonication in 8 M urea buffer to generate an average DNA fragment size of ∼500 bp. Protein–DNA complexes were purified with the Ni<sup>2+</sup>-resin under denatured conditions, and the cross-links removed by heat treatment. DNA fragments co-purified with Spo0J were hybridized with a high-density tiling chip. The hybridization intensities of Spo0J-associated DNA to each oligonucleotide probe on the chip are presented in Fig. <a reveal-id="DSM017F2" data-open="DSM017F2" class="link link-reveal link-table xref-fig">2</a> (see also detailed map in Supplementary Fig. S2). Our ChAP-chip analysis clearly shows that Spo0J forms a stable complex at eight regions, consistent with previous results.<sup><a reveal-id="DSM017C31" data-open="DSM017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Spo0J-binding signals are more broadly distributed than those of DnaA, as described below (Fig. <a reveal-id="DSM017F1" data-open="DSM017F1" class="link link-reveal link-table xref-fig">1</a>B and Supplementary Fig. S2). This result is consistent with the recent finding that Spo0J is associated with several kilobases of DNA flanking its specific binding sites (<em>parS</em>) through a <em>parS</em>-dependent nucleation event that promotes lateral spreading along the chromosome.<sup><a reveal-id="DSM017C37" data-open="DSM017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Thus, it appears that our ChAP-chip method is sufficiently sensitive to detect the <em>in vivo</em> interaction sites of DNA-binding proteins.</p>                        <a id="87195315" scrollto-destination="87195315"></a>
<div id="DSM017F2" data-id="DSM017F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01702.jpeg?Expires=2147483647&Signature=U95SGm~U7L-tZiedCACV0sc5CHTQbNg0XXooY8CkXBqmfANA9Al64SeX1MhLMHVWyKCSwv7Uk0tshtX0kJ3IJPbcJo73fl2KgL936pE8rV6md9fKP0t6nve-8wjujE-TYmti3iEeFZmLsk1EpTCBS1fW9JfaJEq9IjAhth9M08hHzIlLk1xvfgYoH7pW4B5w30PnhoHGUm0l6SHWs6Q04g5hTo-PZjBH96Vzk8hOCZXi2GOm8xvKC6PyfHe6EziqX51SVA49Qa2bztbFLVr5N-5zt1HatlUSPw1a-3GE8dUfIYDWzr3tNVQXCQu3DRGbvsR1DLGi3rU3o0t-Icq1sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA-binding profiles of Spo0J and DnaA proteins on the B. subtilis genome revealed using ChAP-chip analysis. (A) Spo0J (inner blue bars) and DnaA-binding (outer red bars) signals for each 25-mer probe on the chip were calculated by subtracting signal intensities of control DNA from those of DNA in the affinity-purified fraction, and shown at their corresponding genome coordinates. 0J1–0J8 correspond to the sites previously determined by Lin and Grossman.31 DnaA boxes are specified as colored vertical lines under DnaA-binding signals: TTATCCACA: red, TTATACACA, TTATAGACA, and TTATATACA: blue, other DnaA boxes having one base mismatch: orange. (B) Detailed DnaA and Spo0J binding profiles around the ywcI–vpr region, where binding of both proteins was observed, are shown for comparison of binding profiles of two proteins. The arrangement of genes (thick arrows), DnaA-boxes (as in A), and Spo0J binding site (parS site, purple vertical line) are shown schematically at the bottom." path-from-xml="dsm01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195315/dsm01702.jpeg" path-from-xml="dsm01702.jpeg" target="_blank">View large</a><a section="87195315" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01702.jpeg?Expires=2147483647&Signature=MMIB~zJ0LiRNw4UYW5l9HstdwoMkeM-tP8MReYvFFrWYe9QqaTeD7hBTAS6~sQh-bz8I08FOXaTRKBJZaxRQJsNQqN6B9a4UiZc7SXnAKfCKH6DmfwkPPVrqcIw9VfPoNtQZusdnilLmdeSlEK7R7tXUDluWCo8k9bJOElaQXrweI~eTh5FtrrB~1fkr6UA-EGZLFrmnJRFperyRmUP7HFKg2zYrHp24ETvhpUfvvYALkd0LlBVD6oDwpPuvfHC-pUT4mBRlxkV3lm-FOeTGpmV5lbxDRokdBikFGtUzzpfXeoVvK3W6cYbw4ifDKiZwHgAU0Gu8lErd2CkQNO9z2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195315&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA-binding profiles of Spo0J and DnaA proteins on the <em>B. subtilis</em> genome revealed using ChAP-chip analysis. (<strong>A</strong>) Spo0J (inner blue bars) and DnaA-binding (outer red bars) signals for each 25-mer probe on the chip were calculated by subtracting signal intensities of control DNA from those of DNA in the affinity-purified fraction, and shown at their corresponding genome coordinates. 0J1–0J8 correspond to the sites previously determined by Lin and Grossman.<sup>31</sup> DnaA boxes are specified as colored vertical lines under DnaA-binding signals: TTATCCACA: red, TTATACACA, TTATAGACA, and TTATATACA: blue, other DnaA boxes having one base mismatch: orange. (<strong>B</strong>) Detailed DnaA and Spo0J binding profiles around the <em>ywcI–vpr</em> region, where binding of both proteins was observed, are shown for comparison of binding profiles of two proteins. The arrangement of genes (thick arrows), DnaA-boxes (as in A), and Spo0J binding site (<em>parS</em> site, purple vertical line) are shown schematically at the bottom.</p></div></div><div content-id="DSM017F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01702.jpeg?Expires=2147483647&Signature=U95SGm~U7L-tZiedCACV0sc5CHTQbNg0XXooY8CkXBqmfANA9Al64SeX1MhLMHVWyKCSwv7Uk0tshtX0kJ3IJPbcJo73fl2KgL936pE8rV6md9fKP0t6nve-8wjujE-TYmti3iEeFZmLsk1EpTCBS1fW9JfaJEq9IjAhth9M08hHzIlLk1xvfgYoH7pW4B5w30PnhoHGUm0l6SHWs6Q04g5hTo-PZjBH96Vzk8hOCZXi2GOm8xvKC6PyfHe6EziqX51SVA49Qa2bztbFLVr5N-5zt1HatlUSPw1a-3GE8dUfIYDWzr3tNVQXCQu3DRGbvsR1DLGi3rU3o0t-Icq1sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA-binding profiles of Spo0J and DnaA proteins on the B. subtilis genome revealed using ChAP-chip analysis. (A) Spo0J (inner blue bars) and DnaA-binding (outer red bars) signals for each 25-mer probe on the chip were calculated by subtracting signal intensities of control DNA from those of DNA in the affinity-purified fraction, and shown at their corresponding genome coordinates. 0J1–0J8 correspond to the sites previously determined by Lin and Grossman.31 DnaA boxes are specified as colored vertical lines under DnaA-binding signals: TTATCCACA: red, TTATACACA, TTATAGACA, and TTATATACA: blue, other DnaA boxes having one base mismatch: orange. (B) Detailed DnaA and Spo0J binding profiles around the ywcI–vpr region, where binding of both proteins was observed, are shown for comparison of binding profiles of two proteins. The arrangement of genes (thick arrows), DnaA-boxes (as in A), and Spo0J binding site (parS site, purple vertical line) are shown schematically at the bottom." path-from-xml="dsm01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195315/dsm01702.jpeg" path-from-xml="dsm01702.jpeg" target="_blank">View large</a><a section="87195315" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01702.jpeg?Expires=2147483647&Signature=MMIB~zJ0LiRNw4UYW5l9HstdwoMkeM-tP8MReYvFFrWYe9QqaTeD7hBTAS6~sQh-bz8I08FOXaTRKBJZaxRQJsNQqN6B9a4UiZc7SXnAKfCKH6DmfwkPPVrqcIw9VfPoNtQZusdnilLmdeSlEK7R7tXUDluWCo8k9bJOElaQXrweI~eTh5FtrrB~1fkr6UA-EGZLFrmnJRFperyRmUP7HFKg2zYrHp24ETvhpUfvvYALkd0LlBVD6oDwpPuvfHC-pUT4mBRlxkV3lm-FOeTGpmV5lbxDRokdBikFGtUzzpfXeoVvK3W6cYbw4ifDKiZwHgAU0Gu8lErd2CkQNO9z2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195315&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA-binding profiles of Spo0J and DnaA proteins on the <em>B. subtilis</em> genome revealed using ChAP-chip analysis. (<strong>A</strong>) Spo0J (inner blue bars) and DnaA-binding (outer red bars) signals for each 25-mer probe on the chip were calculated by subtracting signal intensities of control DNA from those of DNA in the affinity-purified fraction, and shown at their corresponding genome coordinates. 0J1–0J8 correspond to the sites previously determined by Lin and Grossman.<sup>31</sup> DnaA boxes are specified as colored vertical lines under DnaA-binding signals: TTATCCACA: red, TTATACACA, TTATAGACA, and TTATATACA: blue, other DnaA boxes having one base mismatch: orange. (<strong>B</strong>) Detailed DnaA and Spo0J binding profiles around the <em>ywcI–vpr</em> region, where binding of both proteins was observed, are shown for comparison of binding profiles of two proteins. The arrangement of genes (thick arrows), DnaA-boxes (as in A), and Spo0J binding site (<em>parS</em> site, purple vertical line) are shown schematically at the bottom.</p></div></div>                            <h3 scrollto-destination=87195316 id="87195316" class="section-title" data-legacy-id=s3b>3.2. Identification of high-affinity binding sites of DnaA</h3>
<p>To identify the DnaA-binding sites on the <em>B. subtilis</em> genome, DnaA was expressed as a fusion protein with a 12 × histidine-tag by the authentic promoter using SI003 cells, and ChAP-chip analysis performed. The hybridization profiles of DnaA-associated DNA clearly disclose eight stable binding sites for DnaA on the <em>B. subtilis</em> genome (Fig. <a reveal-id="DSM017F2" data-open="DSM017F2" class="link link-reveal link-table xref-fig">2</a> and Supplementary Fig. S2). Interestingly, binding sites are mapped only in the intergenic regions (between <em>rpmH</em>–<em>dnaA</em>, <em>dnaA</em>–<em>dnaN</em>, <em>yqeG</em>–<em>sda</em>, <em>ywlB</em>–<em>ywlC</em>, <em>jag</em>–<em>thdF</em>, <em>yycS</em>–<em>yydA</em>, <em>ywcI</em>–<em>vpr</em>, and <em>gcp</em>–<em>ydiF</em>). Gene organization surrounding the eight binding sites, location of the DnaA boxes, and detailed DnaA-binding signals are depicted in Fig. <a reveal-id="DSM017F3" data-open="DSM017F3" class="link link-reveal link-table xref-fig">3</a>, and the functions of the surrounding genes are summarized in Table <a reveal-id="DSM017TB1" data-open="DSM017TB1" class="link link-reveal link-table xref-fig">1</a>. The regions upstream and downstream of <em>dnaA</em>, <em>rpmH</em>–<em>dnaA</em> (<em>oriC</em>1), and <em>dnaA</em>–<em>dnaN</em> (<em>oriC</em>2) comprise <em>cis</em>-acting sequences essential for the initiation of chromosome replication,<sup><a reveal-id="DSM017C34" data-open="DSM017C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and the upstream sequence contains the <em>dnaA</em> promoter that is repressed by the corresponding translated protein.<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Expression of <em>sda</em> is dependent on DnaA binding to the upstream sequence,<sup><a reveal-id="DSM017C22" data-open="DSM017C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and promoters of <em>ywlC</em> and <em>yydA</em> were reported to be negatively regulated by DnaA.<sup><a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Multiple DnaA boxes upstream of <em>thdF</em> were identified 17 years ago,<sup><a reveal-id="DSM017C38" data-open="DSM017C38" class="link link-ref link-reveal xref-bibr">38</a></sup> but with no evidence of actual DnaA binding. Our results additionally suggest a possibility that DnaA regulates the expression of <em>vpr</em> and/or <em>ywcI–secT</em>. Binding to the <em>gcp</em>–<em>ydiF</em> region is uncharacteristic for transcriptional regulation, as it is located at the 3′ end of convergent <em>gcp</em> and <em>ydiF</em> genes.</p>                        <a id="87195318" scrollto-destination="87195318"></a>
<div id="DSM017F3" data-id="DSM017F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01703.jpeg?Expires=2147483647&Signature=DAj4mjG9ZPwbqiHQFsFmrj3Lw4mNn~gAT-nUN00p4JcOgX1vlyw1cuNagiIWxmb3qfO8seMnMmGbLCdgaqdiMBhPrSXcqoh4RyOWLzNEvk8-yEpf9LKjswJTeRA1GgSdYZXf1d3Havw7rxdT0PyhU6-TKSi2W3oRXCzzFi8CSYj5DPsfcTnMPnSIXAn2DDTskudMUzMBRqbkpuU~TzSuIdGaC9SDrVAYDGdK~WCc1KjSseDqTs3tfT-uN6VsuIdUSizbSjJilQf4A8NUjMP7vkN6UY-Z-4D9-kfSg1w~wimGJ0SAplywoQBe57yIvm80ujr7vg-fCFxK6qZhbzLFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Local DNA-binding profiles of DnaA. Local DNA binding profiles of DnaA are presented, together with genes and DnaA boxes. DnaA boxes are specified as vertical lines under DnaA-binding signals: TTATCCACA: bold lines, TTATACACA, TTATAGACA and TTATATACA: gray lines, other DnaA boxes having one base mismatch: dotted lines. The top, middle, and bottom lines of ChAP-chip signals indicate signal intensities of 4000, 0, and −500, respectively. Closed boxes represent regions having high-affinity successive DnaA box clusters, as described in Table 2." path-from-xml="dsm01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195318/dsm01703.jpeg" path-from-xml="dsm01703.jpeg" target="_blank">View large</a><a section="87195318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01703.jpeg?Expires=2147483647&Signature=RW95mBP-zHpJfewq2AEzh0HtZzOHBX4tnJJ80YxJ9CMfCKxB8bfLMOKlqWdWYpFd4G11kAUg1gcHGKwdi72s4xC8hWvRTOIhijy87qYfZsq5G4J0aY-EJkKdQRZLDwj9IhMj3MBK5nFgeN41ByqZqcoGkDFjtUBgLph-uZUs~E1HMspoKK8XP6LXbGJEze1XVTfmdc-s37rW4gVUW4uduAP6UOVFFCIb6muFOSLyFwiuvI0EhEBiLqzd0x~jZDSG1zl8wkY4DsKnNpMz5ldooIY9AkVQ0oi88Y9qPbjpgYCCBb7ULTAw~JqDz2-Hnwe7dGOx-g0Ou317gSb437rZ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195318&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Local DNA-binding profiles of DnaA. Local DNA binding profiles of DnaA are presented, together with genes and DnaA boxes. DnaA boxes are specified as vertical lines under DnaA-binding signals: TTATCCACA: bold lines, TTATACACA, TTATAGACA and TTATATACA: gray lines, other DnaA boxes having one base mismatch: dotted lines. The top, middle, and bottom lines of ChAP-chip signals indicate signal intensities of 4000, 0, and −500, respectively. Closed boxes represent regions having high-affinity successive DnaA box clusters, as described in Table <a reveal-id="DSM017TB2" data-open="DSM017TB2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div><div content-id="DSM017F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01703.jpeg?Expires=2147483647&Signature=DAj4mjG9ZPwbqiHQFsFmrj3Lw4mNn~gAT-nUN00p4JcOgX1vlyw1cuNagiIWxmb3qfO8seMnMmGbLCdgaqdiMBhPrSXcqoh4RyOWLzNEvk8-yEpf9LKjswJTeRA1GgSdYZXf1d3Havw7rxdT0PyhU6-TKSi2W3oRXCzzFi8CSYj5DPsfcTnMPnSIXAn2DDTskudMUzMBRqbkpuU~TzSuIdGaC9SDrVAYDGdK~WCc1KjSseDqTs3tfT-uN6VsuIdUSizbSjJilQf4A8NUjMP7vkN6UY-Z-4D9-kfSg1w~wimGJ0SAplywoQBe57yIvm80ujr7vg-fCFxK6qZhbzLFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Local DNA-binding profiles of DnaA. Local DNA binding profiles of DnaA are presented, together with genes and DnaA boxes. DnaA boxes are specified as vertical lines under DnaA-binding signals: TTATCCACA: bold lines, TTATACACA, TTATAGACA and TTATATACA: gray lines, other DnaA boxes having one base mismatch: dotted lines. The top, middle, and bottom lines of ChAP-chip signals indicate signal intensities of 4000, 0, and −500, respectively. Closed boxes represent regions having high-affinity successive DnaA box clusters, as described in Table 2." path-from-xml="dsm01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195318/dsm01703.jpeg" path-from-xml="dsm01703.jpeg" target="_blank">View large</a><a section="87195318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01703.jpeg?Expires=2147483647&Signature=RW95mBP-zHpJfewq2AEzh0HtZzOHBX4tnJJ80YxJ9CMfCKxB8bfLMOKlqWdWYpFd4G11kAUg1gcHGKwdi72s4xC8hWvRTOIhijy87qYfZsq5G4J0aY-EJkKdQRZLDwj9IhMj3MBK5nFgeN41ByqZqcoGkDFjtUBgLph-uZUs~E1HMspoKK8XP6LXbGJEze1XVTfmdc-s37rW4gVUW4uduAP6UOVFFCIb6muFOSLyFwiuvI0EhEBiLqzd0x~jZDSG1zl8wkY4DsKnNpMz5ldooIY9AkVQ0oi88Y9qPbjpgYCCBb7ULTAw~JqDz2-Hnwe7dGOx-g0Ou317gSb437rZ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195318&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Local DNA-binding profiles of DnaA. Local DNA binding profiles of DnaA are presented, together with genes and DnaA boxes. DnaA boxes are specified as vertical lines under DnaA-binding signals: TTATCCACA: bold lines, TTATACACA, TTATAGACA and TTATATACA: gray lines, other DnaA boxes having one base mismatch: dotted lines. The top, middle, and bottom lines of ChAP-chip signals indicate signal intensities of 4000, 0, and −500, respectively. Closed boxes represent regions having high-affinity successive DnaA box clusters, as described in Table <a reveal-id="DSM017TB2" data-open="DSM017TB2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div>                        <a id="87195319" scrollto-destination="87195319"></a>
<div content-id="DSM017TB1" class="table-modal"><div class="table-wrap"><div id="DSM017TB1" class="table-wrap-title" data-id="DSM017TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Genes in transcriptional units adjacent to the DnaA-binding sites</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01706.jpeg?Expires=2147483647&Signature=Rc6njfNgWhgd2L3YWd0IumQCtej1q-OCCcoEO1JLIQSWbnSSbmDdP5hS5~zyOo2D~BsP1wA6vofVvO7sU2hpez2CB~oiMceUSVnkpwQtRgJrbdzcAJt6-BuS-3pn-X~iAnhLpkFLPQMNMCgF8iiMsO9~d-cpyYnIEgJ~RbYeAcZDorPwSX43WLtSHNOSvOTZBypaZcB2KeQLt7QfwpuLtIck3g7Q9dWQaXr45JgpcRS~WPRrwDfTdogOVMgDa8hhM4ytzp-OoVkagRPZJDRK-RSWZ5hZ4Bi3KV2EWjTxiXS8uOCLjjxl5aEQsPL6BUDBtnhXbLaIKoTDIdbUUgbqVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01706.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01706.jpeg?Expires=2147483647&Signature=nYxFOUIBJMbj1aQKaPzqdvXOFoLUv-uhdqZyBpq76uQ5MMvoVmznePsxhUTMVfeuz0vZtPbKCCHoBi2rGt0A0TwBxul08A5Bu8d9tDhf67e43IRKK-KjqMrUwEJ-k8IvSgjnhjnK9tC3A2~qplpPmqZOQJVX9DJBeQFfaevDgsLvJ761e7BMSqqmOISlGC4jowE1qt1Bn578yCz21f~VXoY~dtX7K2lv6NUHfznALuR1~r5FMir-N5oKxGEVfKls1qv8qZszkXoO54TteT31dU~NTV8HmhjKJeNSO9NPXEr7rtNZbgxs1UuL6TATO5Lau~lVdj0iv2QaunfQK9OK5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes in transcriptional units adjacent to the DnaA-binding sites" path-from-xml="dsm01706.jpeg" /></a></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The location of DnaA-binding site is indicated by circle and direction of the adjacent transcriptional units is indicated by arrow.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195319" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM017TB1" class="table-wrap-title" data-id="DSM017TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Genes in transcriptional units adjacent to the DnaA-binding sites</p></div> </div><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01706.jpeg?Expires=2147483647&Signature=Rc6njfNgWhgd2L3YWd0IumQCtej1q-OCCcoEO1JLIQSWbnSSbmDdP5hS5~zyOo2D~BsP1wA6vofVvO7sU2hpez2CB~oiMceUSVnkpwQtRgJrbdzcAJt6-BuS-3pn-X~iAnhLpkFLPQMNMCgF8iiMsO9~d-cpyYnIEgJ~RbYeAcZDorPwSX43WLtSHNOSvOTZBypaZcB2KeQLt7QfwpuLtIck3g7Q9dWQaXr45JgpcRS~WPRrwDfTdogOVMgDa8hhM4ytzp-OoVkagRPZJDRK-RSWZ5hZ4Bi3KV2EWjTxiXS8uOCLjjxl5aEQsPL6BUDBtnhXbLaIKoTDIdbUUgbqVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm01706.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01706.jpeg?Expires=2147483647&Signature=nYxFOUIBJMbj1aQKaPzqdvXOFoLUv-uhdqZyBpq76uQ5MMvoVmznePsxhUTMVfeuz0vZtPbKCCHoBi2rGt0A0TwBxul08A5Bu8d9tDhf67e43IRKK-KjqMrUwEJ-k8IvSgjnhjnK9tC3A2~qplpPmqZOQJVX9DJBeQFfaevDgsLvJ761e7BMSqqmOISlGC4jowE1qt1Bn578yCz21f~VXoY~dtX7K2lv6NUHfznALuR1~r5FMir-N5oKxGEVfKls1qv8qZszkXoO54TteT31dU~NTV8HmhjKJeNSO9NPXEr7rtNZbgxs1UuL6TATO5Lau~lVdj0iv2QaunfQK9OK5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes in transcriptional units adjacent to the DnaA-binding sites" path-from-xml="dsm01706.jpeg" /></a></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The location of DnaA-binding site is indicated by circle and direction of the adjacent transcriptional units is indicated by arrow.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195319" target="_blank">View Large</a></div></div><p>DnaA protein binds to single DnaA box <em>in vitro</em>.<sup><a reveal-id="DSM017C39" data-open="DSM017C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSM017C40" data-open="DSM017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> However, the DnaA-binding regions identified here contain multiple DnaA boxes. These results suggest that the ChAP-chip analysis can be effectively used to detect stable DnaA–DNA complexes, but not transient or weak binding to dispersed DnaA boxes.</p>                            <h3 scrollto-destination=87195321 id="87195321" class="section-title" data-legacy-id=s3c>3.3. Effect of DnaA overproduction and depletion on transcription of genes surrounding the stable binding sites</h3>
<p>To establish the function of DnaA binding to sequences other than <em>oriC</em>, we examined transcriptional profiles in DnaA-overproducing and -depleted cells using the high-density tiling chip employed for ChAP-chip analysis. To this end, we employed NIS2022 cells with an IPTG-inducible <em>dnaA–dnaN</em> operon at the <em>purA</em> locus, and inactivated the authentic <em>dnaA</em> gene by introducing orcher mutations.<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Expression of the DnaA protein in NIS2022 cells is reduced to One-fifth that in non-induced conditions (0 µM IPTG), and increases fivefold upon induction with 100 µM IPTG, compared with the DnaA concentration in wild-type cells.<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Notably, under both non-induced and induced conditions, no apparent growth defects were observed, and the overall profiles of gene expression remained unchanged (Supplementary Fig. S3). Transcriptional initiation from the authentic promoter of <em>dnaA</em> is clearly repressed upon induction of DnaA expression, and stimulated slightly under non-induced conditions (Fig. <a reveal-id="DSM017F4" data-open="DSM017F4" class="link link-reveal link-table xref-fig">4</a>A). This finding is in agreement with a previous study reporting autoregulation of <em>dnaA</em>.<sup><a reveal-id="DSM017C6" data-open="DSM017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> A similar change in transcription was observed for the intergenic region between <em>dnaA</em> and <em>dnaN</em>. The dependence of <em>sda</em> expression on DnaA is well characterized.<sup><a reveal-id="DSM017C22" data-open="DSM017C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We observed <em>sda</em> expression and repression in DnaA-induced and non-induced conditions, respectively (Fig. <a reveal-id="DSM017F4" data-open="DSM017F4" class="link link-reveal link-table xref-fig">4</a>B). Interestingly, we identified another transcript covering the <em>sda</em> gene, but on the non-coding strand. The newly identified transcript was weakly, but significantly repressed by overexpressed DnaA at the DnaA-binding site. We additionally observed weak, but significant repression of <em>ywlC</em>, <em>yydA</em>, and <em>thdF-gidA-gidB-noc</em> transcription in DnaA-induced cells (Fig. <a reveal-id="DSM017F4" data-open="DSM017F4" class="link link-reveal link-table xref-fig">4</a>C–E), although their de-repression in non-induced cells was not apparent. A sufficient amount of DnaA is present in cells to support growth at a normal rate, even under non-induced conditions, thus those genes may still be repressed by DnaA. The suppression of <em>ywlC</em> and <em>yydA</em> expression by DnaA is in accordance with reported microarray data.<sup><a reveal-id="DSM017C23" data-open="DSM017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We have demonstrated DnaA-regulated expression of <em>thdF-gidA-gidB-noc</em> by northern blot analysis using NIS2022 cells (Ogura <em>et al.</em>, unpublished result). These transcriptional alterations were reproducible in two independent experiments (Supplementary Fig. S3). Additionally, the DnaA level affected <em>rocA</em>, <em>rocD</em>, and <em>yvdCD</em> expression positively, and <em>ykuNOP</em> and <em>dhbA</em> expression negatively (Supplementary Fig. S3 and Supplementary Table S2). The molecular mechanisms and biological significance of these changes are currently unclear. On the other hand, the effects of DnaA on the expression of genes around <em>ywcI</em>–<em>vpr</em> and <em>gcp</em>–<em>ydiF</em> could not be distinguished due to weak expression.</p>                        <a id="87195323" scrollto-destination="87195323"></a>
<div id="DSM017F4" data-id="DSM017F4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01704.jpeg?Expires=2147483647&Signature=eZWgwYw196OYVWdEoRlzhbzmNeUrLqz0beueOO4RyTEin3fWNSSMTJKHB1OMc0Xv6lRJuIFqct3gK4HfiJKT~gxcGLLrw2PO4tyc0jymr-CJTWtRYTlSVLK7LfX0pqp8JLvm-3AlAmZfLdWaxdNCPzndy39wue4WYuIcDzHmmJopc11SX0-xsAdNRlZL5WKp3Wty9cy7GMTGMhFVgdcHF2xcePxhkpMZcEmw-bFl5GMsTrAInM1A~8ypYHl2mt7y6X6co9M2X0tIsXXQ9JdqDLqxhoCA1gFxBHwUOus9HJa2rGpvHxCpBj0y51B~wamPlg0PrYGS1UaKMmhXzJ8FKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcriptional changes around the DnaA-binding regions in dnaA-induced and repressed cells. Transcriptional activity of NIS2022 cells in the absence and presence of 100 µM IPTG and wild-type cells were analyzed using tiling chip. Transcriptional signals for each probe were calculated by dividing hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA, and shown by vertical blue bars at their corresponding genome coordinates, for regions surrounding the DnaA-binding sites. Signal intensities of probes on Watson (+) and Crick (−) strands were shown separately to indicate the direction of transcripts. The DnaA-binding signals (DnaA ChAP-chip) and location of genes and DnaA boxes are also shown, as in Fig. 2B. The transcriptional start sites, dnaA and sda, determined previously, are highlighted in the genetic maps with arrows. Possible termination signals of transcription are additionally specified. Signals marked by a bold underline in panel A are due to the expression of dnaA–dnaN placed at the purA locus in NIS2022 cells." path-from-xml="dsm01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195323/dsm01704.jpeg" path-from-xml="dsm01704.jpeg" target="_blank">View large</a><a section="87195323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01704.jpeg?Expires=2147483647&Signature=4OoUEM3ebpsLv6di0HdXOMJ5H1niM7WaImkbGFpLUOcMIwrnGTHhELPaUn-2ImQqxCirF6TNu~ToEzHZIoIF9vxoqutsHFvkkhJYLpuoLVDwmmVnQAi6R7BpO5GzFTkwZfxCdoOt9H0g2WOSPgSH8kzU35E6p5e7d~JgOsoaTUpYJ9EYSaac8Yc8OFUtoXS60clJzwUxn3HtCGMG2diKXaxpoGgbsOi4lyqQNg3e1zpAg3QEOfCsYAoHmpPm0fUT22uiFActiQm75qb80Bjy5YawUdvPdzP~ceJfHJx9BGv4tAYMIj9d8TRkwcysoaWsAqE-HGfvqKy9Ix0Cb-n64w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195323&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcriptional changes around the DnaA-binding regions in <em>dnaA</em>-induced and repressed cells. Transcriptional activity of NIS2022 cells in the absence and presence of 100 µM IPTG and wild-type cells were analyzed using tiling chip. Transcriptional signals for each probe were calculated by dividing hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA, and shown by vertical blue bars at their corresponding genome coordinates, for regions surrounding the DnaA-binding sites. Signal intensities of probes on Watson (+) and Crick (−) strands were shown separately to indicate the direction of transcripts. The DnaA-binding signals (DnaA ChAP-chip) and location of genes and DnaA boxes are also shown, as in Fig. <a reveal-id="DSM017F2" data-open="DSM017F2" class="link link-reveal link-table xref-fig">2</a>B. The transcriptional start sites, <em>dnaA</em> and <em>sda</em>, determined previously, are highlighted in the genetic maps with arrows. Possible termination signals of transcription are additionally specified. Signals marked by a bold underline in panel A are due to the expression of <em>dnaA</em>–<em>dnaN</em> placed at the <em>purA</em> locus in NIS2022 cells.</p></div></div><div content-id="DSM017F4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01704.jpeg?Expires=2147483647&Signature=eZWgwYw196OYVWdEoRlzhbzmNeUrLqz0beueOO4RyTEin3fWNSSMTJKHB1OMc0Xv6lRJuIFqct3gK4HfiJKT~gxcGLLrw2PO4tyc0jymr-CJTWtRYTlSVLK7LfX0pqp8JLvm-3AlAmZfLdWaxdNCPzndy39wue4WYuIcDzHmmJopc11SX0-xsAdNRlZL5WKp3Wty9cy7GMTGMhFVgdcHF2xcePxhkpMZcEmw-bFl5GMsTrAInM1A~8ypYHl2mt7y6X6co9M2X0tIsXXQ9JdqDLqxhoCA1gFxBHwUOus9HJa2rGpvHxCpBj0y51B~wamPlg0PrYGS1UaKMmhXzJ8FKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transcriptional changes around the DnaA-binding regions in dnaA-induced and repressed cells. Transcriptional activity of NIS2022 cells in the absence and presence of 100 µM IPTG and wild-type cells were analyzed using tiling chip. Transcriptional signals for each probe were calculated by dividing hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA, and shown by vertical blue bars at their corresponding genome coordinates, for regions surrounding the DnaA-binding sites. Signal intensities of probes on Watson (+) and Crick (−) strands were shown separately to indicate the direction of transcripts. The DnaA-binding signals (DnaA ChAP-chip) and location of genes and DnaA boxes are also shown, as in Fig. 2B. The transcriptional start sites, dnaA and sda, determined previously, are highlighted in the genetic maps with arrows. Possible termination signals of transcription are additionally specified. Signals marked by a bold underline in panel A are due to the expression of dnaA–dnaN placed at the purA locus in NIS2022 cells." path-from-xml="dsm01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195323/dsm01704.jpeg" path-from-xml="dsm01704.jpeg" target="_blank">View large</a><a section="87195323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01704.jpeg?Expires=2147483647&Signature=4OoUEM3ebpsLv6di0HdXOMJ5H1niM7WaImkbGFpLUOcMIwrnGTHhELPaUn-2ImQqxCirF6TNu~ToEzHZIoIF9vxoqutsHFvkkhJYLpuoLVDwmmVnQAi6R7BpO5GzFTkwZfxCdoOt9H0g2WOSPgSH8kzU35E6p5e7d~JgOsoaTUpYJ9EYSaac8Yc8OFUtoXS60clJzwUxn3HtCGMG2diKXaxpoGgbsOi4lyqQNg3e1zpAg3QEOfCsYAoHmpPm0fUT22uiFActiQm75qb80Bjy5YawUdvPdzP~ceJfHJx9BGv4tAYMIj9d8TRkwcysoaWsAqE-HGfvqKy9Ix0Cb-n64w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195323&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Transcriptional changes around the DnaA-binding regions in <em>dnaA</em>-induced and repressed cells. Transcriptional activity of NIS2022 cells in the absence and presence of 100 µM IPTG and wild-type cells were analyzed using tiling chip. Transcriptional signals for each probe were calculated by dividing hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA, and shown by vertical blue bars at their corresponding genome coordinates, for regions surrounding the DnaA-binding sites. Signal intensities of probes on Watson (+) and Crick (−) strands were shown separately to indicate the direction of transcripts. The DnaA-binding signals (DnaA ChAP-chip) and location of genes and DnaA boxes are also shown, as in Fig. <a reveal-id="DSM017F2" data-open="DSM017F2" class="link link-reveal link-table xref-fig">2</a>B. The transcriptional start sites, <em>dnaA</em> and <em>sda</em>, determined previously, are highlighted in the genetic maps with arrows. Possible termination signals of transcription are additionally specified. Signals marked by a bold underline in panel A are due to the expression of <em>dnaA</em>–<em>dnaN</em> placed at the <em>purA</em> locus in NIS2022 cells.</p></div></div>                            <h3 scrollto-destination=87195324 id="87195324" class="section-title" data-legacy-id=s3d>3.4. Sequence characteristics of stable DnaA-binding sites on the chromosome</h3>
<p>Next, we explored the characteristics defining the stable DnaA-binding sites in the <em>B. subtilis</em> genome. <em>Bacillus subtilis</em> DnaA recognizes DnaA boxes in <em>E. coli oriC</em>, and vice versa, <em>in vitro</em>. Although the numbers and arrangement of DnaA boxes in <em>oriC</em> are different in <em>B. subtilis</em> and <em>E. coli</em>, the <em>oriC</em> regions were incompatible with each other <em>in vivo</em>.<sup><a reveal-id="DSM017C39" data-open="DSM017C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSM017C40" data-open="DSM017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Recently, the crystal structure of the DNA-binding domain of <em>E. coli</em> DnaA complexed with a DnaA box (TTATCCACA) was resolved.<sup><a reveal-id="DSM017C41" data-open="DSM017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> In the complex, amino acids R399, P423, D433, H434, T435, and H439 form base-specific interactions, and are essential for sequence recognition by DnaA. Interestingly, these residues are conserved in <em>B. subtilis</em> DnaA, indicating that DNA-binding specificities are similar for <em>B. subtilis</em> and <em>E. coli</em> DnaA. Thus, we assume that the consensus sequence of the <em>B. subtilis</em> DnaA box is TTATNCACA, similar to the <em>E. coli</em> protein, and allow one base mismatch for identifying DnaA boxes. As shown in Fig. <a reveal-id="DSM017F3" data-open="DSM017F3" class="link link-reveal link-table xref-fig">3</a>, stable DnaA-binding sites contain multiple boxes, as observed in <em>oriC</em> sequences of eubacteria.<sup><a reveal-id="DSM017C42" data-open="DSM017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Initially, we evaluated the density of DnaA boxes along the genome, using a modified method of0 Mackiewicz <em>et al.</em><sup><a reveal-id="DSM017C42" data-open="DSM017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The density of DnaA box distribution was presented on charts as <em>b</em> = 1/<em>d</em>, calculated for each box, where <em>d</em> is the average distance between adjacent DnaA boxes. Mackiewicz and colleagues calculated the <em>b</em> value for three successive DnaA boxes (average of two distances), whereas we additionally calculated for four and five successive boxes, with no clear correlation with the identified binding sites. Recently, the DnaA-binding affinity to DnaA boxes with different fifth nucleotides was evaluated in <em>E. coli</em> cells and the affinity was found to decrease in the order: C &gt; A = G &gt; T.<sup><a reveal-id="DSM017C43" data-open="DSM017C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Accordingly, we assigned the binding affinities as 4, 2, 1.5, 1, and 1 for TTATCCACA, TTATACACA, TTATAGACA, TTATATACA, and other DnaA boxes (having one base mismatch from TTATNCACA), and the binding affinity of DnaA for four successive DnaA boxes was expressed as <em>A</em> = (sum of affinities of four successive DnaA boxes)/<em>d</em>. As the result, we found that the <em>A</em> value is well correlated with the DnaA-binding sites; eight binding sites identified here exclusively have <em>A</em> value &gt;0.3 (Fig. <a reveal-id="DSM017F5" data-open="DSM017F5" class="link link-reveal link-table xref-fig">5</a>A). Sequence characteristics of the eight DnaA-binding regions, including number of DnaA boxes and the <em>A</em> value, are summarized in Table <a reveal-id="DSM017TB2" data-open="DSM017TB2" class="link link-reveal link-table xref-fig">2</a>. Our results suggest that two factors, specifically, closely clustered DnaA boxes of probably four or more, and the affinity of DnaA boxes for the protein, are essential for stable DnaA-binding in <em>B. subtilis</em> cells.</p>                        <a id="87195326" scrollto-destination="87195326"></a>
<div id="DSM017F5" data-id="DSM017F5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01705.jpeg?Expires=2147483647&Signature=H0JgYl0MKEfZ44V5wW5oqgViUY5kDjQJ7BbV2cP1KpN3Q~73kgmyjCz1jAgpxywbHAyKXoTLE4OZYpfqz7iivfpJycQvRJ~~Hs2KQBRog~3iGB29oHj9YDseIVEaouh6E0L0WrU4fH9Cy-yWaVEaBVpv0K0g5eJ5gpKmwvU40BDr53Rx5z9KjBUL-9z-IYl0~QnArEYB3hI8VQqaqkNcUM~Tvd~Ig27ORRmYAqZmk~68~DyVBeRaKQB1RGEE1yiTsNh-Oi33oqg6fJWMpDD-t-2u76vIqhG96Bu0QCGXuTSlQw~UJDREjAQtd36Ii6T-Gl6x9NYdA4q6xNPq1IFn6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Estimation of the distribution of the possible DnaA-binding regions in genome sequences of the Bacillus species. The A value [(sum of affinities of four successive DnaA boxes)/d (average distance of adjacent DnaA boxes)] was calculated for all DnaA boxes, and plotted (closed circles joined by line) at corresponding genome coordinates in B. subtilis (A), B. licheniformis (B), B. halodurans (C), B. anthracis (D), B. thuringiensis (E), B. cereus (F), B. clausii (G), Geobacillus kaustophilus (H), and Oceanobacillus iheyensis (I). Genome sequences, except that of B. subtilis, were retrieved from the NCBI GeneBank database. The genes adjacent to the possible DnaA-binding regions are specified, and “ → ” indicates their orientations (5′ to 3′ direction) relative to DnaA-binding site located between them. Genes or possible DnaA-binding regions that are not conserved in B. subtilis are colored gray." path-from-xml="dsm01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195326/dsm01705.jpeg" path-from-xml="dsm01705.jpeg" target="_blank">View large</a><a section="87195326" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01705.jpeg?Expires=2147483647&Signature=RPn35qt2jqVKT91KXnOUY6dneo9Wgm9ykJT4Vc4Iyz4ft~UzAK5wm4IpwW31LmFdaZpgS7XdbtNW9PXmHVGc0POQomETQb7VNHzKvMk1qQnmNpv7kMlf5VHsyXTtHc6B6hMC0R33GZHbl52NL69Egw~9bIVwifIUIfh7YEPF6RXSnEUCSrcW3cBZxg-ZxOdsC2KHie8s1jzWlIfA2baQ8DZA4YHi6nxd1JkAwN4L3RRmHFOLtYWudPx43aOSeWhFZVWw7mNyF4z1poQIWhc~eEUkUqJYQ5WZFXoNf9VYksdS~zwpmBaLmTQ8xJdkNIo3JW3DCAmU1zraI5yPdcJxeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195326&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Estimation of the distribution of the possible DnaA-binding regions in genome sequences of the <em>Bacillus</em> species. The <em>A</em> value [(sum of affinities of four successive DnaA boxes)/<em>d</em> (average distance of adjacent DnaA boxes)] was calculated for all DnaA boxes, and plotted (closed circles joined by line) at corresponding genome coordinates in <em>B. subtilis</em> (<strong>A</strong>), <em>B. licheniformis</em> (<strong>B</strong>), <em>B. halodurans</em> (<strong>C</strong>), <em>B. anthracis</em> (<strong>D</strong>), <em>B. thuringiensis</em> (<strong>E</strong>), <em>B. cereus</em> (<strong>F</strong>), <em>B. clausii</em> (<strong>G</strong>), <em>Geobacillus kaustophilus</em> (H), and <em>Oceanobacillus iheyensis</em> (<strong>I</strong>). Genome sequences, except that of <em>B. subtilis</em>, were retrieved from the NCBI GeneBank database. The genes adjacent to the possible DnaA-binding regions are specified, and “ → ” indicates their orientations (5′ to 3′ direction) relative to DnaA-binding site located between them. Genes or possible DnaA-binding regions that are not conserved in <em>B. subtilis</em> are colored gray.</p></div></div><div content-id="DSM017F5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/m_dsm01705.jpeg?Expires=2147483647&Signature=H0JgYl0MKEfZ44V5wW5oqgViUY5kDjQJ7BbV2cP1KpN3Q~73kgmyjCz1jAgpxywbHAyKXoTLE4OZYpfqz7iivfpJycQvRJ~~Hs2KQBRog~3iGB29oHj9YDseIVEaouh6E0L0WrU4fH9Cy-yWaVEaBVpv0K0g5eJ5gpKmwvU40BDr53Rx5z9KjBUL-9z-IYl0~QnArEYB3hI8VQqaqkNcUM~Tvd~Ig27ORRmYAqZmk~68~DyVBeRaKQB1RGEE1yiTsNh-Oi33oqg6fJWMpDD-t-2u76vIqhG96Bu0QCGXuTSlQw~UJDREjAQtd36Ii6T-Gl6x9NYdA4q6xNPq1IFn6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Estimation of the distribution of the possible DnaA-binding regions in genome sequences of the Bacillus species. The A value [(sum of affinities of four successive DnaA boxes)/d (average distance of adjacent DnaA boxes)] was calculated for all DnaA boxes, and plotted (closed circles joined by line) at corresponding genome coordinates in B. subtilis (A), B. licheniformis (B), B. halodurans (C), B. anthracis (D), B. thuringiensis (E), B. cereus (F), B. clausii (G), Geobacillus kaustophilus (H), and Oceanobacillus iheyensis (I). Genome sequences, except that of B. subtilis, were retrieved from the NCBI GeneBank database. The genes adjacent to the possible DnaA-binding regions are specified, and “ → ” indicates their orientations (5′ to 3′ direction) relative to DnaA-binding site located between them. Genes or possible DnaA-binding regions that are not conserved in B. subtilis are colored gray." path-from-xml="dsm01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195326/dsm01705.jpeg" path-from-xml="dsm01705.jpeg" target="_blank">View large</a><a section="87195326" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm01705.jpeg?Expires=2147483647&Signature=RPn35qt2jqVKT91KXnOUY6dneo9Wgm9ykJT4Vc4Iyz4ft~UzAK5wm4IpwW31LmFdaZpgS7XdbtNW9PXmHVGc0POQomETQb7VNHzKvMk1qQnmNpv7kMlf5VHsyXTtHc6B6hMC0R33GZHbl52NL69Egw~9bIVwifIUIfh7YEPF6RXSnEUCSrcW3cBZxg-ZxOdsC2KHie8s1jzWlIfA2baQ8DZA4YHi6nxd1JkAwN4L3RRmHFOLtYWudPx43aOSeWhFZVWw7mNyF4z1poQIWhc~eEUkUqJYQ5WZFXoNf9VYksdS~zwpmBaLmTQ8xJdkNIo3JW3DCAmU1zraI5yPdcJxeA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195326&ar=466553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Estimation of the distribution of the possible DnaA-binding regions in genome sequences of the <em>Bacillus</em> species. The <em>A</em> value [(sum of affinities of four successive DnaA boxes)/<em>d</em> (average distance of adjacent DnaA boxes)] was calculated for all DnaA boxes, and plotted (closed circles joined by line) at corresponding genome coordinates in <em>B. subtilis</em> (<strong>A</strong>), <em>B. licheniformis</em> (<strong>B</strong>), <em>B. halodurans</em> (<strong>C</strong>), <em>B. anthracis</em> (<strong>D</strong>), <em>B. thuringiensis</em> (<strong>E</strong>), <em>B. cereus</em> (<strong>F</strong>), <em>B. clausii</em> (<strong>G</strong>), <em>Geobacillus kaustophilus</em> (H), and <em>Oceanobacillus iheyensis</em> (<strong>I</strong>). Genome sequences, except that of <em>B. subtilis</em>, were retrieved from the NCBI GeneBank database. The genes adjacent to the possible DnaA-binding regions are specified, and “ → ” indicates their orientations (5′ to 3′ direction) relative to DnaA-binding site located between them. Genes or possible DnaA-binding regions that are not conserved in <em>B. subtilis</em> are colored gray.</p></div></div>                        <a id="87195327" scrollto-destination="87195327"></a>
<div content-id="DSM017TB2" class="table-modal"><div class="table-wrap"><div id="DSM017TB2" class="table-wrap-title" data-id="DSM017TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Sequence characteristics of the DnaA-binding regions identified by ChAP-chip analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="3">Region<sup>a</sup> </th><th><em>d</em><sup>b</sup> </th><th>Sum of affinity </th><th><em>A</em><sup>c</sup> </th><th>Number of <em>DnaA</em> boxes<sup>d</sup> </th><th>Number of perfect matches<sup>e</sup> </th></tr><tr><th>Surrounding genes </th><th>Start </th><th>End </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>rpmH</em>–<em>dnaA</em> </td><td>193 </td><td>257 </td><td>10.7 </td><td>10 </td><td>0.94 </td><td>8 </td><td>2 (2) </td></tr><tr><td><em>dnaA</em>–<em>dnaN</em> </td><td>1797 </td><td>1860 </td><td>10.3 </td><td>7 </td><td>0.68 </td><td>5 </td><td>1 (1) </td></tr><tr><td><em>gcp</em>–<em>ydiF</em> </td><td>643 951 </td><td>643 999 </td><td>5.3 </td><td>5 </td><td>0.94 </td><td>8 </td><td>4 (3) </td></tr><tr><td><em>yqeG</em>–<em>sda</em> </td><td>2 646 452 </td><td>2 646 536 </td><td>17.3 </td><td>7 </td><td>0.40 </td><td>5 </td><td>2 (2) </td></tr><tr><td><em>ywlC</em>–<em>ywlB</em> </td><td>3 793 128 </td><td>3 793 181 </td><td>7.0 </td><td>8 </td><td>1.14 </td><td>5 </td><td>2 (1) </td></tr><tr><td><em>ywcI</em>–<em>vpr</em> </td><td>3 906 700 </td><td>3 906 743 </td><td>3.7 </td><td>6 </td><td>1.64 </td><td>5 </td><td>3 (0) </td></tr><tr><td><em>yydA</em>–<em>yycS</em> </td><td>4 133 932 </td><td>4 133 997 </td><td>11.0 </td><td>4 </td><td>0.36 </td><td>4 </td><td>0 (0) </td></tr><tr><td><em>thdF</em>–<em>jag</em> </td><td>4 212 013 </td><td>4 212 108 </td><td>21.0 </td><td>10 </td><td>0.48 </td><td>4 </td><td>3 (2) </td></tr></tbody></table></div><div content-id="DSM017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="3">Region<sup>a</sup> </th><th><em>d</em><sup>b</sup> </th><th>Sum of affinity </th><th><em>A</em><sup>c</sup> </th><th>Number of <em>DnaA</em> boxes<sup>d</sup> </th><th>Number of perfect matches<sup>e</sup> </th></tr><tr><th>Surrounding genes </th><th>Start </th><th>End </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>rpmH</em>–<em>dnaA</em> </td><td>193 </td><td>257 </td><td>10.7 </td><td>10 </td><td>0.94 </td><td>8 </td><td>2 (2) </td></tr><tr><td><em>dnaA</em>–<em>dnaN</em> </td><td>1797 </td><td>1860 </td><td>10.3 </td><td>7 </td><td>0.68 </td><td>5 </td><td>1 (1) </td></tr><tr><td><em>gcp</em>–<em>ydiF</em> </td><td>643 951 </td><td>643 999 </td><td>5.3 </td><td>5 </td><td>0.94 </td><td>8 </td><td>4 (3) </td></tr><tr><td><em>yqeG</em>–<em>sda</em> </td><td>2 646 452 </td><td>2 646 536 </td><td>17.3 </td><td>7 </td><td>0.40 </td><td>5 </td><td>2 (2) </td></tr><tr><td><em>ywlC</em>–<em>ywlB</em> </td><td>3 793 128 </td><td>3 793 181 </td><td>7.0 </td><td>8 </td><td>1.14 </td><td>5 </td><td>2 (1) </td></tr><tr><td><em>ywcI</em>–<em>vpr</em> </td><td>3 906 700 </td><td>3 906 743 </td><td>3.7 </td><td>6 </td><td>1.64 </td><td>5 </td><td>3 (0) </td></tr><tr><td><em>yydA</em>–<em>yycS</em> </td><td>4 133 932 </td><td>4 133 997 </td><td>11.0 </td><td>4 </td><td>0.36 </td><td>4 </td><td>0 (0) </td></tr><tr><td><em>thdF</em>–<em>jag</em> </td><td>4 212 013 </td><td>4 212 108 </td><td>21.0 </td><td>10 </td><td>0.48 </td><td>4 </td><td>3 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Regions with <em>A</em> value &gt;0.3 are shown. When DnaA box clusters overlap, those with the highest <em>A</em> value are indicated.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Average distance between four adjacent DnaA boxes.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup><em>A</em> = (sum of affinities of four successive DnaA boxes)/d. Only the highest value in each DnaA box cluster is shown. See text for detail.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Number of DnaA boxes allowing one base mismatch of the consensus DnaA box, TTATNCACA, in successive DnaA box clusters with an <em>A</em> value &gt;0.3.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>The number of perfectly matched sequences to the consensus DnaA box, TTATNCACA. The number of DnaA boxes with highest affinity (TTATCCACA) is indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195327" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM017TB2" class="table-wrap-title" data-id="DSM017TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Sequence characteristics of the DnaA-binding regions identified by ChAP-chip analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="3">Region<sup>a</sup> </th><th><em>d</em><sup>b</sup> </th><th>Sum of affinity </th><th><em>A</em><sup>c</sup> </th><th>Number of <em>DnaA</em> boxes<sup>d</sup> </th><th>Number of perfect matches<sup>e</sup> </th></tr><tr><th>Surrounding genes </th><th>Start </th><th>End </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>rpmH</em>–<em>dnaA</em> </td><td>193 </td><td>257 </td><td>10.7 </td><td>10 </td><td>0.94 </td><td>8 </td><td>2 (2) </td></tr><tr><td><em>dnaA</em>–<em>dnaN</em> </td><td>1797 </td><td>1860 </td><td>10.3 </td><td>7 </td><td>0.68 </td><td>5 </td><td>1 (1) </td></tr><tr><td><em>gcp</em>–<em>ydiF</em> </td><td>643 951 </td><td>643 999 </td><td>5.3 </td><td>5 </td><td>0.94 </td><td>8 </td><td>4 (3) </td></tr><tr><td><em>yqeG</em>–<em>sda</em> </td><td>2 646 452 </td><td>2 646 536 </td><td>17.3 </td><td>7 </td><td>0.40 </td><td>5 </td><td>2 (2) </td></tr><tr><td><em>ywlC</em>–<em>ywlB</em> </td><td>3 793 128 </td><td>3 793 181 </td><td>7.0 </td><td>8 </td><td>1.14 </td><td>5 </td><td>2 (1) </td></tr><tr><td><em>ywcI</em>–<em>vpr</em> </td><td>3 906 700 </td><td>3 906 743 </td><td>3.7 </td><td>6 </td><td>1.64 </td><td>5 </td><td>3 (0) </td></tr><tr><td><em>yydA</em>–<em>yycS</em> </td><td>4 133 932 </td><td>4 133 997 </td><td>11.0 </td><td>4 </td><td>0.36 </td><td>4 </td><td>0 (0) </td></tr><tr><td><em>thdF</em>–<em>jag</em> </td><td>4 212 013 </td><td>4 212 108 </td><td>21.0 </td><td>10 </td><td>0.48 </td><td>4 </td><td>3 (2) </td></tr></tbody></table></div><div content-id="DSM017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="3">Region<sup>a</sup> </th><th><em>d</em><sup>b</sup> </th><th>Sum of affinity </th><th><em>A</em><sup>c</sup> </th><th>Number of <em>DnaA</em> boxes<sup>d</sup> </th><th>Number of perfect matches<sup>e</sup> </th></tr><tr><th>Surrounding genes </th><th>Start </th><th>End </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>rpmH</em>–<em>dnaA</em> </td><td>193 </td><td>257 </td><td>10.7 </td><td>10 </td><td>0.94 </td><td>8 </td><td>2 (2) </td></tr><tr><td><em>dnaA</em>–<em>dnaN</em> </td><td>1797 </td><td>1860 </td><td>10.3 </td><td>7 </td><td>0.68 </td><td>5 </td><td>1 (1) </td></tr><tr><td><em>gcp</em>–<em>ydiF</em> </td><td>643 951 </td><td>643 999 </td><td>5.3 </td><td>5 </td><td>0.94 </td><td>8 </td><td>4 (3) </td></tr><tr><td><em>yqeG</em>–<em>sda</em> </td><td>2 646 452 </td><td>2 646 536 </td><td>17.3 </td><td>7 </td><td>0.40 </td><td>5 </td><td>2 (2) </td></tr><tr><td><em>ywlC</em>–<em>ywlB</em> </td><td>3 793 128 </td><td>3 793 181 </td><td>7.0 </td><td>8 </td><td>1.14 </td><td>5 </td><td>2 (1) </td></tr><tr><td><em>ywcI</em>–<em>vpr</em> </td><td>3 906 700 </td><td>3 906 743 </td><td>3.7 </td><td>6 </td><td>1.64 </td><td>5 </td><td>3 (0) </td></tr><tr><td><em>yydA</em>–<em>yycS</em> </td><td>4 133 932 </td><td>4 133 997 </td><td>11.0 </td><td>4 </td><td>0.36 </td><td>4 </td><td>0 (0) </td></tr><tr><td><em>thdF</em>–<em>jag</em> </td><td>4 212 013 </td><td>4 212 108 </td><td>21.0 </td><td>10 </td><td>0.48 </td><td>4 </td><td>3 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Regions with <em>A</em> value &gt;0.3 are shown. When DnaA box clusters overlap, those with the highest <em>A</em> value are indicated.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Average distance between four adjacent DnaA boxes.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup><em>A</em> = (sum of affinities of four successive DnaA boxes)/d. Only the highest value in each DnaA box cluster is shown. See text for detail.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Number of DnaA boxes allowing one base mismatch of the consensus DnaA box, TTATNCACA, in successive DnaA box clusters with an <em>A</em> value &gt;0.3.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>The number of perfectly matched sequences to the consensus DnaA box, TTATNCACA. The number of DnaA boxes with highest affinity (TTATCCACA) is indicated in parentheses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195327" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195328 id="87195328" class="section-title" data-legacy-id=s3e>3.5. Distribution of probable DnaA-binding sites in other Bacillus species</h3>
<p>We estimated the stable DnaA-binding sites in the genome sequences of other <em>Bacillus</em> species, using the defined <em>A</em> value. Interestingly, regions with high <em>A</em> values locate in a similar arrangement on the genome in all the bacteria examined (Fig. <a reveal-id="DSM017F5" data-open="DSM017F5" class="link link-reveal link-table xref-fig">5</a>B–I). However, close examination of possible DnaA-binding regions in various <em>Bacillus</em> species revealed their conserved and diverse characteristics (Table <a reveal-id="DSM017TB3" data-open="DSM017TB3" class="link link-reveal link-table xref-fig">3</a>). As expected, multiple DnaA-binding sequences upstream and downstream of <em>dnaA</em> (<em>oriC</em>) are essentially conserved in all bacteria. Notably, in <em>B. licheniformis</em>, the <em>dnaA–dnaN</em> region contains only three DnaA boxes, and its <em>A</em> value is below the threshold. DnaA-binding sequences upstream of <em>sda</em>, <em>ywlC</em>, and <em>yydA</em> are also present in most bacteria, supporting the biological significance of DnaA-mediated regulation of these genes. Interestingly, a DnaA box cluster downstream of <em>gcp</em> is highly conserved, although its biological role is not clear at present. On the other hand, DnaA box cluster upstream of the <em>thdF-gidA-gidB-noc</em> operon is not conserved, although those genes are conserved in all bacteria examined (Table <a reveal-id="DSM017TB3" data-open="DSM017TB3" class="link link-reveal link-table xref-fig">3</a>), thus the DnaA-dependent expression of the operon appears not to be general. Similarly, DnaA box clusters upstream of <em>vpr</em> are present in limited bacteria. In addition, other possible DnaA-binding regions (colored red in Fig. <a reveal-id="DSM017F5" data-open="DSM017F5" class="link link-reveal link-table xref-fig">5</a>B–I) that are absent in <em>B. subtilis</em> have been identified. Interestingly, DnaA-binding regions with a high <em>A</em> value are generally located near <em>oriC</em> in all the bacteria examined.</p>                        <a id="87195330" scrollto-destination="87195330"></a>
<div content-id="DSM017TB3" class="table-modal"><div class="table-wrap"><div id="DSM017TB3" class="table-wrap-title" data-id="DSM017TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Conservation of DnaA-binding sequences identified in <em>B. subtilis</em> and adjacent genes in genome sequences of <em>Bacillus</em> spices</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Species </th><th><em>rpmH</em>← </th><th>→<em>dnaA</em>→ </th><th>→<em>dnaN</em> </th><th><em>yydA</em>← </th><th>→<em>yycS</em> </th><th><em>gcp</em>→ </th><th>←<em>ydiF</em> </th><th><em>thdF</em>← </th><th>←<em>jag</em> </th><th><em>ywlC</em>← </th><th>←<em>ywlB</em> </th><th><em>yqeG</em>← </th><th>→<em>sda</em> </th><th><em>ywcI</em>← </th><th>→<em>vpr</em> </th></tr></thead><tbody><tr><td><em>Bacillus</em> </td><td><em>B. anthracis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. cereus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. thuringiensis</em> </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. clausii</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td>▵ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td> </td><td>+ </td></tr><tr><td> </td><td><em>B. halodurans</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td><em>B. licheniformis</em> </td><td>+ </td><td>+ </td><td>▵ </td><td>+ </td><td> </td><td>+ </td><td>▵ </td><td>(+) </td><td>(+) </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>B. subtilis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Geobacillus</em> </td><td><em>G. kaustophilus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td></tr><tr><td><em>Oceanobacillus</em> </td><td><em>O. iheyensis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td> </td><td> </td><td>▵ </td></tr></tbody></table></div><div content-id="DSM017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Species </th><th><em>rpmH</em>← </th><th>→<em>dnaA</em>→ </th><th>→<em>dnaN</em> </th><th><em>yydA</em>← </th><th>→<em>yycS</em> </th><th><em>gcp</em>→ </th><th>←<em>ydiF</em> </th><th><em>thdF</em>← </th><th>←<em>jag</em> </th><th><em>ywlC</em>← </th><th>←<em>ywlB</em> </th><th><em>yqeG</em>← </th><th>→<em>sda</em> </th><th><em>ywcI</em>← </th><th>→<em>vpr</em> </th></tr></thead><tbody><tr><td><em>Bacillus</em> </td><td><em>B. anthracis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. cereus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. thuringiensis</em> </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. clausii</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td>▵ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td> </td><td>+ </td></tr><tr><td> </td><td><em>B. halodurans</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td><em>B. licheniformis</em> </td><td>+ </td><td>+ </td><td>▵ </td><td>+ </td><td> </td><td>+ </td><td>▵ </td><td>(+) </td><td>(+) </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>B. subtilis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Geobacillus</em> </td><td><em>G. kaustophilus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td></tr><tr><td><em>Oceanobacillus</em> </td><td><em>O. iheyensis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td> </td><td> </td><td>▵ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Symbols indicate conservation of genes. When DnaA box clusters with <em>A</em> ≥ 0.3 and <em>A</em> &lt; 0.3 exist in the similar arrangement with that in <em>B. subtilis</em>, cells are indicated as “+” and “(+)”, respectively. When DnaA box clusters do not exit, cells are indicated as “(Δ)”. “ → ” indicates orientation of genes (5′ to 3′) relative to DnaA-binding site located between them.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195330" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM017TB3" class="table-wrap-title" data-id="DSM017TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Conservation of DnaA-binding sequences identified in <em>B. subtilis</em> and adjacent genes in genome sequences of <em>Bacillus</em> spices</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Species </th><th><em>rpmH</em>← </th><th>→<em>dnaA</em>→ </th><th>→<em>dnaN</em> </th><th><em>yydA</em>← </th><th>→<em>yycS</em> </th><th><em>gcp</em>→ </th><th>←<em>ydiF</em> </th><th><em>thdF</em>← </th><th>←<em>jag</em> </th><th><em>ywlC</em>← </th><th>←<em>ywlB</em> </th><th><em>yqeG</em>← </th><th>→<em>sda</em> </th><th><em>ywcI</em>← </th><th>→<em>vpr</em> </th></tr></thead><tbody><tr><td><em>Bacillus</em> </td><td><em>B. anthracis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. cereus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. thuringiensis</em> </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. clausii</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td>▵ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td> </td><td>+ </td></tr><tr><td> </td><td><em>B. halodurans</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td><em>B. licheniformis</em> </td><td>+ </td><td>+ </td><td>▵ </td><td>+ </td><td> </td><td>+ </td><td>▵ </td><td>(+) </td><td>(+) </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>B. subtilis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Geobacillus</em> </td><td><em>G. kaustophilus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td></tr><tr><td><em>Oceanobacillus</em> </td><td><em>O. iheyensis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td> </td><td> </td><td>▵ </td></tr></tbody></table></div><div content-id="DSM017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Species </th><th><em>rpmH</em>← </th><th>→<em>dnaA</em>→ </th><th>→<em>dnaN</em> </th><th><em>yydA</em>← </th><th>→<em>yycS</em> </th><th><em>gcp</em>→ </th><th>←<em>ydiF</em> </th><th><em>thdF</em>← </th><th>←<em>jag</em> </th><th><em>ywlC</em>← </th><th>←<em>ywlB</em> </th><th><em>yqeG</em>← </th><th>→<em>sda</em> </th><th><em>ywcI</em>← </th><th>→<em>vpr</em> </th></tr></thead><tbody><tr><td><em>Bacillus</em> </td><td><em>B. anthracis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. cereus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. thuringiensis</em> </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td> </td><td><em>B. clausii</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td>▵ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td> </td><td>+ </td></tr><tr><td> </td><td><em>B. halodurans</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td></tr><tr><td><em>B. licheniformis</em> </td><td>+ </td><td>+ </td><td>▵ </td><td>+ </td><td> </td><td>+ </td><td>▵ </td><td>(+) </td><td>(+) </td><td>(+) </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>B. subtilis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Geobacillus</em> </td><td><em>G. kaustophilus</em> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>+ </td><td>+ </td><td> </td><td> </td></tr><tr><td><em>Oceanobacillus</em> </td><td><em>O. iheyensis</em> </td><td>+ </td><td>+ </td><td>+ </td><td>▵ </td><td>▵ </td><td>+ </td><td>(+) </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>▵ </td><td> </td><td> </td><td>▵ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Symbols indicate conservation of genes. When DnaA box clusters with <em>A</em> ≥ 0.3 and <em>A</em> &lt; 0.3 exist in the similar arrangement with that in <em>B. subtilis</em>, cells are indicated as “+” and “(+)”, respectively. When DnaA box clusters do not exit, cells are indicated as “(Δ)”. “ → ” indicates orientation of genes (5′ to 3′) relative to DnaA-binding site located between them.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195330" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87195331 id="87195331" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, we developed a ChAP-chip method, based on our original protocol for the purification of protein complexes.<sup><a reveal-id="DSM017C30" data-open="DSM017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The binding sites of Spo0J determined using this technique were consistent with those reported earlier,<sup><a reveal-id="DSM017C31" data-open="DSM017C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSM017C37" data-open="DSM017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> clearly indicating that the ChAP-chip methods are sufficiently sensitive for the genome-wide determination of specific binding sites of DNA-binding proteins. Using this method, we have successfully visualized the distribution of RNA polymerase core enzyme and associated factors, SigA, NusA, and GreA (unpublished results). Thus, the ChAP-chip assay should be effective for proteins that are both stably and dynamically associated with genome DNA.</p><p>The initiator protein of chromosomal replication, DnaA, forms stable complexes at eight intergenic regions on the <em>B. subtilis</em> genome, including the <em>oriC</em> sequences, <em>oriC</em>1 and <em>oriC</em>2. <em>In vitro</em> assays demonstrate that DnaA protein binds to a single DnaA box.<sup><a reveal-id="DSM017C39" data-open="DSM017C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSM017C40" data-open="DSM017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> However, characterization of the DNA sequences bound to DnaA <em>in vivo</em> reveals that the presence of densely clustered DnaA boxes and affinity of these boxes for DnaA protein are critical factors for stable DNA–protein interactions <em>in vivo</em>. The recent crystal structure of <em>Aquifex aeolicus</em> DnaA bound to the non-hydrolyzable ATP analog, AMP-PCP, disclosed that ATP-DnaA forms a right-handed filament defined by specific protein–ATP interactions.<sup><a reveal-id="DSM017C3" data-open="DSM017C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM017C32" data-open="DSM017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> This finding strongly suggests that through filament formation of multiple ATP-DnaA bound to <em>oriC</em>, a right-handed <em>oriC</em> DNA wrap is formed around the initiation nucleoprotein complex. It is plausible that this type of higher-order structure of the DnaA–DNA complex is formed at the stable DnaA-binding sites detected by our group.</p><p>Notably, genes related to basic cellular functions are over-represented in neighboring genes. These include chromosome replication (<em>dnaA</em> and <em>dnaN</em>), translation (<em>yqeI, yqeH</em>: ribosome assembly; <em>gidA, thdF</em>: modification of tRNA; <em>gidB</em>: modification of 16S rRNA), and unknown essential functions (<em>gcp, ywlC</em>). In combination with transcriptome analysis of cells containing increased or decreased amounts of DnaA, we demonstrated that in addition to autoregulation, DnaA proteins stably associated with DNA directly regulate <em>sda</em> expression in a positive and <em>ywlC</em> and <em>yydA</em> expression in a negative manner. Furthermore, DnaA-dependent regulation of these genes was suggested to be maintained in other <em>Bacillus</em> species, indicating their biological significance. The DnaA-dependent control of <em>dnaA</em> and <em>sda</em> is well documented. In contrast, the functions of <em>ywlC</em> and <em>yydA</em> and the rationale for their DnaA-dependent regulation await further investigation. We additionally show that although <em>thdF-gidA-gidB-noc</em> expression is under control of DnaA in <em>B. subtilis</em>, this regulation is not universal in the <em>Bacillus</em> species.</p><p>DnaA generally acts as a repressor of transcription in <em>B. subtilis</em>, probably through blocking the formation of initiation complex of transcription and/or a transcriptional roadblock. However, the location of DnaA boxes and transcriptional organization of the <em>yqeG</em>–<em>sda</em> region suggests an alternative mechanism of transcriptional regulation by DnaA. Interestingly, <em>sda</em> transcription is positively regulated by DnaA. It is possible that DnaA interacts with RNA polymerase as a transcriptional activator, although experimental evidence of these interactions is yet to be obtained. We identified a novel transcript negatively regulated by DnaA, which traverses the <em>sda</em> gene in the opposite orientation. RNA polymerase collisions would occur if both transcriptions were active simultaneously. If these collisions negatively regulate <em>sda</em> transcription, negative regulation of the opposite reaction would have a positive effect on <em>sda</em> expression. Additionally, upregulation of the <em>sda</em> promoter may be mediated through changes in superhelicity of the DNA template. If DnaA forms helical filaments and bound DNA is wrapped around the DnaA helix, as discussed earlier, positive superhelicity will be stabilized, as demonstrated by Erzberger <em>et al.</em><sup><a reveal-id="DSM017C32" data-open="DSM017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Superhelicity of template DNA is an important factor in determining transcriptional activity.<sup><a reveal-id="DSM017C44" data-open="DSM017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSM017C45" data-open="DSM017C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>Estimation of stable DnaA-binding sequences using the <em>A</em> value in genome sequences of <em>Bacillus</em> species suggests that in addition to the upstream regions of <em>dnaA, dnaN, sda</em>, <em>yydA</em>, and <em>ywlC</em>, DnaA binds downstream of <em>gcp</em> in most bacteria examined. In addition, specific DnaA-binding sequences for each bacterium may be present in close proximity to <em>oriC</em>. These observations suggest that DnaA plays several additional roles, such as control of the amount of effective initiator, ATP-DnaA, as proposed for <em>E. coli</em>,<sup><a reveal-id="DSM017C46" data-open="DSM017C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and/or stabilization of domain structure of genome around <em>oriC</em> for the proper initiation of chromosome replication.</p><p>However, although the importance of orientation of adjacent DnaA boxes for stable binding of DnaA has been investigated in <em>E. coli</em>,<sup><a reveal-id="DSM017C47" data-open="DSM017C47" class="link link-ref link-reveal xref-bibr">47</a></sup> we did not take into account this factor to evaluate the DnaA-binding regions. Further improvements of bioinformatic methods to estimate the DnaA-binding regions are necessary.</p><p>In conclusion, we have analyzed the distribution of the DnaA protein on the <em>B. subtilis</em> genome in actively growing cells, and confirmed its role as a direct regulator of several genes. Moreover, we propose that DnaA has additional functions related to chromosome replication. Thus, genome-wide analysis of DnaA-binding sites should facilitate our understanding of the role of <em>B subtilis</em> DnaA as a coordinator of bacterial cell growth.</p>                            <h2 scrollto-destination=87195339 id="87195339" class="section-title" data-legacy-id=s5>Supplementry Data</h2>
<p>Supplementry data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=87195341 id="87195341" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by KAKENHI (Grant-in-Aid for Scientific Research) on Priority Areas Systems Genomics from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=87195343 id="87195343" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leonard</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Grimwade</div> <div class="given-names">J. E.</div></div>. </div><div class="article-title">Building a bacterial orisome: emergence of new regulatory features for replication origin unwinding</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">978</div>-<div class="lpage">985</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Building%20a%20bacterial%20orisome%3A%20emergence%20of%20new%20regulatory%20features%20for%20replication%20origin%20unwinding&amp;author=A.%20C.%20Leonard&amp;author=J.%20E.%20Grimwade&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=55&amp;pages=978-985" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04467.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04467.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04467.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15686547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaguni</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">DnaA: controlling the initiation of bacterial DNA replication and more</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">351</div>-<div class="lpage">375</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%3A%20controlling%20the%20initiation%20of%20bacterial%20DNA%20replication%20and%20more&amp;author=J.%20M.%20Kaguni&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=60&amp;pages=351-375" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.60.080805.142111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.60.080805.142111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.60.080805.142111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16753031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mott</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">DNA replication initiation: mechanisms and regulation in bacteria</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">343</div>-<div class="lpage">354</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20replication%20initiation%3A%20mechanisms%20and%20regulation%20in%20bacteria&amp;author=M.%20L.%20Mott&amp;author=J.%20M.%20Berger&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=5&amp;pages=343-354" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17435790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zakrzewska-Czerwinska</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jakimowicz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zawilak-Pawlik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Regulation of the initiation of chromosomal replication in bacteria</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2007</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">378</div>-<div class="lpage">387</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20the%20initiation%20of%20chromosomal%20replication%20in%20bacteria&amp;author=J.%20Zakrzewska-Czerwinska&amp;author=D.%20Jakimowicz&amp;author=A.%20Zawilak-Pawlik&amp;author=W.%20Messer&amp;publication_year=2007&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=31&amp;pages=378-387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2007.00070.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2007.00070.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2007.00070.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17459114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Braun</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">O'Day</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Autoregulation of the DNA replication gene <em>dnaA</em> in <em>E. coli</em> K-12</div>, <div class="source">Cell</div> , <div class="year">1985</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">159</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autoregulation%20of%20the%20DNA%20replication%20gene%20dnaA%20in%20E.%20coli%20K-12&amp;author=R.%20E.%20Braun&amp;author=K.%20O'Day&amp;author=A.%20Wright&amp;publication_year=1985&amp;journal=Cell&amp;volume=40&amp;pages=159-169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(85)90319-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(85)90319-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(85)90319-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2981626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Imai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Autoregulation of the <em>dnaA-dnaN</em> operon and effects of DnaA protein levels on replication initiation in <em>Bacillus subtilis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">3833</div>-<div class="lpage">3841</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autoregulation%20of%20the%20dnaA-dnaN%20operon%20and%20effects%20of%20DnaA%20protein%20levels%20on%20replication%20initiation%20in%20Bacillus%20subtilis&amp;author=Y.%20Ogura&amp;author=Y.%20Imai&amp;author=N.%20Ogasawara&amp;author=S.%20Moriya&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=3833-3841" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.13.3833-3841.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.13.3833-3841.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.13.3833-3841.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11395445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jakimowicz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Majka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lis</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Konopa</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wegrzyn</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schrempf</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zakrzewska-Czerwinska</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Structure and regulation of the <em>dnaA</em> promoter region in three <em>Streptomyces</em> species</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">1093</div>-<div class="lpage">1102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20regulation%20of%20the%20dnaA%20promoter%20region%20in%20three%20Streptomyces%20species&amp;author=D.%20Jakimowicz&amp;author=J.%20Majka&amp;author=B.%20Lis&amp;author=G.%20Konopa&amp;author=G.%20Wegrzyn&amp;author=W.%20Messer&amp;author=H.%20Schrempf&amp;author=J.%20Zakrzewska-Czerwinska&amp;publication_year=2000&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=262&amp;pages=1093-1102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00008652" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00008652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00008652"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10660070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salazar</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Guerrero</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Casart</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Turcios</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bartoli</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Transcription analysis of the <em>dnaA</em> gene and <em>oriC</em> region of the chromosome of <em>Mycobacterium smegmatis</em> and <em>Mycobacterium bovis</em> BCG, and its regulation by the DnaA protein</div>, <div class="source">Microbiology</div> , <div class="year">2003</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">773</div>-<div class="lpage">784</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20analysis%20of%20the%20dnaA%20gene%20and%20oriC%20region%20of%20the%20chromosome%20of%20Mycobacterium%20smegmatis%20and%20Mycobacterium%20bovis%20BCG%2C%20and%20its%20regulation%20by%20the%20DnaA%20protein&amp;author=L.%20Salazar&amp;author=E.%20Guerrero&amp;author=Y.%20Casart&amp;author=L.%20Turcios&amp;author=F.%20Bartoli&amp;publication_year=2003&amp;journal=Microbiology&amp;volume=149&amp;pages=773-784" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.25832-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.25832-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.25832-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12634345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greendyke</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rajagopalan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Parish</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Madiraju</div> <div class="given-names">M. V.</div></div>. </div><div class="article-title">Conditional expression of <em>Mycobacterium smegmatis dnaA</em>, an essential DNA replication gene</div>, <div class="source">Microbiology</div> , <div class="year">2002</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">3887</div>-<div class="lpage">3900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conditional%20expression%20of%20Mycobacterium%20smegmatis%20dnaA%2C%20an%20essential%20DNA%20replication%20gene&amp;author=R.%20Greendyke&amp;author=M.%20Rajagopalan&amp;author=T.%20Parish&amp;author=M.%20V.%20Madiraju&amp;publication_year=2002&amp;journal=Microbiology&amp;volume=148&amp;pages=3887-3900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-148-12-3887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-148-12-3887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-148-12-3887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12480893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">DnaA initiator—also a transcription factor</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%20initiator%E2%80%94also%20a%20transcription%20factor&amp;author=W.%20Messer&amp;author=C.%20Weigel&amp;publication_year=1997&amp;journal=Mol.%20Microbiol.&amp;volume=24&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.3171678.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1997.3171678.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1997.3171678.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9140960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nozaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Okazaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">In vitro transcription of the origin region of replication of the <em>Escherichia coli</em> chromosome</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1988</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">14176</div>-<div class="lpage">14183</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vitro%20transcription%20of%20the%20origin%20region%20of%20replication%20of%20the%20Escherichia%20coli%20chromosome&amp;author=N.%20Nozaki&amp;author=T.%20Okazaki&amp;author=T.%20Ogawa&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem.&amp;volume=263&amp;pages=14176-14183" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2844760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Q. P.</div></div>,  <div class="name"><div class="surname">Kaguni</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title"><em>dnaA</em> protein regulates transcriptions of the <em>rpoH</em> gene of <em>Escherichia coli</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1989</div>, vol. <div class="volume">264</div> (pg. <div class="fpage">7338</div>-<div class="lpage">7344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dnaA%20protein%20regulates%20transcriptions%20of%20the%20rpoH%20gene%20of%20Escherichia%20coli&amp;author=Q.%20P.%20Wang&amp;author=J.%20M.%20Kaguni&amp;publication_year=1989&amp;journal=J.%20Biol.%20Chem.&amp;volume=264&amp;pages=7338-7344" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2540187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinones</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wandt</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kleinstauber</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>. </div><div class="article-title">DnaA protein stimulates <em>polA</em> gene expression in <em>Escherichia coli</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1193</div>-<div class="lpage">1202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%20protein%20stimulates%20polA%20gene%20expression%20in%20Escherichia%20coli&amp;author=A.%20Quinones&amp;author=G.%20Wandt&amp;author=S.%20Kleinstauber&amp;author=W.%20Messer&amp;publication_year=1997&amp;journal=Mol.%20Microbiol.&amp;volume=23&amp;pages=1193-1202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.2961658.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1997.2961658.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1997.2961658.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9106210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Theisen</div> <div class="given-names">P. W.</div></div>,  <div class="name"><div class="surname">Grimwade</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Leonard</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Bogan</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Helmstetter</div> <div class="given-names">C. E.</div></div>. </div><div class="article-title">Correlation of gene transcription with the time of initiation of chromosome replication in <em>Escherichia coli</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1993</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">575</div>-<div class="lpage">584</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20of%20gene%20transcription%20with%20the%20time%20of%20initiation%20of%20chromosome%20replication%20in%20Escherichia%20coli&amp;author=P.%20W.%20Theisen&amp;author=J.%20E.%20Grimwade&amp;author=A.%20C.%20Leonard&amp;author=J.%20A.%20Bogan&amp;author=C.%20E.%20Helmstetter&amp;publication_year=1993&amp;journal=Mol.%20Microbiol.&amp;volume=10&amp;pages=575-584" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1993.10.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1993.10.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1993.10.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7968535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okazaki</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Cell cycle-dependent transcription from the <em>gid</em> and <em>mioC</em> promoters of <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">1609</div>-<div class="lpage">1615</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20cycle-dependent%20transcription%20from%20the%20gid%20and%20mioC%20promoters%20of%20Escherichia%20coli&amp;author=T.%20Ogawa&amp;author=T.%20Okazaki&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=1609-1615" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.6.1609-1615.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.6.1609-1615.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.6.1609-1615.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8132454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Camara</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Klungsoyr</div> <div class="given-names">H. K.</div></div>,  <div class="name"><div class="surname">Crooke</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Skarstad</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Beckwith</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A novel regulatory mechanism couples deoxyribonucleotide synthesis and DNA replication in <em>Escherichia coli</em></div>, <div class="source">EMBO J.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1137</div>-<div class="lpage">1147</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20regulatory%20mechanism%20couples%20deoxyribonucleotide%20synthesis%20and%20DNA%20replication%20in%20Escherichia%20coli&amp;author=S.%20Gon&amp;author=J.%20E.%20Camara&amp;author=H.%20K.%20Klungsoyr&amp;author=E.%20Crooke&amp;author=K.%20Skarstad&amp;author=J.%20Beckwith&amp;publication_year=2006&amp;journal=EMBO%20J.&amp;volume=25&amp;pages=1137-1147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.emboj.7600990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.emboj.7600990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.emboj.7600990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16482221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Augustin</div> <div class="given-names">L. B.</div></div>,  <div class="name"><div class="surname">Jacobson</div> <div class="given-names">B. A.</div></div>,  <div class="name"><div class="surname">Fuchs</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> Fis and DnaA proteins bind specifically to the <em>nrd</em> promoter region and affect expression of an <em>nrd-lac</em> fusion</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">378</div>-<div class="lpage">387</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20Fis%20and%20DnaA%20proteins%20bind%20specifically%20to%20the%20nrd%20promoter%20region%20and%20affect%20expression%20of%20an%20nrd-lac%20fusion&amp;author=L.%20B.%20Augustin&amp;author=B.%20A.%20Jacobson&amp;author=J.%20A.%20Fuchs&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=378-387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.2.378-387.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.2.378-387.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.2.378-387.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8288532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herrick</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sclavi</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Ribonucleotide reductase and the regulation of DNA replication: an old story and an ancient heritage</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">22</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribonucleotide%20reductase%20and%20the%20regulation%20of%20DNA%20replication%3A%20an%20old%20story%20and%20an%20ancient%20heritage&amp;author=J.%20Herrick&amp;author=B.%20Sclavi&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=63&amp;pages=22-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.63.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.63.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.63.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17229208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kuroda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kishi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The <em>datA</em> locus predominantly contributes to the initiator titration mechanism in the control of replication initiation in <em>Escherichia coli</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">1367</div>-<div class="lpage">1375</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20datA%20locus%20predominantly%20contributes%20to%20the%20initiator%20titration%20mechanism%20in%20the%20control%20of%20replication%20initiation%20in%20Escherichia%20coli&amp;author=T.%20Ogawa&amp;author=Y.%20Yamada&amp;author=T.%20Kuroda&amp;author=T.%20Kishi&amp;author=S.%20Moriya&amp;publication_year=2002&amp;journal=Mol.%20Microbiol.&amp;volume=44&amp;pages=1367-1375" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02969.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2002.02969.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2002.02969.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12068813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morigen</div> <div class="given-names"></div></div>,  <div class="name"><div class="surname">Molina</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Skarstad</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Deletion of the <em>datA</em> site does not affect once-per-cell-cycle timing but induces rifampin-resistant replication</div>, <div class="source">Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">3913</div>-<div class="lpage">3920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion%20of%20the%20datA%20site%20does%20not%20affect%20once-per-cell-cycle%20timing%20but%20induces%20rifampin-resistant%20replication&amp;author=%20Morigen&amp;author=F.%20Molina&amp;author=K.%20Skarstad&amp;publication_year=2005&amp;journal=Bacteriol.&amp;volume=187&amp;pages=3913-3920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.12.3913-3920.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.12.3913-3920.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.12.3913-3920.2005"> </span></div> </div></div></div></div></div><div content-id="DSM017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smulczyk-Krawczyszyn</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jakimowicz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ruban-Osmialowska</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zawilak-Pawlik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Majka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chater</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zakrzewska-Czerwinska</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Cluster of DnaA boxes involved in regulation of <em>Streptomyces</em> chromosome replication: from in silico to in vivo studies</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">6184</div>-<div class="lpage">6194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20of%20DnaA%20boxes%20involved%20in%20regulation%20of%20Streptomyces%20chromosome%20replication%3A%20from%20in%20silico%20to%20in%20vivo%20studies&amp;author=A.%20Smulczyk-Krawczyszyn&amp;author=D.%20Jakimowicz&amp;author=B.%20Ruban-Osmialowska&amp;author=A.%20Zawilak-Pawlik&amp;author=J.%20Majka&amp;author=K.%20Chater&amp;author=J.%20Zakrzewska-Czerwinska&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=6184-6194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00528-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00528-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00528-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16923885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burkholder</div> <div class="given-names">W. F.</div></div>,  <div class="name"><div class="surname">Kurtser</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Grossman</div> <div class="given-names">A. D.</div></div>. </div><div class="article-title">Replication initiation proteins regulate a developmental checkpoint in <em>Bacillus subtilis</em></div>, <div class="source">Cell</div> , <div class="year">2001</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">269</div>-<div class="lpage">279</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20initiation%20proteins%20regulate%20a%20developmental%20checkpoint%20in%20Bacillus%20subtilis&amp;author=W.%20F.%20Burkholder&amp;author=I.%20Kurtser&amp;author=A.%20D.%20Grossman&amp;publication_year=2001&amp;journal=Cell&amp;volume=104&amp;pages=269-279" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(01)00211-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(01)00211-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(01)00211-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11207367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goranov</div> <div class="given-names">A. I.</div></div>,  <div class="name"><div class="surname">Katz</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Breier</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Burge</div> <div class="given-names">C. B.</div></div>,  <div class="name"><div class="surname">Grossman</div> <div class="given-names">A. D.</div></div>. </div><div class="article-title">A transcriptional response to replication status mediated by the conserved bacterial replication protein DnaA</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A.</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">12932</div>-<div class="lpage">12937</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transcriptional%20response%20to%20replication%20status%20mediated%20by%20the%20conserved%20bacterial%20replication%20protein%20DnaA&amp;author=A.%20I.%20Goranov&amp;author=L.%20Katz&amp;author=A.%20M.%20Breier&amp;author=C.%20B.%20Burge&amp;author=A.%20D.%20Grossman&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;volume=102&amp;pages=12932-12937" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506174102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506174102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506174102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16120674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hottes</div> <div class="given-names">A. K.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McAdams</div> <div class="given-names">H. H.</div></div>. </div><div class="article-title">DnaA coordinates replication initiation and cell cycle transcription in <em>Caulobacter crescentus</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">1340</div>-<div class="lpage">1353</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%20coordinates%20replication%20initiation%20and%20cell%20cycle%20transcription%20in%20Caulobacter%20crescentus&amp;author=A.%20K.%20Hottes&amp;author=L.%20Shapiro&amp;author=H.%20H.%20McAdams&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=58&amp;pages=1340-1353" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04912.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2005.04912.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2005.04912.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16313620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Collier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>. </div><div class="article-title">DnaA couples DNA replication and the expression of two cell cycle master regulators</div>, <div class="source">EMBO J.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">346</div>-<div class="lpage">356</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaA%20couples%20DNA%20replication%20and%20the%20expression%20of%20two%20cell%20cycle%20master%20regulators&amp;author=J.%20Collier&amp;author=S.%20R.%20Murray&amp;author=L.%20Shapiro&amp;publication_year=2006&amp;journal=EMBO%20J.&amp;volume=25&amp;pages=346-356" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.emboj.7600927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.emboj.7600927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.emboj.7600927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16395331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneshiro</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Aburatani</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shirahige</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genomic approach for the understanding of dynamic aspect of chromosome behavior</div>, <div class="source">Methods Enzymol.</div> , <div class="year">2006</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">389</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20approach%20for%20the%20understanding%20of%20dynamic%20aspect%20of%20chromosome%20behavior&amp;author=Y.%20Katou&amp;author=K.%20Kaneshiro&amp;author=H.%20Aburatani&amp;author=K.%20Shirahige&amp;publication_year=2006&amp;journal=Methods%20Enzymol.&amp;volume=409&amp;pages=389-410" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16793414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Aiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> histone-like protein H-NS preferentially binds to horizontally acquired DNA in association with RNA polymerase</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">141</div>-<div class="lpage">153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20histone-like%20protein%20H-NS%20preferentially%20binds%20to%20horizontally%20acquired%20DNA%20in%20association%20with%20RNA%20polymerase&amp;author=T.%20Oshima&amp;author=S.%20Ishikawa&amp;author=K.%20Kurokawa&amp;author=H.%20Aiba&amp;author=N.%20Ogasawara&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=141-153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17046956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Navarre</div> <div class="given-names">W. W.</div></div>,  <div class="name"><div class="surname">Porwollik</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">McClelland</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rosen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Libby</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">F. C.</div></div>. </div><div class="article-title">Selective silencing of foreign DNA with low GC content by the H-NS protein in <em>Salmonella</em></div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">236</div>-<div class="lpage">238</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20silencing%20of%20foreign%20DNA%20with%20low%20GC%20content%20by%20the%20H-NS%20protein%20in%20Salmonella&amp;author=W.%20W.%20Navarre&amp;author=S.%20Porwollik&amp;author=Y.%20Wang&amp;author=M.%20McClelland&amp;author=H.%20Rosen&amp;author=S.%20J.%20Libby&amp;author=F.%20C.%20Fang&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=236-238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128794" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128794"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16763111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lucchini</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rowley</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Goldberg</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Hurd</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hinton</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">H-NS mediates the silencing of laterally acquired genes in bacteria</div>, <div class="source">PLoS Pathogens.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">746</div>-<div class="lpage">752</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H-NS%20mediates%20the%20silencing%20of%20laterally%20acquired%20genes%20in%20bacteria&amp;author=S.%20Lucchini&amp;author=G.%20Rowley&amp;author=M.%20D.%20Goldberg&amp;author=D.%20Hurd&amp;author=M.%20Harrison&amp;author=J.C.%20Hinton&amp;publication_year=2006&amp;journal=PLoS%20Pathogens.&amp;volume=2&amp;pages=746-752" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.0020081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.0020081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.0020081"> </span></div> </div></div></div></div></div><div content-id="DSM017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hiramatsu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kuwano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title">A new FtsZ-interacting protein, YlmF, complements the activity of FtsA during progression of cell division in <em>Bacillus subtilis</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1364</div>-<div class="lpage">1380</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20FtsZ-interacting%20protein%2C%20YlmF%2C%20complements%20the%20activity%20of%20FtsA%20during%20progression%20of%20cell%20division%20in%20Bacillus%20subtilis&amp;author=S.%20Ishikawa&amp;author=Y.%20Kawai&amp;author=K.%20Hiramatsu&amp;author=M.%20Kuwano&amp;author=N.%20Ogasawara&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=60&amp;pages=1364-1380" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.60.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.60.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.60.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16796675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">D. C.</div></div>,  <div class="name"><div class="surname">Grossman</div> <div class="given-names">A. D.</div></div>. </div><div class="article-title">Identification and characterization of a bacterial chromosome partitioning site</div>, <div class="source">Cell</div> , <div class="year">1998</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">675</div>-<div class="lpage">685</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20a%20bacterial%20chromosome%20partitioning%20site&amp;author=D.%20C.%20Lin&amp;author=A.%20D.%20Grossman&amp;publication_year=1998&amp;journal=Cell&amp;volume=92&amp;pages=675-685" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81135-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81135-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81135-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9506522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Erzberger</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Mott</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Structural basis for ATP-dependent DnaA assembly and replication-origin remodeling</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">676</div>-<div class="lpage">683</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20ATP-dependent%20DnaA%20assembly%20and%20replication-origin%20remodeling&amp;author=J.%20P.%20Erzberger&amp;author=M.%20L.%20Mott&amp;author=J.%20M.%20Berger&amp;publication_year=2006&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=13&amp;pages=676-683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb1115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb1115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb1115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16829961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">The complete genome sequence of the gram-positive bacterium <em>Bacillus subtilis</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">249</div>-<div class="lpage">256</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-256" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Imai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hassan</div> <div class="given-names">A. K.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Regulation of initiation of <em>Bacillus subtilis</em> chromosome replication</div>, <div class="source">Plasmid</div> , <div class="year">1999</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">17</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20initiation%20of%20Bacillus%20subtilis%20chromosome%20replication&amp;author=S.%20Moriya&amp;author=Y.%20Imai&amp;author=A.%20K.%20Hassan&amp;author=N.%20Ogasawara&amp;publication_year=1999&amp;journal=Plasmid&amp;volume=41&amp;pages=17-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/plas.1998.1381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fplas.1998.1381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fplas.1998.1381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9887303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morimoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Loh</div> <div class="given-names">P. C.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Six GTP-binding proteins of the Era/Obg family are essential for cell growth in <em>Bacillus subtilis</em></div>, <div class="source">Microbiology</div> , <div class="year">2002</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">3539</div>-<div class="lpage">3552</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Six%20GTP-binding%20proteins%20of%20the%20Era%2FObg%20family%20are%20essential%20for%20cell%20growth%20in%20Bacillus%20subtilis&amp;author=T.%20Morimoto&amp;author=P.%20C.%20Loh&amp;author=T.%20Hirai&amp;author=K.%20Asai&amp;author=K.%20Kobayashi&amp;author=S.%20Moriya&amp;author=N.%20Ogasawara&amp;publication_year=2002&amp;journal=Microbiology&amp;volume=148&amp;pages=3539-3552" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-148-11-3539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-148-11-3539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-148-11-3539"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12427945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamane</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sekiguchi</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Regulation and characterization of a newly deduced cell wall hydrolase gene (<em>cwlJ</em>) which affects germination of <em>Bacillus subtilis</em> spores</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">1375</div>-<div class="lpage">1380</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20and%20characterization%20of%20a%20newly%20deduced%20cell%20wall%20hydrolase%20gene%20(cwlJ)%20which%20affects%20germination%20of%20Bacillus%20subtilis%20spores&amp;author=S.%20Ishikawa&amp;author=K.%20Yamane&amp;author=J.%20Sekiguchi&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=1375-1380" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9515903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ferreira</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Errington</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The bacterial chromosome segregation protein Spo0J spreads along DNA from <em>parS</em> nucleation sites</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">1352</div>-<div class="lpage">1361</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20chromosome%20segregation%20protein%20Spo0J%20spreads%20along%20DNA%20from%20parS%20nucleation%20sites&amp;author=H.%20Murray&amp;author=H.%20Ferreira&amp;author=J.%20Errington&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=61&amp;pages=1352-1361" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.61.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.61.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.61.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16925562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshikawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Structure and function of DnaA and the DnaA-box in eubacteria: evolutionary relationships of bacterial replication origins</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1991</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">2589</div>-<div class="lpage">2597</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20function%20of%20DnaA%20and%20the%20DnaA-box%20in%20eubacteria%3A%20evolutionary%20relationships%20of%20bacterial%20replication%20origins&amp;author=H.%20Yoshikawa&amp;author=N.%20Ogasawara&amp;publication_year=1991&amp;journal=Mol.%20Microbiol.&amp;volume=5&amp;pages=2589-2597" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1991.5.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1991.5.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1991.5.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1779750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukuoka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Moriya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshikawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Purification and characterization of an initiation protein for chromosomal replication, DnaA, in <em>Bacillus subtilis</em></div>, <div class="source">J. Biochem.</div> , <div class="year">1990</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">732</div>-<div class="lpage">739</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20an%20initiation%20protein%20for%20chromosomal%20replication%2C%20DnaA%2C%20in%20Bacillus%20subtilis&amp;author=T.%20Fukuoka&amp;author=S.%20Moriya&amp;author=H.%20Yoshikawa&amp;author=N.%20Ogasawara&amp;publication_year=1990&amp;journal=J.%20Biochem.&amp;volume=107&amp;pages=732-739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a123117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a123117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a123117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2168872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>. </div><div class="article-title">The bacterial replication initiator DnaA. DnaA and oriC, the bacterial mode to initiate DNA replication</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2002</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">355</div>-<div class="lpage">374</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20replication%20initiator%20DnaA.%20DnaA%20and%20oriC%2C%20the%20bacterial%20mode%20to%20initiate%20DNA%20replication&amp;author=W.%20Messer&amp;publication_year=2002&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=26&amp;pages=355-374" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12413665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujikawa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kurumizaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nureki</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Terada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shirouzu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yokoyama</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Structural basis of replication origin recognition by the DnaA protein</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2077</div>-<div class="lpage">2086</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20of%20replication%20origin%20recognition%20by%20the%20DnaA%20protein&amp;author=N.%20Fujikawa&amp;author=H.%20Kurumizaka&amp;author=O.%20Nureki&amp;author=T.%20Terada&amp;author=M.%20Shirouzu&amp;author=T.%20Katayama&amp;author=S.%20Yokoyama&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=2077-2086" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12682358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackiewicz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zakrzewska-Czerwinska</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zawilak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dudek</div> <div class="given-names">M. R.</div></div>,  <div class="name"><div class="surname">Cebrat</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Where does bacterial replication start? Rules for predicting the <em>oriC</em> region</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">3781</div>-<div class="lpage">3791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Where%20does%20bacterial%20replication%20start%3F%20Rules%20for%20predicting%20the%20oriC%20region&amp;author=P.%20Mackiewicz&amp;author=J.%20Zakrzewska-Czerwinska&amp;author=A.%20Zawilak&amp;author=M.%20R.%20Dudek&amp;author=S.%20Cebrat&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=3781-3791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15258248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansen</div> <div class="given-names">F. G.</div></div>,  <div class="name"><div class="surname">Christensen</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">C. B.</div></div>,  <div class="name"><div class="surname">Atlung</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Insights into the quality of DnaA boxes and their cooperativity</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">355</div> (pg. <div class="fpage">85</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20quality%20of%20DnaA%20boxes%20and%20their%20cooperativity&amp;author=F.%20G.%20Hansen&amp;author=B.%20B.%20Christensen&amp;author=C.%20B.%20Nielsen&amp;author=T.%20Atlung&amp;publication_year=2006&amp;journal=J.%20Mol.%20Biol.&amp;volume=355&amp;pages=85-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.10.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.10.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.10.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16298387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peter</div> <div class="given-names">B. J.</div></div>,  <div class="name"><div class="surname">Arsuaga</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Breier</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Khodursky</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div>,  <div class="name"><div class="surname">Cozzarelli</div> <div class="given-names">N. R.</div></div>. </div><div class="article-title">Genomic transcriptional response to loss of chromosomal supercoiling in <em>Escherichia coli</em></div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R87</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20transcriptional%20response%20to%20loss%20of%20chromosomal%20supercoiling%20in%20Escherichia%20coli&amp;author=B.%20J.%20Peter&amp;author=J.%20Arsuaga&amp;author=A.%20M.%20Breier&amp;author=A.B.%20Khodursky&amp;author=P.%20O.%20Brown&amp;author=N.%20R.%20Cozzarelli&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-11-r87" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-11-r87" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-11-r87"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15535863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blot</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Mavathur</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Geertz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Travers</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Muskhelishvili</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Homeostatic regulation of supercoiling sensitivity coordinates transcription of the bacterial genome</div>, <div class="source">EMBO Rep.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">710</div>-<div class="lpage">715</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homeostatic%20regulation%20of%20supercoiling%20sensitivity%20coordinates%20transcription%20of%20the%20bacterial%20genome&amp;author=N.%20Blot&amp;author=R.%20Mavathur&amp;author=M.%20Geertz&amp;author=A.%20Travers&amp;author=G.%20Muskhelishvili&amp;publication_year=2006&amp;journal=EMBO%20Rep.&amp;volume=7&amp;pages=710-715" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.embor.7400729" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.embor.7400729" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.embor.7400729"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16799466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansen</div> <div class="given-names">F. G.</div></div>,  <div class="name"><div class="surname">Christensen</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Atlung</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The Initiator titration model: computer simulation of chromosome and minichromosome control</div>, <div class="source">Res. Microbiol.</div> , <div class="year">1991</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">161</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Initiator%20titration%20model%3A%20computer%20simulation%20of%20chromosome%20and%20minichromosome%20control&amp;author=F.%20G.%20Hansen&amp;author=B.%20B.%20Christensen&amp;author=T.%20Atlung&amp;publication_year=1991&amp;journal=Res.%20Microbiol.&amp;volume=142&amp;pages=161-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0923-2508(91)90025-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0923-2508(91)90025-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0923-2508(91)90025-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1925015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM017C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansen</div> <div class="given-names">F. G.</div></div>,  <div class="name"><div class="surname">Christensen</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Atlung</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Sequence characteristics required for cooperative binding and efficient in vivo titration of the replication initiator protein DnaA in <em>E. coli</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">367</div> (pg. <div class="fpage">942</div>-<div class="lpage">952</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20characteristics%20required%20for%20cooperative%20binding%20and%20efficient%20in%20vivo%20titration%20of%20the%20replication%20initiator%20protein%20DnaA%20in%20E.%20coli&amp;author=F.%20G.%20Hansen&amp;author=B.%20B.%20Christensen&amp;author=T.%20Atlung&amp;publication_year=2007&amp;journal=J.%20Mol.%20Biol.&amp;volume=367&amp;pages=942-952" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2007.01.056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2007.01.056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2007.01.056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17316685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=466553&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm017/1/dsm017_Supplementary_Data.zip?Expires=2147483647&amp;Signature=PwN4y2vm2I2NoKUULWPV9w9513iwsa-pK-wj~VbV0Q3i1yghylXL6eT6EkU-ZAznHYfsQm7PywxUUFY5WWX0sphos7ma8mYZR0xyHgnupHhFef7iZOssbg3JzSbbm879ev6uP-GbtD6qdloStQRaHrVcBaN07rCgktYkRDcjlEQ-pmVpZMVBt296kGzxMF~20vkDFd53~83m6AUqujV1x9iKYy8fHmVu5BUlwk56PDrHMxY2qHTnig50~8r1zGK6x6xwi9iORNlpGTsRJiQ7bzepFDgYA0JAyt0uNfamf4p8Qq-YZhOTWwf7ZeAUst~TxZby-cGT5pDYTR12wvDz~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">202</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">51</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="466553"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f4%2f155%2f466553">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000250863200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (51)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F4%2F155%2F466553" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/466553'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Distribution of Stable DnaA-Binding Sites on the Bacillus Subtilis Genome Detected using a Modified ChIP-chip Method" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f155%2f466553">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

