

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/6/367/512658","@type":"ScholarlyArticle","name":"SVD-based Anatomy of Gene Expressions for Correlation Analysis in  Arabidopsis thaliana ","datePublished":"2008-10-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2008-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn025","keywords":["singular value decomposition","gene expression","gene correlation","Arabidopsis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/6/367/512658","author":[{"name":"Fukushima, Atsushi","affiliation":"1  RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Wada, Masayoshi","affiliation":"2  Department of Bioinformatics and Genomes, Graduate School of Information Science ,  Nara Institute of Science and Technology ,  Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Kanaya, Shigehiko","affiliation":"1  RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045 ,  Japan    2  Department of Bioinformatics and Genomes, Graduate School of Information Science ,  Nara Institute of Science and Technology ,  Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Arita, Masanori","affiliation":"1  RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045 ,  Japan    3  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  Kashiwa, Chiba 277-8561 ,  Japan    4  Institute for Advanced Biosciences ,  Keio University ,  Tsuruoka, Yamagata 997-0052 ,  Japan","@type":"Person"}],"description":"Abstract.  Gene co-expression analysis has been widely used in recent years for predicting unknown gene function and its regulatory mechanisms. The predictive a","pageStart":"367","pageEnd":"374","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"SVD-based Anatomy of Gene Expressions for Correlation Analysis in  Arabidopsis thaliana ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="SVD-based Anatomy of Gene Expressions for Correlation Analysis in  Arabidopsis thaliana " />
    <meta property="og:description" content="Abstract.  Gene co-expression analysis has been widely used in recent years for predicting unknown gene function and its regulatory mechanisms. The predictive a" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/6/367/512658" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-10-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Fukushima, Atsushi" /><meta name="citation_author_institution" content="RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Wada, Masayoshi" /><meta name="citation_author_institution" content="Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and Technology, Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Kanaya, Shigehiko" /><meta name="citation_author_institution" content="RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Department of Bioinformatics and Genomes, Graduate School of Information Science, Nara Institute of Science and Technology, Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Arita, Masanori" /><meta name="citation_author_institution" content="RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Yamagata 997-0052, Japan" /><meta name="citation_title" content="SVD-based Anatomy of Gene Expressions for Correlation Analysis in Arabidopsis thaliana" /><meta name="citation_firstpage" content="367" /><meta name="citation_lastpage" content="374" /><meta name="citation_doi" content="10.1093/dnares/dsn025" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2008/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=207-210; " /><meta name="citation_reference" content="citation_title=The Arabidopsis information resource (TAIR): a model organism database providing a centralized, curated gateway to Arabidopsis biology, research materials and community; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=224-228; " /><meta name="citation_reference" content="citation_title=NASCArrays: a repository for microarray data generated by NASC’s transcriptomics service; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D575-D577; " /><meta name="citation_reference" content="citation_title=CSB.DB: a comprehensive systems-biology database; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3647-3651; " /><meta name="citation_reference" content="citation_title=GENEVESTIGATOR. Arabidopsis microarray database and analysis toolbox; Plant Physiol.;  citation_year=2004;  citation_volume=136;  citation_pages=2621-2632; " /><meta name="citation_reference" content="citation_title=ATTED II: a database of co-expressed genes and cis elements for identifying co-regulated gene groups in Arabidopsis; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D863-D869; " /><meta name="citation_reference" content="citation_title=Approaches for extracting practical information from gene co-expression networks in plant biology; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=381-390; " /><meta name="citation_reference" content="citation_title=Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=6478-6483; " /><meta name="citation_reference" content="citation_title=Identification of brassinosteroid-related genes by means of transcript co-response analyses; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=2685-2696; " /><meta name="citation_reference" content="citation_title=Transcriptional coordination of the metabolic network in Arabidopsis; Plant Physiol.;  citation_year=2006;  citation_volume=142;  citation_pages=762-774; " /><meta name="citation_reference" content="citation_title=Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=8633-8638; " /><meta name="citation_reference" content="citation_title=Robust singular value decomposition analysis of microarray data; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=13167-13172; " /><meta name="citation_reference" content="citation_title=Singular value decomposition and principal component analysis; citation_publisher=Kluwer, Norwell, MA; A Practical Approach to Microarray Data Analysis;  citation_year=2003;  citation_pages=91-99; " /><meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=185-193; " /><meta name="citation_reference" content="citation_title=Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock; Science;  citation_year=2001;  citation_volume=293;  citation_pages=880-883; " /><meta name="citation_reference" content="citation_title=The R2R3-MYB transcription factor HAG1/MYB28 is a regulator of methionine-derived glucosinolate biosynthesis in Arabidopsis thaliana; Plant J.;  citation_year=2007;  citation_volume=51;  citation_pages=247-261; " /><meta name="citation_reference" content="citation_title=Principles of transcriptional control in the metabolic network of Saccharomyces cerevisiae; Nat. Biotechnol.;  citation_year=2004;  citation_volume=22;  citation_pages=86-92; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/6/367/1376046/dsn025.pdf" /><meta name="description" content="Abstract.  Gene co-expression analysis has been widely used in recent years for predicting unknown gene function and its regulatory mechanisms. The predictive a" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/6/367/512658" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/6/367/512658" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/6/367/512658");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn025';
    </script>
<title>
	SVD-based Anatomy of Gene Expressions for Correlation Analysis in Arabidopsis thaliana | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./512658" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bc0zPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWt1c2hpbWEsIEF0c3VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBQbGFudCBTY2llbmNlIENlbnRlciwgMS03LTIyIFRzdXJ1bWksIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYWRhLCBNYXNheW9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzIGFuZCBHZW5vbWVzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmF5YSwgU2hpZ2VoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gUGxhbnQgU2NpZW5jZSBDZW50ZXIsIDEtNy0yMiBUc3VydW1pLCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcyBhbmQgR2Vub21lcywgR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcml0YSwgTWFzYW5vcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBQbGFudCBTY2llbmNlIENlbnRlciwgMS03LTIyIFRzdXJ1bWksIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIGZvciBBZHZhbmNlZCBCaW9zY2llbmNlcywgS2VpbyBVbml2ZXJzaXR5LCBUc3VydW9rYSwgWWFtYWdhdGEgOTk3LTAwNTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTVkQtYmFzZWQgQW5hdG9teSBvZiBHZW5lIEV4cHJlc3Npb25zIGZvciBDb3JyZWxhdGlvbiBBbmFseXNpcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIEV4cHJlc3Npb24gT21uaWJ1czogTkNCSSBnZW5lIGV4cHJlc3Npb24gYW5kIGh5YnJpZGl6YXRpb24gYXJyYXkgZGF0YSByZXBvc2l0b3J5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMDctMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgaW5mb3JtYXRpb24gcmVzb3VyY2UgKFRBSVIpOiBhIG1vZGVsIG9yZ2FuaXNtIGRhdGFiYXNlIHByb3ZpZGluZyBhIGNlbnRyYWxpemVkLCBjdXJhdGVkIGdhdGV3YXkgdG8gQXJhYmlkb3BzaXMgYmlvbG9neSwgcmVzZWFyY2ggbWF0ZXJpYWxzIGFuZCBjb21tdW5pdHk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNC0yMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5BU0NBcnJheXM6IGEgcmVwb3NpdG9yeSBmb3IgbWljcm9hcnJheSBkYXRhIGdlbmVyYXRlZCBieSBOQVND4oCZcyB0cmFuc2NyaXB0b21pY3Mgc2VydmljZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDU3NS1ENTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DU0IuREI6IGEgY29tcHJlaGVuc2l2ZSBzeXN0ZW1zLWJpb2xvZ3kgZGF0YWJhc2U7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzY0Ny0zNjUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HRU5FVkVTVElHQVRPUi4gQXJhYmlkb3BzaXMgbWljcm9hcnJheSBkYXRhYmFzZSBhbmQgYW5hbHlzaXMgdG9vbGJveDsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9MjYyMS0yNjMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BVFRFRCBJSTogYSBkYXRhYmFzZSBvZiBjby1leHByZXNzZWQgZ2VuZXMgYW5kIGNpcyBlbGVtZW50cyBmb3IgaWRlbnRpZnlpbmcgY28tcmVndWxhdGVkIGdlbmUgZ3JvdXBzIGluIEFyYWJpZG9wc2lzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EODYzLUQ4Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcHJvYWNoZXMgZm9yIGV4dHJhY3RpbmcgcHJhY3RpY2FsIGluZm9ybWF0aW9uIGZyb20gZ2VuZSBjby1leHByZXNzaW9uIG5ldHdvcmtzIGluIHBsYW50IGJpb2xvZ3k7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0zODEtMzkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbWljcy1iYXNlZCBpZGVudGlmaWNhdGlvbiBvZiBBcmFiaWRvcHNpcyBNeWIgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIHJlZ3VsYXRpbmcgYWxpcGhhdGljIGdsdWNvc2lub2xhdGUgYmlvc3ludGhlc2lzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTY0NzgtNjQ4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYnJhc3Npbm9zdGVyb2lkLXJlbGF0ZWQgZ2VuZXMgYnkgbWVhbnMgb2YgdHJhbnNjcmlwdCBjby1yZXNwb25zZSBhbmFseXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MjY4NS0yNjk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgY29vcmRpbmF0aW9uIG9mIHRoZSBtZXRhYm9saWMgbmV0d29yayBpbiBBcmFiaWRvcHNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTQyOyAgY2l0YXRpb25fcGFnZXM9NzYyLTc3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgZ2VuZXMgcmVxdWlyZWQgZm9yIGNlbGx1bG9zZSBzeW50aGVzaXMgYnkgcmVncmVzc2lvbiBhbmFseXNpcyBvZiBwdWJsaWMgbWljcm9hcnJheSBkYXRhIHNldHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9ODYzMy04NjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2J1c3Qgc2luZ3VsYXIgdmFsdWUgZGVjb21wb3NpdGlvbiBhbmFseXNpcyBvZiBtaWNyb2FycmF5IGRhdGE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTMxNjctMTMxNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmd1bGFyIHZhbHVlIGRlY29tcG9zaXRpb24gYW5kIHByaW5jaXBhbCBjb21wb25lbnQgYW5hbHlzaXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1LbHV3ZXIsIE5vcndlbGwsIE1BOyBBIFByYWN0aWNhbCBBcHByb2FjaCB0byBNaWNyb2FycmF5IERhdGEgQW5hbHlzaXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl9wYWdlcz05MS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJpc29uIG9mIG5vcm1hbGl6YXRpb24gbWV0aG9kcyBmb3IgaGlnaCBkZW5zaXR5IG9saWdvbnVjbGVvdGlkZSBhcnJheSBkYXRhIGJhc2VkIG9uIHZhcmlhbmNlIGFuZCBiaWFzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE4NS0xOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2lwcm9jYWwgcmVndWxhdGlvbiBiZXR3ZWVuIFRPQzEgYW5kIExIWS9DQ0ExIHdpdGhpbiB0aGUgQXJhYmlkb3BzaXMgY2lyY2FkaWFuIGNsb2NrOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5MzsgIGNpdGF0aW9uX3BhZ2VzPTg4MC04ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSMlIzLU1ZQiB0cmFuc2NyaXB0aW9uIGZhY3RvciBIQUcxL01ZQjI4IGlzIGEgcmVndWxhdG9yIG9mIG1ldGhpb25pbmUtZGVyaXZlZCBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0yNDctMjYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmluY2lwbGVzIG9mIHRyYW5zY3JpcHRpb25hbCBjb250cm9sIGluIHRoZSBtZXRhYm9saWMgbmV0d29yayBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz04Ni05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzYvMzY3LzEzNzYwNDYvZHNuMDI1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBHZW5lIGNvLWV4cHJlc3Npb24gYW5hbHlzaXMgaGFzIGJlZW4gd2lkZWx5IHVzZWQgaW4gcmVjZW50IHllYXJzIGZvciBwcmVkaWN0aW5nIHVua25vd24gZ2VuZSBmdW5jdGlvbiBhbmQgaXRzIHJlZ3VsYXRvcnkgbWVjaGFuaXNtcy4gVGhlIHByZWRpY3RpdmUgYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTEyNjU4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZPE7fe27rm1uA9xqkJyRKjZFxGe+" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f367%2f512658" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f367%2f512658" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20280" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&amp;Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/6">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7964698">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964698">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964703">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964703">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964712">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964712">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964747">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964747">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964877">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964877">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964880">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964880">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964882">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964882">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7964884">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7964884">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/6/357/513030">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/6/375/512736">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">SVD-based Anatomy of Gene Expressions for Correlation Analysis in <em>Arabidopsis thaliana</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Fukushima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Fukushima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukushima A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukushima A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Fukushima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayoshi Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioinformatics and Genomes, Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayoshi+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masayoshi Wada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiko Kanaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioinformatics and Genomes, Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiko+Kanaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanaya S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanaya S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigehiko Kanaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masanori Arita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">RIKEN Plant Science Center, 1-7-22 Tsurumi, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Institute for Advanced Biosciences</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Tsuruoka, Yamagata 997-0052</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. E-mail: <a href="mailto:arita@k.u-tokyo.ac.jp" target="_blank">arita@k.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masanori+Arita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arita M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arita M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masanori Arita</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 367–374, <a href='https://doi.org/10.1093/dnares/dsn025'>https://doi.org/10.1093/dnares/dsn025</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 October 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 July 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 September 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="512658" href="/dnaresearch/article-pdf/15/6/367/1376046/dsn025.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Atsushi Fukushima, Masayoshi Wada, Shigehiko Kanaya, Masanori Arita;  SVD-based Anatomy of Gene Expressions for Correlation Analysis in <em>Arabidopsis thaliana</em>, <em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 367–374, <a href='https://doi.org/10.1093/dnares/dsn025'>https://doi.org/10.1093/dnares/dsn025</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/512658?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/512658?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=SVD-based%20Anatomy%20of%20Gene%20Expressions%20for%20Correlation%20Analysis%20in%20%26lt%3Bem%26gt%3BArabidopsis%20thaliana%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2008-10-17&amp;volumeNum=15&amp;issueNum=6&amp;author=Fukushima%2C%20Atsushi%3B%20Wada%2C%20Masayoshi&amp;startPage=367&amp;endPage=374&amp;contentId=10.1093%2Fdnares%2Fdsn025&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f367%2f512658" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7964698 id="7964698" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene co-expression analysis has been widely used in recent years for predicting unknown gene function and its regulatory mechanisms. The predictive accuracy depends on the quality and the diversity of data set used. In this report, we applied singular value decomposition (SVD) to array experiments in public databases to find that co-expression linkage could be estimated by a much smaller number of array data. Correlations of co-expressed gene were assessed using two regulatory mechanisms (feedback loop of the fundamental circadian clock and a global transcription factor Myb28), as well as metabolic pathways in the AraCyc database. Our conclusion is that a smaller number of informative arrays across tissues can suffice to reproduce comparable results with a state-of-the-art co-expression software tool. In our SVD analysis on Arabidopsis data set, array experiments that contributed most as the principal components included stamen development, germinating seed and stress responses on leaf.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;singular value decomposition&quot;">singular value decomposition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene expression&quot;">gene expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene correlation&quot;">gene correlation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis">Arabidopsis</a></div>                            <h2 scrollto-destination=7964703 id="7964703" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Oligonucleotide microarrays such as Affymetrix GeneChip have opened opportunities for the high-throughput observation of gene expressions. For the model plant <em>Arabidopsis thaliana</em> (<em>A. thaliana</em>), &gt;3000 gene-expression data have been measured by different research groups and stored in online repositories such as Gene Expression Omnibus (GEO),<sup><a reveal-id="DSN025C1" data-open="DSN025C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The Arabidopsis Information Resource (TAIR),<sup><a reveal-id="DSN025C2" data-open="DSN025C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and the Nottingham Arabidopsis Stock Centre Arrays (NASC).<sup><a reveal-id="DSN025C3" data-open="DSN025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Also available are the functional prediction tools based on gene co-expression, such as <a href="mailto:AthCoR@CSB.DB" target="_blank">AthCoR@CSB.DB</a>,<sup><a reveal-id="DSN025C4" data-open="DSN025C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Genevestigator,<sup><a reveal-id="DSN025C5" data-open="DSN025C5" class="link link-ref link-reveal xref-bibr">5</a></sup> ATTED-II<sup><a reveal-id="DSN025C6" data-open="DSN025C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and KAGIANA.<sup><a reveal-id="DSN025C7" data-open="DSN025C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Most of the prediction tools measure similarity of co-expression by Pearson’s or Spearman’s rank correlation with <em>P</em>-value across various biological and experimental conditions. Such similarity measure has been exploited to identify functioning genes among candidates otherwise indistinguishable from sequence annotations.<sup><a reveal-id="DSN025C8" data-open="DSN025C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSN025C9" data-open="DSN025C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Since correlation coefficient depends on the quality and the number of data sets, the selection of expression data is crucial for better prediction. For example, Pearson’s correlation results in bad estimates under the existence of outliers, or when the relationship between genes is nonlinear. Revealing complex gene-to-gene relationship such as in primary metabolism therefore requires a careful data pre-processing, i.e. selection of microarray data to delineate ‘true’ gene correlations. For example, Obayashi et al. used empirically weighted Pearson’s correlation in their ATTED-II server to reduce information redundancy in the 1388 GeneChip data from TAIR (see also the help page in the web site <a class="link link-uri" href="http://www.atted.bio.titech.ac.jp/">http://www.atted.bio.titech.ac.jp/</a>). Wei et al.<sup><a reveal-id="DSN025C10" data-open="DSN025C10" class="link link-ref link-reveal xref-bibr">10</a></sup> manually selected 486 so-called ‘high-quality’ GeneChip data from NASC so that computed correlation would be biologically meaningful. Although effectiveness of such strategies has been demonstrated in several studies,<sup><a reveal-id="DSN025C8" data-open="DSN025C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSN025C11" data-open="DSN025C11" class="link link-ref link-reveal xref-bibr">11</a></sup> it is unclear how much data are required, or which data repository are to be used. Data bias such as tissue distribution in repositories is also unknown. We examined three major online repositories (TAIR, NASC and GEO) and confirmed the benefit of using different, but not necessarily all, GeneChip data. Our study is based on singular value decomposition (SVD)<sup><a reveal-id="DSN025C12" data-open="DSN025C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN025C13" data-open="DSN025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and AraCyc metabolic pathways for overall verification of gene co-expressions.</p>                            <h2 scrollto-destination=7964712 id="7964712" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7964717 id="7964717" class="section-title" data-legacy-id=s2a>2.1. Gene-expression data sources and pre-processing</h3>
<p>In this study, we collected and merged data from three major online repositories for <em>A. thaliana</em> gene expressions: TAIR (<a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>), NASC (<a class="link link-uri" href="http://affymetrix.arabidopsis.info/">http://affymetrix.arabidopsis.info/</a>) and GEO (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>). After removing redundancy, the combined data set resulted in 2364 Affymetrix ATH1 GeneChip CEL files. (We used only ATH1 chips, which cover 80% of all genes with 23 000 probes. AG chips with 8000 probes were discarded). Each file was manually classified according to their sample tissue and experimental conditions. The classified data represented 133 experimental series, which are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The raw CEL files were pre-processed by the Robust Multi-chip Average (RMA) Algorithm,<sup><a reveal-id="DSN025C14" data-open="DSN025C14" class="link link-ref link-reveal xref-bibr">14</a></sup> in which perfect match intensities of array probes are modeled as the sum of exponential and Gaussian distributions for the signal and background, respectively.</p>                            <h3 scrollto-destination=7964726 id="7964726" class="section-title" data-legacy-id=s2b>2.2. SVD compression of data matrix</h3>
<div class="block-child-p">SVD was used to reduce the dimension of signal data. Similar to principal component analysis, it produces the best lower rank approximation of the original data matrix. The technique decomposes a data matrix <em>A</em> (<em>m</em> × <em>n</em> matrix) into three matrices, <em>U</em> (<em>m</em> × <em>m</em> matrix), <em>V</em> (<em>n</em> × <em>n</em> matrix), and Σ (<em>m</em> × <em>n</em> diagonal matrix) as follows: <a id="DSN025M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN025M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02507.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02507.gif?Expires=2147483647&amp;Signature=p4DMTHDV~P3z32umxwU4ZbPBn0xgfGzIXUgrp3ZOEm6uuwt~WUMLSrVgW6HylhOiRW~abDvmcNKgiMi90OKCYWga58Z-PKLZDr1r-nkeaqt7V8PEkhNYB9ge3mGn4XAs0UficqHQOjkhHUTrK~LLcNqWEkOkZAIzVJwuecqHAXgsCI49TQ7yu3Mzg5E515h7I440g6gn2dCCPpQVBdwuhzfmxlIucGwxPOGpyfoa1~M5et6SkhcTA3eUczhzGMuwcSRcIEqFMtSYXyQ8MrQgS7EsyOFT8HhOSObY-xFg9w60G53WDWbsTh92t4zHYxbObFcRkPwN6oqLWOsERKT4~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02507.gif" /></a></div></div><div class="label">1</div></div> where T denotes transpose. The diagonal of Σ are called singular values (SVs) and their absolute values plotted against their sorted ranks often display a power-law distribution in real world problems. In our analysis, the distribution was modeled as <em>y</em> = <em>x</em><sup>−0.88</sup> (data not shown). In such cases, the original matrix can be well approximated by zeroing all SVs except <em>k</em> largest ones as in <a id="DSN025M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN025M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02508.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02508.gif?Expires=2147483647&amp;Signature=UrhPklqTkuk25sBHqKzKWrQXf84f9qjDltde3grlPPR9HJ5iu5eiY85kyZAFQDTcS8ye7DF-KRBPkd6sc2rvp27gHZCJPtUGKMADeTX5H65cNaZgLCH4AVdJvq7cT7-ULy7bbZ7zf0WEN-LPoGoN5rYT2TCoHRiK4x7Q5SXiBneTXMlwdMyBPnLv6N8kiIkhuYlWKOKBtFqWxrDmh4tyWj4Zpo6JPYTBOMSVBjOfmvIkgW4945ZCQMkAzxvs2nLZ7mtiJ4cIELBH0MmwJqG0gnB-yBK3Lipdut75QQkJdtReii9h4bHh0guKm2b9A0QjeqPiU3f3fdUI8mtQC97jgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02508.gif" /></a></div></div><div class="label">2</div></div> where Σ<sub><em>k</em></sub> is a <em>m</em> × <em>n</em> diagonal matrix with <em>k</em> largest elements only, and <em>A</em><sub><em>k</em></sub> is the reconstruction. The rank of <em>A</em><sub><em>k</em></sub> is exactly <em>k</em>, i.e. the original dimension <em>n</em> of <em>A</em> is reduced to <em>k</em>.</div>                            <h3 scrollto-destination=7964739 id="7964739" class="section-title" data-legacy-id=s2c>2.3. Rank calculation for pathway genes and its evaluation</h3>
<p>Pearson’s correlation coefficient (<em>r</em>-value) and its significance (<em>P</em>-value) are used to measure the gene co-expression. A list of 1638 probe sets related to 219 pathways was first obtained from AraCyc dump file (ftp://ftp.arabidopsis.org/home/tair/Pathways/aracyc_dump_20070703), to form the <em>m</em> × <em>n</em> matrix <em>A</em>, where <em>m</em> is the number of AraCyc genes (<em>m</em> = 1638), and <em>n</em> the number of arrays (<em>n</em> = 2364), respectively. The computed SVs of the matrix were sorted and the largest <em>k</em> SVs were used to reconstruct the approximated matrix <em>A</em><sub><em>k</em></sub> as in Equation (<a href="#DSN025M2" class="sectionLink xref-sec">2</a>). Using approximated matrices, correlation coefficients between all AraCyc genes were calculated. Co-expressions that did not satisfy each threshold (<em>r</em> &gt; 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8 and 0.9, respectively) were discarded. The cutoff threshold was introduced to better separate inter- and intra-pathway correlations by removing majority of insignificant (low) correlations. For the remaining gene co-expressions, the average rank of intra-pathway co-expressions was calculated on 78 pathways that were associated with ≥10 metabolic genes in the database (see also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                            <h2 scrollto-destination=7964747 id="7964747" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7964752 id="7964752" class="section-title" data-legacy-id=s3a>3.1. Distribution of microarray experiments in public databases</h3>
<p>According to tissue types and experimental conditions, the 2364 array data were manually classified into 133 experimental series, whose complete listing is available as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. TAIR contains 49 experimental series (e.g. development, biotic- or abiotic-treatments, and hormone treatment), NASC provides 55 series (e.g. lignification, plant defense responses, and carbohydrate metabolism through the diurnal cycle and others), and GEO enlists 29 series (e.g. phenotypic diversity, altered environmental plasticity, stamen development and diurnal cycle effect in leaves).</p><p>There are notable differences among the three repositories. First is the tissue distribution in each repository as in Fig. <a reveal-id="DSN025F1" data-open="DSN025F1" class="link link-reveal link-table xref-fig">1</a>. Data from shoot and cell suspension occupy &gt;15% only in TAIR, and data from stamen exist only in GEO. Tissue distribution is almost balanced in TAIR, but significantly biased in NASC and GEO. Another difference is the number of GeneChip data. From this, we can at least conclude that data from all three repositories are necessary to accurately observe gene expressions in different tissue types. In the following study, we merged three data sets into a single collection without duplication.</p>                        <a id="7964763" scrollto-destination="7964763"></a>
<div id="DSN025F1" data-id="DSN025F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02501.jpeg?Expires=2147483647&Signature=nSaikJamh4lvZKnPPvoCnipG9tZkqbmT~2LGHQQFlUMcqMRwx1d1quIPLVQGG3OTV2vdOqXE~oYT4PI0K2nTLGTUdIuF1m9aMNv23qlVsF9CKWDAdYzWBKUoylLzLcCALpAKzfnuvuiryESag4aB5ECkL-z3lfDYYnfc0g4h817kT0mozfEaPb4ICvjQqE5bOMa~pk9NS~U9UL7hKKWN8MU4XYJjw9qiLK8ZZMUH2w4QYCpXlIh0Vfv-W8P2kdWAFHkz8yP8zVgg6fRf6i9GCOVqSnn8GsOW4QElZL03f-u31LoOSJrS-SOh79D8O3w3mjuX4SKRarWjvwBxnspzOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pie chart of the biomaterials of array data in each data repository." path-from-xml="dsn02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964763/dsn02501.jpeg" path-from-xml="dsn02501.jpeg" target="_blank">View large</a><a section="7964763" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02501.jpeg?Expires=2147483647&Signature=eGr8c3LqlDzwXGzKzRelglxExtChUlFi3BFOmoFNj6EIg3hfzdQ7JJv9Az5bO3Vt2u-U9PDNuRdwtYZwjTod3ruKLoa3HJtTiim~QtHjt48I8ZbCUBwVzVBKQGPfswxpv-FdcMrBuQNdbnSBX09Gr1NZsWhwwpiV9RRvAUM5aX9I-Tf6-ncf6y~nBaWx51DY~M-Lk5teJ4r7Fck-f0eWfIkA9DwjrnjV5sZF6hGL~zlRARWCH6sScsbscjLhdBZD-eh5x6SJLmEpgM8Ph7CA35o9UoxEXGb7KK8RMNmTVUIwae2ovN4cuS7RFGIRi-OaSxFrS88tNlQ36qSoaah0Cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964763&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pie chart of the biomaterials of array data in each data repository.</p></div></div><div content-id="DSN025F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02501.jpeg?Expires=2147483647&Signature=nSaikJamh4lvZKnPPvoCnipG9tZkqbmT~2LGHQQFlUMcqMRwx1d1quIPLVQGG3OTV2vdOqXE~oYT4PI0K2nTLGTUdIuF1m9aMNv23qlVsF9CKWDAdYzWBKUoylLzLcCALpAKzfnuvuiryESag4aB5ECkL-z3lfDYYnfc0g4h817kT0mozfEaPb4ICvjQqE5bOMa~pk9NS~U9UL7hKKWN8MU4XYJjw9qiLK8ZZMUH2w4QYCpXlIh0Vfv-W8P2kdWAFHkz8yP8zVgg6fRf6i9GCOVqSnn8GsOW4QElZL03f-u31LoOSJrS-SOh79D8O3w3mjuX4SKRarWjvwBxnspzOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pie chart of the biomaterials of array data in each data repository." path-from-xml="dsn02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964763/dsn02501.jpeg" path-from-xml="dsn02501.jpeg" target="_blank">View large</a><a section="7964763" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02501.jpeg?Expires=2147483647&Signature=eGr8c3LqlDzwXGzKzRelglxExtChUlFi3BFOmoFNj6EIg3hfzdQ7JJv9Az5bO3Vt2u-U9PDNuRdwtYZwjTod3ruKLoa3HJtTiim~QtHjt48I8ZbCUBwVzVBKQGPfswxpv-FdcMrBuQNdbnSBX09Gr1NZsWhwwpiV9RRvAUM5aX9I-Tf6-ncf6y~nBaWx51DY~M-Lk5teJ4r7Fck-f0eWfIkA9DwjrnjV5sZF6hGL~zlRARWCH6sScsbscjLhdBZD-eh5x6SJLmEpgM8Ph7CA35o9UoxEXGb7KK8RMNmTVUIwae2ovN4cuS7RFGIRi-OaSxFrS88tNlQ36qSoaah0Cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964763&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pie chart of the biomaterials of array data in each data repository.</p></div></div>                            <h3 scrollto-destination=7964769 id="7964769" class="section-title" data-legacy-id=s3b>3.2. Dimensional compression by SVD</h3>
<p>We saw that the tissue distribution of microarray data is biased. Another source of bias is hundreds of ‘reference’ (or wild-type) data in the repositories. Even if data look biased, i.e. multiple microarrays seem to show highly similar expression patterns, it is not easy to tell whether they are indeed redundant. The SVD algorithm was employed to check this redundancy (See Materials and methods). Fig. <a reveal-id="DSN025F2" data-open="DSN025F2" class="link link-reveal link-table xref-fig">2</a> shows the distributions of correlation coefficient for all gene pairs calculated by matrix approximation reconstructed using largest 20, 40, 300, 700 SVs and without SVD. The distribution of correlations fitted well with the Gaussian distribution for all reconstructions, and the standard deviations (SD) were 0.34, 0.31, 0.27, 0.26, and 0.26, respectively. The top 20 or 40 SVs could already reproduce the original distribution, implying that we may disregard smaller SVs as noise. The number 20 (or 40) is not an optimal value, but serves as a rough estimate. The reason for choosing these values will be explained later.</p>                        <a id="7964778" scrollto-destination="7964778"></a>
<div id="DSN025F2" data-id="DSN025F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02502.jpeg?Expires=2147483647&Signature=AOOn42y02fjqOlHxuzO8GvPuTgYdpNK3qLtlD0~xj6Zx9sw~7NiQ3JcZVOrNilT1sZULDO5bwalDzWilhr4AtbAQnbglz-TMSrXVDM~b-jxwfsEgVUNUgPenaXZ6ipZNMt69t5s6zbSrQlZEL12iPWhbj9oZFgVFSZAIchPC3ZhSB0UrXcdNNRxOeJ6tWbgJDjlf6G7d7JdEsnKx98N0NMSqZGuDqpkH2E78Y~emWT5OvYMC5QR-BY7gesZ0~TkUztGupGTrc3aPp7uB4v7hMuehUuxGYGR6k6W2SjeP6ExmQK2-fuj90ZwdS42daRvnqU-YVBBo3CFwJmXEBG6yGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of correlation coefficient from five types of data matrices (with- and without-SVD compression) normalized by RMA. Data matrices were reconstructed by largest 20 SVs (solid line), 40 SVs (lower dotted line), 300 and 700 SVs (upper dotted lines), and without-SVD (outermost dotted line). The SD of each distribution are 0.34, 0.31, 0.27, 0.26 and 0.26, respectively." path-from-xml="dsn02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964778/dsn02502.jpeg" path-from-xml="dsn02502.jpeg" target="_blank">View large</a><a section="7964778" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02502.jpeg?Expires=2147483647&Signature=zlfJn9rh5Q1q8u0yqEXUhHrpx2-U-7BynTXd-t67ZkdpFR27ptrl96dFIBliigeGcc7GxR-mvqXVq8N5tbmfJIrfdn-cnyrdcz3lBFGC3GXRlHmM7rEA99k3Kmu4FCs4Asn3TVJB-V7mEvPBkag09MZVZfcyha35m0BOfFNUxmDFbLnrQV4DHgt7c2hVr99yiMbvkH4SZehcLl-Z06z0f6x9~Nh7xGKwHUE4msDdIgRVu43lhD4OwhJ36QFvgsl6YyeX6g~ov3mIy-Wsa1BnwekPy1vYYbJi7suf5SNNY-aRCqGOVYqPQIkWQshFcm5ZZLwrxPRtPrrmCud9~Srvsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964778&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of correlation coefficient from five types of data matrices (with- and without-SVD compression) normalized by RMA. Data matrices were reconstructed by largest 20 SVs (solid line), 40 SVs (lower dotted line), 300 and 700 SVs (upper dotted lines), and without-SVD (outermost dotted line). The SD of each distribution are 0.34, 0.31, 0.27, 0.26 and 0.26, respectively.</p></div></div><div content-id="DSN025F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02502.jpeg?Expires=2147483647&Signature=AOOn42y02fjqOlHxuzO8GvPuTgYdpNK3qLtlD0~xj6Zx9sw~7NiQ3JcZVOrNilT1sZULDO5bwalDzWilhr4AtbAQnbglz-TMSrXVDM~b-jxwfsEgVUNUgPenaXZ6ipZNMt69t5s6zbSrQlZEL12iPWhbj9oZFgVFSZAIchPC3ZhSB0UrXcdNNRxOeJ6tWbgJDjlf6G7d7JdEsnKx98N0NMSqZGuDqpkH2E78Y~emWT5OvYMC5QR-BY7gesZ0~TkUztGupGTrc3aPp7uB4v7hMuehUuxGYGR6k6W2SjeP6ExmQK2-fuj90ZwdS42daRvnqU-YVBBo3CFwJmXEBG6yGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of correlation coefficient from five types of data matrices (with- and without-SVD compression) normalized by RMA. Data matrices were reconstructed by largest 20 SVs (solid line), 40 SVs (lower dotted line), 300 and 700 SVs (upper dotted lines), and without-SVD (outermost dotted line). The SD of each distribution are 0.34, 0.31, 0.27, 0.26 and 0.26, respectively." path-from-xml="dsn02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964778/dsn02502.jpeg" path-from-xml="dsn02502.jpeg" target="_blank">View large</a><a section="7964778" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02502.jpeg?Expires=2147483647&Signature=zlfJn9rh5Q1q8u0yqEXUhHrpx2-U-7BynTXd-t67ZkdpFR27ptrl96dFIBliigeGcc7GxR-mvqXVq8N5tbmfJIrfdn-cnyrdcz3lBFGC3GXRlHmM7rEA99k3Kmu4FCs4Asn3TVJB-V7mEvPBkag09MZVZfcyha35m0BOfFNUxmDFbLnrQV4DHgt7c2hVr99yiMbvkH4SZehcLl-Z06z0f6x9~Nh7xGKwHUE4msDdIgRVu43lhD4OwhJ36QFvgsl6YyeX6g~ov3mIy-Wsa1BnwekPy1vYYbJi7suf5SNNY-aRCqGOVYqPQIkWQshFcm5ZZLwrxPRtPrrmCud9~Srvsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964778&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of correlation coefficient from five types of data matrices (with- and without-SVD compression) normalized by RMA. Data matrices were reconstructed by largest 20 SVs (solid line), 40 SVs (lower dotted line), 300 and 700 SVs (upper dotted lines), and without-SVD (outermost dotted line). The SD of each distribution are 0.34, 0.31, 0.27, 0.26 and 0.26, respectively.</p></div></div><p>To check the effect of dimensional reduction in detail, we first verified Pearson’s correlation coefficient (<em>r</em>), its rank and <em>P</em>-value (<em>P</em>) for two well-known gene regulatory mechanisms: negative feedback loop and transcription factor.</p>                            <h4 scrollto-destination=7964789 id="7964789" class="section-title" data-legacy-id=s3b1>3.2.1. Feedback loop: the central circadian clock</h4>
<p>The central circadian clock (Fig. <a reveal-id="DSN025F3" data-open="DSN025F3" class="link link-reveal link-table xref-fig">3</a>) is a typical non-metabolic regulatory mechanism. When we used all 2364 arrays, strong positive correlation between two Myb-like transcription factor genes, <em>Circadian Clock Associated 1</em> (<em>CCA1</em>) and <em>Late Elongated Hypocotyl</em> (<em>LHY</em>) was observed, as well as weak negative correlation between <em>Timing Of Cab expression 1</em> (<em>TOC1</em>) and <em>LHY</em>, and between <em>TOC1</em> and <em>CCA1</em> (Fig. <a reveal-id="DSN025F3" data-open="DSN025F3" class="link link-reveal link-table xref-fig">3</a>A–C and Table <a reveal-id="DSN025TB1" data-open="DSN025TB1" class="link link-reveal link-table xref-fig">1</a>). These values agreed well with known facts that <em>TOC1</em> is a positive regulator of <em>CCA1</em> and <em>LHY</em>, and that the two clock-associated genes form a negative–positive transcriptional feedback loop.<sup><a reveal-id="DSN025C15" data-open="DSN025C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Table <a reveal-id="DSN025TB1" data-open="DSN025TB1" class="link link-reveal link-table xref-fig">1</a> shows the trend of their correlations and ranks. The approximation kept the rank of interaction even for a small number of SVs such as 20.</p>                        <a id="7964797" scrollto-destination="7964797"></a>
<div id="DSN025F3" data-id="DSN025F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02503.jpeg?Expires=2147483647&Signature=dtJesIBBX~WF0YSRwQ2Pj2G04aJHN59BRtiz4R7zVshPkmDFVrEi4HIgZidsjsz7tHoClZVTchIiWJBwi4r3m-oTa9QMaGmln2nwFI6fq~1mG8MTqBS1cJDsMF3DZGUlS3WlA32r1rdzmmsMUKudy1YujQoTmFaxH2LHUuWh80QrePBvUDNCcEBtqAyKxtBbIkqDM-cjsUet4QPuP8IKlCOYZ6zrAErvIRFJ2OrGJHYtI7~dbpx2N78E58cjEWosBPoKbLp-6uGbWJKo94DSC2QLkgAJBmdm8B7ln~lSWJNdMF9Zm1P4sXHcb93IjliSY2mj-FwCNsHEwJnX9ZvodA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots (with white circles) among three major central oscillator-related genes in Arabidopsis: (A) CCA1 versus LHY, (B) LHY versus TOC1 and (C) CCA1 versus TOC1. Highly overlapped parts look black. (D) The simplest model of the central mechanism of circadian oscillator. Co-expressions were calculated by Pearson’s correlation. See main texts for abbreviations." path-from-xml="dsn02503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964797/dsn02503.jpeg" path-from-xml="dsn02503.jpeg" target="_blank">View large</a><a section="7964797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02503.jpeg?Expires=2147483647&Signature=YZRFipXqu6qlgHQCGF6s04PEbGP3Yp6MsBuYbOwsozN8qfSkX7XAobmEr0ylKxLV3SsflxlV8IRJKLY1C2HrxOtm0-3eIiUyAzBaxKj40NvdcGgHMK6txiNFF1a-gfAiT64MhQvUIe4TFb3i2euUVPUBHXrwPO8MXLnA51bBEZbHzGM5Pa9M0WEtz4uERZZxKFvbposVktzjNlX5kMW2fPA50YFR8DZYHqw0liALaUFr3tF5WpCfFnp3mpomFx7fZpkVFjc6TsibgrhfMkyKq5aw~z6EXDXpiROTTLASuXRDC4twbzz9vvpNSd5G3qZdfryu2O2Fyp3x5uzgeefqLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964797&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots (with white circles) among three major central oscillator-related genes in Arabidopsis: (<strong>A</strong>) <em>CCA1</em> versus <em>LHY</em>, (<strong>B</strong>) <em>LHY</em> versus <em>TOC1</em> and (<strong>C</strong>) <em>CCA1</em> versus <em>TOC1</em>. Highly overlapped parts look black. (<strong>D</strong>) The simplest model of the central mechanism of circadian oscillator. Co-expressions were calculated by Pearson’s correlation. See main texts for abbreviations.</p></div></div><div content-id="DSN025F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02503.jpeg?Expires=2147483647&Signature=dtJesIBBX~WF0YSRwQ2Pj2G04aJHN59BRtiz4R7zVshPkmDFVrEi4HIgZidsjsz7tHoClZVTchIiWJBwi4r3m-oTa9QMaGmln2nwFI6fq~1mG8MTqBS1cJDsMF3DZGUlS3WlA32r1rdzmmsMUKudy1YujQoTmFaxH2LHUuWh80QrePBvUDNCcEBtqAyKxtBbIkqDM-cjsUet4QPuP8IKlCOYZ6zrAErvIRFJ2OrGJHYtI7~dbpx2N78E58cjEWosBPoKbLp-6uGbWJKo94DSC2QLkgAJBmdm8B7ln~lSWJNdMF9Zm1P4sXHcb93IjliSY2mj-FwCNsHEwJnX9ZvodA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plots (with white circles) among three major central oscillator-related genes in Arabidopsis: (A) CCA1 versus LHY, (B) LHY versus TOC1 and (C) CCA1 versus TOC1. Highly overlapped parts look black. (D) The simplest model of the central mechanism of circadian oscillator. Co-expressions were calculated by Pearson’s correlation. See main texts for abbreviations." path-from-xml="dsn02503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964797/dsn02503.jpeg" path-from-xml="dsn02503.jpeg" target="_blank">View large</a><a section="7964797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02503.jpeg?Expires=2147483647&Signature=YZRFipXqu6qlgHQCGF6s04PEbGP3Yp6MsBuYbOwsozN8qfSkX7XAobmEr0ylKxLV3SsflxlV8IRJKLY1C2HrxOtm0-3eIiUyAzBaxKj40NvdcGgHMK6txiNFF1a-gfAiT64MhQvUIe4TFb3i2euUVPUBHXrwPO8MXLnA51bBEZbHzGM5Pa9M0WEtz4uERZZxKFvbposVktzjNlX5kMW2fPA50YFR8DZYHqw0liALaUFr3tF5WpCfFnp3mpomFx7fZpkVFjc6TsibgrhfMkyKq5aw~z6EXDXpiROTTLASuXRDC4twbzz9vvpNSd5G3qZdfryu2O2Fyp3x5uzgeefqLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964797&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plots (with white circles) among three major central oscillator-related genes in Arabidopsis: (<strong>A</strong>) <em>CCA1</em> versus <em>LHY</em>, (<strong>B</strong>) <em>LHY</em> versus <em>TOC1</em> and (<strong>C</strong>) <em>CCA1</em> versus <em>TOC1</em>. Highly overlapped parts look black. (<strong>D</strong>) The simplest model of the central mechanism of circadian oscillator. Co-expressions were calculated by Pearson’s correlation. See main texts for abbreviations.</p></div></div>                        <a id="7964805" scrollto-destination="7964805"></a>
<div content-id="DSN025TB1" class="table-modal"><div class="table-wrap"><div id="DSN025TB1" class="table-wrap-title" data-id="DSN025TB1"><div class="label -label">Table 1</div><div class="caption"><p>Rank of correlations (in parentheses) between three basal genes (<em>CCA1</em>, <em>LHY</em> and <em>TOC1</em>) in the central circadian clock</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SVs used </th><th><em>CCA1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>CCA1</em> </th></tr></thead><tbody><tr><td>20 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.63 (14) </td><td><em>r</em> = −0.70 (4) </td></tr><tr><td>40 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.56 (15) </td><td><em>r</em> = −0.63 (6) </td></tr><tr><td>300 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.49 (15) </td><td><em>r</em> = −0.57 (3) </td></tr><tr><td>700 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.48 (11) </td><td><em>r</em> = −0.56 (4) </td></tr><tr><td>2364 </td><td><em>r</em> = 0.86 (1) </td><td><em>r</em> = −0.48 (12) </td><td><em>r</em> = −0.55 (6) </td></tr></tbody></table></div><div content-id="DSN025TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SVs used </th><th><em>CCA1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>CCA1</em> </th></tr></thead><tbody><tr><td>20 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.63 (14) </td><td><em>r</em> = −0.70 (4) </td></tr><tr><td>40 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.56 (15) </td><td><em>r</em> = −0.63 (6) </td></tr><tr><td>300 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.49 (15) </td><td><em>r</em> = −0.57 (3) </td></tr><tr><td>700 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.48 (11) </td><td><em>r</em> = −0.56 (4) </td></tr><tr><td>2364 </td><td><em>r</em> = 0.86 (1) </td><td><em>r</em> = −0.48 (12) </td><td><em>r</em> = −0.55 (6) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7964805" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN025TB1" class="table-wrap-title" data-id="DSN025TB1"><div class="label -label">Table 1</div><div class="caption"><p>Rank of correlations (in parentheses) between three basal genes (<em>CCA1</em>, <em>LHY</em> and <em>TOC1</em>) in the central circadian clock</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SVs used </th><th><em>CCA1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>CCA1</em> </th></tr></thead><tbody><tr><td>20 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.63 (14) </td><td><em>r</em> = −0.70 (4) </td></tr><tr><td>40 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.56 (15) </td><td><em>r</em> = −0.63 (6) </td></tr><tr><td>300 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.49 (15) </td><td><em>r</em> = −0.57 (3) </td></tr><tr><td>700 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.48 (11) </td><td><em>r</em> = −0.56 (4) </td></tr><tr><td>2364 </td><td><em>r</em> = 0.86 (1) </td><td><em>r</em> = −0.48 (12) </td><td><em>r</em> = −0.55 (6) </td></tr></tbody></table></div><div content-id="DSN025TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SVs used </th><th><em>CCA1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>LHY</em> </th><th><em>TOC1</em>–<em>CCA1</em> </th></tr></thead><tbody><tr><td>20 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.63 (14) </td><td><em>r</em> = −0.70 (4) </td></tr><tr><td>40 </td><td><em>r</em> = 0.90 (1) </td><td><em>r</em> = −0.56 (15) </td><td><em>r</em> = −0.63 (6) </td></tr><tr><td>300 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.49 (15) </td><td><em>r</em> = −0.57 (3) </td></tr><tr><td>700 </td><td><em>r</em> = 0.87 (1) </td><td><em>r</em> = −0.48 (11) </td><td><em>r</em> = −0.56 (4) </td></tr><tr><td>2364 </td><td><em>r</em> = 0.86 (1) </td><td><em>r</em> = −0.48 (12) </td><td><em>r</em> = −0.55 (6) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7964805" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=7964808 id="7964808" class="section-title" data-legacy-id=s3b2>3.2.2 Transcription factor Myb28</h4>
<p>To reconfirm the usefulness of the compressed data using small number of SVs, we checked the correlation values between a well-characterized transcription factor and its downstream genes using different numbers of SVs. Myb28 or R2R3-MYB transcription factor, is a positive regulator of aliphatic methionine-derived glucosinolates (GSL) investigated in the authors’ institution,<sup><a reveal-id="DSN025C8" data-open="DSN025C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSN025C16" data-open="DSN025C16" class="link link-ref link-reveal xref-bibr">16</a></sup> offering a typical example of metabolic regulation by a non-metabolic gene. As in the clock case, the approximation kept the rank of interaction even for 20 SVs (Table <a reveal-id="DSN025TB2" data-open="DSN025TB2" class="link link-reveal link-table xref-fig">2</a>). We also compared the correlation values with that of ATTED-II version 3 (1388 GeneChips from TAIR).<sup><a reveal-id="DSN025C6" data-open="DSN025C6" class="link link-ref link-reveal xref-bibr">6</a></sup> ATTED-II is a widely known and regularly updated correlation analysis software tool for Arabidopsis. Table <a reveal-id="DSN025TB2" data-open="DSN025TB2" class="link link-reveal link-table xref-fig">2</a> demonstrates that correlation values obtained by using largest 20 SVs are comparable with those by ATTED-II.</p>                        <a id="7964816" scrollto-destination="7964816"></a>
<div content-id="DSN025TB2" class="table-modal"><div class="table-wrap"><div id="DSN025TB2" class="table-wrap-title" data-id="DSN025TB2"><div class="label -label">Table 2</div><div class="caption"><p>Correlation coefficients and their ranks (in parentheses) among Myb28-regulated GSL biosynthetic genes [NS, not significant (<em>P</em> ≥ 1E−300)]</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Probe name </th><th>AGI code </th><th>Description </th><th colspan="5">SVs used<hr /> </th><th>ATTED-II </th></tr><tr><th> </th><th> </th><th> </th><th>20 </th><th>40 </th><th>300 </th><th>700 </th><th>All </th><th> </th></tr></thead><tbody><tr><td>247549_at </td><td>At5g61420 </td><td>Myb family transcription factor (Myb28) </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>266395_at </td><td>At2g43100 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD1) </td><td>0.89 (7) </td><td>0.85 (7) </td><td>0.80 (8) </td><td>0.79 (8) </td><td>0.79 (8) </td><td>0.74 </td></tr><tr><td>251524_at </td><td>At3g58990 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD2) </td><td>0.89 (6) </td><td>0.86 (6) </td><td>0.83 (5) </td><td>0.82 (5) </td><td>0.82 (4) </td><td>0.78 </td></tr><tr><td>254687_at </td><td>At4g13770 </td><td>Cytochrome P450 family protein (CYP83A1) </td><td>0.95 (1) </td><td>0.93 (1) </td><td>0.90 (1) </td><td>0.89 (1) </td><td>0.89 (1) </td><td>0.80 </td></tr><tr><td>249866_at </td><td>At5g23010 </td><td>2-Isopropylmalate synthase 3 (IMS3) (MAM-1) </td><td>0.88 (8) </td><td>0.86 (5) </td><td>0.82 (6) </td><td>0.81 (6) </td><td>0.81 (6) </td><td>0.70 </td></tr><tr><td>257021_at </td><td>At3g19710 </td><td>Branched-chain amino acid transaminase, putative (AtBCAT-4) (MAAT) </td><td>0.86 (9) </td><td>0.84 (8) </td><td>0.8 (7) </td><td>0.79 (7) </td><td>0.79 (7) </td><td>0.68 </td></tr><tr><td rowspan="2">262717_s_at </td><td>At1g16410 </td><td>Cytochrome P450, putative (CYP79F1) </td><td>0.85 (12) </td><td>0.82 (11) </td><td>0.76 (12) </td><td>0.74 (12) </td><td>0.74 (12) </td><td>0.67 </td></tr><tr><td>At1g16400 </td><td>No entry (CYP79F2) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>260745_at </td><td>At1g78370 </td><td>glutathione S-transferase, putative (ATGSTU20) </td><td>0.77 (29) </td><td>0.75 (18) </td><td>0.72 (16) </td><td>0.71 (16) </td><td>0.71 (15) </td><td>0.52 </td></tr><tr><td>263477_at </td><td>At2g31790 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74C1) </td><td>0.92 (3) </td><td>0.89 (3) </td><td>0.86 (2) </td><td>0.85 (2) </td><td>0.84 (2) </td><td>0.72 </td></tr><tr><td>255437_at </td><td>At4g03060 </td><td>2-Oxoglutarate-dependent dioxygenase, putative (AOP2) </td><td>0.61 (274) </td><td>0.6 (156) </td><td>0.52 (303) </td><td>0.51 (332) </td><td>0.5 (328) </td><td>0.43 </td></tr><tr><td>255773_at </td><td>At1g18590 </td><td>Sulfotransferase family protein (AtSOT17) </td><td>0.8 (19) </td><td>0.77 (16) </td><td>0.73 (15) </td><td>0.72 (15) </td><td>0.71 (16) </td><td>0.61 </td></tr><tr><td>264873_at </td><td>At1g24100 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74B1) </td><td>0.61 (307) </td><td>0.58 (249) </td><td>0.53 (223) </td><td>0.52 (257) </td><td>0.52 (257) </td><td>0.43 </td></tr><tr><td>260385_at </td><td>At1g74090 </td><td>Sulfotransferase family protein (AtSOT18) </td><td>0.90 (5) </td><td>0.87 (4) </td><td>0.84 (4) </td><td>0.83 (4) </td><td>0.82 (5) </td><td>0.76 </td></tr><tr><td>263706_s_at </td><td>At5g14200 </td><td>AtIMD1 </td><td>0.77 (30) </td><td>0.74 (23) </td><td>0.70 (18) </td><td>0.70 (18) </td><td>0.69 (18) </td><td>NS </td></tr><tr><td>249867_at </td><td>At5g23020 </td><td>2-Isopropylmalate synthase 2 (IMS2) (MAM3) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.41 </td></tr><tr><td>263714_at </td><td>At2g20610 </td><td>Aminotransferase, putative (SUR1) </td><td>0.73 (43) </td><td>0.71 (33) </td><td>0.67 (24) </td><td>0.66 (25) </td><td>0.66 (25) </td><td>0.54 </td></tr><tr><td>250633_at </td><td>At5g07460 </td><td>Peptide methionine sulfoxide reductase, putative (PMSR2) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.44 </td></tr><tr><td>258851_at </td><td>At3g03190 </td><td>Glutathione S-transferase, putative (ATGSTF11) </td><td>0.8 (16) </td><td>0.78 (13) </td><td>0.73 (14) </td><td>0.72 (14) </td><td>0.72 (14) </td><td>0.71 </td></tr><tr><td>254742_at </td><td>At4g13430 </td><td>Aconitate hydratase family protein (AtLeuC1) </td><td>0.68 (97) </td><td>0.65 (64) </td><td>0.60 (53) </td><td>0.6 (55) </td><td>0.59 (59) </td><td>0.62 </td></tr><tr><td>259343_s_at </td><td>At3g03780 </td><td>Cobalamin-independent methionine synthase, putative (AtMS2) </td><td>0.54 (813) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td></tr><tr><td>252274_at </td><td>At3g49680 </td><td>Branched-chain amino acid transaminase 3 (AtBCAT-3) </td><td>0.67 (106) </td><td>0.65 (68) </td><td>0.62 (48) </td><td>0.62 (41) </td><td>0.61 (44) </td><td>0.53 </td></tr></tbody></table></div><div content-id="DSN025TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Probe name </th><th>AGI code </th><th>Description </th><th colspan="5">SVs used<hr /> </th><th>ATTED-II </th></tr><tr><th> </th><th> </th><th> </th><th>20 </th><th>40 </th><th>300 </th><th>700 </th><th>All </th><th> </th></tr></thead><tbody><tr><td>247549_at </td><td>At5g61420 </td><td>Myb family transcription factor (Myb28) </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>266395_at </td><td>At2g43100 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD1) </td><td>0.89 (7) </td><td>0.85 (7) </td><td>0.80 (8) </td><td>0.79 (8) </td><td>0.79 (8) </td><td>0.74 </td></tr><tr><td>251524_at </td><td>At3g58990 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD2) </td><td>0.89 (6) </td><td>0.86 (6) </td><td>0.83 (5) </td><td>0.82 (5) </td><td>0.82 (4) </td><td>0.78 </td></tr><tr><td>254687_at </td><td>At4g13770 </td><td>Cytochrome P450 family protein (CYP83A1) </td><td>0.95 (1) </td><td>0.93 (1) </td><td>0.90 (1) </td><td>0.89 (1) </td><td>0.89 (1) </td><td>0.80 </td></tr><tr><td>249866_at </td><td>At5g23010 </td><td>2-Isopropylmalate synthase 3 (IMS3) (MAM-1) </td><td>0.88 (8) </td><td>0.86 (5) </td><td>0.82 (6) </td><td>0.81 (6) </td><td>0.81 (6) </td><td>0.70 </td></tr><tr><td>257021_at </td><td>At3g19710 </td><td>Branched-chain amino acid transaminase, putative (AtBCAT-4) (MAAT) </td><td>0.86 (9) </td><td>0.84 (8) </td><td>0.8 (7) </td><td>0.79 (7) </td><td>0.79 (7) </td><td>0.68 </td></tr><tr><td rowspan="2">262717_s_at </td><td>At1g16410 </td><td>Cytochrome P450, putative (CYP79F1) </td><td>0.85 (12) </td><td>0.82 (11) </td><td>0.76 (12) </td><td>0.74 (12) </td><td>0.74 (12) </td><td>0.67 </td></tr><tr><td>At1g16400 </td><td>No entry (CYP79F2) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>260745_at </td><td>At1g78370 </td><td>glutathione S-transferase, putative (ATGSTU20) </td><td>0.77 (29) </td><td>0.75 (18) </td><td>0.72 (16) </td><td>0.71 (16) </td><td>0.71 (15) </td><td>0.52 </td></tr><tr><td>263477_at </td><td>At2g31790 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74C1) </td><td>0.92 (3) </td><td>0.89 (3) </td><td>0.86 (2) </td><td>0.85 (2) </td><td>0.84 (2) </td><td>0.72 </td></tr><tr><td>255437_at </td><td>At4g03060 </td><td>2-Oxoglutarate-dependent dioxygenase, putative (AOP2) </td><td>0.61 (274) </td><td>0.6 (156) </td><td>0.52 (303) </td><td>0.51 (332) </td><td>0.5 (328) </td><td>0.43 </td></tr><tr><td>255773_at </td><td>At1g18590 </td><td>Sulfotransferase family protein (AtSOT17) </td><td>0.8 (19) </td><td>0.77 (16) </td><td>0.73 (15) </td><td>0.72 (15) </td><td>0.71 (16) </td><td>0.61 </td></tr><tr><td>264873_at </td><td>At1g24100 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74B1) </td><td>0.61 (307) </td><td>0.58 (249) </td><td>0.53 (223) </td><td>0.52 (257) </td><td>0.52 (257) </td><td>0.43 </td></tr><tr><td>260385_at </td><td>At1g74090 </td><td>Sulfotransferase family protein (AtSOT18) </td><td>0.90 (5) </td><td>0.87 (4) </td><td>0.84 (4) </td><td>0.83 (4) </td><td>0.82 (5) </td><td>0.76 </td></tr><tr><td>263706_s_at </td><td>At5g14200 </td><td>AtIMD1 </td><td>0.77 (30) </td><td>0.74 (23) </td><td>0.70 (18) </td><td>0.70 (18) </td><td>0.69 (18) </td><td>NS </td></tr><tr><td>249867_at </td><td>At5g23020 </td><td>2-Isopropylmalate synthase 2 (IMS2) (MAM3) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.41 </td></tr><tr><td>263714_at </td><td>At2g20610 </td><td>Aminotransferase, putative (SUR1) </td><td>0.73 (43) </td><td>0.71 (33) </td><td>0.67 (24) </td><td>0.66 (25) </td><td>0.66 (25) </td><td>0.54 </td></tr><tr><td>250633_at </td><td>At5g07460 </td><td>Peptide methionine sulfoxide reductase, putative (PMSR2) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.44 </td></tr><tr><td>258851_at </td><td>At3g03190 </td><td>Glutathione S-transferase, putative (ATGSTF11) </td><td>0.8 (16) </td><td>0.78 (13) </td><td>0.73 (14) </td><td>0.72 (14) </td><td>0.72 (14) </td><td>0.71 </td></tr><tr><td>254742_at </td><td>At4g13430 </td><td>Aconitate hydratase family protein (AtLeuC1) </td><td>0.68 (97) </td><td>0.65 (64) </td><td>0.60 (53) </td><td>0.6 (55) </td><td>0.59 (59) </td><td>0.62 </td></tr><tr><td>259343_s_at </td><td>At3g03780 </td><td>Cobalamin-independent methionine synthase, putative (AtMS2) </td><td>0.54 (813) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td></tr><tr><td>252274_at </td><td>At3g49680 </td><td>Branched-chain amino acid transaminase 3 (AtBCAT-3) </td><td>0.67 (106) </td><td>0.65 (68) </td><td>0.62 (48) </td><td>0.62 (41) </td><td>0.61 (44) </td><td>0.53 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7964816" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN025TB2" class="table-wrap-title" data-id="DSN025TB2"><div class="label -label">Table 2</div><div class="caption"><p>Correlation coefficients and their ranks (in parentheses) among Myb28-regulated GSL biosynthetic genes [NS, not significant (<em>P</em> ≥ 1E−300)]</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Probe name </th><th>AGI code </th><th>Description </th><th colspan="5">SVs used<hr /> </th><th>ATTED-II </th></tr><tr><th> </th><th> </th><th> </th><th>20 </th><th>40 </th><th>300 </th><th>700 </th><th>All </th><th> </th></tr></thead><tbody><tr><td>247549_at </td><td>At5g61420 </td><td>Myb family transcription factor (Myb28) </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>266395_at </td><td>At2g43100 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD1) </td><td>0.89 (7) </td><td>0.85 (7) </td><td>0.80 (8) </td><td>0.79 (8) </td><td>0.79 (8) </td><td>0.74 </td></tr><tr><td>251524_at </td><td>At3g58990 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD2) </td><td>0.89 (6) </td><td>0.86 (6) </td><td>0.83 (5) </td><td>0.82 (5) </td><td>0.82 (4) </td><td>0.78 </td></tr><tr><td>254687_at </td><td>At4g13770 </td><td>Cytochrome P450 family protein (CYP83A1) </td><td>0.95 (1) </td><td>0.93 (1) </td><td>0.90 (1) </td><td>0.89 (1) </td><td>0.89 (1) </td><td>0.80 </td></tr><tr><td>249866_at </td><td>At5g23010 </td><td>2-Isopropylmalate synthase 3 (IMS3) (MAM-1) </td><td>0.88 (8) </td><td>0.86 (5) </td><td>0.82 (6) </td><td>0.81 (6) </td><td>0.81 (6) </td><td>0.70 </td></tr><tr><td>257021_at </td><td>At3g19710 </td><td>Branched-chain amino acid transaminase, putative (AtBCAT-4) (MAAT) </td><td>0.86 (9) </td><td>0.84 (8) </td><td>0.8 (7) </td><td>0.79 (7) </td><td>0.79 (7) </td><td>0.68 </td></tr><tr><td rowspan="2">262717_s_at </td><td>At1g16410 </td><td>Cytochrome P450, putative (CYP79F1) </td><td>0.85 (12) </td><td>0.82 (11) </td><td>0.76 (12) </td><td>0.74 (12) </td><td>0.74 (12) </td><td>0.67 </td></tr><tr><td>At1g16400 </td><td>No entry (CYP79F2) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>260745_at </td><td>At1g78370 </td><td>glutathione S-transferase, putative (ATGSTU20) </td><td>0.77 (29) </td><td>0.75 (18) </td><td>0.72 (16) </td><td>0.71 (16) </td><td>0.71 (15) </td><td>0.52 </td></tr><tr><td>263477_at </td><td>At2g31790 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74C1) </td><td>0.92 (3) </td><td>0.89 (3) </td><td>0.86 (2) </td><td>0.85 (2) </td><td>0.84 (2) </td><td>0.72 </td></tr><tr><td>255437_at </td><td>At4g03060 </td><td>2-Oxoglutarate-dependent dioxygenase, putative (AOP2) </td><td>0.61 (274) </td><td>0.6 (156) </td><td>0.52 (303) </td><td>0.51 (332) </td><td>0.5 (328) </td><td>0.43 </td></tr><tr><td>255773_at </td><td>At1g18590 </td><td>Sulfotransferase family protein (AtSOT17) </td><td>0.8 (19) </td><td>0.77 (16) </td><td>0.73 (15) </td><td>0.72 (15) </td><td>0.71 (16) </td><td>0.61 </td></tr><tr><td>264873_at </td><td>At1g24100 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74B1) </td><td>0.61 (307) </td><td>0.58 (249) </td><td>0.53 (223) </td><td>0.52 (257) </td><td>0.52 (257) </td><td>0.43 </td></tr><tr><td>260385_at </td><td>At1g74090 </td><td>Sulfotransferase family protein (AtSOT18) </td><td>0.90 (5) </td><td>0.87 (4) </td><td>0.84 (4) </td><td>0.83 (4) </td><td>0.82 (5) </td><td>0.76 </td></tr><tr><td>263706_s_at </td><td>At5g14200 </td><td>AtIMD1 </td><td>0.77 (30) </td><td>0.74 (23) </td><td>0.70 (18) </td><td>0.70 (18) </td><td>0.69 (18) </td><td>NS </td></tr><tr><td>249867_at </td><td>At5g23020 </td><td>2-Isopropylmalate synthase 2 (IMS2) (MAM3) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.41 </td></tr><tr><td>263714_at </td><td>At2g20610 </td><td>Aminotransferase, putative (SUR1) </td><td>0.73 (43) </td><td>0.71 (33) </td><td>0.67 (24) </td><td>0.66 (25) </td><td>0.66 (25) </td><td>0.54 </td></tr><tr><td>250633_at </td><td>At5g07460 </td><td>Peptide methionine sulfoxide reductase, putative (PMSR2) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.44 </td></tr><tr><td>258851_at </td><td>At3g03190 </td><td>Glutathione S-transferase, putative (ATGSTF11) </td><td>0.8 (16) </td><td>0.78 (13) </td><td>0.73 (14) </td><td>0.72 (14) </td><td>0.72 (14) </td><td>0.71 </td></tr><tr><td>254742_at </td><td>At4g13430 </td><td>Aconitate hydratase family protein (AtLeuC1) </td><td>0.68 (97) </td><td>0.65 (64) </td><td>0.60 (53) </td><td>0.6 (55) </td><td>0.59 (59) </td><td>0.62 </td></tr><tr><td>259343_s_at </td><td>At3g03780 </td><td>Cobalamin-independent methionine synthase, putative (AtMS2) </td><td>0.54 (813) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td></tr><tr><td>252274_at </td><td>At3g49680 </td><td>Branched-chain amino acid transaminase 3 (AtBCAT-3) </td><td>0.67 (106) </td><td>0.65 (68) </td><td>0.62 (48) </td><td>0.62 (41) </td><td>0.61 (44) </td><td>0.53 </td></tr></tbody></table></div><div content-id="DSN025TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Probe name </th><th>AGI code </th><th>Description </th><th colspan="5">SVs used<hr /> </th><th>ATTED-II </th></tr><tr><th> </th><th> </th><th> </th><th>20 </th><th>40 </th><th>300 </th><th>700 </th><th>All </th><th> </th></tr></thead><tbody><tr><td>247549_at </td><td>At5g61420 </td><td>Myb family transcription factor (Myb28) </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>266395_at </td><td>At2g43100 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD1) </td><td>0.89 (7) </td><td>0.85 (7) </td><td>0.80 (8) </td><td>0.79 (8) </td><td>0.79 (8) </td><td>0.74 </td></tr><tr><td>251524_at </td><td>At3g58990 </td><td>Aconitase C-terminal domain-containing protein (AtLeuD2) </td><td>0.89 (6) </td><td>0.86 (6) </td><td>0.83 (5) </td><td>0.82 (5) </td><td>0.82 (4) </td><td>0.78 </td></tr><tr><td>254687_at </td><td>At4g13770 </td><td>Cytochrome P450 family protein (CYP83A1) </td><td>0.95 (1) </td><td>0.93 (1) </td><td>0.90 (1) </td><td>0.89 (1) </td><td>0.89 (1) </td><td>0.80 </td></tr><tr><td>249866_at </td><td>At5g23010 </td><td>2-Isopropylmalate synthase 3 (IMS3) (MAM-1) </td><td>0.88 (8) </td><td>0.86 (5) </td><td>0.82 (6) </td><td>0.81 (6) </td><td>0.81 (6) </td><td>0.70 </td></tr><tr><td>257021_at </td><td>At3g19710 </td><td>Branched-chain amino acid transaminase, putative (AtBCAT-4) (MAAT) </td><td>0.86 (9) </td><td>0.84 (8) </td><td>0.8 (7) </td><td>0.79 (7) </td><td>0.79 (7) </td><td>0.68 </td></tr><tr><td rowspan="2">262717_s_at </td><td>At1g16410 </td><td>Cytochrome P450, putative (CYP79F1) </td><td>0.85 (12) </td><td>0.82 (11) </td><td>0.76 (12) </td><td>0.74 (12) </td><td>0.74 (12) </td><td>0.67 </td></tr><tr><td>At1g16400 </td><td>No entry (CYP79F2) </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>260745_at </td><td>At1g78370 </td><td>glutathione S-transferase, putative (ATGSTU20) </td><td>0.77 (29) </td><td>0.75 (18) </td><td>0.72 (16) </td><td>0.71 (16) </td><td>0.71 (15) </td><td>0.52 </td></tr><tr><td>263477_at </td><td>At2g31790 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74C1) </td><td>0.92 (3) </td><td>0.89 (3) </td><td>0.86 (2) </td><td>0.85 (2) </td><td>0.84 (2) </td><td>0.72 </td></tr><tr><td>255437_at </td><td>At4g03060 </td><td>2-Oxoglutarate-dependent dioxygenase, putative (AOP2) </td><td>0.61 (274) </td><td>0.6 (156) </td><td>0.52 (303) </td><td>0.51 (332) </td><td>0.5 (328) </td><td>0.43 </td></tr><tr><td>255773_at </td><td>At1g18590 </td><td>Sulfotransferase family protein (AtSOT17) </td><td>0.8 (19) </td><td>0.77 (16) </td><td>0.73 (15) </td><td>0.72 (15) </td><td>0.71 (16) </td><td>0.61 </td></tr><tr><td>264873_at </td><td>At1g24100 </td><td>UDP-glucoronosyl/UDP-glucosyl transferase family protein (UGT74B1) </td><td>0.61 (307) </td><td>0.58 (249) </td><td>0.53 (223) </td><td>0.52 (257) </td><td>0.52 (257) </td><td>0.43 </td></tr><tr><td>260385_at </td><td>At1g74090 </td><td>Sulfotransferase family protein (AtSOT18) </td><td>0.90 (5) </td><td>0.87 (4) </td><td>0.84 (4) </td><td>0.83 (4) </td><td>0.82 (5) </td><td>0.76 </td></tr><tr><td>263706_s_at </td><td>At5g14200 </td><td>AtIMD1 </td><td>0.77 (30) </td><td>0.74 (23) </td><td>0.70 (18) </td><td>0.70 (18) </td><td>0.69 (18) </td><td>NS </td></tr><tr><td>249867_at </td><td>At5g23020 </td><td>2-Isopropylmalate synthase 2 (IMS2) (MAM3) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.41 </td></tr><tr><td>263714_at </td><td>At2g20610 </td><td>Aminotransferase, putative (SUR1) </td><td>0.73 (43) </td><td>0.71 (33) </td><td>0.67 (24) </td><td>0.66 (25) </td><td>0.66 (25) </td><td>0.54 </td></tr><tr><td>250633_at </td><td>At5g07460 </td><td>Peptide methionine sulfoxide reductase, putative (PMSR2) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>0.44 </td></tr><tr><td>258851_at </td><td>At3g03190 </td><td>Glutathione S-transferase, putative (ATGSTF11) </td><td>0.8 (16) </td><td>0.78 (13) </td><td>0.73 (14) </td><td>0.72 (14) </td><td>0.72 (14) </td><td>0.71 </td></tr><tr><td>254742_at </td><td>At4g13430 </td><td>Aconitate hydratase family protein (AtLeuC1) </td><td>0.68 (97) </td><td>0.65 (64) </td><td>0.60 (53) </td><td>0.6 (55) </td><td>0.59 (59) </td><td>0.62 </td></tr><tr><td>259343_s_at </td><td>At3g03780 </td><td>Cobalamin-independent methionine synthase, putative (AtMS2) </td><td>0.54 (813) </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td><td>NS </td></tr><tr><td>252274_at </td><td>At3g49680 </td><td>Branched-chain amino acid transaminase 3 (AtBCAT-3) </td><td>0.67 (106) </td><td>0.65 (68) </td><td>0.62 (48) </td><td>0.62 (41) </td><td>0.61 (44) </td><td>0.53 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7964816" target="_blank">View Large</a></div></div><p>The two regulatory examples suggest that blindly increasing the number of GeneChip data does not automatically lead to increased accuracy. By carefully choosing a smaller set of expression data, accurate functional prediction comparable with a state-of-the-art software tool becomes feasible.</p>                            <h3 scrollto-destination=7964825 id="7964825" class="section-title" data-legacy-id=s3c>3.3. Using AraCyc metabolic pathways to evaluate gene co-expressions</h3>
<p>Next, we investigated the correlations among metabolic pathway genes. It is impossible to rigorously assess the effect of dimensional compression due to the absence of a set of ‘true’ gene–gene association inside metabolic pathways. As an alternative, we utilize a credible observation that, on an average, genes associated with the same metabolic pathway are highly co-expressed than genes from different pathways.<sup><a reveal-id="DSN025C10" data-open="DSN025C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSN025C17" data-open="DSN025C17" class="link link-ref link-reveal xref-bibr">17</a></sup> For assessment, we first selected 78 pathways which were associated with ≥10 metabolic genes in the AraCyc database (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p><p>These pathways contained 1638 genes in total. We computed the co-expressions between all pairs of genes and obtained the average rank of intra-pathway co-expressions as in Wei et al.<sup><a reveal-id="DSN025C10" data-open="DSN025C10" class="link link-ref link-reveal xref-bibr">10</a></sup> According to the pathway hypothesis, intra-pathway correlations are ranked lower (i.e. highly correlated) than inter-pathway correlations. Fig. <a reveal-id="DSN025F4" data-open="DSN025F4" class="link link-reveal link-table xref-fig">4</a> shows the trend of the average rank of intra-pathway correlations using reconstructed matrices of the SV index <em>k</em> for different threshold <em>r</em> (see Materials and methods). In the figure, the lowest average rank was achieved ∼20 SVs for most threshold values. In other words, 20 SVs are enough to separate intra-pathway co-expressions, and the set of arrays corresponding to these SVs is considered most informative among 2364 experiments. When <em>r</em> = 0.5, the lowest average rank runs between 15 and 35 and slightly jumps up at ∼40. This effect seems to be an artifact specific to the threshold 0.5 for unknown reason. Also, average ranks for different <em>r</em> look stabilized around <em>k</em> = 20. From these observations, we set the (roughly) minimum number of SVs as 20 (and 40) in our analysis.</p>                        <a id="7964840" scrollto-destination="7964840"></a>
<div id="DSN025F4" data-id="DSN025F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02504.jpeg?Expires=2147483647&Signature=Y1VNvNcDC50oYK22GtULlMPvEYvaBK0B39YulMs9qnQ8ZeKnh3XuqWd2IZr2Q5KNnvY1yjNUhbE37dti7ocQ-F05yqCx-ts85MwVaK20tfkUiAuWeiWp9cQpJFfos33tVQOR2Of7ONnH9qVf2Thjcf8XlHKGHRrSO4vMnbNGFAq0Ny7KCC4bexM-1q9roagOQPZl3zj0w~yTrXBKcb6PnP-J2wOGtprl4IbyF-klnfwawMV88x78-ErCxKlo22IuhSJRA5G092~1HlkoYKhyTeBqtG3MRSLy24uf7UtgTCNKyu77jPtMVsHbWdM~ngBQHcJ6P7~NSd9VXsdzwH9ywA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of AraCyc genes in co-expression rankings against various thresholds (r = 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8 and 0.9). Average ranks of intra-pathway correlations using reconstructed matrices were calculated across the 78 AraCyc pathways that contain ≥10 genes in ATH1 GeneChip." path-from-xml="dsn02504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964840/dsn02504.jpeg" path-from-xml="dsn02504.jpeg" target="_blank">View large</a><a section="7964840" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02504.jpeg?Expires=2147483647&Signature=vCx4xXYbW1oxdoCmL7EXt8sedv59ZEnbpgb2f361W8zGGMnH0-sCiAwSm3q0IgFQYSQle3FDAgzPfO8x-t0~5HECalyMDM-EnFlZwC3obxtWjhw0D087BKrvKgLzZ7DNYXYk5S-Uw6R9-KzTtYwSfiZNXP4y8sz9kl9h8Y7Pw5x4ZYT85LHy0mUeR5GaM2MGCMtcjWGiiaimKjO797GjX~m4fo5IGVsZPScbddCrNgOELN6y7L0WTMzs0NNJBioHWv6Cill1H3Zki1nGq6uWUqJTUNrg1WHOZV~JPDUbyLwrfyUUhVo7om1mitxK7CiiTLB72m95rcesb84SBrQGdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964840&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of AraCyc genes in co-expression rankings against various thresholds (<em>r</em> = 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8 and 0.9). Average ranks of intra-pathway correlations using reconstructed matrices were calculated across the 78 AraCyc pathways that contain ≥10 genes in ATH1 GeneChip.</p></div></div><div content-id="DSN025F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02504.jpeg?Expires=2147483647&Signature=Y1VNvNcDC50oYK22GtULlMPvEYvaBK0B39YulMs9qnQ8ZeKnh3XuqWd2IZr2Q5KNnvY1yjNUhbE37dti7ocQ-F05yqCx-ts85MwVaK20tfkUiAuWeiWp9cQpJFfos33tVQOR2Of7ONnH9qVf2Thjcf8XlHKGHRrSO4vMnbNGFAq0Ny7KCC4bexM-1q9roagOQPZl3zj0w~yTrXBKcb6PnP-J2wOGtprl4IbyF-klnfwawMV88x78-ErCxKlo22IuhSJRA5G092~1HlkoYKhyTeBqtG3MRSLy24uf7UtgTCNKyu77jPtMVsHbWdM~ngBQHcJ6P7~NSd9VXsdzwH9ywA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of AraCyc genes in co-expression rankings against various thresholds (r = 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8 and 0.9). Average ranks of intra-pathway correlations using reconstructed matrices were calculated across the 78 AraCyc pathways that contain ≥10 genes in ATH1 GeneChip." path-from-xml="dsn02504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964840/dsn02504.jpeg" path-from-xml="dsn02504.jpeg" target="_blank">View large</a><a section="7964840" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02504.jpeg?Expires=2147483647&Signature=vCx4xXYbW1oxdoCmL7EXt8sedv59ZEnbpgb2f361W8zGGMnH0-sCiAwSm3q0IgFQYSQle3FDAgzPfO8x-t0~5HECalyMDM-EnFlZwC3obxtWjhw0D087BKrvKgLzZ7DNYXYk5S-Uw6R9-KzTtYwSfiZNXP4y8sz9kl9h8Y7Pw5x4ZYT85LHy0mUeR5GaM2MGCMtcjWGiiaimKjO797GjX~m4fo5IGVsZPScbddCrNgOELN6y7L0WTMzs0NNJBioHWv6Cill1H3Zki1nGq6uWUqJTUNrg1WHOZV~JPDUbyLwrfyUUhVo7om1mitxK7CiiTLB72m95rcesb84SBrQGdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964840&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evaluation of AraCyc genes in co-expression rankings against various thresholds (<em>r</em> = 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8 and 0.9). Average ranks of intra-pathway correlations using reconstructed matrices were calculated across the 78 AraCyc pathways that contain ≥10 genes in ATH1 GeneChip.</p></div></div>                            <h3 scrollto-destination=7964843 id="7964843" class="section-title" data-legacy-id=s3d>3.4. Estimation of the number of informative arrays</h3>
<p>Having confirmed the effectiveness of reconstruction from a small number of SVs, we estimated the informative set of arrays, i.e. array information that are most amplified by the decomposition by regarding the SVs as the amplification factor of orthonormal basis vectors representing array experiments. The matrix <em>A</em><sub><em>k</em></sub> in Equation (<a href="#DSN025M2" class="sectionLink xref-sec">2</a>) was approximated by zeroing elements less than a threshold <em>λ</em> (let <em>B</em><sub><em>k</em></sub> = [<em>A</em><sub><em>k</em></sub>]<sub>&gt;λ</sub> be this matrix), and the dimension of <em>B</em><sub><em>k</em></sub><sup>T</sup> corresponds to the number of significant arrays contributing to the <em>k</em> SVs in <em>A</em><sub><em>k</em></sub>. When the dimension was plotted against the increasing value of <em>λ</em> for different SVs, it rapidly decreased as the <em>λ</em> increased but the dimension was almost consistent for SVs ranging between 10 and 50 (Fig. <a reveal-id="DSN025F5" data-open="DSN025F5" class="link link-reveal link-table xref-fig">5</a>). The result partially supported the dominance of large SVs as in Section 3.2, but we could not determine an appropriate <em>λ</em> to determine the size of informative arrays.</p>                        <a id="7964848" scrollto-destination="7964848"></a>
<div id="DSN025F5" data-id="DSN025F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02505.jpeg?Expires=2147483647&Signature=Al7kFETf8Wf5QsL8CKLDAGl1shUung2W2i-fIYrmo2xknnRn12-oX3HpQrp2-3Am2Ee87otPXydaihWIOMHvM8~W7cda8jpbXNuZMKTvrShthwqvAIS9ipkmy57088eAWcpBmycUd5kKsu15nENlUV~j3N89qa0Vrdl0jLDJJ~xGyzrKrCA9BE-Pn4m9OxzIdoypSIyTLsMhj3oUdS2CoEHOEHr6dh-ZKKIqFVNUh0XdT3FreKE7hBWjqZZzcihZt7kXP-pS9Z~YTPOup3pAevTbSMEYR8H1ndW0K8hy7tNIZllqH5dSEsXTQhK8U08O8r1PJjgilPdtcJAgeokJHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The plot of the number of arrays (y-axis) against λ (x-axis from 1 to 10) for different SVs. Each bar corresponds to 10, 20, 30, 40 and 50 SVs from left to right. The number of significant columns rapidly decreases as the λ increases, and contributing arrays are independent of the number of SVs." path-from-xml="dsn02505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964848/dsn02505.jpeg" path-from-xml="dsn02505.jpeg" target="_blank">View large</a><a section="7964848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02505.jpeg?Expires=2147483647&Signature=cfHsdq8pI0j8hQz-bzRpHTSUs91de0jBW1Rwo963FLXc4vIq5VPa3JB14RSI2T4LYoGRp6neQrbmYQuKpcu2HD~F-ljJt06j6CIAhG4UxuI2Hg9-cDISDZGanHjs7fjyfimB-2WEQX~VlVn9TFJpMGSisS~v6qX2wRfWSWzWIaUjNyWPM9gTagdRwzC3WAyxt5CC-a12O6AUFc4ysj3dCAkEyN3Rqr9JWhJguxgAg5TiprsN~xOxGwwSdWoYnitSF2CahVrDO1j-7Xy3UP5xXaeH~rxgjQ22hLsvx63SZvU5czCq2XDWO0lpLojiz0-APcx08dyo7V-J8J8qLVdwQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964848&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The plot of the number of arrays (<em>y</em>-axis) against <em>λ</em> (<em>x</em>-axis from 1 to 10) for different SVs. Each bar corresponds to 10, 20, 30, 40 and 50 SVs from left to right. The number of significant columns rapidly decreases as the <em>λ</em> increases, and contributing arrays are independent of the number of SVs.</p></div></div><div content-id="DSN025F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02505.jpeg?Expires=2147483647&Signature=Al7kFETf8Wf5QsL8CKLDAGl1shUung2W2i-fIYrmo2xknnRn12-oX3HpQrp2-3Am2Ee87otPXydaihWIOMHvM8~W7cda8jpbXNuZMKTvrShthwqvAIS9ipkmy57088eAWcpBmycUd5kKsu15nENlUV~j3N89qa0Vrdl0jLDJJ~xGyzrKrCA9BE-Pn4m9OxzIdoypSIyTLsMhj3oUdS2CoEHOEHr6dh-ZKKIqFVNUh0XdT3FreKE7hBWjqZZzcihZt7kXP-pS9Z~YTPOup3pAevTbSMEYR8H1ndW0K8hy7tNIZllqH5dSEsXTQhK8U08O8r1PJjgilPdtcJAgeokJHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The plot of the number of arrays (y-axis) against λ (x-axis from 1 to 10) for different SVs. Each bar corresponds to 10, 20, 30, 40 and 50 SVs from left to right. The number of significant columns rapidly decreases as the λ increases, and contributing arrays are independent of the number of SVs." path-from-xml="dsn02505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964848/dsn02505.jpeg" path-from-xml="dsn02505.jpeg" target="_blank">View large</a><a section="7964848" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02505.jpeg?Expires=2147483647&Signature=cfHsdq8pI0j8hQz-bzRpHTSUs91de0jBW1Rwo963FLXc4vIq5VPa3JB14RSI2T4LYoGRp6neQrbmYQuKpcu2HD~F-ljJt06j6CIAhG4UxuI2Hg9-cDISDZGanHjs7fjyfimB-2WEQX~VlVn9TFJpMGSisS~v6qX2wRfWSWzWIaUjNyWPM9gTagdRwzC3WAyxt5CC-a12O6AUFc4ysj3dCAkEyN3Rqr9JWhJguxgAg5TiprsN~xOxGwwSdWoYnitSF2CahVrDO1j-7Xy3UP5xXaeH~rxgjQ22hLsvx63SZvU5czCq2XDWO0lpLojiz0-APcx08dyo7V-J8J8qLVdwQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964848&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The plot of the number of arrays (<em>y</em>-axis) against <em>λ</em> (<em>x</em>-axis from 1 to 10) for different SVs. Each bar corresponds to 10, 20, 30, 40 and 50 SVs from left to right. The number of significant columns rapidly decreases as the <em>λ</em> increases, and contributing arrays are independent of the number of SVs.</p></div></div><p>Most amplified array sets were the stamen development (GSE4733) and the Type III effectors on plant defense response (NASCarrays-59). Other significant arrays included profiles of early germinating seeds (ME00332), the response to bacterial-(LPS, HrpZ, Flg22) and oomycete-(NPP1) derived elicitors (ME00319), oxidative stress (GSE7211) and alternative oxidases (GSE4113 and GSE2406). These results indicated the importance of use of different tissue types in gene correlation analysis.</p>                            <h3 scrollto-destination=7964855 id="7964855" class="section-title" data-legacy-id=s3e>3.5. Correspondence between each SV and genes or experimental conditions</h3>
<p>To evaluate the correspondence between a specific SV (δ) and genes or arrays, δ-dependent reconstructed expression data matrices with the gene sets of AraCyc were examined. The matrices were reconstructed according to the scheme in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. Briefly, we first performed SVD analysis on the data matrix and the resulting diagonal matrix <em>Σ</em> was transformed into <em>δ</em>-only <em>Σ′</em>. The diagonal elements of matrix <em>Σ′</em> are zero values, except for the <em>δ</em> under focus. Using this <em>Σ′</em>, <em>δ</em>-reconstructed expression data matrix was obtained. To see which experimental conditions and genes most contributed to <em>δ</em> (Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>), a hierarchical clustering approach was performed using the data matrix. Let us explain five largest SVs by denoting the <em>i</em>th largest SV as <em>δ</em><sub><em>i</em></sub>. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>, we provide breakdown charts of GO categories for each gene cluster corresponding to these SVs.</p>                        <a id="7964862" scrollto-destination="7964862"></a>
<div id="DSN025F6" data-id="DSN025F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02506.jpeg?Expires=2147483647&Signature=E0BrEDmWHD25hL3RQKtfLan9gwsB6UAfltVe~HCpz~7Ov6AYiniHBNPZZ45g8Kbj0LeSuOuZMgyDjoZcmHKaH6iXK9b~dXWnOuTfTsbm7AjeZ0XCO54aNFG~MHfKrFfpdKyINkdh9CnxLqfXSUsFf9E0cIPfFpulxIE9QN7CaDAGe2dbTiVHMGzm1s44p1JzQOfAD0oeuMOtti6KBUZmt9w2atGcdYE1-NKbCB6worWCBp8AgoEa~IYRngmeYmWd1ZsvoXE153chunfO~P6UYU-I34P-xx~H1ATqLsxVrNApG2CQkGFfSbPBidPOzfA1SVij4aOiG-2bcb4fYKO5ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering of the reconstructed data matrices using only one SV δ. (A–E) Show the matrix reconstructed by the largest SV δ1 to fifth largest value δ5. Columns are experimental series and rows are genes; both of which are hierarchically clustered in each figure. Magenta denotes the positive value of the reconstructed matrix Bk and the cyan the negative value." path-from-xml="dsn02506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964862/dsn02506.jpeg" path-from-xml="dsn02506.jpeg" target="_blank">View large</a><a section="7964862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02506.jpeg?Expires=2147483647&Signature=iyEyV~aCOtPlGdV158gBib7sJDLmLxIuhE4xqIiBK1bETdm9rAwrI0wq9uEihCx8tT5JQLvPiYrPtpaozbC4uHRTXE94BAkgaT9emThM--DxsK37xCRm-VhUIslSMmypZtGUWyBCigDCCKlGa8vO-j7Y-mMOKGWg5GXOM3hNxzW8so5vjZDgR2pzUb44pxE51itmAYaY4rjg7ssRtQ-m0odhNpngBQ7r-nXNfLt3-YX2-ZAXNBOEfCymEH7zHacT16020DndILdKFiyuI508jAyw96D9SRZ9HD3xmdR6qNDHeyj69TCyRwb7kpwzaCryZsGcQDJeeubQVqtxbI98aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964862&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering of the reconstructed data matrices using only one SV <em>δ</em>. (<strong>A</strong>–<strong>E</strong>) Show the matrix reconstructed by the largest SV <em>δ</em><sub><em>1</em></sub> to fifth largest value <em>δ</em><sub><em>5</em></sub>. Columns are experimental series and rows are genes; both of which are hierarchically clustered in each figure. Magenta denotes the positive value of the reconstructed matrix <em>B</em><sub><em>k</em></sub> and the cyan the negative value.</p></div></div><div content-id="DSN025F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/m_dsn02506.jpeg?Expires=2147483647&Signature=E0BrEDmWHD25hL3RQKtfLan9gwsB6UAfltVe~HCpz~7Ov6AYiniHBNPZZ45g8Kbj0LeSuOuZMgyDjoZcmHKaH6iXK9b~dXWnOuTfTsbm7AjeZ0XCO54aNFG~MHfKrFfpdKyINkdh9CnxLqfXSUsFf9E0cIPfFpulxIE9QN7CaDAGe2dbTiVHMGzm1s44p1JzQOfAD0oeuMOtti6KBUZmt9w2atGcdYE1-NKbCB6worWCBp8AgoEa~IYRngmeYmWd1ZsvoXE153chunfO~P6UYU-I34P-xx~H1ATqLsxVrNApG2CQkGFfSbPBidPOzfA1SVij4aOiG-2bcb4fYKO5ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering of the reconstructed data matrices using only one SV δ. (A–E) Show the matrix reconstructed by the largest SV δ1 to fifth largest value δ5. Columns are experimental series and rows are genes; both of which are hierarchically clustered in each figure. Magenta denotes the positive value of the reconstructed matrix Bk and the cyan the negative value." path-from-xml="dsn02506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7964862/dsn02506.jpeg" path-from-xml="dsn02506.jpeg" target="_blank">View large</a><a section="7964862" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn02506.jpeg?Expires=2147483647&Signature=iyEyV~aCOtPlGdV158gBib7sJDLmLxIuhE4xqIiBK1bETdm9rAwrI0wq9uEihCx8tT5JQLvPiYrPtpaozbC4uHRTXE94BAkgaT9emThM--DxsK37xCRm-VhUIslSMmypZtGUWyBCigDCCKlGa8vO-j7Y-mMOKGWg5GXOM3hNxzW8so5vjZDgR2pzUb44pxE51itmAYaY4rjg7ssRtQ-m0odhNpngBQ7r-nXNfLt3-YX2-ZAXNBOEfCymEH7zHacT16020DndILdKFiyuI508jAyw96D9SRZ9HD3xmdR6qNDHeyj69TCyRwb7kpwzaCryZsGcQDJeeubQVqtxbI98aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7964862&ar=512658&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering of the reconstructed data matrices using only one SV <em>δ</em>. (<strong>A</strong>–<strong>E</strong>) Show the matrix reconstructed by the largest SV <em>δ</em><sub><em>1</em></sub> to fifth largest value <em>δ</em><sub><em>5</em></sub>. Columns are experimental series and rows are genes; both of which are hierarchically clustered in each figure. Magenta denotes the positive value of the reconstructed matrix <em>B</em><sub><em>k</em></sub> and the cyan the negative value.</p></div></div><p>The contribution of <em>δ</em><sub><em>1</em></sub> was not limited to any experimental condition or arrays but was related to specific gene clusters. Two clusters of highly positive values were formed (Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data 1</a> displays the full image of the hierarchical clusters of arrays marked in Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>. The upper cluster in Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>A (Group g1 of <em>δ1</em> in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) contained genes associated with aerobic respiration pathway, carbonate dehydratase (in nitrogen metabolism) and photosynthesis. The middle cluster (Group g2) included genes related to glycolysis, aerobic respiration, glutamate metabolism and TCA cycle. The lower cluster (Group g3) included genes for (deoxy) ribose phosphate degradation, steroid biosynthesis, and diterpenoid biosynthesis (gibberellin inactivation). Therefore <em>δ</em><sub><em>1</em></sub> largely corresponded to a variety of major metabolic pathways in primary metabolism irrespective of experiments.</p><p>On the other hand, values from <em>δ</em><sub><em>2</em></sub> to <em>δ</em><sub><em>5</em></sub> were associated with specific experimental conditions. The <em>δ</em><sub><em>2</em></sub> was linked with two large experimental clusters shown in Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>B. The magenta region in the left-hand side corresponded to the shoot data of stress series (heat, UV-B, salt, wound, cold, oxidative and drought; Group atr2 of <em>δ</em><sub><em>2</em></sub> in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>) whereas the right-hand region contained the root data of the same experimental series (Group atr1 of <em>δ</em><sub><em>2</em></sub> in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. See also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data1</a>). Relevant genes were associated with photosynthesis and glycolysis/gluconeogenesis, but many genes show medium or low correlations. Notable observation was therefore the marked contrast between root and shoot irrespective of experimental series.</p><p>Likewise, <em>δ</em><sub><em>3</em></sub> corresponded to two biotic treatment conditions: response to virulent (accession, ME00331) and response to bacterial-(LPS, HrpZ, Flg22) and oomycete-NPP1 (accession, ME00332). The <em>δ</em><sub><em>3</em></sub> still depends on experimental series (vertical direction in Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>), but high correlation in certain group of genes is also observed (horizontal direction in Fig. <a reveal-id="DSN025F6" data-open="DSN025F6" class="link link-reveal link-table xref-fig">6</a>). The correspondences for <em>δ</em><sub><em>4</em></sub> and <em>δ</em><sub><em>5</em></sub> were obscurer, but as their commonly highlighted experimental conditions we could recognize stamen development data set (accession, GSE4733) with gene sets for cytokinins 9-<em>N</em>-glucoside biosynthesis and cytokinins 7-<em>N</em>-glucoside biosynthesis.</p><p>In summary, we could identify biological functions related to the largest five SVs, although each SV did not precisely correspond to specific experimental conditions or genes. We could again confirm the importance of the use of different tissue types (e.g. shoot/root under stress and stamen development).</p>                            <h2 scrollto-destination=7964877 id="7964877" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7964880 id="7964880" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This research was supported by Grant-in-Aid for Scientific Research on Priority Areas ‘Systems Genomics' from MEXT and BIRD, Japan Science and Technology Agency.</p>                            <h2 scrollto-destination=7964882 id="7964882" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Drs Yuji Sawada, and Masami Yokota-Hirai at RIKEN PSC for fruitful discussions. We also thank Yukiko Nakanishi, Hiroaki Osada, Kazuhiro Suwa, and Munehide Itoyama for assistance in classifying GeneChip data, and Tsuyoshi Kato for critical reading of our manuscript.</p>                            <h2 scrollto-destination=7964884 id="7964884" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN025C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Domrachev</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lash</div> <div class="given-names">A. E.</div></div>. </div><div class="article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">207</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Expression%20Omnibus%3A%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository&amp;author=R.%20Edgar&amp;author=M.%20Domrachev&amp;author=A.%20E.%20Lash&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=207-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">P.</div></div>. </div><div class="article-title">The Arabidopsis information resource (TAIR): a model organism database providing a centralized, curated gateway to Arabidopsis biology, research materials and community</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">224</div>-<div class="lpage">228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20information%20resource%20(TAIR)%3A%20a%20model%20organism%20database%20providing%20a%20centralized%2C%20curated%20gateway%20to%20Arabidopsis%20biology%2C%20research%20materials%20and%20community&amp;author=P.%20Zhang&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=224-228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Craigon</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">James</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Okyere</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jotham</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">S.</div></div>. </div><div class="article-title">NASCArrays: a repository for microarray data generated by NASC’s transcriptomics service</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D575</div>-<div class="lpage">D577</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NASCArrays%3A%20a%20repository%20for%20microarray%20data%20generated%20by%20NASC%E2%80%99s%20transcriptomics%20service&amp;author=D.%20J.%20Craigon&amp;author=N.%20James&amp;author=J.%20Okyere&amp;author=J.%20Higgins&amp;author=J.%20Jotham&amp;author=S.%20May&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D575-D577" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steinhauser</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Luedemann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Thimm</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Kopka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">CSB.DB: a comprehensive systems-biology database</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3647</div>-<div class="lpage">3651</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CSB.DB%3A%20a%20comprehensive%20systems-biology%20database&amp;author=D.%20Steinhauser&amp;author=B.%20Usadel&amp;author=A.%20Luedemann&amp;author=O.%20Thimm&amp;author=J.%20Kopka&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3647-3651" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zimmermann</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hirsch-Hoffmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hennig</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Gruissem</div> <div class="given-names">W.</div></div>. </div><div class="article-title">GENEVESTIGATOR. Arabidopsis microarray database and analysis toolbox</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">2621</div>-<div class="lpage">2632</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GENEVESTIGATOR.%20Arabidopsis%20microarray%20database%20and%20analysis%20toolbox&amp;author=P.%20Zimmermann&amp;author=M.%20Hirsch-Hoffmann&amp;author=L.%20Hennig&amp;author=W.%20Gruissem&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=136&amp;pages=2621-2632" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.046367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.046367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.046367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15375207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shibaoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">H.</div></div>. </div><div class="article-title">ATTED II: a database of co-expressed genes and <em>cis</em> elements for identifying co-regulated gene groups in Arabidopsis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D863</div>-<div class="lpage">D869</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ATTED%20II%3A%20a%20database%20of%20co-expressed%20genes%20and%20cis%20elements%20for%20identifying%20co-regulated%20gene%20groups%20in%20Arabidopsis&amp;author=T.%20Obayashi&amp;author=K.%20Kinoshita&amp;author=K.%20Nakai&amp;author=M.%20Shibaoka&amp;author=S.%20Hayashi&amp;author=M.%20Saeki&amp;author=D.%20Shibata&amp;author=K.%20Saito&amp;author=H.%20Ohta&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D863-D869" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl783" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl783" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl783"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Approaches for extracting practical information from gene co-expression networks in plant biology</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">381</div>-<div class="lpage">390</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approaches%20for%20extracting%20practical%20information%20from%20gene%20co-expression%20networks%20in%20plant%20biology&amp;author=K.%20Aoki&amp;author=Y.%20Ogata&amp;author=D.%20Shibata&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=381-390" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17251202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirai</div> <div class="given-names">M. Y.</div></div>,  <div class="name"><div class="surname">Sugiyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tohge</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Araki</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">6478</div>-<div class="lpage">6483</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omics-based%20identification%20of%20Arabidopsis%20Myb%20transcription%20factors%20regulating%20aliphatic%20glucosinolate%20biosynthesis&amp;author=M.%20Y.%20Hirai&amp;author=K.%20Sugiyama&amp;author=Y.%20Sawada&amp;author=T.%20Tohge&amp;author=T.%20Obayashi&amp;author=A.%20Suzuki&amp;author=R.%20Araki&amp;author=N.%20Sakurai&amp;author=H.%20Suzuki&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=6478-6483" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611629104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611629104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611629104"> </span></div> </div></div></div></div></div><div content-id="DSN025C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lisso</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Steinhauser</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Altmann</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kopka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mussig</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Identification of brassinosteroid-related genes by means of transcript co-response analyses</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">2685</div>-<div class="lpage">2696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20brassinosteroid-related%20genes%20by%20means%20of%20transcript%20co-response%20analyses&amp;author=J.%20Lisso&amp;author=D.%20Steinhauser&amp;author=T.%20Altmann&amp;author=J.%20Kopka&amp;author=C.%20Mussig&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=2685-2696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki566"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15891113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Persson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mehta</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Srinivasasainagendra</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">G. P.</div></div>,  <div class="name"><div class="surname">Somerville</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Loraine</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Transcriptional coordination of the metabolic network in Arabidopsis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">762</div>-<div class="lpage">774</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20coordination%20of%20the%20metabolic%20network%20in%20Arabidopsis&amp;author=H.%20Wei&amp;author=S.%20Persson&amp;author=T.%20Mehta&amp;author=V.%20Srinivasasainagendra&amp;author=L.%20Chen&amp;author=G.%20P.%20Page&amp;author=C.%20Somerville&amp;author=A.%20Loraine&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=142&amp;pages=762-774" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.080358" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.080358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.080358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16920875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Persson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Milne</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">G. P.</div></div>,  <div class="name"><div class="surname">Somerville</div> <div class="given-names">C. R.</div></div>. </div><div class="article-title">Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">8633</div>-<div class="lpage">8638</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20required%20for%20cellulose%20synthesis%20by%20regression%20analysis%20of%20public%20microarray%20data%20sets&amp;author=S.%20Persson&amp;author=H.%20Wei&amp;author=J.%20Milne&amp;author=G.%20P.%20Page&amp;author=C.%20R.%20Somerville&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=8633-8638" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0503392102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0503392102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0503392102"> </span></div> </div></div></div></div></div><div content-id="DSN025C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hawkins</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">S. S.</div></div>. </div><div class="article-title">Robust singular value decomposition analysis of microarray data</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">13167</div>-<div class="lpage">13172</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robust%20singular%20value%20decomposition%20analysis%20of%20microarray%20data&amp;author=L.%20Liu&amp;author=D.%20M.%20Hawkins&amp;author=S.%20Ghosh&amp;author=S.%20S.%20Young&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=13167-13172" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1733249100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1733249100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1733249100"> </span></div> </div></div></div></div></div><div content-id="DSN025C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wall</div> <div class="given-names">M. E.</div></div>,  <div class="name"><div class="surname">Rechtsteiner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rocha</div> <div class="given-names">L. M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Berrar</div> <div class="given-names">D. P.</div></div>, et al. </div><div class="article-title">Singular value decomposition and principal component analysis</div>, <div class="source">A Practical Approach to Microarray Data Analysis</div> , <div class="year">2003</div><div class="publisher-loc">Norwell, MA</div><div class="publisher-name">Kluwer</div>(pg. <div class="fpage">91</div>-<div class="lpage">99</div>)</div></div></div></div><div content-id="DSN025C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolstad</div> <div class="given-names">B. M.</div></div>,  <div class="name"><div class="surname">Irizarry</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Astrand</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Speed</div> <div class="given-names">T. P.</div></div>. </div><div class="article-title">A comparison of normalization methods for high density oligonucleotide array data based on variance and bias</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">185</div>-<div class="lpage">193</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias&amp;author=B.%20M.%20Bolstad&amp;author=R.%20A.%20Irizarry&amp;author=M.%20Astrand&amp;author=T.%20P.%20Speed&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=185-193" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.2.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F19.2.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alabadi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Oyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yanovsky</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">F. G.</div></div>,  <div class="name"><div class="surname">Mas</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kay</div> <div class="given-names">S. A.</div></div>. </div><div class="article-title">Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">880</div>-<div class="lpage">883</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reciprocal%20regulation%20between%20TOC1%20and%20LHY%2FCCA1%20within%20the%20Arabidopsis%20circadian%20clock&amp;author=D.%20Alabadi&amp;author=T.%20Oyama&amp;author=M.%20J.%20Yanovsky&amp;author=F.%20G.%20Harmon&amp;author=P.%20Mas&amp;author=S.%20A.%20Kay&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=880-883" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1061320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1061320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1061320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11486091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gigolashvili</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yatusevich</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Berger</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Muller</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Flugge</div> <div class="given-names">U. I.</div></div>. </div><div class="article-title">The R2R3-MYB transcription factor HAG1/MYB28 is a regulator of methionine-derived glucosinolate biosynthesis in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">247</div>-<div class="lpage">261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20R2R3-MYB%20transcription%20factor%20HAG1%2FMYB28%20is%20a%20regulator%20of%20methionine-derived%20glucosinolate%20biosynthesis%20in%20Arabidopsis%20thaliana&amp;author=T.%20Gigolashvili&amp;author=R.%20Yatusevich&amp;author=B.%20Berger&amp;author=C.%20Muller&amp;author=U.%20I.%20Flugge&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=51&amp;pages=247-261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03133.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03133.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03133.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17521412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN025C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN025C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ihmels</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Levy</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Barkai</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Principles of transcriptional control in the metabolic network of <em>Saccharomyces cerevisiae</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">86</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principles%20of%20transcriptional%20control%20in%20the%20metabolic%20network%20of%20Saccharomyces%20cerevisiae&amp;author=J.%20Ihmels&amp;author=R.%20Levy&amp;author=N.%20Barkai&amp;publication_year=2004&amp;journal=Nat.%20Biotechnol.&amp;volume=22&amp;pages=86-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt918" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt918" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt918"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14647306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=512658&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn025-dsn025supp_Table2.xls?Expires=2147483647&amp;Signature=LQ~KWKL2gvY2d9GZ5kjeINaGCz2FcZNOLndO~7-FZLbO98VDurUW5iJY1vSfluDu-AmkizD2B8e9odbrk0ZBnIq8rXIQTTWmGXsqlJe3h4Bn6a6y3Gtl8xHZzZenIiSiGfRrKKv1KjAhe-a0iOGEBphfzYBvbVv100WGt7ZpFI7f~MgtZTbOBK9TrCrZ3EMugkpUUDUtMdKJ2R4XpLpS4quz5jR6ipwSVk6y3JZ1oQ14ha-Rku1eVR8OslZIoJ~Ko4DBN-7Zwnaeyov6iSYCsJiJcNlQjX5Xr0UcMWU3ReczFrJInm5A6cKftFXKu9-p3u27oFlcJEY-XUC~QDpB6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn025-dsn025supp_Table1.xls?Expires=2147483647&amp;Signature=wf0pAUuQhrvITRfbAGc-ldPY7QEU~Gm87w-RFHAfS-byMhALuSZoqKR3E9DdKeZD3AeM66Uk1iwkgCtJ35aZplo59x97mEV3REwibhTI-67IWy~P5zep~9Oa56Ne9nLgu2dLzsOyGGpwWAFDAMNa5rqSxv4bUKW5xTGePowpONCI9qrSbxxmtzQR9nDJeh~TtCb0SmDj4Hxx2RzH~ZDZuc0sONzyelHSPx7R1Z-ShzSmwAhj3WlOkMD1EvMn1mfPZV2hgnbCP177u8IRCGvWOwXU~H9rnY0U4CMCzu-lud-gtRHAxkTW1649hGK6Js-e5ygHsRuH8b7Rbx5~eNmJSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn025/2/dsn025-dsn025supp.pdf?Expires=2147483647&amp;Signature=KcgDrS-bomb9S7WeuFkGJS8-4Plg-sNckFcOsf42PLMLrxNTLkxmJXSSyulzw1JmHn6nLXWW7irJ1rolxlZVplfYYNe4GWa46Vcn3FmqIfJPdLLjt66ZTq-Z-NVVqd242HgUxZLHVgLnNgSSj05HHIpu6FWvatvODeQA78KsXenpWJ2xYwdjVjbWZGOW8relW~e6KFWFaSomnZdgNPCiXgya0wQWcHuamZMOnIfjDtBg3BXLbJ1l9pC7-H6qUV9mnMYg2VsbfPEL1OOuhFyWtOhvEQquUkjRqPsrmSbQLQ72TXWsk-Kmt68VSBUzz8NmeOou12xcunvNLmhDRvgDmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">73</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn025"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="512658"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f6%2f367%2f512658">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000262588400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F6%2F367%2F512658" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/512658'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="SVD-based Anatomy of Gene Expressions for Correlation Analysis in Arabidopsis thaliana" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn025");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f367%2f512658">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

