

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/325/2469971","@type":"ScholarlyArticle","name":" Whole genomic DNA sequencing and comparative genomic analysis of  Arthrospira platensis  : high genome plasticity and genetic diversity ","datePublished":"2016-06-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw023","keywords":["evolution","lateral gene transfer","comparative genomics","chromosomal rearrangement"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/325/2469971","author":[{"name":"Xu, Teng","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China","@type":"Person"},{"name":"Qin, Song","affiliation":"Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China","@type":"Person"},{"name":"Hu, Yongwu","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China    BGI-Shenzhen, Shenzhen 518083, China","@type":"Person"},{"name":"Song, Zhijian","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China","@type":"Person"},{"name":"Ying, Jianchao","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China","@type":"Person"},{"name":"Li, Peizhen","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China","@type":"Person"},{"name":"Dong, Wei","affiliation":"BGI-Shenzhen, Shenzhen 518083, China","@type":"Person"},{"name":"Zhao, Fangqing","affiliation":"Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China","@type":"Person"},{"name":"Yang, Huanming","affiliation":"BGI-Shenzhen, Shenzhen 518083, China    James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, China","@type":"Person"},{"name":"Bao, Qiyu","affiliation":"School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China    BGI-Shenzhen, Shenzhen 518083, China","@type":"Person"}],"description":"Abstract.  Arthrospira platensis is a multi-cellular and filamentous non-N2-fixing cyanobacterium that is capable of performing oxygenic photosynthesis. In this","pageStart":"325","pageEnd":"338","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Whole genomic DNA sequencing and comparative genomic analysis of  Arthrospira platensis  : high genome plasticity and genetic diversity ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Whole genomic DNA sequencing and comparative genomic analysis of  Arthrospira platensis  : high genome plasticity and genetic diversity " />
    <meta property="og:description" content="Abstract.  Arthrospira platensis is a multi-cellular and filamentous non-N2-fixing cyanobacterium that is capable of performing oxygenic photosynthesis. In this" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/325/2469971" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-06-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Xu, Teng" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author" content="Qin, Song" /><meta name="citation_author_institution" content=" Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China " /><meta name="citation_author" content="Hu, Yongwu" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author_institution" content=" BGI-Shenzhen, Shenzhen 518083, China " /><meta name="citation_author" content="Song, Zhijian" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author" content="Ying, Jianchao" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author" content="Li, Peizhen" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author" content="Dong, Wei" /><meta name="citation_author_institution" content=" BGI-Shenzhen, Shenzhen 518083, China " /><meta name="citation_author" content="Zhao, Fangqing" /><meta name="citation_author_institution" content=" Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China " /><meta name="citation_author" content="Yang, Huanming" /><meta name="citation_author_institution" content=" BGI-Shenzhen, Shenzhen 518083, China " /><meta name="citation_author_institution" content=" James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, China " /><meta name="citation_author" content="Bao, Qiyu" /><meta name="citation_author_institution" content=" School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China " /><meta name="citation_author_institution" content=" BGI-Shenzhen, Shenzhen 518083, China " /><meta name="citation_title" content=" Whole genomic DNA sequencing and comparative genomic analysis of Arthrospira platensis : high genome plasticity and genetic diversity " /><meta name="citation_firstpage" content="325" /><meta name="citation_lastpage" content="338" /><meta name="citation_doi" content="10.1093/dnares/dsw023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_author=Habib  M. A. B.; citation_author=Mashuda  P.; citation_author=Huntington  T. C.; citation_author=Hasan  M. R.; citation_publisher=Rome, Italy, Food and Agriculture Organization of The United Nations;  citation_title=A Review on Culture, Production and Use of Spirulina as Food for Humans and Feeds for Domestic Animals and Fish;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=Proteomic analysis and qRT-PCR verification of temperature response to Arthrospira (Spirulina) platensis; citation_author=Huili  W.; citation_author=Xiaokai  Z.; citation_author=Meili  L.;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e83485" /><meta name="citation_reference" content="citation_title=Identification of differentially expressed proteins of Arthrospira (Spirulina) plantensis-YZ under salt-stress conditions by proteomics and qRT-PCR analysis; citation_author=Wang  H.; citation_author=Yang  Y.; citation_author=Chen  W.;  citation_journal_title=Proteome Sci;  citation_year=2013;  citation_volume=11;  citation_pages=6" /><meta name="citation_reference" content="citation_title=Spirulina phycocyanin induces differential protein expression and apoptosis in SKOV-3 cells; citation_author=Pan  R.; citation_author=Lu  R.; citation_author=Zhang  Y.;  citation_journal_title=Int. J. Biol. Macromol;  citation_year=2015;  citation_volume=81;  citation_pages=951-9; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of restriction-modification system in unicellular and filamentous cyanobacteria; citation_author=Zhao  F.; citation_author=Zhang  X.; citation_author=Liang  C.; citation_author=Wu  J.; citation_author=Bao  Q.; citation_author=Qin  S.;  citation_journal_title=Physiol. Genomics;  citation_year=2006;  citation_volume=24;  citation_pages=181-90; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Arthrospira platensis C1 (PCC9438); citation_author=Cheevadhanarak  S.; citation_author=Paithoonrangsarid  K.; citation_author=Prommeenate  P.;  citation_journal_title=Stand. Genomic Sci;  citation_year=2012;  citation_volume=6;  citation_pages=43-53; " /><meta name="citation_reference" content="citation_title=Genome sequence of the edible cyanobacterium Arthrospira sp. PCC 8005; citation_author=Janssen  P. J.; citation_author=Morin  N.; citation_author=Mergeay  M.;  citation_journal_title=J. Bacteriol;  citation_year=2010;  citation_volume=192;  citation_pages=2465-6; " /><meta name="citation_reference" content="citation_title=Genomic structure of an economically important cyanobacterium, Arthrospira (Spirulina) platensis NIES-39; citation_author=Fujisawa  T.; citation_author=Narikawa  R.; citation_author=Okamoto  S.;  citation_journal_title=DNA Res;  citation_year=2010;  citation_volume=17;  citation_pages=85-103; " /><meta name="citation_reference" content="citation_title=A complete sequence of the T. tengcongensis genome; citation_author=Bao  Q.; citation_author=Tian  Y.; citation_author=Li  W.;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=689-700; " /><meta name="citation_reference" content="citation_title=Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus); citation_author=Dong  Y.; citation_author=Xie  M.; citation_author=Jiang  Y.;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=135-41; " /><meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing; citation_author=Gordon  D.; citation_author=Abajian  C.; citation_author=Green  P.;  citation_journal_title=Genome Res;  citation_year=1998;  citation_volume=8;  citation_pages=195-202; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=McKenna  A.; citation_author=Hanna  M.; citation_author=Banks  E.;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer; citation_author=Delcher  A. L.; citation_author=Bratke  K. A.; citation_author=Powers  E. C.; citation_author=Salzberg  S. L.;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=673-9; " /><meta name="citation_reference" content="citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes; citation_author=Lagesen  K.; citation_author=Hallin  P.; citation_author=Rodland  E. A.; citation_author=Staerfeldt  H. H.; citation_author=Rognes  T.; citation_author=Ussery  D. W.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=3100-8; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; citation_author=Lowe  T. M.; citation_author=Eddy  S. R.;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Interactive microbial genome visualization with GView; citation_author=Petkau  A.; citation_author=Stuart-Edwards  M.; citation_author=Stothard  P.; citation_author=Van Domselaar  G.;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=3125-6; " /><meta name="citation_reference" content="citation_title=REBASE–a database for DNA restriction and modification: enzymes, genes and genomes; citation_author=Roberts  R. J.; citation_author=Vincze  T.; citation_author=Posfai  J.; citation_author=Macelis  D.;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=D234-236; " /><meta name="citation_reference" content="citation_title=A simple, fast, and accurate method of phylogenomic inference; citation_author=Wu  M.; citation_author=Eisen  J. A.;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R151" /><meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0; citation_author=Guindon  S.; citation_author=Dufayard  J. F.; citation_author=Lefort  V.; citation_author=Anisimova  M.; citation_author=Hordijk  W.; citation_author=Gascuel  O.;  citation_journal_title=Syst. Biol;  citation_year=2010;  citation_volume=59;  citation_pages=307-321; " /><meta name="citation_reference" content="citation_title=ISfinder: the reference centre for bacterial insertion sequences; citation_author=Siguier  P.; citation_author=Perochon  J.; citation_author=Lestrade  L.; citation_author=Mahillon  J.; citation_author=Chandler  M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D32-6; " /><meta name="citation_reference" content="citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement; citation_author=Darling  A. E.; citation_author=Mau  B.; citation_author=Perna  N. T.;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e11147" /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences; citation_author=Li  W.; citation_author=Godzik  A.;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; citation_author=Benson  G.;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats; citation_author=Grissa  I.; citation_author=Vergnaud  G.; citation_author=Pourcel  C.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W52-7; " /><meta name="citation_reference" content="citation_title=Complete genome and gene expression analyses of Asaia bogorensis reveal unique responses to culture with mammalian cells as a potential opportunistic human pathogen; citation_author=Kawai  M.; citation_author=Higashiura  N.; citation_author=Hayasaki  K.;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=357-66; " /><meta name="citation_reference" content="citation_title=Associations between inverted repeats and the structural evolution of bacterial genomes; citation_author=Achaz  G.; citation_author=Coissac  E.; citation_author=Netter  P.; citation_author=Rocha  E. P.;  citation_journal_title=Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1279-89; " /><meta name="citation_reference" content="citation_title=Improving the coverage of the cyanobacterial phylum using diversity-driven genome sequencing; citation_author=Shih  P. M.; citation_author=Wu  D.; citation_author=Latifi  A.;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2013;  citation_volume=110;  citation_pages=1053-8; " /><meta name="citation_reference" content="citation_title=Concerted and birth-and-death evolution of multigene families; citation_author=Nei  M.; citation_author=Rooney  A. P.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2005;  citation_volume=39;  citation_pages=121-52; " /><meta name="citation_reference" content="citation_title=Whole-Genome Shotgun Sequence of Arthrospira platensis Strain Paraca, a Cultivated and Edible Cyanobacterium; citation_author=Lefort  F.; citation_author=Calmin  G.; citation_author=Crovadore  J.;  citation_journal_title=Genome Announc;  citation_year=2014;  citation_volume=2;  citation_pages=1-2; " /><meta name="citation_reference" content="citation_title=Lateral gene transfer and the nature of bacterial innovation; citation_author=Ochman  H.; citation_author=Lawrence  J. G.; citation_author=Groisman  E. A.;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=405;  citation_pages=299-304; " /><meta name="citation_reference" content="citation_title=Origin and evolution of retroelements based upon their reverse transcriptase sequences; citation_author=Xiong  Y.; citation_author=Eickbush  T. H.;  citation_journal_title=EMBO J;  citation_year=1990;  citation_volume=9;  citation_pages=3353-3362; " /><meta name="citation_reference" content="citation_title=Reverse transcriptase activity of an intron encoded polypeptide; citation_author=Fassbender  S.; citation_author=Bruhl  K. H.; citation_author=Ciriacy  M.; citation_author=Kuck  U.;  citation_journal_title=EMBO J;  citation_year=1994;  citation_volume=13;  citation_pages=2075-83; " /><meta name="citation_reference" content="citation_title=The transposon Tn9 generates a 9 bp repeated sequence during integration; citation_author=Johnsrud  L.; citation_author=Calos  M. P.; citation_author=Miller  J. H.;  citation_journal_title=Cell;  citation_year=1978;  citation_volume=15;  citation_pages=1209-19; " /><meta name="citation_reference" content="citation_title=Relevance of hot spots in the evolution and transmission of Tn1546 in glycopeptide-resistant Enterococcus faecium (GREF) from broiler origin; citation_author=Garcia-Migura  L.; citation_author=Hasman  H.; citation_author=Svendsen  C.; citation_author=Jensen  L. B.;  citation_journal_title=J. Antimicrob. Chemother;  citation_year=2008;  citation_volume=62;  citation_pages=681-687; " /><meta name="citation_reference" content="citation_title=Molecular description and industrial potential of Tn6098 conjugative transfer conferring alpha-galactoside metabolism in Lactococcus lactis; citation_author=Machielsen  R.; citation_author=Siezen  R. J.; citation_author=van Hijum  S. A.; citation_author=van Hylckama Vlieg  J. E.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2011;  citation_volume=77;  citation_pages=555-563; " /><meta name="citation_reference" content="citation_title=Characterization of the ends and target site of a novel tetracycline resistance-encoding conjugative transposon from Enterococcus faecium 664.1H1; citation_author=Roberts  A. P.; citation_author=Davis  I. J.; citation_author=Seville  L.; citation_author=Villedieu  A.; citation_author=Mullany  P.;  citation_journal_title=J. Bacteriol;  citation_year=2006;  citation_volume=188;  citation_pages=4356-61; " /><meta name="citation_reference" content="citation_title=IS-linked movement of a restriction-modification system; citation_author=Takahashi  N.; citation_author=Ohashi  S.; citation_author=Sadykov  M. R.; citation_author=Mizutani-Ui  Y.; citation_author=Kobayashi  I.;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16554" /><meta name="citation_reference" content="citation_title=Evidence for horizontal transfer of the EcoT38I restriction-modification gene to chromosomal DNA by the P2 phage and diversity of defective P2 prophages in Escherichia coli TH38 strains; citation_author=Kita  K.; citation_author=Kawakami  H.; citation_author=Tanaka  H.;  citation_journal_title=J. Bacteriol;  citation_year=2003;  citation_volume=185;  citation_pages=2296-305; " /><meta name="citation_reference" content="citation_title=Characterization of a novel type II restriction-modification system, Sth368I, encoded by the integrative element ICESt1 of Streptococcus thermophilus CNRZ368; citation_author=Burrus  V.; citation_author=Bontemps  C.; citation_author=Decaris  B.; citation_author=Guedon  G.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2001;  citation_volume=67;  citation_pages=1522-8; " /><meta name="citation_reference" content="citation_title=The interplay of restriction-modification systems with mobile genetic elements and their prokaryotic hosts; citation_author=Oliveira  P. H.; citation_author=Touchon  M.; citation_author=Rocha  E. P.;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=10618-31; " /><meta name="citation_reference" content="citation_title=Evolution of prokaryotic gene order: genome rearrangements in closely related species; citation_author=Suyama  M.; citation_author=Bork  P.;  citation_journal_title=Trends Genet;  citation_year=2001;  citation_volume=17;  citation_pages=10-3; " /><meta name="citation_reference" content="citation_title=Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes; citation_author=Gelfand  M. S.; citation_author=Koonin  E. V.;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=2430-9; " /><meta name="citation_reference" content="citation_title=Xer recombination in Escherichia coli. Site-specific DNA topoisomerase activity of the XerC and XerD recombinases; citation_author=Cornet  F.; citation_author=Hallet  B.; citation_author=Sherratt  D. J.;  citation_journal_title=J. Biol. Chem;  citation_year=1997;  citation_volume=272;  citation_pages=21927-31; " /><meta name="citation_reference" content="citation_title=Xer-mediated site-specific recombination in vitro; citation_author=Colloms  S. D.; citation_author=McCulloch  R.; citation_author=Grant  K.; citation_author=Neilson  L.; citation_author=Sherratt  D. J.;  citation_journal_title=EMBO J;  citation_year=1996;  citation_volume=15;  citation_pages=1172-81; " /><meta name="citation_reference" content="citation_title=A nomenclature for restriction enzymes, DNA methyltransferases, homing endonucleases and their genes; citation_author=Roberts  R. J.; citation_author=Belfort  M.; citation_author=Bestor  T.;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=1805-12; " /><meta name="citation_reference" content="citation_title=Restriction and modification systems; citation_author=Wilson  G. G.; citation_author=Murray  N. E.;  citation_journal_title=Annu. Rev. Genet;  citation_year=1991;  citation_volume=25;  citation_pages=585-627; " /><meta name="citation_reference" content="citation_title=Complete nucleotide sequence of the freshwater unicellular cyanobacterium Synechococcus elongatus PCC 6301 chromosome: gene content and organization; citation_author=Sugita  C.; citation_author=Ogata  K.; citation_author=Shikata  M.;  citation_journal_title=Photosynth Res;  citation_year=2007;  citation_volume=93;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Novel metabolic attributes of the genus cyanothece, comprising a group of unicellular nitrogen-fixing Cyanothece; citation_author=Bandyopadhyay  A.; citation_author=Elvitigala  T.; citation_author=Welsh  E.;  citation_journal_title=MBio;  citation_year=2011;  citation_volume=2;  citation_pages=e00214-11; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the genome sequences of Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica; citation_author=Parkhill  J.; citation_author=Sebaihia  M.; citation_author=Preston  A.;  citation_journal_title=Nat. Genet;  citation_year=2003;  citation_volume=35;  citation_pages=32-40; " /><meta name="citation_reference" content="citation_title=DNA repeats lead to the accelerated loss of gene order in bacteria; citation_author=Rocha  E. P.;  citation_journal_title=Trends Genet;  citation_year=2003;  citation_volume=19;  citation_pages=600-3; " /><meta name="citation_reference" content="citation_title=Short-sequence DNA repeats in prokaryotic genomes; citation_author=van Belkum  A.; citation_author=Scherer  S.; citation_author=van Alphen  L.; citation_author=Verbrugh  H.;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=1998;  citation_volume=62;  citation_pages=275-93; " /><meta name="citation_reference" content="citation_title=Adaptive evolution of highly mutable loci in pathogenic bacteria; citation_author=Moxon  E. R.; citation_author=Rainey  P. B.; citation_author=Nowak  M. A.; citation_author=Lenski  R. E.;  citation_journal_title=Curr. Biol;  citation_year=1994;  citation_volume=4;  citation_pages=24-33; " /><meta name="citation_reference" content="citation_title=Order and disorder in bacterial genomes; citation_author=Rocha  E. P.;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2004;  citation_volume=7;  citation_pages=519-27; " /><meta name="citation_reference" content="citation_title=Detection and possible role of two large nondivisible zones on the Escherichia coli chromosome; citation_author=Rebollo  J. E.; citation_author=Francois  V.; citation_author=Louarn  J. M.;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=1988;  citation_volume=85;  citation_pages=9391-5; " /><meta name="citation_reference" content="citation_title=Chromosomal constraints in Gram-positive bacteria revealed by artificial inversions; citation_author=Campo  N.; citation_author=Dias  M. J.; citation_author=Daveran-Mingot  M. L.; citation_author=Ritzenthaler  P.; citation_author=Le Bourgeois  P.;  citation_journal_title=Mol. Microbiol;  citation_year=2004;  citation_volume=51;  citation_pages=511-22; " /><meta name="citation_reference" content="citation_title=Unequal access of chromosomal regions to each other in Salmonella: probing chromosome structure with phage lambda integrase-mediated long-range rearrangements; citation_author=Garcia-Russell  N.; citation_author=Harmon  T. G.; citation_author=Le  T. Q.; citation_author=Amaladas  N. H.; citation_author=Mathewson  R. D.; citation_author=Segall  A. M.;  citation_journal_title=Mol. Microbiol;  citation_year=2004;  citation_volume=52;  citation_pages=329-44; " /><meta name="citation_reference" content="citation_title=Two strains of Crocosphaera watsonii with highly conserved genomes Eare distinguished by strain-specific features; citation_author=Bench  S. R.; citation_author=Ilikchyan  I. N.; citation_author=Tripp  H. J.; citation_author=Zehr  J. P.;  citation_journal_title=Front. Microbiol;  citation_year=2011;  citation_volume=2;  citation_pages=261" /><meta name="citation_reference" content="citation_title=Complete genome structure of the thermophilic cyanobacterium Thermosynechococcus elongatus BP-1; citation_author=Nakamura  Y.; citation_author=Kaneko  T.; citation_author=Sato  S.;  citation_journal_title=DNA Res;  citation_year=2002;  citation_volume=9;  citation_pages=123-30; " /><meta name="citation_reference" content="citation_title=Highly plastic genome of Microcystis aeruginosa PCC 7806, a ubiquitous toxic freshwater cyanobacterium; citation_author=Frangeul  L.; citation_author=Quillardet  P.; citation_author=Castets  A. M.;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=274" /><meta name="citation_reference" content="citation_title=Complete genomic structure of the bloom-forming toxic cyanobacterium Microcystis aeruginosa NIES-843; citation_author=Kaneko  T.; citation_author=Nakajima  N.; citation_author=Okamoto  S.;  citation_journal_title=DNA Res;  citation_year=2007;  citation_volume=14;  citation_pages=247-56; " /><meta name="citation_reference" content="citation_title=Tracing isolates of bacterial species by multilocus variable number of tandem repeat analysis (MLVA); citation_author=van Belkum  A.;  citation_journal_title=FEMS Immunol. Med. Microbiol;  citation_year=2007;  citation_volume=49;  citation_pages=22-7; " /><meta name="citation_reference" content="citation_title=Complex minisatellite rearrangements generated in the total or partial absence of Rad27/hFEN1 activity occur in a single generation and are Rad51 and Rad52 dependent; citation_author=Lopes  J.; citation_author=Ribeyre  C.; citation_author=Nicolas  A.;  citation_journal_title=Mol. Cell Biol;  citation_year=2006;  citation_volume=26;  citation_pages=6675-89; " /><meta name="citation_reference" content="citation_title=CRISPR-Cas systems in bacteria and archaea: versatile small RNAs for adaptive defense and regulation; citation_author=Bhaya  D.; citation_author=Davison  M.; citation_author=Barrangou  R.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2011;  citation_volume=45;  citation_pages=273-297; " /><meta name="citation_reference" content="citation_title=CRISPR-Cas systems for editing, regulating and targeting genomes; citation_author=Sander  J. D.; citation_author=Joung  J. K.;  citation_journal_title=Nat. Biotechnol;  citation_year=2014;  citation_volume=32;  citation_pages=347-55; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/325/17463260/dsw023.pdf" /><meta name="description" content="Abstract.  Arthrospira platensis is a multi-cellular and filamentous non-N2-fixing cyanobacterium that is capable of performing oxygenic photosynthesis. In this" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/325/2469971" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/325/2469971" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/325/2469971");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw023';
    </script>
<title>
	 Whole genomic DNA sequencing and comparative genomic analysis of Arthrospira platensis : high genome plasticity and genetic diversity  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469971" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeDOATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWHUsIFRlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgU2Nob29sIG9mIExhYm9yYXRvcnkgTWVkaWNpbmUgYW5kIExpZmUgU2NpZW5jZS9JbnN0aXR1dGUgb2YgQmlvbWVkaWNhbCBJbmZvcm1hdGljcywgV2VuemhvdSBNZWRpY2FsIFVuaXZlcnNpdHksIFdlbnpob3UgMzI1MDM1LCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJRaW4sIFNvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSW5zdGl0dXRlIG9mIE9jZWFub2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgUWluZ2RhbyAyNjYwNzEsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1LCBZb25nd3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgU2Nob29sIG9mIExhYm9yYXRvcnkgTWVkaWNpbmUgYW5kIExpZmUgU2NpZW5jZS9JbnN0aXR1dGUgb2YgQmlvbWVkaWNhbCBJbmZvcm1hdGljcywgV2VuemhvdSBNZWRpY2FsIFVuaXZlcnNpdHksIFdlbnpob3UgMzI1MDM1LCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQkdJLVNoZW56aGVuLCBTaGVuemhlbiA1MTgwODMsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNvbmcsIFpoaWppYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgU2Nob29sIG9mIExhYm9yYXRvcnkgTWVkaWNpbmUgYW5kIExpZmUgU2NpZW5jZS9JbnN0aXR1dGUgb2YgQmlvbWVkaWNhbCBJbmZvcm1hdGljcywgV2VuemhvdSBNZWRpY2FsIFVuaXZlcnNpdHksIFdlbnpob3UgMzI1MDM1LCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZaW5nLCBKaWFuY2hhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBTY2hvb2wgb2YgTGFib3JhdG9yeSBNZWRpY2luZSBhbmQgTGlmZSBTY2llbmNlL0luc3RpdHV0ZSBvZiBCaW9tZWRpY2FsIEluZm9ybWF0aWNzLCBXZW56aG91IE1lZGljYWwgVW5pdmVyc2l0eSwgV2VuemhvdSAzMjUwMzUsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBQZWl6aGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIFNjaG9vbCBvZiBMYWJvcmF0b3J5IE1lZGljaW5lIGFuZCBMaWZlIFNjaWVuY2UvSW5zdGl0dXRlIG9mIEJpb21lZGljYWwgSW5mb3JtYXRpY3MsIFdlbnpob3UgTWVkaWNhbCBVbml2ZXJzaXR5LCBXZW56aG91IDMyNTAzNSwgQ2hpbmEgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRG9uZywgV2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIEJHSS1TaGVuemhlbiwgU2hlbnpoZW4gNTE4MDgzLCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFvLCBGYW5ncWluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJbnN0aXR1dGUgb2YgT2NlYW5vbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBRaW5nZGFvIDI2NjA3MSwgQ2hpbmEgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgSHVhbm1pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQkdJLVNoZW56aGVuLCBTaGVuemhlbiA1MTgwODMsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBKYW1lcyBELiBXYXRzb24gSW5zdGl0dXRlIG9mIEdlbm9tZSBTY2llbmNlcywgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW8sIFFpeXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgU2Nob29sIG9mIExhYm9yYXRvcnkgTWVkaWNpbmUgYW5kIExpZmUgU2NpZW5jZS9JbnN0aXR1dGUgb2YgQmlvbWVkaWNhbCBJbmZvcm1hdGljcywgV2VuemhvdSBNZWRpY2FsIFVuaXZlcnNpdHksIFdlbnpob3UgMzI1MDM1LCBDaGluYSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQkdJLVNoZW56aGVuLCBTaGVuemhlbiA1MTgwODMsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iIFdob2xlIGdlbm9taWMgRE5BIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGdlbm9taWMgYW5hbHlzaXMgb2YgQXJ0aHJvc3BpcmEgcGxhdGVuc2lzIDogaGlnaCBnZW5vbWUgcGxhc3RpY2l0eSBhbmQgZ2VuZXRpYyBkaXZlcnNpdHkgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1IYWJpYiAgTS4gQS4gQi47IGNpdGF0aW9uX2F1dGhvcj1NYXNodWRhICBQLjsgY2l0YXRpb25fYXV0aG9yPUh1bnRpbmd0b24gIFQuIEMuOyBjaXRhdGlvbl9hdXRob3I9SGFzYW4gIE0uIFIuOyBjaXRhdGlvbl9wdWJsaXNoZXI9Um9tZSwgSXRhbHksIEZvb2QgYW5kIEFncmljdWx0dXJlIE9yZ2FuaXphdGlvbiBvZiBUaGUgVW5pdGVkIE5hdGlvbnM7ICBjaXRhdGlvbl90aXRsZT1BIFJldmlldyBvbiBDdWx0dXJlLCBQcm9kdWN0aW9uIGFuZCBVc2Ugb2YgU3BpcnVsaW5hIGFzIEZvb2QgZm9yIEh1bWFucyBhbmQgRmVlZHMgZm9yIERvbWVzdGljIEFuaW1hbHMgYW5kIEZpc2g7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21pYyBhbmFseXNpcyBhbmQgcVJULVBDUiB2ZXJpZmljYXRpb24gb2YgdGVtcGVyYXR1cmUgcmVzcG9uc2UgdG8gQXJ0aHJvc3BpcmEgKFNwaXJ1bGluYSkgcGxhdGVuc2lzOyBjaXRhdGlvbl9hdXRob3I9SHVpbGkgIFcuOyBjaXRhdGlvbl9hdXRob3I9WGlhb2thaSAgWi47IGNpdGF0aW9uX2F1dGhvcj1NZWlsaSAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lODM0ODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBkaWZmZXJlbnRpYWxseSBleHByZXNzZWQgcHJvdGVpbnMgb2YgQXJ0aHJvc3BpcmEgKFNwaXJ1bGluYSkgcGxhbnRlbnNpcy1ZWiB1bmRlciBzYWx0LXN0cmVzcyBjb25kaXRpb25zIGJ5IHByb3Rlb21pY3MgYW5kIHFSVC1QQ1IgYW5hbHlzaXM7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBILjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFkuOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3Rlb21lIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGlydWxpbmEgcGh5Y29jeWFuaW4gaW5kdWNlcyBkaWZmZXJlbnRpYWwgcHJvdGVpbiBleHByZXNzaW9uIGFuZCBhcG9wdG9zaXMgaW4gU0tPVi0zIGNlbGxzOyBjaXRhdGlvbl9hdXRob3I9UGFuICBSLjsgY2l0YXRpb25fYXV0aG9yPUx1ICBSLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBCaW9sLiBNYWNyb21vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTk1MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiByZXN0cmljdGlvbi1tb2RpZmljYXRpb24gc3lzdGVtIGluIHVuaWNlbGx1bGFyIGFuZCBmaWxhbWVudG91cyBjeWFub2JhY3RlcmlhOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgRi47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWC47IGNpdGF0aW9uX2F1dGhvcj1MaWFuZyAgQy47IGNpdGF0aW9uX2F1dGhvcj1XdSAgSi47IGNpdGF0aW9uX2F1dGhvcj1CYW8gIFEuOyBjaXRhdGlvbl9hdXRob3I9UWluICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5c2lvbC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBBcnRocm9zcGlyYSBwbGF0ZW5zaXMgQzEgKFBDQzk0MzgpOyBjaXRhdGlvbl9hdXRob3I9Q2hlZXZhZGhhbmFyYWsgIFMuOyBjaXRhdGlvbl9hdXRob3I9UGFpdGhvb25yYW5nc2FyaWQgIEsuOyBjaXRhdGlvbl9hdXRob3I9UHJvbW1lZW5hdGUgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TdGFuZC4gR2Vub21pYyBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTQzLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGVkaWJsZSBjeWFub2JhY3Rlcml1bSBBcnRocm9zcGlyYSBzcC4gUENDIDgwMDU7IGNpdGF0aW9uX2F1dGhvcj1KYW5zc2VuICBQLiBKLjsgY2l0YXRpb25fYXV0aG9yPU1vcmluICBOLjsgY2l0YXRpb25fYXV0aG9yPU1lcmdlYXkgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTkyOyAgY2l0YXRpb25fcGFnZXM9MjQ2NS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHN0cnVjdHVyZSBvZiBhbiBlY29ub21pY2FsbHkgaW1wb3J0YW50IGN5YW5vYmFjdGVyaXVtLCBBcnRocm9zcGlyYSAoU3BpcnVsaW5hKSBwbGF0ZW5zaXMgTklFUy0zOTsgY2l0YXRpb25fYXV0aG9yPUZ1amlzYXdhICBULjsgY2l0YXRpb25fYXV0aG9yPU5hcmlrYXdhICBSLjsgY2l0YXRpb25fYXV0aG9yPU9rYW1vdG8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9ODUtMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBsZXRlIHNlcXVlbmNlIG9mIHRoZSBULiB0ZW5nY29uZ2Vuc2lzIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUJhbyAgUS47IGNpdGF0aW9uX2F1dGhvcj1UaWFuICBZLjsgY2l0YXRpb25fYXV0aG9yPUxpICBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY4OS03MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgYW5kIGF1dG9tYXRlZCB3aG9sZS1nZW5vbWUgb3B0aWNhbCBtYXBwaW5nIG9mIHRoZSBnZW5vbWUgb2YgYSBkb21lc3RpYyBnb2F0IChDYXByYSBoaXJjdXMpOyBjaXRhdGlvbl9hdXRob3I9RG9uZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1YaWUgIE0uOyBjaXRhdGlvbl9hdXRob3I9SmlhbmcgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlZDogYSBncmFwaGljYWwgdG9vbCBmb3Igc2VxdWVuY2UgZmluaXNoaW5nOyBjaXRhdGlvbl9hdXRob3I9R29yZG9uICBELjsgY2l0YXRpb25fYXV0aG9yPUFiYWppYW4gIEMuOyBjaXRhdGlvbl9hdXRob3I9R3JlZW4gIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xOTUtMjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2Vub21lIEFuYWx5c2lzIFRvb2xraXQ6IGEgTWFwUmVkdWNlIGZyYW1ld29yayBmb3IgYW5hbHl6aW5nIG5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZyBkYXRhOyBjaXRhdGlvbl9hdXRob3I9TWNLZW5uYSAgQS47IGNpdGF0aW9uX2F1dGhvcj1IYW5uYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1CYW5rcyAgRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgYmFjdGVyaWFsIGdlbmVzIGFuZCBlbmRvc3ltYmlvbnQgRE5BIHdpdGggR2xpbW1lcjsgY2l0YXRpb25fYXV0aG9yPURlbGNoZXIgIEEuIEwuOyBjaXRhdGlvbl9hdXRob3I9QnJhdGtlICBLLiBBLjsgY2l0YXRpb25fYXV0aG9yPVBvd2VycyAgRS4gQy47IGNpdGF0aW9uX2F1dGhvcj1TYWx6YmVyZyAgUy4gTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NjczLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQW1tZXI6IGNvbnNpc3RlbnQgYW5kIHJhcGlkIGFubm90YXRpb24gb2Ygcmlib3NvbWFsIFJOQSBnZW5lczsgY2l0YXRpb25fYXV0aG9yPUxhZ2VzZW4gIEsuOyBjaXRhdGlvbl9hdXRob3I9SGFsbGluICBQLjsgY2l0YXRpb25fYXV0aG9yPVJvZGxhbmQgIEUuIEEuOyBjaXRhdGlvbl9hdXRob3I9U3RhZXJmZWxkdCAgSC4gSC47IGNpdGF0aW9uX2F1dGhvcj1Sb2duZXMgIFQuOyBjaXRhdGlvbl9hdXRob3I9VXNzZXJ5ICBELiBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zMTAwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgY2l0YXRpb25fYXV0aG9yPUxvd2UgIFQuIE0uOyBjaXRhdGlvbl9hdXRob3I9RWRkeSAgUy4gUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGl2ZSBtaWNyb2JpYWwgZ2Vub21lIHZpc3VhbGl6YXRpb24gd2l0aCBHVmlldzsgY2l0YXRpb25fYXV0aG9yPVBldGthdSAgQS47IGNpdGF0aW9uX2F1dGhvcj1TdHVhcnQtRWR3YXJkcyAgTS47IGNpdGF0aW9uX2F1dGhvcj1TdG90aGFyZCAgUC47IGNpdGF0aW9uX2F1dGhvcj1WYW4gRG9tc2VsYWFyICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zMTI1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJFQkFTReKAk2EgZGF0YWJhc2UgZm9yIEROQSByZXN0cmljdGlvbiBhbmQgbW9kaWZpY2F0aW9uOiBlbnp5bWVzLCBnZW5lcyBhbmQgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPVJvYmVydHMgIFIuIEouOyBjaXRhdGlvbl9hdXRob3I9VmluY3plICBULjsgY2l0YXRpb25fYXV0aG9yPVBvc2ZhaSAgSi47IGNpdGF0aW9uX2F1dGhvcj1NYWNlbGlzICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1EMjM0LTIzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUsIGZhc3QsIGFuZCBhY2N1cmF0ZSBtZXRob2Qgb2YgcGh5bG9nZW5vbWljIGluZmVyZW5jZTsgY2l0YXRpb25fYXV0aG9yPVd1ICBNLjsgY2l0YXRpb25fYXV0aG9yPUVpc2VuICBKLiBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPVIxNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgYWxnb3JpdGhtcyBhbmQgbWV0aG9kcyB0byBlc3RpbWF0ZSBtYXhpbXVtLWxpa2VsaWhvb2QgcGh5bG9nZW5pZXM6IGFzc2Vzc2luZyB0aGUgcGVyZm9ybWFuY2Ugb2YgUGh5TUwgMy4wOyBjaXRhdGlvbl9hdXRob3I9R3VpbmRvbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1EdWZheWFyZCAgSi4gRi47IGNpdGF0aW9uX2F1dGhvcj1MZWZvcnQgIFYuOyBjaXRhdGlvbl9hdXRob3I9QW5pc2ltb3ZhICBNLjsgY2l0YXRpb25fYXV0aG9yPUhvcmRpamsgIFcuOyBjaXRhdGlvbl9hdXRob3I9R2FzY3VlbCAgTy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0zMDctMzIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JU2ZpbmRlcjogdGhlIHJlZmVyZW5jZSBjZW50cmUgZm9yIGJhY3RlcmlhbCBpbnNlcnRpb24gc2VxdWVuY2VzOyBjaXRhdGlvbl9hdXRob3I9U2lndWllciAgUC47IGNpdGF0aW9uX2F1dGhvcj1QZXJvY2hvbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1MZXN0cmFkZSAgTC47IGNpdGF0aW9uX2F1dGhvcj1NYWhpbGxvbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1DaGFuZGxlciAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9RDMyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXByb2dyZXNzaXZlTWF1dmU6IG11bHRpcGxlIGdlbm9tZSBhbGlnbm1lbnQgd2l0aCBnZW5lIGdhaW4sIGxvc3MgYW5kIHJlYXJyYW5nZW1lbnQ7IGNpdGF0aW9uX2F1dGhvcj1EYXJsaW5nICBBLiBFLjsgY2l0YXRpb25fYXV0aG9yPU1hdSAgQi47IGNpdGF0aW9uX2F1dGhvcj1QZXJuYSAgTi4gVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTExNDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZC1oaXQ6IGEgZmFzdCBwcm9ncmFtIGZvciBjbHVzdGVyaW5nIGFuZCBjb21wYXJpbmcgbGFyZ2Ugc2V0cyBvZiBwcm90ZWluIG9yIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBjaXRhdGlvbl9hdXRob3I9TGkgIFcuOyBjaXRhdGlvbl9hdXRob3I9R29kemlrICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjU4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSByZXBlYXRzIGZpbmRlcjogYSBwcm9ncmFtIHRvIGFuYWx5emUgRE5BIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPUJlbnNvbiAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFJGaW5kZXI6IGEgd2ViIHRvb2wgdG8gaWRlbnRpZnkgY2x1c3RlcmVkIHJlZ3VsYXJseSBpbnRlcnNwYWNlZCBzaG9ydCBwYWxpbmRyb21pYyByZXBlYXRzOyBjaXRhdGlvbl9hdXRob3I9R3Jpc3NhICBJLjsgY2l0YXRpb25fYXV0aG9yPVZlcmduYXVkICBHLjsgY2l0YXRpb25fYXV0aG9yPVBvdXJjZWwgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVc1Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgYW5kIGdlbmUgZXhwcmVzc2lvbiBhbmFseXNlcyBvZiBBc2FpYSBib2dvcmVuc2lzIHJldmVhbCB1bmlxdWUgcmVzcG9uc2VzIHRvIGN1bHR1cmUgd2l0aCBtYW1tYWxpYW4gY2VsbHMgYXMgYSBwb3RlbnRpYWwgb3Bwb3J0dW5pc3RpYyBodW1hbiBwYXRob2dlbjsgY2l0YXRpb25fYXV0aG9yPUthd2FpICBNLjsgY2l0YXRpb25fYXV0aG9yPUhpZ2FzaGl1cmEgIE4uOyBjaXRhdGlvbl9hdXRob3I9SGF5YXNha2kgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbnMgYmV0d2VlbiBpbnZlcnRlZCByZXBlYXRzIGFuZCB0aGUgc3RydWN0dXJhbCBldm9sdXRpb24gb2YgYmFjdGVyaWFsIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1BY2hheiAgRy47IGNpdGF0aW9uX2F1dGhvcj1Db2lzc2FjICBFLjsgY2l0YXRpb25fYXV0aG9yPU5ldHRlciAgUC47IGNpdGF0aW9uX2F1dGhvcj1Sb2NoYSAgRS4gUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTEyNzktODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyB0aGUgY292ZXJhZ2Ugb2YgdGhlIGN5YW5vYmFjdGVyaWFsIHBoeWx1bSB1c2luZyBkaXZlcnNpdHktZHJpdmVuIGdlbm9tZSBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9U2hpaCAgUC4gTS47IGNpdGF0aW9uX2F1dGhvcj1XdSAgRC47IGNpdGF0aW9uX2F1dGhvcj1MYXRpZmkgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uY2VydGVkIGFuZCBiaXJ0aC1hbmQtZGVhdGggZXZvbHV0aW9uIG9mIG11bHRpZ2VuZSBmYW1pbGllczsgY2l0YXRpb25fYXV0aG9yPU5laSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Sb29uZXkgIEEuIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTIxLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1HZW5vbWUgU2hvdGd1biBTZXF1ZW5jZSBvZiBBcnRocm9zcGlyYSBwbGF0ZW5zaXMgU3RyYWluIFBhcmFjYSwgYSBDdWx0aXZhdGVkIGFuZCBFZGlibGUgQ3lhbm9iYWN0ZXJpdW07IGNpdGF0aW9uX2F1dGhvcj1MZWZvcnQgIEYuOyBjaXRhdGlvbl9hdXRob3I9Q2FsbWluICBHLjsgY2l0YXRpb25fYXV0aG9yPUNyb3ZhZG9yZSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBBbm5vdW5jOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhdGVyYWwgZ2VuZSB0cmFuc2ZlciBhbmQgdGhlIG5hdHVyZSBvZiBiYWN0ZXJpYWwgaW5ub3ZhdGlvbjsgY2l0YXRpb25fYXV0aG9yPU9jaG1hbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1MYXdyZW5jZSAgSi4gRy47IGNpdGF0aW9uX2F1dGhvcj1Hcm9pc21hbiAgRS4gQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDU7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIGV2b2x1dGlvbiBvZiByZXRyb2VsZW1lbnRzIGJhc2VkIHVwb24gdGhlaXIgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPVhpb25nICBZLjsgY2l0YXRpb25fYXV0aG9yPUVpY2tidXNoICBULiBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMzUzLTMzNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldmVyc2UgdHJhbnNjcmlwdGFzZSBhY3Rpdml0eSBvZiBhbiBpbnRyb24gZW5jb2RlZCBwb2x5cGVwdGlkZTsgY2l0YXRpb25fYXV0aG9yPUZhc3NiZW5kZXIgIFMuOyBjaXRhdGlvbl9hdXRob3I9QnJ1aGwgIEsuIEguOyBjaXRhdGlvbl9hdXRob3I9Q2lyaWFjeSAgTS47IGNpdGF0aW9uX2F1dGhvcj1LdWNrICBVLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjA3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zcG9zb24gVG45IGdlbmVyYXRlcyBhIDnigIlicCByZXBlYXRlZCBzZXF1ZW5jZSBkdXJpbmcgaW50ZWdyYXRpb247IGNpdGF0aW9uX2F1dGhvcj1Kb2huc3J1ZCAgTC47IGNpdGF0aW9uX2F1dGhvcj1DYWxvcyAgTS4gUC47IGNpdGF0aW9uX2F1dGhvcj1NaWxsZXIgIEouIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0xOTc4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTIwOS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsZXZhbmNlIG9mIGhvdCBzcG90cyBpbiB0aGUgZXZvbHV0aW9uIGFuZCB0cmFuc21pc3Npb24gb2YgVG4xNTQ2IGluIGdseWNvcGVwdGlkZS1yZXNpc3RhbnQgRW50ZXJvY29jY3VzIGZhZWNpdW0gKEdSRUYpIGZyb20gYnJvaWxlciBvcmlnaW47IGNpdGF0aW9uX2F1dGhvcj1HYXJjaWEtTWlndXJhICBMLjsgY2l0YXRpb25fYXV0aG9yPUhhc21hbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1TdmVuZHNlbiAgQy47IGNpdGF0aW9uX2F1dGhvcj1KZW5zZW4gIEwuIEIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbnRpbWljcm9iLiBDaGVtb3RoZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz02ODEtNjg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZGVzY3JpcHRpb24gYW5kIGluZHVzdHJpYWwgcG90ZW50aWFsIG9mIFRuNjA5OCBjb25qdWdhdGl2ZSB0cmFuc2ZlciBjb25mZXJyaW5nIGFscGhhLWdhbGFjdG9zaWRlIG1ldGFib2xpc20gaW4gTGFjdG9jb2NjdXMgbGFjdGlzOyBjaXRhdGlvbl9hdXRob3I9TWFjaGllbHNlbiAgUi47IGNpdGF0aW9uX2F1dGhvcj1TaWV6ZW4gIFIuIEouOyBjaXRhdGlvbl9hdXRob3I9dmFuIEhpanVtICBTLiBBLjsgY2l0YXRpb25fYXV0aG9yPXZhbiBIeWxja2FtYSBWbGllZyAgSi4gRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTU1NS01NjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGVuZHMgYW5kIHRhcmdldCBzaXRlIG9mIGEgbm92ZWwgdGV0cmFjeWNsaW5lIHJlc2lzdGFuY2UtZW5jb2RpbmcgY29uanVnYXRpdmUgdHJhbnNwb3NvbiBmcm9tIEVudGVyb2NvY2N1cyBmYWVjaXVtIDY2NC4xSDE7IGNpdGF0aW9uX2F1dGhvcj1Sb2JlcnRzICBBLiBQLjsgY2l0YXRpb25fYXV0aG9yPURhdmlzICBJLiBKLjsgY2l0YXRpb25fYXV0aG9yPVNldmlsbGUgIEwuOyBjaXRhdGlvbl9hdXRob3I9VmlsbGVkaWV1ICBBLjsgY2l0YXRpb25fYXV0aG9yPU11bGxhbnkgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9NDM1Ni02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SVMtbGlua2VkIG1vdmVtZW50IG9mIGEgcmVzdHJpY3Rpb24tbW9kaWZpY2F0aW9uIHN5c3RlbTsgY2l0YXRpb25fYXV0aG9yPVRha2FoYXNoaSAgTi47IGNpdGF0aW9uX2F1dGhvcj1PaGFzaGkgIFMuOyBjaXRhdGlvbl9hdXRob3I9U2FkeWtvdiAgTS4gUi47IGNpdGF0aW9uX2F1dGhvcj1NaXp1dGFuaS1VaSAgWS47IGNpdGF0aW9uX2F1dGhvcj1Lb2JheWFzaGkgIEkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE2NTU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIGhvcml6b250YWwgdHJhbnNmZXIgb2YgdGhlIEVjb1QzOEkgcmVzdHJpY3Rpb24tbW9kaWZpY2F0aW9uIGdlbmUgdG8gY2hyb21vc29tYWwgRE5BIGJ5IHRoZSBQMiBwaGFnZSBhbmQgZGl2ZXJzaXR5IG9mIGRlZmVjdGl2ZSBQMiBwcm9waGFnZXMgaW4gRXNjaGVyaWNoaWEgY29saSBUSDM4IHN0cmFpbnM7IGNpdGF0aW9uX2F1dGhvcj1LaXRhICBLLjsgY2l0YXRpb25fYXV0aG9yPUthd2FrYW1pICBILjsgY2l0YXRpb25fYXV0aG9yPVRhbmFrYSAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz0yMjk2LTMwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBhIG5vdmVsIHR5cGUgSUkgcmVzdHJpY3Rpb24tbW9kaWZpY2F0aW9uIHN5c3RlbSwgU3RoMzY4SSwgZW5jb2RlZCBieSB0aGUgaW50ZWdyYXRpdmUgZWxlbWVudCBJQ0VTdDEgb2YgU3RyZXB0b2NvY2N1cyB0aGVybW9waGlsdXMgQ05SWjM2ODsgY2l0YXRpb25fYXV0aG9yPUJ1cnJ1cyAgVi47IGNpdGF0aW9uX2F1dGhvcj1Cb250ZW1wcyAgQy47IGNpdGF0aW9uX2F1dGhvcj1EZWNhcmlzICBCLjsgY2l0YXRpb25fYXV0aG9yPUd1ZWRvbiAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTE1MjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGludGVycGxheSBvZiByZXN0cmljdGlvbi1tb2RpZmljYXRpb24gc3lzdGVtcyB3aXRoIG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzIGFuZCB0aGVpciBwcm9rYXJ5b3RpYyBob3N0czsgY2l0YXRpb25fYXV0aG9yPU9saXZlaXJhICBQLiBILjsgY2l0YXRpb25fYXV0aG9yPVRvdWNob24gIE0uOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNjE4LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgcHJva2FyeW90aWMgZ2VuZSBvcmRlcjogZ2Vub21lIHJlYXJyYW5nZW1lbnRzIGluIGNsb3NlbHkgcmVsYXRlZCBzcGVjaWVzOyBjaXRhdGlvbl9hdXRob3I9U3V5YW1hICBNLjsgY2l0YXRpb25fYXV0aG9yPUJvcmsgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bdm9pZGFuY2Ugb2YgcGFsaW5kcm9taWMgd29yZHMgaW4gYmFjdGVyaWFsIGFuZCBhcmNoYWVhbCBnZW5vbWVzOiBhIGNsb3NlIGNvbm5lY3Rpb24gd2l0aCByZXN0cmljdGlvbiBlbnp5bWVzOyBjaXRhdGlvbl9hdXRob3I9R2VsZmFuZCAgTS4gUy47IGNpdGF0aW9uX2F1dGhvcj1Lb29uaW4gIEUuIFYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI0MzAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WGVyIHJlY29tYmluYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saS4gU2l0ZS1zcGVjaWZpYyBETkEgdG9wb2lzb21lcmFzZSBhY3Rpdml0eSBvZiB0aGUgWGVyQyBhbmQgWGVyRCByZWNvbWJpbmFzZXM7IGNpdGF0aW9uX2F1dGhvcj1Db3JuZXQgIEYuOyBjaXRhdGlvbl9hdXRob3I9SGFsbGV0ICBCLjsgY2l0YXRpb25fYXV0aG9yPVNoZXJyYXR0ICBELiBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0yMTkyNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WGVyLW1lZGlhdGVkIHNpdGUtc3BlY2lmaWMgcmVjb21iaW5hdGlvbiBpbiB2aXRybzsgY2l0YXRpb25fYXV0aG9yPUNvbGxvbXMgIFMuIEQuOyBjaXRhdGlvbl9hdXRob3I9TWNDdWxsb2NoICBSLjsgY2l0YXRpb25fYXV0aG9yPUdyYW50ICBLLjsgY2l0YXRpb25fYXV0aG9yPU5laWxzb24gIEwuOyBjaXRhdGlvbl9hdXRob3I9U2hlcnJhdHQgIEQuIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEo7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMTcyLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vbWVuY2xhdHVyZSBmb3IgcmVzdHJpY3Rpb24gZW56eW1lcywgRE5BIG1ldGh5bHRyYW5zZmVyYXNlcywgaG9taW5nIGVuZG9udWNsZWFzZXMgYW5kIHRoZWlyIGdlbmVzOyBjaXRhdGlvbl9hdXRob3I9Um9iZXJ0cyAgUi4gSi47IGNpdGF0aW9uX2F1dGhvcj1CZWxmb3J0ICBNLjsgY2l0YXRpb25fYXV0aG9yPUJlc3RvciAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTgwNS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzdHJpY3Rpb24gYW5kIG1vZGlmaWNhdGlvbiBzeXN0ZW1zOyBjaXRhdGlvbl9hdXRob3I9V2lsc29uICBHLiBHLjsgY2l0YXRpb25fYXV0aG9yPU11cnJheSAgTi4gRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz01ODUtNjI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBudWNsZW90aWRlIHNlcXVlbmNlIG9mIHRoZSBmcmVzaHdhdGVyIHVuaWNlbGx1bGFyIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jb2NjdXMgZWxvbmdhdHVzIFBDQyA2MzAxIGNocm9tb3NvbWU6IGdlbmUgY29udGVudCBhbmQgb3JnYW5pemF0aW9uOyBjaXRhdGlvbl9hdXRob3I9U3VnaXRhICBDLjsgY2l0YXRpb25fYXV0aG9yPU9nYXRhICBLLjsgY2l0YXRpb25fYXV0aG9yPVNoaWthdGEgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaG90b3N5bnRoIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBtZXRhYm9saWMgYXR0cmlidXRlcyBvZiB0aGUgZ2VudXMgY3lhbm90aGVjZSwgY29tcHJpc2luZyBhIGdyb3VwIG9mIHVuaWNlbGx1bGFyIG5pdHJvZ2VuLWZpeGluZyBDeWFub3RoZWNlOyBjaXRhdGlvbl9hdXRob3I9QmFuZHlvcGFkaHlheSAgQS47IGNpdGF0aW9uX2F1dGhvcj1FbHZpdGlnYWxhICBULjsgY2l0YXRpb25fYXV0aG9yPVdlbHNoICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TUJpbzsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTAwMjE0LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlcyBvZiBCb3JkZXRlbGxhIHBlcnR1c3NpcywgQm9yZGV0ZWxsYSBwYXJhcGVydHVzc2lzIGFuZCBCb3JkZXRlbGxhIGJyb25jaGlzZXB0aWNhOyBjaXRhdGlvbl9hdXRob3I9UGFya2hpbGwgIEouOyBjaXRhdGlvbl9hdXRob3I9U2ViYWloaWEgIE0uOyBjaXRhdGlvbl9hdXRob3I9UHJlc3RvbiAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zMi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHJlcGVhdHMgbGVhZCB0byB0aGUgYWNjZWxlcmF0ZWQgbG9zcyBvZiBnZW5lIG9yZGVyIGluIGJhY3RlcmlhOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz02MDAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvcnQtc2VxdWVuY2UgRE5BIHJlcGVhdHMgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPXZhbiBCZWxrdW0gIEEuOyBjaXRhdGlvbl9hdXRob3I9U2NoZXJlciAgUy47IGNpdGF0aW9uX2F1dGhvcj12YW4gQWxwaGVuICBMLjsgY2l0YXRpb25fYXV0aG9yPVZlcmJydWdoICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTI3NS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRpdmUgZXZvbHV0aW9uIG9mIGhpZ2hseSBtdXRhYmxlIGxvY2kgaW4gcGF0aG9nZW5pYyBiYWN0ZXJpYTsgY2l0YXRpb25fYXV0aG9yPU1veG9uICBFLiBSLjsgY2l0YXRpb25fYXV0aG9yPVJhaW5leSAgUC4gQi47IGNpdGF0aW9uX2F1dGhvcj1Ob3dhayAgTS4gQS47IGNpdGF0aW9uX2F1dGhvcj1MZW5za2kgIFIuIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yNC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JkZXIgYW5kIGRpc29yZGVyIGluIGJhY3RlcmlhbCBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTUxOS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIGFuZCBwb3NzaWJsZSByb2xlIG9mIHR3byBsYXJnZSBub25kaXZpc2libGUgem9uZXMgb24gdGhlIEVzY2hlcmljaGlhIGNvbGkgY2hyb21vc29tZTsgY2l0YXRpb25fYXV0aG9yPVJlYm9sbG8gIEouIEUuOyBjaXRhdGlvbl9hdXRob3I9RnJhbmNvaXMgIFYuOyBjaXRhdGlvbl9hdXRob3I9TG91YXJuICBKLiBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTkzOTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgY29uc3RyYWludHMgaW4gR3JhbS1wb3NpdGl2ZSBiYWN0ZXJpYSByZXZlYWxlZCBieSBhcnRpZmljaWFsIGludmVyc2lvbnM7IGNpdGF0aW9uX2F1dGhvcj1DYW1wbyAgTi47IGNpdGF0aW9uX2F1dGhvcj1EaWFzICBNLiBKLjsgY2l0YXRpb25fYXV0aG9yPURhdmVyYW4tTWluZ290ICBNLiBMLjsgY2l0YXRpb25fYXV0aG9yPVJpdHplbnRoYWxlciAgUC47IGNpdGF0aW9uX2F1dGhvcj1MZSBCb3VyZ2VvaXMgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTUxMS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5lcXVhbCBhY2Nlc3Mgb2YgY2hyb21vc29tYWwgcmVnaW9ucyB0byBlYWNoIG90aGVyIGluIFNhbG1vbmVsbGE6IHByb2JpbmcgY2hyb21vc29tZSBzdHJ1Y3R1cmUgd2l0aCBwaGFnZSBsYW1iZGEgaW50ZWdyYXNlLW1lZGlhdGVkIGxvbmctcmFuZ2UgcmVhcnJhbmdlbWVudHM7IGNpdGF0aW9uX2F1dGhvcj1HYXJjaWEtUnVzc2VsbCAgTi47IGNpdGF0aW9uX2F1dGhvcj1IYXJtb24gIFQuIEcuOyBjaXRhdGlvbl9hdXRob3I9TGUgIFQuIFEuOyBjaXRhdGlvbl9hdXRob3I9QW1hbGFkYXMgIE4uIEguOyBjaXRhdGlvbl9hdXRob3I9TWF0aGV3c29uICBSLiBELjsgY2l0YXRpb25fYXV0aG9yPVNlZ2FsbCAgQS4gTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MzI5LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gc3RyYWlucyBvZiBDcm9jb3NwaGFlcmEgd2F0c29uaWkgd2l0aCBoaWdobHkgY29uc2VydmVkIGdlbm9tZXMgRWFyZSBkaXN0aW5ndWlzaGVkIGJ5IHN0cmFpbi1zcGVjaWZpYyBmZWF0dXJlczsgY2l0YXRpb25fYXV0aG9yPUJlbmNoICBTLiBSLjsgY2l0YXRpb25fYXV0aG9yPUlsaWtjaHlhbiAgSS4gTi47IGNpdGF0aW9uX2F1dGhvcj1UcmlwcCAgSC4gSi47IGNpdGF0aW9uX2F1dGhvcj1aZWhyICBKLiBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHN0cnVjdHVyZSBvZiB0aGUgdGhlcm1vcGhpbGljIGN5YW5vYmFjdGVyaXVtIFRoZXJtb3N5bmVjaG9jb2NjdXMgZWxvbmdhdHVzIEJQLTE7IGNpdGF0aW9uX2F1dGhvcj1OYWthbXVyYSAgWS47IGNpdGF0aW9uX2F1dGhvcj1LYW5la28gIFQuOyBjaXRhdGlvbl9hdXRob3I9U2F0byAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEyMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IHBsYXN0aWMgZ2Vub21lIG9mIE1pY3JvY3lzdGlzIGFlcnVnaW5vc2EgUENDIDc4MDYsIGEgdWJpcXVpdG91cyB0b3hpYyBmcmVzaHdhdGVyIGN5YW5vYmFjdGVyaXVtOyBjaXRhdGlvbl9hdXRob3I9RnJhbmdldWwgIEwuOyBjaXRhdGlvbl9hdXRob3I9UXVpbGxhcmRldCAgUC47IGNpdGF0aW9uX2F1dGhvcj1DYXN0ZXRzICBBLiBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yNzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWljIHN0cnVjdHVyZSBvZiB0aGUgYmxvb20tZm9ybWluZyB0b3hpYyBjeWFub2JhY3Rlcml1bSBNaWNyb2N5c3RpcyBhZXJ1Z2lub3NhIE5JRVMtODQzOyBjaXRhdGlvbl9hdXRob3I9S2FuZWtvICBULjsgY2l0YXRpb25fYXV0aG9yPU5ha2FqaW1hICBOLjsgY2l0YXRpb25fYXV0aG9yPU9rYW1vdG8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjQ3LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFjaW5nIGlzb2xhdGVzIG9mIGJhY3RlcmlhbCBzcGVjaWVzIGJ5IG11bHRpbG9jdXMgdmFyaWFibGUgbnVtYmVyIG9mIHRhbmRlbSByZXBlYXQgYW5hbHlzaXMgKE1MVkEpOyBjaXRhdGlvbl9hdXRob3I9dmFuIEJlbGt1bSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgSW1tdW5vbC4gTWVkLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz0yMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV4IG1pbmlzYXRlbGxpdGUgcmVhcnJhbmdlbWVudHMgZ2VuZXJhdGVkIGluIHRoZSB0b3RhbCBvciBwYXJ0aWFsIGFic2VuY2Ugb2YgUmFkMjcvaEZFTjEgYWN0aXZpdHkgb2NjdXIgaW4gYSBzaW5nbGUgZ2VuZXJhdGlvbiBhbmQgYXJlIFJhZDUxIGFuZCBSYWQ1MiBkZXBlbmRlbnQ7IGNpdGF0aW9uX2F1dGhvcj1Mb3BlcyAgSi47IGNpdGF0aW9uX2F1dGhvcj1SaWJleXJlICBDLjsgY2l0YXRpb25fYXV0aG9yPU5pY29sYXMgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTY2NzUtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUi1DYXMgc3lzdGVtcyBpbiBiYWN0ZXJpYSBhbmQgYXJjaGFlYTogdmVyc2F0aWxlIHNtYWxsIFJOQXMgZm9yIGFkYXB0aXZlIGRlZmVuc2UgYW5kIHJlZ3VsYXRpb247IGNpdGF0aW9uX2F1dGhvcj1CaGF5YSAgRC47IGNpdGF0aW9uX2F1dGhvcj1EYXZpc29uICBNLjsgY2l0YXRpb25fYXV0aG9yPUJhcnJhbmdvdSAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0yNzMtMjk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFItQ2FzIHN5c3RlbXMgZm9yIGVkaXRpbmcsIHJlZ3VsYXRpbmcgYW5kIHRhcmdldGluZyBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9U2FuZGVyICBKLiBELjsgY2l0YXRpb25fYXV0aG9yPUpvdW5nICBKLiBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNC8zMjUvMTc0NjMyNjAvZHN3MDIzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBcnRocm9zcGlyYSBwbGF0ZW5zaXMgaXMgYSBtdWx0aS1jZWxsdWxhciBhbmQgZmlsYW1lbnRvdXMgbm9uLU4yLWZpeGluZyBjeWFub2JhY3Rlcml1bSB0aGF0IGlzIGNhcGFibGUgb2YgcGVyZm9ybWluZyBveHlnZW5pYyBwaG90b3N5bnRoZXNpcy4gSW4gdGhpcyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2OTk3MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSk0jMgzKYloAJIMOoRvtjs78Rl+Q==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f325%2f2469971" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f325%2f2469971" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200332">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200332">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200334">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200334">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200336">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200336">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200340">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200340">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200369">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200369">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200371">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200371">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200373">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200373">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200375">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200375">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/311/2469972">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/4/339/2469970">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> Whole genomic DNA sequencing and comparative genomic analysis of <em>Arthrospira platensis</em> : high genome plasticity and genetic diversity 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Teng Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Teng+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Teng Xu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Song Qin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China </div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Song+Qin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qin S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qin S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Song Qin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongwu Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div><div class="aff"><div class="label -label"><sup>3</sup></div> BGI-Shenzhen, Shenzhen 518083, China </div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongwu+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yongwu Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhijian Song</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhijian+Song">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Song Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Song Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhijian Song</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianchao Ying</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianchao+Ying">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ying J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ying J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jianchao Ying</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peizhen Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peizhen+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Peizhen Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Dong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div> BGI-Shenzhen, Shenzhen 518083, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Dong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dong W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dong W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wei Dong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fangqing Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> Institute of Oceanology, Chinese Academy of Sciences, Qingdao 266071, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fangqing+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fangqing Zhao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huanming Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div> BGI-Shenzhen, Shenzhen 518083, China </div><div class="aff"><div class="label -label"><sup>4</sup></div> James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, China </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huanming+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huanming Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiyu Bao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> School of Laboratory Medicine and Life Science/Institute of Biomedical Informatics, Wenzhou Medical University, Wenzhou 325035, China </div><div class="aff"><div class="label -label"><sup>3</sup></div> BGI-Shenzhen, Shenzhen 518083, China </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw023-cor1" data-open="dsw023-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel: +86-577-86689779. Email: <a href="mailto:baoqy@genomics.org.cn" target="_blank">baoqy@genomics.org.cn</a> . 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qiyu+Bao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bao Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bao Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qiyu Bao</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw023-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw023-FM2"><div class="fn"><div class="footnote-content">Edited by Dr Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 325–338, <a href='https://doi.org/10.1093/dnares/dsw023'>https://doi.org/10.1093/dnares/dsw023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 June 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 May 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469971" href="/dnaresearch/article-pdf/23/4/325/17463260/dsw023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Teng Xu, Song Qin, Yongwu Hu, Zhijian Song, Jianchao Ying, Peizhen Li, Wei Dong, Fangqing Zhao, Huanming Yang, Qiyu Bao;   Whole genomic DNA sequencing and comparative genomic analysis of <em>Arthrospira platensis</em> : high genome plasticity and genetic diversity , <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 325–338, <a href='https://doi.org/10.1093/dnares/dsw023'>https://doi.org/10.1093/dnares/dsw023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469971?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20Whole%20genomic%20DNA%20sequencing%20and%20comparative%20genomic%20analysis%20of%20%26lt%3Bem%26gt%3BArthrospira%20platensis%26lt%3B%2Fem%26gt%3B%20%3A%20high%20genome%20plasticity%20and%20genetic%20diversity%20&amp;publicationDate=2016-06-21&amp;volumeNum=23&amp;issueNum=4&amp;author=Xu%2C%20Teng%3B%20Qin%2C%20Song&amp;startPage=325&amp;endPage=338&amp;contentId=10.1093%2Fdnares%2Fdsw023&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f325%2f2469971" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200332 id="87200332" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Arthrospira platensis</em> is a multi-cellular and filamentous non-N2-fixing cyanobacterium that is capable of performing oxygenic photosynthesis. In this study, we determined the nearly complete genome sequence of <em>A. platensis</em> YZ. <em>A. platensis</em> YZ genome is a single, circular chromosome of 6.62 Mb in size. Phylogenetic and comparative genomic analyses revealed that <em>A. platensis</em> YZ was more closely related to <em>A. platensis</em> NIES-39 than <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. Broad gene gains were identified between <em>A. platensis</em> YZ and three other <em>Arthrospira</em> speices, some of which have been previously demonstrated that can be laterally transferred among different species, such as restriction-modification systems-coding genes. Moreover, unprecedented extensive chromosomal rearrangements among different strains were observed. The chromosomal rearrangements, particularly the chromosomal inversions, were analysed and estimated to be closely related to palindromes that involved long inverted repeat sequences and the extensively distributed type IIR restriction enzyme in the <em>Arthrospira</em> genome. In addition, species from genus <em>Arthrospira</em> unanimously contained the highest rate of repetitive sequence compared with the other species of order <em>Oscillatoriales</em> , suggested that sequence duplication significantly contributed to <em>Arthrospira</em> genome phylogeny. These results provided in-depth views into the genomic phylogeny and structural variation of <em>A. platensis</em> , as well as provide a valuable resource for functional genomics studies. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;lateral gene transfer&quot;">lateral gene transfer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosomal rearrangement&quot;">chromosomal rearrangement</a></div>                            <h2 scrollto-destination=87200334 id="87200334" class="section-title" >1. Introduction</h2>
<p> Spirulina is a multi-cellular and filamentous blue-green microalga that has gained considerable popularity in the health food industry and increasingly as a protein and vitamin supplement. Spirulina taxonomically belongs to two separate genera, namely <em>Spirulina</em> and <em>Arthrospira</em> , and consists of about 15 species. <a reveal-id="dsw023-B1" data-open="dsw023-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>Arthrospira</em> is the most common and widely available spirulina that has been extensively investigated in various health-related studies. <a reveal-id="dsw023-B1" data-open="dsw023-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> It is composed of closely related, genetically and physiologically distinct lineages whose differences enable them to adapt to specific ecological niches. <em>Arthrospira</em><em>platensis</em> from Lake Chenghai is a well-known representative in China, and many previous studies associated with stress tolerance and gene functional characterization were regarding to this cyanobacterial strain. <a reveal-id="dsw023-B2 dsw023-B3 dsw023-B4 dsw023-B5" data-open="dsw023-B2 dsw023-B3 dsw023-B4 dsw023-B5" class="link link-ref link-reveal xref-bibr"><sup>2–5</sup></a><sup>,</sup><em>Arthrospira</em> also contains high amount of proteins, polyunsaturated fatty acids, vitamins, minerals, and photosynthetic pigments. Despite its highly useful applications, very little is known about the phylogeny, physiological and genetic system in <em>A</em> . <em>platensis</em> . Unlike other bacteria, the genome of microorganisms belonging to phylum <em>Cyanobacterium</em> generally possesses a large amount of dispersed repetitive sequences, particularly <em>Arthrospira</em> , thereby causing difficulty in the complete assembly of its genome. <a reveal-id="dsw023-B6" data-open="dsw023-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In the past 6 years, several genomes from genus <em>Arthrospira</em> that belong to geographically distinct lineages have been nearly completely recovered by independent sequencing and assembly into single super-scaffolds, respectively, including <em>A</em> . <em>platensis</em> NIES-39, <em>Arthrospira</em> sp. PCC 8005 and <em>A</em> . <em>platensis</em> C1. <a reveal-id="dsw023-B6 dsw023-B7 dsw023-B8" data-open="dsw023-B6 dsw023-B7 dsw023-B8" class="link link-ref link-reveal xref-bibr"><sup>6–8</sup></a> Previous studies have generated on its genomic constitution, annotation, classification of respective isolates, as well as provided the first opportunity to reconstruct its phylogenetic relationship with other different strains. Nevertheless, there were no in-depth comparative genomics analyses revealing the molecular implications associated with genetic diversity affecting the evolutionary origin, and how these genetic differences generated among distinct lineages. In addition, considering the ubiquity of dispersed repeated sequences and its emerging role in genome evolution, we can hardly be said to comprehend <em>Arthrospira</em> genome without giving an account of what these repetitive sequences are, what they do and how they arise. Present study reports the nearly complete genome sequence of one cultivated strain, namely, <em>A</em> . <em>platensis</em> YZ. We then reconstructed the genomic evolution events involving this strain and compared the gene and genomic structure with those of closely related species to assess the phylogenetic relationship, genetic diversity and genomic structure variation. </p>                            <h2 scrollto-destination=87200336 id="87200336" class="section-title" >2. Materials and methods</h2>
<p><em>Arthrospira</em><em>platensis</em> YZ was initially isolated from Chenghai Lake in Yunnan Province, China. The cyanobacterial strain was cultured under 0.02 M NaCl Zarrouk medium at 30 °C, light intensity of 8 kilolux and 75% humidity. After total DNA was extracted, two genomic shotgun libraries with 1.5- and 4.5-kb insert sizes were constructed described as previously. <a reveal-id="dsw023-B9" data-open="dsw023-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> A fosmid (pCC1FOS fosmid vector) library was constructed by using the CopyControl Fosmid library production kit (Epicentre), following the manufacturer’s recommendations. In addition, a 10-kb Illumina mate-pair library was also constructed as described elsewhere. <a reveal-id="dsw023-B10" data-open="dsw023-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> All libraries were sequenced from both ends. Reads that were derived from the two small-insert sized libraries were initially mixed and assembled by using the Phrap software. Consed was used in the finishing process. <a reveal-id="dsw023-B11" data-open="dsw023-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Initially, the primary assembly generated a total of 2,294 contigs, many of which were determined to be mis-assembled. Then, the primary assembly was manually checked via assessment of the paired-end reads distances of the cloned libraries. The mis-assembled regions were disassembled, and local blocks were established for re-assembly. The Illumina and fosmid reads were mapped onto the assembly to scaffold the contigs. The contiguous relationships between all the contigs at non-repetitive regions were anchored by the Illumina mate-pair or fosmid paired-end reads. Gaps were closed by primer-walking on the PCR products from neighbour extremity of the linked contigs or corresponding fosmid clones. The Illumina reads were also employed to confirm and correct those with low-quality (but not repeated sequence) assembly regions by using the Genome Analysis Toolkit. <a reveal-id="dsw023-B12" data-open="dsw023-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p> We used the Glimmer software to predict protein-coding genes with potential open reading frames of &gt;150 bp in size. <a reveal-id="dsw023-B13" data-open="dsw023-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> RNAmmer and tRNAscan-SE were utilized to identify rRNA and tRNA genes, respectively. <a reveal-id="dsw023-B14" data-open="dsw023-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw023-B15" data-open="dsw023-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Gview was used to construct basic genomic features. <a reveal-id="dsw023-B16" data-open="dsw023-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> BLASTX was used to annotate predicted protein-coding genes against the public protein database with an <em>e</em> -value threshold of 1e-5. Genome-wide identification of restriction-modification (RM) systems were conducted by using BLASTP searching against REbase with a &gt;50% amino acid identity and &gt;50% query coverage after all ORFs were theoretically translated. <a reveal-id="dsw023-B17" data-open="dsw023-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Comparative analysis of RM enzymes between <em>Arthrospira</em> strains was performed by using BLASTP with an e-value threshold of 1e-20. We selected 31 partial or complete genomes of determined species from <em>Oscillatoriales</em> for phylogenetic analysis. A species tree was generated by concatenation of 27 of 31 conserved proteins as previously reported, <a reveal-id="dsw023-B18" data-open="dsw023-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> including <em>dnaG</em> , <em>frr</em> , <em>nusA</em> , <em>pgk</em> , <em>pyrG</em> , <em>rplA</em> , <em>rplB</em> , <em>rplC</em> , <em>rplD</em> , <em>rplE</em> , <em>rplF</em> , <em>rplK</em> , <em>rplL</em> , <em>rplM</em> , <em>rplN</em> , <em>rplP</em> , <em>rplS</em> , <em>rpoB</em> , <em>rpsB</em> , <em>rpsC</em> , <em>rpsE</em> , <em>rpsI</em> , <em>rpsK</em> , <em>rpsM</em> , <em>rpsS</em> , <em>smpB</em> , and <em>tsf</em> , because some species did not harbour all the genes due to various reasons. <em>Gloeobacter violaceus</em> PCC 7421 from order <em>Gloeobacterales</em> was used as outgroup for rooting the tree. Maximum-likelihood phylogenetic trees were generated by using PhyML 3.0 with 1,000 bootstrap replications. <a reveal-id="dsw023-B19" data-open="dsw023-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p> The protein-coding gene derived from four <em>Arthrospira</em> genomes was predicted using the same criteria as earlier described. Gview was used to construct general features that were subsequently employed in comparative analysis. <em>In silico</em> hybridization was performed using the BLASTP program, with an <em>e</em> -value threshold of 1e-5. Using <em>A. platensis</em> YZ as reference, gene order in the other three species was established accordingly, followed by the identification of potential laterally transferred genes (LTGs) by using a custom-derived script combined with a manual check. Insertion sequences were predicted by using IS finder. <a reveal-id="dsw023-B20" data-open="dsw023-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Assessment of global genomic co-linearity relationships among species was performed by using the mauve software. <a reveal-id="dsw023-B21" data-open="dsw023-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> BLASTN was used to identify direct and inverted repeats (IRs) as well as palindromes. Given that all the theoretically predicted genes were protein-coding genes, genome-wide clustering of all genes was performed. Only those species from <em>Oscillatoriales</em> with relatively intact genome sequence (genome scaffolds of &lt; 10) were selected for whole genome clustering analysis, including <em>Lyngbya majuscula</em> 3L, <em>Geitlerinema</em> sp. PCC 7105, <em>Leptolyngbya</em> sp. PCC 7376, <em>Leptolyngbya</em> sp. PCC 7375, <em>Oscillatoria nigro-viridis</em> PCC 7112, <em>Pseudanabaena</em> sp. PCC 6802, <em>Oscillatoria</em> sp. PCC 10802, <em>Microcoleus</em> sp. PCC 7113, <em>Crinalium epipsammum</em> PCC 9333, <em>Oscillatoria acuminata</em> PCC 6304, <em>Leptolyngbya boryanar</em> PCC 6306, <em>Spirulina major</em> PCC 6313, <em>Oscillatoriales cyanobacterium</em> JSC-12, <em>Spirulina subsalsa</em> PCC 9445, <em>Pseudanabaena</em> sp. PCC 7367, <em>Geitlerinema</em> sp. PCC 7407, and four strains from genus <em>Arthrospira</em> , because many genes were doomed to be undetermined in the incomplete genome assembly. All genome sequences were obtained from Integrated Microbial Genomes at DOE Joint Genome Institute or NCBI. Cluster Database at High Identity with Tolerance (CD-HIT) was used to perform gene clustering analysis with 50% alignment coverage for both sequences (-aL 0.5, -aS 0.5) at nine resolutions. <a reveal-id="dsw023-B22" data-open="dsw023-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> C/W ratio was calculated as the number of total clusters divided by the number of total genes. R/T ratio represented the number of redundant gene clusters (the clusters which comprised at least two genes) divided by the total number of clusters. Smaller C/W ratios indicated a higher level of gene redundancy, whereas larger R/T ratios suggested broader gene redundancy spectra. Tandem repeats (TRs) were identified by using TR Finder with the following parameters (2 7 7 80 10 100 2000 -f -d -m). <a reveal-id="dsw023-B23" data-open="dsw023-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Clustering of repeat motifs was conducted by CD-HIT with both 100% identity and 100% query, and hit alignment coverage. Clustered regularly interspaced short palindromic repeat (CRISPR) Finder was used to identify CRISPR arrays. <a reveal-id="dsw023-B24" data-open="dsw023-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The complete genome sequence of <em>A. platensis</em> YZ has been deposited to DDBJ/EMBL/GenBank under accession CP013008. </p>                            <h2 scrollto-destination=87200340 id="87200340" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=87200341 id="87200341" class="section-title" >3.1. General features of <em>A. platensis</em> YZ genome</h3>
<p> We selected one single cultivated cyanobacterial strain, which we designated as <em>A</em> . <em>platensis</em> YZ, for whole genome sequencing. Genomic DNA was sequenced using a whole-genome shotgun sequencing strategy, which was conducted by using a ‘hybrid’ approach that combined Sanger and Illumina sequencing technologies. Four different libraries, including 1.5-, 4.5-, 40-kb insert-sizes clone libraries, and an Illumina mate-pair library with a 10-kb insert-size were constructed and sequenced. Over 130,000 high-quality Sanger reads (∼8.9-fold genome coverage) and 10,434,858 Illumina reads (∼77.2-fold genome coverage) were obtained. We assembled the <em>A. platensis</em> genome into one super-scaffold with terminal overlaps. Taken together, <em>A. platensis</em> has a single, circular chromosome of 6.62 Mb in size, with an average GC content of 44.2% ( <a reveal-id="dsw023-F1" data-open="dsw023-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). The current version of the <em>A. platensis</em> YZ genome contains 10 undetermined regions with a totally estimated gap size of 96 kb, and the remaining gaps apparently consist of TRs and <em>Arthrospira</em> -specific repeats. The genome size of <em>A. platensis</em> YZ is a little smaller than the previously sequenced <em>A. platensis</em> NIES-39 (6.79 Mb), <a reveal-id="dsw023-B8" data-open="dsw023-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> but larger than <em>Arthrospira</em> sp. PCC 8005 (6.23 Mb) <a reveal-id="dsw023-B7" data-open="dsw023-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and <em>A. platensis</em> C1 (6.09 Mb). <a reveal-id="dsw023-B6" data-open="dsw023-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> GC skew analysis to anchor potential origin and terminator for chromosome replication did not show obvious skew shift. This is different from those bacteria whose chromosomal replication origin and terminator could be clearly identified, <a reveal-id="dsw023-B9" data-open="dsw023-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw023-B25" data-open="dsw023-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> but is similar to other cyanobacterial genomes, suggesting the occurrence of potential chromosomal rearrangements, sequence duplications and lateral gene transfer (LGT) events, <a reveal-id="dsw023-B26" data-open="dsw023-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> because the bias of nucleotide compositions between leading and lagging strands were disrupted. In addition, mapping Illumina short reads back to the genome resulted in 99.6% of the total reads uniquely aligned to the assembly and 98.4% paired-end reads represented correct orientation and distance, which was indicative of the high credibility of the assembled genome. </p>                        <a id="87200343" scrollto-destination="87200343"></a>
<div id="dsw023-F1" data-id="dsw023-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f1p.png?Expires=2147483647&Signature=rFlBLchqzmM8K6Ld-Cu7qYdPX0fDAv-lD3A0gyVj44TDz3xkslQhjZc~hPwa9kAQaOpCdtL0607WxNBg2RgUosD5eeq9D-4-ya6N-lqBjRRbzUMzeyTrdhCI0WwwJ4RgsnJUGa8ur8Pf0piFSRyNpzrgRAxDIFwrwfl0Eyv0EoXtFlQ02ROj6Qcr4ckpKb-sg3wDVdn7w6NONgCx~W5EpiBzEIeuWYS96S0RlXsqXvsPUU6eEU6l4v1ANDIi-JibRwb~RGE9yjeXKtmeoVyGC9aEACuU9NNjXv8E6bshvHS6mgV1SEhytT1i8w4yeHK7bell96y2hVK6M2B0iUJWaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Circular representation of the A. platensis YZ genome. From outside to inside: circle 1, physical map scaled to 100 kb; Circles 2 and 3, coding sequences transcribed in the clockwise and counterclockwise directions; Circles 4 and 5 represent GC content and GC skew, respectively. Gaps were labelled with G1-G10. " path-from-xml="dsw023f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200343/dsw023f1p.png" path-from-xml="dsw023f1p.png" target="_blank">View large</a><a section="87200343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f1p.png?Expires=2147483647&Signature=r10IfOMEvzU0YGl3o0xC8G~AHhWZaPA27IP~4TwmRnmGXVv2MW00KGC-M0XmBBTPRFbkSnYtY3SqjMuomGH9oB1jqo9~Zfe1HEQpeM8id580~n8gpeMtyJJFeYvSG~HtSHnxsbKAuFH3UiV407Wud~Lg~RSpxPdMu6lTASYKZKEKck-Bp165RPoB9o6XpxTbUBJlqASfwdm-b8DdSwcb7aL8Jg9piRqQvvts-zdBfo4ycuan1Wqu9ukcUeiH~E-8X00VkaACSpXN0lpbnJCy3XRHWfbMPlwkSA~9gSfwqZRREDwmc6haJ3qyjwJdsetGTJ7PQgx~vs1K-irnmb6sNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200343&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Circular representation of the <em>A. platensis</em> YZ genome. From outside to inside: circle 1, physical map scaled to 100 kb; Circles 2 and 3, coding sequences transcribed in the clockwise and counterclockwise directions; Circles 4 and 5 represent GC content and GC skew, respectively. Gaps were labelled with G1-G10. </p></div></div><div content-id="dsw023-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f1p.png?Expires=2147483647&Signature=rFlBLchqzmM8K6Ld-Cu7qYdPX0fDAv-lD3A0gyVj44TDz3xkslQhjZc~hPwa9kAQaOpCdtL0607WxNBg2RgUosD5eeq9D-4-ya6N-lqBjRRbzUMzeyTrdhCI0WwwJ4RgsnJUGa8ur8Pf0piFSRyNpzrgRAxDIFwrwfl0Eyv0EoXtFlQ02ROj6Qcr4ckpKb-sg3wDVdn7w6NONgCx~W5EpiBzEIeuWYS96S0RlXsqXvsPUU6eEU6l4v1ANDIi-JibRwb~RGE9yjeXKtmeoVyGC9aEACuU9NNjXv8E6bshvHS6mgV1SEhytT1i8w4yeHK7bell96y2hVK6M2B0iUJWaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Circular representation of the A. platensis YZ genome. From outside to inside: circle 1, physical map scaled to 100 kb; Circles 2 and 3, coding sequences transcribed in the clockwise and counterclockwise directions; Circles 4 and 5 represent GC content and GC skew, respectively. Gaps were labelled with G1-G10. " path-from-xml="dsw023f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200343/dsw023f1p.png" path-from-xml="dsw023f1p.png" target="_blank">View large</a><a section="87200343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f1p.png?Expires=2147483647&Signature=r10IfOMEvzU0YGl3o0xC8G~AHhWZaPA27IP~4TwmRnmGXVv2MW00KGC-M0XmBBTPRFbkSnYtY3SqjMuomGH9oB1jqo9~Zfe1HEQpeM8id580~n8gpeMtyJJFeYvSG~HtSHnxsbKAuFH3UiV407Wud~Lg~RSpxPdMu6lTASYKZKEKck-Bp165RPoB9o6XpxTbUBJlqASfwdm-b8DdSwcb7aL8Jg9piRqQvvts-zdBfo4ycuan1Wqu9ukcUeiH~E-8X00VkaACSpXN0lpbnJCy3XRHWfbMPlwkSA~9gSfwqZRREDwmc6haJ3qyjwJdsetGTJ7PQgx~vs1K-irnmb6sNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200343&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Circular representation of the <em>A. platensis</em> YZ genome. From outside to inside: circle 1, physical map scaled to 100 kb; Circles 2 and 3, coding sequences transcribed in the clockwise and counterclockwise directions; Circles 4 and 5 represent GC content and GC skew, respectively. Gaps were labelled with G1-G10. </p></div></div><p> The <em>A. platensis</em> YZ genome was predicted to harbour 6,784 protein-coding genes with average gene length of 795 bp. A total of 3,149 and 6,711 protein-coding genes can be annotated by known function or hypothetical protein in the UniProt/Swiss-Prot and non-redundant protein databases, respectively. Cluster of Orthologous Groups of proteins database annotation showed that a number of genes participated in its replication, recombination, and repair ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span> ). Moreover, the genome sequence was also predicted to contain two sets of rRNA genes and 39 tRNA genes that were predicted to translate 19 types of amino acids, except for lysine. The number of rRNA gene clusters in the <em>A. platensis</em> YZ genome is similar to that observed in <em>A. platensis</em> NIES-39, <em>Arthrospira</em> sp. PCC 8005, and <em>A. platensis</em> C1, whereas the number of tRNA genes differed from those of cynaobacteria (40, 42, and 39 tRNA, respectively). </p>                            <h3 scrollto-destination=87200345 id="87200345" class="section-title" >3.2. Phylogenetic and comparative genomics analyses of <em>Arthrospira</em></h3>
<p> Because <em>Arthrospira</em> belongs to order <em>Oscillatoriales</em> , a species tree was constructed for phylogenomic analysis by concatenating 27 conserved proteins coexisted in the 31 <em>Oscillatoriales</em> species ( <a reveal-id="dsw023-F2" data-open="dsw023-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). As expected, <em>A. platensis</em> YZ was clustered together with the other members of genus <em>Arthrospira</em> , which is highly congruent with the findings of previous study on whole cyanobacteria phylum phylogeny. <a reveal-id="dsw023-B27" data-open="dsw023-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Subclade D of the tree showed that <em>A. platensis</em> YZ was phylogenetically closer to <em>A. platensis</em> str. Paraca and <em>A. platensis</em> NIES-39 than <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1, which was indicative of intimate intragenus relationship among <em>A. platensis</em> YZ, <em>A. platensis</em> str. Paraca, and <em>A. platensis</em> NIES-39. </p>                        <a id="87200347" scrollto-destination="87200347"></a>
<div id="dsw023-F2" data-id="dsw023-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f2p.png?Expires=2147483647&Signature=vIpBuGSRg~C4Xh6IBDMzGRJrYTsaJph7Yf6hxSxFfdfTxQgPF86MnbQpWslcQcBCY0y9O1G-DPvUGUH4ek19kpuy2SKQAy3XqKZsInR7WMFTi4ZHge~wFgX4FnD7QbQTfO7wjRGwO6~nr5Z7Cz84vCZVUyQU~~KuQB7YjfXsZgZ-38CbDvyZP8DIiVH1G81GfBKbMe2OhRQ3o4zTU1oRk3nMK9Qx6lb-mV0Dr80mQjYMu1AWqFnEHyzLHTF3lUyrkpUfOeqtFoCP~YtahVIXHo3Pezb3ZeUshpK-o3wp3jz7ssjPeiOFCt2CXKMSgPTJLngz33IJntgYmCcJeNpleQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" A species tree from Oscillatoriales. Phylogenetic reconstruction of members of Oscillatoriales using the maximum-likelihood method was performed in the present study. Taxa are colour-coded according to the same genus. Nodes supported with a bootstrap of &gt; 70% are indicated by black dots. Phylogenetic subclades are grouped into seven major subclades (A–G)." path-from-xml="dsw023f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200347/dsw023f2p.png" path-from-xml="dsw023f2p.png" target="_blank">View large</a><a section="87200347" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f2p.png?Expires=2147483647&Signature=1YKRCUqHi9rJ3KX9gs-VI~t3HY0FJm2g-lfXdYyEewPFTpliZlHx2FpjdDaeL3isgIJrcYnDbs7MqUu~BhtViZVmLcsAZkz8Ksa0QFpkttq7TEDhI9G~Sn8MQmGmsIIeZx1yTC1U5~ztBTrG9Ed~uLrOfayWNTqP7DsDSVgKNtt3PzzqdBdfxPuj83gkUR6k9nJK5U0N8eVJqNnaA1jFidIAmE-PQUK6wWXZGrvAbG0eVYdWdYLn0n6cz1maIMFtKJVeIN4mA62CQI~~cDZbb~bkk1Mg2g1x4fSNejCQMO8A-Ky4IeBuIjqBlVVvHHAzOmhhgGSRbDsWEvJTNlirTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200347&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> A species tree from <em>Oscillatoriales.</em> Phylogenetic reconstruction of members of <em>Oscillatoriales</em> using the maximum-likelihood method was performed in the present study. Taxa are colour-coded according to the same genus. Nodes supported with a bootstrap of &gt; 70% are indicated by black dots. Phylogenetic subclades are grouped into seven major subclades <strong>(A–G).</strong></p></div></div><div content-id="dsw023-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f2p.png?Expires=2147483647&Signature=vIpBuGSRg~C4Xh6IBDMzGRJrYTsaJph7Yf6hxSxFfdfTxQgPF86MnbQpWslcQcBCY0y9O1G-DPvUGUH4ek19kpuy2SKQAy3XqKZsInR7WMFTi4ZHge~wFgX4FnD7QbQTfO7wjRGwO6~nr5Z7Cz84vCZVUyQU~~KuQB7YjfXsZgZ-38CbDvyZP8DIiVH1G81GfBKbMe2OhRQ3o4zTU1oRk3nMK9Qx6lb-mV0Dr80mQjYMu1AWqFnEHyzLHTF3lUyrkpUfOeqtFoCP~YtahVIXHo3Pezb3ZeUshpK-o3wp3jz7ssjPeiOFCt2CXKMSgPTJLngz33IJntgYmCcJeNpleQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" A species tree from Oscillatoriales. Phylogenetic reconstruction of members of Oscillatoriales using the maximum-likelihood method was performed in the present study. Taxa are colour-coded according to the same genus. Nodes supported with a bootstrap of &gt; 70% are indicated by black dots. Phylogenetic subclades are grouped into seven major subclades (A–G)." path-from-xml="dsw023f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200347/dsw023f2p.png" path-from-xml="dsw023f2p.png" target="_blank">View large</a><a section="87200347" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f2p.png?Expires=2147483647&Signature=1YKRCUqHi9rJ3KX9gs-VI~t3HY0FJm2g-lfXdYyEewPFTpliZlHx2FpjdDaeL3isgIJrcYnDbs7MqUu~BhtViZVmLcsAZkz8Ksa0QFpkttq7TEDhI9G~Sn8MQmGmsIIeZx1yTC1U5~ztBTrG9Ed~uLrOfayWNTqP7DsDSVgKNtt3PzzqdBdfxPuj83gkUR6k9nJK5U0N8eVJqNnaA1jFidIAmE-PQUK6wWXZGrvAbG0eVYdWdYLn0n6cz1maIMFtKJVeIN4mA62CQI~~cDZbb~bkk1Mg2g1x4fSNejCQMO8A-Ky4IeBuIjqBlVVvHHAzOmhhgGSRbDsWEvJTNlirTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200347&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> A species tree from <em>Oscillatoriales.</em> Phylogenetic reconstruction of members of <em>Oscillatoriales</em> using the maximum-likelihood method was performed in the present study. Taxa are colour-coded according to the same genus. Nodes supported with a bootstrap of &gt; 70% are indicated by black dots. Phylogenetic subclades are grouped into seven major subclades <strong>(A–G).</strong></p></div></div><p> Gene gains and losses have been considered as one of the most significant contributors to functional changes. <a reveal-id="dsw023-B28" data-open="dsw023-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> To further characterize the differences among several <em>A. platensis</em> genomes, we performed comparative genomics analysis of <em>A. platensis</em> YZ with three other species, including <em>A. platensis</em> NIES-39, <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1, whereas the complete genome of <em>A. platensis</em> str. Paraca is currently unavailable. <a reveal-id="dsw023-B29" data-open="dsw023-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Extensive sequence gains were identified when using <em>A. platensis</em> YZ as backbone during its comparative analysis to the other three species. At least 20 apparently unique regions, excluding the undetermined sequence, were detected in the <em>A. platensis</em> YZ genome ( <a reveal-id="dsw023-F3" data-open="dsw023-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> ). More particular regions in the <em>A. platensis</em> NIES-39 genome were observed, which might be explained by its larger genome ( <a reveal-id="dsw023-F3" data-open="dsw023-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> ). However, the unique regions in <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1 were markedly reduced relative to that in <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39. Furthermore, <em>in silico</em> hybridization identified 100 <em>A. platensis</em> YZ-specific genes, of which 82 could not be annotated by the known proteins. </p>                        <a id="87200349" scrollto-destination="87200349"></a>
<div id="dsw023-F3" data-id="dsw023-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f3p.png?Expires=2147483647&Signature=O1LLExNF4B-~ygKcv9EwL8nZDBPXOpGlN71ONfySyoda~MjSzzRww3N6vxCANgIgO63MbvXufIc3AX-~iatFlH~vgmhxPzyx20OSJiIdwDj4bLgB9p9TwoMdkQD8PQOiy8l4qOTY-5usu2uis9Hof8s2~wxKh-DwVlpst5RL3fnM2TYFnIdiqpqQvx4MLUJVsMZWYo94LPbhJoU14uMEwqQtYNvhO8fJSrWHbwHkK3UuXjRWCX6iM1rkQdt6tVNV-ZAtnAtf51OkGOxYbUuUNorZMIYrkJH9gcwb4xQp~VhVFjljpPG5Eerxv28fbKPopbse6~jjI47LG~BGEZPgNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparative genomic analysis of four complete sequenced Arthrospira speices. A. platensis YZ (A) and A. platensis NIES-39 (B) were used as backbone (reference or hit) for the comparison to three query genomes, respectively. The genomes scaled to 100 kb. Counting from the outside toward the centre: red colour (slot 1) indicated that the nucleotide sequences were only detected in the reference genomes and undetermined gaps of the reference where query genomes were not mapped by ‘Ns’. Black arrow and R1-R5 in (A) indicate specific or unique regions in A. platensis YZ genome, and the regions R1-R5 were selected for analyses of gene gains in A . platensis YZ relative to other three genomes. G1-G10 in (A) and G1-G18 in (B) indicate 10 and 18 undetermined gaps in A . platensis YZ and A . platensis NIES-39 genome, respectively. Slots 2-5 showed that the corresponding query regions had higher sequence similarity (≥70%) with the reference sequence. Empty regions on the query slots indicated parts without similar hits between the backbone sequence and the query sequences, and those undetermined bases in the reference genomes. The last two circles respectively indicate the GC content and GC skew of the reference genomes. The Arthrospira sp. PCC 8005 was abbreviated as A . sp. PCC 8005 in this and the following figures. " path-from-xml="dsw023f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200349/dsw023f3p.png" path-from-xml="dsw023f3p.png" target="_blank">View large</a><a section="87200349" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f3p.png?Expires=2147483647&Signature=BtxGwL2thCCAXBNDITqdkPYIlN9s~qNdY0zmjMKal2BXiEq1u8bM0Ip66W56v8LW602c2j28XPwnS-4h7Si6A7tQYC9BkXOdsb-RblqN7OstoxFppuiZaO2W9~rOBVXgi7WSGBcbjnm9mPdK0fCaYFcG-iReZRg7EW5sPLnlgu6AK~T3mQ~NdpPdkg7029Q1lG5jkZGN-MoI1jsRNOlaMR5YWRVV57H3AoY~JnTDKX9CJvjX8H6L1FqckgdQezNI1GwYU6o91kwwm44cCBvhDQn0zg23ritr-Clm1m1v7YJ1DOSPunEFizfN4ebfn~vGhBGzfUUvdyBThOrCBzAhNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200349&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparative genomic analysis of four complete sequenced <em>Arthrospira</em> speices. <em>A. platensis</em> YZ <strong>(A)</strong> and <em>A. platensis</em> NIES-39 <strong>(B)</strong> were used as backbone (reference or hit) for the comparison to three query genomes, respectively. The genomes scaled to 100 kb. Counting from the outside toward the centre: red colour (slot 1) indicated that the nucleotide sequences were only detected in the reference genomes and undetermined gaps of the reference where query genomes were not mapped by ‘Ns’. Black arrow and R1-R5 in (A) indicate specific or unique regions in <em>A. platensis</em> YZ genome, and the regions R1-R5 were selected for analyses of gene gains in <em>A</em> . <em>platensis</em> YZ relative to other three genomes. G1-G10 in (A) and G1-G18 in (B) indicate 10 and 18 undetermined gaps in <em>A</em> . <em>platensis</em> YZ and <em>A</em> . <em>platensis</em> NIES-39 genome, respectively. Slots 2-5 showed that the corresponding query regions had higher sequence similarity (≥70%) with the reference sequence. Empty regions on the query slots indicated parts without similar hits between the backbone sequence and the query sequences, and those undetermined bases in the reference genomes. The last two circles respectively indicate the GC content and GC skew of the reference genomes. The <em>Arthrospira</em> sp. PCC 8005 was abbreviated as <em>A</em> . sp. PCC 8005 in this and the following figures. </p></div></div><div content-id="dsw023-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f3p.png?Expires=2147483647&Signature=O1LLExNF4B-~ygKcv9EwL8nZDBPXOpGlN71ONfySyoda~MjSzzRww3N6vxCANgIgO63MbvXufIc3AX-~iatFlH~vgmhxPzyx20OSJiIdwDj4bLgB9p9TwoMdkQD8PQOiy8l4qOTY-5usu2uis9Hof8s2~wxKh-DwVlpst5RL3fnM2TYFnIdiqpqQvx4MLUJVsMZWYo94LPbhJoU14uMEwqQtYNvhO8fJSrWHbwHkK3UuXjRWCX6iM1rkQdt6tVNV-ZAtnAtf51OkGOxYbUuUNorZMIYrkJH9gcwb4xQp~VhVFjljpPG5Eerxv28fbKPopbse6~jjI47LG~BGEZPgNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparative genomic analysis of four complete sequenced Arthrospira speices. A. platensis YZ (A) and A. platensis NIES-39 (B) were used as backbone (reference or hit) for the comparison to three query genomes, respectively. The genomes scaled to 100 kb. Counting from the outside toward the centre: red colour (slot 1) indicated that the nucleotide sequences were only detected in the reference genomes and undetermined gaps of the reference where query genomes were not mapped by ‘Ns’. Black arrow and R1-R5 in (A) indicate specific or unique regions in A. platensis YZ genome, and the regions R1-R5 were selected for analyses of gene gains in A . platensis YZ relative to other three genomes. G1-G10 in (A) and G1-G18 in (B) indicate 10 and 18 undetermined gaps in A . platensis YZ and A . platensis NIES-39 genome, respectively. Slots 2-5 showed that the corresponding query regions had higher sequence similarity (≥70%) with the reference sequence. Empty regions on the query slots indicated parts without similar hits between the backbone sequence and the query sequences, and those undetermined bases in the reference genomes. The last two circles respectively indicate the GC content and GC skew of the reference genomes. The Arthrospira sp. PCC 8005 was abbreviated as A . sp. PCC 8005 in this and the following figures. " path-from-xml="dsw023f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200349/dsw023f3p.png" path-from-xml="dsw023f3p.png" target="_blank">View large</a><a section="87200349" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f3p.png?Expires=2147483647&Signature=BtxGwL2thCCAXBNDITqdkPYIlN9s~qNdY0zmjMKal2BXiEq1u8bM0Ip66W56v8LW602c2j28XPwnS-4h7Si6A7tQYC9BkXOdsb-RblqN7OstoxFppuiZaO2W9~rOBVXgi7WSGBcbjnm9mPdK0fCaYFcG-iReZRg7EW5sPLnlgu6AK~T3mQ~NdpPdkg7029Q1lG5jkZGN-MoI1jsRNOlaMR5YWRVV57H3AoY~JnTDKX9CJvjX8H6L1FqckgdQezNI1GwYU6o91kwwm44cCBvhDQn0zg23ritr-Clm1m1v7YJ1DOSPunEFizfN4ebfn~vGhBGzfUUvdyBThOrCBzAhNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200349&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparative genomic analysis of four complete sequenced <em>Arthrospira</em> speices. <em>A. platensis</em> YZ <strong>(A)</strong> and <em>A. platensis</em> NIES-39 <strong>(B)</strong> were used as backbone (reference or hit) for the comparison to three query genomes, respectively. The genomes scaled to 100 kb. Counting from the outside toward the centre: red colour (slot 1) indicated that the nucleotide sequences were only detected in the reference genomes and undetermined gaps of the reference where query genomes were not mapped by ‘Ns’. Black arrow and R1-R5 in (A) indicate specific or unique regions in <em>A. platensis</em> YZ genome, and the regions R1-R5 were selected for analyses of gene gains in <em>A</em> . <em>platensis</em> YZ relative to other three genomes. G1-G10 in (A) and G1-G18 in (B) indicate 10 and 18 undetermined gaps in <em>A</em> . <em>platensis</em> YZ and <em>A</em> . <em>platensis</em> NIES-39 genome, respectively. Slots 2-5 showed that the corresponding query regions had higher sequence similarity (≥70%) with the reference sequence. Empty regions on the query slots indicated parts without similar hits between the backbone sequence and the query sequences, and those undetermined bases in the reference genomes. The last two circles respectively indicate the GC content and GC skew of the reference genomes. The <em>Arthrospira</em> sp. PCC 8005 was abbreviated as <em>A</em> . sp. PCC 8005 in this and the following figures. </p></div></div><p> The most powerful approach for identification of gene gains and losses is by conducting a direct molecular genetic analysis of DNA sequences. <a reveal-id="dsw023-B30" data-open="dsw023-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> We analysed several syntenic blocks with extensive gene gains in <em>A. platensis</em> YZ relative to the other three <em>A</em> . <em>platensis</em> genomes ( <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). These blocks involved addition of unknown functions and well characterized genes, which indicated the occurrence of potential LGT events. Scanning the upstream and downstream sequences surrounding these additional genes in <em>A. platensis</em> YZ did not identify known transposons or insertion elements. Furthermore, comparative analysis showed an additional 5-kb sequence that encoded five genes that were flanked by a pair of 170-bp direct repeats (DRs) with high similar sequence identity (&gt;91%) that were inserted into the <em>A. platensis</em> YZ genome relative to that of the <em>A. platensis</em> NIES-39, as indicated in R1 ( <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). These potential LTGs included two ATPases that belonged to the AAA superfamily and three hypothetical proteins that were extensively distributed in the various <em>Oscillatoriales</em> species. Moreover, we did not identify typical IRs that flanked these genes, suggesting that the acquisition of these genes might have not been mediated by transposable elements, but was possibly mediated by recombinase through recombination where a pair of 170-bp DRs was presented. However, there was no known recombinase encoded by these genes, which in turn suggesting that this accessory region could have been lost from <em>A. platensis</em> NIES-39 by recombination between two DR sequences. Nucleotides composition analysis did not show obviously abnormal discrepancy of GC content between acquired and flanking regions. Potential LGT associated with acquisition of type I RM system was observed (R2). A complete RM system was acquired in <em>A. platensis</em> YZ relative to <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. More complexity of nested DRs’ composition (DR1, DR2, and DR3) surrounding the acquired gene cluster in <em>A. platensis</em> YZ was identified at this area, whereas only DR3 sequence appeared as a single event in the whole genomes of <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S1</a></span> listed the three DRs appeared in the <em>A. platensis</em> YZ. Moreover, these nested DRs were sporadically scattered across the counterpart region in <em>A. platensis</em> NIES-39 but not throughout the genome, and showed an increased frequency of emergence compared with <em>A. platensis</em> YZ, suggestive of the high instability of this region. Interestingly, we also identified a cluster of probably non- <em>Arthrospira</em> -derived genes including methyltransferase (Y8219) encoding gene that are present in <em>A. platensis</em> YZ genome (R3). The Y8219 and the immediate downstream two function-unknown genes in <em>A. platensis</em> YZ shared only 51.0, 51.2, and 24.2% amino acid identities with the orthologues from <em>Microcoleus vaginatus</em> , <em>Arthrospira</em> sp. PCC 8005 and <em>Pseudoalteromonas agarivorans</em> , respectively, suggesting that these genes were more likely derived from distantly related organisms accompanied by extensive mutations, because two of these three genes had no orthologues in recently sequenced <em>Arthrospira</em> genome at all. The genomic structure in this area among four strains is also largely different, which indicated high variability of this region. In addition, potential LGT between closely related organisms was also observed. Three genes from <em>Arthrospira</em> sp. PCC 8005 that were inserted into the <em>A. platensis</em> NIES-39 backbone resembled the local genomic structure of <em>A. platensis</em> YZ (R4, <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). Finally, LGT potentially mediated by mobile genetic element (MGE) was also identified in <em>A. platensis</em> YZ genome relative to that observed in <em>A. platensis</em> NIES-39 (R5, <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). A reverse transcriptase (RTase)-encoding gene was immediately followed by a pair of long terminal repeats (LTRs, &gt;90% identity), which was further globally enwrapped by a pair of long DRs (LDRs) (&gt;85% identity) that leads to the formation of a typical retrotransposon. The retrotransposons consist of two subclasses, LTR and non-LTR retrotransposons, based on the phylogeny of RTase which they encodes. <a reveal-id="dsw023-B31" data-open="dsw023-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Duplication of retrotransposons is a replicative event through transcribing themselves to RNA intermediate and then reversely transcribing back to DNA that has been demonstrated to be directed by RTase which is the key component of the retrotransposons. <a reveal-id="dsw023-B32" data-open="dsw023-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> However, its integration into the <em>A. platensis</em> genome seems to be a recombination event, because it is unreasonable to generate such LDRs by using transposase. </p>                        <a id="87200351" scrollto-destination="87200351"></a>
<div id="dsw023-F4" data-id="dsw023-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f4p.png?Expires=2147483647&Signature=exB0og8PtPo8uD~iFyzS6hv-6GIxASaQR489Y1G4eyfecMrBUUgvm5qGiQCY2gnuxDIgOGYSF4s6Ia6xv6y228RgwFereCUkVe6oBFJIaCm5HFBkjGx9Bbbcnp4n0LlN~hSJB4Iq~wLrK-dv3JAjEgxI3EaFXM45NhJPP6pQv6y41J0GMhKmPMdy6yf0lJFycJlpzmp2wWUETKnpbIh4hLCtx8-0yAhXwqf6r8QZTLL~IbRKbjjrHxhuW3RPzZ2FWvOfwAr86Lcbndkur-sezzrGn56XMq5czh1kBZEF3h3V0rY9A-oIiLJWUnRYZM6mLNQA30N235F2ZVjAyoAHPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Five typical regions involved in potential LGT events in A. platensis YZ compared with three other Arthrospira species. R1-R5 corresponds to the R1-R5 in the Figure 3. Homologous genes are marked with same colour, whereas non-homologous genes are left blank. Gene-coding products were labelled with abbreviated names provided in the UniProt database, whereas unlabelled arrows indicate unannotated or unknown gene. (+) or (−) suggests the block is located on the non-complementary or complementary strand of the corresponding genome. R1: Five additional genes were gained in the A. platensis YZ genome. A pair of 170-bp sequence (DR) with &gt;95% identity flanked the LTGs in A. platensis YZ, whereas only one copy was detected in A. platensis NIES-39, and entirely absent in Arthrospira sp. PCC 8005 and A. platensis C1; R2: DR1 is 87-bp in length. DR2 and DR3 are parts of DR1. The genome of A. platensis NIES-39 in this region is rich in three types of DRs; R3: A 15-bp imperfect IR flanked the LTGs in Arthrospira sp. PCC 8005 compared with that of A. platensis NIES-39. No DR or IR surrounding the LTG at this region was detected in the A. platensis YZ genome; R4: No DR flanking the LTG was detected in A. platensis YZ compared with that of A. platensis NIES-39; R5: IR and DR are 349- and 187-bp in size, respectively. " path-from-xml="dsw023f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200351/dsw023f4p.png" path-from-xml="dsw023f4p.png" target="_blank">View large</a><a section="87200351" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f4p.png?Expires=2147483647&Signature=4lMoQNdIbhlawiRt-DaPFWBjycCCPfUIHVK8nz2-ThMqD4hNZe5DDh9tn1-RvFysmUBaXBQaiqDPO-6f8pCPnW7RsIPwhEDDKRL0O4QbfA61fPCBCAEL~p9UgfdF81oPoaDWndTmktZd9DO3L3YQpFdJslEI1Wz0YvhQXh4Pf7o8C7yfVCIW2VvP8t02LuIiL7dJ-jpGDkxC9HHtZLk9cZ3SRLue9mcGzH7shwxLsG-747FyX9hdUtzPlmZo83V5GcTVcQZwSKOJwvQJ5YeyuK5DqL62EsEl84sk-vjBJ5nabN6rOIoOnuWs88fFVOR1Oi9kFKXf4WLpEOrLOFXsfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200351&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Five typical regions involved in potential LGT events in <em>A. platensis</em> YZ compared with three other <em>Arthrospira</em> species. R1-R5 corresponds to the R1-R5 in the Figure 3. Homologous genes are marked with same colour, whereas non-homologous genes are left blank. Gene-coding products were labelled with abbreviated names provided in the UniProt database, whereas unlabelled arrows indicate unannotated or unknown gene. (+) or (−) suggests the block is located on the non-complementary or complementary strand of the corresponding genome. R1: Five additional genes were gained in the <em>A. platensis</em> YZ genome. A pair of 170-bp sequence (DR) with &gt;95% identity flanked the LTGs in <em>A. platensis</em> YZ, whereas only one copy was detected in <em>A. platensis</em> NIES-39, and entirely absent in <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1; R2: DR1 is 87-bp in length. DR2 and DR3 are parts of DR1. The genome of <em>A. platensis</em> NIES-39 in this region is rich in three types of DRs; R3: A 15-bp imperfect IR flanked the LTGs in <em>Arthrospira</em> sp. PCC 8005 compared with that of <em>A. platensis</em> NIES-39. No DR or IR surrounding the LTG at this region was detected in the <em>A. platensis</em> YZ genome; R4: No DR flanking the LTG was detected in <em>A. platensis</em> YZ compared with that of <em>A. platensis</em> NIES-39; R5: IR and DR are 349- and 187-bp in size, respectively. </p></div></div><div content-id="dsw023-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f4p.png?Expires=2147483647&Signature=exB0og8PtPo8uD~iFyzS6hv-6GIxASaQR489Y1G4eyfecMrBUUgvm5qGiQCY2gnuxDIgOGYSF4s6Ia6xv6y228RgwFereCUkVe6oBFJIaCm5HFBkjGx9Bbbcnp4n0LlN~hSJB4Iq~wLrK-dv3JAjEgxI3EaFXM45NhJPP6pQv6y41J0GMhKmPMdy6yf0lJFycJlpzmp2wWUETKnpbIh4hLCtx8-0yAhXwqf6r8QZTLL~IbRKbjjrHxhuW3RPzZ2FWvOfwAr86Lcbndkur-sezzrGn56XMq5czh1kBZEF3h3V0rY9A-oIiLJWUnRYZM6mLNQA30N235F2ZVjAyoAHPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Five typical regions involved in potential LGT events in A. platensis YZ compared with three other Arthrospira species. R1-R5 corresponds to the R1-R5 in the Figure 3. Homologous genes are marked with same colour, whereas non-homologous genes are left blank. Gene-coding products were labelled with abbreviated names provided in the UniProt database, whereas unlabelled arrows indicate unannotated or unknown gene. (+) or (−) suggests the block is located on the non-complementary or complementary strand of the corresponding genome. R1: Five additional genes were gained in the A. platensis YZ genome. A pair of 170-bp sequence (DR) with &gt;95% identity flanked the LTGs in A. platensis YZ, whereas only one copy was detected in A. platensis NIES-39, and entirely absent in Arthrospira sp. PCC 8005 and A. platensis C1; R2: DR1 is 87-bp in length. DR2 and DR3 are parts of DR1. The genome of A. platensis NIES-39 in this region is rich in three types of DRs; R3: A 15-bp imperfect IR flanked the LTGs in Arthrospira sp. PCC 8005 compared with that of A. platensis NIES-39. No DR or IR surrounding the LTG at this region was detected in the A. platensis YZ genome; R4: No DR flanking the LTG was detected in A. platensis YZ compared with that of A. platensis NIES-39; R5: IR and DR are 349- and 187-bp in size, respectively. " path-from-xml="dsw023f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200351/dsw023f4p.png" path-from-xml="dsw023f4p.png" target="_blank">View large</a><a section="87200351" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f4p.png?Expires=2147483647&Signature=4lMoQNdIbhlawiRt-DaPFWBjycCCPfUIHVK8nz2-ThMqD4hNZe5DDh9tn1-RvFysmUBaXBQaiqDPO-6f8pCPnW7RsIPwhEDDKRL0O4QbfA61fPCBCAEL~p9UgfdF81oPoaDWndTmktZd9DO3L3YQpFdJslEI1Wz0YvhQXh4Pf7o8C7yfVCIW2VvP8t02LuIiL7dJ-jpGDkxC9HHtZLk9cZ3SRLue9mcGzH7shwxLsG-747FyX9hdUtzPlmZo83V5GcTVcQZwSKOJwvQJ5YeyuK5DqL62EsEl84sk-vjBJ5nabN6rOIoOnuWs88fFVOR1Oi9kFKXf4WLpEOrLOFXsfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200351&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Five typical regions involved in potential LGT events in <em>A. platensis</em> YZ compared with three other <em>Arthrospira</em> species. R1-R5 corresponds to the R1-R5 in the Figure 3. Homologous genes are marked with same colour, whereas non-homologous genes are left blank. Gene-coding products were labelled with abbreviated names provided in the UniProt database, whereas unlabelled arrows indicate unannotated or unknown gene. (+) or (−) suggests the block is located on the non-complementary or complementary strand of the corresponding genome. R1: Five additional genes were gained in the <em>A. platensis</em> YZ genome. A pair of 170-bp sequence (DR) with &gt;95% identity flanked the LTGs in <em>A. platensis</em> YZ, whereas only one copy was detected in <em>A. platensis</em> NIES-39, and entirely absent in <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1; R2: DR1 is 87-bp in length. DR2 and DR3 are parts of DR1. The genome of <em>A. platensis</em> NIES-39 in this region is rich in three types of DRs; R3: A 15-bp imperfect IR flanked the LTGs in <em>Arthrospira</em> sp. PCC 8005 compared with that of <em>A. platensis</em> NIES-39. No DR or IR surrounding the LTG at this region was detected in the <em>A. platensis</em> YZ genome; R4: No DR flanking the LTG was detected in <em>A. platensis</em> YZ compared with that of <em>A. platensis</em> NIES-39; R5: IR and DR are 349- and 187-bp in size, respectively. </p></div></div><p> In some eubacteria and archaea, such as <em>Acinetobacter baumannii</em> , <em>Escherichia coli</em> , <em>Enterococcus faecium</em> , <em>Lactococcus lactis</em> , and <em>Mycobacterium smegmatis</em> , transposon-mediated LGT usually generates a pair of DRs varying from several to a dozen base pairs flanking the foreign gene after its integration into the host genome. <a reveal-id="dsw023-B33 dsw023-B34 dsw023-B35 dsw023-B36" data-open="dsw023-B33 dsw023-B34 dsw023-B35 dsw023-B36" class="link link-ref link-reveal xref-bibr"><sup>33–36</sup></a> Interestingly, the size of DRs in the <em>A. platensis</em> was significantly longer than those of other species. For instance, a 170-bp sequence was only observed in the <em>A. platensis</em> NIES-39 genome, where it duplicated only once and thus generated a DR that flanked the acquired genes in the <em>A. platensis</em> YZ (R1, <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). Likewise, distributions of the 87-bp DR and its’ nested copies were strictly restricted to the neighbouring region of the type I RM system gene cluster, but not ubiquitously in the genome (R2, <a reveal-id="dsw023-F4" data-open="dsw023-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). LGT involved in the RM systems has been widely observed, including insert element-, <a reveal-id="dsw023-B37" data-open="dsw023-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> phage- <a reveal-id="dsw023-B38" data-open="dsw023-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> , and integrative element-mediated transfer. <a reveal-id="dsw023-B39" data-open="dsw023-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Previous bacterial pan-genome statistical analysis suggested that solitary RM genes were most likely to be transferred by large MGEs, whereas complete RM systems were more frequently transferred autonomously or in small MGEs. <a reveal-id="dsw023-B40" data-open="dsw023-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> However, there were no MGE involving in or flanking these acquired genes in R1 and R2. Therefore, it could not exclude that the formation of respective genomic architecture in the three other genomes underwent gene losses at the counterpart regions. The locally distributed DRs may indicate that these DRs were closely related to the gene gains or losses in R1 and R2. </p>                            <h3 scrollto-destination=87200353 id="87200353" class="section-title" >3.3. Chromosomal rearrangements in <em>Arthrospira</em></h3>
<p> Comparative analysis of the four <em>Arthrospira</em> strains showed that orthologous genes within the same block often possessed distinct coding orientations, which in turn was suggestive of the presence of chromosomal rearrangements. Comparison of global genomic synteny revealed that <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39 had strong co-linearity, whereas <em>Arthrospira</em> sp. PCC 8005 was highly collinear with <em>A. platensis</em> C1 ( <a reveal-id="dsw023-F5" data-open="dsw023-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> ), which was consistent with the evolutionary relationship among these species. The observed co-linearity divided the four <em>Arthrospira</em> genomes into two groups. There were few structural variations within groups, except for a 350-kb fragment that was detected <em>in situ</em> to be inverted in orientation between <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1 (region A, <a reveal-id="dsw023-F5" data-open="dsw023-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> ). However, extensive chromosomal rearrangements were observed between the two groups, including inversion, translocation, deletion, as well as combinations of these events, suggesting that speciation between groups underwent multiple and extremely complicated structural variations that are very difficult to interpret. We analysed the sequences surrounding the inverted genomic fragment between <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. Result showed that a pair of 11-kb long inverted repeats (LIRs) with &gt;97% global sequence identities flanked and further extended into the inverted fragment, which resulted in an obscure boundary for the chromosomal inversion. Within the LIR, we identified five pairs of palindromes that constituted a series of restriction enzyme recognition sites (RERS). These molecular evidences suggested that the chromosomal inversion was probably initiated from the symmetrically distributed RERS of the internal region of the LIR and mediated by the restriction enzyme as previously proposed. <a reveal-id="dsw023-B41" data-open="dsw023-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsw023-B42" data-open="dsw023-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> This LIR seems not to be occasionally emerged at the edge of this inverted fragment, but was closely related to this region to some extent, because it appeared only three times and twice throughout the genomes of <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1, respectively. The 11-kb LIR encoded a tyrosine recombinase that belonged to the integrase family. <a reveal-id="dsw023-B43" data-open="dsw023-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw023-B44" data-open="dsw023-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> After a sequence replication event, fusion of the 11-kb copy into adjacent loci of genome might have resulted from the integrase-mediated site-specific recombination, which well explains the duplication events. In addition, the predicted protein-coding genes that harboured RERS that probably interacted with the restriction enzyme within the LIR apparently remained intact after the chromosomal inversion, because the sequence identities between LIR flanking the inverted genomic fragment were extremely high. Chromosomal rearrangements that were potentially associated with sequence duplication were also identified between two groups, such as <a reveal-id="dsw023-F5" data-open="dsw023-F5" class="link link-reveal link-table xref-fig">Figure 5</a> regions B and C. We analysed the sequences that flanked potentially invertible genomic fragment (region B) between the two groups, and two RERS were identified within the LIR in <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39, albeit chromosomal inversion at this region was not recurrent among species in the same group. Five and seven copies of the intact 11.6-kb sequence were respectively detected in the genomes of <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ. However, all the coding genes derived from the 11.6-kb LIR could not be annotated with known functions; therefore, its potential involvement in particular molecular function remains elusive. Likewise, a pair of 2-kb LIR flanking the potentially invertible genomic fragment was also identified in the region C. Therefore, generation of LIR, but not LDR, in the neighbouring region of genome via sequence duplication would not only enhance the frequency of occurrence of RERS and paired RERS, but also would not destroy gene order after chromosomal inversion. Taken together, the sequence duplication followed by the formation of LIRs flanking the genomic fragments, might play an important role in the chromosomal inversions in <em>A. platensis</em> . </p>                        <a id="87200355" scrollto-destination="87200355"></a>
<div id="dsw023-F5" data-id="dsw023-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f5p.png?Expires=2147483647&Signature=Q4AkTUfJfOZFZ3OyxQYmvJHXlgpMVLHYInJO3XYmIO5l82BecWlArctNISh5BA0fC6QNjIhu0MhBb6w-8WvfNUu4kNkLaFqC0yVa8aHyRSec3iNk4KIiQ7vlS88VF2l1ytptyPRvGQm4K2QjimazRF6sU~tdMfn6X4AsX1HhYWIY0J6NKK7wNMkBqq46Wa~VNkoW87H3l6IlfjOgGNXeMh~yhrGVsBIilHw1x0izUn5o7NbmYtU3q9j7vy0eoE5S-K8hBOk2KZTRWONl4kqaWbCJ6Td83L35f00~rlv5vb6f64Tbzf1gWu6C7RwEx0smGky~3uVuUQRZrs9J~tRdjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of global genomic synteny of A. platensis YZ to three other species. Corresponding blocks from these four genomes are shown based on sequence similarities. The homologous sequences at corresponding blocks are lined together and marked with the same colour. Three genomic fragment rearrangements are indicated by frames and labelled A–C . L-LIR and R-LIR represent left and right LIRs, respectively. In these regions, the L-LIR and R-LIR within and between groups both shared &gt;95% global sequence similarities. In the region B, R-LIR in A. platensis YZ is a truncated IR that shared &gt; 95% sequence similarity with L-LIR. R1-R5 showed the palindromic sequences that were distributed in the L-LIR and R-LIR, except for those labelled with an asterisk. The palindromes involved in the restriction enzymes dissection are underlined. " path-from-xml="dsw023f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200355/dsw023f5p.png" path-from-xml="dsw023f5p.png" target="_blank">View large</a><a section="87200355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f5p.png?Expires=2147483647&Signature=t53tV-LdVCq47T4~XhW9Qa-wMb2qR8-NINttsTum4gZ0btpG-RCx824u4yYgpiKshDI5zQPv2iQUEm-1g-Hf-W~b~WawsJKN~R1sBfpveAXiXj7KNRcpnXmAe3KuJJIGGVXZSLeI6s6RK2h9Hn6NmwjDqR7Z35hZzZ1rfoeMbbuyxv0dQDfx1DI6dO1m~FJF3MWDhx4aO4J9lRIDbcYNAUfoVWyV2VPvF4E3Kb0RoQ~v3liqSnW3OooCYSE5C7iZNBsTJrXa~~7hSv9UDN4P-nkrweVC5A~DEJkOdipPsJtzbrX6c75437YOgSjsGsKIJeCv9lr2uQfNJUjfAdOBiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200355&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of global genomic synteny of <em>A. platensis</em> YZ to three other species. Corresponding blocks from these four genomes are shown based on sequence similarities. The homologous sequences at corresponding blocks are lined together and marked with the same colour. Three genomic fragment rearrangements are indicated by frames and labelled <strong>A–C</strong> . L-LIR and R-LIR represent left and right LIRs, respectively. In these regions, the L-LIR and R-LIR within and between groups both shared &gt;95% global sequence similarities. In the region B, R-LIR in <em>A. platensis</em> YZ is a truncated IR that shared &gt; 95% sequence similarity with L-LIR. R1-R5 showed the palindromic sequences that were distributed in the L-LIR and R-LIR, except for those labelled with an asterisk. The palindromes involved in the restriction enzymes dissection are underlined. </p></div></div><div content-id="dsw023-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f5p.png?Expires=2147483647&Signature=Q4AkTUfJfOZFZ3OyxQYmvJHXlgpMVLHYInJO3XYmIO5l82BecWlArctNISh5BA0fC6QNjIhu0MhBb6w-8WvfNUu4kNkLaFqC0yVa8aHyRSec3iNk4KIiQ7vlS88VF2l1ytptyPRvGQm4K2QjimazRF6sU~tdMfn6X4AsX1HhYWIY0J6NKK7wNMkBqq46Wa~VNkoW87H3l6IlfjOgGNXeMh~yhrGVsBIilHw1x0izUn5o7NbmYtU3q9j7vy0eoE5S-K8hBOk2KZTRWONl4kqaWbCJ6Td83L35f00~rlv5vb6f64Tbzf1gWu6C7RwEx0smGky~3uVuUQRZrs9J~tRdjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of global genomic synteny of A. platensis YZ to three other species. Corresponding blocks from these four genomes are shown based on sequence similarities. The homologous sequences at corresponding blocks are lined together and marked with the same colour. Three genomic fragment rearrangements are indicated by frames and labelled A–C . L-LIR and R-LIR represent left and right LIRs, respectively. In these regions, the L-LIR and R-LIR within and between groups both shared &gt;95% global sequence similarities. In the region B, R-LIR in A. platensis YZ is a truncated IR that shared &gt; 95% sequence similarity with L-LIR. R1-R5 showed the palindromic sequences that were distributed in the L-LIR and R-LIR, except for those labelled with an asterisk. The palindromes involved in the restriction enzymes dissection are underlined. " path-from-xml="dsw023f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200355/dsw023f5p.png" path-from-xml="dsw023f5p.png" target="_blank">View large</a><a section="87200355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f5p.png?Expires=2147483647&Signature=t53tV-LdVCq47T4~XhW9Qa-wMb2qR8-NINttsTum4gZ0btpG-RCx824u4yYgpiKshDI5zQPv2iQUEm-1g-Hf-W~b~WawsJKN~R1sBfpveAXiXj7KNRcpnXmAe3KuJJIGGVXZSLeI6s6RK2h9Hn6NmwjDqR7Z35hZzZ1rfoeMbbuyxv0dQDfx1DI6dO1m~FJF3MWDhx4aO4J9lRIDbcYNAUfoVWyV2VPvF4E3Kb0RoQ~v3liqSnW3OooCYSE5C7iZNBsTJrXa~~7hSv9UDN4P-nkrweVC5A~DEJkOdipPsJtzbrX6c75437YOgSjsGsKIJeCv9lr2uQfNJUjfAdOBiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200355&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of global genomic synteny of <em>A. platensis</em> YZ to three other species. Corresponding blocks from these four genomes are shown based on sequence similarities. The homologous sequences at corresponding blocks are lined together and marked with the same colour. Three genomic fragment rearrangements are indicated by frames and labelled <strong>A–C</strong> . L-LIR and R-LIR represent left and right LIRs, respectively. In these regions, the L-LIR and R-LIR within and between groups both shared &gt;95% global sequence similarities. In the region B, R-LIR in <em>A. platensis</em> YZ is a truncated IR that shared &gt; 95% sequence similarity with L-LIR. R1-R5 showed the palindromic sequences that were distributed in the L-LIR and R-LIR, except for those labelled with an asterisk. The palindromes involved in the restriction enzymes dissection are underlined. </p></div></div><p> The RM system, an important immune system for bacteria that prevents the uptake of exogenous DNA, is largely variable in terms of copy number and constitution. <a reveal-id="dsw023-B45" data-open="dsw023-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsw023-B46" data-open="dsw023-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> It was also proposed to possess the potential role in chromosomal rearrangements, because they can cut DNA. <a reveal-id="dsw023-B41" data-open="dsw023-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Genome-wide identification of RMs in the <em>Oscillatoriales</em> order revealed that species from <em>Arthrospira</em> harboured the most abundant RM enzymes ( <a reveal-id="dsw023-T1" data-open="dsw023-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <a reveal-id="dsw023-F6" data-open="dsw023-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> ). In the <em>Arthrospira</em> , four strains all contained types I and II RM enzymes, with type IIR enzymes predomination. Besides types I and II RM <em>, Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1 both harboured a type III RM system, while it was absent in <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ. </p>                        <a id="87200357" scrollto-destination="87200357"></a>
<div id="dsw023-F6" data-id="dsw023-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f6p.png?Expires=2147483647&Signature=EmPvpzbSfYnvw8ESZ-3rE-nYyk0K7DdLlOTKMgf3daEf5r9UZ~rBCZ2iR~~NNTWjXh1TwGR8UsYrA2GYzjy0UFn2FoCYEITtt1z2uYoz0Fe2xIt5wkNX~JRDFw~hcjjzWU3B5UeNQPjagxtnAfSrZa-1WZYgu9NC01x8I2R3sKUQS6knKLEOjpt~7Bv9Ep28qB8cObc3Oinb4MMzadXlkhNmU8t0g22iGO5o5R5HwiZNNPPXr5JPZk2ARATJIWfPl1YEBpLzRDiAhBm0gqbcq~ZOtj4NYnfUfGeEuoWD9WExlmTY4U2CMk6QdfJ2lsbJRMwhl1mOTgfnN7d5nhamJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Overview of RM systems distributed in the four Arthrospira genomes. Colour bars above or below the central line from each strain represent genes with two distinct coding orientations in the genome. Black line is used to connect those best hit RM enzyme between different strains. Arrow indicates the unique enzyme compared with the other three strains. " path-from-xml="dsw023f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200357/dsw023f6p.png" path-from-xml="dsw023f6p.png" target="_blank">View large</a><a section="87200357" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f6p.png?Expires=2147483647&Signature=tjIemMJnsl1s5OKvIakpbZGQFgf58xfyRsHgVGrz3ExyYpjhxuD0BMTMIQgHyPtqv0caHPE6iu5JOTaGkcGfK4D7N0t9x6tor9IECgbjEu4SP7BcpJWA~5I-Ybho503GJAch9HB6PMBo0ViZ9IkjiaqTRoRjSMpmqOMfJlpLurApMdBunBn61tkpZWG0xer8WPYFh-gs~Vvrt37fn4GtaybRVLXKvkCsFSm3q90qJMhYg4E52k85LyZD~9oOpWRjg0CJ2iD9WSWFX3GUyDkpt1kBd4qMkcCkZjEGyTME7lgLfV2s6dx1JhHyFFIxgEMRygfr3yFxSFew9E5PFhLRVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200357&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Overview of RM systems distributed in the four <em>Arthrospira</em> genomes. Colour bars above or below the central line from each strain represent genes with two distinct coding orientations in the genome. Black line is used to connect those best hit RM enzyme between different strains. Arrow indicates the unique enzyme compared with the other three strains. </p></div></div><div content-id="dsw023-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f6p.png?Expires=2147483647&Signature=EmPvpzbSfYnvw8ESZ-3rE-nYyk0K7DdLlOTKMgf3daEf5r9UZ~rBCZ2iR~~NNTWjXh1TwGR8UsYrA2GYzjy0UFn2FoCYEITtt1z2uYoz0Fe2xIt5wkNX~JRDFw~hcjjzWU3B5UeNQPjagxtnAfSrZa-1WZYgu9NC01x8I2R3sKUQS6knKLEOjpt~7Bv9Ep28qB8cObc3Oinb4MMzadXlkhNmU8t0g22iGO5o5R5HwiZNNPPXr5JPZk2ARATJIWfPl1YEBpLzRDiAhBm0gqbcq~ZOtj4NYnfUfGeEuoWD9WExlmTY4U2CMk6QdfJ2lsbJRMwhl1mOTgfnN7d5nhamJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Overview of RM systems distributed in the four Arthrospira genomes. Colour bars above or below the central line from each strain represent genes with two distinct coding orientations in the genome. Black line is used to connect those best hit RM enzyme between different strains. Arrow indicates the unique enzyme compared with the other three strains. " path-from-xml="dsw023f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200357/dsw023f6p.png" path-from-xml="dsw023f6p.png" target="_blank">View large</a><a section="87200357" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f6p.png?Expires=2147483647&Signature=tjIemMJnsl1s5OKvIakpbZGQFgf58xfyRsHgVGrz3ExyYpjhxuD0BMTMIQgHyPtqv0caHPE6iu5JOTaGkcGfK4D7N0t9x6tor9IECgbjEu4SP7BcpJWA~5I-Ybho503GJAch9HB6PMBo0ViZ9IkjiaqTRoRjSMpmqOMfJlpLurApMdBunBn61tkpZWG0xer8WPYFh-gs~Vvrt37fn4GtaybRVLXKvkCsFSm3q90qJMhYg4E52k85LyZD~9oOpWRjg0CJ2iD9WSWFX3GUyDkpt1kBd4qMkcCkZjEGyTME7lgLfV2s6dx1JhHyFFIxgEMRygfr3yFxSFew9E5PFhLRVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200357&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Overview of RM systems distributed in the four <em>Arthrospira</em> genomes. Colour bars above or below the central line from each strain represent genes with two distinct coding orientations in the genome. Black line is used to connect those best hit RM enzyme between different strains. Arrow indicates the unique enzyme compared with the other three strains. </p></div></div>                        <a id="87200358" scrollto-destination="87200358"></a>
<div content-id="dsw023-T1" class="table-modal"><div class="table-wrap"><div id="dsw023-T1" class="table-wrap-title" data-id="dsw023-T1"><div class="label -label">Table 1.</div><div class="caption"><p> Distribution of RM systems in <em>Oscillatoriales</em> order </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>1M </th><th>1R </th><th>1S </th><th>2M </th><th>2R </th><th>3M </th><th>3R </th><th>Total </th></tr></thead><tbody><tr><td><em>Geitlerinema</em> sp. PCC 7407  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">7 </td></tr><tr><td><em>Leptolyngbya</em> sp. 2LT21S03  </td><td char=".">2 </td><td char=".">1 </td><td char=".">2 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7376  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">11 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7104  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">6 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7367  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">7 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>L. boryanar</em> PCC 6306  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>L. majuscula</em> 3L  </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7375  </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">9 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">16 </td></tr><tr><td><em>Geitlerinema</em> sp. PCC 7105  </td><td char=".">1 </td><td char=".">3 </td><td char=".">1 </td><td char=".">9 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">18 </td></tr><tr><td><em>Trichodesmium erythraeum</em> 21-75  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">10 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">19 </td></tr><tr><td><em>O. cyanobacterium</em> JSC-12  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 6802  </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">13 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7429  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">11 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td></tr><tr><td><em>M. vaginatus</em> PCC 9802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">14 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">23 </td></tr><tr><td><em>Oscillatoria formosa</em> PCC 6407  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">6 </td><td char=".">1 </td><td char=".">1 </td><td char=".">28 </td></tr><tr><td><em>Lyngbya</em> sp. CCY8106  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">29 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 6506  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">7 </td><td char=".">1 </td><td char=".">1 </td><td char=".">29 </td></tr><tr><td><em>Microcoleus</em> sp. PCC 7113  </td><td char=".">5 </td><td char=".">5 </td><td char=".">1 </td><td char=".">14 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">30 </td></tr><tr><td><em>S. major</em> PCC 6313  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">11 </td><td char=".">13 </td><td char=".">0 </td><td char=".">0 </td><td char=".">31 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 6406  </td><td char=".">7 </td><td char=".">6 </td><td char=".">2 </td><td char=".">9 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 10802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>O. acuminata</em> PCC 6304  </td><td char=".">4 </td><td char=".">4 </td><td char=".">0 </td><td char=".">20 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>S. subsalsa</em> PCC 9445  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">23 </td><td char=".">7 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>Coleofasciculus chthonoplastes</em> PCC 7420  </td><td char=".">4 </td><td char=".">2 </td><td char=".">0 </td><td char=".">22 </td><td char=".">11 </td><td char=".">0 </td><td char=".">0 </td><td char=".">39 </td></tr><tr><td><em>O. nigro-viridis</em> PCC 7112  </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">21 </td><td char=".">14 </td><td char=".">1 </td><td char=".">1 </td><td char=".">42 </td></tr><tr><td><em>C. epipsammum</em> PCC 9333  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">17 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">43 </td></tr><tr><td><em>A. platensis</em> str. Paraca  </td><td char=".">4 </td><td char=".">3 </td><td char=".">3 </td><td char=".">23 </td><td char=".">18 </td><td char=".">0 </td><td char=".">0 </td><td char=".">51 </td></tr><tr><td><em>A. platensis</em> NIES-39  </td><td char=".">4 </td><td char=".">4 </td><td char=".">4 </td><td char=".">26 </td><td char=".">20 </td><td char=".">0 </td><td char=".">0 </td><td char=".">58 </td></tr><tr><td><em>Arthrospira</em> sp. PCC 8005  </td><td char=".">4 </td><td char=".">4 </td><td char=".">7 </td><td char=".">23 </td><td char=".">19 </td><td char=".">1 </td><td char=".">1 </td><td char=".">59 </td></tr><tr><td><em>A. platensis</em> C1  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">24 </td><td char=".">18 </td><td char=".">1 </td><td char=".">1 </td><td char=".">60 </td></tr><tr><td><em>A. platensis</em> YZ  </td><td char=".">5 </td><td char=".">5 </td><td char=".">3 </td><td char=".">26 </td><td char=".">21 </td><td char=".">0 </td><td char=".">0 </td><td char=".">60 </td></tr></tbody></table></div><div content-id="dsw023-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>1M </th><th>1R </th><th>1S </th><th>2M </th><th>2R </th><th>3M </th><th>3R </th><th>Total </th></tr></thead><tbody><tr><td><em>Geitlerinema</em> sp. PCC 7407  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">7 </td></tr><tr><td><em>Leptolyngbya</em> sp. 2LT21S03  </td><td char=".">2 </td><td char=".">1 </td><td char=".">2 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7376  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">11 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7104  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">6 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7367  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">7 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>L. boryanar</em> PCC 6306  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>L. majuscula</em> 3L  </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7375  </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">9 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">16 </td></tr><tr><td><em>Geitlerinema</em> sp. PCC 7105  </td><td char=".">1 </td><td char=".">3 </td><td char=".">1 </td><td char=".">9 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">18 </td></tr><tr><td><em>Trichodesmium erythraeum</em> 21-75  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">10 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">19 </td></tr><tr><td><em>O. cyanobacterium</em> JSC-12  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 6802  </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">13 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7429  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">11 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td></tr><tr><td><em>M. vaginatus</em> PCC 9802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">14 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">23 </td></tr><tr><td><em>Oscillatoria formosa</em> PCC 6407  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">6 </td><td char=".">1 </td><td char=".">1 </td><td char=".">28 </td></tr><tr><td><em>Lyngbya</em> sp. CCY8106  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">29 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 6506  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">7 </td><td char=".">1 </td><td char=".">1 </td><td char=".">29 </td></tr><tr><td><em>Microcoleus</em> sp. PCC 7113  </td><td char=".">5 </td><td char=".">5 </td><td char=".">1 </td><td char=".">14 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">30 </td></tr><tr><td><em>S. major</em> PCC 6313  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">11 </td><td char=".">13 </td><td char=".">0 </td><td char=".">0 </td><td char=".">31 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 6406  </td><td char=".">7 </td><td char=".">6 </td><td char=".">2 </td><td char=".">9 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 10802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>O. acuminata</em> PCC 6304  </td><td char=".">4 </td><td char=".">4 </td><td char=".">0 </td><td char=".">20 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>S. subsalsa</em> PCC 9445  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">23 </td><td char=".">7 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>Coleofasciculus chthonoplastes</em> PCC 7420  </td><td char=".">4 </td><td char=".">2 </td><td char=".">0 </td><td char=".">22 </td><td char=".">11 </td><td char=".">0 </td><td char=".">0 </td><td char=".">39 </td></tr><tr><td><em>O. nigro-viridis</em> PCC 7112  </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">21 </td><td char=".">14 </td><td char=".">1 </td><td char=".">1 </td><td char=".">42 </td></tr><tr><td><em>C. epipsammum</em> PCC 9333  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">17 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">43 </td></tr><tr><td><em>A. platensis</em> str. Paraca  </td><td char=".">4 </td><td char=".">3 </td><td char=".">3 </td><td char=".">23 </td><td char=".">18 </td><td char=".">0 </td><td char=".">0 </td><td char=".">51 </td></tr><tr><td><em>A. platensis</em> NIES-39  </td><td char=".">4 </td><td char=".">4 </td><td char=".">4 </td><td char=".">26 </td><td char=".">20 </td><td char=".">0 </td><td char=".">0 </td><td char=".">58 </td></tr><tr><td><em>Arthrospira</em> sp. PCC 8005  </td><td char=".">4 </td><td char=".">4 </td><td char=".">7 </td><td char=".">23 </td><td char=".">19 </td><td char=".">1 </td><td char=".">1 </td><td char=".">59 </td></tr><tr><td><em>A. platensis</em> C1  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">24 </td><td char=".">18 </td><td char=".">1 </td><td char=".">1 </td><td char=".">60 </td></tr><tr><td><em>A. platensis</em> YZ  </td><td char=".">5 </td><td char=".">5 </td><td char=".">3 </td><td char=".">26 </td><td char=".">21 </td><td char=".">0 </td><td char=".">0 </td><td char=".">60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200358" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw023-T1" class="table-wrap-title" data-id="dsw023-T1"><div class="label -label">Table 1.</div><div class="caption"><p> Distribution of RM systems in <em>Oscillatoriales</em> order </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>1M </th><th>1R </th><th>1S </th><th>2M </th><th>2R </th><th>3M </th><th>3R </th><th>Total </th></tr></thead><tbody><tr><td><em>Geitlerinema</em> sp. PCC 7407  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">7 </td></tr><tr><td><em>Leptolyngbya</em> sp. 2LT21S03  </td><td char=".">2 </td><td char=".">1 </td><td char=".">2 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7376  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">11 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7104  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">6 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7367  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">7 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>L. boryanar</em> PCC 6306  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>L. majuscula</em> 3L  </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7375  </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">9 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">16 </td></tr><tr><td><em>Geitlerinema</em> sp. PCC 7105  </td><td char=".">1 </td><td char=".">3 </td><td char=".">1 </td><td char=".">9 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">18 </td></tr><tr><td><em>Trichodesmium erythraeum</em> 21-75  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">10 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">19 </td></tr><tr><td><em>O. cyanobacterium</em> JSC-12  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 6802  </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">13 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7429  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">11 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td></tr><tr><td><em>M. vaginatus</em> PCC 9802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">14 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">23 </td></tr><tr><td><em>Oscillatoria formosa</em> PCC 6407  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">6 </td><td char=".">1 </td><td char=".">1 </td><td char=".">28 </td></tr><tr><td><em>Lyngbya</em> sp. CCY8106  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">29 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 6506  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">7 </td><td char=".">1 </td><td char=".">1 </td><td char=".">29 </td></tr><tr><td><em>Microcoleus</em> sp. PCC 7113  </td><td char=".">5 </td><td char=".">5 </td><td char=".">1 </td><td char=".">14 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">30 </td></tr><tr><td><em>S. major</em> PCC 6313  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">11 </td><td char=".">13 </td><td char=".">0 </td><td char=".">0 </td><td char=".">31 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 6406  </td><td char=".">7 </td><td char=".">6 </td><td char=".">2 </td><td char=".">9 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 10802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>O. acuminata</em> PCC 6304  </td><td char=".">4 </td><td char=".">4 </td><td char=".">0 </td><td char=".">20 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>S. subsalsa</em> PCC 9445  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">23 </td><td char=".">7 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>Coleofasciculus chthonoplastes</em> PCC 7420  </td><td char=".">4 </td><td char=".">2 </td><td char=".">0 </td><td char=".">22 </td><td char=".">11 </td><td char=".">0 </td><td char=".">0 </td><td char=".">39 </td></tr><tr><td><em>O. nigro-viridis</em> PCC 7112  </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">21 </td><td char=".">14 </td><td char=".">1 </td><td char=".">1 </td><td char=".">42 </td></tr><tr><td><em>C. epipsammum</em> PCC 9333  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">17 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">43 </td></tr><tr><td><em>A. platensis</em> str. Paraca  </td><td char=".">4 </td><td char=".">3 </td><td char=".">3 </td><td char=".">23 </td><td char=".">18 </td><td char=".">0 </td><td char=".">0 </td><td char=".">51 </td></tr><tr><td><em>A. platensis</em> NIES-39  </td><td char=".">4 </td><td char=".">4 </td><td char=".">4 </td><td char=".">26 </td><td char=".">20 </td><td char=".">0 </td><td char=".">0 </td><td char=".">58 </td></tr><tr><td><em>Arthrospira</em> sp. PCC 8005  </td><td char=".">4 </td><td char=".">4 </td><td char=".">7 </td><td char=".">23 </td><td char=".">19 </td><td char=".">1 </td><td char=".">1 </td><td char=".">59 </td></tr><tr><td><em>A. platensis</em> C1  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">24 </td><td char=".">18 </td><td char=".">1 </td><td char=".">1 </td><td char=".">60 </td></tr><tr><td><em>A. platensis</em> YZ  </td><td char=".">5 </td><td char=".">5 </td><td char=".">3 </td><td char=".">26 </td><td char=".">21 </td><td char=".">0 </td><td char=".">0 </td><td char=".">60 </td></tr></tbody></table></div><div content-id="dsw023-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>1M </th><th>1R </th><th>1S </th><th>2M </th><th>2R </th><th>3M </th><th>3R </th><th>Total </th></tr></thead><tbody><tr><td><em>Geitlerinema</em> sp. PCC 7407  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">7 </td></tr><tr><td><em>Leptolyngbya</em> sp. 2LT21S03  </td><td char=".">2 </td><td char=".">1 </td><td char=".">2 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7376  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">11 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7104  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">6 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7367  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">7 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">12 </td></tr><tr><td><em>L. boryanar</em> PCC 6306  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>L. majuscula</em> 3L  </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">10 </td><td char=".">3 </td><td char=".">0 </td><td char=".">0 </td><td char=".">13 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 7375  </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">9 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">16 </td></tr><tr><td><em>Geitlerinema</em> sp. PCC 7105  </td><td char=".">1 </td><td char=".">3 </td><td char=".">1 </td><td char=".">9 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">18 </td></tr><tr><td><em>Trichodesmium erythraeum</em> 21-75  </td><td char=".">2 </td><td char=".">2 </td><td char=".">0 </td><td char=".">10 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">19 </td></tr><tr><td><em>O. cyanobacterium</em> JSC-12  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 6802  </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">13 </td><td char=".">4 </td><td char=".">0 </td><td char=".">0 </td><td char=".">20 </td></tr><tr><td><em>Pseudanabaena</em> sp. PCC 7429  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">11 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td></tr><tr><td><em>M. vaginatus</em> PCC 9802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">14 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">23 </td></tr><tr><td><em>Oscillatoria formosa</em> PCC 6407  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">6 </td><td char=".">1 </td><td char=".">1 </td><td char=".">28 </td></tr><tr><td><em>Lyngbya</em> sp. CCY8106  </td><td char=".">1 </td><td char=".">1 </td><td char=".">0 </td><td char=".">13 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">29 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 6506  </td><td char=".">1 </td><td char=".">2 </td><td char=".">1 </td><td char=".">16 </td><td char=".">7 </td><td char=".">1 </td><td char=".">1 </td><td char=".">29 </td></tr><tr><td><em>Microcoleus</em> sp. PCC 7113  </td><td char=".">5 </td><td char=".">5 </td><td char=".">1 </td><td char=".">14 </td><td char=".">5 </td><td char=".">0 </td><td char=".">0 </td><td char=".">30 </td></tr><tr><td><em>S. major</em> PCC 6313  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">11 </td><td char=".">13 </td><td char=".">0 </td><td char=".">0 </td><td char=".">31 </td></tr><tr><td><em>Leptolyngbya</em> sp. PCC 6406  </td><td char=".">7 </td><td char=".">6 </td><td char=".">2 </td><td char=".">9 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>Oscillatoria</em> sp. PCC 10802  </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">22 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">33 </td></tr><tr><td><em>O. acuminata</em> PCC 6304  </td><td char=".">4 </td><td char=".">4 </td><td char=".">0 </td><td char=".">20 </td><td char=".">9 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>S. subsalsa</em> PCC 9445  </td><td char=".">3 </td><td char=".">3 </td><td char=".">1 </td><td char=".">23 </td><td char=".">7 </td><td char=".">0 </td><td char=".">0 </td><td char=".">37 </td></tr><tr><td><em>Coleofasciculus chthonoplastes</em> PCC 7420  </td><td char=".">4 </td><td char=".">2 </td><td char=".">0 </td><td char=".">22 </td><td char=".">11 </td><td char=".">0 </td><td char=".">0 </td><td char=".">39 </td></tr><tr><td><em>O. nigro-viridis</em> PCC 7112  </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">21 </td><td char=".">14 </td><td char=".">1 </td><td char=".">1 </td><td char=".">42 </td></tr><tr><td><em>C. epipsammum</em> PCC 9333  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">17 </td><td char=".">10 </td><td char=".">0 </td><td char=".">0 </td><td char=".">43 </td></tr><tr><td><em>A. platensis</em> str. Paraca  </td><td char=".">4 </td><td char=".">3 </td><td char=".">3 </td><td char=".">23 </td><td char=".">18 </td><td char=".">0 </td><td char=".">0 </td><td char=".">51 </td></tr><tr><td><em>A. platensis</em> NIES-39  </td><td char=".">4 </td><td char=".">4 </td><td char=".">4 </td><td char=".">26 </td><td char=".">20 </td><td char=".">0 </td><td char=".">0 </td><td char=".">58 </td></tr><tr><td><em>Arthrospira</em> sp. PCC 8005  </td><td char=".">4 </td><td char=".">4 </td><td char=".">7 </td><td char=".">23 </td><td char=".">19 </td><td char=".">1 </td><td char=".">1 </td><td char=".">59 </td></tr><tr><td><em>A. platensis</em> C1  </td><td char=".">5 </td><td char=".">6 </td><td char=".">5 </td><td char=".">24 </td><td char=".">18 </td><td char=".">1 </td><td char=".">1 </td><td char=".">60 </td></tr><tr><td><em>A. platensis</em> YZ  </td><td char=".">5 </td><td char=".">5 </td><td char=".">3 </td><td char=".">26 </td><td char=".">21 </td><td char=".">0 </td><td char=".">0 </td><td char=".">60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200358" target="_blank">View Large</a></div></div><p> Based on the whole genome co-linear relationship, we performed comparative analyses of RM systems intra- and inter-groups for four <em>Arthrospira</em> strains. As expected, the co-linearity of RM systems-coding genes within groups was consistent with the whole genomic synteny ( <a reveal-id="dsw023-F6" data-open="dsw023-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> ). Within the groups, besides those common RM systems, there were three unique enzymes in the <em>A. platensis</em> YZ genome compared with <em>A. platensis</em> NIES-39, including one type IIR, one type IM and one type IR enzyme, whereas only a type IS was uniquely existence in the <em>A. platensis</em> NIES-39 compared with <em>A. platensis</em> YZ. Likewise, seven (one type IR, one IS, three IIR, two IIM) and six (three type IS and three IIR) enzymes were uniquely presented to each other in <em>A. platensis</em> C1 and <em>Arthrospira</em> sp. PCC 8005, respectively. For inter-groups comparison, two (one IIR and one IIM) and three (two type IS and one IIR) enzymes were uniquely appeared in the <em>A. platensis</em> C1 and <em>Arthrospira</em> sp. PCC 8005 response to the other strains, respectively, while no unique RM enzyme occurred in the <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39 genomes ( <a reveal-id="dsw023-F6" data-open="dsw023-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> ). Generally, the more unique RM gene relative to the smaller genome size was observed in these closely related strains, which may indicate the importance of RM systems for genome plasticity in the <em>Arthrospira</em> . The FASTA sequences of unique enzymes presented in each strain were listed in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S2</a></span> . </p><p> Chromosomal inversions have been frequently identified in other closely related cyanobacterial strains. For instance, an inverted 1.3-Mb fragment was detected between <em>Cyanothece</em> sp. PCC 8801 and PCC 8802, and a 188-kb inversion was identified between <em>Synechococcus elongates</em> PCC 6301 and PCC 7942 chromosome. <a reveal-id="dsw023-B47" data-open="dsw023-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw023-B48" data-open="dsw023-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> However, these authors did not provide possible suggestions underlying the generation of the chromosomal inversion. Indeed, the 1.3-Mb inverted fragment was flanked by a pair of 5.2-kb LIRs with extremely high identities (&gt;99%) both in <em>Cyanothece</em> sp. PCC 8801 and PCC 8802, <a reveal-id="dsw023-B48" data-open="dsw023-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and this LIR sequences were only detected in the flanking regions of the inverted fragment from these two closely related strains. Likewise, the 188-kb inversion between <em>S</em><em>.</em><em>elongates</em> PCC 6301 and PCC 7942 was also flanked by a pair of 0.5-kb LIRs with &gt;81% identities in both strains. <a reveal-id="dsw023-B47" data-open="dsw023-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The above evidences may suggest that LIRs were closely related to the chromosomal inversion. Previous study on phylum <em>Proteobacteria</em> demonstrated that extensively rearranged genomes contain a high number of repeats among closely related bacteria. <a reveal-id="dsw023-B49" data-open="dsw023-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The negative association between the number of repeats and genome stability has been observed and statistically confirmed in γ-proteobacteria, where the genomes contained more repeats undergoing accelerated rearrangement rates, while genomes lacking repeats were more stable. <a reveal-id="dsw023-B50" data-open="dsw023-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Based on these molecular evidences, previous explanations for the possible mechanisms underlying chromosomal inversion included illegitimate recombination, <a reveal-id="dsw023-B51 dsw023-B52 dsw023-B53" data-open="dsw023-B51 dsw023-B52 dsw023-B53" class="link link-ref link-reveal xref-bibr"><sup>51–53</sup></a> intra-chromosomally homologous recombination, <a reveal-id="dsw023-B54" data-open="dsw023-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> as well as integrase-mediated site-specific recombination. <a reveal-id="dsw023-B55" data-open="dsw023-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsw023-B56" data-open="dsw023-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> When compared with these mechanisms, restriction enzyme-mediated chromosomal inversions might be oversimplified, yet could serve as a new mechanistic model for chromosomal rearrangements. The proposed model for restriction enzyme-mediated chromosomal inversion has been described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span> . </p>                            <h3 scrollto-destination=87200361 id="87200361" class="section-title" >3.4. Repeat sequences in <em>Arthrospira</em></h3>
<p> One of the most notable features of the cyanobacterial genome is the predominance of various repeated sequences. Genome-wide clustering analysis of all the genes of <em>Oscillatoriales</em> genomes, respectively, revealed that members of genus <em>Arthrospira</em> possessed the lowest C/W ratios at any level of homology, which in turn suggested that these underwent multiple duplication events during speciation ( <a reveal-id="dsw023-F7" data-open="dsw023-F7" class="link link-reveal link-table xref-fig">Fig. 7</a> ). C/W ratio represented the number of total clusters divided by the number of total genes. Moreover, members of <em>Arthrospira</em> , particularly <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ, showed the highest R/T ratios, which was suggested that gene duplication in <em>Arthrospira</em> widely occurred. This ratio was calculated as the number of redundant gene clusters that comprised at least two genes divided by the total number of clusters. In <em>A. platensis</em> YZ, a total of 327 genes were determined to have been duplicated at least once, which in turn resulted in the generation of 2,020 redundant genes constituting approximately one-third of the total gene and one-sixth of the complete genome size ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). In general, in addition to a large number of uncharacterized genes, these duplicated genes are functionally involved in replication, recombination, energy production, transcription, and post-translational modification. Among these redundant sequences, the most frequently duplicated genes were RTase, transposase, and integrase (194, 159, and 12 copies, respectively), which were derived from 4, 23, and 1 clusters, respectively. We then investigated the probable origin of these enzymes. First, sequence alignment of 12 integrases showed &gt;84% global amino acid similarity, and these integrases were further subdivided into three subtypes, which was suggestive that multiplication of integrase-encoding genes in <em>A. platensis</em> YZ genome was most probably generated by multiple gene duplication events that were accompanied by amino acid substitutions ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span> ). Second, we constructed the phylogenetic relationship of the two largest RTase clusters (clusters 527 and 4099, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ), which contained 79 and 73 members, respectively. The RTases derived from the same cluster were located together, yet showed distinct boundaries between clusters ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> ). This indicated that hundreds of RTase paralogs might have been derived from several common ancestors. Another significant characteristic of duplicated genes was that they were usually adjacently located or duplication-linked with a plenty of unknown functions, some of which can be annotated by one or two of the three enzymes (RTase, transposase, and integrase), and these duplicated fragments varied in size from several to dozens of kilobases. <a reveal-id="dsw023-B57 dsw023-B58 dsw023-B59" data-open="dsw023-B57 dsw023-B58 dsw023-B59" class="link link-ref link-reveal xref-bibr"><sup>57–59</sup></a> This was also partially identified by the observation of contiguous relationships of duplicated genes after sorted by their coordinates in the genome. Therefore, sequence duplication might significantly contribute to the genomic architecture of <em>Arthrospira</em> genomes, which resulted in the formation of various long repeats at both orientations. Previous studies have shown that a large proportion of the transposases and RTase constituted the long repeated sequences in some cyanobacteria. <a reveal-id="dsw023-B57 dsw023-B58 dsw023-B59 dsw023-B60" data-open="dsw023-B57 dsw023-B58 dsw023-B59 dsw023-B60" class="link link-ref link-reveal xref-bibr"><sup>57–60</sup></a> For example, 93% of 362 transposases of <em>Microcystis aeruginosa</em> PCC 7806 emerged in the long repeats. <a reveal-id="dsw023-B59" data-open="dsw023-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> However, because most of the duplicated genes are functionally unknown, besides chromosomal rearrangements, the potential phylogenetic roles of multiple sequence duplication events in the <em>Arthrospira</em> are remaining elusive. </p>                        <a id="87200363" scrollto-destination="87200363"></a>
<div id="dsw023-F7" data-id="dsw023-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f7p.png?Expires=2147483647&Signature=Tn0JtH~HjIexxRisXnLskCx5uJQfoUFasjsr~2ec8l2tfXhCVMZIyb6CtHjvNk-S-koVvP94GNpEaWFCkLLGi0ZSYvfcDRei4GHTVU3o-r7AIK83~qCbu8BMEAno2DLT42hS~EM8QakVJ7vjMM0kmxScR2NoASukJuZwUUeFhkT0JLmjvmlcPcdUsHwnzceKR2-ABwBLRpCzdI8L0Fr3Vz3xFV7J46~59qBGWNP0q6ygKS5zHRbHTOF8Df~3W4GH8yzOL8q1sfVA6nRT8iLPHDTshetMNMYoEBWgrT6R7hh006NXIYGKvEUgkFnNkHGNxsu6ZqCP9OPlvgnnYkuTlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Gene duplication in selected species of Oscillatoriales . The total genes from each species are clustered at 60, 65, 70, 75, 80, 85, 90, 95, and 100% amino acid sequence identities. For one certain species at one certain resolution, the C/W ratio represents the total number of clusters divided by the total number of genes. R/W ratio represents the number of redundant gene clusters (the clusters that comprised at least two genes) divided by the total number of clusters. " path-from-xml="dsw023f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200363/dsw023f7p.png" path-from-xml="dsw023f7p.png" target="_blank">View large</a><a section="87200363" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f7p.png?Expires=2147483647&Signature=AKah5vrAPcXOBkdns2EhE~44M6A55OmPi4Tyab83NPX2PQ5m8OFbjThLLVXsNNduXfE0omC0qUxg07y3ZRIer7PMJQi-5m7FFD0p5~zTWwaoypWwh0wo4P~2EcGEqwUp6fq8uOz5x0U6WdYetnOunFnLZv74tPIN4DRt0Vcytw5Q78Wr0tydvVrTLX1Gl3Zd1mblGM3adfNmAImF9APspmM-5q83ylnVQRHjB9jcA97SjAerYJw4n9kh~du0houeysngPGvnWo68UEClM6-E~bnNEe6I5kNnC1OQTl3rSIh1pCsUSgIliXZ6vapiIvaZAiUuo5JV0MI8zK2BENT0rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200363&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Gene duplication in selected species of <em>Oscillatoriales</em> . The total genes from each species are clustered at 60, 65, 70, 75, 80, 85, 90, 95, and 100% amino acid sequence identities. For one certain species at one certain resolution, the C/W ratio represents the total number of clusters divided by the total number of genes. R/W ratio represents the number of redundant gene clusters (the clusters that comprised at least two genes) divided by the total number of clusters. </p></div></div><div content-id="dsw023-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f7p.png?Expires=2147483647&Signature=Tn0JtH~HjIexxRisXnLskCx5uJQfoUFasjsr~2ec8l2tfXhCVMZIyb6CtHjvNk-S-koVvP94GNpEaWFCkLLGi0ZSYvfcDRei4GHTVU3o-r7AIK83~qCbu8BMEAno2DLT42hS~EM8QakVJ7vjMM0kmxScR2NoASukJuZwUUeFhkT0JLmjvmlcPcdUsHwnzceKR2-ABwBLRpCzdI8L0Fr3Vz3xFV7J46~59qBGWNP0q6ygKS5zHRbHTOF8Df~3W4GH8yzOL8q1sfVA6nRT8iLPHDTshetMNMYoEBWgrT6R7hh006NXIYGKvEUgkFnNkHGNxsu6ZqCP9OPlvgnnYkuTlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Gene duplication in selected species of Oscillatoriales . The total genes from each species are clustered at 60, 65, 70, 75, 80, 85, 90, 95, and 100% amino acid sequence identities. For one certain species at one certain resolution, the C/W ratio represents the total number of clusters divided by the total number of genes. R/W ratio represents the number of redundant gene clusters (the clusters that comprised at least two genes) divided by the total number of clusters. " path-from-xml="dsw023f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200363/dsw023f7p.png" path-from-xml="dsw023f7p.png" target="_blank">View large</a><a section="87200363" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f7p.png?Expires=2147483647&Signature=AKah5vrAPcXOBkdns2EhE~44M6A55OmPi4Tyab83NPX2PQ5m8OFbjThLLVXsNNduXfE0omC0qUxg07y3ZRIer7PMJQi-5m7FFD0p5~zTWwaoypWwh0wo4P~2EcGEqwUp6fq8uOz5x0U6WdYetnOunFnLZv74tPIN4DRt0Vcytw5Q78Wr0tydvVrTLX1Gl3Zd1mblGM3adfNmAImF9APspmM-5q83ylnVQRHjB9jcA97SjAerYJw4n9kh~du0houeysngPGvnWo68UEClM6-E~bnNEe6I5kNnC1OQTl3rSIh1pCsUSgIliXZ6vapiIvaZAiUuo5JV0MI8zK2BENT0rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200363&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Gene duplication in selected species of <em>Oscillatoriales</em> . The total genes from each species are clustered at 60, 65, 70, 75, 80, 85, 90, 95, and 100% amino acid sequence identities. For one certain species at one certain resolution, the C/W ratio represents the total number of clusters divided by the total number of genes. R/W ratio represents the number of redundant gene clusters (the clusters that comprised at least two genes) divided by the total number of clusters. </p></div></div><p> TRs are patterns of nucleotide sequence directly adjacent to each other. Distributions of TRs in the genomic loci are also recognized as the variable number of TR regions that have been widely used to determine an individual's inherited traits as well as for molecular typing both in prokaryotic and eukaryotic species. <a reveal-id="dsw023-B61" data-open="dsw023-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Based on the size of the repeat motif, TRs were classified into microsatellites, minisatellites, and macrosatellites with the motif sizes varying from 1 to 9, 10 to 100, and &gt;100 bp, respectively. <a reveal-id="dsw023-B62" data-open="dsw023-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Examining the four <em>Arthrospira</em> genomes showed that 870, 736, 1,148, and 848 TRs-containing loci (TRL) were presented in <em>Arthrospira</em> sp. PCC 8005, <em>A. platensis</em> C1, <em>A. platensis</em> NIES-39, and <em>A. platensis</em> YZ, respectively ( <a reveal-id="dsw023-F8" data-open="dsw023-F8" class="link link-reveal link-table xref-fig">Fig. 8A</a> ), which represented 789, 666, 1,009, and 766 non-redundant repeat motifs ( <a reveal-id="dsw023-F8" data-open="dsw023-F8" class="link link-reveal link-table xref-fig">Fig. 8B</a> ). The size of these core repeat motifs varied from several base pairs to 1.9 kb that led to form the longest single TRL up to 8.1 kb. Comparative analyses showed that the number of shared repeat motifs among strains derived from the same groups was significantly more than those from different groups ( <a reveal-id="dsw023-F8" data-open="dsw023-F8" class="link link-reveal link-table xref-fig">Fig. 8B</a> ). Only six commonly shared repeat motifs were identified, suggesting that conservation of TRs among four strains was weak. Additionally, more overlapped repeats at both quantity and proportion level may indicate that the evolutionary relationship between <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ was closer than that between <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. </p>                        <a id="87200365" scrollto-destination="87200365"></a>
<div id="dsw023-F8" data-id="dsw023-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f8p.png?Expires=2147483647&Signature=LL59cIMckXsi4lAXfEyM9qzle1~jkF9roo3LCK2UX5Xqp7q2vhgSBLpCPjFDswfdSGnvYDzEwXuUr82GWv5PfZhMxF9CeTYZRKlH1pcesYoO0MZeMJmfhZ~ZSYIjB-NnZsX6UeIVF8mJ4seSYPCHgWMKeZb0E7SnKs-ZTz2rIZiSYJuxE6YmiRJhuFDIO9yC4Cb740q0-woV7tpG575OzYLIEembs3XVZM3~keMws5Sl-jVfc4gERINy2gz-nF0VWJb0MPvFLk-tLJidy7kfcqbKNAZg8~BFDeIcEFzb99bY4azzPfbKRscEVm4ZOT52Kh2njd0SnBvQxY2t9H8gRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of TRs and their relationships among four Arthrospira genomes. (A) length distributions of 870, 736, 1,148, and 848 repeat motifs and their corresponding TRL in Arthrospira sp. PCC 8005, A. platensis C1, A. platensis NIES-39 and A. platensis YZ, respectively. Colour key for each strain is responded to the B panel. (B) 870, 736, 1,148, and 848 repeat motifs from each strain were independently clustered with 100% nucleotide identity, respectively. The non-redundant repeat motifs (789, 666, 1,009, and 766) from each strain were pooled and further clustered with 100% identity to show the common and unique features of these repeat motifs derived from different strains. " path-from-xml="dsw023f8p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200365/dsw023f8p.png" path-from-xml="dsw023f8p.png" target="_blank">View large</a><a section="87200365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f8p.png?Expires=2147483647&Signature=ifbn~DZSOlGaWY5m-R8z8IgUA0UvgqqX9ypGlC~0o3AI4uFa-28pF-9AjNoHS9Iei4m5kmMgn7-5ukO4xfcYCTZx6yVSrNdBSQznzoLyMv6DoN5oMbK4-U59~qxIjL2ykxKM4Vd1c8Hsx8pNxun3R2StalFd8pyGOKBJLFxvrLVsqJ~WFpa8dQOE1R08cuf~o0X1LYnxobEwo8-jHvwGbeJ-NLyKQ8of~o7RA30GjCwUEJFHlfTfEj-zs3Ej-krHMpNaxSaVEdcnvq2HJDwJLBWKeREGW0mvnIS5834fCrSPpgIXX4pKKxI2OFZGhccykvTc9iwMnSJIpEt42kurdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200365&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f8p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of TRs and their relationships among four <em>Arthrospira</em> genomes. <strong>(A)</strong> length distributions of 870, 736, 1,148, and 848 repeat motifs and their corresponding TRL in <em>Arthrospira</em> sp. PCC 8005, <em>A. platensis</em> C1, <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ, respectively. Colour key for each strain is responded to the B panel. <strong>(B)</strong> 870, 736, 1,148, and 848 repeat motifs from each strain were independently clustered with 100% nucleotide identity, respectively. The non-redundant repeat motifs (789, 666, 1,009, and 766) from each strain were pooled and further clustered with 100% identity to show the common and unique features of these repeat motifs derived from different strains. </p></div></div><div content-id="dsw023-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f8p.png?Expires=2147483647&Signature=LL59cIMckXsi4lAXfEyM9qzle1~jkF9roo3LCK2UX5Xqp7q2vhgSBLpCPjFDswfdSGnvYDzEwXuUr82GWv5PfZhMxF9CeTYZRKlH1pcesYoO0MZeMJmfhZ~ZSYIjB-NnZsX6UeIVF8mJ4seSYPCHgWMKeZb0E7SnKs-ZTz2rIZiSYJuxE6YmiRJhuFDIO9yC4Cb740q0-woV7tpG575OzYLIEembs3XVZM3~keMws5Sl-jVfc4gERINy2gz-nF0VWJb0MPvFLk-tLJidy7kfcqbKNAZg8~BFDeIcEFzb99bY4azzPfbKRscEVm4ZOT52Kh2njd0SnBvQxY2t9H8gRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of TRs and their relationships among four Arthrospira genomes. (A) length distributions of 870, 736, 1,148, and 848 repeat motifs and their corresponding TRL in Arthrospira sp. PCC 8005, A. platensis C1, A. platensis NIES-39 and A. platensis YZ, respectively. Colour key for each strain is responded to the B panel. (B) 870, 736, 1,148, and 848 repeat motifs from each strain were independently clustered with 100% nucleotide identity, respectively. The non-redundant repeat motifs (789, 666, 1,009, and 766) from each strain were pooled and further clustered with 100% identity to show the common and unique features of these repeat motifs derived from different strains. " path-from-xml="dsw023f8p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200365/dsw023f8p.png" path-from-xml="dsw023f8p.png" target="_blank">View large</a><a section="87200365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f8p.png?Expires=2147483647&Signature=ifbn~DZSOlGaWY5m-R8z8IgUA0UvgqqX9ypGlC~0o3AI4uFa-28pF-9AjNoHS9Iei4m5kmMgn7-5ukO4xfcYCTZx6yVSrNdBSQznzoLyMv6DoN5oMbK4-U59~qxIjL2ykxKM4Vd1c8Hsx8pNxun3R2StalFd8pyGOKBJLFxvrLVsqJ~WFpa8dQOE1R08cuf~o0X1LYnxobEwo8-jHvwGbeJ-NLyKQ8of~o7RA30GjCwUEJFHlfTfEj-zs3Ej-krHMpNaxSaVEdcnvq2HJDwJLBWKeREGW0mvnIS5834fCrSPpgIXX4pKKxI2OFZGhccykvTc9iwMnSJIpEt42kurdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200365&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f8p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of TRs and their relationships among four <em>Arthrospira</em> genomes. <strong>(A)</strong> length distributions of 870, 736, 1,148, and 848 repeat motifs and their corresponding TRL in <em>Arthrospira</em> sp. PCC 8005, <em>A. platensis</em> C1, <em>A. platensis</em> NIES-39 and <em>A. platensis</em> YZ, respectively. Colour key for each strain is responded to the B panel. <strong>(B)</strong> 870, 736, 1,148, and 848 repeat motifs from each strain were independently clustered with 100% nucleotide identity, respectively. The non-redundant repeat motifs (789, 666, 1,009, and 766) from each strain were pooled and further clustered with 100% identity to show the common and unique features of these repeat motifs derived from different strains. </p></div></div><p> CRISPRs, a type of important repeat elements, are related to the adaptive immunity system that has been widely identified and characterized in many archaea and bacteria. <a reveal-id="dsw023-B63" data-open="dsw023-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw023-B64" data-open="dsw023-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> We comparatively analysed the distribution of CRISPR and CRISPR-associated protein (Cas) systems among four <em>Arthrospira</em> genomes. On the whole, <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39 harboured three typical regions (Block I-III of <a reveal-id="dsw023-F9" data-open="dsw023-F9" class="link link-reveal link-table xref-fig">Fig. 9A</a> ) that contained the CRISPR-Cas system, whereas only two regions were identified in <em>A. platensis</em> C1 and <em>Arthrospira</em> sp. PCC 8005 (Block I and II of <a reveal-id="dsw023-F9" data-open="dsw023-F9" class="link link-reveal link-table xref-fig">Fig. 9B</a> ). It is interesting to find that two CRISPR arrays were adjacently located next to different <em>cas</em> gene organization in all strains as shown in <a reveal-id="dsw023-F9" data-open="dsw023-F9" class="link link-reveal link-table xref-fig">Figure 9</a> Blocks I. The repeat motif of the corresponding CRISPR array was approximately identical to the ones from the members belonging to the same or different groups represented as global genomic synteny, except the slight diversity was presented in the latter array between members from the different groups. Strains derived from the same groups also showed the high similarity in <em>cas</em> gene organization. The main differences for the same CRISPR-Cas systems among strains involved copy number of repeats and the diversity of spacers ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S3</a></span> ). <em>A. platensis</em> NIES-39 and <em>A. platensis</em> C1 harboured more repeats and spacers than <em>A. platensis</em> YZ and <em>Arthrospira</em> sp. PCC 8005, respectively. Furthermore, we also found that one <em>cas</em> gene cluser in all the four strains (Csm1-Csm2-Csm3-Csm4-Csm5-Cas6-CasTM1812 in <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39, Csm1/Csm2-Csm3-Csm4-Csm5-CasTM1812 in <em>A. platensis</em> C1 and <em>Arthrospira</em> sp. PCC 8005) was not CRISPR-associated, which contradicted the general description of <em>cas</em> gene distribution, because we did not identify any CRISPR array that was proximal to the <em>cas</em> gene cluster. In addition to these typical CRISPR-Cas systems, several CRISPR arrays were also detected in the four genomes without a <em>cas</em> gene located within their proximal regions. </p>                        <a id="87200367" scrollto-destination="87200367"></a>
<div id="dsw023-F9" data-id="dsw023-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f9p.png?Expires=2147483647&Signature=kK-sZCR9WSYBzjAhEcqDcmvxX9MOhnWzn62eJS7XoJ0wNjgw9YiekCVJrqHmjRnKWGwCdlwb52D1lgT9Lf1ZnVxPk2wXz4dECgXY7GJna67P2WihIdp0sZHy~mv4vh~C9NIhl3drWnaLbMC0nD15iDJlFjcf~K0vDwtDyzGlK8eutRx6JSNZDJZdRhwkttJeAPQ1LOxVnvcqV7B8a3fbHBIB-YXvMwaO-NOEkT-JMC6xKuV3ExQpOv5eRnOwnIeRlo~YUVx48ReC9ZebyvoMvT3BD8wuCS~9bu9Z8E7epMDTxcZi6nL5L5tRZ~p~nlmCSuqKuZTzWQEssAAwiKFIyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The CRISPR-Cas system loci in the four Arthrospira genomes. Block I-III from left panel (A) showed the organization of CRISPR- Cas systems in A. platensis YZ and A. platensis NIES-39, and the Block I-II from right panel (B) showed the counterparts in Arthrospira sp. PCC 8005 and A. platensis C1. Repeat motifs amid the CRISPR arrays are shared by both strains at the corresponding loci. The unlabelled genes represent unannotated or functionally unknown genes. " path-from-xml="dsw023f9p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200367/dsw023f9p.png" path-from-xml="dsw023f9p.png" target="_blank">View large</a><a section="87200367" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f9p.png?Expires=2147483647&Signature=33OvNDYFC5Wn3N9j6y-K5MDRbIC73~aFIifKB-RR35HQuW7WRW9lq1aVTjY7L9lEppvNLSEmcKovU2krCN6~hOtMAttnbyj16rlAx~mDK19GCvAUtA7q9ZAo3Juk7oMUj23uPatBJ7B2WG1QyH4BQ7XuRa3jI0fjMqcgTx38upf1nSa4WepwObupOy0R0A3UpM93YmD8qr8-s3W-rzdhvFPX-zpXS-06npnK19ihj9QiINm7VXsVWdITN0FG6XWG1lDYksEQtxKgZu1H07CDpQ4Tee7Z6WOhbkr8skaDlanljI7ui0dsw1pjoj-vypkS36DaZN-oABEXGTB1nFslQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200367&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f9p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The CRISPR-Cas system loci in the four <em>Arthrospira</em> genomes. Block I-III from left panel <strong>(A)</strong> showed the organization of CRISPR- <em>Cas</em> systems in <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39, and the Block I-II from right panel <strong>(B)</strong> showed the counterparts in <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. Repeat motifs amid the CRISPR arrays are shared by both strains at the corresponding loci. The unlabelled genes represent unannotated or functionally unknown genes. </p></div></div><div content-id="dsw023-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/m_dsw023f9p.png?Expires=2147483647&Signature=kK-sZCR9WSYBzjAhEcqDcmvxX9MOhnWzn62eJS7XoJ0wNjgw9YiekCVJrqHmjRnKWGwCdlwb52D1lgT9Lf1ZnVxPk2wXz4dECgXY7GJna67P2WihIdp0sZHy~mv4vh~C9NIhl3drWnaLbMC0nD15iDJlFjcf~K0vDwtDyzGlK8eutRx6JSNZDJZdRhwkttJeAPQ1LOxVnvcqV7B8a3fbHBIB-YXvMwaO-NOEkT-JMC6xKuV3ExQpOv5eRnOwnIeRlo~YUVx48ReC9ZebyvoMvT3BD8wuCS~9bu9Z8E7epMDTxcZi6nL5L5tRZ~p~nlmCSuqKuZTzWQEssAAwiKFIyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The CRISPR-Cas system loci in the four Arthrospira genomes. Block I-III from left panel (A) showed the organization of CRISPR- Cas systems in A. platensis YZ and A. platensis NIES-39, and the Block I-II from right panel (B) showed the counterparts in Arthrospira sp. PCC 8005 and A. platensis C1. Repeat motifs amid the CRISPR arrays are shared by both strains at the corresponding loci. The unlabelled genes represent unannotated or functionally unknown genes. " path-from-xml="dsw023f9p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200367/dsw023f9p.png" path-from-xml="dsw023f9p.png" target="_blank">View large</a><a section="87200367" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023f9p.png?Expires=2147483647&Signature=33OvNDYFC5Wn3N9j6y-K5MDRbIC73~aFIifKB-RR35HQuW7WRW9lq1aVTjY7L9lEppvNLSEmcKovU2krCN6~hOtMAttnbyj16rlAx~mDK19GCvAUtA7q9ZAo3Juk7oMUj23uPatBJ7B2WG1QyH4BQ7XuRa3jI0fjMqcgTx38upf1nSa4WepwObupOy0R0A3UpM93YmD8qr8-s3W-rzdhvFPX-zpXS-06npnK19ihj9QiINm7VXsVWdITN0FG6XWG1lDYksEQtxKgZu1H07CDpQ4Tee7Z6WOhbkr8skaDlanljI7ui0dsw1pjoj-vypkS36DaZN-oABEXGTB1nFslQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200367&ar=2469971&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw023f9p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The CRISPR-Cas system loci in the four <em>Arthrospira</em> genomes. Block I-III from left panel <strong>(A)</strong> showed the organization of CRISPR- <em>Cas</em> systems in <em>A. platensis</em> YZ and <em>A. platensis</em> NIES-39, and the Block I-II from right panel <strong>(B)</strong> showed the counterparts in <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1. Repeat motifs amid the CRISPR arrays are shared by both strains at the corresponding loci. The unlabelled genes represent unannotated or functionally unknown genes. </p></div></div><p> In conclusion, <em>A. platensis</em> YZ was more closely related to <em>A. platensis</em> NIES-39 than <em>Arthrospira</em> sp. PCC 8005 and <em>A. platensis</em> C1 at various levels. Comparative genomics and clustering analyses suggested that the <em>Arthrospira</em> genome was highly fluid and exceptionally plastic, which was characterized by extensive chromosomal rearrangements, a large number of repetitive sequences as well as genes encoding transposases, RTases, recombinases, and restriction enzymes. Therefore, speciation of <em>Arthrospira</em> and the specialization for an ecological niche were accompanied by a large scale reorganization of its genomic structure. These findings will facilitate in better understanding the process of speciation and diversification in <em>A</em><em>.</em><em>platensis</em> . </p>                            <h2 scrollto-destination=87200369 id="87200369" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the members of ‘Spirulina Consortium’ at Beijing Genomics Institute, Shenzhen for whole genome DNA sequencing.</p>                            <h2 scrollto-destination=87200371 id="87200371" class="backsection-title" >Accession number</h2>
<p>CP013008</p>                            <h2 scrollto-destination=87200373 id="87200373" class="backsection-title" >Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by National Natural Science Foundation of China (30571009), Natural Science Foundation of Zhejiang Province, China (Z307471) and Science and Technology Foundation of Zhejiang Province, China (2009C33040).</p>                            <h2 scrollto-destination=87200375 id="87200375" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw023-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Habib</div>
               <div class="given-names">M. A. B.</div>
            </div>
             <div class="name">
              <div class="surname">Mashuda</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Huntington</div>
               <div class="given-names">T. C.</div>
            </div>
             <div class="name">
              <div class="surname">Hasan</div>
               <div class="given-names">M. R.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="source">A Review on Culture, Production and Use of Spirulina as Food for Humans and Feeds for Domestic Animals and Fish</div> 
            . 
            <div class="publisher-loc">Food and Agriculture Organization of The United Nations</div>
            : 
            <div class="publisher-name">Rome, Italy</div>
            .
          </div></div></div></div><div content-id="dsw023-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Huili</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Xiaokai</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Meili</div>
               <div class="given-names">L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Proteomic analysis and qRT-PCR verification of temperature response to Arthrospira (Spirulina) platensis</div>
            . 
            <div class="source">PLoS One</div> 
            , 
            <div class="volume">8</div>
            , 
            <div class="fpage">e83485</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomic%20analysis%20and%20qRT-PCR%20verification%20of%20temperature%20response%20to%20Arthrospira%20(Spirulina)%20platensis&amp;author=W.%20Huili&amp;author=Z.%20Xiaokai&amp;author=L.%20Meili&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e83485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0083485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0083485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0083485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24349519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">W.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Identification of differentially expressed proteins of Arthrospira (Spirulina) plantensis-YZ under salt-stress conditions by proteomics and qRT-PCR analysis</div>
            . 
            <div class="source">Proteome Sci</div> 
            ., 
            <div class="volume">11</div>
            , 
            <div class="fpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20differentially%20expressed%20proteins%20of%20Arthrospira%20(Spirulina)%20plantensis-YZ%20under%20salt-stress%20conditions%20by%20proteomics%20and%20qRT-PCR%20analysis&amp;author=H.%20Wang&amp;author=Y.%20Yang&amp;author=W.%20Chen&amp;publication_year=2013&amp;journal=Proteome%20Sci&amp;volume=11&amp;pages=6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1477-5956-11-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1477-5956-11-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1477-5956-11-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23363438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Pan</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Lu</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Spirulina phycocyanin induces differential protein expression and apoptosis in SKOV-3 cells</div>
            . 
            <div class="source">Int. J. Biol. Macromol</div> 
            ., 
            <div class="volume">81</div>
            , 
            <div class="fpage">951</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spirulina%20phycocyanin%20induces%20differential%20protein%20expression%20and%20apoptosis%20in%20SKOV-3%20cells&amp;author=R.%20Pan&amp;author=R.%20Lu&amp;author=Y.%20Zhang&amp;publication_year=2015&amp;journal=Int.%20J.%20Biol.%20Macromol&amp;volume=81&amp;pages=951-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijbiomac.2015.09.039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijbiomac.2015.09.039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijbiomac.2015.09.039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26410814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Liang</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Bao</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Qin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Genome-wide analysis of restriction-modification system in unicellular and filamentous cyanobacteria</div>
            . 
            <div class="source">Physiol. Genomics</div> 
            , 
            <div class="volume">24</div>
            , 
            <div class="fpage">181</div>
            –
            <div class="lpage">90</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20restriction-modification%20system%20in%20unicellular%20and%20filamentous%20cyanobacteria&amp;author=F.%20Zhao&amp;author=X.%20Zhang&amp;author=C.%20Liang&amp;author=J.%20Wu&amp;author=Q.%20Bao&amp;author=S.%20Qin&amp;publication_year=2006&amp;journal=Physiol.%20Genomics&amp;volume=24&amp;pages=181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/physiolgenomics.00255.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fphysiolgenomics.00255.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fphysiolgenomics.00255.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16368872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Cheevadhanarak</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Paithoonrangsarid</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Prommeenate</div>
               <div class="given-names">P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">Draft genome sequence of Arthrospira platensis C1 (PCC9438)</div>
            . 
            <div class="source">Stand. Genomic Sci</div> 
            ., 
            <div class="volume">6</div>
            , 
            <div class="fpage">43</div>
            –
            <div class="lpage">53</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Arthrospira%20platensis%20C1%20(PCC9438)&amp;author=S.%20Cheevadhanarak&amp;author=K.%20Paithoonrangsarid&amp;author=P.%20Prommeenate&amp;publication_year=2012&amp;journal=Stand.%20Genomic%20Sci&amp;volume=6&amp;pages=43-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4056/sigs.2525955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4056%2Fsigs.2525955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4056%2Fsigs.2525955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22675597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Janssen</div>
               <div class="given-names">P. J.</div>
            </div>
             <div class="name">
              <div class="surname">Morin</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Mergeay</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Genome sequence of the edible cyanobacterium Arthrospira sp. PCC 8005</div>
            . 
            <div class="source">J. Bacteriol</div> 
            ., 
            <div class="volume">192</div>
            , 
            <div class="fpage">2465</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20edible%20cyanobacterium%20Arthrospira%20sp.%20PCC%208005&amp;author=P.%20J.%20Janssen&amp;author=N.%20Morin&amp;author=M.%20Mergeay&amp;publication_year=2010&amp;journal=J.%20Bacteriol&amp;volume=192&amp;pages=2465-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00116-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00116-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00116-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fujisawa</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Narikawa</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Okamoto</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Genomic structure of an economically important cyanobacterium, Arthrospira (Spirulina) platensis NIES-39</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">85</div>
            –
            <div class="lpage">103</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20structure%20of%20an%20economically%20important%20cyanobacterium%2C%20Arthrospira%20(Spirulina)%20platensis%20NIES-39&amp;author=T.%20Fujisawa&amp;author=R.%20Narikawa&amp;author=S.%20Okamoto&amp;publication_year=2010&amp;journal=DNA%20Res&amp;volume=17&amp;pages=85-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bao</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Tian</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">A complete sequence of the T. tengcongensis genome</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">12</div>
            , 
            <div class="fpage">689</div>
            –
            <div class="lpage">700</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complete%20sequence%20of%20the%20T.%20tengcongensis%20genome&amp;author=Q.%20Bao&amp;author=Y.%20Tian&amp;author=W.%20Li&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=689-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.219302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.219302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.219302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11997336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dong</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Xie</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Jiang</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus)</div>
            . 
            <div class="source">Nat. Biotechnol</div> 
            ., 
            <div class="volume">31</div>
            , 
            <div class="fpage">135</div>
            –
            <div class="lpage">41</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20automated%20whole-genome%20optical%20mapping%20of%20the%20genome%20of%20a%20domestic%20goat%20(Capra%20hircus)&amp;author=Y.%20Dong&amp;author=M.%20Xie&amp;author=Y.%20Jiang&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=135-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23263233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gordon</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Abajian</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Green</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Consed: a graphical tool for sequence finishing</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">195</div>
            –
            <div class="lpage">202</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing&amp;author=D.%20Gordon&amp;author=C.%20Abajian&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res&amp;volume=8&amp;pages=195-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">McKenna</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Hanna</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Banks</div>
               <div class="given-names">E.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">20</div>
            , 
            <div class="fpage">1297</div>
            –
            <div class="lpage">303</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Delcher</div>
               <div class="given-names">A. L.</div>
            </div>
             <div class="name">
              <div class="surname">Bratke</div>
               <div class="given-names">K. A.</div>
            </div>
             <div class="name">
              <div class="surname">Powers</div>
               <div class="given-names">E. C.</div>
            </div>
             <div class="name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S. L.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer</div>
            . 
            <div class="source">Bioinformatics</div> 
            , 
            <div class="volume">23</div>
            , 
            <div class="fpage">673</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20bacterial%20genes%20and%20endosymbiont%20DNA%20with%20Glimmer&amp;author=A.%20L.%20Delcher&amp;author=K.%20A.%20Bratke&amp;author=E.%20C.%20Powers&amp;author=S.%20L.%20Salzberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=673-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lagesen</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Hallin</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Rodland</div>
               <div class="given-names">E. A.</div>
            </div>
             <div class="name">
              <div class="surname">Staerfeldt</div>
               <div class="given-names">H. H.</div>
            </div>
             <div class="name">
              <div class="surname">Rognes</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Ussery</div>
               <div class="given-names">D. W.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">35</div>
            , 
            <div class="fpage">3100</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;author=K.%20Lagesen&amp;author=P.%20Hallin&amp;author=E.%20A.%20Rodland&amp;author=H.%20H.%20Staerfeldt&amp;author=T.%20Rognes&amp;author=D.%20W.%20Ussery&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=3100-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lowe</div>
               <div class="given-names">T. M.</div>
            </div>
             <div class="name">
              <div class="surname">Eddy</div>
               <div class="given-names">S. R.</div>
            </div>
          </div>
          <div class="year">1997</div>
            , 
            <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">25</div>
            , 
            <div class="fpage">955</div>
            –
            <div class="lpage">64</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Petkau</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Stuart-Edwards</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Stothard</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Van Domselaar</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Interactive microbial genome visualization with GView</div>
            . 
            <div class="source">Bioinformatics</div> 
            , 
            <div class="volume">26</div>
            , 
            <div class="fpage">3125</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactive%20microbial%20genome%20visualization%20with%20GView&amp;author=A.%20Petkau&amp;author=M.%20Stuart-Edwards&amp;author=P.%20Stothard&amp;author=G.%20Van%20Domselaar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=3125-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq588" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq588" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq588"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20956244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Roberts</div>
               <div class="given-names">R. J.</div>
            </div>
             <div class="name">
              <div class="surname">Vincze</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Posfai</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Macelis</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">REBASE–a database for DNA restriction and modification: enzymes, genes and genomes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">38</div>
            , 
            <div class="fpage">D234</div>
            -
            <div class="lpage">236</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=REBASE%E2%80%93a%20database%20for%20DNA%20restriction%20and%20modification%3A%20enzymes%2C%20genes%20and%20genomes&amp;author=R.%20J.%20Roberts&amp;author=T.%20Vincze&amp;author=J.%20Posfai&amp;author=D.%20Macelis&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=D234-236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp874" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp874" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp874"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19846593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Eisen</div>
               <div class="given-names">J. A.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">A simple, fast, and accurate method of phylogenomic inference</div>
            . 
            <div class="source">Genome Biol</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">R151</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20method%20of%20phylogenomic%20inference&amp;author=M.%20Wu&amp;author=J.%20A.%20Eisen&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R151" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-10-r151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-10-r151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-10-r151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18851752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Guindon</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Dufayard</div>
               <div class="given-names">J. F.</div>
            </div>
             <div class="name">
              <div class="surname">Lefort</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Anisimova</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Hordijk</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Gascuel</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>
            . 
            <div class="source">Syst. Biol</div> 
            ., 
            <div class="volume">59</div>
            , 
            <div class="fpage">307</div>
            -
            <div class="lpage">321</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.%20F.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Syst.%20Biol&amp;volume=59&amp;pages=307-321" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsyq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Siguier</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Perochon</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Lestrade</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Mahillon</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Chandler</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">ISfinder: the reference centre for bacterial insertion sequences</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">34</div>
            , 
            <div class="fpage">D32</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISfinder%3A%20the%20reference%20centre%20for%20bacterial%20insertion%20sequences&amp;author=P.%20Siguier&amp;author=J.%20Perochon&amp;author=L.%20Lestrade&amp;author=J.%20Mahillon&amp;author=M.%20Chandler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D32-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Darling</div>
               <div class="given-names">A. E.</div>
            </div>
             <div class="name">
              <div class="surname">Mau</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Perna</div>
               <div class="given-names">N. T.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement</div>
            . 
            <div class="source">PLoS One</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">e11147</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=progressiveMauve%3A%20multiple%20genome%20alignment%20with%20gene%20gain%2C%20loss%20and%20rearrangement&amp;author=A.%20E.%20Darling&amp;author=B.%20Mau&amp;author=N.%20T.%20Perna&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e11147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0011147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0011147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0011147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20593022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Godzik</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>
            . 
            <div class="source">Bioinformatics</div> 
            , 
            <div class="volume">22</div>
            , 
            <div class="fpage">1658</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Benson</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">27</div>
            , 
            <div class="fpage">573</div>
            –
            <div class="lpage">80</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grissa</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Vergnaud</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Pourcel</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">35</div>
            , 
            <div class="fpage">W52</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRFinder%3A%20a%20web%20tool%20to%20identify%20clustered%20regularly%20interspaced%20short%20palindromic%20repeats&amp;author=I.%20Grissa&amp;author=G.%20Vergnaud&amp;author=C.%20Pourcel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W52-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm360"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kawai</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Higashiura</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Hayasaki</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Complete genome and gene expression analyses of Asaia bogorensis reveal unique responses to culture with mammalian cells as a potential opportunistic human pathogen</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">357</div>
            –
            <div class="lpage">66</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20and%20gene%20expression%20analyses%20of%20Asaia%20bogorensis%20reveal%20unique%20responses%20to%20culture%20with%20mammalian%20cells%20as%20a%20potential%20opportunistic%20human%20pathogen&amp;author=M.%20Kawai&amp;author=N.%20Higashiura&amp;author=K.%20Hayasaki&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=357-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26358298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Achaz</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Coissac</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Netter</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E. P.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Associations between inverted repeats and the structural evolution of bacterial genomes</div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">164</div>
            , 
            <div class="fpage">1279</div>
            –
            <div class="lpage">89</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associations%20between%20inverted%20repeats%20and%20the%20structural%20evolution%20of%20bacterial%20genomes&amp;author=G.%20Achaz&amp;author=E.%20Coissac&amp;author=P.%20Netter&amp;author=E.%20P.%20Rocha&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1279-89" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shih</div>
               <div class="given-names">P. M.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Latifi</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Improving the coverage of the cyanobacterial phylum using diversity-driven genome sequencing</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. U S A</div> 
            , 
            <div class="volume">110</div>
            , 
            <div class="fpage">1053</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20coverage%20of%20the%20cyanobacterial%20phylum%20using%20diversity-driven%20genome%20sequencing&amp;author=P.%20M.%20Shih&amp;author=D.%20Wu&amp;author=A.%20Latifi&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=110&amp;pages=1053-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217107110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217107110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217107110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23277585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Rooney</div>
               <div class="given-names">A. P.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Concerted and birth-and-death evolution of multigene families</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">39</div>
            , 
            <div class="fpage">121</div>
            –
            <div class="lpage">52</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Concerted%20and%20birth-and-death%20evolution%20of%20multigene%20families&amp;author=M.%20Nei&amp;author=A.%20P.%20Rooney&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genet&amp;volume=39&amp;pages=121-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.39.073003.112240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.39.073003.112240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lefort</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Calmin</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Crovadore</div>
               <div class="given-names">J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">Whole-Genome Shotgun Sequence of Arthrospira platensis Strain Paraca, a Cultivated and Edible Cyanobacterium</div>
            . 
            <div class="source">Genome Announc</div> 
            ., 
            <div class="volume">2</div>
            , 
            <div class="fpage">1</div>
            –
            <div class="lpage">2</div>
            .
          </div></div></div></div><div content-id="dsw023-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ochman</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Lawrence</div>
               <div class="given-names">J. G.</div>
            </div>
             <div class="name">
              <div class="surname">Groisman</div>
               <div class="given-names">E. A.</div>
            </div>
          </div>
          <div class="year">2000</div>
            , 
            <div class="article-title">Lateral gene transfer and the nature of bacterial innovation</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">405</div>
            , 
            <div class="fpage">299</div>
            –
            <div class="lpage">304</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lateral%20gene%20transfer%20and%20the%20nature%20of%20bacterial%20innovation&amp;author=H.%20Ochman&amp;author=J.%20G.%20Lawrence&amp;author=E.%20A.%20Groisman&amp;publication_year=2000&amp;journal=Nature&amp;volume=405&amp;pages=299-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35012500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35012500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35012500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10830951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Xiong</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Eickbush</div>
               <div class="given-names">T. H.</div>
            </div>
          </div>
          <div class="year">1990</div>
            , 
            <div class="article-title">Origin and evolution of retroelements based upon their reverse transcriptase sequences</div>
            . 
            <div class="source">EMBO J</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">3353</div>
            –
            <div class="lpage">3362</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20evolution%20of%20retroelements%20based%20upon%20their%20reverse%20transcriptase%20sequences&amp;author=Y.%20Xiong&amp;author=T.%20H.%20Eickbush&amp;publication_year=1990&amp;journal=EMBO%20J&amp;volume=9&amp;pages=3353-3362" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1698615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fassbender</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Bruhl</div>
               <div class="given-names">K. H.</div>
            </div>
             <div class="name">
              <div class="surname">Ciriacy</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Kuck</div>
               <div class="given-names">U.</div>
            </div>
          </div>
          <div class="year">1994</div>
            , 
            <div class="article-title">Reverse transcriptase activity of an intron encoded polypeptide</div>
            . 
            <div class="source">EMBO J</div> 
            ., 
            <div class="volume">13</div>
            , 
            <div class="fpage">2075</div>
            –
            <div class="lpage">83</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20transcriptase%20activity%20of%20an%20intron%20encoded%20polypeptide&amp;author=S.%20Fassbender&amp;author=K.%20H.%20Bruhl&amp;author=M.%20Ciriacy&amp;author=U.%20Kuck&amp;publication_year=1994&amp;journal=EMBO%20J&amp;volume=13&amp;pages=2075-83" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7514530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Johnsrud</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Calos</div>
               <div class="given-names">M. P.</div>
            </div>
             <div class="name">
              <div class="surname">Miller</div>
               <div class="given-names">J. H.</div>
            </div>
          </div>
          <div class="year">1978</div>
            , 
            <div class="article-title">The transposon Tn9 generates a 9 bp repeated sequence during integration</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">15</div>
            , 
            <div class="fpage">1209</div>
            –
            <div class="lpage">19</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transposon%20Tn9%20generates%20a%209%E2%80%89bp%20repeated%20sequence%20during%20integration&amp;author=L.%20Johnsrud&amp;author=M.%20P.%20Calos&amp;author=J.%20H.%20Miller&amp;publication_year=1978&amp;journal=Cell&amp;volume=15&amp;pages=1209-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(78)90047-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(78)90047-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(78)90047-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/365355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Garcia-Migura</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Hasman</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Svendsen</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Jensen</div>
               <div class="given-names">L. B.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Relevance of hot spots in the evolution and transmission of Tn1546 in glycopeptide-resistant Enterococcus faecium (GREF) from broiler origin</div>
            . 
            <div class="source">J. Antimicrob. Chemother</div> 
            ., 
            <div class="volume">62</div>
            , 
            <div class="fpage">681</div>
            -
            <div class="lpage">687</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relevance%20of%20hot%20spots%20in%20the%20evolution%20and%20transmission%20of%20Tn1546%20in%20glycopeptide-resistant%20Enterococcus%20faecium%20(GREF)%20from%20broiler%20origin&amp;author=L.%20Garcia-Migura&amp;author=H.%20Hasman&amp;author=C.%20Svendsen&amp;author=L.%20B.%20Jensen&amp;publication_year=2008&amp;journal=J.%20Antimicrob.%20Chemother&amp;volume=62&amp;pages=681-687" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jac/dkn265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjac%2Fdkn265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjac%2Fdkn265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18583326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Machielsen</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Siezen</div>
               <div class="given-names">R. J.</div>
            </div>
             <div class="name">
              <div class="surname">van Hijum</div>
               <div class="given-names">S. A.</div>
            </div>
             <div class="name">
              <div class="surname">van Hylckama Vlieg</div>
               <div class="given-names">J. E.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Molecular description and industrial potential of Tn6098 conjugative transfer conferring alpha-galactoside metabolism in Lactococcus lactis</div>
            . 
            <div class="source">Appl. Environ. Microbiol</div> 
            ., 
            <div class="volume">77</div>
            , 
            <div class="fpage">555</div>
            -
            <div class="lpage">563</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20description%20and%20industrial%20potential%20of%20Tn6098%20conjugative%20transfer%20conferring%20alpha-galactoside%20metabolism%20in%20Lactococcus%20lactis&amp;author=R.%20Machielsen&amp;author=R.%20J.%20Siezen&amp;author=S.%20A.%20van%20Hijum&amp;author=J.%20E.%20van%20Hylckama%20Vlieg&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=77&amp;pages=555-563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02283-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02283-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02283-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21115709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Roberts</div>
               <div class="given-names">A. P.</div>
            </div>
             <div class="name">
              <div class="surname">Davis</div>
               <div class="given-names">I. J.</div>
            </div>
             <div class="name">
              <div class="surname">Seville</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Villedieu</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Mullany</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Characterization of the ends and target site of a novel tetracycline resistance-encoding conjugative transposon from Enterococcus faecium 664.1H1</div>
            . 
            <div class="source">J. Bacteriol</div> 
            ., 
            <div class="volume">188</div>
            , 
            <div class="fpage">4356</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20ends%20and%20target%20site%20of%20a%20novel%20tetracycline%20resistance-encoding%20conjugative%20transposon%20from%20Enterococcus%20faecium%20664.1H1&amp;author=A.%20P.%20Roberts&amp;author=I.%20J.%20Davis&amp;author=L.%20Seville&amp;author=A.%20Villedieu&amp;author=P.%20Mullany&amp;publication_year=2006&amp;journal=J.%20Bacteriol&amp;volume=188&amp;pages=4356-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00129-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00129-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00129-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16740942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Takahashi</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Ohashi</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Sadykov</div>
               <div class="given-names">M. R.</div>
            </div>
             <div class="name">
              <div class="surname">Mizutani-Ui</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">IS-linked movement of a restriction-modification system</div>
            . 
            <div class="source">PLoS One</div> 
            , 
            <div class="volume">6</div>
            , 
            <div class="fpage">e16554</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IS-linked%20movement%20of%20a%20restriction-modification%20system&amp;author=N.%20Takahashi&amp;author=S.%20Ohashi&amp;author=M.%20R.%20Sadykov&amp;author=Y.%20Mizutani-Ui&amp;author=I.%20Kobayashi&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16554" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016554" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016554" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016554"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21305031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kita</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Kawakami</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Tanaka</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Evidence for horizontal transfer of the EcoT38I restriction-modification gene to chromosomal DNA by the P2 phage and diversity of defective P2 prophages in Escherichia coli TH38 strains</div>
            . 
            <div class="source">J. Bacteriol</div> 
            ., 
            <div class="volume">185</div>
            , 
            <div class="fpage">2296</div>
            –
            <div class="lpage">305</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20horizontal%20transfer%20of%20the%20EcoT38I%20restriction-modification%20gene%20to%20chromosomal%20DNA%20by%20the%20P2%20phage%20and%20diversity%20of%20defective%20P2%20prophages%20in%20Escherichia%20coli%20TH38%20strains&amp;author=K.%20Kita&amp;author=H.%20Kawakami&amp;author=H.%20Tanaka&amp;publication_year=2003&amp;journal=J.%20Bacteriol&amp;volume=185&amp;pages=2296-305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.7.2296-2305.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.7.2296-2305.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.7.2296-2305.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Burrus</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Bontemps</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Decaris</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Guedon</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">Characterization of a novel type II restriction-modification system, Sth368I, encoded by the integrative element ICESt1 of Streptococcus thermophilus CNRZ368</div>
            . 
            <div class="source">Appl. Environ. Microbiol</div> 
            ., 
            <div class="volume">67</div>
            , 
            <div class="fpage">1522</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20novel%20type%20II%20restriction-modification%20system%2C%20Sth368I%2C%20encoded%20by%20the%20integrative%20element%20ICESt1%20of%20Streptococcus%20thermophilus%20CNRZ368&amp;author=V.%20Burrus&amp;author=C.%20Bontemps&amp;author=B.%20Decaris&amp;author=G.%20Guedon&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=67&amp;pages=1522-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.4.1522-1528.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.4.1522-1528.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.67.4.1522-1528.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oliveira</div>
               <div class="given-names">P. H.</div>
            </div>
             <div class="name">
              <div class="surname">Touchon</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E. P.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">The interplay of restriction-modification systems with mobile genetic elements and their prokaryotic hosts</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">42</div>
            , 
            <div class="fpage">10618</div>
            –
            <div class="lpage">31</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20interplay%20of%20restriction-modification%20systems%20with%20mobile%20genetic%20elements%20and%20their%20prokaryotic%20hosts&amp;author=P.%20H.%20Oliveira&amp;author=M.%20Touchon&amp;author=E.%20P.%20Rocha&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=10618-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25120263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Suyama</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Bork</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">Evolution of prokaryotic gene order: genome rearrangements in closely related species</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">10</div>
            –
            <div class="lpage">3</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20prokaryotic%20gene%20order%3A%20genome%20rearrangements%20in%20closely%20related%20species&amp;author=M.%20Suyama&amp;author=P.%20Bork&amp;publication_year=2001&amp;journal=Trends%20Genet&amp;volume=17&amp;pages=10-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02159-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02159-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02159-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11163906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gelfand</div>
               <div class="given-names">M. S.</div>
            </div>
             <div class="name">
              <div class="surname">Koonin</div>
               <div class="given-names">E. V.</div>
            </div>
          </div>
          <div class="year">1997</div>
            , 
            <div class="article-title">Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">25</div>
            , 
            <div class="fpage">2430</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avoidance%20of%20palindromic%20words%20in%20bacterial%20and%20archaeal%20genomes%3A%20a%20close%20connection%20with%20restriction%20enzymes&amp;author=M.%20S.%20Gelfand&amp;author=E.%20V.%20Koonin&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=2430-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.12.2430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.12.2430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.12.2430"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9171096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Cornet</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Hallet</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Sherratt</div>
               <div class="given-names">D. J.</div>
            </div>
          </div>
          <div class="year">1997</div>
            , 
            <div class="article-title">Xer recombination in Escherichia coli. Site-specific DNA topoisomerase activity of the XerC and XerD recombinases</div>
            . 
            <div class="source">J. Biol. Chem</div> 
            ., 
            <div class="volume">272</div>
            , 
            <div class="fpage">21927</div>
            –
            <div class="lpage">31</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Xer%20recombination%20in%20Escherichia%20coli.%20Site-specific%20DNA%20topoisomerase%20activity%20of%20the%20XerC%20and%20XerD%20recombinases&amp;author=F.%20Cornet&amp;author=B.%20Hallet&amp;author=D.%20J.%20Sherratt&amp;publication_year=1997&amp;journal=J.%20Biol.%20Chem&amp;volume=272&amp;pages=21927-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.272.35.21927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.35.21927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.272.35.21927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9268326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Colloms</div>
               <div class="given-names">S. D.</div>
            </div>
             <div class="name">
              <div class="surname">McCulloch</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Grant</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Neilson</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Sherratt</div>
               <div class="given-names">D. J.</div>
            </div>
          </div>
          <div class="year">1996</div>
            , 
            <div class="article-title">Xer-mediated site-specific recombination in vitro</div>
            . 
            <div class="source">EMBO J</div> 
            ., 
            <div class="volume">15</div>
            , 
            <div class="fpage">1172</div>
            –
            <div class="lpage">81</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Xer-mediated%20site-specific%20recombination%20in%20vitro&amp;author=S.%20D.%20Colloms&amp;author=R.%20McCulloch&amp;author=K.%20Grant&amp;author=L.%20Neilson&amp;author=D.%20J.%20Sherratt&amp;publication_year=1996&amp;journal=EMBO%20J&amp;volume=15&amp;pages=1172-81" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8605888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Roberts</div>
               <div class="given-names">R. J.</div>
            </div>
             <div class="name">
              <div class="surname">Belfort</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Bestor</div>
               <div class="given-names">T.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2003</div>
            , 
            <div class="article-title">A nomenclature for restriction enzymes, DNA methyltransferases, homing endonucleases and their genes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">31</div>
            , 
            <div class="fpage">1805</div>
            –
            <div class="lpage">12</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20nomenclature%20for%20restriction%20enzymes%2C%20DNA%20methyltransferases%2C%20homing%20endonucleases%20and%20their%20genes&amp;author=R.%20J.%20Roberts&amp;author=M.%20Belfort&amp;author=T.%20Bestor&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=1805-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg274"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wilson</div>
               <div class="given-names">G. G.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">N. E.</div>
            </div>
          </div>
          <div class="year">1991</div>
            , 
            <div class="article-title">Restriction and modification systems</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">25</div>
            , 
            <div class="fpage">585</div>
            –
            <div class="lpage">627</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Restriction%20and%20modification%20systems&amp;author=G.%20G.%20Wilson&amp;author=N.%20E.%20Murray&amp;publication_year=1991&amp;journal=Annu.%20Rev.%20Genet&amp;volume=25&amp;pages=585-627" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.25.120191.003101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.25.120191.003101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.25.120191.003101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1812816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sugita</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Ogata</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Shikata</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">Complete nucleotide sequence of the freshwater unicellular cyanobacterium Synechococcus elongatus PCC 6301 chromosome: gene content and organization</div>
            . 
            <div class="source">Photosynth Res</div> 
            ., 
            <div class="volume">93</div>
            , 
            <div class="fpage">55</div>
            –
            <div class="lpage">67</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20nucleotide%20sequence%20of%20the%20freshwater%20unicellular%20cyanobacterium%20Synechococcus%20elongatus%20PCC%206301%20chromosome%3A%20gene%20content%20and%20organization&amp;author=C.%20Sugita&amp;author=K.%20Ogata&amp;author=M.%20Shikata&amp;publication_year=2007&amp;journal=Photosynth%20Res&amp;volume=93&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11120-006-9122-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11120-006-9122-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11120-006-9122-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17211581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bandyopadhyay</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Elvitigala</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Welsh</div>
               <div class="given-names">E.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Novel metabolic attributes of the genus cyanothece, comprising a group of unicellular nitrogen-fixing Cyanothece</div>
            . 
            <div class="source">MBio</div> 
            , 
            <div class="volume">2</div>
            , 
            <div class="fpage">e00214</div>
            –
            <div class="lpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20metabolic%20attributes%20of%20the%20genus%20cyanothece%2C%20comprising%20a%20group%20of%20unicellular%20nitrogen-fixing%20Cyanothece&amp;author=A.%20Bandyopadhyay&amp;author=T.%20Elvitigala&amp;author=E.%20Welsh&amp;publication_year=2011&amp;journal=MBio&amp;volume=2&amp;pages=e00214-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/mBio.00214-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.00214-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FmBio.00214-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21972240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Parkhill</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Sebaihia</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Preston</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2003</div>
            , 
            <div class="article-title">Comparative analysis of the genome sequences of Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica</div>
            . 
            <div class="source">Nat. Genet</div> 
            ., 
            <div class="volume">35</div>
            , 
            <div class="fpage">32</div>
            –
            <div class="lpage">40</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20genome%20sequences%20of%20Bordetella%20pertussis%2C%20Bordetella%20parapertussis%20and%20Bordetella%20bronchiseptica&amp;author=J.%20Parkhill&amp;author=M.%20Sebaihia&amp;author=A.%20Preston&amp;publication_year=2003&amp;journal=Nat.%20Genet&amp;volume=35&amp;pages=32-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12910271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E. P.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">DNA repeats lead to the accelerated loss of gene order in bacteria</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">19</div>
            , 
            <div class="fpage">600</div>
            –
            <div class="lpage">3</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20repeats%20lead%20to%20the%20accelerated%20loss%20of%20gene%20order%20in%20bacteria&amp;author=E.%20P.%20Rocha&amp;publication_year=2003&amp;journal=Trends%20Genet&amp;volume=19&amp;pages=600-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.09.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.09.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.09.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14585609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">van Belkum</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Scherer</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">van Alphen</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Verbrugh</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Short-sequence DNA repeats in prokaryotic genomes</div>
            . 
            <div class="source">Microbiol. Mol. Biol. Rev</div> 
            ., 
            <div class="volume">62</div>
            , 
            <div class="fpage">275</div>
            –
            <div class="lpage">93</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short-sequence%20DNA%20repeats%20in%20prokaryotic%20genomes&amp;author=A.%20van%20Belkum&amp;author=S.%20Scherer&amp;author=L.%20van%20Alphen&amp;author=H.%20Verbrugh&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=62&amp;pages=275-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9618442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Moxon</div>
               <div class="given-names">E. R.</div>
            </div>
             <div class="name">
              <div class="surname">Rainey</div>
               <div class="given-names">P. B.</div>
            </div>
             <div class="name">
              <div class="surname">Nowak</div>
               <div class="given-names">M. A.</div>
            </div>
             <div class="name">
              <div class="surname">Lenski</div>
               <div class="given-names">R. E.</div>
            </div>
          </div>
          <div class="year">1994</div>
            , 
            <div class="article-title">Adaptive evolution of highly mutable loci in pathogenic bacteria</div>
            . 
            <div class="source">Curr. Biol</div> 
            ., 
            <div class="volume">4</div>
            , 
            <div class="fpage">24</div>
            –
            <div class="lpage">33</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20evolution%20of%20highly%20mutable%20loci%20in%20pathogenic%20bacteria&amp;author=E.%20R.%20Moxon&amp;author=P.%20B.%20Rainey&amp;author=M.%20A.%20Nowak&amp;author=R.%20E.%20Lenski&amp;publication_year=1994&amp;journal=Curr.%20Biol&amp;volume=4&amp;pages=24-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(00)00005-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(00)00005-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(00)00005-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7922307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E. P.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Order and disorder in bacterial genomes</div>
            . 
            <div class="source">Curr. Opin. Microbiol</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">519</div>
            –
            <div class="lpage">27</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Order%20and%20disorder%20in%20bacterial%20genomes&amp;author=E.%20P.%20Rocha&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=7&amp;pages=519-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2004.08.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2004.08.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2004.08.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15451508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rebollo</div>
               <div class="given-names">J. E.</div>
            </div>
             <div class="name">
              <div class="surname">Francois</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Louarn</div>
               <div class="given-names">J. M.</div>
            </div>
          </div>
          <div class="year">1988</div>
            , 
            <div class="article-title">Detection and possible role of two large nondivisible zones on the Escherichia coli chromosome</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. U S A</div> 
            , 
            <div class="volume">85</div>
            , 
            <div class="fpage">9391</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20possible%20role%20of%20two%20large%20nondivisible%20zones%20on%20the%20Escherichia%20coli%20chromosome&amp;author=J.%20E.%20Rebollo&amp;author=V.%20Francois&amp;author=J.%20M.%20Louarn&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=85&amp;pages=9391-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.24.9391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.24.9391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.24.9391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3059345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Campo</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Dias</div>
               <div class="given-names">M. J.</div>
            </div>
             <div class="name">
              <div class="surname">Daveran-Mingot</div>
               <div class="given-names">M. L.</div>
            </div>
             <div class="name">
              <div class="surname">Ritzenthaler</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Le Bourgeois</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Chromosomal constraints in Gram-positive bacteria revealed by artificial inversions</div>
            . 
            <div class="source">Mol. Microbiol</div> 
            ., 
            <div class="volume">51</div>
            , 
            <div class="fpage">511</div>
            –
            <div class="lpage">22</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20constraints%20in%20Gram-positive%20bacteria%20revealed%20by%20artificial%20inversions&amp;author=N.%20Campo&amp;author=M.%20J.%20Dias&amp;author=M.%20L.%20Daveran-Mingot&amp;author=P.%20Ritzenthaler&amp;author=P.%20Le%20Bourgeois&amp;publication_year=2004&amp;journal=Mol.%20Microbiol&amp;volume=51&amp;pages=511-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03847.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03847.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03847.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14756790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Garcia-Russell</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Harmon</div>
               <div class="given-names">T. G.</div>
            </div>
             <div class="name">
              <div class="surname">Le</div>
               <div class="given-names">T. Q.</div>
            </div>
             <div class="name">
              <div class="surname">Amaladas</div>
               <div class="given-names">N. H.</div>
            </div>
             <div class="name">
              <div class="surname">Mathewson</div>
               <div class="given-names">R. D.</div>
            </div>
             <div class="name">
              <div class="surname">Segall</div>
               <div class="given-names">A. M.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Unequal access of chromosomal regions to each other in Salmonella: probing chromosome structure with phage lambda integrase-mediated long-range rearrangements</div>
            . 
            <div class="source">Mol. Microbiol</div> 
            ., 
            <div class="volume">52</div>
            , 
            <div class="fpage">329</div>
            –
            <div class="lpage">44</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unequal%20access%20of%20chromosomal%20regions%20to%20each%20other%20in%20Salmonella%3A%20probing%20chromosome%20structure%20with%20phage%20lambda%20integrase-mediated%20long-range%20rearrangements&amp;author=N.%20Garcia-Russell&amp;author=T.%20G.%20Harmon&amp;author=T.%20Q.%20Le&amp;author=N.%20H.%20Amaladas&amp;author=R.%20D.%20Mathewson&amp;author=A.%20M.%20Segall&amp;publication_year=2004&amp;journal=Mol.%20Microbiol&amp;volume=52&amp;pages=329-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.03976.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.03976.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.03976.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15066024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bench</div>
               <div class="given-names">S. R.</div>
            </div>
             <div class="name">
              <div class="surname">Ilikchyan</div>
               <div class="given-names">I. N.</div>
            </div>
             <div class="name">
              <div class="surname">Tripp</div>
               <div class="given-names">H. J.</div>
            </div>
             <div class="name">
              <div class="surname">Zehr</div>
               <div class="given-names">J. P.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Two strains of Crocosphaera watsonii with highly conserved genomes Eare distinguished by strain-specific features</div>
            . 
            <div class="source">Front. Microbiol</div> 
            ., 
            <div class="volume">2</div>
            , 
            <div class="fpage">261</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20strains%20of%20Crocosphaera%20watsonii%20with%20highly%20conserved%20genomes%20Eare%20distinguished%20by%20strain-specific%20features&amp;author=S.%20R.%20Bench&amp;author=I.%20N.%20Ilikchyan&amp;author=H.%20J.%20Tripp&amp;author=J.%20P.%20Zehr&amp;publication_year=2011&amp;journal=Front.%20Microbiol&amp;volume=2&amp;pages=261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2011.00261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2011.00261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2011.00261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22232617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nakamura</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">Complete genome structure of the thermophilic cyanobacterium Thermosynechococcus elongatus BP-1</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">123</div>
            –
            <div class="lpage">30</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20structure%20of%20the%20thermophilic%20cyanobacterium%20Thermosynechococcus%20elongatus%20BP-1&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;publication_year=2002&amp;journal=DNA%20Res&amp;volume=9&amp;pages=123-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.4.123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.4.123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.4.123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12240834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Frangeul</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Quillardet</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Castets</div>
               <div class="given-names">A. M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">Highly plastic genome of Microcystis aeruginosa PCC 7806, a ubiquitous toxic freshwater cyanobacterium</div>
            . 
            <div class="source">BMC Genomics</div> 
            , 
            <div class="volume">9</div>
            , 
            <div class="fpage">274</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20plastic%20genome%20of%20Microcystis%20aeruginosa%20PCC%207806%2C%20a%20ubiquitous%20toxic%20freshwater%20cyanobacterium&amp;author=L.%20Frangeul&amp;author=P.%20Quillardet&amp;author=A.%20M.%20Castets&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-274"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18534010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Nakajima</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Okamoto</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">Complete genomic structure of the bloom-forming toxic cyanobacterium Microcystis aeruginosa NIES-843</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">14</div>
            , 
            <div class="fpage">247</div>
            –
            <div class="lpage">56</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genomic%20structure%20of%20the%20bloom-forming%20toxic%20cyanobacterium%20Microcystis%20aeruginosa%20NIES-843&amp;author=T.%20Kaneko&amp;author=N.%20Nakajima&amp;author=S.%20Okamoto&amp;publication_year=2007&amp;journal=DNA%20Res&amp;volume=14&amp;pages=247-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">van Belkum</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Tracing isolates of bacterial species by multilocus variable number of tandem repeat analysis (MLVA)</div>
            . 
            <div class="source">FEMS Immunol. Med. Microbiol</div> 
            ., 
            <div class="volume">49</div>
            , 
            <div class="fpage">22</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracing%20isolates%20of%20bacterial%20species%20by%20multilocus%20variable%20number%20of%20tandem%20repeat%20analysis%20(MLVA)&amp;author=A.%20van%20Belkum&amp;publication_year=2007&amp;journal=FEMS%20Immunol.%20Med.%20Microbiol&amp;volume=49&amp;pages=22-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-695X.2006.00173.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-695X.2006.00173.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-695X.2006.00173.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17266711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lopes</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Ribeyre</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Nicolas</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Complex minisatellite rearrangements generated in the total or partial absence of Rad27/hFEN1 activity occur in a single generation and are Rad51 and Rad52 dependent</div>
            . 
            <div class="source">Mol. Cell Biol</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">6675</div>
            –
            <div class="lpage">89</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20minisatellite%20rearrangements%20generated%20in%20the%20total%20or%20partial%20absence%20of%20Rad27%2FhFEN1%20activity%20occur%20in%20a%20single%20generation%20and%20are%20Rad51%20and%20Rad52%20dependent&amp;author=J.%20Lopes&amp;author=C.%20Ribeyre&amp;author=A.%20Nicolas&amp;publication_year=2006&amp;journal=Mol.%20Cell%20Biol&amp;volume=26&amp;pages=6675-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.00649-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.00649-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.00649-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16914748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bhaya</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Davison</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Barrangou</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">CRISPR-Cas systems in bacteria and archaea: versatile small RNAs for adaptive defense and regulation</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">45</div>
            , 
            <div class="fpage">273</div>
            –
            <div class="lpage">297</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR-Cas%20systems%20in%20bacteria%20and%20archaea%3A%20versatile%20small%20RNAs%20for%20adaptive%20defense%20and%20regulation&amp;author=D.%20Bhaya&amp;author=M.%20Davison&amp;author=R.%20Barrangou&amp;publication_year=2011&amp;journal=Annu.%20Rev.%20Genet&amp;volume=45&amp;pages=273-297" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110410-132430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110410-132430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110410-132430"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22060043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw023-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw023-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sander</div>
               <div class="given-names">J. D.</div>
            </div>
             <div class="name">
              <div class="surname">Joung</div>
               <div class="given-names">J. K.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">CRISPR-Cas systems for editing, regulating and targeting genomes</div>
            . 
            <div class="source">Nat. Biotechnol</div> 
            ., 
            <div class="volume">32</div>
            , 
            <div class="fpage">347</div>
            –
            <div class="lpage">55</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR-Cas%20systems%20for%20editing%2C%20regulating%20and%20targeting%20genomes&amp;author=J.%20D.%20Sander&amp;author=J.%20K.%20Joung&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol&amp;volume=32&amp;pages=347-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24584096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a> ), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469971&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw023/2/dsw023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=LSbd8JAP2Sbh015jraNLinEke00UVaOJEDL5aHh57rC0bJCs~Y8Z~Cx1Hsyv8guOwfzvvyfgrvIFVRs3ivLBLk7cKrrlaRvUdMxPwIBYYbu~SxZU9Slppsn~pG~OMI1ITCxXrWOdU9lOFvhI6Mk11DtWueb1uoh2vpgPw9XqycGb4Np5PeRCPqcpCVMyglaL4rOj5HwjZaeYyrqPVABjTUGzDaaRvMxAa~39Gqdf6ccl4Y1j4h0lV-flNyIDZJDhW2dXcqKmCgamUVvFlb1eJlkGzYMIam-kOWUnNgNuF0aIWZKLd1dYIjN9p3CK-yy9~M1Ww~iIyz3VehxuMVSkMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">922</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469971"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f325%2f2469971">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000384768700003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F325%2F2469971" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469971'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" Whole genomic DNA sequencing and comparative genomic analysis of Arthrospira platensis : high genome plasticity and genetic diversity " />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f325%2f2469971">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

