

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/4/339/353499","@type":"ScholarlyArticle","name":"Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to  Sinorhizobium meliloti  2011","datePublished":"2013-04-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2013-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst014","keywords":["genome annotation","prokaryotes","RNA-Seq","rhizobium"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/4/339/353499","author":[{"name":"Sallet, Erika","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Roux, Brice","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Sauviac, Laurent","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Jardinaud, Marie-Franc¸oise","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France    3  INPT-Universite de TOULOUSE ,  ENSAT- Avenue de l'Agrobiopole ,  Auzeville-Tolosane-31326 - Castanet-Tolosan Cédex","@type":"Person"},{"name":"Carrère, Sébastien","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Faraut, Thomas","affiliation":"4  INRA ,  Laboratoire de Génétique Cellulaire, UMR 444 ,  Castanet-Tolosan F-31326 ,  France    5  ENVT ,  Laboratoire de Génétique Cellulaire, UMR 444 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"de Carvalho-Niebel, Fernanda","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Gouzy, Jérôme","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Gamas, Pascal","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Capela, Delphine","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Bruand, Claude","affiliation":"1  INRA ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441 ,  Castanet-Tolosan F-31326 ,  France    2  CNRS ,  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"},{"name":"Schiex, Thomas","affiliation":"6  INRA ,  Unité de Biométrie et d'Intelligence Artificielle, UR 875 ,  Castanet-Tolosan F-31326 ,  France","@type":"Person"}],"description":"Abstract.  The availability of next-generation sequences of transcripts from prokaryotic organisms offers the opportunity to design a new generation of automate","pageStart":"339","pageEnd":"354","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to  Sinorhizobium meliloti  2011","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to  Sinorhizobium meliloti  2011" />
    <meta property="og:description" content="Abstract.  The availability of next-generation sequences of transcripts from prokaryotic organisms offers the opportunity to design a new generation of automate" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/4/339/353499" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-04-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sallet, Erika" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Roux, Brice" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Sauviac, Laurent" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Jardinaud, Marie-Franc¸oise" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="INPT-Universite de TOULOUSE, ENSAT- Avenue de l'Agrobiopole, Auzeville-Tolosane-31326 - Castanet-Tolosan Cédex" /><meta name="citation_author" content="Carrère, Sébastien" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Faraut, Thomas" /><meta name="citation_author_institution" content="INRA, Laboratoire de Génétique Cellulaire, UMR 444, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="ENVT, Laboratoire de Génétique Cellulaire, UMR 444, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="de Carvalho-Niebel, Fernanda" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Gouzy, Jérôme" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Gamas, Pascal" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Capela, Delphine" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Bruand, Claude" /><meta name="citation_author_institution" content="INRA, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441, Castanet-Tolosan F-31326, France" /><meta name="citation_author_institution" content="CNRS, Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594, Castanet-Tolosan F-31326, France" /><meta name="citation_author" content="Schiex, Thomas" /><meta name="citation_author_institution" content="INRA, Unité de Biométrie et d'Intelligence Artificielle, UR 875, Castanet-Tolosan F-31326, France" /><meta name="citation_title" content="Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to Sinorhizobium meliloti 2011" /><meta name="citation_firstpage" content="339" /><meta name="citation_lastpage" content="354" /><meta name="citation_doi" content="10.1093/dnares/dst014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2013/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Structural and operational complexity of the Geobacter sulfurreducens genome; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1304-11; " /><meta name="citation_reference" content="citation_title=A high resolution map of a cyanobacterial transcriptome; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R47" /><meta name="citation_reference" content="citation_title=Pseudomonas putida KT2440 genome update by cDNA sequencing and microarray transcriptomics; Environ. Microbiol.;  citation_year=2011;  citation_volume=13;  citation_pages=1309-26; " /><meta name="citation_reference" content="citation_title=Sequencing illustrates the transcriptional response of Legionella pneumophila during infection and identifies seventy novel small non-coding RNAs; PloS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e17570" /><meta name="citation_reference" content="citation_title=The automatic annotation of bacterial genomes; Brief Bioinform.;  citation_year=2013;  citation_volume=14;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-41; " /><meta name="citation_reference" content="citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=673-79; " /><meta name="citation_reference" content="citation_title=Computational approaches to gene prediction; J. Microbiol.;  citation_year=2006;  citation_volume=44;  citation_pages=137-44; " /><meta name="citation_reference" content="citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=119" /><meta name="citation_reference" content="citation_title=The RAST Server: rapid annotations using subsystems technology; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=75" /><meta name="citation_reference" content="citation_title=BG7: a new approach for bacterial genome annotation designed for next generation sequencing data; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e49239" /><meta name="citation_reference" content="citation_title=VESPA: software to facilitate genomic annotation of prokaryotic organisms through integration of proteomic and transcriptomic data; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=131" /><meta name="citation_reference" content="citation_title=MicroScope—an integrated microbial resource for the curation and comparative analysis of genomic and metabolic data; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D636-47; " /><meta name="citation_reference" content="citation_title=EuGene: an eukaryotic gene finder that combines several sources of evidence; citation_publisher=Springer, Berlin, Germany; Computational biology;  citation_year=2001;  citation_pages=111-25; " /><meta name="citation_reference" content="citation_title=AUGUSTUS: a web server for gene finding in eukaryotes; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=W309-12; " /><meta name="citation_reference" content="citation_title=Computational inference of homologous gene structures in the human genome,; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=803-16; " /><meta name="citation_reference" content="citation_title=SGP-1: prediction and validation of homologous genes based on sequence alignments; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1574-83; " /><meta name="citation_reference" content="citation_title=Comparative gene prediction in human and mouse; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=108-17; " /><meta name="citation_reference" content="citation_title=EuGene'Hom: a generic similarity-based gene finder using multiple homologous sequences; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3742-45; " /><meta name="citation_reference" content="citation_title=Genome annotation in plants and fungi: EuGene as a model platform; Curr. Bioinformatics;  citation_year=2008;  citation_volume=3;  citation_pages=87-97; " /><meta name="citation_reference" content="citation_title=Genome sequence of the metazoan plant–parasitic nematode Meloidogyne incognita; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=909-15; " /><meta name="citation_reference" content="citation_title=The genome of Laccaria bicolor provides insights into mycorrhizal symbiosis; Nature;  citation_year=2008;  citation_volume=452;  citation_pages=88-92; " /><meta name="citation_reference" content="citation_title=The Ectocarpus genome and the independent evolution of multicellularity in brown algae; Nature;  citation_year=2010;  citation_volume=465;  citation_pages=617-21; " /><meta name="citation_reference" content="citation_title=The genome of Tetranychus urticae reveals herbivorous pest adaptations; Nature;  citation_year=2011;  citation_volume=479;  citation_pages=487-92; " /><meta name="citation_reference" content="citation_title=The genome of Theobroma cacao; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=101-08; " /><meta name="citation_reference" content="citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses; Nature;  citation_year=2011;  citation_volume=480;  citation_pages=520-24; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Transcription reprogramming during root nodule development in Medicago truncatula; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e16463" /><meta name="citation_reference" content="citation_title=Differentiation of symbiotic cells and endosymbionts in Medicago truncatula nodulation are coupled to two transcriptome-switches; PloS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e9519" /><meta name="citation_reference" content="citation_title=Sinorhizobium meliloti differentiation during symbiosis with alfalfa: a transcriptomic dissection; Mol. Plant Microbe Interact.;  citation_year=2006;  citation_volume=19;  citation_pages=363-72; " /><meta name="citation_reference" content="citation_title=The composite genome of the legume symbiont Sinorhizobium meliloti; Science;  citation_year=2001;  citation_volume=293;  citation_pages=668-72; " /><meta name="citation_reference" content="citation_title=Physical and genetic characterization of symbiotic and auxotrophic mutants of Rhizobium meliloti induced by transposon Tn5 mutagenesis; J. Bacteriol.;  citation_year=1982;  citation_volume=149;  citation_pages=114-22; " /><meta name="citation_reference" content="citation_title=Analysis of differences between Sinorhizobium meliloti 1021 and 2011 strains using the host calcium spiking response; Mol. Plant Microbe Interact.;  citation_year=2002;  citation_volume=15;  citation_pages=1245-52; " /><meta name="citation_reference" content="citation_title=Global transcriptional analysis of the phosphate starvation response in Sinorhizobium meliloti strains 1021 and 2011; Mol. Genet. Genomics;  citation_year=2004;  citation_volume=272;  citation_pages=1-17; " /><meta name="citation_reference" content="citation_title=Regulation and properties of PstSCAB, a high-affinity, high-velocity phosphate transport system of Sinorhizobium meliloti; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=1089-102; " /><meta name="citation_reference" content="citation_title=The model legume Medicago truncatula A17 is poorly matched for N2 fixation with the sequenced microsymbiont Sinorhizobium meliloti 1021; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=62-6; " /><meta name="citation_reference" content="citation_title=Investigations of Rhizobium biofilm formation; FEMS Microbiol. Ecol.;  citation_year=2006;  citation_volume=56;  citation_pages=195-206; " /><meta name="citation_reference" content="citation_title=Auxotrophy accounts for nodulation defect of most Sinorhizobium meliloti mutants in the branched-chain amino acid biosynthesis pathway; Mol. Plant Microbe Interact.;  citation_year=2008;  citation_volume=21;  citation_pages=1232-41; " /><meta name="citation_reference" content="citation_title=An extracytoplasmic function sigma factor acts as a general stress response regulator in Sinorhizobium meliloti; J. Bacteriol.;  citation_year=2007;  citation_volume=189;  citation_pages=4204-16; " /><meta name="citation_reference" content="citation_title=Dual control of Sinorhizobium meliloti RpoE2 sigma factor activity by two PhyR-type two-component response regulators; J. Bacteriol.;  citation_year=2010;  citation_volume=192;  citation_pages=2255-65; " /><meta name="citation_reference" content="citation_title=Growing Medicago truncatula: choice of substrates and growth conditions; citation_text=Noble foundation. ISBN 0-9754303-1-9. http://www.noble.org/MedicagoHandbook.; The Medicago truncatula handbook;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Medicago truncatula ENOD11: a novel RPRP-encoding early nodulin gene expressed during mycorrhization in arbuscule-containing cells; Mol. Plant Microbe Interact.;  citation_year=2001;  citation_volume=14;  citation_pages=737-48; " /><meta name="citation_reference" content="citation_publisher=Morgan Kaufmann Publishers Inc., Williams College, ;  citation_year=2001;  citation_pages=282-9; " /><meta name="citation_reference" content="citation_title=Global discriminative learning for higher-accuracy computational gene prediction; PLoS Comput. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=e54" /><meta name="citation_reference" content="citation_title=Conrad: gene prediction using conditional random fields; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1389-98; " /><meta name="citation_reference" content="citation_title=Differential expression analysis for sequence count data; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R106" /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate—a practical and powerful approach to multiple testing; J. R. Stat. Soc. B;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets; Cancer Res.;  citation_year=2004;  citation_volume=64;  citation_pages=5245-50; " /><meta name="citation_reference" content="citation_title=Evaluating bacterial gene-finding HMM structures as probabilistic logic programs; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=636-42; " /><meta name="citation_reference" content="citation_title=Carbamoyl-phosphate synthetase—creation of an escape route for ammonia; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=39722-7; " /><meta name="citation_reference" content="citation_title=FixJ-regulated genes evolved through promoter duplication in Sinorhizobium meliloti; Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=2335-45; " /><meta name="citation_reference" content="citation_title=Correlation between ultrastructural differentiation of bacteroids and nitrogen fixation in alfalfa nodules; J. Bacteriol.;  citation_year=1990;  citation_volume=172;  citation_pages=4295-306; " /><meta name="citation_reference" content="citation_title=Strand-specific deep sequencing of the transcriptome; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=989-99; " /><meta name="citation_reference" content="citation_title=Comprehensive comparative analysis of strand-specific RNA sequencing methods; Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=709-U767; " /><meta name="citation_reference" content="citation_title=Apollo: a sequence annotation editor; Genome Biol.;  citation_year=2002;  citation_volume=3;  citation_pages=Research0082" /><meta name="citation_reference" content="citation_title=A genome-wide survey of sRNAs in the symbiotic nitrogen-fixing alpha-proteobacterium Sinorhizobium meliloti; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=245" /><meta name="citation_reference" content="citation_title=Analysis of the regulated transcriptome of Neisseria meningitidis in human blood using a tiling array; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=6217-32; " /><meta name="citation_reference" content="citation_title=Regulatory mechanisms employed by cis-encoded antisense RNAs; Curr. Opin. Microbiol.;  citation_year=2007;  citation_volume=10;  citation_pages=102-9; " /><meta name="citation_reference" content="citation_title=Molecular and expression analysis of the Rhizobium meliloti phosphoenolpyruvate carboxykinase (pckA) gene; J. Bacteriol.;  citation_year=1995;  citation_volume=177;  citation_pages=1452-60; " /><meta name="citation_reference" content="citation_title=Presence of a gene encoding choline sulfatase in Sinorhizobium meliloti bet operon: choline-O-sulfate is metabolized into glycine betaine; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=11394-9; " /><meta name="citation_reference" content="citation_title=Analysis of the chromosome sequence of the legume symbiont Sinorhizobium meliloti strain 1021; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=9877-82; " /><meta name="citation_reference" content="citation_title=Mechanisms for activating bacterial RNA polymerase; FEMS Microbiol. Rev.;  citation_year=2010;  citation_volume=34;  citation_pages=611-27; " /><meta name="citation_reference" content="citation_title=Rhizobium meliloti ntrA (rpoN) gene is required for diverse metabolic functions; J. Bacteriol.;  citation_year=1987;  citation_volume=169;  citation_pages=2424-31; " /><meta name="citation_reference" content="citation_title=The role of sigma factor RpoH1 in the pH stress response of Sinorhizobium meliloti; BMC Microbiol.;  citation_year=2010;  citation_volume=10;  citation_pages=265" /><meta name="citation_reference" content="citation_title=Dual RpoH sigma factors and transcriptional plasticity in a symbiotic bacterium; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=4983-94; " /><meta name="citation_reference" content="citation_title=Sinorhizobium meliloti sigma factors RpoE1 and RpoE4 are activated in stationary phase in response to sulfite; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e50768" /><meta name="citation_reference" content="citation_title=Promoter prediction in the rhizobia; Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=1751-63; " /><meta name="citation_reference" content="citation_title=The transcription unit architecture of the Escherichia coli genome; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=1043-U115; " /><meta name="citation_reference" content="citation_title=Deep sequencing uncovers numerous small RNAs on all four replicons of the plant pathogen Agrobacterium tumefaciens; RNA Biol.;  citation_year=2012;  citation_volume=9;  citation_pages=446-57; " /><meta name="citation_reference" content="citation_title=Sinorhizobium meliloti rpoE2 is necessary for H(2)O(2) stress resistance during the stationary growth phase; FEMS Microbiol. Lett.;  citation_year=2009;  citation_volume=290;  citation_pages=25-31; " /><meta name="citation_reference" content="citation_title=RpoE2 of Sinorhizobium meliloti is necessary for trehalose synthesis and growth in hyperosmotic media; Microbiology;  citation_year=2010;  citation_volume=156;  citation_pages=1708-18; " /><meta name="citation_reference" content="citation_title=Identification of differentially expressed small non-coding RNAs in the legume endosymbiont Sinorhizobium meliloti by comparative genomics; Mol. Microbiol.;  citation_year=2007;  citation_volume=66;  citation_pages=1080-91; " /><meta name="citation_reference" content="citation_title=Identification of chromosomal alpha-proteobacterial small RNAs by comparative genome analysis and detection in Sinorhizobium meliloti strain 1021; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=467" /><meta name="citation_reference" content="citation_title=Prediction of Sinorhizobium meliloti sRNA genes and experimental detection in strain 2011; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=416" /><meta name="citation_reference" content="citation_title=The expression of a novel antisense gene mediates incompatibility within the large repABC family of alpha-proteobacterial plasmids; Mol. Microbiol.;  citation_year=2005;  citation_volume=55;  citation_pages=611-23; " /><meta name="citation_reference" content="citation_title=Transcription attenuation associated with bacterial repetitive extragenic BIME elements; J. Mol. Biol.;  citation_year=2001;  citation_volume=314;  citation_pages=375-86; " /><meta name="citation_reference" content="citation_title=The RNA degradosome and poly(A) polymerase of Escherichia coli are required in vivo for the degradation of small mRNA decay intermediates containing REP-stabilizers; Mol. Microbiol.;  citation_year=2004;  citation_volume=51;  citation_pages=777-90; " /><meta name="citation_reference" content="citation_title=Stability of the pstS transcript of Escherichia coli; Arch. Microbiol.;  citation_year=2009;  citation_volume=191;  citation_pages=105-12; " /><meta name="citation_reference" content="citation_title=Noncoding RNAs binding to the nucleoid protein HU in Escherichia coli; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=6046-55; " /><meta name="citation_reference" content="citation_title=Widespread antisense transcription in Escherichia coli; mBio;  citation_year=2010;  citation_volume=1;  citation_pages=e00024-10; " /><meta name="citation_reference" content="citation_title=RNA expression analysis using a 30 base pair resolution Escherichia coli genome array; Nat. Biotechnol.;  citation_year=2000;  citation_volume=18;  citation_pages=1262-8; " /><meta name="citation_reference" content="citation_title=The primary transcriptome of the major human pathogen Helicobacter pylori; Nature;  citation_year=2010;  citation_volume=464;  citation_pages=250-5; " /><meta name="citation_reference" content="citation_title=Condition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis; Science;  citation_year=2012;  citation_volume=335;  citation_pages=1103-6; " /><meta name="citation_reference" content="citation_title=cis-Antisense RNA, another level of gene regulation in bacteria; Microbiol. Mol. Biol. Rev.;  citation_year=2011;  citation_volume=75;  citation_pages=286-300; " /><meta name="citation_reference" content="citation_title=Poly-3-hydroxybutyrate degradation in Rhizobium (Sinorhizobium) meliloti: isolation and characterization of a gene encoding 3-hydroxybutyrate dehydrogenase; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=849-57; " /><meta name="citation_reference" content="citation_title=Identification of a hydroxyproline transport system in the legume endosymbiont Sinorhizobium meliloti; Mol. Plant Microbe Interact.;  citation_year=2009;  citation_volume=22;  citation_pages=1116-27; " /><meta name="citation_reference" content="citation_title=Structural relationships among Rhizobium meliloti symbiotic promoters; Cell;  citation_year=1983;  citation_volume=35;  citation_pages=479-85; " /><meta name="citation_reference" content="citation_title=Regulation of glutamine synthetase II activity in Rhizobium meliloti 104A14; J. Bacteriol.;  citation_year=1989;  citation_volume=171;  citation_pages=5087-94; " /><meta name="citation_reference" content="citation_title=Rhizobium meliloti 1021 has three differentially regulated loci involved in glutamine biosynthesis, none of which is essential for symbiotic nitrogen fixation; J. Bacteriol.;  citation_year=1989;  citation_volume=171;  citation_pages=1673-82; " /><meta name="citation_reference" content="citation_title=Symbiotic nitrogen fixation does not require adenylylation of glutamine synthetase I in Rhizobium meliloti; FEMS Microbiol. Lett.;  citation_year=1996;  citation_volume=145;  citation_pages=33-40; " /><meta name="citation_reference" content="citation_title=Characterization of sequences downstream from transcriptional start site of Rhizobium meliloti nifHDK promoter; Sci. China C Life Sci.;  citation_year=1997;  citation_volume=40;  citation_pages=217-24; " /><meta name="citation_reference" content="citation_title=The Rhizobium etli amtB gene coding for an NH4+ transporter is down-regulated early during bacteroid differentiation; Mol. Plant Microbe Interact.;  citation_year=1998;  citation_volume=11;  citation_pages=188-98; " /><meta name="citation_reference" content="citation_title=The upstream region of the nodD3 gene of Sinorhizobium meliloti carries enhancer sequences for the transcriptional activator NtrC; FEMS Microbiol. Lett.;  citation_year=1999;  citation_volume=179;  citation_pages=491-9; " /><meta name="citation_reference" content="citation_title=Prediction and overview of the RpoN-regulon in closely related species of the Rhizobiales; Genome Biol.;  citation_year=2002;  citation_volume=3;  citation_pages=Research0076" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/4/339/1010869/dst014.pdf" /><meta name="description" content="Abstract.  The availability of next-generation sequences of transcripts from prokaryotic organisms offers the opportunity to design a new generation of automate" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/4/339/353499" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/4/339/353499" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/4/339/353499");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst014';
    </script>
<title>
	Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to Sinorhizobium meliloti 2011 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353499" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbPYATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FsbGV0LCBFcmlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgNDQxLCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNOUlMsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgMjU5NCwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSb3V4LCBCcmljZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgNDQxLCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNOUlMsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgMjU5NCwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXV2aWFjLCBMYXVyZW50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiA0NDEsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ05SUywgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiAyNTk0LCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkphcmRpbmF1ZCwgTWFyaWUtRnJhbmPCuG9pc2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDQ0MSwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDTlJTLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDI1OTQsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5QVC1Vbml2ZXJzaXRlIGRlIFRPVUxPVVNFLCBFTlNBVC0gQXZlbnVlIGRlIGwnQWdyb2Jpb3BvbGUsIEF1emV2aWxsZS1Ub2xvc2FuZS0zMTMyNiAtIENhc3RhbmV0LVRvbG9zYW4gQ8OpZGV4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FycsOocmUsIFPDqWJhc3RpZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDQ0MSwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDTlJTLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDI1OTQsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFyYXV0LCBUaG9tYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBMYWJvcmF0b2lyZSBkZSBHw6luw6l0aXF1ZSBDZWxsdWxhaXJlLCBVTVIgNDQ0LCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVOVlQsIExhYm9yYXRvaXJlIGRlIEfDqW7DqXRpcXVlIENlbGx1bGFpcmUsIFVNUiA0NDQsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iZGUgQ2FydmFsaG8tTmllYmVsLCBGZXJuYW5kYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgNDQxLCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNOUlMsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgMjU5NCwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb3V6eSwgSsOpcsO0bWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDQ0MSwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDTlJTLCBMYWJvcmF0b2lyZSBkZXMgSW50ZXJhY3Rpb25zIFBsYW50ZXMtTWljcm9vcmdhbmlzbWVzIChMSVBNKSwgVU1SIDI1OTQsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2FtYXMsIFBhc2NhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgNDQxLCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNOUlMsIExhYm9yYXRvaXJlIGRlcyBJbnRlcmFjdGlvbnMgUGxhbnRlcy1NaWNyb29yZ2FuaXNtZXMgKExJUE0pLCBVTVIgMjU5NCwgQ2FzdGFuZXQtVG9sb3NhbiBGLTMxMzI2LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYXBlbGEsIERlbHBoaW5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiA0NDEsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ05SUywgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiAyNTk0LCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJydWFuZCwgQ2xhdWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiA0NDEsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ05SUywgTGFib3JhdG9pcmUgZGVzIEludGVyYWN0aW9ucyBQbGFudGVzLU1pY3Jvb3JnYW5pc21lcyAoTElQTSksIFVNUiAyNTk0LCBDYXN0YW5ldC1Ub2xvc2FuIEYtMzEzMjYsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaGlleCwgVGhvbWFzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVW5pdMOpIGRlIEJpb23DqXRyaWUgZXQgZCdJbnRlbGxpZ2VuY2UgQXJ0aWZpY2llbGxlLCBVUiA4NzUsIENhc3RhbmV0LVRvbG9zYW4gRi0zMTMyNiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJOZXh0LUdlbmVyYXRpb24gQW5ub3RhdGlvbiBvZiBQcm9rYXJ5b3RpYyBHZW5vbWVzIHdpdGggRXVHZW5lLVA6IEFwcGxpY2F0aW9uIHRvIFNpbm9yaGl6b2JpdW0gbWVsaWxvdGkgMjAxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuZCBvcGVyYXRpb25hbCBjb21wbGV4aXR5IG9mIHRoZSBHZW9iYWN0ZXIgc3VsZnVycmVkdWNlbnMgZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEzMDQtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaCByZXNvbHV0aW9uIG1hcCBvZiBhIGN5YW5vYmFjdGVyaWFsIHRyYW5zY3JpcHRvbWU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPVI0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBzZXVkb21vbmFzIHB1dGlkYSBLVDI0NDAgZ2Vub21lIHVwZGF0ZSBieSBjRE5BIHNlcXVlbmNpbmcgYW5kIG1pY3JvYXJyYXkgdHJhbnNjcmlwdG9taWNzOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTMwOS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBpbGx1c3RyYXRlcyB0aGUgdHJhbnNjcmlwdGlvbmFsIHJlc3BvbnNlIG9mIExlZ2lvbmVsbGEgcG5ldW1vcGhpbGEgZHVyaW5nIGluZmVjdGlvbiBhbmQgaWRlbnRpZmllcyBzZXZlbnR5IG5vdmVsIHNtYWxsIG5vbi1jb2RpbmcgUk5BczsgUGxvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNzU3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhdXRvbWF0aWMgYW5ub3RhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczsgQnJpZWYgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIG1pY3JvYmlhbCBnZW5lIGlkZW50aWZpY2F0aW9uIHdpdGggR0xJTU1FUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDYzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgYmFjdGVyaWFsIGdlbmVzIGFuZCBlbmRvc3ltYmlvbnQgRE5BIHdpdGggR2xpbW1lcjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02NzMtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYXBwcm9hY2hlcyB0byBnZW5lIHByZWRpY3Rpb247IEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2RpZ2FsOiBwcm9rYXJ5b3RpYyBnZW5lIHJlY29nbml0aW9uIGFuZCB0cmFuc2xhdGlvbiBpbml0aWF0aW9uIHNpdGUgaWRlbnRpZmljYXRpb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSQVNUIFNlcnZlcjogcmFwaWQgYW5ub3RhdGlvbnMgdXNpbmcgc3Vic3lzdGVtcyB0ZWNobm9sb2d5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qkc3OiBhIG5ldyBhcHByb2FjaCBmb3IgYmFjdGVyaWFsIGdlbm9tZSBhbm5vdGF0aW9uIGRlc2lnbmVkIGZvciBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ5MjM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VkVTUEE6IHNvZnR3YXJlIHRvIGZhY2lsaXRhdGUgZ2Vub21pYyBhbm5vdGF0aW9uIG9mIHByb2thcnlvdGljIG9yZ2FuaXNtcyB0aHJvdWdoIGludGVncmF0aW9uIG9mIHByb3Rlb21pYyBhbmQgdHJhbnNjcmlwdG9taWMgZGF0YTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9TY29wZeKAlGFuIGludGVncmF0ZWQgbWljcm9iaWFsIHJlc291cmNlIGZvciB0aGUgY3VyYXRpb24gYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIGdlbm9taWMgYW5kIG1ldGFib2xpYyBkYXRhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENjM2LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdUdlbmU6IGFuIGV1a2FyeW90aWMgZ2VuZSBmaW5kZXIgdGhhdCBjb21iaW5lcyBzZXZlcmFsIHNvdXJjZXMgb2YgZXZpZGVuY2U7IGNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgQmVybGluLCBHZXJtYW55OyBDb21wdXRhdGlvbmFsIGJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFVR1VTVFVTOiBhIHdlYiBzZXJ2ZXIgZm9yIGdlbmUgZmluZGluZyBpbiBldWthcnlvdGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1XMzA5LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGluZmVyZW5jZSBvZiBob21vbG9nb3VzIGdlbmUgc3RydWN0dXJlcyBpbiB0aGUgaHVtYW4gZ2Vub21lLDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz04MDMtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNHUC0xOiBwcmVkaWN0aW9uIGFuZCB2YWxpZGF0aW9uIG9mIGhvbW9sb2dvdXMgZ2VuZXMgYmFzZWQgb24gc2VxdWVuY2UgYWxpZ25tZW50czsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNTc0LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5lIHByZWRpY3Rpb24gaW4gaHVtYW4gYW5kIG1vdXNlOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEwOC0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXVHZW5lJ0hvbTogYSBnZW5lcmljIHNpbWlsYXJpdHktYmFzZWQgZ2VuZSBmaW5kZXIgdXNpbmcgbXVsdGlwbGUgaG9tb2xvZ291cyBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM3NDItNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBhbm5vdGF0aW9uIGluIHBsYW50cyBhbmQgZnVuZ2k6IEV1R2VuZSBhcyBhIG1vZGVsIHBsYXRmb3JtOyBDdXJyLiBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ODctOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbWV0YXpvYW4gcGxhbnTigJNwYXJhc2l0aWMgbmVtYXRvZGUgTWVsb2lkb2d5bmUgaW5jb2duaXRhOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9OTA5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIExhY2NhcmlhIGJpY29sb3IgcHJvdmlkZXMgaW5zaWdodHMgaW50byBteWNvcnJoaXphbCBzeW1iaW9zaXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTI7ICBjaXRhdGlvbl9wYWdlcz04OC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVjdG9jYXJwdXMgZ2Vub21lIGFuZCB0aGUgaW5kZXBlbmRlbnQgZXZvbHV0aW9uIG9mIG11bHRpY2VsbHVsYXJpdHkgaW4gYnJvd24gYWxnYWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjU7ICBjaXRhdGlvbl9wYWdlcz02MTctMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgVGV0cmFueWNodXMgdXJ0aWNhZSByZXZlYWxzIGhlcmJpdm9yb3VzIHBlc3QgYWRhcHRhdGlvbnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00Nzk7ICBjaXRhdGlvbl9wYWdlcz00ODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgVGhlb2Jyb21hIGNhY2FvOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMS0wODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1lZGljYWdvIGdlbm9tZSBwcm92aWRlcyBpbnNpZ2h0IGludG8gdGhlIGV2b2x1dGlvbiBvZiByaGl6b2JpYWwgc3ltYmlvc2VzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDgwOyAgY2l0YXRpb25fcGFnZXM9NTIwLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdG9tYXRvIGdlbm9tZSBzZXF1ZW5jZSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGZsZXNoeSBmcnVpdCBldm9sdXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODU7ICBjaXRhdGlvbl9wYWdlcz02MzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gcmVwcm9ncmFtbWluZyBkdXJpbmcgcm9vdCBub2R1bGUgZGV2ZWxvcG1lbnQgaW4gTWVkaWNhZ28gdHJ1bmNhdHVsYTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNjQ2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhdGlvbiBvZiBzeW1iaW90aWMgY2VsbHMgYW5kIGVuZG9zeW1iaW9udHMgaW4gTWVkaWNhZ28gdHJ1bmNhdHVsYSBub2R1bGF0aW9uIGFyZSBjb3VwbGVkIHRvIHR3byB0cmFuc2NyaXB0b21lLXN3aXRjaGVzOyBQbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTk1MTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5vcmhpem9iaXVtIG1lbGlsb3RpIGRpZmZlcmVudGlhdGlvbiBkdXJpbmcgc3ltYmlvc2lzIHdpdGggYWxmYWxmYTogYSB0cmFuc2NyaXB0b21pYyBkaXNzZWN0aW9uOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzYzLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcG9zaXRlIGdlbm9tZSBvZiB0aGUgbGVndW1lIHN5bWJpb250IFNpbm9yaGl6b2JpdW0gbWVsaWxvdGk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjkzOyAgY2l0YXRpb25fcGFnZXM9NjY4LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaWNhbCBhbmQgZ2VuZXRpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIHN5bWJpb3RpYyBhbmQgYXV4b3Ryb3BoaWMgbXV0YW50cyBvZiBSaGl6b2JpdW0gbWVsaWxvdGkgaW5kdWNlZCBieSB0cmFuc3Bvc29uIFRuNSBtdXRhZ2VuZXNpczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xMTQtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGRpZmZlcmVuY2VzIGJldHdlZW4gU2lub3JoaXpvYml1bSBtZWxpbG90aSAxMDIxIGFuZCAyMDExIHN0cmFpbnMgdXNpbmcgdGhlIGhvc3QgY2FsY2l1bSBzcGlraW5nIHJlc3BvbnNlOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTI0NS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiB0aGUgcGhvc3BoYXRlIHN0YXJ2YXRpb24gcmVzcG9uc2UgaW4gU2lub3JoaXpvYml1bSBtZWxpbG90aSBzdHJhaW5zIDEwMjEgYW5kIDIwMTE7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gYW5kIHByb3BlcnRpZXMgb2YgUHN0U0NBQiwgYSBoaWdoLWFmZmluaXR5LCBoaWdoLXZlbG9jaXR5IHBob3NwaGF0ZSB0cmFuc3BvcnQgc3lzdGVtIG9mIFNpbm9yaGl6b2JpdW0gbWVsaWxvdGk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9MTA4OS0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb2RlbCBsZWd1bWUgTWVkaWNhZ28gdHJ1bmNhdHVsYSBBMTcgaXMgcG9vcmx5IG1hdGNoZWQgZm9yIE4yIGZpeGF0aW9uIHdpdGggdGhlIHNlcXVlbmNlZCBtaWNyb3N5bWJpb250IFNpbm9yaGl6b2JpdW0gbWVsaWxvdGkgMTAyMTsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9NjItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXN0aWdhdGlvbnMgb2YgUmhpem9iaXVtIGJpb2ZpbG0gZm9ybWF0aW9uOyBGRU1TIE1pY3JvYmlvbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xOTUtMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXhvdHJvcGh5IGFjY291bnRzIGZvciBub2R1bGF0aW9uIGRlZmVjdCBvZiBtb3N0IFNpbm9yaGl6b2JpdW0gbWVsaWxvdGkgbXV0YW50cyBpbiB0aGUgYnJhbmNoZWQtY2hhaW4gYW1pbm8gYWNpZCBiaW9zeW50aGVzaXMgcGF0aHdheTsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTEyMzItNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV4dHJhY3l0b3BsYXNtaWMgZnVuY3Rpb24gc2lnbWEgZmFjdG9yIGFjdHMgYXMgYSBnZW5lcmFsIHN0cmVzcyByZXNwb25zZSByZWd1bGF0b3IgaW4gU2lub3JoaXpvYml1bSBtZWxpbG90aTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODk7ICBjaXRhdGlvbl9wYWdlcz00MjA0LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EdWFsIGNvbnRyb2wgb2YgU2lub3JoaXpvYml1bSBtZWxpbG90aSBScG9FMiBzaWdtYSBmYWN0b3IgYWN0aXZpdHkgYnkgdHdvIFBoeVItdHlwZSB0d28tY29tcG9uZW50IHJlc3BvbnNlIHJlZ3VsYXRvcnM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTkyOyAgY2l0YXRpb25fcGFnZXM9MjI1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3Jvd2luZyBNZWRpY2FnbyB0cnVuY2F0dWxhOiBjaG9pY2Ugb2Ygc3Vic3RyYXRlcyBhbmQgZ3Jvd3RoIGNvbmRpdGlvbnM7IGNpdGF0aW9uX3RleHQ9Tm9ibGUgZm91bmRhdGlvbi4gSVNCTiAwLTk3NTQzMDMtMS05LiBodHRwOi8vd3d3Lm5vYmxlLm9yZy9NZWRpY2Fnb0hhbmRib29rLjsgVGhlIE1lZGljYWdvIHRydW5jYXR1bGEgaGFuZGJvb2s7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGljYWdvIHRydW5jYXR1bGEgRU5PRDExOiBhIG5vdmVsIFJQUlAtZW5jb2RpbmcgZWFybHkgbm9kdWxpbiBnZW5lIGV4cHJlc3NlZCBkdXJpbmcgbXljb3JyaGl6YXRpb24gaW4gYXJidXNjdWxlLWNvbnRhaW5pbmcgY2VsbHM7IE1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03MzctNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Nb3JnYW4gS2F1Zm1hbm4gUHVibGlzaGVycyBJbmMuLCBXaWxsaWFtcyBDb2xsZWdlLCA7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl9wYWdlcz0yODItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGRpc2NyaW1pbmF0aXZlIGxlYXJuaW5nIGZvciBoaWdoZXItYWNjdXJhY3kgY29tcHV0YXRpb25hbCBnZW5lIHByZWRpY3Rpb247IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ucmFkOiBnZW5lIHByZWRpY3Rpb24gdXNpbmcgY29uZGl0aW9uYWwgcmFuZG9tIGZpZWxkczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMzg5LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBhbmFseXNpcyBmb3Igc2VxdWVuY2UgY291bnQgZGF0YTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjEwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2xsaW5nIHRoZSBmYWxzZSBkaXNjb3ZlcnkgcmF0ZeKAlGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyBKLiBSLiBTdGF0LiBTb2MuIEI7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yODktMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3JtYWxpemF0aW9uIG9mIHJlYWwtdGltZSBxdWFudGl0YXRpdmUgcmV2ZXJzZSB0cmFuc2NyaXB0aW9uLVBDUiBkYXRhOiBhIG1vZGVsLWJhc2VkIHZhcmlhbmNlIGVzdGltYXRpb24gYXBwcm9hY2ggdG8gaWRlbnRpZnkgZ2VuZXMgc3VpdGVkIGZvciBub3JtYWxpemF0aW9uLCBhcHBsaWVkIHRvIGJsYWRkZXIgYW5kIGNvbG9uIGNhbmNlciBkYXRhIHNldHM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9NTI0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGluZyBiYWN0ZXJpYWwgZ2VuZS1maW5kaW5nIEhNTSBzdHJ1Y3R1cmVzIGFzIHByb2JhYmlsaXN0aWMgbG9naWMgcHJvZ3JhbXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NjM2LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXJiYW1veWwtcGhvc3BoYXRlIHN5bnRoZXRhc2XigJRjcmVhdGlvbiBvZiBhbiBlc2NhcGUgcm91dGUgZm9yIGFtbW9uaWE7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTM5NzIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpeEotcmVndWxhdGVkIGdlbmVzIGV2b2x2ZWQgdGhyb3VnaCBwcm9tb3RlciBkdXBsaWNhdGlvbiBpbiBTaW5vcmhpem9iaXVtIG1lbGlsb3RpOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9MjMzNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gYmV0d2VlbiB1bHRyYXN0cnVjdHVyYWwgZGlmZmVyZW50aWF0aW9uIG9mIGJhY3Rlcm9pZHMgYW5kIG5pdHJvZ2VuIGZpeGF0aW9uIGluIGFsZmFsZmEgbm9kdWxlczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzI7ICBjaXRhdGlvbl9wYWdlcz00Mjk1LTMwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYW5kLXNwZWNpZmljIGRlZXAgc2VxdWVuY2luZyBvZiB0aGUgdHJhbnNjcmlwdG9tZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz05ODktOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2Ygc3RyYW5kLXNwZWNpZmljIFJOQSBzZXF1ZW5jaW5nIG1ldGhvZHM7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NzA5LVU3Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwb2xsbzogYSBzZXF1ZW5jZSBhbm5vdGF0aW9uIGVkaXRvcjsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1SZXNlYXJjaDAwODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIHN1cnZleSBvZiBzUk5BcyBpbiB0aGUgc3ltYmlvdGljIG5pdHJvZ2VuLWZpeGluZyBhbHBoYS1wcm90ZW9iYWN0ZXJpdW0gU2lub3JoaXpvYml1bSBtZWxpbG90aTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIHJlZ3VsYXRlZCB0cmFuc2NyaXB0b21lIG9mIE5laXNzZXJpYSBtZW5pbmdpdGlkaXMgaW4gaHVtYW4gYmxvb2QgdXNpbmcgYSB0aWxpbmcgYXJyYXk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9NjIxNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdG9yeSBtZWNoYW5pc21zIGVtcGxveWVkIGJ5IGNpcy1lbmNvZGVkIGFudGlzZW5zZSBSTkFzOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTAyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBhbmQgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiB0aGUgUmhpem9iaXVtIG1lbGlsb3RpIHBob3NwaG9lbm9scHlydXZhdGUgY2FyYm94eWtpbmFzZSAocGNrQSkgZ2VuZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzc7ICBjaXRhdGlvbl9wYWdlcz0xNDUyLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVzZW5jZSBvZiBhIGdlbmUgZW5jb2RpbmcgY2hvbGluZSBzdWxmYXRhc2UgaW4gU2lub3JoaXpvYml1bSBtZWxpbG90aSBiZXQgb3Blcm9uOiBjaG9saW5lLU8tc3VsZmF0ZSBpcyBtZXRhYm9saXplZCBpbnRvIGdseWNpbmUgYmV0YWluZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTExMzk0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBjaHJvbW9zb21lIHNlcXVlbmNlIG9mIHRoZSBsZWd1bWUgc3ltYmlvbnQgU2lub3JoaXpvYml1bSBtZWxpbG90aSBzdHJhaW4gMTAyMTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTk4NzctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgZm9yIGFjdGl2YXRpbmcgYmFjdGVyaWFsIFJOQSBwb2x5bWVyYXNlOyBGRU1TIE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTYxMS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umhpem9iaXVtIG1lbGlsb3RpIG50ckEgKHJwb04pIGdlbmUgaXMgcmVxdWlyZWQgZm9yIGRpdmVyc2UgbWV0YWJvbGljIGZ1bmN0aW9uczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0yNDI0LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBzaWdtYSBmYWN0b3IgUnBvSDEgaW4gdGhlIHBIIHN0cmVzcyByZXNwb25zZSBvZiBTaW5vcmhpem9iaXVtIG1lbGlsb3RpOyBCTUMgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTI2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR1YWwgUnBvSCBzaWdtYSBmYWN0b3JzIGFuZCB0cmFuc2NyaXB0aW9uYWwgcGxhc3RpY2l0eSBpbiBhIHN5bWJpb3RpYyBiYWN0ZXJpdW07IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9NDk4My05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lub3JoaXpvYml1bSBtZWxpbG90aSBzaWdtYSBmYWN0b3JzIFJwb0UxIGFuZCBScG9FNCBhcmUgYWN0aXZhdGVkIGluIHN0YXRpb25hcnkgcGhhc2UgaW4gcmVzcG9uc2UgdG8gc3VsZml0ZTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MDc2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb21vdGVyIHByZWRpY3Rpb24gaW4gdGhlIHJoaXpvYmlhOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9MTc1MS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb24gdW5pdCBhcmNoaXRlY3R1cmUgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTA0My1VMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIHNlcXVlbmNpbmcgdW5jb3ZlcnMgbnVtZXJvdXMgc21hbGwgUk5BcyBvbiBhbGwgZm91ciByZXBsaWNvbnMgb2YgdGhlIHBsYW50IHBhdGhvZ2VuIEFncm9iYWN0ZXJpdW0gdHVtZWZhY2llbnM7IFJOQSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDQ2LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5vcmhpem9iaXVtIG1lbGlsb3RpIHJwb0UyIGlzIG5lY2Vzc2FyeSBmb3IgSCgyKU8oMikgc3RyZXNzIHJlc2lzdGFuY2UgZHVyaW5nIHRoZSBzdGF0aW9uYXJ5IGdyb3d0aCBwaGFzZTsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI5MDsgIGNpdGF0aW9uX3BhZ2VzPTI1LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ScG9FMiBvZiBTaW5vcmhpem9iaXVtIG1lbGlsb3RpIGlzIG5lY2Vzc2FyeSBmb3IgdHJlaGFsb3NlIHN5bnRoZXNpcyBhbmQgZ3Jvd3RoIGluIGh5cGVyb3Ntb3RpYyBtZWRpYTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTE3MDgtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBzbWFsbCBub24tY29kaW5nIFJOQXMgaW4gdGhlIGxlZ3VtZSBlbmRvc3ltYmlvbnQgU2lub3JoaXpvYml1bSBtZWxpbG90aSBieSBjb21wYXJhdGl2ZSBnZW5vbWljczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MTA4MC05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY2hyb21vc29tYWwgYWxwaGEtcHJvdGVvYmFjdGVyaWFsIHNtYWxsIFJOQXMgYnkgY29tcGFyYXRpdmUgZ2Vub21lIGFuYWx5c2lzIGFuZCBkZXRlY3Rpb24gaW4gU2lub3JoaXpvYml1bSBtZWxpbG90aSBzdHJhaW4gMTAyMTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIFNpbm9yaGl6b2JpdW0gbWVsaWxvdGkgc1JOQSBnZW5lcyBhbmQgZXhwZXJpbWVudGFsIGRldGVjdGlvbiBpbiBzdHJhaW4gMjAxMTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00MTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXhwcmVzc2lvbiBvZiBhIG5vdmVsIGFudGlzZW5zZSBnZW5lIG1lZGlhdGVzIGluY29tcGF0aWJpbGl0eSB3aXRoaW4gdGhlIGxhcmdlIHJlcEFCQyBmYW1pbHkgb2YgYWxwaGEtcHJvdGVvYmFjdGVyaWFsIHBsYXNtaWRzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz02MTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gYXR0ZW51YXRpb24gYXNzb2NpYXRlZCB3aXRoIGJhY3RlcmlhbCByZXBldGl0aXZlIGV4dHJhZ2VuaWMgQklNRSBlbGVtZW50czsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTQ7ICBjaXRhdGlvbl9wYWdlcz0zNzUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSTkEgZGVncmFkb3NvbWUgYW5kIHBvbHkoQSkgcG9seW1lcmFzZSBvZiBFc2NoZXJpY2hpYSBjb2xpIGFyZSByZXF1aXJlZCBpbiB2aXZvIGZvciB0aGUgZGVncmFkYXRpb24gb2Ygc21hbGwgbVJOQSBkZWNheSBpbnRlcm1lZGlhdGVzIGNvbnRhaW5pbmcgUkVQLXN0YWJpbGl6ZXJzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz03NzctOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YWJpbGl0eSBvZiB0aGUgcHN0UyB0cmFuc2NyaXB0IG9mIEVzY2hlcmljaGlhIGNvbGk7IEFyY2guIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTkxOyAgY2l0YXRpb25fcGFnZXM9MTA1LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25jb2RpbmcgUk5BcyBiaW5kaW5nIHRvIHRoZSBudWNsZW9pZCBwcm90ZWluIEhVIGluIEVzY2hlcmljaGlhIGNvbGk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9NjA0Ni01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBhbnRpc2Vuc2UgdHJhbnNjcmlwdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBtQmlvOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz1lMDAwMjQtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBleHByZXNzaW9uIGFuYWx5c2lzIHVzaW5nIGEgMzAgYmFzZSBwYWlyIHJlc29sdXRpb24gRXNjaGVyaWNoaWEgY29saSBnZW5vbWUgYXJyYXk7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMjYyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcmltYXJ5IHRyYW5zY3JpcHRvbWUgb2YgdGhlIG1ham9yIGh1bWFuIHBhdGhvZ2VuIEhlbGljb2JhY3RlciBweWxvcmk7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz0yNTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uZGl0aW9uLWRlcGVuZGVudCB0cmFuc2NyaXB0b21lIHJldmVhbHMgaGlnaC1sZXZlbCByZWd1bGF0b3J5IGFyY2hpdGVjdHVyZSBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzU7ICBjaXRhdGlvbl9wYWdlcz0xMTAzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWNpcy1BbnRpc2Vuc2UgUk5BLCBhbm90aGVyIGxldmVsIG9mIGdlbmUgcmVndWxhdGlvbiBpbiBiYWN0ZXJpYTsgTWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0yODYtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5LTMtaHlkcm94eWJ1dHlyYXRlIGRlZ3JhZGF0aW9uIGluIFJoaXpvYml1bSAoU2lub3JoaXpvYml1bSkgbWVsaWxvdGk6IGlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhIGdlbmUgZW5jb2RpbmcgMy1oeWRyb3h5YnV0eXJhdGUgZGVoeWRyb2dlbmFzZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODE7ICBjaXRhdGlvbl9wYWdlcz04NDktNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgaHlkcm94eXByb2xpbmUgdHJhbnNwb3J0IHN5c3RlbSBpbiB0aGUgbGVndW1lIGVuZG9zeW1iaW9udCBTaW5vcmhpem9iaXVtIG1lbGlsb3RpOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTExNi0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCByZWxhdGlvbnNoaXBzIGFtb25nIFJoaXpvYml1bSBtZWxpbG90aSBzeW1iaW90aWMgcHJvbW90ZXJzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NDc5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGdsdXRhbWluZSBzeW50aGV0YXNlIElJIGFjdGl2aXR5IGluIFJoaXpvYml1bSBtZWxpbG90aSAxMDRBMTQ7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9NTA4Ny05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Umhpem9iaXVtIG1lbGlsb3RpIDEwMjEgaGFzIHRocmVlIGRpZmZlcmVudGlhbGx5IHJlZ3VsYXRlZCBsb2NpIGludm9sdmVkIGluIGdsdXRhbWluZSBiaW9zeW50aGVzaXMsIG5vbmUgb2Ygd2hpY2ggaXMgZXNzZW50aWFsIGZvciBzeW1iaW90aWMgbml0cm9nZW4gZml4YXRpb247IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MTY3My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3ltYmlvdGljIG5pdHJvZ2VuIGZpeGF0aW9uIGRvZXMgbm90IHJlcXVpcmUgYWRlbnlseWxhdGlvbiBvZiBnbHV0YW1pbmUgc3ludGhldGFzZSBJIGluIFJoaXpvYml1bSBtZWxpbG90aTsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHNlcXVlbmNlcyBkb3duc3RyZWFtIGZyb20gdHJhbnNjcmlwdGlvbmFsIHN0YXJ0IHNpdGUgb2YgUmhpem9iaXVtIG1lbGlsb3RpIG5pZkhESyBwcm9tb3RlcjsgU2NpLiBDaGluYSBDIExpZmUgU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTIxNy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJoaXpvYml1bSBldGxpIGFtdEIgZ2VuZSBjb2RpbmcgZm9yIGFuIE5INCsgdHJhbnNwb3J0ZXIgaXMgZG93bi1yZWd1bGF0ZWQgZWFybHkgZHVyaW5nIGJhY3Rlcm9pZCBkaWZmZXJlbnRpYXRpb247IE1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xODgtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1cHN0cmVhbSByZWdpb24gb2YgdGhlIG5vZEQzIGdlbmUgb2YgU2lub3JoaXpvYml1bSBtZWxpbG90aSBjYXJyaWVzIGVuaGFuY2VyIHNlcXVlbmNlcyBmb3IgdGhlIHRyYW5zY3JpcHRpb25hbCBhY3RpdmF0b3IgTnRyQzsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTQ5MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIGFuZCBvdmVydmlldyBvZiB0aGUgUnBvTi1yZWd1bG9uIGluIGNsb3NlbHkgcmVsYXRlZCBzcGVjaWVzIG9mIHRoZSBSaGl6b2JpYWxlczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1SZXNlYXJjaDAwNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvNC8zMzkvMTAxMDg2OS9kc3QwMTQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBhdmFpbGFiaWxpdHkgb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNlcyBvZiB0cmFuc2NyaXB0cyBmcm9tIHByb2thcnlvdGljIG9yZ2FuaXNtcyBvZmZlcnMgdGhlIG9wcG9ydHVuaXR5IHRvIGRlc2lnbiBhIG5ldyBnZW5lcmF0aW9uIG9mIGF1dG9tYXRlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTM0OTkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkEi9EeAee11hSufOckXhbvwPbbl4=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f339%2f353499" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f339%2f353499" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13681" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&amp;Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/4">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5030350">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030350">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030352">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030352">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030357">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030357">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030381">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030381">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030421">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030421">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030448">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030448">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030453">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030453">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030457">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030457">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5030464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5030464">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/4/325/353411">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/4/355/353793">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to <em>Sinorhizobium meliloti</em> 2011
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erika Sallet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erika+Sallet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sallet E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sallet E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Erika Sallet</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Brice Roux</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Brice+Roux">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Roux B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Roux B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Brice Roux</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laurent Sauviac</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Laurent+Sauviac">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sauviac L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sauviac L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Laurent Sauviac</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marie-Franc&#184;oise Jardinaud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">INPT-Universite de TOULOUSE</div>, <div class="insititution">ENSAT- Avenue de l'Agrobiopole</div>, <div class="addr-line">Auzeville-Tolosane-31326 - Castanet-Tolosan Cédex</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marie-Franc%c2%b8oise+Jardinaud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jardinaud M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jardinaud M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Marie-Franc¸oise Jardinaud</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S&#233;bastien Carr&#232;re</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=S%c3%a9bastien+Carr%c3%a8re">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carr&#232;re S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Carr&#232;re S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sébastien Carrère</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas Faraut</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire de Génétique Cellulaire, UMR 444</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">ENVT</div>, <div class="insititution">Laboratoire de Génétique Cellulaire, UMR 444</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+Faraut">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Faraut T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Faraut T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Thomas Faraut</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fernanda de Carvalho-Niebel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fernanda+de+Carvalho-Niebel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=de Carvalho-Niebel F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;de Carvalho-Niebel F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fernanda de Carvalho-Niebel</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J&#233;r&#244;me Gouzy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J%c3%a9r%c3%b4me+Gouzy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gouzy J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gouzy J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jérôme Gouzy</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pascal Gamas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pascal+Gamas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gamas P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gamas P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Pascal Gamas</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Delphine Capela</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Delphine+Capela">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Capela D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Capela D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Delphine Capela</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Claude Bruand</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">INRA</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 441</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">CNRS</div>, <div class="insititution">Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR 2594</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Claude+Bruand">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bruand C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bruand C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Claude Bruand</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas Schiex</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">INRA</div>, <div class="insititution">Unité de Biométrie et d'Intelligence Artificielle, UR 875</div>, <div class="addr-line">Castanet-Tolosan F-31326</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +33-561285428. Fax. +33-561285335. 
Email: <a href="mailto:thomas.schiex@toulouse.inra.fr" target="_blank">thomas.schiex@toulouse.inra.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+Schiex">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schiex T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schiex T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Thomas Schiex</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Both authors equally contributed to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 339–354, <a href='https://doi.org/10.1093/dnares/dst014'>https://doi.org/10.1093/dnares/dst014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 April 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 February 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 March 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353499" href="/dnaresearch/article-pdf/20/4/339/1010869/dst014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Erika Sallet, Brice Roux, Laurent Sauviac, Marie-Franc¸oise Jardinaud, Sébastien Carrère, Thomas Faraut, Fernanda de Carvalho-Niebel, Jérôme Gouzy, Pascal Gamas, Delphine Capela, Claude Bruand, Thomas Schiex;  Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to <em>Sinorhizobium meliloti</em> 2011, <em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 339–354, <a href='https://doi.org/10.1093/dnares/dst014'>https://doi.org/10.1093/dnares/dst014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353499?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353499?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Next-Generation%20Annotation%20of%20Prokaryotic%20Genomes%20with%20EuGene-P%3A%20Application%20to%20%26lt%3Bem%26gt%3BSinorhizobium%20meliloti%26lt%3B%2Fem%26gt%3B%202011&amp;publicationDate=2013-04-18&amp;volumeNum=20&amp;issueNum=4&amp;author=Sallet%2C%20Erika%3B%20Roux%2C%20Brice&amp;startPage=339&amp;endPage=354&amp;contentId=10.1093%2Fdnares%2Fdst014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f339%2f353499" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5030350 id="5030350" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The availability of next-generation sequences of transcripts from prokaryotic organisms offers the opportunity to design a new generation of automated genome annotation tools not yet available for prokaryotes. In this work, we designed EuGene-P, the first integrative prokaryotic gene finder tool which combines a variety of high-throughput data, including oriented RNA-Seq data, directly into the prediction process. This enables the automated prediction of coding sequences (CDSs), untranslated regions, transcription start sites (TSSs) and non-coding RNA (ncRNA, sense and antisense) genes. EuGene-P was used to comprehensively and accurately annotate the genome of the nitrogen-fixing bacterium <em>Sinorhizobium meliloti</em> strain 2011, leading to the prediction of 6308 CDSs as well as 1876 ncRNAs. Among them, 1280 appeared as antisense to a CDS, which supports recent findings that antisense transcription activity is widespread in bacteria. Moreover, 4077 TSSs upstream of protein-coding or non-coding genes were precisely mapped providing valuable data for the study of promoter regions. By looking for RpoE2-binding sites upstream of annotated TSSs, we were able to extend the <em>S. meliloti</em> RpoE2 regulon by ∼3-fold. Altogether, these observations demonstrate the power of EuGene-P to produce a reliable and high-resolution automatic annotation of prokaryotic genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome annotation&quot;">genome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="prokaryotes">prokaryotes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-Seq">RNA-Seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rhizobium">rhizobium</a></div>                            <h2 scrollto-destination=5030352 id="5030352" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>With the new generation of sequencing (NGS) technologies, bacterial and archeal genome projects now combine deep genomic sequencing with a variety of transcriptome libraries.<sup><a reveal-id="DST014C1  DST014C4" data-open="DST014C1  DST014C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> If the main motivation for transcriptome sequencing is usually the quantification of gene expression, the transcribed sequences generated by deep sequencing can also contribute to prokaryotic genome annotation by the elucidation of gene structural features, including transcription start sites (TSSs), 5′ and 3′ untranslated regions (UTRs) and the identification of non-coding RNA (ncRNA) genes. The quantification of gene expression following deep cDNA sequencing is based on the number of reads that map to a given gene. Therefore, the development of genome annotation tools that enable a better delineation of transcripts should lead to a more reliable expression measurement. In the recent sequencing of bacterial and archeal genomes, the annotation has still been done manually owing to the lack of appropriate tools to integrate RNA-Seq data.<sup><a reveal-id="DST014C5" data-open="DST014C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Indeed, most existing prokaryotic gene finders<sup><a reveal-id="DST014C6  DST014C9" data-open="DST014C6  DST014C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> or high-level bacterial annotation systems<sup><a reveal-id="DST014C10" data-open="DST014C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST014C11" data-open="DST014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> are based on genomic sequence analysis and cannot take into account available expression data in the structural prediction. Expert annotation using RNA-Seq data has been recently facilitated by the use of integrated tools, such as VESPA<sup><a reveal-id="DST014C12" data-open="DST014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> or MicroScope,<sup><a reveal-id="DST014C13" data-open="DST014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> which allow to simultaneously visualize genomic, transcriptomic, proteomic or syntenic data, but the ultimate curation process still remains laborious.</p><p>With the tremendously increasing number of prokaryotic genomes that is being sequenced, there is a clear need for automated prokaryotic genome annotation tools able to integrate the variety of informative data that can be produced either by second-generation sequencing or by other high-throughput analyses, such as tiling arrays and proteomics. The development of such prokaryotic gene finders allowing not only the prediction of coding sequences (CDSs), but also TSSs and non-coding (nc) transcribed genes, should provide improved transcript quantification, facilitated identification of regulatory sequences upstream of mapped TSSs and thus, easier analysis of gene regulation. Because of the higher complexity of eukaryotic gene structures and the usual availability of transcribed sequences (such as expressed sequence tags or ESTs), many eukaryotic gene finders already have the ability to integrate experimental evidence in their gene prediction process. For example, ESTs are exploited in EuGene<sup><a reveal-id="DST014C14" data-open="DST014C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and Augustus,<sup><a reveal-id="DST014C15" data-open="DST014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> GenomScan<sup><a reveal-id="DST014C16" data-open="DST014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> uses similarities with known proteins, whereas SGP/SGP2<sup><a reveal-id="DST014C17" data-open="DST014C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST014C18" data-open="DST014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and EuGene'Hom<sup><a reveal-id="DST014C19" data-open="DST014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> integrates sequence conservation with related organisms.</p><p>In this work, we adapted the eukaryotic gene finder, EuGene<sup><a reveal-id="DST014C14" data-open="DST014C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST014C20" data-open="DST014C20" class="link link-ref link-reveal xref-bibr">20</a></sup>, to the specific requirements of gene identification in prokaryotes, where in particular overlapping CDSs are relatively frequent. EuGene has already been used successfully to annotate a variety of eukaryotic genomes<sup><a reveal-id="DST014C21  DST014C27" data-open="DST014C21  DST014C27" class="link link-ref link-reveal xref-bibr">21–27</a></sup> and has shown its ability to quickly incorporate new types of information for enhancing its predictive power. The generic tool developed here, called EuGene-P, exploits high-throughput data, such as strand-specific RNA-Seq data, to qualitatively improve the prediction contents and to minimize manual expert annotation. The produced annotation contains previously unpredicted important gene structure features such as 5′ and 3′ UTRs, as well as ncRNA genes (including antisense RNAs). The mathematical model behind EuGene-P and its modular software architecture based on plug-ins facilitate the integration of a variety of other high-throughput data, such as PET-Seq, mass spectrometry data, protein similarities, DNA homologies, predicted transcription terminators and others. The source codes of EuGene-P are available under the open-source Artistic licence at <a class="link link-uri" href="https://mulcyber.toulouse.inra.fr/projects/eugene">https://mulcyber.toulouse.inra.fr/projects/eugene</a>. A fully automated generic prokaryotic pipeline annotation relying on EuGene-P is under preparation and will be made available.</p><p>We trained and used EuGene-P for the annotation of the nitrogen-fixing symbiont <em>Sinorhizobium meliloti</em> bacterial strain 2011 (Sm2011). <em>Sinorhizobium meliloti</em> is a Gram-negative bacterium belonging to the alpha subclass of Proteobacteria, which can live either free in the soil, or in symbiotic association with roots of legume plants such as the model legume <em>Medicago truncatula</em>.<sup><a reveal-id="DST014C26" data-open="DST014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The <em>Sinorhizobium–Medicago</em> symbiotic interaction leads to the formation of new root organs called nodules, within which bacteria differentiate into bacteroids that fix nitrogen to the benefit of the host plant. Both nodule organogenesis and bacteroid differentiation are complex developmental processes that involve deep reprogramming of gene expression in both organisms.<sup><a reveal-id="DST014C28  DST014C30" data-open="DST014C28  DST014C30" class="link link-ref link-reveal xref-bibr">28–30</a></sup> The 6.7-Mb genome of Sm2011 is composed of three replicons, one main chromosome and two megaplasmids called pSymA and pSymB. The Sm2011 strain used in this study is closely related to the Sm1021 reference strain that was previously sequenced.<sup><a reveal-id="DST014C31" data-open="DST014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Both strains are independent spontaneous streptomycin-resistant derivatives of the parental SU47 strain.<sup><a reveal-id="DST014C32" data-open="DST014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Despite being originated from the same parental strain, a number of phenotypic differences were reported,<sup><a reveal-id="DST014C33  DST014C38" data-open="DST014C33  DST014C38" class="link link-ref link-reveal xref-bibr">33–38</a></sup> which may be related to specific genetic differences. In this work, we determined both the genome sequence and the transcriptome of the Sm2011 strain under in planta and different growth conditions. These data were integrated into EuGene-P to refine and enrich the annotation of the <em>S. meliloti</em> genome sequence, notably to predict TSSs and ncRNA genes.</p>                            <h2 scrollto-destination=5030357 id="5030357" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5030358 id="5030358" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and growth conditions</h3>
<p>The bacterial strain used in this study was the streptomycin-resistant derivative of Sm2011 (GMI11495). A <em>rpoE2</em> mutant derivative of this strain was generated as previously described.<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Strains were grown under aerobic conditions at 28°C in Vincent minimal medium supplemented with disodium succinate and ammonium chloride as carbon and nitrogen sources as previously described.<sup><a reveal-id="DST014C40" data-open="DST014C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Bacteria were collected either in a mid-exponential phase (OD<sub>600</sub> = 0.6) or in an early stationary phase (∼1 h 30 min after entry in a stationary phase, OD<sub>600</sub> = 1.2). Bacteria were harvested by filtration on 0.2 µm membranes, frozen in liquid nitrogen and stored at −80°C until RNA extraction. Bacterial cultures were collected from three independent biological experiments.</p>                            <h3 scrollto-destination=5030360 id="5030360" class="section-title" data-legacy-id=s2b>2.2. Plant material and growth conditions</h3>
<p><em>Medicago truncatula</em> cv Jemalong A17 seeds were germinated and transferred to aeroponic caissons as described,<sup><a reveal-id="DST014C41" data-open="DST014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> under the following chamber conditions: temperature: 22°C; 75% hygrometry; light intensity: 200 μE m<sup>−2</sup> s<sup>−1</sup>; light–dark photoperiod: 16–8 h. Plants were grown for 18 days in caisson growth medium<sup><a reveal-id="DST014C42" data-open="DST014C42" class="link link-ref link-reveal xref-bibr">42</a></sup> supplemented with 10 mM NH<sub>4</sub>NO<sub>3</sub>, before growth in nitrogen-free medium for 4 days prior to inoculation with <em>S. meliloti</em>. At 10 days post-inoculation, nodules were harvested on ice from at least 20 plants, immediately frozen in liquid nitrogen and stored at −80°C. Each biological repetition corresponded to an independent caisson, with ∼40 plants per caisson.</p>                            <h3 scrollto-destination=5030362 id="5030362" class="section-title" data-legacy-id=s2c>2.3. <em>Sinorhizobium meliloti</em> genome sequencing</h3>
<p>The genome of Sm2011 was sequenced at the Genoscope (CNS, Evry, France) using fractions of 454 Titanium (46 Mb), 454 paired ends (18 Mb, insert size: 8 kb) and Illumina single end reads (1.2 Gb, read length: 76 nt), providing a 190-fold theoretical coverage of the genome. The genome sequence was assembled as described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials and Methods</a>. The nucleotide sequences of Sm2011 and Sm1021 strains were compared using the glint software (Faraut T. and Courcelle E.; <a class="link link-uri" href="http://lipm-bioinfo.toulouse.inra.fr/download/glint/">http://lipm-bioinfo.toulouse.inra.fr/download/glint/</a>, unpublished) to identify polymorphic regions. A set of 71 mutations including 64 putative frameshifts were verified by Sanger sequencing of polymerase chain reaction (PCR) products surrounding these regions generated using either Sm2011 or Sm1021<sup><a reveal-id="DST014C32" data-open="DST014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> DNA as a template. The genome sequence of Sm2011 was submitted to Genbank under accession numbers CP004138, CP004139 and CP004140, and a browser was set up at <a class="link link-uri" href="https://iant.toulouse.inra.fr/S.meliloti2011">https://iant.toulouse.inra.fr/S.meliloti2011</a>.</p>                            <h3 scrollto-destination=5030364 id="5030364" class="section-title" data-legacy-id=s2d>2.4. RNA preparations</h3>
<p>RNAs were prepared as described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials and Methods</a>. Briefly, total RNAs extracted from cultured bacteria and root nodules were depleted of ribosomal RNAs by an oligocapture strategy derived from the Plant Ribominus kit (Invitrogen), in which the oligonucleotide sets were specifically designed to target <em>M. truncatula</em> and <em>S. meliloti</em> rRNAs, as well as the highly abundant <em>S. meliloti</em> tRNA-Ala (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for oligonucleotide sequences). RNAs were then separated in two fractions, short (&lt;200 nt) and long (&gt;200 nt), using Zymo Research RNA Clean &amp; Concentrator™-5 columns (Proteigene).</p>                            <h3 scrollto-destination=5030366 id="5030366" class="section-title" data-legacy-id=s2e>2.5. cDNA library preparation and Illumina sequencing</h3>
<p>Oriented sequencing with a RNA ligation procedure was carried out by Fasteris SA (Geneva, Switzerland) using procedures recommended by Illumina, with adaptors and amplification primers designed by Fasteris, unless specified. For small RNAs, the Small RNA Sequencing Alternative v1.5 Protocol (Illumina) was used, starting with ∼500 ng RNAs that were treated with tobacco acid pyrophosphatase to remove triphosphate at 5′ transcript ends and purified on acrylamide gel before and after the adaptor ligation step. The 3′ adaptor was the Universal miRNA cloning linker (NEB). For large RNAs, the amount of starting RNAs was ∼200 ng, and a fragmentation step by zinc during 8 min was included, after the Illumina procedure. The size of selected inserts was 20–120 nt for short RNA libraries and 50–120 nt for long RNA libraries from cultured bacteria and 150–250 nt for long RNA libraries from nodules. Libraries were sequenced either in paired end or in single end (Table <a reveal-id="DST014TB1" data-open="DST014TB1" class="link link-reveal link-table xref-fig">1</a>). Raw sequence data were submitted to the Gene Expression Omnibus (GEO) database (Accession GSE44083). </p>                        <a id="5030368" scrollto-destination="5030368"></a>
<div content-id="DST014TB1" class="table-modal"><div class="table-wrap"><div id="DST014TB1" class="table-wrap-title" data-id="DST014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>RNA-Seq libraries used for annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GEO sample code </th><th>RNA samples </th><th>RNA 
fraction </th><th>Biological 
replicate 
number </th><th>Sequencing process </th><th>Number of unambiguously 
mapped reads or 
paired-reads </th></tr></thead><tbody><tr><td>GSM1078108 </td><td>Nodule </td><td>Long </td><td>1 </td><td>pe 2 × 54 nt </td><td>79 339 </td></tr><tr><td>GSM1078109 </td><td>Nodule </td><td>Long </td><td>2 </td><td>pe 2 × 54 nt </td><td>103 025 </td></tr><tr><td>GSM1078110 </td><td>Nodule </td><td>Long </td><td>3 </td><td>pe 2 × 54 nt </td><td>55 825 </td></tr><tr><td>GSM1078111 </td><td>Nodule </td><td>Short </td><td>1 </td><td>pe 2 × 54 nt </td><td>785 009 </td></tr><tr><td>GSM1078112 </td><td>Nodule </td><td>Short </td><td>2 </td><td>pe 2 × 54 nt </td><td>1 503 684 </td></tr><tr><td>GSM1078113 </td><td>Nodule </td><td>Short </td><td>3 </td><td>pe 2 × 54 nt </td><td>1 465 610 </td></tr><tr><td>GSM1078114 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>4 158 264 </td></tr><tr><td>GSM1078115 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>4 154 232 </td></tr><tr><td>GSM1078116 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 873 524 </td></tr><tr><td>GSM1078117 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>4 792 283 </td></tr><tr><td>GSM1078118 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 390 729 </td></tr><tr><td>GSM1078119 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>9 061 874 </td></tr><tr><td>GSM1078120 </td><td>Bacteria stationary phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>2 102 607 </td></tr><tr><td>GSM1078121 </td><td>Bacteria stationary phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>3 171 844 </td></tr><tr><td>GSM1078122 </td><td>Bacteria stationary phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 953 260 </td></tr><tr><td>GSM1078123 </td><td>Bacteria stationary phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>11 368 031 </td></tr><tr><td>GSM1078124 </td><td>Bacteria stationary phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 960 882 </td></tr><tr><td>GSM1078125 </td><td>Bacteria stationary phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>5 559 756 </td></tr></tbody></table></div><div content-id="DST014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>GEO sample code </th><th>RNA samples </th><th>RNA 
fraction </th><th>Biological 
replicate 
number </th><th>Sequencing process </th><th>Number of unambiguously 
mapped reads or 
paired-reads </th></tr></thead><tbody><tr><td>GSM1078108 </td><td>Nodule </td><td>Long </td><td>1 </td><td>pe 2 × 54 nt </td><td>79 339 </td></tr><tr><td>GSM1078109 </td><td>Nodule </td><td>Long </td><td>2 </td><td>pe 2 × 54 nt </td><td>103 025 </td></tr><tr><td>GSM1078110 </td><td>Nodule </td><td>Long </td><td>3 </td><td>pe 2 × 54 nt </td><td>55 825 </td></tr><tr><td>GSM1078111 </td><td>Nodule </td><td>Short </td><td>1 </td><td>pe 2 × 54 nt </td><td>785 009 </td></tr><tr><td>GSM1078112 </td><td>Nodule </td><td>Short </td><td>2 </td><td>pe 2 × 54 nt </td><td>1 503 684 </td></tr><tr><td>GSM1078113 </td><td>Nodule </td><td>Short </td><td>3 </td><td>pe 2 × 54 nt </td><td>1 465 610 </td></tr><tr><td>GSM1078114 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>4 158 264 </td></tr><tr><td>GSM1078115 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>4 154 232 </td></tr><tr><td>GSM1078116 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 873 524 </td></tr><tr><td>GSM1078117 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>4 792 283 </td></tr><tr><td>GSM1078118 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 390 729 </td></tr><tr><td>GSM1078119 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>9 061 874 </td></tr><tr><td>GSM1078120 </td><td>Bacteria stationary phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>2 102 607 </td></tr><tr><td>GSM1078121 </td><td>Bacteria stationary phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>3 171 844 </td></tr><tr><td>GSM1078122 </td><td>Bacteria stationary phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 953 260 </td></tr><tr><td>GSM1078123 </td><td>Bacteria stationary phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>11 368 031 </td></tr><tr><td>GSM1078124 </td><td>Bacteria stationary phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 960 882 </td></tr><tr><td>GSM1078125 </td><td>Bacteria stationary phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>5 559 756 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All RNA samples were depleted in ribosomal RNA using the RiboMinus™ protocol and separated in short (&lt;200 nt) and long (&gt;200 nt) fractions. Note that nodule libraries contain a mixture of <em>S. meliloti</em> and <em>M. truncatula</em> transcriptomes. Figures indicated here correspond to <em>S. meliloti</em> sequence reads only.</p></div></div><div content-id=""><div class="fn"><p>pe, paired ends; se, single end.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030368" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST014TB1" class="table-wrap-title" data-id="DST014TB1"><div class="label -label">Table 1.</div><div class="caption"><p>RNA-Seq libraries used for annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GEO sample code </th><th>RNA samples </th><th>RNA 
fraction </th><th>Biological 
replicate 
number </th><th>Sequencing process </th><th>Number of unambiguously 
mapped reads or 
paired-reads </th></tr></thead><tbody><tr><td>GSM1078108 </td><td>Nodule </td><td>Long </td><td>1 </td><td>pe 2 × 54 nt </td><td>79 339 </td></tr><tr><td>GSM1078109 </td><td>Nodule </td><td>Long </td><td>2 </td><td>pe 2 × 54 nt </td><td>103 025 </td></tr><tr><td>GSM1078110 </td><td>Nodule </td><td>Long </td><td>3 </td><td>pe 2 × 54 nt </td><td>55 825 </td></tr><tr><td>GSM1078111 </td><td>Nodule </td><td>Short </td><td>1 </td><td>pe 2 × 54 nt </td><td>785 009 </td></tr><tr><td>GSM1078112 </td><td>Nodule </td><td>Short </td><td>2 </td><td>pe 2 × 54 nt </td><td>1 503 684 </td></tr><tr><td>GSM1078113 </td><td>Nodule </td><td>Short </td><td>3 </td><td>pe 2 × 54 nt </td><td>1 465 610 </td></tr><tr><td>GSM1078114 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>4 158 264 </td></tr><tr><td>GSM1078115 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>4 154 232 </td></tr><tr><td>GSM1078116 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 873 524 </td></tr><tr><td>GSM1078117 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>4 792 283 </td></tr><tr><td>GSM1078118 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 390 729 </td></tr><tr><td>GSM1078119 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>9 061 874 </td></tr><tr><td>GSM1078120 </td><td>Bacteria stationary phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>2 102 607 </td></tr><tr><td>GSM1078121 </td><td>Bacteria stationary phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>3 171 844 </td></tr><tr><td>GSM1078122 </td><td>Bacteria stationary phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 953 260 </td></tr><tr><td>GSM1078123 </td><td>Bacteria stationary phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>11 368 031 </td></tr><tr><td>GSM1078124 </td><td>Bacteria stationary phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 960 882 </td></tr><tr><td>GSM1078125 </td><td>Bacteria stationary phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>5 559 756 </td></tr></tbody></table></div><div content-id="DST014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>GEO sample code </th><th>RNA samples </th><th>RNA 
fraction </th><th>Biological 
replicate 
number </th><th>Sequencing process </th><th>Number of unambiguously 
mapped reads or 
paired-reads </th></tr></thead><tbody><tr><td>GSM1078108 </td><td>Nodule </td><td>Long </td><td>1 </td><td>pe 2 × 54 nt </td><td>79 339 </td></tr><tr><td>GSM1078109 </td><td>Nodule </td><td>Long </td><td>2 </td><td>pe 2 × 54 nt </td><td>103 025 </td></tr><tr><td>GSM1078110 </td><td>Nodule </td><td>Long </td><td>3 </td><td>pe 2 × 54 nt </td><td>55 825 </td></tr><tr><td>GSM1078111 </td><td>Nodule </td><td>Short </td><td>1 </td><td>pe 2 × 54 nt </td><td>785 009 </td></tr><tr><td>GSM1078112 </td><td>Nodule </td><td>Short </td><td>2 </td><td>pe 2 × 54 nt </td><td>1 503 684 </td></tr><tr><td>GSM1078113 </td><td>Nodule </td><td>Short </td><td>3 </td><td>pe 2 × 54 nt </td><td>1 465 610 </td></tr><tr><td>GSM1078114 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>4 158 264 </td></tr><tr><td>GSM1078115 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>4 154 232 </td></tr><tr><td>GSM1078116 </td><td>Bacteria mid-exponential phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 873 524 </td></tr><tr><td>GSM1078117 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>4 792 283 </td></tr><tr><td>GSM1078118 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 390 729 </td></tr><tr><td>GSM1078119 </td><td>Bacteria mid-exponential phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>9 061 874 </td></tr><tr><td>GSM1078120 </td><td>Bacteria stationary phase </td><td>Long </td><td>1 </td><td>se 1 × 50 nt </td><td>2 102 607 </td></tr><tr><td>GSM1078121 </td><td>Bacteria stationary phase </td><td>Long </td><td>2 </td><td>se 1 × 50 nt </td><td>3 171 844 </td></tr><tr><td>GSM1078122 </td><td>Bacteria stationary phase </td><td>Long </td><td>3 </td><td>se 1 × 50 nt </td><td>2 953 260 </td></tr><tr><td>GSM1078123 </td><td>Bacteria stationary phase </td><td>Short </td><td>1 </td><td>pe 2 × 50 nt </td><td>11 368 031 </td></tr><tr><td>GSM1078124 </td><td>Bacteria stationary phase </td><td>Short </td><td>2 </td><td>pe 2 × 50 nt </td><td>5 960 882 </td></tr><tr><td>GSM1078125 </td><td>Bacteria stationary phase </td><td>Short </td><td>3 </td><td>pe 2 × 50 nt </td><td>5 559 756 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All RNA samples were depleted in ribosomal RNA using the RiboMinus™ protocol and separated in short (&lt;200 nt) and long (&gt;200 nt) fractions. Note that nodule libraries contain a mixture of <em>S. meliloti</em> and <em>M. truncatula</em> transcriptomes. Figures indicated here correspond to <em>S. meliloti</em> sequence reads only.</p></div></div><div content-id=""><div class="fn"><p>pe, paired ends; se, single end.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030368" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5030369 id="5030369" class="section-title" data-legacy-id=s2f>2.6. Read mapping</h3>
<p>Reads were mapped to the genome using the procedure as described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials and Methods</a>. For paired-end reads, all positions between the two reads were considered as transcribed. All transcription data can be visualized in the genome browser (<a class="link link-uri" href="https://lipm-browsers.toulouse.inra.fr/gb2/gbrowse/GMI11495-Rm2011G">https://lipm-browsers.toulouse.inra.fr/gb2/gbrowse/GMI11495-Rm2011G</a>).</p>                            <h3 scrollto-destination=5030371 id="5030371" class="section-title" data-legacy-id=s2g>2.7. Semi-conditional random field and 
associated features</h3>
<p>The mathematical model of semi-conditional random field (CRF)<sup><a reveal-id="DST014C43" data-open="DST014C43" class="link link-ref link-reveal xref-bibr">43</a></sup> has been used for gene finding in the eukaryotic gene finders, such as CRAIG<sup><a reveal-id="DST014C44" data-open="DST014C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and CONRAD,<sup><a reveal-id="DST014C45" data-open="DST014C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and implicitly used in EuGene from its creation. The semi-CRF model in EuGene-P is used to define an optimal segmentation of each strand of the genomic sequence into a succession of biologically meaningful regions. For one strand, the segmentation is defined by a succession of regions <strong>s</strong>= (<em>s</em><sub>1</sub> … <em>s<sub>q</sub></em>). Each region <em>s<sub>i</sub></em> = (<em>b<sub>i</sub></em>, <em>l<sub>i</sub></em>, <em>t<sub>i</sub></em>) starts at position <em>b<sub>i</sub></em>, has length <em>l<sub>i</sub></em> and labels <em>t<sub>i</sub></em>. A label can be any of {<em>IG</em>, <em>UTR</em>5<em>′</em>, <em>UIR</em>, <em>UTR</em>3′, <em>ncRNA</em>, <em>CDS</em><sub>1</sub>, <em>CDS</em><sub>2</sub>, <em>CDS</em><sub>3</sub>, <em>CDS</em><sub>1:2</sub>, <em>CDS</em><sub>2:3</sub>, <em>CDS</em><sub>1:3</sub>}, where <em>IG</em> stands for intergenic, <em>UTR</em>5′, <em>UIR</em> and <em>UTR</em>3′ for untranslated regions of coding genes, <em>ncRNA</em> for non-coding RNA genes, <em>CDS<sub>i</sub></em> for coding regions in frame <em>i</em> and <em>CDS<sub>i</sub></em><sub>:<em>j</em></sub> for overlapping coding regions in frame <em>i</em> and <em>j</em>. See Fig. <a reveal-id="DST014F1" data-open="DST014F1" class="link link-reveal link-table xref-fig">1</a> for an example. </p>                        <a id="5030373" scrollto-destination="5030373"></a>
<div id="DST014F1" data-id="DST014F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01401.jpeg?Expires=2147483647&Signature=izrggwvCqyKk9wiipqyJ0Kr0B9y5eqwt5mZAg7XhrgmhyTcvaaynVIyPh9agSe4Vp25g0yYKDNWnd4Mt3A9S-OezPPcnlS7JbPLl0fhGJ4cbLajpzzTJaMrIJs4KAsQeTEsuxLThpRPFEqlePa3Gpt2JtKG8Y9q9SgYmEz92XurXZRpZ0hSOSnrlpM8DOYiQUavDouDvAojRLMFTcWfCVf64Ot74I9BCCDWOrFhTv3atAKZGT8rnzJRRRmhot0TMw3dQYPrBXxUnAHqUbYiVfhTBkoto6iGwp6ZqgPrDnAZq78Z1BtGPJMPNi4S5D1lo634Cl5DgsusK9r2yyfS0Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A prokaryotic genomic sequence and the corresponding annotation defined as a sequence of typed regions. Each region has a specific label (or state) that defines its type. Beyond coding regions (e.g. CDS1) and intergenic regions (IG), an annotation may identify untranslated transcribed regions at the extremities of coding transcripts (5′ and 3′ UTRs), untranslated internal regions (or UIR, between two CDSs in a transcript) and ncRNA genes. Specific region types are also used to label overlapping CDSs. In this figure, the region labelled CDS1:3 corresponds to the overlap of a CDS in frame 1 with another CDS in frame 3." path-from-xml="dst01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030373/dst01401.jpeg" path-from-xml="dst01401.jpeg" target="_blank">View large</a><a section="5030373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01401.jpeg?Expires=2147483647&Signature=WJPcBy3MzMcszcMZk8s~V1boGfvYsfMe6jwnXR~ZKlQLKLLROwDV1ytpxa9hNGx~0gU0TSmb8yg4TLySpWmb5KDnbL3IHCiAdj6m2riaNiUtFBQGSwUKabLzuNpyj66dPtPPCeU1pWhjbsfcGDERj8ZYX6MiAVGmqP8nWGBKap9fXuwzZXR8ZvsFy9rDFyU1EKoR8gzil6n2Fl737VAi33sjhOXieYTZzuKFg6vsEiUzHkQcivJb-wnBm3EMCRkYSG6buJeqadwyfb1uk1U1QLADVG0bPfhTAs~F76UwNpZinJA~zh-AabR3MjFz9ZwLXPq2IUwiZcyOFYuaPnLNWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030373&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A prokaryotic genomic sequence and the corresponding annotation defined as a sequence of typed regions. Each region has a specific label (or state) that defines its type. Beyond coding regions (e.g. CDS<sub>1</sub>) and intergenic regions (IG), an annotation may identify untranslated transcribed regions at the extremities of coding transcripts (5′ and 3′ UTRs), untranslated internal regions (or UIR, between two CDSs in a transcript) and ncRNA genes. Specific region types are also used to label overlapping CDSs. In this figure, the region labelled CDS<sub>1:3</sub> corresponds to the overlap of a CDS in frame 1 with another CDS in frame 3.</p></div></div><div content-id="DST014F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01401.jpeg?Expires=2147483647&Signature=izrggwvCqyKk9wiipqyJ0Kr0B9y5eqwt5mZAg7XhrgmhyTcvaaynVIyPh9agSe4Vp25g0yYKDNWnd4Mt3A9S-OezPPcnlS7JbPLl0fhGJ4cbLajpzzTJaMrIJs4KAsQeTEsuxLThpRPFEqlePa3Gpt2JtKG8Y9q9SgYmEz92XurXZRpZ0hSOSnrlpM8DOYiQUavDouDvAojRLMFTcWfCVf64Ot74I9BCCDWOrFhTv3atAKZGT8rnzJRRRmhot0TMw3dQYPrBXxUnAHqUbYiVfhTBkoto6iGwp6ZqgPrDnAZq78Z1BtGPJMPNi4S5D1lo634Cl5DgsusK9r2yyfS0Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A prokaryotic genomic sequence and the corresponding annotation defined as a sequence of typed regions. Each region has a specific label (or state) that defines its type. Beyond coding regions (e.g. CDS1) and intergenic regions (IG), an annotation may identify untranslated transcribed regions at the extremities of coding transcripts (5′ and 3′ UTRs), untranslated internal regions (or UIR, between two CDSs in a transcript) and ncRNA genes. Specific region types are also used to label overlapping CDSs. In this figure, the region labelled CDS1:3 corresponds to the overlap of a CDS in frame 1 with another CDS in frame 3." path-from-xml="dst01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030373/dst01401.jpeg" path-from-xml="dst01401.jpeg" target="_blank">View large</a><a section="5030373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01401.jpeg?Expires=2147483647&Signature=WJPcBy3MzMcszcMZk8s~V1boGfvYsfMe6jwnXR~ZKlQLKLLROwDV1ytpxa9hNGx~0gU0TSmb8yg4TLySpWmb5KDnbL3IHCiAdj6m2riaNiUtFBQGSwUKabLzuNpyj66dPtPPCeU1pWhjbsfcGDERj8ZYX6MiAVGmqP8nWGBKap9fXuwzZXR8ZvsFy9rDFyU1EKoR8gzil6n2Fl737VAi33sjhOXieYTZzuKFg6vsEiUzHkQcivJb-wnBm3EMCRkYSG6buJeqadwyfb1uk1U1QLADVG0bPfhTAs~F76UwNpZinJA~zh-AabR3MjFz9ZwLXPq2IUwiZcyOFYuaPnLNWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030373&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A prokaryotic genomic sequence and the corresponding annotation defined as a sequence of typed regions. Each region has a specific label (or state) that defines its type. Beyond coding regions (e.g. CDS<sub>1</sub>) and intergenic regions (IG), an annotation may identify untranslated transcribed regions at the extremities of coding transcripts (5′ and 3′ UTRs), untranslated internal regions (or UIR, between two CDSs in a transcript) and ncRNA genes. Specific region types are also used to label overlapping CDSs. In this figure, the region labelled CDS<sub>1:3</sub> corresponds to the overlap of a CDS in frame 1 with another CDS in frame 3.</p></div></div><p>The linear semi-CRF model computes the score of a segmentation (<em>s</em><sub>1</sub>, …, <em>s<sub>q</sub></em>) of a given input sequence as a linear combination of functions representing individual features of the segmentation. Each feature scores a region <em>s<sub>i</sub></em> based on its length <em>l<sub>i</sub></em>, its label <em>t<sub>i</sub></em>, the label of the previous segment <em>t<sub>i</sub></em><sub>−1</sub> and some evidence <strong>x</strong> (including the DNA sequence). EuGene-P relies more specifically on three types of features:
Each feature can be understood as generating votes in favour of some annotations. After a learning phase, each feature receives a weight representing a ‘confidence’. The annotation that collects the maximum weighted sum of votes is considered as the optimal prediction. The usual probabilistic interpretation of CRFs, the formal definition of all features used inside EuGene-P and associated training and prediction algorithms are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials and Methods</a>.</p><ul class="roman-lower"><li><p>Contents features, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst014ileq1.gif?Expires=2147483647&amp;Signature=bldeODmbppNVkkeZDu1fkOscAGAuMj1wehPheNYNftwMgDRgGUe4wILDpnV0N4Zc8eCVMkXbPUQ04P-7uwslCGjbgkaZDJvJyymU4awfRtw2dKI99rgKuNakCNl8~lhroR-rLmE-qrwDDtZXo863N53LuBNLXYIjFJO2ExzEFUDFp4s68kO2UUVQx7ciTr6kHsdCLKL7IXfIOyo1VgvlMd6djMOO-dXG4mSCY0hWHcC14km2OuxIhJbCuLcRmMrAQhQhsB08SjtgZLzpmCeFrtygnev24V2jtwuN3rEO~ul-~Q8TTIOvrRyBBygY5J0ZMm4upVSKLANaXa1p94oAag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dst014ileq1.gif" />⁠</span>, score the fact that a region <em>s<sub>i</sub></em> has received label <em>t<sub>i</sub></em>. For example, if the nucleotides in the region <em>s<sub>i</sub></em> appear in an alignment with a known protein, a ‘protein alignment’ feature will score positively if the associated label <em>t<sub>i</sub></em> represents a coding region in the frame/strand indicated by the alignment.</p></li><li><p>Signal features, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst014ileq2.gif?Expires=2147483647&amp;Signature=hxYEgURImX6Lw9DwauFUZn5zUkCIIoVwm6Hm8cTCCJQ~UwxUqvTVjJ1pIzLt~k26OXLwsjFippOD1oNts--5pWPMgtU7eqnmOCkPFv2keIHtp9i5a6K7dapfjyK6vCt2MEZAd2ySTGFnMQ-Smah7WyC3qci~1k29mxMQwhU~dBEojvWbCtvpNxpSdsC9lzXXKvRIsEsjGGT83gSI406CchT4UmfwQSRoozbUhcSfP~~U3Oh-CqOegE5V4ow~6dmnJsZruBbwsZmx9PYTPEtbGDKyaCFasYObUPQH933NFTIm1kVqZLJvY25dA4-ubrdb-MrFFTB~TcKwchOJzeV53Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dst014ileq2.gif" />⁠</span>, score the fact that a region <em>s<sub>i</sub></em> with label <em>t<sub>i</sub></em> starts at position <em>b<sub>i</sub></em> after a region with label <em>t<sub>i</sub></em><sub>−1</sub>. For example, a ‘RNA-Seq sharp depth upshift’ feature will score positively if <em>s<sub>i</sub></em><sub>−1</sub>, labelled as an intergenic region, is followed by <em>s<sub>i</sub></em> defining a transcribed region, and a sharp upshift in the transcription level is observed on mapped RNA-Seq around position <em>b<sub>i</sub></em>.</p></li><li><p>Length features, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst014ileq3.gif?Expires=2147483647&amp;Signature=GlDpk0S255mzewmwOssLQtuAoKawLMpQK6cv3BTsWFNgcZCsNmeuPOz0mYd6V57igEFzXTLDWy2trA6MBoogZqJuDx2QOeO1e1lglaqj-cmiJ~HsRLiL6eXfrfJumqbcMhh~6HQw82LSb~vDLVhzHZrDMZm0OgsPk9XXnFq0wkyRjLDY26~BbrMg3TJ8ar3gL7iegloO1J0P~lQ9x6eRCS0pCjDjetRNuUFZCJqtP91Q-sOrm4EkLpgpLH4b1SW~ZgTn0FO6tu4Cufz9a-t6Z0oKEOkjlg7NVWpyu0kx69uDoUkxnAmThwI~7CO83LZ49i6nDU6iY~j4zy5psjsVjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dst014ileq3.gif" />⁠</span>, score the fact that a segment <em>s<sub>i</sub></em> has a given length. A typical example would be a feature scoring against extremely short CDSs.</p></li></ul>                            <h3 scrollto-destination=5030376 id="5030376" class="section-title" data-legacy-id=s2h>2.8. Transcriptome analysis</h3>
<p>Differential expression of identified genes was calculated with R v2.13.0 using DESeq v1.4.1<sup><a reveal-id="DST014C46" data-open="DST014C46" class="link link-ref link-reveal xref-bibr">46</a></sup> available in Bioconductor v2.8. DESeq utilizes a negative binomial distribution for modelling read counts per transcript and implements a method for normalizing the counts. Variance was estimated using the per-condition argument. <em>P</em>-values are adjusted for multiple testing using the Benjamini and Hochberg method.<sup><a reveal-id="DST014C47" data-open="DST014C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p>                            <h3 scrollto-destination=5030378 id="5030378" class="section-title" data-legacy-id=s2i>2.9. Quantitative RT-PCR analyses</h3>
<p>Reverse transcription was performed using Superscript II reverse transcriptase (Invitrogen) with random hexamers as primers. RNA samples isolated from at least three independent experiments were tested for each condition. Real-time PCRs were run on a LightCycler system (Roche) using the FastStart DNA MasterPLUS SYBRGreen I kit (Roche) according to the manufacturer's instructions.</p><p>For gene expression normalization, six reference genes were selected from the RNA-Seq data of the current study, on the basis of their similar levels of expression in both culture conditions (exponential and stationary growth phases) and <em>M. truncatula</em> nodules. The expression level of these genes was then examined by qRT-PCR in wild-type and <em>rpoE</em>2 mutant strains grown at 28 and 40°C, and expression data were computed using the NormFinder application.<sup><a reveal-id="DST014C48" data-open="DST014C48" class="link link-ref link-reveal xref-bibr">48</a></sup> SMc00519 and SMb21134 were found as the more stably expressed genes by NormFinder and were therefore used as references for qRT-PCR normalization in our conditions. Oligonucleotides sequences used for PCR are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h2 scrollto-destination=5030381 id="5030381" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5030382 id="5030382" class="section-title" data-legacy-id=s3a>3.1. A new integrative annotation tool for prokaryotic genomes</h3>
<p>One of the main results of this work is the definition of an integrative gene finder for prokaryotic gene prediction, allowing automatic incorporation of various sources of evidence in the prediction process, including oriented RNA-Seq data. The produced annotation not only accounts for statistical properties of observed open reading frames, but also for consistency with a variety of experimental data, thus minimizing subsequent manual expert annotation work.</p><p>We designed EuGene-P on the basis of the eukaryotic gene finder, EuGene.<sup><a reveal-id="DST014C14" data-open="DST014C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST014C20" data-open="DST014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> EuGene is able to incorporate the various types of information for enhancing its predictive power and has been used for the annotation of several genomes.<sup><a reveal-id="DST014C21  DST014C27" data-open="DST014C21  DST014C27" class="link link-ref link-reveal xref-bibr">21–27</a></sup> As all recent integrative gene finders, EuGene does not rely on a full generative probabilistic model, such as Hidden Markov Models,<sup><a reveal-id="DST014C49" data-open="DST014C49" class="link link-ref link-reveal xref-bibr">49</a></sup> that would require the expensive and unrealistic probabilistic modelling of all dependencies between the available information, but on a dedicated discriminative model. Formally, EuGene-P as EuGene can be described as semi-linear CRF-, or SL-CRF-,<sup><a reveal-id="DST014C43" data-open="DST014C43" class="link link-ref link-reveal xref-bibr">43</a></sup> based predictor. A CRF is a variant of Markov random fields, aimed at capturing the conditional probability of a succession of unknown discrete random variables <strong>y</strong>=<strong></strong>(<em>y</em><sub>1</sub> … <em>y<sub>n</sub></em>) given observed variables <strong>x</strong> (the available evidence). From such a model, the values of the unknown variables <strong>y</strong> can be reconstructed as the most probable ones given the available evidence <strong>x</strong>. In gene finding, the genomic sequence and the available information (mapped reads, other similarities …) will be represented as the evidence <strong>x</strong>. The unknown (or hidden) variables <strong>y</strong> are used to represent structural annotations. We therefore associate one variable <em>y<sub>i</sub></em> with every base in the sequence. The variable <em>y<sub>i</sub></em> specifies the annotation label (or state) of the base at position <em>i</em> (inside a CDS, an intergenic region …). In eukaryotic genomes, despite the accumulating evidence of overlapping functional regions, existing gene finders usually assume that each base belongs to just one type of region. The above model, with one variable <em>y<sub>i</sub></em> per base, is perfectly suitable to perform the gene prediction on both strands simultaneously. In gene-dense prokaryotic genomes, overlapping functional regions is a rather frequent event. Genes can overlap with neighbouring genes on either strand. The genomic model we chose is therefore an unusual stranded model. This model describes how genes appear on one strand, independently of the other.</p><p>Formally, we have to enumerate the list of possible states for a nucleotide in an annotation. As shown in Fig. <a reveal-id="DST014F1" data-open="DST014F1" class="link link-reveal link-table xref-fig">1</a>, since we restrict ourselves to a single strand, a typical prokaryotic sequence will contain bases belonging to either an intergenic region (denoted as <em>IG</em>), a transcribed non-translated region of a coding gene (denoted as <em>UTR</em>5<em>′</em>, <em>UIR</em> or <em>UTR</em>3<em>′</em> depending on its location in the gene), a ncRNA gene (denoted as <em>ncRNA</em>), a non-overlapping CDS region in a given coding frame <em>i</em> (denoted as <em>CDS<sub>i</sub></em>) or a region where two CDS in different coding frames <em>i</em> and <em>j</em> overlap (denoted as <em>CDS<sub>i:j</sub></em>).</p><p>Overall, each variable <em>y<sub>i</sub></em>, representing possible annotations for nucleotide <em>i</em>, may take 11 different states. Such states cannot appear arbitrarily in the genome sequence. For example, a CDS must start and end at specific codons. The CRF model can capture gene structures described as simple automaton. The automaton used in EuGene-P is described in Fig. <a reveal-id="DST014F2" data-open="DST014F2" class="link link-reveal link-table xref-fig">2</a>. Transitions between possible states in the automaton correspond to the occurrence of specific biological signals in the sequence. Transcription Starts and Transcription Ends denote the start and end of transcripts (containing coding genes or ncRNA genes), whereas Translation Starts and Translation Ends (denoted as TS<em><sub>i</sub></em> and TE<em><sub>i</sub></em>, respectively, where <em>i</em> is the frame of the corresponding codon in the sequence) enable to, respectively, start or end a CDS inside a transcript and possibly inside another CDS in a different frame. Finally, the conditional probability distribution that relates the evidence in <strong>x</strong> and possible annotations in <strong>y</strong> must be described. In CRF, this is done through a set of features. Every type of experimental or statistical evidence is represented by one (or more) feature. A feature is a small mathematical function that uses some available evidence to vote in favour of (or against) the prediction of specific elements. For example, a ‘protein similarities’ feature would vote in favour of CDS prediction in the regions that have similarities with known proteins. A precise definition of the different features available in EuGene-P is given in Materials and Methods. Once the set of features used for gene finding is fixed, the CRF model can be trained. This training process computes a multiplicative factor for each feature that determines a feature-specific confidence. The prediction is then in charge of finding the annotation that has maximum conditional probability. This is the prediction that accumulates most support from all features. Overall, the mathematical model and associated software provide a qualitative improvement in terms of its abilities in predicting TSSs, untranslated transcribed regions, overlapping CDSs, ncRNA genes and antisense genes. </p>                        <a id="5030387" scrollto-destination="5030387"></a>
<div id="DST014F2" data-id="DST014F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01402.jpeg?Expires=2147483647&Signature=Il01HUR-v95RyHZdFNOLISkU8b3CXVt0FRAuk5UbdU--QTZYWAPmeXrdLnYSfdE1C7As4tG4N2XTq9O~4ESjq~cZNwZG51PY6KJvkBHgDahw7gqSr2MLoO9ziIjMmmIz2IgUtuK6rIIS1a7cBSNmauaWAnPWunqPpqlML5ob7YW3~g31oTjMuWkolCGviGxJmVqM9hw0MxUDOUvdZHyg0GgmNfx75QiYMk99FUoPBVjTSiliROrBa0Ku2e44UfwEmR1IStBqiFOeC5RE0jeezGVPAPSi8yXZ64RE-YPRHuLAHcCnvW8gf-zFtD4iHXSMhz0MixypmCZ~aBnYclLftg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The different states and possible transitions between these states used inside EuGene-P." path-from-xml="dst01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030387/dst01402.jpeg" path-from-xml="dst01402.jpeg" target="_blank">View large</a><a section="5030387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01402.jpeg?Expires=2147483647&Signature=nJcX7v9yRlFhf7qBDfyTieX2ub3MbLBUeQnk4OA3qd1b5pC0Sxp4hLQxoHRRNslX24GHjvGPgYSohg6rBW0Xg8YX2gOYljep-MD2RXe0ggJlWJW7ZH8sD8gGGiMKpx593yOsafHfN2r7m5zO344CSBT-NEeZvREFLQ3ylnA-FNEKW1I9imeLwCQPMd1MPDI-43iTtQg0pMTilz6X1qvpRnJrzok-T4GOktlO0eViVHsbV9FHH~s06HDNVV0tiL2M9jSna6EVNFYkp6glsbkTEpx81EOF8uRhpDgRSxDonZlgYg9O8RNWmYrIv-Fm3FBE7cxU7gdDkEpZnAloRQ5MPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030387&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The different states and possible transitions between these states used inside EuGene-P.</p></div></div><div content-id="DST014F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01402.jpeg?Expires=2147483647&Signature=Il01HUR-v95RyHZdFNOLISkU8b3CXVt0FRAuk5UbdU--QTZYWAPmeXrdLnYSfdE1C7As4tG4N2XTq9O~4ESjq~cZNwZG51PY6KJvkBHgDahw7gqSr2MLoO9ziIjMmmIz2IgUtuK6rIIS1a7cBSNmauaWAnPWunqPpqlML5ob7YW3~g31oTjMuWkolCGviGxJmVqM9hw0MxUDOUvdZHyg0GgmNfx75QiYMk99FUoPBVjTSiliROrBa0Ku2e44UfwEmR1IStBqiFOeC5RE0jeezGVPAPSi8yXZ64RE-YPRHuLAHcCnvW8gf-zFtD4iHXSMhz0MixypmCZ~aBnYclLftg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The different states and possible transitions between these states used inside EuGene-P." path-from-xml="dst01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030387/dst01402.jpeg" path-from-xml="dst01402.jpeg" target="_blank">View large</a><a section="5030387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01402.jpeg?Expires=2147483647&Signature=nJcX7v9yRlFhf7qBDfyTieX2ub3MbLBUeQnk4OA3qd1b5pC0Sxp4hLQxoHRRNslX24GHjvGPgYSohg6rBW0Xg8YX2gOYljep-MD2RXe0ggJlWJW7ZH8sD8gGGiMKpx593yOsafHfN2r7m5zO344CSBT-NEeZvREFLQ3ylnA-FNEKW1I9imeLwCQPMd1MPDI-43iTtQg0pMTilz6X1qvpRnJrzok-T4GOktlO0eViVHsbV9FHH~s06HDNVV0tiL2M9jSna6EVNFYkp6glsbkTEpx81EOF8uRhpDgRSxDonZlgYg9O8RNWmYrIv-Fm3FBE7cxU7gdDkEpZnAloRQ5MPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030387&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The different states and possible transitions between these states used inside EuGene-P.</p></div></div>                            <h3 scrollto-destination=5030388 id="5030388" class="section-title" data-legacy-id=s3b>3.2. Generation of high-quality <em>Sinorhizobium meliloti</em> 2011 genome and transcriptome sequencing data</h3>
<p>The genome sequence of the streptomycin-resistant derivative of <em>S. meliloti</em> strain 2011 was generated using a combination of 454 (Roche) and Solexa (Illumina) technologies that provided a total coverage of ∼190 genome equivalents. The assembly of the complete genome sequence was guided by the <em>S. meliloti</em> 1021 sequence that was determined previously.<sup><a reveal-id="DST014C31" data-open="DST014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The comparison of these two DNA sequences revealed 463 polymorphisms, including 332 SNPs, 119 Indels and 12 large deletions or insertions (&gt;10 bp; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). In addition to these differences, a 3564-nt region was specifically present in the chromosome of Sm2011 but not in Sm1021. This insertion, located between SMc03253 and SMc03254, was checked and confirmed by PCR amplification. This region contains a new gene, referred to as SMc06990, encoding a glutamine synthetase domain fused to a putative carbamoyl-phosphate synthase large chain ATP-binding protein, an enzyme that catalyzes the production of carbamoyl phosphate, which can be subsequently employed in both pyrimidine and arginine biosyntheses,<sup><a reveal-id="DST014C50" data-open="DST014C50" class="link link-ref link-reveal xref-bibr">50</a></sup> as well as the SMc06992 gene which is a duplication (100% identical) of the SMc03253 gene preceded by two copies of its promoter region. The promoter region of SMc03253 was previously shown to be a duplication of the whole promoter region of <em>fixK</em>, a gene whose expression is controlled by the key symbiotic transcription regulator FixJ.<sup><a reveal-id="DST014C51" data-open="DST014C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Sanger DNA sequencing of 71 polymorphic regions including 64 putative frameshifts showed that 55 of them were actually errors on the reference sequence Sm1021, whereas eight were errors on the Sm2011 sequence and only eight were real polymorphisms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). These results suggest that presumably only ∼10% of the 463 polymorphisms are real (most being errors in the Sm1021 sequence).</p><p>To obtain a global view of the transcriptome of Sm2011, RNAs were prepared from bacteria grown in three very different physiological conditions to cover a large number of expressed genes. These include RNAs extracted from bacteria grown in liquid cultures (in both exponential and stationary growth phases) and from 10-day-old nodules in which bacteria were differentiated in nitrogen-fixing bacteroids.<sup><a reveal-id="DST014C52" data-open="DST014C52" class="link link-ref link-reveal xref-bibr">52</a></sup> For each condition, three biological replicates were performed to assess data reproducibility and reliability, and short (&lt;200 nt) and long (&gt;200 nt) RNA fractions were separately analysed. RNA samples were depleted in both ribosomal RNAs and the highly abundant tRNA-Ala using a <em>S. meliloti-</em>specific capture set of oligonucleotides and were sequenced using the stranded Illumina protocol.<sup><a reveal-id="DST014C53" data-open="DST014C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DST014C54" data-open="DST014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> This protocol, based on ligation of adapters directly to the 3′ and 5′ ends of the RNA molecules, has the advantage of preserving the information about the transcript orientation. The RNA-Seq libraries generated in this study are listed in Table <a reveal-id="DST014TB1" data-open="DST014TB1" class="link link-reveal link-table xref-fig">1</a>. The resulting sequences were mapped onto the <em>S. meliloti</em> genome sequence. RNA-Seq data appeared to be highly reproducible as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> (Pearson correlation values varied from 0.899 to 0.998 between biological replicates). Of the 6308 <em>S. meliloti</em> annotated CDSs (see below), the expression of 5717 (90%) was detected in at least one experimental condition [raw expression level summed in the six libraries (short and long) of one condition was above 50 reads]. The number of mapped reads per nucleotide (summed values from triplicates) was visualized using the Apollo interface.<sup><a reveal-id="DST014C55" data-open="DST014C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Figure <a reveal-id="DST014F3" data-open="DST014F3" class="link link-reveal link-table xref-fig">3</a> illustrates a 3-kb region of the genome showing short and long RNAs in two conditions. The expression profiles of bacteria grown in exponential and stationary phases were compared with two previous studies performed in similar conditions, but based on oligonucleotide microarrays.<sup><a reveal-id="DST014C30" data-open="DST014C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Among the 804 genes found to be up-regulated in stationary phase in any of these studies, 631 genes (78%) were consistently found in our study to be up-regulated in the stationary phase (&gt;2-fold, <em>P</em> &lt; 0.05) either in the short or long RNA libraries. This percentage is similar to the percentage of common up-regulated genes found in the two microarray studies (80%), which attest to the good quality of our RNA-Seq data. </p>                        <a id="5030391" scrollto-destination="5030391"></a>
<div id="DST014F3" data-id="DST014F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01403.jpeg?Expires=2147483647&Signature=dcq9V0cDk1FN0fpFLIPMgBzSvzARObMYdp3q6Jf1Bdlq~1JLTiQxpJ-nLLFPIQC-EBq4lpJcGQmsClnj9bDR61heaDzfIoUknU6zqWB7Er~LFESu2cNeQRi-4U~yRrc8zpfZ2EubhEIu8gnOIe5XU~RITx5m0uTmmUCKLHLGbkFBTIV1xV7OV9rAcrjjl4T5Xo2Z5nNWFwQ7NeyzhQraKUc2xw5URgGQ-cC9moym89Nzzr2NwK2H3p8SwK1i61X4BMHeRj9EvtFKHecKr1sUP0H1tqZsIwO12tHmpP3tyO4oiAGh0jNXn4CqkGKS4toOGYw8XUjRyxK2whSDKusaig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of a genomic region in Apollo. Apollo represents the annotation on both strands (upper and lower part of the figure) as well as the expression level of the mapped RNA-Seq data from short and long RNA libraries in exponential and stationary growth phase conditions. Reads mapped on the plus strand are shown in colour, and reads mapped on the minus strand are in grey. Y-axis represents the number of reads summed from triplicates. The upper limit was set at 300 reads. This region contains several annotated non-coding (in green) and protein-coding (in blue) genes, full blue squares correspond to CDSs and open blue squares correspond to 5′ and 3′ UTRs." path-from-xml="dst01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030391/dst01403.jpeg" path-from-xml="dst01403.jpeg" target="_blank">View large</a><a section="5030391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01403.jpeg?Expires=2147483647&Signature=TkbxZTW12eeaUuK~jZs6vxIhHgSf36hmGXN2mKBg2IR-~Ul17GDv7AEmNc58hfp-37gVYF2TfWGnnG0-gTthZjLapxmyUi6dawCz95jsa-oEh2fbB1nBdZZYzvfoNbb5syGRueoDC8uDOWKL~yq3mYMsVjVETBjuZTBkcJQnJ2Cn~rpZhs~Kw1sQWsevyxQfDhoZR0FWJH2HS1zCox4Hpv0RI~-w40rGiGhzQ0svnLG2r5-knL0J8grpY363SX-JHTp7turNgwYP97OsfzmNMjcu8liJoT2zkpNL7OePG~oWSM9L5nQ3ZtVG7fl-nH9S7xPyfKuBy9dio4ndmtFPUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030391&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of a genomic region in Apollo. Apollo represents the annotation on both strands (upper and lower part of the figure) as well as the expression level of the mapped RNA-Seq data from short and long RNA libraries in exponential and stationary growth phase conditions. Reads mapped on the plus strand are shown in colour, and reads mapped on the minus strand are in grey. <em>Y</em>-axis represents the number of reads summed from triplicates. The upper limit was set at 300 reads. This region contains several annotated non-coding (in green) and protein-coding (in blue) genes, full blue squares correspond to CDSs and open blue squares correspond to 5′ and 3′ UTRs.</p></div></div><div content-id="DST014F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/m_dst01403.jpeg?Expires=2147483647&Signature=dcq9V0cDk1FN0fpFLIPMgBzSvzARObMYdp3q6Jf1Bdlq~1JLTiQxpJ-nLLFPIQC-EBq4lpJcGQmsClnj9bDR61heaDzfIoUknU6zqWB7Er~LFESu2cNeQRi-4U~yRrc8zpfZ2EubhEIu8gnOIe5XU~RITx5m0uTmmUCKLHLGbkFBTIV1xV7OV9rAcrjjl4T5Xo2Z5nNWFwQ7NeyzhQraKUc2xw5URgGQ-cC9moym89Nzzr2NwK2H3p8SwK1i61X4BMHeRj9EvtFKHecKr1sUP0H1tqZsIwO12tHmpP3tyO4oiAGh0jNXn4CqkGKS4toOGYw8XUjRyxK2whSDKusaig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of a genomic region in Apollo. Apollo represents the annotation on both strands (upper and lower part of the figure) as well as the expression level of the mapped RNA-Seq data from short and long RNA libraries in exponential and stationary growth phase conditions. Reads mapped on the plus strand are shown in colour, and reads mapped on the minus strand are in grey. Y-axis represents the number of reads summed from triplicates. The upper limit was set at 300 reads. This region contains several annotated non-coding (in green) and protein-coding (in blue) genes, full blue squares correspond to CDSs and open blue squares correspond to 5′ and 3′ UTRs." path-from-xml="dst01403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5030391/dst01403.jpeg" path-from-xml="dst01403.jpeg" target="_blank">View large</a><a section="5030391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst01403.jpeg?Expires=2147483647&Signature=TkbxZTW12eeaUuK~jZs6vxIhHgSf36hmGXN2mKBg2IR-~Ul17GDv7AEmNc58hfp-37gVYF2TfWGnnG0-gTthZjLapxmyUi6dawCz95jsa-oEh2fbB1nBdZZYzvfoNbb5syGRueoDC8uDOWKL~yq3mYMsVjVETBjuZTBkcJQnJ2Cn~rpZhs~Kw1sQWsevyxQfDhoZR0FWJH2HS1zCox4Hpv0RI~-w40rGiGhzQ0svnLG2r5-knL0J8grpY363SX-JHTp7turNgwYP97OsfzmNMjcu8liJoT2zkpNL7OePG~oWSM9L5nQ3ZtVG7fl-nH9S7xPyfKuBy9dio4ndmtFPUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5030391&ar=353499&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of a genomic region in Apollo. Apollo represents the annotation on both strands (upper and lower part of the figure) as well as the expression level of the mapped RNA-Seq data from short and long RNA libraries in exponential and stationary growth phase conditions. Reads mapped on the plus strand are shown in colour, and reads mapped on the minus strand are in grey. <em>Y</em>-axis represents the number of reads summed from triplicates. The upper limit was set at 300 reads. This region contains several annotated non-coding (in green) and protein-coding (in blue) genes, full blue squares correspond to CDSs and open blue squares correspond to 5′ and 3′ UTRs.</p></div></div>                            <h3 scrollto-destination=5030392 id="5030392" class="section-title" data-legacy-id=s3c>3.3. Annotation of the <em>Sinorhizobium meliloti</em> 2011 genome using EuGene-P</h3>
<p>EuGene-P inherits from EuGene its ability to integrate a variety of data. Selecting the most significant or informative sources of evidence is highly beneficial for the quality of the final annotation. We decided to use:
The translation Start and Stop features are generic (see Materials and Methods). EuGene-P allows the user to parameterize the definition of Stop and Start codons to deal with unusual codon tables.</p><ul class="roman-lower"><li><p>Similarities with known protein sequences modelled as a dedicated feature that votes for the prediction of coding regions in the corresponding coding frame (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials and Methods</a>). To identify similarities, we used the SwissProt database as a reliable general source of information for protein similarities. In addition, we used the proteome of the Sm1021 (set of all the protein sequences obtained by translating all CDS of the Sm1021 annotation) as a more specific source of information.</p></li><li><p>Mapped RNA-Seq data that indicate transcription activity. For transcribed sequences, we used RNA libraries of Sm2011 in exponential or stationary growth conditions and libraries of <em>S. meliloti</em>-colonized <em>M. truncatula</em> nodule tissues. All reads were mapped to the <em>S. meliloti</em> genome (Table <a reveal-id="DST014TB1" data-open="DST014TB1" class="link link-reveal link-table xref-fig">1</a>). The absolute expression level and the changes in relative expression levels were each exploited in a specific feature. The absolute expression level was used as an evidence of transcribed regions, while abrupt changes in expression, captured by the derivative of the log-level of the expression, indicate a possible TSS.</p></li><li><p>Interpolated Markov models derived from coding potential to help identifying coding genes. The 3-periodic Markov models were estimated on CDSs from a subset of the genes in the Sm1021 annotation. Those genes have a specific (non-automatic) gene name, indicating that they have gone through expert annotation. Because they are known to have different statistical compositions, one coding model was estimated on pSymA genes and another coding model estimated on genes from pSymB together with the chromosome.</p></li><li><p>Output of ncRNA prediction programs to help identifying RNA genes from known families. The genomic sequence of <em>S. meliloti</em> was analysed using tRNAscan-SE v1.23 (April 2002) for transfer RNA detection, RNAmmer (February 2006) for ribosomal RNAs and rfam_scan v1.0.2 with Rfam v10.0 (1446 families, April 2010) for other known ncRNA gene families. This produced a set of genomic regions predicted as ncRNA genes. Each of these intervals was used in a feature favouring ncRNA prediction in the region that contains them.</p></li></ul><p>Overall, the purely automated annotation of Sm2011 produced a total of 6483 coding genes and 2040 ncRNA (including tRNAs and rRNAs) genes. This raw annotation was then submitted to manual checking, leading to possible curation of predicted CDSs, UTRs and ncRNAs. Manual modifications were done using Apollo<sup><a reveal-id="DST014C55" data-open="DST014C55" class="link link-ref link-reveal xref-bibr">55</a></sup> to simultaneously visualize predicted elements and RNA-Seq expression levels in each condition (Fig. <a reveal-id="DST014F3" data-open="DST014F3" class="link link-reveal link-table xref-fig">3</a>). Each elementary modification typically impacts several levels. For instance, corrections of 5′/3′ ends of UTRs often corresponded to the removal or creation of a new ncRNA. Typically, a predicted ncRNA that appeared close to 5′ was removed, and the UTR enlarged to include the corresponding region. Overall, 100 ncRNAs were removed in this way and the corresponding region included in a UTR (47 5′ UTRs and 53 3′ UTRs), while 87 UTRs (35 5′ UTRs and 52 3′ UTRs) were modified and new ncRNAs annotated. Around 13% of protein-coding genes and 29% of nc genes were modified as described in Table <a reveal-id="DST014TB2" data-open="DST014TB2" class="link link-reveal link-table xref-fig">2</a>. However, it is important to note that nc genes and UTRs are difficult to discriminate even by expert analyses of RNA-Seq data. The manual curation led to the final annotation described in Table <a reveal-id="DST014TB3" data-open="DST014TB3" class="link link-reveal link-table xref-fig">3</a> and is available on the browser <a class="link link-uri" href="https://iant.toulouse.inra.fr/S.meliloti2011">https://iant.toulouse.inra.fr/S.meliloti2011</a>. In total 6308 protein-coding genes, 9 rRNAs, 55 tRNAs, 28 tRNAs precursors and 1876 ncRNAs were annotated. </p>                        <a id="5030396" scrollto-destination="5030396"></a>
<div content-id="DST014TB2" class="table-modal"><div class="table-wrap"><div id="DST014TB2" class="table-wrap-title" data-id="DST014TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Modifications performed on the automatic annotation during the manual curation process</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>5′ ends </th><th>3′ ends </th><th>CDS starts </th><th>CDS stops </th><th>Creations </th><th>Removals </th><th>Total number of modified genes </th></tr></thead><tbody><tr><td>Coding genes </td><td>350 </td><td>275 </td><td>135 </td><td>2 </td><td>19 </td><td>194 </td><td>835 </td></tr><tr><td>Non-coding genes </td><td>31 </td><td>151 </td><td> </td><td> </td><td>180 </td><td>252 </td><td>604 </td></tr></tbody></table></div><div content-id="DST014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>5′ ends </th><th>3′ ends </th><th>CDS starts </th><th>CDS stops </th><th>Creations </th><th>Removals </th><th>Total number of modified genes </th></tr></thead><tbody><tr><td>Coding genes </td><td>350 </td><td>275 </td><td>135 </td><td>2 </td><td>19 </td><td>194 </td><td>835 </td></tr><tr><td>Non-coding genes </td><td>31 </td><td>151 </td><td> </td><td> </td><td>180 </td><td>252 </td><td>604 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030396" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST014TB2" class="table-wrap-title" data-id="DST014TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Modifications performed on the automatic annotation during the manual curation process</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>5′ ends </th><th>3′ ends </th><th>CDS starts </th><th>CDS stops </th><th>Creations </th><th>Removals </th><th>Total number of modified genes </th></tr></thead><tbody><tr><td>Coding genes </td><td>350 </td><td>275 </td><td>135 </td><td>2 </td><td>19 </td><td>194 </td><td>835 </td></tr><tr><td>Non-coding genes </td><td>31 </td><td>151 </td><td> </td><td> </td><td>180 </td><td>252 </td><td>604 </td></tr></tbody></table></div><div content-id="DST014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>5′ ends </th><th>3′ ends </th><th>CDS starts </th><th>CDS stops </th><th>Creations </th><th>Removals </th><th>Total number of modified genes </th></tr></thead><tbody><tr><td>Coding genes </td><td>350 </td><td>275 </td><td>135 </td><td>2 </td><td>19 </td><td>194 </td><td>835 </td></tr><tr><td>Non-coding genes </td><td>31 </td><td>151 </td><td> </td><td> </td><td>180 </td><td>252 </td><td>604 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030396" target="_blank">View Large</a></div></div>                        <a id="5030397" scrollto-destination="5030397"></a>
<div content-id="DST014TB3" class="table-modal"><div class="table-wrap"><div id="DST014TB3" class="table-wrap-title" data-id="DST014TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Structural annotation of the <em>S. meliloti</em> 2011 genome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>CDSs (total number) </td><td>6308 </td></tr><tr><td> New (when compared with Sm1021) </td><td>125 </td></tr><tr><td>tRNAs </td><td>55 </td></tr><tr><td>tRNA primary transcripts </td><td>28 </td></tr><tr><td>rRNAs </td><td>9 </td></tr><tr><td>ncRNAs </td><td>1876 </td></tr><tr><td> Antisense to a protein-coding gene </td><td>1281 </td></tr><tr><td>TSSs (total number) </td><td>4840 </td></tr><tr><td> Predicted with high confidence </td><td>4077 </td></tr><tr><td> Predicted with low confidence </td><td>763 </td></tr><tr><td>Insertion sequences </td><td>94 </td></tr><tr><td>Repeated elements </td><td>618 </td></tr><tr><td> RIME </td><td>209 </td></tr><tr><td> MOTIF </td><td>256 </td></tr><tr><td> Sm-1 repeat </td><td>21 </td></tr><tr><td> Sm-2 repeat </td><td>8 </td></tr><tr><td> Sm-3 repeat </td><td>4 </td></tr><tr><td> Sm-4 repeat </td><td>73 </td></tr><tr><td> Sm-5 repeat </td><td>47 </td></tr></tbody></table></div><div content-id="DST014TB3" class="table-modal"><table><tbody><tr><td>CDSs (total number) </td><td>6308 </td></tr><tr><td> New (when compared with Sm1021) </td><td>125 </td></tr><tr><td>tRNAs </td><td>55 </td></tr><tr><td>tRNA primary transcripts </td><td>28 </td></tr><tr><td>rRNAs </td><td>9 </td></tr><tr><td>ncRNAs </td><td>1876 </td></tr><tr><td> Antisense to a protein-coding gene </td><td>1281 </td></tr><tr><td>TSSs (total number) </td><td>4840 </td></tr><tr><td> Predicted with high confidence </td><td>4077 </td></tr><tr><td> Predicted with low confidence </td><td>763 </td></tr><tr><td>Insertion sequences </td><td>94 </td></tr><tr><td>Repeated elements </td><td>618 </td></tr><tr><td> RIME </td><td>209 </td></tr><tr><td> MOTIF </td><td>256 </td></tr><tr><td> Sm-1 repeat </td><td>21 </td></tr><tr><td> Sm-2 repeat </td><td>8 </td></tr><tr><td> Sm-3 repeat </td><td>4 </td></tr><tr><td> Sm-4 repeat </td><td>73 </td></tr><tr><td> Sm-5 repeat </td><td>47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030397" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST014TB3" class="table-wrap-title" data-id="DST014TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Structural annotation of the <em>S. meliloti</em> 2011 genome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>CDSs (total number) </td><td>6308 </td></tr><tr><td> New (when compared with Sm1021) </td><td>125 </td></tr><tr><td>tRNAs </td><td>55 </td></tr><tr><td>tRNA primary transcripts </td><td>28 </td></tr><tr><td>rRNAs </td><td>9 </td></tr><tr><td>ncRNAs </td><td>1876 </td></tr><tr><td> Antisense to a protein-coding gene </td><td>1281 </td></tr><tr><td>TSSs (total number) </td><td>4840 </td></tr><tr><td> Predicted with high confidence </td><td>4077 </td></tr><tr><td> Predicted with low confidence </td><td>763 </td></tr><tr><td>Insertion sequences </td><td>94 </td></tr><tr><td>Repeated elements </td><td>618 </td></tr><tr><td> RIME </td><td>209 </td></tr><tr><td> MOTIF </td><td>256 </td></tr><tr><td> Sm-1 repeat </td><td>21 </td></tr><tr><td> Sm-2 repeat </td><td>8 </td></tr><tr><td> Sm-3 repeat </td><td>4 </td></tr><tr><td> Sm-4 repeat </td><td>73 </td></tr><tr><td> Sm-5 repeat </td><td>47 </td></tr></tbody></table></div><div content-id="DST014TB3" class="table-modal"><table><tbody><tr><td>CDSs (total number) </td><td>6308 </td></tr><tr><td> New (when compared with Sm1021) </td><td>125 </td></tr><tr><td>tRNAs </td><td>55 </td></tr><tr><td>tRNA primary transcripts </td><td>28 </td></tr><tr><td>rRNAs </td><td>9 </td></tr><tr><td>ncRNAs </td><td>1876 </td></tr><tr><td> Antisense to a protein-coding gene </td><td>1281 </td></tr><tr><td>TSSs (total number) </td><td>4840 </td></tr><tr><td> Predicted with high confidence </td><td>4077 </td></tr><tr><td> Predicted with low confidence </td><td>763 </td></tr><tr><td>Insertion sequences </td><td>94 </td></tr><tr><td>Repeated elements </td><td>618 </td></tr><tr><td> RIME </td><td>209 </td></tr><tr><td> MOTIF </td><td>256 </td></tr><tr><td> Sm-1 repeat </td><td>21 </td></tr><tr><td> Sm-2 repeat </td><td>8 </td></tr><tr><td> Sm-3 repeat </td><td>4 </td></tr><tr><td> Sm-4 repeat </td><td>73 </td></tr><tr><td> Sm-5 repeat </td><td>47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030397" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5030398 id="5030398" class="section-title" data-legacy-id=s3d>3.4. Identification of a high number of putative 
non-coding RNAs in <em>Sinorhizobium meliloti</em></h3>
<p>The number of predicted ncRNAs was remarkable (Table <a reveal-id="DST014TB3" data-open="DST014TB3" class="link link-reveal link-table xref-fig">3</a>). All of them, but five that were only detected by Rfam_scan, were supported by RNA-Seq expression data. Because the number of predicted ncRNAs was surprisingly high, we compared the automated raw predictions (before manual curation) with the set of 1102 small RNA candidates proposed in the previous RNA-Seq study of Schlüter <em>et al.</em><sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> In that study, sRNAs candidates were arbitrarily classified as trans-encoded, cis-encoded sense, cis-encoded antisense and mRNA leaders. Cis-encoded sense regions have been reported as probable mRNA degradation products in Schlüter <em>et al.</em><sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> We therefore excluded cis-encoded sense candidates from the comparison. We found that 77% of cis-encoded antisense candidates, 76% of trans-encoded candidates and 53% of mRNA-leader candidates were covered on &gt;50% of their length by regions that were predicted as non-translated transcribed regions (UTRs or ncRNA regions together covering 503 kb or 3.8% of all chromosomal and plasmid strands).</p><p>Regarding the 1876 ncRNAs that were predicted after the manual curation, a large part (68%) was found located antisense to a protein-coding gene. Antisense RNAs overlap either with the 5′end (10%), the 3′end (19%) or the central part (71%) of the gene found on the opposite strand. These results strongly support the current findings that antisense transcription activity is more widespread in bacteria than initially thought.<sup><a reveal-id="DST014C57" data-open="DST014C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DST014C58" data-open="DST014C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p><p>Our predicted ncRNAs displayed an average size of 107 nt, 94% ranging between 20 and 250 nt (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). This length distribution is consistent with the sizes of 50–348 nt observed by Schlüter <em>et al</em>.<sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Besides the 55 tRNAs, the nine rRNAs and the five well-characterized ncRNAs (<em>ffs</em>, <em>ssrS</em>, <em>ssrA</em>, <em>rnpB</em> and <em>incA</em>), only 36 additional ncRNA were classified by Rfam_scan in 18 known ncRNA families (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). A majority of them has thus a completely unknown function. Interestingly, analysis of expression patterns indicated that a large part of predicted ncRNAs were differentially expressed (&gt; or &lt;2-fold, <em>P</em> &lt; 0.01) between at least two of the three conditions studied: 152 were induced in symbiosis compared with free-living conditions while 1116 were induced, and 317 were repressed in stationary phase when compared with exponential growth phase (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>). These expression patterns support the idea that ncRNAs potentially play important regulatory functions in <em>S. meliloti</em> under these conditions.</p><p>Consistently with the study of Schlüter <em>et al.</em><sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> intergenic repeated elements previously identified in the genome of <em>S. meliloti</em>, like the RIME, MOTIF or Sm-1 to Sm-5 repeats,<sup><a reveal-id="DST014C59  DST014C61" data-open="DST014C59  DST014C61" class="link link-ref link-reveal xref-bibr">59–61</a></sup> were also transcribed and, thus, further increase the number of non-translated transcribed elements. Since reads corresponding to such repeated sequences could not be unambiguously mapped, it was difficult to estimate their relative expression levels and to determine whether they were all transcribed at a similar level.</p>                            <h3 scrollto-destination=5030403 id="5030403" class="section-title" data-legacy-id=s3e>3.5. EuGene-P identifies TSSs and efficiently delineates 5′ UTRs of mRNAs</h3>
<p>The RNA-Seq protocol used here allowed us to precisely predict the 5′ ends of RNAs. This is related to the fact that, prior to library constructions, RNA molecules were treated with the tobacco acid pyrophosphatase that converts the 5′ triphosphate group of native transcripts into a 5′ monophosphate capable of ligation with oligonucleotide adaptors (see Materials and methods). This procedure enabled the sequencing of 5′ RNA ends with a very high precision and thereby the identification of probable TSSs. TSS prediction was based on the identification of abrupt changes in expression level as assessed by the approximation of the derivative of the expression level logarithm. In total, 4077 TSSs of protein-coding genes or nc genes were predicted with good confidence (clear changes in expression), whereas 763 were predicted with a lower confidence. Compared with the existing Sm1021 annotation,<sup><a reveal-id="DST014C31" data-open="DST014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> 505 conserved CDSs had a modified start codon. This was a direct consequence of RNA-Seq data integration since the previously predicted start codon was usually located before the TSS predicted from RNA-Seq data, showing the interest of integrating RNA-Seq data for gene annotation.</p><p>To further evaluate EuGene-P predictions, we compared our data with TSSs experimentally mapped in previous studies. Prokaryotic transcription initiates in promoter DNA regions, defined by the presence of binding sites for a dissociable RNA polymerase subunit called sigma factor.<sup><a reveal-id="DST014C62" data-open="DST014C62" class="link link-ref link-reveal xref-bibr">62</a></sup> To date, seven <em>S. meliloti</em> sigma factors (among 15) are known to be active in at least one of the experimental conditions tested here: the vegetative sigma factor (RpoD, or sigma 70) and the alternative sigma factors RpoN, RpoH1, RpoH2, RpoE2, RpoE1 and RpoE4.<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST014C63" data-open="DST014C63" class="link link-ref link-reveal xref-bibr">63</a>–<a reveal-id="DST014C66" data-open="DST014C66" class="link link-ref link-reveal xref-bibr">66</a></sup> The TSSs of &gt;100 promoters known or supposed to be controlled by either one of these sigma factors were experimentally mapped in various studies (Table <a reveal-id="DST014TB4" data-open="DST014TB4" class="link link-reveal link-table xref-fig">4</a>). The TSSs annotated from the transcript 5′ ends mapped in the present study are in good agreement with these data, as 72% of the experimentally mapped TSSs match (±5 nt) our annotated TSSs (Table <a reveal-id="DST014TB4" data-open="DST014TB4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Several authors used the consensus promoter sequences deduced from these experimentally determined TSSs, combined or not with microarray or Affimetrix data, to predict &gt;200 additional putative targets of these sigma factors (Table <a reveal-id="DST014TB4" data-open="DST014TB4" class="link link-reveal link-table xref-fig">4</a>). The good congruence of these predictions with our data (74%) further strengthens our annotation (Table <a reveal-id="DST014TB4" data-open="DST014TB4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Note, however, that the number of correctly annotated TSSs was found to be positively correlated with the number of reads. Indeed, TSS annotations based on small numbers of reads appeared unreliable (24% of congruence), whereas TSSs covered by &gt;50 sequencing reads were found to match more frequently the experimentally determined or <em>in silico</em> predicted TSSs (82 and 77%, respectively). Caution should therefore be taken with weakly expressed genes. Among other mis-annotated TSSs are those corresponding to processed transcripts, such as tRNA and rRNA, for which only 4 of 14 annotated TSSs match the predicted or experimentally determined TSSs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Finally, to evaluate the proportion of annotated 5′ends corresponding to actual TSSs, we reasoned that most of the promoters not analysed above should be recognized by the vegetative sigma factor RpoD. An <em>in silico</em> search revealed that &gt;1/3 of them contain putative RpoD-binding sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>), as defined by MacLellan <em>et al</em>.<sup><a reveal-id="DST014C67" data-open="DST014C67" class="link link-ref link-reveal xref-bibr">67</a></sup> Altogether, these observations therefore suggest that a large number of the annotated 5′ends indeed correspond to actual TSSs. </p>                        <a id="5030406" scrollto-destination="5030406"></a>
<div content-id="DST014TB4" class="table-modal"><div class="table-wrap"><div id="DST014TB4" class="table-wrap-title" data-id="DST014TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Congruence between TSS annotation and the published literature</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Fraction of annotated TSS<sup>a</sup> matching:<hr /> </th></tr><tr><th>Experimentally 
mapped TSS<sup>b</sup> </th><th><em>In silico</em> predicted TSS<sup>b</sup> </th></tr></thead><tbody><tr><td>RpoD </td><td>22/27 </td><td>63/89 </td></tr><tr><td>RpoH1 and/or RpoH2 </td><td>45/67 </td><td>49/69 </td></tr><tr><td>RpoE1 and/or RpoE4 </td><td>3/4 </td><td>– </td></tr><tr><td>RpoE2 </td><td>1/1 </td><td>29/35 </td></tr><tr><td>RpoN </td><td>3/4 </td><td>5/6<sup>c</sup> </td></tr></tbody></table></div><div content-id="DST014TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Fraction of annotated TSS<sup>a</sup> matching:<hr /> </th></tr><tr><th>Experimentally 
mapped TSS<sup>b</sup> </th><th><em>In silico</em> predicted TSS<sup>b</sup> </th></tr></thead><tbody><tr><td>RpoD </td><td>22/27 </td><td>63/89 </td></tr><tr><td>RpoH1 and/or RpoH2 </td><td>45/67 </td><td>49/69 </td></tr><tr><td>RpoE1 and/or RpoE4 </td><td>3/4 </td><td>– </td></tr><tr><td>RpoE2 </td><td>1/1 </td><td>29/35 </td></tr><tr><td>RpoN </td><td>3/4 </td><td>5/6<sup>c</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All mapped or predicted promoter sequences are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genes for which no TSS was annotated in the current study were not retained for this table.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Data extracted from<sup><a reveal-id="DST014C67" data-open="DST014C67" class="link link-ref link-reveal xref-bibr">67</a>,<a reveal-id="DST014C85" data-open="DST014C85" class="link link-ref link-reveal xref-bibr">85</a>,<a reveal-id="DST014C86" data-open="DST014C86" class="link link-ref link-reveal xref-bibr">86</a></sup> (RpoD)<sup><a reveal-id="DST014C65" data-open="DST014C65" class="link link-ref link-reveal xref-bibr">65</a></sup> (RpoH1/H2),<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST014C40" data-open="DST014C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST014C70" data-open="DST014C70" class="link link-ref link-reveal xref-bibr">70</a></sup> (RpoE2),<sup><a reveal-id="DST014C66" data-open="DST014C66" class="link link-ref link-reveal xref-bibr">66</a></sup> (RpoE1/E4)<sup><a reveal-id="DST014C87  DST014C94" data-open="DST014C87  DST014C94" class="link link-ref link-reveal xref-bibr">87–94</a></sup> (RpoN).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>As the coordinates of RpoN TSS predicted by Dombrecht <em>et al.</em><sup><a reveal-id="DST014C94" data-open="DST014C94" class="link link-ref link-reveal xref-bibr">94</a></sup> were not described in their paper, we kept the promoters carrying the most obvious −24/−12 RpoN-binding sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030406" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST014TB4" class="table-wrap-title" data-id="DST014TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Congruence between TSS annotation and the published literature</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Fraction of annotated TSS<sup>a</sup> matching:<hr /> </th></tr><tr><th>Experimentally 
mapped TSS<sup>b</sup> </th><th><em>In silico</em> predicted TSS<sup>b</sup> </th></tr></thead><tbody><tr><td>RpoD </td><td>22/27 </td><td>63/89 </td></tr><tr><td>RpoH1 and/or RpoH2 </td><td>45/67 </td><td>49/69 </td></tr><tr><td>RpoE1 and/or RpoE4 </td><td>3/4 </td><td>– </td></tr><tr><td>RpoE2 </td><td>1/1 </td><td>29/35 </td></tr><tr><td>RpoN </td><td>3/4 </td><td>5/6<sup>c</sup> </td></tr></tbody></table></div><div content-id="DST014TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Fraction of annotated TSS<sup>a</sup> matching:<hr /> </th></tr><tr><th>Experimentally 
mapped TSS<sup>b</sup> </th><th><em>In silico</em> predicted TSS<sup>b</sup> </th></tr></thead><tbody><tr><td>RpoD </td><td>22/27 </td><td>63/89 </td></tr><tr><td>RpoH1 and/or RpoH2 </td><td>45/67 </td><td>49/69 </td></tr><tr><td>RpoE1 and/or RpoE4 </td><td>3/4 </td><td>– </td></tr><tr><td>RpoE2 </td><td>1/1 </td><td>29/35 </td></tr><tr><td>RpoN </td><td>3/4 </td><td>5/6<sup>c</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All mapped or predicted promoter sequences are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genes for which no TSS was annotated in the current study were not retained for this table.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Data extracted from<sup><a reveal-id="DST014C67" data-open="DST014C67" class="link link-ref link-reveal xref-bibr">67</a>,<a reveal-id="DST014C85" data-open="DST014C85" class="link link-ref link-reveal xref-bibr">85</a>,<a reveal-id="DST014C86" data-open="DST014C86" class="link link-ref link-reveal xref-bibr">86</a></sup> (RpoD)<sup><a reveal-id="DST014C65" data-open="DST014C65" class="link link-ref link-reveal xref-bibr">65</a></sup> (RpoH1/H2),<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST014C40" data-open="DST014C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST014C70" data-open="DST014C70" class="link link-ref link-reveal xref-bibr">70</a></sup> (RpoE2),<sup><a reveal-id="DST014C66" data-open="DST014C66" class="link link-ref link-reveal xref-bibr">66</a></sup> (RpoE1/E4)<sup><a reveal-id="DST014C87  DST014C94" data-open="DST014C87  DST014C94" class="link link-ref link-reveal xref-bibr">87–94</a></sup> (RpoN).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>As the coordinates of RpoN TSS predicted by Dombrecht <em>et al.</em><sup><a reveal-id="DST014C94" data-open="DST014C94" class="link link-ref link-reveal xref-bibr">94</a></sup> were not described in their paper, we kept the promoters carrying the most obvious −24/−12 RpoN-binding sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5030406" target="_blank">View Large</a></div></div><p>Interestingly, manual inspection of transcription data allowed the identification of 33 CDSs having different TSSs depending on experimental conditions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>).</p><p>The length of annotated 5′ UTRs ranges between 1 and 839 nt and displays a median size of 45 nt, which is similar to the median length of 5′ UTRs observed in <em>Escherichia coli</em> (37 nt),<sup><a reveal-id="DST014C68" data-open="DST014C68" class="link link-ref link-reveal xref-bibr">68</a></sup><em>Synechococcus elongatus</em> (33 nt)<sup><a reveal-id="DST014C2" data-open="DST014C2" class="link link-ref link-reveal xref-bibr">2</a></sup>, <em>Geobacter sulfurreducens</em> (37 nt)<sup><a reveal-id="DST014C1" data-open="DST014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> or <em>Agrobacterium tumefaciens</em> (61 nt).<sup><a reveal-id="DST014C69" data-open="DST014C69" class="link link-ref link-reveal xref-bibr">69</a></sup></p>                            <h3 scrollto-destination=5030409 id="5030409" class="section-title" data-legacy-id=s3f>3.6. Reappraisal of the <em>Sinorhizobium meliloti</em> RpoE2 regulon</h3>
<p>The genome-wide determination of TSSs should make it possible to extend our knowledge of regulons by looking for the conserved binding sites of regulators in promoter regions. We tested this idea on the RpoE2 regulon. RpoE2 is an extracytoplasmic function sigma factor involved in the general stress response of <em>S. meliloti</em> and is activated under various conditions, including heat shock, salt stress or entry into stationary phase following nitrogen or carbon starvation.<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> This sigma factor was found in previous studies to target &lt;40 <em>S. meliloti</em> promoters.<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST014C40" data-open="DST014C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST014C70" data-open="DST014C70" class="link link-ref link-reveal xref-bibr">70</a>,<a reveal-id="DST014C71" data-open="DST014C71" class="link link-ref link-reveal xref-bibr">71</a></sup> To re-evaluate the extent of the RpoE2 regulon, we screened all DNA regions located 5–11 nt upstream of 5′ transcript ends for the presence of the strictly conserved RpoE2-binding sequence (GGAAC N<sub>18–19</sub> TT).<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> We identified 108 transcription units that meet this criterion, including 26 putative ncRNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). That most of these sequences correspond to genuine RpoE2-controlled promoters was validated by the following observations : (i) 30 of them were previously reported as RpoE2 targets,<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST014C70" data-open="DST014C70" class="link link-ref link-reveal xref-bibr">70</a>,<a reveal-id="DST014C71" data-open="DST014C71" class="link link-ref link-reveal xref-bibr">71</a></sup> (ii) transcription from 86% of the newly identified promoters (67 of 78) was found in the current study as being up-regulated (&gt;2-fold, <em>P</em> &lt; 0.001) in stationary phase (a known RpoE2-activating condition; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>) and finally (iii) using qRT-PCR, we confirmed that transcription from 6 of 6 randomly chosen promoters (four mRNAs and two ncRNAs) is up-regulated, either following a heat shock or entry in stationary phase (two RpoE2-activating conditions), in the wild type but not in a <em>rpoE2</em> mutant strain (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Altogether, these observations further validate TSS annotations predicted by EuGene-P and give a demonstration of its power to extend the knowledge of a given regulon.</p>                            <h2 scrollto-destination=5030421 id="5030421" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Through RNA sequencing, NGS technologies give access to prokaryotic transcriptomes with an unprecedented resolution and provide a massive amount of novel information on genome organization. In this work, we took advantage of data produced from the legume bacterial symbiont <em>S. meliloti</em> to develop a new bioinformatic tool that exploits transcription data for exhaustive annotation of prokaryotic genomes. The oriented RNA-Seq data that were produced from Sm2011 in stationary and exponential phases as well as in symbiotic condition have excellent reproducibility, with highly consistent triplicates and a good congruence when compared with previously published data. The analysis of both short and long fractions of RNAs enabled the identification of transcribed biological objects of small length, like ncRNAs and short CDSs, which could have been lost with usual RNA preparation protocols. The Sm2011 oriented RNA sequencing also showed a complex landscape of expression on both strands. Such complexity would have been completely hidden by non-oriented sequencing, possibly leading to biased expression level measurements as well as a poorer genome annotation.</p><p>Oriented RNA-Seq data give an opportunity to define a new generation of integrative prokaryotic genome annotation tools. In the area of prokaryotic genome annotation, existing NGS-related studies<sup><a reveal-id="DST014C11" data-open="DST014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> have focussed on the possibly increased level of sequencing errors associated with such technologies. Here, we showed that the quality of the Sm2011 genomic sequence obtained by NGS is comparable with, if not better than, the Sm1021 genomic sequence previously generated by Sanger sequencing.<sup><a reveal-id="DST014C31" data-open="DST014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Using oriented RNA-Seq data, the EuGene-P proved to be able to automatically produce a complex annotation with novel coding and nc genes, including many antisense genes, untranslated 5′ and 3′ regions and precise mapping of 5′ TSSs. To the best of our knowledge, EuGene-P is the first prokaryotic gene finder that is able to predict a comprehensive genome annotation. The ability to predict highly overlapping functional regions is directly inherited from the strand-specific prediction process, which is itself consistent with oriented RNA-Seq data. Predicting genes on each strand independently has historically been considered as a bad idea given that the gene contents of the two DNA strands are highly correlated. However, ncRNA genes and specifically antisense genes blur this idea, which is already shaken by overlapping CDSs and transcripts. Strand-specific prediction and oriented RNA-Seq allow dealing with this complex situation directly.</p><p>The quality of the Sm2011 automatic annotation was validated by in-depth manual curation. A relatively limited number of manual modifications were made using Apollo for the simultaneous visualization of per-triplicated bank expression levels and annotation on both strands. The distinction between 3′ and 5′ UTRs and nearby ncRNA genes remains difficult and is still questionable even in the expert annotation. Beyond this, the resulting final annotation led to the definition of accurate gene structures, which is very useful for biologists to better understand the organization of genes and to characterize their function and regulation.</p><p>The number of predicted ncRNA genes is particularly high in <em>S. meliloti</em>, even though we cannot rule out that some of them encode peptides or small proteins. Most predicted ncRNA regions are consistently supported by RNA-Seq data. The fact that a large proportion of predicted ncRNAs are differentially expressed between the three physiological conditions analysed suggests that they are probably not artefacts introduced either by cDNA library preparation or by sequencing protocols. Moreover, the list of ncRNA genes predicted in a previous RNA-Seq study<sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> is also largely covered by our predicted nc transcripts, despite the fact that it represents only a small fraction of the genome. Among the 1876 predicted ncRNAs, 29 have been experimentally validated by northern blot or 5′-RACE analyses in previous studies.<sup><a reveal-id="DST014C56" data-open="DST014C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DST014C72  DST014C74" data-open="DST014C72  DST014C74" class="link link-ref link-reveal xref-bibr">72–74</a></sup> Beside tRNAs, rRNAs and the five well-conserved and well-characterized ncRNAs, 4.5S RNA (SRP, <em>ffs</em>), 6S RNA (<em>ssrS</em>), tmRNA (<em>ssrA</em>), the ribozyme RNase P (<em>rnpB</em>) and <em>incA</em> that mediates plasmid incompatibility phenotypes,<sup><a reveal-id="DST014C75" data-open="DST014C75" class="link link-ref link-reveal xref-bibr">75</a></sup> 36 ncRNAs belong to known families described in the Rfam database, whereas the remaining predicted ncRNAs could not be assigned to a given class. A lot of work thus remains to be done to validate the existence of predicted ncRNAs and to elucidate their function in <em>S. meliloti</em>. Interestingly, 454-sequencing of small ncRNAs of <em>A. tumefaciens</em>, a bacterium phylogenetically close to <em>S. meliloti</em>, recently revealed the presence of numerous small RNAs on all four replicons.<sup><a reveal-id="DST014C69" data-open="DST014C69" class="link link-ref link-reveal xref-bibr">69</a></sup> The number of ncRNAs in <em>S. meliloti</em> would be even higher if widespread repeated elements like the RIME, MOTIF and Sm-1 to Sm-5 repeats<sup><a reveal-id="DST014C59  DST014C61" data-open="DST014C59  DST014C61" class="link link-ref link-reveal xref-bibr">59–61</a></sup>, that appeared to be highly transcribed elements, were taken into account. Similar repeated regions, like bacterial interspersed mosaic element and <em>boxC</em> DNA repeat elements, have also been shown to be transcribed in <em>E. coli</em> and to play key roles in transcription attenuation<sup><a reveal-id="DST014C76" data-open="DST014C76" class="link link-ref link-reveal xref-bibr">76</a></sup> or mRNA stabilization.<sup><a reveal-id="DST014C77" data-open="DST014C77" class="link link-ref link-reveal xref-bibr">77</a>,<a reveal-id="DST014C78" data-open="DST014C78" class="link link-ref link-reveal xref-bibr">78</a></sup> More recently, they have also been demonstrated to be involved in nucleoid morphology and chromosome formation and maintenance.<sup><a reveal-id="DST014C79" data-open="DST014C79" class="link link-ref link-reveal xref-bibr">79</a></sup></p><p>A large proportion of <em>S. meliloti</em> ncRNAs were found to map antisense to annotated protein-coding genes. With oriented RNA-Seq data, antisense transcription now appears to be a common and widespread phenomenon in bacteria as recently reported for <em>E. coli</em>, in which 1005 antisense RNAs were identified,<sup><a reveal-id="DST014C80" data-open="DST014C80" class="link link-ref link-reveal xref-bibr">80</a>,<a reveal-id="DST014C81" data-open="DST014C81" class="link link-ref link-reveal xref-bibr">81</a></sup> and <em>Helicobacter pylori</em>, in which 46% of CDSs are overlapping with at least one antisense RNA.<sup><a reveal-id="DST014C82" data-open="DST014C82" class="link link-ref link-reveal xref-bibr">82</a>,<a reveal-id="DST014C83" data-open="DST014C83" class="link link-ref link-reveal xref-bibr">83</a></sup> Several mechanisms of the action of antisense RNAs in bacteria have been recently reviewed.<sup><a reveal-id="DST014C84" data-open="DST014C84" class="link link-ref link-reveal xref-bibr">84</a></sup> They include the alteration of target RNA stability, the modulation (inhibition or activation) of translation, transcriptional interference and attenuation. Antisense RNA-mediated regulation thus likely appears as an important component of complex regulatory pathways controlling gene expression in bacteria. However, it was recently suggested by Nicolas <em>et al.</em><sup><a reveal-id="DST014C83" data-open="DST014C83" class="link link-ref link-reveal xref-bibr">83</a></sup> that some antisense RNAs can potentially arise from spurious transcription initiation or from imperfect control of transcription termination.</p><p>In this study, we also provided a detailed map of <em>S. meliloti</em> TSSs. This high-resolution TSS map is in agreement with previous <em>in silico</em> predicted or experimentally determined TSSs, in which 72% of validated TSSs matched our annotated TSSs by ±5 nt. These data will greatly facilitate the study of promoter regions, the identification of protein-binding motifs and the determination of regulons in <em>S. meliloti</em>. This was done here for the RpoE2 regulon that appears to be almost three times larger than previously determined using classical approaches.<sup><a reveal-id="DST014C39" data-open="DST014C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>Oriented bacterial RNA-Seq data also unveil more complex mechanisms, such as alternative transcription starts, depending on the experimental condition (exponential or stationary phase). In our expression data, we identified 33 genes that displayed multiple TSSs. The frequency of multiple TSSs would have probably been higher if more physiological conditions had been analysed. Indeed, it was shown in <em>E. coli</em> and <em>Bacillus subtilis</em> that 35 and 46% of genes, respectively, have multiple TSSs.<sup><a reveal-id="DST014C68" data-open="DST014C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DST014C83" data-open="DST014C83" class="link link-ref link-reveal xref-bibr">83</a></sup> This type of adaptive behaviour is currently difficult to represent and raises new problems for automatic genome annotation and visualization.</p><p>In conclusion, we developed a new generic tool, EuGene-P, to automatically and accurately annotate prokaryotic genomes by integrating genome-wide experimental data, such as RNA-Seq data. This tool was used to re-visit the structural annotation of <em>S. meliloti</em>, providing a much more complete and comprehensive view of its genome architecture. The ability of EuGene-P to identify nc transcribed elements as well as to precisely map TSSs offers a new view of prokaryotic genomes and should greatly contribute to our understanding of gene regulation and function in bacteria.</p>                            <h2 scrollto-destination=5030448 id="5030448" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5030453 id="5030453" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Agence Nationale de la Recherche under grant ANR-08-GENO-106 "SYMbiMICS". This research was done in the Laboratoire des Interactions Plantes-Microorganismes, part of the Laboratoire d'Excellence (LABEX) entitled TULIP (ANR-10-LABX-41). F. Jardinaud was supported by the Institut National Polytechnique de Toulouse.</p>                            <h2 scrollto-destination=5030457 id="5030457" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Bénédicte Bastiat for constructing the Sm2011 <em>rpoE2</em> mutant, and Pierre Dupuy for help with qRT-PCR experiments. We thank Svetlana Yurgel and Michael Kahn (University of Washington) for exchanging the sequence data on putative frameshifts between the two <em>S. meliloti</em> strains, Sm2011 and Sm1021.</p>                            <h2 scrollto-destination=5030464 id="5030464" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Lovley</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Palsson</div> <div class="given-names">B.O.,</div></div>,  <div class="name"><div class="surname">Zengler</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Structural and operational complexity of the <em>Geobacter sulfurreducens</em> genome</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1304</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20operational%20complexity%20of%20the%20Geobacter%20sulfurreducens%20genome&amp;author=Y.%20Qiu&amp;author=B.K.%20Cho&amp;author=Y.S.%20Park&amp;author=D.%20Lovley&amp;author=B.O.%2C%20Palsson&amp;author=K.%20Zengler&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1304-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107540.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107540.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107540.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20592237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vijayan</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">I.H.</div></div>,  <div class="name"><div class="surname">O'Shea</div> <div class="given-names">E.K.</div></div>. </div><div class="article-title">A high resolution map of a cyanobacterial transcriptome</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R47</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20resolution%20map%20of%20a%20cyanobacterial%20transcriptome&amp;author=V.%20Vijayan&amp;author=I.H.%20Jain&amp;author=E.K.%20O'Shea&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-5-r47" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-5-r47" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-5-r47"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21612627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Klockgether</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hagendorf</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title"><em>Pseudomonas putida</em> KT2440 genome update by cDNA sequencing and microarray transcriptomics</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1309</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudomonas%20putida%20KT2440%20genome%20update%20by%20cDNA%20sequencing%20and%20microarray%20transcriptomics&amp;author=S.%20Frank&amp;author=J.%20Klockgether&amp;author=P.%20Hagendorf&amp;publication_year=2011&amp;journal=Environ.%20Microbiol.&amp;volume=13&amp;pages=1309-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2011.13.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2011.13.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2011.13.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21355971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weissenmayer</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Prendergast</div> <div class="given-names">J.G.D.</div></div>,  <div class="name"><div class="surname">Lohan</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Loftus</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Sequencing illustrates the transcriptional response of <em>Legionella pneumophila</em> during infection and identifies seventy novel small non-coding RNAs</div>, <div class="source">PloS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e17570</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20illustrates%20the%20transcriptional%20response%20of%20Legionella%20pneumophila%20during%20infection%20and%20identifies%20seventy%20novel%20small%20non-coding%20RNAs&amp;author=B.A.%20Weissenmayer&amp;author=J.G.D.%20Prendergast&amp;author=A.J.%20Lohan&amp;author=B.J.%20Loftus&amp;publication_year=2011&amp;journal=PloS%20ONE&amp;volume=6&amp;pages=e17570" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0017570" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0017570" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0017570"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21408607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richardson</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Watson</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The automatic annotation of bacterial genomes</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20automatic%20annotation%20of%20bacterial%20genomes&amp;author=E.J.%20Richardson&amp;author=M.%20Watson&amp;publication_year=2013&amp;journal=Brief%20Bioinform.&amp;volume=14&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22408191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Bratke</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Powers</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">673</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20bacterial%20genes%20and%20endosymbiont%20DNA%20with%20Glimmer&amp;author=A.L.%20Delcher&amp;author=K.A.%20Bratke&amp;author=E.C.%20Powers&amp;author=S.L.%20Salzberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=673-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Do</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">D.K.</div></div>. </div><div class="article-title">Computational approaches to gene prediction</div>, <div class="source">J. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">137</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20approaches%20to%20gene%20prediction&amp;author=J.H.%20Do&amp;author=D.K.%20Choi&amp;publication_year=2006&amp;journal=J.%20Microbiol.&amp;volume=44&amp;pages=137-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16728949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyatt</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">G.L.</div></div>,  <div class="name"><div class="surname">Locascio</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Land</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Larimer</div> <div class="given-names">F.W.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Prodigal: prokaryotic gene recognition and translation initiation site identification</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">119</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prodigal%3A%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;author=D.%20Hyatt&amp;author=G.L.%20Chen&amp;author=P.F.%20Locascio&amp;author=M.L.%20Land&amp;author=F.W.%20Larimer&amp;author=J.%20Hauser&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20211023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aziz</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Bartels</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Best</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">The RAST Server: rapid annotations using subsystems technology</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">75</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RAST%20Server%3A%20rapid%20annotations%20using%20subsystems%20technology&amp;author=R.K.%20Aziz&amp;author=D.%20Bartels&amp;author=A.A.%20Best&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18261238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pareja-Tobes</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Manrique</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pareja-Tobes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pareja</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tobes</div> <div class="given-names">R.</div></div>. </div><div class="article-title">BG7: a new approach for bacterial genome annotation designed for next generation sequencing data</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e49239</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BG7%3A%20a%20new%20approach%20for%20bacterial%20genome%20annotation%20designed%20for%20next%20generation%20sequencing%20data&amp;author=P.%20Pareja-Tobes&amp;author=M.%20Manrique&amp;author=E.%20Pareja-Tobes&amp;author=E.%20Pareja&amp;author=R.%20Tobes&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e49239" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0049239" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0049239" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0049239"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23185310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peterson</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">McCue</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Schrimpe-Rutledge</div> <div class="given-names">A.C.</div></div>, et al. </div><div class="article-title">VESPA: software to facilitate genomic annotation of prokaryotic organisms through integration of proteomic and transcriptomic data</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">131</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VESPA%3A%20software%20to%20facilitate%20genomic%20annotation%20of%20prokaryotic%20organisms%20through%20integration%20of%20proteomic%20and%20transcriptomic%20data&amp;author=E.S.%20Peterson&amp;author=L.A.%20McCue&amp;author=A.C.%20Schrimpe-Rutledge&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22480257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vallenet</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Belda</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Calteau</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">MicroScope—an integrated microbial resource for the curation and comparative analysis of genomic and metabolic data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D636</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroScope%E2%80%94an%20integrated%20microbial%20resource%20for%20the%20curation%20and%20comparative%20analysis%20of%20genomic%20and%20metabolic%20data&amp;author=D.%20Vallenet&amp;author=E.%20Belda&amp;author=A.%20Calteau&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D636-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schiex</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Moisan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rouzé</div> <div class="given-names">P.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Sagot</div> <div class="given-names">M.</div></div>. </div><div class="article-title">EuGene: an eukaryotic gene finder that combines several sources of evidence</div>, <div class="source">Computational biology</div> , <div class="year">2001</div><div class="publisher-loc">Berlin, Germany</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">111</div>-<div class="lpage">25</div>)</div></div></div></div><div content-id="DST014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Steinkamp</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>. </div><div class="article-title">AUGUSTUS: a web server for gene finding in eukaryotes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">W309</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20a%20web%20server%20for%20gene%20finding%20in%20eukaryotes&amp;author=M.%20Stanke&amp;author=R.%20Steinkamp&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=W309-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yeh</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">L.P.</div></div>,  <div class="name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div>. </div><div class="article-title">Computational inference of homologous gene structures in the human genome,</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">803</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20inference%20of%20homologous%20gene%20structures%20in%20the%20human%20genome%2C&amp;author=R.F.%20Yeh&amp;author=L.P.%20Lim&amp;author=C.B.%20Burge&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=803-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.175701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.175701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.175701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wiehe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gebauer-Jung</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Guigó</div> <div class="given-names">R.</div></div>. </div><div class="article-title">SGP-1: prediction and validation of homologous genes based on sequence alignments</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1574</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SGP-1%3A%20prediction%20and%20validation%20of%20homologous%20genes%20based%20on%20sequence%20alignments&amp;author=T.%20Wiehe&amp;author=S.%20Gebauer-Jung&amp;author=T.%20Mitchell-Olds&amp;author=R.%20Guig%C3%B3&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1574-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.177401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.177401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.177401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11544202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Agarwal</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Abril</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Wiehe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fickett</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Guigó</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Comparative gene prediction in human and mouse</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">108</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20gene%20prediction%20in%20human%20and%20mouse&amp;author=G.%20Parra&amp;author=P.%20Agarwal&amp;author=J.F.%20Abril&amp;author=T.%20Wiehe&amp;author=J.W.%20Fickett&amp;author=R.%20Guig%C3%B3&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=108-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.871403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.871403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.871403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12529313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foissac</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bardou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Moisan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cros</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Schiex</div> <div class="given-names">T.</div></div>. </div><div class="article-title">EuGene'Hom: a generic similarity-based gene finder using multiple homologous sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3742</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EuGene'Hom%3A%20a%20generic%20similarity-based%20gene%20finder%20using%20multiple%20homologous%20sequences&amp;author=S.%20Foissac&amp;author=P.%20Bardou&amp;author=A.%20Moisan&amp;author=M.J.%20Cros&amp;author=T.%20Schiex&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3742-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foissac</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome annotation in plants and fungi: EuGene as a model platform</div>, <div class="source">Curr. Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">87</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20annotation%20in%20plants%20and%20fungi%3A%20EuGene%20as%20a%20model%20platform&amp;author=S.%20Foissac&amp;author=J.%20Gouzy&amp;author=S.%20Rombauts&amp;publication_year=2008&amp;journal=Curr.%20Bioinformatics&amp;volume=3&amp;pages=87-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2174/157489308784340702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2174%2F157489308784340702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2174%2F157489308784340702"> </span></div> </div></div></div></div></div><div content-id="DST014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abad</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Genome sequence of the metazoan plant–parasitic nematode <em>Meloidogyne incognita</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">909</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20metazoan%20plant%E2%80%93parasitic%20nematode%20Meloidogyne%20incognita&amp;author=P.%20Abad&amp;author=J.%20Gouzy&amp;author=J.M.%20Aury&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=909-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18660804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Aerts</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ahrén</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">The genome of <em>Laccaria bicolor</em> provides insights into mycorrhizal symbiosis</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">88</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Laccaria%20bicolor%20provides%20insights%20into%20mycorrhizal%20symbiosis&amp;author=F.%20Martin&amp;author=A.%20Aerts&amp;author=D.%20Ahr%C3%A9n&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=88-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18322534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cock</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rouzé</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The Ectocarpus genome and the independent evolution of multicellularity in brown algae</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">465</div> (pg. <div class="fpage">617</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ectocarpus%20genome%20and%20the%20independent%20evolution%20of%20multicellularity%20in%20brown%20algae&amp;author=J.M.%20Cock&amp;author=L.%20Sterck&amp;author=P.%20Rouz%C3%A9&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=617-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20520714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grbić</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Van Leeuwen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">R.M.</div></div>, et al. </div><div class="article-title">The genome of <em>Tetranychus urticae</em> reveals herbivorous pest adaptations</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">479</div> (pg. <div class="fpage">487</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Tetranychus%20urticae%20reveals%20herbivorous%20pest%20adaptations&amp;author=M.%20Grbi%C4%87&amp;author=T.%20Van%20Leeuwen&amp;author=R.M.%20Clark&amp;publication_year=2011&amp;journal=Nature&amp;volume=479&amp;pages=487-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22113690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">The genome of <em>Theobroma cacao</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">101</div>-<div class="lpage">08</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.M.%20Aury&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=101-08" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.736" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Debellé</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Oldroyd</div> <div class="given-names">G.E.</div></div>, et al. </div><div class="article-title">The <em>Medicago</em> genome provides insight into the evolution of rhizobial symbioses</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">480</div> (pg. <div class="fpage">520</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses&amp;author=N.D.%20Young&amp;author=F.%20Debell%C3%A9&amp;author=G.E.%20Oldroyd&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=520-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/480162a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F480162a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F480162a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">485</div> (pg. <div class="fpage">635</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tomato%20genome%20sequence%20provides%20insights%20into%20fleshy%20fruit%20evolution&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;author=H.%20Hirakawa&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=635-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moreau</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Verdenaud</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ott</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Transcription reprogramming during root nodule development in <em>Medicago truncatula</em></div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e16463</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20reprogramming%20during%20root%20nodule%20development%20in%20Medicago%20truncatula&amp;author=S.%20Moreau&amp;author=M.%20Verdenaud&amp;author=T.%20Ott&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e16463" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maunoury</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Redondo-Nieto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bourcy</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Differentiation of symbiotic cells and endosymbionts in <em>Medicago truncatula</em> nodulation are coupled to two transcriptome-switches</div>, <div class="source">PloS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e9519</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differentiation%20of%20symbiotic%20cells%20and%20endosymbionts%20in%20Medicago%20truncatula%20nodulation%20are%20coupled%20to%20two%20transcriptome-switches&amp;author=N.%20Maunoury&amp;author=M.%20Redondo-Nieto&amp;author=M.%20Bourcy&amp;publication_year=2010&amp;journal=PloS%20ONE&amp;volume=5&amp;pages=e9519" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009519" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009519" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009519"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20209049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Capela</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Filipe</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bobik</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Batut</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bruand</div> <div class="given-names">C.</div></div>. </div><div class="article-title"><em>Sinorhizobium meliloti</em> differentiation during symbiosis with alfalfa: a transcriptomic dissection</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2006</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">363</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sinorhizobium%20meliloti%20differentiation%20during%20symbiosis%20with%20alfalfa%3A%20a%20transcriptomic%20dissection&amp;author=D.%20Capela&amp;author=C.%20Filipe&amp;author=C.%20Bobik&amp;author=J.%20Batut&amp;author=C.%20Bruand&amp;publication_year=2006&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=19&amp;pages=363-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16610739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galibert</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S.R.</div></div>, et al. </div><div class="article-title">The composite genome of the legume symbiont <em>Sinorhizobium meliloti</em></div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">668</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20composite%20genome%20of%20the%20legume%20symbiont%20Sinorhizobium%20meliloti&amp;author=F.%20Galibert&amp;author=T.M.%20Finan&amp;author=S.R.%20Long&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=668-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1060966" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1060966" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1060966"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11474104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meade</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Ruvkun</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Ausubel</div> <div class="given-names">F.M.</div></div>. </div><div class="article-title">Physical and genetic characterization of symbiotic and auxotrophic mutants of <em>Rhizobium meliloti</em> induced by transposon Tn5 mutagenesis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1982</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">114</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20and%20genetic%20characterization%20of%20symbiotic%20and%20auxotrophic%20mutants%20of%20Rhizobium%20meliloti%20induced%20by%20transposon%20Tn5%20mutagenesis&amp;author=H.M.%20Meade&amp;author=S.R.%20Long&amp;author=G.B.%20Ruvkun&amp;author=S.E.%20Brown&amp;author=F.M.%20Ausubel&amp;publication_year=1982&amp;journal=J.%20Bacteriol.&amp;volume=149&amp;pages=114-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6274841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wais</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Wells</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Analysis of differences between <em>Sinorhizobium meliloti</em> 1021 and 2011 strains using the host calcium spiking response</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2002</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1245</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20differences%20between%20Sinorhizobium%20meliloti%201021%20and%202011%20strains%20using%20the%20host%20calcium%20spiking%20response&amp;author=R.J.%20Wais&amp;author=D.H.%20Wells&amp;author=S.R.%20Long&amp;publication_year=2002&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=15&amp;pages=1245-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2002.15.12.1245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2002.15.12.1245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2002.15.12.1245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12481997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krol</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Global transcriptional analysis of the phosphate starvation response in <em>Sinorhizobium meliloti</em> strains 1021 and 2011</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">1</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptional%20analysis%20of%20the%20phosphate%20starvation%20response%20in%20Sinorhizobium%20meliloti%20strains%201021%20and%202011&amp;author=E.%20Krol&amp;author=A.%20Becker&amp;publication_year=2004&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=272&amp;pages=1-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-004-1030-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-004-1030-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-004-1030-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15221452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yuan</div> <div class="given-names">Z.C.</div></div>,  <div class="name"><div class="surname">Zaheer</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Regulation and properties of PstSCAB, a high-affinity, high-velocity phosphate transport system of <em>Sinorhizobium meliloti</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">1089</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20and%20properties%20of%20PstSCAB%2C%20a%20high-affinity%2C%20high-velocity%20phosphate%20transport%20system%20of%20Sinorhizobium%20meliloti&amp;author=Z.C.%20Yuan&amp;author=R.%20Zaheer&amp;author=T.M.%20Finan&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=1089-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.3.1089-1102.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.3.1089-1102.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.3.1089-1102.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16428413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Terpolilli</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">O'Hara</div> <div class="given-names">G.W.</div></div>,  <div class="name"><div class="surname">Tiwari</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Dilworth</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Howieson</div> <div class="given-names">J.G.</div></div>. </div><div class="article-title">The model legume <em>Medicago truncatula</em> A17 is poorly matched for N2 fixation with the sequenced microsymbiont <em>Sinorhizobium meliloti</em> 1021</div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">62</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20model%20legume%20Medicago%20truncatula%20A17%20is%20poorly%20matched%20for%20N2%20fixation%20with%20the%20sequenced%20microsymbiont%20Sinorhizobium%20meliloti%201021&amp;author=J.J.%20Terpolilli&amp;author=G.W.%20O'Hara&amp;author=R.P.%20Tiwari&amp;author=M.J.%20Dilworth&amp;author=J.G.%20Howieson&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=62-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18422896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujishige</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kapadia</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">De Hoff</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Hirsch</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Investigations of <em>Rhizobium</em> biofilm formation</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2006</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">195</div>-<div class="lpage">206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigations%20of%20Rhizobium%20biofilm%20formation&amp;author=N.A.%20Fujishige&amp;author=N.N.%20Kapadia&amp;author=P.L.%20De%20Hoff&amp;author=A.M.%20Hirsch&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=56&amp;pages=195-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2006.56.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2006.56.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffem.2006.56.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16629750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peltzer</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Roques</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Poinsot</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Auxotrophy accounts for nodulation defect of most <em>Sinorhizobium meliloti</em> mutants in the branched-chain amino acid biosynthesis pathway</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2008</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1232</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Auxotrophy%20accounts%20for%20nodulation%20defect%20of%20most%20Sinorhizobium%20meliloti%20mutants%20in%20the%20branched-chain%20amino%20acid%20biosynthesis%20pathway&amp;author=M.D.%20Peltzer&amp;author=N.%20Roques&amp;author=V.%20Poinsot&amp;publication_year=2008&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=21&amp;pages=1232-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-21-9-1232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-21-9-1232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-21-9-1232"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18700827" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sauviac</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Philippe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Phok</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bruand</div> <div class="given-names">C.</div></div>. </div><div class="article-title">An extracytoplasmic function sigma factor acts as a general stress response regulator in <em>Sinorhizobium meliloti</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2007</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">4204</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20extracytoplasmic%20function%20sigma%20factor%20acts%20as%20a%20general%20stress%20response%20regulator%20in%20Sinorhizobium%20meliloti&amp;author=L.%20Sauviac&amp;author=H.%20Philippe&amp;author=K.%20Phok&amp;author=C.%20Bruand&amp;publication_year=2007&amp;journal=J.%20Bacteriol.&amp;volume=189&amp;pages=4204-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00175-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00175-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00175-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17400745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bastiat</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sauviac</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bruand</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Dual control of <em>Sinorhizobium meliloti</em> RpoE2 sigma factor activity by two PhyR-type two-component response regulators</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2010</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">2255</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20control%20of%20Sinorhizobium%20meliloti%20RpoE2%20sigma%20factor%20activity%20by%20two%20PhyR-type%20two-component%20response%20regulators&amp;author=B.%20Bastiat&amp;author=L.%20Sauviac&amp;author=C.%20Bruand&amp;publication_year=2010&amp;journal=J.%20Bacteriol.&amp;volume=192&amp;pages=2255-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01666-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01666-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01666-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20154128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barker</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pfaff</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Moreau</div> <div class="given-names">D.</div></div>, et al. </div><div class="person-group"><div class="name"><div class="surname">Mathesius</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Journet</div> <div class="given-names">E.P.</div></div>,  <div class="name"><div class="surname">Sumner</div> <div class="given-names">L.W.</div></div>. </div><div class="article-title">Growing Medicago truncatula: choice of substrates and growth conditions</div>, <div class="source">The Medicago truncatula handbook</div> , <div class="year">2006</div> <div class="comment">Noble foundation. ISBN 0-9754303-1-9. http://www.noble.org/MedicagoHandbook.</div></div></div></div></div><div content-id="DST014C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Journet</div> <div class="given-names">E.P.</div></div>,  <div class="name"><div class="surname">El-Gachtouli</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Vernoud</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title"><em>Medicago truncatula</em> ENOD11: a novel RPRP-encoding early nodulin gene expressed during mycorrhization in arbuscule-containing cells</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2001</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">737</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20ENOD11%3A%20a%20novel%20RPRP-encoding%20early%20nodulin%20gene%20expressed%20during%20mycorrhization%20in%20arbuscule-containing%20cells&amp;author=E.P.%20Journet&amp;author=N.%20El-Gachtouli&amp;author=V.%20Vernoud&amp;publication_year=2001&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=14&amp;pages=737-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2001.14.6.737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2001.14.6.737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2001.14.6.737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11386369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafferty</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">McCallum</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pereira</div> <div class="given-names">F.C.N.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Brodley</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Danylu</div> <div class="given-names">A.P.</div></div>. </div>, <div class="year">2001</div><div class="conf-name">Proceedings of the Eighteenth International Conference on Machine Learning, June 28–July 1, 2001</div><div class="conf-loc">Williamstown, MA, USA</div><div class="publisher-name">Morgan Kaufmann Publishers Inc., Williams College</div>(pg. <div class="fpage">282</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="DST014C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernal</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Crammer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hatzigeorgiou</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pereira</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Global discriminative learning for higher-accuracy computational gene prediction</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e54</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20discriminative%20learning%20for%20higher-accuracy%20computational%20gene%20prediction&amp;author=A.%20Bernal&amp;author=K.%20Crammer&amp;author=A.%20Hatzigeorgiou&amp;author=F.%20Pereira&amp;publication_year=2007&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=3&amp;pages=e54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0030054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0030054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0030054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17367206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeCaprio</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Vinson</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Pearson</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Montgomery</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Doherty</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Galagan</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Conrad: gene prediction using conditional random fields</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1389</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conrad%3A%20gene%20prediction%20using%20conditional%20random%20fields&amp;author=D.%20DeCaprio&amp;author=J.P.%20Vinson&amp;author=M.D.%20Pearson&amp;author=P.%20Montgomery&amp;author=M.%20Doherty&amp;author=J.E.%20Galagan&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1389-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6558107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6558107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6558107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17690204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Differential expression analysis for sequence count data</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R106</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-10-r106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-10-r106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate—a practical and powerful approach to multiple testing</div>, <div class="source">J. R. Stat. Soc. B</div> , <div class="year">1995</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DST014C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersen</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Orntoft</div> <div class="given-names">T.F.</div></div>. </div><div class="article-title">Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets</div>, <div class="source">Cancer Res.</div> , <div class="year">2004</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">5245</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Normalization%20of%20real-time%20quantitative%20reverse%20transcription-PCR%20data%3A%20a%20model-based%20variance%20estimation%20approach%20to%20identify%20genes%20suited%20for%20normalization%2C%20applied%20to%20bladder%20and%20colon%20cancer%20data%20sets&amp;author=C.L.%20Andersen&amp;author=J.L.%20Jensen&amp;author=T.F.%20Orntoft&amp;publication_year=2004&amp;journal=Cancer%20Res.&amp;volume=64&amp;pages=5245-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-04-0496" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-04-0496" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-04-0496"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15289330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mørk</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Evaluating bacterial gene-finding HMM structures as probabilistic logic programs</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">636</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluating%20bacterial%20gene-finding%20HMM%20structures%20as%20probabilistic%20logic%20programs&amp;author=S.%20M%C3%B8rk&amp;author=I.%20Holmes&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=636-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22215819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thoden</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">X.Y.</div></div>,  <div class="name"><div class="surname">Raushel</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Holden</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">Carbamoyl-phosphate synthetase—creation of an escape route for ammonia</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">39722</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbamoyl-phosphate%20synthetase%E2%80%94creation%20of%20an%20escape%20route%20for%20ammonia&amp;author=J.B.%20Thoden&amp;author=X.Y.%20Huang&amp;author=F.M.%20Raushel&amp;author=H.M.%20Holden&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=39722-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M206915200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M206915200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M206915200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12130656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferrières</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Francez-Charlot</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rouillé</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kahn</div> <div class="given-names">D.</div></div>. </div><div class="article-title">FixJ-regulated genes evolved through promoter duplication in <em>Sinorhizobium meliloti</em></div>, <div class="source">Microbiology</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">2335</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FixJ-regulated%20genes%20evolved%20through%20promoter%20duplication%20in%20Sinorhizobium%20meliloti&amp;author=L.%20Ferri%C3%A8res&amp;author=A.%20Francez-Charlot&amp;author=J.%20Gouzy&amp;author=S.%20Rouill%C3%A9&amp;author=D.%20Kahn&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=2335-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.27081-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.27081-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.27081-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15256575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vasse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Debilly</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Camut</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Truchet</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Correlation between ultrastructural differentiation of bacteroids and nitrogen fixation in alfalfa nodules</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1990</div>, vol. <div class="volume">172</div> (pg. <div class="fpage">4295</div>-<div class="lpage">306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20ultrastructural%20differentiation%20of%20bacteroids%20and%20nitrogen%20fixation%20in%20alfalfa%20nodules&amp;author=J.%20Vasse&amp;author=F.%20Debilly&amp;author=S.%20Camut&amp;author=G.%20Truchet&amp;publication_year=1990&amp;journal=J.%20Bacteriol.&amp;volume=172&amp;pages=4295-306" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2376562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vivancos</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Guell</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dohm</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Serrano</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Himmelbauer</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Strand-specific deep sequencing of the transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">989</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand-specific%20deep%20sequencing%20of%20the%20transcriptome&amp;author=A.P.%20Vivancos&amp;author=M.%20Guell&amp;author=J.C.%20Dohm&amp;author=L.%20Serrano&amp;author=H.%20Himmelbauer&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=989-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.094318.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.094318.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.094318.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20519413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levin</div> <div class="given-names">J.Z.</div></div>,  <div class="name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Adiconis</div> <div class="given-names">X.A.</div></div>, et al. </div><div class="article-title">Comprehensive comparative analysis of strand-specific RNA sequencing methods</div>, <div class="source">Nat. Methods</div> , <div class="year">2010</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">709</div>-<div class="lpage">U767</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20comparative%20analysis%20of%20strand-specific%20RNA%20sequencing%20methods&amp;author=J.Z.%20Levin&amp;author=M.%20Yassour&amp;author=X.A.%20Adiconis&amp;publication_year=2010&amp;journal=Nat.%20Methods&amp;volume=7&amp;pages=709-U767" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20711195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewis</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Searle</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Apollo: a sequence annotation editor</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">Research0082</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apollo%3A%20a%20sequence%20annotation%20editor&amp;author=S.E.%20Lewis&amp;author=S.M.%20Searle&amp;author=N.%20Harris&amp;publication_year=2002&amp;journal=Genome%20Biol.&amp;volume=3&amp;pages=Research0082" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-12-research0082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-12-research0082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-12-research0082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12537571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlüter</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Reinkensmeier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Daschkey</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A genome-wide survey of sRNAs in the symbiotic nitrogen-fixing alpha-proteobacterium <em>Sinorhizobium meliloti</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">245</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20survey%20of%20sRNAs%20in%20the%20symbiotic%20nitrogen-fixing%20alpha-proteobacterium%20Sinorhizobium%20meliloti&amp;author=J.P.%20Schl%C3%BCter&amp;author=J.%20Reinkensmeier&amp;author=S.%20Daschkey&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=245" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20398411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Del Tordello</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bottini</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Muzzi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Serruto</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Analysis of the regulated transcriptome of <em>Neisseria meningitidis</em> in human blood using a tiling array</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">6217</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20regulated%20transcriptome%20of%20Neisseria%20meningitidis%20in%20human%20blood%20using%20a%20tiling%20array&amp;author=E.%20Del%20Tordello&amp;author=S.%20Bottini&amp;author=A.%20Muzzi&amp;author=D.%20Serruto&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=6217-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01055-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01055-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01055-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22984255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brantl</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Regulatory mechanisms employed by cis-encoded antisense RNAs</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">102</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20mechanisms%20employed%20by%20cis-encoded%20antisense%20RNAs&amp;author=S.%20Brantl&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=10&amp;pages=102-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2007.03.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2007.03.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2007.03.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17387036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osteras</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Driscoll</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Molecular and expression analysis of the <em>Rhizobium meliloti</em> phosphoenolpyruvate carboxykinase (<em>pckA</em>) gene</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1995</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">1452</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20expression%20analysis%20of%20the%20Rhizobium%20meliloti%20phosphoenolpyruvate%20carboxykinase%20(pckA)%20gene&amp;author=M.%20Osteras&amp;author=B.T.%20Driscoll&amp;author=T.M.%20Finan&amp;publication_year=1995&amp;journal=J.%20Bacteriol.&amp;volume=177&amp;pages=1452-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7883700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osteras</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Boncompagni</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Vincent</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Poggi</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Le Rudulier</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Presence of a gene encoding choline sulfatase in <em>Sinorhizobium meliloti bet</em> operon: choline-<em>O</em>-sulfate is metabolized into glycine betaine</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">11394</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Presence%20of%20a%20gene%20encoding%20choline%20sulfatase%20in%20Sinorhizobium%20meliloti%20bet%20operon%3A%20choline-O-sulfate%20is%20metabolized%20into%20glycine%20betaine&amp;author=M.%20Osteras&amp;author=E.%20Boncompagni&amp;author=N.%20Vincent&amp;author=M.C.%20Poggi&amp;author=D.%20Le%20Rudulier&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=11394-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.19.11394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.19.11394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.19.11394"> </span></div> </div></div></div></div></div><div content-id="DST014C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Capela</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Barloy-Hubler</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Analysis of the chromosome sequence of the legume symbiont <em>Sinorhizobium meliloti</em> strain 1021</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">9877</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20chromosome%20sequence%20of%20the%20legume%20symbiont%20Sinorhizobium%20meliloti%20strain%201021&amp;author=D.%20Capela&amp;author=F.%20Barloy-Hubler&amp;author=J.%20Gouzy&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=9877-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.161294398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.161294398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.161294398"> </span></div> </div></div></div></div></div><div content-id="DST014C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghosh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bose</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.D.</div></div>. </div><div class="article-title">Mechanisms for activating bacterial RNA polymerase</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2010</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">611</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20for%20activating%20bacterial%20RNA%20polymerase&amp;author=T.%20Ghosh&amp;author=D.%20Bose&amp;author=X.D.%20Zhang&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=34&amp;pages=611-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2010.00239.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2010.00239.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2010.00239.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20629756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronson</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Nixon</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Albright</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Ausubel</div> <div class="given-names">F.M.</div></div>. </div><div class="article-title"><em>Rhizobium meliloti ntrA</em> (<em>rpoN</em>) gene is required for diverse metabolic functions</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1987</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">2424</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhizobium%20meliloti%20ntrA%20(rpoN)%20gene%20is%20required%20for%20diverse%20metabolic%20functions&amp;author=C.W.%20Ronson&amp;author=B.T.%20Nixon&amp;author=L.M.%20Albright&amp;author=F.M.%20Ausubel&amp;publication_year=1987&amp;journal=J.%20Bacteriol.&amp;volume=169&amp;pages=2424-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3034856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Lucena</div> <div class="given-names">D.K.C.</div></div>,  <div class="name"><div class="surname">Puhler</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Weidner</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The role of sigma factor RpoH1 in the pH stress response of <em>Sinorhizobium meliloti</em></div>, <div class="source">BMC Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">265</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20sigma%20factor%20RpoH1%20in%20the%20pH%20stress%20response%20of%20Sinorhizobium%20meliloti&amp;author=D.K.C.%20De%20Lucena&amp;author=A.%20Puhler&amp;author=S.%20Weidner&amp;publication_year=2010&amp;journal=BMC%20Microbiol.&amp;volume=10&amp;pages=265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-10-265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-10-265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-10-265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20955556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barnett</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Bittner</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Toman</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Oke</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Dual RpoH sigma factors and transcriptional plasticity in a symbiotic bacterium</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">4983</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20RpoH%20sigma%20factors%20and%20transcriptional%20plasticity%20in%20a%20symbiotic%20bacterium&amp;author=M.J.%20Barnett&amp;author=A.N.%20Bittner&amp;author=C.J.%20Toman&amp;author=V.%20Oke&amp;author=S.R.%20Long&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=4983-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00449-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00449-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00449-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22773790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bastiat</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sauviac</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Picheraux</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rossignol</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bruand</div> <div class="given-names">C.</div></div>. </div><div class="article-title"><em>Sinorhizobium meliloti</em> sigma factors RpoE1 and RpoE4 are activated in stationary phase in response to sulfite</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e50768</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sinorhizobium%20meliloti%20sigma%20factors%20RpoE1%20and%20RpoE4%20are%20activated%20in%20stationary%20phase%20in%20response%20to%20sulfite&amp;author=B.%20Bastiat&amp;author=L.%20Sauviac&amp;author=C.%20Picheraux&amp;author=M.%20Rossignol&amp;author=C.%20Bruand&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e50768" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0050768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0050768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0050768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23226379" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacLellan</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">MacLean</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Promoter prediction in the rhizobia</div>, <div class="source">Microbiology</div> , <div class="year">2006</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">1751</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20prediction%20in%20the%20rhizobia&amp;author=S.R.%20MacLellan&amp;author=A.M.%20MacLean&amp;author=T.M.%20Finan&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=1751-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.28743-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28743-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.28743-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16735738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Zengler</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The transcription unit architecture of the <em>Escherichia coli</em> genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1043</div>-<div class="lpage">U115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcription%20unit%20architecture%20of%20the%20Escherichia%20coli%20genome&amp;author=B.K.%20Cho&amp;author=K.%20Zengler&amp;author=Y.%20Qiu&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=1043-U115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilms</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Overloper</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nowrousian</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Narberhaus</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Deep sequencing uncovers numerous small RNAs on all four replicons of the plant pathogen <em>Agrobacterium tumefaciens</em></div>, <div class="source">RNA Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">446</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20uncovers%20numerous%20small%20RNAs%20on%20all%20four%20replicons%20of%20the%20plant%20pathogen%20Agrobacterium%20tumefaciens&amp;author=I.%20Wilms&amp;author=A.%20Overloper&amp;author=M.%20Nowrousian&amp;author=C.M.%20Sharma&amp;author=F.%20Narberhaus&amp;publication_year=2012&amp;journal=RNA%20Biol.&amp;volume=9&amp;pages=446-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/rna.17212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Frna.17212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Frna.17212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22336765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fléchard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fontenelle</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Trautwetter</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ermel</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Blanco</div> <div class="given-names">C.</div></div>. </div><div class="article-title"><em>Sinorhizobium meliloti</em> rpoE2 is necessary for H(2)O(2) stress resistance during the stationary growth phase</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2009</div>, vol. <div class="volume">290</div> (pg. <div class="fpage">25</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sinorhizobium%20meliloti%20rpoE2%20is%20necessary%20for%20H(2)O(2)%20stress%20resistance%20during%20the%20stationary%20growth%20phase&amp;author=M.%20Fl%C3%A9chard&amp;author=C.%20Fontenelle&amp;author=A.%20Trautwetter&amp;author=G.%20Ermel&amp;author=C.%20Blanco&amp;publication_year=2009&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=290&amp;pages=25-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2008.290.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2008.290.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2008.290.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19025578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fléchard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fontenelle</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Blanco</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Goude</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ermel</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Trautwetter</div> <div class="given-names">A.</div></div>. </div><div class="article-title">RpoE2 of <em>Sinorhizobium meliloti</em> is necessary for trehalose synthesis and growth in hyperosmotic media</div>, <div class="source">Microbiology</div> , <div class="year">2010</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1708</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RpoE2%20of%20Sinorhizobium%20meliloti%20is%20necessary%20for%20trehalose%20synthesis%20and%20growth%20in%20hyperosmotic%20media&amp;author=M.%20Fl%C3%A9chard&amp;author=C.%20Fontenelle&amp;author=C.%20Blanco&amp;author=R.%20Goude&amp;author=G.%20Ermel&amp;author=A.%20Trautwetter&amp;publication_year=2010&amp;journal=Microbiology&amp;volume=156&amp;pages=1708-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.034850-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.034850-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.034850-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20223801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Del Val</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rivas</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Torres-Quesada</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Toro</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Jiménez-Zurdo</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Identification of differentially expressed small non-coding RNAs in the legume endosymbiont <em>Sinorhizobium meliloti</em> by comparative genomics</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">1080</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20differentially%20expressed%20small%20non-coding%20RNAs%20in%20the%20legume%20endosymbiont%20Sinorhizobium%20meliloti%20by%20comparative%20genomics&amp;author=C.%20Del%20Val&amp;author=E.%20Rivas&amp;author=O.%20Torres-Quesada&amp;author=N.%20Toro&amp;author=J.I.%20Jim%C3%A9nez-Zurdo&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=66&amp;pages=1080-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.66.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.66.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.66.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17971083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ulvé</div> <div class="given-names">V.M.</div></div>,  <div class="name"><div class="surname">Sevin</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Cheron</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Barloy-Hubler</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Identification of chromosomal alpha-proteobacterial small RNAs by comparative genome analysis and detection in <em>Sinorhizobium meliloti</em> strain 1021</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">467</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20chromosomal%20alpha-proteobacterial%20small%20RNAs%20by%20comparative%20genome%20analysis%20and%20detection%20in%20Sinorhizobium%20meliloti%20strain%201021&amp;author=V.M.%20Ulv%C3%A9&amp;author=E.W.%20Sevin&amp;author=A.%20Cheron&amp;author=F.%20Barloy-Hubler&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18093320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Valverde</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Livny</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schlüter</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Reinkensmeier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Parisi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Prediction of <em>Sinorhizobium meliloti</em> sRNA genes and experimental detection in strain 2011</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">416</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20Sinorhizobium%20meliloti%20sRNA%20genes%20and%20experimental%20detection%20in%20strain%202011&amp;author=C.%20Valverde&amp;author=J.%20Livny&amp;author=J.P.%20Schl%C3%BCter&amp;author=J.%20Reinkensmeier&amp;author=A.%20Becker&amp;author=G.%20Parisi&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacLellan</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Smallbone</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Sibley</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">The expression of a novel antisense gene mediates incompatibility within the large <em>repABC</em> family of alpha-proteobacterial plasmids</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">611</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20expression%20of%20a%20novel%20antisense%20gene%20mediates%20incompatibility%20within%20the%20large%20repABC%20family%20of%20alpha-proteobacterial%20plasmids&amp;author=S.R.%20MacLellan&amp;author=L.A.%20Smallbone&amp;author=C.D.%20Sibley&amp;author=T.M.%20Finan&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=55&amp;pages=611-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04412.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04412.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04412.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Espeli</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Moulin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Boccard</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Transcription attenuation associated with bacterial repetitive extragenic BIME elements</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">314</div> (pg. <div class="fpage">375</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20attenuation%20associated%20with%20bacterial%20repetitive%20extragenic%20BIME%20elements&amp;author=O.%20Espeli&amp;author=L.%20Moulin&amp;author=F.%20Boccard&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=314&amp;pages=375-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.5150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.5150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.5150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khemici</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Carpousis</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">The RNA degradosome and poly(A) polymerase of <em>Escherichia coli</em> are required <em>in vivo</em> for the degradation of small mRNA decay intermediates containing REP-stabilizers</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">777</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RNA%20degradosome%20and%20poly(A)%20polymerase%20of%20Escherichia%20coli%20are%20required%20in%20vivo%20for%20the%20degradation%20of%20small%20mRNA%20decay%20intermediates%20containing%20REP-stabilizers&amp;author=V.%20Khemici&amp;author=A.J.%20Carpousis&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=51&amp;pages=777-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03862.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03862.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03862.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14731278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aguena</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ferreira</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Spira</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Stability of the <em>pstS</em> transcript of <em>Escherichia coli</em></div>, <div class="source">Arch. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">105</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stability%20of%20the%20pstS%20transcript%20of%20Escherichia%20coli&amp;author=M.%20Aguena&amp;author=G.%20Ferreira&amp;author=B.%20Spira&amp;publication_year=2009&amp;journal=Arch.%20Microbiol.&amp;volume=191&amp;pages=105-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00203-008-0433-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00203-008-0433-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00203-008-0433-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18820899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Macvanin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Trostel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zhurkin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Adhya</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Noncoding RNAs binding to the nucleoid protein HU in <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">6046</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noncoding%20RNAs%20binding%20to%20the%20nucleoid%20protein%20HU%20in%20Escherichia%20coli&amp;author=M.%20Macvanin&amp;author=R.%20Edgar&amp;author=F.%20Cui&amp;author=A.%20Trostel&amp;author=V.%20Zhurkin&amp;author=S.%20Adhya&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=6046-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00961-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00961-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00961-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22942248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dornenburg</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">DeVita</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Palumbo</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wade</div> <div class="given-names">J.T.</div></div>. </div><div class="article-title">Widespread antisense transcription in <em>Escherichia coli</em></div>, <div class="source">mBio</div> , <div class="year">2010</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">e00024</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20antisense%20transcription%20in%20Escherichia%20coli&amp;author=J.E.%20Dornenburg&amp;author=A.M.%20DeVita&amp;author=M.J.%20Palumbo&amp;author=J.T.%20Wade&amp;publication_year=2010&amp;journal=mBio&amp;volume=1&amp;pages=e00024-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/mBio.00024-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.00024-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FmBio.00024-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20689751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Selinger</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Cheung</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Mei</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">RNA expression analysis using a 30 base pair resolution <em>Escherichia coli</em> genome array</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2000</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1262</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20expression%20analysis%20using%20a%2030%20base%20pair%20resolution%20Escherichia%20coli%20genome%20array&amp;author=D.W.%20Selinger&amp;author=K.J.%20Cheung&amp;author=R.%20Mei&amp;publication_year=2000&amp;journal=Nat.%20Biotechnol.&amp;volume=18&amp;pages=1262-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/82367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F82367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F82367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11101804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C82" class="ref-content "><div class="label -label">82</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Darfeuille</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">The primary transcriptome of the major human pathogen <em>Helicobacter pylori</em></div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">464</div> (pg. <div class="fpage">250</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20primary%20transcriptome%20of%20the%20major%20human%20pathogen%20Helicobacter%20pylori&amp;author=C.M.%20Sharma&amp;author=S.%20Hoffmann&amp;author=F.%20Darfeuille&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=250-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20164839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C83" class="ref-content "><div class="label -label">83</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicolas</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mäder</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Dervyn</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Condition-dependent transcriptome reveals high-level regulatory architecture in <em>Bacillus subtilis</em></div>, <div class="source">Science</div> , <div class="year">2012</div>, vol. <div class="volume">335</div> (pg. <div class="fpage">1103</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Condition-dependent%20transcriptome%20reveals%20high-level%20regulatory%20architecture%20in%20Bacillus%20subtilis&amp;author=P.%20Nicolas&amp;author=U.%20M%C3%A4der&amp;author=E.%20Dervyn&amp;publication_year=2012&amp;journal=Science&amp;volume=335&amp;pages=1103-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1206848" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1206848" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1206848"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C84" class="ref-content "><div class="label -label">84</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Georg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">cis-Antisense RNA, another level of gene regulation in bacteria</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2011</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">286</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cis-Antisense%20RNA%2C%20another%20level%20of%20gene%20regulation%20in%20bacteria&amp;author=J.%20Georg&amp;author=W.R.%20Hess&amp;publication_year=2011&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=75&amp;pages=286-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00032-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00032-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00032-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21646430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C85" class="ref-content "><div class="label -label">85</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aneja</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Charles</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Poly-3-hydroxybutyrate degradation in <em>Rhizobium</em> (<em>Sinorhizobium</em>) <em>meliloti</em>: isolation and characterization of a gene encoding 3-hydroxybutyrate dehydrogenase</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">849</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poly-3-hydroxybutyrate%20degradation%20in%20Rhizobium%20(Sinorhizobium)%20meliloti%3A%20isolation%20and%20characterization%20of%20a%20gene%20encoding%203-hydroxybutyrate%20dehydrogenase&amp;author=P.%20Aneja&amp;author=T.C.%20Charles&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=849-57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9922248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C86" class="ref-content "><div class="label -label">86</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacLean</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Fowler</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Identification of a hydroxyproline transport system in the legume endosymbiont <em>Sinorhizobium meliloti</em></div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2009</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1116</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20hydroxyproline%20transport%20system%20in%20the%20legume%20endosymbiont%20Sinorhizobium%20meliloti&amp;author=A.M.%20MacLean&amp;author=C.E.%20White&amp;author=J.E.%20Fowler&amp;author=T.M.%20Finan&amp;publication_year=2009&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=22&amp;pages=1116-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-22-9-1116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-22-9-1116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-22-9-1116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19656046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C87" class="ref-content "><div class="label -label">87</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Better</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lewis</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Corbin</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ditta</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Helinski</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Structural relationships among <em>Rhizobium meliloti</em> symbiotic promoters</div>, <div class="source">Cell</div> , <div class="year">1983</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">479</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20relationships%20among%20Rhizobium%20meliloti%20symbiotic%20promoters&amp;author=M.%20Better&amp;author=B.%20Lewis&amp;author=D.%20Corbin&amp;author=G.%20Ditta&amp;author=D.R.%20Helinski&amp;publication_year=1983&amp;journal=Cell&amp;volume=35&amp;pages=479-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(83)90181-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(83)90181-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(83)90181-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6317191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C88" class="ref-content "><div class="label -label">88</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shatters</div> <div class="given-names">R.G.</div></div>,  <div class="name"><div class="surname">Somerville</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Kahn</div> <div class="given-names">M.L.</div></div>. </div><div class="article-title">Regulation of glutamine synthetase II activity in <em>Rhizobium meliloti</em> 104A14</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1989</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">5087</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20glutamine%20synthetase%20II%20activity%20in%20Rhizobium%20meliloti%20104A14&amp;author=R.G.%20Shatters&amp;author=J.E.%20Somerville&amp;author=M.L.%20Kahn&amp;publication_year=1989&amp;journal=J.%20Bacteriol.&amp;volume=171&amp;pages=5087-94" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2570059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C89" class="ref-content "><div class="label -label">89</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Bruijn</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">Rossbach</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schneider</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title"><em>Rhizobium meliloti</em> 1021 has three differentially regulated loci involved in glutamine biosynthesis, none of which is essential for symbiotic nitrogen fixation</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1989</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1673</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhizobium%20meliloti%201021%20has%20three%20differentially%20regulated%20loci%20involved%20in%20glutamine%20biosynthesis%2C%20none%20of%20which%20is%20essential%20for%20symbiotic%20nitrogen%20fixation&amp;author=F.J.%20De%20Bruijn&amp;author=S.%20Rossbach&amp;author=M.%20Schneider&amp;publication_year=1989&amp;journal=J.%20Bacteriol.&amp;volume=171&amp;pages=1673-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2563998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C90" class="ref-content "><div class="label -label">90</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arcondeguy</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Huez</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Fourment</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kahn</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Symbiotic nitrogen fixation does not require adenylylation of glutamine synthetase I in <em>Rhizobium meliloti</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1996</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">33</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Symbiotic%20nitrogen%20fixation%20does%20not%20require%20adenylylation%20of%20glutamine%20synthetase%20I%20in%20Rhizobium%20meliloti&amp;author=T.%20Arcondeguy&amp;author=I.%20Huez&amp;author=J.%20Fourment&amp;author=D.%20Kahn&amp;publication_year=1996&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=145&amp;pages=33-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1996.145.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1996.145.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1996.145.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8931324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C91" class="ref-content "><div class="label -label">91</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">Y.F.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">G.Q.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Characterization of sequences downstream from transcriptional start site of <em>Rhizobium meliloti nifHDK</em> promoter</div>, <div class="source">Sci. China C Life Sci.</div> , <div class="year">1997</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">217</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20sequences%20downstream%20from%20transcriptional%20start%20site%20of%20Rhizobium%20meliloti%20nifHDK%20promoter&amp;author=Y.F.%20Gao&amp;author=T.%20Wu&amp;author=J.B.%20Zhu&amp;author=G.Q.%20Yu&amp;author=S.J.%20Shen&amp;publication_year=1997&amp;journal=Sci.%20China%20C%20Life%20Sci.&amp;volume=40&amp;pages=217-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02882051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02882051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02882051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18726319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C92" class="ref-content "><div class="label -label">92</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Taté</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Riccio</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Merrick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Patriarca</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">The <em>Rhizobium etli amtB</em> gene coding for an NH4+ transporter is down-regulated early during bacteroid differentiation</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">1998</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">188</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Rhizobium%20etli%20amtB%20gene%20coding%20for%20an%20NH4%2B%20transporter%20is%20down-regulated%20early%20during%20bacteroid%20differentiation&amp;author=R.%20Tat%C3%A9&amp;author=A.%20Riccio&amp;author=M.%20Merrick&amp;author=E.J.%20Patriarca&amp;publication_year=1998&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=11&amp;pages=188-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1998.11.3.188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1998.11.3.188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1998.11.3.188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9487694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C93" class="ref-content "><div class="label -label">93</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dusha</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Austin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The upstream region of the <em>nodD3</em> gene of <em>Sinorhizobium meliloti</em> carries enhancer sequences for the transcriptional activator NtrC</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1999</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">491</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20upstream%20region%20of%20the%20nodD3%20gene%20of%20Sinorhizobium%20meliloti%20carries%20enhancer%20sequences%20for%20the%20transcriptional%20activator%20NtrC&amp;author=I.%20Dusha&amp;author=S.%20Austin&amp;author=R.%20Dixon&amp;publication_year=1999&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=179&amp;pages=491-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1999.179.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1999.179.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1999.179.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10518756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST014C94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST014C94" class="ref-content "><div class="label -label">94</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dombrecht</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Marchal</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Vanderleyden</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Michiels</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Prediction and overview of the RpoN-regulon in closely related species of the Rhizobiales</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">Research0076</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20and%20overview%20of%20the%20RpoN-regulon%20in%20closely%20related%20species%20of%20the%20Rhizobiales&amp;author=B.%20Dombrecht&amp;author=K.%20Marchal&amp;author=J.%20Vanderleyden&amp;author=J.%20Michiels&amp;publication_year=2002&amp;journal=Genome%20Biol.&amp;volume=3&amp;pages=Research0076" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-12-research0076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-12-research0076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-12-research0076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12537565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Kenta Nakai</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353499&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table9.xlsx?Expires=2147483647&amp;Signature=T8F~Fz~YdzIX2yQnYfOfhG6ghn5WXDwqXdJzaVaSUIUzPUTfHSa4DSNiBQj3HEDvwKm~s51VeTwKxR-TelBmnwHJeDRb6hTyWgJtJT1eWBsCJYBuAYmNYlNHJ5X5478DOY7iVwOTxzYCHho9RRAuR8IHQCNYu8OLzH5O7U~24g2WguS4M8EpwSDbvgX5LBd8TgfxI0KTiI6H8Tep3S2rp2GFGxQDDttBIvWr7KVvDmJOV06Zxel4~WLOt5bqxMcLrsebkcjaV6ES3OyqSbZ8IKHOVmOoziLSiNChkVC0rYzECeJE63u9Mb5MQB3deG8CZZySx9BeXXn982AsiTr5gQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table8.xlsx?Expires=2147483647&amp;Signature=aJNj4iOVilvcs1Nb1QYKAh99WTosxObGTfHSccBkShAOU2Z-l3zJo3YojlDZB~HzjpxC1aPz7NIjhK6dZWbmwvXMRBw5yupkRXnY6kw1ri3n3kZHyVfsL0IaaBUiYPGaGcU8zTWI4MUBghR1T9LPvD-cPNPbuvsCvz1tfciyM7Ahng52Jup2LbPV9e8KQtRK1OT-5Oygp6GP7-nzg-8bCoojcGdnYC3ipHSv8YWdLV77IxXZnJradYp6mCefXKYd9xEZWLvAoKJdXHyDWdAblxi~VaRoHhc5SfHeT2m~UQQIMQU5~yLcn8l8s6iMNyjqt1YCmrksiTJJxwdSqMzMAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table6.xls?Expires=2147483647&amp;Signature=HYRgst64qtUcFYkxWkmGw4U8fqqiOR682YDJaRIulSt-0nDRPvdovg7Way8lTmYnGoT4MqoIcXDzl31c49ORY4-PJ476NuEW~chAVbqafDxPjR1aScM6N5u2sEFqFGeE3a0eHzC6Y~XhJIDkTjYLJ0DvLe0XJAlLuY3EVbp2cr582hlP617lKVT3MLTeEavZCFa~CgiRWJDB80GMBBWUzNpomz3610eDJyL9TS5Owacv3jdVOvUaeXL0yLfSzInrd8VuZiAWws8bbaNd0TEcmRsFAb9R-3h3~ySw~qZpogbOdjv5LpRdgojg7GPliHlehyLPS07tR~BG9v8th5pqVA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table5.xlsx?Expires=2147483647&amp;Signature=AfMLqJpK9vVJ0-6OOGOKPP-lHMZJP~k1wToHEgXLXtTcj-buMD95t9BYqGeSzCas8p2w7NcBsbmh5MZGQsbBv7jA4lMGiEyxJ6zEO58a2G~B8FTjlmMR4KHLTj2zIkIhzUF0FKS4JYpxAPt97uggBmhXJyE9kCRTJ2vDWEK-DIODjvzg2-BG9AnsEO50nCesEtUouzQpIXoGRgnpRR5nhnQhLNgLejbfuSSpAvEJqQfqSVRjEvfxeSQVWLv-Bf-~wtcjsEKKr95M4uR36xN94FnfoW-QXQVTvWNOL4hmwJGzBdivxppDOFuIYDmYUbpMZoHjyoZ0MzoOvkhxaHtjmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table4.xlsx?Expires=2147483647&amp;Signature=ezXw8nTBTVkEf9EykrUQilARsEXQvZsCM-CmbWFc0OVHMs15JaM5EKh9qfBxRU7-zK4B6jf9u5xwagM86VhaYJRNX74J6TOdpkue83m6NYkOpR5j-gb9PMmMFOU7e~oNqkPR4srzGukrH0vjfyfUut-wa64n-JpbTjxRUddTHJ453UWpVZwSUV06LtSrfrWFxw2JkJe5qmg31VKraPQbNkxmnkM5zjyJxjfXj0FYcDEhCIh-~OVThwVIBSKt963TFjpelZ-87uilPC1mLzPps472cVYfayiaqJzrv4XIkgJaInYkgWLzIrPuRbHtt7JVsjI5wDSZaZ0HptORBeS20Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table3.xlsx?Expires=2147483647&amp;Signature=ue8fhXAMhyPWEiwEVIAV9v1piYpNOo6zJzk57Vn89c9FR0AjYeEOcUcclb30BToRqizAb6890~Bq2qgLYKmLTdae1CUzuwQ-C9uLqmyNHvyx41ptC89dd2245iCMuLp~yoXlhZp4-xJgS2z1sd2EKM3TH-SIVDCIKhRQ~jxCAR8Ct7JUjCWxTZ59vV1wpWOTgNvG-nWOawLsBcWTK3rO4iuOxex77r~FmN-tE0~oSrqPZJSVDR6j19QQmWw8HVTVuE592GGVZG8zAUBCSNwq34KxJYnXnmI~2zTqv~Jr-Hz4NkksSqIZ8AUJBZtAwEx~nn-ZnM7-zA3G5i1AdGNHSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table2.docx?Expires=2147483647&amp;Signature=m7rsnQ8Ksuv3pd6yFqVjrDtvVJjpzvmfheOeuVXkag0nFGbi-KXkDT5bKe8uy29Zo9NZbX0sHNP6FiU~rkSXhbcmwcsoGaRGBA2wnWgfImWp0Uuob~bGxYQeNajlW7iGOz4yy~TYuO~sFiAuisGB5QQzBpj8SvoEFmFv~UuUC8iY8GLc53AqvuzdBFX7imOZdIh23oV25Ayomu5nvFyC3GGX33lgQoMXf8Lk2a74Ne6u8V8TnAhQIG8uD~HaX6RXUo4mB6GP5O1yLbHZCzAUcVyMJNMCQ6vhKjdBBCDfaIJnLF~MtW5izbiRy6RvULZTq8Dghsj18syR3M-M3ZVS1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table10.xlsx?Expires=2147483647&amp;Signature=4wMbpmYHfEZ91-3IuHkNPoZJrGQ1G-FjDT7MgWbqLVO9kU48DsZZzUnu~-w32xOaFMspKaZo40fCevg9x5i-DAYGj5YRgDX2ZkPGIZ8iA6bSoTfvPhC8FrhcXXXBQQAUdYh7fClKorco3y18UEPqqM6XcVEJcHZ8v-iEL4cfVN6xTdZxRK2JrDbmekJLQDpJPhy0llGLk-VgrwpiQRM-hAdoucBM4wwNc4fONbD~XpkB243zz4e7oiQ0uLrknaVGqsiW3EPG8Gw8de36E8Blsnd5mUDmLqhxNT7fvbD3f8lZMIvz8P~-CIrc6FFqytndqyYzDZuiJwMqWqp-aw2yYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_table1.xlsx?Expires=2147483647&amp;Signature=aEu7jlJUl9TPFeFWh1SAkvJ3bWstCTw~dBywqN5g9kbUuj-1jBaUwfMCXNd7nhtVid9sXd1~EGA1hKRcXZ1jF3ayvJd7-jijMFI~gNJ3M8MMeaZziKQ8qzlrb2gJ6Mg2KB42u9kmk2tieNRihFD1w3KX5FWCrBotyNuM77JfwBsQCFPx746g7cESJ3EKqt3yGlKpO~SwoeNAVEWVh7HbOQlX0l-gh9gXw6MoLZ6pcT4hULGTh~cJqBtHMUJMwM9f9e2-HxDdDvFuFkFuHCQpWmqrHQor2ENkcthLLJPvQdFwycvHNbmCtKwbTdLFCvuOF~0zorgu380mayxjZhpztQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_fig3.ppt?Expires=2147483647&amp;Signature=hCr~GDijus9TnhZlhWjdKA5id39ErAL22v-RaCYaQSH7oTIKkh4YjAL67WjtmolNGGBYZmHBCgLWh-dNzKH2DKuZQ9lnVfkUmwxuKpVMeVCZxVQoF68FXjdlfGeCemuJAz48QGGwszYe4QhtQ6cwu4uiN67328GSBZddnTNUN8wA02oG81ViGTfFdbXDqLGH4pLdrCn03Nj~NmEL~W81qW4vNUN8ecbtfnutJlffl1R~EAcZWzQ02qNQRrPxTLoJ-L-2nDVa1chT6L3Wfzlbg0bjjaPlQuE-h7a414EZWXHcEAJaOaJahe~rLr3b6nN-iAjH1a3wCUvLUkHEiQTb1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_fig2.ppt?Expires=2147483647&amp;Signature=Ol~SfSZXd--eZnTe4pn2gkcMlSA6bWXQm6NjiHcpHy8dM0jdcYzCDFJhD-3QFeJaDCBx9pXBwgvwc3bM27jUqOX0LkTaBNyIWQusJYpgFuVrU5wz1lzIevRl44BfEv-Gf8Iv-Vo~kFc1W-qYrPJBnYMQc-GBLWGUAuWlUm-XGO9YNqKn8zCtQYGwAAPPRGkZySRS3qYt44Jnnhkv9W-Coi~n4wRr7qZxExnpTrv3RQW56Pba6p7WK5mwwAdxHTcQ~SRASfDk6SgbkyxAS5JcLyERTkZfCqRQQCosQtdNeKQ6wodR2lfYJFMqzLO6AQl1j7GEtbMN-MullhRBSd5r3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp_fig1.ppt?Expires=2147483647&amp;Signature=i18G1CgQKR83xsN-j7UNf8tXRhQ-YvBAbkWtdQtyDEi2kW3O2jcIIoeAZzk~k8dTOF7Gqe8fKF5vHoSWGKdPZOPmqpXvMBI2LG5oEa9ixEzeimu3GR3EoUPvEowmscwORQIu9RZymVpzVpNBijXR8i73MDRdvLKV3AQFc8-X3GZnTE9GuCo0ElaRo01LysDxTvwLNp1VCeKI2HdlFCVOgzyQf7cL3GmjfyJk1K0fEk1z6NQOIRetsLfdGrFCgIb0ybCdo1TOXBOGOh9QSR2szEMtOCTVzDOU0qJ4j7KwK~wHiduDJbHe3mBnMot8XNya4UvOuPhE9~MZDlKboEf~rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst014/2/dst014-dst014supp.doc?Expires=2147483647&amp;Signature=3o7v8j2DVrj177FJPlHYCe9puBNWIKVX9tUEcjLk4LNAu~fh0A3oB2kAV4Y4gQh2duOg~l2PquhRFqEjcCZC2-BwySTV3r7Ce95zbpuOiKFxVV96F~6sGj3-Dv9m3Q-PeLSvz0fNofO7Hfb9By2LUYGuir~sHnIhVcjQ8~RaGBlvNQyt0KkZbqEuksm4O4kf~gOud8Fysl6tXW~TIiBdY3CgreBlojyx7kR7AW5LJkqlrbKnGrHUd5B8xJkc~plovd7vmcLTsYbFJWDm7-1fIfJFOVIQPcZpijoQM64KRvhSgel5Sy4Lxg4MXarLdIWwcBhIJdtgHb6-TBPV1yM9QA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">845</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">31</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353499"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f4%2f339%2f353499">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000324223800003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (31)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F4%2F339%2F353499" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353499'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Next-Generation Annotation of Prokaryotic Genomes with EuGene-P: Application to Sinorhizobium meliloti 2011" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f339%2f353499">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

