

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1983,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/3/155/366369","@type":"ScholarlyArticle","name":"Analysis of Multiple Occurrences of Alternative Splicing Events in  Arabidopsis thaliana  Using Novel Sequenced Full-Length cDNAs","datePublished":"2009-05-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2009-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp009","keywords":["<italic>Arabidopsis</italic>","alternative splicing","bioinformatics","full-length cDNA"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/3/155/366369","author":[{"name":"Iida, Kei","affiliation":"1  Faculty of Bio-Science ,  Nagahama Institute of Bio-Science and Technology ,  1266 Tamura, Nagahama, Shiga 526-0829 ,  Japan","@type":"Person"},{"name":"Fukami-Kobayashi, Kaoru","affiliation":"2  RIKEN BioResource Center, RIKEN Tsukuba Institute ,  Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"3  Sequence Technology Team ,  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Sakaki, Yoshiyuki","affiliation":"3  Sequence Technology Team ,  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Kobayashi, Masatomo","affiliation":"2  RIKEN BioResource Center, RIKEN Tsukuba Institute ,  Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Seki, Motoaki","affiliation":"4  Plant Genomic Network Research Team ,  RIKEN Plant Science Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Shinozaki, Kazuo","affiliation":"5  Gene Discovery Research Team ,  RIKEN Plant Science Center ,  Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074 ,  Japan","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is a mechanism by which multiple types of mature mRNAs are generated from a single pre-mature mRNA. In this study, we compl","pageStart":"155","pageEnd":"164","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/3/1/m_cover.gif?Expires=2147483647&Signature=AcThEeT5mOHrn~n4-U2ryNv1W60jdGb9iu1L9nlcbclopLicgBN0WDDbBpKjPWlTsx9BHm5I6NX45pGwX6pDvpSXNABBg78sA~cRejaS9CHu0~HvMsRN8l4Tq5wEyqC84J48vL4fbKrMO~DHcPJqtkpkvuATW5d0yUk-CPGrqGF3BQKuy3~OR4IR1reWXGlKLKLLaHg8TsFluNKyfLns8RSK8SbX128fX~LYYY5tl2NKidpRJxcVvxD-AMhHDWtKbVphkBmHy-hpTxfhPnvHKTNpLe2JT36TI2MMR~h2Kz8un9MhjBKpG2GNydqKyfGeHtc04Sf-Fwx1pA9QKagYow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Analysis of Multiple Occurrences of Alternative Splicing Events in  Arabidopsis thaliana  Using Novel Sequenced Full-Length cDNAs","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/3/1/m_cover.gif?Expires=2147483647&Signature=AcThEeT5mOHrn~n4-U2ryNv1W60jdGb9iu1L9nlcbclopLicgBN0WDDbBpKjPWlTsx9BHm5I6NX45pGwX6pDvpSXNABBg78sA~cRejaS9CHu0~HvMsRN8l4Tq5wEyqC84J48vL4fbKrMO~DHcPJqtkpkvuATW5d0yUk-CPGrqGF3BQKuy3~OR4IR1reWXGlKLKLLaHg8TsFluNKyfLns8RSK8SbX128fX~LYYY5tl2NKidpRJxcVvxD-AMhHDWtKbVphkBmHy-hpTxfhPnvHKTNpLe2JT36TI2MMR~h2Kz8un9MhjBKpG2GNydqKyfGeHtc04Sf-Fwx1pA9QKagYow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Analysis of Multiple Occurrences of Alternative Splicing Events in  Arabidopsis thaliana  Using Novel Sequenced Full-Length cDNAs" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is a mechanism by which multiple types of mature mRNAs are generated from a single pre-mature mRNA. In this study, we compl" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/3/155/366369" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-05-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iida, Kei" /><meta name="citation_author_institution" content="Faculty of Bio-Science, Nagahama Institute of Bio-Science and Technology, 1266 Tamura, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_author" content="Fukami-Kobayashi, Kaoru" /><meta name="citation_author_institution" content="RIKEN BioResource Center, RIKEN Tsukuba Institute, Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="Sequence Technology Team, RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Sakaki, Yoshiyuki" /><meta name="citation_author_institution" content="Sequence Technology Team, RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Kobayashi, Masatomo" /><meta name="citation_author_institution" content="RIKEN BioResource Center, RIKEN Tsukuba Institute, Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Seki, Motoaki" /><meta name="citation_author_institution" content="Plant Genomic Network Research Team, RIKEN Plant Science Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Shinozaki, Kazuo" /><meta name="citation_author_institution" content="Gene Discovery Research Team, RIKEN Plant Science Center, Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074, Japan" /><meta name="citation_title" content="Analysis of Multiple Occurrences of Alternative Splicing Events in Arabidopsis thaliana Using Novel Sequenced Full-Length cDNAs" /><meta name="citation_firstpage" content="155" /><meta name="citation_lastpage" content="164" /><meta name="citation_doi" content="10.1093/dnares/dsp009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2009/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomewide comparative analysis of alternative splicing in plants; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=7175-7180; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays; Science;  citation_year=2003;  citation_volume=302;  citation_pages=2141-2144; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1559-1563; " /><meta name="citation_reference" content="citation_title=Functional significance of the alternative transcript processing of the Arabidopsis floral promoter FCA; Plant Cell;  citation_year=2002;  citation_volume=14;  citation_pages=877-888; " /><meta name="citation_reference" content="citation_title=Alternative splicing and mRNA levels of the disease resistance gene RPS4 are induced during defense responses; Plant Physiol.;  citation_year=2007;  citation_volume=145;  citation_pages=1577-1587; " /><meta name="citation_reference" content="citation_title=Evolutionary conservation and regulation of particular alternative splicing events in plant SR proteins; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=4395-4405; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of alternative pre-mRNA splicing in Arabidopsis thaliana based on full-length cDNA sequences; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=5096-5103; " /><meta name="citation_reference" content="citation_title=Characterization of wound-responsive RNA-binding proteins and their splice variants in Arabidopsis; Plant Mol. Biol.;  citation_year=2008;  citation_volume=67;  citation_pages=71-88; " /><meta name="citation_reference" content="citation_title=Differential expression of alternatively spliced mRNAs of Arabidopsis SR protein homologs, atSR30 and atSR45a, in response to environmental stress; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=1036-1049; " /><meta name="citation_reference" content="citation_title=Alternative splicing of pre-mRNAs of Arabidopsis serine/arginine-rich proteins: regulation by hormones and stresses; Plant J.;  citation_year=2007;  citation_volume=49;  citation_pages=1091-1107; " /><meta name="citation_reference" content="citation_title=Alternative splicing and differential gene expression in colon cancer detected by a whole genome exon array; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=325" /><meta name="citation_reference" content="citation_title=Functional annotation of a full-length Arabidopsis cDNA collection; Science;  citation_year=2002;  citation_volume=296;  citation_pages=141-145; " /><meta name="citation_reference" content="citation_title=Empirical analysis of transcriptional activity in the Arabidopsis genome; Science;  citation_year=2003;  citation_volume=302;  citation_pages=842-846; " /><meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=195-202; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1157-1169; " /><meta name="citation_reference" content="citation_title=Alternative splicing at NAGNAG acceptor sites shares common properties in land plants and mammals; Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=709-718; " /><meta name="citation_reference" content="citation_title=The Arabidopsis information resource (TAIR): gene structure and function annotation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D1009-D1014; " /><meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D247-D251; " /><meta name="citation_reference" content="citation_title=Transcription-mediated gene fusion in the human genome; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=30-36; " /><meta name="citation_reference" content="citation_title=A novel testis ubiquitin-binding protein gene arose by exon shuffling in hominoids; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1129-1138; " /><meta name="citation_reference" content="citation_title=Alternative splicing and genome complexity; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=29-30; " /><meta name="citation_reference" content="citation_title=Pre-mRNA splicing in higher plants; Trends Plant Sci.;  citation_year=2000;  citation_volume=5;  citation_pages=160-167; " /><meta name="citation_reference" content="citation_title=Plant serine/arginine-rich proteins and their role in pre-mRNA splicing; Trends Plant Sci.;  citation_year=2004;  citation_volume=9;  citation_pages=541-547; " /><meta name="citation_reference" content="citation_title=A plethora of plant serine/arginine-rich proteins: redundancy or evolution of novel gene functions?; Biochem. Soc. Trans.;  citation_year=2004;  citation_volume=32;  citation_pages=561-564; " /><meta name="citation_reference" content="citation_title=The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses; Plant J.;  citation_year=2007;  citation_volume=50;  citation_pages=347-363; " /><meta name="citation_reference" content="citation_title=Context analysis of termination codons in mRNA that are recognized by plant NMD; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=1072-1078; " /><meta name="citation_reference" content="citation_title=Survey of conserved alternative splicing events of mRNAs encoding SR proteins in land plants; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1085-1094; " /><meta name="citation_reference" content="citation_title=Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=189-192; " /><meta name="citation_reference" content="citation_title=Alternative splicing and protein structure evolution; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=550-558; " /><meta name="citation_reference" content="citation_title=Eukaryotic translation elongation factor 1 gamma contains a glutathione transferase domain–study of a diverse, ancient protein superfamily using motif search and structural modeling; Protein Sci.;  citation_year=1994;  citation_volume=3;  citation_pages=2045-2054; " /><meta name="citation_reference" content="citation_title=The intermolecular interaction between the PH domain and the C-terminal domain of Arabidopsis dynamin-like 6 determines lipid binding specificity; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=31842-31849; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/3/155/1045587/dsp009.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is a mechanism by which multiple types of mature mRNAs are generated from a single pre-mature mRNA. In this study, we compl" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/3/155/366369" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/3/155/366369" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/3/155/366369");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp009';
    </script>
<title>
	Analysis of Multiple Occurrences of Alternative Splicing Events in Arabidopsis thaliana Using Novel Sequenced Full-Length cDNAs | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./366369" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcpPPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJaWRhLCBLZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEJpby1TY2llbmNlLCBOYWdhaGFtYSBJbnN0aXR1dGUgb2YgQmlvLVNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDEyNjYgVGFtdXJhLCBOYWdhaGFtYSwgU2hpZ2EgNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVrYW1pLUtvYmF5YXNoaSwgS2FvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBCaW9SZXNvdXJjZSBDZW50ZXIsIFJJS0VOIFRzdWt1YmEgSW5zdGl0dXRlLCBLb3lhZGFpIDMtMS0xLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRveW9kYSwgQXRzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNlcXVlbmNlIFRlY2hub2xvZ3kgVGVhbSwgUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrYWtpLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTZXF1ZW5jZSBUZWNobm9sb2d5IFRlYW0sIFJJS0VOIEdlbm9taWMgU2NpZW5jZXMgQ2VudGVyLCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvYmF5YXNoaSwgTWFzYXRvbW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBCaW9SZXNvdXJjZSBDZW50ZXIsIFJJS0VOIFRzdWt1YmEgSW5zdGl0dXRlLCBLb3lhZGFpIDMtMS0xLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNla2ksIE1vdG9ha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudCBHZW5vbWljIE5ldHdvcmsgUmVzZWFyY2ggVGVhbSwgUklLRU4gUGxhbnQgU2NpZW5jZSBDZW50ZXIsIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbm96YWtpLCBLYXp1byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdlbmUgRGlzY292ZXJ5IFJlc2VhcmNoIFRlYW0sIFJJS0VOIFBsYW50IFNjaWVuY2UgQ2VudGVyLCBLb3lhZGFpIDMtMS0xLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW5hbHlzaXMgb2YgTXVsdGlwbGUgT2NjdXJyZW5jZXMgb2YgQWx0ZXJuYXRpdmUgU3BsaWNpbmcgRXZlbnRzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIFVzaW5nIE5vdmVsIFNlcXVlbmNlZCBGdWxsLUxlbmd0aCBjRE5BcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNwMDA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDkvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWV3aWRlIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIHBsYW50czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NzE3NS03MTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzdXJ2ZXkgb2YgaHVtYW4gYWx0ZXJuYXRpdmUgcHJlLW1STkEgc3BsaWNpbmcgd2l0aCBleG9uIGp1bmN0aW9uIG1pY3JvYXJyYXlzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMjsgIGNpdGF0aW9uX3BhZ2VzPTIxNDEtMjE0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBsYW5kc2NhcGUgb2YgdGhlIG1hbW1hbGlhbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9MTU1OS0xNTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIHNpZ25pZmljYW5jZSBvZiB0aGUgYWx0ZXJuYXRpdmUgdHJhbnNjcmlwdCBwcm9jZXNzaW5nIG9mIHRoZSBBcmFiaWRvcHNpcyBmbG9yYWwgcHJvbW90ZXIgRkNBOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9ODc3LTg4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIG1STkEgbGV2ZWxzIG9mIHRoZSBkaXNlYXNlIHJlc2lzdGFuY2UgZ2VuZSBSUFM0IGFyZSBpbmR1Y2VkIGR1cmluZyBkZWZlbnNlIHJlc3BvbnNlczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ1OyAgY2l0YXRpb25fcGFnZXM9MTU3Ny0xNTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY29uc2VydmF0aW9uIGFuZCByZWd1bGF0aW9uIG9mIHBhcnRpY3VsYXIgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzIGluIHBsYW50IFNSIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz00Mzk1LTQ0MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIGFsdGVybmF0aXZlIHByZS1tUk5BIHNwbGljaW5nIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGJhc2VkIG9uIGZ1bGwtbGVuZ3RoIGNETkEgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz01MDk2LTUxMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2Ygd291bmQtcmVzcG9uc2l2ZSBSTkEtYmluZGluZyBwcm90ZWlucyBhbmQgdGhlaXIgc3BsaWNlIHZhcmlhbnRzIGluIEFyYWJpZG9wc2lzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9NzEtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBleHByZXNzaW9uIG9mIGFsdGVybmF0aXZlbHkgc3BsaWNlZCBtUk5BcyBvZiBBcmFiaWRvcHNpcyBTUiBwcm90ZWluIGhvbW9sb2dzLCBhdFNSMzAgYW5kIGF0U1I0NWEsIGluIHJlc3BvbnNlIHRvIGVudmlyb25tZW50YWwgc3RyZXNzOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MTAzNi0xMDQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBvZiBwcmUtbVJOQXMgb2YgQXJhYmlkb3BzaXMgc2VyaW5lL2FyZ2luaW5lLXJpY2ggcHJvdGVpbnM6IHJlZ3VsYXRpb24gYnkgaG9ybW9uZXMgYW5kIHN0cmVzc2VzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTEwOTEtMTEwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIGRpZmZlcmVudGlhbCBnZW5lIGV4cHJlc3Npb24gaW4gY29sb24gY2FuY2VyIGRldGVjdGVkIGJ5IGEgd2hvbGUgZ2Vub21lIGV4b24gYXJyYXk7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIGEgZnVsbC1sZW5ndGggQXJhYmlkb3BzaXMgY0ROQSBjb2xsZWN0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MS0xNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtcGlyaWNhbCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgYWN0aXZpdHkgaW4gdGhlIEFyYWJpZG9wc2lzIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz04NDItODQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZWQ6IGEgZ3JhcGhpY2FsIHRvb2wgZm9yIHNlcXVlbmNlIGZpbmlzaGluZzsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgc3RydWN0dXJlIHByZWRpY3Rpb24gZnJvbSBjb25zZW5zdXMgc3BsaWNlZCBhbGlnbm1lbnQgb2YgbXVsdGlwbGUgRVNUcyBtYXRjaGluZyB0aGUgc2FtZSBnZW5vbWljIGxvY3VzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTExNTctMTE2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgYXQgTkFHTkFHIGFjY2VwdG9yIHNpdGVzIHNoYXJlcyBjb21tb24gcHJvcGVydGllcyBpbiBsYW5kIHBsYW50cyBhbmQgbWFtbWFsczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTcwOS03MTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBpbmZvcm1hdGlvbiByZXNvdXJjZSAoVEFJUik6IGdlbmUgc3RydWN0dXJlIGFuZCBmdW5jdGlvbiBhbm5vdGF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1EMTAwOS1EMTAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGZhbTogY2xhbnMsIHdlYiB0b29scyBhbmQgc2VydmljZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQyNDctRDI1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbi1tZWRpYXRlZCBnZW5lIGZ1c2lvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTMwLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHRlc3RpcyB1YmlxdWl0aW4tYmluZGluZyBwcm90ZWluIGdlbmUgYXJvc2UgYnkgZXhvbiBzaHVmZmxpbmcgaW4gaG9taW5vaWRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTExMjktMTEzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIGdlbm9tZSBjb21wbGV4aXR5OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI5LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmUtbVJOQSBzcGxpY2luZyBpbiBoaWdoZXIgcGxhbnRzOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTYwLTE2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgc2VyaW5lL2FyZ2luaW5lLXJpY2ggcHJvdGVpbnMgYW5kIHRoZWlyIHJvbGUgaW4gcHJlLW1STkEgc3BsaWNpbmc7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01NDEtNTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBsZXRob3JhIG9mIHBsYW50IHNlcmluZS9hcmdpbmluZS1yaWNoIHByb3RlaW5zOiByZWR1bmRhbmN5IG9yIGV2b2x1dGlvbiBvZiBub3ZlbCBnZW5lIGZ1bmN0aW9ucz87IEJpb2NoZW0uIFNvYy4gVHJhbnMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NTYxLTU2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEF0R2VuRXhwcmVzcyBnbG9iYWwgc3RyZXNzIGV4cHJlc3Npb24gZGF0YSBzZXQ6IHByb3RvY29scywgZXZhbHVhdGlvbiBhbmQgbW9kZWwgZGF0YSBhbmFseXNpcyBvZiBVVi1CIGxpZ2h0LCBkcm91Z2h0IGFuZCBjb2xkIHN0cmVzcyByZXNwb25zZXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTM2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udGV4dCBhbmFseXNpcyBvZiB0ZXJtaW5hdGlvbiBjb2RvbnMgaW4gbVJOQSB0aGF0IGFyZSByZWNvZ25pemVkIGJ5IHBsYW50IE5NRDsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEwNzItMTA3ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VydmV5IG9mIGNvbnNlcnZlZCBhbHRlcm5hdGl2ZSBzcGxpY2luZyBldmVudHMgb2YgbVJOQXMgZW5jb2RpbmcgU1IgcHJvdGVpbnMgaW4gbGFuZCBwbGFudHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMDg1LTEwOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0aGUgd2lkZXNwcmVhZCBjb3VwbGluZyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBhbmQgbm9uc2Vuc2UtbWVkaWF0ZWQgbVJOQSBkZWNheSBpbiBodW1hbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTE4OS0xOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGFuZCBwcm90ZWluIHN0cnVjdHVyZSBldm9sdXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTU1MC01NTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV1a2FyeW90aWMgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiBmYWN0b3IgMSBnYW1tYSBjb250YWlucyBhIGdsdXRhdGhpb25lIHRyYW5zZmVyYXNlIGRvbWFpbuKAk3N0dWR5IG9mIGEgZGl2ZXJzZSwgYW5jaWVudCBwcm90ZWluIHN1cGVyZmFtaWx5IHVzaW5nIG1vdGlmIHNlYXJjaCBhbmQgc3RydWN0dXJhbCBtb2RlbGluZzsgUHJvdGVpbiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yMDQ1LTIwNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbnRlcm1vbGVjdWxhciBpbnRlcmFjdGlvbiBiZXR3ZWVuIHRoZSBQSCBkb21haW4gYW5kIHRoZSBDLXRlcm1pbmFsIGRvbWFpbiBvZiBBcmFiaWRvcHNpcyBkeW5hbWluLWxpa2UgNiBkZXRlcm1pbmVzIGxpcGlkIGJpbmRpbmcgc3BlY2lmaWNpdHk7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTMxODQyLTMxODQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTYvMy8xNTUvMTA0NTU4Ny9kc3AwMDkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGVybmF0aXZlIHNwbGljaW5nIChBUykgaXMgYSBtZWNoYW5pc20gYnkgd2hpY2ggbXVsdGlwbGUgdHlwZXMgb2YgbWF0dXJlIG1STkFzIGFyZSBnZW5lcmF0ZWQgZnJvbSBhIHNpbmdsZSBwcmUtbWF0dXJlIG1STkEuIEluIHRoaXMgc3R1ZHksIHdlIGNvbXBsIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNjYzNjkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkfMcv7+dydm7Xaoqtmbr/k/+6/sY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f3%2f155%2f366369" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f3%2f155%2f366369" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14252" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/3/1/m_cover.gif?Expires=2147483647&amp;Signature=AcThEeT5mOHrn~n4-U2ryNv1W60jdGb9iu1L9nlcbclopLicgBN0WDDbBpKjPWlTsx9BHm5I6NX45pGwX6pDvpSXNABBg78sA~cRejaS9CHu0~HvMsRN8l4Tq5wEyqC84J48vL4fbKrMO~DHcPJqtkpkvuATW5d0yUk-CPGrqGF3BQKuy3~OR4IR1reWXGlKLKLLaHg8TsFluNKyfLns8RSK8SbX128fX~LYYY5tl2NKidpRJxcVvxD-AMhHDWtKbVphkBmHy-hpTxfhPnvHKTNpLe2JT36TI2MMR~h2Kz8un9MhjBKpG2GNydqKyfGeHtc04Sf-Fwx1pA9QKagYow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/3">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5319538">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319538">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319540">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319540">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319544">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319544">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319561">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319561">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319590">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319590">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319592">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319592">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319594">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319594">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5319596">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5319596">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/3/141/365625">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/3/165/366030">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Analysis of Multiple Occurrences of Alternative Splicing Events in <em>Arabidopsis thaliana</em> Using Novel Sequenced Full-Length cDNAs
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kei Iida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Bio-Science</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">1266 Tamura, Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kei+Iida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iida K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iida K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kei Iida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kaoru Fukami-Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">RIKEN BioResource Center, RIKEN Tsukuba Institute</div>, <div class="addr-line">Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kaoru+Fukami-Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukami-Kobayashi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukami-Kobayashi K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kaoru Fukami-Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Sequence Technology Team</div>, <div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Sakaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Sequence Technology Team</div>, <div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Sakaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiyuki Sakaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masatomo Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">RIKEN BioResource Center, RIKEN Tsukuba Institute</div>, <div class="addr-line">Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masatomo+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masatomo Kobayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoaki Seki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Plant Genomic Network Research Team</div>, <div class="insititution">RIKEN Plant Science Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 29-836-4359. Fax. <div class="fax">+81 29-836-9060</div>. E-mail: <a href="mailto:sinozaki@rtc.riken.jp" target="_blank">sinozaki@rtc.riken.jp</a> (K.S.); Tel. +81 45-503-9587. Fax. <div class="fax">+81 45-503-9584</div>. E-mail: <a href="mailto:mseki@psc.riken.jp" target="_blank">mseki@psc.riken.jp</a> (M.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoaki+Seki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seki M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seki M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Motoaki Seki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuo Shinozaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Gene Discovery Research Team</div>, <div class="insititution">RIKEN Plant Science Center</div>, <div class="addr-line">Koyadai 3-1-1, Tsukuba, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 29-836-4359. Fax. <div class="fax">+81 29-836-9060</div>. E-mail: <a href="mailto:sinozaki@rtc.riken.jp" target="_blank">sinozaki@rtc.riken.jp</a> (K.S.); Tel. +81 45-503-9587. Fax. <div class="fax">+81 45-503-9584</div>. E-mail: <a href="mailto:mseki@psc.riken.jp" target="_blank">mseki@psc.riken.jp</a> (M.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuo+Shinozaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinozaki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shinozaki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuo Shinozaki</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Bioinformatics and Systems Engineering Division, RIKEN Yokohama Institute, 1-7-22, Suehiro, Tsurumi, Yokohama 230-0045, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Comparative Genomics Laboratory, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN3"><div class="fn"><div class="label -label">¶</div><div class="footnote-content">Present address: Toyohashi University of Technology, 1-1, Hibarigaoka, Tenpaku-cho, Toyohashi, Aichi 441-8580, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 3, 1 June 2009, Pages 155–164, <a href='https://doi.org/10.1093/dnares/dsp009'>https://doi.org/10.1093/dnares/dsp009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 May 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 December 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 April 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="366369" href="/dnaresearch/article-pdf/16/3/155/1045587/dsp009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kei Iida, Kaoru Fukami-Kobayashi, Atsushi Toyoda, Yoshiyuki Sakaki, Masatomo Kobayashi, Motoaki Seki, Kazuo Shinozaki;  Analysis of Multiple Occurrences of Alternative Splicing Events in <em>Arabidopsis thaliana</em> Using Novel Sequenced Full-Length cDNAs, <em>DNA Research</em>, Volume 16, Issue 3, 1 June 2009, Pages 155–164, <a href='https://doi.org/10.1093/dnares/dsp009'>https://doi.org/10.1093/dnares/dsp009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/366369?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/366369?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Analysis%20of%20Multiple%20Occurrences%20of%20Alternative%20Splicing%20Events%20in%20%26lt%3Bem%26gt%3BArabidopsis%20thaliana%26lt%3B%2Fem%26gt%3B%20Using%20Novel%20Sequenced%20Full-Length%20cDNAs&amp;publicationDate=2009-05-07&amp;volumeNum=16&amp;issueNum=3&amp;author=Iida%2C%20Kei%3B%20Fukami-Kobayashi%2C%20Kaoru&amp;startPage=155&amp;endPage=164&amp;contentId=10.1093%2Fdnares%2Fdsp009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f3%2f155%2f366369" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5319538 id="5319538" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is a mechanism by which multiple types of mature mRNAs are generated from a single pre-mature mRNA. In this study, we completely sequenced 1800 full-length cDNAs from <em>Arabidopsis thaliana</em>, which had 5′ and/or 3′ sequences that were previously found to have AS events or alternative transcription start sites. Unexpectedly, these sequences gave us further evidence of AS, as 601 out of 1800 transcripts showed novel AS events. We focused on the combination patterns of multiple AS events within individual genes. Interestingly, some specific AS event combination patterns tended to appear more frequently than expected. The two most common patterns were: (i) alternative donor–0∼12 times of exon skips–alternative acceptor and (ii) several times (∼8) of retained introns. We also found that multiple AS events in a transcript tend to have the same effects concerning the length of the mature mRNA. Our current results are consistent with our previous observations, which showed changes in AS profiles under different conditions, and suggest the involvement of hypothetical <em>cis</em>- and <em>trans</em>-acting factors in the regulation of AS events.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis"><em>Arabidopsis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="bioinformatics">bioinformatics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a></div>                            <h2 scrollto-destination=5319540 id="5319540" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Alternative splicing (AS) is a mechanism by which multiple forms of mature mRNAs are generated from a single pre-mature mRNA. The most recent genome-wide analysis of AS events in <em>Arabidopsis thaliana</em> found that &gt;4700 transcribed pre-mature mRNAs were alternatively spliced.<sup><a reveal-id="DSP009C1" data-open="DSP009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The number of known AS events in <em>Arabidopsis</em> has recently been increasing. AS events are important because they diversify the proteome. The impacts of AS events on the human transcriptome and proteome are also well known. In humans, ∼70% of all genes are thought to be alternatively spliced.<sup><a reveal-id="DSP009C2" data-open="DSP009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP009C3" data-open="DSP009C3" class="link link-ref link-reveal xref-bibr">3</a></sup> It is thought that many kinds of proteins are translated from the limited number of genes encoded by the human genome. Compared with humans, there are fewer known AS events in plants. AS events in <em>Arabidopsis</em> occur in around 20% of all known genes.<sup><a reveal-id="DSP009C1" data-open="DSP009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Although the number of AS events is small, some important AS events have been described in <em>Arabidopsis</em>. For example, the floral promoter FCA, disease resistance protein RPS4 and splicing factor atRSZ33 are known to undergo AS.<sup><a reveal-id="DSP009C4  DSP009C6" data-open="DSP009C4  DSP009C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup></p><p>AS events are not only important sources for the diversification of the proteome, but they also have an important role in post-transcriptional regulation. In a previous study, we reported that the pattern of AS events on a genome-wide scale (called AS profiles) was affected by stress conditions and tissue types, especially by cold stress.<sup><a reveal-id="DSP009C7" data-open="DSP009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In a full-length cDNA library from cold-treated plants, cassette exons tend to be selected in mature mRNAs, and introns tend to be retained. Several recent studies have also found that stresses and/or tissue types affect the selectivity of AS events.<sup><a reveal-id="DSP009C8  DSP009C10" data-open="DSP009C8  DSP009C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup></p><p>Recently, several novel technologies have been established, which have improved our ability to study the transcriptome. For studying AS events on a genome-wide scale, a microarray approach, with probes designed for each exon<sup><a reveal-id="DSP009C11" data-open="DSP009C11" class="link link-ref link-reveal xref-bibr">11</a></sup> or covering the whole genome,<sup><a reveal-id="DSP009C2" data-open="DSP009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP009C3" data-open="DSP009C3" class="link link-ref link-reveal xref-bibr">3</a></sup> may prove useful. However, it is difficult to elucidate the entire sequence of AS variants with these methods. The best way to analyze the exact AS events for every transcript is still to make libraries of full-length cDNAs and sequence them. In 2004, we found 2092 AS events after analyzing ∼278 000 sequences from full-length cDNAs.<sup><a reveal-id="DSP009C7" data-open="DSP009C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP009C12" data-open="DSP009C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP009C13" data-open="DSP009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, most of these sequences were partial reads from 5′ or 3′ regions of the full-length cDNAs. Because our previous studies tended to analyze AS events in terminal regions, we could not address impacts of these AS events on the full-length sequences and their coding potentials. In the current study, we chose 1800 cDNAs that had AS events or transcriptional start sites (TSSs) in their 5′/3′ sequences, and sequenced them entirely. On the basis of these novel full-length cDNA sequences, we determined how many AS events occurred in each gene, how multiple AS events within a single gene affected each other and how AS events affected coding potentials.</p>                            <h2 scrollto-destination=5319544 id="5319544" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5319545 id="5319545" class="section-title" data-legacy-id=s2a>2.1. Selecting the full-length cDNAs for determination of the full-length sequences</h3>
<p>We used clones from the RIKEN <em>Arabidopsis</em> full-length (RAFL) cDNA libraries.<sup><a reveal-id="DSP009C12" data-open="DSP009C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP009C13" data-open="DSP009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We classified the fully and partially read RAFL cDNA sequences based on their locations in the genome, AS events and TSSs. As a first step, we constructed transcription units (TUs) based on locations of the sequences in the genome.<sup><a reveal-id="DSP009C7" data-open="DSP009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Next, we checked for the existence of AS events, which were classified as exon skip (ES), alternative donor (AD), alternative acceptor (AA) or retained intron (RI) for each TU. A set of sequences, which did not have any AS events, was classified into an AS group. In some cases, a pair of transcripts in a particular TU lacked overlapping exons. In these cases, although no AS events were found, we classified these two transcripts into different AS groups.</p><p>Finally, we checked the TSSs for each AS group. If a pair of the 5′ end read sequences or fully read sequences had TSSs with differences in length of &gt;500 bp, we separated them into two different TSS groups. As a result, each transcript was classified into a group based on TU, AS and TSS. If a group lacked fully read transcripts, we chose one clone from this group as a target for determination of the full-length sequences.</p>                            <h3 scrollto-destination=5319548 id="5319548" class="section-title" data-legacy-id=s2b>2.2. End and full-length sequencing</h3>
<p>Plasmid DNA extraction was performed using an automated DNA isolation system (PI-1100, Kurabo), and end sequencing of the 1800 clones was performed with BigDye terminator ver.3 sequencing kits and ABI 3730 automated capillary DNA sequencers (Applied Biosystems). For full-length sequencing, template DNA was individually prepared from each plasmid DNA using the rolling-circle amplification (RCA) method (TempliPhi DNA amplification kit, GE Healthcare). A pooled sample, consisting of 768 RCA products, was subjected to shotgun sequencing (ABI 3730). Another pooled sample (1032 RCA products) was sequenced using a Genome Sequencer 20 System (454 Life Sciences/Roche). The generated contig sequences were then incorporated into the end sequence data using the Phrap/Consed system.<sup><a reveal-id="DSP009C14" data-open="DSP009C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Primer walking was used to close gaps and re-sequence low-quality regions of the assembled data. The reconstructed cDNA sequences were completely covered by the Sanger reads. All sequences, excluding 55 sequences that were not mapped to the genome, were submitted to the DNA Data Bank of Japan (DDBJ) under the accession numbers AK316663–AK317782 and AK318618–AK319175.</p>                            <h3 scrollto-destination=5319550 id="5319550" class="section-title" data-legacy-id=s2c>2.3. Classification of transcripts as known or novel</h3>
<p>All novel sequences were mapped to the genome sequence with BLAST and GeneSeqer<sup><a reveal-id="DSP009C15" data-open="DSP009C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSP009C16" data-open="DSP009C16" class="link link-ref link-reveal xref-bibr">16</a></sup> using a previously described method.<sup><a reveal-id="DSP009C17" data-open="DSP009C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Next, the transcripts were classified as known or novel transcripts based on comparisons with annotated gene models. In addition to the four types of AS events (ES, AD, AA and RI), to make a detailed classification of known and novel transcripts, we also checked for alternative structure events, in which transcriptional initiation or termination occurred within the introns of other transcripts (AI and AT). Unlike the method in Section 2.1, we ignored TSS differences in this step, even if those differences were more than 500 bases, because differences found in TSSs could have occurred during the sequencing of potential partial cDNAs.</p><p>For transcript classification, we used novel sequences and annotated gene model in The <em>Arabidopsis</em> Information Resource (TAIR), version 7,<sup><a reveal-id="DSP009C18" data-open="DSP009C18" class="link link-ref link-reveal xref-bibr">18</a></sup> which corresponded with the novel sequences. For these sequences, TU clustering and AS-group clustering were performed again, using a method similar to that described in Section 2.1. Every result was checked manually after the computational classification.</p>                            <h3 scrollto-destination=5319553 id="5319553" class="section-title" data-legacy-id=s2d>2.4. Analysis of the number of AS events per gene</h3>
<p>To analyze the number of AS events per gene, we chose annotated gene loci with AS variants from TAIR. For the data set in the current study, we used novel sequenced transcripts and the annotated gene models, which were derived from corresponding loci with novel sequences. In this analysis, the number of AS events from each of the four types (ES, AD, AA and RI) was counted. The number of AS events was defined based on the type of AS event and the region affected by the AS event. We noted special cases of splice acceptors with more than three acceptor sites. In a case of an intron of three possible splice acceptor sites, two AS events were counted. One event was defined by the region affected between the first and the second acceptor sites, and the second event by the region between the second and third acceptor sites. If we had defined the AS events based on the combination of acceptor sites of the AS variant, there would have been three events. Therefore, the number of AS events would have increased dramatically due to the increased number of acceptor sites. The new method avoided this great increase in event numbers. There was also a similar rule about multiple AD sites. We compared the number of AS events found in each gene using a Mann–Whitney test and used R package (<a class="link link-uri" href="http://cran.r-project.org/">http://cran.r-project.org/</a>) to calculate the <em>P</em>-values.</p>                            <h3 scrollto-destination=5319555 id="5319555" class="section-title" data-legacy-id=s2e>2.5. Analysis of the AS event combination patterns</h3>
<p>In cases of transcripts with more than two AS events, we analyzed the pattern of AS combinations. For these analyses, we sorted AS events that were found in a transcript by the order of their locations. Then, we paired adjacent AS events according to their orders (i.e. first and second, second and third and so forth). Expectation values of each of the AS event combination patterns were calculated based on the probability of a single AS event. Statistical analyses were performed using binomial tests. We also used R package to calculate the <em>P</em>-values.</p><p>AS events could be classified based on their directions. We called AS events that made longer mRNAs ‘in-type’ events, which means regions affected by AS were included in the mRNAs. In the opposite cases, we called AS events that made shorter mRNAs ‘out-type’ events. We classified all the AS events as in- or out-type. Then, we counted the number of combinations for either the in- or out-types from adjacent AS events. Statistical analysis was performed using a chi-squared test with R package.</p>                            <h3 scrollto-destination=5319558 id="5319558" class="section-title" data-legacy-id=s2f>2.6. Prediction of coding sequences and analysis of the effects of as events on functional motifs</h3>
<p>When transcripts were classified into known TUs, we determined their coding sequences (CDSs) based on their corresponding genes. For novel transcripts, we performed BLASTX searches against the annotated protein sequences encoded in mRNAs from the corresponding loci of TAIR.<sup><a reveal-id="DSP009C15" data-open="DSP009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> From the result, we determined the core CDSs and phases, and then extended the CDS to its up- and downstream regions until we found the farthest initial codons and the nearest termination codons. The results were checked manually after the computational prediction. For transcripts classified as novel TUs, we searched CDSs based on a BLASTX search results using all protein sequences in TAIR's database (ftp://ftp.arabidopsis.org).</p><p>To assess the impact of AS events on the encoded protein sequences, we performed motif search analyses for the protein sequences encoded by the novel AS variants and corresponding annotated gene model. In this case, we selected the most similar protein sequences generated from the corresponding loci that were used as reference sequences. The motif search was performed with Pfam.<sup><a reveal-id="DSP009C19" data-open="DSP009C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h2 scrollto-destination=5319561 id="5319561" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5319562 id="5319562" class="section-title" data-legacy-id=s3a>3.1. Full sequencing of 1800 full-length cDNAs</h3>
<p>We determined the full-length sequences of 1800 RAFL cDNAs; 1657 reads (92%) had clear poly-A tails. The average length was 1495 nt, and the GC content was 42.9%. These features were quite similar to our previous sets of full-length cDNAs (average lengths were 1483 nt and GC contents were 42.8%).<sup><a reveal-id="DSP009C12" data-open="DSP009C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP009C13" data-open="DSP009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We mapped the transcripts with BLAST and GeneSequer.<sup><a reveal-id="DSP009C15" data-open="DSP009C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSP009C16" data-open="DSP009C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A total of 1745 transcripts were mapped to the <em>Arabidopsis</em> genome. We could not find the corresponding loci for the other 55 transcripts (Fig. <a reveal-id="DSP009F1" data-open="DSP009F1" class="link link-reveal link-table xref-fig">1</a>).</p>                        <a id="5319564" scrollto-destination="5319564"></a>
<div id="DSP009F1" data-id="DSP009F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00901.jpeg?Expires=2147483647&Signature=uy2j9CD~2K7zQir03yE-0prjqqiPLpleoLbsALDSCBM4PseA-GWRsO4jXpSrFyJeUKe0gY0l3K7~Tsj4TlEppoIbD-ct1SZ1b6HRnqlTJ22AMEv9OnZOiSWprghSxJketK375M7BjKXSyYQ0TXdyYIJTsgdS9-Wl-KhaAQbZb2dbqoG1NFlgaJH30gXCuUbSgx51AujqxrI5re11qS7evR3XPm9t0whaomwFVCaYxMzHNMZJurGrzPKKmrQu14vSBwWW6ywxZ9xAYA93KH02cZNPP0Y6ygQRzsepIqmaynGGU48s1rtNfuu6fGVRRQUmD7RxUdCXxOtdFT9U0PYE0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart and outline of the classifications used for current data set (upper part) and AS events found in the novel AS variants (lower part). The gray-highlighted boxes correspond to the transcripts with novel AS events. In total, 601 transcripts were classified as novel AS variants. And 342 AS events were novel ones found in the current study." path-from-xml="dsp00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319564/dsp00901.jpeg" path-from-xml="dsp00901.jpeg" target="_blank">View large</a><a section="5319564" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00901.jpeg?Expires=2147483647&Signature=BDs6qhT0ZUCeibJcIFUmng~mIk~gfES-t6AmFYNQ-KCG4~BfOIUljwVID3LxtnNIOn18kZ9oggIh0V6c9rGllZFLu52gEJd2YNP8emmrlXeBQjHh5sGm3N19j0sjq~VwMX~aGTmguO64qwsvnfbi6z9UFX~qmhS1lCXpVh0s9lNiWkbEuEo6irOV7SMUgbQlyJeghuugMqdd7QhiNuNuDxPxp4ySMCxDp8qwThrGcjw5Cm2-6DEB4E9XIMW98~MrMwpRU-1qlfxyAYXBSDJ3gwBE3J5c6yEx1gYVYRl~Tt6twJcHh7YoCgFMbBXjFtdx6z7FV3~SxfmBtRoJWOal6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319564&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart and outline of the classifications used for current data set (upper part) and AS events found in the novel AS variants (lower part). The gray-highlighted boxes correspond to the transcripts with novel AS events. In total, 601 transcripts were classified as novel AS variants. And 342 AS events were novel ones found in the current study.</p></div></div><div content-id="DSP009F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00901.jpeg?Expires=2147483647&Signature=uy2j9CD~2K7zQir03yE-0prjqqiPLpleoLbsALDSCBM4PseA-GWRsO4jXpSrFyJeUKe0gY0l3K7~Tsj4TlEppoIbD-ct1SZ1b6HRnqlTJ22AMEv9OnZOiSWprghSxJketK375M7BjKXSyYQ0TXdyYIJTsgdS9-Wl-KhaAQbZb2dbqoG1NFlgaJH30gXCuUbSgx51AujqxrI5re11qS7evR3XPm9t0whaomwFVCaYxMzHNMZJurGrzPKKmrQu14vSBwWW6ywxZ9xAYA93KH02cZNPP0Y6ygQRzsepIqmaynGGU48s1rtNfuu6fGVRRQUmD7RxUdCXxOtdFT9U0PYE0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart and outline of the classifications used for current data set (upper part) and AS events found in the novel AS variants (lower part). The gray-highlighted boxes correspond to the transcripts with novel AS events. In total, 601 transcripts were classified as novel AS variants. And 342 AS events were novel ones found in the current study." path-from-xml="dsp00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319564/dsp00901.jpeg" path-from-xml="dsp00901.jpeg" target="_blank">View large</a><a section="5319564" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00901.jpeg?Expires=2147483647&Signature=BDs6qhT0ZUCeibJcIFUmng~mIk~gfES-t6AmFYNQ-KCG4~BfOIUljwVID3LxtnNIOn18kZ9oggIh0V6c9rGllZFLu52gEJd2YNP8emmrlXeBQjHh5sGm3N19j0sjq~VwMX~aGTmguO64qwsvnfbi6z9UFX~qmhS1lCXpVh0s9lNiWkbEuEo6irOV7SMUgbQlyJeghuugMqdd7QhiNuNuDxPxp4ySMCxDp8qwThrGcjw5Cm2-6DEB4E9XIMW98~MrMwpRU-1qlfxyAYXBSDJ3gwBE3J5c6yEx1gYVYRl~Tt6twJcHh7YoCgFMbBXjFtdx6z7FV3~SxfmBtRoJWOal6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319564&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart and outline of the classifications used for current data set (upper part) and AS events found in the novel AS variants (lower part). The gray-highlighted boxes correspond to the transcripts with novel AS events. In total, 601 transcripts were classified as novel AS variants. And 342 AS events were novel ones found in the current study.</p></div></div>                            <h3 scrollto-destination=5319565 id="5319565" class="section-title" data-legacy-id=s3b>3.2. Classification of transcripts as known or novel</h3>
<p>Out of the 1745 mapped transcripts, three lacked corresponding annotated genes in TAIR. They were classified as novel TUs. Another 14 transcripts were chimeric transcripts that were produced from two adjacent genes. In humans, a large number of such chimeric transcripts have been reported.<sup><a reveal-id="DSP009C3" data-open="DSP009C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSP009C20" data-open="DSP009C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP009C21" data-open="DSP009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The current data set also showed that <em>Arabidopsis</em> had some chimeric transcripts. A total of 1129 transcripts had corresponding gene models in the annotated gene set (Fig. <a reveal-id="DSP009F1" data-open="DSP009F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Six transcripts had additional exons compared with the annotated gene models in TAIR. The remaining 597 transcripts had novel AS events and/or alternative structure variants (as a whole we called them AS variants). Out of them, 572 transcripts contained novel AS events that had been unknown. The others were AS variants that consisted of novel combinations of known AS events. In addition to these 597 novel AS variants, four chimeric transcripts had novel exon–intron structures in comparison with annotated ones. In total, 601 transcripts were classified as novel alternatively spliced/structure variant transcripts (Fig. <a reveal-id="DSP009F1" data-open="DSP009F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). AS events found in the 601 transcripts were summarized in Table <a reveal-id="DSP009TB1" data-open="DSP009TB1" class="link link-reveal link-table xref-fig">1</a>. We checked relative frequencies of each AS event within all AS events found in the novel transcripts and the novel AS events only within the transcripts. In both cases, the frequencies of AS events were similar to the previous studies;<sup><a reveal-id="DSP009C1" data-open="DSP009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> RI events were the most abundant ones and AA events were the next.</p>                        <a id="5319567" scrollto-destination="5319567"></a>
<div content-id="DSP009TB1" class="table-modal"><div class="table-wrap"><div id="DSP009TB1" class="table-wrap-title" data-id="DSP009TB1"><div class="label -label">Table 1</div><div class="caption"><p>The results of AS and alternative structure event prediction</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Event type<sup>a</sup> </th><th colspan="2">Total events in novel transcripts<hr /> </th><th colspan="2">Novel events in novel transcripts<hr /> </th></tr><tr><th> </th><th>Number of AS/structure events </th><th>Ratio </th><th>Number of AS/structure events </th><th>Ratio </th></tr></thead><tbody><tr><td>RI </td><td>610 </td><td>0.45 </td><td>479 </td><td>0.36 </td></tr><tr><td>AA </td><td>279 </td><td>0.21 </td><td>178 </td><td>0.13 </td></tr><tr><td>ES </td><td>185 </td><td>0.14 </td><td>143 </td><td>0.11 </td></tr><tr><td>AD </td><td>161 </td><td>0.12 </td><td>111 </td><td>0.08 </td></tr><tr><td>AI </td><td>66 </td><td>0.05 </td><td>39 </td><td>0.03 </td></tr><tr><td>AT </td><td>48 </td><td>0.04 </td><td>30 </td><td>0.02 </td></tr><tr><td>Total </td><td>1349 </td><td> </td><td>980 </td><td> </td></tr></tbody></table></div><div content-id="DSP009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Event type<sup>a</sup> </th><th colspan="2">Total events in novel transcripts<hr /> </th><th colspan="2">Novel events in novel transcripts<hr /> </th></tr><tr><th> </th><th>Number of AS/structure events </th><th>Ratio </th><th>Number of AS/structure events </th><th>Ratio </th></tr></thead><tbody><tr><td>RI </td><td>610 </td><td>0.45 </td><td>479 </td><td>0.36 </td></tr><tr><td>AA </td><td>279 </td><td>0.21 </td><td>178 </td><td>0.13 </td></tr><tr><td>ES </td><td>185 </td><td>0.14 </td><td>143 </td><td>0.11 </td></tr><tr><td>AD </td><td>161 </td><td>0.12 </td><td>111 </td><td>0.08 </td></tr><tr><td>AI </td><td>66 </td><td>0.05 </td><td>39 </td><td>0.03 </td></tr><tr><td>AT </td><td>48 </td><td>0.04 </td><td>30 </td><td>0.02 </td></tr><tr><td>Total </td><td>1349 </td><td> </td><td>980 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>RI, retained intron; AA, alternative acceptor; ES, exon skip; AD, alternative donor; AI, alternative initiation of transcription; AT, alternative termination of transcription. AI and AT, the alternative structure events were checked for the classification of the transcripts.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5319567" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP009TB1" class="table-wrap-title" data-id="DSP009TB1"><div class="label -label">Table 1</div><div class="caption"><p>The results of AS and alternative structure event prediction</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Event type<sup>a</sup> </th><th colspan="2">Total events in novel transcripts<hr /> </th><th colspan="2">Novel events in novel transcripts<hr /> </th></tr><tr><th> </th><th>Number of AS/structure events </th><th>Ratio </th><th>Number of AS/structure events </th><th>Ratio </th></tr></thead><tbody><tr><td>RI </td><td>610 </td><td>0.45 </td><td>479 </td><td>0.36 </td></tr><tr><td>AA </td><td>279 </td><td>0.21 </td><td>178 </td><td>0.13 </td></tr><tr><td>ES </td><td>185 </td><td>0.14 </td><td>143 </td><td>0.11 </td></tr><tr><td>AD </td><td>161 </td><td>0.12 </td><td>111 </td><td>0.08 </td></tr><tr><td>AI </td><td>66 </td><td>0.05 </td><td>39 </td><td>0.03 </td></tr><tr><td>AT </td><td>48 </td><td>0.04 </td><td>30 </td><td>0.02 </td></tr><tr><td>Total </td><td>1349 </td><td> </td><td>980 </td><td> </td></tr></tbody></table></div><div content-id="DSP009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Event type<sup>a</sup> </th><th colspan="2">Total events in novel transcripts<hr /> </th><th colspan="2">Novel events in novel transcripts<hr /> </th></tr><tr><th> </th><th>Number of AS/structure events </th><th>Ratio </th><th>Number of AS/structure events </th><th>Ratio </th></tr></thead><tbody><tr><td>RI </td><td>610 </td><td>0.45 </td><td>479 </td><td>0.36 </td></tr><tr><td>AA </td><td>279 </td><td>0.21 </td><td>178 </td><td>0.13 </td></tr><tr><td>ES </td><td>185 </td><td>0.14 </td><td>143 </td><td>0.11 </td></tr><tr><td>AD </td><td>161 </td><td>0.12 </td><td>111 </td><td>0.08 </td></tr><tr><td>AI </td><td>66 </td><td>0.05 </td><td>39 </td><td>0.03 </td></tr><tr><td>AT </td><td>48 </td><td>0.04 </td><td>30 </td><td>0.02 </td></tr><tr><td>Total </td><td>1349 </td><td> </td><td>980 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>RI, retained intron; AA, alternative acceptor; ES, exon skip; AD, alternative donor; AI, alternative initiation of transcription; AT, alternative termination of transcription. AI and AT, the alternative structure events were checked for the classification of the transcripts.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5319567" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5319568 id="5319568" class="section-title" data-legacy-id=s3c>3.3. Analysis of the number of AS events per gene</h3>
<p>Six hundred and one genes were classified as novel AS variants, which was unexpected because we chose full-length cDNA clones that already had AS events in their sequenced 5′/3′ regions. These 5′/3′ sequences and AS events have already been registered to the GenBank database.<sup><a reveal-id="DSP009C7" data-open="DSP009C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSP009C12" data-open="DSP009C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP009C13" data-open="DSP009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Since the TAIR group makes efforts to continuously update the gene models, including the AS information,<sup><a reveal-id="DSP009C18" data-open="DSP009C18" class="link link-ref link-reveal xref-bibr">18</a></sup> it was expected that the AS events in previously sequenced regions would have been included in TAIR's annotation and classified as known AS events. Thus, we thought that the 601 novel AS variants must contain additional AS events in their novel sequence regions. We compared the regions containing the AS events with the regions that were sequenced previously (as 5′/3′ sequences). Out of the 980 AS events without corresponding events in TAIR, 638 AS events were present in the previous 5′/3′ sequences (Fig. <a reveal-id="DSP009F1" data-open="DSP009F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). We concluded that the main reason why we found so many novel AS events was that some of the AS events found in the previous 5′/3′ sequences were not reflected in TAIR's annotation. It might be due to TAIR's policy in the database update to exclude alternatively spliced mRNAs with premature termination codons.<sup><a reveal-id="DSP009C18" data-open="DSP009C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Despite that we found many AS events corresponded with the previously sequenced partial sequences, we still found 342 novel AS events in the newly sequenced regions. This result showed that transcripts with AS events tended to have additional AS events in other positions, suggesting that the number of AS events in a single gene should be much greater than what is currently known. Next, we compared the number of AS events per gene between the current data set and TAIR. The average number of AS events per gene was found to be 2.0 in the TUs that corresponded to the current transcripts. On the other hand, it was 1.4 for the genes that had more than two mature mRNA structures in TAIR's annotation. In the current data set, a higher fraction of genes had more than two AS events compared with the genes in TAIR's data set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). These differences were statistically significant (<em>P</em> &lt; 2.2E−16) with a Mann–Whitney test. Therefore, we concluded that <em>Arabidopsis</em> genes should have more AS events per gene. In the current analysis, a further sequencing effort showed many more AS events in <em>Arabidopsis</em>. This finding was consistent with a previous report, in which the frequency of AS events is correlated with the abundance of EST/full-length sequences.<sup><a reveal-id="DSP009C22" data-open="DSP009C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Further sequencing should show that more AS events occur in <em>Arabidopsis</em>.</p>                            <h3 scrollto-destination=5319571 id="5319571" class="section-title" data-legacy-id=s3d>3.4. Analysis of the pattern of AS event combinations in each gene</h3>
<p>In the current study of AS events, specific patterns of AS event combinations preferentially occurred. To confirm this result, we focused on transcripts that had more than two AS events. Among the cDNAs characterized in this study, 354 transcripts had more than two AS events. The number of combinations between adjacent AS events was examined and compared with expectation values that were calculated from the frequencies of every AS event type found in the 354 transcripts (Table <a reveal-id="DSP009TB2" data-open="DSP009TB2" class="link link-reveal link-table xref-fig">2</a>). The frequencies of the individual AS event types were similar between the 601 novel AS variants and the 354 AS variants with multiple AS events (Tables <a reveal-id="DSP009TB1" data-open="DSP009TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSP009TB2" data-open="DSP009TB2" class="link link-reveal link-table xref-fig">2</a>). The four combinations (AD–ES, AD–AA, ES–ES and ES–AA) were observed more frequently than expected, which was statistically significant (<em>P</em> &lt; 0.01 in a binomial test; Table <a reveal-id="DSP009TB2" data-open="DSP009TB2" class="link link-reveal link-table xref-fig">2</a>). The combination of RI–RI was also observed more frequently than expected, although this was not statistically significant (<em>P</em> = 0.106 in a binomial test). The prominence of these five combinations suggests the following two models for AS patterns: combinations of AD–ES (0∼12 times)–AA and multiple repeats of RI (2∼8 times) events. Out of our 601 novel AS variants, the former model was found in 80 transcripts and the latter was found in 125 transcripts (examples in Fig. <a reveal-id="DSP009F2" data-open="DSP009F2" class="link link-reveal link-table xref-fig">2</a>). In a previous study, Wang and Brendel<sup><a reveal-id="DSP009C1" data-open="DSP009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> analyzed AS events in <em>Arabidopsis</em> and rice and concluded that most AS events were mutually independent. The reason why their conclusions and ours were different might be due to that we analyzed the sequences of full-length cDNAs in the current studies, whereas Wang and Brendel treated the data sets including a vast amount of partial read sequences. Analysis of full-length cDNAs showed possibilities that some AS events in a transcript were related each other.</p>                        <a id="5319573" scrollto-destination="5319573"></a>
<div id="DSP009F2" data-id="DSP009F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00902.jpeg?Expires=2147483647&Signature=KNhyeNCKO2mKWGaB9DyJpPDd~NQ4FnjBuZ48hh3sI5CWJaKD5Zk9zNQsfYCnv6QwBs5-GHKejKJ-tTxZpao7hpVxpbf1N3KTP9pM7XMBpOPIBScSSl7o0bM1SwP6qPpN-jHBfJmV9alPW50VWOt~EkqRhdS6ElSIZorSSJtipsV7PmnvibR5rMah0dkvilI0DGT~KHwy8FcvDvebq3BODhqmhALUXmLa~loTVmtG4G5K0L1T-0CXUnNFHwMhmBgMq34lMic66nFcPo20gWB7OPBHpQ86d5A6GF3uD-YiXFFGxccYLW4yq1BhWzTJXoqSb~Ly12cwYoNmFxpM2oxJPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of multiple occurrences of AS events. (A) The AT2G02910 locus which encodes a predicted hydrolase that acts on carbon–nitrogen bonds. The transcript (MAS-001G06) retains introns 1, 2, 5 and 8 compared with AT2G02910.1. (B) The AT3G07300 locus which encodes a protein from the eukaryotic translation initiation factor 2B family. The transcript (MAS-004E14) shows an AD, two ES and an AA AS events. All four of these events were of the ‘out-type’." path-from-xml="dsp00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319573/dsp00902.jpeg" path-from-xml="dsp00902.jpeg" target="_blank">View large</a><a section="5319573" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00902.jpeg?Expires=2147483647&Signature=rGJrOgxsz1g9Aa75cK~9CFWnibqaD3Xj-QveFw3gM8Ce2V0t1NhUz5PxArCREd5Ff9eCVkgNz6Gc90dgyd37C281xL5TFSg01TSeck9d2QjUOvpZvwguEknk8ZQ6Pl7D68wiYb0829zgKMfFHkK50uhB85MLUSQkMFiZSnYSDvUfMf0agR0dzsNu88Qi6UEInYibXRot9-DYa2raCpr-tawEAKgU0~WO7z~CPLU5NWbLdbGb-QewLBZw6BPIG0uXKC3EHeFKjxrTiW~awu9UOd588tv1jkNTjHgC2SK~LLMHOcLKTDzIJllLdm63RVMs97G3aammJNARbJYtomUKpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319573&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of multiple occurrences of AS events. (<strong>A</strong>) The AT2G02910 locus which encodes a predicted hydrolase that acts on carbon–nitrogen bonds. The transcript (MAS-001G06) retains introns 1, 2, 5 and 8 compared with AT2G02910.1. (<strong>B</strong>) The AT3G07300 locus which encodes a protein from the eukaryotic translation initiation factor 2B family. The transcript (MAS-004E14) shows an AD, two ES and an AA AS events. All four of these events were of the ‘out-type’.</p></div></div><div content-id="DSP009F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00902.jpeg?Expires=2147483647&Signature=KNhyeNCKO2mKWGaB9DyJpPDd~NQ4FnjBuZ48hh3sI5CWJaKD5Zk9zNQsfYCnv6QwBs5-GHKejKJ-tTxZpao7hpVxpbf1N3KTP9pM7XMBpOPIBScSSl7o0bM1SwP6qPpN-jHBfJmV9alPW50VWOt~EkqRhdS6ElSIZorSSJtipsV7PmnvibR5rMah0dkvilI0DGT~KHwy8FcvDvebq3BODhqmhALUXmLa~loTVmtG4G5K0L1T-0CXUnNFHwMhmBgMq34lMic66nFcPo20gWB7OPBHpQ86d5A6GF3uD-YiXFFGxccYLW4yq1BhWzTJXoqSb~Ly12cwYoNmFxpM2oxJPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of multiple occurrences of AS events. (A) The AT2G02910 locus which encodes a predicted hydrolase that acts on carbon–nitrogen bonds. The transcript (MAS-001G06) retains introns 1, 2, 5 and 8 compared with AT2G02910.1. (B) The AT3G07300 locus which encodes a protein from the eukaryotic translation initiation factor 2B family. The transcript (MAS-004E14) shows an AD, two ES and an AA AS events. All four of these events were of the ‘out-type’." path-from-xml="dsp00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319573/dsp00902.jpeg" path-from-xml="dsp00902.jpeg" target="_blank">View large</a><a section="5319573" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00902.jpeg?Expires=2147483647&Signature=rGJrOgxsz1g9Aa75cK~9CFWnibqaD3Xj-QveFw3gM8Ce2V0t1NhUz5PxArCREd5Ff9eCVkgNz6Gc90dgyd37C281xL5TFSg01TSeck9d2QjUOvpZvwguEknk8ZQ6Pl7D68wiYb0829zgKMfFHkK50uhB85MLUSQkMFiZSnYSDvUfMf0agR0dzsNu88Qi6UEInYibXRot9-DYa2raCpr-tawEAKgU0~WO7z~CPLU5NWbLdbGb-QewLBZw6BPIG0uXKC3EHeFKjxrTiW~awu9UOd588tv1jkNTjHgC2SK~LLMHOcLKTDzIJllLdm63RVMs97G3aammJNARbJYtomUKpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319573&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of multiple occurrences of AS events. (<strong>A</strong>) The AT2G02910 locus which encodes a predicted hydrolase that acts on carbon–nitrogen bonds. The transcript (MAS-001G06) retains introns 1, 2, 5 and 8 compared with AT2G02910.1. (<strong>B</strong>) The AT3G07300 locus which encodes a protein from the eukaryotic translation initiation factor 2B family. The transcript (MAS-004E14) shows an AD, two ES and an AA AS events. All four of these events were of the ‘out-type’.</p></div></div>                        <a id="5319574" scrollto-destination="5319574"></a>
<div content-id="DSP009TB2" class="table-modal"><div class="table-wrap"><div id="DSP009TB2" class="table-wrap-title" data-id="DSP009TB2"><div class="label -label">Table 2</div><div class="caption"><p>Analysis of the combination of two adjacent AS events</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">First AS events<hr /> </th><th> </th><th colspan="4">Second AS events<hr /> </th></tr><tr><th> </th><th>Number of the AS events<sup>a</sup> </th><th>Frequency<sup>a</sup> </th><th> </th><th>RI </th><th>AA </th><th>ES </th><th>AD </th></tr></thead><tbody><tr><td>RI </td><td>509 </td><td>0.45 </td><td>Expectation value<sup>b</sup> </td><td>156.5 </td><td>82.1 </td><td>58.1 </td><td>54.4 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>221 </td><td>71 </td><td>12 </td><td>25 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>0.106 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AA </td><td>267 </td><td>0.23 </td><td>Expectation value<sup>b</sup> </td><td>82.1 </td><td>43.1 </td><td>30.5 </td><td>28.6 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>60 </td><td>42 </td><td>11 </td><td>21 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>ES </td><td>189 </td><td>0.17 </td><td>Expectation value<sup>b</sup> </td><td>58.1 </td><td>30.5 </td><td>21.6 </td><td>20.2 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>14 </td><td>59 </td><td>85 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>2.30E−05 </td><td>&lt;2.2E −16 </td><td>— </td></tr><tr><td>AD </td><td>177 </td><td>0.15 </td><td>Expectation value<sup>b</sup> </td><td>54.4 </td><td>28.6 </td><td>20.2 </td><td>18.9 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>53 </td><td>45 </td><td>44 </td><td>12 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>0.009 </td><td>1.32E−05 </td><td>— </td></tr></tbody></table></div><div content-id="DSP009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">First AS events<hr /> </th><th> </th><th colspan="4">Second AS events<hr /> </th></tr><tr><th> </th><th>Number of the AS events<sup>a</sup> </th><th>Frequency<sup>a</sup> </th><th> </th><th>RI </th><th>AA </th><th>ES </th><th>AD </th></tr></thead><tbody><tr><td>RI </td><td>509 </td><td>0.45 </td><td>Expectation value<sup>b</sup> </td><td>156.5 </td><td>82.1 </td><td>58.1 </td><td>54.4 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>221 </td><td>71 </td><td>12 </td><td>25 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>0.106 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AA </td><td>267 </td><td>0.23 </td><td>Expectation value<sup>b</sup> </td><td>82.1 </td><td>43.1 </td><td>30.5 </td><td>28.6 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>60 </td><td>42 </td><td>11 </td><td>21 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>ES </td><td>189 </td><td>0.17 </td><td>Expectation value<sup>b</sup> </td><td>58.1 </td><td>30.5 </td><td>21.6 </td><td>20.2 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>14 </td><td>59 </td><td>85 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>2.30E−05 </td><td>&lt;2.2E −16 </td><td>— </td></tr><tr><td>AD </td><td>177 </td><td>0.15 </td><td>Expectation value<sup>b</sup> </td><td>54.4 </td><td>28.6 </td><td>20.2 </td><td>18.9 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>53 </td><td>45 </td><td>44 </td><td>12 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>0.009 </td><td>1.32E−05 </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>These values were counted as novel AS variants with more than two AS events.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Expectation values. Calculated based on the frequencies of independent events. Case number is the observation number: 788.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Results of the binomial test. <em>P</em>-values are shown if the observation values are greater than the expectation values.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5319574" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP009TB2" class="table-wrap-title" data-id="DSP009TB2"><div class="label -label">Table 2</div><div class="caption"><p>Analysis of the combination of two adjacent AS events</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">First AS events<hr /> </th><th> </th><th colspan="4">Second AS events<hr /> </th></tr><tr><th> </th><th>Number of the AS events<sup>a</sup> </th><th>Frequency<sup>a</sup> </th><th> </th><th>RI </th><th>AA </th><th>ES </th><th>AD </th></tr></thead><tbody><tr><td>RI </td><td>509 </td><td>0.45 </td><td>Expectation value<sup>b</sup> </td><td>156.5 </td><td>82.1 </td><td>58.1 </td><td>54.4 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>221 </td><td>71 </td><td>12 </td><td>25 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>0.106 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AA </td><td>267 </td><td>0.23 </td><td>Expectation value<sup>b</sup> </td><td>82.1 </td><td>43.1 </td><td>30.5 </td><td>28.6 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>60 </td><td>42 </td><td>11 </td><td>21 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>ES </td><td>189 </td><td>0.17 </td><td>Expectation value<sup>b</sup> </td><td>58.1 </td><td>30.5 </td><td>21.6 </td><td>20.2 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>14 </td><td>59 </td><td>85 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>2.30E−05 </td><td>&lt;2.2E −16 </td><td>— </td></tr><tr><td>AD </td><td>177 </td><td>0.15 </td><td>Expectation value<sup>b</sup> </td><td>54.4 </td><td>28.6 </td><td>20.2 </td><td>18.9 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>53 </td><td>45 </td><td>44 </td><td>12 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>0.009 </td><td>1.32E−05 </td><td>— </td></tr></tbody></table></div><div content-id="DSP009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">First AS events<hr /> </th><th> </th><th colspan="4">Second AS events<hr /> </th></tr><tr><th> </th><th>Number of the AS events<sup>a</sup> </th><th>Frequency<sup>a</sup> </th><th> </th><th>RI </th><th>AA </th><th>ES </th><th>AD </th></tr></thead><tbody><tr><td>RI </td><td>509 </td><td>0.45 </td><td>Expectation value<sup>b</sup> </td><td>156.5 </td><td>82.1 </td><td>58.1 </td><td>54.4 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>221 </td><td>71 </td><td>12 </td><td>25 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>0.106 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AA </td><td>267 </td><td>0.23 </td><td>Expectation value<sup>b</sup> </td><td>82.1 </td><td>43.1 </td><td>30.5 </td><td>28.6 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>60 </td><td>42 </td><td>11 </td><td>21 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>ES </td><td>189 </td><td>0.17 </td><td>Expectation value<sup>b</sup> </td><td>58.1 </td><td>30.5 </td><td>21.6 </td><td>20.2 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>14 </td><td>59 </td><td>85 </td><td>13 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>2.30E−05 </td><td>&lt;2.2E −16 </td><td>— </td></tr><tr><td>AD </td><td>177 </td><td>0.15 </td><td>Expectation value<sup>b</sup> </td><td>54.4 </td><td>28.6 </td><td>20.2 </td><td>18.9 </td></tr><tr><td> </td><td> </td><td> </td><td>Observation value </td><td>53 </td><td>45 </td><td>44 </td><td>12 </td></tr><tr><td> </td><td> </td><td> </td><td><em>P</em>-value<sup>c</sup> </td><td>— </td><td>0.009 </td><td>1.32E−05 </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>These values were counted as novel AS variants with more than two AS events.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Expectation values. Calculated based on the frequencies of independent events. Case number is the observation number: 788.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Results of the binomial test. <em>P</em>-values are shown if the observation values are greater than the expectation values.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5319574" target="_blank">View Large</a></div></div><p>It should be noted that 26 out of 125 transcripts with multiple RI events consisted of single exons. Most of these transcripts seemed to be mature mRNAs, because 21 out of 26 had clear poly-A tails. However, it was still possible that these transcripts without intron splicing were immature mRNAs, and therefore they were excluded. In this case, the probability of RI events was 0.40. As a result, the number of RI–RI combinations was 157, which was still greater than the expectation value (114.9). Fig. <a reveal-id="DSP009F2" data-open="DSP009F2" class="link link-reveal link-table xref-fig">2</a>B shows an example of a combination pattern in which multiple, but not all, introns were retained in a single transcript. Thus, the model of multiple repeats of RI events was also enriched in the current data set.</p><p>We also classified the AS events by their direction, as in- and out-type events (see Section 2 for details). The direction of AS events showed that for two adjacent AS events, a single direction was preferable (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). This was clear when the adjacent events consisted of the same type of AS event (e.g. ES–ES or RI–RI, which was frequently observed in the analysis of the patterns of AS event combinations). This observation was supported by a chi-squared test with a <em>P</em>-value of &lt;2.2E−16. Interestingly, this trend was still statistically significant in cases of combinations that were made up of two different AS events (<em>P</em> = 1.7E−07 in a chi-squared test; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>For further evidence, we analyzed all public full-length cDNAs of <em>Arabidopsis</em> and rice, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S4</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsp009/DC1" target="_blank">S5</a>). Both results were similar to those of current study concerning the combination of AS events types and directions of the AS events. Thus, we concluded that transcriptome of land plants had preferred combinations of AS event types and directions.</p>                            <h3 scrollto-destination=5319578 id="5319578" class="section-title" data-legacy-id=s3e>3.5. Potential AS regulatory mechanisms</h3>
<p>In the first step of intron splicing in plants, it is thought that the splicing machinery recognizes the intron sequences. In contrast, exon sequences are recognized in mammals.<sup><a reveal-id="DSP009C23" data-open="DSP009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The present study demonstrates that specific patterns of combinations and directions of AS events are preferred in <em>Arabidopsis</em>. Our results are consistent with a model in which the splicing machinery recognizes intronic sequences.<sup><a reveal-id="DSP009C23" data-open="DSP009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> If <em>trans</em>-acting factors, which recognize weak splicing enhancers in introns (called intronic splicing enhancers, ISEs<sup><a reveal-id="DSP009C24" data-open="DSP009C24" class="link link-ref link-reveal xref-bibr">24</a></sup>), are abundant in the nucleus, then the recognized regions are spliced out (Fig. <a reveal-id="DSP009F3" data-open="DSP009F3" class="link link-reveal link-table xref-fig">3</a>). This model was similar to the previous one which had explained abundant RI events in plants.<sup><a reveal-id="DSP009C1" data-open="DSP009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, we thought that this model could also explain the existence of another special type of combination of AS events; AD–ES–AA (Fig. <a reveal-id="DSP009F3" data-open="DSP009F3" class="link link-reveal link-table xref-fig">3</a>B). When some <em>trans</em> factors recognize weak splicing signals which lay on close positions, one huge intron might be spliced out from a pre-mRNA.</p>                        <a id="5319580" scrollto-destination="5319580"></a>
<div id="DSP009F3" data-id="DSP009F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00903.jpeg?Expires=2147483647&Signature=ug7K96GlqV1y4I9xT4oJ~VR0RaKPmr10sNBD1S876feWINOxRaOxpAU1hQYsCMCSCm6w2mXzE9iu~rkiiPQn~MA-glU1laZfhC4TzATBIFhUCz~qAYqLB5Qlote95yTW49Jue4VNPmJWjYUWLSGhavSDIRdqFwn7aG3IeqWJmhGnUPfoh2L6J4B8ZdAms7UHGi23Fzhhi~~5ftrwHbiD9aKpSgA83Ar8g28cHraiwjqqW2~5PzJ~XnIytAZuD5C3ksg4l5GpFnee7tO~RI-U51YE2LkadS-sjFouhlyfvrZ7mPphICyllNflBP0ViROynkzSmblnqSKaXUSqsejJ6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesis regarding the regulation of multiple AS events. (A) Model for cases of multiple RI events. The circles show the putative trans-acting factors. In this model, if the trans-acting factors, which recognize weak splicing signals (gray circles), are decreased, introns with the weak signals remain in the mRNA. (B) The AS model consisting of AD–ES–AA. The gray circles show putative trans-acting factors, which recognize weak splicing signals. If the concentration of the factors increases, several regions are recognized as introns and then the large region is spliced out as a single intron. It is not known if the trans- or cis-acting factors in the model are the same." path-from-xml="dsp00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319580/dsp00903.jpeg" path-from-xml="dsp00903.jpeg" target="_blank">View large</a><a section="5319580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00903.jpeg?Expires=2147483647&Signature=2lkzw4OCIRPo4yPy7kymeQKYaUC6wYFI1jZEjNigAEG5wCyFOwk~B6jFankSsu0KiQanxLAn-5aNSD1daejr~MA8NhrFZgzuQNhYQFV0Su-Q5eTkm-NcMws2ITmJAyPQvlVgxQY0JqbwVRBs3k3Wk4mt0VvdVcIUXN4~aqOS9XwMtvgolElWpFsd-fLYV0pRlqEoBrTHcXcry3xJ3s1RCtn9gNzGn2qo6JX3pULdH3TQ5vNkOxOc2~JEsG3WJxmLhxVcfguBI1O7dQWhNZ49C6~hxRqAJNTmGO4dhbS95nEMoxCoFu3EKPxxUhaK675He-1PedPbrz~ZDn2VPt9c6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319580&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hypothesis regarding the regulation of multiple AS events. (<strong>A</strong>) Model for cases of multiple RI events. The circles show the putative <em>trans</em>-acting factors. In this model, if the <em>trans</em>-acting factors, which recognize weak splicing signals (gray circles), are decreased, introns with the weak signals remain in the mRNA. (<strong>B</strong>) The AS model consisting of AD–ES–AA. The gray circles show putative <em>trans</em>-acting factors, which recognize weak splicing signals. If the concentration of the factors increases, several regions are recognized as introns and then the large region is spliced out as a single intron. It is not known if the <em>trans</em>- or <em>cis</em>-acting factors in the model are the same.</p></div></div><div content-id="DSP009F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00903.jpeg?Expires=2147483647&Signature=ug7K96GlqV1y4I9xT4oJ~VR0RaKPmr10sNBD1S876feWINOxRaOxpAU1hQYsCMCSCm6w2mXzE9iu~rkiiPQn~MA-glU1laZfhC4TzATBIFhUCz~qAYqLB5Qlote95yTW49Jue4VNPmJWjYUWLSGhavSDIRdqFwn7aG3IeqWJmhGnUPfoh2L6J4B8ZdAms7UHGi23Fzhhi~~5ftrwHbiD9aKpSgA83Ar8g28cHraiwjqqW2~5PzJ~XnIytAZuD5C3ksg4l5GpFnee7tO~RI-U51YE2LkadS-sjFouhlyfvrZ7mPphICyllNflBP0ViROynkzSmblnqSKaXUSqsejJ6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesis regarding the regulation of multiple AS events. (A) Model for cases of multiple RI events. The circles show the putative trans-acting factors. In this model, if the trans-acting factors, which recognize weak splicing signals (gray circles), are decreased, introns with the weak signals remain in the mRNA. (B) The AS model consisting of AD–ES–AA. The gray circles show putative trans-acting factors, which recognize weak splicing signals. If the concentration of the factors increases, several regions are recognized as introns and then the large region is spliced out as a single intron. It is not known if the trans- or cis-acting factors in the model are the same." path-from-xml="dsp00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319580/dsp00903.jpeg" path-from-xml="dsp00903.jpeg" target="_blank">View large</a><a section="5319580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00903.jpeg?Expires=2147483647&Signature=2lkzw4OCIRPo4yPy7kymeQKYaUC6wYFI1jZEjNigAEG5wCyFOwk~B6jFankSsu0KiQanxLAn-5aNSD1daejr~MA8NhrFZgzuQNhYQFV0Su-Q5eTkm-NcMws2ITmJAyPQvlVgxQY0JqbwVRBs3k3Wk4mt0VvdVcIUXN4~aqOS9XwMtvgolElWpFsd-fLYV0pRlqEoBrTHcXcry3xJ3s1RCtn9gNzGn2qo6JX3pULdH3TQ5vNkOxOc2~JEsG3WJxmLhxVcfguBI1O7dQWhNZ49C6~hxRqAJNTmGO4dhbS95nEMoxCoFu3EKPxxUhaK675He-1PedPbrz~ZDn2VPt9c6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319580&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hypothesis regarding the regulation of multiple AS events. (<strong>A</strong>) Model for cases of multiple RI events. The circles show the putative <em>trans</em>-acting factors. In this model, if the <em>trans</em>-acting factors, which recognize weak splicing signals (gray circles), are decreased, introns with the weak signals remain in the mRNA. (<strong>B</strong>) The AS model consisting of AD–ES–AA. The gray circles show putative <em>trans</em>-acting factors, which recognize weak splicing signals. If the concentration of the factors increases, several regions are recognized as introns and then the large region is spliced out as a single intron. It is not known if the <em>trans</em>- or <em>cis</em>-acting factors in the model are the same.</p></div></div><p>It is unclear whether or not the same <em>trans</em>-acting factors recognize both weak ISEs and constitutive splicing signals. However, some SR proteins are candidate <em>trans</em>-acting factors in the proposed model. SR proteins are splicing factors that are involved in constitutive and AS.<sup><a reveal-id="DSP009C25" data-open="DSP009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> It is known that the regulation of many SR proteins is complicated and can vary by tissue and stress conditions.<sup><a reveal-id="DSP009C6" data-open="DSP009C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP009C10" data-open="DSP009C10" class="link link-ref link-reveal xref-bibr">10</a></sup> It has also been noted that some SR proteins can recognize ISEs.<sup><a reveal-id="DSP009C24" data-open="DSP009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> On the basis of these evidences, it may be possible that SR proteins work as <em>trans</em>-acting factors. Future work must focus on elucidating the ISEs and <em>trans</em>-acting factors in this model.</p><p>It should also be noted out that our current results are consistent with our previous work on AS profiles. In the previous work, we found a tendency to select cassette exons, retain introns and select AD and acceptor sites to make longer mRNAs in plants during conditions of cold stress.<sup><a reveal-id="DSP009C7" data-open="DSP009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> All these tendencies could be explained by the fact that in-type events are more frequent during cold stress. In the current analysis, we had 32 transcripts with 111 AS events from a library of cold-treated plants. Out of the 111 AS events, 77 (69%) were in-type AS events. In 324 transcripts with multiple AS events, 569 of 1142 (50%) AS events were in-type events. These differences were statistically significant by a chi-squared test (<em>P</em> &lt; 0.001). It may be possible that one or more <em>trans</em>-acting factors that recognize weak ISEs are down-regulated in response to cold stress. Interestingly, microarray analysis<sup><a reveal-id="DSP009C26" data-open="DSP009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> showed that expressions of many SR protein genes largely changed under cold stress conditions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Such regulation on SR proteins might change AS profiles in a large scale.</p>                            <h3 scrollto-destination=5319583 id="5319583" class="section-title" data-legacy-id=s3f>3.6. Analysis of the motifs found in the CDSs of AS variants</h3>
<p>We also analyzed the CDSs encoded in the 601 novel AS variants. Of these, 52 transcripts had no clear CDS and 476 had shorter CDSs compared with the corresponding annotated gene models in TAIR. Out of the 476 transcripts with short CDSs, 96 transcripts had introns in their 3′ UTRs. Including them, we found 112 transcripts with introns in their 3′ UTRs from the data set of the 601 novel AS variants. Similar to mammals, it was reported that nonsense-mediated mRNA decay (NMD) mechanism exists in <em>Arabidopsis</em>. The NMD mechanism takes transcripts that have exon junctions downstream of the termination codon and causes them to decay.<sup><a reveal-id="DSP009C27" data-open="DSP009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Although we may not know the actual CDS encoded by the transcripts, the number of transcripts with 3′ UTRs that contain introns appears to be large. In addition, comparison of functional motifs between the CDSs of AS variants and their reference proteins showed that most AS variants encoded proteins that lacked some functional domains. For 460 out of 549 genes that provided the novel AS variants, we could assign at least one functional domain for the CDS encoded in the AS variant and/or its reference transcript (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). In the protein sequences encoded by the novel AS variants, 392 (85%) had a reduced number of functional domains in comparison with their reference protein sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Only four transcripts had additional motifs when compared with the reference protein sequences. These results seem to show that most AS events decrease the functionality of protein products. We hypothesize that many AS events regulate the amount of functional protein by making mRNAs that are targeted by the NMD or by making mRNAs that encode non-functional proteins. This view is consistent with our previous results from a study on SR proteins in <em>Arabidopsis</em>, rice and moss.<sup><a reveal-id="DSP009C28" data-open="DSP009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In this case, three independent AS events of mRNAs encoding SR proteins with incomplete RNA-binding domains were highly conserved in land plants. Similar models have also been proposed for AS events in animals.<sup><a reveal-id="DSP009C29" data-open="DSP009C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSP009C30" data-open="DSP009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Our current results support the existence of a similar regulatory system in <em>Arabidopsis</em>.</p><p>In the same time, some AS events possibly modulated functions of proteins with multiple domains. A locus AT1G09640 encodes a protein with glutathione <em>S</em>-transferase (GST) domains and elongation factor (EF) 1 gamma domain (Fig. <a reveal-id="DSP009F4" data-open="DSP009F4" class="link link-reveal link-table xref-fig">4</a>A). It was thought that GST activities regulated assembly of a complex containing EF1 subunits and aminoacyl-tRNA synthetases.<sup><a reveal-id="DSP009C31" data-open="DSP009C31" class="link link-ref link-reveal xref-bibr">31</a></sup> An AS variant encoded a protein with GST domains but not EF1 gamma domain (Fig. <a reveal-id="DSP009F4" data-open="DSP009F4" class="link link-reveal link-table xref-fig">4</a>A), which might not be a member of the complexes or have some regulatory roles on assembly of the complex. Another example was found in the locus AT1G10290 encoding dynamin-like protein, ADL6. It was reported that ADL6 had a function in vesicle trafficking from the <em>trans</em>-Golgi network to the central vacuole.<sup><a reveal-id="DSP009C32" data-open="DSP009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> This protein has dynamin domains followed by pleckstrin homology domain which was required for interaction between the protein and lipid<sup><a reveal-id="DSP009C32" data-open="DSP009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> (Fig. <a reveal-id="DSP009F4" data-open="DSP009F4" class="link link-reveal link-table xref-fig">4</a>B). An AS variant encodes a protein without dynamin domains, which should lack dynamin activities but may have activities of interaction with lipids. This AS isoform protein possibly affects activity of the intact proteins. These examples showed that some AS events can modulate functions of proteins especially for ones with multiple domains.</p>                        <a id="5319586" scrollto-destination="5319586"></a>
<div id="DSP009F4" data-id="DSP009F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00904.jpeg?Expires=2147483647&Signature=dSgazj1lv97~pdb1ZU6mnt3~r3X34yf-13VRGoguncVTvtXLzNk-P1ukt1VkMGLZkZeyfm1uig0RQfzckuSP9ChY40FEBi3cgYhFFWkNyVrQpKyeYAW1eX3jcupJLT4t2tAYCUhzDgyA8-OjnK2RyTry3ja8aYHOzvwYs~9a7xefDJ6bw91t5D8dHSJfoDXDxQTR0r4BmY6Kn6L~5ifxe-6qshAQvg9aTz7TmfEt8hjYnVgRNB~jRYY9wGmlovJRg2zfPsYJBu4zkF9uHwxgRDPgQwXGIglMhftobd~6KAkarJJchm0iK~QTV8CO1aqynkULBwTLbKW~uuqCG2OD8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of AS events affecting functional motifs. (A) The AT1G09640 locus which encodes a protein with GST domains and EF1 gamma domain. An AS variant MAS-002I13 encodes a protein lacking EF1 gamma domain. (B) The AT1G10290 locus which encodes dynamin-like protein ADL6. An AS variant MAS-004M08 encodes a protein lacking dynamin domains. Such AS events might modulate protein functions." path-from-xml="dsp00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319586/dsp00904.jpeg" path-from-xml="dsp00904.jpeg" target="_blank">View large</a><a section="5319586" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00904.jpeg?Expires=2147483647&Signature=WcR2uiV115B7w5ePGdOkPgqAf~lRyu08b~NMMu1uQRI8C1G8H~GWTa2QGrXNwEqmfxXweAU1XQ9rHsCUjbvxOuJaXR4wSbgqfosacp65zdgso~sVjZx5S-uLIA9om5ya0Ae-vWhuDwGaCQLpdnb5i8u7GTXVGyR8DFi9Vu5Tq7hmsC6tOQvPrKPatUjU3FppkYiZpJY4M5tymuwP7zQd-bIhksF6zttPkvtJoyGdHx~DoXTrLvY4chKNgprPt~FON2o0JFgWDJHD-deAbhax6if-P-29EMUQbGeZCNdHszqYiMqKZX3iskT97hu2i2pjhaq9QeEeu2YDYrZ8dIhBAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319586&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of AS events affecting functional motifs. (<strong>A</strong>) The AT1G09640 locus which encodes a protein with GST domains and EF1 gamma domain. An AS variant MAS-002I13 encodes a protein lacking EF1 gamma domain. (<strong>B</strong>) The AT1G10290 locus which encodes dynamin-like protein ADL6. An AS variant MAS-004M08 encodes a protein lacking dynamin domains. Such AS events might modulate protein functions.</p></div></div><div content-id="DSP009F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/m_dsp00904.jpeg?Expires=2147483647&Signature=dSgazj1lv97~pdb1ZU6mnt3~r3X34yf-13VRGoguncVTvtXLzNk-P1ukt1VkMGLZkZeyfm1uig0RQfzckuSP9ChY40FEBi3cgYhFFWkNyVrQpKyeYAW1eX3jcupJLT4t2tAYCUhzDgyA8-OjnK2RyTry3ja8aYHOzvwYs~9a7xefDJ6bw91t5D8dHSJfoDXDxQTR0r4BmY6Kn6L~5ifxe-6qshAQvg9aTz7TmfEt8hjYnVgRNB~jRYY9wGmlovJRg2zfPsYJBu4zkF9uHwxgRDPgQwXGIglMhftobd~6KAkarJJchm0iK~QTV8CO1aqynkULBwTLbKW~uuqCG2OD8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of AS events affecting functional motifs. (A) The AT1G09640 locus which encodes a protein with GST domains and EF1 gamma domain. An AS variant MAS-002I13 encodes a protein lacking EF1 gamma domain. (B) The AT1G10290 locus which encodes dynamin-like protein ADL6. An AS variant MAS-004M08 encodes a protein lacking dynamin domains. Such AS events might modulate protein functions." path-from-xml="dsp00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5319586/dsp00904.jpeg" path-from-xml="dsp00904.jpeg" target="_blank">View large</a><a section="5319586" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp00904.jpeg?Expires=2147483647&Signature=WcR2uiV115B7w5ePGdOkPgqAf~lRyu08b~NMMu1uQRI8C1G8H~GWTa2QGrXNwEqmfxXweAU1XQ9rHsCUjbvxOuJaXR4wSbgqfosacp65zdgso~sVjZx5S-uLIA9om5ya0Ae-vWhuDwGaCQLpdnb5i8u7GTXVGyR8DFi9Vu5Tq7hmsC6tOQvPrKPatUjU3FppkYiZpJY4M5tymuwP7zQd-bIhksF6zttPkvtJoyGdHx~DoXTrLvY4chKNgprPt~FON2o0JFgWDJHD-deAbhax6if-P-29EMUQbGeZCNdHszqYiMqKZX3iskT97hu2i2pjhaq9QeEeu2YDYrZ8dIhBAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5319586&ar=366369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of AS events affecting functional motifs. (<strong>A</strong>) The AT1G09640 locus which encodes a protein with GST domains and EF1 gamma domain. An AS variant MAS-002I13 encodes a protein lacking EF1 gamma domain. (<strong>B</strong>) The AT1G10290 locus which encodes dynamin-like protein ADL6. An AS variant MAS-004M08 encodes a protein lacking dynamin domains. Such AS events might modulate protein functions.</p></div></div><p>We also checked the CDSs of the 14 chimeric transcripts. Of these, only one transcript (MAS-001H13) had an ORF that spanned between the first and second genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The transcript was a fused mRNA that began with the AT3G63340 locus (encoding a protein phosphatase 2C-related protein) and ended with the AT3G63330 locus (encoding a protein kinase). Still it is hard to image a function for the protein product of this chimeric transcript, but it is likely that this chimeric protein would function in both adding and removing phosphate groups on target proteins.</p>                            <h3 scrollto-destination=5319588 id="5319588" class="section-title" data-legacy-id=s3g>3.7. Summary</h3>
<p>In the current study, we fully sequenced 1800 full-length cDNAs. Even though this data set was not large, it suggests the possibility that many more AS events occur in <em>Arabidopsis</em> than expected. Furthermore, the analysis of multiple AS events in individual genes suggested a regulatory mechanism that controls AS profiles. We hypothesize that some splicing regulatory elements and <em>trans</em>-acting factors are involved in the regulation of AS profiles. Further computational and experimental approaches should help clarify these <em>cis</em>- and <em>trans</em>-acting factors. We also examined coding potentials and protein sequences in the current study. It was interesting that many transcripts seemed to be targeted by NMD and/or to encode potentially non-functional proteins. More integrated analyses of the regulation of AS profiles and coding potential should give us a greater understanding of the importance of AS in <em>Arabidopsis</em> and other plants.</p>                            <h2 scrollto-destination=5319590 id="5319590" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5319592 id="5319592" class="section-title" data-legacy-id=s5>Funding</h2>
<p>Support to characterize the full-length sequences of RAFL cDNA clones discussed in this manuscript was provided by the National Bio-Resource Project of the MEXT, Japan.</p>                            <h2 scrollto-destination=5319594 id="5319594" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the entire technical staff of the Sequence Technology Team at RIKEN GSC for their assistance. We also thank Mr Issei Sasaki (RIKEN BRC) for technical assistance.</p>                            <h2 scrollto-destination=5319596 id="5319596" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">B. B.</div></div>,  <div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Genomewide comparative analysis of alternative splicing in plants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">7175</div>-<div class="lpage">7180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomewide%20comparative%20analysis%20of%20alternative%20splicing%20in%20plants&amp;author=B.%20B.%20Wang&amp;author=V.%20Brendel&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=7175-7180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0602039103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0602039103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0602039103"> </span></div> </div></div></div></div></div><div content-id="DSP009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Castle</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Garrett-Engele</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Loerch</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Armour</div> <div class="given-names">C. D.</div></div>,  <div class="name"><div class="surname">Santos</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schadt</div> <div class="given-names">E. E.</div></div>,  <div class="name"><div class="surname">Stoughton</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">D. D.</div></div>. </div><div class="article-title">Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">2141</div>-<div class="lpage">2144</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20human%20alternative%20pre-mRNA%20splicing%20with%20exon%20junction%20microarrays&amp;author=J.%20M.%20Johnson&amp;author=J.%20Castle&amp;author=P.%20Garrett-Engele&amp;author=Z.%20Kan&amp;author=P.%20M.%20Loerch&amp;author=C.%20D.%20Armour&amp;author=R.%20Santos&amp;author=E.%20E.%20Schadt&amp;author=R.%20Stoughton&amp;author=D.%20D.%20Shoemaker&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=2141-2144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1090100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1090100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1090100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14684825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gough</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Frith</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Maeda</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Oyama</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ravasi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lenhard</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wells</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">The transcriptional landscape of the mammalian genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1559</div>-<div class="lpage">1563</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20mammalian%20genome&amp;author=P.%20Carninci&amp;author=T.%20Kasukawa&amp;author=S.%20Katayama&amp;author=J.%20Gough&amp;author=M.%20C.%20Frith&amp;author=N.%20Maeda&amp;author=R.%20Oyama&amp;author=T.%20Ravasi&amp;author=B.%20Lenhard&amp;author=C.%20Wells&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1559-1563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1112014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1112014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1112014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Macknight</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Duroux</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Laurie</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dijkwel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dean</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Functional significance of the alternative transcript processing of the <em>Arabidopsis</em> floral promoter FCA</div>, <div class="source">Plant Cell</div> , <div class="year">2002</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">877</div>-<div class="lpage">888</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20significance%20of%20the%20alternative%20transcript%20processing%20of%20the%20Arabidopsis%20floral%20promoter%20FCA&amp;author=R.%20Macknight&amp;author=M.%20Duroux&amp;author=R.%20Laurie&amp;author=P.%20Dijkwel&amp;author=G.%20Simpson&amp;author=C.%20Dean&amp;publication_year=2002&amp;journal=Plant%20Cell&amp;volume=14&amp;pages=877-888" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.010456" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.010456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.010456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11971142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X. C.</div></div>,  <div class="name"><div class="surname">Gassmann</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Alternative splicing and mRNA levels of the disease resistance gene RPS4 are induced during defense responses</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">1577</div>-<div class="lpage">1587</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20mRNA%20levels%20of%20the%20disease%20resistance%20gene%20RPS4%20are%20induced%20during%20defense%20responses&amp;author=X.%20C.%20Zhang&amp;author=W.%20Gassmann&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=145&amp;pages=1577-1587" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.108720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.108720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.108720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17951452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lopato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Voronin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Barta</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evolutionary conservation and regulation of particular alternative splicing events in plant SR proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">4395</div>-<div class="lpage">4405</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20conservation%20and%20regulation%20of%20particular%20alternative%20splicing%20events%20in%20plant%20SR%20proteins&amp;author=M.%20Kalyna&amp;author=S.%20Lopato&amp;author=V.%20Voronin&amp;author=A.%20Barta&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=4395-4405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl570" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl570" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl570"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16936312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Satou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Akiyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Toyoda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Konagaya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genome-wide analysis of alternative pre-mRNA splicing in <em>Arabidopsis thaliana</em> based on full-length cDNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">5096</div>-<div class="lpage">5103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20alternative%20pre-mRNA%20splicing%20in%20Arabidopsis%20thaliana%20based%20on%20full-length%20cDNA%20sequences&amp;author=K.%20Iida&amp;author=M.%20Seki&amp;author=T.%20Sakurai&amp;author=M.%20Satou&amp;author=K.%20Akiyama&amp;author=T.%20Toyoda&amp;author=A.%20Konagaya&amp;author=K.%20Shinozaki&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=5096-5103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15452276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bove</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">C. Y.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Assmann</div> <div class="given-names">S. M.</div></div>. </div><div class="article-title">Characterization of wound-responsive RNA-binding proteins and their splice variants in <em>Arabidopsis</em></div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">71</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20wound-responsive%20RNA-binding%20proteins%20and%20their%20splice%20variants%20in%20Arabidopsis&amp;author=J.%20Bove&amp;author=C.%20Y.%20Kim&amp;author=C.%20A.%20Gibson&amp;author=S.%20M.%20Assmann&amp;publication_year=2008&amp;journal=Plant%20Mol.%20Biol.&amp;volume=67&amp;pages=71-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9302-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9302-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9302-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanabe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yoshimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yabuta</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shigeoka</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Differential expression of alternatively spliced mRNAs of <em>Arabidopsis</em> SR protein homologs, atSR30 and atSR45a, in response to environmental stress</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1036</div>-<div class="lpage">1049</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20alternatively%20spliced%20mRNAs%20of%20Arabidopsis%20SR%20protein%20homologs%2C%20atSR30%20and%20atSR45a%2C%20in%20response%20to%20environmental%20stress&amp;author=N.%20Tanabe&amp;author=K.%20Yoshimura&amp;author=A.%20Kimura&amp;author=Y.%20Yabuta&amp;author=S.%20Shigeoka&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=1036-1049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17556373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palusa</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Ali</div> <div class="given-names">G. S.</div></div>,  <div class="name"><div class="surname">Reddy</div> <div class="given-names">A. S.</div></div>. </div><div class="article-title">Alternative splicing of pre-mRNAs of <em>Arabidopsis</em> serine/arginine-rich proteins: regulation by hormones and stresses</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">1091</div>-<div class="lpage">1107</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20pre-mRNAs%20of%20Arabidopsis%20serine%2Farginine-rich%20proteins%3A%20regulation%20by%20hormones%20and%20stresses&amp;author=S.%20G.%20Palusa&amp;author=G.%20S.%20Ali&amp;author=A.%20S.%20Reddy&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=49&amp;pages=1091-1107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2007.49.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2007.49.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2007.49.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17319848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardina</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Alternative splicing and differential gene expression in colon cancer detected by a whole genome exon array</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">325</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20differential%20gene%20expression%20in%20colon%20cancer%20detected%20by%20a%20whole%20genome%20exon%20array&amp;author=P.%20J.%20Gardina&amp;author=T.%20A.%20Clark&amp;author=B.%20Shimada&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-325" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-325" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-325"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17192196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Narusaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kamiya</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Functional annotation of a full-length <em>Arabidopsis</em> cDNA collection</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">141</div>-<div class="lpage">145</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%20of%20a%20full-length%20Arabidopsis%20cDNA%20collection&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=A.%20Kamiya&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=141-145" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1071006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1071006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1071006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11910074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dale</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shinn</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Palm</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Southwick</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H. C.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">842</div>-<div class="lpage">846</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Empirical%20analysis%20of%20transcriptional%20activity%20in%20the%20Arabidopsis%20genome&amp;author=K.%20Yamada&amp;author=J.%20Lim&amp;author=J.%20M.%20Dale&amp;author=H.%20Chen&amp;author=P.%20Shinn&amp;author=C.%20J.%20Palm&amp;author=A.%20M.%20Southwick&amp;author=H.%20C.%20Wu&amp;author=C.%20Kim&amp;author=M.%20Nguyen&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=842-846" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1088305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1088305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1088305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14593172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Abajian</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Consed: a graphical tool for sequence finishing</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">195</div>-<div class="lpage">202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing&amp;author=D.%20Gordon&amp;author=C.%20Abajian&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=195-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;author=J.%20Zhang&amp;author=Z.%20Zhang&amp;author=W.%20Miller&amp;author=D.%20J.%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1157</div>-<div class="lpage">1169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20structure%20prediction%20from%20consensus%20spliced%20alignment%20of%20multiple%20ESTs%20matching%20the%20same%20genomic%20locus&amp;author=V.%20Brendel&amp;author=L.%20Xing&amp;author=W.%20Zhu&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1157-1169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shionyu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suso</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Alternative splicing at NAGNAG acceptor sites shares common properties in land plants and mammals</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">709</div>-<div class="lpage">718</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20at%20NAGNAG%20acceptor%20sites%20shares%20common%20properties%20in%20land%20plants%20and%20mammals&amp;author=K.%20Iida&amp;author=M.%20Shionyu&amp;author=Y.%20Suso&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=709-718" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18234709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swarbreck</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wilks</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The <em>Arabidopsis</em> information resource (TAIR): gene structure and function annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D1009</div>-<div class="lpage">D1014</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20information%20resource%20(TAIR)%3A%20gene%20structure%20and%20function%20annotation&amp;author=D.%20Swarbreck&amp;author=C.%20Wilks&amp;author=P.%20Lamesch&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D1009-D1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schuster-Böckler</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Pfam: clans, web tools and services</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D247</div>-<div class="lpage">D251</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20clans%2C%20web%20tools%20and%20services&amp;author=R.%20D.%20Finn&amp;author=J.%20Mistry&amp;author=B.%20Schuster-B%C3%B6ckler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D247-D251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akiva</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Toporik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Edelheit</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Peretz</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Diber</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shemesh</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Novik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transcription-mediated gene fusion in the human genome</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">30</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-mediated%20gene%20fusion%20in%20the%20human%20genome&amp;author=P.%20Akiva&amp;author=A.%20Toporik&amp;author=S.%20Edelheit&amp;author=Y.%20Peretz&amp;author=A.%20Diber&amp;author=R.%20Shemesh&amp;author=A.%20Novik&amp;author=R.%20Sorek&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=30-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4137606" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4137606" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4137606"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16344562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Babushok</div> <div class="given-names">D. V.</div></div>,  <div class="name"><div class="surname">Ohshima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ostertag</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mandal</div> <div class="given-names">P. K.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Abrams</div> <div class="given-names">C. S.</div></div>,  <div class="name"><div class="surname">Kazazian</div> <div class="given-names">H. H.</div>Jr</div>. </div><div class="article-title">A novel testis ubiquitin-binding protein gene arose by exon shuffling in hominoids</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1129</div>-<div class="lpage">1138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20testis%20ubiquitin-binding%20protein%20gene%20arose%20by%20exon%20shuffling%20in%20hominoids&amp;author=D.%20V.%20Babushok&amp;author=K.%20Ohshima&amp;author=E.%20M.%20Ostertag&amp;author=X.%20Chen&amp;author=Y.%20Wang&amp;author=P.%20K.%20Mandal&amp;author=N.%20Okada&amp;author=C.%20S.%20Abrams&amp;author=H.%20H.%20Kazazian&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1129-1138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6252107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6252107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6252107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17623810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pospisil</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Valcárcel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Reich</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Alternative splicing and genome complexity</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">29</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20genome%20complexity&amp;author=D.%20Brett&amp;author=H.%20Pospisil&amp;author=J.%20Valc%C3%A1rcel&amp;author=J.%20Reich&amp;author=P.%20Bork&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=29-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lorković</div> <div class="given-names">Z. J.</div></div>,  <div class="name"><div class="surname">Wieczorek Kirk</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Lambermon</div> <div class="given-names">M. H.</div></div>,  <div class="name"><div class="surname">Filipowicz</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Pre-mRNA splicing in higher plants</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2000</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">160</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pre-mRNA%20splicing%20in%20higher%20plants&amp;author=Z.%20J.%20Lorkovi%C4%87&amp;author=D.%20A.%20Wieczorek%20Kirk&amp;author=M.%20H.%20Lambermon&amp;author=W.%20Filipowicz&amp;publication_year=2000&amp;journal=Trends%20Plant%20Sci.&amp;volume=5&amp;pages=160-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(00)01595-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(00)01595-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(00)01595-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10740297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reddy</div> <div class="given-names">A. S.</div></div>. </div><div class="article-title">Plant serine/arginine-rich proteins and their role in pre-mRNA splicing</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">541</div>-<div class="lpage">547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20serine%2Farginine-rich%20proteins%20and%20their%20role%20in%20pre-mRNA%20splicing&amp;author=A.%20S.%20Reddy&amp;publication_year=2004&amp;journal=Trends%20Plant%20Sci.&amp;volume=9&amp;pages=541-547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2004.09.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2004.09.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2004.09.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15501179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barta</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A plethora of plant serine/arginine-rich proteins: redundancy or evolution of novel gene functions?</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">561</div>-<div class="lpage">564</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plethora%20of%20plant%20serine%2Farginine-rich%20proteins%3A%20redundancy%20or%20evolution%20of%20novel%20gene%20functions%3F&amp;author=M.%20Kalyna&amp;author=A.%20Barta&amp;publication_year=2004&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=32&amp;pages=561-564" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0320561" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0320561" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0320561"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15270675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kilian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Whitehead</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Horak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wanke</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Weinl</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Batistic</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">D'Angelo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bornberg-Bauer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kudla</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Harter</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">347</div>-<div class="lpage">363</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20AtGenExpress%20global%20stress%20expression%20data%20set%3A%20protocols%2C%20evaluation%20and%20model%20data%20analysis%20of%20UV-B%20light%2C%20drought%20and%20cold%20stress%20responses&amp;author=J.%20Kilian&amp;author=D.%20Whitehead&amp;author=J.%20Horak&amp;author=D.%20Wanke&amp;author=S.%20Weinl&amp;author=O.%20Batistic&amp;author=C.%20D'Angelo&amp;author=E.%20Bornberg-Bauer&amp;author=J.%20Kudla&amp;author=K.%20Harter&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=50&amp;pages=347-363" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03052.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03052.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03052.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17376166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hori</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Context analysis of termination codons in mRNA that are recognized by plant NMD</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1072</div>-<div class="lpage">1078</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Context%20analysis%20of%20termination%20codons%20in%20mRNA%20that%20are%20recognized%20by%20plant%20NMD&amp;author=K.%20Hori&amp;author=Y.%20Watanabe&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=1072-1078" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17567637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Go</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Survey of conserved alternative splicing events of mRNAs encoding SR proteins in land plants</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1085</div>-<div class="lpage">1094</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20of%20conserved%20alternative%20splicing%20events%20of%20mRNAs%20encoding%20SR%20proteins%20in%20land%20plants&amp;author=K.%20Iida&amp;author=M.%20Go&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1085-1094" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16520337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewis</div> <div class="given-names">B. P.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S. E.</div></div>. </div><div class="article-title">Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">189</div>-<div class="lpage">192</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20widespread%20coupling%20of%20alternative%20splicing%20and%20nonsense-mediated%20mRNA%20decay%20in%20humans&amp;author=B.%20P.%20Lewis&amp;author=R.%20E.%20Green&amp;author=S.%20E.%20Brenner&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=189-192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0136770100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0136770100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0136770100"> </span></div> </div></div></div></div></div><div content-id="DSP009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birzele</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Csaba</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zimmer</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Alternative splicing and protein structure evolution</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">550</div>-<div class="lpage">558</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20protein%20structure%20evolution&amp;author=F.%20Birzele&amp;author=G.%20Csaba&amp;author=R.%20Zimmer&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=550-558" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm1054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm1054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm1054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18055499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Mushegian</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">Tatusov</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Bryant</div> <div class="given-names">S. H.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Valencia</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Eukaryotic translation elongation factor 1 gamma contains a glutathione transferase domain–study of a diverse, ancient protein superfamily using motif search and structural modeling</div>, <div class="source">Protein Sci.</div> , <div class="year">1994</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">2045</div>-<div class="lpage">2054</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Eukaryotic%20translation%20elongation%20factor%201%20gamma%20contains%20a%20glutathione%20transferase%20domain%E2%80%93study%20of%20a%20diverse%2C%20ancient%20protein%20superfamily%20using%20motif%20search%20and%20structural%20modeling&amp;author=E.%20V.%20Koonin&amp;author=A.%20R.%20Mushegian&amp;author=R.%20L.%20Tatusov&amp;author=S.%20F.%20Altschul&amp;author=S.%20H.%20Bryant&amp;author=P.%20Bork&amp;author=A.%20Valencia&amp;publication_year=1994&amp;journal=Protein%20Sci.&amp;volume=3&amp;pages=2045-2054" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pro.v3:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpro.v3:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpro.v3:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7703850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP009C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">S. H.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">J. B.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Min</div> <div class="given-names">M. K.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">D. S.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">Y. W.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">I.</div></div>. </div><div class="article-title">The intermolecular interaction between the PH domain and the C-terminal domain of <em>Arabidopsis</em> dynamin-like 6 determines lipid binding specificity</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">31842</div>-<div class="lpage">31849</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20intermolecular%20interaction%20between%20the%20PH%20domain%20and%20the%20C-terminal%20domain%20of%20Arabidopsis%20dynamin-like%206%20determines%20lipid%20binding%20specificity&amp;author=S.%20H.%20Lee&amp;author=J.%20B.%20Jin&amp;author=J.%20Song&amp;author=M.%20K.%20Min&amp;author=D.%20S.%20Park&amp;author=Y.%20W.%20Kim&amp;author=I.%20Hwang&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=31842-31849" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M204770200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M204770200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M204770200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12105222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Mikio Nishimura</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=366369&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table7.xls?Expires=2147483647&amp;Signature=p00XZ9F4TZR-jLTyW5E23n62ZnZNxjtKyBBuF7HAyEs2SNpNjAAyyv9fGRwbPsXf74dl4katv9xquDY0UGDuqGUwnZi57nqXJ~2EkMibhiBNZuX~49H~lI0DoWo8Cxu9iNqAUDs4W4zRz7mgoSIqVnTLX2dbf2Af5A5eUHbbDUM2dBqUeObz4ystVo7W9zjWVTrjIoojJtTwio0UTUP88awIgHHrfdbLhRNiZsICW94vn4AuVSjN~pETrbyHvfYFhoQx0HSQ8chSCp5PFZ~eCpkgcyxl7IXCZA9NE~vqhO5TEAceAXLEX29~PLQIV63eM3dDXv6oOheF2ezB8QYBow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table6.xls?Expires=2147483647&amp;Signature=ID0UcBNOONiJ5xZm1kePNrE14U1T~sJzVBeRp7gulwkKHyk8FbyjAHFvkILRbDKi-QlBNjC65HWVWDFI1dRJGmVoVbhE57Laq~9HbOG0~eDj6WRl6lVfqzpJdnaoTj0G1ctLHqbY9AxoDDmCm5~Yvn3ZPpEGNTxQdhbupRTYn9OaPCQ8SvyAA1CkTvrPz6m7K0qHRHfqRPO57nWuQTqd5Kn64w7k5Ak5ebWcrF4nGgImaVRlcHfuqbdLKjlGwP3ZyzlzTo6PJ25f2QnPxQUXL0nqZS-1MJ4uwUokBG8htg7OSMWgiItrPTopu-esTheuhppF5-30ILlfG3i0vxoWYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table5.xls?Expires=2147483647&amp;Signature=gche~ZtgzILeUAFxPoS-K6l5LCI60FG4K9ih5PtCdmX6jxliXpRwrxvJlyHjJ~rV61ZqaNuB2~y4lX9Qd3M0wjdEoHvINb4Js46rW7oTOhVmmV8ykeEUXvB4dSwHZ~PY7TBAglFIQDlilyheyndUxkF7smmfOhnO1TkUnYN5aIf9oPt1-rTs2oj6B5Gg3z3MOteHN-QXDMdbNpkwR9iHe6iNBuvNoxUbfcjNncxU6lOOg41ux8-qXO70GVKOALyPc6CZE6Ft6ctzW9xJmeEXH8MAEjOlhHaW-pt2UykKoj~kRsddvOOrJdbA23yao5RNNDqyigKcSJUiIjE90gHE1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table4.xls?Expires=2147483647&amp;Signature=DGtAUkyasNQTo0Cq1DyMiojTY7X6lYAAxA9LPqDBQTZnTX8-GRoKSo4lGFMTYHWGFCz5hbCJIORVq09mfmXH1tkOf-IjwZynwtwH8uHg13RUzyeDqZwkMgxVFE5VnR5Bp1Gyg9cKob12LEd2hltyzkQm6bOZzYdakmHimQcNyk4vGZ3OslwBjppKKjpz24MdoiAcuHL3P1ECHmxVYvtb8CddHfoyn6h97NpJ4juy7sP6HWeMdZLliKyiLcGvBQZJhfKS3VPmCMso-NWMzcuRTFuF5i6~YJoRye~SkyoohLr2xggQSvQ7oIp2iQSYte34Izk~dge4WSHG9eASMmQk2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table3.xls?Expires=2147483647&amp;Signature=R-WLYrqXgs-9FQqFaK7~0YbqpzRs7S33sg8EYMI7ZU~rinXuP3B9b9~1Pix0mx2LDxIO~2AsQZeTCGVSkIE1o2sYkTMmSu23FROBLh7ea03pyPx3nNzdfMpWduZvZyq1qiX9hxeWsPfvlqlzMqx9gj5tPy0GP2ztKB85aMn0CZTZbksF4KEr6W1Mgm~E2SRZQozCbiVtbMilICTNb~N5ceWQTxae1QB33pTtEpRT04eW5akoTd8qdI61N1NMBo6xQk4lTrEmHu5hroY5F3uiVnH-Rq-WrFFbW3Nt7daLeBNT-reWmPL0RJo58RopSPnMnDIWkKUYMJC8Cz6VpW-3xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table2.xls?Expires=2147483647&amp;Signature=MrtB9b-NrPrRGwlkPk~UUKkx8rVgFgWO5~4ZVJoGHYByVW656rxUUoKcdBWAS-l-vhoKFt5b-qTA9-NRlvmFPVyzkQlsxItw~IN-WDelS1SMDLlfPu~jiMcaNwiRnbfL--J4YSejx-3NiKouv-csX9R~zNy0GCvFG-3ijZYkfNkN8HNxGQeiSgCv9~zWGqGkDv~dHzrI6GbcorlvPQdmAZ7-bZlweYWF43~8Cnye2M6vSnhi-29iokvqTwDnBiFDGZZ7Xzn~Ke20Mzf7Ha1DUfOZlYLXRR2z-CEQixDk9eEE1F7YUmxnAeEaN4FBdO0hTpDkMpFLFl8nGL~Ju~yObQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009_Table1.xls?Expires=2147483647&amp;Signature=CGvQZI-BOLQSA-djE0LqP6jwI1O2nfP3Htkp~APPiCafMD7S58ws63MYlqxH~jj12KM66LfEk~AkdTdUaWnlxUNf3gb7mTJlvlhDKdrauD2qWTw2J3aXt7PpMlwqQs7wbsQfgSf39UwMpcv0PbFCYvbNKl47u~XPpV~JoyYwCQv094af4j3OuHYTeucjQ7geIBe3WBVBl7WTlUGdL6t~Ou7WfKF5JO6uHA4N10Xcvr8nTLtcegBhHZOY1v-QpG0W8ukPOHJitwk0DXi4f7Q0c2Py4R-T0v2iCoz3qg1eWGEeK5jjbrYGAGrhtnHLneB9R7cbM0NgcyOI~cX44HvxbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/3/10.1093/dnares/dsp009/2/dsp009-dsp009supp.pdf?Expires=2147483647&amp;Signature=ML~j50MoWfZWsW-lE5xJxCbtBUhz5jGAPSzwAh9G05s0xVkVS2UpA2yqXxmtojT2EHbT8IGM2pe8yG0eyGsC9uqkdsaYrOLYUYvAOI~Vei5O0QpgJai2T2JnyONZn-mmk7NVu05hSv7WLxDlqMyqt91BejUe5Hkv4yQa0jQUctknRm3xdpLLLBThbk1zcGXqgBsnglybfZp4NIpaXsWCnkZsgEM8oRJEDPjBqV6bTsLH0QAjAsop3bGQ5kD7xRdar1jvkN7CT5ZZXXIA07t~OCKH5irPOg3sTrqYXlDEhIXpkp-0CoZL970Sb3u9gAUifsGa7tdPfWp~Mui439QzFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">141</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000267647000002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">9</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="366369"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000267647000002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f3%2f155%2f366369">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000267647000002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (9)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F3%2F155%2F366369" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/366369'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Analysis of Multiple Occurrences of Alternative Splicing Events in Arabidopsis thaliana Using Novel Sequenced Full-Length cDNAs" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f3%2f155%2f366369">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

