

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/215/1745460","@type":"ScholarlyArticle","name":"Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes","datePublished":"2016-04-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw012","keywords":["buckwheat","draft sequence","database usage","GBS marker","heteromorphic self-incompatibility"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/215/1745460","author":[{"name":"Yasui, Yasuo","affiliation":"1  Graduate School of Agriculture ,  Kyoto University ,  Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ueno, Mariko","affiliation":"1  Graduate School of Agriculture ,  Kyoto University ,  Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Matsui, Katsuhiro","affiliation":"3  NARO Kyushu Okinawa Agricultural Research Center ,  2421 Suya, Koshi, Kumamoto 861-1192 ,  Japan","@type":"Person"},{"name":"Katsube-Tanaka, Tomoyuki","affiliation":"1  Graduate School of Agriculture ,  Kyoto University ,  Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Yang, Soo Jung","affiliation":"1  Graduate School of Agriculture ,  Kyoto University ,  Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Aii, Jotaro","affiliation":"4  Faculty of Applied Life Science ,  Niigata University of Pharmacy and Applied Life Science ,  Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Sato, Shingo","affiliation":"4  Faculty of Applied Life Science ,  Niigata University of Pharmacy and Applied Life Science ,  Akiha-ku, Niigata 956-8603 ,  Japan","@type":"Person"},{"name":"Mori, Masashi","affiliation":"5  Research Institute for Bioresources and Biotechnology ,  Ishikawa Prefectural University ,  308 Suematsu, Nonoichi, Ishikawa 912-8836 ,  Japan","@type":"Person"}],"description":"Abstract.  Buckwheat (Fagopyrum esculentum Moench; 2n = 2x = 16) is a nutritionally dense annual crop widely grown in temperate zones. To accelerate molecular b","pageStart":"215","pageEnd":"224","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes" />
    <meta property="og:description" content="Abstract.  Buckwheat (Fagopyrum esculentum Moench; 2n = 2x = 16) is a nutritionally dense annual crop widely grown in temperate zones. To accelerate molecular b" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/215/1745460" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yasui, Yasuo" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ueno, Mariko" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Matsui, Katsuhiro" /><meta name="citation_author_institution" content="NARO Kyushu Okinawa Agricultural Research Center, 2421 Suya, Koshi, Kumamoto 861-1192, Japan" /><meta name="citation_author" content="Katsube-Tanaka, Tomoyuki" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Yang, Soo Jung" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Aii, Jotaro" /><meta name="citation_author_institution" content="Faculty of Applied Life Science, Niigata University of Pharmacy and Applied Life Science, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Sato, Shingo" /><meta name="citation_author_institution" content="Faculty of Applied Life Science, Niigata University of Pharmacy and Applied Life Science, Akiha-ku, Niigata 956-8603, Japan" /><meta name="citation_author" content="Mori, Masashi" /><meta name="citation_author_institution" content="Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, 308 Suematsu, Nonoichi, Ishikawa 912-8836, Japan" /><meta name="citation_title" content="Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes" /><meta name="citation_firstpage" content="215" /><meta name="citation_lastpage" content="224" /><meta name="citation_doi" content="10.1093/dnares/dsw012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The American cranberry: first insights into the whole genome of a species adapted to bog habitat;  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=165" /><meta name="citation_reference" content="citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species;  citation_journal_title=Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=270-8; " /><meta name="citation_reference" content="citation_title=Genome sequence of foxtail millet (Setaria italica) provides insights into grass evolution and biofuel potential;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=549-54; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=Buckwheat as a functional food and its effects on health;  citation_journal_title=J. Agric. Food Chem.;  citation_year=2015;  citation_volume=63;  citation_pages=7896-913; " /><meta name="citation_reference" content="citation_title=The gluten-free diet: testing alternative cereals tolerated by celiac patients;  citation_journal_title=Nutrients;  citation_year=2013;  citation_volume=5;  citation_pages=4250-68; " /><meta name="citation_reference" content="citation_title=Buckwheat allergy: an emerging clinical problem in Europe;  citation_journal_title=J. Allergy Ther.;  citation_year=2014;  citation_volume=5;  citation_pages=168" /><meta name="citation_reference" content="citation_title=Amplified fragment length polymorphism linkage analysis of common buckwheat (Fagopyrum esculentum) and its wild self-pollinated relative Fagopyrum homotropicum;  citation_journal_title=Genome;  citation_year=2004;  citation_volume=47;  citation_pages=345-51; " /><meta name="citation_reference" content="citation_title=A linkage map for common buckwheat based on microsatellite and AFLP markers;  citation_journal_title=Fagopyrum;  citation_year=2006;  citation_volume=23;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=QTL analysis of photoperiod sensitivity in common buckwheat by using markers for expressed sequence tags and photoperiod-sensitivity candidate genes;  citation_journal_title=Breed. Sci.;  citation_year=2011;  citation_volume=61;  citation_pages=394-404; " /><meta name="citation_reference" content="citation_title=Rapid genotyping with DNA micro-arrays for high-density linkage mapping and QTL mapping in common buckwheat (Fagopyrum esculentum Moench);  citation_journal_title=Breed. Sci.;  citation_year=2014;  citation_volume=64;  citation_pages=291-9; " /><meta name="citation_reference" content="citation_title=A robust, simple Genotyping-by-Sequencing (GBS) Approach for high diversity species;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding;  citation_journal_title=Front. Plant Sci.;  citation_year=2014;  citation_volume=5;  citation_pages=484" /><meta name="citation_reference" content="citation_title=Construction of a BAC library for buckwheat genome research—an application to positional cloning of agriculturally valuable traits;  citation_journal_title=Genes Genet. Syst.;  citation_year=2008;  citation_volume=83;  citation_pages=393-401; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=S-LOCUS EARLY FLOWERING 3 is exclusively present in the genomes of short-styled buckwheat plants that exhibit heteromorphic self-incompatibility;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e31264" /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_issue=(Suppl 1);  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Using geneid to identify genes;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2007;  citation_volume=Chapter 4;  citation_pages=Unit 4.3" /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes;  citation_journal_title=BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=The Gypsy Database (GyDB) of mobile genetic elements: release 2.0;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D70-4; " /><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference;  citation_journal_title=Genome Inform.;  citation_year=2009;  citation_volume=23;  citation_pages=205-11; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=ClustalW and ClustalX version 2;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences;  citation_journal_title=Comput. Appl. Biosci.;  citation_year=1992;  citation_volume=8;  citation_pages=275-82; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=Genome size analysis of the genus Fagopyrum;  citation_journal_title=Fagopyrum;  citation_year=2000;  citation_volume=17;  citation_pages=35-9; " /><meta name="citation_reference" content="citation_title=The draft genome of Primula veris yields insights into the molecular basis of heterostyly;  citation_journal_title=Genome Biol.;  citation_year=2015;  citation_volume=16;  citation_pages=12" /><meta name="citation_reference" content="citation_title=Draft sequences of the radish (Raphanus sativus L.) genome;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=481-90; " /><meta name="citation_reference" content="citation_title=Buckwheat as a source of rutin;  citation_journal_title=Science;  citation_year=1946;  citation_volume=15;  citation_pages=197-8; " /><meta name="citation_reference" content="citation_title=Differential expression of flavonoid biosynthesis genes and accumulation of phenolic compounds in common buckwheat (Fagopyrum esculentum);  citation_journal_title=J. Agric. Food Chem.;  citation_year=2010;  citation_volume=58;  citation_pages=12176-81; " /><meta name="citation_reference" content="citation_title=Expression of flavonoid biosynthesis genes vis-à-vis rutin content variation in different growth stages of Fagopyrum species;  citation_journal_title=J. Plant Physiol.;  citation_year=2011;  citation_volume=168;  citation_pages=2117-23; " /><meta name="citation_reference" content="citation_title=Analysis of bHLH and MYB domain proteins: species-specific regulatory differences are caused by divergent evolution of target anthocyanin genes;  citation_journal_title=Plant J.;  citation_year=1998;  citation_volume=13;  citation_pages=475-88; " /><meta name="citation_reference" content="citation_title=The TRANSPARENT TESTA GLABRA1 locus, which regulates trichome differentiation and anthocyanin biosynthesis in Arabidopsis, encodes a WD40 repeat protein;  citation_journal_title=Plant Cell;  citation_year=1999;  citation_volume=11;  citation_pages=1337-49; " /><meta name="citation_reference" content="citation_title=TT2, TT8, and TTG1 synergistically specify the expression of BANYULS and proanthocyanidin biosynthesis in Arabidopsis thaliana;  citation_journal_title=Plant J.;  citation_year=2004;  citation_volume=39;  citation_pages=366-80; " /><meta name="citation_reference" content="citation_title=Regulation of the anthocyanin biosynthetic pathway by the TTG1/bHLH/Myb transcriptional complex in Arabidopsis seedlings;  citation_journal_title=Plant J.;  citation_year=2008;  citation_volume=53;  citation_pages=814-27; " /><meta name="citation_reference" content="citation_title=MYB transcriptiofnactors in plant;  citation_journal_title=Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=67-73; " /><meta name="citation_reference" content="citation_title=The R2R3-MYB gene family in Arabidopsis thaliana;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2001;  citation_volume=4;  citation_pages=447-56; " /><meta name="citation_reference" content="citation_title=MYB transcription factors in Arabidopsis;  citation_journal_title=Trends Plant Sci.;  citation_year=2010;  citation_volume=15;  citation_pages=573-81; " /><meta name="citation_reference" content="citation_title=Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples;  citation_journal_title=Plant Physiol.;  citation_year=2006;  citation_volume=142;  citation_pages=1216-32; " /><meta name="citation_reference" content="citation_title=DkMyb4 is a Myb transcription factor involved in proanthocyanidin biosynthesis in persimmon fruit;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=2028-45; " /><meta name="citation_reference" content="citation_title=Molecular characterization of mutations in white-flowered torenia plants;  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=86" /><meta name="citation_reference" content="citation_title=Characterization of the cis elements in the proximal promoter regions of the anthocyanin pathway genes reveals a common regulatory logic that governs pathway regulation;  citation_journal_title=J. Exp. Bot.;  citation_year=2015;  citation_volume=66;  citation_pages=3775-89; " /><meta name="citation_reference" content="citation_title=Isolation and functional analysis of a MYB transcription factor gene that is a key regulator for the development of red coloration in apple skin;  citation_journal_title=Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=958-70; " /><meta name="citation_reference" content="citation_title=A single-repeat MYB transcription factor, GmMYB176, regulates CHS8 gene expression and affects isoflavonoid biosynthesis in soybean;  citation_journal_title=Plant J.;  citation_year=2010;  citation_volume=62;  citation_pages=1019-34; " /><meta name="citation_reference" content="citation_title=Pepsin-resistant 16-kD buckwheat protein is associated with immediate hypersensitivity reaction in patients with buckwheat allergy;  citation_journal_title=Int. Arch. Allergy Immunol.;  citation_year=2002;  citation_volume=129;  citation_pages=49-56; " /><meta name="citation_reference" content="citation_title=Molecular characterization of a 10-kDa buckwheat molecule reactive to allergic patients’ IgE;  citation_journal_title=Allergy;  citation_year=2004;  citation_volume=59;  citation_pages=533-8; " /><meta name="citation_reference" content="citation_title=Immunological characterization and mutational analysis of the recombinant protein BWp16, a major allergen in buckwheat;  citation_journal_title=Biol. Pharm. Bull.;  citation_year=2008;  citation_volume=31;  citation_pages=1079-85; " /><meta name="citation_reference" content="citation_title=Starches—from current models to genetic engineering;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2013;  citation_volume=11;  citation_pages=223-32; " /><meta name="citation_reference" content="citation_title=Form and fuctionality of starch;  citation_journal_title=Food Hydrocoll.;  citation_year=2009;  citation_volume=23;  citation_pages=1527-34; " /><meta name="citation_reference" content="citation_title=Starches from different botanical sources I: contribution of amylopectin fine structure to thermal properties and enzyme digestibility;  citation_journal_title=Carbohydr. Polym.;  citation_year=2005;  citation_volume=60;  citation_pages=529-38; " /><meta name="citation_reference" content="citation_title=Improving starch for food and industrial applications;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2004;  citation_volume=7;  citation_pages=210-8; " /><meta name="citation_reference" content="citation_title=The function of the waxy locus in starch synthesis in maize endosperm;  citation_journal_title=Biochem. Genet.;  citation_year=1974;  citation_volume=11;  citation_pages=83-96; " /><meta name="citation_reference" content="citation_title=Identification of the soluble starch synthase activities of maize endosperm;  citation_journal_title=Plant Physiol.;  citation_year=1999;  citation_volume=120;  citation_pages=205-15; " /><meta name="citation_reference" content="citation_title=From bacterial glycogen to starch: understanding the biogenesis of the plant starch granule;  citation_journal_title=Annu. Rev. Plant Biol.;  citation_year=2003;  citation_volume=54;  citation_pages=207-33; " /><meta name="citation_reference" content="citation_title=Identification and characterization of granule bound starch synthase I (GBSSI) gene of tartary buckwheat (Fagopyrum tataricum Gaertn.);  citation_journal_title=Gene;  citation_year=2014;  citation_volume=534;  citation_pages=229-35; " /><meta name="citation_reference" content="citation_title=Diversification of genes encoding granule-bound starch synthase in monocots and dicots is marked by multiple genome-wide duplication events;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30088" /><meta name="citation_reference" content="citation_title=Differential effects of Wx-A1, -B1 and -D1 protein deficiencies on apparent amylose content and starch pasting properties in common wheat;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2000;  citation_volume=100;  citation_pages=32-8; " /><meta name="citation_reference" content="citation_title=The inheritance of length of style in buckwheat;  citation_journal_title=J. Agric. Res.;  citation_year=1927;  citation_volume=34;  citation_pages=181-3; " /><meta name="citation_reference" content="citation_title=The genetics of heterostyly; citation_publisher=Springer-Verlag, Berlin;  citation_title=Evolution and function of heterostyly;  citation_year=1992;  citation_pages=129-50; " /><meta name="citation_reference" content="citation_title=The ubiquitin/26S proteasome pathway, the complex last chapter in the life of many plant proteins;  citation_journal_title=Trends Plant Sci.;  citation_year=2003;  citation_volume=8;  citation_pages=135-42; " /><meta name="citation_reference" content="citation_title=Identification and characterization of components of a putative Petunia S-Locus F-box–containing E3 ligase complex involved in S-RNase–based self-incompatibility;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=2531-53; " /><meta name="citation_reference" content="citation_title=Discovery of rare mutations in populations: TILLING by sequencing;  citation_journal_title=Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1257-68; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/215/17463135/dsw012.pdf" /><meta name="description" content="Abstract.  Buckwheat (Fagopyrum esculentum Moench; 2n = 2x = 16) is a nutritionally dense annual crop widely grown in temperate zones. To accelerate molecular b" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/215/1745460" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/215/1745460" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/215/1745460");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw012';
    </script>
<title>
	Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745460" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbqtATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFzdWksIFlhc3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlLCBLeW90byBVbml2ZXJzaXR5LCBLaXRhc2hpcmFrYXdhIE9pd2FrZS1jaG8sIFNha3lvdS1rdSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVZW5vLCBNYXJpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIEt5b3RvIFVuaXZlcnNpdHksIEtpdGFzaGlyYWthd2EgT2l3YWtlLWNobywgU2FreW91LWt1LCBLeW90byA2MDYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdWksIEthdHN1aGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5BUk8gS3l1c2h1IE9raW5hd2EgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciwgMjQyMSBTdXlhLCBLb3NoaSwgS3VtYW1vdG8gODYxLTExOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0c3ViZS1UYW5ha2EsIFRvbW95dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlLCBLeW90byBVbml2ZXJzaXR5LCBLaXRhc2hpcmFrYXdhIE9pd2FrZS1jaG8sIFNha3lvdS1rdSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgU29vIEp1bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIEt5b3RvIFVuaXZlcnNpdHksIEtpdGFzaGlyYWthd2EgT2l3YWtlLWNobywgU2FreW91LWt1LCBLeW90byA2MDYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBaWksIEpvdGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgQXBwbGllZCBMaWZlIFNjaWVuY2UsIE5paWdhdGEgVW5pdmVyc2l0eSBvZiBQaGFybWFjeSBhbmQgQXBwbGllZCBMaWZlIFNjaWVuY2UsIEFraWhhLWt1LCBOaWlnYXRhIDk1Ni04NjAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdG8sIFNoaW5nbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgQXBwbGllZCBMaWZlIFNjaWVuY2UsIE5paWdhdGEgVW5pdmVyc2l0eSBvZiBQaGFybWFjeSBhbmQgQXBwbGllZCBMaWZlIFNjaWVuY2UsIEFraWhhLWt1LCBOaWlnYXRhIDk1Ni04NjAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmksIE1hc2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIEJpb3Jlc291cmNlcyBhbmQgQmlvdGVjaG5vbG9neSwgSXNoaWthd2EgUHJlZmVjdHVyYWwgVW5pdmVyc2l0eSwgMzA4IFN1ZW1hdHN1LCBOb25vaWNoaSwgSXNoaWthd2EgOTEyLTg4MzYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBc3NlbWJseSBvZiB0aGUgZHJhZnQgZ2Vub21lIG9mIGJ1Y2t3aGVhdCBhbmQgaXRzIGFwcGxpY2F0aW9ucyBpbiBpZGVudGlmeWluZyBhZ3Jvbm9taWNhbGx5IHVzZWZ1bCBnZW5lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQW1lcmljYW4gY3JhbmJlcnJ5OiBmaXJzdCBpbnNpZ2h0cyBpbnRvIHRoZSB3aG9sZSBnZW5vbWUgb2YgYSBzcGVjaWVzIGFkYXB0ZWQgdG8gYm9nIGhhYml0YXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgaG90IHBlcHBlciBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIHRoZSBldm9sdXRpb24gb2YgcHVuZ2VuY3kgaW4gQ2Fwc2ljdW0gc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0yNzAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIGZveHRhaWwgbWlsbGV0IChTZXRhcmlhIGl0YWxpY2EpIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZ3Jhc3MgZXZvbHV0aW9uIGFuZCBiaW9mdWVsIHBvdGVudGlhbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIHBpZ2VvbnBlYSAoQ2FqYW51cyBjYWphbiksIGFuIG9ycGhhbiBsZWd1bWUgY3JvcCBvZiByZXNvdXJjZS1wb29yIGZhcm1lcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz04My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CdWNrd2hlYXQgYXMgYSBmdW5jdGlvbmFsIGZvb2QgYW5kIGl0cyBlZmZlY3RzIG9uIGhlYWx0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWdyaWMuIEZvb2QgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz03ODk2LTkxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdsdXRlbi1mcmVlIGRpZXQ6IHRlc3RpbmcgYWx0ZXJuYXRpdmUgY2VyZWFscyB0b2xlcmF0ZWQgYnkgY2VsaWFjIHBhdGllbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdXRyaWVudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQyNTAtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1Y2t3aGVhdCBhbGxlcmd5OiBhbiBlbWVyZ2luZyBjbGluaWNhbCBwcm9ibGVtIGluIEV1cm9wZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWxsZXJneSBUaGVyLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1wbGlmaWVkIGZyYWdtZW50IGxlbmd0aCBwb2x5bW9ycGhpc20gbGlua2FnZSBhbmFseXNpcyBvZiBjb21tb24gYnVja3doZWF0IChGYWdvcHlydW0gZXNjdWxlbnR1bSkgYW5kIGl0cyB3aWxkIHNlbGYtcG9sbGluYXRlZCByZWxhdGl2ZSBGYWdvcHlydW0gaG9tb3Ryb3BpY3VtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0zNDUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGlua2FnZSBtYXAgZm9yIGNvbW1vbiBidWNrd2hlYXQgYmFzZWQgb24gbWljcm9zYXRlbGxpdGUgYW5kIEFGTFAgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RmFnb3B5cnVtOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYW5hbHlzaXMgb2YgcGhvdG9wZXJpb2Qgc2Vuc2l0aXZpdHkgaW4gY29tbW9uIGJ1Y2t3aGVhdCBieSB1c2luZyBtYXJrZXJzIGZvciBleHByZXNzZWQgc2VxdWVuY2UgdGFncyBhbmQgcGhvdG9wZXJpb2Qtc2Vuc2l0aXZpdHkgY2FuZGlkYXRlIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTM5NC00MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGdlbm90eXBpbmcgd2l0aCBETkEgbWljcm8tYXJyYXlzIGZvciBoaWdoLWRlbnNpdHkgbGlua2FnZSBtYXBwaW5nIGFuZCBRVEwgbWFwcGluZyBpbiBjb21tb24gYnVja3doZWF0IChGYWdvcHlydW0gZXNjdWxlbnR1bSBNb2VuY2gpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTI5MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJvYnVzdCwgc2ltcGxlIEdlbm90eXBpbmctYnktU2VxdWVuY2luZyAoR0JTKSBBcHByb2FjaCBmb3IgaGlnaCBkaXZlcnNpdHkgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxOTM3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmctYnktc2VxdWVuY2luZyAoR0JTKSwgYW4gdWx0aW1hdGUgbWFya2VyLWFzc2lzdGVkIHNlbGVjdGlvbiAoTUFTKSB0b29sIHRvIGFjY2VsZXJhdGUgcGxhbnQgYnJlZWRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00ODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBCQUMgbGlicmFyeSBmb3IgYnVja3doZWF0IGdlbm9tZSByZXNlYXJjaOKAlGFuIGFwcGxpY2F0aW9uIHRvIHBvc2l0aW9uYWwgY2xvbmluZyBvZiBhZ3JpY3VsdHVyYWxseSB2YWx1YWJsZSB0cmFpdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIEdlbmV0LiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04MzsgIGNpdGF0aW9uX3BhZ2VzPTM5My00MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgcHJlZGljdGlvbiB3aXRoIGEgaGlkZGVuIE1hcmtvdiBtb2RlbCBhbmQgYSBuZXcgaW50cm9uIHN1Ym1vZGVsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPWlpMjE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TLUxPQ1VTIEVBUkxZIEZMT1dFUklORyAzIGlzIGV4Y2x1c2l2ZWx5IHByZXNlbnQgaW4gdGhlIGdlbm9tZXMgb2Ygc2hvcnQtc3R5bGVkIGJ1Y2t3aGVhdCBwbGFudHMgdGhhdCBleGhpYml0IGhldGVyb21vcnBoaWMgc2VsZi1pbmNvbXBhdGliaWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzEyNjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFsaXR5IGNvbnRyb2wgYW5kIHByZXByb2Nlc3Npbmcgb2YgbWV0YWdlbm9taWMgZGF0YXNldHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9ODYzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3Rpb24gb2YgdGhlIG9jdG9wbG9pZCBzdHJhd2JlcnJ5IGdlbm9tZSBieSBkZWVwIHNlcXVlbmNpbmcgb2YgdGhlIGdlbm9tZXMgb2YgRnJhZ2FyaWEgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNjktODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgaHVtYW4gZ2Vub21lcyB3aXRoIG1hc3NpdmVseSBwYXJhbGxlbCBzaG9ydCByZWFkIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjY1LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2FmZm9sZGluZyBwcmUtYXNzZW1ibGVkIGNvbnRpZ3MgdXNpbmcgU1NQQUNFOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBmYW1pbGllcyBpbiBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbCAxKTsgIGNpdGF0aW9uX3BhZ2VzPWkzNTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBnZW5laWQgdG8gaWRlbnRpZnkgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFByb3RvYy4gQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Q2hhcHRlciA0OyAgY2l0YXRpb25fcGFnZXM9VW5pdCA0LjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1XMTE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHeXBzeSBEYXRhYmFzZSAoR3lEQikgb2YgbW9iaWxlIGdlbmV0aWMgZWxlbWVudHM6IHJlbGVhc2UgMi4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1ENzAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgZ2VuZXJhdGlvbiBvZiBob21vbG9neSBzZWFyY2ggdG9vbHMgYmFzZWQgb24gcHJvYmFiaWxpc3RpYyBpbmZlcmVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBJbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpbW1vbWF0aWM6IGEgZmxleGlibGUgdHJpbW1lciBmb3IgSWxsdW1pbmEgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMTE0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3PigJNXaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU2VxdWVuY2UgQWxpZ25tZW50L01hcCBmb3JtYXQgYW5kIFNBTXRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmcmFtZXdvcmsgZm9yIHZhcmlhdGlvbiBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz00OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2x1c3RhbFcgYW5kIENsdXN0YWxYIHZlcnNpb24gMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yOTQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJhcGlkIGdlbmVyYXRpb24gb2YgbXV0YXRpb24gZGF0YSBtYXRyaWNlcyBmcm9tIHByb3RlaW4gc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIEFwcGwuIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI3NS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTY6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdmVyc2lvbiA2LjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yNzI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzaXplIGFuYWx5c2lzIG9mIHRoZSBnZW51cyBGYWdvcHlydW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZhZ29weXJ1bTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2YgUHJpbXVsYSB2ZXJpcyB5aWVsZHMgaW5zaWdodHMgaW50byB0aGUgbW9sZWN1bGFyIGJhc2lzIG9mIGhldGVyb3N0eWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IHNlcXVlbmNlcyBvZiB0aGUgcmFkaXNoIChSYXBoYW51cyBzYXRpdnVzIEwuKSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CdWNrd2hlYXQgYXMgYSBzb3VyY2Ugb2YgcnV0aW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5NDY7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xOTctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gb2YgZmxhdm9ub2lkIGJpb3N5bnRoZXNpcyBnZW5lcyBhbmQgYWNjdW11bGF0aW9uIG9mIHBoZW5vbGljIGNvbXBvdW5kcyBpbiBjb21tb24gYnVja3doZWF0IChGYWdvcHlydW0gZXNjdWxlbnR1bSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFncmljLiBGb29kIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTIxNzYtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgZmxhdm9ub2lkIGJpb3N5bnRoZXNpcyBnZW5lcyB2aXMtw6AtdmlzIHJ1dGluIGNvbnRlbnQgdmFyaWF0aW9uIGluIGRpZmZlcmVudCBncm93dGggc3RhZ2VzIG9mIEZhZ29weXJ1bSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz0yMTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBiSExIIGFuZCBNWUIgZG9tYWluIHByb3RlaW5zOiBzcGVjaWVzLXNwZWNpZmljIHJlZ3VsYXRvcnkgZGlmZmVyZW5jZXMgYXJlIGNhdXNlZCBieSBkaXZlcmdlbnQgZXZvbHV0aW9uIG9mIHRhcmdldCBhbnRob2N5YW5pbiBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz00NzUtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBUUkFOU1BBUkVOVCBURVNUQSBHTEFCUkExIGxvY3VzLCB3aGljaCByZWd1bGF0ZXMgdHJpY2hvbWUgZGlmZmVyZW50aWF0aW9uIGFuZCBhbnRob2N5YW5pbiBiaW9zeW50aGVzaXMgaW4gQXJhYmlkb3BzaXMsIGVuY29kZXMgYSBXRDQwIHJlcGVhdCBwcm90ZWluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTMzNy00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VFQyLCBUVDgsIGFuZCBUVEcxIHN5bmVyZ2lzdGljYWxseSBzcGVjaWZ5IHRoZSBleHByZXNzaW9uIG9mIEJBTllVTFMgYW5kIHByb2FudGhvY3lhbmlkaW4gYmlvc3ludGhlc2lzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTM2Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiB0aGUgYW50aG9jeWFuaW4gYmlvc3ludGhldGljIHBhdGh3YXkgYnkgdGhlIFRURzEvYkhMSC9NeWIgdHJhbnNjcmlwdGlvbmFsIGNvbXBsZXggaW4gQXJhYmlkb3BzaXMgc2VlZGxpbmdzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTgxNC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVlCIHRyYW5zY3JpcHRpb2ZuYWN0b3JzIGluIHBsYW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSMlIzLU1ZQiBnZW5lIGZhbWlseSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQ0Ny01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVlCIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz01NzMtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZ2h0LWluZHVjZWQgZXhwcmVzc2lvbiBvZiBhIE1ZQiBnZW5lIHJlZ3VsYXRlcyBhbnRob2N5YW5pbiBiaW9zeW50aGVzaXMgaW4gcmVkIGFwcGxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTQyOyAgY2l0YXRpb25fcGFnZXM9MTIxNi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGtNeWI0IGlzIGEgTXliIHRyYW5zY3JpcHRpb24gZmFjdG9yIGludm9sdmVkIGluIHByb2FudGhvY3lhbmlkaW4gYmlvc3ludGhlc2lzIGluIHBlcnNpbW1vbiBmcnVpdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9MjAyOC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gb2YgbXV0YXRpb25zIGluIHdoaXRlLWZsb3dlcmVkIHRvcmVuaWEgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz04NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIGNpcyBlbGVtZW50cyBpbiB0aGUgcHJveGltYWwgcHJvbW90ZXIgcmVnaW9ucyBvZiB0aGUgYW50aG9jeWFuaW4gcGF0aHdheSBnZW5lcyByZXZlYWxzIGEgY29tbW9uIHJlZ3VsYXRvcnkgbG9naWMgdGhhdCBnb3Zlcm5zIHBhdGh3YXkgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9Mzc3NS04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIGEgTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmUgdGhhdCBpcyBhIGtleSByZWd1bGF0b3IgZm9yIHRoZSBkZXZlbG9wbWVudCBvZiByZWQgY29sb3JhdGlvbiBpbiBhcHBsZSBza2luOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9OTU4LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbmdsZS1yZXBlYXQgTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9yLCBHbU1ZQjE3NiwgcmVndWxhdGVzIENIUzggZ2VuZSBleHByZXNzaW9uIGFuZCBhZmZlY3RzIGlzb2ZsYXZvbm9pZCBiaW9zeW50aGVzaXMgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xMDE5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXBzaW4tcmVzaXN0YW50IDE2LWtEIGJ1Y2t3aGVhdCBwcm90ZWluIGlzIGFzc29jaWF0ZWQgd2l0aCBpbW1lZGlhdGUgaHlwZXJzZW5zaXRpdml0eSByZWFjdGlvbiBpbiBwYXRpZW50cyB3aXRoIGJ1Y2t3aGVhdCBhbGxlcmd5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEFyY2guIEFsbGVyZ3kgSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI5OyAgY2l0YXRpb25fcGFnZXM9NDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgMTAta0RhIGJ1Y2t3aGVhdCBtb2xlY3VsZSByZWFjdGl2ZSB0byBhbGxlcmdpYyBwYXRpZW50c+KAmSBJZ0U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFsbGVyZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz01MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5vbG9naWNhbCBjaGFyYWN0ZXJpemF0aW9uIGFuZCBtdXRhdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSByZWNvbWJpbmFudCBwcm90ZWluIEJXcDE2LCBhIG1ham9yIGFsbGVyZ2VuIGluIGJ1Y2t3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gUGhhcm0uIEJ1bGwuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTA3OS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhcmNoZXPigJRmcm9tIGN1cnJlbnQgbW9kZWxzIHRvIGdlbmV0aWMgZW5naW5lZXJpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjIzLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3JtIGFuZCBmdWN0aW9uYWxpdHkgb2Ygc3RhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gb29kIEh5ZHJvY29sbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNTI3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFyY2hlcyBmcm9tIGRpZmZlcmVudCBib3RhbmljYWwgc291cmNlcyBJOiBjb250cmlidXRpb24gb2YgYW15bG9wZWN0aW4gZmluZSBzdHJ1Y3R1cmUgdG8gdGhlcm1hbCBwcm9wZXJ0aWVzIGFuZCBlbnp5bWUgZGlnZXN0aWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FyYm9oeWRyLiBQb2x5bS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz01MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyBzdGFyY2ggZm9yIGZvb2QgYW5kIGluZHVzdHJpYWwgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjEwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW5jdGlvbiBvZiB0aGUgd2F4eSBsb2N1cyBpbiBzdGFyY2ggc3ludGhlc2lzIGluIG1haXplIGVuZG9zcGVybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODMtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRoZSBzb2x1YmxlIHN0YXJjaCBzeW50aGFzZSBhY3Rpdml0aWVzIG9mIG1haXplIGVuZG9zcGVybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MjA1LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gcm9tIGJhY3RlcmlhbCBnbHljb2dlbiB0byBzdGFyY2g6IHVuZGVyc3RhbmRpbmcgdGhlIGJpb2dlbmVzaXMgb2YgdGhlIHBsYW50IHN0YXJjaCBncmFudWxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MjA3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBncmFudWxlIGJvdW5kIHN0YXJjaCBzeW50aGFzZSBJIChHQlNTSSkgZ2VuZSBvZiB0YXJ0YXJ5IGJ1Y2t3aGVhdCAoRmFnb3B5cnVtIHRhdGFyaWN1bSBHYWVydG4uKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MzQ7ICBjaXRhdGlvbl9wYWdlcz0yMjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2lmaWNhdGlvbiBvZiBnZW5lcyBlbmNvZGluZyBncmFudWxlLWJvdW5kIHN0YXJjaCBzeW50aGFzZSBpbiBtb25vY290cyBhbmQgZGljb3RzIGlzIG1hcmtlZCBieSBtdWx0aXBsZSBnZW5vbWUtd2lkZSBkdXBsaWNhdGlvbiBldmVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzAwODgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZWZmZWN0cyBvZiBXeC1BMSwgLUIxIGFuZCAtRDEgcHJvdGVpbiBkZWZpY2llbmNpZXMgb24gYXBwYXJlbnQgYW15bG9zZSBjb250ZW50IGFuZCBzdGFyY2ggcGFzdGluZyBwcm9wZXJ0aWVzIGluIGNvbW1vbiB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0zMi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5oZXJpdGFuY2Ugb2YgbGVuZ3RoIG9mIHN0eWxlIGluIGJ1Y2t3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWdyaWMuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5Mjc7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xODEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbmV0aWNzIG9mIGhldGVyb3N0eWx5OyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW47ICBjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gYW5kIGZ1bmN0aW9uIG9mIGhldGVyb3N0eWx5OyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fcGFnZXM9MTI5LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdWJpcXVpdGluLzI2UyBwcm90ZWFzb21lIHBhdGh3YXksIHRoZSBjb21wbGV4IGxhc3QgY2hhcHRlciBpbiB0aGUgbGlmZSBvZiBtYW55IHBsYW50IHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTM1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBjb21wb25lbnRzIG9mIGEgcHV0YXRpdmUgUGV0dW5pYSBTLUxvY3VzIEYtYm944oCTY29udGFpbmluZyBFMyBsaWdhc2UgY29tcGxleCBpbnZvbHZlZCBpbiBTLVJOYXNl4oCTYmFzZWQgc2VsZi1pbmNvbXBhdGliaWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yNTMxLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2YgcmFyZSBtdXRhdGlvbnMgaW4gcG9wdWxhdGlvbnM6IFRJTExJTkcgYnkgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MTI1Ny02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzMvMjE1LzE3NDYzMTM1L2RzdzAxMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQnVja3doZWF0IChGYWdvcHlydW0gZXNjdWxlbnR1bSBNb2VuY2g7IDJuID0gMnggPSAxNikgaXMgYSBudXRyaXRpb25hbGx5IGRlbnNlIGFubnVhbCBjcm9wIHdpZGVseSBncm93biBpbiB0ZW1wZXJhdGUgem9uZXMuIFRvIGFjY2VsZXJhdGUgbW9sZWN1bGFyIGIiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE3NDU0NjAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkeB4l9mr8/oq9IiMhldu7HKy8KqY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f215%2f1745460" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f215%2f1745460" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199426">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199426">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199428">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199428">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199432">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199432">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199451">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199451">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199484">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199484">4. Conclusion and future perspective</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199486">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199486">5. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199488">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199490">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199490">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199492">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/203/1745366">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/225/1745389">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuo Yasui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed: Tel. +81 75-753-6480. Fax. +81 75-753-6146. E-mail: <a href="mailto:yasyas@kais.kyoto-u.ac.jp" target="_blank">yasyas@kais.kyoto-u.ac.jp</a> (Y.Y.); Tel. +81 76-227-7527. Fax. +81-76-227-7557. E-mail: <a href="mailto:mori@ishikawa-pu.ac.jp" target="_blank">mori@ishikawa-pu.ac.jp</a> (M.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuo+Yasui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasui Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasui Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuo Yasui</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariko Ueno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariko+Ueno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ueno M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ueno M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mariko Ueno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuhiro Matsui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">NARO Kyushu Okinawa Agricultural Research Center</div>, <div class="addr-line">2421 Suya, Koshi, Kumamoto 861-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuhiro+Matsui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsui K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsui K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katsuhiro Matsui</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoyuki Katsube-Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoyuki+Katsube-Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katsube-Tanaka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katsube-Tanaka T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomoyuki Katsube-Tanaka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soo Jung Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kitashirakawa Oiwake-cho, Sakyou-ku, Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soo+Jung+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Soo Jung Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jotaro Aii</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Faculty of Applied Life Science</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Science</div>, <div class="addr-line">Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jotaro+Aii">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aii J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aii J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jotaro Aii</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shingo Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Faculty of Applied Life Science</div>, <div class="insititution">Niigata University of Pharmacy and Applied Life Science</div>, <div class="addr-line">Akiha-ku, Niigata 956-8603</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shingo+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shingo Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masashi Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Research Institute for Bioresources and Biotechnology</div>, <div class="insititution">Ishikawa Prefectural University</div>, <div class="addr-line">308 Suematsu, Nonoichi, Ishikawa 912-8836</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed: Tel. +81 75-753-6480. Fax. +81 75-753-6146. E-mail: <a href="mailto:yasyas@kais.kyoto-u.ac.jp" target="_blank">yasyas@kais.kyoto-u.ac.jp</a> (Y.Y.); Tel. +81 76-227-7527. Fax. +81-76-227-7557. E-mail: <a href="mailto:mori@ishikawa-pu.ac.jp" target="_blank">mori@ishikawa-pu.ac.jp</a> (M.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masashi+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masashi Mori</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Co-first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 215–224, <a href='https://doi.org/10.1093/dnares/dsw012'>https://doi.org/10.1093/dnares/dsw012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745460" href="/dnaresearch/article-pdf/23/3/215/17463135/dsw012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yasuo Yasui, Hideki Hirakawa, Mariko Ueno, Katsuhiro Matsui, Tomoyuki Katsube-Tanaka, Soo Jung Yang, Jotaro Aii, Shingo Sato, Masashi Mori;  Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 215–224, <a href='https://doi.org/10.1093/dnares/dsw012'>https://doi.org/10.1093/dnares/dsw012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745460?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Assembly%20of%20the%20draft%20genome%20of%20buckwheat%20and%20its%20applications%20in%20identifying%20agronomically%20useful%20genes&amp;publicationDate=2016-04-02&amp;volumeNum=23&amp;issueNum=3&amp;author=Yasui%2C%20Yasuo%3B%20Hirakawa%2C%20Hideki&amp;startPage=215&amp;endPage=224&amp;contentId=10.1093%2Fdnares%2Fdsw012&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f215%2f1745460" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199426 id="87199426" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Buckwheat (<em>Fagopyrum esculentum</em> Moench; 2<em>n</em> = 2<em>x</em> = 16) is a nutritionally dense annual crop widely grown in temperate zones. To accelerate molecular breeding programmes of this important crop, we generated a draft assembly of the buckwheat genome using short reads obtained by next-generation sequencing (NGS), and constructed the Buckwheat Genome DataBase. After assembling short reads, we determined 387,594 scaffolds as the draft genome sequence (FES_r1.0). The total length of FES_r1.0 was 1,177,687,305 bp, and the N50 of the scaffolds was 25,109 bp. Gene prediction analysis revealed 286,768 coding sequences (CDSs; FES_r1.0_cds) including those related to transposable elements. The total length of FES_r1.0_cds was 212,917,911 bp, and the N50 was 1,101 bp. Of these, the functions of 35,816 CDSs excluding those for transposable elements were annotated by BLAST analysis. To demonstrate the utility of the database, we conducted several test analyses using BLAST and keyword searches. Furthermore, we used the draft genome as a reference sequence for NGS-based markers, and successfully identified novel candidate genes controlling heteromorphic self-incompatibility of buckwheat. The database and draft genome sequence provide a valuable resource that can be used in efforts to develop buckwheat cultivars with superior agronomic traits<strong>.</strong></p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="buckwheat">buckwheat</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;draft sequence&quot;">draft sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;database usage&quot;">database usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;GBS marker&quot;">GBS marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;heteromorphic self-incompatibility&quot;">heteromorphic self-incompatibility</a></div>                            <h2 scrollto-destination=87199428 id="87199428" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The genomes of model plants, such as <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em> (rice), were fully sequenced by the start of the 21st century, and databases containing chromosomal pseudo-molecules and gene annotation information have subsequently been developed and are widely used as tools and resources for plant genomics and genetics studies. Recently, next-generation sequencing (NGS) has emerged as a powerful technique for analysing the genomes of non-model crops in which few molecular genetic studies have been performed. Genome sequences obtained by NGS can be used to construct databases that contain information of genes inferred from available information of genes in model plants. These genome databases in non-model crops will pave the way for the rapid identification of useful genes for crop breeding, which have already been identified in model plants. These databases will also facilitate the construction of fine genetic maps [based on single nucleotide polymorphism, simple sequence repeat (SSR), and NGS-based markers], and make it possible to identify agronomically important genes by map-based cloning. Thus, genome analyses in various non-model crops are underway, and the genomes of &gt;50 non-model crops have already been sequenced (CoGepedia; <a class="link link-uri" href="https://genomevolution.org/">https://genomevolution.org/</a>). For example, NGS has been used to sequence the genomes of crops that produce beneficial secondary metabolites, such as flavonoid-producing <em>Viburnum trilobum</em> (American cranberry)<sup><a reveal-id="DSW012C1" data-open="DSW012C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and capsaicin-producing <em>Capsicum annuum</em> (hot pepper),<sup><a reveal-id="DSW012C2" data-open="DSW012C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and of crops that are tolerant to environmental stress, such as <em>Setaria italica</em> (foxtail millet)<sup><a reveal-id="DSW012C3" data-open="DSW012C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and <em>Cajanus cajan</em> (pigeonpea),<sup><a reveal-id="DSW012C4" data-open="DSW012C4" class="link link-ref link-reveal xref-bibr">4</a></sup> which grow in semi-arid regions. NGS technology has opened the door to elucidating the molecular mechanisms that control agronomically important traits, and there is much interest in using this technology to analyse the genomes of non-model crops.</p><p>Buckwheat (<em>Fagopyrum esculentum</em> Moench; 2<em>n</em> = 2<em>x</em> = 16) is a widely cultivated annual crop in temperate zones. This nutritionally dense non-model crop contains high levels of starch, protein, flavonoids, and dietary fibre in the grain.<sup><a reveal-id="DSW012C5" data-open="DSW012C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Furthermore, buckwheat flour is gluten-free and can replace wheat flour in a coeliac diet.<sup><a reveal-id="DSW012C6" data-open="DSW012C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Buckwheat, however, has two major defects as a crop. First, its outcrossing nature, caused by heteromorphic self-incompatibility (SI), makes it difficult to produce pure cultivars of buckwheat and to fix useful traits. Second, buckwheat grains contain allergens, which induce anaphylactic reactions in some people.<sup><a reveal-id="DSW012C7" data-open="DSW012C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Improving the nutritional quality of the grain and removing genes responsible for SI and allergens are important breeding objectives in buckwheat, and various genetic molecular marker systems have been developed for this purpose [e.g. amplified fragment length polymorphism (AFLP) markers,<sup><a reveal-id="DSW012C8" data-open="DSW012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> SSR markers,<sup><a reveal-id="DSW012C9" data-open="DSW012C9" class="link link-ref link-reveal xref-bibr">9</a></sup> expressed sequence tag (EST) markers,<sup><a reveal-id="DSW012C10" data-open="DSW012C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and array-based markers<sup><a reveal-id="DSW012C11" data-open="DSW012C11" class="link link-ref link-reveal xref-bibr">11</a></sup>]. However, AFLP markers have not yet been converted to single locus markers in the buckwheat genome. SSR markers have limited utility in buckwheat due to difficulty in amplifying specific loci because of the high level of genetic diversity between buckwheat cultivars, and EST marker systems do not span the entire genome. The newest genome map of buckwheat constructed using array-based markers has sufficient markers to cover the entire genome; however, it requires a specialized instrument to interpret the fluorescence signals of the arrays.<sup><a reveal-id="DSW012C11" data-open="DSW012C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Recently, a versatile NGS-based genotyping method with a low-cost, genotyping-by-sequencing (GBS) marker system was developed.<sup><a reveal-id="DSW012C12" data-open="DSW012C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The GBS system utilizes redundant libraries constructed with PCR fragments that have recognition sites of two kinds of restriction enzymes on both ends. The PCR fragments sequenced using NGS technology are mapped to reference sequences for genome-wide genotyping. The GBS system has been used to genotype various crop species to date.<sup><a reveal-id="DSW012C13" data-open="DSW012C13" class="link link-ref link-reveal xref-bibr">13</a></sup> A draft genome of buckwheat could be used as a reference sequence for developing GBS markers to identify genes that control desirable breeding traits.</p><p>Here, we used NGS-based technology to sequence the buckwheat genome, and constructed the Buckwheat Genome DataBase (BGDB; <a class="link link-uri" href="http://buckwheat.kazusa.or.jp">http://buckwheat.kazusa.or.jp</a>). This database can be used for the rapid detection of homologues of genes previously identified in other plants, and we present three examples of buckwheat genes identified using this approach, i.e. genes controlling flavonoid biosynthesis and genes encoding 2S albumin-type allergens and granule-bound starch synthases (GBSSs). Furthermore, to illustrate that the draft genome can be used as a reference sequence for NGS-based genotyping, we used GBS technology to identify novel candidate genes for controlling heteromorphic SI of buckwheat.</p>                            <h2 scrollto-destination=87199432 id="87199432" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199433 id="87199433" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>A single buckwheat plant with short-styled flowers, a descendant of material used in a previous study to construct a buckwheat BAC library,<sup><a reveal-id="DSW012C14" data-open="DSW012C14" class="link link-ref link-reveal xref-bibr">14</a></sup> was obtained from sib-crossing (BC<sub>1</sub>F<sub>6</sub>). Nuclei were extracted from leaf tissues of the single plant as described previously.<sup><a reveal-id="DSW012C14" data-open="DSW012C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Subsequently DNA was extracted from the nuclei according to a previously described method.<sup><a reveal-id="DSW012C11" data-open="DSW012C11" class="link link-ref link-reveal xref-bibr">11</a></sup> To construct a training set for gene prediction using Augustus 3.0.3,<sup><a reveal-id="DSW012C15" data-open="DSW012C15" class="link link-ref link-reveal xref-bibr">15</a></sup> total RNA was prepared from the anthers of short-styled and long-styled plants, cv ‘KOTO’ using a previously described method.<sup><a reveal-id="DSW012C16" data-open="DSW012C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=87199435 id="87199435" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing of buckwheat</h3>
<p>A paired-end (PE) library with insert sizes of 180–200 bp and a mate-pair (MP) library with insert sizes of 3, 5, 10, and 20 kb were constructed from nuclear DNA according to the manufacture's protocol (Illumina Inc., San Diego, CA, USA). A PE RNA-Seq library with insert sizes of ∼275 bp was also constructed. Sequencing of genomic and RNA-Seq libraries using Illumina HiSeq 2000 was respectively carried out at Hokkaido System Science Co., Ltd and Beijing Genomics Institute. The PE and MP reads were subjected to quality trimming by PRINSEQ 0.20.4,<sup><a reveal-id="DSW012C17" data-open="DSW012C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and further to adaptor trimming by the fastx_clipper program in the FASTX-toolkit 0.0.14 (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a>). The quality value threshold used for quality trimming was 10 from the 3′ terminal, and the adaptor sequence used was ‘AGATCGGAAGAGC’. Then, for the PE library with insert sizes of 180 bp, one base at the 3′ terminal was trimmed from all reads due to low quality, and PE reads shorter than 99 bp and including undetermined nucleotides (Ns) were excluded. For the MP library with insert sizes of 3, 5, and 10 kb, reads shorter than 49 bp and including Ns were excluded, and the 50 bp from the 5′ terminal were used for scaffolding. For the MP library with an insert size of 20 kb, reads shorter than 99 bp and including Ns were excluded, and the 50 bp from the 3′ terminal were used for scaffolding. For the PE RNA-Seq data, reads shorter than 89 bp and including Ns were excluded. The trimmed reads were used for further analyses.</p>                            <h3 scrollto-destination=87199437 id="87199437" class="section-title" data-legacy-id=s2c>2.3. Estimation of genome size</h3>
<p>For genome size estimation, we used PE reads with a k-mer size of 17, as successfully used in a previous study.<sup><a reveal-id="DSW012C18" data-open="DSW012C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The k-mer distribution was investigated using Jellyfish 2.1.3.<sup><a reveal-id="DSW012C19" data-open="DSW012C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The genome size and coverage (i.e. the number of base pairs sequenced as a multiple of the number of base pairs present in the genome) were estimated using the peak at 47 on the k-mer frequency distribution curve (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) according to a previously described method.<sup><a reveal-id="DSW012C18" data-open="DSW012C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=87199439 id="87199439" class="section-title" data-legacy-id=s2d>2.4. Assembly of the buckwheat genome sequences</h3>
<p>The trimmed PE reads were assembled using SOAPdenovo2 rev240<sup><a reveal-id="DSW012C20" data-open="DSW012C20" class="link link-ref link-reveal xref-bibr">20</a></sup> with k-mer sizes of 61, 71, 81, and 91 nt. The option used was –RF –M 1–K [k-mer size]. After the assembly, gaps in scaffolds were closed using GapCloser 1.10 (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>) (<em>P</em>= 31). The trimmed MP reads were used for scaffolding by SSPACE2.0<sup><a reveal-id="DSW012C21" data-open="DSW012C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with parameters –k 5 –x 0 –g 3 –a 0.7. Sequences homologous to bacteria, fungi, and human (hg19) genome sequences, vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>), chloroplast (accession number: NC_000932.1), and mitochondrial (accession number: NC_001284.2) genome sequences from <em>A. thaliana</em>, and the PhiX sequence used in Illumina sequencing by BLASTN<sup><a reveal-id="DSW012C22" data-open="DSW012C22" class="link link-ref link-reveal xref-bibr">22</a></sup> searches with an <em>E</em>-value cut-off of 1E−10 and length coverage of ≥10% were excluded as probable contamination. Finally, scaffolds longer than 300 bp were selected and designated FES_r1.0. Repetitive sequences in FES_r1.0 were detected using RepeatScout 1.0.5<sup><a reveal-id="DSW012C23" data-open="DSW012C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and RepeatMasker 4.0.3 (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) according to a previously described method.<sup><a reveal-id="DSW012C18" data-open="DSW012C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=87199441 id="87199441" class="section-title" data-legacy-id=s2e>2.5. Gene prediction and annotation</h3>
<p>The RNA-Seq reads were mapped onto the draft genome sequence (FES_r1.0) with TopHat 2.0.12.<sup><a reveal-id="DSW012C24" data-open="DSW012C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The bam2hints program installed in Augustus 3.0.3 was used to generate the intronhints.gff file, and Cufflinks was used to reconstruct transcripts in an exonhints.gff file. The two gff files were merged to form a HINTS file. The HINTS file was used as the buckwheat training set to predict genes for Augustus 3.0.3 (Method 1). Furthermore, genes were predicted using Augustus 3.0.2 (Method 2) and geneid 1.4.4<sup><a reveal-id="DSW012C25" data-open="DSW012C25" class="link link-ref link-reveal xref-bibr">25</a></sup> (Method 3) with an <em>A. thaliana</em> training set. Finally, the genes predicted by the three methods were merged.</p><p>The merged genes were subjected to similarity searches against NCBI's NR database (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz">ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz</a>) and amino acid sequences of <em>A. thaliana</em> from TAIR10 (<a class="link link-uri" href="https://www.arabidopsis.org">https://www.arabidopsis.org</a>) using BLASTX with an <em>E</em>-value cut-off of 1E−10. The top hit was used to assign the product name. BLAST searches against UniProt (TrEMBL + Swiss-Prot) with an <em>E</em>-value cut-off of 1E−20 were also carried out. A domain search against InterPro (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>) was conducted using InterProScan<sup><a reveal-id="DSW012C26" data-open="DSW012C26" class="link link-ref link-reveal xref-bibr">26</a></sup> with an <em>E</em>-value cut-off of 1.0. Finally, genes were classified based on NCBI's euKaryotic clusters of Orthologous Groups (KOG) database<sup><a reveal-id="DSW012C27" data-open="DSW012C27" class="link link-ref link-reveal xref-bibr">27</a></sup> by performing BLAST searches with an <em>E</em>-value cut-off of 1E−4.</p><p>Genes related to transposable elements were inferred based on a BLAST search against the NCBI's NR database, and conserved domains were identified based on a search against InterPro and GyDB 2.0<sup><a reveal-id="DSW012C28" data-open="DSW012C28" class="link link-ref link-reveal xref-bibr">28</a></sup> using hmmsearch in HMMER 3.0<sup><a reveal-id="DSW012C29" data-open="DSW012C29" class="link link-ref link-reveal xref-bibr">29</a></sup> with an <em>E</em>-value cut-off of 1.0. Transfer RNA genes (tRNAs) were predicted using tRNAscan-SE v.1.23.<sup><a reveal-id="DSW012C30" data-open="DSW012C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Ribosomal RNA genes (rRNAs) were predicted in BLASTN searches with an <em>E</em>-value cut-off of 1E−10 using <em>A. thaliana</em> 5.8S and 25S rRNAs (accession number: X52320.1) and 18S rRNA (accession number: X16077.1) as queries.</p>                            <h3 scrollto-destination=87199445 id="87199445" class="section-title" data-legacy-id=s2f>2.6. Database construction</h3>
<p>The draft genome sequence (FES_r1.0), predicted gene sequences, deduced amino acid sequences, annotations derived from BLAST searches against the TAIR10 and NCBI's NR databases, and domains identified in the search against InterPro were included in the BGDB. In addition, local BLAST searches and keywords searches for gene names and their annotations were also implemented in the BGDB.</p>                            <h3 scrollto-destination=87199447 id="87199447" class="section-title" data-legacy-id=s2g>2.7. Genotyping-by-sequencing and detection of S-allele-specific sites</h3>
<p>Total DNA extracted from 18 short-styled and 18 long-styled buckwheat landraces from around the world (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) was used for GBS analysis. GBS was carried out according to Elshire et al.,<sup><a reveal-id="DSW012C12" data-open="DSW012C12" class="link link-ref link-reveal xref-bibr">12</a></sup> except that <em>EcoR</em>I and <em>Mse</em>I were used as restriction enzymes. Barcode adaptors are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. Barcode-labeled amplicons were sequenced by Illumina HiSeq 2000 at Hokkaido System Science Co., Ltd. The PE reads were subjected to quality and adaptor trimming by Trimmomatic 0.3.2.<sup><a reveal-id="DSW012C31" data-open="DSW012C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The quality value threshold used for quality trimming was 25 with a window size of 5, and the adaptor sequences used were ‘CACGACGCTCTTCCGATCT’ and ‘ACCGCTCTTCCGATCTGTAA’. Then, PE reads longer than 39 bp were aligned to reference sequences (FES_r1.0) using BWA 0.7.9,<sup><a reveal-id="DSW012C32" data-open="DSW012C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and the mapping results were processed with SAMtools 0.1.18.<sup><a reveal-id="DSW012C33" data-open="DSW012C33" class="link link-ref link-reveal xref-bibr">33</a></sup> To minimize mismatching bases across all the reads, local realignment procedure was carried out using RealignerTargetCreator and IndelRealigner in GATK 3.4.<sup><a reveal-id="DSW012C34" data-open="DSW012C34" class="link link-ref link-reveal xref-bibr">34</a></sup> All the sites on reference sequences that mapped with reads were extracted and combined in a variant call format file using the UnifiedGenotyper in GATK 3.4 with the option of -out_mode EMIT_ALL_CONFIDENT_SITES. Sites at which &gt;50 reads were mapped in long-styled plants but not in short-styled plants were defined as ‘non-SS’. Likewise, sites at which &gt;50 reads were mapped in short-styled plants but not in long-styled plants were defined as ‘non-LS’. Then, the number of short-styled plants with mapped reads at each non-LS site and the number of long-styled plants with mapped reads at each non-SS site were counted. Non-LS sites shared by &gt;10 short-styled plants were regarded as <em>S</em>-allele-specific sites (see Results and discussion). Then, scaffolds harbouring &gt;39 <em>S</em>-allele-specific sites were regarded as <em>S</em>-allelic scaffolds.</p>                            <h3 scrollto-destination=87199449 id="87199449" class="section-title" data-legacy-id=s2h>2.8. Phylogenetic analyses</h3>
<p>Alignments of amino acid sequences were carried out using CLUSTALW,<sup><a reveal-id="DSW012C35" data-open="DSW012C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and the neighbor-joining (NJ) trees<sup><a reveal-id="DSW012C36" data-open="DSW012C36" class="link link-ref link-reveal xref-bibr">36</a></sup> were obtained from pairwise distances corrected by the JTT model.<sup><a reveal-id="DSW012C37" data-open="DSW012C37" class="link link-ref link-reveal xref-bibr">37</a></sup> These analyses were conducted using MEGA6.<sup><a reveal-id="DSW012C38" data-open="DSW012C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p>                            <h2 scrollto-destination=87199451 id="87199451" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87199452 id="87199452" class="section-title" data-legacy-id=s3a>3.1. Genome assembly of buckwheat</h3>
<p>The k-mer frequency distribution curve (k-mer = 17) using PEs with 180 and 200 bp is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>. Based on this curve, the genome size of buckwheat was estimated to be between 1,212,021,130 and 2,424,042,260 bp using peaks at a multiplicity of 94 (coverage = 111.9) and 47 (coverage = 56.0), respectively. The genomic DNA used in this study is expected to contain heterozygous regions, due to the outcrossing nature of buckwheat; however, we used sib-mating descendant plants as material to reduce heterozygous genomic regions. The haploid genome size of 1.2 Gb calculated based on the major peak (multiplicity = 94) is almost the same as that estimated from cytometry analyses (1.34 Gb).<sup><a reveal-id="DSW012C39" data-open="DSW012C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>The numbers of raw and trimmed reads are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. The trimmed reads with k-mer sizes of 61, 71, 81, and 91 nt were assembled using SOAPdenovo2. The N50 values of the assemblies using k-mer sizes of 61, 71, 81, and 91 nt were, respectively, 1,388, 1,419, 1,350, and 770 bp. The longest scaffolds, i.e. those assembled with a k-mer size of 71, were used for further analysis. Gaps in the contigs were closed using GapCloser 1.10 (<a class="link link-uri" href="http://soap.genomics.org.cn">http://soap.genomics.org.cn</a>), and mate-pair reads were used for scaffolding in SSPACE2.0. The 2,693,661 scaffolds that were shorter than 299 bp and the 1,908 scaffolds that exhibited signs of contamination (identified in a BLAST search) were excluded, and the remaining 387,594 scaffolds were designated as the draft genome sequence, FES_r1.0 (Table <a reveal-id="DSW012TB1" data-open="DSW012TB1" class="link link-reveal link-table xref-fig">1</a>). The total length of FES_r1.0 was 1,177,687,305 bp, and the N50 length was 25,109 bp. The scaffolds were named ‘Fes_sc’ followed by a six-digit identifier and the sequence version (e.g. Fes_sc000001.1). </p>                        <a id="87199455" scrollto-destination="87199455"></a>
<div content-id="DSW012TB1" class="table-modal"><div class="table-wrap"><div id="DSW012TB1" class="table-wrap-title" data-id="DSW012TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the draft genome sequence (FES_r1.0)</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of sequences </td><td>387,594 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td>1,177,687,305 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td>3,038 </td></tr><tr><td>Max length of sequences (bases) </td><td>1,053,114 </td></tr><tr><td>Min length of sequences (bases) </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>25,109 </td></tr><tr><td>Number of undetermined bases </td><td>309,030,247 </td></tr><tr><td>G ± C% (GC/ATGC) </td><td>39.1 </td></tr></tbody></table></div><div content-id="DSW012TB1" class="table-modal"><table><tbody><tr><td>Number of sequences </td><td>387,594 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td>1,177,687,305 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td>3,038 </td></tr><tr><td>Max length of sequences (bases) </td><td>1,053,114 </td></tr><tr><td>Min length of sequences (bases) </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>25,109 </td></tr><tr><td>Number of undetermined bases </td><td>309,030,247 </td></tr><tr><td>G ± C% (GC/ATGC) </td><td>39.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199455" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW012TB1" class="table-wrap-title" data-id="DSW012TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the draft genome sequence (FES_r1.0)</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of sequences </td><td>387,594 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td>1,177,687,305 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td>3,038 </td></tr><tr><td>Max length of sequences (bases) </td><td>1,053,114 </td></tr><tr><td>Min length of sequences (bases) </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>25,109 </td></tr><tr><td>Number of undetermined bases </td><td>309,030,247 </td></tr><tr><td>G ± C% (GC/ATGC) </td><td>39.1 </td></tr></tbody></table></div><div content-id="DSW012TB1" class="table-modal"><table><tbody><tr><td>Number of sequences </td><td>387,594 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td>1,177,687,305 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td>3,038 </td></tr><tr><td>Max length of sequences (bases) </td><td>1,053,114 </td></tr><tr><td>Min length of sequences (bases) </td><td>300 </td></tr><tr><td>N50 length (bases) </td><td>25,109 </td></tr><tr><td>Number of undetermined bases </td><td>309,030,247 </td></tr><tr><td>G ± C% (GC/ATGC) </td><td>39.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199455" target="_blank">View Large</a></div></div><p>Considering the genome size, the total length of the assembled genome sequence was close to the estimated size; therefore, the draft genome sequence (FES_r1.0) was considered as the haploid genome sequence. The draft genome sequence spanned 98.3% of the genome size estimated in the k-mer frequency distribution analysis, and 88.1% of that estimated by flow cytometry analysis.<sup><a reveal-id="DSW012C39" data-open="DSW012C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The high coverage rates, low N50 value (25,109 bp), and large total number of scaffolds (387,594) obtained in the present study may be due to the high proportion of heterozygous genomic regions present in the plant materials used, as indicated by the k-mer frequency distribution curve (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Long-read data generated by PacBio RS was found to increase N50 and reduce the total number of scaffolds in the draft genome of <em>Primula veris</em> (cowslip), a heterozygous plant.<sup><a reveal-id="DSW012C40" data-open="DSW012C40" class="link link-ref link-reveal xref-bibr">40</a></sup> A study of <em>Raphanus sativus</em> (radish)<sup><a reveal-id="DSW012C41" data-open="DSW012C41" class="link link-ref link-reveal xref-bibr">41</a></sup> indicated that the lengths of scaffolds assembled based only on short-read data were drastically increased by constructing super-scaffolds with SSPACE2.0 using BAC-end sequences. We have already constructed a BAC library<sup><a reveal-id="DSW012C14" data-open="DSW012C14" class="link link-ref link-reveal xref-bibr">14</a></sup> using parental plants of the material used in this study. Long reads generated by PacBio RS and BAC-end sequencing will be used to expand the scaffold length of the present draft genome of buckwheat.</p>                            <h3 scrollto-destination=87199457 id="87199457" class="section-title" data-legacy-id=s3b>3.2. Gene prediction and annotation</h3>
<p>Gene predictions were performed using Augustus 3.0.3 with the buckwheat training set (Method 1), Augustus 3.0.2 with the <em>A. thaliana</em> training set (Method 2), or geneid with the <em>A. thaliana</em> training set (Method 3), and the results obtained using the three methods (Methods 1–3) are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. If the genes were located at the same locus when using Methods 1–3, the longest gene was selected. After the results were integrated, the total length of the CDSs (FES_r1.0_cds) was 212,917,911 bp composed of 286,768 CDSs, and N50 was 1,101 bp. The gene name was prefixed with a six-digit identifier followed by the prediction method and scaffold number (i.e. auf: Augustus 3.0.3, buckwheat training set, Method 1; aua: Augustus 3.0.2, <em>A. thaliana</em> training set, Method 2; and gia, geneid 1.4.4, <em>A. thaliana</em> training set, Method 3), as in the following example: Fes_sc0012271.1.g000001.aua.1.</p><p>Genes related to transposable elements (TEs) were inferred according to BLAST searches against the NCBI's NR database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The total length of known repeats was 133,362,886 bp (11.32% of FES_r1.0, i.e. 1,177,687,305 bp) and Class I long terminal repeat (LTR) elements were frequently found (8.79% of FES_r1.0). We identified unique repeats that had not previously been sequenced in this analysis, and these had a total length of 475,367,120 bp and accounted for 40.36% of FES_1.0. Genes annotated as transposons were tagged ‘TE’ in the database.</p><p>Based on BLAST searches against the NR database, the genes were tagged as intrinsic (including both of a start codon and stop codon), partial (including a start codon or stop codon, or lacking both start and stop codons), pseudo (pseudogenes; including a stop codon in the coding region), and short (&lt;49 amino acids). Based on BLAST searches against UniProt (TrEMBL + Swiss-Prot) with an E-value cut-off of 1E−20, the genes were tagged ‘f’ (hit region of ≥70% in query length), ‘p’ (hit region of &lt;70% in query length), or ‘n’ (no hits against UniProt). The tags assigned to the genes are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>; 35,816 predicted genes tagged as full length or partial were annotated by BLAST searches against NR and/or UniProt. The predicted genes were classified based on NCBI's KOG database for <em>F. esculentum</em>, <em>Beta vulgaris</em> (which is classified in the same order, Caryophyllales, as <em>F. esculentum</em>), and <em>A. thaliana</em> (the most advanced model plant). KOGs for the predicted genes in the three species were assigned to the 25 functional categories, which were classified into four large groups (Groups 1–4; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), and the percentage of KOGs in each category was calculated for each plant species. Figure <a reveal-id="DSW012F1" data-open="DSW012F1" class="link link-reveal link-table xref-fig">1</a> shows the percentage of KOGs in Groups 1–3. Note that the number of KOGs assigned to the functional categories of N (cell motility) in Group 2 was &lt;10 for all three species and was excluded in Fig. <a reveal-id="DSW012F1" data-open="DSW012F1" class="link link-reveal link-table xref-fig">1</a>. The distribution of KOGs in each large group was similar among these species. For instance, KOGs from all three species were enriched in categories K, O, and G, and were poor in categories B, W, and H. In addition to the protein-coding genes, we identified 1,374 genes for tRNAs in total, and the numbers of genes for each tRNA are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>. </p>                        <a id="87199461" scrollto-destination="87199461"></a>
<div id="DSW012F1" data-id="DSW012F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01201.png?Expires=2147483647&Signature=HtUBqUclZBSgzibcCaY~CN14SqniWsHXfmsg9JzXqLXiLTYn0Ow18jL9xtpA8jtxCIqWR7PaIMkLspt0pPWdep4qwd9PzolkxJheCEgHf1Xu5lAAUicKZNdIaXe8ptsk13UARqqeP8~L33HUQ5kX1wnldEKpkhtU29kQ8RKi3JJOD49PB3d4NZ6ho7huKInooRfIWKRDfRFsXw9AQjwEdFkpZdUOunaEdAOVPjmOqjPGGAZ5aFaUZVojAv2rbwmWyc0VMHxnBDtESA70Vn5LeE1YFaGifY6TTt6ryoPE0HX2T~35HGxHwHoUtpSi0YAPCgrUdTz24miJhE7jTEm2Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Assignment of proteins to KOG functional categories in the three plant species. Genes from F. esculentum (red), B. vulgaris (blue), and A. thaliana (green) were classified based on NCBI's KOG database by performing BLAST searches with an E-value cut-off of 1E−4. KOGs were classified into functional categories. The percentage of KOGs in each functional category is plotted, and percentages are arranged in ascending order within each group. (A) RNA processing and modification; (B) chromatin structure and dynamics; (C) energy production and conversion; (D) cell cycle control, cell division, and chromosome partitioning; (E) amino acid transport and metabolism; (F) nucleotide transport and metabolism; (G) carbohydrate transport and metabolism; (H) coenzyme transport and metabolism; (I) lipid transport and metabolism; (J) translation, ribosomal structure, and biogenesis; (K) transcription; (L) replication, recombination, and repair; (M) cell wall/membrane/envelope biogenesis; (O) posttranslational modification, protein turnover, and chaperones; (P) inorganic ion transport and metabolism; (Q) secondary metabolites biosynthesis, transport, and catabolism; (R) general function prediction only; (S) function unknown; (T) signal transduction mechanisms; (U) intracellular trafficking, secretion, and vesicular transport; (V) defense mechanisms; (W) extracellular structures; (Y) nuclear structure; and (Z) cytoskeleton. Note that KOGs in Groups 1–3 are shown, and that fewer than 10 KOGs were assigned to category N (cell motility) in the three species and were excluded." path-from-xml="dsw01201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199461/dsw01201.tiff" path-from-xml="dsw01201.tiff" target="_blank">View large</a><a section="87199461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01201.png?Expires=2147483647&Signature=DfR27MWLhM4hO~jCz4t9pvtyjXkrRe4~CAfIPS8QUr4QwrI2Obr0Li0HLG28Hm-EhUhAipbDqPS5KKmKOyMBoWPI079zIlaHU9DUKJhtJ64LKwIj3thw7hXL2g0Vn6KujK9uX1Nx-3tgaTXtOLPuWEMr3dzJSwpCFEFHT7jTSD7sZIAO4BQJTkZhmstLEq-pN4Mk~JEbWBrBuwLyMpphtZpRecmmlSTerGY-wfByJtsyn-UAum4uNJNK1PSEq5iN8XhMDDGZC3nOPC42f3-ZZkfzVCX0kea9Rvyp5jf9LMCEK12mUY-RcjmKzKP-Va0qp725HXMWeM7iAwoUA0qNLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199461&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Assignment of proteins to KOG functional categories in the three plant species. Genes from <em>F. esculentum</em> (red), <em>B. vulgaris</em> (blue), and <em>A. thaliana</em> (green) were classified based on NCBI's KOG database by performing BLAST searches with an <em>E</em>-value cut-off of 1E−4. KOGs were classified into functional categories. The percentage of KOGs in each functional category is plotted, and percentages are arranged in ascending order within each group. (A) RNA processing and modification; (B) chromatin structure and dynamics; (C) energy production and conversion; (D) cell cycle control, cell division, and chromosome partitioning; (E) amino acid transport and metabolism; (F) nucleotide transport and metabolism; (G) carbohydrate transport and metabolism; (H) coenzyme transport and metabolism; (I) lipid transport and metabolism; (J) translation, ribosomal structure, and biogenesis; (K) transcription; (L) replication, recombination, and repair; (M) cell wall/membrane/envelope biogenesis; (O) posttranslational modification, protein turnover, and chaperones; (P) inorganic ion transport and metabolism; (Q) secondary metabolites biosynthesis, transport, and catabolism; (R) general function prediction only; (S) function unknown; (T) signal transduction mechanisms; (U) intracellular trafficking, secretion, and vesicular transport; (V) defense mechanisms; (W) extracellular structures; (Y) nuclear structure; and (Z) cytoskeleton. Note that KOGs in Groups 1–3 are shown, and that fewer than 10 KOGs were assigned to category N (cell motility) in the three species and were excluded.</p></div></div><div content-id="DSW012F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01201.png?Expires=2147483647&Signature=HtUBqUclZBSgzibcCaY~CN14SqniWsHXfmsg9JzXqLXiLTYn0Ow18jL9xtpA8jtxCIqWR7PaIMkLspt0pPWdep4qwd9PzolkxJheCEgHf1Xu5lAAUicKZNdIaXe8ptsk13UARqqeP8~L33HUQ5kX1wnldEKpkhtU29kQ8RKi3JJOD49PB3d4NZ6ho7huKInooRfIWKRDfRFsXw9AQjwEdFkpZdUOunaEdAOVPjmOqjPGGAZ5aFaUZVojAv2rbwmWyc0VMHxnBDtESA70Vn5LeE1YFaGifY6TTt6ryoPE0HX2T~35HGxHwHoUtpSi0YAPCgrUdTz24miJhE7jTEm2Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Assignment of proteins to KOG functional categories in the three plant species. Genes from F. esculentum (red), B. vulgaris (blue), and A. thaliana (green) were classified based on NCBI's KOG database by performing BLAST searches with an E-value cut-off of 1E−4. KOGs were classified into functional categories. The percentage of KOGs in each functional category is plotted, and percentages are arranged in ascending order within each group. (A) RNA processing and modification; (B) chromatin structure and dynamics; (C) energy production and conversion; (D) cell cycle control, cell division, and chromosome partitioning; (E) amino acid transport and metabolism; (F) nucleotide transport and metabolism; (G) carbohydrate transport and metabolism; (H) coenzyme transport and metabolism; (I) lipid transport and metabolism; (J) translation, ribosomal structure, and biogenesis; (K) transcription; (L) replication, recombination, and repair; (M) cell wall/membrane/envelope biogenesis; (O) posttranslational modification, protein turnover, and chaperones; (P) inorganic ion transport and metabolism; (Q) secondary metabolites biosynthesis, transport, and catabolism; (R) general function prediction only; (S) function unknown; (T) signal transduction mechanisms; (U) intracellular trafficking, secretion, and vesicular transport; (V) defense mechanisms; (W) extracellular structures; (Y) nuclear structure; and (Z) cytoskeleton. Note that KOGs in Groups 1–3 are shown, and that fewer than 10 KOGs were assigned to category N (cell motility) in the three species and were excluded." path-from-xml="dsw01201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199461/dsw01201.tiff" path-from-xml="dsw01201.tiff" target="_blank">View large</a><a section="87199461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01201.png?Expires=2147483647&Signature=DfR27MWLhM4hO~jCz4t9pvtyjXkrRe4~CAfIPS8QUr4QwrI2Obr0Li0HLG28Hm-EhUhAipbDqPS5KKmKOyMBoWPI079zIlaHU9DUKJhtJ64LKwIj3thw7hXL2g0Vn6KujK9uX1Nx-3tgaTXtOLPuWEMr3dzJSwpCFEFHT7jTSD7sZIAO4BQJTkZhmstLEq-pN4Mk~JEbWBrBuwLyMpphtZpRecmmlSTerGY-wfByJtsyn-UAum4uNJNK1PSEq5iN8XhMDDGZC3nOPC42f3-ZZkfzVCX0kea9Rvyp5jf9LMCEK12mUY-RcjmKzKP-Va0qp725HXMWeM7iAwoUA0qNLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199461&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Assignment of proteins to KOG functional categories in the three plant species. Genes from <em>F. esculentum</em> (red), <em>B. vulgaris</em> (blue), and <em>A. thaliana</em> (green) were classified based on NCBI's KOG database by performing BLAST searches with an <em>E</em>-value cut-off of 1E−4. KOGs were classified into functional categories. The percentage of KOGs in each functional category is plotted, and percentages are arranged in ascending order within each group. (A) RNA processing and modification; (B) chromatin structure and dynamics; (C) energy production and conversion; (D) cell cycle control, cell division, and chromosome partitioning; (E) amino acid transport and metabolism; (F) nucleotide transport and metabolism; (G) carbohydrate transport and metabolism; (H) coenzyme transport and metabolism; (I) lipid transport and metabolism; (J) translation, ribosomal structure, and biogenesis; (K) transcription; (L) replication, recombination, and repair; (M) cell wall/membrane/envelope biogenesis; (O) posttranslational modification, protein turnover, and chaperones; (P) inorganic ion transport and metabolism; (Q) secondary metabolites biosynthesis, transport, and catabolism; (R) general function prediction only; (S) function unknown; (T) signal transduction mechanisms; (U) intracellular trafficking, secretion, and vesicular transport; (V) defense mechanisms; (W) extracellular structures; (Y) nuclear structure; and (Z) cytoskeleton. Note that KOGs in Groups 1–3 are shown, and that fewer than 10 KOGs were assigned to category N (cell motility) in the three species and were excluded.</p></div></div><p>Consequently, we were able to identify &gt;35,000 annotated genes including genes for tRNA. Many of them were classified into similar functional categories as in the other two plant species. The BGDB constructed in this study on the basis of FES_r1.0 is expected to serve as a useful tool for identifying genes to develop buckwheat cultivars with improved agronomic properties. The BGDB is available from the Kazusa DNA Research Institute (<a class="link link-uri" href="http://buckwheat.kazusa.or.jp">http://buckwheat.kazusa.or.jp</a>). To demonstrate the utility of this database, we then conducted four test analyses focusing on agronomically important genes.</p>                            <h4 scrollto-destination=87199463 id="87199463" class="section-title" data-legacy-id=s3b1>3.2.1. Example I: identifying buckwheat genes that regulate flavonoid biosynthesis</h4>
<p>Buckwheat contains several kinds of flavonoids, such as flavonols, proanthocyanidins, and anthocyanins. The flavonol rutin is present at high levels in buckwheat seeds<sup><a reveal-id="DSW012C42" data-open="DSW012C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and seems to be beneficial for human health. Several genes encoding enzymes related to flavonoid biosynthesis in buckwheat have been reported<sup><a reveal-id="DSW012C43" data-open="DSW012C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSW012C44" data-open="DSW012C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and are presumed to be regulated by transcription factors (TFs), such as MYB, bHLH, and WD40, as in other plant species.<sup><a reveal-id="DSW012C45  DSW012C48" data-open="DSW012C45  DSW012C48" class="link link-ref link-reveal xref-bibr">45–48</a></sup> However, little is known about such TFs in buckwheat. The R2R3-MYB TFs are thought to play central roles in plant-specific processes, based on their specific gene expression patterns.<sup><a reveal-id="DSW012C49  DSW012C51" data-open="DSW012C49  DSW012C51" class="link link-ref link-reveal xref-bibr">49–51</a></sup> To provide an overview of genes that regulate plant-specific processes, including flavonoid synthesis in buckwheat, we searched for candidate genes encoding R2R3-MYB TFs using the BGDB.</p><p>By conducting a keyword search using the term ‘MYB’, we identified 274 genes predicted to encode MYB TFs. From these, we excluded partial sequences, pseudogenes, and genes that did not contain fully conserved R2R3 regions. The remaining 71 putative R2R3-MYB TFs obtained from the database are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. Phylogenetic analyses based on the R2R3 domain often reveals functionally characterized groups that are present in a wide range of plant species.<sup><a reveal-id="DSW012C52" data-open="DSW012C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSW012C53" data-open="DSW012C53" class="link link-ref link-reveal xref-bibr">53</a></sup> In the present study, six putative R2R3-MYBs were assigned within known functional groups consisting of representatives from other plant species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Though functional analyses would need to be conducted to determine the role of each gene, this finding shows that R2R3-MYB genes, which likely have different roles, can successfully be obtained from the BGDB.</p><p>To initiate the transcription of genes encoding enzymes in the flavonoid biosynthetic pathway, a TF such as MYB or the MYB-bHLH-WD40 (MBW) complex must bind to TF binding sites (TFBSs) in the promoter region of each gene. Mutation of TFBSs alters the expression of genes.<sup><a reveal-id="DSW012C54" data-open="DSW012C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSW012C55" data-open="DSW012C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Therefore, MYB TFs as well as the TFBSs of target genes can be manipulated to improve flavonoid production. To identify promoter sequences in a non-model plant species, genome walking, which is time-consuming and expensive, would usually be performed. In this study, we tried to identify the promoter sequences of genes in the flavonoid biosynthetic pathway and estimated the TFBSs using PLACE (<a class="link link-uri" href="http://www.dna.affrc.go.jp/PLACE/">http://www.dna.affrc.go.jp/PLACE/</a>). cDNA sequences of nine genes in the pathway have been already registered in GenBank. For three of these genes (chalcone isomerase, CHI; flavonoid 3′-hydroxylase, F3′H; anthocyanidin synthase, ANS), we successfully determined the 1,000–2,000 bp upstream region after a BLASTN search against the BGDB. DNA motifs relating to the MYB or MBW complex predicted by PLACE analysis are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. These results can be confirmed by molecular techniques such as gel-shift assays, as reported for other plant species (e.g. apple,<sup><a reveal-id="DSW012C56" data-open="DSW012C56" class="link link-ref link-reveal xref-bibr">56</a></sup> persimmon,<sup><a reveal-id="DSW012C53" data-open="DSW012C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and soybean<sup><a reveal-id="DSW012C57" data-open="DSW012C57" class="link link-ref link-reveal xref-bibr">57</a></sup>). The BGDB is thus a powerful tool for isolating promoter sequences and accelerating molecular-based analyses of TFs. In this study, however, we could not identify sufficiently long promoter regions (over 1,000 bp) for the remaining six genes, mainly because of gaps between contigs. Gap closing using long reads generated by a PacBio sequencer will greatly improve the ability to search for the promoter regions of target genes.</p>                            <h4 scrollto-destination=87199467 id="87199467" class="section-title" data-legacy-id=s3b2>3.2.2. Example II: identifying a duplicate of a buckwheat allergen gene, Fag e 2</h4>
<p>Buckwheat seeds contain allergens. For instance, Fag e 2 (16 kDa protein) is a pepsin-resistant 2S albumin that causes an immediate allergic reaction.<sup><a reveal-id="DSW012C58" data-open="DSW012C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Although <em>Fag e 2</em> cDNA has been sequenced,<sup><a reveal-id="DSW012C58" data-open="DSW012C58" class="link link-ref link-reveal xref-bibr">58</a></sup> no further genomic information is available. Efforts to develop hypoallergenic buckwheat and establish inspection techniques to minimize allergen contamination in food products require detailed genomic information of <em>Fag e 2</em>.</p><p>A BLAST search of Fag e 2 (accession number: DQ304682) among the predicted proteins in the BGDB yielded one identical gene (Fes_sc0000087.1.g000014.aua.1) and four homologues (Fes_sc0000087.1.g000011.aua.1, Fes_sc0000087.1.g000013.aua.1, Fes_sc0000087.1.g000028.aua.1, and Fes_sc0007211.1.g000003.aua.1) (Table <a reveal-id="DSW012TB2" data-open="DSW012TB2" class="link link-reveal link-table xref-fig">2</a>). The results of a BLAST search against the NCBI's NR database indicated high similarities of four homologues with previously reported allergens of buckwheat. As shown in Fig. <a reveal-id="DSW012F2" data-open="DSW012F2" class="link link-reveal link-table xref-fig">2</a>, the predicted amino acid sequences of Fes_sc0000087.1.g000011.aua.1 and Fes_sc0000087.1.g000028.aua.1 showed high levels of similarity (97 and 98%, respectively) with the buckwheat 8 kDa allergen, which is a member of the 2S-albumin multi-gene family.<sup><a reveal-id="DSW012C59" data-open="DSW012C59" class="link link-ref link-reveal xref-bibr">59</a></sup> In contrast, the predicted proteins of Fes_sc0007211.1.g000003.aua.1 and Fes_sc0000087.1.g000013.aua.1 did not have high levels of amino acid sequence similarity (43 and 79%, respectively) with known buckwheat allergens. Fes_sc0007211.1.g000003.aua.1 had a 55-amino acid deletion at the N terminal and lacked four of eight characteristic Cys residues present in Fag e 2 and 2S albumin family.<sup><a reveal-id="DSW012C60" data-open="DSW012C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Thus, the gene product is not likely to be allergenic. On the other hand, Fes_sc0000087.1.g000013.aua.1, which shows similarity with Fag e 2, might be a novel allergen, because its predicted protein retained the eight characteristic Cys residues. Further immunoblotting analysis will clarify whether or not the protein encoded by Fes_sc0000087.1.g000013.aua.1 is allergenic. It is notable that four genes that retain conserved Cys residues are estimated to be located within a genomic region of 108 kb on single scaffold (Fes_sc0000087.1). Particularly, the <em>Fag e 2</em> gene (Fes_sc0000087.1.g000014.aua.1), and the Fes_sc0000087.1.g000011.aua.1 and Fes_sc0000087.1.g000013.aua.1 homologues, which have similarities with buckwheat allergens, are located within a 17 kb region of the scaffold. Therefore, this region would be a prime candidate for silencing in studies aimed at producing hypoallergenic buckwheat. </p>                        <a id="87199470" scrollto-destination="87199470"></a>
<div content-id="DSW012TB2" class="table-modal"><div class="table-wrap"><div id="DSW012TB2" class="table-wrap-title" data-id="DSW012TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Fag e 2 and its homologues obtained by BLASTP search for buckwheat genome database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID of the BGDB </th><th>Scaffold ID </th><th>Similarity with reported allergen of <em>F. esculentum</em> </th><th><em>E</em>-value </th></tr></thead><tbody><tr><td>Fes_sc0000087.1.g000011.aua.1 </td><td>Fes_sc0000087.1 </td><td>97% (BW 8 kDa allergen protein) </td><td>8e−74 </td></tr><tr><td>Fes_sc0000087.1.g000013.aua.1 </td><td>Fes_sc0000087.1 </td><td>79% (Fag e 2) </td><td>3e−26 </td></tr><tr><td>Fes_sc0000087.1.g000014.aua.1 </td><td>Fes_sc0000087.1 </td><td>100% (Fag e 2) </td><td>7e−104 </td></tr><tr><td>Fes_sc0000087.1.g000028.aua.1 </td><td>Fes_sc0000087.1 </td><td>98% (BW 8 kDa allergen protein) </td><td>5e−85 </td></tr><tr><td>Fes_sc0007211.1.g000003.aua.1 </td><td>Fes_sc0007211.1 </td><td>43% (BW 8 kDa allergen protein) </td><td>4e−21 </td></tr></tbody></table></div><div content-id="DSW012TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID of the BGDB </th><th>Scaffold ID </th><th>Similarity with reported allergen of <em>F. esculentum</em> </th><th><em>E</em>-value </th></tr></thead><tbody><tr><td>Fes_sc0000087.1.g000011.aua.1 </td><td>Fes_sc0000087.1 </td><td>97% (BW 8 kDa allergen protein) </td><td>8e−74 </td></tr><tr><td>Fes_sc0000087.1.g000013.aua.1 </td><td>Fes_sc0000087.1 </td><td>79% (Fag e 2) </td><td>3e−26 </td></tr><tr><td>Fes_sc0000087.1.g000014.aua.1 </td><td>Fes_sc0000087.1 </td><td>100% (Fag e 2) </td><td>7e−104 </td></tr><tr><td>Fes_sc0000087.1.g000028.aua.1 </td><td>Fes_sc0000087.1 </td><td>98% (BW 8 kDa allergen protein) </td><td>5e−85 </td></tr><tr><td>Fes_sc0007211.1.g000003.aua.1 </td><td>Fes_sc0007211.1 </td><td>43% (BW 8 kDa allergen protein) </td><td>4e−21 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Scaffold ID, ID number of the scaffold in which the predicted gene is situated; BW 8 kDa, buckwheat 8 kDa allergen of 2S albumin; Fag e 2, buckwheat 16 kDa allergen of 2S albumin.</p></div></div><div content-id="tblfn2"><div class="fn"><p>The GenBank accession numbers of BW 8 kDa and Fag e 2 are AB055892 and DQ304682, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199470" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW012TB2" class="table-wrap-title" data-id="DSW012TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Fag e 2 and its homologues obtained by BLASTP search for buckwheat genome database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID of the BGDB </th><th>Scaffold ID </th><th>Similarity with reported allergen of <em>F. esculentum</em> </th><th><em>E</em>-value </th></tr></thead><tbody><tr><td>Fes_sc0000087.1.g000011.aua.1 </td><td>Fes_sc0000087.1 </td><td>97% (BW 8 kDa allergen protein) </td><td>8e−74 </td></tr><tr><td>Fes_sc0000087.1.g000013.aua.1 </td><td>Fes_sc0000087.1 </td><td>79% (Fag e 2) </td><td>3e−26 </td></tr><tr><td>Fes_sc0000087.1.g000014.aua.1 </td><td>Fes_sc0000087.1 </td><td>100% (Fag e 2) </td><td>7e−104 </td></tr><tr><td>Fes_sc0000087.1.g000028.aua.1 </td><td>Fes_sc0000087.1 </td><td>98% (BW 8 kDa allergen protein) </td><td>5e−85 </td></tr><tr><td>Fes_sc0007211.1.g000003.aua.1 </td><td>Fes_sc0007211.1 </td><td>43% (BW 8 kDa allergen protein) </td><td>4e−21 </td></tr></tbody></table></div><div content-id="DSW012TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID of the BGDB </th><th>Scaffold ID </th><th>Similarity with reported allergen of <em>F. esculentum</em> </th><th><em>E</em>-value </th></tr></thead><tbody><tr><td>Fes_sc0000087.1.g000011.aua.1 </td><td>Fes_sc0000087.1 </td><td>97% (BW 8 kDa allergen protein) </td><td>8e−74 </td></tr><tr><td>Fes_sc0000087.1.g000013.aua.1 </td><td>Fes_sc0000087.1 </td><td>79% (Fag e 2) </td><td>3e−26 </td></tr><tr><td>Fes_sc0000087.1.g000014.aua.1 </td><td>Fes_sc0000087.1 </td><td>100% (Fag e 2) </td><td>7e−104 </td></tr><tr><td>Fes_sc0000087.1.g000028.aua.1 </td><td>Fes_sc0000087.1 </td><td>98% (BW 8 kDa allergen protein) </td><td>5e−85 </td></tr><tr><td>Fes_sc0007211.1.g000003.aua.1 </td><td>Fes_sc0007211.1 </td><td>43% (BW 8 kDa allergen protein) </td><td>4e−21 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Scaffold ID, ID number of the scaffold in which the predicted gene is situated; BW 8 kDa, buckwheat 8 kDa allergen of 2S albumin; Fag e 2, buckwheat 16 kDa allergen of 2S albumin.</p></div></div><div content-id="tblfn2"><div class="fn"><p>The GenBank accession numbers of BW 8 kDa and Fag e 2 are AB055892 and DQ304682, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199470" target="_blank">View Large</a></div></div>                        <a id="87199471" scrollto-destination="87199471"></a>
<div id="DSW012F2" data-id="DSW012F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01202.png?Expires=2147483647&Signature=V1yA93K51AuZrZ1fu5EpJZRJVc29lhlKpbAgf27f95cQEX9FZ0fXJDlyDL0JKuIaHUedPBuhCpDsK9VToKc9vIMZJv5cMaNbY-E3bXBeGyb5Z2f8dUYsfQXOJQDJNDO~GOJ6E6mV6LUJkn4L1WOAqP9ANnF8mhvG79Hi3OgrV7D2mxkh2sIGLyCfGAwpe3RPr-Qjx8QlEaU2Tx4mC0PzUzLSrMsut7YRMM2g0uS2ZNPKkmuMJpxn9tyaFY0DRnFsQ3ah5rgYaK-GRM2lLR1elzcLp9mzKPUEnOVOtet30wbPMxN5cVjjf2S9jYH2Yjup7b11TtM1EWGWPHzI3AMHRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of amino acid sequences of Fag e 2 (buckwheat 16 kDa allergen of 2S albumin; GenBank accession number: DQ304682), BW 8 kDa (buckwheat 8 kDa allergen of 2S albumin; GenBank accession number: AB055892) and predicted sequences of homologues obtained from the BGDB. The following amino acid colour code is used: orange, small nonpolar (Gly, Ala, Ser, and Thr); green, hydrophobic (Cys, Val, Ile, Leu, Pro, Phe, Tyr, Met, and Trp); magenta, polar (Asn, Gln, and His); red, negatively charged (Asp and Glu); and blue, positively charged (Lys and Arg). Asterisks indicate the eight characteristic Cys residues present in Fag e 2 and 2S albumin family proteins, as described by Satoh et al.60 Note that Fes_sc0007211.1.g000003.aua.1 had low similarity with other amino acid sequences and lacked four of the eight characteristic Cys residues." path-from-xml="dsw01202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199471/dsw01202.tiff" path-from-xml="dsw01202.tiff" target="_blank">View large</a><a section="87199471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01202.png?Expires=2147483647&Signature=mpaDdbqJ2ewByrij-4m3FDSbCPQvuIdrPBHYaLuq~6uW0deVY9flrToplNNVrEvLW-c7Wq3QfjGpebbzU-EDPZEh-PrWYKnk82VWwqoSyKwJhF6RUmr~y4MAk~QcR~qEGK7jOjTBSbfOSrqFnthiBjz2O5SYShOlkfCfrCBhYRykQ-8S-dWY2LdZ2LjJk3TSo6bldFnutUFWjvLvbKT0nv9v8-kNseyd00JSAn2KhZ-7t~0wVz~tOs1VcuVxxyISq0Y-tE391f45ZvOI6xC0d4OLbJ0BZUIhs3eKUW3IvIwOaDapOVt3Y76YqJta03ZR6WT0bgwLxIxzIsGWqfrjwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199471&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of amino acid sequences of Fag e 2 (buckwheat 16 kDa allergen of 2S albumin; GenBank accession number: DQ304682), BW 8 kDa (buckwheat 8 kDa allergen of 2S albumin; GenBank accession number: AB055892) and predicted sequences of homologues obtained from the BGDB. The following amino acid colour code is used: orange, small nonpolar (Gly, Ala, Ser, and Thr); green, hydrophobic (Cys, Val, Ile, Leu, Pro, Phe, Tyr, Met, and Trp); magenta, polar (Asn, Gln, and His); red, negatively charged (Asp and Glu); and blue, positively charged (Lys and Arg). Asterisks indicate the eight characteristic Cys residues present in Fag e 2 and 2S albumin family proteins, as described by Satoh et al.<sup><a reveal-id="DSW012C60" data-open="DSW012C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Note that Fes_sc0007211.1.g000003.aua.1 had low similarity with other amino acid sequences and lacked four of the eight characteristic Cys residues.</p></div></div><div content-id="DSW012F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01202.png?Expires=2147483647&Signature=V1yA93K51AuZrZ1fu5EpJZRJVc29lhlKpbAgf27f95cQEX9FZ0fXJDlyDL0JKuIaHUedPBuhCpDsK9VToKc9vIMZJv5cMaNbY-E3bXBeGyb5Z2f8dUYsfQXOJQDJNDO~GOJ6E6mV6LUJkn4L1WOAqP9ANnF8mhvG79Hi3OgrV7D2mxkh2sIGLyCfGAwpe3RPr-Qjx8QlEaU2Tx4mC0PzUzLSrMsut7YRMM2g0uS2ZNPKkmuMJpxn9tyaFY0DRnFsQ3ah5rgYaK-GRM2lLR1elzcLp9mzKPUEnOVOtet30wbPMxN5cVjjf2S9jYH2Yjup7b11TtM1EWGWPHzI3AMHRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment of amino acid sequences of Fag e 2 (buckwheat 16 kDa allergen of 2S albumin; GenBank accession number: DQ304682), BW 8 kDa (buckwheat 8 kDa allergen of 2S albumin; GenBank accession number: AB055892) and predicted sequences of homologues obtained from the BGDB. The following amino acid colour code is used: orange, small nonpolar (Gly, Ala, Ser, and Thr); green, hydrophobic (Cys, Val, Ile, Leu, Pro, Phe, Tyr, Met, and Trp); magenta, polar (Asn, Gln, and His); red, negatively charged (Asp and Glu); and blue, positively charged (Lys and Arg). Asterisks indicate the eight characteristic Cys residues present in Fag e 2 and 2S albumin family proteins, as described by Satoh et al.60 Note that Fes_sc0007211.1.g000003.aua.1 had low similarity with other amino acid sequences and lacked four of the eight characteristic Cys residues." path-from-xml="dsw01202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199471/dsw01202.tiff" path-from-xml="dsw01202.tiff" target="_blank">View large</a><a section="87199471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01202.png?Expires=2147483647&Signature=mpaDdbqJ2ewByrij-4m3FDSbCPQvuIdrPBHYaLuq~6uW0deVY9flrToplNNVrEvLW-c7Wq3QfjGpebbzU-EDPZEh-PrWYKnk82VWwqoSyKwJhF6RUmr~y4MAk~QcR~qEGK7jOjTBSbfOSrqFnthiBjz2O5SYShOlkfCfrCBhYRykQ-8S-dWY2LdZ2LjJk3TSo6bldFnutUFWjvLvbKT0nv9v8-kNseyd00JSAn2KhZ-7t~0wVz~tOs1VcuVxxyISq0Y-tE391f45ZvOI6xC0d4OLbJ0BZUIhs3eKUW3IvIwOaDapOVt3Y76YqJta03ZR6WT0bgwLxIxzIsGWqfrjwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199471&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment of amino acid sequences of Fag e 2 (buckwheat 16 kDa allergen of 2S albumin; GenBank accession number: DQ304682), BW 8 kDa (buckwheat 8 kDa allergen of 2S albumin; GenBank accession number: AB055892) and predicted sequences of homologues obtained from the BGDB. The following amino acid colour code is used: orange, small nonpolar (Gly, Ala, Ser, and Thr); green, hydrophobic (Cys, Val, Ile, Leu, Pro, Phe, Tyr, Met, and Trp); magenta, polar (Asn, Gln, and His); red, negatively charged (Asp and Glu); and blue, positively charged (Lys and Arg). Asterisks indicate the eight characteristic Cys residues present in Fag e 2 and 2S albumin family proteins, as described by Satoh et al.<sup><a reveal-id="DSW012C60" data-open="DSW012C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Note that Fes_sc0007211.1.g000003.aua.1 had low similarity with other amino acid sequences and lacked four of the eight characteristic Cys residues.</p></div></div>                            <h4 scrollto-destination=87199472 id="87199472" class="section-title" data-legacy-id=s3b3>3.2.3. Example III: identifying <em>GBSS</em> gene in buckwheat</h4>
<p>Starch, which is present in many plant seeds, legumes, and tuber crops, is an important part of the human diet and has industrial applications. Starch contains two types of glucose polymer, i.e. amylopectin and amylose.<sup><a reveal-id="DSW012C61" data-open="DSW012C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Amylopectin is the major component of starch (60–90%),<sup><a reveal-id="DSW012C62" data-open="DSW012C62" class="link link-ref link-reveal xref-bibr">62</a></sup> whereas the amylose content varies among plant species.<sup><a reveal-id="DSW012C63" data-open="DSW012C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Since the amylose content affects the properties of starch, modulating the amylose content has been an important breeding objective in crops.<sup><a reveal-id="DSW012C64" data-open="DSW012C64" class="link link-ref link-reveal xref-bibr">64</a></sup> GBSS catalyses amylose synthesis.<sup><a reveal-id="DSW012C65" data-open="DSW012C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Thus, we searched for genes encoding starch synthases (SSs) from the BGDB, and aimed to identify <em>GBSS</em> genes using phylogenetic approaches.</p><p>To identify buckwheat <em>GBSS</em> genes, we conducted a keyword search using ‘starch synthase’ and obtained 42 hits. We then filtered these hits using modified five conserved sequence motifs proposed by Cao et al.<sup><a reveal-id="DSW012C66" data-open="DSW012C66" class="link link-ref link-reveal xref-bibr">66</a></sup> [i.e. P(2)K(1)GGL(1)D(4)L, VS(5)E, G(2)NG(7)P(2)D, R(3)QKG, D(5)S(2)EPC(1)L(1)Q(5)YG(8)GGL (numbers in parentheses represent numbers of amino acids)]. Of the eight resulting sequences, one was excluded, as it was annotated as a pseudogene. The seven remaining sequences were each derived from a different scaffold.</p><p>To assign these seven putative SSs of buckwheat to previously proposed phylogenetic groups,<sup><a reveal-id="DSW012C67" data-open="DSW012C67" class="link link-ref link-reveal xref-bibr">67</a></sup> we performed NJ analysis using the deduced amino acid sequences of SSs from various plant species. A keyword search using ‘starch synthase’ was also conducted in Phytozome 10.3 (<a class="link link-uri" href="http://phytozome.jgi.doe.gov/">http://phytozome.jgi.doe.gov/</a>) analysing 36 angiosperm species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Of the 27,852 sequences identified, 238 sequences remained after the same filtering procedure as mentioned earlier and were subjected to phylogenetic analysis. Two GBSS sequences of <em>Fagopyrum</em> species, one from <em>F. esculentum</em> deposited at the EMBL/GENBANK/DDBJ (accession number: HW041459) and the other from <em>F. tataricum</em> (AHA36967.1),<sup><a reveal-id="DSW012C68" data-open="DSW012C68" class="link link-ref link-reveal xref-bibr">68</a></sup> were also included in the analyses. A NJ-tree based on the alignment was suggested to contain the following five known phylogenetic groups: SSI, SSII, SSIII, SSIV, and GBSS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). The SS sequences obtained from the BGDB belonged to four of the five classes. This suggests that the BGDB can be used to identify agronomically important genes. However, the previously deposited <em>GBSS</em> (HW041459) sequence is not identical to any of the seven sequences identified in the BGDB, and a BLASTN search using the coding region of HW041459 as a query detected only partially identical sequences over 360-bp and three scaffolds (Fes_sc0195744.1, Fes_sc0059460.1, and Fes_sc0005470.1). This is a shortcoming of the short scaffold size of the assemblies in the BGDB.</p><p>The GBSS clade contained four phylogenetically distinguishable sequences in total: HW041459 and three from BGDB. To clarify the detailed phylogenetic relationship among these four <em>GBSS</em> genes, we performed NJ analysis based on 71 aligned amino acid sequences of GBSS including those from <em>Physicomitrella patens</em> as outgroup (Fig. <a reveal-id="DSW012F3" data-open="DSW012F3" class="link link-reveal link-table xref-fig">3</a>). The copy number of <em>GBSS</em> genes varies in plants; two diverged groups exist in the rosids<sup><a reveal-id="DSW012C69" data-open="DSW012C69" class="link link-ref link-reveal xref-bibr">69</a></sup> and several copies of <em>GBSS</em> in buckwheat seem to also have diverged, at least in two lineages. In the cladogram, a <em>GBSS</em> sequence, Fes sc0004292.1.g000004, clustered with a <em>GBSS</em> sequence from <em>F. tataricum,</em> which belongs to the same genus. <em>GBSS</em> of <em>F. tataricum</em> was confirmed to be expressed in the endosperm,<sup><a reveal-id="DSW012C68" data-open="DSW012C68" class="link link-ref link-reveal xref-bibr">68</a></sup> thus Fes sc0004292.1.g000004 is likely to be expressed in the endosperm too. If more than one <em>GBSS</em> is active within endosperms, the amylose content of buckwheat flour can be controlled by altering their copy number. In hexaploid wheat, Yamamori and Quynh<sup><a reveal-id="DSW012C70" data-open="DSW012C70" class="link link-ref link-reveal xref-bibr">70</a></sup> evaluated the dosage effects of three <em>GBSS</em> genes. Loss of function of these genes is expected to differentiate the starch properties of the grain; moreover, distinct proportions of amylose might be produced according to copy number of active <em>GBSS</em> genes. Studies are underway to examine the expression of the three buckwheat <em>GBSS</em> genes identified here, and also the previously identified one (HW041459). </p>                        <a id="87199477" scrollto-destination="87199477"></a>
<div id="DSW012F3" data-id="DSW012F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01203.png?Expires=2147483647&Signature=ShmrATdCs3-nP22rZ-Qo9ZxWRRRm0JoTpAOrtQvOeN8l1OAsu-1f8YQpqLCgzGHKWHf8iPl7rxI8qDn0in8FmTFKVLao520xfDzhK6rUcBwcx8m3qXCi35g93ylT1L0fSrh2WCVWle8avt4DGFsFpipx9OLPm34gh1arZ9u8L8rTkjdSZbhEHet-~TSSRKe9O3PC2sVBIzL7DI1eG54-AZMFMZJN4Y0Ck~4IgKQlFf11MXY6IkHQmR4WVeZqMxC-SlNVHjFzE6A8D4gdWaEYta2bQ-BiVYJRrqOYOe0p-BW461WvD4hIUimKk34v6GdQxKdmzJVePTqhX-4-yEY9rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="NJ tree based on amino acid sequences of GBSS from buckwheat and other plant species. The bootstrap values (500 replicates) not &lt;50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. Two GBSSs from Physcomitrella patens were used as outgroup sequences. Species names are coloured according to their order: Poales, grey; Ranunculales, cyan; Vitales, purple; Cucurbitales, blue grey; Fabales, green; Malpighiales, blue; Rosales, pink; Myrtales, teal; Brassicales, indigo; Malvales, brown; Sapindales, orange; Caryophyllales, red; Lamiales, yellow; and Solanales, lime. Four GBSSs from F. esculentum are indicated by red circles next to the sequence names. Sequences obtained from BGDB are abbreviated (sc0002521: Fes_sc0002521.1.g000007; sc0004292: Fes_sc0004292.1.g000004; and sc0005258: Fes_sc0005258.1.g000004). Two sequences of Fagopyrum species (AHA36967 and HW041459) were obtained from GenBank. Sequences excluding those from Fagopyrum species were obtained using Phytozome 10.3 and the accession numbers are in parentheses." path-from-xml="dsw01203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199477/dsw01203.tiff" path-from-xml="dsw01203.tiff" target="_blank">View large</a><a section="87199477" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01203.png?Expires=2147483647&Signature=ccMKQI0Z23Ki2zR3RM41ko8Vd7NoJEvZ~MuqRr5ov3ALbx3sNDnct7suuRJLXNr9BniQkRjXqfsvGicnpFDzDTpbH4yKewdvhko59C5rZAzH1TapzqgZEiHI3vs7l~B2JZdSCcb2JcGoy3ghm5Y-Edi0x~Ttx8dE0jZiM-IFXskWRdsN4ZdbO5OLaO81dTa~igdV5-CN1mnIsCOTo1s7H-gTvO11EosIR22ToDIU-lFDZY0pqB54ThsCPQYb8JzCFZt-ibsepTuKGdrhWlomwtksdUTTGo04~8vi8~lHKc1W1YLN7CF4k7R8uCJn3UnfTn6BQQlWLItpfpSuU-wVfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199477&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>NJ tree based on amino acid sequences of GBSS from buckwheat and other plant species. The bootstrap values (500 replicates) not &lt;50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. Two GBSSs from <em>Physcomitrella patens</em> were used as outgroup sequences. Species names are coloured according to their order: Poales, grey; Ranunculales, cyan; Vitales, purple; Cucurbitales, blue grey; Fabales, green; Malpighiales, blue; Rosales, pink; Myrtales, teal; Brassicales, indigo; Malvales, brown; Sapindales, orange; Caryophyllales, red; Lamiales, yellow; and Solanales, lime. Four GBSSs from <em>F. esculentum</em> are indicated by red circles next to the sequence names. Sequences obtained from BGDB are abbreviated (sc0002521: Fes_sc0002521.1.g000007; sc0004292: Fes_sc0004292.1.g000004; and sc0005258: Fes_sc0005258.1.g000004). Two sequences of <em>Fagopyrum</em> species (AHA36967 and HW041459) were obtained from GenBank. Sequences excluding those from <em>Fagopyrum</em> species were obtained using Phytozome 10.3 and the accession numbers are in parentheses.</p></div></div><div content-id="DSW012F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01203.png?Expires=2147483647&Signature=ShmrATdCs3-nP22rZ-Qo9ZxWRRRm0JoTpAOrtQvOeN8l1OAsu-1f8YQpqLCgzGHKWHf8iPl7rxI8qDn0in8FmTFKVLao520xfDzhK6rUcBwcx8m3qXCi35g93ylT1L0fSrh2WCVWle8avt4DGFsFpipx9OLPm34gh1arZ9u8L8rTkjdSZbhEHet-~TSSRKe9O3PC2sVBIzL7DI1eG54-AZMFMZJN4Y0Ck~4IgKQlFf11MXY6IkHQmR4WVeZqMxC-SlNVHjFzE6A8D4gdWaEYta2bQ-BiVYJRrqOYOe0p-BW461WvD4hIUimKk34v6GdQxKdmzJVePTqhX-4-yEY9rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="NJ tree based on amino acid sequences of GBSS from buckwheat and other plant species. The bootstrap values (500 replicates) not &lt;50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. Two GBSSs from Physcomitrella patens were used as outgroup sequences. Species names are coloured according to their order: Poales, grey; Ranunculales, cyan; Vitales, purple; Cucurbitales, blue grey; Fabales, green; Malpighiales, blue; Rosales, pink; Myrtales, teal; Brassicales, indigo; Malvales, brown; Sapindales, orange; Caryophyllales, red; Lamiales, yellow; and Solanales, lime. Four GBSSs from F. esculentum are indicated by red circles next to the sequence names. Sequences obtained from BGDB are abbreviated (sc0002521: Fes_sc0002521.1.g000007; sc0004292: Fes_sc0004292.1.g000004; and sc0005258: Fes_sc0005258.1.g000004). Two sequences of Fagopyrum species (AHA36967 and HW041459) were obtained from GenBank. Sequences excluding those from Fagopyrum species were obtained using Phytozome 10.3 and the accession numbers are in parentheses." path-from-xml="dsw01203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199477/dsw01203.tiff" path-from-xml="dsw01203.tiff" target="_blank">View large</a><a section="87199477" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01203.png?Expires=2147483647&Signature=ccMKQI0Z23Ki2zR3RM41ko8Vd7NoJEvZ~MuqRr5ov3ALbx3sNDnct7suuRJLXNr9BniQkRjXqfsvGicnpFDzDTpbH4yKewdvhko59C5rZAzH1TapzqgZEiHI3vs7l~B2JZdSCcb2JcGoy3ghm5Y-Edi0x~Ttx8dE0jZiM-IFXskWRdsN4ZdbO5OLaO81dTa~igdV5-CN1mnIsCOTo1s7H-gTvO11EosIR22ToDIU-lFDZY0pqB54ThsCPQYb8JzCFZt-ibsepTuKGdrhWlomwtksdUTTGo04~8vi8~lHKc1W1YLN7CF4k7R8uCJn3UnfTn6BQQlWLItpfpSuU-wVfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199477&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>NJ tree based on amino acid sequences of GBSS from buckwheat and other plant species. The bootstrap values (500 replicates) not &lt;50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. Two GBSSs from <em>Physcomitrella patens</em> were used as outgroup sequences. Species names are coloured according to their order: Poales, grey; Ranunculales, cyan; Vitales, purple; Cucurbitales, blue grey; Fabales, green; Malpighiales, blue; Rosales, pink; Myrtales, teal; Brassicales, indigo; Malvales, brown; Sapindales, orange; Caryophyllales, red; Lamiales, yellow; and Solanales, lime. Four GBSSs from <em>F. esculentum</em> are indicated by red circles next to the sequence names. Sequences obtained from BGDB are abbreviated (sc0002521: Fes_sc0002521.1.g000007; sc0004292: Fes_sc0004292.1.g000004; and sc0005258: Fes_sc0005258.1.g000004). Two sequences of <em>Fagopyrum</em> species (AHA36967 and HW041459) were obtained from GenBank. Sequences excluding those from <em>Fagopyrum</em> species were obtained using Phytozome 10.3 and the accession numbers are in parentheses.</p></div></div>                            <h4 scrollto-destination=87199478 id="87199478" class="section-title" data-legacy-id=s3b4>3.2.4. Example IV: isolation of heteromorphic SI genes</h4>
<p>Finally, we screened for candidate genes that control heteromorphic SI in buckwheat. Buckwheat is a heteromorphic self-incompatible crop with dimorphic flowers (i.e. short-styled and long-styled flowers). Short-styled flowers have short styles and long stamens, whereas long-styled flowers have long styles and short stamens. The SI response is expressed between plants bearing the same flower morph, but not between plants bearing different flower morphs. Flower morph and SI response are determined by a diallelic system at the <em>SELF-INCOMPATIBILITY</em> supergene complex locus (<em>S</em> locus); <em>S</em>/<em>s</em> heterozygotes and <em>s</em>/<em>s</em> recessive homozygotes bear short-styled and long-styled flowers, respectively.<sup><a reveal-id="DSW012C71" data-open="DSW012C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSW012C72" data-open="DSW012C72" class="link link-ref link-reveal xref-bibr">72</a></sup> Recently, <em>S-LOCUS EARLY FLOWERING 3</em> (<em>S-ELF3</em>), which controls the short-styled phenotype of buckwheat, was isolated.<sup><a reveal-id="DSW012C16" data-open="DSW012C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Furthermore, it was suggested that recombination is strongly suppressed around <em>S-ELF3</em>.<sup><a reveal-id="DSW012C16" data-open="DSW012C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Based on these findings, we predicted that <em>S-</em>allelic scaffolds exist in which heteromorphic SI-related genes other than <em>S-ELF 3</em> are located. Thus, we tried to detect <em>S-</em>allelic scaffolds in the draft genome and to identify novel candidate genes involved in the SI response in these.</p><p>To obtain <em>S</em>-allele-linked scaffolds from the draft genome, we used GBS reads obtained from each of 18 short-styled and 18 long-styled landraces of buckwheat originating from various countries (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Briefly, GBS reads from 36 plants were mapped to scaffolds of &gt;1,000 bp, and we subsequently extracted the non-LS sites (i.e. sites not present in all the long-styled plants) in which no reads from all of the 18 long-styled plants were mapped. The number of mapped reads for each plant ranged from 1,620,260 to 2,821,338 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). The number of mapped reads per long-styled plant was not significantly different from that per short-styled plant (<em>P</em>-value is 0.536, <em>t</em>-test). We determined how many short-styled plants share each non-LS site, and the number ranged from 1 to 18 (Fig. <a reveal-id="DSW012F4" data-open="DSW012F4" class="link link-reveal link-table xref-fig">4</a>). As a control, non-SS sites (i.e. sites not present in short-styled plants) were also counted as above, and the number of long-styled plants sharing non-SS sites ranged from 1 to 10 (Fig. <a reveal-id="DSW012F4" data-open="DSW012F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87199481" scrollto-destination="87199481"></a>
<div id="DSW012F4" data-id="DSW012F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01204.png?Expires=2147483647&Signature=PIJGlQ-zR7Ehlrmk2zcEjKyQFeuaJxDsB6OY9UAwU8QKKt2Diz-kY40QjbdGEVlqUDqETnesFxoGzEsiHHZnxRjk4tbgGO-zSAkP45uh0JW3Vzb1TncMZIsylPsO6TlVgimkE2fOtmhHLV1ge~a20gMLywARD-a6SALbt3ebjDenZRRGlcS~T9ZfSRZ94MP~5-AwRHSVuaK3-RwUpB841kBSqEdLf4rigJ7I91b7NmnuaK5S52xDNWbnHDPFi0GOi047h8Ggtt5I~TtZ8xU~jVvcxd9jrRoYI2BUn-VpMahFJb1~L7P1V6vbdcuP6y1~w0grX6-CtqkLhReALlC8uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of non-LS sites and non-SS sites identified in GBS reads. Sites at which &gt;50 reads were mapped in long-styled plants but not in short-styled plants were defined as ‘non-SS’. Sites at which &gt;50 reads were mapped in short-styled plants but not in long-styled plants were defined as ‘non-LS’. The number of non-LS (blue bar) and non-SS sites (red bar) was plotted against the number of short-styled plants sharing the non-LS sites and of long-styled plants sharing the non-SS sites, respectively." path-from-xml="dsw01204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199481/dsw01204.tiff" path-from-xml="dsw01204.tiff" target="_blank">View large</a><a section="87199481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01204.png?Expires=2147483647&Signature=tla2KqU6EgjqwIIrYQtiLxjAxv4Zi1smCT7PYQC1pdoCHzn5y4YF62~mkVIdptJMg2EPEODVqWe0HyC5KQsD-BMiOJPjppe2xnPyJ1ws1pGLLHa8xTjPf4GwA2efgHxAvu89zbA2riIilcGQpfmAk-00REfS64gmbXpuAMjgNaW8a0vZXqeu6Lr1vBy46uvyOSCXyjm3BndZem0YqqjUaVcUM0zV7HNM5M7p59ab8OqrBJ1RtZ6Pz6QPbkAhFUCPRyhl9ZavHjUgNgtm64ylBH2uttuXQQyW7xXSa8hqIKgjfHTwvLWrXIQInMd1nZvkDzXk61QKkzsNKZhiZUVA~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199481&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of non-LS sites and non-SS sites identified in GBS reads. Sites at which &gt;50 reads were mapped in long-styled plants but not in short-styled plants were defined as ‘non-SS’. Sites at which &gt;50 reads were mapped in short-styled plants but not in long-styled plants were defined as ‘non-LS’. The number of non-LS (blue bar) and non-SS sites (red bar) was plotted against the number of short-styled plants sharing the non-LS sites and of long-styled plants sharing the non-SS sites, respectively.</p></div></div><div content-id="DSW012F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/m_dsw01204.png?Expires=2147483647&Signature=PIJGlQ-zR7Ehlrmk2zcEjKyQFeuaJxDsB6OY9UAwU8QKKt2Diz-kY40QjbdGEVlqUDqETnesFxoGzEsiHHZnxRjk4tbgGO-zSAkP45uh0JW3Vzb1TncMZIsylPsO6TlVgimkE2fOtmhHLV1ge~a20gMLywARD-a6SALbt3ebjDenZRRGlcS~T9ZfSRZ94MP~5-AwRHSVuaK3-RwUpB841kBSqEdLf4rigJ7I91b7NmnuaK5S52xDNWbnHDPFi0GOi047h8Ggtt5I~TtZ8xU~jVvcxd9jrRoYI2BUn-VpMahFJb1~L7P1V6vbdcuP6y1~w0grX6-CtqkLhReALlC8uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of non-LS sites and non-SS sites identified in GBS reads. Sites at which &gt;50 reads were mapped in long-styled plants but not in short-styled plants were defined as ‘non-SS’. Sites at which &gt;50 reads were mapped in short-styled plants but not in long-styled plants were defined as ‘non-LS’. The number of non-LS (blue bar) and non-SS sites (red bar) was plotted against the number of short-styled plants sharing the non-LS sites and of long-styled plants sharing the non-SS sites, respectively." path-from-xml="dsw01204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199481/dsw01204.tiff" path-from-xml="dsw01204.tiff" target="_blank">View large</a><a section="87199481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw01204.png?Expires=2147483647&Signature=tla2KqU6EgjqwIIrYQtiLxjAxv4Zi1smCT7PYQC1pdoCHzn5y4YF62~mkVIdptJMg2EPEODVqWe0HyC5KQsD-BMiOJPjppe2xnPyJ1ws1pGLLHa8xTjPf4GwA2efgHxAvu89zbA2riIilcGQpfmAk-00REfS64gmbXpuAMjgNaW8a0vZXqeu6Lr1vBy46uvyOSCXyjm3BndZem0YqqjUaVcUM0zV7HNM5M7p59ab8OqrBJ1RtZ6Pz6QPbkAhFUCPRyhl9ZavHjUgNgtm64ylBH2uttuXQQyW7xXSa8hqIKgjfHTwvLWrXIQInMd1nZvkDzXk61QKkzsNKZhiZUVA~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199481&ar=1745460&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of non-LS sites and non-SS sites identified in GBS reads. Sites at which &gt;50 reads were mapped in long-styled plants but not in short-styled plants were defined as ‘non-SS’. Sites at which &gt;50 reads were mapped in short-styled plants but not in long-styled plants were defined as ‘non-LS’. The number of non-LS (blue bar) and non-SS sites (red bar) was plotted against the number of short-styled plants sharing the non-LS sites and of long-styled plants sharing the non-SS sites, respectively.</p></div></div><p>As shown in Fig. <a reveal-id="DSW012F4" data-open="DSW012F4" class="link link-reveal link-table xref-fig">4</a>, a striking U-shaped distribution was obtained when the number of non-LS sites was plotted against the number of short-styled plants sharing the non-LS sites. In contrast, non-SS sites shared by &gt;10 long-styled plants were not detected at all. Considering that there was no significant difference in the number of mapped reads obtained from short-styled and long-styled plants, we regarded the non-LS sites shared by &gt;10 short-styled plants as ‘<em>S</em>-allele-specific sites’. In total, 88,031 of the <em>S</em>-allele-specific sites were detected and found to be located on the <em>S</em>-allelic region, consisting of 332 of scaffolds encompassing 5,393,196 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Of the 332 scaffolds, Fes_sc0003500.1 and Fes_sc0015090.1 harbour <em>S-ELF3</em> and <em>SSG2</em>, respectively. Since we previously established that <em>S-ELF3</em> and <em>SSG2</em> existed only in the genomes of short-styled plants,<sup><a reveal-id="DSW012C16" data-open="DSW012C16" class="link link-ref link-reveal xref-bibr">16</a></sup> this result shows the effectiveness of our procedure in discovering the <em>S</em>-allelic region in the buckwheat genome. However, it should be noted that we used only one restriction enzyme combination (<em>Eco</em>RI and <em>Mse</em>I) to obtain the GBS reads; we did not detect <em>S</em>-allelic scaffolds harbouring no recognition sites for these enzymes. Thus, the total length of the <em>S</em>-allelic region in the buckwheat genome obtained in the present study might be an underestimation.</p><p>In the <em>S</em>-allelic region, repeat sequences were abundant; the ratio of repeat length to the total length of the <em>S</em>-region was 71.43%, which is 1.4-times higher than that of the ratio of repeat length to the total scaffold length (51.69%, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Gypsy elements were particularly abundant in the <em>S</em>-region; the Gypsy elements accounted for 12.15% of the <em>S</em>-region, which is 1.9 times higher than that of the repeat length in the total scaffold length (6.41%, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Excluding TEs, only 32 predicted genes were successfully annotated by our database analyses in the 332 scaffolds (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S12 and S13</a></span>). Among the 32 predicted genes, two were candidates for heteromorphic SI related genes; Fes_sc0024869.1.g000002 and Fes_sc0006594.1.g000005 were predicted to encode proteins with similarity to a RING/U-box superfamily protein and an exoribonuclease 4, respectively. The RING/U-box protein is an E3 type ubiquitin ligase that functions in the 26S proteasome.<sup><a reveal-id="DSW012C73" data-open="DSW012C73" class="link link-ref link-reveal xref-bibr">73</a></sup> It has been suggested that degradation of cytotoxic S-RNases by the 26S proteasome in pollen tubes occurs during compatible pollination in a self-incompatible species, <em>Petunia inflata</em>.<sup><a reveal-id="DSW012C74" data-open="DSW012C74" class="link link-ref link-reveal xref-bibr">74</a></sup> Ongoing studies currently evaluating the expression pattern of these two genes and the effect of mutations in these two genes will clarify their roles in heteromorphic SI of buckwheat.</p>                            <h2 scrollto-destination=87199484 id="87199484" class="section-title" data-legacy-id=s4>4. Conclusion and future perspective</h2>
<p>The genome size of buckwheat is relatively large (∼1.2 Gb), and some genomic regions are expected to be in the heterozygous state due to its outcrossing nature. These factors would reduce the lengths of our scaffolds in the buckwheat draft genome, which was assembled using only Illumina short reads. Though the draft genome was trancated and divided into a large number of scaffolds (387,594 scaffolds), we have successfully identified genes that control agronomically important traits using gene predictions and subsequent annotations in the BGDB. Furthermore, we also identified novel candidate genes involved in heteromorphic SI of buckwheat using the draft genome as a reference sequence for GBS mapping. Even if the scaffolds in a draft genome are truncated, they can nonetheless be used for database construction and as a reference sequence for NGS-based genetic markers. We are now preparing induced mutant pools of buckwheat using heavy-ion beams and chemicals such as ethyl methanesulfonate. Using NGS-based multi-dimensional screening,<sup><a reveal-id="DSW012C75" data-open="DSW012C75" class="link link-ref link-reveal xref-bibr">75</a></sup> mutants of the genes identified in this study will be rapidly identified from the pool and will be used to develop superior varieties of buckwheat.</p>                            <h2 scrollto-destination=87199486 id="87199486" class="section-title" data-legacy-id=s5>5. Data availability</h2>
<p>The Illumina reads used in this study are available from DDBJ/EMBL/NCBI under the accession numbers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. The DRA accession number of the Illumina reads used in GBS analysis is DRA004489. The scaffold sequences are available under the accession numbers BCYN01000001-BCYN01387594 (387,594 entries). The draft genome sequence FES_r1.0, CDS and protein sequences, and annotation file (gff file) are also available from the Buckwheat Genome DataBase (BGDB; <a class="link link-uri" href="http://buckwheat.kazusa.or.jp">http://buckwheat.kazusa.or.jp</a>).</p>                            <h2 scrollto-destination=87199488 id="87199488" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This study was supported by a grant from the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics-based Technology for Agricultural Improvement, SFC3001) and JSPS KAKENHI (grant numbers 22580003, 25292009, 25450011, and 25660006).</p>                            <h2 scrollto-destination=87199490 id="87199490" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank S. Tabata, T. Ota, K. Isono, and the anonymous reviewer for valuable suggestions and comments; R. Kajitani for kind information on genome assemblers; and K. L. Farquharson for language-editing support of the manuscript. We also thank Y. Nakajima for assistance in database construction and management. Computations were partially performed on the NIG supercomputer at ROIS National Institute of Genetics.</p>                            <h2 scrollto-destination=87199492 id="87199492" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW012C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Polashock</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zelzion</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Fajardo</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">The American cranberry: first insights into the whole genome of a species adapted to bog habitat</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20American%20cranberry%3A%20first%20insights%20into%20the%20whole%20genome%20of%20a%20species%20adapted%20to%20bog%20habitat&amp;author=J.%20Polashock&amp;author=E.%20Zelzion&amp;author=D.%20Fajardo&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24927653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S-I.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in <em>Capsicum</em> species</div>, <div class="source">Nat. Genet.</div> , <div class="volume">46</div>, <div class="fpage">270</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20hot%20pepper%20provides%20insights%20into%20the%20evolution%20of%20pungency%20in%20Capsicum%20species&amp;author=S.%20Kim&amp;author=M.%20Park&amp;author=S-I.%20Yeom&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=270-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Quan</div> <div class="given-names">Z.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genome sequence of foxtail millet (<em>Setaria italica</em>) provides insights into grass evolution and biofuel potential</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">549</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;author=G.%20Zhang&amp;author=X.%20Liu&amp;author=Z.%20Quan&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=549-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22580950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Varshney</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Draft genome sequence of pigeonpea (<em>Cajanus cajan</em>), an orphan legume crop of resource-poor farmers</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">83</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(Cajanus%20cajan)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div> </div></div></div></div></div><div content-id="DSW012C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Giménez-Bastida</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Zieliński</div> <div class="given-names">H.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Buckwheat as a functional food and its effects on health</div>, <div class="source">J. Agric. Food Chem.</div> , <div class="volume">63</div>, <div class="fpage">7896</div>–<div class="lpage">913</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Buckwheat%20as%20a%20functional%20food%20and%20its%20effects%20on%20health&amp;author=J.A.%20Gim%C3%A9nez-Bastida&amp;author=H.%20Zieli%C5%84ski&amp;publication_year=2015&amp;journal=J.%20Agric.%20Food%20Chem.&amp;volume=63&amp;pages=7896-913" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/acs.jafc.5b02498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Facs.jafc.5b02498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Facs.jafc.5b02498"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26270637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Comino</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">de Lourdes Moreno</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Real</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rodríguez-Herrera</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Barro</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sousa</div> <div class="given-names">C.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">The gluten-free diet: testing alternative cereals tolerated by celiac patients</div>, <div class="source">Nutrients</div> , <div class="volume">5</div>, <div class="fpage">4250</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20gluten-free%20diet%3A%20testing%20alternative%20cereals%20tolerated%20by%20celiac%20patients&amp;author=I.%20Comino&amp;author=M.%20de%20Lourdes%20Moreno&amp;author=A.%20Real&amp;author=A.%20Rodr%C3%ADguez-Herrera&amp;author=F.%20Barro&amp;author=C.%20Sousa&amp;publication_year=2013&amp;journal=Nutrients&amp;volume=5&amp;pages=4250-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/nu5104250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fnu5104250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fnu5104250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24152755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heffler</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Pizzimenti</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Badiu</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Guida</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rolla</div> <div class="given-names">G.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Buckwheat allergy: an emerging clinical problem in Europe</div>, <div class="source">J. Allergy Ther.</div> , <div class="volume">5</div>, <div class="fpage">168</div>.</div></div></div></div><div content-id="DSW012C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yasui</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ohnishi</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">C.G.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Amplified fragment length polymorphism linkage analysis of common buckwheat (<em>Fagopyrum esculentum</em>) and its wild self-pollinated relative <em>Fagopyrum homotropicum</em></div>, <div class="source">Genome</div> , <div class="volume">47</div>, <div class="fpage">345</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amplified%20fragment%20length%20polymorphism%20linkage%20analysis%20of%20common%20buckwheat%20(Fagopyrum%20esculentum)%20and%20its%20wild%20self-pollinated%20relative%20Fagopyrum%20homotropicum&amp;author=Y.%20Yasui&amp;author=Y.%20Wang&amp;author=O.%20Ohnishi&amp;author=C.G.%20Campbell&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=345-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Konishi</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ohnishi</div> <div class="given-names">O.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">A linkage map for common buckwheat based on microsatellite and AFLP markers</div>, <div class="source">Fagopyrum</div> , <div class="volume">23</div>, <div class="fpage">1</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSW012C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hara</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Okuno</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Matsui</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ohsawa</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">QTL analysis of photoperiod sensitivity in common buckwheat by using markers for expressed sequence tags and photoperiod-sensitivity candidate genes</div>, <div class="source">Breed. Sci.</div> , <div class="volume">61</div>, <div class="fpage">394</div>–<div class="lpage">404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20photoperiod%20sensitivity%20in%20common%20buckwheat%20by%20using%20markers%20for%20expressed%20sequence%20tags%20and%20photoperiod-sensitivity%20candidate%20genes&amp;author=T.%20Hara&amp;author=H.%20Iwata&amp;author=K.%20Okuno&amp;author=K.%20Matsui&amp;author=R.%20Ohsawa&amp;publication_year=2011&amp;journal=Breed.%20Sci.&amp;volume=61&amp;pages=394-404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.61.394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.61.394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yabe</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ueno</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Rapid genotyping with DNA micro-arrays for high-density linkage mapping and QTL mapping in common buckwheat (<em>Fagopyrum esculentum</em> Moench)</div>, <div class="source">Breed. Sci.</div> , <div class="volume">64</div>, <div class="fpage">291</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20genotyping%20with%20DNA%20micro-arrays%20for%20high-density%20linkage%20mapping%20and%20QTL%20mapping%20in%20common%20buckwheat%20(Fagopyrum%20esculentum%20Moench)&amp;author=S.%20Yabe&amp;author=T.%20Hara&amp;author=M.%20Ueno&amp;publication_year=2014&amp;journal=Breed.%20Sci.&amp;volume=64&amp;pages=291-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.64.291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.64.291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.64.291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25914583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Elshire</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">A robust, simple Genotyping-by-Sequencing (GBS) Approach for high diversity species</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20Genotyping-by-Sequencing%20(GBS)%20Approach%20for%20high%20diversity%20species&amp;author=R.J.%20Elshire&amp;author=J.C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">He</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Laroche</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Z-X.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">5</div>, <div class="fpage">484</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20(GBS)%2C%20an%20ultimate%20marker-assisted%20selection%20(MAS)%20tool%20to%20accelerate%20plant%20breeding&amp;author=J.%20He&amp;author=X.%20Zhao&amp;author=A.%20Laroche&amp;author=Z-X.%20Lu&amp;author=H.%20Liu&amp;author=Z.%20Li&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci.&amp;volume=5&amp;pages=484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25324846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yasui</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Mori</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ohnishi</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">C.G.</div></div>, <div class="name string-name"><div class="surname">Ota</div> <div class="given-names">T.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Construction of a BAC library for buckwheat genome research—an application to positional cloning of agriculturally valuable traits</div>, <div class="source">Genes Genet. Syst.</div> , <div class="volume">83</div>, <div class="fpage">393</div>–<div class="lpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20BAC%20library%20for%20buckwheat%20genome%20research%E2%80%94an%20application%20to%20positional%20cloning%20of%20agriculturally%20valuable%20traits&amp;author=Y.%20Yasui&amp;author=M.%20Mori&amp;author=D.%20Matsumoto&amp;author=O.%20Ohnishi&amp;author=C.G.%20Campbell&amp;author=T.%20Ota&amp;publication_year=2008&amp;journal=Genes%20Genet.%20Syst.&amp;volume=83&amp;pages=393-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.83.393" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.83.393" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.83.393"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19168990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stanke</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yasui</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Mori</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Aii</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title"><em>S-LOCUS EARLY FLOWERING 3</em> is exclusively present in the genomes of short-styled buckwheat plants that exhibit heteromorphic self-incompatibility</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e31264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=S-LOCUS%20EARLY%20FLOWERING%203%20is%20exclusively%20present%20in%20the%20genomes%20of%20short-styled%20buckwheat%20plants%20that%20exhibit%20heteromorphic%20self-incompatibility&amp;author=Y.%20Yasui&amp;author=M.%20Mori&amp;author=J.%20Aii&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e31264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0031264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0031264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0031264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22312442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmieder</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Quality control and preprocessing of metagenomic datasets</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">863</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hirakawa</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marçais</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div>
          <div class="year">2011</div>, A <div class="article-title">fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">De novo assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">265</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.097261.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Boetzer</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">De novo identification of repeat families in large genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div><div class="issue">(Suppl 1)</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blanco</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Guigó</div> <div class="given-names">R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Using geneid to identify genes</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <div class="volume">Chapter 4</div>, <div class="fpage">Unit 4.3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20geneid%20to%20identify%20genes&amp;author=E.%20Blanco&amp;author=G.%20Parra&amp;author=R.%20Guig%C3%B3&amp;publication_year=2007&amp;journal=Curr.%20Protoc.%20Bioinformatics&amp;volume=Chapter 4&amp;pages=Unit 4.3" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18428791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quevillon</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tatusov</div>
               <div class="given-names">R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Llorens</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Futami</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Covelli</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">The Gypsy Database (GyDB) of mobile genetic elements: release 2.0</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">D70</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gypsy%20Database%20(GyDB)%20of%20mobile%20genetic%20elements%3A%20release%202.0&amp;author=C.%20Llorens&amp;author=R.%20Futami&amp;author=L.%20Covelli&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D70-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21036865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eddy</div>
               <div class="given-names">S.R.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">A new generation of homology search tools based on probabilistic inference</div>, <div class="source">Genome Inform.</div> , <div class="volume">23</div>, <div class="fpage">205</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Inform.&amp;volume=23&amp;pages=205-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bolger</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows–Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">DePristo</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>, <div class="source">Nat Genet.</div> , <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat%20Genet.&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larkin</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">ClustalW and ClustalX version 2</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ClustalW%20and%20ClustalX%20version%202&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saitou</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div></div>
          <div class="year">1987</div>, <div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">4</div>, <div class="fpage">406</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jones</div>
               <div class="given-names">D.T.</div>
            </div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Thornton</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">1992</div>, <div class="article-title">The rapid generation of mutation data matrices from protein sequences</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="volume">8</div>, <div class="fpage">275</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=D.T.%20Jones&amp;author=W.R.%20Taylor&amp;author=J.M.%20Thornton&amp;publication_year=1992&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=8&amp;pages=275-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1633570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nagano</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Aii</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Adachi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Genome size analysis of the genus <em>Fagopyrum</em></div>, <div class="source">Fagopyrum</div> , <div class="volume">17</div>, <div class="fpage">35</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="DSW012C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nowak</div>
               <div class="given-names">M.D.</div>
            </div>, <div class="name string-name"><div class="surname">Russo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schlapbach</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Huu</div> <div class="given-names">C.N.</div></div>, <div class="name string-name"><div class="surname">Lenhard</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Conti</div> <div class="given-names">E.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">The draft genome of <em>Primula veris</em> yields insights into the molecular basis of heterostyly</div>, <div class="source">Genome Biol.</div> , <div class="volume">16</div>, <div class="fpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Primula%20veris%20yields%20insights%20into%20the%20molecular%20basis%20of%20heterostyly&amp;author=M.D.%20Nowak&amp;author=G.%20Russo&amp;author=R.%20Schlapbach&amp;author=C.N.%20Huu&amp;author=M.%20Lenhard&amp;author=E.%20Conti&amp;publication_year=2015&amp;journal=Genome%20Biol.&amp;volume=16&amp;pages=12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0567-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0567-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0567-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25651398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kitashiba</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Draft sequences of the radish (<em>Raphanus sativus</em> L.) genome</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">481</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20sequences%20of%20the%20radish%20(Raphanus%20sativus%20L.)%20genome&amp;author=H.%20Kitashiba&amp;author=F.%20Li&amp;author=H.%20Hirakawa&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=481-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24848699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cough</div>
               <div class="given-names">J.F.</div>
            </div>, <div class="name string-name"><div class="surname">Naghski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Krewson</div> <div class="given-names">C.F.</div></div></div>
          <div class="year">1946</div>, <div class="article-title">Buckwheat as a source of rutin</div>, <div class="source">Science</div> , <div class="volume">15</div>, <div class="fpage">197</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Buckwheat%20as%20a%20source%20of%20rutin&amp;author=J.F.%20Cough&amp;author=J.%20Naghski&amp;author=C.F.%20Krewson&amp;publication_year=1946&amp;journal=Science&amp;volume=15&amp;pages=197-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.103.2668.197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.103.2668.197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.103.2668.197"> </span></div> </div></div></div></div></div><div content-id="DSW012C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">N.I.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Woo</div> <div class="given-names">S-H.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">C.H.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">S.U.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Differential expression of flavonoid biosynthesis genes and accumulation of phenolic compounds in common buckwheat (<em>Fagopyrum esculentum</em>)</div>, <div class="source">J. Agric. Food Chem.</div> , <div class="volume">58</div>, <div class="fpage">12176</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20flavonoid%20biosynthesis%20genes%20and%20accumulation%20of%20phenolic%20compounds%20in%20common%20buckwheat%20(Fagopyrum%20esculentum)&amp;author=X.%20Li&amp;author=N.I.%20Park&amp;author=H.%20Xu&amp;author=S-H.%20Woo&amp;author=C.H.%20Park&amp;author=S.U.%20Park&amp;publication_year=2010&amp;journal=J.%20Agric.%20Food%20Chem.&amp;volume=58&amp;pages=12176-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf103310g" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf103310g" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf103310g"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21062042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gupta</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Ranab</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Chauhan</div> <div class="given-names">R.S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Expression of flavonoid biosynthesis genes vis-à-vis rutin content variation in different growth stages of <em>Fagopyrum</em> species</div>, <div class="source">J. Plant Physiol.</div> , <div class="volume">168</div>, <div class="fpage">2117</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20flavonoid%20biosynthesis%20genes%20vis-%C3%A0-vis%20rutin%20content%20variation%20in%20different%20growth%20stages%20of%20Fagopyrum%20species&amp;author=N.%20Gupta&amp;author=S.K.%20Sharma&amp;author=J.C.%20Ranab&amp;author=R.S.%20Chauhan&amp;publication_year=2011&amp;journal=J.%20Plant%20Physiol.&amp;volume=168&amp;pages=2117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2011.06.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2011.06.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2011.06.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21872967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quattrocchio</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Wing</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">van der Woude</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mol</div> <div class="given-names">J.N.M.</div></div>, <div class="name string-name"><div class="surname">Koes</div> <div class="given-names">R.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Analysis of bHLH and MYB domain proteins: species-specific regulatory differences are caused by divergent evolution of target anthocyanin genes</div>, <div class="source">Plant J.</div> , <div class="volume">13</div>, <div class="fpage">475</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20bHLH%20and%20MYB%20domain%20proteins%3A%20species-specific%20regulatory%20differences%20are%20caused%20by%20divergent%20evolution%20of%20target%20anthocyanin%20genes&amp;author=F.%20Quattrocchio&amp;author=J.F.%20Wing&amp;author=K.%20van%20der%20Woude&amp;author=J.N.M.%20Mol&amp;author=R.%20Koes&amp;publication_year=1998&amp;journal=Plant%20J.&amp;volume=13&amp;pages=475-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1998.00046.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1998.00046.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1998.00046.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9680994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Walker</div>
               <div class="given-names">A.R.</div>
            </div>, <div class="name string-name"><div class="surname">Davison</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Bolognesi-Winfield</div> <div class="given-names">A.C.</div></div>et al.  </div>. <div class="year">1999</div>, <div class="article-title">The <em>TRANSPARENT TESTA GLABRA1</em> locus, which regulates trichome differentiation and anthocyanin biosynthesis in Arabidopsis, encodes a WD40 repeat protein</div>, <div class="source">Plant Cell</div> , <div class="volume">11</div>, <div class="fpage">1337</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TRANSPARENT%20TESTA%20GLABRA1%20locus%2C%20which%20regulates%20trichome%20differentiation%20and%20anthocyanin%20biosynthesis%20in%20Arabidopsis%2C%20encodes%20a%20WD40%20repeat%20protein&amp;author=A.R.%20Walker&amp;author=P.A.%20Davison&amp;author=A.C.%20Bolognesi-Winfield&amp;publication_year=1999&amp;journal=Plant%20Cell&amp;volume=11&amp;pages=1337-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.11.7.1337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.11.7.1337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.11.7.1337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10402433" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baudry</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Heim</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Dubreucq</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Caboche</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lepiniec</div> <div class="given-names">L.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">TT2, TT8, and TTG1 synergistically specify the expression of <em>BANYULS</em> and proanthocyanidin biosynthesis in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="volume">39</div>, <div class="fpage">366</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TT2%2C%20TT8%2C%20and%20TTG1%20synergistically%20specify%20the%20expression%20of%20BANYULS%20and%20proanthocyanidin%20biosynthesis%20in%20Arabidopsis%20thaliana&amp;author=A.%20Baudry&amp;author=M.A.%20Heim&amp;author=B.%20Dubreucq&amp;author=M.%20Caboche&amp;author=B.%20Weisshaar&amp;author=L.%20Lepiniec&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=39&amp;pages=366-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15255866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gonzalez</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Leavitt</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Lloyd</div> <div class="given-names">A.M.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Regulation of the anthocyanin biosynthetic pathway by the TTG1/bHLH/Myb transcriptional complex in Arabidopsis seedlings</div>, <div class="source">Plant J.</div> , <div class="volume">53</div>, <div class="fpage">814</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20the%20anthocyanin%20biosynthetic%20pathway%20by%20the%20TTG1%2FbHLH%2FMyb%20transcriptional%20complex%20in%20Arabidopsis%20seedlings&amp;author=A.%20Gonzalez&amp;author=M.%20Zhao&amp;author=J.M.%20Leavitt&amp;author=A.M.%20Lloyd&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=53&amp;pages=814-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2008.53.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2008.53.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2008.53.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18036197" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martin</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Paz-Ares</div> <div class="given-names">J.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">MYB transcriptiofnactors in plant</div>, <div class="source">Trends Genet.</div> , <div class="volume">13</div>, <div class="fpage">67</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB%20transcriptiofnactors%20in%20plant&amp;author=C.%20Martin&amp;author=J.%20Paz-Ares&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=67-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(96)10049-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(96)10049-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(96)10049-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9055608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stracke</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Werber</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">The <em>R2R3-MYB</em> gene family in <em>Arabidopsis thaliana</em></div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">4</div>, <div class="fpage">447</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20R2R3-MYB%20gene%20family%20in%20Arabidopsis%20thaliana&amp;author=R.%20Stracke&amp;author=M.%20Werber&amp;author=B.%20Weisshaar&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=4&amp;pages=447-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(00)00199-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(00)00199-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(00)00199-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11597504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dubos</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Stracke</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Grotewold</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lepiniec</div> <div class="given-names">L.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">MYB transcription factors in <em>Arabidopsis</em></div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">15</div>, <div class="fpage">573</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB%20transcription%20factors%20in%20Arabidopsis&amp;author=C.%20Dubos&amp;author=R.%20Stracke&amp;author=E.%20Grotewold&amp;author=B.%20Weisshaar&amp;author=C.%20Martin&amp;author=L.%20Lepiniec&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci.&amp;volume=15&amp;pages=573-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.06.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20674465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Takos</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Jaffé</div> <div class="given-names">F.W.</div></div>, <div class="name string-name"><div class="surname">Jacob</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Bogs</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">A.R.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Light-induced expression of a MYB gene regulates anthocyanin biosynthesis in red apples</div>, <div class="source">Plant Physiol.</div> , <div class="volume">142</div>, <div class="fpage">1216</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Light-induced%20expression%20of%20a%20MYB%20gene%20regulates%20anthocyanin%20biosynthesis%20in%20red%20apples&amp;author=A.M.%20Takos&amp;author=F.W.%20Jaff%C3%A9&amp;author=S.R.%20Jacob&amp;author=J.%20Bogs&amp;author=S.P.%20Robinson&amp;author=A.R.%20Walker&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=142&amp;pages=1216-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.088104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.088104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.088104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17012405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akagi</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ikegami</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tsujimoto</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">DkMyb4 is a Myb transcription factor involved in proanthocyanidin biosynthesis in persimmon fruit</div>, <div class="source">Plant Physiol.</div> , <div class="volume">151</div>, <div class="fpage">2028</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DkMyb4%20is%20a%20Myb%20transcription%20factor%20involved%20in%20proanthocyanidin%20biosynthesis%20in%20persimmon%20fruit&amp;author=T.%20Akagi&amp;author=A.%20Ikegami&amp;author=T.%20Tsujimoto&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=2028-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.146985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.146985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.146985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19783643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nishihara</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nakatsuka</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Molecular characterization of mutations in white-flowered torenia plants</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20mutations%20in%20white-flowered%20torenia%20plants&amp;author=M.%20Nishihara&amp;author=E.%20Yamada&amp;author=M.%20Saito&amp;author=K.%20Fujita&amp;author=H.%20Takahashi&amp;author=T.%20Nakatsuka&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-86" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-86" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-86"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24694353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhu</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Characterization of the <em>cis</em> elements in the proximal promoter regions of the anthocyanin pathway genes reveals a common regulatory logic that governs pathway regulation</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">66</div>, <div class="fpage">3775</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20cis%20elements%20in%20the%20proximal%20promoter%20regions%20of%20the%20anthocyanin%20pathway%20genes%20reveals%20a%20common%20regulatory%20logic%20that%20governs%20pathway%20regulation&amp;author=Z.%20Zhu&amp;author=H.%20Wang&amp;author=Y.%20Wang&amp;publication_year=2015&amp;journal=J.%20Exp.%20Bot.&amp;volume=66&amp;pages=3775-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erv173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferv173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferv173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25911741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ban</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Honda</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hatsuyama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Igarashi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bessho</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Moriguchi</div> <div class="given-names">T.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Isolation and functional analysis of a MYB transcription factor gene that is a key regulator for the development of red coloration in apple skin</div>, <div class="source">Plant Cell Physiol.</div> , <div class="volume">48</div>, <div class="fpage">958</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20functional%20analysis%20of%20a%20MYB%20transcription%20factor%20gene%20that%20is%20a%20key%20regulator%20for%20the%20development%20of%20red%20coloration%20in%20apple%20skin&amp;author=Y.%20Ban&amp;author=C.%20Honda&amp;author=Y.%20Hatsuyama&amp;author=M.%20Igarashi&amp;author=H.%20Bessho&amp;author=T.%20Moriguchi&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=958-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yi</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Derynck</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Telmer</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Marsolais</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Dhaubhadel</div> <div class="given-names">S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">A single-repeat MYB transcription factor, GmMYB176, regulates <em>CHS8</em> gene expression and affects isoflavonoid biosynthesis in soybean</div>, <div class="source">Plant J.</div> , <div class="volume">62</div>, <div class="fpage">1019</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single-repeat%20MYB%20transcription%20factor%2C%20GmMYB176%2C%20regulates%20CHS8%20gene%20expression%20and%20affects%20isoflavonoid%20biosynthesis%20in%20soybean&amp;author=J.%20Yi&amp;author=M.R.%20Derynck&amp;author=X.%20Li&amp;author=P.%20Telmer&amp;author=F.%20Marsolais&amp;author=S.%20Dhaubhadel&amp;publication_year=2010&amp;journal=Plant%20J.&amp;volume=62&amp;pages=1019-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20345602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Akasawa</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Pepsin-resistant 16-kD buckwheat protein is associated with immediate hypersensitivity reaction in patients with buckwheat allergy</div>, <div class="source">Int. Arch. Allergy Immunol.</div> , <div class="volume">129</div>, <div class="fpage">49</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pepsin-resistant%2016-kD%20buckwheat%20protein%20is%20associated%20with%20immediate%20hypersensitivity%20reaction%20in%20patients%20with%20buckwheat%20allergy&amp;author=K.%20Tanaka&amp;author=K.%20Matsumoto&amp;author=A.%20Akasawa&amp;publication_year=2002&amp;journal=Int.%20Arch.%20Allergy%20Immunol.&amp;volume=129&amp;pages=49-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000065173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000065173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000065173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12372998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Matsumoto</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Fujino</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Molecular characterization of a 10-kDa buckwheat molecule reactive to allergic patients’ IgE</div>, <div class="source">Allergy</div> , <div class="volume">59</div>, <div class="fpage">533</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20a%2010-kDa%20buckwheat%20molecule%20reactive%20to%20allergic%20patients%E2%80%99%20IgE&amp;author=R.%20Matsumoto&amp;author=K.%20Fujino&amp;author=Y.%20Nagata&amp;publication_year=2004&amp;journal=Allergy&amp;volume=59&amp;pages=533-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1398-9995.2003.00412.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1398-9995.2003.00412.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1398-9995.2003.00412.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15080835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Satoh</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Koyano</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Teshima</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sawada</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Immunological characterization and mutational analysis of the recombinant protein BWp16, a major allergen in buckwheat</div>, <div class="source">Biol. Pharm. Bull.</div> , <div class="volume">31</div>, <div class="fpage">1079</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunological%20characterization%20and%20mutational%20analysis%20of%20the%20recombinant%20protein%20BWp16%2C%20a%20major%20allergen%20in%20buckwheat&amp;author=R.%20Satoh&amp;author=S.%20Koyano&amp;author=K.%20Takagi&amp;author=R.%20Nakamura&amp;author=R.%20Teshima&amp;author=J.%20Sawada&amp;publication_year=2008&amp;journal=Biol.%20Pharm.%20Bull.&amp;volume=31&amp;pages=1079-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1248/bpb.31.1079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1248%2Fbpb.31.1079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1248%2Fbpb.31.1079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18520034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sonnewald</div>
               <div class="given-names">U.</div>
            </div>, <div class="name string-name"><div class="surname">Kossmann</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Starches—from current models to genetic engineering</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">11</div>, <div class="fpage">223</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Starches%E2%80%94from%20current%20models%20to%20genetic%20engineering&amp;author=U.%20Sonnewald&amp;author=J.%20Kossmann&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=11&amp;pages=223-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2013.11.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2013.11.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2013.11.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23190212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Copeland</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Blazek</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Salman</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">M.C.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Form and fuctionality of starch</div>, <div class="source">Food Hydrocoll.</div> , <div class="volume">23</div>, <div class="fpage">1527</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Form%20and%20fuctionality%20of%20starch&amp;author=L.%20Copeland&amp;author=J.%20Blazek&amp;author=H.%20Salman&amp;author=M.C.%20Tang&amp;publication_year=2009&amp;journal=Food%20Hydrocoll.&amp;volume=23&amp;pages=1527-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodhyd.2008.09.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodhyd.2008.09.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodhyd.2008.09.016"> </span></div> </div></div></div></div></div><div content-id="DSW012C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Srichuwong</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sunarti</div> <div class="given-names">T.C.</div></div>, <div class="name string-name"><div class="surname">Mishima</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Isono</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hisamatsu</div> <div class="given-names">M.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Starches from different botanical sources I: contribution of amylopectin fine structure to thermal properties and enzyme digestibility</div>, <div class="source">Carbohydr. Polym.</div> , <div class="volume">60</div>, <div class="fpage">529</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Starches%20from%20different%20botanical%20sources%20I%3A%20contribution%20of%20amylopectin%20fine%20structure%20to%20thermal%20properties%20and%20enzyme%20digestibility&amp;author=S.%20Srichuwong&amp;author=T.C.%20Sunarti&amp;author=T.%20Mishima&amp;author=N.%20Isono&amp;author=M.%20Hisamatsu&amp;publication_year=2005&amp;journal=Carbohydr.%20Polym.&amp;volume=60&amp;pages=529-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.carbpol.2005.03.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.carbpol.2005.03.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.carbpol.2005.03.004"> </span></div> </div></div></div></div></div><div content-id="DSW012C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jobling</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2004</div>, <div class="article-title">Improving starch for food and industrial applications</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">7</div>, <div class="fpage">210</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20starch%20for%20food%20and%20industrial%20applications&amp;author=S.%20Jobling&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=7&amp;pages=210-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2003.12.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2003.12.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2003.12.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15003223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsai</div>
               <div class="given-names">C-Y.</div>
            </div>
          </div>
          <div class="year">1974</div>, <div class="article-title">The function of the waxy locus in starch synthesis in maize endosperm</div>, <div class="source">Biochem. Genet.</div> , <div class="volume">11</div>, <div class="fpage">83</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20function%20of%20the%20waxy%20locus%20in%20starch%20synthesis%20in%20maize%20endosperm&amp;author=C-Y.%20Tsai&amp;publication_year=1974&amp;journal=Biochem.%20Genet.&amp;volume=11&amp;pages=83-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00485766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00485766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00485766"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4824506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cao</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Imparl-Radosevich</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Guan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Keeling</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">James</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">A.M.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Identification of the soluble starch synthase activities of maize endosperm</div>, <div class="source">Plant Physiol.</div> , <div class="volume">120</div>, <div class="fpage">205</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20soluble%20starch%20synthase%20activities%20of%20maize%20endosperm&amp;author=H.%20Cao&amp;author=J.%20Imparl-Radosevich&amp;author=H.%20Guan&amp;author=P.L.%20Keeling&amp;author=M.G.%20James&amp;author=A.M.%20Myers&amp;publication_year=1999&amp;journal=Plant%20Physiol.&amp;volume=120&amp;pages=205-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.120.1.205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.120.1.205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.120.1.205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10318698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ball</div>
               <div class="given-names">S.G.</div>
            </div>, <div class="name string-name"><div class="surname">Morell</div> <div class="given-names">M.K.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">From bacterial glycogen to starch: understanding the biogenesis of the plant starch granule</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="volume">54</div>, <div class="fpage">207</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20bacterial%20glycogen%20to%20starch%3A%20understanding%20the%20biogenesis%20of%20the%20plant%20starch%20granule&amp;author=S.G.%20Ball&amp;author=M.K.%20Morell&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=54&amp;pages=207-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.54.031902.134927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.54.031902.134927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.54.031902.134927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14502990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Identification and characterization of granule bound starch synthase I (GBSSI) gene of tartary buckwheat (<em>Fagopyrum tataricum</em> Gaertn.)</div>, <div class="source">Gene</div> , <div class="volume">534</div>, <div class="fpage">229</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20granule%20bound%20starch%20synthase%20I%20(GBSSI)%20gene%20of%20tartary%20buckwheat%20(Fagopyrum%20tataricum%20Gaertn.)&amp;author=X.%20Wang&amp;author=B.%20Feng&amp;author=Z.%20Xu&amp;publication_year=2014&amp;journal=Gene&amp;volume=534&amp;pages=229-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2013.10.053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2013.10.053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2013.10.053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24211386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Khan</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">W-M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Diversification of genes encoding granule-bound starch synthase in monocots and dicots is marked by multiple genome-wide duplication events</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e30088</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20genes%20encoding%20granule-bound%20starch%20synthase%20in%20monocots%20and%20dicots%20is%20marked%20by%20multiple%20genome-wide%20duplication%20events&amp;author=J.%20Cheng&amp;author=M.A.%20Khan&amp;author=W-M.%20Qiu&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30088" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22291904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamori</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Quynh</div> <div class="given-names">N.T.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Differential effects of Wx-A1, -B1 and -D1 protein deficiencies on apparent amylose content and starch pasting properties in common wheat</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">100</div>, <div class="fpage">32</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20effects%20of%20Wx-A1%2C%20-B1%20and%20-D1%20protein%20deficiencies%20on%20apparent%20amylose%20content%20and%20starch%20pasting%20properties%20in%20common%20wheat&amp;author=M.%20Yamamori&amp;author=N.T.%20Quynh&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=100&amp;pages=32-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050005"> </span></div> </div></div></div></div></div><div content-id="DSW012C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Garber</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Quisenberry</div> <div class="given-names">K.S.</div></div></div>
          <div class="year">1927</div>, <div class="article-title">The inheritance of length of style in buckwheat</div>, <div class="source">J. Agric. Res.</div> , <div class="volume">34</div>, <div class="fpage">181</div>–<div class="lpage">3</div>.</div></div></div></div><div content-id="DSW012C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lewis</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">D.A.</div></div></div>
          <div class="year">1992</div>, <div class="article-title">The genetics of heterostyly</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Barrett</div> <div class="given-names">S.C.H.</div></div></div> (Ed.), <div class="source">Evolution and function of heterostyly</div> . <div class="publisher-name">Springer-Verlag</div>: <div class="publisher-loc">Berlin</div>, pp. <div class="fpage">129</div>–<div class="lpage">50</div>.</div></div></div></div><div content-id="DSW012C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vierstra</div>
               <div class="given-names">R.D.</div>
            </div>
          </div>
          <div class="year">2003</div>, <div class="article-title">The ubiquitin/26S proteasome pathway, the complex last chapter in the life of many plant proteins</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">8</div>, <div class="fpage">135</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ubiquitin%2F26S%20proteasome%20pathway%2C%20the%20complex%20last%20chapter%20in%20the%20life%20of%20many%20plant%20proteins&amp;author=R.D.%20Vierstra&amp;publication_year=2003&amp;journal=Trends%20Plant%20Sci.&amp;volume=8&amp;pages=135-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(03)00014-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(03)00014-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(03)00014-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hua</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Kao</div> <div class="given-names">T.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Identification and characterization of components of a putative <em>Petunia S</em>-Locus F-box–containing E3 ligase complex involved in S-RNase–based self-incompatibility</div>, <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">2531</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20components%20of%20a%20putative%20Petunia%20S-Locus%20F-box%E2%80%93containing%20E3%20ligase%20complex%20involved%20in%20S-RNase%E2%80%93based%20self-incompatibility&amp;author=Z.%20Hua&amp;author=T.%20Kao&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=2531-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.041061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.041061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.041061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW012C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW012C75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsai</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Howell</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nitcher</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Discovery of rare mutations in populations: TILLING by sequencing</div>, <div class="source">Plant Physiol.</div> , <div class="volume">156</div>, <div class="fpage">1257</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20rare%20mutations%20in%20populations%3A%20TILLING%20by%20sequencing&amp;author=H.%20Tsai&amp;author=T.%20Howell&amp;author=R.%20Nitcher&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1257-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.169748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.169748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.169748"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21531898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745460&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw012/1/dsw012_Supplementary_Data.zip?Expires=2147483647&amp;Signature=mHuorOE2BJqI0KXS85GnFVwFT0Fk7WwJdD70HLDSNnSFYQDVXbQb4bswQL0~QYgc4HNf5JT9k9Eb~Sc4k643cHjjfWfQEQDW-6sQrz4bTgkXnDlAGa3IGivTFts9i16hr6owNUg4ZOqZXOeqLuZDhe~le37cI4d19l~1z4JkN5dwqt2v8FIB2M8OYEaPqFPRzcejRWvOXqnYV1ElyPjFxJmbqLK8s6dBTyOLt~zZfwZC~~CbyVT06Z8AE5KtApveHSdqD69TnCtPapFHt~3vWzGU5Q7UObrNyuRc6R9uVHIysBCG1YhNn8uXA6F5JfTsGUR0j-ZgjkczR5Az3aaBSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,711</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745460"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f215%2f1745460">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F215%2F1745460" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745460'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f215%2f1745460">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

