

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/4/291/502681","@type":"ScholarlyArticle","name":"The Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing","datePublished":"2011-07-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2011-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr022","keywords":["segmented filamentous bacterium","non-culturable bacteria","genome","flagella"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/4/291/502681","author":[{"name":"Kuwahara, Tomomi","affiliation":"1  Department of Microbiology, Faculty of Medicine ,  Kagawa University ,  1750-1 Miki-cho, Kita-gun, Kagawa 761-0793 ,  Japan","@type":"Person"},{"name":"Ogura, Yositoshi","affiliation":"2  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan    3  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"4  Department of Computational Biology, Graduate School of Frontier Sciences ,  University of Tokyo ,  Chiba ,  Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"5  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 Nagatsuta, Yokohama, Kanagawa 226-0026 ,  Japan","@type":"Person"},{"name":"Ooka, Tadasuke","affiliation":"3  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"6  Department of Plant Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"5  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 Nagatsuta, Yokohama, Kanagawa 226-0026 ,  Japan","@type":"Person"},{"name":"Nakayama-Imaohji, Haruyuki","affiliation":"7  Department of Parasitology and Immunology ,  Institute of Health Biosciences, The University of Tokushima Graduate School ,  3-18-15 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Ichimura, Minoru","affiliation":"7  Department of Parasitology and Immunology ,  Institute of Health Biosciences, The University of Tokushima Graduate School ,  3-18-15 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Itoh, Kikuji","affiliation":"8  Graduate School of Agricultural and Life Sciences ,  University of Tokyo ,  1-1-1 Yayoi, Bunkyo, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Ishifune, Chieko","affiliation":"7  Department of Parasitology and Immunology ,  Institute of Health Biosciences, The University of Tokushima Graduate School ,  3-18-15 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Maekawa, Yoichi","affiliation":"7  Department of Parasitology and Immunology ,  Institute of Health Biosciences, The University of Tokushima Graduate School ,  3-18-15 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Yasutomo, Koji","affiliation":"7  Department of Parasitology and Immunology ,  Institute of Health Biosciences, The University of Tokushima Graduate School ,  3-18-15 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"4  Department of Computational Biology, Graduate School of Frontier Sciences ,  University of Tokyo ,  Chiba ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"2  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan    3  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"}],"description":"Abstract.  Numerous microbes inhabit the mammalian intestinal track and strongly impact host physiology; however, our understanding of this ecosystem remains li","pageStart":"291","pageEnd":"303","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing" />
    <meta property="og:description" content="Abstract.  Numerous microbes inhabit the mammalian intestinal track and strongly impact host physiology; however, our understanding of this ecosystem remains li" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/4/291/502681" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-07-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kuwahara, Tomomi" /><meta name="citation_author_institution" content="Department of Microbiology, Faculty of Medicine, Kagawa University, 1750-1 Miki-cho, Kita-gun, Kagawa 761-0793, Japan" /><meta name="citation_author" content="Ogura, Yositoshi" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, Chiba, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsuta, Yokohama, Kanagawa 226-0026, Japan" /><meta name="citation_author" content="Ooka, Tadasuke" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Department of Plant Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 Nagatsuta, Yokohama, Kanagawa 226-0026, Japan" /><meta name="citation_author" content="Nakayama-Imaohji, Haruyuki" /><meta name="citation_author_institution" content="Department of Parasitology and Immunology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-15 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Ichimura, Minoru" /><meta name="citation_author_institution" content="Department of Parasitology and Immunology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-15 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Itoh, Kikuji" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, University of Tokyo, 1-1-1 Yayoi, Bunkyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Ishifune, Chieko" /><meta name="citation_author_institution" content="Department of Parasitology and Immunology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-15 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Maekawa, Yoichi" /><meta name="citation_author_institution" content="Department of Parasitology and Immunology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-15 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Yasutomo, Koji" /><meta name="citation_author_institution" content="Department of Parasitology and Immunology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-15 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, Chiba, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_title" content="The Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing" /><meta name="citation_firstpage" content="291" /><meta name="citation_lastpage" content="303" /><meta name="citation_doi" content="10.1093/dnares/dsr022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2011/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Host-bacterial mutualism in the human intestine; Science;  citation_year=2005;  citation_volume=307;  citation_pages=1915-20; " /><meta name="citation_reference" content="citation_title=How host-microbial interactions shape the nutrient environment of the mammalian intestine; Ann. Rev. Nutr.;  citation_year=2002;  citation_volume=22;  citation_pages=283-307; " /><meta name="citation_reference" content="citation_title=Experimental models of inflammatory bowel disease reveal innate, adaptive, and regulatory mechanisms of host dialogue with the microbiota; Immunol. Rev.;  citation_year=2005;  citation_volume=206;  citation_pages=260-76; " /><meta name="citation_reference" content="citation_title=Microbial influences in inflammatory bowel diseases; Gastroenterology;  citation_year=2008;  citation_volume=134;  citation_pages=577-94; " /><meta name="citation_reference" content="citation_title=Bacteria-induced intestinal cancer in mice with disrupted gpx1 and gpx2 genes; Cancer Res.;  citation_year=2004;  citation_volume=64;  citation_pages=962-8; " /><meta name="citation_reference" content="citation_title=Microbial ecology: human gut microbes associated with obesity; Nature;  citation_year=2006;  citation_volume=444;  citation_pages=1022-3; " /><meta name="citation_reference" content="citation_title=An obesity-associated gut microbiome with increased capacity for energy harvest; Nature;  citation_year=2006;  citation_volume=444;  citation_pages=1027-31; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis of the human distal gut microbiome; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1355-9; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=Bacterial community variation in human body habitats across space and time; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1694-7; " /><meta name="citation_reference" content="citation_title=A human microbial gene catalogue established by metagenomic sequencing; Nature;  citation_year=2010;  citation_volume=464;  citation_pages=59-67; " /><meta name="citation_reference" content="citation_title=Dissecting biological ‘dark matter’ with single-cell genetic analysis of rare and uncultivated TM7 microbes from the human mouth; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=11889-94; " /><meta name="citation_reference" content="citation_title=Intestinal, segmented, filamentous bacteria; FEMS Microbiol. Rev.;  citation_year=1992;  citation_volume=88;  citation_pages=165-80; " /><meta name="citation_reference" content="citation_title=Intestinal, segmented, filamentous bacteria in a wide range of vertebrate species; Lab. Anim.;  citation_year=1993;  citation_volume=27;  citation_pages=141-50; " /><meta name="citation_reference" content="citation_title=Segmented filamentous bacteria are indigenous intestinal bacteria that activate intraepithelial lymphocytes and induce MHC class II molecules and fucosyl asialo GM1 glycolipids on the small intestinal epithelial cells in the ex-germ-free mouse; Microbiol. Immunol.;  citation_year=1995;  citation_volume=39;  citation_pages=555-62; " /><meta name="citation_reference" content="citation_title=Apathogenic, intestinal, segmented, filamentous bacteria stimulate the mucosal immune system of mice; Infect. Immun.;  citation_year=1993;  citation_volume=61;  citation_pages=303-6; " /><meta name="citation_reference" content="citation_title=Aberrant expansion of segmented filamentous bacteria in IgA-deficient gut; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=1981-6; " /><meta name="citation_reference" content="citation_title=Induction of intestinal Th17 cells by segmented filamentous bacteria; Cell;  citation_year=2009;  citation_volume=139;  citation_pages=485-98; " /><meta name="citation_reference" content="citation_title=The key role of segmented filamentous bacteria in the coordinated maturation of gut helper T cell response; Immunity;  citation_year=2009;  citation_volume=31;  citation_pages=677-89; " /><meta name="citation_reference" content="citation_title=Distinct gene expression profiles characterize cellular phenotypes of follicle-associated epithelium and M cells; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=127-37; " /><meta name="citation_reference" content="citation_title=An improved DNA isolation method for metagenomic analysis of the microbial flora of the human intestine; Microb. Environ.;  citation_year=2007;  citation_volume=22;  citation_pages=214-22; " /><meta name="citation_reference" content="citation_title=Microbial Genome Annotation Pipeline (MiGAP) for diverse users;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=The innate immune response to bacterial flagellin is mediated by Toll-like receptor 5; Nature;  citation_year=2001;  citation_volume=410;  citation_pages=1099-103; " /><meta name="citation_reference" content="citation_title=CRISPR/Cas, the immune system of bacteria and archaea; Science;  citation_year=2010;  citation_volume=327;  citation_pages=167-70; " /><meta name="citation_reference" content="citation_title=CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea; Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=181-90; " /><meta name="citation_reference" content="citation_title=The multidrug-resistant human pathogen Clostridium difficile has a highly mobile, mosaic genome; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=779-86; " /><meta name="citation_reference" content="citation_title=Comparison of 16S rRNA sequences of segmented filamentous bacteria isolated from mice, rats, and chickens and proposal of ‘Candidatus arthromitus; Int. J. Syst. Bacteriol.;  citation_year=1995;  citation_volume=133;  citation_pages=445-50; " /><meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life; Science;  citation_year=2006;  citation_volume=311;  citation_pages=1283-7; " /><meta name="citation_reference" content="citation_title=The genome and transcriptomes of the anti-tumor agent Clostridium novyi-NT; Nat. Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=1573-80; " /><meta name="citation_reference" content="citation_title=Clostridium cellulolyticum: model organism of mesophilic cellulolytic clostridia; FEMS Microbiol. Rev.;  citation_year=2005;  citation_volume=29;  citation_pages=741-64; " /><meta name="citation_reference" content="citation_title=A novel class of modular transporters for vitamins in prokaryotes; J. Bacteriol.;  citation_year=2009;  citation_volume=191;  citation_pages=42-51; " /><meta name="citation_reference" content="citation_title=Sporulation of Bacillus subtilis; Curr. Opin. Microbiol.;  citation_year=2004;  citation_volume=7;  citation_pages=579-86; " /><meta name="citation_reference" content="citation_title=Comparative genomic view of clostridial sporulation and physiology; Nat. Rev. Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=969-78; " /><meta name="citation_reference" content="citation_title=Spore germination; Cell Mol. Life Sci.;  citation_year=2002;  citation_volume=59;  citation_pages=403-9; " /><meta name="citation_reference" content="citation_title=Cyanobacterial cell inclusions; Annu. Rev. Microbiol.;  citation_year=1984;  citation_volume=38;  citation_pages=1-25; " /><meta name="citation_reference" content="citation_title=Analysis of genome sequences for genes of cyanophycin metabolism: identifying putative cyanophycin metabolizing prokaryotes; Macromol. Biosci.;  citation_year=2007;  citation_volume=7;  citation_pages=278-96; " /><meta name="citation_reference" content="citation_title=Segmented filamentous bacteria interact with intraepithelial mononuclear cells; Infect. Immun.;  citation_year=2002;  citation_volume=70;  citation_pages=3277-80; " /><meta name="citation_reference" content="citation_title=Transmission electron microscopic demonstration of phagocytosis and intracellular processing of segmented filamentous bacteria by intestinal epithelial cells of the chick ileum; Infect. Immun.;  citation_year=2000;  citation_volume=68;  citation_pages=6496-504; " /><meta name="citation_reference" content="citation_title=Toll-like receptor 5 recognizes a conserved site on flagellin required for protofilament formation and bacterial motility; Nat. Immunol.;  citation_year=2003;  citation_volume=4;  citation_pages=1247-53; " /><meta name="citation_reference" content="citation_title=Detection of pathogenic intestinal bacteria by Toll-like receptor 5 on intestinal CD11c+ lamina propria cells; Nat. Immunol.;  citation_year=2006;  citation_volume=7;  citation_pages=868-74; " /><meta name="citation_reference" content="citation_title=Regulation of humoral and cellular gut immunity by lamina propria dendritic cells expressing Toll-like receptor 5; Nat. Immunol.;  citation_year=2008;  citation_volume=9;  citation_pages=769-76; " /><meta name="citation_reference" content="citation_title=Immune responses of TLR5+ lamina propria dendritic cells in enterobacterial infection; J. Gastroenterol.;  citation_year=2009;  citation_volume=44;  citation_pages=803-11; " /><meta name="citation_reference" content="citation_title=Listeria monocytogenes phospholipase C-dependent calcium signaling modulates bacterial entry into J774 macrophage-like cells; Infect. Immun.;  citation_year=1999;  citation_volume=67;  citation_pages=1770-8; " /><meta name="citation_reference" content="citation_title=Identification and molecular characterization of Salmonella typhimurium gene involved in triggering the internalization of salmonellae into cultured epithelial cells; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=5976-80; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/4/291/1352683/dsr022.pdf" /><meta name="description" content="Abstract.  Numerous microbes inhabit the mammalian intestinal track and strongly impact host physiology; however, our understanding of this ecosystem remains li" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/4/291/502681" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/4/291/502681" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/4/291/502681");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr022';
    </script>
<title>
	Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./502681" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bbp5PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXdhaGFyYSwgVG9tb21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3ksIEZhY3VsdHkgb2YgTWVkaWNpbmUsIEthZ2F3YSBVbml2ZXJzaXR5LCAxNzUwLTEgTWlraS1jaG8sIEtpdGEtZ3VuLCBLYWdhd2EgNzYxLTA3OTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2d1cmEsIFlvc2l0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEJpb2Vudmlyb25tZW50YWwgU2NpZW5jZSwgRnJvbnRpZXIgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIEtlbnNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBUb2t5bywgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCA0MjU5IE5hZ2F0c3V0YSwgWW9rb2hhbWEsIEthbmFnYXdhIDIyNi0wMDI2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9va2EsIFRhZGFzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpcmFrYXdhLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl0b2gsIFRha2VoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEluZm9ybWF0aW9uLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIDQyNTkgTmFnYXRzdXRhLCBZb2tvaGFtYSwgS2FuYWdhd2EgMjI2LTAwMjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhbWEtSW1hb2hqaSwgSGFydXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBhcmFzaXRvbG9neSBhbmQgSW1tdW5vbG9neSwgSW5zdGl0dXRlIG9mIEhlYWx0aCBCaW9zY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9rdXNoaW1hIEdyYWR1YXRlIFNjaG9vbCwgMy0xOC0xNSBLdXJhbW90by1jaG8sIFRva3VzaGltYSA3NzAtODUwMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJY2hpbXVyYSwgTWlub3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQYXJhc2l0b2xvZ3kgYW5kIEltbXVub2xvZ3ksIEluc3RpdHV0ZSBvZiBIZWFsdGggQmlvc2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3VzaGltYSBHcmFkdWF0ZSBTY2hvb2wsIDMtMTgtMTUgS3VyYW1vdG8tY2hvLCBUb2t1c2hpbWEgNzcwLTg1MDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgS2lrdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBhbmQgTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBUb2t5bywgMS0xLTEgWWF5b2ksIEJ1bmt5bywgVG9reW8gMTEzLTg2NTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNoaWZ1bmUsIENoaWVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGFyYXNpdG9sb2d5IGFuZCBJbW11bm9sb2d5LCBJbnN0aXR1dGUgb2YgSGVhbHRoIEJpb3NjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t1c2hpbWEgR3JhZHVhdGUgU2Nob29sLCAzLTE4LTE1IEt1cmFtb3RvLWNobywgVG9rdXNoaW1hIDc3MC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hZWthd2EsIFlvaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGFyYXNpdG9sb2d5IGFuZCBJbW11bm9sb2d5LCBJbnN0aXR1dGUgb2YgSGVhbHRoIEJpb3NjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t1c2hpbWEgR3JhZHVhdGUgU2Nob29sLCAzLTE4LTE1IEt1cmFtb3RvLWNobywgVG9rdXNoaW1hIDc3MC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illhc3V0b21vLCBLb2ppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQYXJhc2l0b2xvZ3kgYW5kIEltbXVub2xvZ3ksIEluc3RpdHV0ZSBvZiBIZWFsdGggQmlvc2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3VzaGltYSBHcmFkdWF0ZSBTY2hvb2wsIDMtMTgtMTUgS3VyYW1vdG8tY2hvLCBUb2t1c2hpbWEgNzcwLTg1MDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgTWFzYWhpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFRva3lvLCBDaGliYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXlhc2hpLCBUZXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgQmlvZW52aXJvbm1lbnRhbCBTY2llbmNlLCBGcm9udGllciBTY2llbmNlIFJlc2VhcmNoIENlbnRlciwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgNTIwMCBLaXlvdGFrZSwgTWl5YXpha2kgODg5LTE2OTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIExpZmVzdHlsZSBvZiB0aGUgU2VnbWVudGVkIEZpbGFtZW50b3VzIEJhY3Rlcml1bTogQSBOb24tQ3VsdHVyYWJsZSBHdXQtQXNzb2NpYXRlZCBJbW11bm9zdGltdWxhdGluZyBNaWNyb2JlIEluZmVycmVkIGJ5IFdob2xlLUdlbm9tZSBTZXF1ZW5jaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvc3QtYmFjdGVyaWFsIG11dHVhbGlzbSBpbiB0aGUgaHVtYW4gaW50ZXN0aW5lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwNzsgIGNpdGF0aW9uX3BhZ2VzPTE5MTUtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyBob3N0LW1pY3JvYmlhbCBpbnRlcmFjdGlvbnMgc2hhcGUgdGhlIG51dHJpZW50IGVudmlyb25tZW50IG9mIHRoZSBtYW1tYWxpYW4gaW50ZXN0aW5lOyBBbm4uIFJldi4gTnV0ci47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yODMtMzA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBlcmltZW50YWwgbW9kZWxzIG9mIGluZmxhbW1hdG9yeSBib3dlbCBkaXNlYXNlIHJldmVhbCBpbm5hdGUsIGFkYXB0aXZlLCBhbmQgcmVndWxhdG9yeSBtZWNoYW5pc21zIG9mIGhvc3QgZGlhbG9ndWUgd2l0aCB0aGUgbWljcm9iaW90YTsgSW1tdW5vbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDY7ICBjaXRhdGlvbl9wYWdlcz0yNjAtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBpbmZsdWVuY2VzIGluIGluZmxhbW1hdG9yeSBib3dlbCBkaXNlYXNlczsgR2FzdHJvZW50ZXJvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz01NzctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhLWluZHVjZWQgaW50ZXN0aW5hbCBjYW5jZXIgaW4gbWljZSB3aXRoIGRpc3J1cHRlZCBncHgxIGFuZCBncHgyIGdlbmVzOyBDYW5jZXIgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTk2Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgZWNvbG9neTogaHVtYW4gZ3V0IG1pY3JvYmVzIGFzc29jaWF0ZWQgd2l0aCBvYmVzaXR5OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQ0OyAgY2l0YXRpb25fcGFnZXM9MTAyMi0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvYmVzaXR5LWFzc29jaWF0ZWQgZ3V0IG1pY3JvYmlvbWUgd2l0aCBpbmNyZWFzZWQgY2FwYWNpdHkgZm9yIGVuZXJneSBoYXJ2ZXN0OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQ0OyAgY2l0YXRpb25fcGFnZXM9MTAyNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWMgYW5hbHlzaXMgb2YgdGhlIGh1bWFuIGRpc3RhbCBndXQgbWljcm9iaW9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0xMzU1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1ldGFnZW5vbWljcyByZXZlYWxlZCBjb21tb25seSBlbnJpY2hlZCBnZW5lIHNldHMgaW4gaHVtYW4gZ3V0IG1pY3JvYmlvbWVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGNvbW11bml0eSB2YXJpYXRpb24gaW4gaHVtYW4gYm9keSBoYWJpdGF0cyBhY3Jvc3Mgc3BhY2UgYW5kIHRpbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9MTY5NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGh1bWFuIG1pY3JvYmlhbCBnZW5lIGNhdGFsb2d1ZSBlc3RhYmxpc2hlZCBieSBtZXRhZ2Vub21pYyBzZXF1ZW5jaW5nOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY0OyAgY2l0YXRpb25fcGFnZXM9NTktNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3RpbmcgYmlvbG9naWNhbCDigJhkYXJrIG1hdHRlcuKAmSB3aXRoIHNpbmdsZS1jZWxsIGdlbmV0aWMgYW5hbHlzaXMgb2YgcmFyZSBhbmQgdW5jdWx0aXZhdGVkIFRNNyBtaWNyb2JlcyBmcm9tIHRoZSBodW1hbiBtb3V0aDsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0xMTg4OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXN0aW5hbCwgc2VnbWVudGVkLCBmaWxhbWVudG91cyBiYWN0ZXJpYTsgRkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz0xNjUtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVzdGluYWwsIHNlZ21lbnRlZCwgZmlsYW1lbnRvdXMgYmFjdGVyaWEgaW4gYSB3aWRlIHJhbmdlIG9mIHZlcnRlYnJhdGUgc3BlY2llczsgTGFiLiBBbmltLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE0MS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VnbWVudGVkIGZpbGFtZW50b3VzIGJhY3RlcmlhIGFyZSBpbmRpZ2Vub3VzIGludGVzdGluYWwgYmFjdGVyaWEgdGhhdCBhY3RpdmF0ZSBpbnRyYWVwaXRoZWxpYWwgbHltcGhvY3l0ZXMgYW5kIGluZHVjZSBNSEMgY2xhc3MgSUkgbW9sZWN1bGVzIGFuZCBmdWNvc3lsIGFzaWFsbyBHTTEgZ2x5Y29saXBpZHMgb24gdGhlIHNtYWxsIGludGVzdGluYWwgZXBpdGhlbGlhbCBjZWxscyBpbiB0aGUgZXgtZ2VybS1mcmVlIG1vdXNlOyBNaWNyb2Jpb2wuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NTU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcGF0aG9nZW5pYywgaW50ZXN0aW5hbCwgc2VnbWVudGVkLCBmaWxhbWVudG91cyBiYWN0ZXJpYSBzdGltdWxhdGUgdGhlIG11Y29zYWwgaW1tdW5lIHN5c3RlbSBvZiBtaWNlOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTMwMy02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYmVycmFudCBleHBhbnNpb24gb2Ygc2VnbWVudGVkIGZpbGFtZW50b3VzIGJhY3RlcmlhIGluIElnQS1kZWZpY2llbnQgZ3V0OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE5ODEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWN0aW9uIG9mIGludGVzdGluYWwgVGgxNyBjZWxscyBieSBzZWdtZW50ZWQgZmlsYW1lbnRvdXMgYmFjdGVyaWE7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTM5OyAgY2l0YXRpb25fcGFnZXM9NDg1LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUga2V5IHJvbGUgb2Ygc2VnbWVudGVkIGZpbGFtZW50b3VzIGJhY3RlcmlhIGluIHRoZSBjb29yZGluYXRlZCBtYXR1cmF0aW9uIG9mIGd1dCBoZWxwZXIgVCBjZWxsIHJlc3BvbnNlOyBJbW11bml0eTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTY3Ny04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGVzIGNoYXJhY3Rlcml6ZSBjZWxsdWxhciBwaGVub3R5cGVzIG9mIGZvbGxpY2xlLWFzc29jaWF0ZWQgZXBpdGhlbGl1bSBhbmQgTSBjZWxsczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGltcHJvdmVkIEROQSBpc29sYXRpb24gbWV0aG9kIGZvciBtZXRhZ2Vub21pYyBhbmFseXNpcyBvZiB0aGUgbWljcm9iaWFsIGZsb3JhIG9mIHRoZSBodW1hbiBpbnRlc3RpbmU7IE1pY3JvYi4gRW52aXJvbi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yMTQtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBHZW5vbWUgQW5ub3RhdGlvbiBQaXBlbGluZSAoTWlHQVApIGZvciBkaXZlcnNlIHVzZXJzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5uYXRlIGltbXVuZSByZXNwb25zZSB0byBiYWN0ZXJpYWwgZmxhZ2VsbGluIGlzIG1lZGlhdGVkIGJ5IFRvbGwtbGlrZSByZWNlcHRvciA1OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDEwOyAgY2l0YXRpb25fcGFnZXM9MTA5OS0xMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUi9DYXMsIHRoZSBpbW11bmUgc3lzdGVtIG9mIGJhY3RlcmlhIGFuZCBhcmNoYWVhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1JJU1BSIGludGVyZmVyZW5jZTogUk5BLWRpcmVjdGVkIGFkYXB0aXZlIGltbXVuaXR5IGluIGJhY3RlcmlhIGFuZCBhcmNoYWVhOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbXVsdGlkcnVnLXJlc2lzdGFudCBodW1hbiBwYXRob2dlbiBDbG9zdHJpZGl1bSBkaWZmaWNpbGUgaGFzIGEgaGlnaGx5IG1vYmlsZSwgbW9zYWljIGdlbm9tZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz03NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgMTZTIHJSTkEgc2VxdWVuY2VzIG9mIHNlZ21lbnRlZCBmaWxhbWVudG91cyBiYWN0ZXJpYSBpc29sYXRlZCBmcm9tIG1pY2UsIHJhdHMsIGFuZCBjaGlja2VucyBhbmQgcHJvcG9zYWwgb2Yg4oCYQ2FuZGlkYXR1cyBhcnRocm9taXR1czsgSW50LiBKLiBTeXN0LiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTQ0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIGF1dG9tYXRpYyByZWNvbnN0cnVjdGlvbiBvZiBhIGhpZ2hseSByZXNvbHZlZCB0cmVlIG9mIGxpZmU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzExOyAgY2l0YXRpb25fcGFnZXM9MTI4My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIGFuZCB0cmFuc2NyaXB0b21lcyBvZiB0aGUgYW50aS10dW1vciBhZ2VudCBDbG9zdHJpZGl1bSBub3Z5aS1OVDsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE1NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb3N0cmlkaXVtIGNlbGx1bG9seXRpY3VtOiBtb2RlbCBvcmdhbmlzbSBvZiBtZXNvcGhpbGljIGNlbGx1bG9seXRpYyBjbG9zdHJpZGlhOyBGRU1TIE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTc0MS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBjbGFzcyBvZiBtb2R1bGFyIHRyYW5zcG9ydGVycyBmb3Igdml0YW1pbnMgaW4gcHJva2FyeW90ZXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTkxOyAgY2l0YXRpb25fcGFnZXM9NDItNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwb3J1bGF0aW9uIG9mIEJhY2lsbHVzIHN1YnRpbGlzOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWMgdmlldyBvZiBjbG9zdHJpZGlhbCBzcG9ydWxhdGlvbiBhbmQgcGh5c2lvbG9neTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTk2OS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BvcmUgZ2VybWluYXRpb247IENlbGwgTW9sLiBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz00MDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3lhbm9iYWN0ZXJpYWwgY2VsbCBpbmNsdXNpb25zOyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBnZW5vbWUgc2VxdWVuY2VzIGZvciBnZW5lcyBvZiBjeWFub3BoeWNpbiBtZXRhYm9saXNtOiBpZGVudGlmeWluZyBwdXRhdGl2ZSBjeWFub3BoeWNpbiBtZXRhYm9saXppbmcgcHJva2FyeW90ZXM7IE1hY3JvbW9sLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNzgtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlZ21lbnRlZCBmaWxhbWVudG91cyBiYWN0ZXJpYSBpbnRlcmFjdCB3aXRoIGludHJhZXBpdGhlbGlhbCBtb25vbnVjbGVhciBjZWxsczsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0zMjc3LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc21pc3Npb24gZWxlY3Ryb24gbWljcm9zY29waWMgZGVtb25zdHJhdGlvbiBvZiBwaGFnb2N5dG9zaXMgYW5kIGludHJhY2VsbHVsYXIgcHJvY2Vzc2luZyBvZiBzZWdtZW50ZWQgZmlsYW1lbnRvdXMgYmFjdGVyaWEgYnkgaW50ZXN0aW5hbCBlcGl0aGVsaWFsIGNlbGxzIG9mIHRoZSBjaGljayBpbGV1bTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz02NDk2LTUwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9sbC1saWtlIHJlY2VwdG9yIDUgcmVjb2duaXplcyBhIGNvbnNlcnZlZCBzaXRlIG9uIGZsYWdlbGxpbiByZXF1aXJlZCBmb3IgcHJvdG9maWxhbWVudCBmb3JtYXRpb24gYW5kIGJhY3RlcmlhbCBtb3RpbGl0eTsgTmF0LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTI0Ny01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIHBhdGhvZ2VuaWMgaW50ZXN0aW5hbCBiYWN0ZXJpYSBieSBUb2xsLWxpa2UgcmVjZXB0b3IgNSBvbiBpbnRlc3RpbmFsIENEMTFjKyBsYW1pbmEgcHJvcHJpYSBjZWxsczsgTmF0LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGh1bW9yYWwgYW5kIGNlbGx1bGFyIGd1dCBpbW11bml0eSBieSBsYW1pbmEgcHJvcHJpYSBkZW5kcml0aWMgY2VsbHMgZXhwcmVzc2luZyBUb2xsLWxpa2UgcmVjZXB0b3IgNTsgTmF0LiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NzY5LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbW11bmUgcmVzcG9uc2VzIG9mIFRMUjUrIGxhbWluYSBwcm9wcmlhIGRlbmRyaXRpYyBjZWxscyBpbiBlbnRlcm9iYWN0ZXJpYWwgaW5mZWN0aW9uOyBKLiBHYXN0cm9lbnRlcm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTgwMy0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlzdGVyaWEgbW9ub2N5dG9nZW5lcyBwaG9zcGhvbGlwYXNlIEMtZGVwZW5kZW50IGNhbGNpdW0gc2lnbmFsaW5nIG1vZHVsYXRlcyBiYWN0ZXJpYWwgZW50cnkgaW50byBKNzc0IG1hY3JvcGhhZ2UtbGlrZSBjZWxsczsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0xNzcwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBtb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbiBvZiBTYWxtb25lbGxhIHR5cGhpbXVyaXVtIGdlbmUgaW52b2x2ZWQgaW4gdHJpZ2dlcmluZyB0aGUgaW50ZXJuYWxpemF0aW9uIG9mIHNhbG1vbmVsbGFlIGludG8gY3VsdHVyZWQgZXBpdGhlbGlhbCBjZWxsczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTU5NzYtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOC80LzI5MS8xMzUyNjgzL2RzcjAyMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTnVtZXJvdXMgbWljcm9iZXMgaW5oYWJpdCB0aGUgbWFtbWFsaWFuIGludGVzdGluYWwgdHJhY2sgYW5kIHN0cm9uZ2x5IGltcGFjdCBob3N0IHBoeXNpb2xvZ3k7IGhvd2V2ZXIsIG91ciB1bmRlcnN0YW5kaW5nIG9mIHRoaXMgZWNvc3lzdGVtIHJlbWFpbnMgbGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwMjY4MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQuI0S15J7KaxbOgy6PRYUZ8u2Zzw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f291%2f502681" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f291%2f502681" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19820" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&amp;Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/4">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7753986">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7753986">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7753988">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7753988">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7753997">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7753997">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754014">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754014">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754051">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754051">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754053">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754053">Supplemental data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754055">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754055">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754057">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754057">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7754059">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7754059">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/4/277/502564">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomomi Kuwahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology, Faculty of Medicine</div>, <div class="insititution">Kagawa University</div>, <div class="addr-line">1750-1 Miki-cho, Kita-gun, Kagawa 761-0793</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomomi+Kuwahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuwahara T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuwahara T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomomi Kuwahara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yositoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yositoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yositoshi Ogura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">University of Tokyo</div>, <div class="addr-line">Chiba</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 Nagatsuta, Yokohama, Kanagawa 226-0026</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasuke Ooka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasuke+Ooka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ooka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ooka T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tadasuke Ooka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Plant Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 Nagatsuta, Yokohama, Kanagawa 226-0026</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruyuki Nakayama-Imaohji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Parasitology and Immunology</div>, <div class="insititution">Institute of Health Biosciences, The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-15 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruyuki+Nakayama-Imaohji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama-Imaohji H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama-Imaohji H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Haruyuki Nakayama-Imaohji</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Minoru Ichimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Parasitology and Immunology</div>, <div class="insititution">Institute of Health Biosciences, The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-15 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Minoru+Ichimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ichimura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ichimura M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Minoru Ichimura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kikuji Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Graduate School of Agricultural and Life Sciences</div>, <div class="insititution">University of Tokyo</div>, <div class="addr-line">1-1-1 Yayoi, Bunkyo, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kikuji+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kikuji Itoh</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chieko Ishifune</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Parasitology and Immunology</div>, <div class="insititution">Institute of Health Biosciences, The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-15 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chieko+Ishifune">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishifune C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishifune C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chieko Ishifune</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoichi Maekawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Parasitology and Immunology</div>, <div class="insititution">Institute of Health Biosciences, The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-15 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoichi+Maekawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maekawa Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maekawa Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoichi Maekawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Yasutomo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Parasitology and Immunology</div>, <div class="insititution">Institute of Health Biosciences, The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-15 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Yasutomo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasutomo K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasutomo K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Koji Yasutomo</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">University of Tokyo</div>, <div class="addr-line">Chiba</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 985-85-0871. Fax. +81 985-85-6475. E-mail: <a href="mailto:thayash@miyazaki-u.ac.jp" target="_blank">thayash@miyazaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this study.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 291–303, <a href='https://doi.org/10.1093/dnares/dsr022'>https://doi.org/10.1093/dnares/dsr022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 July 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 May 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 June 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="502681" href="/dnaresearch/article-pdf/18/4/291/1352683/dsr022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Tomomi Kuwahara, Yositoshi Ogura, Kenshiro Oshima, Ken Kurokawa, Tadasuke Ooka, Hideki Hirakawa, Takehiko Itoh, Haruyuki Nakayama-Imaohji, Minoru Ichimura, Kikuji Itoh, Chieko Ishifune, Yoichi Maekawa, Koji Yasutomo, Masahira Hattori, Tetsuya Hayashi;  The Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing, <em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 291–303, <a href='https://doi.org/10.1093/dnares/dsr022'>https://doi.org/10.1093/dnares/dsr022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/502681?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/502681?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20Lifestyle%20of%20the%20Segmented%20Filamentous%20Bacterium%3A%20A%20Non-Culturable%20Gut-Associated%20Immunostimulating%20Microbe%20Inferred%20by%20Whole-Genome%20Sequencing&amp;publicationDate=2011-07-26&amp;volumeNum=18&amp;issueNum=4&amp;author=Kuwahara%2C%20Tomomi%3B%20Ogura%2C%20Yositoshi&amp;startPage=291&amp;endPage=303&amp;contentId=10.1093%2Fdnares%2Fdsr022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f291%2f502681" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7753986 id="7753986" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Numerous microbes inhabit the mammalian intestinal track and strongly impact host physiology; however, our understanding of this ecosystem remains limited owing to the high complexity of the microbial community and the presence of numerous non-culturable microbes. Segmented filamentous bacteria (SFBs), which are clostridia-related Gram-positive bacteria, are among such non-culturable populations and are well known for their unique morphology and tight attachment to intestinal epithelial cells. Recent studies have revealed that SFBs play crucial roles in the post-natal maturation of gut immune function, especially the induction of Th17 lymphocytes. Here, we report the complete genome sequence of mouse SFBs. The genome, which comprises a single circular chromosome of 1 620 005 bp, lacks genes for the biosynthesis of almost all amino acids, vitamins/cofactors and nucleotides, but contains a full set of genes for sporulation/germination and, unexpectedly, for chemotaxis/flagella-based motility. These findings suggest a triphasic lifestyle of the SFB, which comprises two types of vegetative (swimming and epicellular parasitic) phases and a dormant (spore) phase. Furthermore, SFBs encode four types of flagellin, three of which are recognized by Toll-like receptor 5 and could elicit the innate immune response. Our results reveal the non-culturability, lifestyle and immunostimulation mechanisms of SFBs and provide a genetic basis for the future development of the SFB cultivation and gene-manipulation techniques.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;segmented filamentous bacterium&quot;">segmented filamentous bacterium</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-culturable bacteria&quot;">non-culturable bacteria</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="flagella">flagella</a></div>                            <h2 scrollto-destination=7753988 id="7753988" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The mammalian digestive tract harbours diverse microbes, with populations as high as 100 trillion organisms. Their metabolic activities, such as energy harvesting from otherwise indigestible dietary polysaccharides and the synthesis of essential vitamins, and colonization resistance against enteric pathogens are considered beneficial for the host.<sup><a reveal-id="DSR022C1" data-open="DSR022C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR022C2" data-open="DSR022C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The gut microbiota play pivotal roles in the maturation and differentiation of the gut immune system. However, alteration of the gut microbial composition is often linked to various pathological conditions, such as inflammatory bowel diseases, colon cancer and obesity.<sup><a reveal-id="DSR022C3  DSR022C7" data-open="DSR022C3  DSR022C7" class="link link-ref link-reveal xref-bibr">3–7</a></sup></p><p>Our understanding of gut microbiota has been severely hampered by its extreme complexity; however, the recent advent of culture independent techniques that employ 16S rRNA gene sequencing has revealed that mammalian gut microbiota comprise several hundred or more species, and the majority of the constituents are yet to be cultured or characterized.<sup><a reveal-id="DSR022C8  DSR022C11" data-open="DSR022C8  DSR022C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> Moreover, 20–80% of gut microbes are predicted to be non-culturable by current cultivation techniques. Therefore, genomic information of these non-culturable populations is essential to a more thorough understanding of the biology of gut microbiota; however, genome sequencing of non-culturable microbes is still a significant scientific challenge even with metagenome analysis or single cell sequencing using new sequencing technologies.<sup><a reveal-id="DSR022C8  DSR022C12" data-open="DSR022C8  DSR022C12" class="link link-ref link-reveal xref-bibr">8–12</a></sup></p><p>Segmented filamentous bacteria (SFBs), which are clostridia-related, spore-forming, Gram-positive bacteria, are well-known members of these non-culturable populations. SFBs inhabit the terminal ilea of a wide range of mammals.<sup><a reveal-id="DSR022C13" data-open="DSR022C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR022C14" data-open="DSR022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Their characteristic prevalence at weaning periods and unique morphology of being tightly attached to the epithelial cells around Peyer's patches strongly suggest their involvement in the maturation of the gut immune system.<sup><a reveal-id="DSR022C15" data-open="DSR022C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSR022C16" data-open="DSR022C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In fact, it was found that SFB colonization enhances the luminal IgA production.<sup><a reveal-id="DSR022C17" data-open="DSR022C17" class="link link-ref link-reveal xref-bibr">17</a></sup> More recently, several groups have shown that SFBs play a crucial role in the post-natal maturation of gut immune function, especially in the induction of a specific subset of proinflammatory T helper lymphocytes, Th17 cells.<sup><a reveal-id="DSR022C18" data-open="DSR022C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR022C19" data-open="DSR022C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Thus, more attention is being paid to SFB-derived molecules and the signal transduction pathways responsible for such immunomodulatory effects; however, no technique is available to cultivate and genetically manipulate the SFB <em>in vitro</em>.</p><p>Here, we report the genome sequence determination of mouse SFBs, the first complete genome of a non-culturable mammalian gut microbe, using SFB cells grown in germ-free mice. An analysis of the genome sequence revealed the genomic features underlying the non-culturability of SFBs and their unique triphasic lifestyle, which would help us to establish an SFB cultivation protocol. In addition, our results are the first to provide insight into the mechanisms of the immunostimulating activities of SFBs and some of the genomic information relevant for human-associated SFBs.</p>                            <h2 scrollto-destination=7753997 id="7753997" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7754000 id="7754000" class="section-title" data-legacy-id=s2a>2.1. Preparation of the genomic DNA</h3>
<p>SFB-gnotobiotic mice were prepared according to the method described by Umesaki <em>et al</em>.<sup><a reveal-id="DSR022C15" data-open="DSR022C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Briefly, the terminal ileum of an 8-week-old male BALB/c mouse (Clea Japan) was cut into 1-cm pieces. Mucosal epithelial cells were separated from the lamina propria (LP) as described previously.<sup><a reveal-id="DSR022C20" data-open="DSR022C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The epithelial cells were transferred to a fresh tube containing 1 ml of phosphate-buffered saline (PBS) and homogenized using a Teflon grinder. The homogenate was treated with 3% chloroform to lyse the host epithelial cells and non-spore-forming bacteria. After removing the chloroform by bubbling with carbon dioxide, 5-week-old male germ-free BALB/c mice (Clea Japan) that had been kept in vinyl isolators were orally inoculated with the sample. At 2 weeks after inoculation, caecal contents were collected and bacterial DNA was extracted as described previously.<sup><a reveal-id="DSR022C21" data-open="DSR022C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                            <h3 scrollto-destination=7754003 id="7754003" class="section-title" data-legacy-id=s2b>2.2. Preparation of mouse LP lymphocytes</h3>
<p>The small intestines were opened longitudinally, washed with PBS and incubated for 20 min at 37°C in a FACS buffer [PBS containing 10% fetal bovine serum (FBS), 10 mM EDTA, 20 mM HEPES, 1 mM sodium pyruvate, 10 µg/ml polymyxin B sulphate, 100 units/ml penicillin and 100 µg/ml streptomycin]. The intestines were then cut into small pieces and incubated with RPMI1640 containing 10% FBS, 400 units/ml collagenase D and 100 µg/ml DNase I (all from Roche Diagnostics) for 1 h at 37°C. The digested tissues were washed with the FACS buffer, resuspended in 10 ml of 40% Percoll (GE Healthcare) and overlaid on 5 ml of 80% Percoll to perform Percoll gradient separation. Interface cells [LP lymphocytes (LPLs)] were collected and suspended in PBS containing 2% FBS and 0.05% NaN<sub>3</sub>. For IFN-γ and interleukin (IL)-17A detection, the LPLs were stimulated for 4 h with 50 ng/ml of phorbol 12-myristate 13-acetate, 1 µg/ml of ionomycin and 2 µM monensin (all from Sigma). Cells were first stained for surface CD4 and then for intracellular IFN-γ and IL-17A after fixation with a 4% paraformaldehyde phosphate buffer solution and washing with a 0.1% saponin-containing buffer. Antibodies used in this part of the study included biotinylated anti-CD4 monoclonal antibody (mAb), FITC-labelled anti-IFN-γ mAb, PE-labelled anti-IL-17A mAb and streptavidin-PE-Cy7 (all from eBioscience). Flow cytometry analysis was performed using a FACSCanto II with the FACSDiva software (BD Biosciences).</p>                            <h3 scrollto-destination=7754005 id="7754005" class="section-title" data-legacy-id=s2c>2.3. Genome sequencing, annotation and comparative genomics</h3>
<p>The genome sequence of SFBs was determined using a combined strategy of 454 pyrosequencing (GS FLX; Roche) and Sanger sequencing. A total of 234 915 GS FLX reads were assembled with the GS Assembler software into 242 large contigs (≥500 bp). These contigs were then reassembled using the Phred/Phrap/Consed software (<a class="link link-uri" href="http://www.phrap.com">http://www.phrap.com</a>) with 25 257 Sanger sequencing reads from plasmid libraries (3 and 10 kb inserts). Gap closing and the re-sequencing of low-quality regions were performed by sequencing the PCR products and the appropriate plasmid clones. The genome sequence was automatically annotated with the Microbial Genome Annotation Pipeline<sup><a reveal-id="DSR022C22" data-open="DSR022C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and manually curated using the IMC-GE software (In Silico Biology, Inc.). The genome sequence of SFBs has been deposited in the DDBJ/EMBL/GenBank databases under accession number AP012202.</p><p>The SFB gene contents were compared with those of <em>Clostridium difficile</em> strain 630 (CD) and <em>C. novyi</em> strain NT (CN). The orthologous protein coding sequences (CDSs or CDS groups) for CD and CN were taken from the MBGD (<a class="link link-uri" href="http://mbgd.genome.ad.jp/">http://mbgd.genome.ad.jp/</a>). On the basis of the results of a reciprocal BLASTP search between SFB and CD and between SFB and CN (threshold; ≥25% sequence identity and ≥60% aligned length coverage of a query sequence), we categorized the CDSs (or CDS groups) from the three genomes into groups of ‘conserved in all’, ‘conserved in SFB and CD but not in CN’, ‘conserved in SFB and CN but not in CD’, ‘conserved in CD and CN but not in SFB’ and those unique to each species.</p>                            <h3 scrollto-destination=7754008 id="7754008" class="section-title" data-legacy-id=s2d>2.4. Analysis of Toll-like receptor 5-linked NF-κB signaling pathway activation</h3>
<p>Four flagellin genes were PCR-amplified and cloned into pGEX-6P-1 (GE Healthcare) to generate glutathione-S-transferase (GST)-flagellin fusion genes. GST-fusion proteins were expressed in <em>Escherichia coli</em> BL21 via incubation with 1 mM IPTG at 20°C overnight and purified with Sepharose 4B GST resin (GE Healthcare). The Toll-like receptor 5 (TLR5)-stimulating activities of SFB flagellins were determined by a luciferase reporter assay system as described previously.<sup><a reveal-id="DSR022C23" data-open="DSR022C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Briefly, the 293T cells were transfected with pNF-κB-Luc (Clontech) and pRL-TK (Promega) in addition to the mouse TLR5-encoding pGA vector or empty vector. pNF-κB-Luc contains the firefly luciferase gene, expression of which is controlled by the upstream NF-κB response elements, and pRL-TK contains the <em>Renilla</em> luciferase gene, which is control by the promoter of the thymidine kinase. At 20 h after transfection, the cells were stimulated with 1 µg/ml of recombinant <em>Salmonella</em> flagellin (IMGENEX Corp.), GST or GST-fusion SFB flagellins for 24 h. After stimulation, the cells were harvested and the luciferase activities in the cell lysates were measured by the Dual Luciferase Reporter Assay System (Promega) using a Lumat LB9507 (Berthold). Luciferase activity was expressed as a ratio of the NF-κB-dependent firefly luciferase activity divided by the control <em>Renilla</em> luciferase activity (relative luciferase units).</p>                            <h3 scrollto-destination=7754010 id="7754010" class="section-title" data-legacy-id=s2e>2.5. The search for human SFB-derived sequences in human gut metagenome sequences</h3>
<p>From the Illumina sequence reads produced in the MetaHIT project,<sup><a reveal-id="DSR022C11" data-open="DSR022C11" class="link link-ref link-reveal xref-bibr">11</a></sup> we removed reads containing one or more Ns. The remaining 7 345 361 234 reads were trimmed for low-quality sequences and used as queries in a BLAST search against the SFB genome sequence, 1334 RefSeq complete microbial genome sequences (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/RefSeq/">http://www.ncbi.nlm.nih.gov/RefSeq/</a>) and 16 276 contig sequences produced by the Human Microbiome Project (<a class="link link-uri" href="http://www.hmpdacc.org/">http://www.hmpdacc.org/</a>). If a query sequence returned a top hit to the mouse SFB genome with a ≥95% sequence identity for ≥95% of the query sequence and the difference in the BLAST bit score between the top (to SFB) and second hits was larger than zero, we defined it as a human SFB-related sequence. In the 277 reads we identified, BLASTN bit score differences between the top and second hits ranged from 2.0 to 121.0 (hits for only SFB) and the median value was 11.9, indicating that the 277 sequences showed a significantly higher sequence identity to SFBs than to other known genomes.</p>                            <h3 scrollto-destination=7754012 id="7754012" class="section-title" data-legacy-id=s2f>2.6. Ethics</h3>
<p>All animal experiments were performed at the University of Tokushima according to the university's guidelines for animal experiments. The experimental designs were approved by the university animal committee.</p>                            <h2 scrollto-destination=7754014 id="7754014" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7754015 id="7754015" class="section-title" data-legacy-id=s3a>3.1. Genome sequencing</h3>
<p>To obtain genomic DNA for sequencing, we generated SFB-associated gnotobiotic mice by orally inoculating SFBs into germ-free BALB/c mice. The SFB cells used for inoculation (mostly in the endospore form) were obtained from the ileal mucosa of an 8-week-old male BALB/c mouse. Monoassociation was confirmed by Gram-staining and sequencing of the 16S rRNA gene. For the latter, a chromatography comprising single peaks was detected. The SFB cells in the gnotobiotic mice had a segmented filamentous morphology and were tightly attached to the gut epithelia with a deep indentation of the epithelial cell surface where the SFB was attached (Fig. <a reveal-id="DSR022F1" data-open="DSR022F1" class="link link-reveal link-table xref-fig">1</a>A). We further confirmed that the colonization of SFBs in the germ-free mouse intestine induced Th17 and Th1 subsets in the gut LP CD4<sup>+</sup> lymphocyte preparation as reported previously<sup><a reveal-id="DSR022C18" data-open="DSR022C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR022C19" data-open="DSR022C19" class="link link-ref link-reveal xref-bibr">19</a></sup>; the IL-17-producing Th17 population increased from 0.6% (germ-free mouse) to 8.4% (SFB-gnotobiotic mouse) and the IFN-γ-producing Th1 population from 20.5 to 37.0% (Fig. <a reveal-id="DSR022F1" data-open="DSR022F1" class="link link-reveal link-table xref-fig">1</a>B). </p>                        <a id="7754017" scrollto-destination="7754017"></a>
<div id="DSR022F1" data-id="DSR022F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02201.jpeg?Expires=2147483647&Signature=dIkksq1acsKFqw-h48sSvE1MimdioPrKluj1sn7ElikYowxKW~mKrbNgJBs3rfinlIAgX4a5fVPfwpbtXLb3FUCaMHgXIsZWTnAKK0VFdbw2FrSXN3mwLfGubNmygz9L92rOQuG5NOZfffuwOy8Hhh2zdZGJ-PnjQyf88M-0rGU94m-0QYajbsxAYASB0eyVek3Qeu3L~ccFEi6YoAfcLudlLw814uRAG-hfKoQPKZfBJKILe29nu1ecZ6dW0yqTDVOmM1T-Bv8C1zDg1RDgR4mAWnQwjKIilfx~yEx8sqdDraZZqQEu0ieokgglIppgmkKpVTt5mzc4TFX3hzDwNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Morphology and immunostimulatory activity of SFBs grown in SFB-monoassociated gnotobiotic mice. (A) Electron microscopic examination of SFBs. BALB/c germ-free mice were colonized with the SFB, and the terminal ilea collected 14 days after inoculation were examined by scanning electron microscopy. SFBs show a characteristic long filamentous morphology and preferentially colonized around Peyer's patches. Epithelial cell surfaces to which SFB attach are deeply indented. (B) Induction of Th1 and Th17 cells by SFB colonization. Intestinal LPLs were isolated from BALB/c germ-free mice or SFB-monoassociated gnotobiotic mice and the percentages of IFN-γ+ or IL-17+ cells within the CD4+ lymphocyte population in each LPL preparation were determined by FACS analysis." path-from-xml="dsr02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754017/dsr02201.jpeg" path-from-xml="dsr02201.jpeg" target="_blank">View large</a><a section="7754017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02201.jpeg?Expires=2147483647&Signature=jfUezHFrMdcFwIfF8N1Fg9c~x3K9GqKlRI1LFWZg5cc6~-P4MqJ~m~tE5iEGUX3cWUX-XfEpKwuSFemjlaYwpyNKAUecmKCnZVSwdAXN96XHlyAZ4SwyHWuSALKZ2HPtr2TsipiD1DU7R9g47UW3zRDIUCH82fnj2S-Bj2wUNxNwcp~ojIh~VfrIqWVHwrtZqk9crwHwKQPNg-vrQXGOgrfUBFGNyGEbZVRM~syB0iHj00I8ArM0Fl3WFAm0vKivXHA4rKUYzhTfH-Gqj4RSSxhQ6N~~a0klwPOxcrOcxoCphBbm5EKiQ0a6i1gspSJ5c5~QXFv6Lj33x1AfO67OHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754017&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Morphology and immunostimulatory activity of SFBs grown in SFB-monoassociated gnotobiotic mice. (A) Electron microscopic examination of SFBs. BALB/c germ-free mice were colonized with the SFB, and the terminal ilea collected 14 days after inoculation were examined by scanning electron microscopy. SFBs show a characteristic long filamentous morphology and preferentially colonized around Peyer's patches. Epithelial cell surfaces to which SFB attach are deeply indented. (B) Induction of Th1 and Th17 cells by SFB colonization. Intestinal LPLs were isolated from BALB/c germ-free mice or SFB-monoassociated gnotobiotic mice and the percentages of IFN-γ<sup>+</sup> or IL-17<sup>+</sup> cells within the CD4<sup>+</sup> lymphocyte population in each LPL preparation were determined by FACS analysis.</p></div></div><div content-id="DSR022F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02201.jpeg?Expires=2147483647&Signature=dIkksq1acsKFqw-h48sSvE1MimdioPrKluj1sn7ElikYowxKW~mKrbNgJBs3rfinlIAgX4a5fVPfwpbtXLb3FUCaMHgXIsZWTnAKK0VFdbw2FrSXN3mwLfGubNmygz9L92rOQuG5NOZfffuwOy8Hhh2zdZGJ-PnjQyf88M-0rGU94m-0QYajbsxAYASB0eyVek3Qeu3L~ccFEi6YoAfcLudlLw814uRAG-hfKoQPKZfBJKILe29nu1ecZ6dW0yqTDVOmM1T-Bv8C1zDg1RDgR4mAWnQwjKIilfx~yEx8sqdDraZZqQEu0ieokgglIppgmkKpVTt5mzc4TFX3hzDwNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Morphology and immunostimulatory activity of SFBs grown in SFB-monoassociated gnotobiotic mice. (A) Electron microscopic examination of SFBs. BALB/c germ-free mice were colonized with the SFB, and the terminal ilea collected 14 days after inoculation were examined by scanning electron microscopy. SFBs show a characteristic long filamentous morphology and preferentially colonized around Peyer's patches. Epithelial cell surfaces to which SFB attach are deeply indented. (B) Induction of Th1 and Th17 cells by SFB colonization. Intestinal LPLs were isolated from BALB/c germ-free mice or SFB-monoassociated gnotobiotic mice and the percentages of IFN-γ+ or IL-17+ cells within the CD4+ lymphocyte population in each LPL preparation were determined by FACS analysis." path-from-xml="dsr02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754017/dsr02201.jpeg" path-from-xml="dsr02201.jpeg" target="_blank">View large</a><a section="7754017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02201.jpeg?Expires=2147483647&Signature=jfUezHFrMdcFwIfF8N1Fg9c~x3K9GqKlRI1LFWZg5cc6~-P4MqJ~m~tE5iEGUX3cWUX-XfEpKwuSFemjlaYwpyNKAUecmKCnZVSwdAXN96XHlyAZ4SwyHWuSALKZ2HPtr2TsipiD1DU7R9g47UW3zRDIUCH82fnj2S-Bj2wUNxNwcp~ojIh~VfrIqWVHwrtZqk9crwHwKQPNg-vrQXGOgrfUBFGNyGEbZVRM~syB0iHj00I8ArM0Fl3WFAm0vKivXHA4rKUYzhTfH-Gqj4RSSxhQ6N~~a0klwPOxcrOcxoCphBbm5EKiQ0a6i1gspSJ5c5~QXFv6Lj33x1AfO67OHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754017&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Morphology and immunostimulatory activity of SFBs grown in SFB-monoassociated gnotobiotic mice. (A) Electron microscopic examination of SFBs. BALB/c germ-free mice were colonized with the SFB, and the terminal ilea collected 14 days after inoculation were examined by scanning electron microscopy. SFBs show a characteristic long filamentous morphology and preferentially colonized around Peyer's patches. Epithelial cell surfaces to which SFB attach are deeply indented. (B) Induction of Th1 and Th17 cells by SFB colonization. Intestinal LPLs were isolated from BALB/c germ-free mice or SFB-monoassociated gnotobiotic mice and the percentages of IFN-γ<sup>+</sup> or IL-17<sup>+</sup> cells within the CD4<sup>+</sup> lymphocyte population in each LPL preparation were determined by FACS analysis.</p></div></div><p>The genomic DNA of SFBs was extracted from the caecal contents of SFB-gnotobiotic mice and used for genome sequencing via a combination of 454 pyrosequencing and Sanger sequencing. Because it is currently impossible to generate SFB-gnotobiotic mice from a single SFB cell, the DNA sample contained DNA molecules derived from multiple SFB clones. In fact, we observed many single-nucleotide polymorphisms and small indels. Therefore, when sequence polymorphisms were observed, we regarded the most frequently appearing sequence as representative of the SFB genome sequence.</p>                            <h3 scrollto-destination=7754019 id="7754019" class="section-title" data-legacy-id=s3b>3.2. General genome features</h3>
<p>The genome of SFBs comprises a single circular chromosome of 1 620 005 bp with an average 28.8% GC content (Table <a reveal-id="DSR022TB1" data-open="DSR022TB1" class="link link-reveal link-table xref-fig">1</a>). The chromosome encodes 1491 CDSs, 6 rRNA (<em>rrn</em>) operons, 37 tRNA genes and only 24 pseudogenes. Of the 1491 CDSs, 994 (66.7%) were functionally assigned. Like other low GC content Gram-positive bacteria, the SFB chromosome exhibits clear GC skew transitions at the replication origin and terminus and a strong coding bias with 82.4% of the CDSs being encoded on the leading strand (Fig. <a reveal-id="DSR022F2" data-open="DSR022F2" class="link link-reveal link-table xref-fig">2</a>). Chromosome regions showing GC bias anomalies correspond to <em>rrn</em> or prophage regions. </p>                        <a id="7754021" scrollto-destination="7754021"></a>
<div content-id="DSR022TB1" class="table-modal"><div class="table-wrap"><div id="DSR022TB1" class="table-wrap-title" data-id="DSR022TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of the SFB chromosome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Size (bp) </td><td>1 620 005 </td></tr><tr><td>G + C content (%) </td><td>28.3 </td></tr><tr><td>CDSs </td><td>1491<sup>a</sup> </td></tr><tr><td>Coding density </td><td>89.8 </td></tr><tr><td>Average CDS length (bp) </td><td>960 </td></tr><tr><td>rRNA operons </td><td>6 </td></tr><tr><td>tRNA genes </td><td>37 </td></tr><tr><td>Prophages </td><td>4 </td></tr></tbody></table></div><div content-id="DSR022TB1" class="table-modal"><table><tbody><tr><td>Size (bp) </td><td>1 620 005 </td></tr><tr><td>G + C content (%) </td><td>28.3 </td></tr><tr><td>CDSs </td><td>1491<sup>a</sup> </td></tr><tr><td>Coding density </td><td>89.8 </td></tr><tr><td>Average CDS length (bp) </td><td>960 </td></tr><tr><td>rRNA operons </td><td>6 </td></tr><tr><td>tRNA genes </td><td>37 </td></tr><tr><td>Prophages </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Twenty-four pseudogenes are not included.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7754021" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR022TB1" class="table-wrap-title" data-id="DSR022TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of the SFB chromosome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Size (bp) </td><td>1 620 005 </td></tr><tr><td>G + C content (%) </td><td>28.3 </td></tr><tr><td>CDSs </td><td>1491<sup>a</sup> </td></tr><tr><td>Coding density </td><td>89.8 </td></tr><tr><td>Average CDS length (bp) </td><td>960 </td></tr><tr><td>rRNA operons </td><td>6 </td></tr><tr><td>tRNA genes </td><td>37 </td></tr><tr><td>Prophages </td><td>4 </td></tr></tbody></table></div><div content-id="DSR022TB1" class="table-modal"><table><tbody><tr><td>Size (bp) </td><td>1 620 005 </td></tr><tr><td>G + C content (%) </td><td>28.3 </td></tr><tr><td>CDSs </td><td>1491<sup>a</sup> </td></tr><tr><td>Coding density </td><td>89.8 </td></tr><tr><td>Average CDS length (bp) </td><td>960 </td></tr><tr><td>rRNA operons </td><td>6 </td></tr><tr><td>tRNA genes </td><td>37 </td></tr><tr><td>Prophages </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Twenty-four pseudogenes are not included.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7754021" target="_blank">View Large</a></div></div>                        <a id="7754022" scrollto-destination="7754022"></a>
<div id="DSR022F2" data-id="DSR022F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02202.jpeg?Expires=2147483647&Signature=VnIPB-tYDouhsiYdm4nHTU-Ac3kLd3uUeP7~A-zLhnHokH61QPvqO69qWrpmlfdp4TjBQMMr1CXw-B0k4yWBpeamu~pm-FTseDx0whuUKymHOP2CbdgVRxgtXT08BVgq~htXwf8OwNTg-fHPk2Vzel1AArGwIO-2iJKP1qtZt21IcnN-rPTxOjxZdHBsqvMxJWVWFUZXzvNs5wvkb3DRTvm9aK3~7rK1TtCm32JWxtnED4fzRs-R5ATBNqzVRJ7bOh2NodjoBswF6mrnnj9FzSA3A-Y5lxQtwaN-aVlgJVobcdGRjDQXjbld1ME94V8ktsg~3opsFwsPMuVS-0leUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular map of the SFB chromosome. From the outside in: the first circle depicts the nucleotide sequence positions (in bp). The second and third depict CDSs that are transcribed clockwise and counterclockwise, respectively, and CDSs were coloured according to the results of the BLASTP homology search of each CDS (E-value &lt;1e−5) against the RefSeq database at the NCBI (blue, top hit to order Clostridiales; light blue, top hit to other species; red, no hit). The fourth and fifth depict CDSs conserved in C. difficile and C. novyi, respectively (≥25% sequence identity and ≥60% aligned length coverage of a query sequence). The sixth depicts prophages, the seventh GC content, the eighth GC skew, the ninth rRNA operons and the tenth tRNA genes." path-from-xml="dsr02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754022/dsr02202.jpeg" path-from-xml="dsr02202.jpeg" target="_blank">View large</a><a section="7754022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02202.jpeg?Expires=2147483647&Signature=gVBT8jbIB8TymyP~cHvJzT6w3GUUQRvJClDr6JpaOVI~WsPs2QPgLSbAZ85LlcdrdxNNtpepz5hz534mXXAatx0vblNvmbwuKvuO4F4tVjKCJhiqAXNaULeazydKH-2vdPBlmQKevWcyvFcwsftu4ZWXdJXocbz81dXGwcP65~E0K3vrvM7jpD65s-SBsjgYkG~bwiRUlb3ahnMRxZiRabQK2fBq8LNcH9MS5Ke5l-mFizfUGM6xniWM18MLCq1~IDmuIbJTN5Yc-drGODt0l7LLjLaotYB6VeucMzWE~TO0cJENE147j~rlMJDXXRTY4VxS~LFaAAjH2bukQaTzlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754022&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular map of the SFB chromosome. From the outside in: the first circle depicts the nucleotide sequence positions (in bp). The second and third depict CDSs that are transcribed clockwise and counterclockwise, respectively, and CDSs were coloured according to the results of the BLASTP homology search of each CDS (<em>E</em>-value &lt;1e−5) against the RefSeq database at the NCBI (blue, top hit to order Clostridiales; light blue, top hit to other species; red, no hit). The fourth and fifth depict CDSs conserved in <em>C. difficile</em> and <em>C. novyi</em>, respectively (≥25% sequence identity and ≥60% aligned length coverage of a query sequence). The sixth depicts prophages, the seventh GC content, the eighth GC skew, the ninth rRNA operons and the tenth tRNA genes.</p></div></div><div content-id="DSR022F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02202.jpeg?Expires=2147483647&Signature=VnIPB-tYDouhsiYdm4nHTU-Ac3kLd3uUeP7~A-zLhnHokH61QPvqO69qWrpmlfdp4TjBQMMr1CXw-B0k4yWBpeamu~pm-FTseDx0whuUKymHOP2CbdgVRxgtXT08BVgq~htXwf8OwNTg-fHPk2Vzel1AArGwIO-2iJKP1qtZt21IcnN-rPTxOjxZdHBsqvMxJWVWFUZXzvNs5wvkb3DRTvm9aK3~7rK1TtCm32JWxtnED4fzRs-R5ATBNqzVRJ7bOh2NodjoBswF6mrnnj9FzSA3A-Y5lxQtwaN-aVlgJVobcdGRjDQXjbld1ME94V8ktsg~3opsFwsPMuVS-0leUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular map of the SFB chromosome. From the outside in: the first circle depicts the nucleotide sequence positions (in bp). The second and third depict CDSs that are transcribed clockwise and counterclockwise, respectively, and CDSs were coloured according to the results of the BLASTP homology search of each CDS (E-value &lt;1e−5) against the RefSeq database at the NCBI (blue, top hit to order Clostridiales; light blue, top hit to other species; red, no hit). The fourth and fifth depict CDSs conserved in C. difficile and C. novyi, respectively (≥25% sequence identity and ≥60% aligned length coverage of a query sequence). The sixth depicts prophages, the seventh GC content, the eighth GC skew, the ninth rRNA operons and the tenth tRNA genes." path-from-xml="dsr02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754022/dsr02202.jpeg" path-from-xml="dsr02202.jpeg" target="_blank">View large</a><a section="7754022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02202.jpeg?Expires=2147483647&Signature=gVBT8jbIB8TymyP~cHvJzT6w3GUUQRvJClDr6JpaOVI~WsPs2QPgLSbAZ85LlcdrdxNNtpepz5hz534mXXAatx0vblNvmbwuKvuO4F4tVjKCJhiqAXNaULeazydKH-2vdPBlmQKevWcyvFcwsftu4ZWXdJXocbz81dXGwcP65~E0K3vrvM7jpD65s-SBsjgYkG~bwiRUlb3ahnMRxZiRabQK2fBq8LNcH9MS5Ke5l-mFizfUGM6xniWM18MLCq1~IDmuIbJTN5Yc-drGODt0l7LLjLaotYB6VeucMzWE~TO0cJENE147j~rlMJDXXRTY4VxS~LFaAAjH2bukQaTzlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754022&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular map of the SFB chromosome. From the outside in: the first circle depicts the nucleotide sequence positions (in bp). The second and third depict CDSs that are transcribed clockwise and counterclockwise, respectively, and CDSs were coloured according to the results of the BLASTP homology search of each CDS (<em>E</em>-value &lt;1e−5) against the RefSeq database at the NCBI (blue, top hit to order Clostridiales; light blue, top hit to other species; red, no hit). The fourth and fifth depict CDSs conserved in <em>C. difficile</em> and <em>C. novyi</em>, respectively (≥25% sequence identity and ≥60% aligned length coverage of a query sequence). The sixth depicts prophages, the seventh GC content, the eighth GC skew, the ninth rRNA operons and the tenth tRNA genes.</p></div></div><p>The SFB contains no IS elements but four prophages (Fig. <a reveal-id="DSR022F2" data-open="DSR022F2" class="link link-reveal link-table xref-fig">2</a>). Among the four prophages, one (SFBMP01) is apparently intact, whereas the others (SFBMP02∼SFBMP04) are highly degraded phage remnants and SFBMP02 is similar to a part of SFBMP01 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The presence of multiple prophages suggests that phages are the main driving force for horizontal gene transfer between SFBs and other microbes. Frequent exposure to invading foreign DNAs is also suggested by the presence of three loci of clustered, regularly interspaced and short palindromic repeats (CRISPRs) in the SFB genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). CRISPRs are involved in a recently discovered microbial adaptive immunity system that protects microbial cells from invading foreign genetic elements, such as bacteriophages and conjugative plasmids.<sup><a reveal-id="DSR022C24" data-open="DSR022C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR022C25" data-open="DSR022C25" class="link link-ref link-reveal xref-bibr">25</a></sup> CRISPR loci typically consist of several non-contiguous direct repeats separated by stretches of variable sequences called spacers, which correspond to segments of captured phage and plasmid sequences. The CRISPR sequences thus provide an adaptive, heritable record of past infections and express small RNAs that target invading nucleic acids. One of the CRISPR loci in the SFB genome (CRISPR3) contains 13 spacer-repeat units, which are preceded by a gene cluster encoding six CRISPR-associated (Cas) proteins. Interestingly, the other two loci (each contains three and four spacer-repeat units, respectively) are located on prophage SFBMP01. Neither is associated with <em>cas</em> genes, but they share the same 160-bp leader sequence and 32-bp repeat sequences with CRISPR3, suggesting that the precursor RNAs derived from these prophage-encoded CRISPRs may also be matured to short CRISPR RNAs by the CRISPR3-associated Cas proteins. Prophage-encoded CRISPRs were previously described in <em>C. difficille</em>.<sup><a reveal-id="DSR022C26" data-open="DSR022C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Any spacer sequences found in the three CRISPR loci of SFBs showed no significant homology to the sequences in databases.</p>                            <h3 scrollto-destination=7754024 id="7754024" class="section-title" data-legacy-id=s3c>3.3. The phylogenetic position of SFBs</h3>
<p>In our initial BLAST analysis against the RefSeq database, 66.2% of the SFB CDSs had best-hit homologues of the order Clostridiales but with relatively low levels of sequence identity (Fig. <a reveal-id="DSR022F2" data-open="DSR022F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These results are consistent with the phylogenetic position of SFBs previously inferred by 16S rRNA sequence analysis; SFBs belong to a distinct lineage within the <em>Clostridium</em> subphylum at the rank of genus, whose proposed genus name was <em>Candidatus Arthromitus</em> (Fig. <a reveal-id="DSR022F3" data-open="DSR022F3" class="link link-reveal link-table xref-fig">3</a>A).<sup><a reveal-id="DSR022C27" data-open="DSR022C27" class="link link-ref link-reveal xref-bibr">27</a></sup> We further confirmed this phylogenetic position by constructing a genomic phylogenetic tree based on the sequences of 31 universal genes<sup><a reveal-id="DSR022C28" data-open="DSR022C28" class="link link-ref link-reveal xref-bibr">28</a></sup> from fully sequenced species belonging to the order Clostridiales (Fig. <a reveal-id="DSR022F3" data-open="DSR022F3" class="link link-reveal link-table xref-fig">3</a>B). </p>                        <a id="7754026" scrollto-destination="7754026"></a>
<div id="DSR022F3" data-id="DSR022F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02203.jpeg?Expires=2147483647&Signature=3uhAOX2crMwreKiSj2dvWJDhA2vmO1TE7SVumZiqazcdSnL7g9KnpX6BS-B6N8oA~1t6UnCRi58dhIr1ZLxvdodg18Z66WFqAUjkAQ87pab1Uu52LsLKxdc7VbEGQw1i~9n-~NfMqfxcVq1G64HSlh7JzFwlUZOk-vEMfpY26zBx2GPkmSfqzTbOYHeMdQLwqrRrNeME~xMMSEdp3cKZxFPooNOrLuE-2Wfkjq87iY-niIBmlVXNgnvbabXT1TA5FoSMLIk5Kxm9v1TAOgDGEbHgX4pBEDWy4ZfttlgFUrtEe-kWvmA1A3JhYLq~DKsLWMqPqEKvBXqYJS0mJFmJtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The phylogenetic position of SFBs among the fully sequenced Clostridium species. Phylogenetic trees constructed based on the 16S rDNA sequences (A) and the concatenated sequences of 31 universal genes28 (B) are shown." path-from-xml="dsr02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754026/dsr02203.jpeg" path-from-xml="dsr02203.jpeg" target="_blank">View large</a><a section="7754026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02203.jpeg?Expires=2147483647&Signature=odwBQUFITzuzn2E4fUMyJDB8C7~rBMCik2Kglyez7LojsiVNC6iBcFQ8yaTSqJTLnIB7A0Qkv~DkJaqaWxmCa9k~~CvM0kRcFjQUZRr0GRWcwWXdmKaf9CS0~BaKvjU3v8z-4i8iYKrxbyrOzAV6VZN3pngSnr60wfwKwBWlVDLC3NAl4X-wrG7KL17bErotd0m9k3J--wCOaLUUbzI8fxeWUviEv0MnLisJapcjH~F4NQAyfFBGDgYgRGacpiiZNCvIt-URBa3gmWgq1hd-1-zkjT6hzFxtA2pGLBEXsvV7X5FX7WLtiMfrSWYUvd~sNO3BGE0j9jhiB2Gj7qZVJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754026&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The phylogenetic position of SFBs among the fully sequenced <em>Clostridium</em> species. Phylogenetic trees constructed based on the 16S rDNA sequences (A) and the concatenated sequences of 31 universal genes<sup><a reveal-id="DSR022C28" data-open="DSR022C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (B) are shown.</p></div></div><div content-id="DSR022F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02203.jpeg?Expires=2147483647&Signature=3uhAOX2crMwreKiSj2dvWJDhA2vmO1TE7SVumZiqazcdSnL7g9KnpX6BS-B6N8oA~1t6UnCRi58dhIr1ZLxvdodg18Z66WFqAUjkAQ87pab1Uu52LsLKxdc7VbEGQw1i~9n-~NfMqfxcVq1G64HSlh7JzFwlUZOk-vEMfpY26zBx2GPkmSfqzTbOYHeMdQLwqrRrNeME~xMMSEdp3cKZxFPooNOrLuE-2Wfkjq87iY-niIBmlVXNgnvbabXT1TA5FoSMLIk5Kxm9v1TAOgDGEbHgX4pBEDWy4ZfttlgFUrtEe-kWvmA1A3JhYLq~DKsLWMqPqEKvBXqYJS0mJFmJtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The phylogenetic position of SFBs among the fully sequenced Clostridium species. Phylogenetic trees constructed based on the 16S rDNA sequences (A) and the concatenated sequences of 31 universal genes28 (B) are shown." path-from-xml="dsr02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754026/dsr02203.jpeg" path-from-xml="dsr02203.jpeg" target="_blank">View large</a><a section="7754026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02203.jpeg?Expires=2147483647&Signature=odwBQUFITzuzn2E4fUMyJDB8C7~rBMCik2Kglyez7LojsiVNC6iBcFQ8yaTSqJTLnIB7A0Qkv~DkJaqaWxmCa9k~~CvM0kRcFjQUZRr0GRWcwWXdmKaf9CS0~BaKvjU3v8z-4i8iYKrxbyrOzAV6VZN3pngSnr60wfwKwBWlVDLC3NAl4X-wrG7KL17bErotd0m9k3J--wCOaLUUbzI8fxeWUviEv0MnLisJapcjH~F4NQAyfFBGDgYgRGacpiiZNCvIt-URBa3gmWgq1hd-1-zkjT6hzFxtA2pGLBEXsvV7X5FX7WLtiMfrSWYUvd~sNO3BGE0j9jhiB2Gj7qZVJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754026&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The phylogenetic position of SFBs among the fully sequenced <em>Clostridium</em> species. Phylogenetic trees constructed based on the 16S rDNA sequences (A) and the concatenated sequences of 31 universal genes<sup><a reveal-id="DSR022C28" data-open="DSR022C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (B) are shown.</p></div></div>                            <h3 scrollto-destination=7754027 id="7754027" class="section-title" data-legacy-id=s3d>3.4. Comparative genomics</h3>
<p>Because no closely related species has yet been sequenced, we chose CD and CN for genomic comparison because both are gut-associated clostridia and they have the largest or smallest genome among the fully sequenced Clostridiales (4.3 and 2.5 Mb, respectively).<sup><a reveal-id="DSR022C26" data-open="DSR022C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR022C29" data-open="DSR022C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Based on the orthologous gene group classification of CD and CN genes in the MBGD database (<a class="link link-uri" href="http://mbgd.genome.ad.jp/">http://mbgd.genome.ad.jp/</a>), in which 3741 CDSs of CD were grouped into 2851 CDSs (or CDS groups) and 2315 CDSs of CN were grouped into 2129 CDSs (or CDS groups), we compared the gene content of SFBs with these two gut-associated clostridia via an all-to-all BLAST analysis. The results revealed that 940 and 890 of the SFB CDSs (or CDSs groups) are conserved in CD and CN, respectively; 756 are conserved in both strains, whereas 425 are unique to SFB (Fig. <a reveal-id="DSR022F4" data-open="DSR022F4" class="link link-reveal link-table xref-fig">4</a>A). Most of the unique genes are of unknown function. More importantly, we identified 577 CDSs that are shared by CD and CN but not SFB. The distribution of the 577 CDSs into the functional categories of the clusters of orthologous groups (COGs) of proteins indicates that, in comparison to CD and CN, the SFB lacks remarkable numbers of genes involved in the following processes: ‘energy production and conversion (C in Fig. <a reveal-id="DSR022F4" data-open="DSR022F4" class="link link-reveal link-table xref-fig">4</a>B)’, ‘amino acid transport and metabolism (E)’, ‘nucleotide transport and metabolism (F)’ and ‘coenzyme metabolism (H)’. Significant portions of the CD/CN shared genes that are related to ‘carbohydrate transport and metabolism (G)’, ‘transcription (K)’ and ‘inorganic ion transport and metabolism (P)’ are also absent in SFBs. These results and the smaller genome size of SFBs suggest that during its strict niche specialization, genome reduction has taken place and many metabolic functions have been lost (see below). </p>                        <a id="7754029" scrollto-destination="7754029"></a>
<div id="DSR022F4" data-id="DSR022F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02204.jpeg?Expires=2147483647&Signature=gi-5g3mV~pPSyNS~6xmojo3AqDkzAeTZ5jKwUeOuJmTVh3WnIXXfJeanJBMICL~HW9L1g1KCE5yj6RTsQ8Z3rXzz2VerSfEQJGrkRm5J-I0ygeZDAJWS99mjp1HLtniijqHzUuKVk~n1BArkJDPv3Y9wid~4LUgnz8LmAOoFpk-RO2JNO0M5n3~QYmDgUWhGJ9qypc3I0~vjW9ENqKi4TdsxJItx-kNRCl0PD5gYazPr4wzlQTV7MYD2zRQhjDTXEfxI~gJphehoNsg2sHoYwv5CweIbMAPkvT3RD46gjww8BWZ-X740GjvcoP9tujDt9ryo3HTshQ92q8AUeYM2YA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A comparison of SFB gene content to C. difficile and C. novyi. (A) A Venn diagram showing the conservation of orthologous CDSs (or CDS groups) between SFBs, CD and CN. The numbers of orthologous CDSs in each category are shown. (B) The functional distributions of orthologous CDSs belonging to three conservation categories; ‘conserved in the three bacteria’ (left bars), ‘conserved in CD and CN but not in SFB’ (middle) and ‘unique to SFB’ (right). The functional classification follows the COG categories (http://www.ncbi.nlm.nih.gov/COG/) (C: energy production and conversion; D: Cell cycle control, mitosis and meiosis; E: amino acid transport and metabolism; F: nucleotide transport and metabolism; G: carbohydrate transport and metabolism; H: coenzyme transport and metabolism; I: lipid transport and metabolism; J: translation; K: transcription; L: replication, recombination and repair; M: cell wall/membrane biogenesis; N: cell motility; O: posttranslational modification, protein turnover, chaperones; P: inorganic ion transport and metabolism; Q: secondary metabolites biosynthesis, transport and catabolism; R: general function prediction only; S: function unknown; T: signal transduction mechanisms; U: intracellular trafficking and secretion; V: defense mechanisms) and ‘X’ indicates CDSs that are not assigned to any category. Each bar represents the percentage of CDSs belonging to each functional category in each of three conservation categories." path-from-xml="dsr02204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754029/dsr02204.jpeg" path-from-xml="dsr02204.jpeg" target="_blank">View large</a><a section="7754029" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02204.jpeg?Expires=2147483647&Signature=kLDJOiWPObN9jxdxqObdYSH0c-Vu7ouqNUGYBocKqMZC-HeEXLfiyoGemogLzYAYPDW3PHydQhi~loo1e1E2WzlDAQhJOhmQqRLH-opTFn6~NQFp4gEnlovnigqF6GtoXngCThcd~BE0EM8X9bEK-Y7IE3nAVUkIr413vA~GPOt-5U-fQrVaxyfQhtiL5FdbvVJw~qZvvN8NfnlscvhbVYdWInar66owasrXflxl7Yr2XcOpXvM7m6ALmsjNSxzQRG5c49cTX2pygeCNGZPk1NYRuX4KAR6j4T2hNhmf1qBayOQ06pUH7csA8LNbHNvPsGHqd9RgnWEwcLSD62hSdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754029&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A comparison of SFB gene content to <em>C. difficile</em> and <em>C. novyi</em>. (A) A Venn diagram showing the conservation of orthologous CDSs (or CDS groups) between SFBs, CD and CN. The numbers of orthologous CDSs in each category are shown. (B) The functional distributions of orthologous CDSs belonging to three conservation categories; ‘conserved in the three bacteria’ (left bars), ‘conserved in CD and CN but not in SFB’ (middle) and ‘unique to SFB’ (right). The functional classification follows the COG categories (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/COG/">http://www.ncbi.nlm.nih.gov/COG/</a>) (C: energy production and conversion; D: Cell cycle control, mitosis and meiosis; E: amino acid transport and metabolism; F: nucleotide transport and metabolism; G: carbohydrate transport and metabolism; H: coenzyme transport and metabolism; I: lipid transport and metabolism; J: translation; K: transcription; L: replication, recombination and repair; M: cell wall/membrane biogenesis; N: cell motility; O: posttranslational modification, protein turnover, chaperones; P: inorganic ion transport and metabolism; Q: secondary metabolites biosynthesis, transport and catabolism; R: general function prediction only; S: function unknown; T: signal transduction mechanisms; U: intracellular trafficking and secretion; V: defense mechanisms) and ‘X’ indicates CDSs that are not assigned to any category. Each bar represents the percentage of CDSs belonging to each functional category in each of three conservation categories.</p></div></div><div content-id="DSR022F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02204.jpeg?Expires=2147483647&Signature=gi-5g3mV~pPSyNS~6xmojo3AqDkzAeTZ5jKwUeOuJmTVh3WnIXXfJeanJBMICL~HW9L1g1KCE5yj6RTsQ8Z3rXzz2VerSfEQJGrkRm5J-I0ygeZDAJWS99mjp1HLtniijqHzUuKVk~n1BArkJDPv3Y9wid~4LUgnz8LmAOoFpk-RO2JNO0M5n3~QYmDgUWhGJ9qypc3I0~vjW9ENqKi4TdsxJItx-kNRCl0PD5gYazPr4wzlQTV7MYD2zRQhjDTXEfxI~gJphehoNsg2sHoYwv5CweIbMAPkvT3RD46gjww8BWZ-X740GjvcoP9tujDt9ryo3HTshQ92q8AUeYM2YA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A comparison of SFB gene content to C. difficile and C. novyi. (A) A Venn diagram showing the conservation of orthologous CDSs (or CDS groups) between SFBs, CD and CN. The numbers of orthologous CDSs in each category are shown. (B) The functional distributions of orthologous CDSs belonging to three conservation categories; ‘conserved in the three bacteria’ (left bars), ‘conserved in CD and CN but not in SFB’ (middle) and ‘unique to SFB’ (right). The functional classification follows the COG categories (http://www.ncbi.nlm.nih.gov/COG/) (C: energy production and conversion; D: Cell cycle control, mitosis and meiosis; E: amino acid transport and metabolism; F: nucleotide transport and metabolism; G: carbohydrate transport and metabolism; H: coenzyme transport and metabolism; I: lipid transport and metabolism; J: translation; K: transcription; L: replication, recombination and repair; M: cell wall/membrane biogenesis; N: cell motility; O: posttranslational modification, protein turnover, chaperones; P: inorganic ion transport and metabolism; Q: secondary metabolites biosynthesis, transport and catabolism; R: general function prediction only; S: function unknown; T: signal transduction mechanisms; U: intracellular trafficking and secretion; V: defense mechanisms) and ‘X’ indicates CDSs that are not assigned to any category. Each bar represents the percentage of CDSs belonging to each functional category in each of three conservation categories." path-from-xml="dsr02204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754029/dsr02204.jpeg" path-from-xml="dsr02204.jpeg" target="_blank">View large</a><a section="7754029" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02204.jpeg?Expires=2147483647&Signature=kLDJOiWPObN9jxdxqObdYSH0c-Vu7ouqNUGYBocKqMZC-HeEXLfiyoGemogLzYAYPDW3PHydQhi~loo1e1E2WzlDAQhJOhmQqRLH-opTFn6~NQFp4gEnlovnigqF6GtoXngCThcd~BE0EM8X9bEK-Y7IE3nAVUkIr413vA~GPOt-5U-fQrVaxyfQhtiL5FdbvVJw~qZvvN8NfnlscvhbVYdWInar66owasrXflxl7Yr2XcOpXvM7m6ALmsjNSxzQRG5c49cTX2pygeCNGZPk1NYRuX4KAR6j4T2hNhmf1qBayOQ06pUH7csA8LNbHNvPsGHqd9RgnWEwcLSD62hSdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754029&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A comparison of SFB gene content to <em>C. difficile</em> and <em>C. novyi</em>. (A) A Venn diagram showing the conservation of orthologous CDSs (or CDS groups) between SFBs, CD and CN. The numbers of orthologous CDSs in each category are shown. (B) The functional distributions of orthologous CDSs belonging to three conservation categories; ‘conserved in the three bacteria’ (left bars), ‘conserved in CD and CN but not in SFB’ (middle) and ‘unique to SFB’ (right). The functional classification follows the COG categories (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/COG/">http://www.ncbi.nlm.nih.gov/COG/</a>) (C: energy production and conversion; D: Cell cycle control, mitosis and meiosis; E: amino acid transport and metabolism; F: nucleotide transport and metabolism; G: carbohydrate transport and metabolism; H: coenzyme transport and metabolism; I: lipid transport and metabolism; J: translation; K: transcription; L: replication, recombination and repair; M: cell wall/membrane biogenesis; N: cell motility; O: posttranslational modification, protein turnover, chaperones; P: inorganic ion transport and metabolism; Q: secondary metabolites biosynthesis, transport and catabolism; R: general function prediction only; S: function unknown; T: signal transduction mechanisms; U: intracellular trafficking and secretion; V: defense mechanisms) and ‘X’ indicates CDSs that are not assigned to any category. Each bar represents the percentage of CDSs belonging to each functional category in each of three conservation categories.</p></div></div>                            <h3 scrollto-destination=7754030 id="7754030" class="section-title" data-legacy-id=s3e>3.5. Metabolic capabilities</h3>
<p>Consistent with the prediction that SFBs are strict anaerobes, no CDS for tricarboxylic acid cycle- or respiratory chain-related proteins were identified, whereas the SFB contains two catalases and one peroxidase. They may confer some level of oxygen tolerance, which is required for the SFB to pursue an epicellular life on intestinal epithelia where a microaerobic environment is formed. However, SFBs contain a complete enzyme set for the glycolytic pathway from glucose to pyruvate (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2A</a>). SFBs can further metabolize pyruvate using the pyruvate:ferredoxin oxidoreductase–hydrogenase system to produce acetate or ethanol but not butyrate and butanol. Pyruvate can also be metabolized to lactate. These are the primary energy production pathways for the SFB, like many other clostridia.<sup><a reveal-id="DSR022C30" data-open="DSR022C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Furthermore, SFBs possess a D-ribose ATP-binding cassette (ABC) transporter, ribokinase and transketolase and can, therefore, uptake extracellular ribose and metabolize it to glyceraldehyde-3-phosphate, an intermediate of the glycolytic pathway. Although the pentose phosphate pathway (PPP) is conserved, components for the <em>de novo</em> synthesis of purine and pyrimidine nucleotides from phosphoribosyl pyrophosphate (PRPP) are missing. Thus, it is not likely that the PPP of SFBs is involved in nucleotide biosynthesis.</p><p>A more striking metabolic feature of SFBs is the almost complete lack of <em>de novo</em> biosynthesis pathways for amino acids, vitamins/cofactors and nucleotides (Table <a reveal-id="DSR022TB2" data-open="DSR022TB2" class="link link-reveal link-table xref-fig">2</a>). As for amino acid biosynthesis, we identified only 20 genes related to this function. Genes for lysine biosynthesis are relatively well conserved, but they probably just function to synthesize diaminopimeric acid as a component of peptidoglycan. Similarly, SFBs possess only 14 genes for vitamin and cofactor biosynthesis, and almost all the genes required for the biosynthesis of thiamin, riboflavin, pyridoxine, nicotinamide, pantothenate, coenzyme A, biotin, folate and vitamin B<sub>12</sub> are absent. As mentioned above, SFBs are unable to synthesize purines and pyrimidines from PRPP. In addition, key enzymes for nucleotide interconversion, such as thymidylate synthase and guanine monophosphate reductase, are also absent in the SFB. </p>                        <a id="7754033" scrollto-destination="7754033"></a>
<div content-id="DSR022TB2" class="table-modal"><div class="table-wrap"><div id="DSR022TB2" class="table-wrap-title" data-id="DSR022TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Gene repertoires for amino acid and vitamin/cofactor biosynthesis in <em>C. difficile</em> (CD), <em>C. novyi</em> (CN) and SFB</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organisms </th><th>CD </th><th>CN </th><th>SFB </th></tr></thead><tbody><tr><td colspan="4">Amino acids </td></tr><tr><td rowspan="5"> Glycine, serine and threonine </td><td><em>serA</em> </td><td><em>serA</em> </td><td><em>serA</em> </td></tr><tr><td><em>serB</em> </td><td><em>serB</em> </td><td> </td></tr><tr><td><em>glyA</em> </td><td><em>glyA</em> </td><td><em>glyA</em> </td></tr><tr><td><em>thrBC</em> </td><td><em>thrBC</em> </td><td> </td></tr><tr><td><em>hom</em> </td><td><em>hom</em> </td><td> </td></tr><tr><td rowspan="5"> Cysteine and methionine </td><td><em>cysEK</em> </td><td><em>cysEK</em> </td><td> </td></tr><tr><td><em>metAY</em> </td><td><em>metAY</em> </td><td> </td></tr><tr><td><em>metC</em> </td><td> </td><td><em>metC</em> </td></tr><tr><td><em>metH</em> </td><td><em>metH</em> </td><td> </td></tr><tr><td><em>metK</em> </td><td><em>metK</em> </td><td><em>metK</em> </td></tr><tr><td rowspan="4"> Valine, leucine and isoleucine </td><td><em>ilvA</em> </td><td><em>ilvA</em> </td><td> </td></tr><tr><td><em>ilvBCD</em> </td><td><em>ilvBCD</em> </td><td> </td></tr><tr><td> </td><td><em>ilvE</em> </td><td><em>ilvE</em> </td></tr><tr><td><em>leuABCD</em> </td><td> </td><td> </td></tr><tr><td rowspan="3"> Lysine </td><td><em>dapABCDEFG</em> </td><td><em>dapABCDEFG</em> </td><td><em>dapABDFG</em> </td></tr><tr><td><em>lysA</em> </td><td><em>lysA</em> </td><td><em>lysA</em> </td></tr><tr><td><em>glnA</em> </td><td><em>glnA</em> </td><td><em>glnA</em> </td></tr><tr><td rowspan="3"> Alanine, aspartate  and glutamate </td><td><em>asnA</em> </td><td><em>asnA</em> </td><td> </td></tr><tr><td><em>asnB</em> </td><td> </td><td><em>asnB</em> </td></tr><tr><td><em>gltD</em> </td><td><em>gltD</em> </td><td> </td></tr><tr><td rowspan="5"> Arginine and proline </td><td><em>aspC</em> </td><td> </td><td><em>aspC</em> </td></tr><tr><td><em>aspD</em> </td><td><em>aspD</em> </td><td> </td></tr><tr><td><em>proC</em> </td><td><em>proABC</em> </td><td> </td></tr><tr><td><em>argBCDEFJ</em> </td><td><em>argBCDFJ</em> </td><td> </td></tr><tr><td><em>argGH</em> </td><td><em>argGH</em> </td><td> </td></tr><tr><td rowspan="2"> Histidine </td><td><em>hisABCDFGHIZ</em> </td><td><em>hisABCDFGIZ</em> </td><td> </td></tr><tr><td><em>hisK</em> </td><td><em>hisK</em> </td><td> </td></tr><tr><td rowspan="5"> Phenylalanine, tyrosine and tryptophan </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td></tr><tr><td><em>aroD</em> </td><td><em>aroD</em> </td><td> </td></tr><tr><td><em>trpB</em> </td><td><em>trpB</em> </td><td> </td></tr><tr><td><em>tyrA</em> </td><td><em>tyrA</em> </td><td> </td></tr><tr><td><em>pheA</em> </td><td><em>pheA</em> </td><td> </td></tr><tr><td colspan="4">Vitamins and cofactors </td></tr><tr><td rowspan="5"> Thiamin </td><td><em>thiCFGH</em> </td><td><em>thiCFGH</em> </td><td><em>thiH</em> </td></tr><tr><td><em>thiD</em> </td><td><em>thiD</em> </td><td> </td></tr><tr><td><em>thiI</em> </td><td><em>thiI</em> </td><td> </td></tr><tr><td><em>thiN</em> </td><td><em>thiN</em> </td><td><em>thiN</em> </td></tr><tr><td><em>isc</em> </td><td><em>isc</em> </td><td><em>isc</em> </td></tr><tr><td rowspan="2"> Riboflavin </td><td><em>ribABDH</em> </td><td><em>ribABDH</em> </td><td> </td></tr><tr><td><em>ribC</em> </td><td><em>ribC</em> </td><td><em>ribC</em> </td></tr><tr><td rowspan="3"> Vitamin B<sub>6</sub> </td><td> </td><td><em>pdxA</em> </td><td> </td></tr><tr><td><em>pdxK</em> </td><td> </td><td> </td></tr><tr><td> </td><td><em>pdxS</em> </td><td><em>pdxS</em> </td></tr><tr><td rowspan="3"> Nicotinate and nicotinamide </td><td><em>nadABC</em> </td><td><em>nadABC</em> </td><td> </td></tr><tr><td><em>nadD</em> </td><td><em>nadD</em> </td><td><em>nadD</em> </td></tr><tr><td><em>nadE</em> </td><td> </td><td> </td></tr><tr><td rowspan="2"> Pantothenate </td><td><em>panBC</em> </td><td><em>panBCD</em> </td><td> </td></tr><tr><td><em>panE</em> </td><td><em>panE</em> </td><td> </td></tr><tr><td rowspan="4"> Coenzyme A </td><td><em>coaX</em> </td><td><em>coaX</em> </td><td> </td></tr><tr><td><em>coaBC</em> </td><td><em>coaBC</em> </td><td> </td></tr><tr><td><em>coaD</em> </td><td><em>coaD</em> </td><td><em>coaD</em> </td></tr><tr><td><em>coaE</em> </td><td><em>coaE</em> </td><td><em>coaE</em> </td></tr><tr><td rowspan="3"> Biotin </td><td><em>bioA</em> </td><td><em>bioAD</em> </td><td> </td></tr><tr><td><em>bioB</em> </td><td><em>bioB</em> </td><td> </td></tr><tr><td><em>birA</em> </td><td><em>birA</em> </td><td><em>birA</em> </td></tr><tr><td rowspan="4"> Folate </td><td><em>pabABC</em> </td><td><em>pabABC</em> </td><td><em>pabC</em> </td></tr><tr><td> </td><td><em>folA</em> </td><td><em>folA</em> </td></tr><tr><td><em>folBEKP</em> </td><td><em>folBEKP</em> </td><td> </td></tr><tr><td><em>folC</em> </td><td><em>folC</em> </td><td> </td></tr><tr><td rowspan="6"> Vitamin B<sub>12</sub> </td><td><em>hemBCD</em> </td><td><em>hemABCDL</em> </td><td> </td></tr><tr><td><em>hemN</em> </td><td><em>hemN</em> </td><td> </td></tr><tr><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiA</em> </td></tr><tr><td><em>cobCDSTU</em> </td><td><em>cobDSTU</em> </td><td><em>cobC</em> </td></tr><tr><td> </td><td><em>cobO</em> </td><td><em>cobO</em> </td></tr><tr><td><em>eutT</em> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Organisms </th><th>CD </th><th>CN </th><th>SFB </th></tr></thead><tbody><tr><td colspan="4">Amino acids </td></tr><tr><td rowspan="5"> Glycine, serine and threonine </td><td><em>serA</em> </td><td><em>serA</em> </td><td><em>serA</em> </td></tr><tr><td><em>serB</em> </td><td><em>serB</em> </td><td> </td></tr><tr><td><em>glyA</em> </td><td><em>glyA</em> </td><td><em>glyA</em> </td></tr><tr><td><em>thrBC</em> </td><td><em>thrBC</em> </td><td> </td></tr><tr><td><em>hom</em> </td><td><em>hom</em> </td><td> </td></tr><tr><td rowspan="5"> Cysteine and methionine </td><td><em>cysEK</em> </td><td><em>cysEK</em> </td><td> </td></tr><tr><td><em>metAY</em> </td><td><em>metAY</em> </td><td> </td></tr><tr><td><em>metC</em> </td><td> </td><td><em>metC</em> </td></tr><tr><td><em>metH</em> </td><td><em>metH</em> </td><td> </td></tr><tr><td><em>metK</em> </td><td><em>metK</em> </td><td><em>metK</em> </td></tr><tr><td rowspan="4"> Valine, leucine and isoleucine </td><td><em>ilvA</em> </td><td><em>ilvA</em> </td><td> </td></tr><tr><td><em>ilvBCD</em> </td><td><em>ilvBCD</em> </td><td> </td></tr><tr><td> </td><td><em>ilvE</em> </td><td><em>ilvE</em> </td></tr><tr><td><em>leuABCD</em> </td><td> </td><td> </td></tr><tr><td rowspan="3"> Lysine </td><td><em>dapABCDEFG</em> </td><td><em>dapABCDEFG</em> </td><td><em>dapABDFG</em> </td></tr><tr><td><em>lysA</em> </td><td><em>lysA</em> </td><td><em>lysA</em> </td></tr><tr><td><em>glnA</em> </td><td><em>glnA</em> </td><td><em>glnA</em> </td></tr><tr><td rowspan="3"> Alanine, aspartate  and glutamate </td><td><em>asnA</em> </td><td><em>asnA</em> </td><td> </td></tr><tr><td><em>asnB</em> </td><td> </td><td><em>asnB</em> </td></tr><tr><td><em>gltD</em> </td><td><em>gltD</em> </td><td> </td></tr><tr><td rowspan="5"> Arginine and proline </td><td><em>aspC</em> </td><td> </td><td><em>aspC</em> </td></tr><tr><td><em>aspD</em> </td><td><em>aspD</em> </td><td> </td></tr><tr><td><em>proC</em> </td><td><em>proABC</em> </td><td> </td></tr><tr><td><em>argBCDEFJ</em> </td><td><em>argBCDFJ</em> </td><td> </td></tr><tr><td><em>argGH</em> </td><td><em>argGH</em> </td><td> </td></tr><tr><td rowspan="2"> Histidine </td><td><em>hisABCDFGHIZ</em> </td><td><em>hisABCDFGIZ</em> </td><td> </td></tr><tr><td><em>hisK</em> </td><td><em>hisK</em> </td><td> </td></tr><tr><td rowspan="5"> Phenylalanine, tyrosine and tryptophan </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td></tr><tr><td><em>aroD</em> </td><td><em>aroD</em> </td><td> </td></tr><tr><td><em>trpB</em> </td><td><em>trpB</em> </td><td> </td></tr><tr><td><em>tyrA</em> </td><td><em>tyrA</em> </td><td> </td></tr><tr><td><em>pheA</em> </td><td><em>pheA</em> </td><td> </td></tr><tr><td colspan="4">Vitamins and cofactors </td></tr><tr><td rowspan="5"> Thiamin </td><td><em>thiCFGH</em> </td><td><em>thiCFGH</em> </td><td><em>thiH</em> </td></tr><tr><td><em>thiD</em> </td><td><em>thiD</em> </td><td> </td></tr><tr><td><em>thiI</em> </td><td><em>thiI</em> </td><td> </td></tr><tr><td><em>thiN</em> </td><td><em>thiN</em> </td><td><em>thiN</em> </td></tr><tr><td><em>isc</em> </td><td><em>isc</em> </td><td><em>isc</em> </td></tr><tr><td rowspan="2"> Riboflavin </td><td><em>ribABDH</em> </td><td><em>ribABDH</em> </td><td> </td></tr><tr><td><em>ribC</em> </td><td><em>ribC</em> </td><td><em>ribC</em> </td></tr><tr><td rowspan="3"> Vitamin B<sub>6</sub> </td><td> </td><td><em>pdxA</em> </td><td> </td></tr><tr><td><em>pdxK</em> </td><td> </td><td> </td></tr><tr><td> </td><td><em>pdxS</em> </td><td><em>pdxS</em> </td></tr><tr><td rowspan="3"> Nicotinate and nicotinamide </td><td><em>nadABC</em> </td><td><em>nadABC</em> </td><td> </td></tr><tr><td><em>nadD</em> </td><td><em>nadD</em> </td><td><em>nadD</em> </td></tr><tr><td><em>nadE</em> </td><td> </td><td> </td></tr><tr><td rowspan="2"> Pantothenate </td><td><em>panBC</em> </td><td><em>panBCD</em> </td><td> </td></tr><tr><td><em>panE</em> </td><td><em>panE</em> </td><td> </td></tr><tr><td rowspan="4"> Coenzyme A </td><td><em>coaX</em> </td><td><em>coaX</em> </td><td> </td></tr><tr><td><em>coaBC</em> </td><td><em>coaBC</em> </td><td> </td></tr><tr><td><em>coaD</em> </td><td><em>coaD</em> </td><td><em>coaD</em> </td></tr><tr><td><em>coaE</em> </td><td><em>coaE</em> </td><td><em>coaE</em> </td></tr><tr><td rowspan="3"> Biotin </td><td><em>bioA</em> </td><td><em>bioAD</em> </td><td> </td></tr><tr><td><em>bioB</em> </td><td><em>bioB</em> </td><td> </td></tr><tr><td><em>birA</em> </td><td><em>birA</em> </td><td><em>birA</em> </td></tr><tr><td rowspan="4"> Folate </td><td><em>pabABC</em> </td><td><em>pabABC</em> </td><td><em>pabC</em> </td></tr><tr><td> </td><td><em>folA</em> </td><td><em>folA</em> </td></tr><tr><td><em>folBEKP</em> </td><td><em>folBEKP</em> </td><td> </td></tr><tr><td><em>folC</em> </td><td><em>folC</em> </td><td> </td></tr><tr><td rowspan="6"> Vitamin B<sub>12</sub> </td><td><em>hemBCD</em> </td><td><em>hemABCDL</em> </td><td> </td></tr><tr><td><em>hemN</em> </td><td><em>hemN</em> </td><td> </td></tr><tr><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiA</em> </td></tr><tr><td><em>cobCDSTU</em> </td><td><em>cobDSTU</em> </td><td><em>cobC</em> </td></tr><tr><td> </td><td><em>cobO</em> </td><td><em>cobO</em> </td></tr><tr><td><em>eutT</em> </td><td> </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7754033" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR022TB2" class="table-wrap-title" data-id="DSR022TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Gene repertoires for amino acid and vitamin/cofactor biosynthesis in <em>C. difficile</em> (CD), <em>C. novyi</em> (CN) and SFB</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organisms </th><th>CD </th><th>CN </th><th>SFB </th></tr></thead><tbody><tr><td colspan="4">Amino acids </td></tr><tr><td rowspan="5"> Glycine, serine and threonine </td><td><em>serA</em> </td><td><em>serA</em> </td><td><em>serA</em> </td></tr><tr><td><em>serB</em> </td><td><em>serB</em> </td><td> </td></tr><tr><td><em>glyA</em> </td><td><em>glyA</em> </td><td><em>glyA</em> </td></tr><tr><td><em>thrBC</em> </td><td><em>thrBC</em> </td><td> </td></tr><tr><td><em>hom</em> </td><td><em>hom</em> </td><td> </td></tr><tr><td rowspan="5"> Cysteine and methionine </td><td><em>cysEK</em> </td><td><em>cysEK</em> </td><td> </td></tr><tr><td><em>metAY</em> </td><td><em>metAY</em> </td><td> </td></tr><tr><td><em>metC</em> </td><td> </td><td><em>metC</em> </td></tr><tr><td><em>metH</em> </td><td><em>metH</em> </td><td> </td></tr><tr><td><em>metK</em> </td><td><em>metK</em> </td><td><em>metK</em> </td></tr><tr><td rowspan="4"> Valine, leucine and isoleucine </td><td><em>ilvA</em> </td><td><em>ilvA</em> </td><td> </td></tr><tr><td><em>ilvBCD</em> </td><td><em>ilvBCD</em> </td><td> </td></tr><tr><td> </td><td><em>ilvE</em> </td><td><em>ilvE</em> </td></tr><tr><td><em>leuABCD</em> </td><td> </td><td> </td></tr><tr><td rowspan="3"> Lysine </td><td><em>dapABCDEFG</em> </td><td><em>dapABCDEFG</em> </td><td><em>dapABDFG</em> </td></tr><tr><td><em>lysA</em> </td><td><em>lysA</em> </td><td><em>lysA</em> </td></tr><tr><td><em>glnA</em> </td><td><em>glnA</em> </td><td><em>glnA</em> </td></tr><tr><td rowspan="3"> Alanine, aspartate  and glutamate </td><td><em>asnA</em> </td><td><em>asnA</em> </td><td> </td></tr><tr><td><em>asnB</em> </td><td> </td><td><em>asnB</em> </td></tr><tr><td><em>gltD</em> </td><td><em>gltD</em> </td><td> </td></tr><tr><td rowspan="5"> Arginine and proline </td><td><em>aspC</em> </td><td> </td><td><em>aspC</em> </td></tr><tr><td><em>aspD</em> </td><td><em>aspD</em> </td><td> </td></tr><tr><td><em>proC</em> </td><td><em>proABC</em> </td><td> </td></tr><tr><td><em>argBCDEFJ</em> </td><td><em>argBCDFJ</em> </td><td> </td></tr><tr><td><em>argGH</em> </td><td><em>argGH</em> </td><td> </td></tr><tr><td rowspan="2"> Histidine </td><td><em>hisABCDFGHIZ</em> </td><td><em>hisABCDFGIZ</em> </td><td> </td></tr><tr><td><em>hisK</em> </td><td><em>hisK</em> </td><td> </td></tr><tr><td rowspan="5"> Phenylalanine, tyrosine and tryptophan </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td></tr><tr><td><em>aroD</em> </td><td><em>aroD</em> </td><td> </td></tr><tr><td><em>trpB</em> </td><td><em>trpB</em> </td><td> </td></tr><tr><td><em>tyrA</em> </td><td><em>tyrA</em> </td><td> </td></tr><tr><td><em>pheA</em> </td><td><em>pheA</em> </td><td> </td></tr><tr><td colspan="4">Vitamins and cofactors </td></tr><tr><td rowspan="5"> Thiamin </td><td><em>thiCFGH</em> </td><td><em>thiCFGH</em> </td><td><em>thiH</em> </td></tr><tr><td><em>thiD</em> </td><td><em>thiD</em> </td><td> </td></tr><tr><td><em>thiI</em> </td><td><em>thiI</em> </td><td> </td></tr><tr><td><em>thiN</em> </td><td><em>thiN</em> </td><td><em>thiN</em> </td></tr><tr><td><em>isc</em> </td><td><em>isc</em> </td><td><em>isc</em> </td></tr><tr><td rowspan="2"> Riboflavin </td><td><em>ribABDH</em> </td><td><em>ribABDH</em> </td><td> </td></tr><tr><td><em>ribC</em> </td><td><em>ribC</em> </td><td><em>ribC</em> </td></tr><tr><td rowspan="3"> Vitamin B<sub>6</sub> </td><td> </td><td><em>pdxA</em> </td><td> </td></tr><tr><td><em>pdxK</em> </td><td> </td><td> </td></tr><tr><td> </td><td><em>pdxS</em> </td><td><em>pdxS</em> </td></tr><tr><td rowspan="3"> Nicotinate and nicotinamide </td><td><em>nadABC</em> </td><td><em>nadABC</em> </td><td> </td></tr><tr><td><em>nadD</em> </td><td><em>nadD</em> </td><td><em>nadD</em> </td></tr><tr><td><em>nadE</em> </td><td> </td><td> </td></tr><tr><td rowspan="2"> Pantothenate </td><td><em>panBC</em> </td><td><em>panBCD</em> </td><td> </td></tr><tr><td><em>panE</em> </td><td><em>panE</em> </td><td> </td></tr><tr><td rowspan="4"> Coenzyme A </td><td><em>coaX</em> </td><td><em>coaX</em> </td><td> </td></tr><tr><td><em>coaBC</em> </td><td><em>coaBC</em> </td><td> </td></tr><tr><td><em>coaD</em> </td><td><em>coaD</em> </td><td><em>coaD</em> </td></tr><tr><td><em>coaE</em> </td><td><em>coaE</em> </td><td><em>coaE</em> </td></tr><tr><td rowspan="3"> Biotin </td><td><em>bioA</em> </td><td><em>bioAD</em> </td><td> </td></tr><tr><td><em>bioB</em> </td><td><em>bioB</em> </td><td> </td></tr><tr><td><em>birA</em> </td><td><em>birA</em> </td><td><em>birA</em> </td></tr><tr><td rowspan="4"> Folate </td><td><em>pabABC</em> </td><td><em>pabABC</em> </td><td><em>pabC</em> </td></tr><tr><td> </td><td><em>folA</em> </td><td><em>folA</em> </td></tr><tr><td><em>folBEKP</em> </td><td><em>folBEKP</em> </td><td> </td></tr><tr><td><em>folC</em> </td><td><em>folC</em> </td><td> </td></tr><tr><td rowspan="6"> Vitamin B<sub>12</sub> </td><td><em>hemBCD</em> </td><td><em>hemABCDL</em> </td><td> </td></tr><tr><td><em>hemN</em> </td><td><em>hemN</em> </td><td> </td></tr><tr><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiA</em> </td></tr><tr><td><em>cobCDSTU</em> </td><td><em>cobDSTU</em> </td><td><em>cobC</em> </td></tr><tr><td> </td><td><em>cobO</em> </td><td><em>cobO</em> </td></tr><tr><td><em>eutT</em> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Organisms </th><th>CD </th><th>CN </th><th>SFB </th></tr></thead><tbody><tr><td colspan="4">Amino acids </td></tr><tr><td rowspan="5"> Glycine, serine and threonine </td><td><em>serA</em> </td><td><em>serA</em> </td><td><em>serA</em> </td></tr><tr><td><em>serB</em> </td><td><em>serB</em> </td><td> </td></tr><tr><td><em>glyA</em> </td><td><em>glyA</em> </td><td><em>glyA</em> </td></tr><tr><td><em>thrBC</em> </td><td><em>thrBC</em> </td><td> </td></tr><tr><td><em>hom</em> </td><td><em>hom</em> </td><td> </td></tr><tr><td rowspan="5"> Cysteine and methionine </td><td><em>cysEK</em> </td><td><em>cysEK</em> </td><td> </td></tr><tr><td><em>metAY</em> </td><td><em>metAY</em> </td><td> </td></tr><tr><td><em>metC</em> </td><td> </td><td><em>metC</em> </td></tr><tr><td><em>metH</em> </td><td><em>metH</em> </td><td> </td></tr><tr><td><em>metK</em> </td><td><em>metK</em> </td><td><em>metK</em> </td></tr><tr><td rowspan="4"> Valine, leucine and isoleucine </td><td><em>ilvA</em> </td><td><em>ilvA</em> </td><td> </td></tr><tr><td><em>ilvBCD</em> </td><td><em>ilvBCD</em> </td><td> </td></tr><tr><td> </td><td><em>ilvE</em> </td><td><em>ilvE</em> </td></tr><tr><td><em>leuABCD</em> </td><td> </td><td> </td></tr><tr><td rowspan="3"> Lysine </td><td><em>dapABCDEFG</em> </td><td><em>dapABCDEFG</em> </td><td><em>dapABDFG</em> </td></tr><tr><td><em>lysA</em> </td><td><em>lysA</em> </td><td><em>lysA</em> </td></tr><tr><td><em>glnA</em> </td><td><em>glnA</em> </td><td><em>glnA</em> </td></tr><tr><td rowspan="3"> Alanine, aspartate  and glutamate </td><td><em>asnA</em> </td><td><em>asnA</em> </td><td> </td></tr><tr><td><em>asnB</em> </td><td> </td><td><em>asnB</em> </td></tr><tr><td><em>gltD</em> </td><td><em>gltD</em> </td><td> </td></tr><tr><td rowspan="5"> Arginine and proline </td><td><em>aspC</em> </td><td> </td><td><em>aspC</em> </td></tr><tr><td><em>aspD</em> </td><td><em>aspD</em> </td><td> </td></tr><tr><td><em>proC</em> </td><td><em>proABC</em> </td><td> </td></tr><tr><td><em>argBCDEFJ</em> </td><td><em>argBCDFJ</em> </td><td> </td></tr><tr><td><em>argGH</em> </td><td><em>argGH</em> </td><td> </td></tr><tr><td rowspan="2"> Histidine </td><td><em>hisABCDFGHIZ</em> </td><td><em>hisABCDFGIZ</em> </td><td> </td></tr><tr><td><em>hisK</em> </td><td><em>hisK</em> </td><td> </td></tr><tr><td rowspan="5"> Phenylalanine, tyrosine and tryptophan </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td><td><em>aroABCEGK</em> </td></tr><tr><td><em>aroD</em> </td><td><em>aroD</em> </td><td> </td></tr><tr><td><em>trpB</em> </td><td><em>trpB</em> </td><td> </td></tr><tr><td><em>tyrA</em> </td><td><em>tyrA</em> </td><td> </td></tr><tr><td><em>pheA</em> </td><td><em>pheA</em> </td><td> </td></tr><tr><td colspan="4">Vitamins and cofactors </td></tr><tr><td rowspan="5"> Thiamin </td><td><em>thiCFGH</em> </td><td><em>thiCFGH</em> </td><td><em>thiH</em> </td></tr><tr><td><em>thiD</em> </td><td><em>thiD</em> </td><td> </td></tr><tr><td><em>thiI</em> </td><td><em>thiI</em> </td><td> </td></tr><tr><td><em>thiN</em> </td><td><em>thiN</em> </td><td><em>thiN</em> </td></tr><tr><td><em>isc</em> </td><td><em>isc</em> </td><td><em>isc</em> </td></tr><tr><td rowspan="2"> Riboflavin </td><td><em>ribABDH</em> </td><td><em>ribABDH</em> </td><td> </td></tr><tr><td><em>ribC</em> </td><td><em>ribC</em> </td><td><em>ribC</em> </td></tr><tr><td rowspan="3"> Vitamin B<sub>6</sub> </td><td> </td><td><em>pdxA</em> </td><td> </td></tr><tr><td><em>pdxK</em> </td><td> </td><td> </td></tr><tr><td> </td><td><em>pdxS</em> </td><td><em>pdxS</em> </td></tr><tr><td rowspan="3"> Nicotinate and nicotinamide </td><td><em>nadABC</em> </td><td><em>nadABC</em> </td><td> </td></tr><tr><td><em>nadD</em> </td><td><em>nadD</em> </td><td><em>nadD</em> </td></tr><tr><td><em>nadE</em> </td><td> </td><td> </td></tr><tr><td rowspan="2"> Pantothenate </td><td><em>panBC</em> </td><td><em>panBCD</em> </td><td> </td></tr><tr><td><em>panE</em> </td><td><em>panE</em> </td><td> </td></tr><tr><td rowspan="4"> Coenzyme A </td><td><em>coaX</em> </td><td><em>coaX</em> </td><td> </td></tr><tr><td><em>coaBC</em> </td><td><em>coaBC</em> </td><td> </td></tr><tr><td><em>coaD</em> </td><td><em>coaD</em> </td><td><em>coaD</em> </td></tr><tr><td><em>coaE</em> </td><td><em>coaE</em> </td><td><em>coaE</em> </td></tr><tr><td rowspan="3"> Biotin </td><td><em>bioA</em> </td><td><em>bioAD</em> </td><td> </td></tr><tr><td><em>bioB</em> </td><td><em>bioB</em> </td><td> </td></tr><tr><td><em>birA</em> </td><td><em>birA</em> </td><td><em>birA</em> </td></tr><tr><td rowspan="4"> Folate </td><td><em>pabABC</em> </td><td><em>pabABC</em> </td><td><em>pabC</em> </td></tr><tr><td> </td><td><em>folA</em> </td><td><em>folA</em> </td></tr><tr><td><em>folBEKP</em> </td><td><em>folBEKP</em> </td><td> </td></tr><tr><td><em>folC</em> </td><td><em>folC</em> </td><td> </td></tr><tr><td rowspan="6"> Vitamin B<sub>12</sub> </td><td><em>hemBCD</em> </td><td><em>hemABCDL</em> </td><td> </td></tr><tr><td><em>hemN</em> </td><td><em>hemN</em> </td><td> </td></tr><tr><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiABCDEFGHJKLPT</em> </td><td><em>cbiA</em> </td></tr><tr><td><em>cobCDSTU</em> </td><td><em>cobDSTU</em> </td><td><em>cobC</em> </td></tr><tr><td> </td><td><em>cobO</em> </td><td><em>cobO</em> </td></tr><tr><td><em>eutT</em> </td><td> </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7754033" target="_blank">View Large</a></div></div><p>Severe deficiency in the biosynthesis capability of amino acids, vitamins/cofactors and nucleotides suggests that SFBs uptake these compounds from extracellular environments. Consistent with this, at least 103 genes are devoted to transport functions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). They include ABC transporter systems for oligopeptides, amino acids, sugars, phosphate, phosphonate, iron compounds and cobalt. SFBM_1365, SFBM_1366 and SFBM_1367 may constitute an energy-coupling factor transporter to uptake vitamins and cofactors,<sup><a reveal-id="DSR022C31" data-open="DSR022C31" class="link link-ref link-reveal xref-bibr">31</a></sup> although the substrate-binding components therein have yet to be identified. In addition, SFBs possess nine phosphotransferase systems to uptake various simple sugars and several other types of transport systems for nucleotides, amino acids and other compounds.</p><p>Although the Tat protein secretion system is not present in the SFB, the Sec system is conserved. It may be noteworthy that, among the eight glycosyl hydrolases of SFBs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), only SFBM_0066 was predicted as an extracellular enzyme by the PSORT program (<a class="link link-uri" href="http://psort.hgc.jp/">http://psort.hgc.jp/</a>). This suggests that SFBs have a very limited ability to digest food- or host-derived polysaccharides by themselves and, instead, primarily uptake simple sugars present in ready-for-use states in food or provided by the host or other gut bacteria. In contrast, 41 protease/peptidases are encoded by SFB. Of these, 4 are extracellular enzymes and 20 are membrane-associated (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). In SFBs, it is very likely that these enzymes, together with transporters for oligopeptides and amino acids, contribute to the acquisition of amino acids derived from external polypeptides. Similarly, two extracellular nucleases (SFBM_0242 and SFBM_0633) probably contribute to nucleotide acquisition from external sources.</p><p>In summary, SFB growth heavily relies on the uptake of essential compounds from extracellular environments, some of which may be supplied by host cells that SFB tightly attaches to. These metabolic features form the basis of the non-culturability of SFBs <em>in vitro</em> and would be valuable information for developing culture media and protocols for the SFB.</p>                            <h3 scrollto-destination=7754037 id="7754037" class="section-title" data-legacy-id=s3f>3.6. Sporulation and germination</h3>
<p>During the epicellular parasitic phase, SFB probably obtain various nutritional components via intimate interactions with host cells; however, once they are detached from the host cells or SFB-attached host cells slough off in the process of epithelial cell turnover, such nutrition supply from the host cells would be shutdown, which probably induces the sporulation process in the SFB. We identified 66 genes related to sporulation or germination in the SFB, which are dispersed throughout the chromosome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). They include key proteins for the sporulation signalling cascade, such as sporulation sigma factors, and other stage-specific sporulation proteins (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2B</a>); however, in common with other sequenced clostridia, SFBs lack several proteins that have been shown in <em>B. subtilis</em> to be involved in the phosphorelays that trigger sporulation, such as Spo0F, Spo0B and the Kin proteins.<sup><a reveal-id="DSR022C32" data-open="DSR022C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR022C33" data-open="DSR022C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In <em>B. subtilis</em>, all the five sensor histidine kinases (KinA to KinE) are orphan kinases. Among the 10 sensor kinases identified in the SFB (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>), SFBM_0504 is the sole orphan kinase and has a PAS domain-like domain, like the <em>B. subtilis</em> KinA protein<sup><a reveal-id="DSR022C32" data-open="DSR022C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and, thus, could be a candidate sensor kinase for sporulation initiation.</p><p>Key elements in the germination process involve germinant receptors that sense appropriate germinants, such as specific amino acids, to trigger germination. The GerA family of germinant receptors are encoded by tricistronic <em>ger</em> operons,<sup><a reveal-id="DSR022C34" data-open="DSR022C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and bacilli and clostridia often contain multiple <em>ger</em> operons to sense different germinants. For example, CN and <em>C. botulinum</em> (CB) possess two and three operons, respectively (Fig. <a reveal-id="DSR022F5" data-open="DSR022F5" class="link link-reveal link-table xref-fig">5</a>), although the <em>ger</em> operon was not identified in the CD genome.<sup><a reveal-id="DSR022C26" data-open="DSR022C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In SFBs, we identified only one <em>ger</em> operon (Fig. <a reveal-id="DSR022F5" data-open="DSR022F5" class="link link-reveal link-table xref-fig">5</a>), indicating that SFBs can sense limited types of germinants. A comparison of the analogous <em>ger</em> operon loci between SFB, CN and CB have revealed that, although many genes downstream of the <em>ger</em> operon are missing in the SFB, the overall genetic organization of this locus is conserved. </p>                        <a id="7754040" scrollto-destination="7754040"></a>
<div id="DSR022F5" data-id="DSR022F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02205.jpeg?Expires=2147483647&Signature=Ip76TgouIJVkKAvqM-uLQLFIhScOvXW42KNOUUThx~zpVD5H~elgsz9P6Olq3yifhqNBllATJAP4A3zSrY970RUQ6MX~Hfekr0P526V~DhThs7R6cRegXas1Z1tltvayDedrU8YsKzdoN2IlHVV766JF7Vwd0XMWLJwdY0OQR6QfzCB6o7z4aTp8FzP-WsfsmjtnGtsBIrAIV~-xw2POxsJnX5ylQNYdACYxndrtcHpP4Pz7JMCYTaw6sC7s0EowQ8kv6L4EG9WaFUuKVsmfx7SwYBno1q9dQMgO67h4cT60SGzrs7~Fh2C4Cj96RJxrCo-5oiQWtEhIWgJjwRRlKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the SFB ger operon locus with C. novyi (CN) and C. botulinum (CB). SFB contains a single ger operon encoding a germinant receptor, whereas CN and CB possess two and three ger operons, respectively. Among these, only the loci analogous to that of SFBs are shown. The genes conserved in the three species are depicted by blue arrows, those conserved in the CN and CB but not in SFB by green and those unique to each species by white. Regions missing in each genome are indicated by a dotted line." path-from-xml="dsr02205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754040/dsr02205.jpeg" path-from-xml="dsr02205.jpeg" target="_blank">View large</a><a section="7754040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02205.jpeg?Expires=2147483647&Signature=weqzcV-WVFKKSgvKgRpsn2gEHu9r1SpJe1Pwm3Ig7C~-xPul~6WTT5gjQvzhsNlugKmVBqh3etiXoryTNKOnYj4ibgJJn-4vdVKWq7lwzVvdT-u~c4tS3u07vRFJa31K7ZCkztqVeglLJG7ngSTHZyYgRHDvo3rcEjjhEalHJjqPHJ5k3x9bOuF4BdzFtb0igbKY-xj02EK4gfPCSo7dTycktTcSUMWHiODbInZz-LkOE23dvNXha2QGUzuHFcNrVBHWt8MG-K02kg3FPLqVtLUAC-h4jQ0dZ6ulk6z-gR6yd74uTVW94omHkduTCBAR9QJFd20FfER4fN8mSFHqAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754040&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the SFB <em>ger</em> operon locus with <em>C. novyi</em> (CN) and <em>C. botulinum</em> (CB). SFB contains a single <em>ger</em> operon encoding a germinant receptor, whereas CN and CB possess two and three <em>ger</em> operons, respectively. Among these, only the loci analogous to that of SFBs are shown. The genes conserved in the three species are depicted by blue arrows, those conserved in the CN and CB but not in SFB by green and those unique to each species by white. Regions missing in each genome are indicated by a dotted line.</p></div></div><div content-id="DSR022F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02205.jpeg?Expires=2147483647&Signature=Ip76TgouIJVkKAvqM-uLQLFIhScOvXW42KNOUUThx~zpVD5H~elgsz9P6Olq3yifhqNBllATJAP4A3zSrY970RUQ6MX~Hfekr0P526V~DhThs7R6cRegXas1Z1tltvayDedrU8YsKzdoN2IlHVV766JF7Vwd0XMWLJwdY0OQR6QfzCB6o7z4aTp8FzP-WsfsmjtnGtsBIrAIV~-xw2POxsJnX5ylQNYdACYxndrtcHpP4Pz7JMCYTaw6sC7s0EowQ8kv6L4EG9WaFUuKVsmfx7SwYBno1q9dQMgO67h4cT60SGzrs7~Fh2C4Cj96RJxrCo-5oiQWtEhIWgJjwRRlKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the SFB ger operon locus with C. novyi (CN) and C. botulinum (CB). SFB contains a single ger operon encoding a germinant receptor, whereas CN and CB possess two and three ger operons, respectively. Among these, only the loci analogous to that of SFBs are shown. The genes conserved in the three species are depicted by blue arrows, those conserved in the CN and CB but not in SFB by green and those unique to each species by white. Regions missing in each genome are indicated by a dotted line." path-from-xml="dsr02205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754040/dsr02205.jpeg" path-from-xml="dsr02205.jpeg" target="_blank">View large</a><a section="7754040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02205.jpeg?Expires=2147483647&Signature=weqzcV-WVFKKSgvKgRpsn2gEHu9r1SpJe1Pwm3Ig7C~-xPul~6WTT5gjQvzhsNlugKmVBqh3etiXoryTNKOnYj4ibgJJn-4vdVKWq7lwzVvdT-u~c4tS3u07vRFJa31K7ZCkztqVeglLJG7ngSTHZyYgRHDvo3rcEjjhEalHJjqPHJ5k3x9bOuF4BdzFtb0igbKY-xj02EK4gfPCSo7dTycktTcSUMWHiODbInZz-LkOE23dvNXha2QGUzuHFcNrVBHWt8MG-K02kg3FPLqVtLUAC-h4jQ0dZ6ulk6z-gR6yd74uTVW94omHkduTCBAR9QJFd20FfER4fN8mSFHqAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754040&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the SFB <em>ger</em> operon locus with <em>C. novyi</em> (CN) and <em>C. botulinum</em> (CB). SFB contains a single <em>ger</em> operon encoding a germinant receptor, whereas CN and CB possess two and three <em>ger</em> operons, respectively. Among these, only the loci analogous to that of SFBs are shown. The genes conserved in the three species are depicted by blue arrows, those conserved in the CN and CB but not in SFB by green and those unique to each species by white. Regions missing in each genome are indicated by a dotted line.</p></div></div><p>As shown in Fig. <a reveal-id="DSR022F5" data-open="DSR022F5" class="link link-reveal link-table xref-fig">5</a>, the <em>ispE</em> and <em>cphAB</em> genes are located just upstream of the <em>ger</em> operon in the SFB. The presence of these genes may be noteworthy in terms of the metabolism of SFBs. The <em>ispE</em> gene encodes 4-(cytidine-5′-diphospho)-2-C-methyl-<span class="small-caps">d</span>-erythritol kinase, a component of the methyl-erythritol phosphate (MEP) pathway to synthesize terpenoids that are essential for membrane and peptidoglycan biogenesis. Because other components for the pathway were also identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2C</a>), the MEP pathway is completely conserved in the SFB. The <em>cphAB</em> genes encode cyanophycin synthase and cyanophycinase (Fig. <a reveal-id="DSR022F5" data-open="DSR022F5" class="link link-reveal link-table xref-fig">5</a>). Cyanophycin is a prokaryote-specific copolymer of aspartate and arginine, which has been shown in many cyanobacteria to serve as a storage compound for nitrogen, carbon and energy.<sup><a reveal-id="DSR022C35" data-open="DSR022C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The presence of the <em>cphAB</em> and isoaspartyl dipeptidase (SFBM_1358) genes suggests that SFBs can synthesize cyanophycin and may utilize it as an amino acid pool in the germination process. In a previous analysis of 570 microbial genomes, it was predicted that only 24 strains, which include several clostridia (<em>C. beijerinckii</em>, <em>C. botulinum</em>, <em>C. perfringens</em> and <em>C. thermocellum</em>), can synthesize cyanopycin.<sup><a reveal-id="DSR022C36" data-open="DSR022C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                            <h3 scrollto-destination=7754042 id="7754042" class="section-title" data-legacy-id=s3g>3.7. Chemotaxis and flagella-mediated motility</h3>
<p>SFBs possess a full set of genes for the chemotaxis and flagella-mediated motility system. These genes are separately located in five genomic loci (Locus-1 to Locus-5; Fig. <a reveal-id="DSR022F6" data-open="DSR022F6" class="link link-reveal link-table xref-fig">6</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In addition, two methyl-accepting chemotaxis proteins (receptors for attractants) are encoded in other loci (SFBM_0442 and SFBM_1032). Flagellated SFB cells were not observed in our electron microscopic examination of SFB-monoassociated gnotobiotic mice (from 3 days to 2 weeks after oral inoculation) as well as in previous examinations.<sup><a reveal-id="DSR022C13" data-open="DSR022C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR022C37" data-open="DSR022C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSR022C38" data-open="DSR022C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Thus, it is possible that in SFBs, the system has been inactivated or is used for some other purposes, such as a type III secretion system (flagella type); however, the presence of a complete set for chemotaxis genes and four flagellin genes and the absence of recognizable pseudogenes in the gene set (except for one flagellin gene; see below) strongly suggest that SFBs possess an active chemotaxis and flagella-mediated motility system. This information provides us a previously unknown aspect of SFB biology and life cycle. It is unknown at which life cycle stage SFBs produce flagella, but the most plausible scenario may be that after germination, SFBs swim in the intestine, by sensing some chemoattractant(s), to the distal parts of the small intestine where they colonize. In this regard, it is worth mentioning that SFBs contain only two methyl-accepting chemotaxis proteins (SFBM_0442 is probably the intracellular type). This may suggest that SFBs can sense very limited types of attractants. </p>                        <a id="7754044" scrollto-destination="7754044"></a>
<div id="DSR022F6" data-id="DSR022F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02206.jpeg?Expires=2147483647&Signature=JyryrtWiPhxgfTJNXTU8J3w0cH5g-5z7QOByo1Tz01n3VNoO61oV6jMWfawf7PABFqBo7MSOSn4EO21V1Z3E9nncZiS23UCZjwYKJtiugRVO9WYp~LZlU16LcpkAqOtUXT3VMvWT6bd2~yw1s4mZISDH7oNMeShUM57wk8FSZAs3xvvhKHzJW8rnCRWPrZwdvFSJZU50BFR9vDpVvfKtv4PxuK1s3YmxDEstA1A1ylzSypb2y0ZUepCYWIOTKxurvv~TUN-LLVMh~ov3UbJ3vVhpztMT-9sUPVVQrlCA67wiaspt5tfJrbyYul3UMM9IdMdryAXwzN7G~lhDrWnGmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chemotaxis/flagella biosynthesis gene loci and the activation of the TLR5-linked NF-κB signalling pathway by SFB flagellins. (A) Genetic structures of five SFB genomic loci containing genes for chemotaxis and flagella-mediated motility are shown. Two loci containing genes for methyl-accepting chemotaxis proteins are not included in this figure. See Supplementary Table S4 for detail on each gene. (B) Luciferase activities in the 293T cells that express (black bar) or do not express mouse TLR5 (white bar) after stimulation with the recombinant Salmonella flagellin, recombinant GST fusion proteins of SFB flagellins (FliC1, FliC2, FliC3 or FliC4) or GST are shown. Results are presented as fold-increase relative to the control value." path-from-xml="dsr02206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754044/dsr02206.jpeg" path-from-xml="dsr02206.jpeg" target="_blank">View large</a><a section="7754044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02206.jpeg?Expires=2147483647&Signature=rTlTOwnjNcnDC2hG0iErRVArVogjpu6VcXBhrnIGoTZMp2MT~gsXqIIay54PqnRTWdvxXbv1RAP9MHXNR5SxYoH9LgHMogrjil9KYaWDxRttaTgfwLUY0iIOkn-i8mCDC9uR1esb4WROuePHtH8hjYLjmcNbpgiJDmwwuJUSnG0vSn~PVrhRuM59tN5dyT7dfhbAbToPLQj47n7k598a1-uIvl8ctmn6ex8n5jDHV8-YnWk188JMADUpeDEN-mIrwJ6C~nir4ad8BiQXwR4euGi22PMegezFjj7SmcD7etmVR3-d51zFxXVRXJz1G7W0kCRV6TZt6CUBGxT-8ilLMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754044&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chemotaxis/flagella biosynthesis gene loci and the activation of the TLR5-linked NF-κB signalling pathway by SFB flagellins. (A) Genetic structures of five SFB genomic loci containing genes for chemotaxis and flagella-mediated motility are shown. Two loci containing genes for methyl-accepting chemotaxis proteins are not included in this figure. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for detail on each gene. (B) Luciferase activities in the 293T cells that express (black bar) or do not express mouse TLR5 (white bar) after stimulation with the recombinant <em>Salmonella</em> flagellin, recombinant GST fusion proteins of SFB flagellins (FliC1, FliC2, FliC3 or FliC4) or GST are shown. Results are presented as fold-increase relative to the control value.</p></div></div><div content-id="DSR022F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/m_dsr02206.jpeg?Expires=2147483647&Signature=JyryrtWiPhxgfTJNXTU8J3w0cH5g-5z7QOByo1Tz01n3VNoO61oV6jMWfawf7PABFqBo7MSOSn4EO21V1Z3E9nncZiS23UCZjwYKJtiugRVO9WYp~LZlU16LcpkAqOtUXT3VMvWT6bd2~yw1s4mZISDH7oNMeShUM57wk8FSZAs3xvvhKHzJW8rnCRWPrZwdvFSJZU50BFR9vDpVvfKtv4PxuK1s3YmxDEstA1A1ylzSypb2y0ZUepCYWIOTKxurvv~TUN-LLVMh~ov3UbJ3vVhpztMT-9sUPVVQrlCA67wiaspt5tfJrbyYul3UMM9IdMdryAXwzN7G~lhDrWnGmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chemotaxis/flagella biosynthesis gene loci and the activation of the TLR5-linked NF-κB signalling pathway by SFB flagellins. (A) Genetic structures of five SFB genomic loci containing genes for chemotaxis and flagella-mediated motility are shown. Two loci containing genes for methyl-accepting chemotaxis proteins are not included in this figure. See Supplementary Table S4 for detail on each gene. (B) Luciferase activities in the 293T cells that express (black bar) or do not express mouse TLR5 (white bar) after stimulation with the recombinant Salmonella flagellin, recombinant GST fusion proteins of SFB flagellins (FliC1, FliC2, FliC3 or FliC4) or GST are shown. Results are presented as fold-increase relative to the control value." path-from-xml="dsr02206.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7754044/dsr02206.jpeg" path-from-xml="dsr02206.jpeg" target="_blank">View large</a><a section="7754044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr02206.jpeg?Expires=2147483647&Signature=rTlTOwnjNcnDC2hG0iErRVArVogjpu6VcXBhrnIGoTZMp2MT~gsXqIIay54PqnRTWdvxXbv1RAP9MHXNR5SxYoH9LgHMogrjil9KYaWDxRttaTgfwLUY0iIOkn-i8mCDC9uR1esb4WROuePHtH8hjYLjmcNbpgiJDmwwuJUSnG0vSn~PVrhRuM59tN5dyT7dfhbAbToPLQj47n7k598a1-uIvl8ctmn6ex8n5jDHV8-YnWk188JMADUpeDEN-mIrwJ6C~nir4ad8BiQXwR4euGi22PMegezFjj7SmcD7etmVR3-d51zFxXVRXJz1G7W0kCRV6TZt6CUBGxT-8ilLMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7754044&ar=502681&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02206.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chemotaxis/flagella biosynthesis gene loci and the activation of the TLR5-linked NF-κB signalling pathway by SFB flagellins. (A) Genetic structures of five SFB genomic loci containing genes for chemotaxis and flagella-mediated motility are shown. Two loci containing genes for methyl-accepting chemotaxis proteins are not included in this figure. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> for detail on each gene. (B) Luciferase activities in the 293T cells that express (black bar) or do not express mouse TLR5 (white bar) after stimulation with the recombinant <em>Salmonella</em> flagellin, recombinant GST fusion proteins of SFB flagellins (FliC1, FliC2, FliC3 or FliC4) or GST are shown. Results are presented as fold-increase relative to the control value.</p></div></div>                            <h3 scrollto-destination=7754045 id="7754045" class="section-title" data-legacy-id=s3h>3.8. Multiple flagellins and immunostimulation activity</h3>
<p>A unique feature of the SFB flagella system is the presence of four types of flagellins (FliC1 to FliC4). Of these, FliC3 and FliC4 are encoded in tandem in Locus-1 (Fig. <a reveal-id="DSR022F6" data-open="DSR022F6" class="link link-reveal link-table xref-fig">6</a>A) and show a high sequence similarity (82.3% identity). All contain conserved N- and C-terminal flagellin domains, but FliC1 lacks a variable region between the N- and the C-terminal domains.</p><p>Flagellins are recognized by TLR5 as pathogen-associated molecular patterns (PAMPs) and induce cytokine production to activate the host innate immune system.<sup><a reveal-id="DSR022C39" data-open="DSR022C39" class="link link-ref link-reveal xref-bibr">39</a></sup> TLR5 is highly expressed in LP in the small intestine, wherein dendritic cells are the dominant antigen-presenting cells and only CD11c<sup>hi</sup>CD11b<sup>hi</sup> LPDCs express TLR5.<sup><a reveal-id="DSR022C40  DSR022C42" data-open="DSR022C40  DSR022C42" class="link link-ref link-reveal xref-bibr">40–42</a></sup> These flagellin-exposed LPDCs produce IL-6 and IL-12, which results in the generation of IgA<sup>+</sup> plasma cells and polarization towards the Th17 and Th1 functions.<sup><a reveal-id="DSR022C42" data-open="DSR022C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Therefore, the identified flagellins could be the key molecules responsible for the unique immunostimulation activity of SFBs. To test this presumption, we prepared recombinant proteins for each type of the SFB flagellins and examined whether they could activate the TLR5-linked NF-κB signalling pathway. As shown in Fig. <a reveal-id="DSR022F6" data-open="DSR022F6" class="link link-reveal link-table xref-fig">6</a>B, although FliC1 was unable to activate the pathway, the other three activated it like that achieved by <em>Salmonella</em> flagellin. These results indicate that the immunostimulation activity of SFBs is attributable to the FliC2, FliC3 and FliC4 flagellins, although other molecules, such as cyanophycin and sporulation-related byproducts, may also be involved.</p><p>The tight association of SFBs to Peyer's patches, which could make LPDCs more easily accessible to SFB-derived PAMPs, may also directly or indirectly contribute to innate immunity activation. In this regard, the presence of a fibronectin-binding protein (SFBM_0986) and a phosphatidylinositol-specific phospholipase C (PI-PLC; SFBM_0755) is noteworthy. The elevation of intracellular free calcium levels induced by PI-PLC from various pathogens plays key roles in various processes of microbe–host cell interaction, such as bacterial internalization and host cell actin rearrangement.<sup><a reveal-id="DSR022C43" data-open="DSR022C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSR022C44" data-open="DSR022C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Therefore, SFB PI-PLC may also be involved in the formation of dense aggregates consisting of actin-like microfibrils beneath the host cell membrane where the SFB attached.<sup><a reveal-id="DSR022C13" data-open="DSR022C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Other unidentified factors are also probably required for the intimate binding of SFBs to host epithelial cells, and many of such factors may be encoded by SFB-specific genes, most of which are of unknown function (Fig. <a reveal-id="DSR022F4" data-open="DSR022F4" class="link link-reveal link-table xref-fig">4</a>).</p>                            <h3 scrollto-destination=7754049 id="7754049" class="section-title" data-legacy-id=s3i>3.9. Human SFB-related sequences in the human gut metagenome sequence data</h3>
<p>SFBs have also been found in humans, but SFBs are known to have strict host specificities and the SFB from a different host species may represent a different species.<sup><a reveal-id="DSR022C13" data-open="DSR022C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR022C14" data-open="DSR022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The availability of the mouse SFB genome sequence provided us a first opportunity to search for human SFB-related sequences in the large data set of the human gut metagenome sequences.<sup><a reveal-id="DSR022C11" data-open="DSR022C11" class="link link-ref link-reveal xref-bibr">11</a></sup> By searching 7 345 361 234 high-quality Illumina sequences in the data set, we identified 277 reads that are highly homologous to the mouse SFB sequence (sequence identity; ≥95%, alignment length; ≥95% of the read, excluded reads related to rRNA and tRNA sequences) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Only a small number of reads corresponding to 0.31% of the mouse SFB genome were identified. This may be because non-mapped regions exhibit significant sequence divergence between mouse and human SFB (our search was conservative) or, more likely, because the currently available metagenome sequence data set does not include samples from children at weaning periods. In mice, SFBs become prevalent at this age. Thus, the metagenome sequence data from such children would be required to obtain more genomic sequence information on human SFB.</p>                            <h2 scrollto-destination=7754051 id="7754051" class="section-title" data-legacy-id=s4>4. Conclusion</h2>
<p>The whole-genome sequence of mouse SFBs revealed that this unique gut microbe has a very limited range of metabolic capabilities similar to those of obligate intracellular parasites. In particular, SFBs lack nearly all <em>de novo</em> biosynthesis pathways for amino acids, vitamins and cofactors and nucleotides. These features well explain the non-culturability of SFBs and further suggest that they acquire these compounds, probably from the host intestinal epithelia to which SFBs are tightly attached, via well-conserved transport systems. In addition to a full set of genes for sporulation and germination, SFBs possess a chemotaxis and flagella-based motility system. This finding suggests a triphasic lifestyle of SFBs that comprises swimming, epicellular and dormant (spore) phases. Moreover, SFBs encode four different types of flagellins, and three of these are capable of activating the TLR5-linked NF-κB signalling pathway, which could promote the luminal IgA production and the induction of particular T helper cell subclasses (Th1 and Th17). Thus, SFB genome sequence determination discloses the genetic basis for their non-culturability, which could help us to develop SFB-specific cultivation techniques. In addition, it reveals a previously unknown lifestyle including a flagellated swimming phase, and it provides a first glimpse into understanding the molecular basis for SFB-induced immunostimulation. Furthermore, through a systematic search of the human gut metagenome sequence database, the first pieces of the genomic information on human-associated SFB were obtained.</p>                            <h2 scrollto-destination=7754053 id="7754053" class="section-title" data-legacy-id=s5>Supplemental data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7754055 id="7754055" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by grants-in-aid for Scientific Researches (C) and (B) (to T.K. and T.H., respectively), for Young Scientists (B) (to H.N-I.) and for Scientific Research on Priority Areas ‘Applied Genomics’ (to T.H.) and ‘Comprehensive Genomics’ (to M.H.) from the Ministry of Education, Science and Technology of Japan and by that from the Institute for Bioinformatics Research and Development, the Japan Science and Technology Agency (BIRD-JST) (to K.K.).</p>                            <h2 scrollto-destination=7754057 id="7754057" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Drs K. J. Ishii (Osaka University) and F. Takeshita (Yokohama City University) for providing the TLR5 cDNA and the NF-κB reporter plasmid, respectively. The 16 276-contig sequences used in this study were generated in part with US federal funds from the NIH Human Microbiome Project, the Common Fund, National Institutes of Health and the Department of Health and Human Services.</p>                            <h2 scrollto-destination=7754059 id="7754059" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR022C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Backhed</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Sonnenburg</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Host-bacterial mutualism in the human intestine</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1915</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host-bacterial%20mutualism%20in%20the%20human%20intestine&amp;author=F.%20Backhed&amp;author=R.E.%20Ley&amp;author=J.L.%20Sonnenburg&amp;author=D.A.%20Peterson&amp;author=J.I.%20Gordon&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1915-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1104816" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1104816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1104816"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15790844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hooper</div> <div class="given-names">L.V.</div></div>,  <div class="name"><div class="surname">Midtvedt</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">How host-microbial interactions shape the nutrient environment of the mammalian intestine</div>, <div class="source">Ann. Rev. Nutr.</div> , <div class="year">2002</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">283</div>-<div class="lpage">307</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20host-microbial%20interactions%20shape%20the%20nutrient%20environment%20of%20the%20mammalian%20intestine&amp;author=L.V.%20Hooper&amp;author=T.%20Midtvedt&amp;author=J.I.%20Gordon&amp;publication_year=2002&amp;journal=Ann.%20Rev.%20Nutr.&amp;volume=22&amp;pages=283-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.nutr.22.011602.092259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.nutr.22.011602.092259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.nutr.22.011602.092259"> </span></div> </div></div></div></div></div><div content-id="DSR022C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elson</div> <div class="given-names">C.O.</div></div>,  <div class="name"><div class="surname">Cong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">McCracken</div> <div class="given-names">V.J.</div></div>,  <div class="name"><div class="surname">Dimmitt</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Lorenz</div> <div class="given-names">R.G.</div></div>,  <div class="name"><div class="surname">Weaver</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Experimental models of inflammatory bowel disease reveal innate, adaptive, and regulatory mechanisms of host dialogue with the microbiota</div>, <div class="source">Immunol. Rev.</div> , <div class="year">2005</div>, vol. <div class="volume">206</div> (pg. <div class="fpage">260</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20models%20of%20inflammatory%20bowel%20disease%20reveal%20innate%2C%20adaptive%2C%20and%20regulatory%20mechanisms%20of%20host%20dialogue%20with%20the%20microbiota&amp;author=C.O.%20Elson&amp;author=Y.%20Cong&amp;author=V.J.%20McCracken&amp;author=R.A.%20Dimmitt&amp;author=R.G.%20Lorenz&amp;author=C.T.%20Weaver&amp;publication_year=2005&amp;journal=Immunol.%20Rev.&amp;volume=206&amp;pages=260-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imr.2005.206.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimr.2005.206.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimr.2005.206.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16048554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sartor</div> <div class="given-names">R.B.</div></div>. </div><div class="article-title">Microbial influences in inflammatory bowel diseases</div>, <div class="source">Gastroenterology</div> , <div class="year">2008</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">577</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20influences%20in%20inflammatory%20bowel%20diseases&amp;author=R.B.%20Sartor&amp;publication_year=2008&amp;journal=Gastroenterology&amp;volume=134&amp;pages=577-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1053/j.gastro.2007.11.059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1053%2Fj.gastro.2007.11.059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1053%2Fj.gastro.2007.11.059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18242222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chu</div> <div class="given-names">F.F.</div></div>,  <div class="name"><div class="surname">Esworthy</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">P.G.</div></div>, et al. </div><div class="article-title">Bacteria-induced intestinal cancer in mice with disrupted <em>gpx1</em> and <em>gpx2</em> genes</div>, <div class="source">Cancer Res.</div> , <div class="year">2004</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">962</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteria-induced%20intestinal%20cancer%20in%20mice%20with%20disrupted%20gpx1%20and%20gpx2%20genes&amp;author=F.F.%20Chu&amp;author=R.S.%20Esworthy&amp;author=P.G.%20Chu&amp;publication_year=2004&amp;journal=Cancer%20Res.&amp;volume=64&amp;pages=962-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-03-2272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F0008-5472.CAN-03-2272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F0008-5472.CAN-03-2272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14871826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Turnbauch</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Microbial ecology: human gut microbes associated with obesity</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">444</div> (pg. <div class="fpage">1022</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20ecology%3A%20human%20gut%20microbes%20associated%20with%20obesity&amp;author=R.E.%20Ley&amp;author=P.J.%20Turnbauch&amp;author=S.%20Klein&amp;author=J.I.%20Gordon&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=1022-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/4441022a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F4441022a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F4441022a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Mahowald</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Magrini</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Mardis</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">An obesity-associated gut microbiome with increased capacity for energy harvest</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">444</div> (pg. <div class="fpage">1027</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20obesity-associated%20gut%20microbiome%20with%20increased%20capacity%20for%20energy%20harvest&amp;author=P.J.%20Turnbaugh&amp;author=R.E.%20Ley&amp;author=M.A.%20Mahowald&amp;author=V.%20Magrini&amp;author=E.R.%20Mardis&amp;author=J.I.%20Gordon&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=1027-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gill</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Deboy</div> <div class="given-names">R.T.</div></div>, et al. </div><div class="article-title">Metagenomic analysis of the human distal gut microbiome</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1355</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20of%20the%20human%20distal%20gut%20microbiome&amp;author=S.R.%20Gill&amp;author=M.%20Pop&amp;author=R.T.%20Deboy&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1355-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1124234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1124234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1124234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16741115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">169</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20revealed%20commonly%20enriched%20gene%20sets%20in%20human%20gut%20microbiomes&amp;author=K.%20Kurokawa&amp;author=T.%20Itoh&amp;author=T.%20Kuwahara&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17916580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Castello</div> <div class="given-names">E.K.</div></div>,  <div class="name"><div class="surname">Lauber</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fierer</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Bacterial community variation in human body habitats across space and time</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1694</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20community%20variation%20in%20human%20body%20habitats%20across%20space%20and%20time&amp;author=E.K.%20Castello&amp;author=C.L.%20Lauber&amp;author=M.%20Hamady&amp;author=N.%20Fierer&amp;author=J.I.%20Gordon&amp;author=R.%20Knight&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1694-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1177486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1177486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1177486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19892944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A human microbial gene catalogue established by metagenomic sequencing</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">464</div> (pg. <div class="fpage">59</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20human%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;author=J.%20Qin&amp;author=R.%20Li&amp;author=J.%20Raes&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=59-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marcy</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ouverney</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bik</div> <div class="given-names">E.M.</div></div>, et al. </div><div class="article-title">Dissecting biological ‘dark matter’ with single-cell genetic analysis of rare and uncultivated TM7 microbes from the human mouth</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">11889</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20biological%20%E2%80%98dark%20matter%E2%80%99%20with%20single-cell%20genetic%20analysis%20of%20rare%20and%20uncultivated%20TM7%20microbes%20from%20the%20human%20mouth&amp;author=Y.%20Marcy&amp;author=C.%20Ouverney&amp;author=E.M.%20Bik&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=11889-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0704662104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0704662104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0704662104"> </span></div> </div></div></div></div></div><div content-id="DSR022C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klaasen</div> <div class="given-names">H.L.B.M.</div></div>,  <div class="name"><div class="surname">Koopman</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Poelma</div> <div class="given-names">F.G.J.</div></div>,  <div class="name"><div class="surname">Beynen</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Intestinal, segmented, filamentous bacteria</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">1992</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">165</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intestinal%2C%20segmented%2C%20filamentous%20bacteria&amp;author=H.L.B.M.%20Klaasen&amp;author=J.P.%20Koopman&amp;author=F.G.J.%20Poelma&amp;author=A.C.%20Beynen&amp;publication_year=1992&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=88&amp;pages=165-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1992.88.issue-3-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1992.88.issue-3-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1992.88.issue-3-4"> </span></div> </div></div></div></div></div><div content-id="DSR022C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klaasen</div> <div class="given-names">H.L.B.M.</div></div>,  <div class="name"><div class="surname">Koopman</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Van den Brink</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Bakker</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Poelma</div> <div class="given-names">F.G.</div></div>,  <div class="name"><div class="surname">Beynen</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Intestinal, segmented, filamentous bacteria in a wide range of vertebrate species</div>, <div class="source">Lab. Anim.</div> , <div class="year">1993</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">141</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intestinal%2C%20segmented%2C%20filamentous%20bacteria%20in%20a%20wide%20range%20of%20vertebrate%20species&amp;author=H.L.B.M.%20Klaasen&amp;author=J.P.%20Koopman&amp;author=M.E.%20Van%20den%20Brink&amp;author=M.H.%20Bakker&amp;author=F.G.%20Poelma&amp;author=A.C.%20Beynen&amp;publication_year=1993&amp;journal=Lab.%20Anim.&amp;volume=27&amp;pages=141-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1258/002367793780810441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1258%2F002367793780810441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1258%2F002367793780810441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8501895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Umesaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Matsmoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Imaoka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Setoyama</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Segmented filamentous bacteria are indigenous intestinal bacteria that activate intraepithelial lymphocytes and induce MHC class II molecules and fucosyl asialo GM1 glycolipids on the small intestinal epithelial cells in the ex-germ-free mouse</div>, <div class="source">Microbiol. Immunol.</div> , <div class="year">1995</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">555</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmented%20filamentous%20bacteria%20are%20indigenous%20intestinal%20bacteria%20that%20activate%20intraepithelial%20lymphocytes%20and%20induce%20MHC%20class%20II%20molecules%20and%20fucosyl%20asialo%20GM1%20glycolipids%20on%20the%20small%20intestinal%20epithelial%20cells%20in%20the%20ex-germ-free%20mouse&amp;author=Y.%20Umesaki&amp;author=Y.%20Okada&amp;author=S.%20Matsmoto&amp;author=A.%20Imaoka&amp;author=H.%20Setoyama&amp;publication_year=1995&amp;journal=Microbiol.%20Immunol.&amp;volume=39&amp;pages=555-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mim.1995.39.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmim.1995.39.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmim.1995.39.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7494493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klaasen</div> <div class="given-names">H.L.B.M.</div></div>,  <div class="name"><div class="surname">Van der Heijden</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Stok</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Apathogenic, intestinal, segmented, filamentous bacteria stimulate the mucosal immune system of mice</div>, <div class="source">Infect. Immun.</div> , <div class="year">1993</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">303</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Apathogenic%2C%20intestinal%2C%20segmented%2C%20filamentous%20bacteria%20stimulate%20the%20mucosal%20immune%20system%20of%20mice&amp;author=H.L.B.M.%20Klaasen&amp;author=P.J.%20Van%20der%20Heijden&amp;author=W.%20Stok&amp;publication_year=1993&amp;journal=Infect.%20Immun.&amp;volume=61&amp;pages=303-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8418051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Meek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Doi</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Aberrant expansion of segmented filamentous bacteria in IgA-deficient gut</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">1981</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aberrant%20expansion%20of%20segmented%20filamentous%20bacteria%20in%20IgA-deficient%20gut&amp;author=K.%20Suzuki&amp;author=B.%20Meek&amp;author=Y.%20Doi&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=1981-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307317101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307317101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307317101"> </span></div> </div></div></div></div></div><div content-id="DSR022C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ivanov</div> <div class="given-names">I.I.</div></div>,  <div class="name"><div class="surname">Atarashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Manel</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Induction of intestinal Th17 cells by segmented filamentous bacteria</div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">485</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induction%20of%20intestinal%20Th17%20cells%20by%20segmented%20filamentous%20bacteria&amp;author=I.I.%20Ivanov&amp;author=K.%20Atarashi&amp;author=N.%20Manel&amp;publication_year=2009&amp;journal=Cell&amp;volume=139&amp;pages=485-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.09.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.09.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.09.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19836068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaboriau-Routhiau</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Rakotobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lécuyer</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The key role of segmented filamentous bacteria in the coordinated maturation of gut helper T cell response</div>, <div class="source">Immunity</div> , <div class="year">2009</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">677</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20key%20role%20of%20segmented%20filamentous%20bacteria%20in%20the%20coordinated%20maturation%20of%20gut%20helper%20T%20cell%20response&amp;author=V.%20Gaboriau-Routhiau&amp;author=S.%20Rakotobe&amp;author=E.%20L%C3%A9cuyer&amp;publication_year=2009&amp;journal=Immunity&amp;volume=31&amp;pages=677-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.immuni.2009.08.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.immuni.2009.08.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.immuni.2009.08.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19833089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hase</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohshima</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawano</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Distinct gene expression profiles characterize cellular phenotypes of follicle-associated epithelium and M cells</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">127</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20gene%20expression%20profiles%20characterize%20cellular%20phenotypes%20of%20follicle-associated%20epithelium%20and%20M%20cells&amp;author=K.%20Hase&amp;author=S.%20Ohshima&amp;author=K.%20Kawano&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=127-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morita</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ohshima</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">An improved DNA isolation method for metagenomic analysis of the microbial flora of the human intestine</div>, <div class="source">Microb. Environ.</div> , <div class="year">2007</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">214</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20DNA%20isolation%20method%20for%20metagenomic%20analysis%20of%20the%20microbial%20flora%20of%20the%20human%20intestine&amp;author=H.%20Morita&amp;author=T.%20Kuwahara&amp;author=K.%20Ohshima&amp;publication_year=2007&amp;journal=Microb.%20Environ.&amp;volume=22&amp;pages=214-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.22.214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.22.214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.22.214"> </span></div> </div></div></div></div></div><div content-id="DSR022C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microbial Genome Annotation Pipeline (MiGAP) for diverse users</div>, <div class="year">2009</div><div class="conf-name">The 20th International Conference on Genome Informatics (GIW2009) Poster and Software Demonstrations (Yokohama), S001-1-2</div></div></div></div></div><div content-id="DSR022C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Ozinsky</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The innate immune response to bacterial flagellin is mediated by Toll-like receptor 5</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">410</div> (pg. <div class="fpage">1099</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20innate%20immune%20response%20to%20bacterial%20flagellin%20is%20mediated%20by%20Toll-like%20receptor%205&amp;author=F.%20Hayashi&amp;author=K.D.%20Smith&amp;author=A.%20Ozinsky&amp;publication_year=2001&amp;journal=Nature&amp;volume=410&amp;pages=1099-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35074106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35074106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35074106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11323673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horvath</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Barrangou</div> <div class="given-names">R.</div></div>. </div><div class="article-title">CRISPR/Cas, the immune system of bacteria and archaea</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">167</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%2FCas%2C%20the%20immune%20system%20of%20bacteria%20and%20archaea&amp;author=P.%20Horvath&amp;author=R.%20Barrangou&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=167-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1179555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1179555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1179555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20056882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marraffini</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Sontheimer</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">181</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%20interference%3A%20RNA-directed%20adaptive%20immunity%20in%20bacteria%20and%20archaea&amp;author=L.A.%20Marraffini&amp;author=E.J.%20Sontheimer&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20125085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sebaihia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wren</div> <div class="given-names">B.W.</div></div>,  <div class="name"><div class="surname">Mullany</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The multidrug-resistant human pathogen <em>Clostridium difficile</em> has a highly mobile, mosaic genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">779</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20multidrug-resistant%20human%20pathogen%20Clostridium%20difficile%20has%20a%20highly%20mobile%2C%20mosaic%20genome&amp;author=M.%20Sebaihia&amp;author=B.W.%20Wren&amp;author=P.%20Mullany&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=779-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1830" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1830" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1830"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16804543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Heinen</div> <div class="given-names">P.P.</div></div>,  <div class="name"><div class="surname">Blok</div> <div class="given-names">H.J.</div></div>, et al. </div><div class="article-title">Comparison of 16S rRNA sequences of segmented filamentous bacteria isolated from mice, rats, and chickens and proposal of ‘<em>Candidatus arthromitus</em></div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1995</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">445</div>-<div class="lpage">50</div>)</div></div></div></div><div content-id="DSR022C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ciccarelli</div> <div class="given-names">F.D.</div></div>,  <div class="name"><div class="surname">Doerks</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">von Mering</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Creevey</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Toward automatic reconstruction of a highly resolved tree of life</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">311</div> (pg. <div class="fpage">1283</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20automatic%20reconstruction%20of%20a%20highly%20resolved%20tree%20of%20life&amp;author=F.D.%20Ciccarelli&amp;author=T.%20Doerks&amp;author=C.%20von%20Mering&amp;author=C.J.%20Creevey&amp;author=B.%20Snel&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Science&amp;volume=311&amp;pages=1283-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1123061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1123061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1123061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16513982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bettegowda</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The genome and transcriptomes of the anti-tumor agent <em>Clostridium novyi-</em>NT</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1573</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20and%20transcriptomes%20of%20the%20anti-tumor%20agent%20Clostridium%20novyi-NT&amp;author=C.%20Bettegowda&amp;author=X.%20Huang&amp;author=J.%20Lin&amp;publication_year=2006&amp;journal=Nat.%20Biotechnol.&amp;volume=24&amp;pages=1573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1256"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17115055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Desvaux</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>Clostridium cellulolyticum</em>: model organism of mesophilic cellulolytic clostridia</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2005</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">741</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clostridium%20cellulolyticum%3A%20model%20organism%20of%20mesophilic%20cellulolytic%20clostridia&amp;author=M.%20Desvaux&amp;publication_year=2005&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=29&amp;pages=741-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.femsre.2004.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.femsre.2004.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.femsre.2004.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16102601" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodionov</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Hebbeln</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Eudes</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A novel class of modular transporters for vitamins in prokaryotes</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2009</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">42</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20modular%20transporters%20for%20vitamins%20in%20prokaryotes&amp;author=D.A.%20Rodionov&amp;author=P.%20Hebbeln&amp;author=A.%20Eudes&amp;publication_year=2009&amp;journal=J.%20Bacteriol.&amp;volume=191&amp;pages=42-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01208-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01208-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01208-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piggot</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Hilbert</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Sporulation of <em>Bacillus subtilis</em></div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">579</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sporulation%20of%20Bacillus%20subtilis&amp;author=P.J.%20Piggot&amp;author=D.W.%20Hilbert&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=7&amp;pages=579-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2004.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2004.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2004.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15556029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paredes</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Alsaker</div> <div class="given-names">K.V.</div></div>,  <div class="name"><div class="surname">Papoutsakis</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Comparative genomic view of clostridial sporulation and physiology</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">969</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20view%20of%20clostridial%20sporulation%20and%20physiology&amp;author=C.J.%20Paredes&amp;author=K.V.%20Alsaker&amp;author=T.A.%20Papoutsakis&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;pages=969-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16261177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moir</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Corfe</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Behravan</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Spore germination</div>, <div class="source">Cell Mol. Life Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">403</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spore%20germination&amp;author=A.%20Moir&amp;author=B.M.%20Corfe&amp;author=J.%20Behravan&amp;publication_year=2002&amp;journal=Cell%20Mol.%20Life%20Sci.&amp;volume=59&amp;pages=403-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-002-8432-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-002-8432-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-002-8432-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11964118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allen</div> <div class="given-names">M.M.</div></div>. </div><div class="article-title">Cyanobacterial cell inclusions</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">1984</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cyanobacterial%20cell%20inclusions&amp;author=M.M.%20Allen&amp;publication_year=1984&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=38&amp;pages=1-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.mi.38.100184.000245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.mi.38.100184.000245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.mi.38.100184.000245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6437321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fuser</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Steinbuchel</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Analysis of genome sequences for genes of cyanophycin metabolism: identifying putative cyanophycin metabolizing prokaryotes</div>, <div class="source">Macromol. Biosci.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">278</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20genome%20sequences%20for%20genes%20of%20cyanophycin%20metabolism%3A%20identifying%20putative%20cyanophycin%20metabolizing%20prokaryotes&amp;author=G.%20Fuser&amp;author=A.%20Steinbuchel&amp;publication_year=2007&amp;journal=Macromol.%20Biosci.&amp;volume=7&amp;pages=278-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1616-5195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1616-5195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1616-5195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17390395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyerholz</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Stabel</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Cheville</div> <div class="given-names">N.F.</div></div>. </div><div class="article-title">Segmented filamentous bacteria interact with intraepithelial mononuclear cells</div>, <div class="source">Infect. Immun.</div> , <div class="year">2002</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">3277</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmented%20filamentous%20bacteria%20interact%20with%20intraepithelial%20mononuclear%20cells&amp;author=D.K.%20Meyerholz&amp;author=T.J.%20Stabel&amp;author=N.F.%20Cheville&amp;publication_year=2002&amp;journal=Infect.%20Immun.&amp;volume=70&amp;pages=3277-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.70.6.3277-3280.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.70.6.3277-3280.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.70.6.3277-3280.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamauchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Snel</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Transmission electron microscopic demonstration of phagocytosis and intracellular processing of segmented filamentous bacteria by intestinal epithelial cells of the chick ileum</div>, <div class="source">Infect. Immun.</div> , <div class="year">2000</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">6496</div>-<div class="lpage">504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transmission%20electron%20microscopic%20demonstration%20of%20phagocytosis%20and%20intracellular%20processing%20of%20segmented%20filamentous%20bacteria%20by%20intestinal%20epithelial%20cells%20of%20the%20chick%20ileum&amp;author=K.%20Yamauchi&amp;author=J.%20Snel&amp;publication_year=2000&amp;journal=Infect.%20Immun.&amp;volume=68&amp;pages=6496-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.68.11.6496-6504.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.68.11.6496-6504.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.68.11.6496-6504.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11035767" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Andersen-Nissen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Toll-like receptor 5 recognizes a conserved site on flagellin required for protofilament formation and bacterial motility</div>, <div class="source">Nat. Immunol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1247</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toll-like%20receptor%205%20recognizes%20a%20conserved%20site%20on%20flagellin%20required%20for%20protofilament%20formation%20and%20bacterial%20motility&amp;author=K.D.%20Smith&amp;author=E.%20Andersen-Nissen&amp;author=F.%20Hayashi&amp;publication_year=2003&amp;journal=Nat.%20Immunol.&amp;volume=4&amp;pages=1247-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ni1011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fni1011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fni1011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14625549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uematsu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jang</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Chevrier</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Detection of pathogenic intestinal bacteria by Toll-like receptor 5 on intestinal CD11c<sup>+</sup> lamina propria cells</div>, <div class="source">Nat. Immunol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">868</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20pathogenic%20intestinal%20bacteria%20by%20Toll-like%20receptor%205%20on%20intestinal%20CD11c%2B%20lamina%20propria%20cells&amp;author=S.%20Uematsu&amp;author=M.H.%20Jang&amp;author=N.%20Chevrier&amp;publication_year=2006&amp;journal=Nat.%20Immunol.&amp;volume=7&amp;pages=868-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ni1362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fni1362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fni1362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16829963" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uematsu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fujimoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jang</div> <div class="given-names">M.H.</div></div>, et al. </div><div class="article-title">Regulation of humoral and cellular gut immunity by lamina propria dendritic cells expressing Toll-like receptor 5</div>, <div class="source">Nat. Immunol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">769</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20humoral%20and%20cellular%20gut%20immunity%20by%20lamina%20propria%20dendritic%20cells%20expressing%20Toll-like%20receptor%205&amp;author=S.%20Uematsu&amp;author=K.%20Fujimoto&amp;author=M.H.%20Jang&amp;publication_year=2008&amp;journal=Nat.%20Immunol.&amp;volume=9&amp;pages=769-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ni.1622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fni.1622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fni.1622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uematsu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Akira</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Immune responses of TLR5<sup>+</sup> lamina propria dendritic cells in enterobacterial infection</div>, <div class="source">J. Gastroenterol.</div> , <div class="year">2009</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">803</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immune%20responses%20of%20TLR5%2B%20lamina%20propria%20dendritic%20cells%20in%20enterobacterial%20infection&amp;author=S.%20Uematsu&amp;author=S.%20Akira&amp;publication_year=2009&amp;journal=J.%20Gastroenterol.&amp;volume=44&amp;pages=803-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00535-009-0094-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00535-009-0094-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00535-009-0094-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19547909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wadsworth</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Goldfine</div> <div class="given-names">H.</div></div>. </div><div class="article-title"><em>Listeria monocytogenes</em> phospholipase C-dependent calcium signaling modulates bacterial entry into J774 macrophage-like cells</div>, <div class="source">Infect. Immun.</div> , <div class="year">1999</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">1770</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Listeria%20monocytogenes%20phospholipase%20C-dependent%20calcium%20signaling%20modulates%20bacterial%20entry%20into%20J774%20macrophage-like%20cells&amp;author=S.J.%20Wadsworth&amp;author=H.%20Goldfine&amp;publication_year=1999&amp;journal=Infect.%20Immun.&amp;volume=67&amp;pages=1770-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10085017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR022C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR022C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ginocchio</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pace</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Galan</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Identification and molecular characterization of <em>Salmonella typhimurium</em> gene involved in triggering the internalization of salmonellae into cultured epithelial cells</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">5976</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20molecular%20characterization%20of%20Salmonella%20typhimurium%20gene%20involved%20in%20triggering%20the%20internalization%20of%20salmonellae%20into%20cultured%20epithelial%20cells&amp;author=C.%20Ginocchio&amp;author=J.%20Pace&amp;author=J.E.%20Galan&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=5976-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.13.5976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.13.5976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.13.5976"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=502681&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr022-dsr022supp_tables.pdf?Expires=2147483647&amp;Signature=0o~Zv91o7Ib5FkpA8sZZYfCk0XBIz4gLp5-Lnx29sQBoLwCehNqEEHHk91cIXNF9dBtj4WT-LO7pwKxlb9QX9Fm-Nz4X0JmOFkN~VEbz1iT1cpbyi7O24IyYOQb90EMrjbNxXezwer3kclLr0u5zB-Qpp6B8gHjvQWCSGar9g3mRxYTWrzccDxerQhb8y9L4mKsZYn9kimkGadBYwX8QZSAlT8DR7cMiR3fUvz-V6AeUNa9NllKzB-TLukkpuIkQJmvvDs0TDj0pBur3y7K6keqF3ni~5AcrV6m78R6ecyRNqqGAqVYdp1sBD7yWH2RKHZBM5kvNT7exj5c3xTq~uw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr022/2/dsr022-dsr022supp_figs.pdf?Expires=2147483647&amp;Signature=K3cE6bBRHC6daxBjFpWofUY7DU8p2kLSnz~zRsdh3C7dZqTIgKSxleR9Ux-2eS~5Ew2eLbQHHzJF5a-HfxG-urWxdTjgX0gYOuIGpVcaMehj8RYG2R7O5aHwPiKprqcU7FXFReGlNR~mn6whh6ITBcUbJQ7nUB4qXDxbW8GqTIQiDeX0nMIPKCGsSsLSgl8U96wVPiBNxtPGi9KVpNe-TePfvBM-Jod5mrYblM8Co9-Vh0qr-mm3hJFvmC4mzFD1blDcf7UBWa7JqeJ32M6vY-mLt~kIyUiopRamVpwn-cu6rJ5S4~7Z7TI1VLvc0Kb9E2IBsReYkEyQBYaGv4OQWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">331</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">53</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="502681"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f4%2f291%2f502681">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000294936100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (53)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F4%2F291%2F502681" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/502681'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Lifestyle of the Segmented Filamentous Bacterium: A Non-Culturable Gut-Associated Immunostimulating Microbe Inferred by Whole-Genome Sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f291%2f502681">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

