

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/4/325/353411","@type":"ScholarlyArticle","name":"High-Resolution Mapping of  In vivo  Genomic Transcription Factor Binding Sites Using  In situ  DNase I Footprinting and ChIP-seq","datePublished":"2013-04-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2013-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst013","keywords":["GeF-seq","ChIP-seq","AbrB","<italic>Bacillus subtilis</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/4/325/353411","author":[{"name":"Chumsakul, Onuma","affiliation":"1  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Nakamura, Kensuke","affiliation":"2  Department of Life Science and Informatics ,  Maebashi Institute of Technology ,  460-1, Kamisadori, Maebashi-City, Gunma ,  Japan","@type":"Person"},{"name":"Kurata, Tetsuya","affiliation":"3  Plant Global Education Project, Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Sakamoto, Tomoaki","affiliation":"3  Plant Global Education Project, Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Hobman, Jon L.","affiliation":"4  School of Biosciences ,  The University of Nottingham ,  Sutton Bonington Campus, Sutton Bonington, Loughborough, Leicestershire LE12 5RD ,  UK","@type":"Person"},{"name":"Ogasawara, Naotake","affiliation":"1  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Oshima, Taku","affiliation":"1  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Ishikawa, Shu","affiliation":"1  Graduate School of Biological Sciences ,  Nara Institute of Science and Technology ,  8916-5, Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"}],"description":"Abstract.  Accurate identification of the DNA-binding sites of transcription factors and other DNA-binding proteins on the genome is crucial to understanding th","pageStart":"325","pageEnd":"338","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-Resolution Mapping of  In vivo  Genomic Transcription Factor Binding Sites Using  In situ  DNase I Footprinting and ChIP-seq","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-Resolution Mapping of  In vivo  Genomic Transcription Factor Binding Sites Using  In situ  DNase I Footprinting and ChIP-seq" />
    <meta property="og:description" content="Abstract.  Accurate identification of the DNA-binding sites of transcription factors and other DNA-binding proteins on the genome is crucial to understanding th" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/4/325/353411" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-04-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chumsakul, Onuma" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Nakamura, Kensuke" /><meta name="citation_author_institution" content="Department of Life Science and Informatics, Maebashi Institute of Technology, 460-1, Kamisadori, Maebashi-City, Gunma, Japan" /><meta name="citation_author" content="Kurata, Tetsuya" /><meta name="citation_author_institution" content="Plant Global Education Project, Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Sakamoto, Tomoaki" /><meta name="citation_author_institution" content="Plant Global Education Project, Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Hobman, Jon L." /><meta name="citation_author_institution" content="School of Biosciences, The University of Nottingham, Sutton Bonington Campus, Sutton Bonington, Loughborough, Leicestershire LE12 5RD, UK" /><meta name="citation_author" content="Ogasawara, Naotake" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Oshima, Taku" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Ishikawa, Shu" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_title" content="High-Resolution Mapping of In vivo Genomic Transcription Factor Binding Sites Using In situ DNase I Footprinting and ChIP-seq" /><meta name="citation_firstpage" content="325" /><meta name="citation_lastpage" content="338" /><meta name="citation_doi" content="10.1093/dnares/dst013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2013/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of protein-DNA interactions; Annu. Rev. Genomics Hum. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=81-102; " /><meta name="citation_reference" content="citation_title=High-resolution profiling of histone methylations in the human genome; Cell;  citation_year=2007;  citation_volume=129;  citation_pages=823-37; " /><meta name="citation_reference" content="citation_title=Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=651-7; " /><meta name="citation_reference" content="citation_title=Genome-wide maps of chromatin state in pluripotent and lineage-committed cells; Nature;  citation_year=2007;  citation_volume=448;  citation_pages=553-60; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of in vivo protein-DNA interactions; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1497-502; " /><meta name="citation_reference" content="citation_title=Computation for ChIP-seq and RNA-seq studies; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=S22-32; " /><meta name="citation_reference" content="citation_title=Genome-wide identification of in vivo protein-DNA binding sites from ChIP-Seq data; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=5221-31; " /><meta name="citation_reference" content="citation_title=Design and analysis of ChIP-seq experiments for DNA-binding proteins; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1351-9; " /><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R137" /><meta name="citation_reference" content="citation_title=Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution; Cell;  citation_year=2011;  citation_volume=147;  citation_pages=1408-19; " /><meta name="citation_reference" content="citation_title=Footprinting: a method for determining the sequence selectivity, affinity and kinetics of DNA-binding ligands; Methods;  citation_year=2007;  citation_volume=42;  citation_pages=128-40; " /><meta name="citation_reference" content="citation_title=Protein occupancy landscape of a bacterial genome; Mol. Cell.;  citation_year=2009;  citation_volume=35;  citation_pages=247-53; " /><meta name="citation_reference" content="citation_title=Distribution of stable DnaA-binding sites on the Bacillus subtilis genome detected using a modified ChIP-chip method; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=155-68; " /><meta name="citation_reference" content="citation_title=Genome-wide binding profiles of the Bacillus subtilis transition state regulator AbrB and its homolog Abh reveals their interactive role in transcriptional regulation; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=414-28; " /><meta name="citation_reference" content="citation_title=Insights into the nature of DNA binding of AbrB-like transcription factors; Structure;  citation_year=2008;  citation_volume=16;  citation_pages=1702-13; " /><meta name="citation_reference" content="citation_title=Delineation of AbrB-binding sites on the Bacillus subtilis spo0H, kinB, ftsAZ, and pbpE promoters and use of a derived homology to identify a previously unsuspected binding site in the bsuB1 methylase promote; J. Bacteriol.;  citation_year=1995;  citation_volume=177;  citation_pages=6999-7002; " /><meta name="citation_reference" content="citation_title=In vitro selection of optimal AbrB-binding sites: comparison to known in vivo sites indicates flexibility in AbrB binding and recognition of three-dimensional DNA structures; Mol. Microbiol.;  citation_year=1996;  citation_volume=19;  citation_pages=145-58; " /><meta name="citation_reference" content="citation_title=Abh and AbrB control of Bacillus subtilis antimicrobial gene expression; J. Bacteriol.;  citation_year=2007;  citation_volume=189;  citation_pages=7720-32; " /><meta name="citation_reference" content="citation_title=BIPAD: a web server for modeling bipartite sequence elements; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=76" /><meta name="citation_reference" content="citation_title=Isolation and characterization of cell wall-defective variants of Bacillus subtilis and Bacillus licheniformis; J. Bacteriol.;  citation_year=1973;  citation_volume=116;  citation_pages=456-65; " /><meta name="citation_reference" content="citation_title=Sequence-specific error profile of Illumina sequencers; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e90" /><meta name="citation_reference" content="citation_title=Regulatory sequence analysis tools; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3593-6; " /><meta name="citation_reference" content="citation_title=DBTBS: a database of transcriptional regulation in Bacillus subtilis containing upstream intergenic conservation information; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D93-96; " /><meta name="citation_reference" content="citation_title=SigmaX is involved in controlling Bacillus subtilis biofilm architecture through the AbrB homologue Abh; J. Bacteriol.;  citation_year=2009;  citation_volume=191;  citation_pages=6822-32; " /><meta name="citation_reference" content="citation_title=The lac repressor displays facilitated diffusion in living cells; Science;  citation_year=2012;  citation_volume=336;  citation_pages=1595-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/4/325/1010561/dst013.pdf" /><meta name="description" content="Abstract.  Accurate identification of the DNA-binding sites of transcription factors and other DNA-binding proteins on the genome is crucial to understanding th" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/4/325/353411" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/4/325/353411" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/4/325/353411");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst013';
    </script>
<title>
	High-Resolution Mapping of In vivo Genomic Transcription Factor Binding Sites Using In situ DNase I Footprinting and ChIP-seq | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353411" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf1EPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaHVtc2FrdWwsIE9udW1hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSwgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FtdXJhLCBLZW5zdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2UgYW5kIEluZm9ybWF0aWNzLCBNYWViYXNoaSBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgNDYwLTEsIEthbWlzYWRvcmksIE1hZWJhc2hpLUNpdHksIEd1bm1hLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1cmF0YSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBsYW50IEdsb2JhbCBFZHVjYXRpb24gUHJvamVjdCwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSwgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNha2Ftb3RvLCBUb21vYWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgR2xvYmFsIEVkdWNhdGlvbiBQcm9qZWN0LCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01LCBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9ibWFuLCBKb24gTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgQmlvc2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIE5vdHRpbmdoYW0sIFN1dHRvbiBCb25pbmd0b24gQ2FtcHVzLCBTdXR0b24gQm9uaW5ndG9uLCBMb3VnaGJvcm91Z2gsIExlaWNlc3RlcnNoaXJlIExFMTIgNVJELCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9nYXNhd2FyYSwgTmFvdGFrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUsIFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIFRha3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01LCBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNoaWthd2EsIFNodSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUsIFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtUmVzb2x1dGlvbiBNYXBwaW5nIG9mIEluIHZpdm8gR2Vub21pYyBUcmFuc2NyaXB0aW9uIEZhY3RvciBCaW5kaW5nIFNpdGVzIFVzaW5nIEluIHNpdHUgRE5hc2UgSSBGb290cHJpbnRpbmcgYW5kIENoSVAtc2VxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHByb3RlaW4tRE5BIGludGVyYWN0aW9uczsgQW5udS4gUmV2LiBHZW5vbWljcyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODEtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gcHJvZmlsaW5nIG9mIGhpc3RvbmUgbWV0aHlsYXRpb25zIGluIHRoZSBodW1hbiBnZW5vbWU7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI5OyAgY2l0YXRpb25fcGFnZXM9ODIzLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBwcm9maWxlcyBvZiBTVEFUMSBETkEgYXNzb2NpYXRpb24gdXNpbmcgY2hyb21hdGluIGltbXVub3ByZWNpcGl0YXRpb24gYW5kIG1hc3NpdmVseSBwYXJhbGxlbCBzZXF1ZW5jaW5nOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTY1MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtYXBzIG9mIGNocm9tYXRpbiBzdGF0ZSBpbiBwbHVyaXBvdGVudCBhbmQgbGluZWFnZS1jb21taXR0ZWQgY2VsbHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDg7ICBjaXRhdGlvbl9wYWdlcz01NTMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIG1hcHBpbmcgb2YgaW4gdml2byBwcm90ZWluLUROQSBpbnRlcmFjdGlvbnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE2OyAgY2l0YXRpb25fcGFnZXM9MTQ5Ny01MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uIGZvciBDaElQLXNlcSBhbmQgUk5BLXNlcSBzdHVkaWVzOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPVMyMi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaWRlbnRpZmljYXRpb24gb2YgaW4gdml2byBwcm90ZWluLUROQSBiaW5kaW5nIHNpdGVzIGZyb20gQ2hJUC1TZXEgZGF0YTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NTIyMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVzaWduIGFuZCBhbmFseXNpcyBvZiBDaElQLXNlcSBleHBlcmltZW50cyBmb3IgRE5BLWJpbmRpbmcgcHJvdGVpbnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMzUxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGVsLWJhc2VkIGFuYWx5c2lzIG9mIENoSVAtU2VxIChNQUNTKTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SMTM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBnZW5vbWUtd2lkZSBwcm90ZWluLUROQSBpbnRlcmFjdGlvbnMgZGV0ZWN0ZWQgYXQgc2luZ2xlLW51Y2xlb3RpZGUgcmVzb2x1dGlvbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xNDA4LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb290cHJpbnRpbmc6IGEgbWV0aG9kIGZvciBkZXRlcm1pbmluZyB0aGUgc2VxdWVuY2Ugc2VsZWN0aXZpdHksIGFmZmluaXR5IGFuZCBraW5ldGljcyBvZiBETkEtYmluZGluZyBsaWdhbmRzOyBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTI4LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIG9jY3VwYW5jeSBsYW5kc2NhcGUgb2YgYSBiYWN0ZXJpYWwgZ2Vub21lOyBNb2wuIENlbGwuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MjQ3LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24gb2Ygc3RhYmxlIERuYUEtYmluZGluZyBzaXRlcyBvbiB0aGUgQmFjaWxsdXMgc3VidGlsaXMgZ2Vub21lIGRldGVjdGVkIHVzaW5nIGEgbW9kaWZpZWQgQ2hJUC1jaGlwIG1ldGhvZDsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGJpbmRpbmcgcHJvZmlsZXMgb2YgdGhlIEJhY2lsbHVzIHN1YnRpbGlzIHRyYW5zaXRpb24gc3RhdGUgcmVndWxhdG9yIEFickIgYW5kIGl0cyBob21vbG9nIEFiaCByZXZlYWxzIHRoZWlyIGludGVyYWN0aXZlIHJvbGUgaW4gdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTQxNC0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byB0aGUgbmF0dXJlIG9mIEROQSBiaW5kaW5nIG9mIEFickItbGlrZSB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7IFN0cnVjdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE3MDItMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlbGluZWF0aW9uIG9mIEFickItYmluZGluZyBzaXRlcyBvbiB0aGUgQmFjaWxsdXMgc3VidGlsaXMgc3BvMEgsIGtpbkIsIGZ0c0FaLCBhbmQgcGJwRSBwcm9tb3RlcnMgYW5kIHVzZSBvZiBhIGRlcml2ZWQgaG9tb2xvZ3kgdG8gaWRlbnRpZnkgYSBwcmV2aW91c2x5IHVuc3VzcGVjdGVkIGJpbmRpbmcgc2l0ZSBpbiB0aGUgYnN1QjEgbWV0aHlsYXNlIHByb21vdGU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9Njk5OS03MDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXRybyBzZWxlY3Rpb24gb2Ygb3B0aW1hbCBBYnJCLWJpbmRpbmcgc2l0ZXM6IGNvbXBhcmlzb24gdG8ga25vd24gaW4gdml2byBzaXRlcyBpbmRpY2F0ZXMgZmxleGliaWxpdHkgaW4gQWJyQiBiaW5kaW5nIGFuZCByZWNvZ25pdGlvbiBvZiB0aHJlZS1kaW1lbnNpb25hbCBETkEgc3RydWN0dXJlczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTQ1LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYmggYW5kIEFickIgY29udHJvbCBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBhbnRpbWljcm9iaWFsIGdlbmUgZXhwcmVzc2lvbjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODk7ICBjaXRhdGlvbl9wYWdlcz03NzIwLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CSVBBRDogYSB3ZWIgc2VydmVyIGZvciBtb2RlbGluZyBiaXBhcnRpdGUgc2VxdWVuY2UgZWxlbWVudHM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgY2VsbCB3YWxsLWRlZmVjdGl2ZSB2YXJpYW50cyBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBhbmQgQmFjaWxsdXMgbGljaGVuaWZvcm1pczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk3MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz00NTYtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlLXNwZWNpZmljIGVycm9yIHByb2ZpbGUgb2YgSWxsdW1pbmEgc2VxdWVuY2VyczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9ZTkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdG9yeSBzZXF1ZW5jZSBhbmFseXNpcyB0b29sczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzU5My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EQlRCUzogYSBkYXRhYmFzZSBvZiB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbiBpbiBCYWNpbGx1cyBzdWJ0aWxpcyBjb250YWluaW5nIHVwc3RyZWFtIGludGVyZ2VuaWMgY29uc2VydmF0aW9uIGluZm9ybWF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1EOTMtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ21hWCBpcyBpbnZvbHZlZCBpbiBjb250cm9sbGluZyBCYWNpbGx1cyBzdWJ0aWxpcyBiaW9maWxtIGFyY2hpdGVjdHVyZSB0aHJvdWdoIHRoZSBBYnJCIGhvbW9sb2d1ZSBBYmg7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTkxOyAgY2l0YXRpb25fcGFnZXM9NjgyMi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGxhYyByZXByZXNzb3IgZGlzcGxheXMgZmFjaWxpdGF0ZWQgZGlmZnVzaW9uIGluIGxpdmluZyBjZWxsczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzY7ICBjaXRhdGlvbl9wYWdlcz0xNTk1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC80LzMyNS8xMDEwNTYxL2RzdDAxMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWNjdXJhdGUgaWRlbnRpZmljYXRpb24gb2YgdGhlIEROQS1iaW5kaW5nIHNpdGVzIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBhbmQgb3RoZXIgRE5BLWJpbmRpbmcgcHJvdGVpbnMgb24gdGhlIGdlbm9tZSBpcyBjcnVjaWFsIHRvIHVuZGVyc3RhbmRpbmcgdGgiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1MzQxMSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQyr3ISx+ZDeZGjUEsGWWV67pxm7Q==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f325%2f353411" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f325%2f353411" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13681" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/4/1/m_cover.gif?Expires=2147483647&amp;Signature=kDpSlJOATic2pdO69CWwGnx~EeVVupFOwOpjIaJNVT0SJS-ZIZRvENqW~Qcw~svxb1i19QQ5Ifn2XSwMVW3omPsYiWl2q5mpuh3ULHTOxJVMCnbgaueMmdt4m-QrXpozwSwG-xUXXoU8lSjVZAti7YN4euxw8MoIiPyepZp~m4HTtQb6hmGXoGj96UA6VylA3Vl~xHlps4hDe08v5OVYpSfLGteJJjWrY56XnToeSfJskVz1r9Ox3e8MCbHm1BL2tdKZcbxHa2lRgFr-reVp8DuAQ~DCkYevKlLmt2eq5yLULh3MjpkvpbZwLWeQqvi-W-TjOzKoTuoM7uKiU9jbEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/4">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5028439">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028439">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028441">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028446">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028446">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028475">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028475">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028495">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028495">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028501">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028503">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028503">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028505">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5028507">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5028507">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/4/315/353210">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/4/339/353499">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-Resolution Mapping of <em>In vivo</em> Genomic Transcription Factor Binding Sites Using <em>In situ</em> DNase I Footprinting and ChIP-seq
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Onuma Chumsakul</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Onuma+Chumsakul">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chumsakul O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chumsakul O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Onuma Chumsakul</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kensuke Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Life Science and Informatics</div>, <div class="insititution">Maebashi Institute of Technology</div>, <div class="addr-line">460-1, Kamisadori, Maebashi-City, Gunma</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kensuke+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kensuke Nakamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Kurata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Plant Global Education Project, Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Kurata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurata T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurata T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Kurata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoaki Sakamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Plant Global Education Project, Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoaki+Sakamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakamoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakamoto T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomoaki Sakamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jon L. Hobman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">School of Biosciences</div>, <div class="insititution">The University of Nottingham</div>, <div class="addr-line">Sutton Bonington Campus, Sutton Bonington, Loughborough, Leicestershire LE12 5RD</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jon+L.+Hobman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hobman J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hobman J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jon L. Hobman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naotake Ogasawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naotake+Ogasawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogasawara N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogasawara N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naotake Ogasawara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taku Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-743-72-5431 (S.I. and T.O.). Fax. +81-743-72-5439 (S.I. and T.O.). Email: <a href="mailto:shu@bs.naist.jp" target="_blank">shu@bs.naist.jp</a> (S.I.); <a href="mailto:taku@bs.naist.jp" target="_blank">taku@bs.naist.jp</a> (T.O.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taku+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taku Oshima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shu Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biological Sciences</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-743-72-5431 (S.I. and T.O.). Fax. +81-743-72-5439 (S.I. and T.O.). Email: <a href="mailto:shu@bs.naist.jp" target="_blank">shu@bs.naist.jp</a> (S.I.); <a href="mailto:taku@bs.naist.jp" target="_blank">taku@bs.naist.jp</a> (T.O.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shu+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shu Ishikawa</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 325–338, <a href='https://doi.org/10.1093/dnares/dst013'>https://doi.org/10.1093/dnares/dst013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 April 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 February 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 March 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353411" href="/dnaresearch/article-pdf/20/4/325/1010561/dst013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Onuma Chumsakul, Kensuke Nakamura, Tetsuya Kurata, Tomoaki Sakamoto, Jon L. Hobman, Naotake Ogasawara, Taku Oshima, Shu Ishikawa;  High-Resolution Mapping of <em>In vivo</em> Genomic Transcription Factor Binding Sites Using <em>In situ</em> DNase I Footprinting and ChIP-seq, <em>DNA Research</em>, Volume 20, Issue 4, 1 August 2013, Pages 325–338, <a href='https://doi.org/10.1093/dnares/dst013'>https://doi.org/10.1093/dnares/dst013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353411?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353411?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-Resolution%20Mapping%20of%20%26lt%3Bem%26gt%3BIn%20vivo%26lt%3B%2Fem%26gt%3B%20Genomic%20Transcription%20Factor%20Binding%20Sites%20Using%20%26lt%3Bem%26gt%3BIn%20situ%26lt%3B%2Fem%26gt%3B%20DNase%20I%20Footprinting%20and%20ChIP-seq&amp;publicationDate=2013-04-11&amp;volumeNum=20&amp;issueNum=4&amp;author=Chumsakul%2C%20Onuma%3B%20Nakamura%2C%20Kensuke&amp;startPage=325&amp;endPage=338&amp;contentId=10.1093%2Fdnares%2Fdst013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f325%2f353411" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5028439 id="5028439" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Accurate identification of the DNA-binding sites of transcription factors and other DNA-binding proteins on the genome is crucial to understanding their molecular interactions with DNA. Here, we describe a new method: Genome Footprinting by high-throughput sequencing (GeF-seq), which combines <em>in vivo</em> DNase I digestion of genomic DNA with ChIP coupled with high-throughput sequencing. We have determined the <em>in vivo</em> binding sites of a <em>Bacillus subtilis</em> global regulator, AbrB, using GeF-seq. This method shows that exact DNA-binding sequences, which were protected from <em>in vivo</em> DNase I digestion, were resolved at a comparable resolution to that achieved by <em>in vitro</em> DNase I footprinting, and this was simply attained without the necessity of prediction by peak-calling programs. Moreover, DNase I digestion of the bacterial nucleoid resolved the closely positioned AbrB-binding sites, which had previously appeared as one peak in ChAP-chip and ChAP-seq experiments. The high-resolution determination of AbrB-binding sites using GeF-seq enabled us to identify bipartite TGGNA motifs in 96% of the AbrB-binding sites. Interestingly, in a thousand binding sites with very low-binding intensities, single TGGNA motifs were also identified. Thus, GeF-seq is a powerful method to elucidate the molecular mechanism of target protein binding to its cognate DNA sequences.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="GeF-seq">GeF-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ChIP-seq">ChIP-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="AbrB">AbrB</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Bacillus subtilis&quot;"><em>Bacillus subtilis</em></a></div>                            <h2 scrollto-destination=5028441 id="5028441" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genome-wide mapping of the <em>in vivo</em> DNA-binding sites of transcription factors or other DNA-binding proteins either by Chromatin Immunoprecipitation coupled with microarray (ChIP-chip)<sup><a reveal-id="DST013C1" data-open="DST013C1" class="link link-ref link-reveal xref-bibr">1</a></sup> or by the recently developed ChIP coupled with high-throughput sequencing (ChIP-seq) method have become widely used techniques in protein–DNA interaction research.<sup><a reveal-id="DST013C2  DST013C5" data-open="DST013C2  DST013C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> The resolution of the DNA-binding sites determined by ChIP-seq was a dramatic improvement on the resolution that was possible using ChIP-chip, because of the higher resolution of high-throughput sequencing compared with oligonucleotide arrays. However, for both techniques, the DNA fragments, co-purified with the target protein (ChIP-DNA), are generated by sonication and generally fall within the size range of 100–500 bp. These sonicated fragments are often much longer than the actual protein-binding site and, thus, the sequence tags of the ChIP-DNA distribute in broad regions around the actual binding sites. In addition, as only the terminal sequences of ChIP-DNA fragments can be obtained by high-throughput sequencing, piled ChIP-seq tags on the forward (+) and reverse (−) strands usually show bimodal peaks.<sup><a reveal-id="DST013C6" data-open="DST013C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST013C7" data-open="DST013C7" class="link link-ref link-reveal xref-bibr">7</a></sup> To overcome these problems and determine the actual protein-binding sites to within a few 10 bp, algorithms for the processing of ChIP-seq data have been proposed, although the results obtained by them are still predictive.<sup><a reveal-id="DST013C6  DST013C9" data-open="DST013C6  DST013C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> Thus, more precise experimental mapping methods are required to determine the exact binding sites of DNA-binding proteins using ChIP-seq technology.</p><p>Recently, the ChIP-exo method, which trims the 5′-region of the protein-unbound region of ChIP-DNA by the use of 5′–3′ lambda (<em>λ</em>) exonuclease, has been developed, and this method demonstrated an improvement in resolution in determining the DNA-binding sites of target eukaryotic proteins through the determination of the edge positions of protein-bound genomic sequences.<sup><a reveal-id="DST013C10" data-open="DST013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In contrast to DNA exonucleases, DNase I preferentially cleaves endogenous DNA regions that are not protected by bound proteins and, thus, has been employed for <em>in vitro</em> footprinting to precisely determine the DNA-binding sites of DNA-binding proteins.<sup><a reveal-id="DST013C11" data-open="DST013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Using DNase I digestion, Vora <em>et al</em>.<sup><a reveal-id="DST013C12" data-open="DST013C12" class="link link-ref link-reveal xref-bibr">12</a></sup> proposed a method, designated <em>in vivo</em> protein occupancy display (IPOD), which visualizes the <em>in vivo</em> binding profile of total DNA-binding proteins on genomic DNA.<sup><a reveal-id="DST013C12" data-open="DST013C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In this method, genomic DNA cross-linked with total proteins and extracted from formaldehyde-treated cells was digested with DNase I, and the DNase I-resistant DNA fragments were purified by phenol extraction and mapped using a tiling array.</p><p>We report here a novel method designated as Genome Footprinting by high-throughput sequencing (GeF-seq; <em>in vivo</em> GeF-seq). This method combines <em>in situ</em> DNase I digestion of bacterial genomic DNA with a modified ChIP-chip method (ChAP-chip, Chromatin Affinity Precipitation-chip) we previously developed.<sup><a reveal-id="DST013C13" data-open="DST013C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Unlike IPOD, GeF-seq can visualize the binding profile of a specific target protein at a resolution seen at the <em>in vitro</em> footprinting level. We evaluated the resolution achieved using the GeF-seq method by examining the binding profile of the <em>Bacillus subtilis</em> transition state regulator, AbrB, in comparison with results obtained by ChAP-chip and a modified ChIP-seq method (ChAP coupled with high-throughput sequencing) utilizing sonication to fragment the genomic DNA. AbrB represses the expression of many genes during exponential growth, and we have demonstrated using ChAP-chip that AbrB binds to hundreds of sites throughout the entire <em>B. subtilis</em> genome during exponential growth.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> AbrB is a small protein (10.4 kDa), having a unique structure. The N-terminal domains of two AbrB molecules form a single DNA-binding domain, and AbrB forms a tetramer having a stable DNA-binding ability, via both N-terminal and C-terminal interactions. Structural modelling of AbrB bound to the target sequence indicated that the AbrB tetramer would interact with ∼20 bp sequences,<sup><a reveal-id="DST013C15" data-open="DST013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> whereas <em>in vitro</em> footprinting studies detected a wider range of binding regions from 25 to 80 bp, suggesting that a higher order structure of the AbrB tetramer may be involved in DNA binding at some sites on the chromosome.<sup><a reveal-id="DST013C16  DST013C18" data-open="DST013C16  DST013C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> We previously proposed that AbrB binds to bipartite TGGNA motifs based on the <em>in vivo</em> AbrB-binding regions determined by ChAP-chip analysis,<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> which is in accordance with a motif identified by the <em>in vitro</em> SELEX method.<sup><a reveal-id="DST013C17" data-open="DST013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> However, the consensus sequence was detected in a small number of AbrB-binding regions, and the consensus DNA-binding sequence for AbrB is not completely understood at present.</p><p>We demonstrate here that, by mapping the sequences of short DNA fragments co-purified with AbrB after <em>in situ</em> DNase I digestion of the genomic DNA, the AbrB-binding profile could be visualized with a resolution comparable with that of <em>in vitro</em> footprinting. Importantly, the BiPad web server for modelling bipartite sequence elements<sup><a reveal-id="DST013C19" data-open="DST013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> automatically detected consensus sequences for AbrB binding in &gt;95% of the experimentally determined binding sites. Moreover, highly accurate DNA-binding site information obtained by GeF-seq enabled us to obtain a comprehensive view of the correlation between AbrB-binding signals and cognate recognition sequences; AbrB not only binds to bipartite motifs in sequences with high binding signals, but also to single-sequence motifs in sequences with low signals. These results demonstrate the usefulness of the GeF-seq method.</p>                            <h2 scrollto-destination=5028446 id="5028446" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5028447 id="5028447" class="section-title" data-legacy-id=s2a>2.1. Bacterial strain</h3>
<p><em>Bacillus subtilis</em> strain OC001 expressing C-terminal 2HC (12 histidines plus a chitin-binding domain)-tagged AbrB (AbrB-2HC) was used throughout.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                            <h3 scrollto-destination=5028449 id="5028449" class="section-title" data-legacy-id=s2b>2.2. ChAP-chip and ChAP-seq</h3>
<p>ChAP-chip data for AbrB binding on the <em>B. subtilis</em> genome were taken from our previous report.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> DNA fragments for ChAP-seq analysis were prepared, as previously described.<sup><a reveal-id="DST013C13" data-open="DST013C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Construction of the DNA library for Illumina sequencing was as described below except for the size of the DNA fragments used: ∼250 bp fragments, corresponding to ∼150 bp DNA fragments isolated by ChAP without adapter sequences, were selected for PCR enrichment.</p>                            <h3 scrollto-destination=5028451 id="5028451" class="section-title" data-legacy-id=s2c>2.3. <em>In situ</em> DNase I digestion of genomic DNA</h3>
<p>The GeF-seq method is schematically illustrated in Fig. <a reveal-id="DST013F1" data-open="DST013F1" class="link link-reveal link-table xref-fig">1</a>A. To cross-link protein–DNA complexes, 400 ml of OC001 (<em>abrB-2HC</em>) cells grown to the exponential phase in Luria-Bertani medium at 37°C were treated with formaldehyde as previously described.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> To hydrolyze the cell wall without osmotic burst, cells were treated with 5 mg/ml lysozyme in 3 ml of isotonic sucrose-malate-magnesium buffer (0.02 M maleic acid, 0.5 M sucrose, and 0.02 M MgCl<sub>2</sub>, pH 6.5 adjusted with NaOH)<sup><a reveal-id="DST013C20" data-open="DST013C20" class="link link-ref link-reveal xref-bibr">20</a></sup> in the presence of 1 mM phenylmethylsulfonyl fluoride (PMSF). After 20-min incubation at 37°C with mixing, cells were collected by centrifugation at 6000 <em>g</em> for 5 min at 4°C. Cells were resuspended in 0.5 ml of a buffer containing 0.1 M Tris–HCl (pH 7.5), 0.2 M NaCl, 1% (v/v) Triton X-100, 0.1% (w/v) Na-deoxycholate, 0.2% (w/v) Brij 58, and 20% (v/v) glycerol. </p>                        <a id="5028453" scrollto-destination="5028453"></a>
<div id="DST013F1" data-id="DST013F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01301.jpeg?Expires=2147483647&Signature=wzuvP-d6l4L0Q20FZCS8-N8tUzI3s7SK2VRzTdoTDH91ompldMwW7CcWou9rF6aMfQS7MlGEePTM4YTMrRUw7KVXA1OSQfhWvy6jQCZUZWqZZYoDoSKxSnw0Qc-vqdXWYGuv1xzj2zTVxnF7KACldLKaJIzh5MibjvaXUcYpgTTJhkj4F5xHwj-rk1qJfcfzOyVRn5IdLN3NBSl02YKj3HjsUxiLz5z90VWjjavbB7x~iH5S2Jnq66RW-LYPC~K~xGT9UcFKLi6T4ifhmgpoRqdsNg6WwXUmTsp4dmFKRkHEKnPujHa4APZHVKZKN4JZOE9DyBOBEN~mAxa8xPvFXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Schematic workflow for GeF-seq (see Materials and methods for detail). For comparison purposes, the ChIP-chip and ChIP-seq methods are also illustrated. (B) In situ DNase I digestion. The size of DNA fragments digested by various concentrations of DNase I (1, 0.6, 0.4, and 0.2 U/ml, Lanes 3–6, respectively) and analysed by gel electrophoresis. DNA fragments generated by sonication are run alongside the DNase I digested DNA for comparison purposes (Lane 2)." path-from-xml="dst01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028453/dst01301.jpeg" path-from-xml="dst01301.jpeg" target="_blank">View large</a><a section="5028453" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01301.jpeg?Expires=2147483647&Signature=yRMMR0NIEDDTSH6DI0w5pPxq4-gYMHU~2TAL2Hf7BfqXRNyZ8tzVQLRR6CRtEVHQlYEimblGKiqufTXOLwIte0Mz4SMgbODeZIZoF7fGSQoQHFUay5PtjYsKICc2YY7v6YEeo4UPxynbucAnDJdpmQlk5TWqVc1UIkUhdGXpkK4TUcc6uDJYxoqsQlG5sicXWeCdlV-o6MPE6uzJaoHtLI3dHp5mPvCm-q6kzXylWEK8YlQuPm~uyokhnYhkJ~9kWnecbVaj9e66ObVF2fdQD0WaCJ0Pi8kQfLjVrosGP8p14TXNyj0YS1byL14GP1PRpxX3-UXrhX5Zgf2~6y62Bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028453&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Schematic workflow for GeF-seq (see Materials and methods for detail). For comparison purposes, the ChIP-chip and ChIP-seq methods are also illustrated. (B) <em>In situ</em> DNase I digestion. The size of DNA fragments digested by various concentrations of DNase I (1, 0.6, 0.4, and 0.2 U/ml, Lanes 3–6, respectively) and analysed by gel electrophoresis. DNA fragments generated by sonication are run alongside the DNase I digested DNA for comparison purposes (Lane 2).</p></div></div><div content-id="DST013F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01301.jpeg?Expires=2147483647&Signature=wzuvP-d6l4L0Q20FZCS8-N8tUzI3s7SK2VRzTdoTDH91ompldMwW7CcWou9rF6aMfQS7MlGEePTM4YTMrRUw7KVXA1OSQfhWvy6jQCZUZWqZZYoDoSKxSnw0Qc-vqdXWYGuv1xzj2zTVxnF7KACldLKaJIzh5MibjvaXUcYpgTTJhkj4F5xHwj-rk1qJfcfzOyVRn5IdLN3NBSl02YKj3HjsUxiLz5z90VWjjavbB7x~iH5S2Jnq66RW-LYPC~K~xGT9UcFKLi6T4ifhmgpoRqdsNg6WwXUmTsp4dmFKRkHEKnPujHa4APZHVKZKN4JZOE9DyBOBEN~mAxa8xPvFXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Schematic workflow for GeF-seq (see Materials and methods for detail). For comparison purposes, the ChIP-chip and ChIP-seq methods are also illustrated. (B) In situ DNase I digestion. The size of DNA fragments digested by various concentrations of DNase I (1, 0.6, 0.4, and 0.2 U/ml, Lanes 3–6, respectively) and analysed by gel electrophoresis. DNA fragments generated by sonication are run alongside the DNase I digested DNA for comparison purposes (Lane 2)." path-from-xml="dst01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028453/dst01301.jpeg" path-from-xml="dst01301.jpeg" target="_blank">View large</a><a section="5028453" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01301.jpeg?Expires=2147483647&Signature=yRMMR0NIEDDTSH6DI0w5pPxq4-gYMHU~2TAL2Hf7BfqXRNyZ8tzVQLRR6CRtEVHQlYEimblGKiqufTXOLwIte0Mz4SMgbODeZIZoF7fGSQoQHFUay5PtjYsKICc2YY7v6YEeo4UPxynbucAnDJdpmQlk5TWqVc1UIkUhdGXpkK4TUcc6uDJYxoqsQlG5sicXWeCdlV-o6MPE6uzJaoHtLI3dHp5mPvCm-q6kzXylWEK8YlQuPm~uyokhnYhkJ~9kWnecbVaj9e66ObVF2fdQD0WaCJ0Pi8kQfLjVrosGP8p14TXNyj0YS1byL14GP1PRpxX3-UXrhX5Zgf2~6y62Bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028453&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Schematic workflow for GeF-seq (see Materials and methods for detail). For comparison purposes, the ChIP-chip and ChIP-seq methods are also illustrated. (B) <em>In situ</em> DNase I digestion. The size of DNA fragments digested by various concentrations of DNase I (1, 0.6, 0.4, and 0.2 U/ml, Lanes 3–6, respectively) and analysed by gel electrophoresis. DNA fragments generated by sonication are run alongside the DNase I digested DNA for comparison purposes (Lane 2).</p></div></div><p>To determine suitable conditions for <em>in situ</em> DNase I digestion of genomic DNA, four samples of OC001 cells were prepared as described above and mixed with 10 μl of RNase A (10 mg/ml) and 50 μl of a solution containing 100 mM MgCl<sub>2</sub> and 50 mM CaCl<sub>2</sub>. DNase I digestion was started with the addition of 0.5, 0.3, 0.2, and 0.1 units (U) of DNase I (corresponding to a final concentration of 1, 0.6, 0.4, and 0.2 U/ml) (Takara) and incubated at 37°C with shaking (230 rpm) for 30 min. The reaction was terminated by urea denaturation upon the addition of 3 ml of urea-Triton buffer [0.1 M 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (pH 7.5), 0.01 M imidazole, 8 M urea, 0.5 M NaCl, 1% Triton X-100, 10 mM β-mercaptoethanol, and 1 mM PMSF] instead of ethylenediaminetetraacetic acid, which severely inhibits protein purification by Dynabeads TALON (invitrogen). The samples were then sonicated on ice using an Astrason Ultrasonic Processor XL (Misonix) for 10 min (4 s ‘on’ and 10 s ‘off’, at output level 5). After centrifugation to remove cell debris, 30 μl of the supernatant was mixed with 70 μl of M-wash buffer (0.1 M Tris–HCl, pH 7.5, 1% sodium dodecyl sulfate, 0.01 M dithiothreitol) and incubated at 65°C overnight to reverse the cross-linking. After the removal of proteins by phenol–chloroform–isoamyl alcohol treatment, DNA was recovered by ethanol precipitation in the presence of glycogen, resuspended in 50 μl of nuclease-free water and run on a 2% agarose gel (Fig. <a reveal-id="DST013F1" data-open="DST013F1" class="link link-reveal link-table xref-fig">1</a>B). Treatment with 0.5 units of DNase I (1 U/ml) generated DNA fragments &lt;100 bp in size, and incubation with higher amounts of DNase I resulted in a decrease in the amount of DNA detected by agarose gel electrophoresis (data not shown). Thus, we selected 0.5 units (1 U/ml) of DNase I for further analysis.</p>                            <h3 scrollto-destination=5028455 id="5028455" class="section-title" data-legacy-id=s2d>2.4. Affinity purification of DNA fragments bound to AbrB</h3>
<p>AbrB–DNA complexes were affinity-purified from the clarified DNase I-treated cell lysate, using Dynabeads TALON as described previously,<sup><a reveal-id="DST013C13" data-open="DST013C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> but with the following modification: after protein–DNA complexes were purified and reverse cross-linked by heat treatment at 65°C overnight, proteins were removed using two phenol–chloroform–isoamyl alcohol extractions, and DNA fragments were recovered by ethanol precipitation in the presence of glycogen.</p>                            <h3 scrollto-destination=5028457 id="5028457" class="section-title" data-legacy-id=s2e>2.5. Sequencing of DNA fragments co-purified with AbrB</h3>
<p>The DNA library for sequencing by the Illumina Genome Analyzer IIx (GAIIx) was generated using the NEB Next DNA Sample Prep Reagent kit (New England BioLabs) according to manufacturer's instructions for ‘Preparing Samples for Sequencing Genomic DNA’ (Illumina) with the following modification; after ligation of the adapters to the DNA fragments, the ligated product was run on a 2% [Tris-acetate-EDTA (TAE)] low-range agarose gel (Biorad) at 50 V for 2.5 h in TAE buffer and the region of the gel ∼150 bp (although the DNA was not visible on the gel), corresponding to ∼50 bp fragments without adapter sequences, was excised. The DNA fragments were then purified using a QIAquick Gel Extraction kit (Qaigen) and amplified using 14 cycles of PCR, to obtain at least 1 fmol of DNA library. The amount of DNA was determined by an Agilent 2100 Bioanalyzer using the High-Sensitivity DNA Kit (Agilent). The sequence of the library was then determined by 75-bp single-ended sequencing using the Illumina GAIIx sequencer according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=5028459 id="5028459" class="section-title" data-legacy-id=s2f>2.6. Mapping of read sequences and normalization of tag counts</h3>
<p>A total of 10 369 855 read sequences obtained from the Illumina GAIIx were mapped on the reference genome (<em>B. subtilis</em> str. 168, NC_000964.3), and the mapping results were visualized using the mpsmap and psmap softwares (<a class="link link-uri" href="http://metalmine.naist.jp/maps/gefseq">http://metalmine.naist.jp/maps/gefseq</a>), respectively.<sup><a reveal-id="DST013C21" data-open="DST013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Because DNA fragments of ∼50 bp (without adapter sequences for PCR amplification) were selected in the sample preparation process to obtain complete sequences of the ChAP-DNA fragments, most of the reads reached into the adapter sequence attached to the 3′-end of ChAP-DNA. Thus, unlike general Illumina™ sequencing results obtained by following the instruction manual, most of the read sequences consisted of ∼50 bp of ChAP-DNA sequence followed by the adapter sequence, and both of these sequences varied in length. Since mapping of such different lengths of sequence containing the unmappable adapter sequence was not possible using a standard sequence mapping/assembly program, we utilized the property of mpsmap that maps different length sequences to the best chromosomal location, while allowing up to a specified number of mismatches without a gap. In this study, the read sequences were initially mapped allowing a maximum of 35 mismatches, and the adapter sequences were finally removed. As a result of the first mapping, 9 685 519 (93%) of the read sequences were uniquely mapped to the reference genome. (Thus, the genomic regions encoding the 10 rRNA operons were not included in the present analysis.) Then, to remove the adapter sequences, the starting positions were assigned to seven or more bases allowing a two-base mismatch matched with 5′-end of the primer sequence (AGATCGGAAGAGCTCGTATGCCGTCTTCTGCTTGA) in the 3′-region of the read sequences. In addition, mapped sequences (without adapter sequences) with &gt;2 bp mismatches against the reference sequence were removed, and 8 571 055 (83%) of the read sequences remained for further analysis. Finally, in order to normalize the difference in the local copy number of genomic DNA, counts of mapped reads at each nucleotide position along the genome sequence were linearly scaled by using the <em>oriC</em>/<em>terC</em> ratio (5.15), estimated by sequencing and mapping of whole genomic DNA fragments digested by DNase I, to define the AbrB-binding signals (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>Results shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> suggested that there was preferential digestion of AT-rich genomic sequences by DNase I. However, mapping results of the distribution of ChAP-DNA sequences suggested that the preferentiality apparently did not affect the quantitative estimation of the AbrB-binding profile (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=5028462 id="5028462" class="section-title" data-legacy-id=s2g>2.7. Detection of protein-binding regions</h3>
<p>Most of the read sequences were mapped on distinct regions along the genome surrounded by regions where ends of the read sequences accumulated (Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>C). We used this feature to define the AbrB-binding sites. To estimate the end points of the genomic sequences in the read sequences more precisely, we reanalysed them so that adapter sequences at the 3′-ends of the sequences could be subtracted from the genomic DNA they had been attached to during generation of the library for sequencing. We assigned sequences as ‘adapter sequences’ when five bases at the 3′-end of the read sequence were identical to the adapter primer sequence and the following sequences matched to the primer sequence with no more than two bases mismatched. The accumulation profile of the 3′-ends thus determined across the genome sequence was similar to that of 5′-ends, which was defined as the first base of the read sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), strongly suggesting that the procedure to estimate the 3′-ends of read sequences was reliable.</p><p>Then, the left ends of the read sequences relative to the reference genome sequence were defined as a sum of the 5′-ends of read sequences mapped on the plus strand and the 3′-ends of read sequences mapped on the minus strand, whereas the right ends were defined as a sum of the 3′-ends of read sequences mapped on the plus strand and the 5′-ends of read sequences mapped on the minus strand (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). We counted the numbers of left and right ends mapped to each nucleotide, and positions with ≥10 ends of read sequences and with the highest number of ends within ±30 bp windows were determined in 1 bp steps, as candidates for the left and the right boundaries of the DNA-binding sequences. Then, we extracted regions surrounded by a pair of possible left and right boundaries positioned within a range from 25 to 80 bp (considering the <em>in vitro</em> AbrB footprinting results), and regions, where AbrB-binding signal intensities exceeded a threshold value at more than half of the nucleotides between them, were extracted as AbrB-binding sites. In this study, we first extracted AbrB-binding sequences using the signal intensity corresponding to the top 10th percentile of all nucleotides across the genome as the threshold (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). At some regions, different combinations of boundaries surrounding the overlapping sequences satisfied the criteria. In such cases, the innermost sequences were selected as AbrB-binding sequences for further analysis. Finally, the average of the AbrB-binding signals within the individual binding sequence was calculated, as the AbrB-binding signal intensities of each binding site.</p>                            <h3 scrollto-destination=5028466 id="5028466" class="section-title" data-legacy-id=s2h>2.8. Motif analysis</h3>
<p>AbrB-binding DNA motifs were analysed by the BiPad web server (<a class="link link-uri" href="http://bipad.cmh.edu">http://bipad.cmh.edu</a>) for modelling bipartite sequence elements.<sup><a reveal-id="DST013C19" data-open="DST013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The BiPad program performs multiple local alignment by entropy minimization and cyclic refinement using a stochastic greedy search strategy, and we used the following settings: left half-site, gap range lengths, right half-site, and the iteration cycles were set to 9, 0 or 1, 9, and 500, respectively. To examine the possibility of whether the AbrB-binding motif was discovered by chance, we selected three sets of data, each of which consists of 300 50 bp sequences randomly selected from the <em>B. subtilis</em> 168 genome sequence by the RSA tool,<sup><a reveal-id="DST013C22" data-open="DST013C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and analysed by Bipad.</p>                            <h3 scrollto-destination=5028470 id="5028470" class="section-title" data-legacy-id=s2i>2.9. Sequencing data</h3>
<p>Sequencing data in this study have been submitted to the DDBJ Sequence Read Archive (DRA) and the BioProject database under accession code DRA000758 and PRJDB675, respectively.</p>                            <h2 scrollto-destination=5028475 id="5028475" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5028476 id="5028476" class="section-title" data-legacy-id=s3a>3.1. <em>In vivo</em> GeF-seq</h3>
<p>To improve the resolution of protein-binding site determination by ChIP-seq or ChAP-seq methodologies, we attempted <em>in situ</em> DNase I digestion of the cross-linked bacterial nucleoid to restrict the size of DNA fragments co-purified with the target protein to directly interacting sequences (Fig. <a reveal-id="DST013F1" data-open="DST013F1" class="link link-reveal link-table xref-fig">1</a>A). We employed <em>B. subtilis</em> AbrB as a model protein, whose binding sites were recently determined by use of the ChAP-chip method to be &gt;600 sites scattered across the genome.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Exponentially growing <em>B. subtilis</em> OC001 cells expressing C-terminal 2HC (12 histidines plus a chitin-binding domain)-tagged AbrB (AbrB-2HC) were treated with formaldehyde to stabilize the protein–DNA interactions by cross-linking, and the collected cells were treated with lysozyme in isotonic buffer to facilitate an efficient penetration of DNase I into cells. Then, the genomic DNA was fragmented to &lt;100 bp by the DNase I treatment, followed by the affinity purification of the cross-linked AbrB–DNA complexes using cobalt-coated magnetic beads. DNA fragments co-purified with AbrB (ChAP-DNA) were isolated after reversing the cross-linking between proteins and DNA. As we intended to obtain whole sequences of ChAP-DNA to avoid the bimodal distribution of sequence tags, DNA fragments containing ∼50 bp of inserted DNA, after ligation of adapter sequences, were selected to prepare the library for high-throughput sequencing by Illumina GAIIx. It has been demonstrated that AbrB interacts with ∼20 bp sequences<sup><a reveal-id="DST013C15" data-open="DST013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and, thus, we also expected that 50 bp fragments would be enough to cover single AbrB-binding sites. Single-ended 75-bp sequencing by the Illumina GAIIx generated 9 685 519 (uniquely mapped) sequence reads. As expected, most of the read sequences (88.5%) contained the adapter sequences for PCR amplification at the 3′-end portion, with an average insert size of 50 bp after removal of them (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>), and insert sequences were mapped on distinct sites on the <em>B. subtilis</em> genome. Then, counts of the mapped reads at each nucleotide position along the genome were normalized for differences in the local copy number of genomic DNA, to define the AbrB-binding signals.</p>                            <h3 scrollto-destination=5028478 id="5028478" class="section-title" data-legacy-id=s3b>3.2. Comparison of the distribution of AbrB-binding signals determined by GeF-seq, ChAP-seq, and ChAP-chip</h3>
<p>To evaluate the resolution of the GeF-seq method in identifying genomic protein-binding sites, we initially compared the distributions of AbrB-binding signals along the genome as determined by three methods: GeF-seq, ChAP-seq, and ChAP-chip. The distributions of the AbrB-binding signals on the genome determined by GeF-seq and ChAP-seq in the present study were highly consistent with that of ChAP-chip we reported previously.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Typical examples of the comparison are presented in Fig. <a reveal-id="DST013F2" data-open="DST013F2" class="link link-reveal link-table xref-fig">2,</a> and the complete profiles of the binding signals across the genome obtained by the three methods are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. Close-up views of profiles of AbrB-binding signals (Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>A and B) indicated that, although the ChAP-seq method improved the resolution of detection of the binding regions compared with ChAP-chip, the GeF-seq method dramatically improved the resolution even when compared with ChAP-seq. Importantly, GeF-seq could resolve the closely positioned binding sites that appear as one peak in the ChAP-seq method, as shown in Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>A. </p>                        <a id="5028480" scrollto-destination="5028480"></a>
<div id="DST013F2" data-id="DST013F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01302.jpeg?Expires=2147483647&Signature=bHNZvu74fmxBZ6RjwEKEgPsMGiSm4YZQrUmc7gi-zBXrLNoBGQ44RXOYyZrTu8Qpa6KFTkN2tYjwBy7tBc8iPkfJKe~rsH99mUvOQkJEov~R7V9KZ7BoAGDv-p94dr0IDvDGNshtZ0DBBNboZl1YGJazpF8krdvZS~xX2~-x6ZR5y95eRFFtxSfuj9dJD3i~p-AcsnAFiA6leNqIVOIXJVLtDNGz6T7F3Hz7qsYs4cHFV2yewtjD7E~E7qWCF1lxAwpEEVa7fkcg3fDJ9lUhSK9DqO6WP0E08YgJTfWfobTyZppwoPA1kMy1q4KTJwWrihzvQvZAW3DRUCm43qoRDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of AbrB-binding profiles detected by ChAP-chip, ChAP-seq, and GeF-seq on the B. subtilis genome. The distribution of AbrB-binding sites on the B. subtilis genome is shown in the indicated genome region at the top of the figure. For the ChAP-chip results, previously published data were used. The top, middle, and bottom lines indicate binding intensities of 7, 0.4, and 0, respectively, as defined by the previous report.14 For ChAP-seq and GeF-seq, protein-binding intensities (number of reads after normalization of the local copy number of the genomic DNA) are presented as a sum of binding intensities on the plus and − strands. The top lines in both the ChAP-seq and GeF-seq columns indicate binding intensities of 15 000 and 25 000, respectively, and the bottom lines indicate binding intensities of 0. The gene organization is schematically shown at the bottom." path-from-xml="dst01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028480/dst01302.jpeg" path-from-xml="dst01302.jpeg" target="_blank">View large</a><a section="5028480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01302.jpeg?Expires=2147483647&Signature=gBbHM-ePYbaU9ZufRc5TTq-J4yGQwnhA9GyjWa6ixXm39pWuB6w0fQ-K7rl0e3TKOhb7Q8p5tw1xuu-xvuS7nhwTc9x5DktJP~I2DHb2NXbDq1OJf~SRZSgeZkUlGJprby~96Ies-ZPNkm~iO0mErQaIcs1gQ4pU81JPMD-BgeC1ki11ub4RBxaw8OZTPS9glFYay3-evKCCsfMAkcgmzxyN12-27HOXxfEIoJ78h9hLa1ygBqVjuxN7RnL4zFA5f6YxlFM7LlbOqx-cbth0y3IrpdkhR9FGAtRaYfJJ7NCSri8QfoAkkEQV8usvJtg318XaoAJgEmqcQeYNjINSwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028480&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of AbrB-binding profiles detected by ChAP-chip, ChAP-seq, and GeF-seq on the <em>B. subtilis</em> genome. The distribution of AbrB-binding sites on the <em>B. subtilis</em> genome is shown in the indicated genome region at the top of the figure. For the ChAP-chip results, previously published data were used. The top, middle, and bottom lines indicate binding intensities of 7, 0.4, and 0, respectively, as defined by the previous report.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For ChAP-seq and GeF-seq, protein-binding intensities (number of reads after normalization of the local copy number of the genomic DNA) are presented as a sum of binding intensities on the plus and − strands. The top lines in both the ChAP-seq and GeF-seq columns indicate binding intensities of 15 000 and 25 000, respectively, and the bottom lines indicate binding intensities of 0. The gene organization is schematically shown at the bottom.</p></div></div><div content-id="DST013F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01302.jpeg?Expires=2147483647&Signature=bHNZvu74fmxBZ6RjwEKEgPsMGiSm4YZQrUmc7gi-zBXrLNoBGQ44RXOYyZrTu8Qpa6KFTkN2tYjwBy7tBc8iPkfJKe~rsH99mUvOQkJEov~R7V9KZ7BoAGDv-p94dr0IDvDGNshtZ0DBBNboZl1YGJazpF8krdvZS~xX2~-x6ZR5y95eRFFtxSfuj9dJD3i~p-AcsnAFiA6leNqIVOIXJVLtDNGz6T7F3Hz7qsYs4cHFV2yewtjD7E~E7qWCF1lxAwpEEVa7fkcg3fDJ9lUhSK9DqO6WP0E08YgJTfWfobTyZppwoPA1kMy1q4KTJwWrihzvQvZAW3DRUCm43qoRDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of AbrB-binding profiles detected by ChAP-chip, ChAP-seq, and GeF-seq on the B. subtilis genome. The distribution of AbrB-binding sites on the B. subtilis genome is shown in the indicated genome region at the top of the figure. For the ChAP-chip results, previously published data were used. The top, middle, and bottom lines indicate binding intensities of 7, 0.4, and 0, respectively, as defined by the previous report.14 For ChAP-seq and GeF-seq, protein-binding intensities (number of reads after normalization of the local copy number of the genomic DNA) are presented as a sum of binding intensities on the plus and − strands. The top lines in both the ChAP-seq and GeF-seq columns indicate binding intensities of 15 000 and 25 000, respectively, and the bottom lines indicate binding intensities of 0. The gene organization is schematically shown at the bottom." path-from-xml="dst01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028480/dst01302.jpeg" path-from-xml="dst01302.jpeg" target="_blank">View large</a><a section="5028480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01302.jpeg?Expires=2147483647&Signature=gBbHM-ePYbaU9ZufRc5TTq-J4yGQwnhA9GyjWa6ixXm39pWuB6w0fQ-K7rl0e3TKOhb7Q8p5tw1xuu-xvuS7nhwTc9x5DktJP~I2DHb2NXbDq1OJf~SRZSgeZkUlGJprby~96Ies-ZPNkm~iO0mErQaIcs1gQ4pU81JPMD-BgeC1ki11ub4RBxaw8OZTPS9glFYay3-evKCCsfMAkcgmzxyN12-27HOXxfEIoJ78h9hLa1ygBqVjuxN7RnL4zFA5f6YxlFM7LlbOqx-cbth0y3IrpdkhR9FGAtRaYfJJ7NCSri8QfoAkkEQV8usvJtg318XaoAJgEmqcQeYNjINSwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028480&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of AbrB-binding profiles detected by ChAP-chip, ChAP-seq, and GeF-seq on the <em>B. subtilis</em> genome. The distribution of AbrB-binding sites on the <em>B. subtilis</em> genome is shown in the indicated genome region at the top of the figure. For the ChAP-chip results, previously published data were used. The top, middle, and bottom lines indicate binding intensities of 7, 0.4, and 0, respectively, as defined by the previous report.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> For ChAP-seq and GeF-seq, protein-binding intensities (number of reads after normalization of the local copy number of the genomic DNA) are presented as a sum of binding intensities on the plus and − strands. The top lines in both the ChAP-seq and GeF-seq columns indicate binding intensities of 15 000 and 25 000, respectively, and the bottom lines indicate binding intensities of 0. The gene organization is schematically shown at the bottom.</p></div></div>                        <a id="5028481" scrollto-destination="5028481"></a>
<div id="DST013F3" data-id="DST013F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01303.jpeg?Expires=2147483647&Signature=pGbcwdQHnMDjt44DUXGv7F2krNXXgc~DcuYUso3AU6--SI4--UGcGvtW1hnKyuUn~kzc-7OMxLMGoMBS7ube85ODW6dMfrN9hucb2i1VSrYw56c3r-IFDUlC5HrZXu7XN-VHp6hQXi5DFWsXunWKpGQrccgy~hvjtHMlCg3D~PNIUvUDVKFRpZz5W7OKD4EGbkqmecX2PgmEgdvcu9oSUe1uuea89-NPj6qp9MdzSL72zAcgdYS5LhuIE3vDuEfW91qbAwVUO4~gQOmcI9zeqfz0KNSXzqGUjaI-Adge4oQ7MHnC31WHJnCY3jWDs7zTuGVudESLIhYymKn5LQSVbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of AbrB-binding sites determined by GeF-seq, ChAP-Seq, and ChAP-chip. (A) A typical example of AbrB-binding regions that are merged into one peak in ChAP-Seq and ChAP-chip, but separately detected by GeF-seq. For the ChAP-chip results, previously published data were used. For ChAP-seq and GeF-seq, binding intensities on the forward (+) and reverse (−) strands are inversely shown by transparent blue and red peaks, respectively (thus, overlapping regions appear purple) in the respective inverted orientation (+ and −) and their summed binding intensities are shown by a dark blue peak in the upper part of the figure (merged). The gene organization is schematically shown at the bottom. (B) A typical example of AbrB-binding regions, where ChAP-seq detected two peaks on the plus and minus strands with a gap, but GeF-seq detected two peaks at the same position (see Results for details). (C) Resolution of detection within an AbrB-binding region of the chromosome by GeF-seq analysis. The intergenic region between yqxM and yqzG in (B) is magnified. The number of left and right ends is depicted by green and red lines, respectively, at each base. The scale for the protein-binding intensity and the number of left and right ends are indicated on the right inside and outside, respectively, and the highest positions on the left and the right edges are marked by vertical dotted lines whose boundaries are defined as the protein-binding region. The AbrB-binding region, which was previously determined by in vitro footprinting,18 is also shown by green boxes in (A) and (B) and red letters in the DNA sequence on the top in (C)." path-from-xml="dst01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028481/dst01303.jpeg" path-from-xml="dst01303.jpeg" target="_blank">View large</a><a section="5028481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01303.jpeg?Expires=2147483647&Signature=09fQ-LLUpWzvE2RHBxao12EhIESDDauhstL6kmbUPQBMn2CL7gzDzS2Qsm6WqtMNW6ig4dGGfkGdZwJ1NzKKCqI4gLIlHdZgMhaohGieohpJcPUw4bKMEWh8vdmoWme2PzBCFU7Iv3u39o~oTUkHXyp6ThE6Q~z4h3pEqTU0D2ytdpw8QHutUtmVlJFDGbLgQk-t-23Zru0RhldfxUgUCP~XckBoVbo3eBIMPiKqyqYsHwcxK23H0vlYzC-5fziI9riqAR0rU117iHSmclx84IPvgOZfjMQ~~SaaO1M37MrP2mmybaom1xn7QR8WSbWix-yaPxvUY4w6or57Ml0m2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028481&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of AbrB-binding sites determined by GeF-seq, ChAP-Seq, and ChAP-chip. (A) A typical example of AbrB-binding regions that are merged into one peak in ChAP-Seq and ChAP-chip, but separately detected by GeF-seq. For the ChAP-chip results, previously published data were used. For ChAP-seq and GeF-seq, binding intensities on the forward (+) and reverse (−) strands are inversely shown by transparent blue and red peaks, respectively (thus, overlapping regions appear purple) in the respective inverted orientation (+ and −) and their summed binding intensities are shown by a dark blue peak in the upper part of the figure (merged). The gene organization is schematically shown at the bottom. (B) A typical example of AbrB-binding regions, where ChAP-seq detected two peaks on the plus and minus strands with a gap, but GeF-seq detected two peaks at the same position (see Results for details). (C) Resolution of detection within an AbrB-binding region of the chromosome by GeF-seq analysis. The intergenic region between <em>yqxM</em> and <em>yqzG</em> in (B) is magnified. The number of left and right ends is depicted by green and red lines, respectively, at each base. The scale for the protein-binding intensity and the number of left and right ends are indicated on the right inside and outside, respectively, and the highest positions on the left and the right edges are marked by vertical dotted lines whose boundaries are defined as the protein-binding region. The AbrB-binding region, which was previously determined by <em>in vitro</em> footprinting,<sup><a reveal-id="DST013C18" data-open="DST013C18" class="link link-ref link-reveal xref-bibr">18</a></sup> is also shown by green boxes in (A) and (B) and red letters in the DNA sequence on the top in (C).</p></div></div><div content-id="DST013F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01303.jpeg?Expires=2147483647&Signature=pGbcwdQHnMDjt44DUXGv7F2krNXXgc~DcuYUso3AU6--SI4--UGcGvtW1hnKyuUn~kzc-7OMxLMGoMBS7ube85ODW6dMfrN9hucb2i1VSrYw56c3r-IFDUlC5HrZXu7XN-VHp6hQXi5DFWsXunWKpGQrccgy~hvjtHMlCg3D~PNIUvUDVKFRpZz5W7OKD4EGbkqmecX2PgmEgdvcu9oSUe1uuea89-NPj6qp9MdzSL72zAcgdYS5LhuIE3vDuEfW91qbAwVUO4~gQOmcI9zeqfz0KNSXzqGUjaI-Adge4oQ7MHnC31WHJnCY3jWDs7zTuGVudESLIhYymKn5LQSVbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of AbrB-binding sites determined by GeF-seq, ChAP-Seq, and ChAP-chip. (A) A typical example of AbrB-binding regions that are merged into one peak in ChAP-Seq and ChAP-chip, but separately detected by GeF-seq. For the ChAP-chip results, previously published data were used. For ChAP-seq and GeF-seq, binding intensities on the forward (+) and reverse (−) strands are inversely shown by transparent blue and red peaks, respectively (thus, overlapping regions appear purple) in the respective inverted orientation (+ and −) and their summed binding intensities are shown by a dark blue peak in the upper part of the figure (merged). The gene organization is schematically shown at the bottom. (B) A typical example of AbrB-binding regions, where ChAP-seq detected two peaks on the plus and minus strands with a gap, but GeF-seq detected two peaks at the same position (see Results for details). (C) Resolution of detection within an AbrB-binding region of the chromosome by GeF-seq analysis. The intergenic region between yqxM and yqzG in (B) is magnified. The number of left and right ends is depicted by green and red lines, respectively, at each base. The scale for the protein-binding intensity and the number of left and right ends are indicated on the right inside and outside, respectively, and the highest positions on the left and the right edges are marked by vertical dotted lines whose boundaries are defined as the protein-binding region. The AbrB-binding region, which was previously determined by in vitro footprinting,18 is also shown by green boxes in (A) and (B) and red letters in the DNA sequence on the top in (C)." path-from-xml="dst01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028481/dst01303.jpeg" path-from-xml="dst01303.jpeg" target="_blank">View large</a><a section="5028481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01303.jpeg?Expires=2147483647&Signature=09fQ-LLUpWzvE2RHBxao12EhIESDDauhstL6kmbUPQBMn2CL7gzDzS2Qsm6WqtMNW6ig4dGGfkGdZwJ1NzKKCqI4gLIlHdZgMhaohGieohpJcPUw4bKMEWh8vdmoWme2PzBCFU7Iv3u39o~oTUkHXyp6ThE6Q~z4h3pEqTU0D2ytdpw8QHutUtmVlJFDGbLgQk-t-23Zru0RhldfxUgUCP~XckBoVbo3eBIMPiKqyqYsHwcxK23H0vlYzC-5fziI9riqAR0rU117iHSmclx84IPvgOZfjMQ~~SaaO1M37MrP2mmybaom1xn7QR8WSbWix-yaPxvUY4w6or57Ml0m2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028481&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of AbrB-binding sites determined by GeF-seq, ChAP-Seq, and ChAP-chip. (A) A typical example of AbrB-binding regions that are merged into one peak in ChAP-Seq and ChAP-chip, but separately detected by GeF-seq. For the ChAP-chip results, previously published data were used. For ChAP-seq and GeF-seq, binding intensities on the forward (+) and reverse (−) strands are inversely shown by transparent blue and red peaks, respectively (thus, overlapping regions appear purple) in the respective inverted orientation (+ and −) and their summed binding intensities are shown by a dark blue peak in the upper part of the figure (merged). The gene organization is schematically shown at the bottom. (B) A typical example of AbrB-binding regions, where ChAP-seq detected two peaks on the plus and minus strands with a gap, but GeF-seq detected two peaks at the same position (see Results for details). (C) Resolution of detection within an AbrB-binding region of the chromosome by GeF-seq analysis. The intergenic region between <em>yqxM</em> and <em>yqzG</em> in (B) is magnified. The number of left and right ends is depicted by green and red lines, respectively, at each base. The scale for the protein-binding intensity and the number of left and right ends are indicated on the right inside and outside, respectively, and the highest positions on the left and the right edges are marked by vertical dotted lines whose boundaries are defined as the protein-binding region. The AbrB-binding region, which was previously determined by <em>in vitro</em> footprinting,<sup><a reveal-id="DST013C18" data-open="DST013C18" class="link link-ref link-reveal xref-bibr">18</a></sup> is also shown by green boxes in (A) and (B) and red letters in the DNA sequence on the top in (C).</p></div></div><p>Using ChAP-seq, binding sites were often detected as two broad peaks on the forward (+) and reverse (−) strands, as previously reported.<sup><a reveal-id="DST013C6" data-open="DST013C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In contrast, using GeF-seq, the distributions of sequence tags on the plus and − strands overlapped in the middle of the two ChAP-seq peaks (Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>B). Thus, the use of short DNA fragments enabled the conclusive determination of protein-bound regions of DNA without the necessity for the bioinformatic prediction of the binding sites. In addition, AbrB-binding signals at each binding site generally distributed in a trapezoid form, and the ends of the read sequences accumulated at the left and right edges (Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>C). These observations strongly suggested that <em>in situ</em> DNase I digestion occurred at the boundaries of protein-binding sites, as observed in <em>in vitro</em> DNase I footprinting. Furthermore, the lengths of sequences protected from DNase I digestion (27–80 bp) suggested that these sequences would be interacting with one to three AbrB tetramer(s). We used this feature to define AbrB-binding sequences, as described below.</p>                            <h3 scrollto-destination=5028483 id="5028483" class="section-title" data-legacy-id=s3c>3.3. Determination of AbrB-binding sequences</h3>
<p>To automatically extract DNA sequences bound by AbrB from the GeF-seq results, we developed an analytical pipeline as described in Materials and methods. Briefly, we first surveyed pairs of nucleotide positions showing the highest accumulation of ends of read sequences, as candidates for the borders of the protein-binding regions. Then, AbrB-binding signals between them were evaluated using a relaxed threshold value, corresponding to the signal intensity at the top 10th percentile of all nucleotide positions across the genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). This resulted in 5897 possible AbrB-binding sites being detected (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), which included not only specific, but also non-specific AbrB-binding, sites. These were extracted and ranked by their average binding signal intensities of nucleotides included in each site. The peak ID was given from 1 to 5897 by their intensity ranked from high to low, respectively.</p><p>The top 700 binding sites accompanied by high-binding signal intensities were first examined, because this number was approximately similar to that obtained by previous ChAP-chip analysis (694 binding sites).<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The length of the AbrB-binding regions determined by the GeF-seq ranged from 27 to 79 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>), which was consistent with the results of <em>in vitro</em> footprinting experiments listed in a database of transcriptional regulation in <em>B. subtilis</em><sup><a reveal-id="DST013C23" data-open="DST013C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and in recent reports.<sup><a reveal-id="DST013C17" data-open="DST013C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST013C18" data-open="DST013C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST013C24" data-open="DST013C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Among 32 AbrB-binding sites previously determined by <em>in vitro</em> DNase I footprinting, our GeF-seq experiment detected 11 AbrB-binding sequences (Fig. <a reveal-id="DST013F3" data-open="DST013F3" class="link link-reveal link-table xref-fig">3</a>C and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). GeF-seq also detected 10 AbrB-binding sequences within the 5897 possible AbrB-binding sites, although binding intensities were lower than those of the top 700 binding sites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Thus, we found that these 21 binding sites matched those obtained by <em>in vitro</em> DNase I footprinting. These results indicate that our GeF-seq method has the ability to detect protein-bound DNA sequences with a resolution comparable with that of the <em>in vitro</em> footprinting method, although differences in boundaries are observed between our GeF-seq result and the <em>in vitro</em> footprinting result, which may result from differences in conditions between <em>in vivo</em> and <em>in vitro</em> experiments.</p>                            <h3 scrollto-destination=5028486 id="5028486" class="section-title" data-legacy-id=s3d>3.4. Identification of consensus sequences for the AbrB binding</h3>
<p>In previous ChAP-chip analysis,<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> we found a possible consensus sequence for AbrB binding to be TNCCA–4 bp–TGGNA, which is composed of a pair of two AbrB-binding motifs previously identified by the <em>in vitro</em> SELEX method.<sup><a reveal-id="DST013C17" data-open="DST013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> However, those motifs were detected in a limited number of AbrB-bound sequences. In addition, we found that not only TNCCA–4 bp–TGGNA, but also other bipartite TGGNA motifs, in palindromic or tandem orientation, separated by 4–5 bp were enriched in AbrB-bound DNA sequences on the <em>B. subtilis</em> genome.</p><p>In the present GeF-seq analysis, the lengths of automatically extracted AbrB-binding sequences were restricted to an <em>in vitro</em> DNase I footprinting level. Thus, we expected that the large amount of precise information on AbrB-binding sequences might give us a clear view on the consensus AbrB-binding sequence. We then utilized the BiPad web server, a web interface to predict sequence elements embedded within unaligned sequences, to analyse the experimentally derived AbrB-binding sequences. BiPad predicts various pairs of bipartite motifs with different gaps in different orientations as one consensus sequence.<sup><a reveal-id="DST013C19" data-open="DST013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> BiPad successfully identified a mixture of bipartite TGGNA motifs in 96% (678) of the 700 experimentally identified sequences (Fig. <a reveal-id="DST013F4" data-open="DST013F4" class="link link-reveal link-table xref-fig">4</a>A), and we found that we could classify them into six patterns by manually sorting the predicted consensus in each AbrB-binding sequence (Fig. <a reveal-id="DST013F4" data-open="DST013F4" class="link link-reveal link-table xref-fig">4</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). As a result, consensus sequences were found to be composed of bipartite TGGNA motifs separated by 4 or 5 bp AT-rich sequences arranged in direct, reverse direct, inverted, and everted repeat orientations (Fig. <a reveal-id="DST013F4" data-open="DST013F4" class="link link-reveal link-table xref-fig">4</a>B). Importantly, the location of the consensus sequence was usually close to the middle of the experimentally identified binding sequences (Fig. <a reveal-id="DST013F4" data-open="DST013F4" class="link link-reveal link-table xref-fig">4</a>C). Thus, we not only confirmed that the AbrB-binding consensus sequence we proposed previously was indeed detectable in almost all of the AbrB-binding sequences with high binding signals, but we also demonstrated that the information on the protein-binding sequences automatically extracted by the GeF-seq analysis enabled us to clearly identify a consensus sequence for protein binding, at least in the case of AbrB. </p>                        <a id="5028489" scrollto-destination="5028489"></a>
<div id="DST013F4" data-id="DST013F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01304.jpeg?Expires=2147483647&Signature=dHxA3SKIuhOfI~1IXZUhwMxu9ovar77OlZymJ3MoGOspqqQjhqC6Lb8tzdfswSefLb2LNVh-ZvSwKzmO-7kLysk6sXQ8-GO7Me-7mk5N0wFStMXa5PeZgSL45QplNL0saUKwiOOTBuuk0Bx5iSOtYwN1gYZcnKJ~HOT18W1uxz~~vCTxilfOoRjpnrg~7CzLdrByeFTeN8MZE65PSezMGdtRH2NC4tB5DXKOM~djwDzxLHjCIHo8OXO2qqMtL32Yx68DwKpxfmN4yLSn-0a2VMDtFX68KOLlmSS33v-dNCw50k544bAkFiwBeoVvKYgR6sXqwk1fzbDCCc5dc4YAHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus sequences of AbrB-binding sites identified by BiPad. (A) The top 700 AbrB-binding sequences ranked by their average binding intensity from high to low were analysed. The results of BiPad analysis were plotted regardless of the orientation of the TGGNA/TNCCA motif; direct repeat (DI), reversed direct repeat (RDR), inverted repeat (IR), and everted repeat (ER). The number of input sequences and sequences judged to contain the consensus sequences by the algorithms are indicated. The mixture of four types of bipartite patterns with 4- and 5-bp spacing in (A) were sorted and the graphical logo for each pattern was re-created in (B). The DI and RDR bipartite pattern were plotted in the same logo. The number of sequences belonging to each bipartite pattern is shown. (C) Distribution of bipartite DNA motifs within the 700 binding sequences. The binding sequences were sorted by length, and the motifs identified by Bipad were marked red." path-from-xml="dst01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028489/dst01304.jpeg" path-from-xml="dst01304.jpeg" target="_blank">View large</a><a section="5028489" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01304.jpeg?Expires=2147483647&Signature=vZ8QIKnXVKGMYlcXkpGUJDWSaA0U3OegW0S0Jolwap13egRfBhCTw6zifMFNXX5KhRZ79WkvLMg0WQYqTo~X~SAltqWw2MGPrdtSPZjPHuqo9FM8W~IUBsbOHjn1MWjDDJ-JdGaKAWfcdAiUAvr-mUlSGtWkCC64Oj5oxdFQeL~CDL4IgRO1RtlbiBU5iATQALtQjVf5jvZYNxycjiAsQ1K7Bv7xN4A2k01HkWfAXjUQmDHr-m582-lvaSw9rK~X4bC2p13O3KzziT6QQw7UsuRayqLgXnJiOYLcAOIrOKEfZCvC~5PRAwonq7VaTTO4yxIDYPc-bZJvzx1W-l4r4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028489&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus sequences of AbrB-binding sites identified by BiPad. (A) The top 700 AbrB-binding sequences ranked by their average binding intensity from high to low were analysed. The results of BiPad analysis were plotted regardless of the orientation of the TGGNA/TNCCA motif; direct repeat (DI), reversed direct repeat (RDR), inverted repeat (IR), and everted repeat (ER). The number of input sequences and sequences judged to contain the consensus sequences by the algorithms are indicated. The mixture of four types of bipartite patterns with 4- and 5-bp spacing in (A) were sorted and the graphical logo for each pattern was re-created in (B). The DI and RDR bipartite pattern were plotted in the same logo. The number of sequences belonging to each bipartite pattern is shown. (C) Distribution of bipartite DNA motifs within the 700 binding sequences. The binding sequences were sorted by length, and the motifs identified by Bipad were marked red.</p></div></div><div content-id="DST013F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01304.jpeg?Expires=2147483647&Signature=dHxA3SKIuhOfI~1IXZUhwMxu9ovar77OlZymJ3MoGOspqqQjhqC6Lb8tzdfswSefLb2LNVh-ZvSwKzmO-7kLysk6sXQ8-GO7Me-7mk5N0wFStMXa5PeZgSL45QplNL0saUKwiOOTBuuk0Bx5iSOtYwN1gYZcnKJ~HOT18W1uxz~~vCTxilfOoRjpnrg~7CzLdrByeFTeN8MZE65PSezMGdtRH2NC4tB5DXKOM~djwDzxLHjCIHo8OXO2qqMtL32Yx68DwKpxfmN4yLSn-0a2VMDtFX68KOLlmSS33v-dNCw50k544bAkFiwBeoVvKYgR6sXqwk1fzbDCCc5dc4YAHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus sequences of AbrB-binding sites identified by BiPad. (A) The top 700 AbrB-binding sequences ranked by their average binding intensity from high to low were analysed. The results of BiPad analysis were plotted regardless of the orientation of the TGGNA/TNCCA motif; direct repeat (DI), reversed direct repeat (RDR), inverted repeat (IR), and everted repeat (ER). The number of input sequences and sequences judged to contain the consensus sequences by the algorithms are indicated. The mixture of four types of bipartite patterns with 4- and 5-bp spacing in (A) were sorted and the graphical logo for each pattern was re-created in (B). The DI and RDR bipartite pattern were plotted in the same logo. The number of sequences belonging to each bipartite pattern is shown. (C) Distribution of bipartite DNA motifs within the 700 binding sequences. The binding sequences were sorted by length, and the motifs identified by Bipad were marked red." path-from-xml="dst01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028489/dst01304.jpeg" path-from-xml="dst01304.jpeg" target="_blank">View large</a><a section="5028489" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01304.jpeg?Expires=2147483647&Signature=vZ8QIKnXVKGMYlcXkpGUJDWSaA0U3OegW0S0Jolwap13egRfBhCTw6zifMFNXX5KhRZ79WkvLMg0WQYqTo~X~SAltqWw2MGPrdtSPZjPHuqo9FM8W~IUBsbOHjn1MWjDDJ-JdGaKAWfcdAiUAvr-mUlSGtWkCC64Oj5oxdFQeL~CDL4IgRO1RtlbiBU5iATQALtQjVf5jvZYNxycjiAsQ1K7Bv7xN4A2k01HkWfAXjUQmDHr-m582-lvaSw9rK~X4bC2p13O3KzziT6QQw7UsuRayqLgXnJiOYLcAOIrOKEfZCvC~5PRAwonq7VaTTO4yxIDYPc-bZJvzx1W-l4r4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028489&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus sequences of AbrB-binding sites identified by BiPad. (A) The top 700 AbrB-binding sequences ranked by their average binding intensity from high to low were analysed. The results of BiPad analysis were plotted regardless of the orientation of the TGGNA/TNCCA motif; direct repeat (DI), reversed direct repeat (RDR), inverted repeat (IR), and everted repeat (ER). The number of input sequences and sequences judged to contain the consensus sequences by the algorithms are indicated. The mixture of four types of bipartite patterns with 4- and 5-bp spacing in (A) were sorted and the graphical logo for each pattern was re-created in (B). The DI and RDR bipartite pattern were plotted in the same logo. The number of sequences belonging to each bipartite pattern is shown. (C) Distribution of bipartite DNA motifs within the 700 binding sequences. The binding sequences were sorted by length, and the motifs identified by Bipad were marked red.</p></div></div><p>It should be also noted that any clear consensus sequence was not detected in the remaining 22 sequences, although a degenerate single TGGNA motif was detected (data not shown). Since AbrB binding to these sequences was clearly detected with high signal intensity (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), this result indicates that AbrB also binds to sequences without bipartite motifs by some mechanism, for example, when the DNA sequence forms structure(s) to fit the AbrB-binding surface.</p>                            <h3 scrollto-destination=5028491 id="5028491" class="section-title" data-legacy-id=s3e>3.5. Correlation between AbrB-binding signals and motif discovery</h3>
<p>Here, using 700 AbrB-binding DNA sequences with high GeF-seq AbrB-binding signals, we identified bipartite AbrB-binding motifs across the <em>Bacillus</em> genome arranged in any orientation with a 4- or 5-bp spacing, which is consistent with our previous ChAP-chip analysis<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and the <em>in vitro</em> SELEX results reported by Xu and Strauch.<sup><a reveal-id="DST013C17" data-open="DST013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> These results strongly suggested that, when binding signal intensities are high, the sequences are those specifically recognized by AbrB. We usually use a threshold value to discriminate ‘real’ protein-binding peaks and possible ‘artificial’ binding peaks in ChIP-chip and ChIP-seq experiments. However, actually, these threshold values have been operationally defined by researchers; for example, aiming to remove false positives or to remove false negatives, and examination of actual protein binding to extract possible binding sequences has rarely been examined thoroughly. The finding that we could identify AbrB-binding consensus sequences in almost all of the binding sequences accompanied with high binding signals in GeF-seq prompted us to comprehensively examine whether there was conservation of the binding motifs in sequences with lower binding signals.</p><p>To this end, we divided the 5897 possible AbrB-binding sequences into 20 sets each containing 300 sequences, according to average AbrB-binding signal intensities (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>C), and the consensus sequence for each set was extracted by the Bipad program (Fig. <a reveal-id="DST013F5" data-open="DST013F5" class="link link-reveal link-table xref-fig">5</a>). In the three datasets with the top AbrB-binding signal intensities (1–300, 301–600, and 601–900), the bipartite TGGNA motifs with a 4- or 5-bp spacer sequence were detected in almost all sequences (98, 96, and 96% respectively, Fig. <a reveal-id="DST013F5" data-open="DST013F5" class="link link-reveal link-table xref-fig">5</a>). In the next group of datasets with lower binding signal intensities (901–1200, 1201–1500, and 1501–1800), although the consensus sequences containing bipartite TGGNA motifs with a 4- or 5-bp spacer sequence could be detected, one half-site became degenerate. Interestingly, in the following 10 sets (from 1801 to 4800), only a single TGGNA motif was detected, whereas, in the remaining four sets with the lowest binding signal intensities (from 4801 to 5897, Fig. <a reveal-id="DST013F5" data-open="DST013F5" class="link link-reveal link-table xref-fig">5</a>), the single motif becomes very degenerate. We confirmed that the TGGNA motif was not detected by chance because no motif was detected in similar sets of DNA sequences (300 50-bp sequences) that were randomly extracted from the genome sequence of <em>B. subtilis</em> 168 (data not shown). These results strongly suggested that AbrB not only binds stably, with high experimentally derived binding signals, to bipartite TGGNA motifs, but also interacts with single TGGNA motifs in sequences with lower but significant experimentally derived signal intensities (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). </p>                        <a id="5028494" scrollto-destination="5028494"></a>
<div id="DST013F5" data-id="DST013F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01305.jpeg?Expires=2147483647&Signature=LHsqvWeofuja3QdNVDWhT-ZZdVO07LmQbDPQagwgev4IZlucZpu~3AauFBjE~JbK5ee8ArWYdjgcijcUBm0ZOGFL5KSck7fZvKMypKDjtt4RwgNPdAHNpgbx0th~1S4gKBGRkCBvW3YdRO-sG4x8SN2VOlSYemc-ONRlQLkC-p1RKSIRr-Q8ZljVelRaVXTge78Z1qU6w7SNPtwZXr1~rlOWpbKe4ExQ13ODN-rwXMek-UDY1f2GLBUBsCb9P1m3MV~SHodxx3FlHvtZE8M3CnVwGip8LVeOO~jU2Ybw2NMAjSkEqtgJfJuUFW~dhKpvabhi6FidVlNSZotYbe84~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus sequences of the AbrB-binding sequences detected in the top 10th percentile threshold. Each group of 300 signals in the 20 sets of AbrB-binding sequences were ranked from high to low by binding intensities and were analysed by Bipad to detect respective consensus sequences. The number of sequences containing the motif is shown in parallel." path-from-xml="dst01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028494/dst01305.jpeg" path-from-xml="dst01305.jpeg" target="_blank">View large</a><a section="5028494" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01305.jpeg?Expires=2147483647&Signature=LyMFrWWyJqqsbYDP0QPyOuDmrRN7~nikp7jyoRTiaL5xwPUn7nbZ-TKN6FYNiCEswLQlZ7ADLjYAJYzmpREqVY0jYkemfdKRIgXQfENWHuA0nQ20hvTXqEK55qjMD99XmIycR3z1qyxb6XYOov7vee4rrsuMa6CBj0R7yhC8rOKSgnJ0i39f14~gQTq9Aj0CHxymVNbVtNY-qfZjyqXex-aaxd8H-PujJmacRMhhtBbuJ4YjaGZDevRWb5FHZDzvLvI7rIeZPPXx2rTGPJTBcaiUmV17VrQH7QkzcGkNH~bCTO5ofhoS~bTfn0K1Td1EM~TgMVLFHDL4sKAbO8tSqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028494&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus sequences of the AbrB-binding sequences detected in the top 10th percentile threshold. Each group of 300 signals in the 20 sets of AbrB-binding sequences were ranked from high to low by binding intensities and were analysed by Bipad to detect respective consensus sequences. The number of sequences containing the motif is shown in parallel.</p></div></div><div content-id="DST013F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/m_dst01305.jpeg?Expires=2147483647&Signature=LHsqvWeofuja3QdNVDWhT-ZZdVO07LmQbDPQagwgev4IZlucZpu~3AauFBjE~JbK5ee8ArWYdjgcijcUBm0ZOGFL5KSck7fZvKMypKDjtt4RwgNPdAHNpgbx0th~1S4gKBGRkCBvW3YdRO-sG4x8SN2VOlSYemc-ONRlQLkC-p1RKSIRr-Q8ZljVelRaVXTge78Z1qU6w7SNPtwZXr1~rlOWpbKe4ExQ13ODN-rwXMek-UDY1f2GLBUBsCb9P1m3MV~SHodxx3FlHvtZE8M3CnVwGip8LVeOO~jU2Ybw2NMAjSkEqtgJfJuUFW~dhKpvabhi6FidVlNSZotYbe84~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus sequences of the AbrB-binding sequences detected in the top 10th percentile threshold. Each group of 300 signals in the 20 sets of AbrB-binding sequences were ranked from high to low by binding intensities and were analysed by Bipad to detect respective consensus sequences. The number of sequences containing the motif is shown in parallel." path-from-xml="dst01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5028494/dst01305.jpeg" path-from-xml="dst01305.jpeg" target="_blank">View large</a><a section="5028494" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst01305.jpeg?Expires=2147483647&Signature=LyMFrWWyJqqsbYDP0QPyOuDmrRN7~nikp7jyoRTiaL5xwPUn7nbZ-TKN6FYNiCEswLQlZ7ADLjYAJYzmpREqVY0jYkemfdKRIgXQfENWHuA0nQ20hvTXqEK55qjMD99XmIycR3z1qyxb6XYOov7vee4rrsuMa6CBj0R7yhC8rOKSgnJ0i39f14~gQTq9Aj0CHxymVNbVtNY-qfZjyqXex-aaxd8H-PujJmacRMhhtBbuJ4YjaGZDevRWb5FHZDzvLvI7rIeZPPXx2rTGPJTBcaiUmV17VrQH7QkzcGkNH~bCTO5ofhoS~bTfn0K1Td1EM~TgMVLFHDL4sKAbO8tSqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5028494&ar=353411&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus sequences of the AbrB-binding sequences detected in the top 10th percentile threshold. Each group of 300 signals in the 20 sets of AbrB-binding sequences were ranked from high to low by binding intensities and were analysed by Bipad to detect respective consensus sequences. The number of sequences containing the motif is shown in parallel.</p></div></div>                            <h2 scrollto-destination=5028495 id="5028495" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We demonstrate here that, by mapping the sequences of ∼50 bp fragments co-purified with AbrB after <em>in situ</em> DNase I digestion of genomic DNA, <em>in vivo</em> AbrB-binding sites could be determined with a resolution comparable with that of <em>in vitro</em> footprinting. Furthermore, comprehensive and precise information on the DNA sequences that AbrB binds gave us a clear view of AbrB binding on the <em>B. subtilis</em> genome—it would stably bind to bipartite TGGNA motifs, but it also interacted with many single TGGNA motifs on the genome.</p><p><em>In vitro</em> DNase I footprinting has currently been one of the most widely used methods to determine at high resolution the precise DNA sequences bound by transcription factors and other DNA-binding proteins. However, this method is laborious and can be performed against only a few DNA sequence targets in one experiment. In addition, the synthetic conditions under which DNase I foorprinting assays have been conducted risks leading to artifactual results for several reasons, e.g. the use of purified proteins that are not modified as would occur <em>in vivo</em> and may not work in the same way, the low-ionic strength of solutions used in <em>in vitro</em> footprinting experiments that may allow non-specific DNA–protein interactions, the use of short DNA sequences that may lack the secondary structure of DNA found <em>in vivo</em>, experiments conducted at non-physiological temperatures, and the absence of essential effectors, which may impair the specific binding of the protein to the corresponding DNA sequence. In contrast, in the GeF-seq method, DNA–protein interactions in the nucleoid are stabilized in the living cells by formaldehyde treatment, and then DNA digestion is carried out <em>in situ</em> to retain the native DNA-binding state of the target protein. Thus, the GeF-seq method identifies the actual DNA-binding sequences of target proteins across the whole genome simultaneously, with minimal risk of artefacts, and at high resolution, which is comparable with that of <em>in vitro</em> footprinting. In analysing the resolution of the method, we confirmed that 21 AbrB-binding regions we found using GeF-seq were consistent with <em>in vitro</em> footprinting results that have been reported previously (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). It should also be noted that some AbrB-binding sites previously detected by <em>in vitro</em> footprinting (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were not detected using GeF-seq. Although it is possible that such regions might be occupied by AbrB under some specific conditions, AbrB binding to these DNA sequences may occur only in <em>in vitro</em> experimental conditions.</p><p>Here, we detected not only bipartite TGGNA motifs in probable high-affinity AbrB-binding sites, but also single TGGNA motifs in possible low-affinity AbrB-binding sites, that would be generally ignored as non-specific, using the binding-site prediction software. Such low-affinity AbrB binding may not be biologically important, but it is possible that those binding sites may have a role to concentrate AbrB molecules on the nucleoid to increase the chance of finding high-affinity binding sites, which are directly involved in gene regulation.<sup><a reveal-id="DST013C25" data-open="DST013C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We usually use a threshold value to discriminate ‘real’ protein-binding peaks on the genome and possible ‘artificial’ binding peaks in ChIP-chip and ChIP-seq experiments. However, our results clearly demonstrated that the use of threshold values could discard important information. Our results suggest that comprehensive and precise information on protein-binding sequences obtained by GeF-seq analysis, in combination with the identification of consensus sequences in them, would give us a clear and comprehensive view of protein binding on the genome.</p><p>Specifically, we clearly demonstrate here that the consensus sequence for the high-affinity AbrB binding is comprised of bipartite TGGNA motifs gapped by a 4- or 5-bp AT-rich sequence arranged in direct, reverse direct, inverted, and everted repeat orientations. This result is consistent with a previous <em>in vitro</em> SELEX study,<sup><a reveal-id="DST013C17" data-open="DST013C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and our informatics analysis showing that various bipartite motifs are enriched in AbrB-binding regions determined by ChIP-chip.<sup><a reveal-id="DST013C14" data-open="DST013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Thus, the GeF-seq results reported here show, for the first time, the highly flexible proposed consensus sequences, which are actually recognized by AbrB molecules in <em>in vivo</em>. Previous structural modelling of AbrB bound to the target DNA sequence indicated that the AbrB tetramer would interact with ∼20 bp sequences,<sup><a reveal-id="DST013C15" data-open="DST013C15" class="link link-ref link-reveal xref-bibr">15</a></sup> whereas <em>in vitro</em> footprinting studies detected a wider range of binding regions from 25 to 80 bp. In this study, GeF-seq also detected a similar range of AbrB-binding regions from 27 to 80 bp in size. When the positions of the bipartite motifs within the binding sequences are depicted, the motifs are usually located in the middle of the binding sequences, but some are not centrally located in the long binding sequences (Fig. <a reveal-id="DST013F4" data-open="DST013F4" class="link link-reveal link-table xref-fig">4</a>C, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Interestingly, we observed that the binding region is generally composed of multiple TGGNA motifs almost covering the full length of the sequenced region (data not shown), suggesting that higher oligomers of AbrB may interact with multiple TGGNA motifs.</p><p>Here, we have demonstrated that GeF-seq is a powerful tool for helping to understand the <em>in vivo</em> distribution of DNA-binding proteins on the genome. However, several issues remain to be explored, in order to fully establish the GeF-seq method. (i) We have not yet examined how DNase I digestion conditions would affect the results, although the results shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1 and S2</a> suggest that the GeF-seq results would be robust against changes in DNase I digestion conditions. (ii) We empirically selected criteria to map read sequences and to define protein-binding sites on the genome. Further improvements in the sequence data processing algorithms are desirable to automate this process. (iii) The Bipad program outputs one consensus sequence for each input sequence, and a method to identify multiple motifs in each sequence is desirable. (iv) GeF-seq data suggest that protein-binding signal intensities to the genome should correlate with protein-binding affinities to the cognate target sequences, but this needs to be shown experimentally. (v) GeF-seq has successfully determined protein-binding sites across a bacterial genome, but examination of whether this method is applicable for much larger genomes of higher organisms is necessary.</p>                            <h2 scrollto-destination=5028501 id="5028501" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5028503 id="5028503" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work has been supported by the Advanced Low Carbon Technology Research and Development Program (ALCA) of the Japan Science and Technology Agency (JST) and a UK Royal Society International Joint Project to T.O. and J.L.H. Interactions between the authors' laboratories have been facilitated by a BBSRC/JST Japan Partnering Award.</p>                            <h2 scrollto-destination=5028505 id="5028505" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr C. Bi for useful advice on motifs analysis using BiPad web server.</p>                            <h2 scrollto-destination=5028507 id="5028507" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST013C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">T.H.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genome-wide analysis of protein-DNA interactions</div>, <div class="source">Annu. Rev. Genomics Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">81</div>-<div class="lpage">102</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.genom.7.080505.115634" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.genom.7.080505.115634</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20protein-DNA%20interactions&amp;author=T.H.%20Kim&amp;author=B.%20Ren&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&amp;volume=7&amp;pages=81-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genom.7.080505.115634" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genom.7.080505.115634" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genom.7.080505.115634"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16722805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">High-resolution profiling of histone methylations in the human genome</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">823</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.05.009" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2007.05.009</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20profiling%20of%20histone%20methylations%20in%20the%20human%20genome&amp;author=A.%20Barski&amp;author=S.%20Cuddapah&amp;author=K.%20Cui&amp;publication_year=2007&amp;journal=Cell&amp;volume=129&amp;pages=823-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.05.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.05.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.05.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robertson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bainbridge</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide profiles of STAT1 DNA association using chromatin immunoprecipitation and massively parallel sequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">651</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth1068" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth1068</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiles%20of%20STAT1%20DNA%20association%20using%20chromatin%20immunoprecipitation%20and%20massively%20parallel%20sequencing&amp;author=G.%20Robertson&amp;author=M.%20Hirst&amp;author=M.%20Bainbridge&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=651-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17558387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mikkelsen</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Ku</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jaffe</div> <div class="given-names">D.B.</div></div>, et al. </div><div class="article-title">Genome-wide maps of chromatin state in pluripotent and lineage-committed cells</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">448</div> (pg. <div class="fpage">553</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature06008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature06008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20maps%20of%20chromatin%20state%20in%20pluripotent%20and%20lineage-committed%20cells&amp;author=T.S.%20Mikkelsen&amp;author=M.%20Ku&amp;author=D.B.%20Jaffe&amp;publication_year=2007&amp;journal=Nature&amp;volume=448&amp;pages=553-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17603471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genome-wide mapping of in vivo protein-DNA interactions</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1497</div>-<div class="lpage">502</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1141319" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1141319</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20in%20vivo%20protein-DNA%20interactions&amp;author=D.S.%20Johnson&amp;author=A.%20Mortazavi&amp;author=R.M.%20Myers&amp;author=B.%20Wold&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1497-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1141319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1141319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1141319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pepke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Computation for ChIP-seq and RNA-seq studies</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">S22</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1371" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1371</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computation%20for%20ChIP-seq%20and%20RNA-seq%20studies&amp;author=S.%20Pepke&amp;author=B.%20Wold&amp;author=A.%20Mortazavi&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=S22-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19844228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jothi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Barski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genome-wide identification of in vivo protein-DNA binding sites from ChIP-Seq data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">5221</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn488" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn488</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20in%20vivo%20protein-DNA%20binding%20sites%20from%20ChIP-Seq%20data&amp;author=R.%20Jothi&amp;author=S.%20Cuddapah&amp;author=A.%20Barski&amp;author=K.%20Cui&amp;author=K.%20Zhao&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=5221-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18684996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kharchenko</div> <div class="given-names">P.V.</div></div>,  <div class="name"><div class="surname">Tolstorukov</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">Design and analysis of ChIP-seq experiments for DNA-binding proteins</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1351</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.1508" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.1508</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20and%20analysis%20of%20ChIP-seq%20experiments%20for%20DNA-binding%20proteins&amp;author=P.V.%20Kharchenko&amp;author=M.Y.%20Tolstorukov&amp;author=P.J.%20Park&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1351-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029915" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">Model-based analysis of ChIP-Seq (MACS)</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R137</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2008-9-9-r137</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Model-based%20analysis%20of%20ChIP-Seq%20(MACS)&amp;author=Y.%20Zhang&amp;author=T.%20Liu&amp;author=C.A.%20Meyer&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R137" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-9-r137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-9-r137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-9-r137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18798982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rhee</div> <div class="given-names">H.S.,</div></div>,  <div class="name"><div class="surname">Pugh</div> <div class="given-names">B.F.</div></div>. </div><div class="article-title">Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution</div>, <div class="source">Cell</div> , <div class="year">2011</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">1408</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2011.11.013" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2011.11.013</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20genome-wide%20protein-DNA%20interactions%20detected%20at%20single-nucleotide%20resolution&amp;author=H.S.%2C%20Rhee&amp;author=B.F.%20Pugh&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=1408-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.11.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22153082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hampshire</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Rusling</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Broughton-Head</div> <div class="given-names">V.J.,</div></div>,  <div class="name"><div class="surname">Fox</div> <div class="given-names">K.R.</div></div>. </div><div class="article-title">Footprinting: a method for determining the sequence selectivity, affinity and kinetics of DNA-binding ligands</div>, <div class="source">Methods</div> , <div class="year">2007</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">128</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ymeth.2007.01.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ymeth.2007.01.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Footprinting%3A%20a%20method%20for%20determining%20the%20sequence%20selectivity%2C%20affinity%20and%20kinetics%20of%20DNA-binding%20ligands&amp;author=A.J.%20Hampshire&amp;author=D.A.%20Rusling&amp;author=V.J.%2C%20Broughton-Head&amp;author=K.R.%20Fox&amp;publication_year=2007&amp;journal=Methods&amp;volume=42&amp;pages=128-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2007.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2007.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2007.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17472895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vora</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hottes</div> <div class="given-names">A.K.,</div></div>,  <div class="name"><div class="surname">Tavazoie</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Protein occupancy landscape of a bacterial genome</div>, <div class="source">Mol. Cell.</div> , <div class="year">2009</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">247</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.molcel.2009.06.035" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.molcel.2009.06.035</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20occupancy%20landscape%20of%20a%20bacterial%20genome&amp;author=T.%20Vora&amp;author=A.K.%2C%20Hottes&amp;author=S.%20Tavazoie&amp;publication_year=2009&amp;journal=Mol.%20Cell.&amp;volume=35&amp;pages=247-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2009.06.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2009.06.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2009.06.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19647521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ogura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yoshimura</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Distribution of stable DnaA-binding sites on the <em>Bacillus subtilis</em> genome detected using a modified ChIP-chip method</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">155</div>-<div class="lpage">68</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsm017" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsm017</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20stable%20DnaA-binding%20sites%20on%20the%20Bacillus%20subtilis%20genome%20detected%20using%20a%20modified%20ChIP-chip%20method&amp;author=S.%20Ishikawa&amp;author=Y.%20Ogura&amp;author=M.%20Yoshimura&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chumsakul</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide binding profiles of the <em>Bacillus subtilis</em> transition state regulator AbrB and its homolog Abh reveals their interactive role in transcriptional regulation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">414</div>-<div class="lpage">28</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkq780" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkq780</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20binding%20profiles%20of%20the%20Bacillus%20subtilis%20transition%20state%20regulator%20AbrB%20and%20its%20homolog%20Abh%20reveals%20their%20interactive%20role%20in%20transcriptional%20regulation&amp;author=O.%20Chumsakul&amp;author=H.%20Takahashi&amp;author=T.%20Oshima&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=414-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20817675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sullivan</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Bobay</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Kojetin</div> <div class="given-names">D.J.</div></div>, et al. </div><div class="article-title">Insights into the nature of DNA binding of AbrB-like transcription factors</div>, <div class="source">Structure</div> , <div class="year">2008</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1702</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.str.2008.08.014" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.str.2008.08.014</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20nature%20of%20DNA%20binding%20of%20AbrB-like%20transcription%20factors&amp;author=D.M.%20Sullivan&amp;author=B.G.%20Bobay&amp;author=D.J.%20Kojetin&amp;publication_year=2008&amp;journal=Structure&amp;volume=16&amp;pages=1702-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.str.2008.08.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.str.2008.08.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.str.2008.08.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19000822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strauch</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Delineation of AbrB-binding sites on the <em>Bacillus subtilis</em> spo0H, kinB, ftsAZ, and pbpE promoters and use of a derived homology to identify a previously unsuspected binding site in the bsuB1 methylase promote</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1995</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">6999</div>-<div class="lpage">7002</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Delineation%20of%20AbrB-binding%20sites%20on%20the%20Bacillus%20subtilis%20spo0H%2C%20kinB%2C%20ftsAZ%2C%20and%20pbpE%20promoters%20and%20use%20of%20a%20derived%20homology%20to%20identify%20a%20previously%20unsuspected%20binding%20site%20in%20the%20bsuB1%20methylase%20promote&amp;author=M.A.%20Strauch&amp;publication_year=1995&amp;journal=J.%20Bacteriol.&amp;volume=177&amp;pages=6999-7002" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7592498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Strauch</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">In vitro selection of optimal AbrB-binding sites: comparison to known in vivo sites indicates flexibility in AbrB binding and recognition of three-dimensional DNA structures</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1996</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">145</div>-<div class="lpage">58</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-2958.1996.358882.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-2958.1996.358882.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vitro%20selection%20of%20optimal%20AbrB-binding%20sites%3A%20comparison%20to%20known%20in%20vivo%20sites%20indicates%20flexibility%20in%20AbrB%20binding%20and%20recognition%20of%20three-dimensional%20DNA%20structures&amp;author=K.%20Xu&amp;author=M.A.%20Strauch&amp;publication_year=1996&amp;journal=Mol.%20Microbiol.&amp;volume=19&amp;pages=145-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.358882.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1996.358882.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1996.358882.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8821944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strauch</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Bobay</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Cavanagh</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Le Breton</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Abh and AbrB control of <em>Bacillus subtilis</em> antimicrobial gene expression</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2007</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">7720</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/JB.01081-07" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/JB.01081-07</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abh%20and%20AbrB%20control%20of%20Bacillus%20subtilis%20antimicrobial%20gene%20expression&amp;author=M.A.%20Strauch&amp;author=B.G.%20Bobay&amp;author=J.%20Cavanagh&amp;author=F.%20Yao&amp;author=A.%20Wilson&amp;author=Y.%20Le%20Breton&amp;publication_year=2007&amp;journal=J.%20Bacteriol.&amp;volume=189&amp;pages=7720-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01081-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01081-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01081-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17720793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bi</div> <div class="given-names">C.,</div></div>,  <div class="name"><div class="surname">Rogan</div> <div class="given-names">P.K.</div></div>. </div><div class="article-title">BIPAD: a web server for modeling bipartite sequence elements</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">76</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-7-76" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-7-76</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BIPAD%3A%20a%20web%20server%20for%20modeling%20bipartite%20sequence%20elements&amp;author=C.%2C%20Bi&amp;author=P.K.%20Rogan&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-76" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-76" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-76"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16503993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wyrick</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Rogers</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Isolation and characterization of cell wall-defective variants of <em>Bacillus subtilis</em> and <em>Bacillus licheniformis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1973</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">456</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20cell%20wall-defective%20variants%20of%20Bacillus%20subtilis%20and%20Bacillus%20licheniformis&amp;author=P.B.%20Wyrick&amp;author=H.J.%20Rogers&amp;publication_year=1973&amp;journal=J.%20Bacteriol.&amp;volume=116&amp;pages=456-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4126820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Morimoto</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Sequence-specific error profile of Illumina sequencers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> pg. <div class="fpage">e90</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkr344" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkr344</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-specific%20error%20profile%20of%20Illumina%20sequencers&amp;author=K.%20Nakamura&amp;author=T.%20Oshima&amp;author=T.%20Morimoto&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21576222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Helden</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Regulatory sequence analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3593</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkg567" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkg567</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20sequence%20analysis%20tools&amp;author=J.%20Van%20Helden&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3593-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sierro</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">de Hoon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">DBTBS: a database of transcriptional regulation in <em>Bacillus subtilis</em> containing upstream intergenic conservation information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D93</div>-<div class="lpage">96</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm910" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm910</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTBS%3A%20a%20database%20of%20transcriptional%20regulation%20in%20Bacillus%20subtilis%20containing%20upstream%20intergenic%20conservation%20information&amp;author=N.%20Sierro&amp;author=Y.%20Makita&amp;author=M.%20de%20Hoon&amp;author=K.%20Nakai&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D93-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm910" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm910" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm910"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17962296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Strauch</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Stanley-Wall</div> <div class="given-names">N.R.</div></div>. </div><div class="article-title">SigmaX is involved in controlling <em>Bacillus subtilis</em> biofilm architecture through the AbrB homologue Abh</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2009</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">6822</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/JB.00618-09" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/JB.00618-09</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SigmaX%20is%20involved%20in%20controlling%20Bacillus%20subtilis%20biofilm%20architecture%20through%20the%20AbrB%20homologue%20Abh&amp;author=E.J.%20Murray&amp;author=M.A.%20Strauch&amp;author=N.R.%20Stanley-Wall&amp;publication_year=2009&amp;journal=J.%20Bacteriol.&amp;volume=191&amp;pages=6822-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00618-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00618-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00618-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19767430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST013C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST013C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hammar</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Leroy</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Mahmutovic</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Marklund</div> <div class="given-names">E.G.</div></div>,  <div class="name"><div class="surname">Berg</div> <div class="given-names">O.G.</div></div>,  <div class="name"><div class="surname">Elf</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The lac repressor displays facilitated diffusion in living cells</div>, <div class="source">Science</div> , <div class="year">2012</div>, vol. <div class="volume">336</div> (pg. <div class="fpage">1595</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1221648" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1221648</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20lac%20repressor%20displays%20facilitated%20diffusion%20in%20living%20cells&amp;author=P.%20Hammar&amp;author=P.%20Leroy&amp;author=A.%20Mahmutovic&amp;author=E.G.%20Marklund&amp;author=O.G.%20Berg&amp;author=J.%20Elf&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=1595-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1221648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1221648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1221648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353411&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_tables.xls?Expires=2147483647&amp;Signature=ZPMXJaxnhBHlOKKxqEk~WRryEFeLOQ0rD0JHW9eukUH~p~290d6kagcEzJDpAM7yDS81zr2ETXUDzL-0NymE04HEbMKdFsDcjCq8rOaWuz6rJnN7Uxcf4VESky9rH87TJ~1GGmiMkbqsRWr8N7VZLK9KA4AtfC7JjFdiWtXCcVGY~Cpw-GypcMZ0vcQciaiLdA5fjlUiMk0QMGxIBYTi8ESs4SbE0-BDxjuJ29CfFfl7bZnCSC855gIAo9u7I-ivyuoOntU7Fjl8vEJpdhjyE5vwI2ugy8xzlYqDXBYseTfn2M6iBFX046vJZ-IratZeNlQUJiUmA~Pdw3ThTcaYwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig6.pdf?Expires=2147483647&amp;Signature=AJktwtNyv-2DtHUqvYAFK3wC1h29Yqpbe51-J9c3SSeB5Mv~ms8JJCAP22jgKJN3tgCjnEvgT1zLSVz6~jh8Ce75UHHC1iXB9doSlZsIWOnULXF4alZOFSKxA7JnvLlhnI~Xv7m6pvZDhTAp1ZZwvOBBy99Q~Uei53BOeOccVXjXS5h0dIGq4cKR2SLqgG9nazqsVgdG9H-0z9B1XNQmb3uZXxufx4gM1e9krtGG5NIDGQdKw8-lJ6BXJvJFjEVuxjxqrSoYyIkHwjUQRARjmPjYpfFWiafZta7vHnrVUEsNjppEI5rRTCkNkxIouyLVAA9qt8eB7vhP3P5yHty63A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig5.pdf?Expires=2147483647&amp;Signature=Tktixwfz1l9104QG087gMbbmn84ENH72DeYcodPczuGYP~EBF64th9Oz75HUbtd9OJCF1nb6260htLkH7boj~CYUyTIjY6jh71-UmX-8XUsbcI0cC3oIxxGDNVKHMqXNOQEi6u6EKFtMDRjkkMNV443hqJc5AqeWhK7w5TXl4vg0zxLvyyPqb5Mz6PCuKBQoGbSjY~5-O9gOkYzJWQa-tq0xcPml5sIYJVKkGvvdCA9rlJxHDFq7bB73QNl7QnZHAbBYKt49yU-goS4-R8DDVVEVMBLokyhDQ~WdWm9Li9e0ABk3FzCgPEq29bmHJHCvooV2SmqPVtfNonnXN-qODQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig4.pdf?Expires=2147483647&amp;Signature=HTInyDdGLTS65aF0DW3bRJ~5O0-7r2cFpnXzlybZPWSje9y4Dkapwaap~4Yuev9RB0xBYSDcV3pJryn9NM1OQARHq4Ba1ApDM-f5Y3tFd5~Sq4uME92bA6XBSV33EIbGxk9ZpO2Y09H5NFYIdzRLbDgzdWCfTri48JdG3Eb54yWuLE4cbIfgX~XBtNEMKoTPbCwZ652GUpBK9KA84bF6aF7qBVu7NFGX~hWvdij~G24yrGFG9h3e0UamJdF6Vk02ovCBameLwyMPltzAoBpnGuyw4F8vZobl8NGvsrUMflWQA5ZyvpsKzYCUVjVuPYAxvjrow-reI6~1PrhMNQfIOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig3.pdf?Expires=2147483647&amp;Signature=vCKuiHJBUa2AzpRLM6TSoxVgKcgcgKj7vf8aHdu7kQlximn9xRdsGkh2gnaD~VFG6eyE9RNwoibSYhHhgfXL6Q7pWNjTTP4nAI7v0Cxek7iFhj13R8F0T-BdxXtIQqrYVobOJmaP0JyozcduR-m-8aNHg6JinsijuN47Y-zAHsQyg5aH-qHvOuMOzuPiAX~OG9CJFOCteMGIvUO5eDksB3pluk6PIYhUaP24i~dFe3p~LD6MxtYzZDqPmonXNUnFGRyUDBbsq4ZW~jZjZdEBQ6TiFVL1nC0HEwZe1DY0ZoFEqrurXU3lk6wOul5ASyz-eeGyxqhG4598ywp1hdadcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig2.pdf?Expires=2147483647&amp;Signature=PpkKuievcxibaSUUkylUk2GQCilh3N-ynyUl7EvEhIWCoj6ExNfZd-DLiILLyObsOSTyr44LF9TXy8OWHJqqGxaKx5ewLihO1YYhIEKGpnFl0H14vaRbZEkroJMcf0sF-Ua1jkY2c3lBelCnffAbEiZXaqFxVg48RsXKzBY~LCzQup2svfZVPy78~vg7F0yJheAxpH3abN4gDqwL6SCe2WUl3g404WCn7~YJ6GU8Ldub-oRVgzLTZUR4P07CusjfSapw2LJm1QB5t~wiZwMmSv4E4acZuskBuegQ7OZSvLEWoDGgY2R5PS6cCA2HZ~tj0iEBiVS5U32hLM~V-RN9mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp_fig1.pdf?Expires=2147483647&amp;Signature=InIHaBJBP8n2pDsu9Oxd5elOjBSZ2-NeLqbemyAQr88N~cuJ8Rqim9X0BfcwTIVrf11Lj5Dj6DSz3j8p97gEhoc1LLUSrmCJ2-gCemU97yqEGQI105LuAYKZgwoSETBstUACvuCcqmt~TtD8zg8S7HvlAP73Cype-O8KKzeuvx-ejv3yuJbCNqjD-r-1US8OCWKaGwO9CzVd6QWvEYiNzMqT5mj0WiC6rQjAmByn0XBAM0nq9ZSiirlZztLigovJ-btAUsack8wENZHCc2uQRPHACHCMasrQfibOOxcLGIZJzUn-rOZupDawLzsWGOhfRSyOM8H~TJhimLnafYdFvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/4/10.1093/dnares/dst013/2/dst013-dst013supp.doc?Expires=2147483647&amp;Signature=VKAL1bDeVGdzyaZvuIVKz2Av~glaH7bWxpRvwD97ktvgU2Zid6B1pqDmHWZNopalgx~2gmw~n4PBH9RVPGLOo1XQRUUCazVD5TZw4iLylCaF96atuB0Jbpo5tRS2tpW2lTfLMYYXW-~Y1FO50ZFv9H7NJWsaMIAYoxjKMO1hOJSy0Pv6aNhSQSegABH3Nh-ZGENE7cLo8kcMq-6VTG5ADr7P5ZgRtbHdcODyh-knUqiiEqT~n1gHJU0qj1DSFxiLKauirSZQQSyzkfNOd0lK2Z8517KFIOXNq0Gykkp0UT-bgEl5zqhj~P4v91FjyCqoO4qLgioUwK8Vpd68DPPjFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">400</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353411"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000324223800002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f4%2f325%2f353411">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000324223800002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F4%2F325%2F353411" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353411'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-Resolution Mapping of In vivo Genomic Transcription Factor Binding Sites Using In situ DNase I Footprinting and ChIP-seq" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f4%2f325%2f353411">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

