

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":2,"applicationTime":2044,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/3/317/4898127","@type":"ScholarlyArticle","name":"An efficient approach for the development of genome-specific markers in allohexaploid wheat ( Triticum aestivum  L.) and its application in the construction of high-density linkage maps of the D genome","datePublished":"2018-02-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2018-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy004","keywords":["<italic>Triticum aestivum</italic>","allohexaploid","next-generation sequencing","amplicon sequencing","wheat D genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/3/317/4898127","author":[{"name":"Ishikawa, Goro","affiliation":"Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan  Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan","@type":"Person"},{"name":"Saito, Mika","affiliation":"Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan","@type":"Person"},{"name":"Tanaka, Tsuyoshi","affiliation":"Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan","@type":"Person"},{"name":"Katayose, Yuichi","affiliation":"Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan","@type":"Person"},{"name":"Kanamori, Hiroyuki","affiliation":"Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan","@type":"Person"},{"name":"Kurita, Kanako","affiliation":"Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan","@type":"Person"},{"name":"Nakamura, Toshiki","affiliation":"Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan","@type":"Person"}],"description":"Abstract.  In common wheat, the development of genotyping platforms has been hampered by the large size of the genome, its highly repetitive elements and its al","pageStart":"317","pageEnd":"326","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An efficient approach for the development of genome-specific markers in allohexaploid wheat ( Triticum aestivum  L.) and its application in the construction of high-density linkage maps of the D genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="efficient approach for the development of genome-specific markers in allohexaploid wheat ( Triticum aestivum  L.) and its application in the construction of high-density linkage maps of the D genome" />
    <meta property="og:description" content="Abstract.  In common wheat, the development of genotyping platforms has been hampered by the large size of the genome, its highly repetitive elements and its al" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/3/317/4898127" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-02-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ishikawa, Goro" /><meta name="citation_author_institution" content="Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan" /><meta name="citation_author_institution" content="Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan" /><meta name="citation_author" content="Saito, Mika" /><meta name="citation_author_institution" content="Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan" /><meta name="citation_author" content="Tanaka, Tsuyoshi" /><meta name="citation_author_institution" content="Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan" /><meta name="citation_author" content="Katayose, Yuichi" /><meta name="citation_author_institution" content="Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan" /><meta name="citation_author" content="Kanamori, Hiroyuki" /><meta name="citation_author_institution" content="Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan" /><meta name="citation_author" content="Kurita, Kanako" /><meta name="citation_author_institution" content="Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan" /><meta name="citation_author" content="Nakamura, Toshiki" /><meta name="citation_author_institution" content="Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan" /><meta name="citation_title" content="An efficient approach for the development of genome-specific markers in allohexaploid wheat (Triticum aestivum L.) and its application in the construction of high-density linkage maps of the D genome" /><meta name="citation_firstpage" content="317" /><meta name="citation_lastpage" content="326" /><meta name="citation_doi" content="10.1093/dnares/dsy004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2018/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Array-based high-throughput DNA markers for crop improvement;  citation_journal_title=Heredity;  citation_year=2008;  citation_volume=101;  citation_pages=5-18; " /><meta name="citation_reference" content="citation_title=Whole-genome strategies for marker-assisted plant breeding;  citation_journal_title=Mol. Breed;  citation_year=2012;  citation_volume=29;  citation_pages=833-54; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=8057-62; " /><meta name="citation_reference" content="citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90,000 single nucleotide polymorphism array;  citation_journal_title=Plant Biotechnol. J;  citation_year=2014;  citation_volume=12;  citation_pages=787-96; " /><meta name="citation_reference" content="citation_title=A high-density genetic map with array-based markers facilitates structural and quantitative trait locus analyses of the common wheat genome;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=555-67; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis identified SNPs closely linked to a gene resistant to soil-borne wheat mosaic virus;  citation_journal_title=Theor. Appl. Genet;  citation_year=2014;  citation_volume=127;  citation_pages=1039-47; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping for stripe rust (Puccinia striiformis F. sp. tritici) in US Pacific Northwest winter wheat (Triticum aestivum L.);  citation_journal_title=Theor. Appl. Genet;  citation_year=2015;  citation_volume=128;  citation_pages=1083-101; " /><meta name="citation_reference" content="citation_title=High-density genetic linkage map construction and QTL mapping of grain shape and size in the wheat population Yanda1817 x Beinong6;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0118144" /><meta name="citation_reference" content="citation_title=Genetic architecture of resistance to stripe rust in a global winter wheat germplasm collection;  citation_journal_title=G3-Genes Genomes Genet;  citation_year=2016;  citation_volume=6;  citation_pages=2237-53; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis on pre-harvest sprouting resistance and grain color in U.S. winter wheat;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=794" /><meta name="citation_reference" content="citation_title=Identification and validation of single nucleotide polymorphic markers linked to Ug99 stem rust resistance in spring wheat;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0171963" /><meta name="citation_reference" content="citation_title=Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.);  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=727" /><meta name="citation_reference" content="citation_title=Association mapping and validation of QTLs for flour yield in the soft winter wheat variety Kitahonami;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e111337" /><meta name="citation_reference" content="citation_title=QTL analysis of spike morphological traits and plant height in winter wheat (Triticum aestivum L.) using a high-density SNP and SSR-based linkage map;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7;  citation_pages=1617" /><meta name="citation_reference" content="citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat;  citation_journal_title=Plant Biotechnol. J;  citation_year=2012;  citation_volume=10;  citation_pages=826-39; " /><meta name="citation_reference" content="citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e32253" /><meta name="citation_reference" content="citation_title=Characterization of a mini core collection of Japanese wheat varieties using single-nucleotide polymorphisms generated by genotyping-by-sequencing;  citation_journal_title=Breed Sci;  citation_year=2016;  citation_volume=66;  citation_pages=213-25; " /><meta name="citation_reference" content="citation_title=Ancient hybridizations among the ancestral genomes of bread wheat;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=1250092" /><meta name="citation_reference" content="citation_title=Analysis of the bread wheat genome using whole-genome shotgun sequencing;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=705-10; " /><meta name="citation_reference" content="citation_title=Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=1382-97; " /><meta name="citation_reference" content="citation_title=Using next generation sequencing for multiplexed trait-linked markers in wheat;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0143890" /><meta name="citation_reference" content="citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome; citation_author=IWGSC;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=1251788" /><meta name="citation_reference" content="citation_title=TriFLDB: a database of clustered full-length coding sequences from Triticeae with applications to comparative grass genomics;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=150;  citation_pages=1135-46; " /><meta name="citation_reference" content="citation_title=PCR-based landmark unique gene (PLUG) markers effectively assign homoeologous wheat genes to A, B and D genomes;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=135" /><meta name="citation_reference" content="citation_title=A chromosome bin map of 16,000 expressed sequence tag loci and distribution of genes among the three genomes of polyploid wheat;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=701-12; " /><meta name="citation_reference" content="citation_title=A physical, genetic and functional sequence assembly of the barley genome;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=711-6; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso;  citation_journal_title=Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Recent developments in the MAFFT multiple sequence alignment program;  citation_journal_title=Brief Bioinform;  citation_year=2008;  citation_volume=9;  citation_pages=286-98; " /><meta name="citation_reference" content="citation_title=Functional markers in wheat: current status and future prospects;  citation_journal_title=Theor. Appl. Genet;  citation_year=2012;  citation_volume=125;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=MapDisto: fast and efficient computation of genetic linkage maps;  citation_journal_title=Mol. Breed;  citation_year=2012;  citation_volume=30;  citation_pages=1231-5; " /><meta name="citation_reference" content="citation_title=Reconstruction of the synthetic W7984 x Opata M85 wheat reference population;  citation_journal_title=Genome;  citation_year=2011;  citation_volume=54;  citation_pages=875-82; " /><meta name="citation_reference" content="citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.);  citation_journal_title=Plant Biotechnol. J;  citation_year=2011;  citation_volume=9;  citation_pages=1086-99; " /><meta name="citation_reference" content="citation_title=Nucleotide diversity maps reveal variation in diversity among wheat genomes and chromosomes;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=702" /><meta name="citation_reference" content="citation_title=Localization of anchor loci representing five hundred annotated rice genes to wheat chromosomes using PLUG markers;  citation_journal_title=Theor. Appl. Genet;  citation_year=2009;  citation_volume=118;  citation_pages=499-514; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay;  citation_journal_title=Theor. Appl. Genet;  citation_year=2009;  citation_volume=119;  citation_pages=507-17; " /><meta name="citation_reference" content="citation_title=Comparative LD mapping using single SNPs and haplotypes identifies QTL for plant height and biomass as secondary traits of drought tolerance in maize;  citation_journal_title=Mol. Breeding;  citation_year=2012;  citation_volume=30;  citation_pages=407-18; " /><meta name="citation_reference" content="citation_title=Evolution of polyploid Triticum wheats under cultivation: the role of domestication, natural hybridization and allopolyploid speciation in their diversification;  citation_journal_title=Plant Cell Physiol;  citation_year=2011;  citation_volume=52;  citation_pages=750-64; " /><meta name="citation_reference" content="McIntosh R. A., Yamazaki Y., Dubcovsky J. , et al  2013 , Catalogue of gene symbols for wheat. https://shigen.nig.ac.jp/wheat/komugi/genes/symbolClassList.jsp (5 February 2018, date last accessed) " /><meta name="citation_reference" content="citation_title=Genome-wide association study for pre-harvest sprouting resistance in a large germplasm collection of Chinese wheat landraces;  citation_journal_title=Front. Plant Sci;  citation_year=2017;  citation_volume=8;  citation_pages=401" /><meta name="citation_reference" content="citation_title=Genome-wide association mapping of preharvest sprouting resistance in a diversity panel of European winter wheats;  citation_journal_title=J. Appl. Genet;  citation_year=2015;  citation_volume=56;  citation_pages=277-85; " /><meta name="citation_reference" content="citation_title=Mapping a gene conferring resistance to Wheat yellow mosaic virus in European winter wheat cultivar ‘Ibis’ (Triticum aestivum L.);  citation_journal_title=Euphytica;  citation_year=2010;  citation_volume=176;  citation_pages=223-9; " /><meta name="citation_reference" content="citation_title=Gene content and virtual gene order of barley chromosome 1H;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=151;  citation_pages=496-505; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/3/317/25072190/dsy004.pdf" /><meta name="description" content="Abstract.  In common wheat, the development of genotyping platforms has been hampered by the large size of the genome, its highly repetitive elements and its al" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/3/317/4898127" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/3/317/4898127" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/3/317/4898127");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy004';
    </script>
<title>
	efficient approach for the development of genome-specific markers in allohexaploid wheat (Triticum aestivum L.) and its application in the construction of high-density linkage maps of the D genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4898127" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeZyPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpa2F3YSwgR29ybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRvaG9rdSBBZ3JpY3VsdHVyYWwgUmVzZWFyY2ggQ2VudGVyLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8pLCBNb3Jpb2thLCBJd2F0ZSAwMjAtMDE5OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQ3JvcCBTY2llbmNlLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8pLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhaXRvLCBNaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVG9ob2t1IEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiAoTkFSTyksIE1vcmlva2EsIEl3YXRlIDAyMC0wMTk4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmFrYSwgVHN1eW9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQ3JvcCBTY2llbmNlLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8pLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthdGF5b3NlLCBZdWljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQ3JvcCBTY2llbmNlLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8pLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmFtb3JpLCBIaXJveXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBDcm9wIFNjaWVuY2UsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiAoTkFSTyksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VyaXRhLCBLYW5ha28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQ3JvcCBTY2llbmNlLCBOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24gKE5BUk8pLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FtdXJhLCBUb3NoaWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVG9ob2t1IEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIsIE5hdGlvbmFsIEFncmljdWx0dXJlIGFuZCBGb29kIFJlc2VhcmNoIE9yZ2FuaXphdGlvbiAoTkFSTyksIE1vcmlva2EsIEl3YXRlIDAyMC0wMTk4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW4gZWZmaWNpZW50IGFwcHJvYWNoIGZvciB0aGUgZGV2ZWxvcG1lbnQgb2YgZ2Vub21lLXNwZWNpZmljIG1hcmtlcnMgaW4gYWxsb2hleGFwbG9pZCB3aGVhdCAoVHJpdGljdW0gYWVzdGl2dW0gTC4pIGFuZCBpdHMgYXBwbGljYXRpb24gaW4gdGhlIGNvbnN0cnVjdGlvbiBvZiBoaWdoLWRlbnNpdHkgbGlua2FnZSBtYXBzIG9mIHRoZSBEIGdlbm9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN5MDA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcnJheS1iYXNlZCBoaWdoLXRocm91Z2hwdXQgRE5BIG1hcmtlcnMgZm9yIGNyb3AgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTUtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzdHJhdGVnaWVzIGZvciBtYXJrZXItYXNzaXN0ZWQgcGxhbnQgYnJlZWRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz04MzMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGNvbXBhcmF0aXZlIGRpdmVyc2l0eSB1bmNvdmVycyBtdWx0aXBsZSB0YXJnZXRzIG9mIHNlbGVjdGlvbiBmb3IgaW1wcm92ZW1lbnQgaW4gaGV4YXBsb2lkIHdoZWF0IGxhbmRyYWNlcyBhbmQgY3VsdGl2YXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz04MDU3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHBvbHlwbG9pZCB3aGVhdCBnZW5vbWljIGRpdmVyc2l0eSB1c2luZyBhIGhpZ2gtZGVuc2l0eSA5MCwwMDAgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGFycmF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9Nzg3LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSBnZW5ldGljIG1hcCB3aXRoIGFycmF5LWJhc2VkIG1hcmtlcnMgZmFjaWxpdGF0ZXMgc3RydWN0dXJhbCBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIGFuYWx5c2VzIG9mIHRoZSBjb21tb24gd2hlYXQgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTU1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBhbmFseXNpcyBpZGVudGlmaWVkIFNOUHMgY2xvc2VseSBsaW5rZWQgdG8gYSBnZW5lIHJlc2lzdGFudCB0byBzb2lsLWJvcm5lIHdoZWF0IG1vc2FpYyB2aXJ1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyNzsgIGNpdGF0aW9uX3BhZ2VzPTEwMzktNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIG1hcHBpbmcgZm9yIHN0cmlwZSBydXN0IChQdWNjaW5pYSBzdHJpaWZvcm1pcyBGLiBzcC4gdHJpdGljaSkgaW4gVVMgUGFjaWZpYyBOb3J0aHdlc3Qgd2ludGVyIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0xMDgzLTEwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGdlbmV0aWMgbGlua2FnZSBtYXAgY29uc3RydWN0aW9uIGFuZCBRVEwgbWFwcGluZyBvZiBncmFpbiBzaGFwZSBhbmQgc2l6ZSBpbiB0aGUgd2hlYXQgcG9wdWxhdGlvbiBZYW5kYTE4MTcgeCBCZWlub25nNjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDExODE0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYXJjaGl0ZWN0dXJlIG9mIHJlc2lzdGFuY2UgdG8gc3RyaXBlIHJ1c3QgaW4gYSBnbG9iYWwgd2ludGVyIHdoZWF0IGdlcm1wbGFzbSBjb2xsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMy1HZW5lcyBHZW5vbWVzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMjM3LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBhbmFseXNpcyBvbiBwcmUtaGFydmVzdCBzcHJvdXRpbmcgcmVzaXN0YW5jZSBhbmQgZ3JhaW4gY29sb3IgaW4gVS5TLiB3aW50ZXIgd2hlYXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTc5NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCB2YWxpZGF0aW9uIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGljIG1hcmtlcnMgbGlua2VkIHRvIFVnOTkgc3RlbSBydXN0IHJlc2lzdGFuY2UgaW4gc3ByaW5nIHdoZWF0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUwMTcxOTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbi0gYW5kIGdlbm9tZS1zcGVjaWZpYyBwYXR0ZXJucyBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGFuZCBTTlAgdmFyaWF0aW9uIGluIHNwcmluZyBhbmQgd2ludGVyIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTcyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG1hcHBpbmcgYW5kIHZhbGlkYXRpb24gb2YgUVRMcyBmb3IgZmxvdXIgeWllbGQgaW4gdGhlIHNvZnQgd2ludGVyIHdoZWF0IHZhcmlldHkgS2l0YWhvbmFtaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTEzMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYW5hbHlzaXMgb2Ygc3Bpa2UgbW9ycGhvbG9naWNhbCB0cmFpdHMgYW5kIHBsYW50IGhlaWdodCBpbiB3aW50ZXIgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKSB1c2luZyBhIGhpZ2gtZGVuc2l0eSBTTlAgYW5kIFNTUi1iYXNlZCBsaW5rYWdlIG1hcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTYxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXVsdGlwYXJlbnQgYWR2YW5jZWQgZ2VuZXJhdGlvbiBpbnRlci1jcm9zcyBwb3B1bGF0aW9uIGZvciBnZW5ldGljIGFuYWx5c2lzIGluIHdoZWF0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ODI2LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXBzIGZvciBiYXJsZXkgYW5kIHdoZWF0IHVzaW5nIGEgbm92ZWwgdHdvLWVuenltZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgYXBwcm9hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzIyNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGEgbWluaSBjb3JlIGNvbGxlY3Rpb24gb2YgSmFwYW5lc2Ugd2hlYXQgdmFyaWV0aWVzIHVzaW5nIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXMgZ2VuZXJhdGVkIGJ5IGdlbm90eXBpbmctYnktc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9MjEzLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmNpZW50IGh5YnJpZGl6YXRpb25zIGFtb25nIHRoZSBhbmNlc3RyYWwgZ2Vub21lcyBvZiBicmVhZCB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDU7ICBjaXRhdGlvbl9wYWdlcz0xMjUwMDkyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGJyZWFkIHdoZWF0IGdlbm9tZSB1c2luZyB3aG9sZS1nZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9NzA1LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZ2Vub21lLXdpZGUgZGV0ZWN0aW9uIGFuZCBjYXRhbG9naW5nIG9mIEVNUy1pbmR1Y2VkIG11dGF0aW9ucyB1c2luZyBleG9tZSBjYXB0dXJlIGFuZCBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEzODItOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIG5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGZvciBtdWx0aXBsZXhlZCB0cmFpdC1saW5rZWQgbWFya2VycyBpbiB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDE0Mzg5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tZS1iYXNlZCBkcmFmdCBzZXF1ZW5jZSBvZiB0aGUgaGV4YXBsb2lkIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSkgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SVdHU0M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ1OyAgY2l0YXRpb25fcGFnZXM9MTI1MTc4OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyaUZMREI6IGEgZGF0YWJhc2Ugb2YgY2x1c3RlcmVkIGZ1bGwtbGVuZ3RoIGNvZGluZyBzZXF1ZW5jZXMgZnJvbSBUcml0aWNlYWUgd2l0aCBhcHBsaWNhdGlvbnMgdG8gY29tcGFyYXRpdmUgZ3Jhc3MgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9MTEzNS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UENSLWJhc2VkIGxhbmRtYXJrIHVuaXF1ZSBnZW5lIChQTFVHKSBtYXJrZXJzIGVmZmVjdGl2ZWx5IGFzc2lnbiBob21vZW9sb2dvdXMgd2hlYXQgZ2VuZXMgdG8gQSwgQiBhbmQgRCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tZSBiaW4gbWFwIG9mIDE2LDAwMCBleHByZXNzZWQgc2VxdWVuY2UgdGFnIGxvY2kgYW5kIGRpc3RyaWJ1dGlvbiBvZiBnZW5lcyBhbW9uZyB0aGUgdGhyZWUgZ2Vub21lcyBvZiBwb2x5cGxvaWQgd2hlYXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2ODsgIGNpdGF0aW9uX3BhZ2VzPTcwMS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlzaWNhbCwgZ2VuZXRpYyBhbmQgZnVuY3Rpb25hbCBzZXF1ZW5jZSBhc3NlbWJseSBvZiB0aGUgYmFybGV5IGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTcxMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb2dyYW0gZm9yIGFubm90YXRpbmcgYW5kIHByZWRpY3RpbmcgdGhlIGVmZmVjdHMgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcywgU25wRWZmOiBTTlBzIGluIHRoZSBnZW5vbWUgb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgc3RyYWluIHcxMTE4OyBpc287ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZseTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODAtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgZGV2ZWxvcG1lbnRzIGluIHRoZSBNQUZGVCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgcHJvZ3JhbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJpZWYgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yODYtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgbWFya2VycyBpbiB3aGVhdDogY3VycmVudCBzdGF0dXMgYW5kIGZ1dHVyZSBwcm9zcGVjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz0xLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBEaXN0bzogZmFzdCBhbmQgZWZmaWNpZW50IGNvbXB1dGF0aW9uIG9mIGdlbmV0aWMgbGlua2FnZSBtYXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTIzMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbnN0cnVjdGlvbiBvZiB0aGUgc3ludGhldGljIFc3OTg0IHggT3BhdGEgTTg1IHdoZWF0IHJlZmVyZW5jZSBwb3B1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz04NzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQtc3BlY2lmaWMsIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbSBkaXNjb3ZlcnkgYW5kIGxpbmthZ2UgYW5hbHlzaXMgaW4gaGV4YXBsb2lkIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3RpZGUgZGl2ZXJzaXR5IG1hcHMgcmV2ZWFsIHZhcmlhdGlvbiBpbiBkaXZlcnNpdHkgYW1vbmcgd2hlYXQgZ2Vub21lcyBhbmQgY2hyb21vc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTcwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsaXphdGlvbiBvZiBhbmNob3IgbG9jaSByZXByZXNlbnRpbmcgZml2ZSBodW5kcmVkIGFubm90YXRlZCByaWNlIGdlbmVzIHRvIHdoZWF0IGNocm9tb3NvbWVzIHVzaW5nIFBMVUcgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTQ5OS01MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBnZW5vdHlwaW5nIGluIHBvbHlwbG9pZCB3aGVhdCB3aXRoIHRoZSBJbGx1bWluYSBHb2xkZW5HYXRlIGFzc2F5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE5OyAgY2l0YXRpb25fcGFnZXM9NTA3LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBMRCBtYXBwaW5nIHVzaW5nIHNpbmdsZSBTTlBzIGFuZCBoYXBsb3R5cGVzIGlkZW50aWZpZXMgUVRMIGZvciBwbGFudCBoZWlnaHQgYW5kIGJpb21hc3MgYXMgc2Vjb25kYXJ5IHRyYWl0cyBvZiBkcm91Z2h0IHRvbGVyYW5jZSBpbiBtYWl6ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQwNy0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHBvbHlwbG9pZCBUcml0aWN1bSB3aGVhdHMgdW5kZXIgY3VsdGl2YXRpb246IHRoZSByb2xlIG9mIGRvbWVzdGljYXRpb24sIG5hdHVyYWwgaHlicmlkaXphdGlvbiBhbmQgYWxsb3BvbHlwbG9pZCBzcGVjaWF0aW9uIGluIHRoZWlyIGRpdmVyc2lmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NzUwLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJNY0ludG9zaCBSLiBBLiwgWWFtYXpha2kgWS4sIER1YmNvdnNreSBKLiAsIGV0IGFsICAyMDEzICwgQ2F0YWxvZ3VlIG9mIGdlbmUgc3ltYm9scyBmb3Igd2hlYXQuIGh0dHBzOi8vc2hpZ2VuLm5pZy5hYy5qcC93aGVhdC9rb211Z2kvZ2VuZXMvc3ltYm9sQ2xhc3NMaXN0LmpzcCAoNSBGZWJydWFyeSAyMDE4LCBkYXRlIGxhc3QgYWNjZXNzZWQpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IGZvciBwcmUtaGFydmVzdCBzcHJvdXRpbmcgcmVzaXN0YW5jZSBpbiBhIGxhcmdlIGdlcm1wbGFzbSBjb2xsZWN0aW9uIG9mIENoaW5lc2Ugd2hlYXQgbGFuZHJhY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00MDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIG9mIHByZWhhcnZlc3Qgc3Byb3V0aW5nIHJlc2lzdGFuY2UgaW4gYSBkaXZlcnNpdHkgcGFuZWwgb2YgRXVyb3BlYW4gd2ludGVyIHdoZWF0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0yNzctODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYSBnZW5lIGNvbmZlcnJpbmcgcmVzaXN0YW5jZSB0byBXaGVhdCB5ZWxsb3cgbW9zYWljIHZpcnVzIGluIEV1cm9wZWFuIHdpbnRlciB3aGVhdCBjdWx0aXZhciDigJhJYmlz4oCZIChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz0yMjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBjb250ZW50IGFuZCB2aXJ0dWFsIGdlbmUgb3JkZXIgb2YgYmFybGV5IGNocm9tb3NvbWUgMUg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9NDk2LTUwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzMvMzE3LzI1MDcyMTkwL2RzeTAwNC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSW4gY29tbW9uIHdoZWF0LCB0aGUgZGV2ZWxvcG1lbnQgb2YgZ2Vub3R5cGluZyBwbGF0Zm9ybXMgaGFzIGJlZW4gaGFtcGVyZWQgYnkgdGhlIGxhcmdlIHNpemUgb2YgdGhlIGdlbm9tZSwgaXRzIGhpZ2hseSByZXBldGl0aXZlIGVsZW1lbnRzIGFuZCBpdHMgYWwiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ4OTgxMjcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkgJIsNdEIFp0GVzKPs3jHQAqGkQk=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f317%2f4898127" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f317%2f4898127" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109141" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&amp;Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/3">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118057467">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057467">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057469">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057469">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057475">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057475">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057495">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057495">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057509">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057509">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057516">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057516">Data Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057518">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057518">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057522">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057522">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057524">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057524">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/3/307/4831046">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/3/327/4898128">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An efficient approach for the development of genome-specific markers in allohexaploid wheat (<em>Triticum aestivum</em> L.) and its application in the construction of high-density linkage maps of the D genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Goro Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan</div><div class="aff">Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy004-cor1" data-open="dsy004-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 19 643 3514. Fax. +81 19 643 3514. Email: <a href="mailto:goro@affrc.go.jp" target="_blank">goro@affrc.go.jp</a> (G.I.), <a href="mailto:tnaka@affrc.go.jp" target="_blank">tnaka@affrc.go.jp</a> (T.N.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Goro+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Goro Ishikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mika Saito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mika+Saito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saito M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saito M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mika Saito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyoshi Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyoshi+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tsuyoshi Tanaka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichi Katayose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichi+Katayose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katayose Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katayose Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuichi Katayose</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Kanamori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Kanamori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanamori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanamori H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Kanamori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanako Kurita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki 305-8518, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanako+Kurita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurita K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurita K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kanako Kurita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshiki Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Tohoku Agricultural Research Center, National Agriculture and Food Research Organization (NARO), Morioka, Iwate 020-0198, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy004-cor1" data-open="dsy004-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 19 643 3514. Fax. +81 19 643 3514. Email: <a href="mailto:goro@affrc.go.jp" target="_blank">goro@affrc.go.jp</a> (G.I.), <a href="mailto:tnaka@affrc.go.jp" target="_blank">tnaka@affrc.go.jp</a> (T.N.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshiki+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshiki Nakamura</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy004-FM2"><div class="fn"><div class="footnote-content">Co-first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy004-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 317–326, <a href='https://doi.org/10.1093/dnares/dsy004'>https://doi.org/10.1093/dnares/dsy004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 February 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 October 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 February 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4898127" href="/dnaresearch/article-pdf/25/3/317/25072190/dsy004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Goro Ishikawa, Mika Saito, Tsuyoshi Tanaka, Yuichi Katayose, Hiroyuki Kanamori, Kanako Kurita, Toshiki Nakamura;  An efficient approach for the development of genome-specific markers in allohexaploid wheat (<em>Triticum aestivum</em> L.) and its application in the construction of high-density linkage maps of the D genome, <em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 317–326, <a href='https://doi.org/10.1093/dnares/dsy004'>https://doi.org/10.1093/dnares/dsy004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898127?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20efficient%20approach%20for%20the%20development%20of%20genome-specific%20markers%20in%20allohexaploid%20wheat%20(%26lt%3Bem%26gt%3BTriticum%20aestivum%26lt%3B%2Fem%26gt%3B%20L.)%20and%20its%20application%20in%20the%20construction%20of%20high-density%20linkage%20maps%20of%20the%20D%20genome&amp;publicationDate=2018-02-21&amp;volumeNum=25&amp;issueNum=3&amp;author=Ishikawa%2C%20Goro%3B%20Saito%2C%20Mika&amp;startPage=317&amp;endPage=326&amp;contentId=10.1093%2Fdnares%2Fdsy004&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f317%2f4898127" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118057467 id="118057467" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In common wheat, the development of genotyping platforms has been hampered by the large size of the genome, its highly repetitive elements and its allohexaploid nature. However, recent advances in sequencing technology provide opportunities to resolve these difficulties. Using next-generation sequencing and gene-targeting sequence capture, 12,551 nucleotide polymorphisms were detected in the common wheat varieties ‘Hatsumochi’ and ‘Kitahonami’ and were assigned to chromosome arms using International Wheat Genome Sequencing Consortium survey sequences. Because the number of markers for D genome chromosomes in commercially available wheat single nucleotide polymorphism arrays is insufficient, we developed markers using a genome-specific amplicon sequencing strategy. Approximately 80% of the designed primers successfully amplified D genome-specific products, suggesting that by concentrating on a specific subgenome, we were able to design successful markers as efficiently as could be done in a diploid species. The newly developed markers were uniformly distributed across the D genome and greatly extended the total coverage. Polymorphisms were surveyed in six varieties, and 31,542 polymorphic sites and 5,986 potential marker sites were detected in the D genome. The marker development and genotyping strategies are cost effective, robust and flexible and may enhance multi-sample studies in the post-genomic era in wheat.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Triticum aestivum&quot;"><em>Triticum aestivum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="allohexaploid">allohexaploid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing&quot;">next-generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;amplicon sequencing&quot;">amplicon sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;wheat D genome&quot;">wheat D genome</a></div>                            <h2 scrollto-destination=118057469 id="118057469" class="section-title" >1. Introduction</h2>
<p>High-throughput genotyping platforms are essential tools for various genetic studies that involve genetic mapping, genome-wide association studies, phylogenetic analyses, marker-assisted selection (MAS) and genomic selection.<a reveal-id="dsy004-B1" data-open="dsy004-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy004-B2" data-open="dsy004-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Recent advances in sequencing technologies have greatly facilitated the discovery of polymorphisms in the whole genome. The 9K iSelect<a reveal-id="dsy004-B3" data-open="dsy004-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and 90K iSelect<a reveal-id="dsy004-B4" data-open="dsy004-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> single-nucleotide polymorphism (SNP) arrays have been developed for allohexaploid wheat (<em>Triticum aestivum</em> L.) by transcriptome sequencing using next-generation sequencing (NGS) technology. These arrays comprise many gene-based SNPs that allow an individual plant to be genotyped at all sites simultaneously and tend to be robust marker platforms. Therefore, these arrays have been widely used for germplasm characterization and quantitative trait locus (QTL) mapping.<a reveal-id="dsy004-B5 dsy004-B6 dsy004-B7 dsy004-B8 dsy004-B9 dsy004-B10 dsy004-B11 dsy004-B12" data-open="dsy004-B5 dsy004-B6 dsy004-B7 dsy004-B8 dsy004-B9 dsy004-B10 dsy004-B11 dsy004-B12" class="link link-ref link-reveal xref-bibr"><sup>5–12</sup></a></p><p>However, array-based markers are inflexible and have a relatively high per-sample cost. The number of polymorphisms detected by the markers on an array chip tends to substantially decrease in lines that are genetically distant from those used to design the array. When the 9K iSelect array, which was designed based on SNP information of American and Australian varieties, is used in Japanese varieties, a low polymorphic rate is detected, particularly in varieties grown in the southern regions (latitude 33–37°N).<a reveal-id="dsy004-B13" data-open="dsy004-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> In this region, the wheat harvest season overlaps with the rainy season, and mainly domestic genetic resources have been used to breed varieties with resistance to pre-harvest sprouting and <em>Fusarium</em> head blight. On the other hand, in northern regions from Hokkaido (42–45°N) to Tohoku (37–41°N), there is less rain during the harvest season and several North American varieties were introduced to improve flour quality. In many cases, the polymorphisms detected in American and Australian varieties are not conserved in Japanese varieties. Furthermore, when Axiom HD wheat genotyping arrays (Affymetrix, Santa Clara, CA, USA) were used in Japanese materials, only 3.1% of the markers were categorized as ‘PolyHighResolution’, which defines markers with good cluster resolution and at least two examples of the minor allele (unpublished). However, a different set of markers in this array were of high quality and were more polymorphic in Japanese materials than in samples from varieties processed by the WISP (<a class="link link-uri" href="http://www.wheatisp.org/" target="_blank">http://www.wheatisp.org/</a> (5 February 2018, date last accessed)) in the design of the array. Thus, the usefulness of array-based markers largely depends on the source of the polymorphisms. Additionally, genotype calls remain complicated because of the polyploid nature of wheat, and allele data should be interpreted with caution. These disadvantages are deleterious for MAS in wheat breeding programs because a cost-effective genotyping platform with a rapid turnaround time, low per-sample cost and very low rate of missing data is required for efficient MAS. Furthermore, MAS usually relies on a set of specific markers for specific QTLs and genes using a medium throughput system, rather than random genes or markers.</p><p>Available array platforms are limited in their application to the D genome, because the genetic coverage of the D genome is highly inadequate.<a reveal-id="dsy004-B14" data-open="dsy004-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> This tendency is also observed using other whole-genome genotyping platforms, such as DArT and genotyping by sequencing (GBS), which also have comparatively fewer D genome markers.<a reveal-id="dsy004-B15 dsy004-B16 dsy004-B17" data-open="dsy004-B15 dsy004-B16 dsy004-B17" class="link link-ref link-reveal xref-bibr"><sup>15–17</sup></a> Marcussen et al. reported that the origin of the D genome was 1–2 million years later than that of the A and B genomes.<a reveal-id="dsy004-B18" data-open="dsy004-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Furthermore, the D subgenome of hexaploid wheat was established by polyploidization ∼0.4 million years later than the other subgenomes. The shorter time for accumulation of nucleotide polymorphisms induced by natural mutations might influence the scarcity of markers in the D genome. Therefore, to conduct studies focused on chromosomes in the D genome, new markers must be developed. However, the discovery of polymorphisms in wheat is hampered by its large genome size (16 Gb) and high repeat content (approximately 80%).<a reveal-id="dsy004-B19" data-open="dsy004-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Although the cost of sequencing is decreasing, sequencing the whole genome remains prohibitively expensive, particularly in species with large genomes. Henry et al.<a reveal-id="dsy004-B20" data-open="dsy004-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> reported that exome capture combined with NGS can be used to successfully and efficiently detect Ethyl methanesulfonate-induced mutations. These authors compared the cost effectiveness among plant species with various genome sizes and concluded that this approach is increasingly advantageous as the genome size increases. Therefore, wheat is a suitable material for sequence capture in terms of cost effectiveness. Furthermore, this technique is useful in controlling the distribution of targets along chromosomes.</p><p>Genotyping by multiplexing amplicon sequencing (GBMAS) (Lab protocols of Schnable Lab., Iowa State University: <a class="link link-uri" href="http://schnablelab.plantgenomics.iastate.edu/resources/protocols/" target="_blank">http://schnablelab.plantgenomics.iastate.edu/resources/protocols/</a> (5 February 2018, date last accessed)) and genome-tagged amplification (GTA)<a reveal-id="dsy004-B21" data-open="dsy004-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> are new genotyping platforms that combine multiplexed PCR and multiplexed samples using bar codes with NGS. These methods appear to be suitable for MAS due to higher flexibility in choosing combinations of marker number and sample number. However, designing PCR primers for wheat is hindered by the close homology of the three genomes (A, B and D) and the high sequence similarity among the genes and gene family members. Recently, the International Wheat Genome Sequencing Consortium (IWGSC) offered a solution for distinguishing genomes using the chromosome-arm sorting technique.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Using the differences among homoeologous sequences, genome-specific primers that span target polymorphic sites can be designed. Genotyping by amplicon sequencing using genome-specific amplicons could be highly beneficial for simplifying genotype calls and achieving robust analysis for MAS.</p><p>The objective of this study is to develop an efficient approach for locating polymorphisms across the wheat genomes using the advantages of a sequence capture technique. To evaluate this approach, we designed D genome-specific primers and constructed linkage maps using multi-sample genotyping by amplicon sequencing.</p>                            <h2 scrollto-destination=118057475 id="118057475" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=118057476 id="118057476" class="section-title" >2.1. Plant materials and DNA extraction</h3>
<p>Genomic DNA was extracted from leaves of <em>Triticum aestivum</em> cv. ‘Kitahonami’ and cv. ‘Hatsumochi’ using a DNeasy Plant Mini kit (Qiagen, Hilden, Germany). ‘Kitahonami’ is a winter wheat variety adapted to Hokkaido and has superior properties, such as high yield, high flour yield and high noodle-making quality. In contrast, ‘Hatsumochi’ was the first registered waxy (glutinous) variety and is adapted to the Kanto region (Central Japan). Diversity analyses indicated that modern Japanese varieties were separated into three groups corresponding to adaptation regions, and these two varieties fall into separate groups<a reveal-id="dsy004-B13" data-open="dsy004-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy004-B17" data-open="dsy004-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>; thus, we expected to observe a reasonable level of polymorphism between them. To construct the genetic map, a population of 94 recombinant inbred lines (RILs) was developed by crossing ‘Hatsumochi’ and ‘Kitahonami’ then using the single-seed descent method to advance to the F<sub>9</sub> generation. The total genomic DNA was extracted from the leaves of the RILs using the automated DNA extracting machine PI-50α (Kurabo Industries Ltd., Osaka, Japan).</p>                            <h3 scrollto-destination=118057478 id="118057478" class="section-title" >2.2. Capture probe design and preparation of libraries</h3>
<p>A workflow of sequence capture, polymorphism detection and designing genome-specific primers is described in <a reveal-id="dsy004-F1" data-open="dsy004-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. To design the capture probes, we selected 8,892 wheat expressed sequence tag (EST) and full-length cDNA sequences from NCBI (<a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/" target="_blank">https://www.ncbi.nlm.nih.gov/</a> (5 February 2018, date last accessed)) and TriFLDB<a reveal-id="dsy004-B23" data-open="dsy004-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Of these, 4,895 EST sequences were selected earlier during our design of PCR-based Landmark Unique Gene (PLUG) markers<a reveal-id="dsy004-B24" data-open="dsy004-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and were known to be evenly distributed across the wheat chromosomes. Most of the remaining 3,997 sequences were EST sequences derived from mapped 9K iSelect probes<a reveal-id="dsy004-B3" data-open="dsy004-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> showing polymorphisms in Japanese varieties, and additional sequences to fill in gaps in coverage were based on bin-mapped wheat ESTs<a reveal-id="dsy004-B25" data-open="dsy004-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> that were selected based on the barley physical map.<a reveal-id="dsy004-B26" data-open="dsy004-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Using the syntenic relationships among rice, barley and wheat, the selected sequences were estimated to be evenly distributed across the seven <em>Triticeae</em> chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). All sequences were compared with one another, and only one representative homoeologous copy of each gene was selected for probe design. DNA capture of genomic DNA of ‘Hatsumochi’ and ‘Kitahonami’ was performed with a SeqCap EZ Developer Kit (Roche Diagnostics, Basel, Switzerland) following the manufacturer’s protocol. </p>                        <a id="118057480" scrollto-destination="118057480"></a>
<div id="dsy004-F1" data-id="dsy004-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f1.png?Expires=2147483647&Signature=amdlzMHdCyLSSKlH8kRb2a5~ddi6JaI97ijMhDgBlC~KDY9aFlR6CsOaYORwtz05ZtemoVPalj6UOikb72VS1kVgRtPQA6tX6Qdi9U4tvlYjogH~FFOJdP1NNQgeaQ9~LJhPov6aqtXroJEeL4FA23JTZNyiKxUPZE-zmgml25akmEW3YxTPurrhwSxN8H5i8eR8EJ2YObMwRAbLXHVPqXq48U9JYkHQdky7LeKI79yiYlp8sKbpQYdn8D3jhZtK2XSsod6Nr77-KzOslrKCd9eueuwbXV0RyStUZlIoiLrJOn~D5GJVxm4i2-rg2hlnSZSULe3GbwAylncK5H7MAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A workflow of sequence capture, polymorphism detection and designing genome-specific primers for amplicon sequencing. HTM; Hatsumochi, KH: Kitahonami, SurvSeq: IWGSC survey sequence.22" path-from-xml="dsy004f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057480/dsy004f1.tif" path-from-xml="dsy004f1.tif" target="_blank">View large</a><a section="118057480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f1.png?Expires=2147483647&Signature=2QMRw6H4VyiRcVgVfAX0YK2WUqntQ-RJnzlctce3E7pelyVLWgXtIueKDmi3HJ8VCuWH-E4tkBCQFJdHQ8P17~Uu-BLU7xpKCyNSLFAzUqS7d3v-K2~hV~F30jCyUUsaoWPU2MOS79to22Dgk3F9JqBwJ2GMTYeo9lbfv72J2sUOBaTul8Rv8w-BGlvx4Ms0J1Ib0wVojNvh7nQISPmvULbSAcc-Jtqp0LTObXtMoej55xju4dPjjQdql-jttK~LO~faU-akNXVbPt5iyFYpbEhUgtWub5SL24Sf1vQCZJ3zngbIlkVyK6PM3r88v9szbwK-tDsEfCcoMzbuR7ltLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057480&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A workflow of sequence capture, polymorphism detection and designing genome-specific primers for amplicon sequencing. HTM; Hatsumochi, KH: Kitahonami, SurvSeq: IWGSC survey sequence.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p></div></div><div content-id="dsy004-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f1.png?Expires=2147483647&Signature=amdlzMHdCyLSSKlH8kRb2a5~ddi6JaI97ijMhDgBlC~KDY9aFlR6CsOaYORwtz05ZtemoVPalj6UOikb72VS1kVgRtPQA6tX6Qdi9U4tvlYjogH~FFOJdP1NNQgeaQ9~LJhPov6aqtXroJEeL4FA23JTZNyiKxUPZE-zmgml25akmEW3YxTPurrhwSxN8H5i8eR8EJ2YObMwRAbLXHVPqXq48U9JYkHQdky7LeKI79yiYlp8sKbpQYdn8D3jhZtK2XSsod6Nr77-KzOslrKCd9eueuwbXV0RyStUZlIoiLrJOn~D5GJVxm4i2-rg2hlnSZSULe3GbwAylncK5H7MAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A workflow of sequence capture, polymorphism detection and designing genome-specific primers for amplicon sequencing. HTM; Hatsumochi, KH: Kitahonami, SurvSeq: IWGSC survey sequence.22" path-from-xml="dsy004f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057480/dsy004f1.tif" path-from-xml="dsy004f1.tif" target="_blank">View large</a><a section="118057480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f1.png?Expires=2147483647&Signature=2QMRw6H4VyiRcVgVfAX0YK2WUqntQ-RJnzlctce3E7pelyVLWgXtIueKDmi3HJ8VCuWH-E4tkBCQFJdHQ8P17~Uu-BLU7xpKCyNSLFAzUqS7d3v-K2~hV~F30jCyUUsaoWPU2MOS79to22Dgk3F9JqBwJ2GMTYeo9lbfv72J2sUOBaTul8Rv8w-BGlvx4Ms0J1Ib0wVojNvh7nQISPmvULbSAcc-Jtqp0LTObXtMoej55xju4dPjjQdql-jttK~LO~faU-akNXVbPt5iyFYpbEhUgtWub5SL24Sf1vQCZJ3zngbIlkVyK6PM3r88v9szbwK-tDsEfCcoMzbuR7ltLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057480&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A workflow of sequence capture, polymorphism detection and designing genome-specific primers for amplicon sequencing. HTM; Hatsumochi, KH: Kitahonami, SurvSeq: IWGSC survey sequence.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p></div></div><p>In total, 1,000 ng of each DNA sample was sheared using a Covaris LE220 (Covaris Inc., Woburn, MA, USA) focused ultrasonicator to fragments that averaged 600 bp. An NEBNext Quick DNA Library Prep Master Mix set for 454 (New England Biolabs Inc., Beverly, MA, USA) was used to construct two genomic libraries. These libraries were amplified and fractionated from 500 to 700 bp before hybridization using a SeqCap EZ reagent Kit. To avoid hybridization with repetitive regions in the wheat genome, 1,500 ng of each amplified genomic library was mixed with 10 µL of SeqCap EZ reagent Kit in the presence of 10 µg PCE (plant capture enhancer, Roche Diagnostics) in a 0.2-mL thin-wall tube.</p>                            <h3 scrollto-destination=118057482 id="118057482" class="section-title" >2.3. Detection of nucleotide polymorphisms between ‘Hatsumochi’ and ‘Kitahonami’</h3>
<p>The enriched genomic DNAs were sequenced using the next-generation sequencer GS FLX plus (Roche Diagnostics). Reads were mapped to the wheat survey sequences<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using 454 Sequencing System Software 2.7 (option: -mi 98). The detected polymorphisms against the survey sequences, including SNPs and Indels, were supported with at least three reads in each variety. From them, polymorphic sites between ‘Hatsumochi’ and ‘Kitahonami’ were extracted. The predicted effects of the polymorphisms were analysed using SnpEff<a reveal-id="dsy004-B27" data-open="dsy004-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with IWGSC ver. 2.2 annotation data (<a class="link link-uri" href="https://wheat-urgi.versailles.inra.fr/Seq-Repository/Genes-annotations" target="_blank">https://wheat-urgi.versailles.inra.fr/Seq-Repository/Genes-annotations</a> (5 February 2018, date last accessed)).</p>                            <h3 scrollto-destination=118057484 id="118057484" class="section-title" >2.4. Design of genome-specific primers</h3>
<p>The flanking sequences of target polymorphism sites were obtained and blasted against the IWGSC survey sequences using PSI-BLAST,<a reveal-id="dsy004-B28" data-open="dsy004-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and three homoeologous sequences in terms of their chromosomal locations were identified. The three sequences were aligned by MAFFT ver. 6.864<a reveal-id="dsy004-B29" data-open="dsy004-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> with a default parameters and imported into an in-house Java pipeline. This pipeline requires an alignment file of the homoeologous sequences in a target region. Based on the polymorphisms among the genomes, the pipeline automatically detects genome-specific primer pairs of 18–22 nucleotides at an amplicon length of approximately 300 bp. The primers contained nucleotides specific to the target sequence at the first and/or second positions from their 3′ end. The forward and reverse primers for the first PCR were tailed with the common sequence tags CS1 (5′-ACACTGACGACATGGTTCTACA-3′) and CS2 (5′-TACGGTAGCAGAGACTTGGTCT-3′), respectively, to allow for the addition of adapters during the second round of PCR according to the protocol of the Access Array for Ion Torrent PGM Sequencing System (Fluidigm Corporation, San Francisco, CA, USA).</p>                            <h3 scrollto-destination=118057486 id="118057486" class="section-title" >2.5. Library preparation and amplicon sequencing</h3>
<p>The reaction mix for the first PCR contained a total volume of 10 µL consisting of 1× Multiplex PCR Master Mix (Qiagen), a primer mix (described below) and 20–50 ng of template DNA. The primer mix contained 48 sets of randomly selected locus-specific primers, with CS1 or CS2 tails attached. The first PCR program consisted of a denaturation step at 95 °C for 15 min, followed by 32 cycles of 30 s at 94 °C, 90 s at 60 °C and 1 min at 72 °C, and a final extension for 10 min at 72 °C. The product from the first PCR of each sample was diluted 100-fold with sterilized distilled water, and 2 µL of the diluted product was used as the template for the second PCR. To perform bidirectional amplicon tagging in the second PCR, a forward fusion primer containing Ion A adapter, barcode, and the CS1 sequence and a reverse fusion primer containing Ion P1 adapter and the CS2 sequence were used with a portion of the template, while a second portion was amplified with an A-adaptor–barcode–CS2 and P1 adaptor–CS1 primer combination.</p><p>The 10-µL second PCR mix contained 1× Multiplex PCR Master Mix (Qiagen) and 400 n<span class="small-caps">m</span> forward and reverse fusion primers, and PCR was run using the following profile: 15 min at 95 °C, followed by 15 cycles of 30 s at 94 °C, 90 s at 60 °C and 1 min at 72 °C, and a final extension step of 10 min at 72 °C. All second PCR products were mixed in equivalent volumes (2 µL of PCR product per sample). The pooled product was purified using Agencourt AMPure XP Reagent beads (Beckman-Coulter, Fullerton, CA, USA) as follows: 12 µL of pooled products, 24 µL of TE buffer and 36 µL of well-mixed AMPure XP beads were vortexed. After a 10-min incubation at room temperature, the sample was placed onto a magnetic separator for 1 min, and the supernatant was discarded. The beads with sample attached were washed twice with 180 µL of freshly prepared 70% ethanol. Finally, the purified PCR products were suspended in 40 µL of low TE buffer. The quality of the amplicon library was assessed using an Agilent 2100 Bioanalyzer, and a high sensitivity kit (Agilent Technologies, Santa Clara, CA, USA) was used to define the region covering all PCR library peaks (300–450 bp). The purified library was quantified using a Qubit dsDNA HS assay kit (Thermo Fisher Scientific), with dilution to a concentration of 5 pM. Sequencing was performed using an Ion Torrent PGM system with an Ion PGM 400 sequencing kit and 318 chips (Thermo Fisher Scientific). A schematic diagram of the experimental procedure is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>.</p>                            <h3 scrollto-destination=118057489 id="118057489" class="section-title" >2.6. Data processing and genotype calling</h3>
<p>The removal of the Ion Torrent sequencing adaptor sites and demultiplexing of the barcodes to separate the different samples were automatically performed by Torrent Suite ver. 5 (Thermo Fisher Scientific). Further analysis was conducted using a custom Java script. First, the CS1 and CS2 tail sequences were removed. Second, the sequences were aligned to reference sequences using a BLAST-like alignment tool. The reference sequences consisted of internal sequences in the primers of each marker. Third, based on the alignment results of each sample-marker combination, the total number of aligned reads containing A, C, G and T bases, null alleles or missing values (NaN) and nucleotide deletions (“–”) in the base position of interest were counted. Fourth, the alleles were defined according to the bases with the highest read counts. The minimum read count was set to 5. Heterozygous site (H) was called when two alleles each had more than 40% of the total reads. Finally, the results of the genotype calls were summarized and exported into a tabular form.</p>                            <h3 scrollto-destination=118057491 id="118057491" class="section-title" >2.7. Construction of the linkage map</h3>
<p>The wheat <em>Pst</em>I (<em>Taq</em>I) v.3 DArT array (Diversity Array Technology Pty Ltd., Bruce, Australia) and 9K SNP array<a reveal-id="dsy004-B3" data-open="dsy004-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> were used to genotype 94 RILs. Publicly available SSR markers (GrainGenes 2.0) and functional markers<a reveal-id="dsy004-B30" data-open="dsy004-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> of the <em>Wx-A1</em>, <em>Wx-D1</em> and <em>Pina-D1</em> genes were also used. The data obtained using DArT, 9K, SSR and the newly developed markers (TARC) were merged, and polymorphic markers between parents were extracted. Poor-quality genotypes and genotypes with more than 10% missing data or segregation distortion were removed. Because redundant markers are completely correlated or identical and cannot provide additional information, only one of a redundant set of markers was used in map construction. MapDisto 1.7.5<a reveal-id="dsy004-B31" data-open="dsy004-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> was used to identify linkage groups. Mapped markers corresponding to those on the hexaploid wheat consensus map<a reveal-id="dsy004-B3" data-open="dsy004-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy004-B15" data-open="dsy004-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy004-B32" data-open="dsy004-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> were used as anchors to assign each linkage group to a particular chromosome and to orient linkage groups on short and long chromosome arms.</p>                            <h3 scrollto-destination=118057493 id="118057493" class="section-title" >2.8. Polymorphism survey using multiple genotypes</h3>
<p>To increase the resources for nucleotide polymorphism detection, four additional varieties, namely ‘Shunyou’, ‘Tohoku224’, ‘Kinuhime’ and ‘Yumechikara’, were subjected to capture sequencing. The first three varieties are adopted to the Kanto and Tohoku regions (Central and Northeastern Japan), while the ‘Yumechikara’ variety is adopted to the Hokkaido region. The procedures used for DNA extraction, library preparation, sequencing by GS FLX plus (Roche Diagnostics) and mapping to the reference sequences<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> were identical to those described above. To identify highly polymorphic sites, we performed pair-wise comparisons among the six varieties used. The detected polymorphisms, including SNPs and Indels, were supported with at least one read as either “Reference” or “SNP” in a given variety. If reads supported both “Reference” and “SNP”, we identified that variety as heterozygous at that site.</p>                            <h2 scrollto-destination=118057495 id="118057495" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=118057496 id="118057496" class="section-title" >3.1. Detection of polymorphisms between ‘Hatsumochi’ and ‘Kitahonami’</h3>
<p>Using the next-generation sequencer GS FLX plus, 1,114,867 (536,055,413 bp) and 1,304,168 (615,634,987 bp) reads were obtained from the enriched genomic DNA of ‘Hatsumochi’ and ‘Kitahonami’, respectively. The average lengths of the sequences were 481 bp for ‘Hatsumochi’ and 472 bp for ‘Kitahonami’. Based on the criteria described above, 12,551 nucleotide polymorphisms were detected between these two varieties (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Using the survey sequences,<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> we localized these polymorphisms to each chromosome or chromosome arm. As shown in <a reveal-id="dsy004-T1" data-open="dsy004-T1" class="link link-reveal link-table xref-fig">Table 1</a>, the number of polymorphisms varied among the chromosomes and genomes. The number of polymorphisms on the D genome were approximately one-sixth and one-fifth of those on the B and A genomes, respectively. Relatively fewer polymorphisms were observed in the Group 4 chromosomes than in the other groups. The cumulative numbers of polymorphic sites according to the gene models are shown in <a reveal-id="dsy004-F2" data-open="dsy004-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>. Most polymorphisms were found in genic regions, including both the 500-bp upstream and downstream regions (<a reveal-id="dsy004-F2" data-open="dsy004-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In the intron regions, 5,220 polymorphisms were detected, meaning the intron regions carried the highest percentage (41.6%) of the total polymorphisms. <div content-id="dsy004-T1" class="table-modal"><div class="table-wrap"><div id="dsy004-T1" data-id="dsy004-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Number of polymorphisms between ‘Hatsumochi’ and ‘Kitahonami’ in each wheat chromosome arm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="7">Group<hr /> </th><th> </th></tr><tr><th>Genome </th><th>Arm </th><th>1 </th><th>2 </th><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>Total </th></tr></thead><tbody><tr><td>A </td><td>S </td><td>293 </td><td>409 </td><td>137 </td><td>160 </td><td>150 </td><td>287 </td><td>490 </td><td>5,293 </td></tr><tr><td> </td><td>L </td><td>718 </td><td>651 </td><td>529 </td><td>218 </td><td>513 </td><td>292 </td><td>446 </td><td> </td></tr><tr><td>B </td><td>S </td><td>174 </td><td>519 </td><td>1,197<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>123 </td><td>241 </td><td>428 </td><td>151 </td><td>6,316 </td></tr><tr><td> </td><td>L </td><td>648 </td><td>929 </td><td> </td><td>398 </td><td>723 </td><td>512 </td><td>273 </td><td> </td></tr><tr><td>D </td><td>S </td><td>62 </td><td>143 </td><td>23 </td><td>13 </td><td>16 </td><td>63 </td><td>64 </td><td>942 </td></tr><tr><td> </td><td>L </td><td>56 </td><td>126 </td><td>86 </td><td>17 </td><td>146 </td><td>75 </td><td>52 </td><td> </td></tr><tr><td>Total </td><td> </td><td>1,951 </td><td>2,777 </td><td>1,972 </td><td>929 </td><td>1,789 </td><td>1,657 </td><td>1,476 </td><td>12,551 </td></tr></tbody></table></div><div content-id="dsy004-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="7">Group<hr /> </th><th> </th></tr><tr><th>Genome </th><th>Arm </th><th>1 </th><th>2 </th><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>Total </th></tr></thead><tbody><tr><td>A </td><td>S </td><td>293 </td><td>409 </td><td>137 </td><td>160 </td><td>150 </td><td>287 </td><td>490 </td><td>5,293 </td></tr><tr><td> </td><td>L </td><td>718 </td><td>651 </td><td>529 </td><td>218 </td><td>513 </td><td>292 </td><td>446 </td><td> </td></tr><tr><td>B </td><td>S </td><td>174 </td><td>519 </td><td>1,197<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>123 </td><td>241 </td><td>428 </td><td>151 </td><td>6,316 </td></tr><tr><td> </td><td>L </td><td>648 </td><td>929 </td><td> </td><td>398 </td><td>723 </td><td>512 </td><td>273 </td><td> </td></tr><tr><td>D </td><td>S </td><td>62 </td><td>143 </td><td>23 </td><td>13 </td><td>16 </td><td>63 </td><td>64 </td><td>942 </td></tr><tr><td> </td><td>L </td><td>56 </td><td>126 </td><td>86 </td><td>17 </td><td>146 </td><td>75 </td><td>52 </td><td> </td></tr><tr><td>Total </td><td> </td><td>1,951 </td><td>2,777 </td><td>1,972 </td><td>929 </td><td>1,789 </td><td>1,657 </td><td>1,476 </td><td>12,551 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Total number of short and long arms.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy004-T1" data-id="dsy004-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Number of polymorphisms between ‘Hatsumochi’ and ‘Kitahonami’ in each wheat chromosome arm</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="7">Group<hr /> </th><th> </th></tr><tr><th>Genome </th><th>Arm </th><th>1 </th><th>2 </th><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>Total </th></tr></thead><tbody><tr><td>A </td><td>S </td><td>293 </td><td>409 </td><td>137 </td><td>160 </td><td>150 </td><td>287 </td><td>490 </td><td>5,293 </td></tr><tr><td> </td><td>L </td><td>718 </td><td>651 </td><td>529 </td><td>218 </td><td>513 </td><td>292 </td><td>446 </td><td> </td></tr><tr><td>B </td><td>S </td><td>174 </td><td>519 </td><td>1,197<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>123 </td><td>241 </td><td>428 </td><td>151 </td><td>6,316 </td></tr><tr><td> </td><td>L </td><td>648 </td><td>929 </td><td> </td><td>398 </td><td>723 </td><td>512 </td><td>273 </td><td> </td></tr><tr><td>D </td><td>S </td><td>62 </td><td>143 </td><td>23 </td><td>13 </td><td>16 </td><td>63 </td><td>64 </td><td>942 </td></tr><tr><td> </td><td>L </td><td>56 </td><td>126 </td><td>86 </td><td>17 </td><td>146 </td><td>75 </td><td>52 </td><td> </td></tr><tr><td>Total </td><td> </td><td>1,951 </td><td>2,777 </td><td>1,972 </td><td>929 </td><td>1,789 </td><td>1,657 </td><td>1,476 </td><td>12,551 </td></tr></tbody></table></div><div content-id="dsy004-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="7">Group<hr /> </th><th> </th></tr><tr><th>Genome </th><th>Arm </th><th>1 </th><th>2 </th><th>3 </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>Total </th></tr></thead><tbody><tr><td>A </td><td>S </td><td>293 </td><td>409 </td><td>137 </td><td>160 </td><td>150 </td><td>287 </td><td>490 </td><td>5,293 </td></tr><tr><td> </td><td>L </td><td>718 </td><td>651 </td><td>529 </td><td>218 </td><td>513 </td><td>292 </td><td>446 </td><td> </td></tr><tr><td>B </td><td>S </td><td>174 </td><td>519 </td><td>1,197<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>123 </td><td>241 </td><td>428 </td><td>151 </td><td>6,316 </td></tr><tr><td> </td><td>L </td><td>648 </td><td>929 </td><td> </td><td>398 </td><td>723 </td><td>512 </td><td>273 </td><td> </td></tr><tr><td>D </td><td>S </td><td>62 </td><td>143 </td><td>23 </td><td>13 </td><td>16 </td><td>63 </td><td>64 </td><td>942 </td></tr><tr><td> </td><td>L </td><td>56 </td><td>126 </td><td>86 </td><td>17 </td><td>146 </td><td>75 </td><td>52 </td><td> </td></tr><tr><td>Total </td><td> </td><td>1,951 </td><td>2,777 </td><td>1,972 </td><td>929 </td><td>1,789 </td><td>1,657 </td><td>1,476 </td><td>12,551 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Total number of short and long arms.</p></div></div></div></div></p>                        <a id="118057498" scrollto-destination="118057498"></a>
<div id="dsy004-F2" data-id="dsy004-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f2.png?Expires=2147483647&Signature=pzoRbuaeK2eBY0RO8edO4N63-bGcYmg53W5w~tWGWI2AgFcHS52CeQXFEiZcUQ2LldMbr7X9nhpsaJ2Q4dlN267xRpAkOfpII0CwuyO6hBsOtbj3dtYM6hyitGqzY9CTFPQ6DxqTGFmD~qkpy6RfcNqnftez7DkTvBbm5Vr-cZ82nZoy~NH5On5L2jLlq8hqq-R2mYuS3~G9ZYPkiejLiwYYMinA2RPi~nc3BMYDC6kXls2nzkCKsrQN0chMrIHw0rAzBv1KiIfgh-oeldcFH98DW0vWQCmcrVsHWIGe0C~lO6YlHKY4rnQ6HIuwueCAqsURaolx~BAU8yCU7o65bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cumulative number of polymorphic sites between ‘Hatsumochi’ and ‘Kitahonami’ in each genic region. The IWGSC gene models were used to define the genic regions.22" path-from-xml="dsy004f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057498/dsy004f2.tif" path-from-xml="dsy004f2.tif" target="_blank">View large</a><a section="118057498" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f2.png?Expires=2147483647&Signature=qv58gTeleBKhyO25LNhLDLZ09KCQIhYj~pp6HVMNAOeFnPMOHSg58d7nPQUW0VPm7VIh-i1~E96OlihraEX~gfTENq4lZgYZ-iiixNXyU4bvtJVVUAwncf3lx~lTI2LxlPcXvwD-SrNfyUVpTfM66grQImAh4fnAgR3DiI~TbBN~4koBFZg425J-~ZAW8kV3AgPfsvtuNSubKsNSPy4pAUV-EqhHIyPID6zoBhjRLaOuyQ9nh7tjmF4RXwTuqGY3F6ZiYUWFrdov582pWr3UZG1EM7ZzQi13DdwFhiqZQF2k17oobTup7wIOceC2zrA8eu522jmjejdCo4cJDuUtEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057498&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cumulative number of polymorphic sites between ‘Hatsumochi’ and ‘Kitahonami’ in each genic region. The IWGSC gene models were used to define the genic regions.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p></div></div><div content-id="dsy004-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f2.png?Expires=2147483647&Signature=pzoRbuaeK2eBY0RO8edO4N63-bGcYmg53W5w~tWGWI2AgFcHS52CeQXFEiZcUQ2LldMbr7X9nhpsaJ2Q4dlN267xRpAkOfpII0CwuyO6hBsOtbj3dtYM6hyitGqzY9CTFPQ6DxqTGFmD~qkpy6RfcNqnftez7DkTvBbm5Vr-cZ82nZoy~NH5On5L2jLlq8hqq-R2mYuS3~G9ZYPkiejLiwYYMinA2RPi~nc3BMYDC6kXls2nzkCKsrQN0chMrIHw0rAzBv1KiIfgh-oeldcFH98DW0vWQCmcrVsHWIGe0C~lO6YlHKY4rnQ6HIuwueCAqsURaolx~BAU8yCU7o65bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cumulative number of polymorphic sites between ‘Hatsumochi’ and ‘Kitahonami’ in each genic region. The IWGSC gene models were used to define the genic regions.22" path-from-xml="dsy004f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057498/dsy004f2.tif" path-from-xml="dsy004f2.tif" target="_blank">View large</a><a section="118057498" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f2.png?Expires=2147483647&Signature=qv58gTeleBKhyO25LNhLDLZ09KCQIhYj~pp6HVMNAOeFnPMOHSg58d7nPQUW0VPm7VIh-i1~E96OlihraEX~gfTENq4lZgYZ-iiixNXyU4bvtJVVUAwncf3lx~lTI2LxlPcXvwD-SrNfyUVpTfM66grQImAh4fnAgR3DiI~TbBN~4koBFZg425J-~ZAW8kV3AgPfsvtuNSubKsNSPy4pAUV-EqhHIyPID6zoBhjRLaOuyQ9nh7tjmF4RXwTuqGY3F6ZiYUWFrdov582pWr3UZG1EM7ZzQi13DdwFhiqZQF2k17oobTup7wIOceC2zrA8eu522jmjejdCo4cJDuUtEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057498&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cumulative number of polymorphic sites between ‘Hatsumochi’ and ‘Kitahonami’ in each genic region. The IWGSC gene models were used to define the genic regions.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p></div></div>                            <h3 scrollto-destination=118057499 id="118057499" class="section-title" >3.2. Validation of polymorphisms by amplicon sequencing</h3>
<p>Three hundred and ninety-six D genome-specific primers were designed using the primer-picking pipeline described above. The details regarding the primers are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. Preliminary analysis of primers using gel electrophoresis of PCR products indicated that single PCR products with the expected fragment sizes were obtained using 380 of the 396 primer sets. Multiple products or a lack of bands were observed using the remaining primer sets. The PCR products from the 380 successful primer sets were mixed and sequenced using GS FLX plus. In total, 442,564 (average length 322.7 bp) and 640,147 (average length 327.2 bp) reads were obtained from the amplicons of ‘Hatsumochi’ and ‘Kitahonami’, respectively. The sequences derived from 312 markers were mapped to reference sequences without interference from off-target or homoeologous sequences (<a reveal-id="dsy004-T2" data-open="dsy004-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Mixtures of homoeologous sequences were observed using 44 markers, which were classified according to the degree of the mixture as follows: low (&lt;30%), medium (30–50%) and high (&gt;50%). The alleles in markers with a low level of mixture could automatically be defined; in contrast, manual examination of the data was required to define alleles in markers with medium and high levels of mixture. The polymorphisms detected with 334 of the markers were consistent with those obtained using sequence capture (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Thirty-six markers failed to validate the polymorphisms because of mixtures of homoeologous sequences, while 17 markers showed low read coverage. Nine markers did not show polymorphisms at the target sites. <div content-id="dsy004-T2" class="table-modal"><div class="table-wrap"><div id="dsy004-T2" data-id="dsy004-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Classification of markers based on the mapping results of the amplicon sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="2">Validation by parents<hr /> </th><th colspan="2">Genotype of RILs<hr /> </th></tr><tr><th>Class </th><th>Description </th><th>No. of markers </th><th>% in total </th><th>No. of markers </th><th>% in total </th></tr></thead><tbody><tr><td>1 </td><td>Genome specific </td><td>312 </td><td>78.8 </td><td>247 </td><td>62.4 </td></tr><tr><td>2 </td><td>Mix off-target </td><td>6 </td><td>1.5 </td><td>11 </td><td>2.8 </td></tr><tr><td>3 </td><td>Mix low (&lt;30%) </td><td>16 </td><td>4.0 </td><td>34 </td><td>8.6 </td></tr><tr><td>4 </td><td>Mix medium (30–50%) </td><td>21 </td><td>5.3 </td><td>43 </td><td>10.9 </td></tr><tr><td>5 </td><td>Mix high (&gt;50%) </td><td>7 </td><td>1.8 </td><td>9 </td><td>2.3 </td></tr><tr><td>6 </td><td>Indel </td><td>2 </td><td>0.5 </td><td>15 </td><td>3.8 </td></tr><tr><td>7 </td><td>Other<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>11 </td><td>2.8 </td><td>24 </td><td>6.1 </td></tr><tr><td>8 </td><td>Low or no reads </td><td>21 </td><td>5.3 </td><td>13 </td><td>3.3 </td></tr><tr><td> </td><td>Total </td><td>396 </td><td> </td><td>396 </td><td> </td></tr></tbody></table></div><div content-id="dsy004-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="2">Validation by parents<hr /> </th><th colspan="2">Genotype of RILs<hr /> </th></tr><tr><th>Class </th><th>Description </th><th>No. of markers </th><th>% in total </th><th>No. of markers </th><th>% in total </th></tr></thead><tbody><tr><td>1 </td><td>Genome specific </td><td>312 </td><td>78.8 </td><td>247 </td><td>62.4 </td></tr><tr><td>2 </td><td>Mix off-target </td><td>6 </td><td>1.5 </td><td>11 </td><td>2.8 </td></tr><tr><td>3 </td><td>Mix low (&lt;30%) </td><td>16 </td><td>4.0 </td><td>34 </td><td>8.6 </td></tr><tr><td>4 </td><td>Mix medium (30–50%) </td><td>21 </td><td>5.3 </td><td>43 </td><td>10.9 </td></tr><tr><td>5 </td><td>Mix high (&gt;50%) </td><td>7 </td><td>1.8 </td><td>9 </td><td>2.3 </td></tr><tr><td>6 </td><td>Indel </td><td>2 </td><td>0.5 </td><td>15 </td><td>3.8 </td></tr><tr><td>7 </td><td>Other<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>11 </td><td>2.8 </td><td>24 </td><td>6.1 </td></tr><tr><td>8 </td><td>Low or no reads </td><td>21 </td><td>5.3 </td><td>13 </td><td>3.3 </td></tr><tr><td> </td><td>Total </td><td>396 </td><td> </td><td>396 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Markers with both mixtures of sequences and indels.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy004-T2" data-id="dsy004-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Classification of markers based on the mapping results of the amplicon sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="2">Validation by parents<hr /> </th><th colspan="2">Genotype of RILs<hr /> </th></tr><tr><th>Class </th><th>Description </th><th>No. of markers </th><th>% in total </th><th>No. of markers </th><th>% in total </th></tr></thead><tbody><tr><td>1 </td><td>Genome specific </td><td>312 </td><td>78.8 </td><td>247 </td><td>62.4 </td></tr><tr><td>2 </td><td>Mix off-target </td><td>6 </td><td>1.5 </td><td>11 </td><td>2.8 </td></tr><tr><td>3 </td><td>Mix low (&lt;30%) </td><td>16 </td><td>4.0 </td><td>34 </td><td>8.6 </td></tr><tr><td>4 </td><td>Mix medium (30–50%) </td><td>21 </td><td>5.3 </td><td>43 </td><td>10.9 </td></tr><tr><td>5 </td><td>Mix high (&gt;50%) </td><td>7 </td><td>1.8 </td><td>9 </td><td>2.3 </td></tr><tr><td>6 </td><td>Indel </td><td>2 </td><td>0.5 </td><td>15 </td><td>3.8 </td></tr><tr><td>7 </td><td>Other<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>11 </td><td>2.8 </td><td>24 </td><td>6.1 </td></tr><tr><td>8 </td><td>Low or no reads </td><td>21 </td><td>5.3 </td><td>13 </td><td>3.3 </td></tr><tr><td> </td><td>Total </td><td>396 </td><td> </td><td>396 </td><td> </td></tr></tbody></table></div><div content-id="dsy004-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="2">Validation by parents<hr /> </th><th colspan="2">Genotype of RILs<hr /> </th></tr><tr><th>Class </th><th>Description </th><th>No. of markers </th><th>% in total </th><th>No. of markers </th><th>% in total </th></tr></thead><tbody><tr><td>1 </td><td>Genome specific </td><td>312 </td><td>78.8 </td><td>247 </td><td>62.4 </td></tr><tr><td>2 </td><td>Mix off-target </td><td>6 </td><td>1.5 </td><td>11 </td><td>2.8 </td></tr><tr><td>3 </td><td>Mix low (&lt;30%) </td><td>16 </td><td>4.0 </td><td>34 </td><td>8.6 </td></tr><tr><td>4 </td><td>Mix medium (30–50%) </td><td>21 </td><td>5.3 </td><td>43 </td><td>10.9 </td></tr><tr><td>5 </td><td>Mix high (&gt;50%) </td><td>7 </td><td>1.8 </td><td>9 </td><td>2.3 </td></tr><tr><td>6 </td><td>Indel </td><td>2 </td><td>0.5 </td><td>15 </td><td>3.8 </td></tr><tr><td>7 </td><td>Other<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>11 </td><td>2.8 </td><td>24 </td><td>6.1 </td></tr><tr><td>8 </td><td>Low or no reads </td><td>21 </td><td>5.3 </td><td>13 </td><td>3.3 </td></tr><tr><td> </td><td>Total </td><td>396 </td><td> </td><td>396 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Markers with both mixtures of sequences and indels.</p></div></div></div></div></p>                            <h3 scrollto-destination=118057501 id="118057501" class="section-title" >3.3. Genotyping by amplicon sequencing in 96 multiplexed samples</h3>
<p>The ‘Hatsumochi’/‘Kitahonami’ RILs were genotyped using an Ion PGM (Thermo Fisher Scientific) according to the procedure described above. The variations in the number of reads per sample and per maker are shown in <a reveal-id="dsy004-F3" data-open="dsy004-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. The number of reads per sample was relatively stable, and the average read count was 34,873. However, the variation in the number of reads per marker was high. The difference between the most and least sequenced markers was more than 150-fold. The markers with large product sizes tended to show a low number of reads (data not shown). Based on mapping to the reference sequences, we classified markers into eight classes (<a reveal-id="dsy004-T2" data-open="dsy004-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The alleles in markers in classes 1–3 could automatically be defined, while those in markers in classes 4–6 had to be defined manually. Class 7 indicates markers with mixtures of sequences and indels, and class 8 represents no or quite low reads. The genotypes of 96 samples using 359 markers were obtained, and missing values accounted for only 1.2% (392 of 33,746 data points). </p>                        <a id="118057503" scrollto-destination="118057503"></a>
<div id="dsy004-F3" data-id="dsy004-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f3.png?Expires=2147483647&Signature=rPeo7T83DTsbFLkK9SOqgIconlWCLO5rnZblsCV1KxQGT5njchiT~svRzEp7wvhLoRNXdHBEwE6xp7Xz1a1IEsz0ECRtZWC4Kwta352L1mmdW6u8IRqkarKklyk05SDAJ0ZNppUE~LGj2RhaTIldqBe6CYJTrt6xu0bspbC2ZCrnCrotWR~Z~1hcX6DddU1ClMe6HC-9CRGN-IVAoKpQxvGIWYUhDDj1Z9xD3x3f4vgaG32wHRlSS3VBm4FlxqWQ6T8Ha7~rk0Mg~bStkDsAgZU1N4FZ~W~nzvHaf3xBlC1bnLi7jfd8e8HeBeKoEG74H1FFi8sxEwhQ2Qph08R93g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Variations in the number of reads per sample (a) and per marker (b). Error bars in the lower figure indicate standard deviations among 96 samples." path-from-xml="dsy004f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057503/dsy004f3.tif" path-from-xml="dsy004f3.tif" target="_blank">View large</a><a section="118057503" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f3.png?Expires=2147483647&Signature=N5yIgC~hq4tOIczfo8wu51QOdgHInHeoBH1yN9Zglc-P1issas1I4yy-A2hQSb83yqbVn8FLXQKCBIuWNWRyssa3l5Tf9JLN9OrrGdRuLbBfEoZHdSBd8biQ31Nw9iSpa-M7WjO6A7-DuySE~9lsiKX4xRR0kmRUzI-8FKv9ihLbSN9oK3aO~tbRizeL4Y98Nj5TyOuGOSCL8KxcHpYJhtmE2JFu9QbhhAVCkFMPZOx0xrFY2j3hsJV3LlVvDGnc~dzl1fYBP7KQX-QiyyhuuF6NZXQ8iazmqK~01Dd8uuKEwh48fRztMhjnS9wvh3eUF0~J8S1VLkdPj4Oq5GY3rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057503&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Variations in the number of reads per sample (a) and per marker (b). Error bars in the lower figure indicate standard deviations among 96 samples.</p></div></div><div content-id="dsy004-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f3.png?Expires=2147483647&Signature=rPeo7T83DTsbFLkK9SOqgIconlWCLO5rnZblsCV1KxQGT5njchiT~svRzEp7wvhLoRNXdHBEwE6xp7Xz1a1IEsz0ECRtZWC4Kwta352L1mmdW6u8IRqkarKklyk05SDAJ0ZNppUE~LGj2RhaTIldqBe6CYJTrt6xu0bspbC2ZCrnCrotWR~Z~1hcX6DddU1ClMe6HC-9CRGN-IVAoKpQxvGIWYUhDDj1Z9xD3x3f4vgaG32wHRlSS3VBm4FlxqWQ6T8Ha7~rk0Mg~bStkDsAgZU1N4FZ~W~nzvHaf3xBlC1bnLi7jfd8e8HeBeKoEG74H1FFi8sxEwhQ2Qph08R93g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Variations in the number of reads per sample (a) and per marker (b). Error bars in the lower figure indicate standard deviations among 96 samples." path-from-xml="dsy004f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057503/dsy004f3.tif" path-from-xml="dsy004f3.tif" target="_blank">View large</a><a section="118057503" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f3.png?Expires=2147483647&Signature=N5yIgC~hq4tOIczfo8wu51QOdgHInHeoBH1yN9Zglc-P1issas1I4yy-A2hQSb83yqbVn8FLXQKCBIuWNWRyssa3l5Tf9JLN9OrrGdRuLbBfEoZHdSBd8biQ31Nw9iSpa-M7WjO6A7-DuySE~9lsiKX4xRR0kmRUzI-8FKv9ihLbSN9oK3aO~tbRizeL4Y98Nj5TyOuGOSCL8KxcHpYJhtmE2JFu9QbhhAVCkFMPZOx0xrFY2j3hsJV3LlVvDGnc~dzl1fYBP7KQX-QiyyhuuF6NZXQ8iazmqK~01Dd8uuKEwh48fRztMhjnS9wvh3eUF0~J8S1VLkdPj4Oq5GY3rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057503&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Variations in the number of reads per sample (a) and per marker (b). Error bars in the lower figure indicate standard deviations among 96 samples.</p></div></div>                            <h3 scrollto-destination=118057504 id="118057504" class="section-title" >3.4. Extension of D genome maps using the newly developed markers</h3>
<p>By combining the data obtained using the newly developed markers (defined as TARC markers) with those obtained using DArT, 9K array and SSR, genotyping data for 3,956 markers was obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). According to the grouping of redundant markers, 1,408 markers were considered non-redundant and were used for map construction using MapDisto software.<a reveal-id="dsy004-B31" data-open="dsy004-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> After filtering by redundancy, the number of DArT and 9K markers was reduced to 42.9% and 25.9% of the initial number, respectively. In contrast, of the 359 TARC markers, 261 (72.7%) remained after filtering. Using the default settings of MapDisto, 32 linkage groups were extracted from the 1,399 markers. Nine markers were unassigned. Three linkage groups were not assigned to chromosomes because of the lack of sequence information for the markers. The linkage map covered 3,994.7 cM, with an average chromosome length of 189.6 cM, and the lengths of the individual chromosomes varied from 95.3 cM (6D) to 275.4 cM (3B). Chromosomes 1A, 2A, 3A, 5D, 7B and 7D consisted of two linkage groups, and chromosome 6D consisted of three linkage groups. Only maps of the D genome are illustrated in <a reveal-id="dsy004-F4" data-open="dsy004-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>. The complete set of linkage maps is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. The number of loci per chromosome varied from 29 (4D) to 112 (2B), with an average of 66 loci per chromosome. Using a maximum of 0.61 for chromosome 1A and a minimum of 0.21 for chromosome 4D, the overall marker density was 0.36 markers per cM. The D genome linkage maps were compared with previous maps without the TARC markers. The total length of the D genome increased from 878.4 cM to 1333.1 cM, and the average distance between markers changed from 1.45 to 2.09 cM. In comparison, the A genome map length was 1222.7 cM with 2.58 cM between markers on average and the B genome 1425.4 cM with 2.80 cM between markers. The TARC markers were distributed across the seven wheat chromosomes and significantly filled the previously observed gaps (<a reveal-id="dsy004-F4" data-open="dsy004-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Furthermore, the TARC markers successfully extended the long arms of 1D, 2D, 4D and 6D and the short arms of 2D and 4D. </p>                        <a id="118057506" scrollto-destination="118057506"></a>
<div id="dsy004-F4" data-id="dsy004-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f4.png?Expires=2147483647&Signature=Es3i3MiD5~2Ue6g26KjZJp4i2JrOvrGBTtsvA-mNJuflw-QDvWXQhtlNvlcGIJKiItzMShuHuAXMLaeVnh5Dy1XVi4G7EYfywvHXYqPcH5Ad2ePGOpOJE7fjRB-5rbucLP3ziCSrzUfRFH1g9GER65hnu07bLnYwiZ9raqYzvCNKKw4S1uSWJAwydyzICbtP0ks0fCiEOm3SQOW9FqT650PhsZoGcfWoRfEZkRHbifEAYhU74RXXqs7oZQ0N8YlWF3Y~wvKlo9T5PixIxbTbodISeSz~MRiJIK4~fjLhdQUS1ccakjVQ-vueJammMnB7AwzKKNVu94MWlYdGSUSinA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of the D genome using the ‘Hatsumochi’/‘Kitahonami’ RILs. Newly developed markers are prefixed with ‘tarc’. Box on the left of each chromosome indicates the putative position of the centromere." path-from-xml="dsy004f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057506/dsy004f4.tif" path-from-xml="dsy004f4.tif" target="_blank">View large</a><a section="118057506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f4.png?Expires=2147483647&Signature=CxEUsgHNbaRo-VJEznxlWEr2hoiGYpMoy6the6UVRwnI-t3JZwBjEiolgTKcuLLOGrytY0nKoTjnZhTBpktix9GslljfS5-EANRV5ZS6EsxQ-z0BJ5J6Nhka7G5sL1cJ1ycs3UdDbjyfl5S6TQzfZi-I-aETchPTgV-3Hg~aJV2JP0n0GMAfrysiPXlIz8mg93~4X10U2MPtsT8~vv8D5~GEKDZ6cKqi~ZNlMCRSAKxQfpTs0jrDyaLC0LRy6I4GHLFOwu-2IPdmgWpNkvPRd3ujl3G~-qQdU0tA6zYfKpP-govgpbMDk8~VmKONDOV4AuoEOjCxvUy51fnPxAjTZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057506&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage maps of the D genome using the ‘Hatsumochi’/‘Kitahonami’ RILs. Newly developed markers are prefixed with ‘tarc’. Box on the left of each chromosome indicates the putative position of the centromere.</p></div></div><div content-id="dsy004-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/m_dsy004f4.png?Expires=2147483647&Signature=Es3i3MiD5~2Ue6g26KjZJp4i2JrOvrGBTtsvA-mNJuflw-QDvWXQhtlNvlcGIJKiItzMShuHuAXMLaeVnh5Dy1XVi4G7EYfywvHXYqPcH5Ad2ePGOpOJE7fjRB-5rbucLP3ziCSrzUfRFH1g9GER65hnu07bLnYwiZ9raqYzvCNKKw4S1uSWJAwydyzICbtP0ks0fCiEOm3SQOW9FqT650PhsZoGcfWoRfEZkRHbifEAYhU74RXXqs7oZQ0N8YlWF3Y~wvKlo9T5PixIxbTbodISeSz~MRiJIK4~fjLhdQUS1ccakjVQ-vueJammMnB7AwzKKNVu94MWlYdGSUSinA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Linkage maps of the D genome using the ‘Hatsumochi’/‘Kitahonami’ RILs. Newly developed markers are prefixed with ‘tarc’. Box on the left of each chromosome indicates the putative position of the centromere." path-from-xml="dsy004f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057506/dsy004f4.tif" path-from-xml="dsy004f4.tif" target="_blank">View large</a><a section="118057506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004f4.png?Expires=2147483647&Signature=CxEUsgHNbaRo-VJEznxlWEr2hoiGYpMoy6the6UVRwnI-t3JZwBjEiolgTKcuLLOGrytY0nKoTjnZhTBpktix9GslljfS5-EANRV5ZS6EsxQ-z0BJ5J6Nhka7G5sL1cJ1ycs3UdDbjyfl5S6TQzfZi-I-aETchPTgV-3Hg~aJV2JP0n0GMAfrysiPXlIz8mg93~4X10U2MPtsT8~vv8D5~GEKDZ6cKqi~ZNlMCRSAKxQfpTs0jrDyaLC0LRy6I4GHLFOwu-2IPdmgWpNkvPRd3ujl3G~-qQdU0tA6zYfKpP-govgpbMDk8~VmKONDOV4AuoEOjCxvUy51fnPxAjTZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057506&ar=4898127&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy004f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Linkage maps of the D genome using the ‘Hatsumochi’/‘Kitahonami’ RILs. Newly developed markers are prefixed with ‘tarc’. Box on the left of each chromosome indicates the putative position of the centromere.</p></div></div>                            <h3 scrollto-destination=118057507 id="118057507" class="section-title" >3.5. Detection of polymorphisms among the six varieties</h3>
<p>Using the GS FLX plus, 8,253,381 reads were obtained from the six varieties. Of these reads, 5,473,010 reads were uniquely aligned to the IWGSC survey sequences.<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Using the criteria described above, 31,542 polymorphic site candidates were detected (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The pair-wise comparisons among the six varieties are shown in <a reveal-id="dsy004-T3" data-open="dsy004-T3" class="link link-reveal link-table xref-fig">Table 3</a>. The number of polymorphisms among the varieties ranged from 9,491 (‘Tohoku224’ vs. ‘Kinuhime’) to 19,926 (‘Hatsumochi’ vs. ‘Yumechikara’) and averaged 15,498 sites. Based on the mapping of the results to the IWGSC survey sequences, 5,986 polymorphic sites were predicted to be on the D genome and, therefore, can be considered potential markers for the D genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). <div content-id="dsy004-T3" class="table-modal"><div class="table-wrap"><div id="dsy004-T3" data-id="dsy004-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Number of polymorphic site candidates detected in pair-wise comparisons of six wheat varieties</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Kitahonami </th><th>Yumechikara </th><th>Tohoku224 </th><th>Hatsumochi </th><th>Kinuhime </th><th>Shunyou </th></tr></thead><tbody><tr><td>Kitahonami </td><td>— </td><td>15,638 </td><td>16,934 </td><td>16,800 </td><td>16,670 </td><td>15,774 </td></tr><tr><td>Yumechikara </td><td> </td><td>— </td><td>18,526 </td><td>19,926 </td><td>19,216 </td><td>18,219 </td></tr><tr><td>Tohoku224 </td><td> </td><td> </td><td>— </td><td>12,518 </td><td>9,491 </td><td>13,542 </td></tr><tr><td>Hatsumochi </td><td> </td><td> </td><td> </td><td>— </td><td>11,561 </td><td>14,008 </td></tr><tr><td>Kinuhime </td><td> </td><td> </td><td> </td><td> </td><td>— </td><td>13,661 </td></tr><tr><td>Shunyou </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>— </td></tr></tbody></table></div><div content-id="dsy004-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Kitahonami </th><th>Yumechikara </th><th>Tohoku224 </th><th>Hatsumochi </th><th>Kinuhime </th><th>Shunyou </th></tr></thead><tbody><tr><td>Kitahonami </td><td>— </td><td>15,638 </td><td>16,934 </td><td>16,800 </td><td>16,670 </td><td>15,774 </td></tr><tr><td>Yumechikara </td><td> </td><td>— </td><td>18,526 </td><td>19,926 </td><td>19,216 </td><td>18,219 </td></tr><tr><td>Tohoku224 </td><td> </td><td> </td><td>— </td><td>12,518 </td><td>9,491 </td><td>13,542 </td></tr><tr><td>Hatsumochi </td><td> </td><td> </td><td> </td><td>— </td><td>11,561 </td><td>14,008 </td></tr><tr><td>Kinuhime </td><td> </td><td> </td><td> </td><td> </td><td>— </td><td>13,661 </td></tr><tr><td>Shunyou </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>— </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy004-T3" data-id="dsy004-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Number of polymorphic site candidates detected in pair-wise comparisons of six wheat varieties</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Kitahonami </th><th>Yumechikara </th><th>Tohoku224 </th><th>Hatsumochi </th><th>Kinuhime </th><th>Shunyou </th></tr></thead><tbody><tr><td>Kitahonami </td><td>— </td><td>15,638 </td><td>16,934 </td><td>16,800 </td><td>16,670 </td><td>15,774 </td></tr><tr><td>Yumechikara </td><td> </td><td>— </td><td>18,526 </td><td>19,926 </td><td>19,216 </td><td>18,219 </td></tr><tr><td>Tohoku224 </td><td> </td><td> </td><td>— </td><td>12,518 </td><td>9,491 </td><td>13,542 </td></tr><tr><td>Hatsumochi </td><td> </td><td> </td><td> </td><td>— </td><td>11,561 </td><td>14,008 </td></tr><tr><td>Kinuhime </td><td> </td><td> </td><td> </td><td> </td><td>— </td><td>13,661 </td></tr><tr><td>Shunyou </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>— </td></tr></tbody></table></div><div content-id="dsy004-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Kitahonami </th><th>Yumechikara </th><th>Tohoku224 </th><th>Hatsumochi </th><th>Kinuhime </th><th>Shunyou </th></tr></thead><tbody><tr><td>Kitahonami </td><td>— </td><td>15,638 </td><td>16,934 </td><td>16,800 </td><td>16,670 </td><td>15,774 </td></tr><tr><td>Yumechikara </td><td> </td><td>— </td><td>18,526 </td><td>19,926 </td><td>19,216 </td><td>18,219 </td></tr><tr><td>Tohoku224 </td><td> </td><td> </td><td>— </td><td>12,518 </td><td>9,491 </td><td>13,542 </td></tr><tr><td>Hatsumochi </td><td> </td><td> </td><td> </td><td>— </td><td>11,561 </td><td>14,008 </td></tr><tr><td>Kinuhime </td><td> </td><td> </td><td> </td><td> </td><td>— </td><td>13,661 </td></tr><tr><td>Shunyou </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>— </td></tr></tbody></table></div></div></p>                            <h2 scrollto-destination=118057509 id="118057509" class="section-title" >4. Discussion</h2>
<p>In allohexaploid wheat, marker development is hampered by a large genome size, a high proportion of highly repetitive sequence (&gt;80%), and the presence of three different genomes in which corresponding genes share a high level of sequence similarity. Because of these characteristics of the wheat genome, nucleotide polymorphisms have been obtained primarily using transcript sequences<a reveal-id="dsy004-B33" data-open="dsy004-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsy004-B34" data-open="dsy004-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> or restriction-site flanking sequences.<a reveal-id="dsy004-B16" data-open="dsy004-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy004-B17" data-open="dsy004-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Although these methods have greatly contributed to increasing the number of markers, they do not provide a method of controlling the chromosomal locations of the detected polymorphisms. Therefore, although the number of polymorphisms is high, a significant bias occurs in the distribution of polymorphisms across the genome. Thus, targeted development of markers is required to saturate linkage maps. In this study, gene-enriched libraries were prepared using custom capture probes. To design the probes, we used positional information from sources such as the consensus linkage map (International Triticeae Mapping Initiative, ITMI),<a reveal-id="dsy004-B32" data-open="dsy004-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> bin-mapped ESTs,<a reveal-id="dsy004-B25" data-open="dsy004-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> PLUG markers<a reveal-id="dsy004-B35" data-open="dsy004-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and the barley physical map.<a reveal-id="dsy004-B26" data-open="dsy004-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Through this process, the map length in the D genome increased dramatically from 878.4 cM to 1333.1 cM (<a reveal-id="dsy004-F4" data-open="dsy004-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), and we were able to successfully develop markers in regions that were not covered by commercially available marker platforms, including DArT and 9K arrays. Because of the large genome size of wheat (16 Gb), the extraction of reliable polymorphic sites supported by sufficient read depth was believed to require a high-performance sequencer. However, in this study, using the GS FLX plus (Roche diagnostics), we obtained only 536 and 616 Mb sequences from the genomic DNA of ‘Hatsumochi’ and ‘Kitahonami’, respectively, indicating that less than 4% of the genome was sequenced in each variety. Despite this low coverage, we detected 12,551 polymorphic sites with an average read depth of 5.4 and 5.9 in ‘Hatsumochi’ and ‘Kitahonami’, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These observations indicated that the SeqCap EZ reagent kit (Roche Diagnostics) enriched the target sites by approximately 160-fold compared with whole genome analysis and greatly contributed to the effectiveness of polymorphism detection. In reference to the gene models of the IWGSC survey sequences,<a reveal-id="dsy004-B22" data-open="dsy004-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> most polymorphic sites were located near genes, indicating that we successfully eliminated repetitive sequences and effectively enriched the targets. The elimination of repetitive elements is important for genome mapping because these elements usually distort the mapping results. In this study, more than 40% of the polymorphic sites were in intron regions, and thus could not be found using transcriptome analyses. Because intron regions have a higher level of polymorphisms than exon regions, enriched genomic sequencing is a beneficial approach for developing gene-related markers. Furthermore, based on SnpEff analysis,<a reveal-id="dsy004-B27" data-open="dsy004-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> 60 and 1,630 polymorphisms were predicted to have high and moderate effects on the corresponding gene functions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Therefore, the strategy used in this study effectively identified polymorphisms that could potentially be related to agronomically important traits.</p><p>In allohexaploid wheat, the presence of three homoeologous genomes poses a challenge in SNP detection. Although a difference in degree is observed, many markers on the commercially available SNP arrays are affected by interference from the other two homoeologous sequences.<a reveal-id="dsy004-B36" data-open="dsy004-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In this study, the genome-specific primers were highly beneficial for defining alleles. Approximately 80% of the marker loci were successfully genotyped using the locus-specific approach when we validated the polymorphisms using the two parental varieties (<a reveal-id="dsy004-T2" data-open="dsy004-T2" class="link link-reveal link-table xref-fig">Table 2</a>). When we used the RIL populations, more than 70% of the markers were grouped into classes 1–3, indicating that these markers could be genotyped similarly to diploid species. Based on these results, our strategy of designing genome-specific primers was effective and demonstrated the importance of obtaining a priori knowledge of the polymorphisms among genomes by comparing the homoeologous sequences of interest.</p><p>The variations in the read number among samples and markers were investigated (<a reveal-id="dsy004-F3" data-open="dsy004-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In this study, we did not equilibrate the multiplex PCR samples from individual wells before mixing samples. Despite this simplification of the process, the read numbers among the samples were relatively uniform, except for one outlier sample that had an extremely low number of reads. Therefore, the protocol used in this study is beneficial for processing many samples. In contrast, the read numbers varied among the markers, with the read number ranging from nearly zero to more than 200 per sample. Markers with longer amplicons tended to have fewer reads (data not shown); however, other factors, such as the annealing efficiencies of the primers, also affected the read numbers. Because the multiplex levels of the samples and markers were determined according to the minimum number of reads required for each marker, a uniform distribution of read numbers among the markers is important. Further optimization of the multiplex PCR conditions will be beneficial in minimizing missing genotype values.</p><p>Because amplicon sequencing primers are typically designed to flank the site of interest, other polymorphisms in addition to the site of interest can sometimes be identified. When applying the method described in this paper to breeding or genetic analysis, the detection of new polymorphisms that are independent of the target sites could provide additional haplotype information for the samples of interest, and such haplotype information substantially improves power in the detection of marker-trait associations.<a reveal-id="dsy004-B37" data-open="dsy004-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Informative markers that contain more than two polymorphic sites could reduce the marker number required for an investigation, thereby reducing the cost for genotyping, and help in performing various genetic studies, such as association mapping and genomic selection in breeding programs.</p><p>Because the existing SNP arrays are substantially deficient in the number of D genome markers, we developed markers for the D genome to demonstrate the effectiveness of our approach. The polymorphism rate of the D genome is lower than that of the other two genomes, which is attributed to the evolutionary history of the D genome. According to recent studies, interploidy natural hybridization and subsequent introgression played a significant role in the diversification of common wheat,<a reveal-id="dsy004-B38" data-open="dsy004-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and therefore, the D genome had fewer opportunities to exchange genetic material than the A and B genomes. Although the polymorphic rate is low, comparable numbers of important genes and QTLs on the D genome are described in the catalog of gene symbols for wheat.<a reveal-id="dsy004-B39" data-open="dsy004-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Recent genome-wide association studies for pre-harvest sprouting using Chinese wheat landraces<a reveal-id="dsy004-B40" data-open="dsy004-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and European winter wheats<a reveal-id="dsy004-B41" data-open="dsy004-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> reported QTLs on 1D, 3D and 5D chromosomes. Additionally, resistant genes against wheat yellow mosaic virus and soil-borne wheat mosaic virus were found on chromosomes 2D and 5D, respectively.<a reveal-id="dsy004-B6" data-open="dsy004-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy004-B42" data-open="dsy004-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> For the fine mapping of these agronomically important genes or QTLs, the number of markers must be increased around the regions of interest. From the polymorphism survey using six varieties, around 6,000 polymorphic sites were detected on the D genome, and some sites have been successfully used to develop markers (data not shown). Therefore, the polymorphic information obtained in this study is a useful resource for the further development of markers across the genome.</p><p>In this study, we proposed efficient strategies for the detection of nucleotide polymorphisms among varieties of interest and the design of locus-specific primers to achieve robust high-throughput genotyping. The IWGSC’s continuous effort to obtain the first reference sequence of the spring wheat variety ‘Chinese Spring’ is opening the post-genomic era (<a class="link link-uri" href="http://www.wheatgenome.org/" target="_blank">http://www.wheatgenome.org/</a> (5 February 2018, date last accessed)). In this era, the motivation for collecting genomic resources using in-house materials is likely to increase. By comparing the polymorphic sites found in this study with probe sequences in publicly available SNP arrays, at least two-thirds of the total sites did not show any similarity with those probes in BLASTn searches (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S5</a></span>). The high percentage of new polymorphic sites indicates the importance of polymorphism surveys using materials of interest. In addition to the number of polymorphic sites, the polymorphic frequencies among materials are also important. Compared with the existing SNP arrays, the polymorphic sites among the six varieties in this study provided highly polymorphic markers among Japanese materials, particularly among varieties from Central and Western Japan (data not shown). To date, the germplasm used in genomic studies has been limited to the leading varieties in developed countries. However, for specific traits, such as resistance to disease or abiotic stress, many sources of germplasm from around the world remain unexamined. Because the method described in this study is less expensive, more flexible and more reliable than previous methods, this method is suitable for pan-genome studies that must process many haplotypes.</p>                            <h2 scrollto-destination=118057516 id="118057516" class="section-title" >Data Availability</h2>
<p>All sequences analysed in the present study were deposited into the DDBJ/GenBank/MMBL database with accession numbers DRA006270. Sample indices are prefixed to sequence names as follows: ‘Kitahonami’, H7U3MUP; ‘Hatsumochi’, H7YSFDH; ‘Shunyou’, IY4LCZI; ‘Kinuhime’, IZHNKMJ; ‘Tohoku224’, IZO1F6G; and ‘Yumechikara’, HKAWHI.</p>                            <h2 scrollto-destination=118057518 id="118057518" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This study was supported by a grant from the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics-based Technology for Agricultural Improvement, NGB-1002 and NGB-1007).</p>                            <h3 scrollto-destination=118057520 id="118057520" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=118057522 id="118057522" class="backsection-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118057524 id="118057524" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy004-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">P. K.</div></div>, <div class="name string-name"><div class="surname">Rustgi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mir</div> <div class="given-names">R. R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Array-based high-throughput DNA markers for crop improvement</div>, <div class="source">Heredity</div> , <div class="volume">101</div>, <div class="fpage">5</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array-based%20high-throughput%20DNA%20markers%20for%20crop%20improvement&amp;author=P.%20K.%20Gupta&amp;author=S.%20Rustgi&amp;author=R.%20R.%20Mir&amp;publication_year=2008&amp;journal=Heredity&amp;volume=101&amp;pages=5-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2008.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2008.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2008.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18461083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y. B.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y. L.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">C. X.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">S. B.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Prasanna</div> <div class="given-names">B. M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Whole-genome strategies for marker-assisted plant breeding</div>, <div class="source">Mol. Breed</div> ., <div class="volume">29</div>, <div class="fpage">833</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20strategies%20for%20marker-assisted%20plant%20breeding&amp;author=Y.%20B.%20Xu&amp;author=Y.%20L.%20Lu&amp;author=C.%20X.%20Xie&amp;author=S.%20B.%20Gao&amp;author=J.%20M.%20Wan&amp;author=B.%20M.%20Prasanna&amp;publication_year=2012&amp;journal=Mol.%20Breed&amp;volume=29&amp;pages=833-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-012-9699-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-012-9699-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-012-9699-6"> </span></div> </div></div></div></div></div><div content-id="dsy004-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cavanagh</div> <div class="given-names">C. R.</div></div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">110</div>, <div class="fpage">8057</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20diversity%20uncovers%20multiple%20targets%20of%20selection%20for%20improvement%20in%20hexaploid%20wheat%20landraces%20and%20cultivars&amp;author=C.%20R.%20Cavanagh&amp;author=S.%20Chao&amp;author=S.%20Wang&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=8057-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217133110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217133110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217133110"> </span></div> </div></div></div></div></div><div content-id="dsy004-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Forrest</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Characterization of polyploid wheat genomic diversity using a high-density 90,000 single nucleotide polymorphism array</div>, <div class="source">Plant Biotechnol. J</div> ., <div class="volume">12</div>, <div class="fpage">787</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20polyploid%20wheat%20genomic%20diversity%20using%20a%20high-density%2090%2C000%20single%20nucleotide%20polymorphism%20array&amp;author=S.%20Wang&amp;author=D.%20Wong&amp;author=K.%20Forrest&amp;publication_year=2014&amp;journal=Plant%20Biotechnol.%20J&amp;volume=12&amp;pages=787-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2014.12.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2014.12.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2014.12.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24646323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iehisa</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A high-density genetic map with array-based markers facilitates structural and quantitative trait locus analyses of the common wheat genome</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">555</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20genetic%20map%20with%20array-based%20markers%20facilitates%20structural%20and%20quantitative%20trait%20locus%20analyses%20of%20the%20common%20wheat%20genome&amp;author=J.%20C.%20Iehisa&amp;author=R.%20Ohno&amp;author=T.%20Kimura&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=555-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24972598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bockus</div> <div class="given-names">W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome-wide association analysis identified SNPs closely linked to a gene resistant to soil-borne wheat mosaic virus</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">127</div>, <div class="fpage">1039</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20identified%20SNPs%20closely%20linked%20to%20a%20gene%20resistant%20to%20soil-borne%20wheat%20mosaic%20virus&amp;author=S.%20Liu&amp;author=X.%20Yang&amp;author=D.%20Zhang&amp;author=G.%20Bai&amp;author=S.%20Chao&amp;author=W.%20Bockus&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet&amp;volume=127&amp;pages=1039-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2277-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2277-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2277-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Naruoka</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Garland-Campbell</div> <div class="given-names">K. A.</div></div>, <div class="name string-name"><div class="surname">Carter</div> <div class="given-names">A. H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide association mapping for stripe rust (<em>Puccinia striiformis</em> F. sp. <em>tritici</em>) in US Pacific Northwest winter wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">128</div>, <div class="fpage">1083</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20for%20stripe%20rust%20(Puccinia%20striiformis%20F.%20sp.%20tritici)%20in%20US%20Pacific%20Northwest%20winter%20wheat%20(Triticum%20aestivum%20L.)&amp;author=Y.%20Naruoka&amp;author=K.%20A.%20Garland-Campbell&amp;author=A.%20H.%20Carter&amp;publication_year=2015&amp;journal=Theor.%20Appl.%20Genet&amp;volume=128&amp;pages=1083-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-015-2492-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-015-2492-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-015-2492-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25754424" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Q. H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y. X.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">S. H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">High-density genetic linkage map construction and QTL mapping of grain shape and size in the wheat population Yanda1817 x Beinong6</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0118144</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20genetic%20linkage%20map%20construction%20and%20QTL%20mapping%20of%20grain%20shape%20and%20size%20in%20the%20wheat%20population%20Yanda1817%20x%20Beinong6&amp;author=Q.%20H.%20Wu&amp;author=Y.%20X.%20Chen&amp;author=S.%20H.%20Zhou&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0118144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0118144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0118144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0118144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25675376" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bulli</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Pumphrey</div> <div class="given-names">M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genetic architecture of resistance to stripe rust in a global winter wheat germplasm collection</div>, <div class="source">G3-Genes Genomes Genet</div> ., <div class="volume">6</div>, <div class="fpage">2237</div>–<div class="lpage">53</div>.</div></div></div></div><div content-id="dsy004-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genome-wide association analysis on pre-harvest sprouting resistance and grain color in U.S. winter wheat</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">794</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20on%20pre-harvest%20sprouting%20resistance%20and%20grain%20color%20in%20U.S.%20winter%20wheat&amp;author=M.%20Lin&amp;author=D.%20Zhang&amp;author=S.%20Liu&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=794" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-3148-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-3148-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-3148-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27729004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">L. X.</div></div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Sorrells</div> <div class="given-names">M. E.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Identification and validation of single nucleotide polymorphic markers linked to Ug99 stem rust resistance in spring wheat</div>, <div class="source">PLoS One</div> , <div class="volume">12</div>, <div class="fpage">e0171963</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20validation%20of%20single%20nucleotide%20polymorphic%20markers%20linked%20to%20Ug99%20stem%20rust%20resistance%20in%20spring%20wheat&amp;author=L.%20X.%20Yu&amp;author=S.%20Chao&amp;author=R.%20P.%20Singh&amp;author=M.%20E.%20Sorrells&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0171963" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0171963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0171963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0171963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28241006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S. M.</div></div>, <div class="name string-name"><div class="surname">Dubcovsky</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dvorak</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">727</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population-%20and%20genome-specific%20patterns%20of%20linkage%20disequilibrium%20and%20SNP%20variation%20in%20spring%20and%20winter%20wheat%20(Triticum%20aestivum%20L.)&amp;author=S.%20M.%20Chao&amp;author=J.%20Dubcovsky&amp;author=J.%20Dvorak&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=727" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-727" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-727" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-727"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21190581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ito</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Association mapping and validation of QTLs for flour yield in the soft winter wheat variety Kitahonami</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e111337</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20and%20validation%20of%20QTLs%20for%20flour%20yield%20in%20the%20soft%20winter%20wheat%20variety%20Kitahonami&amp;author=G.%20Ishikawa&amp;author=K.%20Nakamura&amp;author=H.%20Ito&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e111337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0111337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0111337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0111337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25360619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">QTL analysis of spike morphological traits and plant height in winter wheat (<em>Triticum aestivum</em> L.) using a high-density SNP and SSR-based linkage map</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">1617</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20spike%20morphological%20traits%20and%20plant%20height%20in%20winter%20wheat%20(Triticum%20aestivum%20L.)%20using%20a%20high-density%20SNP%20and%20SSR-based%20linkage%20map&amp;author=H.%20Zhai&amp;author=Z.%20Feng&amp;author=J.%20Li&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=7&amp;pages=1617" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27872629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">B. E.</div></div>, <div class="name string-name"><div class="surname">George</div> <div class="given-names">A. W.</div></div>, <div class="name string-name"><div class="surname">Forrest</div> <div class="given-names">K. L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat</div>, <div class="source">Plant Biotechnol. J</div> ., <div class="volume">10</div>, <div class="fpage">826</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20multiparent%20advanced%20generation%20inter-cross%20population%20for%20genetic%20analysis%20in%20wheat&amp;author=B.%20E.%20Huang&amp;author=A.%20W.%20George&amp;author=K.%20L.%20Forrest&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J&amp;volume=10&amp;pages=826-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00702.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00702.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00702.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22594629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Poland</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Sorrells</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Jannink</div> <div class="given-names">J. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e32253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20high-density%20genetic%20maps%20for%20barley%20and%20wheat%20using%20a%20novel%20two-enzyme%20genotyping-by-sequencing%20approach&amp;author=J.%20A.%20Poland&amp;author=P.%20J.%20Brown&amp;author=M.%20E.%20Sorrells&amp;author=J.%20L.%20Jannink&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e32253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0032253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0032253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0032253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kanamori</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Katayose</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Handa</div> <div class="given-names">H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Characterization of a mini core collection of Japanese wheat varieties using single-nucleotide polymorphisms generated by genotyping-by-sequencing</div>, <div class="source">Breed Sci</div> ., <div class="volume">66</div>, <div class="fpage">213</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20mini%20core%20collection%20of%20Japanese%20wheat%20varieties%20using%20single-nucleotide%20polymorphisms%20generated%20by%20genotyping-by-sequencing&amp;author=F.%20Kobayashi&amp;author=T.%20Tanaka&amp;author=H.%20Kanamori&amp;author=J.%20Wu&amp;author=Y.%20Katayose&amp;author=H.%20Handa&amp;publication_year=2016&amp;journal=Breed%20Sci&amp;volume=66&amp;pages=213-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.66.213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.66.213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.66.213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27162493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcussen</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sandve</div> <div class="given-names">S. R.</div></div>, <div class="name string-name"><div class="surname">Heier</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Ancient hybridizations among the ancestral genomes of bread wheat</div>, <div class="source">Science</div> , <div class="volume">345</div>, <div class="fpage">1250092</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ancient%20hybridizations%20among%20the%20ancestral%20genomes%20of%20bread%20wheat&amp;author=T.%20Marcussen&amp;author=S.%20R.%20Sandve&amp;author=L.%20Heier&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=1250092" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1250092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1250092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1250092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25035499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brenchley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Spannagl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pfeifer</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Analysis of the bread wheat genome using whole-genome shotgun sequencing</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">705</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20bread%20wheat%20genome%20using%20whole-genome%20shotgun%20sequencing&amp;author=R.%20Brenchley&amp;author=M.%20Spannagl&amp;author=M.%20Pfeifer&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=705-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Henry</div> <div class="given-names">I. M.</div></div>, <div class="name string-name"><div class="surname">Nagalakshmi</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Lieberman</div> <div class="given-names">M. C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">1382</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20genome-wide%20detection%20and%20cataloging%20of%20EMS-induced%20mutations%20using%20exome%20capture%20and%20next-generation%20sequencing&amp;author=I.%20M.%20Henry&amp;author=U.%20Nagalakshmi&amp;author=M.%20C.%20Lieberman&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=1382-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.121590" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.121590" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.121590"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24728647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bernardo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">St Amand</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Using next generation sequencing for multiplexed trait-linked markers in wheat</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0143890</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20next%20generation%20sequencing%20for%20multiplexed%20trait-linked%20markers%20in%20wheat&amp;author=A.%20Bernardo&amp;author=S.%20Wang&amp;author=P.%20St%20Amand&amp;author=G.%20Bai&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0143890" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0143890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0143890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0143890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26625271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="collab">IWGSC</div> <div class="year">2014</div>, 
<div class="article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome</div>, <div class="source">Science</div> , <div class="volume">345</div>, <div class="fpage">1251788</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1251788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1251788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1251788"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25035500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TriFLDB: a database of clustered full-length coding sequences from <em>Triticeae</em> with applications to comparative grass genomics</div>, <div class="source">Plant Physiol</div> ., <div class="volume">150</div>, <div class="fpage">1135</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TriFLDB%3A%20a%20database%20of%20clustered%20full-length%20coding%20sequences%20from%20Triticeae%20with%20applications%20to%20comparative%20grass%20genomics&amp;author=K.%20Mochida&amp;author=T.%20Yoshida&amp;author=T.%20Sakurai&amp;author=Y.%20Ogihara&amp;author=K.%20Shinozaki&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=150&amp;pages=1135-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.138214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.138214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.138214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19448038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Yonemaru</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PCR-based landmark unique gene (PLUG) markers effectively assign homoeologous wheat genes to A, B and D genomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">8</div>, <div class="fpage">135</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR-based%20landmark%20unique%20gene%20(PLUG)%20markers%20effectively%20assign%20homoeologous%20wheat%20genes%20to%20A%2C%20B%20and%20D%20genomes&amp;author=G.%20Ishikawa&amp;author=J.%20Yonemaru&amp;author=M.%20Saito&amp;author=T.%20Nakamura&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17535443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qi</div> <div class="given-names">L. L.</div></div>, <div class="name string-name"><div class="surname">Echalier</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Chao</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">A chromosome bin map of 16,000 expressed sequence tag loci and distribution of genes among the three genomes of polyploid wheat</div>, <div class="source">Genetics</div> , <div class="volume">168</div>, <div class="fpage">701</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosome%20bin%20map%20of%2016%2C000%20expressed%20sequence%20tag%20loci%20and%20distribution%20of%20genes%20among%20the%20three%20genomes%20of%20polyploid%20wheat&amp;author=L.%20L.%20Qi&amp;author=B.%20Echalier&amp;author=S.%20Chao&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=701-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.034868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.034868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.034868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15514046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mayer</div> <div class="given-names">K. F.</div></div>, <div class="name string-name"><div class="surname">Waugh</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">J. W.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A physical, genetic and functional sequence assembly of the barley genome</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">711</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20physical%2C%20genetic%20and%20functional%20sequence%20assembly%20of%20the%20barley%20genome&amp;author=K.%20F.%20Mayer&amp;author=R.%20Waugh&amp;author=J.%20W.%20Brown&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=711-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cingolani</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Platts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang le</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso</div>, <div class="source">Fly</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20Wang%20le&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div></div>, et al.   <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Toh</div> <div class="given-names">H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Recent developments in the MAFFT multiple sequence alignment program</div>, <div class="source">Brief Bioinform</div> ., <div class="volume">9</div>, <div class="fpage">286</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20developments%20in%20the%20MAFFT%20multiple%20sequence%20alignment%20program&amp;author=K.%20Katoh&amp;author=H.%20Toh&amp;publication_year=2008&amp;journal=Brief%20Bioinform&amp;volume=9&amp;pages=286-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18372315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Appels</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">X.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Functional markers in wheat: current status and future prospects</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">125</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20markers%20in%20wheat%3A%20current%20status%20and%20future%20prospects&amp;author=Y.%20Liu&amp;author=Z.%20He&amp;author=R.%20Appels&amp;author=X.%20Xia&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet&amp;volume=125&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1829-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1829-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1829-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22366867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lorieux</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">MapDisto: fast and efficient computation of genetic linkage maps</div>, <div class="source">Mol. Breed</div> ., <div class="volume">30</div>, <div class="fpage">1231</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapDisto%3A%20fast%20and%20efficient%20computation%20of%20genetic%20linkage%20maps&amp;author=M.%20Lorieux&amp;publication_year=2012&amp;journal=Mol.%20Breed&amp;volume=30&amp;pages=1231-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-012-9706-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-012-9706-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-012-9706-y"> </span></div> </div></div></div></div></div><div content-id="dsy004-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sorrells</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Gustafson</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Somers</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Reconstruction of the synthetic W7984 x Opata M85 wheat reference population</div>, <div class="source">Genome</div> , <div class="volume">54</div>, <div class="fpage">875</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20synthetic%20W7984%20x%20Opata%20M85%20wheat%20reference%20population&amp;author=M.%20E.%20Sorrells&amp;author=J.%20P.%20Gustafson&amp;author=D.%20Somers&amp;publication_year=2011&amp;journal=Genome&amp;volume=54&amp;pages=875-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21999208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Allen</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Barker</div> <div class="given-names">G. L.</div></div>, <div class="name string-name"><div class="surname">Berry</div> <div class="given-names">S. T.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Plant Biotechnol. J</div> ., <div class="volume">9</div>, <div class="fpage">1086</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%2C%20single-nucleotide%20polymorphism%20discovery%20and%20linkage%20analysis%20in%20hexaploid%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=A.%20M.%20Allen&amp;author=G.%20L.%20Barker&amp;author=S.%20T.%20Berry&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J&amp;volume=9&amp;pages=1086-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21627760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akhunov</div> <div class="given-names">E. D.</div></div>, <div class="name string-name"><div class="surname">Akhunova</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">O. D.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Nucleotide diversity maps reveal variation in diversity among wheat genomes and chromosomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20diversity%20maps%20reveal%20variation%20in%20diversity%20among%20wheat%20genomes%20and%20chromosomes&amp;author=E.%20D.%20Akhunov&amp;author=A.%20R.%20Akhunova&amp;author=O.%20D.%20Anderson&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21156062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ashida</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Localization of anchor loci representing five hundred annotated rice genes to wheat chromosomes using PLUG markers</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">118</div>, <div class="fpage">499</div>–<div class="lpage">514</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localization%20of%20anchor%20loci%20representing%20five%20hundred%20annotated%20rice%20genes%20to%20wheat%20chromosomes%20using%20PLUG%20markers&amp;author=G.%20Ishikawa&amp;author=T.%20Nakamura&amp;author=T.%20Ashida&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet&amp;volume=118&amp;pages=499-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0916-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0916-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0916-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19057889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akhunov</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Nicolet</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dvorak</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">119</div>, <div class="fpage">507</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20genotyping%20in%20polyploid%20wheat%20with%20the%20Illumina%20GoldenGate%20assay&amp;author=E.%20Akhunov&amp;author=C.%20Nicolet&amp;author=J.%20Dvorak&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet&amp;volume=119&amp;pages=507-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1059-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1059-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1059-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19449174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y. L.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Z. M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Comparative LD mapping using single SNPs and haplotypes identifies QTL for plant height and biomass as secondary traits of drought tolerance in maize</div>, <div class="source">Mol. Breeding</div> , <div class="volume">30</div>, <div class="fpage">407</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20LD%20mapping%20using%20single%20SNPs%20and%20haplotypes%20identifies%20QTL%20for%20plant%20height%20and%20biomass%20as%20secondary%20traits%20of%20drought%20tolerance%20in%20maize&amp;author=Y.%20L.%20Lu&amp;author=J.%20Xu&amp;author=Z.%20M.%20Yuan&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=30&amp;pages=407-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9631-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9631-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9631-5"> </span></div> </div></div></div></div></div><div content-id="dsy004-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Evolution of polyploid <em>Triticum</em> wheats under cultivation: the role of domestication, natural hybridization and allopolyploid speciation in their diversification</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">52</div>, <div class="fpage">750</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20polyploid%20Triticum%20wheats%20under%20cultivation%3A%20the%20role%20of%20domestication%2C%20natural%20hybridization%20and%20allopolyploid%20speciation%20in%20their%20diversification&amp;author=Y.%20Matsuoka&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol&amp;volume=52&amp;pages=750-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21317146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McIntosh</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Dubcovsky</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, Catalogue of gene symbols for wheat. <a class="link link-uri" href="https://shigen.nig.ac.jp/wheat/komugi/genes/symbolClassList.jsp" target="_blank">https://shigen.nig.ac.jp/wheat/komugi/genes/symbolClassList.jsp</a> (5 February 2018, date last accessed)</div></div></div></div><div content-id="dsy004-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">M. P.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Genome-wide association study for pre-harvest sprouting resistance in a large germplasm collection of Chinese wheat landraces</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">8</div>, <div class="fpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20for%20pre-harvest%20sprouting%20resistance%20in%20a%20large%20germplasm%20collection%20of%20Chinese%20wheat%20landraces&amp;author=Y.%20Zhou&amp;author=H.%20Tang&amp;author=M.%20P.%20Cheng&amp;publication_year=2017&amp;journal=Front.%20Plant%20Sci&amp;volume=8&amp;pages=401" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28428791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albrecht</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Oberforster</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kempf</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome-wide association mapping of preharvest sprouting resistance in a diversity panel of European winter wheats</div>, <div class="source">J. Appl. Genet</div> ., <div class="volume">56</div>, <div class="fpage">277</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20of%20preharvest%20sprouting%20resistance%20in%20a%20diversity%20panel%20of%20European%20winter%20wheats&amp;author=T.%20Albrecht&amp;author=M.%20Oberforster&amp;author=H.%20Kempf&amp;publication_year=2015&amp;journal=J.%20Appl.%20Genet&amp;volume=56&amp;pages=277-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13353-015-0286-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13353-015-0286-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13353-015-0286-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25924791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy004-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nishio</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kojima</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hayata</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Mapping a gene conferring resistance to Wheat yellow mosaic virus in European winter wheat cultivar ‘Ibis’ (<em>Triticum aestivum</em> L.)</div>, <div class="source">Euphytica</div> , <div class="volume">176</div>, <div class="fpage">223</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20a%20gene%20conferring%20resistance%20to%20Wheat%20yellow%20mosaic%20virus%20in%20European%20winter%20wheat%20cultivar%20%E2%80%98Ibis%E2%80%99%20(Triticum%20aestivum%20L.)&amp;author=Z.%20Nishio&amp;author=H.%20Kojima&amp;author=A.%20Hayata&amp;publication_year=2010&amp;journal=Euphytica&amp;volume=176&amp;pages=223-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0229-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0229-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0229-5"> </span></div> </div></div></div></div></div><div content-id="dsy004-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy004-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mayer</div> <div class="given-names">K. F.</div></div>, <div class="name string-name"><div class="surname">Taudien</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Martis</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Gene content and virtual gene order of barley chromosome 1H</div>, <div class="source">Plant Physiol</div> ., <div class="volume">151</div>, <div class="fpage">496</div>–<div class="lpage">505</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20content%20and%20virtual%20gene%20order%20of%20barley%20chromosome%201H&amp;author=K.%20F.%20Mayer&amp;author=S.%20Taudien&amp;author=M.%20Martis&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=151&amp;pages=496-505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.142612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.142612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.142612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19692534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4898127&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy004/1/dsy004_supp.zip?Expires=2147483647&amp;Signature=A~ohJXg0vDrCbnv1oBuFqq4WKDXA~fn8uJdmjnZ3pZHGitNLgZP0~~Guv3HeBrSbHBW6-D7fyG6UfZEeWJ3neUe1KLw~C0PH8UsMgBgZ5LkH2i7-b-ye3MYFhGBpuEScIJqmNItiXg9kD13Cmwyj4xZRLNxLvD8bgIYkkMzhkOiCwdOzz1PPvrW8t-2wUdjAgnt3wOuk-WLG1zoLlJ9F5K-23zh3VOiBm6jCWjkewuL9O4hmO5lgeqN7YdBsDIGXKKFeOjL3TSmr7Wn59u8nY5dRSEHAOplc62yvikQ4XZYxwZbuQL-BlWu5XTxiuouLXOrylncf3DFYvrT-txYWDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">904</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4898127"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f3%2f317%2f4898127">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F3%2F317%2F4898127" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4898127'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="efficient approach for the development of genome-specific markers in allohexaploid wheat (Triticum aestivum L.) and its application in the construction of high-density linkage maps of the D genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f317%2f4898127">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

