

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/4/343/3056574","@type":"ScholarlyArticle","name":"The evolutionary fate of the chloroplast and nuclear  rps16  genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from  Lupinus ","datePublished":"2017-02-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2017-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx006","keywords":["chloroplast genomes","<italic>Lupinus</italic>","functional gene relocation","repeated sequences","phylogeny"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/4/343/3056574","author":[{"name":"Keller, J.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"},{"name":"Rousseau-Gueutin, M.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France   2 IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France","@type":"Person"},{"name":"Martin, G.E.","affiliation":"3 CIRAD (Centre de coopération Internationale en Recherche Agronomique pour le Développement), UMR AGAP, F-34398 Montpellier, France","@type":"Person"},{"name":"Morice, J.","affiliation":"2 IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France","@type":"Person"},{"name":"Boutte, J.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"},{"name":"Coissac, E.","affiliation":"4 Laboratoire d'Ecologie Alpine, CNRS - Université de Grenoble 1 - Université de Savoie, 38041 Grenoble, France","@type":"Person"},{"name":"Ourari, M.","affiliation":"5 Département des Sciences Biologiques, Faculté des Sciences de la Nature et de la Vie, Université Abderrahmane Mira, 06000 Bejaia, Algeria","@type":"Person"},{"name":"Aïnouche, M.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"},{"name":"Salmon, A.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"},{"name":"Cabello-Hurtado, F.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"},{"name":"Aïnouche, A.","affiliation":"1 UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France","@type":"Person"}],"description":"Abstract.  The Fabaceae family is considered as a model system for understanding chloroplast genome evolution due to the presence of extensive structural rearra","pageStart":"343","pageEnd":"358","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The evolutionary fate of the chloroplast and nuclear  rps16  genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from  Lupinus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="evolutionary fate of the chloroplast and nuclear  rps16  genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from  Lupinus " />
    <meta property="og:description" content="Abstract.  The Fabaceae family is considered as a model system for understanding chloroplast genome evolution due to the presence of extensive structural rearra" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/4/343/3056574" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Keller, J." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author" content="Rousseau-Gueutin, M." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author_institution" content="IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France" /><meta name="citation_author" content="Martin, G.E." /><meta name="citation_author_institution" content="CIRAD (Centre de coopération Internationale en Recherche Agronomique pour le Développement), UMR AGAP, F-34398 Montpellier, France" /><meta name="citation_author" content="Morice, J." /><meta name="citation_author_institution" content="IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France" /><meta name="citation_author" content="Boutte, J." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author" content="Coissac, E." /><meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS - Université de Grenoble 1 - Université de Savoie, 38041 Grenoble, France" /><meta name="citation_author" content="Ourari, M." /><meta name="citation_author_institution" content="Département des Sciences Biologiques, Faculté des Sciences de la Nature et de la Vie, Université Abderrahmane Mira, 06000 Bejaia, Algeria" /><meta name="citation_author" content="Aïnouche, M." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author" content="Salmon, A." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author" content="Cabello-Hurtado, F." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_author" content="Aïnouche, A." /><meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France" /><meta name="citation_title" content="The evolutionary fate of the chloroplast and nuclear rps16 genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from Lupinus" /><meta name="citation_firstpage" content="343" /><meta name="citation_lastpage" content="358" /><meta name="citation_doi" content="10.1093/dnares/dsx006" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2017/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Legume comparative genomics: progress in phylogenetics and phylogenomics;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2006;  citation_volume=9;  citation_pages=99-103; " /><meta name="citation_reference" content="citation_publisher=Royal Botanic Gardens, Kew;  citation_title=Legumes of the World;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Towards a new classification system for legumes: progress report from the 6th International Legume Conference; citation_author=LPWG;  citation_journal_title=South Afr. J. Bot;  citation_year=2013;  citation_volume=89;  citation_pages=3-9; " /><meta name="citation_reference" content="citation_title=The rest of the iceberg. Legume diversity and evolution in a phylogenetic context;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=900-10; " /><meta name="citation_reference" content="citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses;  citation_journal_title=Nature;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Legumes: importance and constraints to greater use;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=872-7; " /><meta name="citation_reference" content="citation_title=Extensive reorganization of the plastid genome of Trifolium subterraneum (Fabaceae) is associated with numerous repeated sequences and novel DNA insertions;  citation_journal_title=J. Mol. Evol;  citation_year=2008;  citation_volume=67;  citation_pages=696-704; " /><meta name="citation_reference" content="citation_title=Multiple independent losses of two genes and one intron from legume chloroplast genomes;  citation_journal_title=Syst. Bot;  citation_year=1995;  citation_volume=20;  citation_pages=272" /><meta name="citation_reference" content="citation_title=Extreme reconfiguration of plastid genomes in the angiosperm family Geraniaceae: rearrangements, repeats, and codon usage;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=583-600; " /><meta name="citation_reference" content="citation_title=Extensive rearrangements in the chloroplast genome of Trachelium caeruleum are associated with repeats and tRNA genes;  citation_journal_title=J. Mol. Evol;  citation_year=2008;  citation_volume=66;  citation_pages=350-61; " /><meta name="citation_reference" content="citation_publisher=Springer, Netherlands, Dordrecht;  citation_title=Genomics of Chloroplasts and Mitochondria;  citation_year=2012;  citation_pages=103-26; " /><meta name="citation_reference" content="citation_title=Localized hypermutation and associated gene losses in legume chloroplast genomes;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1700-10; " /><meta name="citation_reference" content="citation_title=Reconstruction of the ancestral plastid genome in Geraniaceae reveals a correlation between genome rearrangements, repeats, and nucleotide substitution rates;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=645-59; " /><meta name="citation_reference" content="citation_title=Mimosoid legume plastome evolution: IR expansion, tandem repeat expansions, and accelerated rate of evolution in clpP;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=16958" /><meta name="citation_reference" content="citation_title=The first complete chloroplast genome of the Genistoid legume Lupinus luteus: evidence for a novel major lineage-specific rearrangement and new insights regarding plastome evolution in the legume family;  citation_journal_title=Ann. Bot;  citation_year=2014;  citation_volume=113;  citation_pages=1197-210; " /><meta name="citation_reference" content="citation_title=Plastid genome sequences of legumes reveal parallel inversions and multiple losses of rps16 in papilionoids: parallel inversions and rps16 losses in legumes;  citation_journal_title=J. Syst. Evol;  citation_year=2015;  citation_volume=53;  citation_pages=458-68; " /><meta name="citation_reference" content="citation_title=Complete plastome sequences from Glycine syndetika and six additional perennial wild relatives of soybean;  citation_journal_title=G3amp58 GenesGenomesGenetics;  citation_year=2014;  citation_volume=4;  citation_pages=2023-33; " /><meta name="citation_reference" content="citation_title=The complete sequence of the Acacia ligulata chloroplast genome reveals a highly divergent clpP1 gene. C.-H. Kuo (ed.);  citation_journal_title=PLOS ONE;  citation_year=2015;  citation_volume=10;  citation_pages=e0125768" /><meta name="citation_reference" content="citation_title=Unusual structure of geranium chloroplast DNA: a triple-sized inverted repeat, extensive gene duplications, multiple inversions, and two repeat families;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1987;  citation_volume=84;  citation_pages=769-73; " /><meta name="citation_reference" content="citation_title=The distribution and phylogenetic significance of a 50-kb chloroplast DNA inversion in the flowering plant family Leguminosae;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=1996;  citation_volume=5;  citation_pages=429-438; " /><meta name="citation_reference" content="citation_title=Complete plastid genome sequence of the chickpea (Cicer arietinum) and the phylogenetic distribution of rps12 and clpP intron losses among legumes (Leguminosae);  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2008;  citation_volume=48;  citation_pages=1204-17; " /><meta name="citation_reference" content="citation_title=A phylogeny of legumes (Leguminosae) based on analysis of the plastid matK gene resolves many well-supported subclades within the family;  citation_journal_title=Am. J. Bot;  citation_year=2004;  citation_volume=91;  citation_pages=1846-1862; " /><meta name="citation_reference" content="citation_title=A chloroplast DNA inversion as a subtribal character in the phaseoleae (Leguminosae);  citation_journal_title=Syst. Bot;  citation_year=1990;  citation_volume=15;  citation_pages=378" /><meta name="citation_reference" content="citation_title=Rapid evolutionary change of common bean (Phaseolus vulgaris L.) plastome, and the genomic diversification of legume chloroplasts;  citation_journal_title=BMC Genom;  citation_year=2007;  citation_volume=8;  citation_pages=228" /><meta name="citation_reference" content="citation_title=The chloroplast genome sequence of mungbean (Vigna radiata) determined by high-throughput pyrosequencing: structural organization and phylogenetic relationships;  citation_journal_title=DNA Res;  citation_year=2010;  citation_volume=17;  citation_pages=11-22; " /><meta name="citation_reference" content="citation_title=Capturing the biofuel wellhead and powerhouse: the chloroplast and mitochondrial genomes of the leguminous feedstock tree Pongamia pinnata. J.H. Badger (ed.);  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e51687" /><meta name="citation_reference" content="citation_title=Endosymbiotic gene transfer: organelle genomes forge eukaryotic chromosomes;  citation_journal_title=Nat. Rev. Genet;  citation_year=2004;  citation_volume=5;  citation_pages=123-35; " /><meta name="citation_reference" content="citation_title=Substitution of the gene for chloroplast RPS16 was assisted by generation of a dual targeting signal;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=1566-75; " /><meta name="citation_reference" content="citation_title=Nonessential plastid-encoded ribosomal proteins in tobacco: a developmental role for plastid translation and implications for reductive genome evolution;  citation_journal_title=Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=3137-55; " /><meta name="citation_reference" content="citation_title=The tobacco plastid accD gene is essential and is required for leaf development;  citation_journal_title=Plant J;  citation_year=2005;  citation_volume=44;  citation_pages=237-44; " /><meta name="citation_reference" content="citation_title=The chloroplast clpP gene, encoding a proteolytic subunit of ATP-dependent protease, is indispensable for chloroplast development in tobacco;  citation_journal_title=Plant Cell Physiol;  citation_year=2001;  citation_volume=42;  citation_pages=264-73; " /><meta name="citation_reference" content="citation_title=Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2007;  citation_volume=104;  citation_pages=19369-19374; " /><meta name="citation_reference" content="citation_title=Revisiting the phylogeny of papilionoid legumes: new insights from comprehensively sampled early-branching lineages;  citation_journal_title=Am. J. Bot;  citation_year=2012;  citation_volume=99;  citation_pages=1991-2013; " /><meta name="citation_reference" content="citation_title=Evolutionary origin of highly repetitive plastid genomes within the clover genus (Trifolium);  citation_journal_title=BMC Evol. Biol;  citation_year=2014;  citation_volume=14;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Legumes as a model plant family. Genomics for food and feed report of the cross-legume advances through genomics conference;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=137;  citation_pages=1228-35; " /><meta name="citation_reference" content="citation_publisher=CAB International, Wallingford, Oxon, UK; New York, NY, USA;  citation_title=Lupins as Crop Plants: Biology, Production, and Utilization;  citation_year=1998; " /><meta name="citation_reference" content="citation_title=Proteomics for exploiting diversity of lupin seed storage proteins and their use as nutraceuticals for health and welfare;  citation_journal_title=J. Proteomics;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Draft genome sequence, and a sequence-defined genetic linkage map of the legume crop species Lupinus angustifolius. X.L. Cui, (ed.);  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e64799" /><meta name="citation_reference" content="citation_title=Phylogenetic relationships in Lupinus (Fabaceae: Papilionoideae) based on internal transcribed spacer sequences (ITS) of nuclear ribosomal DNA;  citation_journal_title=Am. J. Bot;  citation_year=1999;  citation_volume=86;  citation_pages=590-607; " /><meta name="citation_reference" content="citation_title=Multiple continental radiations and correlates of diversification in Lupinus (Leguminosae): testing for key innovation with incomplete taxon sampling;  citation_journal_title=Syst. Biol;  citation_year=2012;  citation_volume=61;  citation_pages=443-60; " /><meta name="citation_reference" content=" Eastwood R.J.,Drummond C.S.,Schifino-Wittmann M.T.,Hughes . 2008 , Lupins for health and wealth: 12th International Lupin Conference, Fremantle, Western Australia, 14–18 September 2008. In: J.A., Palta, J.D., Berger, and International Lupin Association (eds.). International Lupin Association, Canterbury, New Zealand. " /><meta name="citation_reference" content="citation_title=Island radiation on a continental scale: exceptional rates of plant diversification after uplift of the Andes;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2006;  citation_volume=103;  citation_pages=10334-10339; " /><meta name="citation_reference" content="citation_title=Molecular phylogeny and phylogeography of Lupinus (Leguminosae) inferred from nucleotide sequences of therbcL gene and ITS 1 + 2 regions of rDNA;  citation_journal_title=Plant Syst. Evol;  citation_year=1997;  citation_volume=208;  citation_pages=139-167; " /><meta name="citation_reference" content="citation_title=Isolation, phylogeny and evolution of the SymRK gene in the legume genus Lupinus L;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2011;  citation_volume=60;  citation_pages=49-61; " /><meta name="citation_reference" content="citation_title=New data and phylogenetic placement of the enigmatic Old World lupin: Lupinus mariae-josephi H. Pascual;  citation_journal_title=Genet. Resour. Crop Evol;  citation_year=2011;  citation_volume=58;  citation_pages=101-14; " /><meta name="citation_reference" content="citation_title=Molecular phylogeny, diversification and character evolution in Lupinus (Fabaceae) with special attention to Mediterranean and African lupines;  citation_journal_title=Plant Syst. Evol;  citation_year=2004;  citation_volume=246; " /><meta name="citation_reference" content="citation_title=From barcodes to genomes: extending the concept of DNA barcoding;  citation_journal_title=Mol. Ecol;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=Automatic annotation of organellar genomes with DOGMA;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3252-5; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1639-1645; " /><meta name="citation_reference" content="citation_title=BaCelLo: a Balanced subCellular Localization predictor;  citation_journal_title=Protoc. Exch;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Prediction of subcellular localization using sequence-biased recurrent networks;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Predicting subcellular localization of proteins based on their N-terminal amino acid sequence;  citation_journal_title=J. Mol. Biol;  citation_year=2000;  citation_volume=300;  citation_pages=1005-16; " /><meta name="citation_reference" content="citation_title=MultiLoc2: integrating phylogeny and Gene Ontology terms improves subcellular protein localization prediction;  citation_journal_title=BMC Bioinf;  citation_year=2009;  citation_volume=10;  citation_pages=274" /><meta name="citation_reference" content="citation_title=Predotar: a tool for rapidly screening proteomes forN-terminal targeting sequences;  citation_journal_title=Proteomics;  citation_year=2004;  citation_volume=4;  citation_pages=1581-90; " /><meta name="citation_reference" content="citation_title=Structure and activities of group II introns;  citation_journal_title=Annu. Rev. Biochem;  citation_year=1995;  citation_volume=64;  citation_pages=435-61; " /><meta name="citation_reference" content="citation_title=Comparative and functional anatomy of group II catalytic introns—a review;  citation_journal_title=Gene;  citation_year=1989;  citation_volume=82;  citation_pages=5-30; " /><meta name="citation_reference" content="citation_title=Group II introns: structure and catalytic versatility of large natural ribozymes;  citation_journal_title=Crit. Rev. Biochem. Mol. Biol;  citation_year=2003;  citation_volume=38;  citation_pages=249-303; " /><meta name="citation_reference" content="citation_title=Different status of the gene for ribosomal protein S16 in the chloroplast genome during evolution of the genus Arabidopsis and closely related species;  citation_journal_title=Genes Genet. Syst;  citation_year=2010;  citation_volume=85;  citation_pages=319-26; " /><meta name="citation_reference" content="citation_title=Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1647-9; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=APE: analyses of phylogenetics and evolution in R language;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=289-90; " /><meta name="citation_reference" content="citation_title=A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences;  citation_journal_title=J. Mol. Evol;  citation_year=1980;  citation_volume=16;  citation_pages=111-20; " /><meta name="citation_reference" content="citation_title=Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models;  citation_journal_title=Mol. Biol. Evol;  citation_year=2000;  citation_volume=17;  citation_pages=32-43; " /><meta name="citation_reference" content="citation_title=REPuter: the manifold applications of repeat analysis on a genomic scale;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=4633-42; " /><meta name="citation_reference" content="citation_title=Complete chloroplast genome sequence of glycine max and comparative analyses with other legume genomes;  citation_journal_title=Plant Mol. Biol;  citation_year=2005;  citation_volume=59;  citation_pages=309-22; " /><meta name="citation_reference" content="  Mayer C.   2007 , October, Phobos: Highly Accurate Search for Perfect and Imperfect Tandem Repeats in Complete Genomes by Christoph Mayer. " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers;  citation_journal_title=Methods Mol. Biol. Clifton NJ;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-3066; " /><meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap;  citation_journal_title=Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783" /><meta name="citation_reference" content="citation_title=jModelTest: phylogenetic model averaging;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=1253-6; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=A molecular timeline for the origin of photosynthetic eukaryotes;  citation_journal_title=Mol. Biol. Evol;  citation_year=2004;  citation_volume=21;  citation_pages=809-18; " /><meta name="citation_reference" content="citation_title=Why chloroplasts and mitochondria contain genomes;  citation_journal_title=Comp. Funct. Genomics;  citation_year=2003;  citation_volume=4;  citation_pages=31-6; " /><meta name="citation_reference" content="citation_title=Energy transduction anchors genes in organelles: Problems;  citation_journal_title=and paradigms. BioEssays;  citation_year=2005;  citation_volume=27;  citation_pages=426-35; " /><meta name="citation_reference" content="citation_title=Why chloroplasts and mitochondria retain their own genomes and genetic systems: colocation for redox regulation of gene expression;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2015;  citation_volume=112;  citation_pages=10231-8; " /><meta name="citation_reference" content="citation_title=Direct transcriptional control of the chloroplast genes psbA and psaAB adjusts photosynthesis to light energy distribution in plants;  citation_journal_title=IUBMB Life;  citation_year=1999;  citation_volume=48;  citation_pages=271-6; " /><meta name="citation_reference" content="citation_title=Massively convergent evolution for ribosomal protein gene content in plastid and mitochondrial genomes;  citation_journal_title=Genome Biol. Evol;  citation_year=2013;  citation_volume=5;  citation_pages=2318-29; " /><meta name="citation_reference" content="citation_title=Uncovering the protein translocon at the chloroplast inner envelope membrane;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=571-574; " /><meta name="citation_reference" content="citation_title=ycf1, the most promising plastid DNA barcode of land plants;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=8348" /><meta name="citation_reference" content="citation_title=Highly variable chloroplast markers for evaluating plant phylogeny at low taxonomic levels and for DNA barcoding. A. Moustafa, (ed.);  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e35071" /><meta name="citation_reference" content="  Swofford D.L.   2001 , PAUP*: Phylogenetic Analysis Using Parsimony (and other methods) 4.0.b5. " /><meta name="citation_reference" content="citation_publisher=Springer, US;  citation_title=Molecular Systematics of Plants II;  citation_year=1998;  citation_pages=297-348; " /><meta name="citation_reference" content="citation_title=Diversification of Lupinus (Leguminosae) in the western New World: derived evolution of perennial life history and colonization of montane habitats;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2008;  citation_volume=48;  citation_pages=408-21; " /><meta name="citation_reference" content="citation_title=The highly rearranged chloroplast genome of Trachelium caeruleum (Campanulaceae): multiple inversions, inverted repeat expansion and contraction, transposition, insertions/deletions, and several repeat families;  citation_journal_title=Curr. Genet;  citation_year=1997;  citation_volume=31;  citation_pages=419-429; " /><meta name="citation_reference" content="citation_title=Dispersed repeats and structural reorganization in subclover chloroplast DNA;  citation_journal_title=Mol. Biol. Evol;  citation_year=1989;  citation_volume=6;  citation_pages=355-68; " /><meta name="citation_reference" content="citation_title=Intramolecular recombination of chloroplast genome mediated by short direct-repeat sequences in wheat species;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1988;  citation_volume=85;  citation_pages=8573-8577; " /><meta name="citation_reference" content="citation_title=Length polymorphisms of simple sequence repeat DNA in soybean;  citation_journal_title=Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=1131-9; " /><meta name="citation_reference" content="citation_title=Chloroplast microsatellite diversity in Phaseolus vulgaris;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Development of 12 chloroplast microsatellite markers in Vigna unguiculata (Fabaceae) and amplification in Phaseolus vulgaris;  citation_journal_title=Appl. Plant Sci;  citation_year=2014;  citation_volume=2;  citation_pages=1300075" /><meta name="citation_reference" content="citation_title=Hypervariable microsatellites provide a general source of polymorphic DNA markers for the chloroplast genome;  citation_journal_title=Curr. Biol;  citation_year=1995;  citation_volume=5;  citation_pages=1023-1029; " /><meta name="citation_reference" content="citation_title=Chloroplast microsatellites: new tools for studies in plant ecology and evolution;  citation_journal_title=Trends Ecol. Evol;  citation_year=2001;  citation_volume=16;  citation_pages=142-147; " /><meta name="citation_reference" content="citation_title=Polymorphic chloroplast simple sequence repeat primers for systematic and population studies in the genus Hordeum;  citation_journal_title=Mol. Ecol;  citation_year=1999;  citation_volume=8;  citation_pages=505-511; " /><meta name="citation_reference" content="citation_title=Extraordinarily polymorphic microsatellite DNA in barley: species diversity, chromosomal locations, and population dynamics;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1994;  citation_volume=91;  citation_pages=5466-70; " /><meta name="citation_reference" content="citation_title=Abundance, polymorphism and genetic mapping of microsatellites in rice;  citation_journal_title=Mol. Gen. Genet. MGG;  citation_year=1993;  citation_volume=241;  citation_pages=225-235; " /><meta name="citation_reference" content="citation_title=The eukaryotic tree of life from a global phylogenomic perspective;  citation_journal_title=Cold Spring Harb. Perspect. Biol;  citation_year=2014;  citation_volume=6;  citation_pages=a016147" /><meta name="citation_reference" content="citation_title=Phylogeny and molecular evolution of the green algae;  citation_journal_title=Crit. Rev. Plant Sci;  citation_year=2012;  citation_volume=31;  citation_pages=1-46; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/4/343/22892844/dsx006.pdf" /><meta name="description" content="Abstract.  The Fabaceae family is considered as a model system for understanding chloroplast genome evolution due to the presence of extensive structural rearra" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/4/343/3056574" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/4/343/3056574" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/4/343/3056574");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx006';
    </script>
<title>
	evolutionary fate of the chloroplast and nuclear rps16 genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from Lupinus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3056574" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb3nATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2VsbGVyLCBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVNUiBDTlJTIDY1NTMgRWNvYmlvLCBPU1VSIChPYnNlcnZhdG9pcmUgZGVzIFNjaWVuY2VzIGRlIGwnVW5pdmVycyBkZSBSZW5uZXMpLCBVbml2ZXJzaXTDqSBkZSBSZW5uZXMgMSwgMzUwNDIgUmVubmVzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSb3Vzc2VhdS1HdWV1dGluLCBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVNUiBDTlJTIDY1NTMgRWNvYmlvLCBPU1VSIChPYnNlcnZhdG9pcmUgZGVzIFNjaWVuY2VzIGRlIGwnVW5pdmVycyBkZSBSZW5uZXMpLCBVbml2ZXJzaXTDqSBkZSBSZW5uZXMgMSwgMzUwNDIgUmVubmVzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJR0VQUCwgSU5SQSwgQWdyb2NhbXB1cyBPdWVzdCwgVW5pdmVyc2l0w6kgZGUgUmVubmVzIDEsIEJQMzUzMjcsIDM1NjUzIExlIFJoZXUgQ2VkZXgsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hcnRpbiwgRy5FLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFEIChDZW50cmUgZGUgY29vcMOpcmF0aW9uIEludGVybmF0aW9uYWxlIGVuIFJlY2hlcmNoZSBBZ3Jvbm9taXF1ZSBwb3VyIGxlIETDqXZlbG9wcGVtZW50KSwgVU1SIEFHQVAsIEYtMzQzOTggTW9udHBlbGxpZXIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmljZSwgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJR0VQUCwgSU5SQSwgQWdyb2NhbXB1cyBPdWVzdCwgVW5pdmVyc2l0w6kgZGUgUmVubmVzIDEsIEJQMzUzMjcsIDM1NjUzIExlIFJoZXUgQ2VkZXgsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJvdXR0ZSwgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVTVIgQ05SUyA2NTUzIEVjb2JpbywgT1NVUiAoT2JzZXJ2YXRvaXJlIGRlcyBTY2llbmNlcyBkZSBsJ1VuaXZlcnMgZGUgUmVubmVzKSwgVW5pdmVyc2l0w6kgZGUgUmVubmVzIDEsIDM1MDQyIFJlbm5lcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ29pc3NhYywgRS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b2lyZSBkJ0Vjb2xvZ2llIEFscGluZSwgQ05SUyAtIFVuaXZlcnNpdMOpIGRlIEdyZW5vYmxlIDEgLSBVbml2ZXJzaXTDqSBkZSBTYXZvaWUsIDM4MDQxIEdyZW5vYmxlLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPdXJhcmksIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRMOpcGFydGVtZW50IGRlcyBTY2llbmNlcyBCaW9sb2dpcXVlcywgRmFjdWx0w6kgZGVzIFNjaWVuY2VzIGRlIGxhIE5hdHVyZSBldCBkZSBsYSBWaWUsIFVuaXZlcnNpdMOpIEFiZGVycmFobWFuZSBNaXJhLCAwNjAwMCBCZWphaWEsIEFsZ2VyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBw69ub3VjaGUsIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVU1SIENOUlMgNjU1MyBFY29iaW8sIE9TVVIgKE9ic2VydmF0b2lyZSBkZXMgU2NpZW5jZXMgZGUgbCdVbml2ZXJzIGRlIFJlbm5lcyksIFVuaXZlcnNpdMOpIGRlIFJlbm5lcyAxLCAzNTA0MiBSZW5uZXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhbG1vbiwgQS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVTVIgQ05SUyA2NTUzIEVjb2JpbywgT1NVUiAoT2JzZXJ2YXRvaXJlIGRlcyBTY2llbmNlcyBkZSBsJ1VuaXZlcnMgZGUgUmVubmVzKSwgVW5pdmVyc2l0w6kgZGUgUmVubmVzIDEsIDM1MDQyIFJlbm5lcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FiZWxsby1IdXJ0YWRvLCBGLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVNUiBDTlJTIDY1NTMgRWNvYmlvLCBPU1VSIChPYnNlcnZhdG9pcmUgZGVzIFNjaWVuY2VzIGRlIGwnVW5pdmVycyBkZSBSZW5uZXMpLCBVbml2ZXJzaXTDqSBkZSBSZW5uZXMgMSwgMzUwNDIgUmVubmVzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBw69ub3VjaGUsIEEuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVU1SIENOUlMgNjU1MyBFY29iaW8sIE9TVVIgKE9ic2VydmF0b2lyZSBkZXMgU2NpZW5jZXMgZGUgbCdVbml2ZXJzIGRlIFJlbm5lcyksIFVuaXZlcnNpdMOpIGRlIFJlbm5lcyAxLCAzNTA0MiBSZW5uZXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIGV2b2x1dGlvbmFyeSBmYXRlIG9mIHRoZSBjaGxvcm9wbGFzdCBhbmQgbnVjbGVhciBycHMxNiBnZW5lcyBhcyByZXZlYWxlZCB0aHJvdWdoIHRoZSBzZXF1ZW5jaW5nIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNlcyBvZiBmb3VyIG5vdmVsIGxlZ3VtZSBjaGxvcm9wbGFzdCBnZW5vbWVzIGZyb20gTHVwaW51cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWUgY29tcGFyYXRpdmUgZ2Vub21pY3M6IHByb2dyZXNzIGluIHBoeWxvZ2VuZXRpY3MgYW5kIHBoeWxvZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTk5LTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVJveWFsIEJvdGFuaWMgR2FyZGVucywgS2V3OyAgY2l0YXRpb25fdGl0bGU9TGVndW1lcyBvZiB0aGUgV29ybGQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZHMgYSBuZXcgY2xhc3NpZmljYXRpb24gc3lzdGVtIGZvciBsZWd1bWVzOiBwcm9ncmVzcyByZXBvcnQgZnJvbSB0aGUgNnRoIEludGVybmF0aW9uYWwgTGVndW1lIENvbmZlcmVuY2U7IGNpdGF0aW9uX2F1dGhvcj1MUFdHOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Tb3V0aCBBZnIuIEouIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlc3Qgb2YgdGhlIGljZWJlcmcuIExlZ3VtZSBkaXZlcnNpdHkgYW5kIGV2b2x1dGlvbiBpbiBhIHBoeWxvZ2VuZXRpYyBjb250ZXh0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTkwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1lZGljYWdvIGdlbm9tZSBwcm92aWRlcyBpbnNpZ2h0IGludG8gdGhlIGV2b2x1dGlvbiBvZiByaGl6b2JpYWwgc3ltYmlvc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZXM6IGltcG9ydGFuY2UgYW5kIGNvbnN0cmFpbnRzIHRvIGdyZWF0ZXIgdXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTg3Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgcmVvcmdhbml6YXRpb24gb2YgdGhlIHBsYXN0aWQgZ2Vub21lIG9mIFRyaWZvbGl1bSBzdWJ0ZXJyYW5ldW0gKEZhYmFjZWFlKSBpcyBhc3NvY2lhdGVkIHdpdGggbnVtZXJvdXMgcmVwZWF0ZWQgc2VxdWVuY2VzIGFuZCBub3ZlbCBETkEgaW5zZXJ0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9Njk2LTcwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgaW5kZXBlbmRlbnQgbG9zc2VzIG9mIHR3byBnZW5lcyBhbmQgb25lIGludHJvbiBmcm9tIGxlZ3VtZSBjaGxvcm9wbGFzdCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yNzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRyZW1lIHJlY29uZmlndXJhdGlvbiBvZiBwbGFzdGlkIGdlbm9tZXMgaW4gdGhlIGFuZ2lvc3Blcm0gZmFtaWx5IEdlcmFuaWFjZWFlOiByZWFycmFuZ2VtZW50cywgcmVwZWF0cywgYW5kIGNvZG9uIHVzYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01ODMtNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgcmVhcnJhbmdlbWVudHMgaW4gdGhlIGNobG9yb3BsYXN0IGdlbm9tZSBvZiBUcmFjaGVsaXVtIGNhZXJ1bGV1bSBhcmUgYXNzb2NpYXRlZCB3aXRoIHJlcGVhdHMgYW5kIHRSTkEgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTM1MC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBOZXRoZXJsYW5kcywgRG9yZHJlY2h0OyAgY2l0YXRpb25fdGl0bGU9R2Vub21pY3Mgb2YgQ2hsb3JvcGxhc3RzIGFuZCBNaXRvY2hvbmRyaWE7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsaXplZCBoeXBlcm11dGF0aW9uIGFuZCBhc3NvY2lhdGVkIGdlbmUgbG9zc2VzIGluIGxlZ3VtZSBjaGxvcm9wbGFzdCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTcwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3Rpb24gb2YgdGhlIGFuY2VzdHJhbCBwbGFzdGlkIGdlbm9tZSBpbiBHZXJhbmlhY2VhZSByZXZlYWxzIGEgY29ycmVsYXRpb24gYmV0d2VlbiBnZW5vbWUgcmVhcnJhbmdlbWVudHMsIHJlcGVhdHMsIGFuZCBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiByYXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NjQ1LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW1vc29pZCBsZWd1bWUgcGxhc3RvbWUgZXZvbHV0aW9uOiBJUiBleHBhbnNpb24sIHRhbmRlbSByZXBlYXQgZXhwYW5zaW9ucywgYW5kIGFjY2VsZXJhdGVkIHJhdGUgb2YgZXZvbHV0aW9uIGluIGNscFA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNjk1OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmaXJzdCBjb21wbGV0ZSBjaGxvcm9wbGFzdCBnZW5vbWUgb2YgdGhlIEdlbmlzdG9pZCBsZWd1bWUgTHVwaW51cyBsdXRldXM6IGV2aWRlbmNlIGZvciBhIG5vdmVsIG1ham9yIGxpbmVhZ2Utc3BlY2lmaWMgcmVhcnJhbmdlbWVudCBhbmQgbmV3IGluc2lnaHRzIHJlZ2FyZGluZyBwbGFzdG9tZSBldm9sdXRpb24gaW4gdGhlIGxlZ3VtZSBmYW1pbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gQm90OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTExOTctMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFzdGlkIGdlbm9tZSBzZXF1ZW5jZXMgb2YgbGVndW1lcyByZXZlYWwgcGFyYWxsZWwgaW52ZXJzaW9ucyBhbmQgbXVsdGlwbGUgbG9zc2VzIG9mIHJwczE2IGluIHBhcGlsaW9ub2lkczogcGFyYWxsZWwgaW52ZXJzaW9ucyBhbmQgcnBzMTYgbG9zc2VzIGluIGxlZ3VtZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFN5c3QuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz00NTgtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHBsYXN0b21lIHNlcXVlbmNlcyBmcm9tIEdseWNpbmUgc3luZGV0aWthIGFuZCBzaXggYWRkaXRpb25hbCBwZXJlbm5pYWwgd2lsZCByZWxhdGl2ZXMgb2Ygc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzNhbXA1OCBHZW5lc0dlbm9tZXNHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjAyMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIHNlcXVlbmNlIG9mIHRoZSBBY2FjaWEgbGlndWxhdGEgY2hsb3JvcGxhc3QgZ2Vub21lIHJldmVhbHMgYSBoaWdobHkgZGl2ZXJnZW50IGNscFAxIGdlbmUuIEMuLUguIEt1byAoZWQuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExPUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDEyNTc2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVudXN1YWwgc3RydWN0dXJlIG9mIGdlcmFuaXVtIGNobG9yb3BsYXN0IEROQTogYSB0cmlwbGUtc2l6ZWQgaW52ZXJ0ZWQgcmVwZWF0LCBleHRlbnNpdmUgZ2VuZSBkdXBsaWNhdGlvbnMsIG11bHRpcGxlIGludmVyc2lvbnMsIGFuZCB0d28gcmVwZWF0IGZhbWlsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz03NjktNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkaXN0cmlidXRpb24gYW5kIHBoeWxvZ2VuZXRpYyBzaWduaWZpY2FuY2Ugb2YgYSA1MC1rYiBjaGxvcm9wbGFzdCBETkEgaW52ZXJzaW9uIGluIHRoZSBmbG93ZXJpbmcgcGxhbnQgZmFtaWx5IExlZ3VtaW5vc2FlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBoeWxvZ2VuZXQuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQyOS00Mzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHBsYXN0aWQgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSBhbmQgdGhlIHBoeWxvZ2VuZXRpYyBkaXN0cmlidXRpb24gb2YgcnBzMTIgYW5kIGNscFAgaW50cm9uIGxvc3NlcyBhbW9uZyBsZWd1bWVzIChMZWd1bWlub3NhZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGh5bG9nZW5ldC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEyMDQtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGh5bG9nZW55IG9mIGxlZ3VtZXMgKExlZ3VtaW5vc2FlKSBiYXNlZCBvbiBhbmFseXNpcyBvZiB0aGUgcGxhc3RpZCBtYXRLIGdlbmUgcmVzb2x2ZXMgbWFueSB3ZWxsLXN1cHBvcnRlZCBzdWJjbGFkZXMgd2l0aGluIHRoZSBmYW1pbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xODQ2LTE4NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hsb3JvcGxhc3QgRE5BIGludmVyc2lvbiBhcyBhIHN1YnRyaWJhbCBjaGFyYWN0ZXIgaW4gdGhlIHBoYXNlb2xlYWUgKExlZ3VtaW5vc2FlKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQm90OyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9Mzc4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgZXZvbHV0aW9uYXJ5IGNoYW5nZSBvZiBjb21tb24gYmVhbiAoUGhhc2VvbHVzIHZ1bGdhcmlzIEwuKSBwbGFzdG9tZSwgYW5kIHRoZSBnZW5vbWljIGRpdmVyc2lmaWNhdGlvbiBvZiBsZWd1bWUgY2hsb3JvcGxhc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTIyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjaGxvcm9wbGFzdCBnZW5vbWUgc2VxdWVuY2Ugb2YgbXVuZ2JlYW4gKFZpZ25hIHJhZGlhdGEpIGRldGVybWluZWQgYnkgaGlnaC10aHJvdWdocHV0IHB5cm9zZXF1ZW5jaW5nOiBzdHJ1Y3R1cmFsIG9yZ2FuaXphdGlvbiBhbmQgcGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FwdHVyaW5nIHRoZSBiaW9mdWVsIHdlbGxoZWFkIGFuZCBwb3dlcmhvdXNlOiB0aGUgY2hsb3JvcGxhc3QgYW5kIG1pdG9jaG9uZHJpYWwgZ2Vub21lcyBvZiB0aGUgbGVndW1pbm91cyBmZWVkc3RvY2sgdHJlZSBQb25nYW1pYSBwaW5uYXRhLiBKLkguIEJhZGdlciAoZWQuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MTY4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZG9zeW1iaW90aWMgZ2VuZSB0cmFuc2Zlcjogb3JnYW5lbGxlIGdlbm9tZXMgZm9yZ2UgZXVrYXJ5b3RpYyBjaHJvbW9zb21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMjMtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YnN0aXR1dGlvbiBvZiB0aGUgZ2VuZSBmb3IgY2hsb3JvcGxhc3QgUlBTMTYgd2FzIGFzc2lzdGVkIGJ5IGdlbmVyYXRpb24gb2YgYSBkdWFsIHRhcmdldGluZyBzaWduYWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE1NjYtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbmVzc2VudGlhbCBwbGFzdGlkLWVuY29kZWQgcmlib3NvbWFsIHByb3RlaW5zIGluIHRvYmFjY286IGEgZGV2ZWxvcG1lbnRhbCByb2xlIGZvciBwbGFzdGlkIHRyYW5zbGF0aW9uIGFuZCBpbXBsaWNhdGlvbnMgZm9yIHJlZHVjdGl2ZSBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzEzNy01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRvYmFjY28gcGxhc3RpZCBhY2NEIGdlbmUgaXMgZXNzZW50aWFsIGFuZCBpcyByZXF1aXJlZCBmb3IgbGVhZiBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTIzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNobG9yb3BsYXN0IGNscFAgZ2VuZSwgZW5jb2RpbmcgYSBwcm90ZW9seXRpYyBzdWJ1bml0IG9mIEFUUC1kZXBlbmRlbnQgcHJvdGVhc2UsIGlzIGluZGlzcGVuc2FibGUgZm9yIGNobG9yb3BsYXN0IGRldmVsb3BtZW50IGluIHRvYmFjY287ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTI2NC03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgODEgZ2VuZXMgZnJvbSA2NCBwbGFzdGlkIGdlbm9tZXMgcmVzb2x2ZXMgcmVsYXRpb25zaGlwcyBpbiBhbmdpb3NwZXJtcyBhbmQgaWRlbnRpZmllcyBnZW5vbWUtc2NhbGUgZXZvbHV0aW9uYXJ5IHBhdHRlcm5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTkzNjktMTkzNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJldmlzaXRpbmcgdGhlIHBoeWxvZ2VueSBvZiBwYXBpbGlvbm9pZCBsZWd1bWVzOiBuZXcgaW5zaWdodHMgZnJvbSBjb21wcmVoZW5zaXZlbHkgc2FtcGxlZCBlYXJseS1icmFuY2hpbmcgbGluZWFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xOTkxLTIwMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBvcmlnaW4gb2YgaGlnaGx5IHJlcGV0aXRpdmUgcGxhc3RpZCBnZW5vbWVzIHdpdGhpbiB0aGUgY2xvdmVyIGdlbnVzIChUcmlmb2xpdW0pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWd1bWVzIGFzIGEgbW9kZWwgcGxhbnQgZmFtaWx5LiBHZW5vbWljcyBmb3IgZm9vZCBhbmQgZmVlZCByZXBvcnQgb2YgdGhlIGNyb3NzLWxlZ3VtZSBhZHZhbmNlcyB0aHJvdWdoIGdlbm9taWNzIGNvbmZlcmVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9MTIyOC0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNBQiBJbnRlcm5hdGlvbmFsLCBXYWxsaW5nZm9yZCwgT3hvbiwgVUs7IE5ldyBZb3JrLCBOWSwgVVNBOyAgY2l0YXRpb25fdGl0bGU9THVwaW5zIGFzIENyb3AgUGxhbnRzOiBCaW9sb2d5LCBQcm9kdWN0aW9uLCBhbmQgVXRpbGl6YXRpb247ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21pY3MgZm9yIGV4cGxvaXRpbmcgZGl2ZXJzaXR5IG9mIGx1cGluIHNlZWQgc3RvcmFnZSBwcm90ZWlucyBhbmQgdGhlaXIgdXNlIGFzIG51dHJhY2V1dGljYWxzIGZvciBoZWFsdGggYW5kIHdlbGZhcmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSwgYW5kIGEgc2VxdWVuY2UtZGVmaW5lZCBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHRoZSBsZWd1bWUgY3JvcCBzcGVjaWVzIEx1cGludXMgYW5ndXN0aWZvbGl1cy4gWC5MLiBDdWksIChlZC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTY0Nzk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgaW4gTHVwaW51cyAoRmFiYWNlYWU6IFBhcGlsaW9ub2lkZWFlKSBiYXNlZCBvbiBpbnRlcm5hbCB0cmFuc2NyaWJlZCBzcGFjZXIgc2VxdWVuY2VzIChJVFMpIG9mIG51Y2xlYXIgcmlib3NvbWFsIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEJvdDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTU5MC02MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIGNvbnRpbmVudGFsIHJhZGlhdGlvbnMgYW5kIGNvcnJlbGF0ZXMgb2YgZGl2ZXJzaWZpY2F0aW9uIGluIEx1cGludXMgKExlZ3VtaW5vc2FlKTogdGVzdGluZyBmb3Iga2V5IGlubm92YXRpb24gd2l0aCBpbmNvbXBsZXRlIHRheG9uIHNhbXBsaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NDQzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgRWFzdHdvb2QgUi5KLixEcnVtbW9uZCBDLlMuLFNjaGlmaW5vLVdpdHRtYW5uIE0uVC4sSHVnaGVzIC4gMjAwOCAsIEx1cGlucyBmb3IgaGVhbHRoIGFuZCB3ZWFsdGg6IDEydGggSW50ZXJuYXRpb25hbCBMdXBpbiBDb25mZXJlbmNlLCBGcmVtYW50bGUsIFdlc3Rlcm4gQXVzdHJhbGlhLCAxNOKAkzE4IFNlcHRlbWJlciAyMDA4LiBJbjogSi5BLiwgUGFsdGEsIEouRC4sIEJlcmdlciwgYW5kIEludGVybmF0aW9uYWwgTHVwaW4gQXNzb2NpYXRpb24gKGVkcy4pLiBJbnRlcm5hdGlvbmFsIEx1cGluIEFzc29jaWF0aW9uLCBDYW50ZXJidXJ5LCBOZXcgWmVhbGFuZC4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNsYW5kIHJhZGlhdGlvbiBvbiBhIGNvbnRpbmVudGFsIHNjYWxlOiBleGNlcHRpb25hbCByYXRlcyBvZiBwbGFudCBkaXZlcnNpZmljYXRpb24gYWZ0ZXIgdXBsaWZ0IG9mIHRoZSBBbmRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTEwMzM0LTEwMzM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgcGh5bG9nZW55IGFuZCBwaHlsb2dlb2dyYXBoeSBvZiBMdXBpbnVzIChMZWd1bWlub3NhZSkgaW5mZXJyZWQgZnJvbSBudWNsZW90aWRlIHNlcXVlbmNlcyBvZiB0aGVyYmNMIGdlbmUgYW5kIElUUyAxICsgMiByZWdpb25zIG9mIHJETkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFN5c3QuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjA4OyAgY2l0YXRpb25fcGFnZXM9MTM5LTE2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uLCBwaHlsb2dlbnkgYW5kIGV2b2x1dGlvbiBvZiB0aGUgU3ltUksgZ2VuZSBpbiB0aGUgbGVndW1lIGdlbnVzIEx1cGludXMgTDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9NDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBkYXRhIGFuZCBwaHlsb2dlbmV0aWMgcGxhY2VtZW50IG9mIHRoZSBlbmlnbWF0aWMgT2xkIFdvcmxkIGx1cGluOiBMdXBpbnVzIG1hcmlhZS1qb3NlcGhpIEguIFBhc2N1YWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0LiBSZXNvdXIuIENyb3AgRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTEwMS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHBoeWxvZ2VueSwgZGl2ZXJzaWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXIgZXZvbHV0aW9uIGluIEx1cGludXMgKEZhYmFjZWFlKSB3aXRoIHNwZWNpYWwgYXR0ZW50aW9uIHRvIE1lZGl0ZXJyYW5lYW4gYW5kIEFmcmljYW4gbHVwaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgU3lzdC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb20gYmFyY29kZXMgdG8gZ2Vub21lczogZXh0ZW5kaW5nIHRoZSBjb25jZXB0IG9mIEROQSBiYXJjb2Rpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGljIGFubm90YXRpb24gb2Ygb3JnYW5lbGxhciBnZW5vbWVzIHdpdGggRE9HTUE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzI1Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjb3M6IGFuIGluZm9ybWF0aW9uIGFlc3RoZXRpYyBmb3IgY29tcGFyYXRpdmUgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjM5LTE2NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhQ2VsTG86IGEgQmFsYW5jZWQgc3ViQ2VsbHVsYXIgTG9jYWxpemF0aW9uIHByZWRpY3RvcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvdG9jLiBFeGNoOyAgY2l0YXRpb25feWVhcj0yMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHN1YmNlbGx1bGFyIGxvY2FsaXphdGlvbiB1c2luZyBzZXF1ZW5jZS1iaWFzZWQgcmVjdXJyZW50IG5ldHdvcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpbmcgc3ViY2VsbHVsYXIgbG9jYWxpemF0aW9uIG9mIHByb3RlaW5zIGJhc2VkIG9uIHRoZWlyIE4tdGVybWluYWwgYW1pbm8gYWNpZCBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTMwMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMDUtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpTG9jMjogaW50ZWdyYXRpbmcgcGh5bG9nZW55IGFuZCBHZW5lIE9udG9sb2d5IHRlcm1zIGltcHJvdmVzIHN1YmNlbGx1bGFyIHByb3RlaW4gbG9jYWxpemF0aW9uIHByZWRpY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmY7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkb3RhcjogYSB0b29sIGZvciByYXBpZGx5IHNjcmVlbmluZyBwcm90ZW9tZXMgZm9yTi10ZXJtaW5hbCB0YXJnZXRpbmcgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIGFjdGl2aXRpZXMgb2YgZ3JvdXAgSUkgaW50cm9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9NDM1LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmQgZnVuY3Rpb25hbCBhbmF0b215IG9mIGdyb3VwIElJIGNhdGFseXRpYyBpbnRyb25z4oCUYSByZXZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz01LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Hcm91cCBJSSBpbnRyb25zOiBzdHJ1Y3R1cmUgYW5kIGNhdGFseXRpYyB2ZXJzYXRpbGl0eSBvZiBsYXJnZSBuYXR1cmFsIHJpYm96eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JpdC4gUmV2LiBCaW9jaGVtLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yNDktMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnQgc3RhdHVzIG9mIHRoZSBnZW5lIGZvciByaWJvc29tYWwgcHJvdGVpbiBTMTYgaW4gdGhlIGNobG9yb3BsYXN0IGdlbm9tZSBkdXJpbmcgZXZvbHV0aW9uIG9mIHRoZSBnZW51cyBBcmFiaWRvcHNpcyBhbmQgY2xvc2VseSByZWxhdGVkIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIEdlbmV0LiBTeXN0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9MzE5LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5laW91cyBiYXNpYzogYW4gaW50ZWdyYXRlZCBhbmQgZXh0ZW5kYWJsZSBkZXNrdG9wIHNvZnR3YXJlIHBsYXRmb3JtIGZvciB0aGUgb3JnYW5pemF0aW9uIGFuZCBhbmFseXNpcyBvZiBzZXF1ZW5jZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE2NDctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFNTCA0OiBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTg2LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUEU6IGFuYWx5c2VzIG9mIHBoeWxvZ2VuZXRpY3MgYW5kIGV2b2x1dGlvbiBpbiBSIGxhbmd1YWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgbWV0aG9kIGZvciBlc3RpbWF0aW5nIGV2b2x1dGlvbmFyeSByYXRlcyBvZiBiYXNlIHN1YnN0aXR1dGlvbnMgdGhyb3VnaCBjb21wYXJhdGl2ZSBzdHVkaWVzIG9mIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgc3lub255bW91cyBhbmQgbm9uc3lub255bW91cyBzdWJzdGl0dXRpb24gcmF0ZXMgdW5kZXIgcmVhbGlzdGljIGV2b2x1dGlvbmFyeSBtb2RlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTMyLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SRVB1dGVyOiB0aGUgbWFuaWZvbGQgYXBwbGljYXRpb25zIG9mIHJlcGVhdCBhbmFseXNpcyBvbiBhIGdlbm9taWMgc2NhbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9NDYzMy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgY2hsb3JvcGxhc3QgZ2Vub21lIHNlcXVlbmNlIG9mIGdseWNpbmUgbWF4IGFuZCBjb21wYXJhdGl2ZSBhbmFseXNlcyB3aXRoIG90aGVyIGxlZ3VtZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0zMDktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgTWF5ZXIgQy4gICAyMDA3ICwgT2N0b2JlciwgUGhvYm9zOiBIaWdobHkgQWNjdXJhdGUgU2VhcmNoIGZvciBQZXJmZWN0IGFuZCBJbXBlcmZlY3QgVGFuZGVtIFJlcGVhdHMgaW4gQ29tcGxldGUgR2Vub21lcyBieSBDaHJpc3RvcGggTWF5ZXIuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2wuIENsaWZ0b24gTko7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MzY1LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVDogYSBub3ZlbCBtZXRob2QgZm9yIHJhcGlkIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBiYXNlZCBvbiBmYXN0IEZvdXJpZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMwNTktMzA2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uZmlkZW5jZSBsaW1pdHMgb24gcGh5bG9nZW5pZXM6IGFuIGFwcHJvYWNoIHVzaW5nIHRoZSBib290c3RyYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTc4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWpNb2RlbFRlc3Q6IHBoeWxvZ2VuZXRpYyBtb2RlbCBhdmVyYWdpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTY6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdmVyc2lvbiA2LjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGZhbTogdGhlIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDIyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtb2xlY3VsYXIgdGltZWxpbmUgZm9yIHRoZSBvcmlnaW4gb2YgcGhvdG9zeW50aGV0aWMgZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9ODA5LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaHkgY2hsb3JvcGxhc3RzIGFuZCBtaXRvY2hvbmRyaWEgY29udGFpbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wLiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTMxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZXJneSB0cmFuc2R1Y3Rpb24gYW5jaG9ycyBnZW5lcyBpbiBvcmdhbmVsbGVzOiBQcm9ibGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9YW5kIHBhcmFkaWdtcy4gQmlvRXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDI2LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaHkgY2hsb3JvcGxhc3RzIGFuZCBtaXRvY2hvbmRyaWEgcmV0YWluIHRoZWlyIG93biBnZW5vbWVzIGFuZCBnZW5ldGljIHN5c3RlbXM6IGNvbG9jYXRpb24gZm9yIHJlZG94IHJlZ3VsYXRpb24gb2YgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9MTAyMzEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IHRyYW5zY3JpcHRpb25hbCBjb250cm9sIG9mIHRoZSBjaGxvcm9wbGFzdCBnZW5lcyBwc2JBIGFuZCBwc2FBQiBhZGp1c3RzIHBob3Rvc3ludGhlc2lzIHRvIGxpZ2h0IGVuZXJneSBkaXN0cmlidXRpb24gaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JVUJNQiBMaWZlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MjcxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmVseSBjb252ZXJnZW50IGV2b2x1dGlvbiBmb3Igcmlib3NvbWFsIHByb3RlaW4gZ2VuZSBjb250ZW50IGluIHBsYXN0aWQgYW5kIG1pdG9jaG9uZHJpYWwgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTIzMTgtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuY292ZXJpbmcgdGhlIHByb3RlaW4gdHJhbnNsb2NvbiBhdCB0aGUgY2hsb3JvcGxhc3QgaW5uZXIgZW52ZWxvcGUgbWVtYnJhbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzM5OyAgY2l0YXRpb25fcGFnZXM9NTcxLTU3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9eWNmMSwgdGhlIG1vc3QgcHJvbWlzaW5nIHBsYXN0aWQgRE5BIGJhcmNvZGUgb2YgbGFuZCBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz04MzQ4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IHZhcmlhYmxlIGNobG9yb3BsYXN0IG1hcmtlcnMgZm9yIGV2YWx1YXRpbmcgcGxhbnQgcGh5bG9nZW55IGF0IGxvdyB0YXhvbm9taWMgbGV2ZWxzIGFuZCBmb3IgRE5BIGJhcmNvZGluZy4gQS4gTW91c3RhZmEsIChlZC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTM1MDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBTd29mZm9yZCBELkwuICAgMjAwMSAsIFBBVVAqOiBQaHlsb2dlbmV0aWMgQW5hbHlzaXMgVXNpbmcgUGFyc2ltb255IChhbmQgb3RoZXIgbWV0aG9kcykgNC4wLmI1LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIFVTOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIFN5c3RlbWF0aWNzIG9mIFBsYW50cyBJSTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3BhZ2VzPTI5Ny0zNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2lmaWNhdGlvbiBvZiBMdXBpbnVzIChMZWd1bWlub3NhZSkgaW4gdGhlIHdlc3Rlcm4gTmV3IFdvcmxkOiBkZXJpdmVkIGV2b2x1dGlvbiBvZiBwZXJlbm5pYWwgbGlmZSBoaXN0b3J5IGFuZCBjb2xvbml6YXRpb24gb2YgbW9udGFuZSBoYWJpdGF0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NDA4LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaGlnaGx5IHJlYXJyYW5nZWQgY2hsb3JvcGxhc3QgZ2Vub21lIG9mIFRyYWNoZWxpdW0gY2FlcnVsZXVtIChDYW1wYW51bGFjZWFlKTogbXVsdGlwbGUgaW52ZXJzaW9ucywgaW52ZXJ0ZWQgcmVwZWF0IGV4cGFuc2lvbiBhbmQgY29udHJhY3Rpb24sIHRyYW5zcG9zaXRpb24sIGluc2VydGlvbnMvZGVsZXRpb25zLCBhbmQgc2V2ZXJhbCByZXBlYXQgZmFtaWxpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NDE5LTQyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzcGVyc2VkIHJlcGVhdHMgYW5kIHN0cnVjdHVyYWwgcmVvcmdhbml6YXRpb24gaW4gc3ViY2xvdmVyIGNobG9yb3BsYXN0IEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNTUtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJhbW9sZWN1bGFyIHJlY29tYmluYXRpb24gb2YgY2hsb3JvcGxhc3QgZ2Vub21lIG1lZGlhdGVkIGJ5IHNob3J0IGRpcmVjdC1yZXBlYXQgc2VxdWVuY2VzIGluIHdoZWF0IHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTg1NzMtODU3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVuZ3RoIHBvbHltb3JwaGlzbXMgb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBETkEgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MTEzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGxvcm9wbGFzdCBtaWNyb3NhdGVsbGl0ZSBkaXZlcnNpdHkgaW4gUGhhc2VvbHVzIHZ1bGdhcmlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIDEyIGNobG9yb3BsYXN0IG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgaW4gVmlnbmEgdW5ndWljdWxhdGEgKEZhYmFjZWFlKSBhbmQgYW1wbGlmaWNhdGlvbiBpbiBQaGFzZW9sdXMgdnVsZ2FyaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTMwMDA3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5cGVydmFyaWFibGUgbWljcm9zYXRlbGxpdGVzIHByb3ZpZGUgYSBnZW5lcmFsIHNvdXJjZSBvZiBwb2x5bW9ycGhpYyBETkEgbWFya2VycyBmb3IgdGhlIGNobG9yb3BsYXN0IGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTAyMy0xMDI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGxvcm9wbGFzdCBtaWNyb3NhdGVsbGl0ZXM6IG5ldyB0b29scyBmb3Igc3R1ZGllcyBpbiBwbGFudCBlY29sb2d5IGFuZCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBFY29sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTQyLTE0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1vcnBoaWMgY2hsb3JvcGxhc3Qgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBwcmltZXJzIGZvciBzeXN0ZW1hdGljIGFuZCBwb3B1bGF0aW9uIHN0dWRpZXMgaW4gdGhlIGdlbnVzIEhvcmRldW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NTA1LTUxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmFvcmRpbmFyaWx5IHBvbHltb3JwaGljIG1pY3Jvc2F0ZWxsaXRlIEROQSBpbiBiYXJsZXk6IHNwZWNpZXMgZGl2ZXJzaXR5LCBjaHJvbW9zb21hbCBsb2NhdGlvbnMsIGFuZCBwb3B1bGF0aW9uIGR5bmFtaWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz01NDY2LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFuY2UsIHBvbHltb3JwaGlzbSBhbmQgZ2VuZXRpYyBtYXBwaW5nIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbi4gR2VuZXQuIE1HRzsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDE7ICBjaXRhdGlvbl9wYWdlcz0yMjUtMjM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXVrYXJ5b3RpYyB0cmVlIG9mIGxpZmUgZnJvbSBhIGdsb2JhbCBwaHlsb2dlbm9taWMgcGVyc3BlY3RpdmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbGQgU3ByaW5nIEhhcmIuIFBlcnNwZWN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1hMDE2MTQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IGFuZCBtb2xlY3VsYXIgZXZvbHV0aW9uIG9mIHRoZSBncmVlbiBhbGdhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JpdC4gUmV2LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNC8zNDMvMjI4OTI4NDQvZHN4MDA2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgRmFiYWNlYWUgZmFtaWx5IGlzIGNvbnNpZGVyZWQgYXMgYSBtb2RlbCBzeXN0ZW0gZm9yIHVuZGVyc3RhbmRpbmcgY2hsb3JvcGxhc3QgZ2Vub21lIGV2b2x1dGlvbiBkdWUgdG8gdGhlIHByZXNlbmNlIG9mIGV4dGVuc2l2ZSBzdHJ1Y3R1cmFsIHJlYXJyYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzA1NjU3NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTc7nWGmbWj185B6aPFOQhgSqc67Q==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f343%2f3056574" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f343%2f3056574" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=98073" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&amp;Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/4">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106354973">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354973">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354975">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354975">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354983">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354983">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355001">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355001">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355028">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355028">4. Conclusion and further perspectives</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355030">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355030">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355041">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355041">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/4/333/3058514">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/4/359/3077284">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The evolutionary fate of the chloroplast and nuclear <em>rps16</em> genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from <em>Lupinus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Keller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These two authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Keller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Keller J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Keller J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Keller</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. Rousseau-Gueutin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div><div class="aff"><div class="label -label">2</div>IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These two authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+Rousseau-Gueutin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rousseau-Gueutin M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rousseau-Gueutin M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. Rousseau-Gueutin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">G.E. Martin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>CIRAD (Centre de coopération Internationale en Recherche Agronomique pour le Développement), UMR AGAP, F-34398 Montpellier, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=G.E.+Martin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Martin G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Martin G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">G.E. Martin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Morice</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>IGEPP, INRA, Agrocampus Ouest, Université de Rennes 1, BP35327, 35653 Le Rheu Cedex, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Morice">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morice J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morice J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Morice</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Boutte</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Boutte">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boutte J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boutte J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Boutte</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">E. Coissac</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Laboratoire d'Ecologie Alpine, CNRS - Université de Grenoble 1 - Université de Savoie, 38041 Grenoble, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=E.+Coissac">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Coissac E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Coissac E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">E. Coissac</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. Ourari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Département des Sciences Biologiques, Faculté des Sciences de la Nature et de la Vie, Université Abderrahmane Mira, 06000 Bejaia, Algeria</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+Ourari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ourari M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ourari M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. Ourari</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. A&#239;nouche</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+A%c3%afnouche">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=A&#239;nouche M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;A&#239;nouche M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. Aïnouche</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A. Salmon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A.+Salmon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Salmon A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Salmon A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">A. Salmon</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">F. Cabello-Hurtado</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=F.+Cabello-Hurtado">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cabello-Hurtado F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cabello-Hurtado F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">F. Cabello-Hurtado</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A. A&#239;nouche</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l'Univers de Rennes), Université de Rennes 1, 35042 Rennes, France</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx006-cor1" data-open="dsx006-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +33 223236119. Fax. +33 223235047. Email: <a href="mailto:abdelkader.ainouche@univ-rennes1.fr" target="_blank">abdelkader.ainouche@univ-rennes1.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A.+A%c3%afnouche">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=A&#239;nouche A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;A&#239;nouche A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">A. Aïnouche</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx006-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These two authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx006-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 343–358, <a href='https://doi.org/10.1093/dnares/dsx006'>https://doi.org/10.1093/dnares/dsx006</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 October 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">24 January 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 February 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3056574" href="/dnaresearch/article-pdf/24/4/343/22892844/dsx006.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>J. Keller, M. Rousseau-Gueutin, G.E. Martin, J. Morice, J. Boutte, E. Coissac, M. Ourari, M. Aïnouche, A. Salmon, F. Cabello-Hurtado, A. Aïnouche;  The evolutionary fate of the chloroplast and nuclear <em>rps16</em> genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from <em>Lupinus</em>, <em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 343–358, <a href='https://doi.org/10.1093/dnares/dsx006'>https://doi.org/10.1093/dnares/dsx006</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3056574?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20evolutionary%20fate%20of%20the%20chloroplast%20and%20nuclear%20%26lt%3Bem%26gt%3Brps16%26lt%3B%2Fem%26gt%3B%20genes%20as%20revealed%20through%20the%20sequencing%20and%20comparative%20analyses%20of%20four%20novel%20legume%20chloroplast%20genomes%20from%20%26lt%3Bem%26gt%3BLupinus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-02-27&amp;volumeNum=24&amp;issueNum=4&amp;author=Keller%2C%20J.%3B%20Rousseau-Gueutin%2C%20M.&amp;startPage=343&amp;endPage=358&amp;contentId=10.1093%2Fdnares%2Fdsx006&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f343%2f3056574" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106354973 id="106354973" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The Fabaceae family is considered as a model system for understanding chloroplast genome evolution due to the presence of extensive structural rearrangements, gene losses and localized hypermutable regions. Here, we provide sequences of four chloroplast genomes from the <em>Lupinus</em> genus, belonging to the underinvestigated Genistoid clade. Notably, we found in <em>Lupinus</em> species the functional loss of the essential <em>rps16</em> gene, which was most likely replaced by the nuclear <em>rps16</em> gene that encodes chloroplast and mitochondrion targeted RPS16 proteins. To study the evolutionary fate of the <em>rps16</em> gene, we explored all available plant chloroplast, mitochondrial and nuclear genomes. Whereas no plant mitochondrial genomes carry an <em>rps16</em> gene, many plants still have a functional nuclear and chloroplast <em>rps16</em> gene. Ka/Ks ratios revealed that both chloroplast and nuclear <em>rps16</em> copies were under purifying selection. However, due to the dual targeting of the nuclear <em>rps16</em> gene product and the absence of a mitochondrial copy, the chloroplast gene may be lost. We also performed comparative analyses of lupine plastomes (SNPs, indels and repeat elements), identified the most variable regions and examined their phylogenetic utility. The markers identified here will help to reveal the evolutionary history of lupines, Genistoids and closely related clades.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chloroplast genomes&quot;">chloroplast genomes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Lupinus"><em>Lupinus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional gene relocation&quot;">functional gene relocation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;repeated sequences&quot;">repeated sequences</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogeny">phylogeny</a></div>                            <h2 scrollto-destination=106354975 id="106354975" class="section-title" >1. Introduction</h2>
<p>The Fabaceae (or Leguminosae) is one of the largest flowering plant families, with ca. 19,500 herbaceous to tree species (ca. 751 genera) distributed in very diverse ecogeographical areas around the World.<a reveal-id="dsx006-B1" data-open="dsx006-B1" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> Because of their ability to establish specific associations with nitrogen-fixing rhizobial bacteria,<a reveal-id="dsx006-B4" data-open="dsx006-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx006-B5" data-open="dsx006-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> many legume species are of great ecological and economic interest. They provide valuable biological nitrogen for better productivity and ecosystem functioning, and supply significant sources of protein for human and animal nutrition and health.<a reveal-id="dsx006-B6" data-open="dsx006-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Within Fabaceae, the Papillionoideae clade includes several major crops for human and animal consumption, such as soybean (<em>Glycine</em>), barrel medic (<em>Medicago</em>), bean (<em>Phaseolus</em>), cowpea (<em>Vigna</em>), chickpea (<em>Cicer</em>), pea (<em>Pisum</em>), peanut (<em>Arachis</em>), pigeon pea (<em>Cajanus</em>) and lupine (<em>Lupinus</em>). Increasing our knowledge of the evolutionary history of this family, as well as of the mechanisms involved in its physiological and ecological properties will improve management of natural and agricultural ecosystems and guide plant breeding programs.</p><p>During the last decade, our understanding of the structural and functional evolutionary dynamics of legume genomes increased significantly due to progress in Next Generation Sequencing (NGS) technologies. This recent sequencing of many plant plastomes revealed the unusual evolution of the Fabaceae, Geraniaceae and Campanulaceae plastomes.<a reveal-id="dsx006-B7" data-open="dsx006-B7" class="link link-ref link-reveal xref-bibr"><sup>7–13</sup></a> To date, 34 complete Fabaceae plastomes have been sequenced (including 17 in the last three years), mainly from Papilionoid lineages (25), and a few from the Cesalpinioid (5) and Mimosoid (4) lineages.<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14–18</sup></a> Comparative analyses of Fabaceae plastomes showed that they have undergone major structural evolution compared with other plant families, including the lack of one inverted repeat (IR),<a reveal-id="dsx006-B19" data-open="dsx006-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> a 51-kb inversion shared by most Papilionoid clades (including species from the IR lacking clade also called IRLC),<a reveal-id="dsx006-B20" data-open="dsx006-B20" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> a 78-kb inversion in Phaseolae,<a reveal-id="dsx006-B23" data-open="dsx006-B23" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> a 5.6-kb inversion in <em>Milletia</em>,<a reveal-id="dsx006-B26" data-open="dsx006-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> a 36-kb inversion in the Genistoid clade<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and a 39-kb inversion in <em>Robinia</em>.<a reveal-id="dsx006-B16" data-open="dsx006-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p>Although gene content is relatively well conserved in angiosperm plastomes,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx006-B27" data-open="dsx006-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> it has been shown that several genes, such as <em>accD</em>,<a reveal-id="dsx006-B7" data-open="dsx006-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><em>psaI</em>,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>ycf4</em>, <em>rpl33</em>,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>clpP</em><a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx006-B18" data-open="dsx006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> or <em>rps16</em><a reveal-id="dsx006-B28" data-open="dsx006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a>, have been functionally lost in various legume lineages. Some of these chloroplast genes (<em>accD</em> and <em>rps16</em>), which are essential for plant survival,<a reveal-id="dsx006-B29" data-open="dsx006-B29" class="link link-ref link-reveal xref-bibr"><sup>29–31</sup></a> were shown to be functionally replaced by a nuclear gene.<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx006-B28" data-open="dsx006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p><p>In contrast to the plastomes of most angiosperm families, Fabaceae plastomes have regions with accelerated mutation rates, including genic regions such as the <em>rps16</em>-<em>ycf4</em> region in the IRLC clade<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> or the <em>clpP</em> gene in Mimosoids.<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx006-B18" data-open="dsx006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> It has been suggested that this remarkable pattern of variation most likely results from the functional alteration of genes involved in DNA replication, repair and recombination,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx006-B32" data-open="dsx006-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> which may also facilitate the expansion of repeat sequences and the formation of structural rearrangements. For instance, the extensive reorganization of the plastid genome in <em>Trifolium</em> was correlated with an increase in repeat number,<a reveal-id="dsx006-B7" data-open="dsx006-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and the increase in size of the plastid genome in Mimosoids was correlated with tandem repeat expansions.<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>Until recently, most of the knowledge of legume plastome evolutionary dynamics derived from model and crop plants in the Papilionoid lineage, and specifically the non-protein amino acid-accumulating (NPAA) clade (including Millettioids, Robinioids and IRLC).<a reveal-id="dsx006-B33" data-open="dsx006-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> In the last three years, plastomes of other Papilionoids, Mimosoids and Cesalpinioids lineages have been sequenced and have provided additional insights into the unusual plastome evolution of the Fabaceae.<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14–18</sup></a> In a few genera (<em>Glycine</em>, <em>Lathyrus</em>, <em>Trifolium</em>), the plastomes of several species were sequenced,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx006-B17" data-open="dsx006-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx006-B34" data-open="dsx006-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> contributing to a better understanding of the origin of specific structural variations. Localized hypermutations, gene losses and plastome size variations were identified as well as useful sequence resources were found for phylogenetic inference. However, additional sequencing efforts in key genera of the highly diverse legume family is essential for understanding key features of plastome evolution, and to resolve phylogenetic relationships at these taxonomic levels.</p><p>The Genistoid clade contains ∼18% of the 13,800 Papilionoid taxa.<a reveal-id="dsx006-B35" data-open="dsx006-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Within this poorly-studied lineage, the diverse <em>Lupinus</em> genus is considered as a good model system. <em>Lupinus</em> is composed of hundreds of annual and perennial herbaceous species and a few soft-woody shrubs and trees, which occur in a wide range of ecogeographical conditions. Lupines are mainly distributed in the New World (NW) (from Alaska to southern Chile and Argentina), whereas ∼20 species and subspecies are native to the Old World (OW) where two groups are distinguished: the smooth-seeded (circum-Mediterranean) and the rough-seeded lupines (scattered in North-equatorial Africa).<a reveal-id="dsx006-B36" data-open="dsx006-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In addition, this genus includes some crop species<a reveal-id="dsx006-B37" data-open="dsx006-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (<em>Lupinus albus</em>, <em>Lupinus luteus</em>, <em>Lupinus angustifolius</em>, <em>Lupinus mutabilis</em>), which are of growing interest due to their high seed protein content, their potential as nitrogen producers and for their health benefits.<a reveal-id="dsx006-B37" data-open="dsx006-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsx006-B38" data-open="dsx006-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Molecular phylogenetic investigations using nuclear (ITS, nrDNA, <em>LEGCYC1A</em>, <em>SymRK</em>) and chloroplastic (<em>rbcL</em>, <em>matK</em>, <em>trnL</em>-<em>trnF</em>, <em>trnL</em>-<em>trnT</em>, <em>trnL</em>-intron, <em>trnS</em>-<em>trnG</em>) regions drastically improved knowledge of the evolutionary history of this complex genus.<a reveal-id="dsx006-B39" data-open="dsx006-B39" class="link link-ref link-reveal xref-bibr"><sup>39–44</sup></a> Many clades have been well circumscribed, and patterns of diversification were identified in both the NW and the OW. In spite of these significant advances, there are still uncertainties and unresolved relationships to be elucidated, such as for instance: (i) basal relationships between the NW and OW lineages in the <em>Lupinus</em> phylogeny; (ii) relationships amongst the OW lineages and within the African clade; and (iii) the enigmatic position of some taxa (e.g. Floridian lupines).<a reveal-id="dsx006-B44" data-open="dsx006-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx006-B45" data-open="dsx006-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a></p><p>Recently, the plastome of <em>L. luteus</em> was published, representing the first chloroplast genome sequenced in <em>Lupinus</em> and in the Genistoid lineage.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Comparison with other legume plastomes allowed the discovery that the Genistoids share a 36-kb inversion, and the identification of mutational hotspots representing potentially informative regions for evolutionary studies. However, these identified regions, such as the <em>ycf4</em> gene in the NPAA clade<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> or the <em>clpP</em> in the IRLC and Mimosoids,<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> may be of interest only in particular clades, due to a specific accelerated evolutionary rate in these lineages. Thus, additional plastomes are needed to specifically understand the plastome evolution of the lupine/Genistoid lineage and to accurately identify their most variable regions that are of phylogenetic significance. In this context, we sequenced four novel lupine plastomes: two Mediterranean smooth-seeded species, <em>L. albus</em> and <em>Lupinus micranthus</em> Guss. and two rough-seeded species, <em>Lupinus atlanticus</em> Glads. and <em>Lupinus princei</em> Harms. Comparative analyses were performed among the five OW lupine plastomes (including the previously published <em>L. luteus</em><a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>) at the structural (inversions, indels, repeat numbers and distribution), gene and sequence levels, in order to better understand their evolutionary dynamics and to identify novel phylogenetically informative regions. More specifically, sequencing of these four additional plastomes revealed the pseudogenization of the chloroplast <em>rps16</em> gene in <em>Lupinus</em> species. Analyses of the Ka/Ks ratios of the functional chloroplast and the nuclear <em>rps16</em> genes (both encoding the same chloroplast RPS16 protein) from some representative Angiosperm species revealed that both copies were under purifying selection. However, since the nuclear <em>rps16</em> gene also encodes the mitochondrial RPS16 protein and that this gene is lost in the mitochondrial genomes of all plants sequenced to date, the loss of the nuclear <em>rps16</em> gene would be detrimental for plant survival. This could explain why only the chloroplast <em>rps16</em> gene has been functionally lost many times during plant evolution, despite being under purifying selection. In addition, investigations on the evolutionary dynamic of the lupine plastomes (mutations, indels and repeated elements) allowed identification of variable characters and regions. The phylogenetic interest of these regions in the genus <em>Lupinus</em> was tested using representative species of the main lupine clades.</p>                            <h2 scrollto-destination=106354983 id="106354983" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=106354984 id="106354984" class="section-title" >2.1. Plant material and DNA isolation</h3>
<p>Genomic DNA of 30 lupine species was extracted from fresh leaves using the NucleoSpin<sup>®</sup> Plant II kit (Macherey-Nagel), following the manufacturer’s instructions. The genomic DNA extracts of four <em>Lupinus</em> species (<em>L. albus</em>, <em>L. micranthus</em>, <em>L. atlanticus</em> and <em>L. princei</em>) were subjected to NGS for plastome reconstruction. DNA extracts from the other 26 lupine species were used in different evolutionary tests on genes and regions of interest; including four OW rough-seeded species (<em>L. digitatus</em>, <em>L. cosentinii</em>, <em>L. anatolicus</em> and <em>L. pilosus</em>); three OW smooth-seeded species (<em>L. hispanicus</em> subsp. <em>bicolor</em>, <em>L. angustifolius</em> subsp. a<em>ngustifolius</em> and <em>L. angustifolius</em> subsp. <em>reticulatus</em>); and fourteen species representing the main known groups in the NW lupines. Among these are (i) five members of the North and South East American clade (<em>L. texensis</em>, <em>L. paraguariensis</em>, <em>L. gibertianus</em> and <em>L. sellowianus</em>); (ii) nine members from various groups mainly occurring in western regions of North, Central and South America (<em>L. affinis</em>, <em>L. hirsitussimus</em>, <em>L. luteolus</em>, <em>L. nanus</em>, <em>L. polyphyllus</em>, <em>L. mutabilis</em>, <em>L. mexicanus</em>, <em>L. elegans</em>, two unidentified samples <em>L. sp.</em> from Equator); (iii) and two Florida endemic species (<em>L. diffusus</em> and <em>L. villosus</em>). Moreover, the DNA of three representatives from the <em>Genista-Cytisus</em> complex, sister group to <em>Lupinus</em> in the Genisteae<a reveal-id="dsx006-B46" data-open="dsx006-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> tribe were obtained: <em>Retama sphaerocarpa</em>, <em>Cytisus battandieri</em> and <em>Genista erioclada</em>. More details on geographic origins and reference numbers of these plant materials are presented in <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs1_dsx006.xls?Expires=2147483647&Signature=sCMnLQBkGQ7LtF-novZzfiqUEbb-ONEANNCXRG9L9CM3lFkU7RJ9oP0NrXR5wlNtylK0zdoa5Aj5gHq4Nte53Rt7W~4Y9XoHf9sqig3tcmIdoFw4uGguD7nb4ZW16pd4oRlBM35Cb3Eb6OJdgPJD1LzkqPpduPNnd6IuHSk9DxxjNF3QMwuoZnXIHo5p8RjyIQ9t33Kv0UaNie0FBkF1M6nSiwriU17kv63CUBuIdlCsYAko2ZXuUvg1jqIqdD0wtMjXa6qcte8KsYStysMVWRNV7LmRW7wUtwmteFaSl4bXchHjmRF9QZZ~ya5YfnN-BmAxCE89cQSSR2ZQuD7Ebg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=106354986 id="106354986" class="section-title" >2.2. High throughput sequencing, plastome assembly and annotation</h3>
<p>The genomic DNA of <em>L. albus</em>, <em>L. micranthus</em>, <em>L. atlanticus</em> and <em>L. princei</em> were subjected to high-throughput sequencing using an Illumina HiSeq 2000 platform (BGI, Hong-Kong). One flow cell containing a library of each species was used, yielding ∼11 millions of 100 bp paired-end (PE) reads (insert size = 500 bp) for each library, except <em>L. micranthus</em>, for which ∼5.5 millions of PE reads were obtained.</p><p><em>De novo</em> chloroplast genome (plastomes) assemblies were performed using Paired End Illumina reads and ‘The organelle assembler’ software (<a class="link link-uri" href="http://metabarcoding.org/asm" target="_blank">http://metabarcoding.org/asm</a> (January 2015, date last accessed))<a reveal-id="dsx006-B47" data-open="dsx006-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>: its aim is to assemble over represented sequences such as organelle genomes (chloroplast or mitochondrion), or the rDNA cistron. Each draft plastome sequence was then verified and corrected by mapping the Illumina reads against each genome using Bowtie 2 v2.0.<a reveal-id="dsx006-B48" data-open="dsx006-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> A few uncertain nucleotides were verified by Sanger sequencing. Plastome annotation was performed using DOGMA (Dual Organellar GenoMe Annotator, <a class="link link-uri" href="http://dogma.ccbb.utexas.edu" target="_blank">http://dogma.ccbb.utexas.edu</a> (January 2015, date last accessed))<a reveal-id="dsx006-B49" data-open="dsx006-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> and by aligning each of the four newly constructed plastomes with the published <em>L. luteus</em> plastome (KC695666<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>). A graphical representation of each plastome was drawn using Circos<a reveal-id="dsx006-B50" data-open="dsx006-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs1_dsx006.ppt?Expires=2147483647&amp;Signature=NUbYoeFLPQhq3ZfnoTeIZrc3s0teLmtrnCzvV6bqmW1q2PWoidgMR2Ayok94SKOfmo8ui8CJ2ot91PwlnKZlngB5y13bOK7HX5nT7GB0wmPXj6IWCr7d8z6SL7itC-glhwtgyw3t4kgDStVTmAov6WtRvykmKe6JGQXwOj4yQy8k-rnOKdgm1HPh~UcXu6LhqvDAw0ESHXoCQLE1lqaOO05J0RLHuYBVRIPAt0K12yg0HlY2t5mXzPH~5xMxAGkrcuvInrpkuWkdnlBB2q-CfpZSyG7y-I2Otb~OeVfIBIsRe7vz6b1rf7mVDjlRnBgKeau9zP76pG6oETOStt9b0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs4_dsx006.ppt?Expires=2147483647&amp;Signature=0vT1yYSotVvpRxoM5WLzUw8XZc6QYkVc12-u~wGw2vzwrokRjU~JxVw7ZKDd43KlsbFdBLJV4AW-N1NUH2tLeruzP9~IlVS~smZat5YIc~lK8o5ueQOB90uSx72F~KjnfQFY~vWmKX1lpcYT522Pko5YgZ4vWJozN6x8-vTYGXBuxqHl8QEJJi6LuJtU-~PMNh9JW-WKiOzeF15AQiIwPjxXnDQuquuLV4U1dSYv4lcByw8c3o8TP0TWi6NAbYFWKyBNa5qdCULIY-uZAvpWJIpLZ3kKUNumqTooChQRpcy~1ZAyFgoH~ZP-S5qDAaOkmD4IsvhRWyUMoVDMRvZYEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S4</a></span>).</p>                            <h3 scrollto-destination=106354989 id="106354989" class="section-title" >2.3. Identification of <em>rps16</em> gene sequences in plant mitochondrion, chloroplast and nuclear genomes</h3>
<p>Sequences of the <em>rps16</em> gene were searched for all non-parasitic plant mitochondrion, chloroplast and nuclear genomes available to date. Organelle and nuclear genomes were downloaded from GenBank (<a class="link link-uri" href="https://ncbi.nlm.nih.gov" target="_blank">https://ncbi.nlm.nih.gov</a> (November 2016, date last accessed)) and Phytozome v11 (<a class="link link-uri" href="https://phytozome.jgi.doe.gov/pz/portal.html" target="_blank">https://phytozome.jgi.doe.gov/pz/portal.html</a> (November 2016, date last accessed)), respectively. For the nuclear <em>rps16</em> genes, presence in mature proteins of a signal peptide targeting the proteins to the organelles was tested using BaCelLo,<a reveal-id="dsx006-B51" data-open="dsx006-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> ProteinProwler,<a reveal-id="dsx006-B52" data-open="dsx006-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> TargetP,<a reveal-id="dsx006-B53" data-open="dsx006-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> MultiLoc2<a reveal-id="dsx006-B54" data-open="dsx006-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and Predotar.<a reveal-id="dsx006-B55" data-open="dsx006-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> In addition, as the chloroplast <em>rps16</em> gene has a subgroup IIB intron, we looked for the presence of the correct splicing of this intron by verifying the presence of the strictly conserved splicing sites (GTGYG and AY at the 5ʹ and 3ʹ splice sites of the intron, respectively)<a reveal-id="dsx006-B56" data-open="dsx006-B56" class="link link-ref link-reveal xref-bibr"><sup>56–59</sup></a> in all chloroplast <em>rps16</em> genes with a complete coding sequence (742 species).</p>                            <h3 scrollto-destination=106354991 id="106354991" class="section-title" >2.4. Selective pressure acting on the nuclear and chloroplast <em>rps16 genes</em></h3>
<p>Within a subset of plants representing the main clades of Angiosperms (<em>Arabidopsis lyrata</em>: Brassicales; <em>Citrus sinensis</em>: Sapindales; <em>Cucumis sativus</em>: Cucurbitales; <em>Glycine max</em>: Fabales; <em>Manihot esculentum</em>: Malpighiales; <em>Musa acuminata</em>: Zingiberales; <em>Oryza sativa</em>: Poales; <em>Panicum virgatum</em>: Poales; <em>Prunus persica</em>: Rosales; <em>Solanum lycopersicum</em>: Solanales; <em>Theobroma cacao</em>: Malvales; <em>Vitis vinifera</em>: Vitales), we retrieved the functional nuclear and chloroplast <em>rps16</em> gene sequences, which both encode the chloroplast RPS16 proteins. The different nuclear or chloroplast copies were aligned using Geneious v6.1.8<a reveal-id="dsx006-B60" data-open="dsx006-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and the alignments were adjusted manually. Non-synonymous and synonymous nucleotide substitution rates were evaluated using the yn00 method implemented in PAML<a reveal-id="dsx006-B61" data-open="dsx006-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> for the nuclear and chloroplast <em>rps16</em> sequences. A list of species considered and the accession numbers of nuclear and chloroplast <em>rps16</em> sequences used are presented in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs2_dsx006.xls?Expires=2147483647&Signature=sVlH6PVnYVU-tIBLs-yiAbUlcEMODKALjSE1QL6Vip8PeGAebwjUBvSMAnmwyDkBS6EdCLwEBYcc09YxAKf~A-iLilEaJ~h8ugD93D7QOTALAf4cuTe3ihqcA~uXCKwfdvuuJD-EmL5DKSLWhdzze-tINQkopMkXzoOuZFUG1XHgG8NNm-zOzLlsql2UrLqdiIb4u0RMTrdT4nOHAaEpeWgzQ9oAMnTHoo5N4N5lfC9VIpDhsqRW7AyfVWVzKGnaLnaXrwkcRhbRE8Ga7peWZw67xU2JA4hQod~UC1G1TWoS8c66PcuGRkj~kwjhY5RFZtwcVEPZh6fu43yTvWreaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. Ka/Ks analyses of chloroplast <em>rps16</em> gene were also performed using only the representatives of the following Angiosperm families: Asteraceae, Brassicaceae, Fabaceae, Poaceae and Solanaceae.</p>                            <h3 scrollto-destination=106354993 id="106354993" class="section-title" >2.5. Sequence divergence among lupine plastomes</h3>
<p>To identify the most variable regions among lupine chloroplast genomes, the five plastomes were aligned using Geneious v6.1.8<a reveal-id="dsx006-B60" data-open="dsx006-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and pairwise comparisons between each of the five plastomes were performed to evaluate the percentage of identity in sliding window frames of 1 kb with a Python custom script. Using this script, insertion-deletions (indels) with a minimum size of 20 bp were identified. These large indels as well as pairwise comparisons results were represented graphically using Circos.<a reveal-id="dsx006-B50" data-open="dsx006-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Additionally, the five aligned plastomes were screened to identify autapomorphous (single) and shared indels of at least 2 bp, and the excluding regions with homopolymers or with ambiguous overlapping indels. Sequence divergence among the five lupine plastomes (including <em>L. luteus</em>) was also evaluated independently for intergenic spacers, introns, exons, rRNAs and tRNAs by calculating pairwise distances between homologous regions. Pairwise distances were determined with the <strong><em>ape</em></strong> R-cran Package<a reveal-id="dsx006-B62" data-open="dsx006-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> (available at: <a class="link link-uri" href="http://cran.r-project.org/web/packages/ape/ape.pdf" target="_blank">http://cran.r-project.org/web/packages/ape/ape.pdf</a>) using the Kimura 2-parameters (K2p) evolution model for introns and intergenic spacers.<a reveal-id="dsx006-B63" data-open="dsx006-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Additionally, sequence divergence of protein encoding sequences was estimated using the synonymous (Ks) and non-synonymous (Ka) nucleotide substitution rates with the yn00 method<a reveal-id="dsx006-B64" data-open="dsx006-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> from the PAML package.<a reveal-id="dsx006-B61" data-open="dsx006-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a></p><p>Repeat sequences in each lupine plastome were identified using REPuter<a reveal-id="dsx006-B65" data-open="dsx006-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> with similar parameters as previously described for the analysis of Fabaceae plastomes<a reveal-id="dsx006-B7" data-open="dsx006-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx006-B25" data-open="dsx006-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsx006-B66" data-open="dsx006-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> and excluding one copy of the IR. Palindrome sequences as well as dispersed direct and IRs of a minimum length of 30 bp and presenting at least 90% sequence identity were identified (Hamming distance of three). Additionally, mono-, di-, tri-, tetra- and penta-nucleotides Short Sequence Repeats (SSRs) with a mininimal size of 12 bp and a minimal repeat number of five were detected using the Phobos software<a reveal-id="dsx006-B67" data-open="dsx006-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> implemented in Geneious v6.1.8.<a reveal-id="dsx006-B60" data-open="dsx006-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a></p>                            <h3 scrollto-destination=106354996 id="106354996" class="section-title" >2.6. PCR amplification of the <em>rps</em>16 gene and the most variable regions in lupines</h3>
<p>Several primer pairs were designed using Primer 3.0<a reveal-id="dsx006-B68" data-open="dsx006-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> to examine the variation of the <em>rps16</em> gene and two fast evolving regions (<em>psaA</em>-<em>ycf4</em> and <em>ycf1</em>-<em>rps15</em>) in <em>Lupinus</em>. For <em>rps16</em>, the following primer pairs were used: F-CCGTCCCAGAGCATATTCAG, R-GCAACGATTCGATAAATGGC and F-CCCATTCATATCGAAGGAAAACT, R-CCATCATGTACTATTTACATCATCAATC and R-CTATATACAAGTCATCCACACCCTC. Within the fast evolving regions, primer pairs were designed to amplify four sub-regions (<em>accD</em> and <em>ycf1</em> genes, <em>ycf1</em>-<em>rps15</em> and <em>trnF<sup>GAA</sup>-trnL<sup>UAA</sup></em> 5ʹ-3ʹ intergenic spacers): <em>accD</em> with F-GTCTATAAATACATTACCCCCG, R-TGTCTTCATCCATAGGATTCC; <em>ycf1-rps15</em> with F-GATTTATGTTGCACAAACCG, R-CATTGATGGGTGGTGAGG; <em>trnF<sup>GAA</sup></em>-<em>trnL<sup>UAA</sup></em> with F-TTGAACTGGTGACACGAGG, R-TGGCGAAATTGGTAGACG. Because of the large size of the <em>ycf1</em>, two primer pairs were designed: <em>ycf1</em> part1 with F-AATCAAGCAGAAAGTTATGGG, R-CTTACATCTTTTGAGCTTTCACTC; <em>ycf1</em> part2 with F-GGAATGGAAGTAGAATTGCC, R-TTTTGTTTACGCGTCTTGT.</p><p>PCR amplifications of these regions were carried out for 32 taxa (including three Genisteae outgroups, <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs1_dsx006.xls?Expires=2147483647&Signature=sCMnLQBkGQ7LtF-novZzfiqUEbb-ONEANNCXRG9L9CM3lFkU7RJ9oP0NrXR5wlNtylK0zdoa5Aj5gHq4Nte53Rt7W~4Y9XoHf9sqig3tcmIdoFw4uGguD7nb4ZW16pd4oRlBM35Cb3Eb6OJdgPJD1LzkqPpduPNnd6IuHSk9DxxjNF3QMwuoZnXIHo5p8RjyIQ9t33Kv0UaNie0FBkF1M6nSiwriU17kv63CUBuIdlCsYAko2ZXuUvg1jqIqdD0wtMjXa6qcte8KsYStysMVWRNV7LmRW7wUtwmteFaSl4bXchHjmRF9QZZ~ya5YfnN-BmAxCE89cQSSR2ZQuD7Ebg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) in a total volume of 50 μl, containing 5× Green GoTaq flexi Reaction Buffer (Promega), 0.2 mM of dNTP, 0.2 μM of each primer, 1.25 Unit of G2 flexi DNA polymerase (Promega), mqH<sub>2</sub>O and 20 ng of template DNA. Cycling conditions were 94 °C for 2 min followed by 35 cycles at 94 °C for 30 s, 48–52 °C (adapted according to the primer pairs used) for 30 s, 72 °C for 90 s and a final extension at 72 °C for 7 min. PCR products were purified using the NucleoSpin gel and PCR clean up kit (Macherey Nagel), following the manufacturer’s instructions. Purified PCR products were sequenced directly (in both directions) by Sanger at Macrogen Europe (Amsterdam, The Netherlands). All sequences were deposited in Genbank under the accession numbers, KX147685 to KX147753 and KX787895 to KX787910.</p>                            <h3 scrollto-destination=106354999 id="106354999" class="section-title" >2.7. Phylogenetic analyses</h3>
<p>For each of the four chloroplast regions investigated, all lupine sequences were aligned using MAFFT implemented in Geneious v6.1.8.<a reveal-id="dsx006-B60" data-open="dsx006-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx006-B69" data-open="dsx006-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> The resulting alignments were adjusted manually. In addition, a concatenated data matrix was constructed using the sequences obtained from the four regions (<em>accD</em> and <em>ycf1</em> genes, <em>ycf1</em>-<em>rps15</em> and <em>trnF<sup>GAA</sup>-trnL<sup>UAA</sup></em> 5ʹ-3ʹ intergenic spacers). These matrices were first subjected to phylogenetic analyses using Maximum Parsimony (MP). Bootstrap analyses were performed with 1,000 replicates.<a reveal-id="dsx006-B70" data-open="dsx006-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> These data matrices were also subjected to Maximum Likelihood (ML) phylogenetic analyses. The best-fitted model of sequence evolution for each region (individual or concatenated) was determined using JModeltest<a reveal-id="dsx006-B71" data-open="dsx006-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> and ML analyses were then performed for each matrix with 1,000 bootstrap replicates using MEGA 6.0.<a reveal-id="dsx006-B72" data-open="dsx006-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a></p>                            <h2 scrollto-destination=106355001 id="106355001" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=106355002 id="106355002" class="section-title" >3.1. Structure, organization and gene content of lupine plastomes</h3>
<p>The Illumina PE reads obtained for <em>L. albus</em>, <em>L. atlanticus</em>, <em>L. micranthus</em> and <em>L. princei</em>, were used to assemble their chloroplast genome sequences (deposited in GenBank under accession numbers KU726826; KU726827; KU726828; KU726829, respectively). The four plastomes harbor a quadripartite structure (a Large Single Copy and a Small Single Copy separated by two IRs) with a total length ranging from 151,808 bp to 152,272 bp. As expected from previous PCR-based evidence, they all have the 36-kb inversion that occurred at the base of the Genistoid emergence or soon after.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The different <em>Lupinus</em> plastomes have similar gene, intron and GC content (<a reveal-id="dsx006-T1" data-open="dsx006-T1" class="link link-reveal link-table xref-fig">Table 1</a>) as do most photosynthetic and non-parasitic angiosperm plastomes.<a reveal-id="dsx006-B11" data-open="dsx006-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The genes are distributed into three main categories: self-replication (58 genes), photosynthesis (47 genes) and other functions (six genes) (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs3_dsx006.xls?Expires=2147483647&amp;Signature=jl0VKE7lzqsYW7~RPgm~9pdfFWZlJk1NIGLY7VO8LpoaGeF9WG7z-Bja19CNWuRtIxMbBG6igFx3bzptnPP8XkOz39HcrL9hun~UXAD63RSErqMgdB8RZBBBMy-0Ve3P2iFOCe6ZAz3JxF5gN9cVqJUfThmbDUxFL4YcmECR03EHXaPe44xoUVF3gPTpjtuV-8H3Qf2vCCkzd2T2jNC2rtFCRx1DPe2pCe6c3q9Cm8-ivG5JAbK76VGRl3CXJ0PDKHdPBD14HT3s0IK8kM-wJkgY9qZJoYuvOWRGG0LArO6C59Z8~upbg9Leh84O6f~zvPfnPVbWMNPpL1v1oFWioQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Among these genes, 76 are protein-encoding genes, 30 encode tRNAs and four encode rRNAs. None of the genes known to be lost or pseudogenized in other legume lineages, such as <em>accD</em>,<a reveal-id="dsx006-B7" data-open="dsx006-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><em>psaI</em>, <em>ycf4</em>, <em>rpl23</em> or <em>rpl33</em><a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> are missing in the lupine plastomes. Interestingly, comparative analyses of the lupine plastomes (including <em>L. luteus</em>) revealed a likely loss of functionality of the <em>rps16</em> gene in <em>L. albus</em> and <em>L. micranthus</em> but not in the other species. Both pseudogenes showed a deletion (verified by Sanger sequencing), which lead to a pre-mature stop codon (19 and 20 amino acids earlier in <em>L. albus</em> and <em>L. micranthus</em>, respectively) within the functional domain of the RPS16 protein (<a reveal-id="dsx006-F1" data-open="dsx006-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). To determine if these truncated <em>rps16</em> genes in <em>L. albus</em> and <em>L. micranthus</em> are still functional, we used pfam (pfam-A, default parameters)<a reveal-id="dsx006-B73" data-open="dsx006-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> to search for the presence of a functional domain in the five lupine species investigated in this study. No RPS16 functional domain could be identified in <em>L. albus</em> and <em>L. micranthus</em> only, clearly suggesting that <em>rps16</em> is a pseudogene in these two lupine species. Recently, an additional way at the origin of the loss of functionality of the chloroplast <em>rps16</em> gene was identified and corresponds to the loss of its splicing capacity.<a reveal-id="dsx006-B59" data-open="dsx006-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> In lupines, we observed that the <em>rps16</em> intron is not correctly spliced. This suggests that the <em>rps16</em> is not functional in the five chloroplast genomes (all five lupine plastomes must therefore have 76 functional protein-coding genes), and that the loss of functionality most likely occurred first via the loss of the ability to splice the intron. Thereafter, additional mutations in <em>L. albus</em> and <em>L. micranthus</em> led to pre-mature stop codons. Whether this shared pre-mature stop codon results from a common ancestor or from independent mutational events needs to more accurately resolve phylogenetic relationships of these two species among the OW lupines (see later in the phylogenetic section). Sequencing of the <em>rps16</em> gene in other lupines and closely related species revealed that another population of <em>L. micranthus</em> has a pseudo <em>rps16</em>, and that it is also defunct in <em>L. angustifolius</em>, <em>Lupinus mariae-josephae</em>, <em>L. villosus</em> and in a member of the <em>Lupinus</em> sister group, <em>G. erioclada</em> (data not shown). <div content-id="dsx006-T1" class="table-modal"><div class="table-wrap"><div id="dsx006-T1" data-id="dsx006-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Characteristics of <em>Lupinus</em> plastomes</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Plastome characteristics </th><th><em>L. luteus</em> </th><th><em>L. albus</em> </th><th><em>L. atlanticus</em> </th><th><em>L. princei</em> </th><th><em>L. micranthus</em> </th></tr></thead><tbody><tr><td>Overall size in bp </td><td>151,894 </td><td>151,921 </td><td>152,272 </td><td>152,243 </td><td>151,808 </td></tr><tr><td>LSC size in bp (%) </td><td>82,327 (54.2) </td><td>82,280 (54.2) </td><td>82,674 (54.3) </td><td>82,663 (54.3) </td><td>82,145 (54.1) </td></tr><tr><td>SSC size in bp (%) </td><td>17,847 (11.7) </td><td>17,841 (11.7) </td><td>17,894 (11.8) </td><td>17,876 (11.7) </td><td>17,857 (11.8) </td></tr><tr><td>IR size in bp (%) </td><td>25,860 (34.1) </td><td>25,900 (34.1) </td><td>25,852 (34) </td><td>25,852 (34) </td><td>25,903 (34.1) </td></tr><tr><td>Coding regions size in bp (%) </td><td>90,217 (59.4) </td><td>90,002 (59.2) </td><td>90,125 (59.2) </td><td>90,104 (59.2) </td><td>90,083 (59.3) </td></tr><tr><td>Protein-coding region in bp (%) </td><td>78,363 (51.6) </td><td>78,148 (51.4) </td><td>78,271 (51.4) </td><td>78,250 (51.4) </td><td>78,229 (51.5) </td></tr><tr><td>Introns size in bp (%) </td><td>19,136 (12.6) </td><td>19,115 (12.6) </td><td>19,111 (12.6) </td><td>19,121 (12.6) </td><td>18,754 (12.4) </td></tr><tr><td>rRNA size in bp (%) </td><td>9,056 (6) </td><td>9,056 (6) </td><td>9,056 (5.9) </td><td>9,056 (5.9) </td><td>9,056 (6) </td></tr><tr><td>tRNA size in bp (%) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td></tr><tr><td>IGSs size in bp (%) </td><td>42,541 (28) </td><td>42,804 (28.2) </td><td>43,036 (28.3) </td><td>43,018 (28.3) </td><td>42,971 (28.3) </td></tr><tr><td>No. of different genes </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td></tr><tr><td>No. of different protein-coding genes </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td></tr><tr><td>No. of different rRNA genes </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td></tr><tr><td>No. of different duplicated genes by IR </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>No. of different genes with introns </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>Overall % of GC content </td><td>36.6 </td><td>36.7 </td><td>36.6 </td><td>36.7 </td><td>36.6 </td></tr><tr><td>% of GC content in protein-coding regions </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td></tr><tr><td>% of GC content in introns </td><td>36.3 </td><td>36.9 </td><td>36.8 </td><td>36.8 </td><td>36.8 </td></tr><tr><td>% of GC content in IGSs </td><td>30.3 </td><td>30.4 </td><td>30.3 </td><td>30.4 </td><td>30.3 </td></tr><tr><td>% of GC content in rRNA </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td></tr><tr><td>% of GC content in tRNA </td><td>53.3 </td><td>53.2 </td><td>53.3 </td><td>53.3 </td><td>53.3 </td></tr></tbody></table></div><div content-id="dsx006-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Plastome characteristics </th><th><em>L. luteus</em> </th><th><em>L. albus</em> </th><th><em>L. atlanticus</em> </th><th><em>L. princei</em> </th><th><em>L. micranthus</em> </th></tr></thead><tbody><tr><td>Overall size in bp </td><td>151,894 </td><td>151,921 </td><td>152,272 </td><td>152,243 </td><td>151,808 </td></tr><tr><td>LSC size in bp (%) </td><td>82,327 (54.2) </td><td>82,280 (54.2) </td><td>82,674 (54.3) </td><td>82,663 (54.3) </td><td>82,145 (54.1) </td></tr><tr><td>SSC size in bp (%) </td><td>17,847 (11.7) </td><td>17,841 (11.7) </td><td>17,894 (11.8) </td><td>17,876 (11.7) </td><td>17,857 (11.8) </td></tr><tr><td>IR size in bp (%) </td><td>25,860 (34.1) </td><td>25,900 (34.1) </td><td>25,852 (34) </td><td>25,852 (34) </td><td>25,903 (34.1) </td></tr><tr><td>Coding regions size in bp (%) </td><td>90,217 (59.4) </td><td>90,002 (59.2) </td><td>90,125 (59.2) </td><td>90,104 (59.2) </td><td>90,083 (59.3) </td></tr><tr><td>Protein-coding region in bp (%) </td><td>78,363 (51.6) </td><td>78,148 (51.4) </td><td>78,271 (51.4) </td><td>78,250 (51.4) </td><td>78,229 (51.5) </td></tr><tr><td>Introns size in bp (%) </td><td>19,136 (12.6) </td><td>19,115 (12.6) </td><td>19,111 (12.6) </td><td>19,121 (12.6) </td><td>18,754 (12.4) </td></tr><tr><td>rRNA size in bp (%) </td><td>9,056 (6) </td><td>9,056 (6) </td><td>9,056 (5.9) </td><td>9,056 (5.9) </td><td>9,056 (6) </td></tr><tr><td>tRNA size in bp (%) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td></tr><tr><td>IGSs size in bp (%) </td><td>42,541 (28) </td><td>42,804 (28.2) </td><td>43,036 (28.3) </td><td>43,018 (28.3) </td><td>42,971 (28.3) </td></tr><tr><td>No. of different genes </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td></tr><tr><td>No. of different protein-coding genes </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td></tr><tr><td>No. of different rRNA genes </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td></tr><tr><td>No. of different duplicated genes by IR </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>No. of different genes with introns </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>Overall % of GC content </td><td>36.6 </td><td>36.7 </td><td>36.6 </td><td>36.7 </td><td>36.6 </td></tr><tr><td>% of GC content in protein-coding regions </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td></tr><tr><td>% of GC content in introns </td><td>36.3 </td><td>36.9 </td><td>36.8 </td><td>36.8 </td><td>36.8 </td></tr><tr><td>% of GC content in IGSs </td><td>30.3 </td><td>30.4 </td><td>30.3 </td><td>30.4 </td><td>30.3 </td></tr><tr><td>% of GC content in rRNA </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td></tr><tr><td>% of GC content in tRNA </td><td>53.3 </td><td>53.2 </td><td>53.3 </td><td>53.3 </td><td>53.3 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx006-T1" data-id="dsx006-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Characteristics of <em>Lupinus</em> plastomes</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Plastome characteristics </th><th><em>L. luteus</em> </th><th><em>L. albus</em> </th><th><em>L. atlanticus</em> </th><th><em>L. princei</em> </th><th><em>L. micranthus</em> </th></tr></thead><tbody><tr><td>Overall size in bp </td><td>151,894 </td><td>151,921 </td><td>152,272 </td><td>152,243 </td><td>151,808 </td></tr><tr><td>LSC size in bp (%) </td><td>82,327 (54.2) </td><td>82,280 (54.2) </td><td>82,674 (54.3) </td><td>82,663 (54.3) </td><td>82,145 (54.1) </td></tr><tr><td>SSC size in bp (%) </td><td>17,847 (11.7) </td><td>17,841 (11.7) </td><td>17,894 (11.8) </td><td>17,876 (11.7) </td><td>17,857 (11.8) </td></tr><tr><td>IR size in bp (%) </td><td>25,860 (34.1) </td><td>25,900 (34.1) </td><td>25,852 (34) </td><td>25,852 (34) </td><td>25,903 (34.1) </td></tr><tr><td>Coding regions size in bp (%) </td><td>90,217 (59.4) </td><td>90,002 (59.2) </td><td>90,125 (59.2) </td><td>90,104 (59.2) </td><td>90,083 (59.3) </td></tr><tr><td>Protein-coding region in bp (%) </td><td>78,363 (51.6) </td><td>78,148 (51.4) </td><td>78,271 (51.4) </td><td>78,250 (51.4) </td><td>78,229 (51.5) </td></tr><tr><td>Introns size in bp (%) </td><td>19,136 (12.6) </td><td>19,115 (12.6) </td><td>19,111 (12.6) </td><td>19,121 (12.6) </td><td>18,754 (12.4) </td></tr><tr><td>rRNA size in bp (%) </td><td>9,056 (6) </td><td>9,056 (6) </td><td>9,056 (5.9) </td><td>9,056 (5.9) </td><td>9,056 (6) </td></tr><tr><td>tRNA size in bp (%) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td></tr><tr><td>IGSs size in bp (%) </td><td>42,541 (28) </td><td>42,804 (28.2) </td><td>43,036 (28.3) </td><td>43,018 (28.3) </td><td>42,971 (28.3) </td></tr><tr><td>No. of different genes </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td></tr><tr><td>No. of different protein-coding genes </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td></tr><tr><td>No. of different rRNA genes </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td></tr><tr><td>No. of different duplicated genes by IR </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>No. of different genes with introns </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>Overall % of GC content </td><td>36.6 </td><td>36.7 </td><td>36.6 </td><td>36.7 </td><td>36.6 </td></tr><tr><td>% of GC content in protein-coding regions </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td></tr><tr><td>% of GC content in introns </td><td>36.3 </td><td>36.9 </td><td>36.8 </td><td>36.8 </td><td>36.8 </td></tr><tr><td>% of GC content in IGSs </td><td>30.3 </td><td>30.4 </td><td>30.3 </td><td>30.4 </td><td>30.3 </td></tr><tr><td>% of GC content in rRNA </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td></tr><tr><td>% of GC content in tRNA </td><td>53.3 </td><td>53.2 </td><td>53.3 </td><td>53.3 </td><td>53.3 </td></tr></tbody></table></div><div content-id="dsx006-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Plastome characteristics </th><th><em>L. luteus</em> </th><th><em>L. albus</em> </th><th><em>L. atlanticus</em> </th><th><em>L. princei</em> </th><th><em>L. micranthus</em> </th></tr></thead><tbody><tr><td>Overall size in bp </td><td>151,894 </td><td>151,921 </td><td>152,272 </td><td>152,243 </td><td>151,808 </td></tr><tr><td>LSC size in bp (%) </td><td>82,327 (54.2) </td><td>82,280 (54.2) </td><td>82,674 (54.3) </td><td>82,663 (54.3) </td><td>82,145 (54.1) </td></tr><tr><td>SSC size in bp (%) </td><td>17,847 (11.7) </td><td>17,841 (11.7) </td><td>17,894 (11.8) </td><td>17,876 (11.7) </td><td>17,857 (11.8) </td></tr><tr><td>IR size in bp (%) </td><td>25,860 (34.1) </td><td>25,900 (34.1) </td><td>25,852 (34) </td><td>25,852 (34) </td><td>25,903 (34.1) </td></tr><tr><td>Coding regions size in bp (%) </td><td>90,217 (59.4) </td><td>90,002 (59.2) </td><td>90,125 (59.2) </td><td>90,104 (59.2) </td><td>90,083 (59.3) </td></tr><tr><td>Protein-coding region in bp (%) </td><td>78,363 (51.6) </td><td>78,148 (51.4) </td><td>78,271 (51.4) </td><td>78,250 (51.4) </td><td>78,229 (51.5) </td></tr><tr><td>Introns size in bp (%) </td><td>19,136 (12.6) </td><td>19,115 (12.6) </td><td>19,111 (12.6) </td><td>19,121 (12.6) </td><td>18,754 (12.4) </td></tr><tr><td>rRNA size in bp (%) </td><td>9,056 (6) </td><td>9,056 (6) </td><td>9,056 (5.9) </td><td>9,056 (5.9) </td><td>9,056 (6) </td></tr><tr><td>tRNA size in bp (%) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td><td>2,798 (1.8) </td></tr><tr><td>IGSs size in bp (%) </td><td>42,541 (28) </td><td>42,804 (28.2) </td><td>43,036 (28.3) </td><td>43,018 (28.3) </td><td>42,971 (28.3) </td></tr><tr><td>No. of different genes </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td><td>110 </td></tr><tr><td>No. of different protein-coding genes </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td><td>76 </td></tr><tr><td>No. of different rRNA genes </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td><td>30 </td></tr><tr><td>No. of different duplicated genes by IR </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>No. of different genes with introns </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td><td>18 </td></tr><tr><td>Overall % of GC content </td><td>36.6 </td><td>36.7 </td><td>36.6 </td><td>36.7 </td><td>36.6 </td></tr><tr><td>% of GC content in protein-coding regions </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td><td>37.3 </td></tr><tr><td>% of GC content in introns </td><td>36.3 </td><td>36.9 </td><td>36.8 </td><td>36.8 </td><td>36.8 </td></tr><tr><td>% of GC content in IGSs </td><td>30.3 </td><td>30.4 </td><td>30.3 </td><td>30.4 </td><td>30.3 </td></tr><tr><td>% of GC content in rRNA </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td><td>55.3 </td></tr><tr><td>% of GC content in tRNA </td><td>53.3 </td><td>53.2 </td><td>53.3 </td><td>53.3 </td><td>53.3 </td></tr></tbody></table></div></div></p>                        <a id="106355004" scrollto-destination="106355004"></a>
<div id="dsx006-F1" data-id="dsx006-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f1.png?Expires=2147483647&Signature=dPS~RqsQPgiegGq-OyusJOB3-5HQVBH7mHUOCWOObgunQS7UPBcuhZso0F7xXeUPJptJRlGsrXWoplhZWpHKOIuU8l2NLI5TKo0D08ld3BiX35U1sQlZdxwHINyk3~NrrxGWr3ea-BLipTjgGOGy-p90gQ9oo1X7~hhKSSRcbRQ1HieVmP2K~Mmhsux1kpdgUSpNyS~iaLwJDAN~KfJ92tDBksxZd9SEhjQOgcW1P6Dnz1fXIuuorjvaXOQFd1Q6xO6VmK5GUCgsfgodzpO5x6JL9kSbFk-nGKm1O-278h-whLCNMaqf7FcZUcoLK0Wj4eWyE~ZRWguUx3P9hizP1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of lupine chloroplast rps16 coding-sequences with legume rps16 sequences (Glycine max and Lotus japonicus) and Cucumis sativus rps16 sequence (outgroup). The ribosomal protein S16 domain is indicated between brackets. The presence of a pre-mature stop codon within the rps16 functional domain of L. albus and L. micranthus is represented by a black asterisk. The black triangle denotes the position of rps16 intron. It is important to note that the five lupine species present incorrect splicing sites according to.59" path-from-xml="dsx006f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355004/dsx006f1.png" path-from-xml="dsx006f1.png" target="_blank">View large</a><a section="106355004" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f1.png?Expires=2147483647&Signature=aSZoW5p8MCyKmgoExh1szW9c8lnf65wZmEAcffCbWg2ChotxFBqzqgnogdSzcPm1ikrokdHjmxQvfYH7GM7te1FK9KwvYUj~d-ePOP9qO25oeEif0xS4q1F-wTQDesltgs1vc6609DN8kEe-kJGsPTv~osGV6G6RXMcI0gpRA2EC~S80YzmXneqKI7S3LXcSr4FkRuJNF3Do~8Pw5XfaGEbVIKozjoFqKCSFe6Xedfalx8iuEJSjU~PlqChztfniZTFSt02-Ury8nxTlc3ZjPCMTAg0nv~dSswB40n2Bf2bzqBXPm~DQVkOpt3P~afO28MFZEgUZVWT31iZTxPAS0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355004&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of lupine chloroplast <em>rps16</em> coding-sequences with legume <em>rps16</em> sequences (<em>Glycine max</em> and <em>Lotus japonicus</em>) and <em>Cucumis sativus rps16</em> sequence (outgroup). The ribosomal protein S16 domain is indicated between brackets. The presence of a pre-mature stop codon within the <em>rps16</em> functional domain of <em>L. albus</em> and <em>L. micranthus</em> is represented by a black asterisk. The black triangle denotes the position of <em>rps16</em> intron. It is important to note that the five lupine species present incorrect splicing sites according to.<a reveal-id="dsx006-B59" data-open="dsx006-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a></p></div></div><div content-id="dsx006-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f1.png?Expires=2147483647&Signature=dPS~RqsQPgiegGq-OyusJOB3-5HQVBH7mHUOCWOObgunQS7UPBcuhZso0F7xXeUPJptJRlGsrXWoplhZWpHKOIuU8l2NLI5TKo0D08ld3BiX35U1sQlZdxwHINyk3~NrrxGWr3ea-BLipTjgGOGy-p90gQ9oo1X7~hhKSSRcbRQ1HieVmP2K~Mmhsux1kpdgUSpNyS~iaLwJDAN~KfJ92tDBksxZd9SEhjQOgcW1P6Dnz1fXIuuorjvaXOQFd1Q6xO6VmK5GUCgsfgodzpO5x6JL9kSbFk-nGKm1O-278h-whLCNMaqf7FcZUcoLK0Wj4eWyE~ZRWguUx3P9hizP1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of lupine chloroplast rps16 coding-sequences with legume rps16 sequences (Glycine max and Lotus japonicus) and Cucumis sativus rps16 sequence (outgroup). The ribosomal protein S16 domain is indicated between brackets. The presence of a pre-mature stop codon within the rps16 functional domain of L. albus and L. micranthus is represented by a black asterisk. The black triangle denotes the position of rps16 intron. It is important to note that the five lupine species present incorrect splicing sites according to.59" path-from-xml="dsx006f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355004/dsx006f1.png" path-from-xml="dsx006f1.png" target="_blank">View large</a><a section="106355004" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f1.png?Expires=2147483647&Signature=aSZoW5p8MCyKmgoExh1szW9c8lnf65wZmEAcffCbWg2ChotxFBqzqgnogdSzcPm1ikrokdHjmxQvfYH7GM7te1FK9KwvYUj~d-ePOP9qO25oeEif0xS4q1F-wTQDesltgs1vc6609DN8kEe-kJGsPTv~osGV6G6RXMcI0gpRA2EC~S80YzmXneqKI7S3LXcSr4FkRuJNF3Do~8Pw5XfaGEbVIKozjoFqKCSFe6Xedfalx8iuEJSjU~PlqChztfniZTFSt02-Ury8nxTlc3ZjPCMTAg0nv~dSswB40n2Bf2bzqBXPm~DQVkOpt3P~afO28MFZEgUZVWT31iZTxPAS0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355004&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of lupine chloroplast <em>rps16</em> coding-sequences with legume <em>rps16</em> sequences (<em>Glycine max</em> and <em>Lotus japonicus</em>) and <em>Cucumis sativus rps16</em> sequence (outgroup). The ribosomal protein S16 domain is indicated between brackets. The presence of a pre-mature stop codon within the <em>rps16</em> functional domain of <em>L. albus</em> and <em>L. micranthus</em> is represented by a black asterisk. The black triangle denotes the position of <em>rps16</em> intron. It is important to note that the five lupine species present incorrect splicing sites according to.<a reveal-id="dsx006-B59" data-open="dsx006-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a></p></div></div>                            <h3 scrollto-destination=106355005 id="106355005" class="section-title" >3.2. Evolutionary dynamic and fate of the <em>rps16</em> gene in plant mitochondrial, chloroplast and nuclear genomes</h3>
<p>As in some <em>Lupinus</em> species, the chloroplast <em>rps16</em> gene was missing in many other Fabaceae, including <em>P. vulgaris</em> and the IRLC.<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In this family, the chloroplast <em>rps16</em> gene, which is essential for plant survival, has been functionally replaced by a nuclear gene that can encode both mitochondrial and chloroplast RPS16 proteins.<a reveal-id="dsx006-B28" data-open="dsx006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> To better understand the origin and evolutionary fate of <em>rps16</em> genes residing in different genome compartments but with similar functions, we searched for plant chloroplast, mitochondrial and nuclear <em>rps16</em> genes in the currently available nuclear and organelle genomes. In total, we investigated 52 nuclear, 289 mitochondrion and 1,166 chloroplast genomes from the non-parasitic brown and green plant lineages (Haptophytes, Stramenopiles, Glaucophytes, Rhodophytes, Chlorophytes and Streptophytes). Within all the sequenced mitochondrial genomes, no <em>rps16</em> gene was found, whereas a functional (no pre-mature stop codon) nuclear <em>rps16</em> gene copy was observed in all species investigated. The loss of the mitochondrial <em>rps16</em> gene before the divergence of the Glaucophyta from Rhodophyta, Chlorophyta and Streptophyta lineages suggests that the transfer of <em>rps16</em> from the mitochondrion to the nucleus occurred ∼1,500 million years ago<a reveal-id="dsx006-B74" data-open="dsx006-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> (<a reveal-id="dsx006-F2" data-open="dsx006-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), which is much earlier than previously determined (i.e. before the emergence of angiosperms).<a reveal-id="dsx006-B28" data-open="dsx006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> As the <em>rps16</em> gene was lost from the mitochondrion before divergence of the green lineage and as the nuclear <em>rps16</em> gene encodes mitochondrion RPS16 proteins, its functional loss from the nuclear genome would be detrimental. As expected, a peptide signal targeting the nuclear encoded RPS16 proteins to the mitochondrion was identified in all species investigated, while the presence of a chloroplast target peptide was predicted in only a few species (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs4_dsx006.xls?Expires=2147483647&amp;Signature=EcnotUJeYrsGfe0p~NrpLcwyqp-xzvlM6auxjs~pxT27Nbei4T4KoBv0b9fMu3S8~YQlICXYx7wbwjvoRnxvxHc3Dxcs11JpohZXMZ3zC3c46-BRRWklOJXfcqlF7j2rHXn6VEvF2AYLdoJHteJJSb3uDwK8~hQ0~Lcvoc9P2D753znTShrPbEz45J8Sw~RrUdG5md1BnGVqS6Y3X8U-GNGpkXJviIuudxt3moClhITirGaNoc9DWCmt5RnWaXsOGeZ0847h92ASLIsZnBmysLngdadAnL0JKVng~9HJEi9-w34r7CE4AE~Y87YPm0ON3vPOGRo4l4JPAB4uPAakKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). However, it is likely that all species present a nuclear <em>rps16</em> gene that can target the protein to both the mitochondria and plastids, as previously demonstrated by.<a reveal-id="dsx006-B28" data-open="dsx006-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Indeed, these authors showed experimentally that in two species that have lost the <em>rps16</em> gene from their chloroplast (<em>Medicago truncatula</em> and <em>Populus alba</em>), and for which the nuclear encoded RPS16 proteins were only predicted to be localized in the mitochondrion (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs4_dsx006.xls?Expires=2147483647&amp;Signature=EcnotUJeYrsGfe0p~NrpLcwyqp-xzvlM6auxjs~pxT27Nbei4T4KoBv0b9fMu3S8~YQlICXYx7wbwjvoRnxvxHc3Dxcs11JpohZXMZ3zC3c46-BRRWklOJXfcqlF7j2rHXn6VEvF2AYLdoJHteJJSb3uDwK8~hQ0~Lcvoc9P2D753znTShrPbEz45J8Sw~RrUdG5md1BnGVqS6Y3X8U-GNGpkXJviIuudxt3moClhITirGaNoc9DWCmt5RnWaXsOGeZ0847h92ASLIsZnBmysLngdadAnL0JKVng~9HJEi9-w34r7CE4AE~Y87YPm0ON3vPOGRo4l4JPAB4uPAakKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>), the RPS16 proteins were targeted to both organelles. It is thus very likely that despite the absence of a predicted chloroplast target peptide, nuclear-encoded RPS16 proteins are targeted to both mitochondria and chloroplasts. </p>                        <a id="106355007" scrollto-destination="106355007"></a>
<div id="dsx006-F2" data-id="dsx006-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f2.png?Expires=2147483647&Signature=U5enWrT3y3xP1HCqasRldfeF1EpuzUjxSfeVAmPAIig~Htfwrc5Bie7wFKi~-4amlxBV4SsZDzvCQqe8ylGZWjoTIw~jjUUKD6Q9JYfD6fTpbgO~okBUmCjM3Qv52hTOOeun6UBKeiCFixEYAW5whir5yM~phmflGiw14kgpkgZDvkjjCOuMC-~ojxaJWP3HMa~v76B9KGIGzuutvgGcgX5PK0SuIM2fdIdxsP0CgZD~qLXd7RuCc8V7Cykbu8nHy1Nl123YaXoBkp4GbKBTzbG9bCryr6CDNGOZtWjfGxuVMw653QZcw5eGuxF0fHcb8l9PG-UK1EwbYu23h1TPlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome localization of the rps16 gene(s) encoding the mitochondrial and chloroplast RPS16 proteins in Archeaplastida. Early in Archeaplastida, the mitochondrial rps16 gene was transferred to the nucleus (nuc) and acquired a signal peptide targeting both mitochondrion (mt) and chloroplast (cp). In Glaucophytes and Red Algae, the mitochondrial rps16 gene is always absent whereas it is present in 13 Rhodophyta chloroplast genomes (no plastome sequence available from Glaucophytes). In the core Chlorophyta lineage, none of the 76 species having a fully sequenced chloroplast and mitochondrial genomes have a rps16 gene. In the Streptophyta lineage, no rps16 gene was found in the mitochondrial genomes, whereas the chloroplast rps16 gene may either be functional or loss its functionality (complete gene loss, presence of a pre-mature stop codon or loss of the splicing capacity). Tree was redrawn according to Ref.97,98" path-from-xml="dsx006f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355007/dsx006f2.png" path-from-xml="dsx006f2.png" target="_blank">View large</a><a section="106355007" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f2.png?Expires=2147483647&Signature=4lMyQ5ZQyESchvKC7flM116A7lsnaHm7dZmjb2JWCqM-HFAdOltzNxQsumT7MC5M~GPx9lQS2q1mqzVTQ1-NgPUiz7s1ZqRf8LVE71g5HVEJKFIWwQ0CnM-ST~p~e1L-vz230vx5-lVUHwsDuz8J~3OLQTq6cyopnUcxaqhurD4SzRiq4tVINrl3X4-BirRSzuCsOr-KvFXC0Qa2LRsSiwkii3lkV5q-vg8B6~INfHpOp-cMyT5BencS1oY7tTNrmXGGCAuADAdFX7ugP7OuPMB3WTn-LOGtSxJ08i~rQ9m7MOHF1ktznotephkCIhXVjDHYwsL-1kwLi1YI7si5xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355007&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome localization of the <em>rps16</em> gene(s) encoding the mitochondrial and chloroplast RPS16 proteins in Archeaplastida. Early in Archeaplastida, the mitochondrial <em>rps16</em> gene was transferred to the nucleus (nuc) and acquired a signal peptide targeting both mitochondrion (mt) and chloroplast (cp). In Glaucophytes and Red Algae, the mitochondrial <em>rps16</em> gene is always absent whereas it is present in 13 Rhodophyta chloroplast genomes (no plastome sequence available from Glaucophytes). In the core Chlorophyta lineage, none of the 76 species having a fully sequenced chloroplast and mitochondrial genomes have a <em>rps16</em> gene. In the Streptophyta lineage, no <em>rps16</em> gene was found in the mitochondrial genomes, whereas the chloroplast <em>rps16</em> gene may either be functional or loss its functionality (complete gene loss, presence of a pre-mature stop codon or loss of the splicing capacity). Tree was redrawn according to Ref.<a reveal-id="dsx006-B97" data-open="dsx006-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a><sup>,</sup><a reveal-id="dsx006-B98" data-open="dsx006-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a></p></div></div><div content-id="dsx006-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f2.png?Expires=2147483647&Signature=U5enWrT3y3xP1HCqasRldfeF1EpuzUjxSfeVAmPAIig~Htfwrc5Bie7wFKi~-4amlxBV4SsZDzvCQqe8ylGZWjoTIw~jjUUKD6Q9JYfD6fTpbgO~okBUmCjM3Qv52hTOOeun6UBKeiCFixEYAW5whir5yM~phmflGiw14kgpkgZDvkjjCOuMC-~ojxaJWP3HMa~v76B9KGIGzuutvgGcgX5PK0SuIM2fdIdxsP0CgZD~qLXd7RuCc8V7Cykbu8nHy1Nl123YaXoBkp4GbKBTzbG9bCryr6CDNGOZtWjfGxuVMw653QZcw5eGuxF0fHcb8l9PG-UK1EwbYu23h1TPlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome localization of the rps16 gene(s) encoding the mitochondrial and chloroplast RPS16 proteins in Archeaplastida. Early in Archeaplastida, the mitochondrial rps16 gene was transferred to the nucleus (nuc) and acquired a signal peptide targeting both mitochondrion (mt) and chloroplast (cp). In Glaucophytes and Red Algae, the mitochondrial rps16 gene is always absent whereas it is present in 13 Rhodophyta chloroplast genomes (no plastome sequence available from Glaucophytes). In the core Chlorophyta lineage, none of the 76 species having a fully sequenced chloroplast and mitochondrial genomes have a rps16 gene. In the Streptophyta lineage, no rps16 gene was found in the mitochondrial genomes, whereas the chloroplast rps16 gene may either be functional or loss its functionality (complete gene loss, presence of a pre-mature stop codon or loss of the splicing capacity). Tree was redrawn according to Ref.97,98" path-from-xml="dsx006f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355007/dsx006f2.png" path-from-xml="dsx006f2.png" target="_blank">View large</a><a section="106355007" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f2.png?Expires=2147483647&Signature=4lMyQ5ZQyESchvKC7flM116A7lsnaHm7dZmjb2JWCqM-HFAdOltzNxQsumT7MC5M~GPx9lQS2q1mqzVTQ1-NgPUiz7s1ZqRf8LVE71g5HVEJKFIWwQ0CnM-ST~p~e1L-vz230vx5-lVUHwsDuz8J~3OLQTq6cyopnUcxaqhurD4SzRiq4tVINrl3X4-BirRSzuCsOr-KvFXC0Qa2LRsSiwkii3lkV5q-vg8B6~INfHpOp-cMyT5BencS1oY7tTNrmXGGCAuADAdFX7ugP7OuPMB3WTn-LOGtSxJ08i~rQ9m7MOHF1ktznotephkCIhXVjDHYwsL-1kwLi1YI7si5xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355007&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome localization of the <em>rps16</em> gene(s) encoding the mitochondrial and chloroplast RPS16 proteins in Archeaplastida. Early in Archeaplastida, the mitochondrial <em>rps16</em> gene was transferred to the nucleus (nuc) and acquired a signal peptide targeting both mitochondrion (mt) and chloroplast (cp). In Glaucophytes and Red Algae, the mitochondrial <em>rps16</em> gene is always absent whereas it is present in 13 Rhodophyta chloroplast genomes (no plastome sequence available from Glaucophytes). In the core Chlorophyta lineage, none of the 76 species having a fully sequenced chloroplast and mitochondrial genomes have a <em>rps16</em> gene. In the Streptophyta lineage, no <em>rps16</em> gene was found in the mitochondrial genomes, whereas the chloroplast <em>rps16</em> gene may either be functional or loss its functionality (complete gene loss, presence of a pre-mature stop codon or loss of the splicing capacity). Tree was redrawn according to Ref.<a reveal-id="dsx006-B97" data-open="dsx006-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a><sup>,</sup><a reveal-id="dsx006-B98" data-open="dsx006-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a></p></div></div><p>Within the chloroplast genomes investigated (1,166), the <em>rps16</em> gene was found to be missing (total absence of the gene or truncated proteins due to pre-mature stop codon) in 312 genomes. We looked for the presence of correct splicing sites in chloroplast <em>rps16</em> genes with a complete coding sequence and an intron. We found that 434 and 306 had or did not have the splicing capacity, respectively. Among the latter, 197 exhibited mutations in both 5ʹ and 3ʹ splice sites, whereas 22 and 87 had mutations only at the 5ʹ or the 3ʹ splice sites, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs5_dsx006.xls?Expires=2147483647&amp;Signature=u8zRXIJj3GgCOD7Y1P5FBeCGQgOlKLVSK9cG~rZi~BCSvg8-pIcLTsKIWEqKeMez7ZPqrQnpT-~AL9B9cWdZU1e1vUEsvv3AISfSSmxdZ0ZY51qNRpxCubInUjdaK9LIqKYF9kvY9~V9auqx5QqQl9dk7byX78e8MULav12RD40P1LFGcqoLB4GU2CTJexzac9cFfWbkrOpslOj3qBwRagE6djewJtzIrEiiNFyBWvpPvKUgi8k0T16Wt6msvkBNinwA5Bx5qGnxTaFXivtYdd0CxFJNqmNgplGNojg1Cy-UeJGHoSB75guVc83bBnUHBvf9bP5Z3Ii1SXTG3NZZKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). As previously observed, this gene has lost its functionality many times during flowering plant evolution,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> by the loss of either all or part of the coding sequence or of the splicing sites. Dating back to 1,500 MYA, chloroplast RPS16 proteins can be produced by either nuclear or chloroplast <em>rps16</em> genes. Our results highlight the fact that even though the chloroplast <em>rps16</em> gene could have been non-functional in all plant genomes since then, it is still present and functional in most plants.</p><p>To better understand the evolutionary dynamics of the <em>rps16</em> gene, we analysed the selective pressure acting on functional chloroplast and nuclear <em>rps16</em> genes among 12 species representing the main angiosperms clades. As the <em>rps16</em> gene was functional in both chloroplast and nuclear genomes and as only the chloroplast copy is likely to be lost, the selective constraints acting on the chloroplast gene could be relaxed. Results of Ka/Ks ratios revealed a strong purifying selection for both chloroplast and nuclear <em>rps16</em> (average Ka/Ks ratio: 0.045 ± se 0.010 and 0.1707 ± se 0.003 for nuclear and chloroplast copies, respectively; <span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs6_dsx006.xls?Expires=2147483647&amp;Signature=iAt2gOjVRj6I1PdOTyMQsi8W0iIdzPY105sKyW8UITsk7EQs-sA40HylOjm4ehzoerX-oPYepoboUQcrqXvHtWL8dPDy3P4pEU-Eb0aOyxUknJ79d-DU3KV7a5rtO21USO9MxFJ4t9uVzg2UGL8ZgypxTHadWEtWJpTE8muvb5jA35VUbZUjc3ECAuPQmxEXSIHV8bnmrwP4luNRGbHpEiZR9YwKRLAtRyxnCtjfw3CLrjZ~PsNC1TSGIGPn7seHgdGRw933CkEZ3yi~dNdLbwtLqVswyjUb1cHMrhvkCCH9UmZuM3uTBXsAXEORf7Ac4rqlL2hLQE-O97b6KIibKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs7_dsx006.xls?Expires=2147483647&amp;Signature=Cr7YSxCvQGze5pLxm3QwN8WhQCdSPaqpVWn8NG1WsnrbMMkvvegsVbCOV4GA5rW7gcvljwWZjW0QptB6Lsmu2QlnlO99HE5GA-AgkjvfdLqGB9yIz3UnMI-adpfKglBXvQfU0iY13Zhd~T5DHh91rzSjiCObShvW8Ulukcx2jmJb7yOpDyYDvn8MKlHKnOona6Y-sfVnBnHH3VMjqbaILRqq1TF7fKZfOK6lnyv5UU9vAaejkYh2ppja9iT~-MZYt9KNvIEuybyA4wzfc2vlwmt5lmURLA7Vo23vlzAlBUU8N4xHVHhM0rMiqLzlJVBNzXElcIFTQVsSE3QIbYCEPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S7</a></span>). Ka/Ks ratios of cp-<em>rps16</em> for each of the main Angiosperm families (Asteraceae, Brassicaceae, Fabaceae, Poaceae and Solanaceae) were also calculated for all cp-<em>rps16</em> found with a complete coding sequence and a correctly spliced intron. Results were similar for each of the five families investigated and revealed a strong purifying selection acting on all the tested datasets (average of Ka/Ks ratios always lower than 0.23; <span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs8_dsx006.xls?Expires=2147483647&amp;Signature=AuC-cKZ5nra7AGvPbN0sdfTTVDSKlMqo0sfzrDd6MTHlJcpQscporn6AZ6kyXohwnLjTgrvyWD1EyM5EPs~~NfgOPYlDYXMuDSjmVustcGWBAFEqCgMRnYo-BrQXE-J4AOUgFE-JtasI3A98c1LA4ow28FE9xhFq9ioCHNBF3TfjebaJNU9LGeL~sroOX-88iAFE2N64e0ivzT1kfl5V2DZrFju2nEKRImxdIr85wZ6P6IMhVxKS9Q88qylHXIP7ZEEkRGMPVO2ktQYMnvPelOz~29p3EzHc5uSQAlcZmeSKcAhT1sC718d5qtpQyjHZ1kTnr-SgS7m-T~56vi~X9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). A possible explanation of this strong negative selection pressure still acting on the chloroplast <em>rps16</em> gene and the presence of a functional chloroplast <em>rps16</em> gene in many plant genomes (in contrast with plant mitochondrial genomes) is that the chloroplast <em>rps16</em> gene may function or be regulated slightly better than the nuclear gene under certain conditions.</p><p>Although these results revealed the multiple status (absent, truncated, incorrectly spliced, functional) of the chloroplast <em>rps16</em> gene among the plant kingdom and confirmed the hypothesis of<a reveal-id="dsx006-B59" data-open="dsx006-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> that the loss of splicing capacity is widely spread through plant species, mechanisms beyond the conservation of the chloroplast copy in most species remain unknown. Indeed, the chloroplast <em>rps16</em> gene is still essential in certain plant species as revealed by knock-down studies in tobacco.<a reveal-id="dsx006-B29" data-open="dsx006-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Different hypotheses have been proposed to explain the retention of some genes within the organelle genomes. The current most widely accepted hypothesis corresponds to the Colocation of gene and gene product for redox regulation of gene expression (or CoRR).<a reveal-id="dsx006-B75" data-open="dsx006-B75" class="link link-ref link-reveal xref-bibr"><sup>75–77</sup></a> This hypothesis concerns genes that are redox-dependant (such as <em>rbcL</em>; <em>rps2,3,4,7,8,11,12,14,19</em>; <em>rpl2,14,16,20,36</em>).<a reveal-id="dsx006-B77" data-open="dsx006-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> However, the chloroplast <em>rps16</em> gene has been found to be redox independent.<a reveal-id="dsx006-B78" data-open="dsx006-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> An alternative hypothesis that was considered concerned the retention of the ribosomal assembly genes in the organelle.<a reveal-id="dsx006-B79" data-open="dsx006-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> A ‘core set’ of ribosomal genes were identified in all plants investigated,<a reveal-id="dsx006-B79" data-open="dsx006-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> however, <em>rps16</em> was not included. Another possible explanation of the retention of a functional chloroplast <em>rps16</em> gene in many species may be due to the loss of the chloroplast target peptide of the nuclear-encoded RPS16 protein (despite the fact that the mitochondrion <em>rps16</em> target signal remain retained).</p>                            <h3 scrollto-destination=106355011 id="106355011" class="section-title" >3.3. Lupine plastome variability</h3>
                            <h4 scrollto-destination=106355012 id="106355012" class="section-title" >3.3.1. Identification of single nucleotide polymorphims and indels in lupine plastomes</h4>
<p>To identify putative mutation hotspots, pairwise comparisons of the five lupine plastomes were performed and showed that they have a very high level of sequence identity (98% on average). The two African species (<em>L. atlanticus</em> and <em>L. princei</em>), which are the most closely related lupine species investigated in this study, exhibit the highest identity (99.7%); whereas the species with the lowest sequence identity are <em>L. luteus</em> and <em>L. micranthus</em> (97.9%). These comparisons also enabled identification of 164 non-ambiguous indels along the chloroplast genomes, including 14 with a size ranging from 20 to 357 bp. Of the 164 indels, 50% are 5–6 bp long. These analyses revealed two highly variable regions (<a reveal-id="dsx006-F3" data-open="dsx006-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The first region spans from <em>psaA</em> to <em>ycf4</em> (∼11.5 kb) and was already identified as a hypermutable region.<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> This region contains 11 genes: <em>psaA</em>, <em>ycf4</em>, <em>ycf3</em>, <em>trnS-GGA</em>, <em>psbI</em>, <em>psbK</em>, <em>trnQ-UUG</em>, <em>rps16</em>, <em>accD</em> and <em>psaI</em> genes, for which four genes (<em>accD</em>, <em>rps16</em>, <em>ycf4</em>, <em>psaI</em>) were shown to be functionally lost in at least one legume species. The second most variable region includes the <em>ycf1</em>-<em>rps15</em> genes (∼6.5 kb). The <em>ycf1</em> gene, which encodes a translocon protein of the inner chloroplast membrane,<a reveal-id="dsx006-B80" data-open="dsx006-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> is larger than 5 kb in lupines and is highly variable with the exception of a 5ʹ fragment duplicated in the IR (519 bp in lupines). This gene was recently identified as one of the most variable chloroplast genes in Angiosperms and is considered as a powerful tool for DNA barcoding.<a reveal-id="dsx006-B81" data-open="dsx006-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a><sup>,</sup><a reveal-id="dsx006-B82" data-open="dsx006-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> The longest hypervariable region (<em>psaA</em>-<em>ycf4</em>) contains the highest number of indels (25), with 11 (among the 14 present in the genome) between 20 and 357 bp. Some of these large indels will most likely be useful to discriminate lupine species and/or groups of species from other <em>Lupinus</em> lineages or from other closely related genera. </p>                        <a id="106355014" scrollto-destination="106355014"></a>
<div id="dsx006-F3" data-id="dsx006-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f3.png?Expires=2147483647&Signature=XDxhi6KY7Q8EN7RO54-415rTNcxfWL1bZfEv10bSLKIgtXP0rp1qtNNiZfnqrSDY78bjQBZjdN8XU1BHW9D8y7TNgcbNmCiS1spI9hrWP0fvykBJ8DMmry1CKfMjBxLBqc6pqT-elXnzMDgCpj04rYy~Zyj9unS4gFXgjMVOdNubnZTEfb7ID018bwVz4P-W0v3-nTFycG8Sxo8MuPnNwj86X21K7AlkBL2zoeW0dweh45uuqj~9MYD9-kaO6JbR6x1cjibVOve536KnS1u3RBtCUOcQv9MZ3EOPGQSh5hu3sbGTqDicTM62y8YhKfOM-oNsiNVzGn3~blGK8C95HA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparison of lupine plastomes to identify single nucleotide polymorphisms and indels. The outer circle represents the gene map of lupine plastomes; the boxes outside this first circle indicate a counterclockwise of transcription direction whereas inside boxes indicate a clockwise transcript direction. In the second circle, potentially informative sites are indicated by black bars. The following ten inner circles represent pairwise comparisons between the five available lupine plastomes; pairwise identity level is indicated and indels &gt;20 bp are represented by black triangle. The central black circle represents the different parts of the chloroplast genome (LSC, SSC and IRs). The endpoints of the 50-kb inversion, specific to the Papilionoid legumes20–22 and of the 36-kb inversion, specific to the Genistoid clade,15 are represented by arrows." path-from-xml="dsx006f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355014/dsx006f3.png" path-from-xml="dsx006f3.png" target="_blank">View large</a><a section="106355014" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f3.png?Expires=2147483647&Signature=grr5bHig~rXzbfZynS5q7e9HhcPtrzZ008gcRxuTXlj-jrynrORK7TfB9yXedtMvFO1wdAlHnxSbwWDLvTilEzjuGAEeIFinWr8U8GGOtARlduyZm5fh4YGbigcAC8SrQp8AqkMhq8H~QJXY~JYfl5ZqE1sKBAfS83d5GA7IFi67eqDnvvbeaBmSb9u0bx7dd02eu9i3WhlEyIz0hnwW4RoHvqf-G1kkfk5YwWeLZ~FzCJaGIEe0oQPij4vRAkXjOSGN0dFVDAMHKkzOfHShTFdN0k9HXgnA934F0UYKSBvr2BNBCwKLxwOgM~CpsteGK2IxvviPrvyqSxX9DHvXEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355014&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparison of lupine plastomes to identify single nucleotide polymorphisms and indels. The outer circle represents the gene map of lupine plastomes; the boxes outside this first circle indicate a counterclockwise of transcription direction whereas inside boxes indicate a clockwise transcript direction. In the second circle, potentially informative sites are indicated by black bars. The following ten inner circles represent pairwise comparisons between the five available lupine plastomes; pairwise identity level is indicated and indels &gt;20 bp are represented by black triangle. The central black circle represents the different parts of the chloroplast genome (LSC, SSC and IRs). The endpoints of the 50-kb inversion, specific to the Papilionoid legumes<a reveal-id="dsx006-B20" data-open="dsx006-B20" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> and of the 36-kb inversion, specific to the Genistoid clade,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> are represented by arrows.</p></div></div><div content-id="dsx006-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f3.png?Expires=2147483647&Signature=XDxhi6KY7Q8EN7RO54-415rTNcxfWL1bZfEv10bSLKIgtXP0rp1qtNNiZfnqrSDY78bjQBZjdN8XU1BHW9D8y7TNgcbNmCiS1spI9hrWP0fvykBJ8DMmry1CKfMjBxLBqc6pqT-elXnzMDgCpj04rYy~Zyj9unS4gFXgjMVOdNubnZTEfb7ID018bwVz4P-W0v3-nTFycG8Sxo8MuPnNwj86X21K7AlkBL2zoeW0dweh45uuqj~9MYD9-kaO6JbR6x1cjibVOve536KnS1u3RBtCUOcQv9MZ3EOPGQSh5hu3sbGTqDicTM62y8YhKfOM-oNsiNVzGn3~blGK8C95HA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparison of lupine plastomes to identify single nucleotide polymorphisms and indels. The outer circle represents the gene map of lupine plastomes; the boxes outside this first circle indicate a counterclockwise of transcription direction whereas inside boxes indicate a clockwise transcript direction. In the second circle, potentially informative sites are indicated by black bars. The following ten inner circles represent pairwise comparisons between the five available lupine plastomes; pairwise identity level is indicated and indels &gt;20 bp are represented by black triangle. The central black circle represents the different parts of the chloroplast genome (LSC, SSC and IRs). The endpoints of the 50-kb inversion, specific to the Papilionoid legumes20–22 and of the 36-kb inversion, specific to the Genistoid clade,15 are represented by arrows." path-from-xml="dsx006f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355014/dsx006f3.png" path-from-xml="dsx006f3.png" target="_blank">View large</a><a section="106355014" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f3.png?Expires=2147483647&Signature=grr5bHig~rXzbfZynS5q7e9HhcPtrzZ008gcRxuTXlj-jrynrORK7TfB9yXedtMvFO1wdAlHnxSbwWDLvTilEzjuGAEeIFinWr8U8GGOtARlduyZm5fh4YGbigcAC8SrQp8AqkMhq8H~QJXY~JYfl5ZqE1sKBAfS83d5GA7IFi67eqDnvvbeaBmSb9u0bx7dd02eu9i3WhlEyIz0hnwW4RoHvqf-G1kkfk5YwWeLZ~FzCJaGIEe0oQPij4vRAkXjOSGN0dFVDAMHKkzOfHShTFdN0k9HXgnA934F0UYKSBvr2BNBCwKLxwOgM~CpsteGK2IxvviPrvyqSxX9DHvXEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355014&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparison of lupine plastomes to identify single nucleotide polymorphisms and indels. The outer circle represents the gene map of lupine plastomes; the boxes outside this first circle indicate a counterclockwise of transcription direction whereas inside boxes indicate a clockwise transcript direction. In the second circle, potentially informative sites are indicated by black bars. The following ten inner circles represent pairwise comparisons between the five available lupine plastomes; pairwise identity level is indicated and indels &gt;20 bp are represented by black triangle. The central black circle represents the different parts of the chloroplast genome (LSC, SSC and IRs). The endpoints of the 50-kb inversion, specific to the Papilionoid legumes<a reveal-id="dsx006-B20" data-open="dsx006-B20" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> and of the 36-kb inversion, specific to the Genistoid clade,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> are represented by arrows.</p></div></div>                            <h4 scrollto-destination=106355015 id="106355015" class="section-title" >3.3.2. Sequence divergence between lupine plastomes</h4>
<p>Pairwise distance (K2p) comparisons among the five lupine plastomes were calculated for non-coding sequences. As expected, the lowest rates of variation were observed for tRNA and rRNA (maximum K2p value: 0.0141, <span class="content-section supplementary-material"><a path-from-xml="sup16" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs9_dsx006.xls?Expires=2147483647&amp;Signature=MWpsBn4cHNf-cR7pSTXl52PM1lwp-dNx6KmtLV7pxYjSXsJp9WGDasKX2TX1h~fNPjguncxvNwfxuLoyUBohpwJEkm6oPiuELT~h3HwI8D2a6oYhlXOnsHgwfLyG12fN5hDTa7I0sprFEoJ23zDBWyyefwKvhT3NKpYTuXWkT9d8E4F35Ejpx2ShQ-h9NRzNvHwWI0ZImYH35EAaP1yGxGskZZewP6fk6XvC3iUqTaKJ-d2do2fdocIoub3Jw50p5LmueCmi8u0QH079N08tzekJXxHcduFvt1Z0sp~IV5XuBozjmuoWpYOGVamSHTDqBppzBPvxrBTtukUSNHu1Qg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). For introns (<span class="content-section supplementary-material"><a path-from-xml="sup16" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs9_dsx006.xls?Expires=2147483647&amp;Signature=MWpsBn4cHNf-cR7pSTXl52PM1lwp-dNx6KmtLV7pxYjSXsJp9WGDasKX2TX1h~fNPjguncxvNwfxuLoyUBohpwJEkm6oPiuELT~h3HwI8D2a6oYhlXOnsHgwfLyG12fN5hDTa7I0sprFEoJ23zDBWyyefwKvhT3NKpYTuXWkT9d8E4F35Ejpx2ShQ-h9NRzNvHwWI0ZImYH35EAaP1yGxGskZZewP6fk6XvC3iUqTaKJ-d2do2fdocIoub3Jw50p5LmueCmi8u0QH079N08tzekJXxHcduFvt1Z0sp~IV5XuBozjmuoWpYOGVamSHTDqBppzBPvxrBTtukUSNHu1Qg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs6_dsx006.ppt?Expires=2147483647&amp;Signature=Il96bTJUYPlXDT2qUzeroyXp5TCWyse-8Dt57lSAIXPl7UZ7qhlOX8jSqYFQ6ZVD0yGvRC8hHq24yYR1HV9sC78wCR7sz~EwliXHBg3bjn-rgZ90VVurhcyfTeIbYV2aaIvQVgy41zqZpTdStRo4Uq2x2C2D3x07MUfEH2Qksij8GD0aMin4CFWNoaaQ7lesQMtCkF8PsGnFcZtIGyDRU0~QNEuUf5P4IConHmMkhWOxOtQOc8ukNEYDjF4IpH1-FqmwiqphdY9OeLL5Ei8lbkEFwfGZi4GiksuxtUzEPlteVA4c8I28qMto401sSH2oJkElMBX1Cju5F0uvegFSMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>A), average of K2p rates ranged from 0.0006 (<em>ndhB</em> intron) to 0.0263 (<em>clpP</em> intron 1). Compared with the K2p analyses performed by,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> who estimated sequence divergence between <em>L. luteus</em> and other legume species, our overall K2p values obtained by comparing only lupine species are, as expected, significantly lower (Wilcoxon test, <em>P</em>-value = 0.05; see <a reveal-id="dsx006-F4" data-open="dsx006-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Among the five lupine species considered, the <em>clpP</em> intron 1, <em>rpl16</em>, <em>rpoC1</em> and <em>ndhA</em> introns exhibit the higher K2p values. The most variable intron in lupines corresponds to the first intron of <em>clpP</em>, which also showed accelerated mutation rate in Mimosoideae.<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx006-B18" data-open="dsx006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The <em>trnK</em> and <em>trnL</em> introns previously used for legumes and lupines phylogenies were found to only exhibit high variation when comparing <em>L. luteus</em> to other Fabaceae. </p>                        <a id="106355017" scrollto-destination="106355017"></a>
<div id="dsx006-F4" data-id="dsx006-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f4.png?Expires=2147483647&Signature=yR7v~MK3nAJeSWiSbvwDQh7g7TnChxc4Tb6dtYKJ9FzPkjhjm74w5c1yo5cAu3uXrhAzunuCBRNmkupaDnHJ7~yzdCGuOJBc~cDBX4LYgHoLQ2vQwRFcllLLjGift7wutD0OB4SxzCwuYfU6XJlkR7uwRJFWCxjs515xa0d4gvaDHvt5e118a98FG237wMCl5--3EFdSHO-Xy586XPrSNrPaKQPpQf0MFxaoZ~qUJ~lFo3Kcg7BQfGuKJZ5bPqq92ENuQluu1na8djkIiKwFr1oxStTqv4uqdz8iE2LmOMUu89M0tnjlkxPnxTuDe5iZru4b0z~n1DPQhnY7JfYQqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="K2p mean values ± standard error for (A) introns and (B) intergenic spacers between (i) L. albus, L. atlanticus, L. luteus, L. micranthus and L. princei (black cicles) and (ii) L. luteus, Phaseolus vulgaris, Pisum sativum, Vigna radiata, Glycine max, Lathyrus sativus, Cicer arietinum, Trifolium subterraneum, Medicago truncatula, Lotus japonicus and Millettia pinnata (black squares).15 The x-axis corresponds to intron and intergenic regions. Asterisks represent a statistically significant difference (P.value: 0.05)." path-from-xml="dsx006f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355017/dsx006f4.png" path-from-xml="dsx006f4.png" target="_blank">View large</a><a section="106355017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f4.png?Expires=2147483647&Signature=B5hCVDDFyhC9QKHFvGPMY2zg~1SQfRwVgIeygw1GHglHwcySMyS6IWTlP2fjJJZqSA58SCMMdry2dJmW14aHhogTl4xiRacU9fUV2fNuGGHwaXpR~VObZKGJ18QLVWtWobtxYBFq2CUjm5oTLVlT41TUCgYT8LY8jPURGCS6bnn6U9Rs9bSsoIMgKQWKE5KiD8vS43-p5Nh5CIc~2pNZS2eQ9wmZBznLsDpJcheCAmRAOx-qQJM1Kl-dmfgFMJaEMOuB-kUuWoyzXy~PXhDXg~Jn68CwiRgO-XX-i~iot7b1Icl2bZMjsY~A-iQdvnGdrZTjqSJwlGu~skHeXBrNvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355017&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>K2p mean values ± standard error for (A) introns and (B) intergenic spacers between (i) <em>L. albus</em>, <em>L. atlanticus</em>, <em>L. luteus</em>, <em>L. micranthus</em> and <em>L. princei</em> (black cicles) and (ii) <em>L. luteus</em>, <em>Phaseolus vulgaris</em>, <em>Pisum sativum</em>, <em>Vigna radiata</em>, <em>Glycine max</em>, <em>Lathyrus sativus</em>, <em>Cicer arietinum</em>, <em>Trifolium subterraneum</em>, <em>Medicago truncatula</em>, <em>Lotus japonicus</em> and <em>Millettia pinnata</em> (black squares).<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The <em>x</em>-axis corresponds to intron and intergenic regions. Asterisks represent a statistically significant difference (<em>P</em>.value: 0.05).</p></div></div><div content-id="dsx006-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f4.png?Expires=2147483647&Signature=yR7v~MK3nAJeSWiSbvwDQh7g7TnChxc4Tb6dtYKJ9FzPkjhjm74w5c1yo5cAu3uXrhAzunuCBRNmkupaDnHJ7~yzdCGuOJBc~cDBX4LYgHoLQ2vQwRFcllLLjGift7wutD0OB4SxzCwuYfU6XJlkR7uwRJFWCxjs515xa0d4gvaDHvt5e118a98FG237wMCl5--3EFdSHO-Xy586XPrSNrPaKQPpQf0MFxaoZ~qUJ~lFo3Kcg7BQfGuKJZ5bPqq92ENuQluu1na8djkIiKwFr1oxStTqv4uqdz8iE2LmOMUu89M0tnjlkxPnxTuDe5iZru4b0z~n1DPQhnY7JfYQqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="K2p mean values ± standard error for (A) introns and (B) intergenic spacers between (i) L. albus, L. atlanticus, L. luteus, L. micranthus and L. princei (black cicles) and (ii) L. luteus, Phaseolus vulgaris, Pisum sativum, Vigna radiata, Glycine max, Lathyrus sativus, Cicer arietinum, Trifolium subterraneum, Medicago truncatula, Lotus japonicus and Millettia pinnata (black squares).15 The x-axis corresponds to intron and intergenic regions. Asterisks represent a statistically significant difference (P.value: 0.05)." path-from-xml="dsx006f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355017/dsx006f4.png" path-from-xml="dsx006f4.png" target="_blank">View large</a><a section="106355017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f4.png?Expires=2147483647&Signature=B5hCVDDFyhC9QKHFvGPMY2zg~1SQfRwVgIeygw1GHglHwcySMyS6IWTlP2fjJJZqSA58SCMMdry2dJmW14aHhogTl4xiRacU9fUV2fNuGGHwaXpR~VObZKGJ18QLVWtWobtxYBFq2CUjm5oTLVlT41TUCgYT8LY8jPURGCS6bnn6U9Rs9bSsoIMgKQWKE5KiD8vS43-p5Nh5CIc~2pNZS2eQ9wmZBznLsDpJcheCAmRAOx-qQJM1Kl-dmfgFMJaEMOuB-kUuWoyzXy~PXhDXg~Jn68CwiRgO-XX-i~iot7b1Icl2bZMjsY~A-iQdvnGdrZTjqSJwlGu~skHeXBrNvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355017&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>K2p mean values ± standard error for (A) introns and (B) intergenic spacers between (i) <em>L. albus</em>, <em>L. atlanticus</em>, <em>L. luteus</em>, <em>L. micranthus</em> and <em>L. princei</em> (black cicles) and (ii) <em>L. luteus</em>, <em>Phaseolus vulgaris</em>, <em>Pisum sativum</em>, <em>Vigna radiata</em>, <em>Glycine max</em>, <em>Lathyrus sativus</em>, <em>Cicer arietinum</em>, <em>Trifolium subterraneum</em>, <em>Medicago truncatula</em>, <em>Lotus japonicus</em> and <em>Millettia pinnata</em> (black squares).<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The <em>x</em>-axis corresponds to intron and intergenic regions. Asterisks represent a statistically significant difference (<em>P</em>.value: 0.05).</p></div></div><p>K2p values for IGSs ranged from 0 to 0.0434 (<span class="content-section supplementary-material"><a path-from-xml="sup16" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs9_dsx006.xls?Expires=2147483647&amp;Signature=MWpsBn4cHNf-cR7pSTXl52PM1lwp-dNx6KmtLV7pxYjSXsJp9WGDasKX2TX1h~fNPjguncxvNwfxuLoyUBohpwJEkm6oPiuELT~h3HwI8D2a6oYhlXOnsHgwfLyG12fN5hDTa7I0sprFEoJ23zDBWyyefwKvhT3NKpYTuXWkT9d8E4F35Ejpx2ShQ-h9NRzNvHwWI0ZImYH35EAaP1yGxGskZZewP6fk6XvC3iUqTaKJ-d2do2fdocIoub3Jw50p5LmueCmi8u0QH079N08tzekJXxHcduFvt1Z0sp~IV5XuBozjmuoWpYOGVamSHTDqBppzBPvxrBTtukUSNHu1Qg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs6_dsx006.ppt?Expires=2147483647&amp;Signature=Il96bTJUYPlXDT2qUzeroyXp5TCWyse-8Dt57lSAIXPl7UZ7qhlOX8jSqYFQ6ZVD0yGvRC8hHq24yYR1HV9sC78wCR7sz~EwliXHBg3bjn-rgZ90VVurhcyfTeIbYV2aaIvQVgy41zqZpTdStRo4Uq2x2C2D3x07MUfEH2Qksij8GD0aMin4CFWNoaaQ7lesQMtCkF8PsGnFcZtIGyDRU0~QNEuUf5P4IConHmMkhWOxOtQOc8ukNEYDjF4IpH1-FqmwiqphdY9OeLL5Ei8lbkEFwfGZi4GiksuxtUzEPlteVA4c8I28qMto401sSH2oJkElMBX1Cju5F0uvegFSMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6B</a></span>). In comparison to the commonly-used IGSs in legume phylogenetic studies (<em>trnF</em>_<em>trnL</em>, mean K2p = 0.0185, 428 bp; <em>trnL</em>_<em>trnT</em>, 0.0182, 633 bp; <em>trnS</em>_<em>trnG</em>, 0.0181, 799 bp), 36 IGS regions present a higher K2p values, and 15 of them are larger than 300 bp, and thus may be useful for phylogenetic studies. This analysis allowed detection of two relatively variable IGS sequences, corresponding to <em>ycf1</em>_<em>rps15</em> (mean K2p = 0.0355, aligned length = 470 bp) and <em>rpl32</em>_<em>ndhF</em> (0.0322, 486 bp) that were not detected in previous analysis (<a reveal-id="dsx006-F4" data-open="dsx006-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>).</p><p>Non-synonymous (Ka) and synonymous (Ks) nucleotide substitution rates were calculated for protein-coding sequences, as well as the Ka/Ks ratio (<span class="content-section supplementary-material"><a path-from-xml="sup17" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs10_dsx006.xls?Expires=2147483647&amp;Signature=GjAaLlO5bkwaKRiJLygPScRqJVc5A51vkKy0TKue4uhP~uLyik22f6qn6056T9BaOqg7Ni5WuuTAgcrbkNYuQ4xXnArw97H4hR570A2W0CPGpnp8oYpmLCPncvviZpTRVcedz8HHt-FHHbtgYYyxY-64eTUCp4VtLJVNgsH4CfOJ5XANtETxSm6Qaq7QCVjxE8OKIt9cIqmLKFj1I-yeNKDBz0oSN5xSfuo0xKeRv~Rs7-QHKVqJxpl5Ym8I6QMPid7Oz8KsA7ztD68aZ85AEXSlDdckdNFxqETMoqhkDeL7OlgPq13nuX8R82ztzC-jYLv6z~qUrLl5HXi0Ht5TQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S10</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup19" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs12_dsx006.xls?Expires=2147483647&amp;Signature=3hZH0tTs8L0HXAEL0VOF5dqbkpxbWcVubvFhHJ0kz-~8CAF7E-TvufrjjKnIKUAtHR7jYSKuSiA0Uu~UXa0gKyLgtXrjE-He4k7XZZZmcP7QE1nlON-xgZ2cjuorHTnJGH0Zb72ZtMiDoyQMM8KH6W5YHPTDiknaUHiVHuvkyQSSPSPh0rCcka2vzVOtDMHKq33hjRIal6GDx1hRLSYEpCWDWR~lvQ5N1oqOhSWn4T6thxkw~WAbEVdC17QIvrttDWkls77V-dwb7ZevIOxm574OIipQ8pggiv5nqmvHZHPAGNCW~znkrJ8U3Bp03Eno3O8UOdJNJh~MOMM68d2~Bw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S12</a></span>). The mean Ks among the five lupines studied ranged from 0 (<em>petG</em>, <em>petL</em>, <em>petN</em>, <em>psaJ</em>, <em>psbF</em>, <em>psbI</em>, <em>psbM</em>, <em>rpl23</em>, <em>rpl33</em>, <em>rps7</em>) to 0.05049 (<em>psbT</em>). All protein-encoding genes have a Ks value lower than 0.1. Similarly, the non-synonymous substitution rate (Ka) was lower than 0.025 for all genes. Finally, Ka/Ks ratios were calculated for each protein-coding region in order to determine the selective constraint acting on each gene. Almost all genes evolved under high purifying selective constraint (53 of the 77 genes have a Ka/Ks ratio lower than 0.2), except for six genes (<em>matK</em>, <em>rpoA</em>, <em>ycf2</em>, <em>rpoC2</em>, <em>accD</em> and <em>ycf1</em>) that show a ratio &gt; 0.5 (including three genes evolving almost neutrally (<em>matK</em>, <em>rpoA</em>, <em>ycf2</em>; <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs7_dsx006.ppt?Expires=2147483647&amp;Signature=mzo8ZN~MfW3vOBe9YmW2Td4yy61MJHW8ZI-hjnoy-FtlFEYeFS0oHb0Pxv71O8py0qCsmqopqa1NrS08sURnfLfe5bK1YI9TkFCOt0PWd8n1HRcTYbK7IRTxm-CAbAL4GftRN4UbAanq6-cJOknKK-8z0vPaUu4pEqqmoYOnU2Zs5KsHIuw48tiPpm3YHc72U9jtxBuWmrdjlXNXyFSX3Y3sRqc8LIFo19J7Vao2vsCAwrOhDFE~LU5U6lGH1K-pgZYep4-F8oVrzCvWxKI9SZ0UyQDZD0zvYqxP7SXuXpq4IBp5vwSfX2errj6hivU9MFLWTHwB7vaL~A12QATiaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Except for <em>ycf1</em> and <em>ycf2</em>, the other genes were not identified as neutrally evolving between legumes and <em>L. luteus</em>.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Comparison of Ka/Ks ratios obtained when considering only lupine species to the Ka/Ks ratios obtained when comparing <em>L. luteus</em><a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> to other legumes, revealed 14 genes that exhibit higher Ka/Ks ratios between lupines than between <em>L. luteus</em> and legumes. Among these genes, only six are significantly higher (<em>accD</em>, <em>ndhF</em>, <em>psbB</em>, <em>rbcL</em>, <em>rpoB</em> and <em>rsp2</em>; <a reveal-id="dsx006-F5" data-open="dsx006-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). However, detailed analysis of these genes (synonymous and non-synonymous substitution comparisons; and codon-based ML phylogenetic analyses; results not shown) did not reveal significant accelerated mutation rates at either synonymous or non-synonymous sites. Conversely, the <em>ycf4</em> gene and the flanking <em>cemA</em> and <em>accD</em> genes, which were found to be highly variable in the <em>Lathyrus</em> and <em>Desmodium</em> clades,<a reveal-id="dsx006-B12" data-open="dsx006-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> were more stable, lacking major rearrangements in <em>Lupinus</em>. These results highlight that fast-evolving regions may strongly differ among clades within a family. </p>                        <a id="106355020" scrollto-destination="106355020"></a>
<div id="dsx006-F5" data-id="dsx006-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f5.png?Expires=2147483647&Signature=i38-22PSaZ1Xgh~FPv29qIeZaR1Yc3W~jLEkD4mNlugHLKRDYVUubsnf~EVVx90XZHQG72ydOsC7g5E4jSoHnsN1Mh7JeO~Vdfc4BYsuiTXoM5uk0N-sTg81P-kgjN4X924rto2KFS-tW11jyIdZDgqjq8jupUAfr-Is4IFqcdbxCEfBx~Hn4oymfIvTrNfUX7JVXgTPardAxyJhgL7hf1Gc~zHRLrX9iDVjwBcjv~KWzAm9frvnq7rQhgsN9lYYOoTuTLFTi9jxueZu5HRLp5EZGvFCZ6ul~2xkd8RFSOwxT~ym1Ab2a~CvskUG1qfYGHOpUK6Tsec5kjzbh8IwFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mean Ka/Ks ratio values ± standard error between homologous regions of (i) the five Lupines L. albus, L. atlanticus, L. luteus, L. micranthus and L. princei (black circles) and (ii) L. luteus, Phaseolus vulgaris, Pisum sativum, Vigna radiata, Glycine max, Lathyrus sativus, Cicer arietinum, Trifolium subterraneum, Medicago truncatula, Lotus japonicus and Millettia pinnata (black squares15). The x-axis corresponds to the CDS regions. Asterisks represent a statistically significant difference (P.value: 0.05)." path-from-xml="dsx006f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355020/dsx006f5.png" path-from-xml="dsx006f5.png" target="_blank">View large</a><a section="106355020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f5.png?Expires=2147483647&Signature=CCQuNpgrNIPhYeqRNecb4tsG-9vo7Xz8NTdNxTKuHYfVo~GvIWT7dk7OCD4Ou~A6a4coB0yFwgcZPUwoJW94tQnCYkFLpEgY-oXFc8bq0JTzgfSk8qHT8wZ-8THRrE~Hps0EunbKj~Gf22i9ojJzMtZVW4NwcvGXaCNp-kzVO8dwIiHxJr8ZlRPY--i9jiNBMELrMzmtp4p5LBKgMkx7bPdoOTmFikR6Qs4aLw4yje0Df8hf7xOLUqM~QgvgFhLDvn5hrfrX1oqEP6V-UOG8nXwECPTqGQJJkiIwAhc-ZnKwANXMm4ViaH~iARsRUgiy4GhgcHybHi~7gbtr5zAAkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355020&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mean Ka/Ks ratio values ± standard error between homologous regions of (i) the five Lupines <em>L. albus</em>, <em>L. atlanticus</em>, <em>L. luteus</em>, <em>L. micranthus</em> and <em>L. princei</em> (black circles) and (ii) <em>L. luteus</em>, <em>Phaseolus vulgaris</em>, <em>Pisum sativum</em>, <em>Vigna radiata</em>, <em>Glycine max</em>, <em>Lathyrus sativus</em>, <em>Cicer arietinum</em>, <em>Trifolium subterraneum</em>, <em>Medicago truncatula</em>, <em>Lotus japonicus</em> and <em>Millettia pinnata</em> (black squares<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>). The <em>x</em>-axis corresponds to the CDS regions. Asterisks represent a statistically significant difference (<em>P.</em>value: 0.05).</p></div></div><div content-id="dsx006-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f5.png?Expires=2147483647&Signature=i38-22PSaZ1Xgh~FPv29qIeZaR1Yc3W~jLEkD4mNlugHLKRDYVUubsnf~EVVx90XZHQG72ydOsC7g5E4jSoHnsN1Mh7JeO~Vdfc4BYsuiTXoM5uk0N-sTg81P-kgjN4X924rto2KFS-tW11jyIdZDgqjq8jupUAfr-Is4IFqcdbxCEfBx~Hn4oymfIvTrNfUX7JVXgTPardAxyJhgL7hf1Gc~zHRLrX9iDVjwBcjv~KWzAm9frvnq7rQhgsN9lYYOoTuTLFTi9jxueZu5HRLp5EZGvFCZ6ul~2xkd8RFSOwxT~ym1Ab2a~CvskUG1qfYGHOpUK6Tsec5kjzbh8IwFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mean Ka/Ks ratio values ± standard error between homologous regions of (i) the five Lupines L. albus, L. atlanticus, L. luteus, L. micranthus and L. princei (black circles) and (ii) L. luteus, Phaseolus vulgaris, Pisum sativum, Vigna radiata, Glycine max, Lathyrus sativus, Cicer arietinum, Trifolium subterraneum, Medicago truncatula, Lotus japonicus and Millettia pinnata (black squares15). The x-axis corresponds to the CDS regions. Asterisks represent a statistically significant difference (P.value: 0.05)." path-from-xml="dsx006f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355020/dsx006f5.png" path-from-xml="dsx006f5.png" target="_blank">View large</a><a section="106355020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f5.png?Expires=2147483647&Signature=CCQuNpgrNIPhYeqRNecb4tsG-9vo7Xz8NTdNxTKuHYfVo~GvIWT7dk7OCD4Ou~A6a4coB0yFwgcZPUwoJW94tQnCYkFLpEgY-oXFc8bq0JTzgfSk8qHT8wZ-8THRrE~Hps0EunbKj~Gf22i9ojJzMtZVW4NwcvGXaCNp-kzVO8dwIiHxJr8ZlRPY--i9jiNBMELrMzmtp4p5LBKgMkx7bPdoOTmFikR6Qs4aLw4yje0Df8hf7xOLUqM~QgvgFhLDvn5hrfrX1oqEP6V-UOG8nXwECPTqGQJJkiIwAhc-ZnKwANXMm4ViaH~iARsRUgiy4GhgcHybHi~7gbtr5zAAkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355020&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mean Ka/Ks ratio values ± standard error between homologous regions of (i) the five Lupines <em>L. albus</em>, <em>L. atlanticus</em>, <em>L. luteus</em>, <em>L. micranthus</em> and <em>L. princei</em> (black circles) and (ii) <em>L. luteus</em>, <em>Phaseolus vulgaris</em>, <em>Pisum sativum</em>, <em>Vigna radiata</em>, <em>Glycine max</em>, <em>Lathyrus sativus</em>, <em>Cicer arietinum</em>, <em>Trifolium subterraneum</em>, <em>Medicago truncatula</em>, <em>Lotus japonicus</em> and <em>Millettia pinnata</em> (black squares<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>). The <em>x</em>-axis corresponds to the CDS regions. Asterisks represent a statistically significant difference (<em>P.</em>value: 0.05).</p></div></div>                            <h4 scrollto-destination=106355021 id="106355021" class="section-title" >3.3.3. <em>Lupinus</em> plastid sequences of phylogenetic utility</h4>
<p>To explore the putative phylogenetic utility of different chloroplast regions, potentially informative sites (Pi) were evaluated among lupines in: (i) complete chloroplast genomes, (ii) protein-coding sequences, (iii) intergenic spacers, (iv) introns and (v) in the two hypervariable regions (<em>psaA-ycf4</em> and <em>ycf1-rps15</em>) (<a reveal-id="dsx006-T2" data-open="dsx006-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Results revealed 666 Pi (among 2,874 variables sites) in the five aligned plastomes, which are distributed as follow: 45.3% of the Pi in IGS, 38.3% in CDS and only 11.4% in introns. The remaining five percent are located in tRNA and rRNA genes. The two hypervariable regions containing <em>psaA-ycf4</em> and <em>ycf1-rps15</em>, account for 8.7 and 14.3% of the total number of Pi, respectively. Molecular phylogenies were performed using either complete plastomes, or introns, or IGS or CDS, and revealed a similar topology (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs5_dsx006.ppt?Expires=2147483647&Signature=SDxDcGGSaK0Rmzs7qIWSZYHo8Et7R9jfHTcl9-apzZxfvHkgyP5N3AbboS-kLIkuKRqrGpfGfmW2uyYx9s82dgoBY-Q9p86DkcYvhN2s1Wl5JD097N7iQo7zeUT6nxgI-NgS8YBd8iTiyvl8Y48aF8tiqm-FBh9FVqHsaA5uONWNu~JM8g5uGaBRvBOuKNK35Vc0mP4mSkIPvY0we5ve6CM3sHdI9uR5rfn3pVKxbrvkrMLG2BtQHyMfzpqfEe5QOdXIgjv4ec0NUR1XLBqzOeqq0CUcky7liLNGEABf5icBjOQJ8nE-e7sL0Cd1~zwl7INLq4pyHuJwPskHbs-moA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), with the rough-seeded species (<em>L. atlanticus</em> and <em>L. princei</em>) in a well-supported clade (always with 100% of bootstrap support) clearly distinct from the smooth-seeded species <em>L. albus</em>, <em>L. luteus</em> and <em>L. micranthus</em>. Among the latter, <em>L. albus</em> was always the closest Mediterranean lupine to the rough-seeded group (86–87% of bootstrap value based on either IGS or complete plastome data). These results differ from previous phylogenies<a reveal-id="dsx006-B40" data-open="dsx006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx006-B41" data-open="dsx006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsx006-B44" data-open="dsx006-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx006-B45" data-open="dsx006-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> based on single or few genes (chloroplast and nuclear genes), which found <em>L. micranthus</em> to be the closest Mediterranean lupine to the rough-seeded species. In addition, the whole plastome phylogenies provide, for the first time, strong evidence (97–99% bootstrap support) of a common ancestor for <em>L. micranthus</em>, <em>L. albus</em> and the rough-seeded lupines, which are positioned as sister group to <em>L. luteus</em>. MP analyses (using PAUP4<a reveal-id="dsx006-B83" data-open="dsx006-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a>) of the five aligned lupine plastomes, with or without the 164 non-ambiguous indels (coded as 0 or 1 for the presence of a deletion or an insertion, respectively), led to the same results (not shown). To further investigate the phylogenetic utility of the most variable lupine regions identified, we amplified and sequenced five chloroplast regions (<em>accD</em>, two parts of the <em>ycf1</em> gene and the <em>ycf1</em>-<em>rps15</em> and <em>trnF<sup>GAA</sup>-trnL<sup>UAA</sup></em> 5ʹ-3ʹ intergenic spacers, length ranging from 800 to 2,000 bp) from 16 lupine species. Each matrix was subjected to ML analysis. After verifying the absence of incongruence between the trees obtained for the five different regions, a concatenated matrix of all regions was analysed, following the conditional-combination approach.<a reveal-id="dsx006-B84" data-open="dsx006-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a><sup>,</sup><em>L. villosus</em> and <em>L. anatolicus</em> were not considered in this analysis, as not all regions were amplified in these two species. The ML tree obtained from this latter matrix is presented in <a reveal-id="dsx006-F6" data-open="dsx006-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>. Despite low resolution of the basal nodes, the topology is consistent with an early divergence of the lupines into two main lineages: the OW lineage comprising all the smooth- and rough-seeded Mediterranean and African taxa, which includes the representative of the Floridian species (<em>L. diffusus</em>) and the NW lineage composed of all American taxa from diverse origins (except <em>L. diffusus</em>). Within these two main lineages, most clades are consistent with previous phylogenies<a reveal-id="dsx006-B40" data-open="dsx006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx006-B41" data-open="dsx006-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsx006-B44" data-open="dsx006-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx006-B45" data-open="dsx006-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and some of them present very high support using these cpDNA data, such as: (i) the OW rough-seeded species (<em>L. atlanticus</em>, <em>L. cosentinii</em>, <em>L. digitatus</em>, <em>L. pilosus</em> and <em>L. princei</em>) with a bootstrap value of 99%; (ii) the Mediterranean smooth-seeded lupines <em>L. luteus</em> and <em>L. hispanicus</em> subsp. <em>bicolor</em> (which together form the <em>lutei</em> section) with 100% bootstrap support, linked to <em>L. angustifolius</em> as sister group; (iii) the clade including the Texan lupines and the eastern South American species (<em>L. texensis</em>, <em>L. paraguariensis</em> and <em>L. gibertianus</em>) with 100% bootstrap support, (iv) and a clade (100% bootstrap support) corresponding to the Western American and Mexican species (<em>L. polyphyllus</em>, <em>L. mutabilis</em>, <em>L. mexicanus</em> and the undetermined lupine from Ecuador). Support for these clades is reinforced by synapomorphic indels (<a reveal-id="dsx006-F6" data-open="dsx006-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Within the OW lineage, the Mediterranean smooth-seeded species do not form a distinct clade and appear as paraphyletic to the rough-seeded group and the Floridian <em>L. diffusus</em>, <em>L. albus</em> and <em>L. micranthus</em> are placed (with 88% bootstrap support) as the closest Mediterranean smooth-seeded lupines to the rough-seeded species. In this phylogeny, <em>L. albus</em> is sister to <em>L. micranthus</em>, with moderate bootstrap support (75%) rather than to the rough-seeded lupins (with a bootstrap support of 86–87%), as observed in the whole plastome based phylogenies (see above and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs5_dsx006.ppt?Expires=2147483647&Signature=SDxDcGGSaK0Rmzs7qIWSZYHo8Et7R9jfHTcl9-apzZxfvHkgyP5N3AbboS-kLIkuKRqrGpfGfmW2uyYx9s82dgoBY-Q9p86DkcYvhN2s1Wl5JD097N7iQo7zeUT6nxgI-NgS8YBd8iTiyvl8Y48aF8tiqm-FBh9FVqHsaA5uONWNu~JM8g5uGaBRvBOuKNK35Vc0mP4mSkIPvY0we5ve6CM3sHdI9uR5rfn3pVKxbrvkrMLG2BtQHyMfzpqfEe5QOdXIgjv4ec0NUR1XLBqzOeqq0CUcky7liLNGEABf5icBjOQJ8nE-e7sL0Cd1~zwl7INLq4pyHuJwPskHbs-moA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). This incongruence might be explained by the low number of taxa or to the different sequence datasets analysed in these phylogenies (<a reveal-id="dsx006-F6" data-open="dsx006-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>, <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs5_dsx006.ppt?Expires=2147483647&Signature=SDxDcGGSaK0Rmzs7qIWSZYHo8Et7R9jfHTcl9-apzZxfvHkgyP5N3AbboS-kLIkuKRqrGpfGfmW2uyYx9s82dgoBY-Q9p86DkcYvhN2s1Wl5JD097N7iQo7zeUT6nxgI-NgS8YBd8iTiyvl8Y48aF8tiqm-FBh9FVqHsaA5uONWNu~JM8g5uGaBRvBOuKNK35Vc0mP4mSkIPvY0we5ve6CM3sHdI9uR5rfn3pVKxbrvkrMLG2BtQHyMfzpqfEe5QOdXIgjv4ec0NUR1XLBqzOeqq0CUcky7liLNGEABf5icBjOQJ8nE-e7sL0Cd1~zwl7INLq4pyHuJwPskHbs-moA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Further investigation to resolve such phylogenetic uncertainty is needed. Compared with previous <em>Lupinus</em> phylogenies, based on chloroplast (<em>matK</em>, <em>rbcL</em>, <em>trnL</em> intron, <em>trnL-trnF</em>, <em>trnS-trnG</em>, <em>trnT-trnL</em>) or nuclear sequences (<em>LEGCYC1A</em>, <em>LEGCYC1B</em>, ITS1 + 2, <em>GPAT1</em>, <em>GPAT2</em>, <em>SymRK</em>, ETS),<a reveal-id="dsx006-B39" data-open="dsx006-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx006-B40" data-open="dsx006-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx006-B42" data-open="dsx006-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsx006-B44" data-open="dsx006-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx006-B85" data-open="dsx006-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> our five hypervariable chloroplast regions may not have revealed novel relationships but strongly reinforced support for some known clades (such as the West and the East American groups, or the OW rough-seeded section). Moreover, they provided additional and significant data supporting the singular Floridian unifoliolate lupines (represented here by <em>L. diffusus</em>), for which phylogenetic placement has always been questionable, as close relative to the OW lupines rather than to the NW ones (at least from the maternally inherited plastome). Finally, we showed the phylogenetic utility of these two identified regions but the consideration of a higher number of lupines and related species will allow for optimal exploitation of their potential to inform these phylogenies, and improve our knowledge of the evolutionary history of lupine and closely related Genistoid clades that are poorly investigated. <div content-id="dsx006-T2" class="table-modal"><div class="table-wrap"><div id="dsx006-T2" data-id="dsx006-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Number of potentially informative sites in complete plastomes (cp), protein-coding sequences (CDS), intergenic spacers (IGS), introns as well as in the two hypervariable regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Regions </th><th>Complete cp </th><th>CDS </th><th>IGS </th><th>Introns </th><th>tRNA-rRNA </th><th>psaA-ycf4 </th><th>ycf1-rps15 </th></tr></thead><tbody><tr><td>Number of Pi </td><td>666 </td><td>255 </td><td>302 </td><td>76 </td><td>33 </td><td>58 </td><td>95 </td></tr><tr><td>% of total Pi </td><td>100 </td><td>38.3 </td><td>45.3 </td><td>11.4 </td><td>5 </td><td>8.7 </td><td>14.3 </td></tr><tr><td>Aligned length </td><td>153,462 </td><td>76,518 </td><td>37,948 </td><td>16,093 </td><td>22,903 </td><td>11,534 </td><td>6,092 </td></tr><tr><td>% of Pi by region </td><td>0.4 </td><td>0.3 </td><td>0.7 </td><td>0.5 </td><td>0.1 </td><td>0.5 </td><td>1.6 </td></tr></tbody></table></div><div content-id="dsx006-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Regions </th><th>Complete cp </th><th>CDS </th><th>IGS </th><th>Introns </th><th>tRNA-rRNA </th><th>psaA-ycf4 </th><th>ycf1-rps15 </th></tr></thead><tbody><tr><td>Number of Pi </td><td>666 </td><td>255 </td><td>302 </td><td>76 </td><td>33 </td><td>58 </td><td>95 </td></tr><tr><td>% of total Pi </td><td>100 </td><td>38.3 </td><td>45.3 </td><td>11.4 </td><td>5 </td><td>8.7 </td><td>14.3 </td></tr><tr><td>Aligned length </td><td>153,462 </td><td>76,518 </td><td>37,948 </td><td>16,093 </td><td>22,903 </td><td>11,534 </td><td>6,092 </td></tr><tr><td>% of Pi by region </td><td>0.4 </td><td>0.3 </td><td>0.7 </td><td>0.5 </td><td>0.1 </td><td>0.5 </td><td>1.6 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx006-T2" data-id="dsx006-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Number of potentially informative sites in complete plastomes (cp), protein-coding sequences (CDS), intergenic spacers (IGS), introns as well as in the two hypervariable regions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Regions </th><th>Complete cp </th><th>CDS </th><th>IGS </th><th>Introns </th><th>tRNA-rRNA </th><th>psaA-ycf4 </th><th>ycf1-rps15 </th></tr></thead><tbody><tr><td>Number of Pi </td><td>666 </td><td>255 </td><td>302 </td><td>76 </td><td>33 </td><td>58 </td><td>95 </td></tr><tr><td>% of total Pi </td><td>100 </td><td>38.3 </td><td>45.3 </td><td>11.4 </td><td>5 </td><td>8.7 </td><td>14.3 </td></tr><tr><td>Aligned length </td><td>153,462 </td><td>76,518 </td><td>37,948 </td><td>16,093 </td><td>22,903 </td><td>11,534 </td><td>6,092 </td></tr><tr><td>% of Pi by region </td><td>0.4 </td><td>0.3 </td><td>0.7 </td><td>0.5 </td><td>0.1 </td><td>0.5 </td><td>1.6 </td></tr></tbody></table></div><div content-id="dsx006-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Regions </th><th>Complete cp </th><th>CDS </th><th>IGS </th><th>Introns </th><th>tRNA-rRNA </th><th>psaA-ycf4 </th><th>ycf1-rps15 </th></tr></thead><tbody><tr><td>Number of Pi </td><td>666 </td><td>255 </td><td>302 </td><td>76 </td><td>33 </td><td>58 </td><td>95 </td></tr><tr><td>% of total Pi </td><td>100 </td><td>38.3 </td><td>45.3 </td><td>11.4 </td><td>5 </td><td>8.7 </td><td>14.3 </td></tr><tr><td>Aligned length </td><td>153,462 </td><td>76,518 </td><td>37,948 </td><td>16,093 </td><td>22,903 </td><td>11,534 </td><td>6,092 </td></tr><tr><td>% of Pi by region </td><td>0.4 </td><td>0.3 </td><td>0.7 </td><td>0.5 </td><td>0.1 </td><td>0.5 </td><td>1.6 </td></tr></tbody></table></div></div></p>                        <a id="106355023" scrollto-destination="106355023"></a>
<div id="dsx006-F6" data-id="dsx006-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f6.png?Expires=2147483647&Signature=tLJGtdwIfGsWDfe0ueZPnQ0vnyMYL4YjTBbycNFRoGqLtezboKdsItToZVIgsnwduPzkyl67mfba5HbZalFO70pRh7G6eTrGvdh5WLdHb0FUHV39VZYcPtsA4~wmZAKLtz5SKWQbTA2WRcAbDCbWao5w6WTFRcXYdnxBkWNPa9R5luM1iC5GWK5OVmOwTIMV-x-lHSqBaPSE2BAuElRKX06Bel6XGddXd0iEY~hCujLUe1vd2tQNfF78zohD9ftzo-~xoLPYdeek2pGRSCOgA6rsO8QCfO38uLsmOqmv0GVD2xaUQ7e0Z78VT7zfH0zBIBGcBlM4bEuP0Jw5qJXVNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Maximum likelihood unrooted tree (General Time Reversible model, rates Gamma distributed with Invariant Sites, 1,000 bootstraps) of concatenated regions (part of accD and ycf1 genes, ycf1-rps15 IGS and trnF-trnL regions). Bootstrap support values are indicated above branches. Grey diamonds represent indels specific to a node. The numbers above the diamonds indicate the number of additional indels supporting the node, with corresponding indel sizes (in bp) between brackets. The Old Wold (OW) and New World (NW) ancestral nodes are indicated on the tree by solid black points." path-from-xml="dsx006f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355023/dsx006f6.png" path-from-xml="dsx006f6.png" target="_blank">View large</a><a section="106355023" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f6.png?Expires=2147483647&Signature=stVweAewpX8gVy7fbDAgTLnhLVqXMwg8AWDF9Tuwo3lA~mW84m8-QnhPK4sIaM760a39ftzEIzxvZFUC4pIO2Q2vXS9RfEloK9I6P4iQ4To24z2EOdZH~GcfWvbqKjz3P2BzGDPvP4u4HCkM~vLpS2MpPLDA0SVCfyoCaxU0U63F98c4AQXnP0lQJJIZtC1bir7IiLyKnCiF6CYOXx7yDgk8RA1kyqXsAHjiNsfQA6gQ3nK73Czr6C5iA1ODw7Y3xk9nAKmCTY7I0y2vRJmCguSoCvO7~oIyTpxNnlolNAuCp08ejGc67eIfolXLweBbh9P02U06YkstYGiz3-wmZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355023&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Maximum likelihood unrooted tree (General Time Reversible model, rates Gamma distributed with Invariant Sites, 1,000 bootstraps) of concatenated regions (part of <em>accD</em> and <em>ycf1</em> genes, <em>ycf1-rps15</em> IGS and <em>trnF</em>-<em>trnL</em> regions). Bootstrap support values are indicated above branches. Grey diamonds represent indels specific to a node. The numbers above the diamonds indicate the number of additional indels supporting the node, with corresponding indel sizes (in bp) between brackets. The Old Wold (OW) and New World (NW) ancestral nodes are indicated on the tree by solid black points.</p></div></div><div content-id="dsx006-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f6.png?Expires=2147483647&Signature=tLJGtdwIfGsWDfe0ueZPnQ0vnyMYL4YjTBbycNFRoGqLtezboKdsItToZVIgsnwduPzkyl67mfba5HbZalFO70pRh7G6eTrGvdh5WLdHb0FUHV39VZYcPtsA4~wmZAKLtz5SKWQbTA2WRcAbDCbWao5w6WTFRcXYdnxBkWNPa9R5luM1iC5GWK5OVmOwTIMV-x-lHSqBaPSE2BAuElRKX06Bel6XGddXd0iEY~hCujLUe1vd2tQNfF78zohD9ftzo-~xoLPYdeek2pGRSCOgA6rsO8QCfO38uLsmOqmv0GVD2xaUQ7e0Z78VT7zfH0zBIBGcBlM4bEuP0Jw5qJXVNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Maximum likelihood unrooted tree (General Time Reversible model, rates Gamma distributed with Invariant Sites, 1,000 bootstraps) of concatenated regions (part of accD and ycf1 genes, ycf1-rps15 IGS and trnF-trnL regions). Bootstrap support values are indicated above branches. Grey diamonds represent indels specific to a node. The numbers above the diamonds indicate the number of additional indels supporting the node, with corresponding indel sizes (in bp) between brackets. The Old Wold (OW) and New World (NW) ancestral nodes are indicated on the tree by solid black points." path-from-xml="dsx006f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355023/dsx006f6.png" path-from-xml="dsx006f6.png" target="_blank">View large</a><a section="106355023" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f6.png?Expires=2147483647&Signature=stVweAewpX8gVy7fbDAgTLnhLVqXMwg8AWDF9Tuwo3lA~mW84m8-QnhPK4sIaM760a39ftzEIzxvZFUC4pIO2Q2vXS9RfEloK9I6P4iQ4To24z2EOdZH~GcfWvbqKjz3P2BzGDPvP4u4HCkM~vLpS2MpPLDA0SVCfyoCaxU0U63F98c4AQXnP0lQJJIZtC1bir7IiLyKnCiF6CYOXx7yDgk8RA1kyqXsAHjiNsfQA6gQ3nK73Czr6C5iA1ODw7Y3xk9nAKmCTY7I0y2vRJmCguSoCvO7~oIyTpxNnlolNAuCp08ejGc67eIfolXLweBbh9P02U06YkstYGiz3-wmZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355023&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Maximum likelihood unrooted tree (General Time Reversible model, rates Gamma distributed with Invariant Sites, 1,000 bootstraps) of concatenated regions (part of <em>accD</em> and <em>ycf1</em> genes, <em>ycf1-rps15</em> IGS and <em>trnF</em>-<em>trnL</em> regions). Bootstrap support values are indicated above branches. Grey diamonds represent indels specific to a node. The numbers above the diamonds indicate the number of additional indels supporting the node, with corresponding indel sizes (in bp) between brackets. The Old Wold (OW) and New World (NW) ancestral nodes are indicated on the tree by solid black points.</p></div></div>                            <h4 scrollto-destination=106355024 id="106355024" class="section-title" >3.3.4. Repeated sequences</h4>
<p>Repeated sequences are known to play a major role in genome evolution. In chloroplast genomes repeats are involved in various structural rearrangements, such as inversions, insertions or deletions. These structural modifications sometimes lead to pseudogenization or duplication as well as to plastome expansion or contraction.<a reveal-id="dsx006-B10" data-open="dsx006-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx006-B19" data-open="dsx006-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx006-B86" data-open="dsx006-B86" class="link link-ref link-reveal xref-bibr"><sup>86–88</sup></a> The most striking example of the involvement of repeat sequences in genome size change was observed in Geraniaceae, where the plastome size varies from 128,787 bp to 217,942 bp in <em>Monsonia speciosa</em> and <em>Pelargonium hortorum</em> species, respectively.<a reveal-id="dsx006-B9" data-open="dsx006-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In Fabaceae, repeat sequences were also shown to be involved in LSC extension in Mimosoids<a reveal-id="dsx006-B14" data-open="dsx006-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> or related to structural rearrangements, as in <em>Trifolium subterraneum</em> that presents numerous reorganization events and a very high percentage of repeated elements (20% of its genome). Recently, a 29 bp IR in the <em>trnS<sup>GGA</sup></em> and <em>trnS<sup>GCU</sup></em> was found to be at the origin of a large 36 kb inversion discovered in <em>L. luteus</em> (and Genisteae), through a flip-flop recombination event.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> This inversion was regarded as a new powerful clade marker for most Genistoids in legumes and our study confirmed the presence of this inversion in the four additional <em>Lupinus</em> plastomes investigated here. Since these short inverted repeats (separated by at least 30 kb) are present in almost all known Fabaceae plastomes,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> it has been underlined that such inversion events could have occurred and could occur again elsewhere via the same mechanism.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Interestingly,<a reveal-id="dsx006-B16" data-open="dsx006-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> recently discovered an independent 39 kb inversion at exactly the same location in <em>Robinia pseudoacacia</em> among 13 taxa investigated. This result confirms the potential of such repeats in plastome dynamics, and demonstrates that even rarely occurring, large inversions might result from independent events in distantly related taxa, such as here in Robinioids and Genistoids, biasing their phylogenetic utility. Despite the homoplasious nature of these inversions, such remarkable parallel inversions could be cautiously used as clade evolutionary markers in each of the affected lineages. Because of the importance of repeated elements in plastome evolution (particularly in Fabaceae), we investigated the type and number of repeats present in each <em>Lupinus</em> plastome using REPuter.<a reveal-id="dsx006-B65" data-open="dsx006-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> A total of 142 repeats were identified across the five <em>Lupinus</em> species. These repeats, which are relatively well distributed along the plastome sequences (<a reveal-id="dsx006-F7" data-open="dsx006-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>), were divided into three categories, (i) palindromes (60 repeats), (ii) forward repeats (45 repeats) and (iii) reverse repeats (37 repeats). Although all five chloroplast genomes show a relatively similar number of repeats (24 in <em>L. albus</em> to 33 in <em>L. princei</em>) and confirm previous results obtained by<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>, we identified three, four, nine and six repeats specific to <em>L. atlanticus, L. albus</em>, <em>L. micranthus</em> and <em>L. luteus</em>, respectively. </p>                        <a id="106355026" scrollto-destination="106355026"></a>
<div id="dsx006-F7" data-id="dsx006-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f7.png?Expires=2147483647&Signature=V9d9WldLfoavpaKH~FthZF0zJ-T~-rDff1gH-ePdpqBq9i10juwrllzJcLCU0OTi6RFQbB0r1HuqAcUeY0gefqOHQ-BR9ZUtQImAU9em1U2j2vAJW74QOhUr2stfs1wL0vhbSs6m3c5q9JT8s3EL5qyW59AVa44u24uLQRd4sZMtzIwVIR9Zb88DY8gF2-rOQWAA7mZPQZsztfpSrv3EnqqmpkZHhOlCqaVM0KKFyFjk9KNDFl2-7ofqs1NFiMw2Hqmrefo6Fxc1wDiRX51AjXzEnL7wMzxUwhba2USMnx6nIqZqH09NrkQI7-P2hD7sN124BFCCVKfic7oF39hCVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of repeated sequences and potentially informative SNPs in lupine plastomes. From the outer to the most inner circles. First circle: representation of genes content; second circle: potentially informative sites; third circle: SSRs (circles correspond to mononucleotides, squares stand for dinucleotides and triangles represent trinucleotides); fourth circle: direct repeat interspaced by &lt; 3 kb; fifth circle: inverted repeat interspaced by &lt; 3 kb; sixth circle: palindromic repeats. In the middle, full and dotted lines represent direct and inverted dispersed repeats (separated by &gt; 3 kb), respectively. The endpoints of the 50-kb inversion, specific to the Papilionoid legumes20–22 and of the 36-kb inversion, specific to the Genistoid clade,15 are represented arrows." path-from-xml="dsx006f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355026/dsx006f7.png" path-from-xml="dsx006f7.png" target="_blank">View large</a><a section="106355026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f7.png?Expires=2147483647&Signature=D8i~Muy7AbrKZ0LdE6CDs0wUhKO2nUf7R3YkONfr66faaaOocY8jlo9lA66nl46ZDONdGALQmbPbyJZhNvJKnhO6Si4jEnoowIl8DxJJwKV0hVMpDk8VzEYWjsKWcCXGAdlYohI5~f~ZnD8TF2~9aOqYqbgLWKsWNGWS1a7uJFDpmjLP18cD6Gn4hphGXMP26jPTentGCc3j7dyQognrQ3vPy7xNbxd-~Kt4FxKvNBItSJ4vxm6gvWXllWqaGXEM6uO7mHYlQH9fdV0u8mhgbiKLZBngI3rR4KHRDFd1c2HgiSM4XLPua9u9xElVkCffx5o3PPzYh-JBCEotBO2qow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355026&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of repeated sequences and potentially informative SNPs in lupine plastomes. From the outer to the most inner circles. First circle: representation of genes content; second circle: potentially informative sites; third circle: SSRs (circles correspond to mononucleotides, squares stand for dinucleotides and triangles represent trinucleotides); fourth circle: direct repeat interspaced by &lt; 3 kb; fifth circle: inverted repeat interspaced by &lt; 3 kb; sixth circle: palindromic repeats. In the middle, full and dotted lines represent direct and inverted dispersed repeats (separated by &gt; 3 kb), respectively. The endpoints of the 50-kb inversion, specific to the Papilionoid legumes<a reveal-id="dsx006-B20" data-open="dsx006-B20" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> and of the 36-kb inversion, specific to the Genistoid clade,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> are represented arrows.</p></div></div><div content-id="dsx006-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/m_dsx006f7.png?Expires=2147483647&Signature=V9d9WldLfoavpaKH~FthZF0zJ-T~-rDff1gH-ePdpqBq9i10juwrllzJcLCU0OTi6RFQbB0r1HuqAcUeY0gefqOHQ-BR9ZUtQImAU9em1U2j2vAJW74QOhUr2stfs1wL0vhbSs6m3c5q9JT8s3EL5qyW59AVa44u24uLQRd4sZMtzIwVIR9Zb88DY8gF2-rOQWAA7mZPQZsztfpSrv3EnqqmpkZHhOlCqaVM0KKFyFjk9KNDFl2-7ofqs1NFiMw2Hqmrefo6Fxc1wDiRX51AjXzEnL7wMzxUwhba2USMnx6nIqZqH09NrkQI7-P2hD7sN124BFCCVKfic7oF39hCVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of repeated sequences and potentially informative SNPs in lupine plastomes. From the outer to the most inner circles. First circle: representation of genes content; second circle: potentially informative sites; third circle: SSRs (circles correspond to mononucleotides, squares stand for dinucleotides and triangles represent trinucleotides); fourth circle: direct repeat interspaced by &lt; 3 kb; fifth circle: inverted repeat interspaced by &lt; 3 kb; sixth circle: palindromic repeats. In the middle, full and dotted lines represent direct and inverted dispersed repeats (separated by &gt; 3 kb), respectively. The endpoints of the 50-kb inversion, specific to the Papilionoid legumes20–22 and of the 36-kb inversion, specific to the Genistoid clade,15 are represented arrows." path-from-xml="dsx006f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355026/dsx006f7.png" path-from-xml="dsx006f7.png" target="_blank">View large</a><a section="106355026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/dsx006f7.png?Expires=2147483647&Signature=D8i~Muy7AbrKZ0LdE6CDs0wUhKO2nUf7R3YkONfr66faaaOocY8jlo9lA66nl46ZDONdGALQmbPbyJZhNvJKnhO6Si4jEnoowIl8DxJJwKV0hVMpDk8VzEYWjsKWcCXGAdlYohI5~f~ZnD8TF2~9aOqYqbgLWKsWNGWS1a7uJFDpmjLP18cD6Gn4hphGXMP26jPTentGCc3j7dyQognrQ3vPy7xNbxd-~Kt4FxKvNBItSJ4vxm6gvWXllWqaGXEM6uO7mHYlQH9fdV0u8mhgbiKLZBngI3rR4KHRDFd1c2HgiSM4XLPua9u9xElVkCffx5o3PPzYh-JBCEotBO2qow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355026&ar=3056574&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx006f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of repeated sequences and potentially informative SNPs in lupine plastomes. From the outer to the most inner circles. First circle: representation of genes content; second circle: potentially informative sites; third circle: SSRs (circles correspond to mononucleotides, squares stand for dinucleotides and triangles represent trinucleotides); fourth circle: direct repeat interspaced by &lt; 3 kb; fifth circle: inverted repeat interspaced by &lt; 3 kb; sixth circle: palindromic repeats. In the middle, full and dotted lines represent direct and inverted dispersed repeats (separated by &gt; 3 kb), respectively. The endpoints of the 50-kb inversion, specific to the Papilionoid legumes<a reveal-id="dsx006-B20" data-open="dsx006-B20" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> and of the 36-kb inversion, specific to the Genistoid clade,<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> are represented arrows.</p></div></div><p>The number of repeats found across lupine plastomes is much lower than the number observed in some other legumes, such as in <em>T. subterraneum</em>, <em>M. truncatula</em>, <em>G. max</em>, <em>Pisum sativum</em>, or <em>Lathyrus sativus</em> and <em>Cicer arietinum</em> which contain ∼500, 190, 100, 74, 78 and 75 repeats of a similar size, respectively.<a reveal-id="dsx006-B17" data-open="dsx006-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx006-B18" data-open="dsx006-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx006-B66" data-open="dsx006-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> It is thus not surprising to observe a more conserved genome size, gene content and less structural rearrangement in the <em>Lupinus</em> genus. Distribution of these repeats (mononucleotides, dispersed and palindromic) along plastomes was characterized, revealing that around 70% of the repeats are in the LSC, whereas 20% and 10% are localized in the SSC and IR, respectively. Within the plastome, most of the repeats are situated in the highly variable regions of the LSC, in the <em>rps12-trnV<sup>GAC</sup></em> intergenic spacer of the IR and in the intron of <em>ndhA</em> in the SSC. The five lupine plastomes exhibit a similar pattern of repeat distribution, with more than half of the repeats localized in the non-coding regions, around 30% are in protein-coding sequences and around 15% are in introns. Only two dispersed repeats (shared by all lupines) were found in tRNA genes, including the inverted repeat found to be involved in the 36-kb inversion.<a reveal-id="dsx006-B15" data-open="dsx006-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> While performing these analyses, we also paid particular attention to Simple Sequence Repeats (SSRs or microsatellites) that are particularly interesting in a wide range of genetic studies in population genetics, plant evolution and domestication, or for the estimation of gene and pollen flow.<a reveal-id="dsx006-B89" data-open="dsx006-B89" class="link link-ref link-reveal xref-bibr"><sup>89–96</sup></a> We found between 37 (<em>L. princei</em>) and 51 (<em>L. luteus</em>) microsatellites longer than 12 bp in lupine plastomes, with mononucleotide repeats representing between 70 and 80% of these microsatellites, compared with only 16–30% and 0–5%, of di- and tri-nucleotide SSRs, respectively. The <em>ycf1</em> gene, which corresponds to the most variable lupine plastome regions, is the richest region in SSRs, with ∼20% of the total microsatellites. In comparison, the second hypervariable region (<em>psaA-ycf4</em>) presents only zero to five percent of SSRs (<span class="content-section supplementary-material"><a path-from-xml="sup20" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs13_dsx006.xls?Expires=2147483647&amp;Signature=mJeDIiobjofGUmlPDYXpwA81neWyRPdt56mwhHrUXz9N1B5rwvfUWyOw3LxG-bqk0TMNjXWN2g9hOLd6UY16a-3gQb6sLwa3J-RFjnTH64J3VKDtcCx5-yzFAfCTkbh72gFWSIe4hqKe3OVGx9LIXgQWj7veHKLrzbdYsdxyK2C2Knyyo~nyHgVKSHTHGzPB8LP3uzwClsdOz1budEf33pENlCyV3RkZkvUe6YJ6Lts5TRbeH42bURng1HM-EJ6r~UPbd9ZYl322BqsFxZ7IS5bIqj9cfGuLswewE753bKtRBE-HyzbimS-Vk2BQIpEolldol0a74Xxzp57yUwpb1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). Among the 213 SSRs identified within the five plastomes, nine are perfectly shared by all species, two additional shared SSRs vary in size, whereas the others are species or group-specific and thus represent potentially useful markers. Taken together, the various kinds of markers revealed from this study (Single Nucleotide Polymorphisms or SNPs, indels, repeats and inversions) represent important resources of genetic/genomic markers with which to deepen our investigations of <em>Lupinus</em> and its Genistoid allies, and for comparative analyses in legumes.</p>                            <h2 scrollto-destination=106355028 id="106355028" class="section-title" >4. Conclusion and further perspectives</h2>
<p>In this work, four additional lupine chloroplast genomes were sequenced, assembled and analysed at different levels. This study provides novel insights into the chloroplast genome evolutionary dynamics in the poorly studied Genistoid clade. Our results revealed highly conserved structure and gene content among the five <em>Lupinus</em> species with the exception of the <em>rps16</em> gene, which is very likely pseudogenized in the different lupine species investigated. Detailed surveys of mitochondrion, nuclear and chloroplast genomes available to date revealed that <em>rps16</em> gene is absent from all plant mitochondria, strongly suggesting that this gene was functionally replaced by the nuclear <em>rps16</em> gene since the divergence of plants. Compared with the mitochondrion, the chloroplast <em>rps16</em> gene is still present in many plants but has lost its functionality many times independently. Analysis of the evolution rate of functional <em>rps16</em> genes present in both the nuclear and chloroplast genomes of some representative angiosperm species revealed that these genes are both under purifying selection, whereas a relaxed selective constraint was expected for the chloroplast copy. Comparative analyses of lupine plastomes also enabled identification of two hypervariable regions: <em>psaA-ycf4</em> (11.5 kb) and <em>ycf1-rps15</em> (6.5 kb). We demonstrate that these regions, which contain a high number of potentially informative sites and the highest number of SSRs, were highly consistent with, and reinforced the support for, previous phylogenies. The analyses of the short repeated sequences present in <em>Lupinus</em> plastomes allowed us to identify different types of chloroplast markers that could be very useful, low cost and easy to use for studying genetic diversity and evolutionary history of lupines or Genistoids.</p>                            <h2 scrollto-destination=106355030 id="106355030" class="backacknowledgements-title" >Acknowledgments</h2>
<p>The authors thank all institutions for kindly providing seed samples: Institut National d’Agronomie, Alger, Algérie; Institut National de la Recherche Agronomique, Dijon, France; Western Australian Department of Agriculture, Baron-Hay Court, South Perth, Western Australia; EMBRAPA-CENARGEN, Brasilia, Brasil; United States Department of Agriculture ARS, Washington DC, USA; University of Edmonton, Alberta, Canada.</p><p>The authors also thank M.T. Schifino-Wittman (Federal University of South Rio Grande, Brasil); D. Jones (University of Florida, USA); M.T. Misset (Université Rennes 1, France); R. Pasquet (IRD, Montpellier, France); J. Pascual (previously IMIA, Spain); M. Sahnoune (Université de Bejaia, Algeria). The authors greatly thank Thierry Fontaine-Breton and Fouad Nassur for taking care of the plant material. We also thank Christina Richards (Univ. of Florida, USA) and Jeremy Timmis (Univ. of Adelaide, Australia) for critically reading the manuscript as well as the two anonymous reviewers for their careful evaluation of our manuscript and their helpful comments and suggestions.</p>                            <h3 scrollto-destination=106355033 id="106355033" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=106355035 id="106355035" class="section-title" >Accession numbers</h3>
<p>KX787895, KX787896, KX787897, KX787898, KX787899, KX787900, KX787901, KX787902, KX787903, KX787904, KX787905, KX787906, KX787907, KX787908, KX787909, KX787910, KX147685, KX147686, KX147687, KX147688, KX147689, KX147690, KX147691, KX147692, KX147693, KX147694, KX147695, KX147696, KX147697, KX147698, KX147699, KX147700, KX147701, KX147702, KX147703, KX147704, KX147705, KX147706, KX147707, KX147708, KX147709, KX147710, KX147711, KX147712, KX147713, KX147714, KX147715, KX147716, KX147717, KX147718, KX147719, KX147720, KX147721, KX147722, KX147723, KX147724, KX147725, KX147726, KX147727, KX147728, KX147729, KX147730, KX147731, KX147732, KX147733, KX147734, KX147735, KX147736, KX147737, KX147738, KX147739, KX147740, KX147741, KX147742, KX147743, KX147744, KX147745, KX147746, KX147747, KX147748, KX147749, KX147750, KX147751, KX147752, KX147753, KU726826, KU726827, KU726828, KU726829</p>                            <h3 scrollto-destination=106355037 id="106355037" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs1_dsx006.ppt?Expires=2147483647&amp;Signature=NUbYoeFLPQhq3ZfnoTeIZrc3s0teLmtrnCzvV6bqmW1q2PWoidgMR2Ayok94SKOfmo8ui8CJ2ot91PwlnKZlngB5y13bOK7HX5nT7GB0wmPXj6IWCr7d8z6SL7itC-glhwtgyw3t4kgDStVTmAov6WtRvykmKe6JGQXwOj4yQy8k-rnOKdgm1HPh~UcXu6LhqvDAw0ESHXoCQLE1lqaOO05J0RLHuYBVRIPAt0K12yg0HlY2t5mXzPH~5xMxAGkrcuvInrpkuWkdnlBB2q-CfpZSyG7y-I2Otb~OeVfIBIsRe7vz6b1rf7mVDjlRnBgKeau9zP76pG6oETOStt9b0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S1</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs7_dsx006.ppt?Expires=2147483647&amp;Signature=mzo8ZN~MfW3vOBe9YmW2Td4yy61MJHW8ZI-hjnoy-FtlFEYeFS0oHb0Pxv71O8py0qCsmqopqa1NrS08sURnfLfe5bK1YI9TkFCOt0PWd8n1HRcTYbK7IRTxm-CAbAL4GftRN4UbAanq6-cJOknKK-8z0vPaUu4pEqqmoYOnU2Zs5KsHIuw48tiPpm3YHc72U9jtxBuWmrdjlXNXyFSX3Y3sRqc8LIFo19J7Vao2vsCAwrOhDFE~LU5U6lGH1K-pgZYep4-F8oVrzCvWxKI9SZ0UyQDZD0zvYqxP7SXuXpq4IBp5vwSfX2errj6hivU9MFLWTHwB7vaL~A12QATiaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs1_dsx006.xls?Expires=2147483647&amp;Signature=sCMnLQBkGQ7LtF-novZzfiqUEbb-ONEANNCXRG9L9CM3lFkU7RJ9oP0NrXR5wlNtylK0zdoa5Aj5gHq4Nte53Rt7W~4Y9XoHf9sqig3tcmIdoFw4uGguD7nb4ZW16pd4oRlBM35Cb3Eb6OJdgPJD1LzkqPpduPNnd6IuHSk9DxxjNF3QMwuoZnXIHo5p8RjyIQ9t33Kv0UaNie0FBkF1M6nSiwriU17kv63CUBuIdlCsYAko2ZXuUvg1jqIqdD0wtMjXa6qcte8KsYStysMVWRNV7LmRW7wUtwmteFaSl4bXchHjmRF9QZZ~ya5YfnN-BmAxCE89cQSSR2ZQuD7Ebg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Tables S1</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup20" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs13_dsx006.xls?Expires=2147483647&amp;Signature=mJeDIiobjofGUmlPDYXpwA81neWyRPdt56mwhHrUXz9N1B5rwvfUWyOw3LxG-bqk0TMNjXWN2g9hOLd6UY16a-3gQb6sLwa3J-RFjnTH64J3VKDtcCx5-yzFAfCTkbh72gFWSIe4hqKe3OVGx9LIXgQWj7veHKLrzbdYsdxyK2C2Knyyo~nyHgVKSHTHGzPB8LP3uzwClsdOz1budEf33pENlCyV3RkZkvUe6YJ6Lts5TRbeH42bURng1HM-EJ6r~UPbd9ZYl322BqsFxZ7IS5bIqj9cfGuLswewE753bKtRBE-HyzbimS-Vk2BQIpEolldol0a74Xxzp57yUwpb1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S13</a></span> are available at <em>DNARES</em> Online.</p>                            <h3 scrollto-destination=106355039 id="106355039" class="section-title" >Funding</h3>
<p>Jean Keller was supported by a doctoral research grant from the University of Rennes 1 - French Ministry of Higher Education and Research. Mathieu Rousseau-Gueutin was supported by the European Union Seventh Framework Programme (FP7-CIG-2013-2017; Grant no. 333709). This work benefited from the International Associated Laboratory “Ecological Genomics of Polyploidy” supported by CNRS (INEE, UMR CNRS 6553 Ecobio), University of Rennes 1, Iowa State University (Ames, USA).</p>                            <h2 scrollto-destination=106355041 id="106355041" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx006-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cronk</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Ojeda</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Pennington</div> <div class="given-names">R.T.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Legume comparative genomics: progress in phylogenetics and phylogenomics</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">9</div>, <div class="fpage">99</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20comparative%20genomics%3A%20progress%20in%20phylogenetics%20and%20phylogenomics&amp;author=Q.%20Cronk&amp;author=I.%20Ojeda&amp;author=R.T.%20Pennington&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=9&amp;pages=99-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2006.01.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2006.01.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2006.01.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16480916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lewis</div>
               <div class="given-names">G.P.</div>
            </div>
          </div>
          <div class="year">2005</div>, <div class="source">Legumes of the World</div> . 
<div class="publisher-loc">Royal Botanic Gardens, Kew</div>.</div></div></div></div><div content-id="dsx006-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="collab">LPWG</div>. <div class="year">2013</div>, 
<div class="article-title">Towards a new classification system for legumes: progress report from the 6th International Legume Conference</div>. <div class="source">South Afr. J. Bot</div> ., <div class="volume">89</div>, <div class="fpage">3</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sajb.2013.07.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sajb.2013.07.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sajb.2013.07.022"> </span></div> </div></div></div></div></div><div content-id="dsx006-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Doyle</div>
               <div class="given-names">J.J.</div>
            </div>, <div class="name string-name"><div class="surname">Luckow</div> <div class="given-names">M.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">The rest of the iceberg. Legume diversity and evolution in a phylogenetic context</div>. <div class="source">Plant Physiol</div> ., <div class="volume">131</div>, <div class="fpage">900</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rest%20of%20the%20iceberg.%20Legume%20diversity%20and%20evolution%20in%20a%20phylogenetic%20context&amp;author=J.J.%20Doyle&amp;author=M.%20Luckow&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=900-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.018150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.018150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.018150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Young</div>
               <div class="given-names">N.D.</div>
            </div>, <div class="name string-name"><div class="surname">Debellé</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Oldroyd</div> <div class="given-names">G.E.D.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The Medicago genome provides insight into the evolution of rhizobial symbioses</div>. <div class="source">Nature</div> , <strong>480</strong>, 520–524.</div></div></div></div><div content-id="dsx006-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Graham</div>
               <div class="given-names">P.H.</div>
            </div>, <div class="name string-name"><div class="surname">Vance</div> <div class="given-names">C.P.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Legumes: importance and constraints to greater use</div>. <div class="source">Plant Physiol</div> ., <div class="volume">131</div>, <div class="fpage">872</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%3A%20importance%20and%20constraints%20to%20greater%20use&amp;author=P.H.%20Graham&amp;author=C.P.%20Vance&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.017004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.017004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.017004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cai</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Guisinger</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.-G.</div></div></div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Extensive reorganization of the plastid genome of <em>Trifolium subterraneum</em> (Fabaceae) is associated with numerous repeated sequences and novel DNA insertions</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">67</div>, <div class="fpage">696</div>–<div class="lpage">704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20reorganization%20of%20the%20plastid%20genome%20of%20Trifolium%20subterraneum%20(Fabaceae)%20is%20associated%20with%20numerous%20repeated%20sequences%20and%20novel%20DNA%20insertions&amp;author=Z.%20Cai&amp;author=M.%20Guisinger&amp;author=H.-G.%20Kim&amp;publication_year=2008&amp;journal=J.%20Mol.%20Evol&amp;volume=67&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9180-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9180-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9180-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19018585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Doyle</div>
               <div class="given-names">J.J.</div>
            </div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">1995</div>, 
<div class="article-title">Multiple independent losses of two genes and one intron from legume chloroplast genomes</div>. <div class="source">Syst. Bot</div> ., <div class="volume">20</div>, <div class="fpage">272</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20independent%20losses%20of%20two%20genes%20and%20one%20intron%20from%20legume%20chloroplast%20genomes&amp;author=J.J.%20Doyle&amp;author=J.L.%20Doyle&amp;author=J.D.%20Palmer&amp;publication_year=1995&amp;journal=Syst.%20Bot&amp;volume=20&amp;pages=272" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2419496" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2419496" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2419496"> </span></div> </div></div></div></div></div><div content-id="dsx006-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guisinger</div>
               <div class="given-names">M.M.</div>
            </div>, <div class="name string-name"><div class="surname">Kuehl</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Extreme reconfiguration of plastid genomes in the angiosperm family Geraniaceae: rearrangements, repeats, and codon usage</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">583</div>–<div class="lpage">600</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extreme%20reconfiguration%20of%20plastid%20genomes%20in%20the%20angiosperm%20family%20Geraniaceae%3A%20rearrangements%2C%20repeats%2C%20and%20codon%20usage&amp;author=M.M.%20Guisinger&amp;author=J.V.%20Kuehl&amp;author=J.L.%20Boore&amp;author=R.K.%20Jansen&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=583-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Haberle</div>
               <div class="given-names">R.C.</div>
            </div>, <div class="name string-name"><div class="surname">Fourcade</div> <div class="given-names">H.M.</div></div>, <div class="name string-name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Extensive rearrangements in the chloroplast genome of <em>Trachelium caeruleum</em> are associated with repeats and tRNA genes</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">66</div>, <div class="fpage">350</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20rearrangements%20in%20the%20chloroplast%20genome%20of%20Trachelium%20caeruleum%20are%20associated%20with%20repeats%20and%20tRNA%20genes&amp;author=R.C.%20Haberle&amp;author=H.M.%20Fourcade&amp;author=J.L.%20Boore&amp;author=R.K.%20Jansen&amp;publication_year=2008&amp;journal=J.%20Mol.%20Evol&amp;volume=66&amp;pages=350-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9086-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9086-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9086-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18330485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jansen</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Ruhlman</div> <div class="given-names">T.A.</div></div></div>
          <div class="year">2012</div>, Plastid genomes of seed plants In: <div class="person-group"><div class="name string-name"><div class="surname">Bock</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Knoop</div> <div class="given-names">V.</div></div></div> (eds.), <div class="source">Genomics of Chloroplasts and Mitochondria</div> . 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">Netherlands, Dordrecht</div>, pp. <div class="fpage">103</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Chloroplasts%20and%20Mitochondria&amp;author=R.K.%20Jansen&amp;author=T.A.%20Ruhlman&amp;author=R.%20Bock&amp;author=V.%20Knoop&amp;publication_year=2012&amp;book=Genomics%20of%20Chloroplasts%20and%20Mitochondria" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-94-007-2920-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-94-007-2920-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-94-007-2920-9"> </span></div> </div></div></div></div></div><div content-id="dsx006-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Magee</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Aspinall</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rice</div> <div class="given-names">D.W.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Localized hypermutation and associated gene losses in legume chloroplast genomes</div>. <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1700</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localized%20hypermutation%20and%20associated%20gene%20losses%20in%20legume%20chloroplast%20genomes&amp;author=A.M.%20Magee&amp;author=S.%20Aspinall&amp;author=D.W.%20Rice&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1700-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.111955.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.111955.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.111955.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20978141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Weng</div>
               <div class="given-names">M.-L.</div>
            </div>, <div class="name string-name"><div class="surname">Blazier</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Govindu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">R. K.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Reconstruction of the ancestral plastid genome in Geraniaceae reveals a correlation between genome rearrangements, repeats, and nucleotide substitution rates</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">31</div>, <div class="fpage">645</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20ancestral%20plastid%20genome%20in%20Geraniaceae%20reveals%20a%20correlation%20between%20genome%20rearrangements%2C%20repeats%2C%20and%20nucleotide%20substitution%20rates&amp;author=M.-L.%20Weng&amp;author=J.C.%20Blazier&amp;author=M.%20Govindu&amp;author=R.%20K.%20Jansen&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=645-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24336877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dugas</div>
               <div class="given-names">D.V.</div>
            </div>, <div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Koenen</div> <div class="given-names">E.J.M.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Mimosoid legume plastome evolution: IR expansion, tandem repeat expansions, and accelerated rate of evolution in clpP</div>. <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">16958</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mimosoid%20legume%20plastome%20evolution%3A%20IR%20expansion%2C%20tandem%20repeat%20expansions%2C%20and%20accelerated%20rate%20of%20evolution%20in%20clpP&amp;author=D.V.%20Dugas&amp;author=D.%20Hernandez&amp;author=E.J.M.%20Koenen&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=16958" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep16958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep16958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep16958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26592928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martin</div>
               <div class="given-names">G.E.</div>
            </div>, <div class="name string-name"><div class="surname">Rousseau-Gueutin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cordonnier</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">The first complete chloroplast genome of the Genistoid legume <em>Lupinus luteus</em>: evidence for a novel major lineage-specific rearrangement and new insights regarding plastome evolution in the legume family</div>. <div class="source">Ann. Bot</div> ., <div class="volume">113</div>, <div class="fpage">1197</div>–<div class="lpage">210</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20complete%20chloroplast%20genome%20of%20the%20Genistoid%20legume%20Lupinus%20luteus%3A%20evidence%20for%20a%20novel%20major%20lineage-specific%20rearrangement%20and%20new%20insights%20regarding%20plastome%20evolution%20in%20the%20legume%20family&amp;author=G.E.%20Martin&amp;author=M.%20Rousseau-Gueutin&amp;author=S.%20Cordonnier&amp;publication_year=2014&amp;journal=Ann.%20Bot&amp;volume=113&amp;pages=1197-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcu050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcu050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcu050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24769537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schwarz</div>
               <div class="given-names">E.N.</div>
            </div>, <div class="name string-name"><div class="surname">Ruhlman</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Sabir</div> <div class="given-names">J.S.M.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Plastid genome sequences of legumes reveal parallel inversions and multiple losses of <em>rps16</em> in papilionoids: parallel inversions and <em>rps16</em> losses in legumes</div>. <div class="source">J. Syst. Evol</div> ., <div class="volume">53</div>, <div class="fpage">458</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plastid%20genome%20sequences%20of%20legumes%20reveal%20parallel%20inversions%20and%20multiple%20losses%20of%20rps16%20in%20papilionoids%3A%20parallel%20inversions%20and%20rps16%20losses%20in%20legumes&amp;author=E.N.%20Schwarz&amp;author=T.A.%20Ruhlman&amp;author=J.S.M.%20Sabir&amp;publication_year=2015&amp;journal=J.%20Syst.%20Evol&amp;volume=53&amp;pages=458-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jse.12179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjse.12179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjse.12179"> </span></div> </div></div></div></div></div><div content-id="dsx006-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sherman-Broyles</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Bombarely</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Grimwood</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Complete plastome sequences from <em>Glycine syndetika</em> and six additional perennial wild relatives of soybean</div>. <div class="source">G3amp58 GenesGenomesGenetics</div> , <div class="volume">4</div>, <div class="fpage">2023</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20plastome%20sequences%20from%20Glycine%20syndetika%20and%20six%20additional%20perennial%20wild%20relatives%20of%20soybean&amp;author=S.%20Sherman-Broyles&amp;author=A.%20Bombarely&amp;author=J.%20Grimwood&amp;author=J.%20Schmutz&amp;author=J.%20Doyle&amp;publication_year=2014&amp;journal=G3amp58%20GenesGenomesGenetics&amp;volume=4&amp;pages=2023-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.012690" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.012690" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.012690"> </span></div> </div></div></div></div></div><div content-id="dsx006-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Williams</div>
               <div class="given-names">A.V.</div>
            </div>, <div class="name string-name"><div class="surname">Boykin</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Howell</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Nevill</div> <div class="given-names">P.G.</div></div>, <div class="name string-name"><div class="surname">Small</div> <div class="given-names">I.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">The complete sequence of the <em>Acacia ligulata</em> chloroplast genome reveals a highly divergent clpP1 gene. C.-H. Kuo (ed.)</div>. <div class="source">PLOS ONE</div> , <div class="volume">10</div>, <div class="fpage">e0125768</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20sequence%20of%20the%20Acacia%20ligulata%20chloroplast%20genome%20reveals%20a%20highly%20divergent%20clpP1%20gene.%20C.-H.%20Kuo%20(ed.)&amp;author=A.V.%20Williams&amp;author=L.M.%20Boykin&amp;author=K.A.%20Howell&amp;author=P.G.%20Nevill&amp;author=I.%20Small&amp;publication_year=2015&amp;journal=PLOS%20ONE&amp;volume=10&amp;pages=e0125768" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0125768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0125768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0125768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25955637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Palmer</div>
               <div class="given-names">J.D.</div>
            </div>, <div class="name string-name"><div class="surname">Nugent</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Herbon</div> <div class="given-names">L.A.</div></div></div>
          <div class="year">1987</div>, 
<div class="article-title">Unusual structure of geranium chloroplast DNA: a triple-sized inverted repeat, extensive gene duplications, multiple inversions, and two repeat families</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">84</div>, <div class="fpage">769</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unusual%20structure%20of%20geranium%20chloroplast%20DNA%3A%20a%20triple-sized%20inverted%20repeat%2C%20extensive%20gene%20duplications%2C%20multiple%20inversions%2C%20and%20two%20repeat%20families&amp;author=J.D.%20Palmer&amp;author=J.M.%20Nugent&amp;author=L.A.%20Herbon&amp;publication_year=1987&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=84&amp;pages=769-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.3.769" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.3.769" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.3.769"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16593810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Doyle</div>
               <div class="given-names">J.J.</div>
            </div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Ballenger</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">1996</div>, 
<div class="article-title">The distribution and phylogenetic significance of a 50-kb chloroplast DNA inversion in the flowering plant family Leguminosae</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">5</div>, <div class="fpage">429</div>–<div class="lpage">438</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20distribution%20and%20phylogenetic%20significance%20of%20a%2050-kb%20chloroplast%20DNA%20inversion%20in%20the%20flowering%20plant%20family%20Leguminosae&amp;author=J.J.%20Doyle&amp;author=J.L.%20Doyle&amp;author=J.A.%20Ballenger&amp;author=J.D.%20Palmer&amp;publication_year=1996&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=5&amp;pages=429-438" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mpev.1996.0038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmpev.1996.0038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmpev.1996.0038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8728401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jansen</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Wojciechowski</div> <div class="given-names">M.F.</div></div>, <div class="name string-name"><div class="surname">Sanniyasi</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.-B.</div></div>, <div class="name string-name"><div class="surname">Daniell</div> <div class="given-names">H.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Complete plastid genome sequence of the chickpea (<em>Cicer arietinum</em>) and the phylogenetic distribution of rps12 and clpP intron losses among legumes (Leguminosae)</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">48</div>, <div class="fpage">1204</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20plastid%20genome%20sequence%20of%20the%20chickpea%20(Cicer%20arietinum)%20and%20the%20phylogenetic%20distribution%20of%20rps12%20and%20clpP%20intron%20losses%20among%20legumes%20(Leguminosae)&amp;author=R.K.%20Jansen&amp;author=M.F.%20Wojciechowski&amp;author=E.%20Sanniyasi&amp;author=S.-B.%20Lee&amp;author=H.%20Daniell&amp;publication_year=2008&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=48&amp;pages=1204-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.06.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.06.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.06.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18638561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wojciechowski</div>
               <div class="given-names">M.F.</div>
            </div>, <div class="name string-name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sanderson</div> <div class="given-names">M.J.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">A phylogeny of legumes (Leguminosae) based on analysis of the plastid matK gene resolves many well-supported subclades within the family</div>. <div class="source">Am. J. Bot</div> ., <div class="volume">91</div>, <div class="fpage">1846</div>–<div class="lpage">1862</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogeny%20of%20legumes%20(Leguminosae)%20based%20on%20analysis%20of%20the%20plastid%20matK%20gene%20resolves%20many%20well-supported%20subclades%20within%20the%20family&amp;author=M.F.%20Wojciechowski&amp;author=M.%20Lavin&amp;author=M.J.%20Sanderson&amp;publication_year=2004&amp;journal=Am.%20J.%20Bot&amp;volume=91&amp;pages=1846-1862" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.91.11.1846" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.91.11.1846" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.91.11.1846"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21652332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bruneau</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">1990</div>, 
<div class="article-title">A chloroplast DNA inversion as a subtribal character in the phaseoleae (Leguminosae)</div>. <div class="source">Syst. Bot</div> ., <div class="volume">15</div>, <div class="fpage">378</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chloroplast%20DNA%20inversion%20as%20a%20subtribal%20character%20in%20the%20phaseoleae%20(Leguminosae)&amp;author=A.%20Bruneau&amp;author=J.J.%20Doyle&amp;author=J.D.%20Palmer&amp;publication_year=1990&amp;journal=Syst.%20Bot&amp;volume=15&amp;pages=378" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2419351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2419351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2419351"> </span></div> </div></div></div></div></div><div content-id="dsx006-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Castillo-Ramírez</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">González</div> <div class="given-names">V.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Rapid evolutionary change of common bean (<em>Phaseolus vulgaris</em> L.) plastome, and the genomic diversification of legume chloroplasts</div>. <div class="source">BMC Genom</div> ., <div class="volume">8</div>, <div class="fpage">228</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20evolutionary%20change%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20plastome%2C%20and%20the%20genomic%20diversification%20of%20legume%20chloroplasts&amp;author=X.%20Guo&amp;author=S.%20Castillo-Ram%C3%ADrez&amp;author=V.%20Gonz%C3%A1lez&amp;publication_year=2007&amp;journal=BMC%20Genom&amp;volume=8&amp;pages=228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-228"> </span></div> </div></div></div></div></div><div content-id="dsx006-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tangphatsornruang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sangsrakru</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chanprasert</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The chloroplast genome sequence of mungbean (<em>Vigna radiata</em>) determined by high-throughput pyrosequencing: structural organization and phylogenetic relationships</div>. <div class="source">DNA Res</div> ., <div class="volume">17</div>, <div class="fpage">11</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chloroplast%20genome%20sequence%20of%20mungbean%20(Vigna%20radiata)%20determined%20by%20high-throughput%20pyrosequencing%3A%20structural%20organization%20and%20phylogenetic%20relationships&amp;author=S.%20Tangphatsornruang&amp;author=D.%20Sangsrakru&amp;author=J.%20Chanprasert&amp;publication_year=2010&amp;journal=DNA%20Res&amp;volume=17&amp;pages=11-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20007682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kazakoff</div>
               <div class="given-names">S.H.</div>
            </div>, <div class="name string-name"><div class="surname">Imelfort</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Capturing the biofuel wellhead and powerhouse: the chloroplast and mitochondrial genomes of the leguminous feedstock tree <em>Pongamia pinnata</em>. J.H. Badger (ed.)</div>. <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e51687</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20the%20biofuel%20wellhead%20and%20powerhouse%3A%20the%20chloroplast%20and%20mitochondrial%20genomes%20of%20the%20leguminous%20feedstock%20tree%20Pongamia%20pinnata.%20J.H.%20Badger%20(ed.)&amp;author=S.H.%20Kazakoff&amp;author=M.%20Imelfort&amp;author=D.%20Edwards&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e51687" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0051687" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0051687" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0051687"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23272141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Timmis</div>
               <div class="given-names">J.N.</div>
            </div>, <div class="name string-name"><div class="surname">Ayliffe</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">C.Y.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">W.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Endosymbiotic gene transfer: organelle genomes forge eukaryotic chromosomes</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">5</div>, <div class="fpage">123</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endosymbiotic%20gene%20transfer%3A%20organelle%20genomes%20forge%20eukaryotic%20chromosomes&amp;author=J.N.%20Timmis&amp;author=M.A.%20Ayliffe&amp;author=C.Y.%20Huang&amp;author=W.%20Martin&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet&amp;volume=5&amp;pages=123-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14735123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ueda</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Substitution of the gene for chloroplast RPS16 was assisted by generation of a dual targeting signal</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">1566</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substitution%20of%20the%20gene%20for%20chloroplast%20RPS16%20was%20assisted%20by%20generation%20of%20a%20dual%20targeting%20signal&amp;author=M.%20Ueda&amp;author=T.%20Nishikawa&amp;author=M.%20Fujimoto&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=1566-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18453549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fleischmann</div>
               <div class="given-names">T.T.</div>
            </div>, <div class="name string-name"><div class="surname">Scharff</div> <div class="given-names">L.B.</div></div>, <div class="name string-name"><div class="surname">Alkatib</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hasdorf</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schöttler</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Bock</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Nonessential plastid-encoded ribosomal proteins in tobacco: a developmental role for plastid translation and implications for reductive genome evolution</div>. <div class="source">Plant Cell</div> , <div class="volume">23</div>, <div class="fpage">3137</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonessential%20plastid-encoded%20ribosomal%20proteins%20in%20tobacco%3A%20a%20developmental%20role%20for%20plastid%20translation%20and%20implications%20for%20reductive%20genome%20evolution&amp;author=T.T.%20Fleischmann&amp;author=L.B.%20Scharff&amp;author=S.%20Alkatib&amp;author=S.%20Hasdorf&amp;author=M.%20A.%20Sch%C3%B6ttler&amp;author=R.%20Bock&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=3137-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.088906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.088906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.088906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21934145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kode</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Mudd</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Iamtham</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Day</div> <div class="given-names">A.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">The tobacco plastid accD gene is essential and is required for leaf development</div>. <div class="source">Plant J</div> ., <div class="volume">44</div>, <div class="fpage">237</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tobacco%20plastid%20accD%20gene%20is%20essential%20and%20is%20required%20for%20leaf%20development&amp;author=V.%20Kode&amp;author=E.A.%20Mudd&amp;author=S.%20Iamtham&amp;author=A.%20Day&amp;publication_year=2005&amp;journal=Plant%20J&amp;volume=44&amp;pages=237-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2005.02533.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2005.02533.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2005.02533.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16212603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shikanai</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ueda</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nishimura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kuroiwa</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hashimoto</div> <div class="given-names">T.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">The chloroplast clpP gene, encoding a proteolytic subunit of ATP-dependent protease, is indispensable for chloroplast development in tobacco</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">42</div>, <div class="fpage">264</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chloroplast%20clpP%20gene%2C%20encoding%20a%20proteolytic%20subunit%20of%20ATP-dependent%20protease%2C%20is%20indispensable%20for%20chloroplast%20development%20in%20tobacco&amp;author=T.%20Shikanai&amp;author=K.%20Shimizu&amp;author=K.%20Ueda&amp;author=Y.%20Nishimura&amp;author=T.%20Kuroiwa&amp;author=T.%20Hashimoto&amp;publication_year=2001&amp;journal=Plant%20Cell%20Physiol&amp;volume=42&amp;pages=264-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pce031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpce031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpce031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11266577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jansen</div>
               <div class="given-names">R.K.</div>
            </div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Raubeson</div> <div class="given-names">L.A.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Analysis of 81 genes from 64 plastid genomes resolves relationships in angiosperms and identifies genome-scale evolutionary patterns</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">104</div>, <div class="fpage">19369</div>–<div class="lpage">19374</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%2081%20genes%20from%2064%20plastid%20genomes%20resolves%20relationships%20in%20angiosperms%20and%20identifies%20genome-scale%20evolutionary%20patterns&amp;author=R.K.%20Jansen&amp;author=Z.%20Cai&amp;author=L.A.%20Raubeson&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=104&amp;pages=19369-19374" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0709121104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0709121104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0709121104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18048330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cardoso</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">de Queiroz</div> <div class="given-names">L.P.</div></div>, <div class="name string-name"><div class="surname">Pennington</div> <div class="given-names">R.T.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Revisiting the phylogeny of papilionoid legumes: new insights from comprehensively sampled early-branching lineages</div>. <div class="source">Am. J. Bot</div> ., <div class="volume">99</div>, <div class="fpage">1991</div>–<div class="lpage">2013</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Revisiting%20the%20phylogeny%20of%20papilionoid%20legumes%3A%20new%20insights%20from%20comprehensively%20sampled%20early-branching%20lineages&amp;author=D.%20Cardoso&amp;author=L.P.%20de%20Queiroz&amp;author=R.T.%20Pennington&amp;publication_year=2012&amp;journal=Am.%20J.%20Bot&amp;volume=99&amp;pages=1991-2013" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1200380" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1200380" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1200380"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23221500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sveinsson</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Cronk</div> <div class="given-names">Q.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Evolutionary origin of highly repetitive plastid genomes within the clover genus (Trifolium)</div>. <div class="source">BMC Evol. Biol</div> ., <div class="volume">14</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origin%20of%20highly%20repetitive%20plastid%20genomes%20within%20the%20clover%20genus%20(Trifolium)&amp;author=S.%20Sveinsson&amp;author=Q.%20Cronk&amp;publication_year=2014&amp;journal=BMC%20Evol.%20Biol&amp;volume=14&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-014-0228-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-014-0228-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-014-0228-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24382122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gepts</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Beavis</div> <div class="given-names">W.D.</div></div>, <div class="name string-name"><div class="surname">Brummer</div> <div class="given-names">E.C.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Legumes as a model plant family. Genomics for food and feed report of the cross-legume advances through genomics conference</div>. <div class="source">Plant Physiol</div> ., <div class="volume">137</div>, <div class="fpage">1228</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%20as%20a%20model%20plant%20family.%20Genomics%20for%20food%20and%20feed%20report%20of%20the%20cross-legume%20advances%20through%20genomics%20conference&amp;author=P.%20Gepts&amp;author=W.D.%20Beavis&amp;author=E.C.%20Brummer&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=137&amp;pages=1228-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.060871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.060871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.060871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15824285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gladstones</div>
               <div class="given-names">J.S.</div>
            </div>, <div class="name string-name"><div class="surname">Atkins</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Hamblin</div> <div class="given-names">J.</div></div></div> (eds.). <div class="year">1998</div>, <div class="source">Lupins as Crop Plants: Biology, Production, and Utilization</div> . 
<div class="publisher-name">CAB International</div>, 
<div class="publisher-loc">Wallingford, Oxon, UK; New York, NY, USA</div>.</div></div></div></div><div content-id="dsx006-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cabello-Hurtado</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Keller</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sanchez-Lucas</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Jorrín-Novo</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Aïnouche</div> <div class="given-names">A.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">Proteomics for exploiting diversity of lupin seed storage proteins and their use as nutraceuticals for health and welfare</div>. <div class="source">J. Proteomics</div> , <strong>143</strong>, 57–68.</div></div></div></div><div content-id="dsx006-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">Z.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence, and a sequence-defined genetic linkage map of the legume crop species <em>Lupinus angustifolius</em>. X.L. Cui, (ed.)</div>. <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e64799</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%2C%20and%20a%20sequence-defined%20genetic%20linkage%20map%20of%20the%20legume%20crop%20species%20Lupinus%20angustifolius.%20X.L.%20Cui%2C%20(ed.)&amp;author=H.%20Yang&amp;author=Y.%20Tao&amp;author=Z.%20Zheng&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e64799" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0064799" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0064799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0064799"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23734219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ainouche</div>
               <div class="given-names">A.-K.</div>
            </div>, <div class="name string-name"><div class="surname">Bayer</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Phylogenetic relationships in <em>Lupinus</em> (Fabaceae: Papilionoideae) based on internal transcribed spacer sequences (ITS) of nuclear ribosomal DNA</div>. <div class="source">Am. J. Bot</div> ., <div class="volume">86</div>, <div class="fpage">590</div>–<div class="lpage">607</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20in%20Lupinus%20(Fabaceae%3A%20Papilionoideae)%20based%20on%20internal%20transcribed%20spacer%20sequences%20(ITS)%20of%20nuclear%20ribosomal%20DNA&amp;author=A.-K.%20Ainouche&amp;author=R.J.%20Bayer&amp;publication_year=1999&amp;journal=Am.%20J.%20Bot&amp;volume=86&amp;pages=590-607" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2656820" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2656820" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2656820"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10205079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Drummond</div>
               <div class="given-names">C.S.</div>
            </div>, <div class="name string-name"><div class="surname">Eastwood</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Miotto</div> <div class="given-names">S.T.S.</div></div>, <div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">C.E.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Multiple continental radiations and correlates of diversification in <em>Lupinus</em> (Leguminosae): testing for key innovation with incomplete taxon sampling</div>. <div class="source">Syst. Biol</div> ., <div class="volume">61</div>, <div class="fpage">443</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20continental%20radiations%20and%20correlates%20of%20diversification%20in%20Lupinus%20(Leguminosae)%3A%20testing%20for%20key%20innovation%20with%20incomplete%20taxon%20sampling&amp;author=C.S.%20Drummond&amp;author=R.J.%20Eastwood&amp;author=S.T.S.%20Miotto&amp;author=C.E.%20Hughes&amp;publication_year=2012&amp;journal=Syst.%20Biol&amp;volume=61&amp;pages=443-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syr126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyr126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsyr126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22228799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eastwood</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Schifino-Wittmann</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Hughes</div></div></div>. <div class="year">2008</div>, <em>Lupins for health and wealth: 12th International Lupin Conference, Fremantle, Western Australia, 14–18 September 2008.</em> In: J.A., Palta, J.D., Berger, and International Lupin Association (eds.). International Lupin Association, Canterbury, New Zealand.</div></div></div></div><div content-id="dsx006-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hughes</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Eastwood</div> <div class="given-names">R.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Island radiation on a continental scale: exceptional rates of plant diversification after uplift of the Andes</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">103</div>, <div class="fpage">10334</div>–<div class="lpage">10339</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Island%20radiation%20on%20a%20continental%20scale%3A%20exceptional%20rates%20of%20plant%20diversification%20after%20uplift%20of%20the%20Andes&amp;author=C.%20Hughes&amp;author=R.%20Eastwood&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=103&amp;pages=10334-10339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0601928103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0601928103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0601928103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16801546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Käss</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Wink</div> <div class="given-names">M.</div></div></div>
          <div class="year">1997</div>, 
<div class="article-title">Molecular phylogeny and phylogeography of <em>Lupinus</em> (Leguminosae) inferred from nucleotide sequences of therbcL gene and ITS 1 + 2 regions of rDNA</div>. <div class="source">Plant Syst. Evol</div> ., <div class="volume">208</div>, <div class="fpage">139</div>–<div class="lpage">167</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20and%20phylogeography%20of%20Lupinus%20(Leguminosae)%20inferred%20from%20nucleotide%20sequences%20of%20therbcL%20gene%20and%20ITS%201%20%2B%202%20regions%20of%20rDNA&amp;author=E.%20K%C3%A4ss&amp;author=M.%20Wink&amp;publication_year=1997&amp;journal=Plant%20Syst.%20Evol&amp;volume=208&amp;pages=139-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00985439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00985439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00985439"> </span></div> </div></div></div></div></div><div content-id="dsx006-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahé</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Markova</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pasquet</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Misset</div> <div class="given-names">M.-T.</div></div>, <div class="name string-name"><div class="surname">Aïnouche</div> <div class="given-names">A.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Isolation, phylogeny and evolution of the SymRK gene in the legume genus <em>Lupinus</em> L</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">60</div>, <div class="fpage">49</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%2C%20phylogeny%20and%20evolution%20of%20the%20SymRK%20gene%20in%20the%20legume%20genus%20Lupinus%20L&amp;author=F.%20Mah%C3%A9&amp;author=D.%20Markova&amp;author=R.%20Pasquet&amp;author=M.-T.%20Misset&amp;author=A.%20A%C3%AFnouche&amp;publication_year=2011&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=60&amp;pages=49-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2011.04.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2011.04.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2011.04.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21550410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahé</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Pascual</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Coriton</div> <div class="given-names">O.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">New data and phylogenetic placement of the enigmatic Old World lupin: <em>Lupinus</em> mariae-josephi H. Pascual</div>. <div class="source">Genet. Resour. Crop Evol</div> ., <div class="volume">58</div>, <div class="fpage">101</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20data%20and%20phylogenetic%20placement%20of%20the%20enigmatic%20Old%20World%20lupin%3A%20Lupinus%20mariae-josephi%20H.%20Pascual&amp;author=F.%20Mah%C3%A9&amp;author=H.%20Pascual&amp;author=O.%20Coriton&amp;publication_year=2011&amp;journal=Genet.%20Resour.%20Crop%20Evol&amp;volume=58&amp;pages=101-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-010-9580-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-010-9580-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-010-9580-6"> </span></div> </div></div></div></div></div><div content-id="dsx006-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aïnouche</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bayer</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Misset</div> <div class="given-names">M.-T.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Molecular phylogeny, diversification and character evolution in <em>Lupinus</em> (Fabaceae) with special attention to Mediterranean and African lupines</div>. <div class="source">Plant Syst. Evol</div> ., <div class="volume">246</div>.</div></div></div></div><div content-id="dsx006-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coissac</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Hollingsworth</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Lavergne</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Taberlet</div> <div class="given-names">P.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">From barcodes to genomes: extending the concept of DNA barcoding</div>. <div class="source">Mol. Ecol</div> , <strong>25</strong>, 1423–8.</div></div></div></div><div content-id="dsx006-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wyman</div>
               <div class="given-names">S.K.</div>
            </div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Automatic annotation of organellar genomes with DOGMA</div>. <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">3252</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20annotation%20of%20organellar%20genomes%20with%20DOGMA&amp;author=S.K.%20Wyman&amp;author=R.K.%20Jansen&amp;author=J.L.%20Boore&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3252-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15180927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krzywinski</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Circos: an information aesthetic for comparative genomics</div>. <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1639</div>–<div class="lpage">1645</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1639-1645" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pierleoni</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Martelli</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Fariselli</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Casadio</div> <div class="given-names">R.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">BaCelLo: a Balanced subCellular Localization predictor</div>. <div class="source">Protoc. Exch</div> , <strong>22</strong>, 408–16.</div></div></div></div><div content-id="dsx006-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Boden</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Hawkins</div> <div class="given-names">J.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Prediction of subcellular localization using sequence-biased recurrent networks</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">2279</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20subcellular%20localization%20using%20sequence-biased%20recurrent%20networks&amp;author=M.%20Boden&amp;author=J.%20Hawkins&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15746276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Emanuelsson</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Predicting subcellular localization of proteins based on their N-terminal amino acid sequence</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">300</div>, <div class="fpage">1005</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20subcellular%20localization%20of%20proteins%20based%20on%20their%20N-terminal%20amino%20acid%20sequence&amp;author=O.%20Emanuelsson&amp;author=H.%20Nielsen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol&amp;volume=300&amp;pages=1005-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3903" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10891285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blum</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Briesemeister</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kohlbacher</div> <div class="given-names">O.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">MultiLoc2: integrating phylogeny and Gene Ontology terms improves subcellular protein localization prediction</div>. <div class="source">BMC Bioinf</div> ., <div class="volume">10</div>, <div class="fpage">274</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MultiLoc2%3A%20integrating%20phylogeny%20and%20Gene%20Ontology%20terms%20improves%20subcellular%20protein%20localization%20prediction&amp;author=T.%20Blum&amp;author=S.%20Briesemeister&amp;author=O.%20Kohlbacher&amp;publication_year=2009&amp;journal=BMC%20Bioinf&amp;volume=10&amp;pages=274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-274"> </span></div> </div></div></div></div></div><div content-id="dsx006-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Small</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Peeters</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Legeai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lurin</div> <div class="given-names">C.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Predotar: a tool for rapidly screening proteomes forN-terminal targeting sequences</div>. <div class="source">Proteomics</div> , <div class="volume">4</div>, <div class="fpage">1581</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predotar%3A%20a%20tool%20for%20rapidly%20screening%20proteomes%20forN-terminal%20targeting%20sequences&amp;author=I.%20Small&amp;author=N.%20Peeters&amp;author=F.%20Legeai&amp;author=C.%20Lurin&amp;publication_year=2004&amp;journal=Proteomics&amp;volume=4&amp;pages=1581-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15174128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Michel</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Ferat</div> <div class="given-names">J.L.</div></div></div>
          <div class="year">1995</div>, 
<div class="article-title">Structure and activities of group II introns</div>. <div class="source">Annu. Rev. Biochem</div> ., <div class="volume">64</div>, <div class="fpage">435</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20activities%20of%20group%20II%20introns&amp;author=F.%20Michel&amp;author=J.L.%20Ferat&amp;publication_year=1995&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=64&amp;pages=435-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.64.070195.002251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.64.070195.002251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.64.070195.002251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7574489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Michel</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Kazuhiko</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Haruo</div> <div class="given-names">O.</div></div></div>
          <div class="year">1989</div>, 
<div class="article-title">Comparative and functional anatomy of group II catalytic introns—a review</div>. <div class="source">Gene</div> , <div class="volume">82</div>, <div class="fpage">5</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20and%20functional%20anatomy%20of%20group%20II%20catalytic%20introns%E2%80%94a%20review&amp;author=F.%20Michel&amp;author=U.%20Kazuhiko&amp;author=O.%20Haruo&amp;publication_year=1989&amp;journal=Gene&amp;volume=82&amp;pages=5-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(89)90026-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(89)90026-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(89)90026-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2684776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lehmann</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">U.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Group II introns: structure and catalytic versatility of large natural ribozymes</div>. <div class="source">Crit. Rev. Biochem. Mol. Biol</div> ., <div class="volume">38</div>, <div class="fpage">249</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Group%20II%20introns%3A%20structure%20and%20catalytic%20versatility%20of%20large%20natural%20ribozymes&amp;author=K.%20Lehmann&amp;author=U.%20Schmidt&amp;publication_year=2003&amp;journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol&amp;volume=38&amp;pages=249-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/713609236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F713609236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F713609236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12870716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Roy</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Ueda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kadowaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tsutsumi</div> <div class="given-names">N.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Different status of the gene for ribosomal protein S16 in the chloroplast genome during evolution of the genus Arabidopsis and closely related species</div>. <div class="source">Genes Genet. Syst</div> ., <div class="volume">85</div>, <div class="fpage">319</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20status%20of%20the%20gene%20for%20ribosomal%20protein%20S16%20in%20the%20chloroplast%20genome%20during%20evolution%20of%20the%20genus%20Arabidopsis%20and%20closely%20related%20species&amp;author=S.%20Roy&amp;author=M.%20Ueda&amp;author=K.%20Kadowaki&amp;author=N.%20Tsutsumi&amp;publication_year=2010&amp;journal=Genes%20Genet.%20Syst&amp;volume=85&amp;pages=319-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.85.319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.85.319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.85.319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21317544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kearse</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Moir</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1647</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geneious%20basic%3A%20an%20integrated%20and%20extendable%20desktop%20software%20platform%20for%20the%20organization%20and%20analysis%20of%20sequence%20data&amp;author=M.%20Kearse&amp;author=R.%20Moir&amp;author=A.%20Wilson&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1647-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
          </div>
          <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paradis</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Claude</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Strimmer</div> <div class="given-names">K.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">APE: analyses of phylogenetics and evolution in R language</div>. <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">289</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=APE%3A%20analyses%20of%20phylogenetics%20and%20evolution%20in%20R%20language&amp;author=E.%20Paradis&amp;author=J.%20Claude&amp;author=K.%20Strimmer&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=289-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg412" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg412" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg412"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14734327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kimura</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1980</div>, 
<div class="article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">16</div>, <div class="fpage">111</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20estimating%20evolutionary%20rates%20of%20base%20substitutions%20through%20comparative%20studies%20of%20nucleotide%20sequences&amp;author=M.%20Kimura&amp;publication_year=1980&amp;journal=J.%20Mol.%20Evol&amp;volume=16&amp;pages=111-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01731581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01731581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01731581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">17</div>, <div class="fpage">32</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20synonymous%20and%20nonsynonymous%20substitution%20rates%20under%20realistic%20evolutionary%20models&amp;author=Z.%20Yang&amp;author=R.%20Nielsen&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol&amp;volume=17&amp;pages=32-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kurtz</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Choudhuri</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Ohlebusch</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Schleiermacher</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Stoye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Giegerich</div> <div class="given-names">R.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">REPuter: the manifold applications of repeat analysis on a genomic scale</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">4633</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=REPuter%3A%20the%20manifold%20applications%20of%20repeat%20analysis%20on%20a%20genomic%20scale&amp;author=S.%20Kurtz&amp;author=J.V.%20Choudhuri&amp;author=E.%20Ohlebusch&amp;author=C.%20Schleiermacher&amp;author=J.%20Stoye&amp;author=R.%20Giegerich&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=4633-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.22.4633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.22.4633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.22.4633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11713313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saski</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.-B.</div></div>, <div class="name string-name"><div class="surname">Daniell</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Complete chloroplast genome sequence of glycine max and comparative analyses with other legume genomes</div>. <div class="source">Plant Mol. Biol</div> ., <div class="volume">59</div>, <div class="fpage">309</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20chloroplast%20genome%20sequence%20of%20glycine%20max%20and%20comparative%20analyses%20with%20other%20legume%20genomes&amp;author=C.%20Saski&amp;author=S.-B.%20Lee&amp;author=H.%20Daniell&amp;publication_year=2005&amp;journal=Plant%20Mol.%20Biol&amp;volume=59&amp;pages=309-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-005-8882-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-005-8882-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-005-8882-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16247559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mayer</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2007</div>, October, Phobos: Highly Accurate Search for Perfect and Imperfect Tandem Repeats in Complete Genomes by Christoph Mayer.</div></div></div></div><div content-id="dsx006-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rozen</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>. <div class="source">Methods Mol. Biol. Clifton NJ</div> , <div class="volume">132</div>, <div class="fpage">365</div>–<div class="lpage">86</div>.</div></div></div></div><div content-id="dsx006-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Katoh</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">3066</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-3066" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Felsenstein</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">1985</div>, 
<div class="article-title">Confidence limits on phylogenies: an approach using the bootstrap</div>. <div class="source">Evolution</div> , <div class="volume">39</div>, <div class="fpage">783</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;author=J.%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1558-5646.1985.tb00420.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1558-5646.1985.tb00420.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1558-5646.1985.tb00420.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Posada</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">jModelTest: phylogenetic model averaging</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">1253</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=jModelTest%3A%20phylogenetic%20model%20averaging&amp;author=D.%20Posada&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=1253-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Finn</div>
               <div class="given-names">R.D.</div>
            </div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yoon</div>
               <div class="given-names">H.S.</div>
            </div>, <div class="name string-name"><div class="surname">Hackett</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Ciniglia</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bhattacharya</div> <div class="given-names">D.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">A molecular timeline for the origin of photosynthetic eukaryotes</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">21</div>, <div class="fpage">809</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20timeline%20for%20the%20origin%20of%20photosynthetic%20eukaryotes&amp;author=H.S.%20Yoon&amp;author=J.D.%20Hackett&amp;author=C.%20Ciniglia&amp;author=G.%20Pinto&amp;author=D.%20Bhattacharya&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol&amp;volume=21&amp;pages=809-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="year">2003</div>, 
<div class="article-title">Why chloroplasts and mitochondria contain genomes</div>. <div class="source">Comp. Funct. Genomics</div> , <div class="volume">4</div>, <div class="fpage">31</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20chloroplasts%20and%20mitochondria%20contain%20genomes&amp;author=J.F.%20Allen&amp;publication_year=2003&amp;journal=Comp.%20Funct.%20Genomics&amp;volume=4&amp;pages=31-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/cfg.245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcfg.245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fcfg.245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18629105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">J.F.</div>
            </div>, <div class="name string-name"><div class="surname">Puthiyaveetil</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ström</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Allen</div> <div class="given-names">C.A.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Energy transduction anchors genes in organelles: Problems</div><div class="source">and paradigms. BioEssays</div> , <div class="volume">27</div>, <div class="fpage">426</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Energy%20transduction%20anchors%20genes%20in%20organelles%3A%20Problems&amp;author=J.F.%20Allen&amp;author=S.%20Puthiyaveetil&amp;author=J.%20Str%C3%B6m&amp;author=C.A.%20Allen&amp;publication_year=2005&amp;journal=and%20paradigms.%20BioEssays&amp;volume=27&amp;pages=426-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bies.v27:4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.v27:4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbies.v27:4"> </span></div> </div></div></div></div></div><div content-id="dsx006-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Why chloroplasts and mitochondria retain their own genomes and genetic systems: colocation for redox regulation of gene expression</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">112</div>, <div class="fpage">10231</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20chloroplasts%20and%20mitochondria%20retain%20their%20own%20genomes%20and%20genetic%20systems%3A%20colocation%20for%20redox%20regulation%20of%20gene%20expression&amp;author=J.F.%20Allen&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=112&amp;pages=10231-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1500012112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1500012112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1500012112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26286985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pfannschmidt</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Nilsson</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tullberg</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Link</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Allen</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Direct transcriptional control of the chloroplast genes psbA and psaAB adjusts photosynthesis to light energy distribution in plants</div>. <div class="source">IUBMB Life</div> , <div class="volume">48</div>, <div class="fpage">271</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20transcriptional%20control%20of%20the%20chloroplast%20genes%20psbA%20and%20psaAB%20adjusts%20photosynthesis%20to%20light%20energy%20distribution%20in%20plants&amp;author=T.%20Pfannschmidt&amp;author=A.%20Nilsson&amp;author=A.%20Tullberg&amp;author=G.%20Link&amp;author=J.F.%20Allen&amp;publication_year=1999&amp;journal=IUBMB%20Life&amp;volume=48&amp;pages=271-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/713803507" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F713803507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F713803507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10690637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maier</div>
               <div class="given-names">U.-G.</div>
            </div>, <div class="name string-name"><div class="surname">Zauner</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Woehle</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Massively convergent evolution for ribosomal protein gene content in plastid and mitochondrial genomes</div>. <div class="source">Genome Biol. Evol</div> ., <div class="volume">5</div>, <div class="fpage">2318</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massively%20convergent%20evolution%20for%20ribosomal%20protein%20gene%20content%20in%20plastid%20and%20mitochondrial%20genomes&amp;author=U.-G.%20Maier&amp;author=S.%20Zauner&amp;author=C.%20Woehle&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol&amp;volume=5&amp;pages=2318-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24259312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kikuchi</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Bédard</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hirano</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Uncovering the protein translocon at the chloroplast inner envelope membrane</div>. <div class="source">Science</div> , <div class="volume">339</div>, <div class="fpage">571</div>–<div class="lpage">574</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uncovering%20the%20protein%20translocon%20at%20the%20chloroplast%20inner%20envelope%20membrane&amp;author=S.%20Kikuchi&amp;author=J.%20B%C3%A9dard&amp;author=M.%20Hirano&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=571-574" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1229262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1229262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1229262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23372012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dong</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">ycf1, the most promising plastid DNA barcode of land plants</div>. <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">8348</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ycf1%2C%20the%20most%20promising%20plastid%20DNA%20barcode%20of%20land%20plants&amp;author=W.%20Dong&amp;author=C.%20Xu&amp;author=C.%20Li&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=8348" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep08348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep08348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep08348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25672218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dong</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">S.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Highly variable chloroplast markers for evaluating plant phylogeny at low taxonomic levels and for DNA barcoding. A. Moustafa, (ed.)</div>. <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e35071</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20variable%20chloroplast%20markers%20for%20evaluating%20plant%20phylogeny%20at%20low%20taxonomic%20levels%20and%20for%20DNA%20barcoding.%20A.%20Moustafa%2C%20(ed.)&amp;author=W.%20Dong&amp;author=J.%20Liu&amp;author=J.%20Yu&amp;author=L.%20Wang&amp;author=S.%20Zhou&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e35071" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0035071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0035071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0035071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22511980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Swofford</div>
               <div class="given-names">D.L.</div>
            </div>
          </div>
          <div class="year">2001</div>, <em>PAUP*: Phylogenetic Analysis Using Parsimony (and other methods) 4.0.b5.</em></div></div></div></div><div content-id="dsx006-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Johnson</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">D.E.</div></div></div>
          <div class="year">1998</div>, Assessing congruence: empirical examples from molecular data In: <div class="person-group"><div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.J.</div></div></div>, (eds.), <div class="source">Molecular Systematics of Plants II</div> . 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">US</div>, pp. <div class="fpage">297</div>–<div class="lpage">348</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20Systematics%20of%20Plants%20II&amp;author=L.A.%20Johnson&amp;author=D.E.%20Soltis&amp;author=D.E.%20Soltis&amp;author=P.S.%20Soltis&amp;author=J.J.%20Doyle&amp;publication_year=1998&amp;book=Molecular%20Systematics%20of%20Plants%20II" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-4615-5419-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-4615-5419-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-4615-5419-6"> </span></div> </div></div></div></div></div><div content-id="dsx006-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Drummond</div>
               <div class="given-names">C.S.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Diversification of <em>Lupinus</em> (Leguminosae) in the western New World: derived evolution of perennial life history and colonization of montane habitats</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">48</div>, <div class="fpage">408</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20Lupinus%20(Leguminosae)%20in%20the%20western%20New%20World%3A%20derived%20evolution%20of%20perennial%20life%20history%20and%20colonization%20of%20montane%20habitats&amp;author=C.S.%20Drummond&amp;publication_year=2008&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=48&amp;pages=408-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.03.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.03.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.03.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18534869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cosner</div>
               <div class="given-names">M.E.</div>
            </div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Downie</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, 
<div class="article-title">The highly rearranged chloroplast genome of <em>Trachelium caeruleum</em> (Campanulaceae): multiple inversions, inverted repeat expansion and contraction, transposition, insertions/deletions, and several repeat families</div>. <div class="source">Curr. Genet</div> ., <div class="volume">31</div>, <div class="fpage">419</div>–<div class="lpage">429</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20highly%20rearranged%20chloroplast%20genome%20of%20Trachelium%20caeruleum%20(Campanulaceae)%3A%20multiple%20inversions%2C%20inverted%20repeat%20expansion%20and%20contraction%2C%20transposition%2C%20insertions%2Fdeletions%2C%20and%20several%20repeat%20families&amp;author=M.E.%20Cosner&amp;author=R.K.%20Jansen&amp;author=J.D.%20Palmer&amp;author=S.R.%20Downie&amp;publication_year=1997&amp;journal=Curr.%20Genet&amp;volume=31&amp;pages=419-429" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002940050225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002940050225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002940050225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9162114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Milligan</div>
               <div class="given-names">B.G.</div>
            </div>, <div class="name string-name"><div class="surname">Hampton</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">1989</div>, 
<div class="article-title">Dispersed repeats and structural reorganization in subclover chloroplast DNA</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">6</div>, <div class="fpage">355</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dispersed%20repeats%20and%20structural%20reorganization%20in%20subclover%20chloroplast%20DNA&amp;author=B.G.%20Milligan&amp;author=J.N.%20Hampton&amp;author=J.D.%20Palmer&amp;publication_year=1989&amp;journal=Mol.%20Biol.%20Evol&amp;volume=6&amp;pages=355-68" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2615639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ogihara</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Terachi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sasakuma</div> <div class="given-names">T.</div></div></div>
          <div class="year">1988</div>, 
<div class="article-title">Intramolecular recombination of chloroplast genome mediated by short direct-repeat sequences in wheat species</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">85</div>, <div class="fpage">8573</div>–<div class="lpage">8577</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intramolecular%20recombination%20of%20chloroplast%20genome%20mediated%20by%20short%20direct-repeat%20sequences%20in%20wheat%20species&amp;author=Y.%20Ogihara&amp;author=T.%20Terachi&amp;author=T.%20Sasakuma&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=85&amp;pages=8573-8577" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.22.8573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.22.8573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.22.8573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3186748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akkaya</div>
               <div class="given-names">M.S.</div>
            </div>, <div class="name string-name"><div class="surname">Bhagwat</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Cregan</div> <div class="given-names">P.B.</div></div></div>
          <div class="year">1992</div>, 
<div class="article-title">Length polymorphisms of simple sequence repeat DNA in soybean</div>. <div class="source">Genetics</div> , <div class="volume">132</div>, <div class="fpage">1131</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Length%20polymorphisms%20of%20simple%20sequence%20repeat%20DNA%20in%20soybean&amp;author=M.S.%20Akkaya&amp;author=A.A.%20Bhagwat&amp;author=P.B.%20Cregan&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=1131-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1459432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Desiderio</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Bitocchi</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Bellucci</div> <div class="given-names">E.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Chloroplast microsatellite diversity in <em>Phaseolus vulgaris</em></div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">3</div>.</div></div></div></div><div content-id="dsx006-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pan</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Development of 12 chloroplast microsatellite markers in <em>Vigna unguiculata</em> (Fabaceae) and amplification in <em>Phaseolus vulgaris</em></div>. <div class="source">Appl. Plant Sci</div> ., <div class="volume">2</div>, <div class="fpage">1300075</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%2012%20chloroplast%20microsatellite%20markers%20in%20Vigna%20unguiculata%20(Fabaceae)%20and%20amplification%20in%20Phaseolus%20vulgaris&amp;author=L.%20Pan&amp;author=Y.%20Li&amp;author=R.%20Guo&amp;author=H.%20Wu&amp;author=Z.%20Hu&amp;author=C.%20Chen&amp;publication_year=2014&amp;journal=Appl.%20Plant%20Sci&amp;volume=2&amp;pages=1300075" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/apps.1300075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fapps.1300075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fapps.1300075"> </span></div> </div></div></div></div></div><div content-id="dsx006-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Powell</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Andre</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">1995</div>, 
<div class="article-title">Hypervariable microsatellites provide a general source of polymorphic DNA markers for the chloroplast genome</div>. <div class="source">Curr. Biol</div> ., <div class="volume">5</div>, <div class="fpage">1023</div>–<div class="lpage">1029</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypervariable%20microsatellites%20provide%20a%20general%20source%20of%20polymorphic%20DNA%20markers%20for%20the%20chloroplast%20genome&amp;author=W.%20Powell&amp;author=M.%20Morgante&amp;author=C.%20Andre&amp;publication_year=1995&amp;journal=Curr.%20Biol&amp;volume=5&amp;pages=1023-1029" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(95)00206-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(95)00206-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(95)00206-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8542278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Provan</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Hollingsworth</div> <div class="given-names">P.M.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">Chloroplast microsatellites: new tools for studies in plant ecology and evolution</div>. <div class="source">Trends Ecol. Evol</div> ., <div class="volume">16</div>, <div class="fpage">142</div>–<div class="lpage">147</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chloroplast%20microsatellites%3A%20new%20tools%20for%20studies%20in%20plant%20ecology%20and%20evolution&amp;author=J.%20Provan&amp;author=W.%20Powell&amp;author=P.M.%20Hollingsworth&amp;publication_year=2001&amp;journal=Trends%20Ecol.%20Evol&amp;volume=16&amp;pages=142-147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(00)02097-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(00)02097-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(00)02097-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11179578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Provan</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Booth</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Powell</div> <div class="given-names">W.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Polymorphic chloroplast simple sequence repeat primers for systematic and population studies in the genus Hordeum</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">8</div>, <div class="fpage">505</div>–<div class="lpage">511</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphic%20chloroplast%20simple%20sequence%20repeat%20primers%20for%20systematic%20and%20population%20studies%20in%20the%20genus%20Hordeum&amp;author=J.%20Provan&amp;author=J.R.%20Russell&amp;author=A.%20Booth&amp;author=W.%20Powell&amp;publication_year=1999&amp;journal=Mol.%20Ecol&amp;volume=8&amp;pages=505-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294X.1999.00545.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294X.1999.00545.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294X.1999.00545.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10199011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saghai Maroof</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Biyashev</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">G.P.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Allard</div> <div class="given-names">R.W.</div></div></div>
          <div class="year">1994</div>, 
<div class="article-title">Extraordinarily polymorphic microsatellite DNA in barley: species diversity, chromosomal locations, and population dynamics</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">91</div>, <div class="fpage">5466</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraordinarily%20polymorphic%20microsatellite%20DNA%20in%20barley%3A%20species%20diversity%2C%20chromosomal%20locations%2C%20and%20population%20dynamics&amp;author=M.A.%20Saghai%20Maroof&amp;author=R.M.%20Biyashev&amp;author=G.P.%20Yang&amp;author=Q.%20Zhang&amp;author=R.W.%20Allard&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=91&amp;pages=5466-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.12.5466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.12.5466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.12.5466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8202509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">K.-S.</div>
            </div>, <div class="name string-name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div></div>
          <div class="year">1993</div>, 
<div class="article-title">Abundance, polymorphism and genetic mapping of microsatellites in rice</div>. <div class="source">Mol. Gen. Genet. MGG</div> , <div class="volume">241</div>, <div class="fpage">225</div>–<div class="lpage">235</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20polymorphism%20and%20genetic%20mapping%20of%20microsatellites%20in%20rice&amp;author=K.-S.%20Wu&amp;author=S.D.%20Tanksley&amp;publication_year=1993&amp;journal=Mol.%20Gen.%20Genet.%20MGG&amp;volume=241&amp;pages=225-235" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00280220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00280220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00280220"> </span></div> </div></div></div></div></div><div content-id="dsx006-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burki</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">The eukaryotic tree of life from a global phylogenomic perspective</div>. <div class="source">Cold Spring Harb. Perspect. Biol</div> ., <div class="volume">6</div>, <div class="fpage">a016147</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20eukaryotic%20tree%20of%20life%20from%20a%20global%20phylogenomic%20perspective&amp;author=F.%20Burki&amp;publication_year=2014&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol&amp;volume=6&amp;pages=a016147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/cshperspect.a016147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fcshperspect.a016147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fcshperspect.a016147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24789819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx006-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx006-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Leliaert</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Moreau</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Phylogeny and molecular evolution of the green algae</div>. <div class="source">Crit. Rev. Plant Sci</div> ., <div class="volume">31</div>, <div class="fpage">1</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20molecular%20evolution%20of%20the%20green%20algae&amp;author=F.%20Leliaert&amp;author=D.R.%20Smith&amp;author=H.%20Moreau&amp;publication_year=2012&amp;journal=Crit.%20Rev.%20Plant%20Sci&amp;volume=31&amp;pages=1-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07352689.2011.615705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352689.2011.615705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07352689.2011.615705"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3056574&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs1_dsx006.ppt?Expires=2147483647&amp;Signature=NUbYoeFLPQhq3ZfnoTeIZrc3s0teLmtrnCzvV6bqmW1q2PWoidgMR2Ayok94SKOfmo8ui8CJ2ot91PwlnKZlngB5y13bOK7HX5nT7GB0wmPXj6IWCr7d8z6SL7itC-glhwtgyw3t4kgDStVTmAov6WtRvykmKe6JGQXwOj4yQy8k-rnOKdgm1HPh~UcXu6LhqvDAw0ESHXoCQLE1lqaOO05J0RLHuYBVRIPAt0K12yg0HlY2t5mXzPH~5xMxAGkrcuvInrpkuWkdnlBB2q-CfpZSyG7y-I2Otb~OeVfIBIsRe7vz6b1rf7mVDjlRnBgKeau9zP76pG6oETOStt9b0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S1</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs2_dsx006.ppt?Expires=2147483647&amp;Signature=LpKXc8BnHt~NpaizeMErepyoREWjcGP6RW4by4HHjJPc0qoyuw9YHOeqanj5yknGpGr~mzCj9w1p4Drj8Zh~lbj6Y9nbF79YgZCwD3v~rA-PzFTQFGj~ZtMtew7jROFmFgUQB0VU8qsNn3SIlvXzZXg3bONejR3aGzTwmTzrolG~Wu0O79ssX1v89OnRsQ46JsEl-ha1KHQOSl8wqX~cW0TpInNER4MqbyRGyEj3EfI4og50lT~kr3BAw3CCEVHqLdCJhCkP3WzOCSdysAfB63vMOXy-O8Q-ln52qUw3iLMuYEt1VdRXo9LBvDJY3jFlr4gYJRI~Z9~8E4xcXIYs5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S2</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs3_dsx006.ppt?Expires=2147483647&amp;Signature=Cmg5llk1ncu6sCoGVUKsOZHT560FYYkTCdBBEtd5SCZuUUmZ1tom6SjOMJM-sTNLrgHwpOatrkgRxL0-cvRc846Uc0Stghk-xrAXMrgF2199AsVy0cnzI~FJ7PJQL7GkvZdIOQ20DgRztPi~wHC1jrz-dk8DnLFwilUydcZu0jYJb4JwqRLtSFSA~mBkyq0~9Ctuju0m4ymzizcHYmZ9bs0qxnSO1InT0M18Jbh10Z5pSqRfezKE99bVetn3YMy3HZV4ahe9e7N5Fb97-KhyCHHnhNWcu1sDOJ9HleaWfY~dVA4OtgJWn65QHHJ8Gr98EfAAw9h1opfNXwpH9rsV6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S3</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs4_dsx006.ppt?Expires=2147483647&amp;Signature=0vT1yYSotVvpRxoM5WLzUw8XZc6QYkVc12-u~wGw2vzwrokRjU~JxVw7ZKDd43KlsbFdBLJV4AW-N1NUH2tLeruzP9~IlVS~smZat5YIc~lK8o5ueQOB90uSx72F~KjnfQFY~vWmKX1lpcYT522Pko5YgZ4vWJozN6x8-vTYGXBuxqHl8QEJJi6LuJtU-~PMNh9JW-WKiOzeF15AQiIwPjxXnDQuquuLV4U1dSYv4lcByw8c3o8TP0TWi6NAbYFWKyBNa5qdCULIY-uZAvpWJIpLZ3kKUNumqTooChQRpcy~1ZAyFgoH~ZP-S5qDAaOkmD4IsvhRWyUMoVDMRvZYEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S4</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs5_dsx006.ppt?Expires=2147483647&amp;Signature=SDxDcGGSaK0Rmzs7qIWSZYHo8Et7R9jfHTcl9-apzZxfvHkgyP5N3AbboS-kLIkuKRqrGpfGfmW2uyYx9s82dgoBY-Q9p86DkcYvhN2s1Wl5JD097N7iQo7zeUT6nxgI-NgS8YBd8iTiyvl8Y48aF8tiqm-FBh9FVqHsaA5uONWNu~JM8g5uGaBRvBOuKNK35Vc0mP4mSkIPvY0we5ve6CM3sHdI9uR5rfn3pVKxbrvkrMLG2BtQHyMfzpqfEe5QOdXIgjv4ec0NUR1XLBqzOeqq0CUcky7liLNGEABf5icBjOQJ8nE-e7sL0Cd1~zwl7INLq4pyHuJwPskHbs-moA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S5</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs6_dsx006.ppt?Expires=2147483647&amp;Signature=Il96bTJUYPlXDT2qUzeroyXp5TCWyse-8Dt57lSAIXPl7UZ7qhlOX8jSqYFQ6ZVD0yGvRC8hHq24yYR1HV9sC78wCR7sz~EwliXHBg3bjn-rgZ90VVurhcyfTeIbYV2aaIvQVgy41zqZpTdStRo4Uq2x2C2D3x07MUfEH2Qksij8GD0aMin4CFWNoaaQ7lesQMtCkF8PsGnFcZtIGyDRU0~QNEuUf5P4IConHmMkhWOxOtQOc8ukNEYDjF4IpH1-FqmwiqphdY9OeLL5Ei8lbkEFwfGZi4GiksuxtUzEPlteVA4c8I28qMto401sSH2oJkElMBX1Cju5F0uvegFSMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S6</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/supfigs7_dsx006.ppt?Expires=2147483647&amp;Signature=mzo8ZN~MfW3vOBe9YmW2Td4yy61MJHW8ZI-hjnoy-FtlFEYeFS0oHb0Pxv71O8py0qCsmqopqa1NrS08sURnfLfe5bK1YI9TkFCOt0PWd8n1HRcTYbK7IRTxm-CAbAL4GftRN4UbAanq6-cJOknKK-8z0vPaUu4pEqqmoYOnU2Zs5KsHIuw48tiPpm3YHc72U9jtxBuWmrdjlXNXyFSX3Y3sRqc8LIFo19J7Vao2vsCAwrOhDFE~LU5U6lGH1K-pgZYep4-F8oVrzCvWxKI9SZ0UyQDZD0zvYqxP7SXuXpq4IBp5vwSfX2errj6hivU9MFLWTHwB7vaL~A12QATiaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Fig S7</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs1_dsx006.xls?Expires=2147483647&amp;Signature=sCMnLQBkGQ7LtF-novZzfiqUEbb-ONEANNCXRG9L9CM3lFkU7RJ9oP0NrXR5wlNtylK0zdoa5Aj5gHq4Nte53Rt7W~4Y9XoHf9sqig3tcmIdoFw4uGguD7nb4ZW16pd4oRlBM35Cb3Eb6OJdgPJD1LzkqPpduPNnd6IuHSk9DxxjNF3QMwuoZnXIHo5p8RjyIQ9t33Kv0UaNie0FBkF1M6nSiwriU17kv63CUBuIdlCsYAko2ZXuUvg1jqIqdD0wtMjXa6qcte8KsYStysMVWRNV7LmRW7wUtwmteFaSl4bXchHjmRF9QZZ~ya5YfnN-BmAxCE89cQSSR2ZQuD7Ebg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs2_dsx006.xls?Expires=2147483647&amp;Signature=sVlH6PVnYVU-tIBLs-yiAbUlcEMODKALjSE1QL6Vip8PeGAebwjUBvSMAnmwyDkBS6EdCLwEBYcc09YxAKf~A-iLilEaJ~h8ugD93D7QOTALAf4cuTe3ihqcA~uXCKwfdvuuJD-EmL5DKSLWhdzze-tINQkopMkXzoOuZFUG1XHgG8NNm-zOzLlsql2UrLqdiIb4u0RMTrdT4nOHAaEpeWgzQ9oAMnTHoo5N4N5lfC9VIpDhsqRW7AyfVWVzKGnaLnaXrwkcRhbRE8Ga7peWZw67xU2JA4hQod~UC1G1TWoS8c66PcuGRkj~kwjhY5RFZtwcVEPZh6fu43yTvWreaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs3_dsx006.xls?Expires=2147483647&amp;Signature=jl0VKE7lzqsYW7~RPgm~9pdfFWZlJk1NIGLY7VO8LpoaGeF9WG7z-Bja19CNWuRtIxMbBG6igFx3bzptnPP8XkOz39HcrL9hun~UXAD63RSErqMgdB8RZBBBMy-0Ve3P2iFOCe6ZAz3JxF5gN9cVqJUfThmbDUxFL4YcmECR03EHXaPe44xoUVF3gPTpjtuV-8H3Qf2vCCkzd2T2jNC2rtFCRx1DPe2pCe6c3q9Cm8-ivG5JAbK76VGRl3CXJ0PDKHdPBD14HT3s0IK8kM-wJkgY9qZJoYuvOWRGG0LArO6C59Z8~upbg9Leh84O6f~zvPfnPVbWMNPpL1v1oFWioQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs4_dsx006.xls?Expires=2147483647&amp;Signature=EcnotUJeYrsGfe0p~NrpLcwyqp-xzvlM6auxjs~pxT27Nbei4T4KoBv0b9fMu3S8~YQlICXYx7wbwjvoRnxvxHc3Dxcs11JpohZXMZ3zC3c46-BRRWklOJXfcqlF7j2rHXn6VEvF2AYLdoJHteJJSb3uDwK8~hQ0~Lcvoc9P2D753znTShrPbEz45J8Sw~RrUdG5md1BnGVqS6Y3X8U-GNGpkXJviIuudxt3moClhITirGaNoc9DWCmt5RnWaXsOGeZ0847h92ASLIsZnBmysLngdadAnL0JKVng~9HJEi9-w34r7CE4AE~Y87YPm0ON3vPOGRo4l4JPAB4uPAakKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs5_dsx006.xls?Expires=2147483647&amp;Signature=u8zRXIJj3GgCOD7Y1P5FBeCGQgOlKLVSK9cG~rZi~BCSvg8-pIcLTsKIWEqKeMez7ZPqrQnpT-~AL9B9cWdZU1e1vUEsvv3AISfSSmxdZ0ZY51qNRpxCubInUjdaK9LIqKYF9kvY9~V9auqx5QqQl9dk7byX78e8MULav12RD40P1LFGcqoLB4GU2CTJexzac9cFfWbkrOpslOj3qBwRagE6djewJtzIrEiiNFyBWvpPvKUgi8k0T16Wt6msvkBNinwA5Bx5qGnxTaFXivtYdd0CxFJNqmNgplGNojg1Cy-UeJGHoSB75guVc83bBnUHBvf9bP5Z3Ii1SXTG3NZZKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs6_dsx006.xls?Expires=2147483647&amp;Signature=iAt2gOjVRj6I1PdOTyMQsi8W0iIdzPY105sKyW8UITsk7EQs-sA40HylOjm4ehzoerX-oPYepoboUQcrqXvHtWL8dPDy3P4pEU-Eb0aOyxUknJ79d-DU3KV7a5rtO21USO9MxFJ4t9uVzg2UGL8ZgypxTHadWEtWJpTE8muvb5jA35VUbZUjc3ECAuPQmxEXSIHV8bnmrwP4luNRGbHpEiZR9YwKRLAtRyxnCtjfw3CLrjZ~PsNC1TSGIGPn7seHgdGRw933CkEZ3yi~dNdLbwtLqVswyjUb1cHMrhvkCCH9UmZuM3uTBXsAXEORf7Ac4rqlL2hLQE-O97b6KIibKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs7_dsx006.xls?Expires=2147483647&amp;Signature=Cr7YSxCvQGze5pLxm3QwN8WhQCdSPaqpVWn8NG1WsnrbMMkvvegsVbCOV4GA5rW7gcvljwWZjW0QptB6Lsmu2QlnlO99HE5GA-AgkjvfdLqGB9yIz3UnMI-adpfKglBXvQfU0iY13Zhd~T5DHh91rzSjiCObShvW8Ulukcx2jmJb7yOpDyYDvn8MKlHKnOona6Y-sfVnBnHH3VMjqbaILRqq1TF7fKZfOK6lnyv5UU9vAaejkYh2ppja9iT~-MZYt9KNvIEuybyA4wzfc2vlwmt5lmURLA7Vo23vlzAlBUU8N4xHVHhM0rMiqLzlJVBNzXElcIFTQVsSE3QIbYCEPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S7</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs8_dsx006.xls?Expires=2147483647&amp;Signature=AuC-cKZ5nra7AGvPbN0sdfTTVDSKlMqo0sfzrDd6MTHlJcpQscporn6AZ6kyXohwnLjTgrvyWD1EyM5EPs~~NfgOPYlDYXMuDSjmVustcGWBAFEqCgMRnYo-BrQXE-J4AOUgFE-JtasI3A98c1LA4ow28FE9xhFq9ioCHNBF3TfjebaJNU9LGeL~sroOX-88iAFE2N64e0ivzT1kfl5V2DZrFju2nEKRImxdIr85wZ6P6IMhVxKS9Q88qylHXIP7ZEEkRGMPVO2ktQYMnvPelOz~29p3EzHc5uSQAlcZmeSKcAhT1sC718d5qtpQyjHZ1kTnr-SgS7m-T~56vi~X9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs9_dsx006.xls?Expires=2147483647&amp;Signature=MWpsBn4cHNf-cR7pSTXl52PM1lwp-dNx6KmtLV7pxYjSXsJp9WGDasKX2TX1h~fNPjguncxvNwfxuLoyUBohpwJEkm6oPiuELT~h3HwI8D2a6oYhlXOnsHgwfLyG12fN5hDTa7I0sprFEoJ23zDBWyyefwKvhT3NKpYTuXWkT9d8E4F35Ejpx2ShQ-h9NRzNvHwWI0ZImYH35EAaP1yGxGskZZewP6fk6XvC3iUqTaKJ-d2do2fdocIoub3Jw50p5LmueCmi8u0QH079N08tzekJXxHcduFvt1Z0sp~IV5XuBozjmuoWpYOGVamSHTDqBppzBPvxrBTtukUSNHu1Qg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S9</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs10_dsx006.xls?Expires=2147483647&amp;Signature=GjAaLlO5bkwaKRiJLygPScRqJVc5A51vkKy0TKue4uhP~uLyik22f6qn6056T9BaOqg7Ni5WuuTAgcrbkNYuQ4xXnArw97H4hR570A2W0CPGpnp8oYpmLCPncvviZpTRVcedz8HHt-FHHbtgYYyxY-64eTUCp4VtLJVNgsH4CfOJ5XANtETxSm6Qaq7QCVjxE8OKIt9cIqmLKFj1I-yeNKDBz0oSN5xSfuo0xKeRv~Rs7-QHKVqJxpl5Ym8I6QMPid7Oz8KsA7ztD68aZ85AEXSlDdckdNFxqETMoqhkDeL7OlgPq13nuX8R82ztzC-jYLv6z~qUrLl5HXi0Ht5TQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S10</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs11_dsx006.xls?Expires=2147483647&amp;Signature=zMhSEdJESnUlGUZETF7zF2bEgqC5MHCrA1JXSPN1DFED-Z4DYj1VfhHeQS1iAvN5af2Gmq510v0pIKiycGhtW4mWUHBs~1Yt9MP9g1FCanSG9KVLlCAvikCev2AfBDf4lyb~V~QbaMcOLep5fjMjpXG1yqT-B2SkbyCNl0KvwLpbILCDkwtxe4RQ~SnH0Y-SHFKzjr9Ae2X1RV3W-NorRgGfafWn8Bn8EpojNQfdqR1hzkET7PJVtrtJ696ej2X4iIpQN20F~Vo6FR0e6~5ro6BzyvLWXM14VfsSAuZRDI1iYAr~4xMrdq8z4efYxHFYhf7G8tAvwl7tT7JWpusyWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S11</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs12_dsx006.xls?Expires=2147483647&amp;Signature=3hZH0tTs8L0HXAEL0VOF5dqbkpxbWcVubvFhHJ0kz-~8CAF7E-TvufrjjKnIKUAtHR7jYSKuSiA0Uu~UXa0gKyLgtXrjE-He4k7XZZZmcP7QE1nlON-xgZ2cjuorHTnJGH0Zb72ZtMiDoyQMM8KH6W5YHPTDiknaUHiVHuvkyQSSPSPh0rCcka2vzVOtDMHKq33hjRIal6GDx1hRLSYEpCWDWR~lvQ5N1oqOhSWn4T6thxkw~WAbEVdC17QIvrttDWkls77V-dwb7ZevIOxm574OIipQ8pggiv5nqmvHZHPAGNCW~znkrJ8U3Bp03Eno3O8UOdJNJh~MOMM68d2~Bw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S12</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx006/2/suptabs13_dsx006.xls?Expires=2147483647&amp;Signature=mJeDIiobjofGUmlPDYXpwA81neWyRPdt56mwhHrUXz9N1B5rwvfUWyOw3LxG-bqk0TMNjXWN2g9hOLd6UY16a-3gQb6sLwa3J-RFjnTH64J3VKDtcCx5-yzFAfCTkbh72gFWSIe4hqKe3OVGx9LIXgQWj7veHKLrzbdYsdxyK2C2Knyyo~nyHgVKSHTHGzPB8LP3uzwClsdOz1budEf33pENlCyV3RkZkvUe6YJ6Lts5TRbeH42bURng1HM-EJ6r~UPbd9ZYl322BqsFxZ7IS5bIqj9cfGuLswewE753bKtRBE-HyzbimS-Vk2BQIpEolldol0a74Xxzp57yUwpb1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S13</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">917</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx006"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3056574"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f4%2f343%2f3056574">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000407965100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F4%2F343%2F3056574" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3056574'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="evolutionary fate of the chloroplast and nuclear rps16 genes as revealed through the sequencing and comparative analyses of four novel legume chloroplast genomes from Lupinus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx006");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f343%2f3056574">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

